{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "source": [
        "## Installing Libraries\n"
      ],
      "metadata": {
        "id": "IZFw3vEhT7f3"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "U5kPA9afT6Pl"
      },
      "outputs": [],
      "source": [
        "!pip install llama-index llama-parse python-dotenv"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from llama_parse import LlamaParse\n",
        "from llama_index.core import VectorStoreIndex, SimpleDirectoryReader\n",
        "import os\n",
        "import nest_asyncio\n",
        "from dotenv import load_dotenv\n",
        "from IPython.display import Markdown, display\n",
        "from google.colab import userdata\n",
        "\n",
        "\n",
        "nest_asyncio.apply()"
      ],
      "metadata": {
        "id": "O7lxQXoOT7D0"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "os.environ[\"llamaparse_api\"] = userdata.get(\"LLAMA_CLOUD_API_KEY\")\n",
        "os.environ[\"OPENAI_API_KEY\"] = userdata.get(\"OPENAI_APIKEY\")"
      ],
      "metadata": {
        "id": "D4XSeCiPUc7b"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Setting up LlamaParser"
      ],
      "metadata": {
        "id": "EX8ivFTbUsT6"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "parser = LlamaParse(\n",
        "    api_key=llamaparse_api,\n",
        "    result_type=\"markdown\"  # \"markdown\" and \"text\" are available\n",
        ")"
      ],
      "metadata": {
        "id": "JFqyqNF3UiGA"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Using DirectoryReader with Llamaparse and without Llamaparse"
      ],
      "metadata": {
        "id": "sbmmQLSDUvJ9"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "file_extractor = {\".pdf\": parser}\n",
        "documents1 = SimpleDirectoryReader(input_files=['/content/data/cellbiology.pdf'], file_extractor=file_extractor).load_data()\n",
        "print(documents1)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "-mzhEfKkUjh6",
        "outputId": "f527cfc2-4bf6-45e8-8b5f-766bf73e2fb0"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Started parsing the file under job_id 16d9ef9e-7e79-4210-bc9d-d153c8e3fa60\n",
            "...............[Document(id_='b5fd9c1b-75ca-4294-82a0-af86c07b4e81', embedding=None, metadata={'file_path': '/content/data/cellbiology.pdf', 'file_name': 'cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='CELL BIOLOGY\\n         SECOND EDITION\\n---\\n# CELL BIOLOGY\\n\\nA Short Course\\n\\nSECOND EDITION\\n\\nStephen R. Bolsover\\n\\nDepartment of Physiology\\n\\nUniversity College London\\n\\nJeremy S. Hyams\\n\\nDepartment of Biology\\n\\nUniversity College London\\n\\nElizabeth A. Shephard\\n\\nDepartment of Biochemistry and Molecular Biology\\n\\nUniversity College London\\n\\nHugh A. White\\n\\nDepartment of Biochemistry and Molecular Biology\\n\\nUniversity College London\\n\\nClaudia G. Wiedemann\\n\\nDepartment of Physiology\\n\\nUniversity College London\\n\\nA JOHN WILEY & SONS, INC., PUBLICATION\\n---\\nCopyright ©C 2004 by John Wiley & Sons, Inc. All rights reserved.\\n\\nPublished by John Wiley & Sons, Inc., Hoboken, New Jersey.\\n\\nPublished simultaneously in Canada.\\n\\nNo part of this publication may be reproduced, stored in a retrieval system, or transmitted in any form or by any\\nmeans, electronic, mechanical, photocopying, recording, scanning, or otherwise, except as permitted under\\nSection 107 or 108 of the 1976 United States Copyright Act, without either the prior written permission of the\\nPublisher, or authorization through payment of the appropriate per-copy fee to the Copyright Clearance Center,\\nInc., 222 Rosewood Drive, Danvers, MA 01923, 978-750-8400, fax 978-646-8600, or on the web at\\nwww.copyright.com. Requests to the Publisher for permission should be addressed to the Permissions\\nDepartment, John Wiley & Sons, Inc., 111 River Street, Hoboken, NJ 07030, (201) 748-6011,\\nfax (201) 748-6008.\\n\\nLimit of Liability/Disclaimer of Warranty: While the publisher and author have used their best efforts in\\npreparing this book, they make no representations or warranties with respect to the accuracy or completeness of\\nthe contents of this book and specifically disclaim any implied warranties of merchantability or fitness for a\\nparticular purpose. No warranty may be created or extended by sales representatives or written sales materials.\\nThe advice and strategies contained herein may not be suitable for your situation. You should consult with a\\nprofessional where appropriate. Neither the publisher nor author shall be liable for any loss of profit or any other\\ncommercial damages, including but not limited to special, incidental, consequential, or other damages.\\n\\nFor general information on our other products and services please contact our Customer Care Department within\\nthe U.S. at 877-762-2974, outside the U.S. at 317-572-3993 or fax 317-572-4002.\\n\\nWiley also publishes its books in a variety of electronic formats. Some content that appears in print, however,\\nmay not be available in electronic format.\\n\\nLibrary of Congress Cataloging-in-Publication Data:\\n\\n|Cell biology : a short course / Stephen R. Bolsover . . . [et al.].—2nd ed.|p. cm.|\\n|---|---|\\n|Includes bibliographical references and index.| |\\n|ISBN 0-471-26393-1 (Paper)| |\\n|1. Cytology. I. Bolsover, Stephen R., 1954–| |\\n|QH581.2.C425 2003|571.6—dc21 2003000577|\\n\\nPrinted in the United States of America\\n\\n10 9 8 7 6 5 4 3 2 1\\n---\\n# CONTENTS IN BRIEF\\n\\n|1|CELLS AND TISSUES|1|\\n|---|---|---|\\n|2|FROM WATER TO DNA: THE CHEMISTRY OF LIFE|19|\\n|3|MEMBRANES AND ORGANELLES|51|\\n|4|DNA STRUCTURE AND THE GENETIC CODE|65|\\n|5|DNA AS A DATA STORAGE MEDIUM|87|\\n|6|TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION|105|\\n|7|RECOMBINANT DNA AND GENETIC ENGINEERING|129|\\n|8|MANUFACTURING PROTEIN|163|\\n|9|PROTEIN STRUCTURE|183|\\n|10|INTRACELLULAR PROTEIN TRAFFICKING|213|\\n|11|HOW PROTEINS WORK|237|\\n|12|ENERGY TRADING WITHIN THE CELL|257|\\n|13|METABOLISM|281|\\n|14|IONS AND VOLTAGES|309|\\n|15|THE ACTION POTENTIAL|325|\\n|16|INTRACELLULAR SIGNALING|341|\\n|17|INTERCELLULAR COMMUNICATION|363|\\n|18|MECHANICAL MOLECULES|381|\\n|19|CELL CYCLE AND CONTROL OF CELL NUMBER|401|\\n|20|CASE STUDY: CYSTIC FIBROSIS|423|\\n---\\n# CONTENTS\\n\\n# PREFACE\\n\\nxv Bases Are Molecules That Take H          + from Water, 25\\n\\nIsoelectric Point, 25\\n\\n# ACKNOWLEDGMENTS\\n\\nxvii\\n\\n# INSTRUCTOR NOTES\\n\\nxix A Hydrogen Bond Forms When a Hydrogen Atom Is Shared, 25\\n\\nBiological Macromolecules, 27\\n\\n# 1   CELLS AND TISSUES\\n\\nPrinciples of Microscopy, 2\\n\\nThe Light Microscope, 3\\n\\nThe Electron Microscope, 8\\n\\nThe Scanning Electron Microscope, 9\\n\\nOnly Two Types of Cell, 9\\n\\nSpecial Properties of Plant Cells, 11\\n\\nViruses, 11\\n\\nOrigin of Eukaryotic Cells, 12\\n\\nCell Specialization, 12\\n\\nEpithelia, 12\\n\\nConnective Tissue, 13\\n\\nNervous Tissue, 13\\n\\nMuscle, 14\\n\\nPlants, 15\\n\\nSummary, 16\\n\\nReview Questions, 16\\n\\nAnswers to Review Questions, 17\\n\\n# 2   FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\nThe Chemical Bond: Sharing Electrons, 19\\n\\nInteractions with Water: Solutions, 21\\n\\nIonic Compounds Will Dissolve Only in Polar Solvents, 21\\n\\nAcids Are Molecules That Give H        +  to Water, 21\\n\\n# 3   MEMBRANES AND ORGANELLES\\n\\nBasic Properties of Cell Membranes, 51\\n\\nStraight Through the Membrane: Diffusion Through the Bilayer, 53\\n\\nBeyond the Cell Membrane: The Extracellular Matrix, 53\\n\\nCell Junctions, 54\\n\\nOrganelles Bounded by Double-Membrane Envelopes, 56\\n\\nThe Nucleus, 56\\n\\nMitochondria and Chloroplasts, 58\\n---\\n# CONTENTS\\n\\n|Organelles Bounded by Single-Membrane Envelopes|58|\\n|---|---|\\n|DNA Replication|87|\\n|Peroxisomes|59|\\n|Endoplasmic Reticulum|60|\\n|Golgi Apparatus|60|\\n|Lysosomes|61|\\n|Summary|61|\\n|Review Questions|62|\\n|Answers to Review Questions|63|\\n|4 DNA STRUCTURE AND THE GENETIC CODE|65|\\n|Introduction|65|\\n|The Structure of DNA|65|\\n|The DNA Molecule Is a Double Helix|68|\\n|The Two DNA Chains Are Complementary|69|\\n|Different Forms of DNA|71|\\n|DNA as the Genetic Material|71|\\n|Packaging of DNA Molecules into Chromosomes|71|\\n|Eukaryotic Chromosomes and Chromatin Structure|71|\\n|Prokaryotic Chromosomes|73|\\n|Plasmids|74|\\n|Viruses|74|\\n|The Genetic Code|75|\\n|Amino Acid Names Are Abbreviated|79|\\n|The Code Is Degenerate But Unambiguous|79|\\n|Start and Stop Codons and the Reading Frame|79|\\n|The Code Is Nearly Universal|80|\\n|Missense Mutations|80|\\n|Summary|81|\\n|Further Reading|84|\\n|Review Questions|84|\\n|Answers to Review Questions|85|\\n|5 DNA AS A DATA STORAGE MEDIUM|87|\\n|Introduction|87|\\n\\n# 6 TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\n\\n|Structure of RNA|105|\\n|---|---|\\n|RNA Polymerase|106|\\n|Gene Notation|106|\\n|Bacterial RNA Synthesis|106|\\n|Control of Bacterial Gene Expression|109|\\n|lac, an Inducible Operon|111|\\n|trp, a Repressible Operon|116|\\n|Eukaryotic RNA Synthesis|118|\\n|Messenger RNA Processing|118|\\n|Control of Eukaryotic Gene Expression|119|\\n---\\n# CONTENTS\\n\\n|Glucocorticoids Cross the Cell Membrane to Activate Transcription, 121|Ribosome-Binding Site, 168|\\n|---|---|\\n|Summary, 125| |\\n|Further Reading, 125| |\\n|Review Questions, 126| |\\n|Answers to Review Questions, 127| |\\n\\n# 7  RECOMBINANT DNA AND GENETIC ENGINEERING, 129\\n\\n- DNA Cloning, 129\\n- Creating the Clone, 130\\n- Introduction of Foreign DNA Molecules into Bacteria, 130\\n- Selection of cDNA Clones, 134\\n- Genomic DNA Clones, 139\\n\\nUses of DNA Clones, 143\\n\\n- DNA Sequencing, 143\\n- Southern Blotting, 146\\n- In situ Hybridization, 147\\n- Northern Blotting, 148\\n- Production of Mammalian Proteins in Bacteria, 149\\n- Protein Engineering, 149\\n- Polymerase Chain Reaction, 150\\n- Identifying the Gene Responsible for a Disease, 152\\n- Reverse Genetics, 152\\n- Transgenic Animals, 157\\n\\nEthics of DNA Testing for Inherited Disease, 157\\n\\n|Summary, 158| |\\n|---|---|\\n|Further Reading, 159| |\\n|Review Questions, 159| |\\n|Answers to Review Questions, 160| |\\n\\n# 8  MANUFACTURING PROTEIN, 163\\n\\n- Attachment of an Amino Acid to Its tRNA, 163\\n- Transfer RNA, the Anticodon, and the Wobble, 164\\n- The Ribosome, 165\\n- Bacterial Protein Synthesis, 168\\n---\\n# CONTENTS\\n\\n|10|INTRACELLULAR PROTEIN TRAFFICKING, 213|\\n|---|---|\\n| |Three Modes of Intracellular Protein Transport, 213|\\n| |Targeting Sequences, 215|\\n| |Retention, 215|\\n| |Transport to and from the Nucleus, 215|\\n| |The Nuclear Pore Complex, 216|\\n| |Gated Transport Through the Nuclear Pore, 216|\\n| |GTPases and the GDP/GTP Cycle, 218|\\n| |GTPases in Nuclear Transport, 218|\\n| |Transport Across Membranes, 221|\\n| |Transport to Mitochondria, 221|\\n| |Chaperones and Protein Folding, 221|\\n| |Transport to Peroxisomes, 221|\\n| |Synthesis on the Rough Endoplasmic Reticulum, 223|\\n| |Glycosylation: The Endoplasmic Reticulum and Golgi System, 225|\\n| |Vesicular Trafficking Between Intracellular Compartments, 226|\\n| |The Principle of Fission and Fusion, 226|\\n| |Vesicle Formation, 228|\\n| |Coatomer-Coated Vesicles, 228|\\n| |Clathrin-Coated Vesicles, 229|\\n| |The Trans-Golgi Network and Protein Secretion, 229|\\n| |Targeting Proteins to the Lysosome, 230|\\n| |Fusion, 231|\\n| |Summary, 232|\\n| |Further Reading, 233|\\n| |Review Questions, 233|\\n| |Answers to Review Questions, 234|\\n|11|HOW PROTEINS WORK, 237|\\n| |How Proteins Bind Other Molecules, 237|\\n| |Dynamic Protein Structures, 238|\\n| |Allosteric Effects, 238|\\n| |Chemical Changes That Shift the Preferred Shape of a Protein, 240|\\n| |Enzymes Are Protein Catalysts, 241|\\n\\n# 12 ENERGY TRADING WITHIN THE CELL, 257\\n\\nCellular Energy Currencies, 258\\n\\nReduced Nicotinamide Adenine Dinucleotide (NADH), 259\\n\\nNucleoside Triphosphates (ATP plus GTP, CTP, TTP, and UTP), 259\\n\\nThe Hydrogen Ion Gradient Across the Mitochondrial Membrane, 261\\n\\nThe Sodium Gradient Across the Plasma Membrane, 262\\n\\nEnergy Currencies Are Interconvertible, 263\\n\\nExchange Mechanisms Convert Between the Four Energy Currencies, 263\\n\\nElectron Transport Chain, 265\\n\\nATP Synthase, 269\\n\\nSodium/Potassium ATPase, 270\\n\\nADP/ATP Exchanger, 271\\n\\nPhotosynthesis, 271\\n\\nAll Carriers Can Change Direction, 275\\n\\nSummary, 278\\n\\nFurther Reading, 278\\n\\nReview Questions, 278\\n\\nAnswers to Review Questions, 279\\n\\n# 13 METABOLISM, 281\\n\\nThe Krebs Cycle: The Central Switching Yard of Metabolism, 283\\n---\\n# CONTENTS\\n\\nFrom Glucose to Pyruvate: Glycolysis, 284\\n\\nReview Questions, 322\\n\\n- Glycolysis Without Oxygen, 286\\n- Glycogen Can Provide Glucose for Glycolysis, 288\\n- Glucose May Be Oxidized to Produce Pentose Sugars, 289\\n\\nFrom Fats to Acetyl-CoA: β Oxidation, 290\\n\\nAmino Acids as Another Source of Metabolic Energy, 292\\n\\nMaking Glucose: Gluconeogenesis, 295\\n\\nMaking Glycogen: Glycogenesis, 298\\n\\nMaking Fatty Acids and Glycerides, 300\\n\\nSynthesis of Amino Acids, 300\\n\\nCarbon Fixation in Plants, 302\\n\\nControl of Energy Production, 303\\n\\n- Feedback and Feedforward, 303\\n- Negative Feedback Control of Glycolysis, 304\\n- Feedforward Control in Muscle Cells, 304\\n\\nSummary, 306\\n\\nFurther Reading, 306\\n\\nReview Questions, 307\\n\\nAnswers to Review Questions, 308\\n\\nTHE ACTION POTENTIAL, 325\\n\\n- The Calcium Action Potential in Sea Urchin Eggs, 325\\n- Effect of Egg Transmembrane Voltage on Sperm Fusion, 325\\n- The Voltage-Gated Calcium Channel, 327\\n- The Calcium Action Potential, 328\\n- The Voltage-Gated Sodium Channel in Nerve Cells, 330\\n- The Voltage-Gated Sodium Channel, 330\\n- Electrical Transmission down a Nerve Cell Axon, 332\\n- Myelination and Rapid Action Potential Transmission, 334\\n\\nSummary, 337\\n\\nFurther Reading, 338\\n\\nReview Questions, 338\\n\\nAnswers to Review Questions, 339\\n\\nINTRACELLULAR SIGNALING, 341\\n\\n- Calcium, 341\\n- Calcium Can Enter from the Extracellular Medium, 341\\n- Calcium Can Be Released from the Endoplasmic Reticulum, 344\\n- Processes Activated by Cytosolic Calcium Are Extremely Diverse, 348\\n- Return of Calcium to Resting Levels, 350\\n- Cyclic Adenosine Monophosphate, 350\\n- Cyclic Guanosine Monophosphate, 353\\n- Multiple Messengers, 353\\n- Biochemical Signaling, 353\\n- Receptor Tyrosine Kinases and the MAP Kinase Cascade, 353\\n- Growth Factors Can Trigger a Calcium Signal, 356\\n- Protein Kinase B and the Glucose Transporter: How Insulin Works, 356\\n---\\n# CONTENTS\\n\\n17    INTERCELLULAR COMMUNICATION, 363\\nClassifying Transmitters and Receptors, 363\\nIonotropic Cell Surface Receptors, 364\\nMetabotropic Cell Surface Receptors, 365\\nIntracellular Receptors, 365\\nIntercellular Communication in Action: The Gastrocnemius Muscle, 365\\nTelling pe Muscle to Contract: The Action of Motoneurones, 367\\nControlling pe Blood Supply: Paracrine Transmitters, 368\\nNew Blood Vessels in Growing Muscle, 371\\nSynapses Between Neurons, 372\\nSummary, 376\\nFurper Reading, 377\\nReview Questions, 377\\nAnswers to Review Questions, 378\\n\\n18    MECHANICAL MOLECULES, 381\\nThe Cytoskeleton is Bop Strong and Motile, 381\\nMicrotubules, 381\\nMicrotubule-Based Motility, 386\\nCilia and Flagella, 386\\nIntracellular Transport, 389\\nMicrofilaments, 390\\nMuscle Contraction, 393\\nCell Locomotion, 395\\nCytoplasmic Streaming, 395\\nIntermediate Filaments, 396\\nAnchoring Cell Junctions, 396\\nSummary, 398\\n\\n19     CELL CYCLE AND CONTROL OF CELL NUMBER, 401\\nStages of Mitosis, 402\\nMeiosis and Fertilization, 404\\nMeiosis, 405\\nFertilization and Inheritance, 406\\nDominant Genetic Disease, 408\\nCrossing Over and Linkage, 408\\nControl of pe Cell Division Cycle, 408\\nMolecular Regulation of pe G2/M (Interphase/Mitosis) Cell Cycle Control Point, 410\\nWhat About pe G1/S Control Point?, 412\\nApoptosis, 415\\nInstructed Deap: Deap Domain Receptors, 416\\nDefault Deap: Absence of Growp Factors, 416\\nThe Sick Are Left to Die: Stress-Activated Apoptosis, 417\\nSummary, 419\\nFurper Reading, 420\\nReview Questions, 420\\nAnswers to Review Questions, 421\\n\\n20     CASE STUDY: CYSTIC FIBROSIS, 423\\nIntroduction, 423\\nCystic Fibrosis is a Severe Genetic Disease, 423\\nThe Fundamental Lesion in Cystic Fibrosis Lies in Chloride Transport, 424\\nHoming in on pe CF Gene, 425\\nCloning pe Gene for CF, 426\\nThe CFTR Gene Codes for a Chloride Ion Channel, 426\\nGene Therapy for CF, 427\\nDiagnostic Tests for CF, 431\\n---\\n# CONTENTS\\n\\nThe Future, 432\\n\\nSummary, 433\\n\\nFurther Reading, 433\\n\\nReview Questions, 434\\n\\nAnswers to Review Questions, 435\\n\\n# APPENDIX: CHANNELS AND CARRIERS\\n\\nGLOSSARY, 441\\n\\nINDEX, 501\\n---\\n# PREFACE\\n\\nCell Biology, A Short Course aims to cover a wide area of cell biology in a form especially suitable for first year undergraduates. We have deliberately kept the book to a manageable size so that neither the cost, the content, nor the weight is too daunting for the student.\\n\\nThe overall theme for the book is the cell as the unit of life. We begin (Chapters 1–3) by describing the components of the cell as seen under the microscope. We then (Chapters 4–8) turn to the central dogma of molecular biology and describe how DNA is used to make RNA which in turn is used to make protein. The next section (Chapters 9–11) describes how proteins are delivered to the appropriate location inside or outside the cell, and how proteins perform their many functions. We then (Chapters 12–14) turn to cell energetics and metabolism. Signaling within and between cells is covered in Chapters 15 through 17. To conclude the book, Chapter 18 describes the composition and function of the cytoskeleton, Chapter 19 covers cell birth and cell death, while Chapter 20 uses the example of the common and severe genetic disease cystic fibrosis to illustrate many of the themes discussed earlier in the book.\\n\\nBoxed material throughout the book is divided into examples to illustrate the topics covered in the main text, explanations of the medical relevance of the material, and in-depth sections that extend the coverage beyond the content of the main text. Questions are provided at the end of each chapter to help the reader assess how well they have assimilated and understood the material.\\n\\nAs well as giving references to printed material, we reference material available on the internet in many places in the book. Rather than give detailed addresses, we provide links to all these sites and many others from the book’s homepage at http://www.physiol.ucl.ac.uk/sbolsover/teaching/cbasc/cbasc.html.\\n---\\n# ACKNOWLEDGMENTS\\n\\nWe thank all the students, colleagues and family members who read the initial versions of the book and whose suggestions and constructive criticism helped enormously.\\n\\nxvii\\n---\\n# INSTRUCTOR NOTES\\n\\nMolecular cell biology courses now form a foundation for many subsequent specializations in areas outside cell biology. We therefore cover molecular genetics, metabolic pathways, and electrophysiology in sufficient detail to make Cell Biology a suitable course book for first-year students who will later specialize in genetics, biochemistry, pharmacology, or physiology.\\n\\nEach chapter comprises:\\n\\n- The main text, with figures and tables.\\n- A numbered summary.\\n- Review questions with answers for student self-assessment. These questions concern the main text only; no knowledge of the boxed material is required.\\n- Example boxes that illustrate the points made in the main text.\\n- Medical relevance boxes to show how basic cell biological knowledge illuminates medical problems or has provided solutions.\\n- In-Depth boxes that extend the content.\\n\\nSelf-assessment questions can form the basics for tutorials, with students asked to defend the correct answer. They are also easily modified to generate new questions for student assessment. Instructors are encouraged to submit new questions for inclusion on the CBASC website.\\n\\nInstructors may wish to specify parts of Cell Biology as core material for courses targeted to particular specialties. The parts chosen can be customized to the particular specialty in two ways:\\n\\n1. By selecting from the complete set of twenty chapters. The following sections could be used to support particular teaching modules:\\n\\n|Chapters 4 through 7|DNA, RNA and genetic engineering.|\\n|---|---|\\n|Chapters 8 through 10|Protein synthesis, structure and trafficking.|\\n|Chapters 11 through 13|Metabolism and cellular energetics.|\\n|Chapters 14 through 17|Electrophysiology and cell signaling.|\\n|Chapter 18|The cytoskeleton and cell motility.|\\n|Chapter 19|Cell division and apoptosis.|\\n\\nChapters 16, 17, and 19 might, in contrast, be selected in a module concentrating on the control of development, since these describe how growth factors and other extracellular chemicals regulate cell division and cell death.\\n---\\n# INSTRUCTOR NOTES\\n\\n|Fluorescence Microscopy|How We Study Proteins in One Dimension describes SDS-PAGE|\\n|---|---|\\n|Chirality and Amino Acids|Hydropathy Plotting—The PDGF Receptor|\\n|Curing Mad Mice with Smelly Fish introduces the concept of osmolarity and osmosis and extends the coverage of chaotropic and structure stabilizing agents| |\\n|What to Measure in an Enzyme Assay|Determination of V m and K M the Lineweaver-Burk plot|\\n|ATP Synthase, Rotary Motor, and Synthetic Machine| |\\n|Can It Happen? The Concept of Free Energy| |\\n|The Urea Cycle—The First Metabolic Cycle Discovered|The Glyoxylate Shunt|\\n|The Nernst Equation|Measuring the Transmembrane Voltage|\\n|Frequency Coding in the Nervous System| |\\n|Ryanodine Receptors| |\\n|A Worm’s Eye View of Cell Death| |\\n|Lipid Bilayer Voltage Clamp| |\\n\\nFor example, a course emphasizing protein structure would include In Depth 8.1, 9.1, 9.2 and 9.3, while a course concentrating on metabolic pathways would include In Depth 13.1 and 13.2.\\n\\nThe CBASC website is maintained by the authors. As well as providing over one hundred links to sites with information that extends or illustrates the material in the book, we will use the site to post typological or other errors, comments and test questions sent to us by readers. The full address is http://www.physiol.ucl.ac.uk/sbolsover/teaching/cbasc/cbasc.html or simply type ‘CBASC’ into a search engine.\\n---\\n# CELLS AND TISSUES\\n\\nThe cell is the basic unit of life. Microorganisms such as bacteria, yeast, and amoebae exist as single cells. By contrast, the adult human is made up of about 30 trillion cells (1 trillion = 1012) which are mostly organized into collectives called tissues. Cells are, with a few notable exceptions, small (Fig. 1.1) with lengths measured in micrometers (μm, where 1000 μm = 1 mm) and their discovery stemmed from the conviction of a small group of seventeenth-century microscope makers that a new and undiscovered world lay beyond the limits of the human eye. These pioneers set in motion a science and an industry that continues to the present day.\\n\\nThe first person to observe and record cells was Robert Hooke (1635–1703) who described the cella (open spaces) of plant tissues. But the colossus of this era of discovery was a Dutchman, Anton van Leeuwenhoek (1632–1723), a man with no university education but with unrivaled talents as both a microscope maker and as an observer and recorder of the microscopic living world. van Leeuwenhoek was a contemporary and friend of the Delft artist Johannes Vermeer (1632–1675) who pioneered the use of light and shade in art at the same time that van Leeuwenhoek was exploring the use of light to discover the microscopic world. Sadly, none of van Leeuwenhoek’s microscopes have survived to the present day. Despite van Leeuwenhoek’s Herculean efforts, it was to be another 150 years before, in 1838, the botanist Matthias Schleiden and the zoologist Theodor Schwann formally proposed that all living organisms are composed of cells. Their “cell theory,” which nowadays seems so obvious, was a milestone in the development of modern biology. Nevertheless general acceptance took many years, in large part because the plasma membrane, the membrane\\n\\n*Cell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C2004 by John Wiley & Sons, Inc.*\\n---\\n# CELLS AND TISSUES\\n\\nviruses are submicroscopic particles 50–100 nm\\n\\nprokaryotes are the smallest true cells. Most are 1–2 μm\\n\\nmost eukaryotic cells are in the range 5 μm–100 μm\\n\\n| |yeast 5 μm|human cell 20 μm|\\n|---|---|---|\\n|some exceptional cells can be seen with the naked eye|ciliated protists 0.25 mm| |\\n|frog egg 1 mm| | |\\n\\nFigure 1.1. Dimensions of some example cells. 1 mm = 10-3 m; 1 μm = 10-6 m; 1 nm = 10-9 m.\\n\\nsurrounding the cell that divides the living inside from the nonliving extracellular medium (Fig. 1.2) is too thin to be seen using a light microscope.\\n\\n# PRINCIPLES OF MICROSCOPY\\n\\nMicroscopes make small objects appear bigger. A light microscope will magnify an image up to 1500 times its original size. Electron microscopes can achieve magnifications up to 1 million times. However, bigger is only better when more details are revealed. The fineness of detail that a microscope can reveal is its resolving power. This is defined as the smallest distance that two objects can approach one another yet still be recognized as being separate. The resolution that a microscope achieves is mainly a function of the wavelength of the illumination source it employs. The smaller the wavelength, the smaller the object that will cause diffraction, and the better the resolving power. The light microscope, because it uses visible light of wavelength around 500 nanometers (nm, where 1000 nm = 1 μm), can distinguish objects as small as about half this: 250 nm. It can therefore be used to visualize the smallest cells and the major intracellular structures or organelles. The microscopic study of cell structure organization is known as cytology. An electron microscope is required to reveal the ultrastructure (the fine detail) of the organelles and other cytoplasmic structures (Fig. 1.2).\\n\\nThe wavelength of an electron beam is about 100,000 times less than that of white light. In theory, this should lead to a corresponding increase in resolution. In practice, the\\n---\\n# PRINCIPLES OF MICROSCOPY\\n\\n|heterochromatin|light microscope image|electron microscope image|\\n|---|---|---|\\n| |nucleolus|plasma membrane|\\n| |nucleus|vesicle|\\n| | |centrosome|\\n| | |microtubule|\\n| |cytoplasm|mitochondrion|\\n|Golgi apparatus| | |\\n|internal membranes| | |\\n|mitochondrial ribosome| |lysosome|\\n|peroxisome|cytoplasmic ribosome|nuclear pore|\\n|mitochondrion|rough endoplasmic reticulum|nuclear envelope|\\n\\nFigure 1.2. Cell structure as seen through the light and transmission electron microscopes.\\n\\nelectron microscope can distinguish structures about 1000 times smaller than is possible in the light microscope, that is, down to about 0.2 nm in size.\\n\\nThe Light Microscope\\n\\nA light microscope consists of a light source, which may be the sun or an artificial light, plus three glass lenses: a condenser lens to focus light on the specimen, an objective lens to form the magnified image, and a projector lens, usually called the eyepiece, to convey the magnified image to the eye. Depending on the focal length of the various lenses and their arrangement, a given magnification is achieved. In bright-field\\n---\\n# CELLS AND TISSUES\\n\\n|light source|electron source|electron source|\\n|---|---|---|\\n|condenser lens|condenser lens|lens|\\n|specimen|specimen|electrons|\\n|light|electrons|beam scanner|\\n|objective lens|objective lens|camera|\\n|projector|projector lens|lens|\\n|mirror|lens|image introduced for direct viewing|\\n|by eye|camera|viewed directly|\\n|viewing| |light|\\n| | |light|\\n| | |reflected electrons|\\n| |specimen| |\\n|light image|scanning electron microscope|image|\\n|processing computer system| | |\\n|display monitor|transmission electron microscope| |\\n| | |display monitor|\\n\\nFigure 1.3. Basic design of light and electron microscopes.\\n\\nFigure 1.4. Simple upright light microscope.\\n---\\n# PRINCIPLES OF MICROSCOPY\\n\\nBright field microscopy\\n\\nBright field microscopy\\nFormyl-met-leu-phe added\\n\\nPhase contrast microscopy\\n\\nPhase contrast microscopy\\nFormyl-met-leu-phe added\\n\\nFigure 1.5. Human blood cells viewed by bright-field and phase-contrast light microscopy. Arrow indicates a white blood cell. Formyl-met-leu-phe (page 171) causes the white blood cell to spread out and become very thin. It becomes almost invisible by bright-field microscopy but can still be detected by phase-contrast microscopy.\\n\\nIn microscopy, the image that reaches the eye consists of the colors of white light less that absorbed by the cell. Most living cells have little color (plant cells are an obvious exception) and are therefore largely transparent to transmitted light. This problem can be overcome by cytochemistry, the use of colored stains to selectively highlight particular structures and organelles. However, many of these compounds are highly toxic and to be effective they often require that the cell or tissue is first subjected to a series of harsh chemical treatments.\\n\\nA different approach, and one that can be applied to living cells, is the use of phase-contrast microscopy. This relies on the fact that light travels at different speeds through regions of the cell that differ in composition. The phase-contrast microscope converts these differences in refractive index into differences in contrast, and considerably more detail is revealed (Fig. 1.5). Light microscopes come in a number of physical orientations (upright, inverted, etc.) but whatever the orientation of the microscope the optical principles are the same.\\n---\\n# CELLS AND TISSUES\\n\\n# IN DEPTH 1.1 Fluorescence Microscopy\\n\\nFluorescent molecules emit light when they are illuminated with light of a shorter wavelength. Familiar examples are the hidden signature in bank passbooks, which is written in fluorescent ink that glows blue (wavelength about 450 nm) when illuminated with ultraviolet light (UV) (wavelength about 360 nm), and the whitener in fabric detergents that causes your white shirt to glow blue when illuminated by the ultraviolet light in a club. The fluorescent dye Hoechst 33342 has a similar wavelength dependence: It is excited by UV light and emits blue light. However, it differs from the dyes used in ink or detergent in that it binds tightly to the DNA in the nucleus and only fluoresces when so bound. Diagram a shows the optical path through a microscope set up so as to look at a preparation stained with Hoechst. White light from an arc lamp passes through an excitation filter that allows only UV light to pass. This light then strikes the heart of the fluorescent microscope: a special mirror called a dichroic mirror that reflects light of wavelengths shorter than a designed cutoff but transmits light of longer wavelength. To view Hoechst, we use a dichroic mirror of cutoff wavelength 400 nm, which therefore reflects the UV excitation light up through the objective lens and onto the specimen. Any Hoechst bound to DNA in the preparation will emit blue light. Some of this will be\\n\\n|(a)|specimen|\\n|---|---|\\n| |objective lens|\\n| |arc lamp|\\n|dichroic mirror|excitation filter|\\n|emission filter|projector lens|\\n|mirror|image viewed directly|\\n|introduced for direct viewing by eye|camera|\\n|image processing computer system| |\\n|display monitor| |\\n\\n(b)\\n---\\n# PRINCIPLES OF MICROSCOPY\\n\\nCaptured by the objective lens and, because its wavelength is greater than 400 nm, will not be reflected by the dichroic mirror but will instead pass through. An emission filter, set to pass only blue light, cuts out any scattered UV light. The blue light now passes to the eye or camera in the usual way. Image b shows a field of cells cultured from rat brain (gift of Dr. Charles Krieger, Simon Fraser University) after staining with Hoechst. Only the nuclei are seen, as bright ovals.\\n\\nAlthough some of the structures and chemicals found in cells can be selectively stained by specific fluorescent dyes, others are most conveniently revealed by using antibodies. In this technique an animal (usually a mouse, rabbit, or goat) is injected with a protein or other chemical of interest. The animal’s immune system recognizes the chemical as foreign and generates antibodies that bind to (and therefore help neutralize) the chemical. Some blood is then taken from the animal and the antibodies purified. The antibodies can then be labeled by attaching a fluorescent dye. Images c and d show the same field of brain cells but with the excitation filter, dichroic mirror, and emission filter changed so as to reveal in c a protein called ELAV that is found only in nerve cells; then in d an intermediate filament protein (page 000) found only in glial cells. The antibody that binds to ELAV is labeled with a fluorescent dye that is excited by blue light and emits green light. The antibody that binds to the glial filaments is labeled with a dye that is excited by green light and emits red light. Because these wavelength characteristics are different, the location of the three chemicals—DNA, ELAV, and intermediate filament—can be revealed independently in the same specimen. See the CBASC website for an image of all three signals in color and superimposed.\\n\\n(c)\\n(d)\\n\\nThe technique just described is called primary immunofluorescence and requires that the antibody to the chemical of interest be labeled with a dye. Only antibodies to chemicals that many laboratories study are so labeled. In order to reveal other chemicals, scientists use secondary immunofluorescence. In this approach, a commercial company injects an animal (e.g., a goat) with an antibody from another animal (e.g., a rabbit). The goat then makes “goat anti rabbit” antibody. This, called the secondary antibody, is purified and labeled with a dye. All the scientist has to do is make or buy a rabbit antibody that binds to the chemical of interest. No further modification of this specialized, primary antibody is necessary. Once the primary antibody has bound to the specimen and excess antibody rinsed off, the specimen is then exposed to the secondary antibody that binds selectively to the primary antibody. Viewing the stained preparation in a fluorescence microscope then reveals the location of the chemical of interest. The same dye-labeled secondary antibody can be used in other laboratories or at other times to reveal the location of many different chemicals because the specificity is determined by the unlabeled primary antibody.\\n---\\n# CELLS AND TISSUES\\n\\nThe Electron Microscope\\n\\nThe most commonly used type of electron microscope in biology is called the transmission electron microscope because electrons are transmitted through the specimen to the observer. The transmission electron microscope has essentially the same design as a light microscope, but the lenses, rather than being glass, are electromagnets that bend beams of electrons (Fig. 1.3b). An electron gun generates a beam of electrons by heating a thin, V-shaped piece of tungsten wire to 3000 ◦ C. A large voltage accelerates the beam down the microscope column, which is under vacuum because the electrons would be slowed and scattered if they collided with air molecules. The magnified image can be viewed on a fluorescent screen that emits light when struck by electrons. While the electron microscope offers great improvements in resolution, electron beams are potentially highly destructive, and biological material must be subjected to a complex processing schedule before it can be examined. The preparation of cells for electron microscopy is summarized in Figure 1.6.\\n\\nA small piece of tissue (~1 mm3) is immersed in glutaraldehyde and osmium tetroxide. These chemicals bind all pe component parts of pe cells togeper; pe tissue is said to be fixed. It is pen washed poroughly.\\nThe tissue is dehydrated by soaking in acetone or epanol.\\nThe tissue is embedded in resin which is pen baked hard.\\nSections (pin slices less pan 100 nm pick) are cut wip a machine called an ultramicrotome.\\nThe sections are placed on a small copper grid and stained wip uranyl acetate and lead citrate. When viewed in pe electron microscope, regions pat have bound lots of uranium and lead will appear dark because pey are a barrier to pe electron beam.\\n\\nFigure 1.6. Preparation of tissue for electron microscopy.\\n---\\nThe transmission electron microscope produces a detailed image but one that is static, two-dimensional, and highly processed. Often, only a small region of what was once a dynamic, living, three-dimensional cell is revealed. Moreover, the picture revealed is essentially a snapshot taken at the particular instant that the cell was killed. Clearly, such images must be interpreted with great care. Electron microscopes are large and require a skilled operator. Nevertheless, they are the main source of information on the structure of the cell at the nanometer scale, called the ultrastructure.\\n\\nThe Scanning Electron Microscope\\n\\nWhereas the image in a transmission electron microscope is formed by electrons transmitted through the specimen, in the scanning electron microscope it is formed from electrons that are reflected back from the surface of a specimen as the electron beam scans rapidly back and forth over it. These reflected electrons are processed to generate a picture on a display monitor. The scanning electron microscope operates over a wide magnification range, from 10× to 100,000×. Its greatest advantage, however, is a large depth of focus that gives a three-dimensional image. The scanning electron microscope is particularly useful for providing topographical information on the surfaces of cells or tissues. Modern instruments have a resolution of about 1 nm.\\n\\nSuch has been the importance of microscopy to developments in biology that two scientists have been awarded the Nobel prize for their contributions to microscopy. Frits Zernike was awarded the Nobel prize for physics in 1953 for the development of phase-contrast microscopy and Ernst Ruska the same award in 1986 for the invention of the transmission electron microscope. Ruska’s prize marks one of the longest gaps between a discovery (in the 1930s in the research labs of the Siemens Corporation in Berlin) and the award of a Nobel prize. Anton van Leeuwenhoek died almost two centuries before the Nobel prizes were introduced in 1901 and the prize is not awarded posthumously.\\n\\nSuperficially at least, cells exhibit a staggering diversity. Some lead a solitary existence; others live in communities; some have defined, geometric shapes; others have flexible boundaries; some swim, some crawl, and some are sedentary; many are green (some are even red, blue, or purple); others have no obvious coloration. Given these differences, it is perhaps surprising that there are only two types of cell. Bacterial cells are said to be prokaryotic (Greek for “before nucleus”) because they have very little visible internal organization so that, for instance, the genetic material is free within the cell. They are also small, the vast majority being 1–2 μm in length. The cells of all other organisms, from protists to mammals to fungi to plants, are eukaryotic (Greek for “with a nucleus”). These are generally larger (5–100 μm, although some eukaryotic cells are large enough to be seen with the naked eye) and structurally more complex. Eukaryotic cells contain a variety of specialized structures known collectively as organelles, surrounded by a viscous substance called cytosol. The largest organelle, the nucleus, contains the genetic information stored in the molecule deoxyribonucleic\\n---\\n|prokaryotic| | | | |\\n|---|---|---|---|---|\\n|ribosome| | | | |\\n|DNA| | | | |\\n|plasma membrane| | | | |\\n|bacterial cell|1 μm| | | |\\n|eukaryotic| | | | |\\n|nucleus| | | | |\\n|chromatin| | | | |\\n|nucleolus| | | | |\\n|Golgi apparatus| | | | |\\n|mitochondrion| | | | |\\n|rough endoplasmic reticulum| | | | |\\n|plasma membrane| | | |vacuole|\\n|animal cell| | | |vacuole|\\n|plasmodesma| | |cell wall| |\\n|chloroplast| | | |plant cell|\\n\\nFigure 1.7. Organization of prokaryotic and eukaryotic cells.\\n\\nThe structure and function of organelles will be described in detail in subsequent chapters. Table 1.1 provides a brief glossary of the major organelles and summarizes the differences between prokaryotic and eukaryotic cells.\\n\\nExample Box 1.1 Sterilization by Filtration\\n\\nBecause even the smallest cells are larger than 1 μm, harmful bacteria and protists can be removed from drinking water by passing through a filter with 200-nm-diameter holes. Filters can vary in size from huge, such as those used in various commercial processes, to small enough to be easily transportable by backpackers. Filtering drinking water greatly reduces the chances of bringing back an unwanted souvenir from your camping trip!\\n---\\n# Table 1.1. Differences Between Prokaryotic and Eukaryotic Cells\\n\\n| |Prokaryotes|Eukaryotes|\\n|---|---|---|\\n|Size|Usually 1–2 μm|Usually 5–100 μm|\\n|Nucleus|Absent|Present, bounded by nuclear envelope|\\n|DNA|Usually a single circular molecule (=chromosome)|Multiple molecules (=chromosomes), linear, associated with protein.|\\n|Cell division|Simple fission|Mitosis or meiosis|\\n|Internal membranes|Rare|Complex (nuclear envelope, Golgi apparatus, endoplasmic reticulum, etc.—Fig. 1.2)|\\n|Ribosomes|70S|80S (70S in mitochondria and chloroplasts)|\\n|Cytoskeleton|Absent|Microtubules, microfilaments, intermediate filaments|\\n|Motility|Rotary motor (drives bacterial flagellum)|Dynein (drives cilia and eukaryote flagellum); kinesin, myosin|\\n|First appeared|3.5 × 10^9 years ago|1.5 × 10^9 years ago|\\n\\na The tiny chromosomes of mitochondria and chloroplasts are exceptions; like prokaryotic chromosomes they are often circular.\\n\\nb The S value, or Svedberg unit, is a sedimentation rate. It is a measure of how fast a molecule moves in a gravitational field, and therefore in an ultracentrifuge.\\n\\n# Special Properties of Plant Cells\\n\\nAmong eukaryotic cells the most striking difference is between those of animals and plants (Fig. 1.7). Plants have evolved a sedentary lifestyle and a mode of nutrition that means they must support a leaf canopy. Their cells are enclosed within a rigid cell wall that gives shape to the cell and structural rigidity to the organism (page 53). This is in contrast to the flexible boundaries of animal cells. Plant cells frequently contain one or more vacuoles that can occupy up to 75% of the cell volume. Vacuoles accumulate a high concentration of sugars and other soluble compounds. Water enters the vacuole to dilute these sugars, generating hydrostatic pressure that is counterbalanced by the rigid wall. In this way the cells of the plant become stiff or turgid, in the same way that when an inner tube is inflated inside a bicycle tire the combination becomes stiff. Vacuoles are often pigmented, and the spectacular colors of petals and fruit reflect the presence of compounds such as the purple anthocyanins in the vacuole. Cells of photosynthetic plant tissues contain a special organelle, the chloroplast, that houses the light-harvesting and carbohydrate-generating systems of photosynthesis (page 271). Plant cells lack centrosomes (page 382) (Fig. 1.2) although these are found in many algae.\\n\\n# Viruses\\n\\nViruses occupy a unique space between the living and nonliving worlds. On one hand they are made of the same molecules as living cells. On the other hand they are incapable of independent existence, being completely dependent on a host cell to reproduce. Almost all living organisms have viruses that infect them. Human viruses include polio, influenza, herpes, rabies, ebola, smallpox, chickenpox, and the AIDS (acquired immunodeficiency\\n---\\n# CELLS AND TISSUES\\n\\nViruses are submicroscopic particles consisting of a core of genetic material enclosed within a protein coat called the capsid. Some viruses have an extra membrane layer called the envelope. Viruses are metabolically inert until they enter a host cell, whereupon the viral genetic material directs the host cell machinery to produce viral protein and viral genetic material. Viruses often insert their genome into that of the host, an ability that is widely made use of in molecular genetics (Chapter 7). Bacterial viruses, called bacteriophages (page 74), are used by scientists to transfer genes between bacterial strains. Human viruses are used as vehicles for gene therapy. By exploiting the natural infection cycle of a virus such as adenovirus, it is possible to introduce a functional copy of a human gene into a patient suffering from a genetic disease such as cystic fibrosis (Chapter 20).\\n\\n# ORIGIN OF EUKARYOTIC CELLS\\n\\nProkaryotic cells are simpler and more primitive in their organization than eukaryotic cells. According to the fossil record, prokaryotic organisms antedate, by at least 2 billion years, the first eukaryotes that appeared some 1.5 billion years ago. It seems highly likely that eukaryotes evolved from prokaryotes, and the most likely explanation of this process is the endosymbiotic theory. The basis of this hypothesis is that some eukaryotic organelles originated as free-living prokaryotes that were engulfed by larger cells in which they established a mutually beneficial relationship. For example, mitochondria would have originated as free-living aerobic bacteria and chloroplasts as cyanobacteria, photosynthetic prokaryotes formerly known as blue-green algae.\\n\\nThe endosymbiotic theory provides an attractive explanation for the fact that both mitochondria and chloroplasts contain DNA and ribosomes of the prokaryotic type (Table 1.1). The case for the origin of other eukaryotic organelles is less persuasive. While it is clearly not perfect, most biologists are now prepared to accept that the endosymbiotic theory provides at least a partial explanation for the evolution of the eukaryotic cell from a prokaryotic ancestor. Unfortunately, living forms having a cellular organization intermediate between prokaryotes and eukaryotes are rare. Some primitive protists possess a nucleus but lack mitochondria and other typical eukaryotic organelles. They also have the prokaryotic type of ribosomes. These organisms are all intracellular parasites and they include Microspora, an organism that infects AIDS patients.\\n\\n# CELL SPECIALIZATION\\n\\nAll the body cells that comprise a single organism share the same set of genetic instructions in their nuclei. Nevertheless, the cells are not all identical. Rather, plants and animals are composed of different tissues, groups of cells that are specialized to carry out a common function. This specialization occurs because different cell types read out different parts of the DNA blueprint and therefore make different proteins, as we will see in Chapter 6. In animals there are four major tissue types: epithelium, connective tissue, nervous tissue, and muscle.\\n\\n# Epithelia\\n\\nEpithelia are sheets of cells that cover the surface of the body and line its internal cavities such as the lungs and intestine. The cells may be columnar, taller than they are broad, or\\n---\\n# CELL SPECIALIZATION\\n\\nsquamous, meaning flat. In the intestine, the single layer of columnar cells lining the inside, or lumen, has an absorptive function that is increased by the folding of the surface into villi (Fig. 1.8). The luminal surfaces of these cells have microvilli that increase the surface area even further. The basal surface sits on a supporting layer of extracellular fibers called the basement membrane. Many of the epithelial cells of the airways, for instance, those lining the trachea and bronchioles, have cilia on their surfaces. These are hairlike appendages that actively beat back and forth, moving a layer of mucus away from the lungs (Chapter 18). Particles and bacteria are trapped in the mucus layer, preventing them from reaching the delicate air exchange membranes in the lung. In the case of the skin, the epithelium is said to be stratified because it is composed of several layers.\\n\\nMedical Smoking Is Bad for Your Cilia\\n\\nRelevance The cilia on the epithelial cells that line our airways move a belt of mucus that carries trapped particles and microorganisms away from the lungs. These cilia are paralyzed by cigarette smoke. As a consequence, smokers have severely reduced mucociliary clearance with the result that they have to cough to expel the mucus that continuously accumulates in their lungs. Because of this impaired ciliary function, smokers are much more susceptible to respiratory diseases such as emphysema and asthma. The cilia of passive smokers, people who breathe smoky air, are also affected.\\n\\n# Connective Tissue\\n\\nConnective tissues provide essential support for the other tissues of the body. They include bone, cartilage, and adipose (fat) tissue. Unlike other tissues, connective tissue contains relatively few cells within a large volume of extracellular matrix that consists of different types of fiber embedded in amorphous ground substance (Fig. 1.8). The most abundant of the fibers is collagen, a protein with the tensile properties of steel that accounts for about a third of the protein of the human body. Other fibers have elastic properties that permit the supported tissues to be displaced and then to return to their original position. The amorphous ground substance absorbs large quantities of water, facilitating the diffusion of metabolites, oxygen, and carbon dioxide to and from the cells in other tissues and organs. Of the many cell types found in connective tissue, two of the most important are fibroblasts, which secrete the ground substance and fibers, and macrophages, which remove foreign, dead, and defective material from it. A number of inherited diseases are associated with defects in connective tissue. Marfan’s syndrome, for example, is characterized by long arms, legs, and torso and by a weakness of the cardiovascular system and eyes. These characteristics result from a defect in the organization of the collagen fibers.\\n\\n# Nervous Tissue\\n\\nNervous tissue is a highly modified epithelium that is composed of several cell types. Principal among these are the nerve cells, also called neurons, along with a variety of supporting cells that help maintain them. Neurons extend processes called axons, which can be over a meter in length. Neurons constantly monitor what is occurring inside and outside the body. They integrate and summarize this information and mount appropriate responses to it (Chapters 15–17). Another type of cell called glia has other roles in nervous tissue including forming the electrical insulation around axons.\\n---\\n# CELLS AND TISSUES\\n\\none brush border\\nbrush\\nborder\\nepipelial cell\\ngoblet cell\\none villus\\ntight junction\\nnucleus\\nbasement membrane\\nepipelial cell\\nfibroblast\\ncollagen\\nconnective tissue\\nfibers\\nextracellular matrix\\ncapillary\\nred blood cell\\nsmoop muscle cells orientated circularly around intestine\\nsmoop muscle cells orientated longitudinally along intestine (cut in cross section)\\n\\nFigure 1.8. Tissues and structures of the intestine wall.\\n\\nMuscle tissue can be of two types, smooth or striated. Smooth muscle cells are long and slender and are usually found in the walls of tubular organs such as the intestine and many blood vessels. In general, smooth muscle cells contract slowly and can maintain the contracted state for a long period of time. There are two classes of striated muscle: cardiac and skeletal. Cardiac muscle cells make up the walls of the heart chambers. These\\n---\\n# CELL SPECIALIZATION\\n\\n|A|B|\\n|---|---|\\n|C|D|\\n\\nPlants are composed of several tissues.\\n\\nSkeletal muscle cells are branched cells that are connected electrically by gap junctions (page 55), and their automatic rhythmical contraction powers the beating of the heart. Each skeletal muscle is a bundle of hundreds to thousands of fibers, each fiber being a giant single cell with many nuclei. This rather unusual situation is the result of an event that occurs in the embryo when the cells that give rise to the fibers fuse together, pooling their nuclei in a common cytoplasm. The mechanism of contraction of skeletal muscle will be described in Chapter 18.\\n\\nPlants\\n\\nPlant cells are also organized into tissues (Fig. 1.9). The basic organization of a shoot or root is into an outer protective layer, or epidermis, a vascular tissue that provides support and\\n---\\n# CELLS AND TISSUES\\n\\ntransport, and a cortex that fills the space between the two. The epidermis consists of one or more layers of closely packed cells. Above the ground these cells secrete a waxy layer, the cuticle, which helps the plant retain water. The cuticle is perforated by pores called stomata that allow gas exchange between the air and the photosynthetic cells and also constitute the major route for water loss by a process called transpiration. Below ground, the epidermal cells give rise to root hairs that are important in the absorption of water and minerals. The vascular tissue is composed of xylem, which transports water and its dissolved solutes from the roots, and phloem, which conveys the products of photosynthesis, predominantly sugars, to their site of use or storage. The cortex consists primarily of parenchyma cells, unspecialized cells whose cell walls are usually thin and bendable. They are the major site of metabolic activity and photosynthesis in leaves and green shoots.\\n\\n# SUMMARY\\n\\n1. All living organisms are made of cells.\\n2. Our understanding of cell structure and function has gone hand in hand with developments in microscopy and its associated techniques.\\n3. Light microscopy revealed the diversity of cell types and the existence of the major organelles: nucleus, mitochondrion and, in plants, the vacuole and chloroplast.\\n4. The electron microscope revealed the detailed structure of the larger organelles and resolved the cell ultrastructure, the fine detail at the nanometer scale.\\n5. There are two types of cells, prokaryotes and eukaryotes.\\n6. Prokaryotic cells have very little visible internal organization. They usually measure 1–2 μm across.\\n7. Eukaryotic cells usually measure 5–100 μm across. They contain a variety of specialized internal organelles, the largest of which, the nucleus, contains the genetic material.\\n8. The endosymbiotic theory proposes that some eukaryotic organelles, such as mitochondria and chloroplasts, originated as free-living prokaryotes.\\n9. The cells of plants and animals are organized into tissues. In animals there are four tissue types: epithelium, connective tissue, nervous tissue, and muscle. Plants are formed of epidermis, cortex, and vascular tissues.\\n\\n# REVIEW QUESTIONS\\n\\nFor each question, choose the ONE BEST answer or completion.\\n\\n1. Eukaryotic cells usually contain\\n- A. a nucleus.\\n- B. mitochondria.\\n- C. ribosomes.\\n---\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n|1.|E. These are all typical features of eukaryotic cells.|\\n|---|---|\\n|2.|B. None of the others are true. In particular the fact that an electron beam is used means that the lenses are electromagnets, not glass, and means that the interior of an electron microscope is held at a vacuum, which means in turn that it is not a suitable environment for living tissue and tends to mean that electron microscopes are bulky, heavy pieces of equipment.|\\n\\n# 1.\\n\\n|A.|microfilaments.|\\n|---|---|\\n|B.|microvilli.|\\n|C.|microtubules.|\\n|D.|all of the above.|\\n\\n# 2.\\n\\n|A.|is ideal for looking at living cells.|\\n|---|---|\\n|B.|has a resolution of 0.2 nm.|\\n|C.|uses lenses made of glass.|\\n|D.|was invented by Anton van Leeuwenhoek.|\\n|E.|is easily transportable.|\\n\\n# 3.\\n\\n|A.|Microvilli are typically associated with nervous tissue.|\\n|---|---|\\n|B.|Columnar and squamous are types of epithelial tissue.|\\n|C.|Collagen is a component of the extracellular matrix.|\\n|D.|Glial cells form the insulation around nerve cells.|\\n|E.|Epidermis, vascular tissue, and cortex are types of plant tissue.|\\n\\n# 4.\\n\\n|A.|1 μm.|\\n|---|---|\\n|B.|10 μm.|\\n|C.|1 mm.|\\n|D.|10 mm.|\\n|E.|1 m.|\\n\\n# 5.\\n\\n|A.|a plasma membrane.|\\n|---|---|\\n|B.|DNA.|\\n|C.|ribosomes.|\\n|D.|nuclei.|\\n|E.|molecular motors.|\\n\\n# 6.\\n\\n|A.|condenser, specimen, objective, projector.|\\n|---|---|\\n|B.|condenser, objective, specimen, projector.|\\n|C.|condenser, projector, specimen, objective.|\\n|D.|objective, specimen, projector, condenser.|\\n|E.|objective, projector, specimen, condenser.|\\n\\n# 7.\\n\\n|A.|Smooth.|\\n|---|---|\\n|B.|Ciliated.|\\n|C.|Striated.|\\n|D.|Cardiac|\\n|E.|Skeletal.|\\n---\\n# CELLS AND TISSUES\\n\\n3. A. Microvilli are found on epithelia, for example, the epithelium lining the intestine.\\n\\n4. C. The international system prefixes used in this book are m for milli, meaning 10-3, μ for micro,\\nmeaning 10-6, and n for nano meaning 10-9. Thus 1,000,000 times 10-9 is 10-3 so 1,000,000 nm\\nis equal to 1 mm.\\n\\n5. D. Eukaryotes have all of these, but prokaryotes do not have nuclei. Rather, the DNA is free in the\\ncytoplasm.\\n\\n6. A\\n\\n7. B. No muscle cells are ciliated; as we will see in Chapter 18, the molecular motor systems used\\nin cilia and in muscles are different. Muscle cells are divided into smooth and striated; the striated\\ntype is then further subdivided into cardiac and skeletal.\\n---\\nFROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\nOrganisms are made up of a lot of different chemicals. These vary in size, from small molecules like water to large molecules like DNA, and interact and associate in many different ways to generate the processes of life. In this chapter we will introduce some basic concepts of how these chemicals are made and interact. We will then describe the most important of these chemicals: water, carbohydrates, nucleotides, amino acids, and lipids.\\n\\nTHE CHEMICAL BOND: SHARING ELECTRONS\\n\\nWater is the most abundant substance in organisms. Cells are rich in water. Cytoplasm consists of organelles floating in a watery medium called cytosol that also contains proteins. The situation is not so different outside our cells. Although we are land animals living in air, most of our cells are bathed in a watery fluid called extracellular medium. We will therefore start by considering water itself.\\n\\nFigure 2.1a\\nshows a molecule of water, consisting of one atom of oxygen and two hydrogen atoms, joined to form an open V shape. The lines represent covalent bonds formed when atoms share electrons, each seeking the most stable structure. Oxygen has a greater affinity for electrons than does hydrogen so the electrons are not distributed equally. The oxygen grabs a greater share of the available negative charge than do the hydrogen atoms. The molecule of water is polarized, with partial negative charge on the oxygen and partial positive charges on the two hydrogens. We write the charge on each hydrogen as δ+\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C2004 by John Wiley & Sons, Inc.\\n---\\nFROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\nFigure 2.1. Water is a polar molecule while the chlorine molecule is nonpolar.\\n\\nFigure 2.1a shows a water molecule. The hydrogen atoms have a small positive charge δ+ to indicate that it is smaller than the charge on a single hydrogen nucleus. The oxygen atom has the small net negative charge 2δ−. Molecules that, like water, have positive regions sticking out one side and negative regions sticking out the other are called polar.\\n\\nFigure 2.1b shows a molecule of chlorine gas. It consists of two chlorine atoms, each of which consists of a positively charged nucleus surrounded by negatively charged electrons. Like oxygen, chlorine atoms tend to accept electrons when they become available, but the battle is equal in the chlorine molecule: The two atoms share their electrons equally and the molecule is nonpolar.\\n\\nFigure 2.2. Formation of sodium chloride, an ionic compound.\\n\\n|Na| |Na +|\\n|---|---|---|\\n|e−| | |\\n|Cl| |Cl−|\\n|Cl| |Cl−|\\n|e−| | |\\n|Na|(a)|Na +|\\n|− + − + −| | |\\n|+ − + − +| | |\\n|− + − + −| | |\\n|+ − + − +| | |\\n---\\n# INTERACTIONS WITH WATER: SOLUTIONS\\n\\nFigure 2.2a shows what happens when a chlorine molecule is allowed to react with the metal sodium. Each atom of chlorine takes over one electron from a sodium atom. This leaves the sodium atoms with a single positive charge because there is now one more positive charge on the sodium nucleus than negatively charged surrounding electrons. Similarly, each chlorine atom now has a single negative charge because it now has one more electron than there are positive charges in its nucleus. Chemical species that have either gained or lost electrons, and that therefore bear an overall charge, are called ions. The reaction of chlorine and sodium has produced sodium ions and chloride ions. Positively charged ions like sodium are called cations while negatively charged ones like chloride are called anions. The positively charged sodium ions and the negatively charged chloride ions now attract each other strongly. If there are no other chemicals around, the ions will arrange themselves to minimize the distance between sodium and chloride, and the resulting well-packed array of ions is a crystal of sodium chloride, shown in Figure 2.2b.\\n\\nFigure 2.3a shows one molecule of octane, the main constituent of gasoline. Octane is an example of a nonpolar solvent. Electrons are shared equally between carbon and hydrogen, and the component atoms do not bear a net charge. Figure 2.3b shows a small crystal of sodium chloride immersed in octane. At the edge of the crystal, positively charged sodium ions are being pulled in toward the center of the crystal by the negative charge on chloride ions, and negatively charged chloride ions are being pulled in toward the center of the crystal by the positive charge on sodium ions. The sodium and chloride ions will not leave the crystal. Sodium chloride is insoluble in octane. However, sodium chloride will dissolve in water, and Figure 2.4a shows why. The chloride ion at the top left is being pulled into the crystal by the positive charge on its sodium ion neighbors, but at the same time it is being pulled out of the crystal by the positive charge on the hydrogen atoms of nearby water molecules. Similarly, the sodium ion at the bottom left is being pulled into the crystal by the negative charge on its chloride ion neighbors, but at the same time it is being pulled out of the crystal by the negative charge on the oxygen atoms of nearby water molecules. The ions are not held in the crystal so tightly and can leave. Once the ions have left the crystal, they become surrounded by a hydration shell of water molecules, all oriented in the appropriate direction (Fig. 2.4b)—oxygen inward for a positive ion like sodium, hydrogen inward for a negative ion like chloride. A chemical species in solution, whether in water or in any other solvent, is called a solute. Liquids whose main constituent is water are called aqueous.\\n\\nWhen we exercise, our muscle cells can become acid, and this is what creates the pain of cramping muscles and the heart pain of angina. Acidity is important in all areas of biology, from the acidity gradient that drives our mitochondria (page 261) to the ecological consequences of acid rain. Acid solutions contain a high concentration of hydrogen ions. The hydrogen atom is unusual in that it only has one electron while, in its most common isotope, its nucleus\\n---\\n# FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\nFigure 2.3. (a) Structure of the nonpolar compound octane. (b) Ionic compounds are insoluble in nonpolar solvents.\\n\\nThe proton is a positively charged subatomic particle that comprises a single proton. In gases at very low pressure, it is possible for bare protons to exist alone and be manipulated, for example, in linear accelerators. However, in water protons never exist alone but always associate with another molecule, for example, with water to create the H3O+ ion. Acid solutions are those with an H3O+ concentration higher than 100 nmol liter−1.\\n\\nSour cream contains lactic acid. Pure lactic acid has the structure shown at the left of Figure 2.5a. The —COOH part in the box is called a carboxyl group. Both oxygens have a tendency to pull electrons away from the hydrogen and, in aqueous solution, the hydrogen is donated with a full positive charge to a molecule of water. The electron is left behind on the now negatively charged lactate ion.\\n\\nCH3CH(OH)COOH + H2O\\nCH3CH(OH)COO− + H3O+\\n---\\n# INTERACTIONS WITH WATER: SOLUTIONS\\n\\n|H| | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |\\n|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\\n| |H|O| |H| | |H| |O| | |H| |O| |H| |O| |H| |O| |H| |O| |H| |O| |H| |O| |H| |O| |H| |O| |H| |O| |H| |O| |H| |O| |H| |O| |H| |O| |H| |O| |H| |O| |H| |O| |H| |O| | | | | | | | | | | | | |\\n| | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |\\n| | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |\\n\\n(a)\\n\\n|H| | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |\\n|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\\n| |H|O| | | | |H| |O| | |H| |O| | |H| |O| | |H| |O| | |H| |O| | |H| |O| | |H| |O| | |H| |O| | |H| |O| | |H| |O| | |H| |O| | |H| |O| | |H| |O| | |H| |O| | |H| |O| | |H| |O| | | | | | | | | | | | | |\\n| | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | | |\\n\\n(b)\\n\\nFigure 2.4. Ionic compounds dissolve readily in water.\\n---\\n# FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\nFigure 2.5. Acids and bases, respectively, give up and accept H+ when dissolved in water.\\n\\nFor convenience we often write this as\\n\\nCH3CH(OH)COOH → CH3CH(OH)COO− + H3O+\\n\\nHere we are using H3O+ as a convenient symbol to denote H3O+. We do not mean that there are real H3O+ ions, that is, bare hydrogen nuclei, in aqueous solutions.\\n\\nThe equilibrium constant, K for the dissociation of lactic acid, is defined as\\n\\n|Ka =|([CH3CH(OH)COO−] [H3O+])| |\\n|---|---|---|\\n| |([CH3CH(OH)COOH])| |\\n\\nwhere the square brackets refer, by convention, to concentrations and the subscripts, e, denote that these are the concentrations of each species at equilibrium.\\n\\nDissolving lots of lactic acid in water produces an acid solution, that is, one with a high concentration of H3O+ (really H3O+) ions. For historical reasons, the acidity of a solution is given as the pH, defined thus:\\n\\npH = − log ([H3O+]) where [H3O+] is measured in moles per liter\\n\\nPure water has a pH of 7, corresponding to [H3O+] = 100 nmol liter−1 —this is said to be neutral as regards pH. If the pH is lower than this, then there is more H3O+ about and the solution is acid. Cytosol has a pH that lies very slightly on the alkaline side of neutrality, at about 7.2.\\n\\nThe pH of a solution determines the ratio of lactate to undissociated lactic acid. As the H3O+ concentration rises and pH falls, the equilibrium is pushed over from lactate toward lactic acid. At pH 3.9, the concentrations of lactate and lactic acid become equal. Looking at\\n---\\nINTERACTIONS WITH WATER: SOLUTIONS\\n\\nthe equation above, we can therefore see that when this happens K                     a = [H   + ]. Just as acidity\\nis given on the logarithmic pH scale so is the scale of strengths of different acids, pK                    a being\\ndefined as − log     10 K  a. The pK    a is the pH at which the concentration of the dissociated acid\\nis equal to the concentration of the undissociated acid. For an acid that is weaker than lactic\\nacid, the pK    a is higher than 3.9, meaning that the acid is less readily dissociated and needs\\na lower concentration of protons before it will give up its H                  + . For an acid that is stronger\\nthan lactic acid, the pK      a is less than 3.9: Such an acid is more readily dissociated and needs\\na higher H    +  concentration before it will accept H          +  and form undissociated acid.\\n\\nBases Are Molecules That Take H                        +   from Water\\nTrimethylamine is the compound that gives rotting fish its unpleasant smell. Pure trimethy-\\nlamine has the structure shown at the left of Figure 2.5b. When trimethylamine is dissolved\\nin water, it accepts an H     +  to become the positively charged trimethylamine ion shown on the\\nright of the figure. We refer to molecules that have accepted H                  +  ions as protonated, using\\n“proton” as a short way of saying “hydrogen nucleus.” Dissolving lots of trimethylamine\\nin water produces an alkaline solution, that is, one with a low concentration of H                      +  ions and\\nhence a pH greater than 7. The solution never runs out of H                   +  completely because new H           +\\nare formed from water:\\n\\nH  O → OH       − + H   +\\n2\\n\\nThus if we keep adding trimethylamine to water and using up H                        + , we end up with a low\\nconcentration of H       + , but lots of OH    − .\\nThe pH of a solution determines the position of equilibrium between protonated and\\ndeprotonated trimethylamine, and as before we define the pK                         a as the pH at which the\\nconcentration of protonated and deprotonated base are the same. The pK                       a of trimethylamine\\nis 9.74, meaning that the concentration of H            + must fall to the low level of 10      −9.74  mol liter  −1 ,\\nthat is, 0.2 nmol liter    −1 , before half of the trimethylamines will give up their H                + s.\\n\\nIsoelectric Point\\nThe large molecules called proteins (page 183) have many acidic and basic sites that will\\ngive up or accept an H       +  as the pH changes. In alkaline solutions proteins will tend to have\\nan overall negative charge because the acidic sites have lost an H                      +   and bear a negative\\ncharge. As the pH falls, the acidic sites accept an H                 +  and become uncharged, and basic\\nsites also accept an H      +  to gain a positive charge. Thus as the pH falls from an initial high\\nvalue, the overall charge on the protein becomes less and less negative and then more and\\nmore positive. The pH at which the protein has no overall charge is called the isoelectric\\npoint. The isoelectric points of different proteins are different, and this property is useful\\nin separating them during analysis (page 178). The majority of intracellular proteins have\\nan isoelectric point that is less than 7.2, so that at normal intracellular pH they bear a net\\nnegative charge.\\n\\nA Hydrogen Bond Forms When a Hydrogen Atom Is Shared\\nWe have seen how oxygen tends to grab electrons from hydrogen, forming a polar bond.\\nNitrogen and sulfur are similarly electron-grabbing. If a hydrogen attached to an oxygen,\\n---\\n26 FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\n(a)\\n\\na hydrogen bond\\ncannot form here\\nbecause the N H N\\natoms are not in a\\nstraight line\\n\\n||||||||||||\\n|---|---|---|---|---|---|---|---|---|---|---|\\n|| |||||| ||||\\n|| |||| |||| | |\\n\\n(b)\\n\\nFigure 2.6. The hydrogen bond.\\n\\nnitrogen or sulfur by a covalent bond gets close to a second electron-grabbing atom; then that\\nsecond atom also grabs a small share of the electrons to form what is known as a hydrogen\\nbond. The atom to which the hydrogen is covalently bonded is called the donor because it\\nis losing some of its share of electrons; the other electron-grabbing atom is the acceptor.\\nFor a hydrogen bond to form, the donor and acceptor must be within a fixed distance of one\\nanother (typically 0.3 nm) with the hydrogen on a straight line between them.\\n---\\n# CARBOHYDRATES: CANDY AND CANES\\n\\nLiquid water is so stable because the individual molecules can hydrogen bond, as illustrated in Figure 2.6a. Hydrogen bonding also plays a critical role in allowing DNA to store and replicate genetic information. Figure 2.6b shows how the base pairs (page 69) of DNA form hydrogen bonds in which hydrogen atoms are shared between nitrogen and oxygen and between nitrogen and nitrogen.\\n\\nBIOLOGICAL MACROMOLECULES\\n\\nVery large molecules, or macromolecules, are central to the working of cells. Large biological molecules are polymers: they are assembled by joining together small, simpler molecules, which are therefore called monomers. Chemical technology has mimicked nature by producing many important polymers—polyethylene is a polymer of ethylene monomers. Cells make a number of macromolecules that we will introduce, together with their monomer building blocks, in this chapter.\\n\\nCARBOHYDRATES: CANDY AND CANES\\n\\nCarbohydrates—sugars and the macromolecules built from them—have many different roles in cells and organisms.\\n\\nAn Assortment of Sweets\\n\\nAll carbohydrates are formed from the simple sugars called monosaccharides. Figure 2.7 shows the monosaccharide called glucose. The form shown at the top has five carbons joined each to the other with an oxygen atom completing the ring. As well as the oxygen in the ring, glucose has five other oxygens, each in an −OH (hydroxyl) group. Glucose easily switches between the three forms, or isomers, illustrated in Figure 2.7. The two ring structures are stereo isomers: although they comprise the same atoms connected by the same bonds, they represent two different ways of arranging the atoms in space. The two stereo isomers, named α and β, continually interconvert in solution via the open-chain form. Figure 2.8 shows five other monosaccharides that we will meet again in this book. Like glucose, each of these monosaccharides can adopt an open-chain form and a number of ring structures. In Figure 2.8 we show each sugar in a form that it adopts quite often. These sugars share with glucose the two characteristics of monosaccharides: they can adopt a form in which an oxygen atom completes a ring of carbons, and they have many hydroxyl groups. The generic names for monosaccharides are derived from the Greek for the number of their carbon atoms, so glucose, galactose, mannose, and fructose are hexoses (6 carbons) while ribose and ribulose are pentoses (5 carbons). Classically, a monosaccharide has the general formula Cn(H2O)n, hence the name carbohydrate. All the monosaccharides shown in Figures 2.7 and 2.8 fit this rule—the four hexoses can be written as C6(H2O)6 and the two pentoses as C5(H2O)5.\\n\\nIt is worth noting that although both sugars and organic acids contain an —OH group, the behavior of an —OH group that forms part of a carboxyl group is very different from one that is not next to a double-bonded oxygen. In general the —OH group in a carboxyl group will readily give up an H+ to water or other acceptors; this is not true of —OH groups.\\n---\\n# FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\n| | |CH OH|\\n|---|---|\\n|2|H|C 5 O H|\\n| |4 C|H H C 1 α-glucose|\\n| | |OH|\\n|OH|C 3|C 2 OH|\\n| |H|OH|\\n\\n1 CHO\\nH 2 C OH\\nHO 3 C H\\nH 4 C OH\\nH 5 C OH\\n\\n| | |CH OH|\\n|---|---|\\n|2|H|C 5 O OH|\\n| |4 C|H C 1 β-glucose|\\n| | |OH H|\\n|OH|C 3|C 2 H|\\n| |H|OH|\\n\\nFigure 2.7. Glucose, a monosaccharide, easily switches between three isomers.\\n\\nin general. We therefore reserve the term hydroxyl group for —OH groups on carbon atoms that are not also double-bonded to oxygen.\\n\\nDisaccharides Monosaccharides can be easily joined by glycosidic bonds in which the carbon backbones are linked through oxygen and a water molecule is lost. The bond is identified by the\\n---\\n# CARBOHYDRATES: CANDY AND CANES\\n\\n|6CH OH|\\n|---|\\n|OH|C 5 O OH|\\n|4C H|C 1 β-galactose|\\n|OH|2 H|\\n|H|3C C H|\\n| |OH|\\n|6CH OH|\\n|H|C 5 O H|\\n|4C H|C 1 α-mannose|\\n|OH|OH|\\n|OH|3C 2 C OH|\\n| |H H|\\n|6CH OH|O 1CH OH|\\n| |2|2|\\n|5C H OH|C 2 α-fructose|\\n|H|4C C 3 OH|\\n| |OH H|\\n|5CH 2OH|O OH|\\n|4C H|C 1 β-ribose|\\n|H|3C C 2 H|\\n| |OH OH|\\n|H|O 1CH OH|\\n| |2|\\n|5C H|C 2 α-ribulose|\\n|H|4C C 3 OH|\\n| |OH OH|\\n\\nFigure 2.8. Some monosaccharides, each shown as a common isomer.\\n---\\n# FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\nFigure 2.9. The disaccharide lactose.\\n\\n|carbons linked.|For example, Figure 2.9 shows lactose, a sugar found in milk that is formed when galactose and glucose are linked by a (1→4) glycosidic bond. Sugars formed from two monosaccharide monomers are called disaccharides. Whenever a more complicated molecule is made from simpler building blocks, we call the remnant of the individual building blocks residues. Thus lactose is made of one galactose residue and one glucose residue.|\\n|---|---|\\n|A complication arises here. Although free monosaccharides can easily switch between the α and β forms, formation of the glycosidic bond locks the shape. Thus, although galactose in solution spends equal time in the α and β forms, the galactose residue in lactose is locked into the β form, so the full specification of the bond in lactose is β1→4.| |\\n\\nOut of the Sweet Comes Forth Strength\\n\\nFormation of glycosidic bonds can continue almost indefinitely. A chain of up to 100 or so monosaccharides is called an oligosaccharide, from the Greek word oligo meaning few. Longer polymers are called polysaccharides and have characteristics very different from those of their monosaccharide building blocks. Figure 2.10a shows part of a molecule of glycogen, a polymer made exclusively of glucose monomers organized in long chains with α1→4 links. The glycogen chain branches at intervals, each branch being an α(1→4) linked chain of glucose linked to the main chain with an α(1→6) bond. Solid lumps of glycogen are found in the cytoplasm of muscle, liver, and some other cells. These glycogen granules are 10–40 nm in diameter with up to 120,000 glucose residues. Glycogen is broken down to release glucose when the cell needs energy (Chapter 13).\\n\\nCellulose makes up the cell wall of plants and is the world’s most abundant macro-molecule. Like glycogen, cellulose is a polymer of glucose, but this time the links are β1→4. It is shown in Figure 2.10b. Drawn on the flat page, it looks very like glycogen. However, the difference in bond type is critical. Glucoses linked by α(1→4) links arrange themselves in a floppy helix, while glucoses linked by β(1→4) links form extended chains.\\n---\\n# CARBOHYDRATES: CANDY AND CANES\\n\\n| | | | | | | | | | | | | | |\\n|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\\n| | | | | | | | | | | | | | |\\n| | | | | | | | | | | | | | |\\n| | | | | | | | | | | | | | |\\n| | | | | | | | | | | | | | |\\n| | | | | | | | | | | | | | |\\n\\nideal for building the rigid plant cell wall. Animals have enzymes (protein catalysts) that\\ncan break down the α(1→4) bond in glycogen, but only certain bacteria and fungi can break\\nthe β(1→4) link in cellulose. All animals that eat plants rely on bacteria in their intestines\\nto provide the enzymes to digest cellulose.\\n\\nModified Sugars\\nA number of chemically modified sugars are important in biology. Figure 2.11 shows\\nthree. Deoxyribose is a ribose that is missing the —OH group on carbon number two. Its\\n\\nFigure 2.10. The polysaccharides glycogen and cellulose.\\n---\\n# FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\n|2|5|C|OH|O| |OH|\\n|---|---|---|---|---|---|---|\\n|4|C|H| |H|C|1|\\n| | |H| |3|C| |C|2|H|\\n| | |OH| |H| | |\\n| | | |deoxyribose| | | |\\n\\n|2|6|C|H|OH|\\n|---|---|---|---|---|\\n|H|5|C|O|OH|\\n| | |4|C|H| |C|1|\\n| | |OH| |H|\\n\\n| |OH|C|3|C|2|H|\\n|---|---|---|---|---|---|---|\\n| | | |H| |N|H|\\n| | | |C|O| | |\\n| | | |CH3| | | |\\n| | |N-acetyl glucosamine| | | | |\\n\\n| |O|\\n|---|---|\\n| |O|P|O|\\n| |O|\\n| |H|C|6|H|\\n\\n| |H|\\n|---|---|\\n| |H|5|C|O|H|\\n|4|C|H| |C|1|\\n| |OH|3|C|2|OH|\\n| |H| |H|\\n| |mannose-6-phosphate|\\n\\nFigure 2.11. Deoxyribose, N-acetyl glucosamine and mannose-6-phosphate are modified sugars.\\n---\\n# CARBOHYDRATES: CANDY AND CANES\\n\\nMain use is in making deoxyribonucleic acid—DNA. N-acetyl glucosamine is a glucose in which an —NHCOCH3 group replaces the —OH on carbon two. It is used in a number of oligosaccharides and polysaccharides, for example, in chitin, which forms the hard parts of insects. Mannose-6-phosphate is a mannose that has been phosphorylated, that is, had a phosphate group attached, on carbon number six. We will meet more phosphorylated sugars in the next section.\\n\\nH adenine H N C C N C H N C N C N 5′ H HO CH 2 4′ O 1′ C H H C ribose H C 3′ C2′ H OH OH (a) adenosine, a nucleoside\\nH adenine triphosphate H NN C C CN N C H O O O 5′ H C N O P O P O P O CH 2 O O O 4′C H O H C 1′ribose γγ αα H C C H phosphatephosphateββ phosphatephosphate 3′ 2′ phosphatephosphate OH OH (b) adenosine triphosphate, a nucleotide\\n\\nFigure 2.12. Adenosine, a nucleoside, and adenosine triphosphate, a nucleotide.\\n---\\n# FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\n|purines|pyrimidines|\\n|---|---|\\n|H H N C C N C H N C N C N H H adenine (A) O C C N C H H N C N C N H H N H guanine (G) O C C N C H H N C N C N H Hhypoxanthine |H H C C N H C N C N H O cytosine (C) H C H O H C C N H H N C O thymine (T) H C C O H C C N H Huracil (U)O |\\n\\nFigure 2.13. Seven bases found in nucleotides.\\n---\\n# NUCLEOSIDES, PHOSPHATE, AND NUCLEOTIDES\\n\\nFigure 2.12a shows a nucleoside called adenosine. It is composed of ribose coupled to a nitrogen-rich compound called adenine. The numbers on the sugar—1, 2; and so forth—are the same numbering system we have seen before; the symbol is pronounced “prime” and is there to indicate that we are identifying the atoms of the sugar, not the atoms of the adenine. The name nucleoside reflects the fact that phosphorylated nucleosides (see below) are the building blocks of the nucleic acids that form the genetic material in the nucleus. However, nucleosides also play important roles in other places inside and outside the cell. Seven different compounds can be used to generate nucleosides (Fig. 2.13). All seven contain many nitrogen atoms and one or more ring structures. They are the three purines called adenine, guanine and hypoxanthine, the three pyrimidines called cytosine, thymine, and uracil, and an odd man out called nicotinamide. These ring compounds are called bases. Historically, the name arose because the compounds are indeed bases in the sense used earlier in this chapter—they will exchange an H+ with water. The roots of the name are now forgotten by most molecular biologists, who now use the word base to mean a purine.\\n\\n||||\\n|---|---|---|\\n|(a)|(b)|(c)|\\n\\nFigure 2.14. Phosphate groups can attach to COH groups or to other phosphate groups.\\n---\\n# FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\nIn general, a nucleoside is formed by attaching one of these bases to the 1-carbon atom of ribose.\\n\\nPhosphorous, although by weight a relatively minor fraction of the whole cell, plays a number of critical roles. In solution phosphorous is mainly found as a phosphate ion with a single hydrogen atom still attached, HPO42−. We often indicate phosphate ions with the symbol Pi, meaning inorganic phosphate. Where phosphate becomes important, however, is when it is attached to an organic, that is, carbon-containing, molecule. Phosphate can substitute into any C—OH group with the loss of a water molecule. The equilibrium in the reaction shown lies far to the left, but cells have other strategies for attaching phosphate groups to organic molecules. Once one phosphate group has been added, more can be added to form a chain. Once again, the equilibrium in the reaction shown lies far to the left, but cells can achieve this result using other strategies.\\n---\\n# AMINO ACIDS, POLYPEPTIDES, AND PROTEINS\\n\\nFigure 2.16. Inositol trisphosphate, a multiply phosphorylated polyalcohol.\\n\\nFigure 2.12b shows adenosine with a chain of three phosphate groups attached to the 5′ carbon of the ribose. This important molecule is adenosine triphosphate (ATP), and we will meet it many times in the course of this book. The three phosphate groups are denoted by the Greek letters α, β, and γ. Phosphorylated nucleosides are called nucleotides.\\n\\nFigure 2.15 shows another important molecule formed from adenosine. Here adenosine and a nicotinamide nucleotide are joined through their phosphate groups. The resulting molecule, called nicotinamide adenine dinucleotide (NAD+), plays a critical role in the cellular energy budget.\\n\\nMolecules with several OH groups can become multiply phosphorylated. Figure 2.16 shows inositol trisphosphate (IP3), an important messenger molecule we will meet again in Chapter 16. Both ATP and IP3 have three phosphate groups, but to indicate the fact that in ATP these are arranged in a chain, while in IP3 they are attached to different carbons, we use the prefix tri in adenosine triphosphate and the prefix tris in inositol trisphosphate. Similarly, a compound with two phosphates in a chain is called a diphosphate, while one with one phosphate on each of two different carbons is called a bisphosphate.\\n\\n# AMINO ACIDS, POLYPEPTIDES, AND PROTEINS\\n\\nAmino acids contain both a COOH group, which readily gives an H+ to water and is therefore acidic, and a basic NH2 group, which readily accepts H+ to become NH3+. Figure 2.17a shows two amino acids, leucine and γ-amino butyric acid (GABA), in the form in which they are found at normal pH: the COOH groups have each lost an H+ and the NH2 groups have each gained one, so that the molecules bear both a negative and a positive charge. We name organic acids by labeling the carbon that bears the carboxyl group α, the next one β, and so on. When we add an amino group, making an amino acid, we state the letter of the carbon to which the amino group is attached. Hence leucine is an α-amino acid.\\n---\\n# FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\nwhile GABA stands for gamma-amino butyric acid. α-Amino acids are the building blocks\\nof proteins. They have the general structure shown in Figure 2.17b where R is called the\\nside chain. Leucine has a simple side chain of carbon and hydrogen. Other amino acids\\nhave different side chains and so have different properties. It is the diversity of amino side\\nchains that give proteins their characteristic properties (page 185).\\n\\nα-Amino acids can link together to form long chains through the formation of a peptide\\nbond between the —COO− group of one amino acid and the —NH3+ group of the next.\\nFigure 2.17c shows the generalized stucture of such a chain of α-amino acids. If there are\\nfewer than about 50 amino acids in a polymer we tend to call it a peptide. More and it\\nis a polypeptide. Polypeptides that fold into a specific shape are called proteins. Most of\\nthe components of the machinery of the cell—the chemicals that carry out the processes\\nof life—are proteins. Peptides and polypeptides are formed inside cells by adding specific\\namino acids to the end of a growing polymer. As we will see in Chapter 4, the particular\\namino acid to be added at each step is defined by the instructions on a molecule called\\n\\nFigure 2.17. Amino acids and pe peptide bond.\\n\\nδCH3\\nγCH CH3\\nγCH2 NH3\\nβCH2\\nOOC αCH NH3 OOC αCH2\\nleucine (a) γ-amino butyric acid\\nR\\nOOC CH NH3\\nα-amino acid general form (b)\\nR1 H R2 H R3 H Rn\\nNH3 CH C N CH C N CH C - - - - - - - N CH COO\\nO O O\\npeptide bond peptide bond generalized peptide (c)\\n---\\n# LIPIDS\\n\\nMessenger ribonucleic acid (RNA), which in turn contains a copy of the information recorded on the cell’s DNA that is stored in the nucleus. This is the central dogma of molecular biology: DNA makes RNA makes protein.\\n\\n# LIPIDS\\n\\nFigure 2.18a shows the fatty acid oleic acid. It comprises a carboxyl group plus a long tail of carbons and hydrogens. At neutral pH oleic acid gives up its H+ to become the oleate ion (Fig. 2.18b). The two ends of the oleate ion are very different. The carboxyl group is negatively charged and hence will associate readily with water molecules. The hydrocarbon tail is nonpolar and does not readily associate with water. The molecule is said to be amphipathic, from the Greek for “hating both,” meaning that one half does not like to be in water while the other half does not like to be in a nonpolar environment like octane.\\n\\nFigure 2.18c shows the small molecule glycerol. Like sugars, glycerol has many hydroxyl groups and, like sugars, tastes sweet, as its name suggests. However, it is not a sugar because it cannot adopt an oxygen-containing ring structure. Rather, because compounds containing hydroxyl groups are called alcohols, glycerol is a polyalcohol.\\n\\nAs we shall see in Chapter 13, cells can join fatty acids and glycerol to make glycerides. The bond is formed by removing the elements of water between the carboxyl group of the fatty acid and a hydroxyl group of glycerol. Any bond of this type, between a carboxyl group and a hydroxyl group, is called an ester bond. Figure 2.19 shows two glycerides. In Figure 2.19a is trioleoylglycerol, the main component of olive oil. It is a triacylglycerol (or triglyceride) formed from three molecules of oleic acid and one molecule of glycerol. The fatty acid residues are called acyl groups. Formation of the ester bonds has removed the charged carboxyl groups that rendered the oleate amphipathic. Almost all of the triacylglycerol molecule is simple hydrocarbon chain that cannot hydrogen bond with water molecules. For this reason, olive oil and other triacylglycerols do not mix with water. They are therefore said to be hydrophobic. Triacylglycerols that are liquid at room temperature are called oils, those that are solid are called fats, but they are all the same sort of molecule.\\n\\n# Example 2.1 Salad Dressing Is a Mixture of Solvents\\n\\nVinegar is a dilute solution of acetic acid in water. Acetic acid gives up an H+ to water to leave the negatively charged acetate ion. The simplest salad dressing is a shaken-up mixture of olive oil and vinegar. Olive oil is hydrophobic and so does not dissolve in the water, the two liquids remain as separate droplets and soon separate again after shaking. However much one shakes, all the acetate will remain in the water because it is an ion. If salt (Na+ Cl−) is added to the dressing, it also dissolves in the water, with none dissolving in the oil. In contrast if you add chilies to the salad dressing, the active chemical component, capsacin, will dissolve in the oil because it is nonpolar. Only by shaking up the mixture can you get all the tastes together.\\n\\nCells contain droplets of triacylglycerol within their cytoplasm. These are usually small, but in fat cells, which are specialized as fat stores, the droplets coalesce into a single large globule so that the cell’s cytoplasm is squeezed into a thin layer surrounding the fat.\\n---\\nFROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\n|H|O|\\n|---|---|\\n|C|O|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n|H|C|\\n|H|C|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|\\n| |H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|C|H|\\n| |H|\\n| |H|\\n| |H|H|H|\\n| |H|C|CC|C|H|\\n| |O|O|O|\\n| |H|H|H|\\n\\n(a) oleic acid\\n\\n(b) oleate\\n\\nFigure 2.18. Oleic acid and glycerol.\\n\\nglobule. During fasting triacylglycerols are broken down into free fatty acids and glycerol, a process called lipolysis. The fatty acids and the glycerol then enter the circulation for use by tissues.\\n\\nFigure 2.19b shows phosphatidylcholine, an example of the phospholipids that make the plasma membrane and other cell membranes. Like triacylglycerols, phospholipids have\\n---\\n# CH 3\\n\\nCH3   N + CH3  charged\\n\\nCH 2     headgroup\\n\\nCH 2\\n\\nO          phospho-\\n\\nO    P    O –   diester\\n\\nH              HH            H                            H             H             O          link\\n\\nH    C              CC            C   H  glycerol        H     C             C             C     H  glycerol\\n\\nO              O             O                            OOO           OOO           H             ester\\n\\nC    O         CC  OO        C  O                         C   O         C    O                      bonds\\n\\nH   C    H    H    C   H     H   C  H                    H    C   H     H   C    H\\n\\nH   C    H    H    C   H     H   C  H                    H    C   H     H   C    H\\n\\nH   C    H    H    C   H     H   C  H                    H    C   H     H   C    H\\n\\nH   C    H    H    C   H     H   C  H                    H    C   H         C    H\\n\\nH   C    H    H    C   H     H   C  H                    H    C   H         C    H\\n\\nH   C    H    H    C   H     H   C  H                    H    C   H     H   C    H\\n\\nH   C    H    H    CC  H     H   C  H                    H    C   H         C    H\\n\\nH   C         H    CC        H   C                       H    C   H         C    H\\n\\nH   C         H    C         H   C      acyl             H    C   H     H   C    H                 acyl\\n\\nH   C    H    H    C   H     H   C  H   group            H    C   H         C    H                 group\\n\\nH   C    H    H    C   H     H   C  H                    H    C   H         C    H\\n\\nH   C    H    H    C   H     H   C  H                    H    C   H     H   C    H\\n\\nH   C    H    H    C   H     H   C  H                    H    C   H         C    H\\n\\nH   C    H    H    C   H     H   C  H                    H    C   H         C    H\\n\\nH   C    H    H    C   H     H   C  H                    H    C   H     H   C    H\\n\\nH   C    H    H    C   H     H   C  H                    H    C   H     H   C    H\\n\\nH   C    H    H    C   H     H   C  H                    H    C   H     H   C    H\\n\\nH              H             H                            H         H   C    H\\n\\nH   C    H    H    C   H     H   C  H                    H    C   H     H   C    H\\n\\ntrioleoylglycerol, a triacylglycerol                    phosphatidylcholine, aHphospholipid\\n\\n(a)                                                        (b)\\n\\naqueous medium\\n\\nlipid bilayer                                                    hydrophobic               hydrophilic\\n\\nlipid tails               lipid\\n\\nheadgroups\\n\\n(c)                        aqueous medium\\n\\nFigure 2.19.     Lipids.\\n\\n41\\n---\\n# FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\nfatty acids attached to glycerol, but only two of them. In place of the third fatty acid is a polar, often electrically charged head group. The head group is joined to the glycerol through phosphate in a structure called a phosphodiester link. The combination of head group and negatively charged phosphate is able to associate strongly with water—it is said to be hydrophilic. The two fatty acids, on the other hand, form a tail that, like olive oil, is hydrophobic. Phospholipids can therefore neither dissolve in water (because of their hydrophobic tails) nor remain completely separate, like olive oil (because then the head group could not associate with water). Phospholipid molecules therefore spontaneously form lipid bilayers between 5 and 10 nm thick (Fig. 2.19c) in which each part of the molecule is in its preferred environment. Cell membranes are lipid bilayers plus some added protein.\\n\\n|Medical Relevance|The Kinks Have It: Double Bonds, Membrane Fluidity, and Evening Primroses|\\n|---|---|\\n|2.1|One of the fatty acids commonly found as a component of animal fat is stearic acid, illustrated as (a) opposite. Its 18 carbon atoms are joined by single bonds, making a long straight molecule. In contrast, oleic acid (b) has a double bond between the ninth and tenth carbons. This introduces a kink in the chain. A fatty acid with kinks is less able to solidify because it is less able to pack in a regular fashion. The more double bonds in a fatty acid, the lower its melting point. Thus stearic acid melts at 69.6°C, while oleic acid melts at 13.4°C and is therefore liquid at room temperature. Fatty acids containing double bonds between the carbon atoms are said to be unsaturated. Polyunsaturated fatty acids have more than one double bond, more than one bend, and therefore even lower melting points. Linoleic acid (c) has two, and melts at -9°C; linolenic acid (d) has three and a melting point of -17°C.|\\n\\nFor the triacylglycerols inside our cells, and the phospholipids in our membranes, the same rule applies—the more double bonds in the acyl groups, the lower the melting point. The large masses of triacylglycerols containing stearic acid in animal bodies are liquid at body temperature but form solid lumps of fat at room temperature, while trioleoylglycerol is liquid at room temperature (but should not be kept in the fridge!). Membranes must not solidify—if they did, then they would crack and the cell contents would leak out each time the cell was flexed. Unsaturated fatty acids play an essential role in maintaining membrane liquidity. Mammals are unable to introduce double bonds beyond carbon 9 in the fatty acid chain. This means that linoleic and linolenic acids must be present in the diet. They are known as essential fatty acids. Fortunately, the biochemical abilities of plants are not so restricted, and plant oils form a valuable source of unsaturated fatty acids. The normal form of linolenic acid, shown in (d), is α-linolenic acid, which has its double bonds between carbons 9 and 10, 12 and 13, and 15 and 16. Some plant seed oils contain an isomer of linolenic acid with double bonds between carbons 6 and 7, 9 and 10, and 12 and 13. This is called γ-linolenic acid (e). The attractive, yellow-flowered garden plant called the evening primrose (Oenethera perennis) has seeds that contain an oil with γ-linolenic in its triacyglycerols. The 6 to 7 double bond introduces a kink closer to the glycerol. This is thought to increase fluidity when incorporated into membrane lipids. No one really knows if it really has the marvelous health effects that some claim for it, or if it does, how it works. This ignorance has not stopped people from making lots of money from evening primrose oil. It is included in cosmetics and in alternative medicines for internal and external application. γ-Linolenic acid occurs in other plants—the seeds of borage (Borago officinalis) are one of the richest sources. It is even found in some fungi. However, these lack the romance of the evening primrose!\\n---\\n# LIPIDS\\n\\n|Stearic acid, C18 no double bonds, melts at 69.6°|(a)|\\n|---|---|\\n|Oleic acid, C18 one double bond, melts at 13.4°|(b)|\\n|Linoleic acid, C18 two double bonds, melts at -9°|(c)|\\n|Linolenic acid, C18 three double bonds, melts at -17°|(d)|\\n|γ-Linolenic acid, C18 three double bonds|(e)|\\n---\\n# FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\nHYDROLYSIS\\n\\nMany of the macromolecules of which cells are made are generated from their individual building blocks by the removal of the elements of water. Equally, macromolecules can be broken into their individual building blocks by hydrolysis—breakage by the addition of water. Figure 2.20 shows four examples. Lactose is hydrolyzed to the monosaccharides galactose and glucose by the addition of one water molecule. Next, we show a dipeptide being broken into individual amino acids by hydrolysis. These two hydrolysis reactions occur all the time in our intestines, though some people lose the ability to hydrolyze lactose as they get older: they are lactose intolerant. Third, we show a dimeric (formed of two parts) inorganic phosphate ion called pyrophosphate being hydrolyzed to regular phosphate ions. The hydrolysis of pyrophosphate is catalyzed by enzymes found throughout the body, both inside cells and out. Later in the book we will meet a number of instances where a reaction creates\\n\\n6CH OH 6 lactose H 5C 2 O H CH OH 4C H C 1 2 OH H OH 5C O O C 3 C 2 OH 4C H C 1 H OH OH H CH3 H C 3 C 2 H CH CH 3 OH H OH CH2 H CH 2 H N CH CC N CH COO pyrophosphate H O 3 O O 2 O O P O P O H O OH O\\n6 CH OH 2 H O H C5 O H 2 4 C H C 1 OH OH H CH 2 HO C3 C2 OH OOC CH NH 3 6 H glucose OH serine O CH2OH O P OH OH C5 O OH CH3 O O 4C H C 1 CH CH 3 OH H HO P O H C3 C2 H CH2 O H OH OOC CH NH 3 H galactose leucine (a) (b) (c)\\n\\nFigure 2.20. Hydrolysis is the breakage of covalent bonds by the addition of the elements of water.\\n---\\nCH3\\n\\n|4H O|CH3|N|CH3|\\n|---|---|---|---|\\n|2| |CH2| |\\n| | |CH2| |\\n|O| | | |\\n| | |O|P|O|\\n| | |H| |\\n| | |H|C|\\n|O| |C|O|\\n|H| |C|H|H|C|H|C|O|\\n|O|H|C|H|H|C|H|H|C|H|C|O|\\n| | |H|C|H|H|C|H|H|C|H|\\n| | |H|C|H|H|C|H|H|C|H|\\n|H| |C|H|H|C|H|H|C|H|C|O|\\n|O|H|C|H|H|C|H|H|C|H|\\n|H| |C|O|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H|C|H|\\n| | |H| |\\n|stearate| |CH3|arachidonate|\\n| | |CH3|N|CH3|\\n| | |CH2|OH|O|\\n|CH|OH|O|P|OH|\\n| | |CH2|OH|O|\\n|glycerol|(d)|choline| |\\n\\nFigure 2.20. Continued\\n\\n45\\n---\\n# FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\npyrophosphate, but as soon as the pyrophosphate is produced it will be hydrolyzed to regular phosphate. Lastly, we show the complete hydrolysis of the phospholipid phosphatidylcholine. Five molecules result: one phosphate ion together with choline, glycerol, and two fatty acids. This hydrolysis also occurs in our intestines, where it is a multistage process. Hydrolysis reactions usually proceed rapidly in living tissue when the appropriate catalyst is present, because water is present at high concentration. In contrast the generation of macromolecules from their constituent building blocks, often by the elimination of the elements of water between constituent building blocks, usually requires the expenditure of energy by the cell. We will see how these reactions are performed in later chapters.\\n\\n# SUMMARY\\n\\n1. When two atoms interact, electrons may be shared between the two to form a covalent bond, or may pass completely from one atom to another, forming ions.\\n2. In water the electrons are not shared equally but are displaced toward the oxygen atom, so that the molecule is polar.\\n3. Ionic compounds will only dissolve in polar solvents.\\n4. Acids are molecules that give an H+ to water, forming H3O+. Dissolving an acid in water produces a solution of pH less than 7.\\n5. Bases are molecules that accept an H+ from water, leaving OH-. Dissolving a base in water produces a solution of pH greater than 7.\\n6. Pure water has the neutral pH of 7.0.\\n7. A hydrogen bond can form when a hydrogen atom takes up a position between two electron-grabbing atoms (oxygen, nitrogen, or sulfur), the three forming a straight line.\\n8. Monosaccharides are compounds with a central skeleton of carbon to which are attached many OH groups. They can switch between two oxygen-containing ring structures via an open-chain configuration.\\n9. Monosaccharides can join together to form disaccharides such as lactose and to form long chains such as glycogen and cellulose.\\n10. Adenine, guanine, hypoxanthine, cytosine, thymine, uracil, and nicotinamide are nitrogen-rich ring-shaped molecules called bases. Bases combine with ribose or deoxyribose to form nucleosides. Phosphorylation of a nucleoside on the 5\\' carbon produces a nucleotide.\\n11. Amino acids are compounds with a —COOH acidic group and an —NH2 basic group. At neutral pH these groups will, respectively, lose and gain an H+ to become —COO- and —NH3+.\\n12. Polypeptides are polymers of α-amino acids linked by peptide bonds. The sequence of amino acids is determined by instructions on the cell’s DNA. Proteins are polypeptides that fold into a specific shape.\\n---\\n# REVIEW QUESTIONS\\n\\n13. Lipids are formed by the attachment of hydrophobic long-chain fatty acids to a glycerol backbone. Phospholipids also have a hydrophilic head group and spontaneously form lipid bilayers.\\n\\n14. Hydrolysis is the breakage of covalent bonds by the addition of the elements of water.\\n\\n# FURTHER READING\\n\\nVoet, D., and Voet, J. 2003. Biochemistry. New York: Wiley.\\n\\n# REVIEW QUESTIONS\\n\\nFor each question, choose the ONE BEST answer or completion.\\n\\n|1. Water| |\\n|---|---|\\n|A. can give up an H+, becoming OH-.| |\\n|B. can accept an H+, becoming H3O+.| |\\n|C. can form hydrogen bonds.| |\\n|D. is a good solute for ionic substances.| |\\n|E. all of the above.| |\\n\\n|2. The amino acid leucine has an acidic COOH group and a basic NH2 group. The reaction| |\\n|---|---|\\n|—COOH → —COO- + H+ has a pKa of 2.3, while the reaction —NH3+ → —NH2 + H+ has a pKa of 9.7. At pH 1.0| |\\n|A. there will be more —OOC—CH(CH3)2)—NH2|than either —OOC—CH(CH3)2)—NH3+ or HOOC—CH(CH2CH(CH3)2)—NH3+.|\\n|B. there will be equal amounts of —OOC—CH(CH2CH(CH3)2)—NH2|and —OOC—CH(CH3)2)—NH3+, but not much HOOC—CH(CH2CH(CH3)2)—NH3+.|\\n|C. there will be more —OOC—CH(CH3)2)—NH3+ than either|HOOC—CH(CH2CH(CH3)2)—NH3+ or —OOC—CH(CH2CH(CH3)2)—NH2.|\\n|D. there will be equal amounts of HOOC—CH(CH2CH(CH3)2)—NH3+ and|—OOC—CH(CH2CH(CH3)2)—NH3+, but not much —OOC—CH(CH2CH(CH3)2)—NH2.|\\n|E. there will be more HOOC—CH(CH2CH(CH3)2)—NH3+ than either|—OOC—CH(CH2CH(CH3)2)—NH3+ or —OOC—CH(CH2CH(CH3)2)—NH2.|\\n\\n|3. Which of the following statements is incorrect?| |\\n|---|---|\\n|A. Glycogen is formed from units that include glucose.| |\\n|B. Cellulose is formed from units that include glucose.| |\\n|C. Lactose is formed from units that include ribose.| |\\n|D. Adenosine is formed from units that include ribose.| |\\n|E. Nicotinamide adenine dinucleotide is formed from units that include ribose.| |\\n\\n|4. Phospholipids| |\\n|---|---|\\n|A. have hydrophilic fatty acid tails.| |\\n|B. have charged, hydrophobic head groups.| |\\n---\\n# FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n\\n5. When an acid is added to pure water\\n\\n- A. it will dissolve easily.\\n- B. it will give up an H+ to form a negatively charged ion.\\n- C. it will form a solution with a high concentration of H3O+ ions.\\n- D. it will form a solution with a low pH.\\n- E. all of the above.\\n\\n6. Hydrogen bonds cannot form between\\n\\n- A. water and glucose.\\n- B. water and water.\\n- C. water and phosphate.\\n- D. phosphate and octane.\\n- E. phosphate and glucose.\\n\\n7. Which of these statements about the polypeptides in cells is wrong?\\n\\n- A. They are a polymer of γ -amino acids linked by peptide bonds.\\n- B. The sequence of amino acids is determined by instructions on the cell’s DNA.\\n- C. They have an —NH3+ group at one end of the polymer.\\n- D. They have a —COO− group at one end of the polymer.\\n- E. They are called proteins if they fold into a specific shape.\\n\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n1. E. All these statements about water are true.\\n\\n2. E. A pH of 1.0 is very acid (an H+ concentration of 100 mmol liter−1), more acid even than 2.3, the pH at which the COOH group is half-protonated. Both the —COOH and NH2 groups on leucine are therefore protonated at pH 1.0. If you are still confused, consider the situation at pH 11. At this pH, the concentration of H+ is extremely low (10−11 mol liter−1 or 10 pmol liter−1). Leucine therefore loses all its hydrogen ions and has the form —OOC—CH(CH2.CH(CH3)2)—NH2. Now consider adding H+ so that the pH falls. The NH2 group is basic and accepts H+ readily, even when H+ is present at quite a low concentration. At pH 9.7 (an H+ concentration of 200 pmol liter−1) half the NH2 groups have accepted an H+ to become NH3+, and at pH values less than 9.7 a majority of the NH2 groups are protonated. This is the situation at neutral pH: the NH2 groups are protonated while the COOH groups are deprotonated, so leucine has the form —OOC—CH(CH2.CH(CH3)2)—NH3+. If we keep adding H+, we eventually approach a pH of 2.3, where the H+ concentration has the high value of 5 mmol liter−1. Even though the COOH groups like to give up H+, they are forced to accept H+ when the concentration of H+ in solution gets this high. At pH 2.3 half the COOH groups are protonated; half are still COO−. At a pH less than 2.3, that is, at H+ concentrations higher than 5 mmol liter−1, the majority of the COOH groups are protonated and leucine has the form HOOC—CH(CH2.CH(CH3)2)—NH3+.\\n\\n3. C. Lactose is a disaccharide formed from glucose and galactose.\\n\\n4. C. All the other statements are false. In particular: (A) The fatty acid tails are hydrophobic. (B) The head groups are hydrophilic because they are polar and often are charged. (D) Phospholipids will not form droplets because this would place the hydrophilic head group in a hydrophobic environment.\\n---\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n|5.|E. All these statements are true. In particular, the definition of pH means that a solution with a high concentration of H3O+ ions is one with a low pH.|\\n|---|---|\\n|6.|D. Octane is formed of carbon and hydrogen only. It does not contain any of the electron-grabbing atoms that, together with hydrogen, form a hydrogen bond. Octane therefore cannot form hydrogen bonds with other chemicals.|\\n|7.|A. Cellular polypeptides are polymers of α-amino acids, not γ -amino acids. Changing γ to α in this question generates an accurate summary of the biology of polypeptides.|\\n---\\n# MEMBRANES AND ORGANELLES\\n\\nThe vast majority of the reactions that cells carry out take place in water. Eukaryotic cells are, at any one time, carrying out an enormous range of such chemical manipulations: collectively, these reactions are called metabolism (Chapter 13). In much the same way that our homes are divided into rooms that are adapted for particular activities, so eukaryotic cells contain distinct compartments or organelles to house specific functions. The term organelle is used rather loosely. At one extreme some scientists use it to mean any distinct cellular structure that has a more or less well-defined job to do; at the other are scientists who would reserve the name organelle for those cellular compartments that contain their own DNA and have some limited genetic autonomy. In this book, we will define organelles as those cellular components whose limits, like those of the cell itself, are defined by membranes. It is first necessary to consider some of the fundamental properties of cell membranes.\\n\\n# BASIC PROPERTIES OF CELL MEMBRANES\\n\\nIt is difficult to overstate the importance of membranes to living cells; without them life could not exist. The plasma membrane, also known as the cell surface membrane or plasmalemma, defines the boundary of the cell. It regulates the movement of materials into and out of the cell and facilitates electrical signaling between cells. Other membranes define the boundaries of organelles and provide a matrix upon which complex chemical reactions can occur. Some of these themes will be developed in subsequent chapters. In the following section the basic structure of the cell membrane will be outlined.\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C 2004 by John Wiley & Sons, Inc.\\n---\\nMEMBRANES AND ORGANELLES\\n\\n|extracellular medium|\\n|---|\\n|sugar residues|\\n|hydrophobic lipid tails|hydrophilic lipid headgroups|\\n|lipid bilayer|\\n|integral membrane protein|\\n|cytosol|peripheral membrane protein|\\n\\nFigure 3.1. Membranes comprise a lipid bilayer plus integral and peripheral proteins.\\n\\nThe basic structure of a biological membrane is shown in Figure 3.1. Approximately half the mass is phospholipid, which spontaneously organizes to form a lipid bilayer (page 42). All the membranes of the cell, including the plasma membrane, also contain proteins. These may be tightly associated with the membrane and extracted from it only with great difficulty, in which case they are called integral proteins (e.g., the gap junction channel, page 55); or they may be separated with relative ease, in which case they are termed peripheral\\n\\n|O2|CO2|NO|H2O|Na+|K+|Ca++|Glucose|\\n|---|---|---|---|---|---|---|---|\\n|Steroid hormones| | | | | | | |\\n\\nFigure 3.2. Small uncharged molecules can pass through membranes by simple diffusion, but ions cannot.\\n---\\n# BASIC PROPERTIES OF CELL MEMBRANES\\n\\nproteins (e.g., clathrin adaptor protein, page 229). Membrane proteins are free to move laterally, within the plane of the membrane. Integral membrane proteins are often glycosylated—that is, have sugar residues attached—on the side facing the extracellular medium.\\n\\nStraight Through the Membrane: Diffusion Through the Bilayer Molecules of oxygen are uncharged. Although they dissolve readily enough in water, they are also able to dissolve in the hydrophobic interior of lipid bilayers. Oxygen molecules can therefore pass from the extracellular fluid into the interior of the plasma membrane, and from there pass on into the cytoplasm, in a simple diffusion process (Fig. 3.2). Three other small molecules with important roles in biology—carbon dioxide, nitric oxide, and water itself—also pass across the plasma membrane by simple diffusion, as do the uncharged hormones of the steroid family. In contrast, charged ions cannot dissolve in hydrophobic regions (page 21) and therefore cannot cross membranes by simple diffusion.\\n\\n|✄|▯|\\n|---|---|\\n|✄✄|▯▯|\\n\\nExample 3.1 Rapid Diffusion in the Lungs Blood is composed of cells, red and white, and plasma, a solution of sodium chloride, other ions, various organic molecules and proteins. Red blood cells are very simple. They have no nucleus and their plasma membrane encloses a cytosol packed with the oxygen-carrying protein hemoglobin, bathed in a salt solution. The sodium concentration in the red blood cytosol is much lower than the sodium concentration in plasma. It is important that the sodium concentration in the cytosol remain low: if it increases, the red blood cells will swell and then burst as water rushes in. As red blood cells pass through the lungs, they quickly gain oxygen because oxygen molecules can pass rapidly across the plasma membrane by simple diffusion. The cells do not, however, gain sodium ions from the plasma as they pass around the body because the lipid bilayer is impermeable to ions. Red blood cells therefore remain the right size to pass though even the tiniest blood vessels in our bodies.\\n\\nBeyond the Cell Membrane: The Extracellular Matrix In connective tissue (page 13), the spaces between cells are filled by an extracellular matrix consisting of polysaccharides and proteins such as collagen together with combinations of the two called proteoglycans. The proteins provide tensile strength and elasticity while the polysaccharides form a hydrated gel that expands to fill the extracellular space. The sugar-based extracellular matrix reaches its highest form of expression in the cell walls of plants (Fig. 3.3). The plant cell wall consists primarily of cellulose microfibrils linked together by other polysaccharide molecules such as hemicellulose and pectin. The thickness of the wall is determined largely by its pectin content. In healthy plant cells the plasma membrane is constantly pressing against the cell wall. In plants the extracellular medium is a much more dilute solution than the intracellular solution. If no other forces operated, water would diffuse into plant cells down its concentration gradient, a process called osmosis. However, plant cells are prevented from expanding by the inextensible cellulose microfibrils in their cell walls. As water diffuses in, the hydrostatic pressure in the cells rises, making the combination of cell plus cell wall stiff. The same effect can be seen when a bicycle inner tube is blown up inside the tire, and is called turgor. If plants are starved of water, water will leave the cell interior, hydrostatic pressure drops, and the cells lose their turgor pressure, like a tire when the tube inside is deflated. As a result the nonwoody parts of the plant wilt.\\n---\\n54\\n\\nMEMBRANES AND ORGANELLES\\n\\n| | |vacuole|\\n|---|---|---|\\n| |direction of expansion|nucleus|\\n| | |primary cell wall|\\n| | |secondary cell wall|\\n|Primary cell wall| | |\\n| |cellulose microfibril| |\\n\\nFigure 3.3. The cell wall directs plant cell growth.\\n\\nNewly generated plant cells first lay down a primary cell wall of cellulose microfibrils that are orientated in one direction around the plant cell (Fig. 3.3a). Hydrostatic pressure therefore tends to cause a growing plant cell to elongate in one direction, perpendicular to the axis of the microfibrils. Once the growth of the cell is complete, more layers of cellulose and/or other compounds are added, most notably the polyphenolic compound lignin, to form the secondary cell wall (Fig. 3.3b).\\n\\nCell Junctions\\n\\nIn multicellular organisms, and particularly in epithelia, it is often necessary for neighboring cells within a tissue to be connected together. This function is provided by cell junctions. In animal cells there are three types of junctions. Those that form a tight seal between adjacent cells are known as tight junctions; those that allow communication between cells are known as gap junctions. A third class of cell junction that anchors cells together, allowing the tissue to be stretched without tearing, are called anchoring junctions. Plant cells do\\n---\\n# BASIC PROPERTIES OF CELL MEMBRANES\\n\\nCells may not have tight junctions, gap junctions, or anchoring junctions but do contain a unique class of communicating junction known as plasmodesmata.\\n\\nTight junctions are found wherever flow of extracellular medium is to be restricted and are particularly common in epithelial cells such as those lining the small intestine. The plasma membranes of adjacent cells are pressed together so tightly that no intercellular space exists between them. Tight junctions between the epithelial cells of the intestine ensure that the only way that molecules can get from the lumen of the intestine to the blood supply that lies beneath is by passing through the cells, a route that can be selective.\\n\\nGap junctions are specialized structures that allow cell-to-cell communication in animals. When two cells form a gap junction, ions and small molecules can pass directly from the cytosol of one cell to the cytosol of the other cell without going into the extracellular fluid. Since ions can move through the junction, changes in electrical voltage\\n\\n|Gap junction|cytosol of right hand cell|\\n|---|---|\\n|hydrophilic solute| |\\n|cytosol of left hand cell| |\\n|||\\n\\nFigure 3.4. Gap junctions allow solute and electrical current to pass from the cytosol of one cell to the cytosol of its neighbor.\\n---\\nMEMBRANES AND ORGANELLES\\n\\nare also rapidly transmitted from cell to cell by this route. The structure that makes this possible is the gap junction channel. Channels, as we will see in Chapter 14, are water-filled holes through membranes. When two gap junction channels or connexons meet, they form a water-filled tube that runs all the way through the plasma membrane of the first cell, across the small gap between the cells, and through the plasma membrane of the second cell. In the middle of the channel is a continuous hole about 1.5 nm in diameter. This hole is large enough to allow small ions through (and therefore to pass electrical current) together with amino acids and nucleotides, but it is too small for proteins or nucleic acids. Gap junctions are especially important in the heart, where they allow an electrical signal to pass rapidly between all the cardiac muscle cells, ensuring that they all contract at the proper time. Each gap junction channel is composed of six protein subunits that can twist against each other to open and close the central channel in a process called gating (page 314) that allows the cell to control the degree to which it shares solute with its neighbor. The plasmodesmata that perforate the cell walls of many plant tissues (Fig. 1.7 on page 10) serve much the same purpose as the gap junctions of animal cells but are much bigger and cannot shut quickly. Some plant viruses use plasmodesmata to spread from cell to cell.\\n\\nExample 3.2 Gap Junctions Keep Eggs Ready But Waiting\\n\\nIn the days leading up to ovulation, oocytes (cells that undergo meiosis to give rise to eggs) that are to be released from the follicle are kept in a state of suspended development by a chemical in their cytosol called cyclic AMP (page 350). The oocytes themselves do not make cAMP. Rather, the follicle cells that surround them in the ovary make cAMP, which then passes through gap junctions into the oocyte. Only when the oocyte is released and begins its passage down the Fallopian tube does it prepare for fertilization. In particular, the process of meiosis (page 404) is completed so as to remove half the chromosomes, so that when the sperm adds its complement of chromosomes the fertilized egg will have the correct number.\\n\\nAnchoring junctions bind cells tightly together and are found in tissues such as the skin and heart that are subjected to mechanical stress. These junctions are described later (page 396).\\n\\nORGANELLES BOUNDED BY DOUBLE-MEMBRANE ENVELOPES\\n\\nThree of the major cell organelles, the nucleus, mitochondrion, and, in plant cells, the chloroplast, share two distinctive features. They are all enclosed within an envelope consisting of two parallel membranes and they all contain the genetic material DNA.\\n\\nThe Nucleus\\n\\nThe nucleus is often the most prominent cell organelle. It contains the genome, the cell’s database, which is encoded in molecules of the nucleic acid, DNA. The nucleus is bounded by a nuclear envelope composed of two membranes separated by an intermembrane space (Fig. 3.5). The inner membrane of the nuclear envelope is lined by a meshwork of proteins\\n---\\n# ORGANELLES BOUNDED BY DOUBLE-MEMBRANE ENVELOPES\\n\\n|ribosome|nucleolus|\\n|---|---|\\n|euchromatin|nuclear lamina|\\n|heterochromatin|rough endoplasmic reticulum|\\n|nuclear pore|ribosome|\\n|inner nuclear membrane| |\\n|intermembrane space| |\\n|outer nuclear membrane| |\\n\\nFigure 3.5. The nucleus and the relationship of its membranes to those of the endoplasmic reticulum.\\n\\ncalled the nuclear lamina which provides rigidity to the nucleus. A two-way traffic of proteins and nucleic acids between the nucleus and the cytoplasm passes through holes in the nuclear envelope called nuclear pores. The nucleus of a cell that is synthesizing proteins at a low level will have few nuclear pores. In cells that are undergoing active protein synthesis, however, virtually the whole nuclear surface is perforated.\\n\\nWithin the nucleus it is usually possible to recognize discrete areas. Most of it is occupied by chromatin, a complex of DNA and certain DNA-binding proteins such as histones (page 73). In most cells it is possible to recognize two types of chromatin; lightly staining euchromatin is that portion of the cell’s DNA that is being actively transcribed into RNA; heterochromatin, on the other hand, represents the inactive portion of the genome where no RNA synthesis is occurring. The DNA in heterochromatin is tightly coiled then supercoiled (see Fig. 4.5 on page 72), leading to its dense appearance in both the light and electron microscopes.\\n\\nUnlike DNA, RNA is not confined within the nucleus and other organelles but is also found within the cytoplasm associated with particles called ribosomes whose function is to make proteins. Ribosomes are made in the nucleus, in specialized regions called nucleoli that form at specific sites on the DNA called nucleolar organizer regions. These contain blocks of genes that code for the ribosomal RNA.\\n\\nIt should be stressed that the appearance of the nucleus we have described thus far relates to the cell in interphase, the period between successive rounds of cell division. As the cell enters mitosis (Chapter 19) the organization of the nucleus changes dramatically.\\n---\\n# MEMBRANES AND ORGANELLES\\n\\nDNA becomes more and more tightly packed and is revealed as a number of separate rods called chromosomes—46 in human cells. The nucleolus disperses, and the nuclear envelope fragments. Upon completion of mitosis, these structural rearrangements are reversed and the nucleus resumes its typical interphase organization.\\n\\nMitochondria and Chloroplasts\\n\\nAlthough most of the genetic information of a eukaryotic cell resides in nuclear DNA, some of the information necessary to make both chloroplasts and mitochondria is stored within these organelles themselves. This information resides on small circular DNA molecules that are similar to the chromosomes of bacteria but very different from the long linear DNA molecules in the nucleus. This is strong evidence for the endosymbiotic theory of the origin of mitochondria and chloroplasts, which proposes that the small circular DNA molecules found in these organelles is all that is left of the chromosomes of the original symbiotic bacteria. Chloroplasts and mitochondria both contain ribosomes and synthesize a small subset of their own proteins, although the great majority of proteins that form chloroplasts and mitochondria are encoded by nuclear genes and synthesized in the cytoplasm. Perhaps the most distinctive feature of these organelles, however, is that the inner of their two membranes is markedly elaborated and folded to increase its surface area.\\n\\nMitochondria are among the most easily recognizable organelles due to the extensive folding of their inner membrane to form shelflike projections named cristae. The number of cristae, like the number of mitochondria themselves, depends upon the energy budget of the cell in which they are found. In striated muscle cells, which must contract and relax repeatedly over long periods of time, there are many mitochondria that contain numerous cristae; in fat cells, which generate little energy, there are few mitochondria and their cristae are less well developed. This gives a clue as to the function of mitochondria: they are the cell’s power stations. Mitochondria produce the molecule adenosine triphosphate (ATP), one of the cell’s energy currencies that provide the energy to drive a host of cellular reactions and mechanisms. The double-membrane structure provides four distinct domains: the outer membrane, the inner membrane, the intermembrane space, and the matrix. Each domain houses a distinct set of functions, details of which are given in Chapters 12 and 13.\\n\\nChloroplasts are found only in photosynthetic protists and plant cells. In addition to the two bounding membranes, they contain internal membranes called thylakoids which, in plants, form stacks called grana. The thylakoids contain the proteins and other molecules responsible for light capture. The dark reactions of photosynthesis, on the other hand, takes place in the matrix, called the stroma, which also contains the DNA and ribosomes.\\n\\n# ORGANELLES BOUNDED BY SINGLE-MEMBRANE ENVELOPES\\n\\nEukaryotic cells contain many sacs and tubes bounded by a single membrane. Although these are often rather similar in appearance, they can be subdivided into different types specialized to carry out distinct functions.\\n---\\nORGANELLES BOUNDED BY SINGLE-MEMBRANE ENVELOPES\\n\\n|ribosome|cristae|intermembrane space|\\n|---|---|---|\\n| |matrix|inner membrane|\\n| | |outer membrane|\\n\\nmitochondrion\\n\\nribosome\\n\\n|thylakoid| |\\n|---|---|\\n|granum|stroma|\\n|thylakoid| |\\n| |inner membrane|\\n| |intermembrane space|\\n| |outer membrane|\\n\\nchloroplast\\n\\nFigure 3.6. Mitochondrion and chloroplast.\\n\\nPeroxisomes\\nMitochondria and chloroplasts are frequently found close to another membrane-bound organelle, the peroxisome. In human cells peroxisomes have a diameter of about 500 nm, and their dense matrix contains a heterogeneous collection of proteins concerned with a variety of metabolic functions, some of which are only now beginning to be understood. Peroxisomes are so named because they are frequently responsible for the conversion of the highly reactive molecule hydrogen peroxide (H2O2), which is formed as a by-product\\n---\\n# MEMBRANES AND ORGANELLES\\n\\nof the reactions in the mitochondrion, into water:\\n\\n2H2O2 → 2H2O + O2\\n\\nThis reaction is carried out by a protein called catalase, which sometimes forms an obvious crystal within the peroxisome. Catalase is an enzyme—a protein catalyst that increases the rate of a chemical reaction (page 241). In fact, it was one of the first enzymes to be discovered. In humans, peroxisomes are primarily associated with lipid metabolism. Understanding peroxisome function is important for a number of inherited human diseases such as X-linked adrenoleukodystrophy where peroxisome malfunction and the consequent inability to metabolize lipid properly typically leads to death in childhood or early adulthood unless dietary lipid is extremely restricted.\\n\\n# IN DEPTH 3.1 Nobel Prizes for Organelle Researchers\\n\\nIn addition to Camillo Golgi’s Nobel prize in 1906, modern studies of cell organelles have also been recognized by the Nobel committee. George Palade, Albert Claude, and Christian de Duve shared the 1974 Nobel prize in physiology or medicine for their work on the identification and isolation of cell organelles. Günter Blobel, a former student of Palade’s, was awarded the same prize in 1999 for the discovery that proteins carry so-called targeting sequences that determine where in the cell they should reside (page 215).\\n\\n# Endoplasmic Reticulum\\n\\nThe endoplasmic reticulum is a network of membrane enclosed channels that run throughout the cell, forming a continuous network whose lumen (inside) is at all points separated from the cytosol by a single membrane. The membrane of the endoplasmic reticulum is continuous with the outer nuclear membrane (Fig. 3.5). Two regions can be recognized in most cells, known as smooth endoplasmic reticulum and rough endoplasmic reticulum (Fig. 1.2 on page 3). The basic difference is that the rough endoplasmic reticulum is covered in ribosomes, which give it its rough appearance in the electron microscope.\\n\\nThe function of the smooth endoplasmic reticulum varies from tissue to tissue. In the ovaries, testes, and the adrenal gland it is where steroid hormones are made; in the liver it is the site of detoxication of foreign chemicals including drugs. Probably the most universal role of the smooth endoplasmic reticulum is the storage and sudden release of calcium ions. Calcium ions are pumped from the cytosol into the lumen of the smooth endoplasmic reticulum to more than 100 times the concentration found in the cytosol. Many stimuli can cause this calcium to be released back into the cytosol, where it activates many cell processes (Chapter 16).\\n\\nThe rough endoplasmic reticulum is where the cells make the proteins that will end up as integral membrane proteins in the plasma membrane, and proteins that the cell will export to the extracellular medium (such as the proteins of the extracellular matrix, page 13).\\n\\nGolgi Apparatus\\n\\nThe Golgi apparatus, named after its discoverer, 1906 Nobel prize winner Camillo Golgi, is a distinctive stack of flattened sacks called cisternae. The Golgi apparatus is the distribution\\n---\\n# SUMMARY\\n\\npoint of the cell where proteins made within the rough endoplasmic reticulum are further processed and then directed to their final destination. Appropriately, given this central role, the Golgi apparatus is situated at the so-called cell center, a point immediately adjacent to the nucleus that is also occupied by a structure called the centrosome (Fig. 1.2 on page 3). The centrosome helps to organize the cytoskeleton (Chapter 18).\\n\\n# Lysosomes\\n\\nLysosomes are sometimes called “cell stomachs” because they contain enzymes that digest cellular components. They are particularly plentiful in cells that digest and destroy other cells, such as macrophages. Lysosomes are roughly spherical and usually 250–500 nm in diameter.\\n\\n# SUMMARY\\n\\n- Membranes are made of phospholipids and protein.\\n- Cells are bounded by membranes while cell functions are compartmentalized into membrane-bound organelles.\\n- Solutes with a significant solubility in hydrophobic solvents can pass across biological membranes by simple diffusion.\\n- The extracellular matrix is found on the outside of animal cells. A cellulose-based cell wall is found outside plant cells.\\n- Turgor pressure gives nonlignified plant tissue its stiffness.\\n- Tight junctions prevent the passage of extracellular water or solute between the cells of an epithelium.\\n- Gap junctions allow solute and electrical current to pass from the cytosol of one cell to the cytosol of its neighbor.\\n- The nucleus, mitochondrion, and chloroplast are bounded by double-membrane envelopes. In the case of the nucleus, this is perforated by nuclear pores. All three organelles contain DNA.\\n- Mitochondria produce the energy currency adenosine triphosphate (ATP).\\n- Chloroplasts capture light energy for use by plant cells.\\n- Peroxisomes carry out a number of reactions including the destruction of hydrogen peroxide.\\n- The endoplasmic reticulum is the site of much protein synthesis. Cell stimulation can often cause calcium ions stored in the endoplasmic reticulum to be released into the cytosol.\\n- The Golgi apparatus is concerned with the modification of proteins after they have been synthesized.\\n- Lysosomes contain powerful degradative enzymes.\\n---\\n# MEMBRANES AND ORGANELLES\\n\\n# REVIEW QUESTIONS\\n\\nFor each question, choose the ONE BEST answer or completion.\\n\\n|1. Biological membranes|E. all of the above.|\\n|---|---|\\n|2. The extracellular matrix|E. all of the above.|\\n|3. Which of the following statements about gap junctions is incorrect?|B. They are found in both plants and animals.|\\n|4. A cell with a high density of nuclear pores is likely to be|C. synthesizing lots of protein.|\\n|5. In diffusing from the mitochondrial matrix of one animal cell to the interior of the Golgi apparatus of another cell 50 μm away in the same animal a molecule of CO2 would cross|C. four membranes.|\\n|6. One organelle often found at the cell center is the|A. nucleus.|\\n|7. A stimulated cell will often release calcium ions from its|B. endoplasmic reticulum.|\\n---\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n|1.|E. All these statements about biological membranes are true.|\\n|---|---|\\n|2.|A. Collagen is a major constituent of the extracellular matrix. Considering the other answers: B and C are wrong because phospholipid is found in membranes while RNA is intracellular. Answer D is wrong because the extracellular matrix is an open meshwork through which the aqueous extracellular medium and its dissolved solutes can easily pass.|\\n|3.|B. Plants do not have gap junctions, indeed the presence of the cellulose cell wall means that even if the cells produced the protein subunits neighboring cells never approach each other closely enough to form gap junctions.|\\n|4.|C. Proteins are synthesized using the information on RNA that must be exported from the nucleus to the cytoplasm through nuclear pores. None of the other choices is correct. In particular, the nuclear envelope with all its nuclear pores breaks down in cells undergoing mitosis.|\\n|5.|D. The CO2 must diffuse across the inner mitochondrial membrane, the outer mitochondrial membrane, the plasma membrane of the first cell, the plasma membrane of the second cell, and the membrane of the Golgi apparatus.|\\n|6.|D. The cell center is occupied by the centrioles (which are not organelles because they are not membrane-bound) and by the Golgi apparatus.|\\n|7.|B. The vast majority of cells are able to release calcium ions from the endoplasmic reticulum when stimulated. The nuclear envelope, mitochondria, and lysosomes may be able to release calcium ions in certain more limited circumstances.|\\n|8.|A. Most ribosomes that are not free in the cytoplasm are found on the rough endoplasmic reticulum, but some are found on the outer nuclear membrane, which is continuous with the membrane of the endoplasmic reticulum (see Fig. 3.5). Mitochondria and chloroplasts have their own ribosomes, but these are found inside these organelles, not on the outer surface.|\\n---\\n# DNA STRUCTURE AND THE GENETIC CODE\\n\\nINTRODUCTION\\n\\nOur genes are made of deoxyribonucleic acid (DNA). This remarkable molecule contains all the information necessary to make a cell, and DNA is able to pass on this information when a cell divides. This chapter describes the structure and properties of DNA molecules, the way in which our DNA is packaged into chromosomes, and how the information stored within DNA is retrieved via the genetic code.\\n\\nTHE STRUCTURE OF DNA\\n\\nDeoxyribonucleic acid is an extremely long polymer made from units called deoxyribonucleotides, which are often simply called nucleotides. These nucleotides differ from those described in Chapter 2 in one respect: the sugar is deoxyribose, not ribose. Figure 4.1 shows one deoxyribonucleotide, deoxyadenosine triphosphate. Note that deoxyribose, unlike ribose (page 29), has no OH group on its 2\\' carbon. Four bases are found in DNA; they are the two purines adenine (A) and guanine (G) and the two pyrimidines cytosine (C) and thymine (T) (Fig. 4.2). The combined base and sugar is known as a nucleoside to distinguish it from the phosphorylated form, which is called a nucleotide. Four different nucleotides join to make DNA. They are 2\\'-deoxyadenosine-5\\'-triphosphate (dATP).\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C2004 by John Wiley & Sons, Inc.\\n---\\n# DNA STRUCTURE AND THE GENETIC CODE\\n\\nExample 4.1 Gift from the Dead\\n\\nDeoxyribonucleic acid was known to contain genetic information several years before its structure was finally determined. As early as 1928 Fred Griffith carried out the now famous pneumococcus transformation experiment. There are several strains of the bacterium Diplococcus pneumoniae. Some strains cause pneumonia and are said to be virulent; others do not and are nonvirulent. The virulent bacteria possess a polysaccharide coat and, when grown on an agar plate, create colonies with a smooth appearance—these are S bacteria. Nonvirulent bacteria do not have this polysaccharide coat, and their colonies are rough in appearance—these are R bacteria. Mice injected with S bacteria developed pneumonia and died, whereas those injected with R bacteria were unaffected. When S bacteria were killed by heat treatment before injection, the mice remained healthy. However, those mice injected with a mixture of live R bacteria and heat-killed S bacteria died of pneumonia. This observation meant that something in the heat-killed S bacteria carried the information that enabled bacteria to make the polysaccharide coat and therefore to change—transform—the R bacteria into a virulent strain. Later experiments in the mid-1940s by Oswald Avery, Maclyn McCarty, and Colin MacLeod clearly demonstrated that the transforming factor was DNA. They made extracts of S bacteria and treated them with enzymes that destroy either DNA, RNA, or protein. These extracts were then mixed with R bacteria. Only the extract rich in DNA was able to transform R bacteria into S bacteria.\\n\\n|S|S|S|mouse injected with S bacteria dies|\\n|---|---|---|---|\\n|R|R|R|mouse injected with R bacteria stays healthy|\\n|S|S|S|mouse injected with heat-killed S bacteria stays healthy|\\n|R|S|R|mouse injected with live R bacteria + heat-killed S bacteria dies|\\n\\na deoxynucleoside triphosphate (dATP)\\n---\\n# THE STRUCTURE OF DNA\\n\\n|purines|pyrimidines|\\n|---|---|\\n|H|H|\\n|H|N C C N C H|\\n| |N C N C N H|\\n| |H|\\n|adenine (A)|cytosine (C)|\\n|O C C N C H|H C N C N H|\\n|H N C N C N H|H C C C N H|\\n|H N|N C|\\n| |H|\\n|guanine (G)|thymine (T)|\\n\\nFigure 4.2. The Four bases found in DNA.\\n\\n2-deoxyguanosine-5-triphosphate (dGTP), 2-deoxycytidine-5-triphosphate (dCTP), and\\n2-deoxythymidine-5-triphosphate (dTTP).\\nDNA molecules are very large. The single chromosome of the bacterium Escherichia coli is made up of two strands of DNA that are hydrogen-bonded together to form a single circular molecule comprising 9 million nucleotides. Humans have 46 DNA molecules in each cell, each forming one chromosome. We inherit 23 chromosomes from each parent. Each set of 23 chromosomes encodes a complete copy of our genome and is made up of 6 × 10^9 nucleotides (or 3 × 10^9 base pairs—see below). We do not yet know the exact number of genes that encode messenger RNA and therefore proteins in the human genome. The current estimate is in the range of 30,000. Table 4.1 compares the number of predicted messenger RNA genes in the genomes of different organisms. In each organism, there are also approximately 100 genes that code for ribosomal RNAs and transfer RNAs. The role these three types of RNA play in protein synthesis is described in Chapter 8.\\nFigure 4.3 illustrates the structure of the DNA chain. As nucleotides are added to the chain by the enzyme DNA polymerase (Chapter 5), they lose two phosphate groups. The last (the α phosphate) remains and forms a phosphodiester link between successive deoxyribose residues. The bond forms between the hydroxyl group on the 3\\' carbon of the deoxyribose of one nucleotide and the α-phosphate group attached to the 5\\' carbon of the next nucleotide. Adjacent nucleotides are hence joined by a 3\\'–5\\' phosphodiester link. The linkage gives rise\\n---\\n# Table 4.1. Numbers of Predicted Genes in Various Organisms\\n\\n|Organism|Number of Predicted Genes|\\n|---|---|\\n|Bacterium—Haemophilus influenzae|1,709|\\n|Yeast—Saccharomyces cerevisiae|6,241|\\n|Fruit fly—Drosophila melannogaster|13,601|\\n|Worm—Caenorhabdites elegans|18,424|\\n|Plant—Arabidopsis thaliana|25,498|\\n|Human—Homo sapiens|∼30,000|\\n\\nTo the sugar–phosphate backbone of a DNA molecule. A DNA chain has polarity because its two ends are different. In the first nucleotide in the chain, the 5′ carbon of the deoxyribose is phosphorylated but otherwise free. This is called the 5′ end of the DNA chain. At the other end is a deoxyribose with a free hydroxyl group on its 3′ carbon. This is called the 3′ end.\\n\\nThe DNA Molecule Is a Double Helix. In 1953 Rosalind Franklin used X-ray diffraction to show that DNA was a helical (i.e., twisted) polymer. James Watson and Francis Crick demonstrated, by building three-dimensional models, that the molecule is a double helix.\\n\\n# IN DEPTH 4.1 Gene Number and Complexity\\n\\nAs the genomes of more and more organisms were sequenced, the most surprising feature to emerge was just how few genes supposedly “complex” organisms possess. The first eukaryotic genome to be sequenced was that of the lowly budding yeast, Saccharomyces cerevisiae, the simple unicellular fungus that we use to make bread and beer. S. cerevisiae has about 6000 genes. The fruit fly, Drosophila melanogaster, a much more complex organism with a brain, nervous and digestive systems, and the ability to fly and navigate, on the other hand, has 13,600 genes, or roughly twice the number in a yeast. Even more surprising was the case of the human genome. Prior to the completion of the Human Genome Project predictions of the number of human genes were in the order of 100,000. Surely this complex vertebrate that could send a spaceship to Mars and write War and Peace would need vastly more genes than the fruit fly. In the event, the number of human genes turned out to be much lower than expected, about 30,000, only twice as many genes as in Drosophila.\\n\\nIf complex biological and social achievements are not the result of having more genes, where do they come from? Two factors help the human genome generate a more complex organism. First, having more DNA per gene means that more DNA can be used in enhancer sequences (page 120), allowing more subtle control of where, when, and to what extent a gene is expressed. Second, there is not a straightforward one-to-one relationship between genes and proteins. Alternative splicing (page 118) allows the cell to “cut and paste” a messenger RNA molecule in different ways to produce many different proteins from the same gene. Estimates are that something like 50% of human genes show alternative splicing with the pattern of splicing (the range of proteins produced) varying from tissue to tissue. Drosophila genes also show alternative splicing but those of yeast, which contain few introns (page 98), do not.\\n---\\n# THE STRUCTURE OF DNA\\n\\nsugar–phosphate backbones lie on the outside of the molecule, and the purines and pyrimidines lie on the inside of the molecule. There is just enough space for one purine and one pyrimidine in the center of the double helix. The Watson–Crick model showed that the purine guanine (G) would fit nicely with the pyrimidine cytosine (C), forming three hydrogen bonds. The purine adenine (A) would fit nicely with the pyrimidine thymine (T), forming two hydrogen bonds. Thus A always pairs with T, and G always pairs with C. The three hydrogen bonds formed between G and C produce a relatively strong base pair. Because only two hydrogen bonds are formed between A and T, this weaker base pair is more easily broken. The difference in strengths between a G–C and an A–T base pair is important in the initiation and termination of RNA synthesis (page 108). The two chains of DNA are said to be antiparallel because they lie in the opposite orientation with respect to one another, with the 3\\'-hydroxyl terminus of one strand opposite the 5\\'-phosphate terminus of the second strand. The sugar–phosphate backbones do not completely conceal the bases inside. There are two grooves along the surface of the DNA molecule. One is wide and deep—the major groove—and the other is narrow and shallow—the minor groove (Fig. 4.4). Proteins can use the grooves to gain access to the bases (page 123).\\n\\n# The Two DNA Chains Are Complementary\\n\\nA consequence of the base pairs formed between the two strands of DNA is that if the base sequence of one strand is known, then that of its partner can be inferred. A G in one strand\\n---\\n70\\n\\n|cytosine-guanine|H|hydrogen bond|\\n|---|---|---|\\n|base pair (C G)|H C C N H O C C N C H| |\\n| |H C N C N H N C N C N deoxyribose| |\\n| |deoxyribose O H N| |\\n| | | |\\n| |cytosine (C)|H guanine (G)|\\n| | | |\\n|thymine-adenine|H H C H O H N H N C H| |\\n|base pair (T A)|H C C C N H N C C C N deoxyribose| |\\n| |deoxyribose N C O H C N| |\\n| | | |\\n| |thymine (T)|adenine (A)|\\n| | | |\\n|schematic| | |\\n|representation of| | |\\n|the double-helical| | |\\n|structure of DNA| | |\\n| | |C G|\\n| |A CT G AG T| |\\n|minor groove| | |\\n| |T A sugar-phosphate| |\\n| | |backbone| |\\n| | | |\\n| |T TA A base pairs| |\\n|major groove| | |\\n| |C G G| |\\n| | | |\\n| |A T| |\\n| | | |\\n|Figure 4.4. The DNA double helix is held together by hydrogen bonds.| | |\\n---\\n# PACKAGING OF DNA MOLECULES INTO CHROMOSOMES\\n\\nwill always be paired with a C in the other. Similarly an A will always pair with a T. The two strands are therefore said to be complementary.\\n\\nExample 4.2\\nErwin Chargaff’s Puzzling Data\\n\\nIn a key discovery of the 1950s, Erwin Chargaff analyzed the purine and pyrimidine content of DNA isolated from many different organisms and found that the amounts of A and T were always the same, as were the amounts of G and C. Such an identity was inexplicable at the time, but helped James Watson and Francis Crick build their double-helix model in which every A on one strand of the DNA helix has a matching T on the other strand, and every G on one strand has a matching C on the other.\\n\\nDifferent Forms of DNA\\n\\nThe original Watson–Crick model of DNA is now called the B-form. In this form, the two strands of DNA form a right-handed helix. If viewed from either end, it turns in a clockwise direction. B-DNA is the predominant form in which DNA is found. Our genome, however, also contains several variations of the B-form double helix. One of these, Z-DNA, so-called because its backbone has a zig-zag shape, forms a left-handed helix and occurs when the DNA sequence is made of alternating purines and pyrimidines. Thus the structure adopted by DNA is a function of its base sequence.\\n\\nDNA AS THE GENETIC MATERIAL\\n\\nDeoxyribonucleic acid carries the genetic information encoded in the sequence of the four bases—adenine, guanine, cytosine, and thymine. The information in DNA is transferred to its daughter molecules through replication (the duplication of DNA molecules) and subsequent cell division. DNA directs the synthesis of proteins through the intermediary molecule RNA. The DNA code is transferred to RNA by a process known as transcription (Chapter 6). The RNA code is then translated into a sequence of amino acids during protein synthesis (Chapter 8). This is the central dogma of molecular biology: DNA makes RNA makes protein.\\n\\nRetroviruses such as human immunodeficiency virus, the cause of AIDS, are an exception to this rule. As their name suggests, they reverse the normal order of data transfer. Inside the virus coat is a molecule of RNA plus an enzyme that can make DNA from an RNA template by the process known as reverse transcription.\\n\\nPACKAGING OF DNA MOLECULES INTO CHROMOSOMES\\n\\nEukaryotic Chromosomes and Chromatin Structure\\n\\nA human cell contains 46 chromosomes (23 pairs), each of which is a single DNA molecule bundled up with various proteins. On average, each human chromosome contains about 1.3 × 10^8 base pairs (bp) of DNA. If the DNA in a human chromosome were stretched as\\n---\\n#  DNA STRUCTURE AND THE GENETIC CODE\\n\\n|DNA has to be highly compacted in order to fit into the cell|\\n|---|\\n|1400 nm|\\n|30nm|histone octamer|\\n|solenoid|\\n|30nm|\\n|nucleosomes (\"beads on a string\")|\\n\\nFigure 4.5. How DNA is packaged into chromosomes.\\n---\\n# PACKAGING OF DNA MOLECULES INTO CHROMOSOMES\\n\\nAs far as it would go without breaking it would be about 5 cm long, so the 46 chromosomes in all represent about 2 m of DNA. The nucleus in which this DNA must be contained has a diameter of only about 10μm, so large amounts of DNA must be packaged into a small space. This represents a formidable problem that is dealt with by binding the DNA to proteins to form chromatin. As shown in Figure 4.5, the DNA double helix is packaged at both small and larger scales. In the first stage, shown on the right of the figure, the DNA double helix with a diameter of 2 nm is bound to proteins known as histones. Histones are positively charged because they contain high amounts of the amino acids arginine and lysine (page 185) and bind tightly to the negatively charged phosphates on DNA. A 146 bp length of DNA is wound around a protein complex composed of two molecules each of four different histones—H2A, H2B, H3, and H4—to form a nucleosome. Because each nucleosome is separated from its neighbor by about 50 bp of linker DNA, this unfolded chromatin state looks like beads on a string when viewed in an electron microscope. Nucleosomes undergo further packaging. A fifth type of histone, H1, binds to the linker DNA and pulls the nucleosomes together helping to further coil the DNA into chromatin fibers 30 nm in diameter, which are referred to as 30-nm solenoids. The fibers then form loops with the help of a class of proteins known as nonhistones, and this further condenses the DNA (panels on left-hand side of Fig. 4.5) into a higher order set of coils in a process called supercoiling.\\n\\nIn a normal interphase cell about 10% of the chromatin is highly compacted and visible under the light microscope (page 57). This form of chromatin is called heterochromatin and is the portion of the genome where no RNA synthesis is occurring. The remaining interphase chromatin is less compacted and is known as euchromatin.\\n\\nChromatin is in its most compacted form when the cell is preparing for mitosis, as shown at the top left of Figure 4.5. The chromatin folds and condenses further to form the 1400-nm-wide chromosomes we see under the light microscope. Because the cell is to divide, the DNA has been replicated, so that each chromosome is now formed by two chromatids, each one a DNA double helix. This means the progeny cell, produced by division of the progenitor cell, will receive a full set of 46 chromosomes. Figure 4.6 is a photograph of human chromosomes as they appear at cell division.\\n\\n# Prokaryotic Chromosomes\\n\\nThe chromosome of the bacterium E. coli is a single circular DNA molecule of about 4.5 × 10 6 base pairs. It has a circumference of 1 mm, yet must fit into the 1-μm cell, so like eukaryotic chromosomes it is coiled, supercoiled, and packaged with basic proteins that are similar to eukaryotic histones. However, an ordered nucleosome structure similar to the “beads on a string” seen in eukaryotic cells is not observed in prokaryotes. Prokaryotes do not have nuclear envelopes so the condensed chromosome together with its associated proteins lies free in the cytoplasm, forming a mass that is called the nucleoid to emphasize its functional equivalence to the eukaryotic nucleus.\\n---\\n# DNA STRUCTURE AND THE GENETIC CODE\\n\\nPlasmids\\n\\nPlasmids are small circular minichromosomes found in bacteria and some eukaryotes. They are several thousand base pairs long and are probably tightly coiled and supercoiled inside the cell. Plasmids often code for proteins that confer resistance to a particular antibiotic. In Chapter 7 we describe how plasmids are used by scientists and genetic engineers to artificially introduce foreign DNA molecules into bacterial cells.\\n\\nViruses\\n\\nViruses (page 11) rely on the host cell to make more virus. Once viruses have entered cells, the cells’ machinery is used to copy the viral genome. Depending on the virus type, the genome may be single- or double-stranded DNA, or even RNA. A viral genome is packaged within a protective protein coat. Viruses that infect bacteria are called bacteriophages. One of these, called lambda, has a fixed-size DNA molecule of 4.5 × 104 base pairs. In contrast, the bacteriophage M13 can change its chromosome size, its protein coat expanding in parallel to accommodate the chromosome. This makes M13 useful in genetic engineering (Chapter 7).\\n\\n# IN DEPTH 4.2 DNA—A Gordian Knot\\n\\nAt the start of his career Alexander the Great was shown the Gordian Knot, a tangled ball of knotted rope, and told that whoever untied the knot would conquer Asia. Alexander cut through the knot with his sword. A similar problem occurs in the nucleus, where the 46 chromosomes form 2 m of tangled, knotted DNA. How does the DNA ever untangle at mitosis? The cell adopts Alexander’s solution—it cuts the rope. At any place where the DNA helix is under strain, for instance, where two chromosomes press against each other, an enzyme called topoisomerase II cuts one chromosome double helix so that the other can pass through the gap. Then, surpassing Alexander, the enzyme rejoins the cut ends. Topoisomerases are active all the time in the nucleus, relieving any strain that develops in the tangled mass of DNA.\\n\\nchromosome under tension\\ntopoisomerase II moves to the region of most strain\\n---\\nTHE GENETIC CODE\\n\\n|1st chromosome|is bound to the|enzyme|topoisomerase II|\\n|---|---|---|---|\\n|topoisomerase II|ATP| | |\\n|2nd chromosome|is bound to the|enzyme|topoisomerase II|\\n|1st chromosome is|cut and the cut|ends are separated| |\\n|2nd chromosome|passes through|the gap in the 1st|chromosome|\\n|ADP + Pi| | | |\\n|DNA of 1st chromosome|is resealed| | |\\n|2nd chromosome is|released by the enzyme| | |\\n\\nTHE GENETIC CODE\\n\\nProteins are linear polymers of individual building blocks called amino acids (page 37). The sequence of bases along the DNA strand determines the sequence of the amino acids in proteins. There are 20 different amino acids in proteins but only 4 different bases in DNA (A, T, C, and G). Each amino acid is specified by a codon, a group of three bases. Because\\n---\\n# DNA STRUCTURE AND THE GENETIC CODE\\n\\nThere are 4 bases in DNA, a three-letter code gives 64 (4 × 4 × 4) possible codons. These 64 codons form the genetic code—the set of instructions that tells a cell the order in which amino acids are to be joined to form a protein. Despite the fact that the sequence of codons on DNA determines the sequence of amino acids in proteins, the DNA helix does not itself play a role in protein synthesis. The translation of the sequence from codons into amino acids occurs through the intervention of members of a third class of molecule—messenger RNAs (mRNA) (Fig. 4.7). Messenger RNA acts as a template, guiding the assembly of amino acids into a polypeptide chain. Messenger RNA uses the same code as the one used in DNA with one difference: In mRNA the base uracil (U) (Fig. 2.13 on page 34) is used in place of thymine (T). When we write the genetic code, we usually use the RNA format, that is, we use U instead of T.\\n\\nThe code is read in sequential groups of three, codon by codon. Adjacent codons do not overlap, and each triplet of bases specifies one particular amino acid. This discovery\\n\\nFigure 4.6. A spread of human chromosomes (at metaphase—see page 403). The green signal reveals pe gene called FMO3, which when mutated causes trimepylaminuria (fish odor syndrome) (Medical Relevance 4.1). There are two copies of pe gene, one inherited from each parent, indicated by pe arrows. The FMO3 gene is located on pe long arm of chromosome 1, pe longest human chromosome.\\n---\\n# THE GENETIC CODE\\n\\nDNA\\n\\n|5′|ATG|GGC|TAC|CCC|TGC|CTG|3′|\\n|---|---|---|---|---|---|---|---|\\n|3′|TAC|CCG|ATG|GGG|ACG|GAC|5′|\\n\\nmRNA\\n\\n5′\\n\\uf8f5AUG\\uf8f5\\nGGC\\uf8f5\\nUAC\\uf8f5\\nCCC\\uf8f5\\nUGC\\uf8f5\\nCUG\\uf8f5\\n3′\\n\\nprotein N-terminus\\n\\n|met|gly|tyr|pro|cys|leu|C-terminus|\\n|---|---|---|---|---|---|---|\\n| |M|G|Y| |P|C|L|\\n\\nThe 3′ to 5′ DNA strand is transcribed into an mRNA molecule that is translated,\\naccording to the genetic code, into protein. Note that mRNA is made in the 5′\\nto 3′ direction and proteins are synthesized from the N terminus. The 3-and\\n1- letter amino acid codes are shown.\\n\\nFigure 4.7. The central dogma of molecular biology.\\n\\nwas made by Sydney Brenner, Francis Crick, and their colleagues by studying the effect of\\nvarious mutations (changes in the DNA sequence) on the bacteriophage T4, which infects\\nthe common bacterium E. coli. If a mutation caused either one or two nucleotides to be\\nadded or deleted, from one end of the T4 DNA, then a defective polypeptide was produced,\\nwith a completely different sequence of amino acids. However, if three bases were added\\nor deleted, then the protein made often retained its normal function. These proteins were\\nfound to be identical to the original protein, except for the addition or loss of one amino\\nacid. The identification of the triplets encoding each amino acid began in 1961. This was made\\npossible by using a cell-free protein synthesis system prepared by breaking open E. coli\\ncells. Synthetic RNA polymers, of known sequence, were added to the cell-free system\\ntogether with the 20 amino acids. When the RNA template contained only uridine residues\\n(poly-U) the polypeptide produced contained only phenylalanine. The codon specifying\\nthis amino acid must therefore be UUU. A poly-A template produced a polypeptide of\\nlysine, and poly-C one of proline: AAA and CCC must therefore specify lysine and proline,\\nrespectively. Synthetic RNA polymers containing all possible combinations of the bases A,\\nC, G, and U, were added to the cell-free system to determine the codons for the other amino\\nacids. A template made of the repeating unit CU gave a polypeptide with the alternating\\nsequence leucine–serine. Because the first amino acid in the chain was found to be leucine,\\nits codon must be CUC and that for serine UCU. Although much of the genetic code\\nwas read in this way, the amino acids defined by some codons were particularly hard\\nto determine. Only when specific transfer RNA molecules (page 163) were used was it\\npossible to demonstrate that GUU codes for valine. The genetic code was finally solved\\nby the combined efforts of several research teams. The leaders of two of these, Marshall\\nNirenberg and Gobind Khorana, received the Nobel prize in 1968 for their part in cracking\\nthe code.\\n---\\n# DNA STRUCTURE AND THE GENETIC CODE\\n\\n|Amino Acid|Three-letter Code|One-letter Code|Amino Acid|Three-letter Code|One-letter Code|\\n|---|---|---|---|---|---|\\n|Alanine|GCU|A|Asparagine|AAU|N|\\n| |GCC|A| |AAG|N|\\n| |GCA|A| |CCC|N|\\n| |GCG|A| |CCC|N|\\n|Cysteine|UGU|C|Glutamine|CAA|Q|\\n| |UGC|C| |CAG|Q|\\n| | | | |CCC|Q|\\n| | | | |CCC|Q|\\n|Histidine|CAU|H|Isoleucine|AUU|I|\\n| |CAC|H| |AUC|I|\\n| | | | |UUA|I|\\n| | | | |UUG|I|\\n|Methionine|AUG|M|Phenylalanine|UUU|F|\\n| | | | |UUC|F|\\n| | | |Proline|CCU|P|\\n| |CH2| | |AGU|S|\\n| | | | |AGC|S|\\n|Threonine|ACU|T|Tryptophan|UGG|W|\\n| |ACC|T|Tyrosine|UAU|Y|\\n| |ACG|T| | | |\\n| |ACA|T| | | |\\n| |CH2| | |UGA|STOP|\\n| | | | |UAA|STOP|\\n| | | | |UAG|STOP|\\n\\nFigure 4.8. The genetic code and the corresponding amino acid side chains. The three- and one-letter amino acid abbreviations are shown. Hydrophilic side chains are shown in green, hydrophobic side chains in black. The significance of this distinction is discussed in Chapter 9.\\n---\\n# THE GENETIC CODE\\n\\nAmino Acid Names Are Abbreviated\\n\\nTo save time we usually write an amino acid as either a three-letter abbreviation, for example, glycine is written as gly and leucine as leu, or as a one-letter code, for example, glycine is G and leucine is L. Figure 4.8 shows the full name, and the three- and one-letter abbreviations, used for each of the 20 amino acids found in proteins.\\n\\nThe Code Is Degenerate But Unambiguous\\n\\nTo introduce the terms degenerate and ambiguous, consider the English language. English shows considerable degeneracy, meaning that the same concept can be indicated using a number of different words—think, for example, of lockup, cell, pen, pound, brig, and dungeon. English also shows ambiguity, so that it is only by context that one can tell whether cell means a lockup or the basic unit of life. Like the English language the genetic code shows degeneracy, but unlike language the code is unambiguous.\\n\\nThe 64 codons of the genetic code are shown in Figure 4.8 together with the side chains of the amino acids for which each codes. Amino acids with hydrophilic side chains are shown in green while those with hydrophobic side chains are in black. The importance of this distinction will be discussed in Chapter 9. Sixty-one codons specify an amino acid, and the remaining three act as stop signals for protein synthesis. Methionine and tryptophan are the only amino acids coded for by single codons. The other 18 amino acids are encoded by either 2, 3, 4, or 6 codons and so the code is degenerate. No triplet codes for more than one amino acid and so the code is unambiguous. Notice that when two or more codons specify the same amino acid, they usually only differ in the third base of the triplet. Thus mutations can arise in this position of the codon without altering amino acid sequences. Perhaps degeneracy evolved in the triplet system to avoid a situation in which 20 codons each meant one amino acid, and 44 specified none. If this were the case, then most mutations would stop protein synthesis dead.\\n\\nStart and Stop Codons and the Reading Frame\\n\\nThe order of the codons in DNA and the amino acid sequence of a protein are colinear. The start signal for protein synthesis is the codon AUG specifying the incorporation of methionine. Because the genetic code is read in blocks of three, there are three potential reading frames in any mRNA. Figure 4.9 shows that only one of these results in the synthesis of the correct protein. When we look at a sequence of bases, it is not obvious which of the reading frames should be used to code for protein. As we shall see later\\n\\n|met|his|glu|tyr|\\n|---|---|---|---|\\n|A U G|C U A|G A A|U A C|\\n| | |cys|stop|asn|\\n|A U G|C U A|G A A|U A C|\\n| | |ala|arg|ile|\\n| |A U|G C|U A|G A|U A|C|\\n\\nFigure 4.9. Reading frames—the genetic code is read in blocks of three.\\n---\\n#  DNA STRUCTURE AND THE GENETIC CODE\\n\\n|frameshift mutation|met|gln|trp|val|glu|\\n|---|---|---|---|---|---|\\n|normal|A U G|C A A|U G G|G U C|G A G|\\n|mutant|A U G|C A A|G G G|U C G|A G A|\\n\\n|nonsense mutation|met|gln|trp|val|glu|\\n|---|---|---|---|---|---|\\n|normal|A U G|C A A|U G G|G U C|G A G|\\n|mutant|A U G|C A A|U G A|G U C|G A G|\\n\\n|missense mutation|met|gln|trp|val|glu|\\n|---|---|---|---|---|---|\\n|normal|A U G|C A A|U G G|G U C|G A G|\\n|mutant|A U G|C A U|U G G|G U C|G A G|\\n\\nFigure 4.10. Mutations that alter the sequence of bases.\\n\\n(page 168), the ribosome scans along the mRNA until it encounters an AUG. This both\\ndefines the first amino acid of the protein and the reading frame used from that point on.\\nA mutation that inserts or deletes a nucleotide will change the normal reading frame and is\\ncalled a frameshift mutation (Fig. 4.10).\\nThe codons UAA, UAG, and UGA are stop signals for protein synthesis. A base\\nchange that causes an amino acid codon to become a stop codon is known as a nonsense\\nmutation (Fig. 4.10). If, for example, the codon for tryptophan UGG changes to UGA,\\nthen a premature stop signal will have been introduced into the messenger RNA template.\\nA shortened protein, usually without function, is produced.\\n\\nThe Code Is Nearly Universal\\nThe code shown in Figure 4.8 is the one used by organisms as diverse as E. coli and humans\\nfor their nuclear-encoded proteins. It was originally thought that the code would be universal.\\nHowever, several mitochondrial genes use UGA to mean tryptophan rather than stop. The\\nnuclear code for some protists uses UAA and UAG to code for glutamine rather than stop.\\n\\nMissense Mutations\\nA mutation that changes the codon from one amino acid to that for another is a missense\\nmutation (Fig. 4.10). As shown in Figure 4.8, the second base of each codon shows the most\\nconsistency with the chemical nature of the amino acid it encodes. Amino acids with charged,\\n---\\n# SUMMARY\\n\\nhydrophilic side chains usually have A or G—a purine—in the second position. Those with hydrophobic side chains usually have C or U—a pyrimidine—in that position. This has implications for mutations of the second base. Substitution of a purine for a pyrimidine is very likely to change the chemical nature of the amino acid side chain significantly and can therefore seriously affect the protein. Sickle cell anemia is an example of such a mutation. At position 6 in the β-globin chain of hemoglobin the mutation in DNA changes a glutamate residue encoded by GAG to a valine residue encoded by GTG (GUG in RNA). The shorthand notation for this mutation is E6V, meaning that the glutamate (E) at position 6 of the protein becomes a valine (V). This change in amino acid alters the overall charge of the chain, and the hemoglobin tends to precipitate in the red blood cells of those affected. The cells adopt a sickle shape and therefore tend to block blood vessels, causing painful cramplike symptoms and progressive damage to vital organs.\\n\\n# SUMMARY\\n\\n1. DNA, the cell’s database, contains the genetic information necessary to encode RNA and protein.\\n2. The information is stored in the sequence of four bases. These are the purines, adenine and guanine, and the pyrimidines, cytosine and thymine. Each base is attached to the 1-carbon atom of the sugar deoxyribose. A phosphate group is attached to the 5-carbon atom of the sugar. The base + sugar + phosphate is called a nucleotide.\\n3. The enzyme DNA polymerase joins nucleotides together by forming a phosphodiester link between the hydroxyl group on the 3\\' carbon of deoxyribose of one nucleotide and the 5\\'-phosphate group of another. This gives rise to the sugar–phosphate backbone structure of DNA.\\n4. The two strands of DNA are held together in a double-helical structure because guanine hydrogen bonds with cytosine and adenine hydrogen bonds with thymine. This means that if the sequence of one strand is known, that of the other can be inferred. The two strands are complementary in sequence.\\n5. DNA binds to histone and nonhistone proteins to form chromatin. DNA is wrapped around histones to form a nucleosome structure. This is then folded again and again. This packaging compresses the DNA molecule to a size that fits into the cell.\\n6. The genetic code specifies the sequence of amino acids in a polypeptide. The code is transferred from DNA to mRNA and is read in groups of three bases (a codon) during protein synthesis. There are 64 codons; 61 specify an amino acid and 3 are the stop signals for protein synthesis.\\n---\\n# DNA STRUCTURE AND THE GENETIC CODE\\n\\n|Medical Relevance|Trimethylaminuria|\\n|---|---|\\n|Trimethylaminuria is an inherited disorder characterized by an unpleasant body odor similar to rotting fish. Mutations in the gene FMO3, located on human chromosome 1 (Fig. 4.6) produce a defective version of a protein called flavin-containing monooxygenase 3 (FMO3). One of the functions of this protein, which is found in the liver, is to convert a chemical called trimethylamine (very smelly) to trimethylamine N -oxide (nonsmelly). FMO3 carries out this reaction by adding an oxygen onto the nitrogen of trimethylamine. Defective FMO3 protein cannot carry out this reaction. Affected individuals therefore excrete large amounts of trimethylamine in their breath, sweat, and urine. Trimethylamine is the chemical that gives rotting fish its distinctive and unpleasant smell, and this gives rise to the alternative name for trimethylaminuria—fish odor syndrome.| |\\n|Trimethylamine is derived from our diet in two ways. Fish contain lots of trimethylamine N-oxide (page 208). The bacteria that live in our intestines remove the oxygen from the nitrogen of the N-oxide to produce trimethylamine. However, one cannot avoid trimethylamine simply by avoiding fish. The phospholipids in the cell membranes of food include phosphatidylcholine, and this is hydrolyzed by our digestive enzymes to release free choline. Gut bacteria then break down the choline further, producing trimethylamine together with acetaldehyde (also called ethanal).| |\\n|Trimethylaminuria is a difficult disorder to live with. Because of their body odor, sufferers often experience rejection and social isolation. A diet very low in choline, and other chemicals that contain a trimethylamine group, can help to reduce the problem.| |\\n|Analysis of the faulty genes in different families has revealed different mutations in the FMO3 gene. Some families carry the missense mutation P153L in which a CCC that codes for a proline as the 153rd amino acid in the normal protein is mutated to CTC (leu), a change that turns out to be critical for the operation of the protein. Other families carry a nonsense mutation in which the GGA (gly) at amino acid 148 in the normal protein, is mutated to TGA, a stop codon. Another family carries a frameshift mutation in which the sequence . . . GGA AAG CGT GTC CTG G . . . , which specifies amino acids 188gly lys arg val leu192 in the normal protein, is mutated, by the deletion of the highlighted (bold) G, to . . . GGA AAG CGT TCC TGG . . . , thus generating the amino acid sequence 188gly lys arg ser trp192. The frameshift mutation changes not only the amino acid sequence, but also introduces a stop codon further down the mRNA. Because different mutations can give rise to trimethylaminuria, the development of a general diagnostic test for carriers of the faulty gene (who also have one good copy and are therefore carriers who do not themselves have the disorder; page 407) is not possible.| |\\n|In contrast, a disorder such as sickle cell anemia is caused, in all individuals, by the same mutation (page 81). One test will diagnose the defective gene and prospective parents can be advised and counseled.| |\\n---\\n# TRIMETHYLAMINURIA\\n\\n|CH3|3H|CH3|choline|CH3|CH3|\\n|---|---|---|---|---|---|\\n|CH3|N|CH3| | | |\\n|CH2| |OH| | | |\\n| | | |trimethylamine|acetaldehyde|(ethanal)|\\n|CH2|O|PP|O| | |\\n|O| |O|OH|- O2| |\\n|O|P|O| | | |\\n|H| |H| | | |\\n|C| |C| | | |\\n|OOO| |OOO| | | |\\n|C|O|C|O| | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H|C|H| | | |\\n|H| |H| | | |\\n| | | | | | |\\n|phosphatidyl choline| | | | | |\\n| | | |stearate| | |\\n| | | | | | |\\n| | | |arachidonate| | |\\n---\\n# FURTHER READING\\n\\nDiGuilo, M. 1997. The origin of the genetic code. Trends Biochem. Sci. 22: 49–50.\\n\\nDolphin, C. T., Janmohamed, A., Smith, R. L., Shephard, E. A., and Phillips, I. R. 1997. Missense mutation in flavin-containing monooxygenase 3 gene, FMO3, underlies fish-odour syndrome. Nature Genetics 17: 491–494.\\n\\nMaddox, B. 2002. Rosalind Franklin: The Dark Lady of DNA. New York: Harper Collins.\\n\\nRoca, J. 1995. The mechanisms of DNA topoisomerases. Trends Biochem. Sci. 20: 156–160\\n\\nWatson, J. D., and Crick, F. H. C. 1953. A structure for deoxyribose nucleic acid. Nature 171: 737.\\n\\nWoodcock, C. L., and Horowitz, R. A. 1995. Chromatin organization re-viewed. Trends Cell Biol. 5: 272–277.\\n\\n# REVIEW QUESTIONS\\n\\nFor each question, choose the ONE BEST answer or completion.\\n\\n|1. Which base is not found in the DNA of most organisms?|E. Thymine|\\n|---|---|\\n|2. The bonds that join the nucleotides to form a single strand of DNA are|C. phosphodiester links.|\\n|3. DNA is packaged into chromatin with the help of the protein called|B. histone.|\\n|4. Histone proteins are rich in the amino acids|D. arginine and lysine.|\\n|5. The 20 amino acids found in proteins are coded for by|C. 61 codons.|\\n---\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n|6.|Methionine is coded for by|\\n|---|---|\\n| |A. one codon only.|\\n| |B. two different codons.|\\n| |C. three different codons.|\\n| |D. four different codons.|\\n| |E. six different codons.|\\n|7.|A missense mutation is one that|\\n| |A. creates a stop codon in the mRNA.|\\n| |B. changes the codon of one amino acid to that of another.|\\n| |C. changes the reading frame of the mRNA.|\\n| |D. creates a break in the DNA strand.|\\n| |E. has no effect on the amino acid sequence.|\\n\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n1. D. Uracil, which hydrogen bonds with adenine, is present in RNA but is absent from the DNA of most organisms.\\n2. C. The deoxyribose residues of adjacent nucleotides are joined by phosphodiester links.\\n3. B. Histones help to package DNA into chromatin.\\n4. D. Histones bind to negatively charged DNA because they are rich in the positively charged amino acids arginine and lysine.\\n5. C. Of the 64 possible permutations of a triplet of four types of base, 61 code for the 20 amino acids, leaving 3 (UGA, UAG, and UAA) to code for stop.\\n6. A. Only AUG codes for methionine, which is the first amino acid found in all newly synthesized proteins of most species.\\n7. B. A missense mutation changes the codon of one amino acid to that of another. A nonsense mutation is one that creates a stop codon and a frameshift mutation is one that changes the reading frame of the mRNA.\\n---\\n# DNA AS A DATA STORAGE MEDIUM\\n\\nINTRODUCTION\\n\\nThe genetic material DNA must be faithfully replicated every time a cell divides to ensure that the information encoded in it is passed unaltered to the progeny cells. DNA molecules have to last a long time compared to RNA and protein. The sugar–phosphate backbone of DNA is a very stable structure because there are no free hydroxyl groups on the sugar— they are all used up in bonds, either to the base or to phosphate. The bases themselves are protected from chemical attack because they are hidden within the DNA double helix. Nevertheless, chemical changes—mutations—do occur in the DNA molecule, and cells have had to evolve mechanisms to ensure that mutation is kept to a minimum. Repair systems are essential for both cell survival and to ensure that the correct DNA sequence is passed on to daughter cells. This chapter describes how new DNA molecules are made during chromosome duplication and how the cell acts to correct base changes in DNA.\\n\\nDNA REPLICATION\\n\\nDuring replication the two strands of the double helix unwind. Each then acts as a template for the synthesis of a new strand. This process generates two double-stranded daughter DNA molecules, each of which is identical to the parent molecule. The base sequences of the new strands are complementary in sequence to the template strands upon which they were built. This means that G, A, C, and T in the old strand cause C, T, G, and A, respectively, to be placed in the new strand.\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\n\\nISBN 0-471-26393-1 Copyright ©C2004 by John Wiley & Sons, Inc.\\n---\\n88\\n\\n| |5′|lagging strand|3′|\\n|---|---|---|---|\\n|single-strand| |RNA|DNA|\\n|binding protein|primase|polymerase 5′| |\\n|DnaB helicase| |5′| |\\n|5′| |Okazaki| |\\n|3′|RNA primer|fragment| |\\n|DnaB helicase|DNA polymerase|5′|3′|\\n\\n3′ leading strand 5′\\n\\nreplication fork\\n\\nFigure 5.1. DNA replication. The helicases, and the replication fork, are moving to the left.\\n\\nThe DNA Replication Fork\\n\\nReplication of a new DNA strand starts at specific sequences known as origins of replication. The small circular chromosome of Escherichia coli has only one of these, whereas eukaryotic chromosomes, which are much larger, have many. At each origin of replication, the parental strands of DNA untwist to give rise to a structure known as the replication fork (Fig. 5.1). This unwinding permits each parental strand to act as a template for the synthesis of a new strand. The structure of the double helix and the nature of DNA replication pose a mechanical problem. How do the two strands unwind and how do they stay unwound so that each can act as a template for a new strand?\\n\\nPROTEINS OPEN UP THE DNA DOUBLE HELIX DURING REPLICATION\\n\\nThe DNA molecule must be opened up before replication can proceed. The helix is a very stable structure, and in a test tube the two strands separate only when the temperature reaches about 90 ◦ C. In the cell the combined actions of several proteins help to separate the two strands. Much of our knowledge of replication comes from studying E. coli, but similar systems operate in all organisms, prokaryote and eukaryote. The proteins this bacterium uses to open up the double helix during replication include DnaA, DnaB, DnaC, and single-strand binding proteins.\\n\\nDnaA Protein\\n\\nSeveral copies of the protein DnaA bind to four sequences of nine base pairs within the E. coli origin of replication (ori C). This causes the two strands to begin to separate (or “melt”) because the hydrogen bonds in DNA are broken near to where the DnaA protein\\n---\\nPROTEINS OPEN UP THE DNA DOUBLE HELIX DURING REPLICATION 89\\n\\nbinds. The DNA is now in the open complex formation and has been prepared for the next stage in replication, which is to open up the helix even further.\\n\\n✎ Medical Anthrax, Cipro, and the Need to Unwind\\n\\nRelevance The DNA helix must be able to rotate if it is to relieve the rotational stress caused by the formation of the replication fork. This is achieved by an enzyme called topoisomerase I, which breaks a phosphodiester link in one strand of the helix. The DNA helix can now rotate around the remaining unbroken strand to relieve the stress. The phosphodiester link is then reformed. The repeated operation of topoisomerase 1 allows progressive unwinding of the two template strands as replication proceeds. Topoisomerase I should not be confused with topoisomerase II (page 74), which cuts both strands of a DNA double helix to allow a second chromosome to pass through the gap. Topoisomerases II play an essential role during DNA replication in prokaryotes. When the chromosome\\n\\nTopoisomerase I cleaves a phosphodiester bond in one of the DNA strands\\n\\nDNA helix now rotates around the remaining unbroken strand\\n\\nThe intact strand passes through the nicked strand and the phosphodiester bond is reformed\\n\\nunwinds to form the replication fork, the region of DNA in front of the fork overwinds; the twists cannot dissipate because the prokaryotic chromosome is circular. Topoisomerase II relieves this stress and allows DNA replication to proceed by performing the same action as described on page 000, except that both the DNA double helix that is cut and then resealed, and the double helix that is passed through the gap, belong to the same circular chromosome. Concerns that a terrorist organization might release large amounts of anthrax spores have caused several governments to stockpile large amounts of the antibiotic Cipro. This works by inhibiting the prokaryotic form of topoisomerase II (sometimes called gyrase) and hence preventing cell replication.\\n\\n✍ cell replication. ✌\\n---\\n# DNA AS A DATA STORAGE MEDIUM\\n\\nDnaB and DnaC Proteins\\n\\nDnaB is a helicase. It moves along a DNA strand, breaking hydrogen bonds, and in the process unwinds the helix (Fig. 5.1). Two molecules of DnaB are needed, one for each strand of DNA. One DnaB attaches to one of the template strands and moves in the 5\\' to 3\\' direction; the second DnaB attaches to the other strand and moves in the 3\\' to 5\\' direction. The unwinding of the DNA double helix by DnaB is an ATP-dependent process. DnaB is escorted to the DNA strands by another protein DnaC. However, having delivered DnaB to its destination, DnaC plays no further role in replication.\\n\\nSingle-Strand Binding Proteins\\n\\nAs soon as DnaB unwinds the two parental strands, they are engulfed by single-strand binding proteins. These proteins bind to adjacent groups of 32 nucleotides. DNA covered by single-strand binding proteins is rigid, without bends or kinks. It is therefore an excellent template for DNA synthesis (Fig. 5.1). Single-strand binding proteins are sometimes called helix-destabilizing proteins.\\n\\n# BIOCHEMISTRY OF DNA REPLICATION\\n\\nThe synthesis of a new DNA molecule is catalyzed by DNA polymerase III. Its substrates are the four deoxyribonucleoside triphosphates, dATP, dCTP, dGTP, and dTTP. DNA polymerase III catalyzes the formation of a phosphodiester link (Fig. 4.3 on page 69) between the 3\\'-hydroxyl group of one sugar residue and the 5\\'-phosphate group of a second sugar residue (Fig. 5.2a). The base sequence of a newly synthesized DNA strand is dictated by the base sequence of its parental strand. If the sequence of the template strand is 3\\' CATCGA 5\\', then that of the daughter strand is 5\\' GTAGCT 3\\'.\\n\\nDNA polymerase III can only add a nucleotide to a free 3\\'-hydroxyl group and therefore synthesizes DNA in the 5\\' to 3\\' direction. The template strand is read in the 3\\' to 5\\' direction. However, the two strands of the double helix are antiparallel. They cannot be synthesized in the same direction because only one has a free 3\\'-hydroxyl group, the other has a free 5\\'-phosphate group. No DNA polymerase has been found that can synthesize DNA in the 3\\' to 5\\' direction, that is, by attaching a nucleotide to a 5\\'-phosphate, so the synthesis of the two daughter strands must differ. One strand, the leading strand, is synthesized continuously while the other, the lagging strand, is synthesized discontinuously. DNA polymerase III can synthesize both daughter strands, but must make the lagging strand as a series of short 5\\' to 3\\' sections. The fragments of DNA, called Okazaki fragments after Reiji Okazaki who discovered them in 1968, are then joined together by DNA polymerase and ligase (Fig. 5.1).\\n\\nDNA Synthesis Requires an RNA Primer\\n\\nDNA polymerase III cannot itself initiate the synthesis of DNA. An enzyme called primase is needed to catalyze the formation of a short stretch of RNA complementary in sequence to the DNA template strand (Fig. 5.1). This RNA chain, the primer, is needed to prime (or start) the synthesis of the new DNA strand. DNA polymerase III catalyzes the formation of a phosphodiester link between the 3\\'-hydroxyl group of the RNA primer and the 5\\'-phosphate group of the appropriate deoxyribonucleotide. Several RNA primers are made along the length of the lagging strand template. Each is extended in the 5\\' to 3\\' direction by DNA.\\n---\\n# BIOCHEMISTRY OF DNA REPLICATION\\n\\n|5′ G|T|A|G|OH|3′|5′ G|T|A|G|OH|3′|\\n|---|---|---|---|---|---|---|---|---|---|---|---|\\n|3′ C|A|T|C|G|A|5′|3′ C|A|T|C|G|A|5′|\\n\\nDNA polymerase adds a C to the growing chain\\n\\nDNA polymerase makes a mistake and adds an A to the growing chain\\n\\n| | | | | | | | | | | | | | |\\n|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\\n|OH| | | | | |A|OH| | | | | | |\\n\\n|5′ G|T|A|G|C|3′|5′ G|T|A|G| | |3′|\\n|---|---|---|---|---|---|---|---|---|---|---|---|---|\\n|3′ C|A|T|C|G|A|5′|3′ C|A|T|C|G|A|5′|\\n\\nDNA polymerase III uses its 3′ 5′ exonuclease activity to remove the incorrect deoxynucleotide. This is known as \\'proof-reading\\'.\\n\\nReplication continues (a)\\n\\n|5′ G|T|A|G|OH|3′|\\n|---|---|---|---|---|---|\\n|3′ C|A|T|C|G|A|5′|\\n\\nDNA polymerase III uses its 5′ 3′ polymerase activity to insert the correct deoxynucleotide\\n\\n|5′ G|T|A|G|C|OH|3′|\\n|---|---|---|---|---|---|---|\\n|3′ C|A|T|C|G|A|5′|\\n\\nReplication continuesReplication continues (b)\\n\\nFigure 5.2. DNA polymerase III can correct its own mistakes.\\n---\\nDNA AS A DATA STORAGE MEDIUM\\n\\npolymerase III until it reaches the 5 ′ end of the next RNA primer (Fig. 5.1). In prokaryotes the lagging strand is primed about every 1000 nucleotides whereas in eukaryotes this takes place each 200 nucleotides.\\n\\nRNA Primers Are Removed Once the synthesis of the DNA fragment is complete, the RNA primers must be replaced by deoxyribonucleotides. In prokaryotes the enzyme DNA polymerase I removes ribonucleotides using its 5 ′ → 3 ′ exonuclease activity and then uses its 5 ′ → 3 ′ polymerizing activity to incorporate deoxyribonucleotides. In this way, the entire RNA primer gets replaced by DNA. Synthesis of the lagging strand is completed by the enzyme DNA ligase, which joins the DNA fragments together by catalyzing formation of phosphodiester links between adjacent fragments. Eukaryotic organisms probably use an enzyme called ribonuclease H to remove their RNA primers. This enzyme breaks phosphodiester links in an RNA strand that is hydrogen-bonded to a DNA strand.\\n\\nThe Self-Correcting DNA Polymerase The human genome consists of about 3 × 10 9 base pairs of DNA. DNA polymerase III makes a mistake about every 1 in 10 4 bases and joins an incorrect deoxyribonucleotide to the growing chain. If unchecked, these mistakes would lead to a catastrophic mutation rate. Fortunately, DNA polymerase III has an built-in proofreading mechanism that corrects its own errors. If an incorrect base is inserted into the newly synthesized daughter strand, the enzyme recognizes the change in shape of the double-stranded molecule, which arises through incorrect base pairing, and DNA synthesis stops (Fig. 5.2b). DNA polymerase III then uses its 3 ′to 5 ′exonuclease activity to remove the incorrect deoxyribonucleotide and replace it with the correct one. DNA synthesis then proceeds. DNA polymerase III hence functions as a self-correcting enzyme.\\n\\n✄✄✄ ▯▯▯ ✂✂Example 5.1 The Meselson–Stahl Experiment ✂ ✁✁✁ In 1958 Matthew Meselson and Franklin Stahl designed an ingenious experiment to test whether each strand of the double helix does indeed act as a template for the synthesis of a new strand. They grew the bacterium Escherichia coli in a medium containing the heavy isotope 15N that could be incorporated into new DNA molecules. After several cell divisions they transferred the bacteria, now containing “heavy” DNA, to a medium containing only the lighter, normal, isotope 14N. Any newly synthesized DNA molecules would therefore be lighter than the original parent DNA molecules containing 15N. The difference in density between the heavy and light DNAs allows their separation using very high speed centrifugation. The results of this experiment are illustrated in the figure. DNA isolated from cells grown in the 15 N medium had the highest density and migrated the furthest during centrifugation. The lightest DNA was found in cells grown in the 14N medium for two generations, whereas DNA from bacteria grown for only one generation in the lighter 14N medium had a density half way between these two. This is exactly the pattern expected if each strand of the double helix acts as a template for the synthesis of a new strand. The two heavy parental strands separated during replication, with each acting as a template for a newly synthesized light strand, which remained bound to the heavy strand in a double helix. The resulting DNA was therefore of intermediate density. Only in the second round of DNA replication, when the light strands created during the first round of replication were allowed to act as templates for the construction of complementary light strands, did DNA double helices composed entirely of 14N-containing building blocks appear. The two scientists were awarded the Nobel prize for this discovery that DNA replication is “semiconservative,” meaning that the results are not completely new but are half new and half old.\\n---\\n|BIOCHEMISTRY OF DNA REPLICATION|\\n|---|\\n|bacteria grown in medium enriched with the heavy isotope 15N|bacteria transferred to medium containing 14N|bacteria continue to grow in medium containing 14N|\\n|1st cell division 2nd cell division|\\n|the density of the DNA isolated from the bacteria was analyzed by high speed centrifugation|\\n|least dense| |most dense|\\n|This experiment demonstrated that the two strands of the parental DNA unwind and each acts as a template for a new daughter strand|\\n|new old|old new|\\n---\\n# DNA AS A DATA STORAGE MEDIUM\\n\\nDNA REPAIR\\n\\nDeoxyribonucleic Acid can be damaged by a number of agents, which include oxygen, water, naturally occurring chemicals in our diet, and radiation. Because damage to DNA can change the sequence of bases, a cell must be able to repair alterations in the DNA code if it is to survive and pass on the DNA database unaltered to its progeny cells.\\n\\nSpontaneous and Chemically Induced Base Changes\\n\\nThe most common damage suffered by a DNA molecule is depurination—the loss of an adenine or guanine because the bond between the purine base and the deoxyribose sugar to which it is attached is hydrolyzed by water. Within each human cell about 5000–10,000 depurinations occur every day.\\n\\nDeamination is a less frequent event; it happens about 100 times a day in every human cell. Collision of water molecules with the bond linking the amine group to cytosine sets off a spontaneous deamination that produces uracil. Cytosine base pairs with guanine, whereas uracil pairs with adenine. If this change were not corrected, then a CG base pair would mutate to a UA base pair the next time the DNA strand was replicated.\\n\\nUltraviolet light or chemical carcinogens such as benzopyrene, found in cigarette smoke, can also disrupt the structure of DNA. The absorption of ultraviolet light can cause two adjacent thymine residues to link and form a thymine dimer. If uncorrected, thymine dimers create a distortion in the DNA helix known as a bulky lesion. This inhibits normal base pairing between the two strands of the double helix and blocks the replication process. Ultraviolet light has a powerful germicidal action and is widely used to sterilize equipment. One of the reasons why bacteria are killed by this treatment is because the formation of large numbers of thymine dimers prevents replication.\\n\\nRepair Processes\\n\\nIf there were no way to correct altered DNA, the rate of mutation would be intolerable. DNA excision and DNA repair enzymes have evolved to detect and to repair altered DNA. The role of the repair enzymes is to cut out (excise) the damaged portion of DNA and then to repair the base sequence. Much of our knowledge on DNA repair has been derived from studies on E. coli, but the general principles apply to other organisms such as ourselves. Repair is possible because DNA comprises two complementary strands. If the repair mechanisms can identify which of the two strands is the damaged one, it can then be repaired as good as new by rebuilding it to be complementary to the undamaged one.\\n\\nTwo types of excision repair are described in this section: base excision repair and nucleotide excision repair. The common themes of each of these repair mechanisms are:\\n(1) An enzyme recognizes the damaged DNA,\\n(2) the damaged portion is removed,\\n(3) DNA polymerase inserts the correct nucleotide(s) into position (according to the base sequence of the second DNA strand), and\\n(4) DNA ligase joins the newly repaired section to the remainder of the DNA strand.\\n\\nBase excision–repair is needed to repair DNAs that have lost a purine (depurination), or where a cytosine has been deaminated to uracil (U). Although uracil is a normal constituent of RNA, it does not form part of undamaged DNA and is recognized and removed by the repair mechanisms.\\n---\\n#  DNA REPAIR\\n\\n|depurination|deamination|\\n|---|---|\\n|5′|H C N C C O|\\n|O CH 2 O N C N H|O CH2 O N C|\\n|C H H C N C|C H H C O|\\n|H C3′ C H N H|H C 3′ C H cytosine|\\n|O H H|O H|\\n|guanine| |\\n|H2O|NH + H3O+|\\n|guanine|4 H O|\\n|5′|H C N H|\\n|O CH 2 O OH|O CH2 O N C|\\n|C H H C|C H H C O|\\n|H C 3′ C H|H C 3′ C H uracil|\\n|O H|O H|\\n|depurinated sugar residue| |\\n|removal of uracil by the enzyme uracil-DNA-glycosidase| |\\n\\nREPAIR (Figure 5.5)\\n\\n|5′|O CCH2 H O H COH|\\n|---|---|\\n|H C 3′ C H|O H|\\n|depyrimidated sugar residue| |\\n\\nFigure 5.3. Spontaneous reactions corrupt the DNA database.\\n\\nrepair enzyme uracil–DNA glycosidase (Fig. 5.3). This leaves a gap in the DNA where the\\nbase had been attached to deoxyribose. There is no enzyme that can simply reattach a C into\\nthe vacant space on the sugar. Instead, an enzyme called AP endonuclease recognizes the\\ngap and removes the sugar by breaking the phosphodiester links on either side (Fig. 5.5).\\nWhen DNA has been damaged by the loss of a purine (Fig. 5.3), AP endonuclease also\\nremoves the sugar that has lost its base. The AP in the enzyme’s name means apyrimidinic\\n(without a pyrimidine) or apurinic (without a purine).\\nThe repair process for reinserting a purine, or a pyrimidine, into DNA is now the\\nsame (Fig. 5.5). DNA polymerase I replaces the appropriate deoxyribonucleotide into\\n---\\n# DNA AS A DATA STORAGE MEDIUM\\n\\n| |H|O|\\n|---|---|---|\\n|5′|N|C|\\n|O|CH A|O|C|C|CH 3|\\n|C|2|H|O|H|C|N|C|H|\\n|H|C|3′|C|H| |H|O|\\n| |O| |H| |O|C|N|C|C|CH 3|\\n| |5′| |O|P|O|CH 2|O| |N|C|\\n| |O| |C| |C|CH| | |3|\\n| |C|H|O|H|C|N|H|C|\\n| |H|C| |C|H| | |O|\\n| | |O|3′| |H| | |O|\\n| | | | | | | | | | |ultraviolet light|\\n\\n| |H|O|\\n|---|---|---|\\n|5′|N|C|\\n|O|CH 2| | | | | |3|\\n|C|H|O|H|C|N|H|C|\\n|H|C| |C| | |O|\\n| | |O|3′| |H| | |O|\\n| | | | | | | | | | | |\\n\\nFigure 5.4. Formation of a thymine dimer in DNA.\\n---\\n#  DNA REPAIR\\n\\n|Stage 1|Stage 2|\\n|---|---|\\n|Hydrolysis of the phosphodiester links surrounding the sugar residue from which the base has been removed|DNA pol I inserts the correct deoxyribonucleotide and DNA ligase catalyzes the formation of the phosphodiester links|\\n\\nIn a specific AP endonuclease position, phosphodiester links are cleaved. DNA ligase then seals the strand by catalyzing the reformation of a phosphodiester link.\\n\\nExample 5.2 A Curious Code\\n\\nA bacteriophage called PBS2, which infects the bacterium Bacillus subtilis, is unusual in that it uses uracil in place of thymine in its DNA. This would be expected to have the unfortunate result—for the virus—that as soon as the bacteriophage DNA entered the bacterial cytoplasm, bacterial repair enzymes would begin cutting the uracils out. PBS2 gets around this problem by quickly making a protein called uracil glycosylase inhibitor, which binds irreversibly to the bacterium’s uracil DNA glycosylase and inactivates it. Once this DNA repair enzyme is inactivated, the next problem is to persuade the bacterium to make lots of uracil nucleotides rather than thymine ones. Thymine is normally made from uracil, so all PBS2 does is to produce a second protein that inhibits this conversion, ensuring a good supply of dUTP for replication of its DNA.\\n\\nNucleotide excision–repair is required to correct a thymine dimer. The thymine dimer, together with some 30 surrounding nucleotides, is excised from the DNA. Repairing damage of this bulky type requires several proteins because the exposed, undamaged, DNA strand must be protected from nuclease attack while the damaged strand is repaired by the actions of DNA polymerase I and DNA ligase.\\n---\\n# DNA AS A DATA STORAGE MEDIUM\\n\\n|Medical Relevance|Bloom’s Syndrome and Xeroderma Pigmentosum|\\n|---|---|\\n|5.2|DNA helicases are essential proteins required to open up the DNA helix during replication. In Bloom’s syndrome, mutations give rise to a defective helicase. The result is excessive chromosome breakage, and affected people are predisposed to many different types of cancers when they are young.|\\n| |People who suffer from the genetic disorder known as xeroderma pigmentosum are deficient in one of the enzymes for excision repair. As a result, they are very sensitive to ultraviolet light. They contract skin cancer even when they have been exposed to sunlight for very short periods because thymine dimers produced by ultraviolet light are not excised from their genomes.|\\n\\n# GENE STRUCTURE AND ORGANIZATION IN EUKARYOTES\\n\\nIntrons and Exons—Additional Complexity in Eukaryotic Genes\\n\\nGenes that code for proteins should be simple things: DNA makes RNA makes protein, and a gene codes for the amino acids of a protein by the three-base genetic code. In prokaryotes, indeed, a gene is a continuous series of bases that, read in threes, code for the protein. This simple and apparently sensible system does not apply in eukaryotes. Instead, the protein-coding regions of almost all eukaryotic genes are organized as a series of separate bits interspersed with noncoding regions. The protein-coding regions of the split genes are called exons. The regions between are called introns, short for intervening sequences. In Figure 5.6 we show the structure of the β-globin gene, which contains three exons and two introns. Introns are often very long compared to exons. As happens in prokaryotes, messenger RNA complementary to the DNA is synthesized, but then the introns are spliced out before the mRNA leaves the nucleus (page 118). This means that a gene is much longer\\n\\n|α-globin gene cluster - chromosome 16|embryo|fetus & adult|\\n|---|---|---|\\n| |ζ|ψ|ψ|α 2|α 1|\\n|β-globin gene cluster - chromosome 11|embryo|fetus|adult|\\n| |ε|γ G|γA|ψ|δ|β|\\n\\nPromoter and enhancers\\n\\nFigure 5.6.\\nThe human α- and β-globin gene family clusters. Adults only express α, β, and δ, and of these the expression of δ is very low. The exon/intron boundaries of the β-globin gene are indicated at the bottom.\\n---\\n# GENE STRUCTURE AND ORGANIZATION IN EUKARYOTES\\n\\nthan the mRNA that ultimately codes for the protein. The name exon derives from the fact that these are the regions of the gene that, when transcribed into mRNA, exit from the nucleus.\\n\\nIn fact there is an evolutionary rationale to this apparently perverse arrangement. As we will see (page 200), a single protein is often composed of a series of domains, with each domain performing a different role. The breaks between exons usually correspond to domain boundaries. During evolution, reordering of exons has created new genes that have some of the exons of one gene, and some of the exons of another, and hence generates novel proteins composed of new arrangements of domains, each of which still does its job.\\n\\nThe Major Classes of Eukaryotic DNA\\n\\nWe do not yet fully understand the construction of our nuclear genome. Only about 1.5% of the human genome codes for exonic sequences (i.e., makes protein) with about 23.5% coding for introns, tRNA genes, and ribosomal RNA (rRNA) genes. Most protein-coding genes occur only once in the genome and are called single-copy genes.\\n\\nIN DEPTH 5.1 Genome Projects\\n\\nA major goal of biology in the latter decades of the twentieth century was to determine the base sequence of the human genome. This was published in 2001. The goal of biology in the twenty-first century is to determine what all the genes do. This is a difficult task, which is made easier by a comparative genomics approach. Humans share many genes with other organisms. Genes such as those encoding proteins important in metabolism, development, and the control of cell division are especially conserved through evolution.\\n\\nThe genome of the pufferfish, Fugu rubripes rupribes, has about the same number of genes as the human genome, but is only one seventh of the size because it has less extragenic DNA. The pufferfish genome has been sequenced and now can be compared with the human genome to help identify human protein-coding genes.\\n\\nThe genomes of other organisms are also being sequenced. The mouse is a very good model to compare to the human genome. The mouse is a mammal but has a rapid generation time. Mutant mouse models are proving very useful in defining the function of human genes. Caenorhabditis elegans is a tiny nematode worm with just 959 body cells. It is also transparent. Many mutant strains of C. elegans now exist, and it is a good model for analyzing genes important in development and aging. The zebrafish (Danio rerio) is a vertebrate whose embryo is transparent. It is thus easy to spot developmental mutants. A comparison of the mutant gene with that of the human genome will help to identify genes important in human development.\\n\\nSophisticated databases have been created to store and analyze base sequence information from the genome projects. Computer programs analyze the data for exon sequences and compare the sequence of one genome to that of another. In this way sequences encoding related proteins (proteins that share stretches of similar amino acids) can be identified. Some important programs, which can be easily accessed through the Internet (see CBASC website) are BLASTN for the comparison of a nucleotide sequence to other sequences stored in a nucleotide database and BLASTP, which compares an amino acid sequence to protein sequence databases.\\n---\\n#  DNA AS A DATA STORAGE MEDIUM\\n\\nMany genes have been duplicated at some time during their evolution. Mutation over the succeeding generations causes the initially identical copies to diverge in sequence and produce what is known as a gene family. Members of a gene family usually have a related function—the immunoglobulin gene family, for example, makes antibodies. Different members of a family sometimes encode proteins that carry out the same specialized function but at different times during development. These genes then generate related proteins or isoforms, which are often distinguished by placing a Greek letter after the protein name, for example, hemoglobin α and hemoglobin β. Figure 5.6 shows the α- and β-globin gene families. The β-globin gene cluster is on human chromosome 11, and the α-globin gene cluster is on human chromosome 16. Hemoglobin is composed of two α globins and two β globins (page 204). The gene clusters encode proteins produced at specific times during development: from embryo to fetus to adult. The different globin proteins are produced at different stages of gestation to cope with the different oxygen transport requirements during development. The duplication of genes and their subsequent divergence allows the expansion of the gene repertoire, the production of new protein molecules, and the elaboration of ever more specialized gene functions during evolution.\\n\\nSome sections of DNA are very similar in sequence to other members of their gene family but do not produce mRNA. These are known as pseudogenes. There are two in the α-globin gene cluster (ψ in Fig. 5.6). Pseudogenes may be former genes that have mutated to such an extent that they can no longer be transcribed into RNA. Some pseudogenes have arisen because an mRNA molecule has inadvertently been integrated into the genome, copied back into mRNA by an enzyme called reverse transcriptase found in some viruses (page 71). Such pseudogenes are immediately recognizable because some or all of their introns were spliced out before the integration occurred. Some have the poly-A tail characteristic of intact mRNA (page 118). These are called processed pseudogenes.\\n\\nSometimes DNA that encodes RNA is repeated as a series of copies that follow one after the other along the chromosome. Such genes are said to be tandemly repeated and include the genes that code for ribosomal RNAs (about 250 copies/cell), transfer RNAs (50 copies/cell), and histone proteins (20–50 copies/cell). The products of these genes are required in large amounts.\\n\\nThis still leaves about 75% of our genome with no very clearly understood function. A large proportion of this so-called extragenic DNA is made up of repetitious DNA sequences that are repeated many times in the genome. Some sequences are repeated more than a million times and are called satellite DNA. The repeating unit is usually several hundred base pairs long, and many copies are often lined up next to each other in tandem repeats. Most of the satellite DNA is found in a region called the centromere, which plays a role in the physical movement of the chromosomes that occurs at cell division (page 403), and one theory is that it has a structural function.\\n\\nOur genome also contains minisatellite DNA where the tandem repeat is about 25 bp long. Minisatellite DNA stretches can be up to 20,000 bp in length and are often found near the ends of chromosomes, a region called the telomere. Microsatellite DNA has an even smaller repeat unit of about 4 bp or less. Again the function of these repeated sequences is unknown, but microsatellites, because their number varies between different individuals, have proved very useful in DNA testing (page 146).\\n\\nOther extragenic sequences, known as LINEs (long interspersed nuclear elements) and SINEs (short interspersed nuclear elements) occur in our genome. There are about 50,000 copies of LINEs in a mammalian genome. SINEs probably arose by the insertion of small\\n---\\n# SUMMARY\\n\\nRNAs into the genome. The most abundant SINE in the human genome is known as the Alu repeat. It occurs about one million times.\\n\\n# GENE NOMENCLATURE\\n\\nOne of the great difficulties that has arisen out of genome-sequencing projects is how to name the genes and the proteins they encode. This has not been easy and a number of committees have been set up to deal with this problem. In general, each gene is designated by an abbreviation, written in capitalized italics. For example, the human gene for flavin-containing monooxygenase is designated FMO. Because there is more than one FMO gene, we assign a number to identify the specific gene to which we are referring. The gene which when mutated gives rise to trimethylaminuria (page 82) is called FMO3. The protein encoded by the FMO3 gene is written in normal capitals, as FMO3. Similarly, cytochrome P-450 genes (page 249) are abbreviated to CYP. CYP3A4 is a gene that belongs to the CYP3 family. This family has several members so we must include additional information in the gene name to specify precisely the member of the CYP3 gene family we are referring to. The protein name is written as CYP3A4.\\n\\n# SUMMARY\\n\\n1. During replication each parent DNA strand acts as the template for the synthesis of a new daughter strand. The base sequence of the newly synthesized strand is complementary to that of the template strand.\\n2. Replication starts at specific sequences called origins of replication. The two strands untwist and form the replication fork. Helicase enzymes unwind the double helix, and single-strand binding proteins keep it unwound during replication. DNA polymerase III synthesizes the leading strand continuously in the 5\\' to 3\\' direction. The lagging strand is made discontinuously in short pieces in the 5\\' to 3\\' direction. These are joined together by DNA ligase.\\n3. DNA polymerase is a self-correcting enzyme. It can remove an incorrect base using its 3\\' to 5\\'-exonuclease activity and then replace it.\\n4. DNA repair enzymes can correct mutations. Uracil in DNA, resulting from the spontaneous deamination of cytosine, is removed by uracil–DNA glycosidase. The depyrimidinated sugar is cleaved from the sugar–phosphate backbone by AP endonuclease, and DNA polymerase then inserts the correct nucleotide. The phosphodiester link is reformed by DNA ligase.\\n5. Protein-coding genes are split into exons and introns. Only exons code for protein. The human genome has a large amount of DNA whose function is not obvious. This includes much repetitious DNA, whose sequence is multiplied many times.\\n6. Protein-coding genes may be found in repeated groups of slightly diverging structure called gene families, either close together or scattered over the genome. Some of the family members have lost the ability to operate—they are pseudogenes.\\n---\\n# DNA AS A DATA STORAGE MEDIUM\\n\\n# FURTHER READING\\n\\nBrenner, S., Elgar, G., Sandford, R., Macrae, A., Venkatesh, B., and Aparicio, S. 1993. Characterization of pe pufferfish (Fugu) genome as a compact model vertebrate genome. Nature 366: 265–268.\\nFriedberg, E. C. 2001. How nucleotide excision repair protects against cancer. Nature Rev. Cancer 1: 22–33.\\nRadman, M., and Wagner, R. 1988. The high fidelity of DNA duplication. Sci. Am. 259: 40–46.2.\\nScharer, O. D., and Jiricny, J. 2001. Recent progress in pe biology, chemistry and structural biology of DNA glycosylases. Bioessays 23: 270–281.\\n\\n# REVIEW QUESTIONS\\n\\n1. 1. When a DNA strand with the sequence 3\\' GACTAACGATGC 5\\' is used as a template on which a daughter DNA sequence is formed, the daughter strand has the sequence\\n\\nA. 3\\' CTGATTGCTACG 5\\'\\n\\nB. 5\\' GCATCGTTAGTC 3\\'\\n\\nC. 5\\' CTGATTGCTACG 3\\'\\n\\nD. 3\\' CGTAGCAATCAG 5\\'\\n\\nE. 3\\' CGUAGCAAUCAG 5\\'\\n2. 2. Okazaki fragments are\\n\\nA. short peptide sequences.\\n\\nB. RNA primers.\\n\\nC. newly synthesized stretches of DNA produced during replication of the lagging template strand.\\n\\nD. degraded fragments of DNA.\\n\\nE. the lengths of DNA between origins of replication.\\n3. 3. Following a spontaneous DNA deamination of cytosine that generates uracil, repair proceeds in the following order:\\n\\nA. Removal of the uracil from its ribose, removal of the sugar to leave a gap, addition of an appropriate new deoxyribonucleotide, reformation of the phosphodiester links.\\n\\nB. Removal of the sugar from the sugar–phosphate backbone, removal of the hydrogen-bonded uracil, addition of an appropriate new deoxyribonucleotide, reformation of the phosphodiester links.\\n\\nC. Removal of the uracil from its ribose, insertion of cytosine which hydrogen bonds to guanosine, reformation of the bond between cytosine and ribose.\\n\\nD. Addition of an appropriate new deoxyribonucleotide to the opposite side of the DNA helix, rotation of the helix by topisomerase 1, reformation of the phosphodiester links.\\n\\nE. Transamination of uracil to regenerate cytosine, checking of the conversion by self-correcting DNA polymerase.\\n4. 4. Which of the following do not play a role in replication?\\n\\nA. RNA primers\\n\\nB. Deoxynucleotides\\n\\nC. Helicase\\n\\nD. Stop codons\\n\\nE. DNA polymerase\\n---\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n|5.|During DNA replication, the new DNA is formed on the lagging strand in a series of independent short sections because|\\n|---|---|\\n| |A. DNA polymerase III can synthesize DNA in both the 3\\' to 5\\' and 5\\' to 3\\' directions but makes more mistakes when working in the 5\\' to 3\\' direction.|\\n| |B. DNA polymerase III can only operate in the 5\\' to 3\\' direction.|\\n| |C. Thymine dimers are excised from the growing strand, leaving it cut into short fragments.|\\n| |D. The hydrogen bonding between adenine and thymine is relatively weak.|\\n| |E. DnaA causes the DNA strands to melt.|\\n\\n|6.|The human genome comprises|\\n|---|---|\\n| |A. protein-coding genes.|\\n| |B. genes encoding mRNAs.|\\n| |C. sequences that do not encode genes.|\\n| |D. genes for ribosomal RNAs and transfer RNAs.|\\n| |E. all of the above.|\\n\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n1. C. 5\\' CTGATTGCTACG 3\\'—DNA is always synthesized in the 5\\' to 3\\' direction, and the sequence of the newly synthesized strand is complementary to that of the template strand.\\n\\n2. C. Okazaki fragments are newly synthesized stretches of DNA produced during replication of the lagging template strand.\\n\\n3. A. Answers B, C, and E are more or less topologically and chemically plausible but are not the way repair proceeds. Answer D is nonsense.\\n\\n4. D. Stop codons are used to halt protein synthesis during translation of mRNA. None of the DNA transcription machinery takes any special notice of bases that will generate a stop codon in mRNA; they are copied in exactly the same way as the rest of the genome.\\n\\n5. B. Because DNA polymerase can only proceed in the 5\\' to 3\\' direction, the lagging strand must be replicated as a series of short sections called Okazaki fragments.\\n\\n6. E. All are correct statements; the genome represents all the information on the DNA.\\n---\\n# TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\n\\nTranscription (or RNA synthesis) is the process whereby the information held in the nucleotide sequence of DNA is transferred to RNA. The three major classes of RNA are ribosomal RNA (rRNA), transfer RNA (tRNA), and messenger RNA (mRNA). All play key roles in protein synthesis. Genes encoding mRNAs are known as protein-coding genes. A gene is said to be expressed when its genetic information is transferred to mRNA and then to protein. Two important questions are addressed in this chapter: how is RNA synthesized, and what factors control how much is made?\\n\\n# STRUCTURE OF RNA\\n\\nRibonucleic acid is a polymer made up of monomeric nucleotide units. RNA has a chemical structure similar to that of DNA, but there are two major differences. First, the sugar in RNA is ribose instead of deoxyribose. Second, although RNA contains the two purine bases adenine and guanine and the pyrimidine cytosine, the fourth base is different. The pyrimidine uracil (U) replaces thymine. The building blocks of RNA are therefore the four ribonucleoside triphosphates adenosine 5′-triphosphate, guanosine 5′-triphosphate, cytidine 5′-triphosphate, and uridine 5′-triphosphate. These four nucleotides are joined together by phosphodiester links. Like DNA, the RNA chain has direction. In the first nucleotide in the chain, the 5′ carbon of the ribose is phosphorylated and is available for the formation of phosphodiester links. This is called the 5′ end of the RNA chain. At the\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C2004 by John Wiley & Sons, Inc.\\n---\\n# TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\n\\nHOCH5′ 2 O OH H N C C C 4′C H H C1′ O C N C H H C 3′ C 2′ H OH OH ribose uracil (U)\\n\\nThe RNA contains the sugar ribose and the base uracil in place of deoxyribose and thymine. At one end is a phosphate group attached to the 5′ carbon of the ribose, and at the other end is a ribose with a free hydroxyl group on its 3′ carbon. This is called the 3′ end. RNA molecules are single-stranded along much of their length, although they often contain regions that are double-stranded due to intramolecular base pairing.\\n\\n# RNA POLYMERASE\\n\\nIn any gene, only one DNA strand acts as the template for transcription. The sequence of nucleotides in RNA depends on their sequence in the DNA template. The bases T, A, G, and C in the DNA template will specify the bases A, U, C, and G, respectively, in RNA. DNA is transcribed into RNA by the enzyme RNA polymerase. Transcription requires that this enzyme recognize the beginning of the gene to be transcribed and catalyze the formation of phosphodiester links between nucleotides that have been selected according to the sequence within the DNA template.\\n\\n# GENE NOTATION\\n\\nFigure 6.3 shows the notation used in describing the positions of nucleotides within and adjacent to a gene. The nucleotide in the template strand at which transcription begins is designated with the number +1. Transcription proceeds in the downstream direction, and nucleotides in the transcribed DNA are given successive positive numbers. Downstream sequences are drawn, by convention, to the right of the transcription start site. Nucleotides that lie to the left of this site are called the upstream sequences and are identified by negative numbers.\\n\\n# BACTERIAL RNA SYNTHESIS\\n\\nEscherichia coli genes are all transcribed by the same RNA polymerase. This enzyme is made up of five subunits (polypeptide chains). The subunits are named α (there are two of these), β, β\\', and σ. Each of the subunits has its own job to do in transcription. The role of the sigma (σ) factor is to recognize a specific DNA sequence called the promoter, which lies just upstream of the gene to be transcribed. E. coli promoters contain two important regions. One centered around nucleotide −10 usually has the sequence TATATT.\\n---\\n# Figure 6.2. Synthesis of an RNA strand.\\n\\n|O|γP|O|βP|O|αP|O|CH2|O|\\n|---|---|---|---|---|---|---|---|---|\\n|O| |O| |O| |O|C|H|\\n| | |O| | | |O|C3′|H|\\n| | | | | |OH| |OH| |\\n\\nO P O P O\\n\\nO O RNA\\n\\npyrophosphate polymerase\\n\\n3′\\n\\n|O|γP|O|βP|O|αP|O|CH2|O|\\n|---|---|---|---|---|---|---|---|---|\\n|O| |O| |O| |O|C|H|\\n| | |O| | | |O|C3′|H|\\n| | | | | |OH| |OH| |\\n\\n# Figure 6.3. Numbering of a DNA sequence.\\n\\nDNA upstream downstream\\n\\n–35 box –10 box –1 +1\\n\\nTTGACApromoter TATATTT direction of RNA synthesis\\n---\\n# TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\n\\n**Figure 6.4.** (a) RNA polymerase binds to the promoter. (b) DNA helix unwinds and RNA polymerase synthesizes an RNA molecule.\\n\\nDNA template -35 box -10 box +1 transcription\\n\\nσ factor RNA polymerase\\n\\n(a)\\n\\n|RNA 5′|Unwinding of DNA molecule|\\n|---|---|\\n|DNA template 5′|3′ transcription bubble 5′|\\n|3′|Rewinding of DNA molecule|\\n|RNA polymerase|(b)|\\n\\nThis sequence is called the -10 box (or the Pribnow box). The second, centered near nucleotide -35 often has the sequence TTGACA. This is the -35 box. On binding to the promoter sequence (Fig. 6.4a), the σ factor brings the other subunits (two of α plus one each of β and β\\') of RNA polymerase into contact with the DNA to be transcribed. This forms the closed promoter complex. For transcription to begin, the two strands of DNA must separate, enabling one strand to act as the template for the synthesis of an RNA molecule. This formation is called the open promoter complex. The separation of the two DNA strands is helped by the AT-rich sequence of the -10 box. There are only two hydrogen bonds between the bases adenine and thymine; thus it is relatively easy to separate the two strands at this point. DNA unwinds and rewinds as RNA polymerase advances along the double helix, synthesizing an RNA chain as it goes. This produces a transcription bubble (Fig. 6.4b). The RNA chain grows in the 5\\' to 3\\' direction, and the template strand is read in the 3\\' to 5\\' direction (Fig. 6.4). When the RNA chain is about 10 bases long, the σ factor is released from RNA polymerase and plays no further role in transcription. The β subunit of RNA polymerase\\n---\\n# CONTROL OF BACTERIAL GENE EXPRESSION\\n\\nbinds ribonucleotides and joins them together by catalyzing the formation of phosphodiester\\nlinks as it moves along the DNA template. The β               ′ subunit helps to keep the RNA polymerase\\nattached to DNA. The two              α  subunits are important as they help RNA polymerase to\\nassemble on the promoter (see discussion of the lac operon below).\\n\\nRNA polymerase has to know when it has reached the end of a gene. Escherichia coli has\\nspecific sequences, called terminators, at the ends of its genes that cause RNA polymerase\\nto stop transcribing DNA. A terminator sequence consists of two regions rich in the bases\\nG and C that are separated by about 10 bp. This sequence is followed by a stretch of A\\nbases. Figure 6.5 shows how the terminator halts transcription. When the GC-rich regions\\nare transcribed, a hairpin loop forms in the RNA with the first and second GC-rich regions\\naligning and pairing up. Formation of this structure within the RNA molecule causes the\\ntranscription bubble to shrink because where the template DNA strand can no longer bind\\nto the RNA molecule it reconnects to its sister DNA strand. The remaining interactions\\nbetween the adenines in the DNA template and the uracils in the RNA chain have only\\ntwo hydrogen bonds per base pair and are therefore too weak to maintain the transcription\\nbubble. The RNA molecule is then released, transcription terminates, and the double helix\\nreforms. This type of transcription termination is known as rho-independent termination.\\n\\nSome E. coli genes contain different terminator sites. These are recognized by a protein,\\nknown as rho, which frees the RNA from the DNA. In this case transcription is terminated\\nby a process known as rho-dependent termination.\\n\\n# CONTROL OF BACTERIAL GENE EXPRESSION\\n\\nMany bacterial proteins are always present in the cell in a constant amount. However, the\\namount of other proteins is regulated by the presence or absence of a particular nutrient. To\\ngrow and divide and make the most efficient use of the available nutrients, bacteria have to\\nadjust quickly to changes in their environment. They do this by regulating the production of\\nproteins required for either breakdown or synthesis of a particular compound. Gene expres-\\nsion in bacteria is controlled mainly at the level of transcription. This is because bacterial\\ncells have no nuclear envelope, and RNA synthesis and protein synthesis are not separate but\\noccur simultaneously. This is one reason why bacteria lack the more sophisticated control\\nmechanisms that regulate gene expression in eukaryotes.\\n\\nEach bacterial promoter usually controls the transcription of a cluster of genes coding\\nfor proteins that work together on a particular task. This collection of related genes is called\\nan operon and is transcribed as a single mRNA molecule called a polycistronic mRNA.\\nAs shown in Figure 6.6, translation of this mRNA produces the required proteins because\\nthere are several start and stop codons for protein synthesis along its length. Each start and\\nstop codon (page 79) specifies a region of RNA that will be translated into one particular\\nprotein. The organization of genes into operons ensures that all the proteins necessary to\\nmetabolize a particular compound are made at the same time and hence helps bacteria to\\nrespond quickly to environmental changes.\\n\\nThe three major factors involved in regulating how much RNA is made are (1) nucleotide\\nsequences within or flanking a gene, (2) proteins that bind to these sequences, and (3)\\nthe environment. The human intestine contains many millions of E. coli cells that must\\nrespond very quickly to the sudden appearance of a particular nutrient. For instance, most\\nfoods do not contain the disaccharide lactose (Fig. 6.7), but milk contains large amounts.\\nWithin minutes of our drinking a glass of milk, E. coli in our intestines start to produce the\\nenzyme β-galactosidase that cleaves lactose to glucose and galactose (Fig. 6.7). In general,\\n---\\n# TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\n\\n|||\\n|---|\\n|5′AA U G AG A G G CCG 3′ AUAUGGCCUUUUU 3′|5′ CC GGAAAAA 3′|\\n|5′|3′|\\n\\n(a)\\n\\n|U A|AG CU RNA hairpin loop|\\n|---|---|\\n|5′ AA C G|3′ C G|\\n|UG G C AAAA|A GAG CU UUUU|\\n|5′|3′|\\n\\ntranscription bubble shortens\\n\\n(b)\\n\\n|AU AU|G C|\\n|---|---|\\n|C G RNA chain|released|\\n|C G|G C|\\n|AAUGAG A G C U UUUU|DNA double helix reforms|\\n|3′|5′|\\n\\n(c)\\n\\nFigure 6.5. Transcription termination in Escherichia coli.\\n---\\n# CONTROL OF BACTERIAL GENE EXPRESSION\\n\\n|promoter|gene A|gene B|gene C|\\n|---|---|---|---|\\n| |AUG start|AUG start|AUG start|\\n| |stop|stop|stop|\\n| |polycistronic mRNA|polycistronic mRNA|polycistronic mRNA|\\n| |translation produces three proteins|translation produces three proteins|translation produces three proteins|\\n| |protein A|protein B|protein C|\\n\\nFigure 6.6. A bacterial operon is transcribed into a polycistronic mRNA.\\n\\nThe substrates of β-galactosidase are compounds like lactose that contain a β-galactoside linkage and are therefore called β-galactosides.\\n\\nlac, an Inducible Operon\\n\\nβ-Galactosidase is encoded by one of the genes that make up the lactose (lac) operon, which is shown in Figure 6.8. The operon contains three protein-coding genes called lac z, lac y, and lac a. β-Galactosidase is encoded by the lac z gene. As noted before, gene names are always italicized, while the protein product is always in standard type. Lac y encodes β-galactoside permease, a carrier (page 316) that helps lactose get into the cell. The lac a gene codes for transacetylase. This protein is thought to remove compounds that have a structure similar to lactose but that are not useful to the cell.\\n\\nIn the absence of β-galactoside compounds like lactose, there is no need for E. coli to produce β-galactosidase or β-galactoside permease, and the cell contains only a few molecules of these proteins. The lac operon is said to be inducible because the rate of transcription into RNA increases greatly when a β-galactoside is present. How is the transcription of the lac z, lac y, and lac a genes switched on and off? A repressor protein (the product of the lac i gene) binds to a sequence in the lac operon known as the operator. The operator lies next to the promoter so that, when the repressor is bound, RNA polymerase is unable to bind to the promoter. In the absence of a β-galactoside, the lac operon spends most of its time in the state shown in Figure 6.8a. The repressor is bound to the operator, RNA polymerase cannot bind, and no transcription occurs. Only for the small fraction of time\\n---\\n# TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\n\\n| | | | | | | | | | | | | |\\n|---|---|---|---|---|---|---|---|---|---|---|---|---|\\n| | | | | | | | | | | | | |\\n| | | | | | | | | | | | | |\\n\\nFigure 6.7. Reactions catalyzed by β-galactosidase.\\n---\\n# CONTROL OF BACTERIAL GENE EXPRESSION\\n\\n|lac i|promoter|operator|lac z|lac y|lac a|\\n|---|---|---|---|---|---|\\n|repressor|repressor|repressor|repressor|repressor|repressor|\\n|active repressor protein|(a) no β-galactoside sugars present—operon repressed|(a) no β-galactoside sugars present—operon repressed|(a) no β-galactoside sugars present—operon repressed|(a) no β-galactoside sugars present—operon repressed|(a) no β-galactoside sugars present—operon repressed|\\n\\n|lac i|promoter|operator|lac z|lac y|lac a|\\n|---|---|---|---|---|---|\\n|RNA polymerase|RNA polymerase|RNA polymerase|RNA polymerase|RNA polymerase|RNA polymerase|\\n|inactive repressor protein|ribosome|ribosome|ribosome|ribosome|ribosome|\\n\\ninducer\\n(e.g. allolactose)\\n(e.g. allolactose)\\n(e.g. allolactose)\\n(e.g. allolactose)\\n(e.g. allolactose)\\n\\nβ-galactosidase\\npermease\\n(b) β-galactoside sugars present—operon derepressed\\ntransacetylase\\n\\nFigure 6.8. Transcription of the lac operon requires the presence of an inducer.\\n\\nthat the operator is unoccupied by the repressor can RNA polymerase bind and generate mRNA. Thus in the absence of a β-galactoside, only very small amounts of β-galactosidase, β-galactoside permease, and transacetylase are synthesized. If lactose appears, it is converted to an isomer called allolactose. This conversion is carried out by β-galactosidase (Fig. 6.7); as we have seen, a small amount of β-galactosidase is made even when β-galactoside is absent. The repressor protein has a binding site for allolactose and undergoes a conformational change when bound to this compound (Fig. 6.8b). This means that the repressor is no longer able to bind to the operator. The way is then clear for RNA polymerase to bind to the promoter and to transcribe the operon. Thus in\\n---\\nTRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\n\\na short time the bacteria produce the proteins necessary for utilizing the new food source. The concentration of the substrate (lactose in this case) determines whether or not mRNA is synthesized. The lac operon is said to be under negative regulation by the repressor protein. The transcription of the lac operon is controlled not only by the repressor protein but also by another protein, the catabolite activator protein (CAP). If both glucose and lactose are present, it is more efficient for the cell to use glucose as the carbon source because the utilization of glucose requires no new RNA and protein synthesis, all the proteins necessary being already present in the cell. Only in the absence of glucose, therefore, does E. coli transcribe the lac operon at a high rate. This control operates through an intracellular messenger molecule called cyclic adenosine monophosphate (cyclic AMP) (Fig. 6.9). When glucose concentrations are low, the concentration of cyclic AMP increases. Cyclic AMP binds to CAP, and the complex then binds to a sequence upstream of the lac operon promoter (Fig. 6.10) where it has a remarkable effect. The DNA surrounding CAP bends by about 90°. Both α subunits of RNA polymerase are now able to make contact with CAP at the same time so that the affinity of RNA polymerase for the lac promoter is increased. The result is that now the lac z, lac y, and lac a genes can be transcribed very efficiently. The lac operon is said to be under positive regulation by the CAP–cAMP complex. To recap, the control of the lac operon is not simple. Several requirements need to be met before it can be transcribed. The repressor must not be bound to the operator, and the CAP–cyclic AMP complex and RNA polymerase must be bound to their respective DNA binding sites. These requirements are only met when glucose is absent and a β-galactoside, such as the sugar lactose, is present. Other compounds such as isopropylthio-β- D-galactoside (IPTG) (Fig. 6.11) can bind to the repressor but are not metabolized. These gratuitous inducers are very useful in DNA research and in biotechnology. Chapter 7 deals with this and with some of the industrial applications of the lac operon.\\n\\nFigure 6.9. Cyclic adenosine monophosphate, also called cyclic AMP or just cAMP.\\n---\\n# low glucose\\n\\nATP\\n\\ncAMP\\n\\n|lac i|CAP|promoter|operator|lac z|lac y|lac a|\\n|---|---|---|---|---|---|---|\\n|RNA polymerase|RNA polymerase|RNA polymerase|RNA polymerase|RNA polymerase|RNA polymerase|RNA polymerase|\\n\\ninactive repressor protein\\n\\ninducer (e.g. allolactose)\\n\\nβ-galactosidase\\n\\npermease\\n\\ntransacetylase\\n\\nFigure 6.10. For efficient transcription of the lac operon, both cAMP and a β-galactoside sugar must be present.\\n\\nCH OH CH2\\n\\nOH C O S C CH3\\n\\nC H H C H\\n\\nOH C C H\\n\\nH C C H\\n\\nH OH\\n\\nIPTG (isopropyl thio-β-D-galactoside)\\n\\nFigure 6.11. Isopropylthio-β- D -galactoside (IPTG), which can bind to the lac repressor protein but which is not metabolized.\\n\\n115\\n---\\n# TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\n\\nExample 6.1 Quorum Sensing: Squids That Glow in the Dark\\n\\nThe bacterium Vibrio fischeri lives free in seawater but is also found at high densities in the light-emitting organs of the nocturnal squid Euprymna scolopes, where it synthesizes an enzyme called luciferase that generates light. This phenomenon is called bioluminescence. When living free in seawater V. fischeri synthesizes almost no luciferase; indeed, there would be little point in doing so because the light emitted by a single bacterium would be too dim for anything to see. V. fischeri only begins making lots of luciferase when the density of bacteria is high—just as it is in the squid light organs. The word quorum is defined in Webster’s Dictionary as “the number of . . . members of a body that when duly assembled is legally competent to transact business.” Thus quorum sensing is a good description of V. fischeri’s behavior. How does it work?\\n\\nLuciferase is a product of the lux A and lux B genes in a bacterial operon called the lux operon. A region in the lux operon promoter binds a transcription factor, LuxR, which is only active when it has bound a small, uncharged molecule called N -acyl-HSL (also called VAI for V. fischeri autoinducer). N -acyl-HSL in turn is made by an enzyme called N -acyl-HSL synthase (or VAI synthase) that is encoded by the gene lux I, which is part of the lux operon. In free-living V. fischeri the lux operon is transcribed at a low level. Small amounts of N -acyl-HSL are made, which immediately leak out of the cell into the open sea without binding to luxR. When the bacterium is concentrated in the squid’s light organs, then some N -acyl-HSL binds to LuxR, increasing transcription of the lux operon. This makes more luciferase—but it also makes more N -acyl-HSL synthase. The concentration of N -acyl-HSL therefore rises—so transcription of the lux operon increases further. This means that the genes for N -acyl-HSL synthase, luciferase, and the enzymes that produce the substrate for luciferase are now transcribed at a high rate. This autoinduction of the lux operon by N -acyl-HSL is a form of positive feedback (page 303). When luciferase carries out its reaction, the squid will luminesce at the intensity of moonlight so that as it glides at night over the coral reefs of Hawaii it does not cast a dark shadow; thus its prey are less likely to notice its presence.\\n\\n|lux R|operator|promoter|lux I|lux A|lux B|\\n|---|---|---|---|---|---|\\n|inactive transcription factor|N-acyl-HSL| |N-acyl HSL synthase|Luciferase| |\\n\\ntrp, a Repressible Operon\\n\\nOperons that code for proteins that synthesize amino acids are regulated in a different way from the lac operon. These operons are only transcribed if the amino acid is not present, and transcription is switched off if there is already enough of the amino acid around. In this\\n---\\n# CONTROL OF BACTERIAL GENE EXPRESSION\\n\\n|trp R|promoter operator|trp E|trp D|trp C|trp B|trp A|\\n|---|---|---|---|---|---|---|\\n| |RNA polymerase| | | | | |\\n\\naporepressor (a) no tryptophan; operon transcribed\\n\\n|trp R|promoter operator|trp E|trp D|trp C|trp B|trp A|\\n|---|---|---|---|---|---|---|\\n| |repressor| | | | | |\\n| | |no transcription| | | | |\\n\\naporepressor + tryptophan corepressor (b) tryptophan present; operon repressed\\n\\nFigure 6.12. Transcription of the trp operon is controlled by the concentration of the amino acid tryptophan. (a) No tryptophan, operon transcribed. (b) Tryptophan present, operon repressed.\\n\\nway the cell carefully controls the concentration of free amino acids. The tryptophan (trp) operon is made up of five structural genes encoding enzymes that synthesize the amino acid tryptophan (Fig. 6.12). This is a repressible operon. The cell regulates the amount of tryptophan produced by preventing transcription of the trp operon mRNA when there is sufficient tryptophan about. As with the lac operon, the transcription of the trp operon is controlled by a regulatory protein. The gene trp R encodes an inactive repressor protein that is called an aporepressor. Tryptophan binds to this to produce an active repressor complex and is therefore called the corepressor. The active repressor complex binds to the operator sequence of the trp operon and prevents the attachment of RNA polymerase to the trp promoter sequence. Therefore, when the concentration of tryptophan in the cell is high, the active repressor complex will form, and transcription of the trp operon is prevented. However, when the amount of tryptophan in the cell decreases, the active repressor complex cannot be formed. RNA polymerase binds to the promoter, transcription of the trp operon\\n---\\n# TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\n\\nproceeds, and the enzymes needed to synthesize tryptophan are produced. This is an example of negative feedback (page 303). Many other operons are regulated by similar mechanisms in which specific regulatory proteins interact with specific small molecules.\\n\\n# EUARYOTIC RNA SYNTHESIS\\n\\nEukaryotes have three types of RNA polymerase. RNA polymerase I transcribes the genes that code for most of the ribosomal RNAs. All messenger RNAs are synthesized using RNA polymerase II. Transfer RNA genes are transcribed by RNA polymerase III. This last enzyme also catalyzes the synthesis of several small RNAs including the 5S ribosomal RNA. The chemical reaction catalyzed by these three RNA polymerases, the formation of phosphodiester links between nucleotides, is the same in eukaryotes and bacteria.\\n\\n# Messenger RNA Processing\\n\\nA newly synthesized eukaryotic mRNA undergoes several modifications before it leaves the nucleus (Fig. 6.13). The first is known as capping. Very early in transcription the 5 -terminal triphosphate group is modified by the addition of a guanosine via a 5 -5 -phosphodiester link. The guanosine is subsequently methylated to form the 7-methyl guanosine cap. The 3′ ends of nearly all eukaryotic mRNAs are modified by the addition of a long stretch of adenosine residues, the poly-A tail (Fig. 6.13). A sequence AAUAAA is found in most eukaryotic mRNAs about 20 bases from where the poly-A tail is added and is probably a signal for the enzyme poly-A polymerase to bind and to begin the polyadenylation process. The length of the poly-A tail varies, it can be as long as 250 nucleotides. Unlike DNA, RNA is an unstable molecule, and the capping of eukaryotic mRNAs at their 5 ′ ends and the addition of a poly-A tail to their 3 ′end increases the lifetime of mRNA molecules by protecting them from digestion by nucleases. Many eukaryotic protein-coding genes are split into exon and intron sequences. Both the exons and introns are transcribed into mRNA. The introns have to be removed and the exons joined together by a process known as RNA splicing before the mRNA can be used to make protein. Removal of introns takes place within the nucleus. Splicing is complex and not yet fully understood. It has, however, certain rules. Within an mRNA the first two bases following an exon are always GU and the last two bases of the intron are AG. Several small nuclear RNAs (snRNAs) are involved in splicing. These are complexed with a number of proteins to form a structure known as the spliceosome. One of the snRNAs is complementary in sequence to either end of the intron sequence. It is thought that binding of this snRNA to the intron, by complementary base pairing, brings the two exon sequences together, which causes the intron to loop out (Fig. 6.13). The proteins in the spliceosome remove the intron and join the exons together. Splicing is the final modification made to the mRNA in the nucleus. The mRNA is now transported to the cytoplasm for protein synthesis. As well as removing introns, splicing can sometimes remove exons in a process called alternative splicing. This allows the same gene to give rise to different proteins at different times or in different cells. For example, alternative splicing of the gene for the molecular motor dynein produces motors that transport different types of cargo (page 390).\\n---\\n# CONTROL OF EUKARYOTIC GENE EXPRESSION\\n\\nSince most eukaryotes are multicellular organisms with many cell types, gene expression must be controlled so that different cell lineages develop differently and remain different. A brain cell is quite different from a liver cell because it contains different proteins even though the DNA in the two cell types is identical. During development and differentiation, different sets of genes are switched on and off. Hemoglobin, for example, is only expressed in developing red blood cells even though the globin genes are present in all types of cell. Genetic engineering technology (Chapter 7) has made the isolation and manipulation of eukaryotic genes possible. This has given us some insight into the extraordinarily complex\\n\\n5′ exon 1 intron 1 exon 2 3′\\n\\n|precursor mRNA|transcription|\\n|---|---|\\n|5′ m Gppp|3′|\\n| |polyadenylation|\\n\\n5′ m Gppp A A U A A A AAAA 3\\' poly A tail\\n\\nspliceosome formation polyadenylation \"signal\"\\n\\nintron loops out\\n\\n5′ m Gppp U spliceosome G A G AAAA 3\\'\\n\\n5′ m Gppp exon 1 exon 2 processed mRNA AAAA 3\\'\\n\\nnucleus cytoplasm\\n---\\n# TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\n\\nProcesses that regulate transcription of eukaryotic genes and allow a fertilized egg to develop into a multicellular, multitissue adult.\\n\\nUnlike the situation in bacteria, the eukaryotic cell is divided by the nuclear envelope into nucleus and cytoplasm. Transcription and translation are therefore separated in space and in time. This means that the expression of eukaryotic genes can be regulated at more than one place in the cell. Although gene expression in eukaryotes is controlled primarily by regulating transcription in the nucleus, there are many instances in which expression is controlled at the level of translation in the cytoplasm or by altering the way in which the primary mRNA transcript is processed.\\n\\nThe interaction of RNA polymerase with its promoter is far more complex in eukaryotes than it is in bacteria. This section describes how the transcription of a gene, encoding mRNA, is transcribed by RNA polymerase II. In contrast to bacterial RNA polymerase, RNA polymerase II cannot recognize a promoter sequence. Instead, other proteins known as transcription factors bind to the promoter and guide RNA polymerase II to the beginning of the gene to be transcribed.\\n\\nThe promoter sequence of most eukaryotic genes encoding mRNAs contains an AT-rich region about 25 bp upstream of the transcription start site. This sequence, called the TATA box, binds a protein called the transcription factor IID (TFIID), one of whose subunits is called the TATA-binding protein, or TBP (Fig. 6.14a). Several other transcription factors (TFIIA, TFIIB, TFIIE, TFIIF, and TFIIH) then bind to TFIID and to the promoter region (Fig. 6.14b). TFIIF is the protein that guides RNA polymerase II to the beginning of the gene to be transcribed. The complex formed between the TATA box, TFIID, the other transcription factors, and RNA polymerase is known as the transcription preinitiation complex. Note that the proteins with the prefix TFII are so named because they are transcription factors that help RNA polymerase II to bind to promoter sequences.\\n\\nAlthough many gene promoters contain a TATA box, some do not. These TATA-less genes usually encode proteins that are needed in every cell and are hence called housekeeping genes. The promoters of these genes contain the sequence GGGGCGGGGC, called the GC box. A protein called Sp1 binds to the GC box and is then able to recruit TATA-binding protein to the DNA even though there is no TATA box for the latter to bind to. TATA-binding protein then recruits the rest of the transcription preinitiation complex so that transcription can proceed.\\n\\nIn either case, transcription begins when the carboxy-terminal domain of RNA polymerase II is phosphorylated. This region is rich in the amino acids serine and threonine each of which contains an OH group in their side chain. When these OH groups are phosphorylated (pages 190, 252), RNA polymerase II breaks away from the preinitiation complex and proceeds to transcribe DNA into mRNA (Fig. 6.14c).\\n\\nAlthough the formation of the initiation protein complex is sometimes enough to produce a few molecules of RNA, the binding of other proteins to sequences next to the gene greatly increases the rate of transcription producing much more mRNA. These proteins are also called transcription factors, and the DNA sequences to which they bind are called enhancers, so named because their presence enhances transcription. Enhancer sequences often lie upstream of a promoter, but they have also been found downstream. Enhancer sequences and the proteins that bind to them play an important role in determining whether a particular gene is to be transcribed. Some transcription factors bind to a gene to ensure that it is transcribed at the right stage of development or in the right tissue. Figure 6.15 shows how one gene can be transcribed in skeletal muscle but not\\n---\\n# CONTROL OF EUKARYOTIC GENE EXPRESSION\\n\\n**Figure 6.14.** RNA polymerase is guided to the promoter by TFII accessory proteins.\\n\\nIn the liver simply because of the presence or absence of proteins that bind to enhancer sequences.\\n\\n**Glucocorticoids Cross the Cell Membrane to Activate Transcription**\\n\\nGlucocorticoids are steroid hormones produced by the adrenal cortex that increase the transcription of several genes important in carbohydrate and protein metabolism. Because they are uncharged and relatively nonpolar, steroid hormones can pass through the plasma membrane by simple diffusion to enter the cytosol. Here they encounter a class of transcription factors that have a binding site for steroid hormones and are therefore called steroid hormone receptors (Fig. 6.16). In the absence of glucocorticoid hormone, its receptor remains.\\n---\\n# TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\n\\n**Figure 6.15. Tissue-specific transcription.**\\n\\n|skeletal muscle|NFAT, a transcription factor|myosin IIa gene transcription|\\n|---|---|---|\\n|liver|enhancer sequence|promoter - no transcription|\\n\\nIn the cytosol and is inactive because it is complexed to two molecules of an inhibitor protein known as Hsp90. However, when the glucocorticoid hormone enters the cell and binds to its receptor, the Hsp90 protein is displaced. The targeting sequence (page 215) that targets the receptor to the nucleus is uncovered, and the glucocorticoid receptor-hormone complex can now move into the nucleus. Here, two molecules of the complex bind to a plasma membrane.\\n\\n|cytoplasm|glucocorticoid receptor hormone complex - inhibitory protein released|\\n|---|---|\\n| |targeting sequence exposed|\\n| |inhibitory protein (hsp 90)|\\n\\n**nucleus**\\nreceptor protein is now able to bind to DNA\\n\\n**TFIID**\\nH R E TATA box RNA transcription follows polymerase II mRNA\\n\\n**Figure 6.16. The steroid hormone receptor acts to increase gene transcription in the presence of hormone.**\\n---\\n# CONTROL OF EUKARYOTIC GENE EXPRESSION\\n\\nA  5′            T   3′                    to rest of glucocorticoid\\nG A     T      C           Zn ++               hormone receptor protein\\n\\nG  T         A      C\\nT                    A\\nN              N                          ++++++\\nN     N                         ZnZnZn\\nN            N                       Zn ++++++\\nG   T            C     A                          ZnZn\\nT     A        Zn  ++\\n\\nG  AA        T      C  T                        to rest of glucocorticoid\\n5′                  3′                     hormone receptor protein\\n\\n**Figure 6.17. The palindromic HRE binds to dimerized steroid hormone receptor.**\\n\\n15-bp sequence known as the hormone response element (HRE) that lies upstream of the\\nTATA box. The HRE is an enhancer sequence. The glucocorticoid receptor–hormone com-\\nplex interacts with the preinitiation complex bound to the TATA box, and the rate at which\\nRNA polymerase transcribes genes containing the HRE is increased.\\nFigure 6.17 shows why the glucorticoid hormone receptor binds to DNA as a dimer. The\\nHRE is a palindrome—the sequence on both the top and bottom strands is the same when\\nread in the 5   ′to 3 ′direction. Each strand of the HRE has the 6-bp sequence 5               ′ AGAACA 3  ′\\nthat is known as the core recognition motif. This is the sequence to which a single receptor\\nmolecule binds. Because the HRE contains two recognition motifs, it binds two molecules\\nof glucocorticoid receptor. The two 6-bp sequences are separated by 3 base pairs, which\\npresumably are there to provide sufficient space for the receptor homodimer to fit snugly\\non the double helix. The glucocorticoid receptor is unaffected by the identity of these three\\nbase pairs, so the nucleotides are simply indicated as N in Fig. 6.17.\\nChemicals that are released from one cell and that alter the behavior of other cells\\nare called transmitters. Glucocorticoids are an example of transmitters that alter gene\\ntranscription. The cells of multicellular organisms turn the transcription of genes on and\\noff in response to many extracellular chemicals. Unlike steroid hormones, most of these\\ntransmitters cannot enter the cell and must activate intracellular messenger systems inside\\nthe cell that in turn carry the signal onward from the plasma membrane to the nucleus\\n(Chapter 16).\\n---\\n# TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\n\\nExample 6.2 Glucocorticoid Hormones Can Repress Transcription: Rheumatoid Arthritis\\n\\nGlucocorticoid treatment has been shown to give some relief to individuals who suffer from the debilitating disorder rheumatoid arthritis. Collagenase, an enzyme that digests collagen, is generated in the joints of these patients, causing destruction of the extracellular matrix and therefore chronic inflammation. Transcription of the collagenase gene is controlled by an enhancer sequence called the AP1 site. For transcription to occur, the enhancer must be occupied by a transcription factor called AP1, which, like the active glucocorticoid receptor, is a dimer. Glucocorticoid hormones inhibit transcription of the collagenase gene by an ingenious mechanism. The glucocorticoid hormone, on entering the cell interacts with the glucocorticoid receptor as shown in Figure 6.16. The receptor–hormone complex moves to the nucleus and binds to the proteins that would otherwise dimerize to form AP1. The heterodimer is unable to activate transcription of the collagenase gene. By depleting the pool of AP1 subunits, the glucocorticoid receptor–hormone complex prevents transcription of the collagenase gene.\\n\\n# Medical Relevance St. John’s Wort and the Pill\\n\\nWhen we ingest plant products and other chemicals foreign to us, the body responds by increasing the amounts of a group of proteins known as the cytochromes P450 (CYPs). This remarkable response, found largely in the liver, is our built-in detoxication system. The foreign chemical signals the appropriate CYP gene to activate, transcription takes place, and more CYP protein is produced. The CYP protein then metabolizes the foreign chemical so that it will be cleared from the body quickly and efficiently through the urine or feces. To the liver, a medicinal drug is just another foreign chemical, and the cytochrome P450 system is used to destroy it.\\n\\nSt. John’s wort is a herbal remedy, taken by many as a natural antidepressant. One of the active components is the chemical hyperforin, which has been shown to decrease the effectiveness of several therapeutic drugs including the oral contraceptive pill, the HIV protease inhibitor indinavir, and immunosuppressants such as cyclosporin. This is because each of these drugs is metabolized by a cytochrome P450 called CYP3A4, and the gene for CYP3A4 is activated by hyperforin. Because the activation of the CYP3A4 gene results in more CYP3A4 protein, the prescribed drug, in a person also taking St. John’s wort, is metabolized more rapidly, cleared from the body quickly, and is therefore less effective.\\n\\nThe transcription of the CYP3A4 gene is increased because hyperforin passes to the nucleus and binds to a receptor called the pregnane X receptor (PXR). The receptor–hyperforin complex is then able to dimerize with another receptor called RXR. The dimer binds to an enhancer sequence of the CYP3A4 gene, interacts with the transcription preinitiation complex, and triggers RNA polymerase II to begin transcription of CYP3A4 mRNA.\\n\\nThis activation of the CYP3A4 gene by St. John’s wort means that this herbal remedy should be taken with caution if a patient has been prescribed a therapeutic drug. CYP3A4 is responsible for the metabolism of about 80% of currently prescribed drugs.\\n---\\n# SUMMARY\\n\\n1. DNA is transcribed into RNA by the enzyme RNA polymerase. The three types of RNA are ribosomal RNA (rRNA), transfer RNA (tRNA), and messenger RNA (mRNA). Uracil, adenine, cytosine, and guanine are the four bases in RNA.\\n\\n2. In bacteria, RNA polymerase binds to the promoter sequence just upstream of the start site of transcription. The enzyme moves down the DNA template and synthesizes an RNA molecule. RNA synthesis stops once the enzyme has transcribed a terminator sequence.\\n\\n3. Bacterial genes encoding proteins for the same metabolic pathway are often clustered into operons. Some operons are induced in the presence of the substrate of their pathway, for example, the lac operon. Others are repressed in the presence of the product of the pathway, for example, the trp operon.\\n\\n4. Eukaryotic mRNAs are modified by the addition of a 7-methyl-guanosine cap at their 5′ end. A poly-A tail is added to their 3′ end. Intron sequences are removed, and the exon sequences joined together by the process known as splicing. The fully processed mRNA is then ready for transport to the cytoplasm and protein synthesis.\\n\\n5. In eukaryotes, there are three RNA polymerases—RNA polymerases I, II, and III. RNA polymerase II needs the help of the TATA-binding protein and other transcription factors to become bound to a promoter. This group of proteins is called the transcription preinitiation complex, and this is sufficient to make a small number of RNA molecules. However, to make a lot of RNA in response to a signal, such as a hormone, other proteins bind to sequences called enhancers. These proteins interact with the initiation complex and increase the rate of RNA synthesis.\\n\\n# FURTHER READING\\n\\nLatchman, D. 2002. Gene Regulation, A Eukaryotic Perspective. 4p ed. Cheltenham: Nelson Thornes Lp.\\nLloyd, G., Landini, P., and Busby, S. 2001. Activation and repression of transcription initiation in bacteria. Essays Biochem. 37: 17–31.\\nRoberts, G. C., and Smip, C. W. 2000. Alternative splicing: Combinatorial output from pe genome. Curr. Opin. Chem. Biol. 6: 375–383.\\nTjian, R. 1995. Molecular machines pat control genes. Sci. Am. 272: 38–45.\\nQuorum sensing: Hastings, J. W., and Greenberg, E. P. 1999. Quorum sensing: The explanation of a curious phenomenon reveals a common characteristic of bacteria. J. Bacteriol. 188: 2667–2668.\\nSt. John’s wort: Moore, L. B., Goodwin, B., Jones, S. A., Wisely, G. B., Serabjit-Singh, C. J., Willson, T. M., Collins, and Kliewer, J. L., 2000. S. A. St. John’s wort induces hepatic drug metabolism prough activation of pe pregnane X receptor. Proc Natl Acad Sci U S A 13: 7500–7502.\\n---\\n# TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\n\\n# REVIEW QUESTIONS\\n\\nFor each question, choose the ONE BEST answer or completion.\\n\\n1. A DNA strand of sequence 3\\' GTCAAGGATATTCGAT 5\\' would be transcribed in pe nucleus to form a polymer of sequence\\nA. 3\\' GTCAAGGATATTCGAT 5\\'.\\nB. 5\\' GUCAAGGAUAUUCGAU 3\\'.\\nC. 5\\' CAGUUCCUAUAAGCUA 3\\'.\\nD. 3\\' AUCGAAUAUCCUUGAC 5\\'.\\nE. NH3+-Gln-Phe-Leu-COOH.\\n\\n2. RNA polymerase from E. coli is a multisubunit protein composed of α, β, β\\', and σ. The promoter sequence is recognized and bound by\\nA. α.\\nB. β\\'.\\nC. β.\\nD. a β:β\\' dimer.\\nE. σ.\\n\\n3. A rho-independent transcription terminator sequence in prokaryotic DNA has\\nA. a GC-rich region followed by a string of uracils.\\nB. a GC-rich region followed by a string of adenines.\\nC. several stop codons.\\nD. pe sequences TTGACA and TATATT separated by about 17 bases.\\nE. pe sequence AAUAAA and a poly-A tail up to 250 nucleotides long, separated by about 20 bases.\\n\\n4. The lac operon is transcribed when\\nA. lactose is present and glucose is absent.\\nB. cAMP concentrations in pe cell are high.\\nC. The cAMP–CAP protein is bound to pe lac promoter region.\\nD. pe lac repressor is bound to allolactose or a similar shaped molecule.\\nE. all of pe above.\\n\\n5. The trp operon is transcribed when\\nA. tryptophan concentrations in pe cell are high.\\nB. pe trp repressor is bound to tryptophan or a similar shaped molecule.\\nC. tryptophan is bound to its aporepressor.\\nD. pe appropriate corepressor is absent.\\nE. all of pe above.\\n\\n6. Before a newly synpesized eukaryotic mRNA attaches to pe ribosome, it undergoes\\nA. capping.\\nB. polyadenylation.\\nC. splicing.\\nD. transport to pe cytosol.\\nE. all of pe above.\\n\\n7. The preinitiation complex of proteins forms at a eukaryotic protein-coding gene promoter in order to\\nA. align RNA polymerase I.\\nB. align RNA polymerase II.\\n---\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n1. C. 5 CAGUUCCUAUAAGCUA 3. Transcription generates RNA (not protein, which is made at pe next step, translation, outside pe nucleus in eukaryotes). RNA is synpesized in pe 5\\' to 3\\' direction, and pe sequence of pe newly synpesized strand is complementary to pat of pe template DNA strand.\\n2. E. It is pe σ factor pat binds pe promoter sequence and pen brings pe oper subunits onto pe DNA.\\n3. B. The terminator sequence in DNA comprises a GC-rich region followed by a string of adenines. Concerning oper answers: (C) stop codons exert peir effect during translation, not during transcription. (D) The sequences TTGACA and TATATT, pe -35 and -10 boxes, are found in pe promoter region of prokaryotic genes, not at pe end. (E) Processed eukaryotic RNA terminates wip pe sequence AAUAAA and a poly-A tail up to 250 nucleotides long, separated by about 20 bases. The presence of uracil should have been enough to tell you pat pis was not DNA!\\n4. E. The lac operon is transcribed when lactose is present, generating allolactose, which binds to pe repressor, preventing its binding to pe promoter, AND if glucose is absent, causing an increase in pe cAMP concentration and perefore allowing pe cAMP–CAP protein to bind to pe lac promoter region.\\n5. D. The corepressor is tryptophan itself, and pe trp operon is only transcribed when tryptophan is absent and perefore needs to be made by pe cell. When tryptophan is present at a sufficiently high concentration (viz. A), pen it binds to its aporepressor (B, C), which pen binds to pe promoter, preventing transcription.\\n6. E. All pese processes occur before eukaryotic mRNA is translated at pe ribosome.\\n7. B. RNA polymerase II transcribes protein-coding genes in eukaryotes. DnaB is part of pe machinery of DNA replication pat assembles, at pe appropriate time, at pe origin of replication—not at a promoter.\\n---\\n# RECOMBINANT DNA AND GENETIC ENGINEERING\\n\\nDeoxyribonucleic acid is the cell’s database. Within the base sequence is all the information necessary to encode RNA and protein. A number of biological and chemical methods now give us the ability to isolate DNA molecules and to determine their base sequence. Once we have the DNA and know the sequence, many possibilities open up. We can identify mutations that cause disease, make a human vaccine in a bacterial cell, or alter a sequence and hence the protein it encodes. The knowledge of the entire base sequence of the human genome, and of the genomes of many other organisms, such as bacteria that cause disease, is set to revolutionize medicine and biology. In future years the power of genetic engineering is likely to impact ever more strongly on industry and on the way we live. This chapter describes some of the important methods involved in recombinant DNA technology at the heart of which is DNA cloning.\\n\\n# DNA CLONING\\n\\nSince DNA molecules are composed of only four nucleotides, their physical and chemical properties are very similar. Hence it is extremely difficult to purify individual species of DNA by classical biochemical techniques similar to those used successfully for the purification of proteins. However, we can use DNA cloning to help us to separate DNA molecules from each other. A clone is a population of cells that arose from one original cell (the progenitor) and, in the absence of mutation, all members of a clone will be genetically identical. If a foreign gene or gene fragment is introduced into a cell and the cell then grows and divides\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C2004 by John Wiley & Sons, Inc.\\n---\\n# RECOMBINANT DNA AND GENETIC ENGINEERING\\n\\nrepeatedly, many copies of the foreign gene can be produced, and the gene is then said to have been cloned. A DNA fragment can be cloned from any organism. The basic approach to cloning a gene is to take the genetic material from the cell of interest, which in the examples we will describe is a human cell, and to introduce this DNA into bacterial cells. Clones of bacteria are then generated, each of which contains and replicates one fragment of the human genetic material. The clones that contain the gene we are interested in are then identified and grown separately. We therefore use a biological approach to isolate DNA molecules rather than physical or chemical techniques.\\n\\n# CREATING THE CLONE\\n\\nHow do we clone a human DNA sequence? The human genome has 3 × 109 base pairs of DNA, and the DNA content of each cell (other than eggs and sperm) is identical. However, each cell expresses only a fraction of its genes. Different types of cells express different sets of genes and thus their mRNA content is not the same. In addition, processed mRNA is shorter than its parent DNA sequence and contains no introns (page 98). Consequently, it is much easier to isolate a DNA sequence by starting with its mRNA. We therefore start the cloning process by isolating mRNA from the cells of interest. The mRNA is then copied into DNA by an enzyme called reverse transcriptase that is found in some viruses. As the newly synthesized DNA is complementary in sequence to the mRNA template, it is known as complementary DNA, or cDNA. The sample of cDNAs, produced from the mRNA, will represent the product of many different genes.\\n\\nThe way in which a cDNA molecule is synthesized from mRNA is shown in Figure 7.1. Most eukaryotic mRNA molecules have a string of A’s at their 3\\' end, the poly-A tail (page 118). A short run of T residues can therefore be used to prime the synthesis of DNA from an mRNA template using reverse transcriptase. The resulting double-stranded molecule is a hybrid containing one strand of DNA and one of RNA. The RNA strand is removed by digestion with the enzyme ribonuclease H. This enzyme cleaves phosphodiester links in the RNA strand of the paired RNA-DNA complex, making a series of nicks down the length of the RNA. DNA polymerase (page 90) is then added. This homes in on the nicks and then moves along replacing ribonucleotides with deoxyribonucleotides. Lastly, DNA ligase (page 92) is used to reform any missing phosphodiester links. In this way a double-stranded DNA molecule is generated by the replacement of the RNA strand with a DNA strand. If the starting point had been mRNA isolated from liver cells, then a collection of cDNA molecules representative of all the mRNA molecules within the liver will have been produced. These DNA molecules have now to be introduced into bacteria.\\n\\nIntroduction of Foreign DNA Molecules into Bacteria\\n\\nCloning Vectors. To ensure the survival and propagation of foreign DNAs, they must be inserted into a vector that can replicate inside bacterial cells and be passed on to subsequent generations of the bacteria. The vectors used for cloning are derived from naturally occurring bacterial plasmids or bacteriophages. Plasmids (page 74) are small circular DNA molecules found within bacteria. Each contains an origin of replication (page 88) and thus can replicate independently of the bacterial chromosome and produce many copies of itself. Plasmids often carry genes that confer antibiotic resistance on the host bacterium. The advantage of this to the scientist is that bacteria containing the plasmid can be selected for in a population of other bacteria simply by applying the antibiotic. Those bacteria with the antibiotic resistance gene will survive, whereas those without it will die. Figure 7.2 shows\\n---\\n# CREATING THE CLONE\\n\\n|mRNA 5′|A AAA 3′|\\n|---|---|\\n|Reverse transcriptase using dATP, dCTP, dGTP, dTTP, (T) primer| |\\n\\n|mRNA 5′|A AAA 3′|\\n|---|---|\\n|DNA 3′|T TTT 5′|\\n|Ribonuclease H cleaves phosphodiester links in the mRNA molecule| |\\n\\n|chopped mRNA 5′|A AAA 3′|\\n|---|---|\\n|DNA 3′|T TTT 5′|\\n|DNA polymerase replaces RNA strand with a DNA strand| |\\n\\n|DNA with breaks 5′|A AAA 3′|\\n|---|---|\\n|DNA 3′|T TTT 5′|\\n|DNA ligase catalyzes formation of phosphodiester links| |\\n\\n|DNA 5′|A AAA 3′|\\n|---|---|\\n|DNA 3′|Double-stranded DNA molecule|T TTT 5′|\\n\\nFigure 7.1. Synthesis of a double-stranded DNA molecule.\\n\\nrestriction endonuclease site\\nantibiotic resistance gene\\n\\nFigure 7.2. A plasmid cloning vector.\\n---\\n# RECOMBINANT DNA AND GENETIC ENGINEERING\\n\\nThe basic components of a typical plasmid cloning vector: an antibiotic resistance gene, a restriction endonuclease site (see next section) at which foreign DNA can be inserted, and an origin of replication so the plasmid can copy itself many times inside the bacterial cell.\\n\\nExample 7.1\\nThe Cloning Vector pBluescript\\n\\nThe plasmid pBluescript is based on a naturally occurring plasmid that has been engineered to include several valuable features. pBluescript has an origin of replication, and the ampicillin resistance gene for selecting bacterial cells that have taken up the plasmid. The multiple cloning site (MCS) allows the scientist to cut the plasmid with the most appropriate restriction endonuclease for the task at hand. The MCS lies within the lac Z gene, which codes for the enzyme β-galactosidase (page 112). β-Galactosidase converts a substrate known as X-gal to a bright blue product. Cells containing pBluescript without a foreign DNA in the MCS will produce blue colonies when grown on agar plates. However, when the lac Z gene is disrupted by insertion of a foreign DNA in the MCS, the cells containing the recombinant plasmids will grow to produce a colony with the normal color of white. This is because the function of the lac Z gene is destroyed and β-galactosidase is not produced. This is the basis of a test, called the blue/white assay, to identify colonies containing recombinant plasmids.\\n\\nAnother important feature of pBluescript is the presence of the bacteriophage T7 and T3 promoter sequences, which flank the MCS. These promoter sequences are used to transcribe mRNA from a cDNA cloned into one of the sites within the MCS. By selecting the promoter sequence, and adding the appropriate RNA polymerase (T3 or T7 RNA polymerase), either the sense or the antisense mRNA can be synthesized. Antisense RNAs are used in a number of techniques, for example, in situ hybridization (page 147) to detect cells producing a specific mRNA.\\n\\nlac Z ↑ T7\\namp Γ pBluescript MCS ↑ T3\\n\\nTranscription from the T3 promoter will transcribe the bottom strand into mRNA. Transcription from the T3 promoter will transcribe the bottom strand into mRNA. Transcription from the T7 promoter will transcribe the top strand into mRNA.\\n---\\n# CREATING THE CLONE\\n\\n|Bacterial species/strain|Enzyme name|Recognition sequences and cleavage sites|\\n|---|---|---|\\n|Bacillus amyloliquefaciens H|Bam H1|GGATCC CCTAGG|\\n|Escherichia coli Ry13|Eco R1|GAATTC CTTAAG|\\n|Providencia stuartii 164|Pst 1|CTGCAG GACGTC|\\n|Serratia marcescens SB|Sma H1|CCCGGG GGGCC|\\n|Rhodopseudomonas sphaeroides|Rsa 1|GTAC CATG|\\n\\nFigure 7.3. Recognition sites of some common restriction endonucleases.\\n\\nBacteriophages (page 74) are viruses that infect bacteria and utilize the host cell’s components for their own replication. The bacteriophage genome is, like a plasmid, circular, although many viruses use RNA rather than DNA as their genetic material. If human DNA is inserted into a bacteriophage, the bacteriophage will do the job of introducing it into a bacterium.\\n\\nJoining Foreign DNAs to a Cloning Vector. Enzymes known as restriction endonucleases are used to insert foreign DNA into a cloning vector. Each restriction endonuclease recognizes a particular DNA sequence of (usually) 4 or 6 bp. The enzyme binds to this sequence and then cuts both strands of the double helix. Many restriction endonucleases have been isolated from bacteria. The names and recognition sequences of a few of the common ones are shown in Figure 7.3. Restriction endonuclease names are conventionally written in italics because they are derived from the Latin name for the bacterium in which the protein occurs.\\n\\nSome enzymes such as Bam HI, Eco RI, and Pst I make staggered cuts on each strand. The resultant DNA molecules are said to have sticky ends (Fig. 7.4) because such fragments can associate by complementary base pairing to any other fragment of DNA generated by the same enzyme. Other enzymes such as Sma HI cleave the DNA smoothly to produce\\n---\\n# RECOMBINANT DNA AND GENETIC ENGINEERING\\n\\nBlunt ends (Fig. 7.4). DNA fragments produced in this way can be joined to any other blunt-ended fragment.\\n\\nFigure 7.5 illustrates how human DNA is inserted into a plasmid that contains a Bam H1 restriction endonuclease site. A short length of synthetic DNA (an oligonucleotide) that includes a Bam H1 recognition site is added to each end of the human DNA fragment. Both the human DNA and the cloning vector are cut with Bam H1. The cut ends are now complementary and will anneal together. DNA ligase then catalyzes the formation of phosphodiester links between the vector and the human DNA. The resultant molecule is known as a recombinant plasmid. If our starting material was mRNA from a sample of liver, we would now have a collection of plasmids each carrying a cDNA from one of the genes that was being transcribed in this organ.\\n\\nIntroduction of Recombinant Plasmids into Bacteria.\\n\\nFigure 7.6 summarizes how recombinant plasmids are introduced into bacteria such as Escherichia coli. Bacteria are first treated with concentrated calcium chloride to make the cell wall more permeable to DNA. DNA can now enter these cells, which are said to have been made competent. Cells that take up DNA in this way are said to be transformed. The transformation process is very inefficient, and only a small percentage of cells actually take up the recombinant molecules. This means that it is extremely unlikely that any one bacterium has taken up two plasmids. The presence of an antibiotic resistance gene in the cloning vector makes it possible to select those bacteria that have taken up a molecule of foreign DNA, since only the transformed cells can survive in the presence of the antibiotic. The collection of bacterial colonies produced after this selection process is a clone library. All the cells of a single colony harbor identical recombinant molecules that began as one mRNA molecule in the original cell sample. Other colonies in the same clone library contain plasmids carrying different DNA inserts. Isolating individual bacterial colonies will produce different clones of foreign DNA. In the example we have described, where the starting DNA material used to produce these clones was a population of cDNA molecules, the collection of clones is called a cDNA library.\\n\\nSelection of cDNA Clones\\n\\nHaving constructed a cDNA library—which may contain many thousands of different clones—the next step is to identify the clones that contain the cDNA of interest.\\n---\\n# CREATING THE CLONE\\n\\n|human DNA fragment|\\n|---|\\n|oligonucleotide with restriction endonuclease site|oligonucleotide with restriction endonuclease site|vector DNA|\\n|GGA TCC|GGA TCC|\\n|CCTAGG|CCTAGG|\\n|BamH1|TA G|BamH1|GATCCG|GCCTAG|GCCG|\\n\\nDNA strands anneal via complementary base pairing\\n\\nTCC GGA\\n\\nAAGG CCTATC\\n\\nGGCCT GG\\n\\nDNA ligase repairs the recombinant plasmid phosphodiester bonds\\n\\nFigure 7.5. Generation of a recombinant plasmid.\\n---\\n# Figure 7.6. Introduction of recombinant plasmids into bacteria.\\n\\nThere are many ingenious ways of doing this. We will describe two ways of selecting cDNA clones from a library. One method simply detects the presence of the foreign DNA attached to the plasmid vector, and the second detects the protein encoded for by the foreign DNA. We call the process of selecting specific clones “screening the library”.\\n\\nPreparation of the cDNA Library for Screening. Bacterial colonies are plated onto agar plates, and the colonies are then replica-plated onto a nylon membrane, which is then treated with detergent to burst (or lyse) the bound cells. If the clone is to be selected by virtue of its DNA sequence, the nylon membrane is processed with sodium hydroxide. This is necessary to break all hydrogen bonds between the DNA strands bound to the nylon membrane and ensures that the DNA is single-stranded. The processed nylon membrane is an exact replica of the DNA contained within each bacterial colony on the agar plate. If the clone is to be selected from the library by detecting the protein encoded by the foreign DNA, then colonies are again replica-plated onto a nylon membrane. This time, however, the nylon membrane is processed to produce an exact copy of the proteins synthesized by each bacterial colony.\\n\\nOligonucleotide Probes for cDNA Clones. If some of the amino acid sequence is known for the protein whose cDNA is to be cloned, an oligonucleotide can be synthesized in vitro (i.e., in a machine rather than in a cell) that has a sequence complementary to the protein\\'s sequence.\\n---\\nCREATING THE CLONE\\n\\n|met|gln|lys|phe|asn|known protein sequence|\\n|---|---|---|---|---|---|\\n| |AUG|CA|A|AA|UU|U|AA|U|all possible mRNA sequences|\\n|G|G|C|C|synthesize 16|primers|\\n| |ATG|CA|A|AA|TT|T|AA|T|DNA oligonucleotides|\\n|G| |G| |C| |C|\\n| | | |label each oligonucleotide| | | | | | | | |\\n| | | |with polynucleotide kinase| | | | | | | | |\\n| | | |+ γ 32 PATP| | | | | | | | |\\n| | | | | |membrane lift incubated with| | | | | | | | |\\n| | | | | |radio-labelled oligonucleotides| | | | | | | | |\\n| | | | | |clone hybridized| | | | | | | | |\\n|X-ray film| | | | | | | | |\\n\\nFigure 7.7. Selection of a cDNA clone with a radioactive oligonucleotide probe.\\n---\\n# RECOMBINANT DNA AND GENETIC ENGINEERING\\n\\n|transform bacterial cells|human proteins expressed in bacterial cells|\\n|---|---|\\n|Human cDNA sequence|placed on surface of agar plate|\\n\\n1. incubate filter with primary antibody\\n\\n2. incubate filter with secondary antibody linked to alkaline phosphatase\\n\\nproteins bind to membrane\\n\\nadd substrate for alkaline phosphatase—a colored product is produced\\n\\nspecific bacterial colony identified\\n\\nFigure 7.8. Selection of a cDNA clone with an antibody probe.\\n---\\n# CREATING THE CLONE\\n\\none of the strands of the cDNA. The first step is to use the genetic code to predict all possible DNA sequences that could code for a short stretch of amino acids within the protein. This strategy is shown in Figure 7.7. The sequence—met gln lys phe asn—can be coded for by 16 possible sequences, because of the redundancy of the genetic code. All 16 oligonucleotide sequences are synthesized. One of the 16 sequences will be complementary to the cDNA we want to select from the library. The oligonucleotides are tagged with a radioactive phosphate group (32P) at their 5\\' ends using the enzyme polynucleotide kinase (PNK) and the substrate [γ-32P]ATP, that is, ATP whose γ phosphate is the radioactive isotope 32P. PNK removes the 5-phosphate group from each oligonucleotide, leaving a 5-hydroxyl group. The enzyme then transfers the γ (32P) phosphate group of [γ-32P]ATP to the 5-hydroxyl.\\n\\nThe nylon membrane to which the library DNA is attached is incubated together with the mixture of radiolabeled oligonucleotides. This process is called hybridization, a word used whenever two nucleic acid strands associate together by hydrogen bonding. In this case the oligonucleotide complementary in sequence to the clone we want to select will hydrogen bond to the single-stranded DNA on the nylon membrane. Once hybridization is complete, excess oligonucleotide is washed from the nylon membrane, which is now covered with a sheet of X-ray film and placed in a light-tight cassette. The radioactivity in the oligonucleotide will darken the silver grains on the X-ray film—a process known as autoradiography. A positive clone will show up as a black spot on the film. Superimposing the X-ray film back onto the original bacterial plate will identify the living bacterial colony that contains the desired foreign DNA clone.\\n\\nAntibody Probes for cDNA Clones. This method makes use of specific antibodies to detect bacteria expressing the protein product of the DNA to be cloned. For this to work, the foreign DNA must be expressed in the bacterial cells; that is to say, its information must be copied first into mRNA and then into protein. To ensure efficient expression, the plasmid vector contains a bacterial promoter sequence that is used to control transcription of foreign DNA. Such cloning vectors are known as expression vectors. The promoter of the lac operon is commonly used in this way. The clone library is plated onto agar plates containing an inducer of the lac operon such as IPTG (page 114) to ensure that lots of mRNA and in turn lots of protein is synthesized. Figure 7.8 shows how an antibody, linked to an enzyme (usually alkaline phosphatase), can detect a positive clone by generating a colored product. The pattern of colored spots on the nylon membrane is used to identify the bacterial clones of interest on the original agar plate.\\n\\n# Genomic DNA Clones\\n\\nThe approach described in the previous section permits the isolation of cDNA clones. Complementary DNA clones have many important uses, some of which are described below. However, as a cDNA is a copy of mRNA only, when we want to isolate a gene to investigate its structure and function, we need to create genomic DNA clones. Genes contain exons and introns and have regulatory sequences at their 5\\' and 3\\' ends and are therefore much larger than cDNAs. The vectors used to clone genes must therefore be able to hold long stretches of DNA. Plasmids used for cDNA cloning cannot do this. A selection of vectors used to clone genes is shown in Table 7.1. Vectors such as the P1 artificial chromosomes (PACs)—based on the bacteriophage P1—can hold about 150,000 bp of DNA. PACs have been used very successfully in the Human Genome Project (page 144). Yeast artificial chromosome\\n---\\n# Table 7.1. Vectors Used for Cloning Genomic DNA\\n\\n|Genomic DNA Cloning Vector|Size of Insert (kb)|\\n|---|---|\\n|Bacteriophage λ|9–23|\\n|Cosmid|30–44|\\n|PAC (P1 artificial chromosome)|130–150|\\n|BAC (bacterial artificial chromosome)|Up to 300|\\n|YAC (yeast artificial chromosome)|200–600|\\n\\nYeast artificial chromosome (YAC) vectors can hold between 200,000 and 600,000 million base pairs of foreign DNA. The choice of genomic vector is governed by the size of DNA the scientist needs to clone. PAC, Bacterial artificial chromosome (BAC), and YAC vectors are needed if an entire gene sequence is to be represented in a single clone. PAC, BAC, and YAC vectors contain all the sequences needed to produce a minichromosome in the appropriate cell type: PACs and BACs in bacterial cells and YACs, as their name implies, to replicate foreign sequences in a yeast cell. The YAC vector therefore contains sequences that will allow the DNA to be processed by the host yeast cell as if it were a normal chromosome and to allow replication alongside the other chromosomes in the cell. Thus a YAC vector has sequences that specify the yeast centromere (page 403), telomeres (the ends of the new chromosome), and the yeast origin of replication. A YAC vector also contains selectable marker genes so that only those cells that have been transformed by a correctly constructed YAC chromosome will survive.\\n\\nTo generate the large DNA fragments needed for genomic cloning, the chromosomal DNA is incubated with a restriction endonuclease for a very short time. Not all the recognition sites for that enzyme are cleaved, and large fragments of DNA are hence produced by partial digestion. Genomic DNA fragments are joined to genomic cloning vectors in the same way we join cDNAs to cDNA cloning vectors. In the example shown in Figure 7.9, human DNA has been introduced into the genome of a bacteriophage known as lambda (λ). This particular vector can accommodate up to 23,000 bp of foreign DNA in its genome. Bacteria are then infected, generating a collection of bacteria called a genomic DNA library.\\n\\nTo select the genomic DNA sequence of interest, the library is plated onto a layer (or lawn) of cultured bacteria so that many copies of the recombinant bacteriophage can be produced. A single λ bacteriophage infects a single E. coli. The recombinant bacteriophages then multiply inside the host cells. The cells die and lyse, and the bacteriophages spread to the surrounding layer of bacteria and infect them. These cells lyse, in turn, and the process continues.\\n\\n# Example 7.2 Cloning a Receptor Protein cDNA\\n\\nGlutamate is one of the most important transmitters in the brain. The gene coding for the glutamate receptor, the protein on the surface of nerve cells that, upon binding glutamate, allows an influx of sodium ions (page 372), remained uncloned for a number of years. Success came with the use of a very clever cloning strategy, based on the function of the receptor. mRNA was isolated from brain cells and used as the template for the production of cDNA molecules. These were inserted into a plasmid expression vector. Following the introduction of these cDNAs into bacteria, a cDNA library representative of all the mRNAs in the brain was produced. The many thousands of cDNA clones in the library were then divided into pools. Each of the many pools was then injected into a frog egg (Xenopus oocyte), which transcribed the cDNAs into RNA and translated the RNA into protein. To see which of the oocytes had been injected with the cDNA for the glutamate receptor, these cells were whole cell patch clamped (page 321). Glutamate was applied to the oocytes, and the oocyte\\n---\\n# CLONING A RECEPTOR PROTEIN\\n\\nWhose injected pool had included the cDNA for the glutamate receptor responded with an inward current of sodium ions indicating the presence of glutamate receptors in the plasma membrane. The pool of cDNAs giving this response was further divided into smaller pools. Each of these was rescreened for the presence of glutamate receptor activity. This was followed by several rounds of rescreening. For each round a further subdivision was made of the cDNAs into pools containing fewer and fewer cDNA molecules. Eventually each pool contained only a single cDNA so that the cDNA for the glutamate receptor could be identified. A number of other receptors have now been isolated using the same strategy in which a functional assay is used to identify the cDNA encoding the receptor.\\n\\n|1. mRNA from brain used to make cDNA|\\n|---|\\n|2. cDNA clones|\\n|subdivide into pools|\\n|3. pool 1|pool 2|pool 3|pool 4|\\n|4. extract cDNAs and inject into oocytes|\\n|5. Na+|\\n|Na+|\\n|glutamate|no receptor activity|\\n|oocytes producing an inward sodium current on glutamate application identify the pool containing the cDNA coding for the glutamate receptor|\\n|repeat steps 4 and 5 on pool 1 cDNAs, each time using smaller cDNA subpools until a single cDNA giving a positive response in the receptor assay has been identified|\\n---\\n# RECOMBINANT DNA AND GENETIC ENGINEERING\\n\\n|Human genomic DNA|bacteriophage λ vector|\\n|---|---|\\n|The nonessential region of the λ genome can be replaced by a piece of foreign DNA|The nonessential region of the λ genome can be replaced by a piece of foreign DNA|\\n|Partially digest with Eco Rl (E) to generate large fragments| |\\n|DNA ligase| |\\n| |digest with Eco Rl|\\n|left λ arm|right λ arm|\\n|λ genome containing a fragment of human DNA| |\\n|packaged to produce bacteriophage particles| |\\n|infect a lawn of E. coli growing on an agar medium| |\\n|infected cells replicate the recombinant bacteriophage and plaques develop| |\\n|nylon membrane placed on surface of agar plate| |\\n|remove nylon membrane and treat with alkali| |\\n|incubate nylon membrane with radiolabeled cDNA probe| |\\n|specific plaque identified| |\\n|X-ray film autoradiogram| |\\n|wash nylon membrane and expose to X-ray film| |\\n|probe will hybridize to complementary gene sequences| |\\n\\nFigure 7.9. Generation and selection of genomic DNA clones.\\n---\\nUSES OF DNA CLONES\\n\\nDNA Sequencing\\nThe ability to determine pe order of pe bases wipin a DNA molecule has been one of pe greatest technical contributions to molecular biology. DNA is made by pe polymerization of pe four deoxynucleotides dATP, dGTP, dCTP, and dTTP. These are joined togeper when DNA polymerase catalyzes pe formation of a phosphodiester link between a free 3 - ′ hydroxyl on pe deoxyribose sugar moiety of one nucleotide and a free 5 -phosphate group ′ on pe sugar residue of a second nucleotide. However, pe ′artificial dideoxynucleotides ddATP, ddGTP, ddCTP, and ddTTP have no 3 -hydroxyl on peir sugar residue (Fig. 7.10), and so if pey are incorporated into a growing DNA chain, synpesis will stop. This is pe basis of pe dideoxy chain termination DNA sequencing technique devised by Frederick Sanger and for which he was awarded a Nobel prize in 1980.\\n\\ngeneral base structure of a dideoxynucleotide\\n\\n|O|O|O|\\n|---|---|---|\\n|O|γ P|O|β P|O|α P|O|CH 2|O|\\n|O|O|O|\\n|4′ C|H|C 1′|\\n|triphosphate|H|C 3′|C 2′|H|\\n|H|H| |\\n|cannot form a phosphodiester bond with an incoming deoxynucleotide| | |\\n\\nFigure 7.10. A dideoxynucleotide.\\n---\\n# RECOMBINANT DNA AND GENETIC ENGINEERING\\n\\nThis technique is illustrated in Figure 7.11. A cloned piece of DNA of unknown sequence is first joined to a short oligonucleotide whose sequence is known. The DNA is then made single-stranded so that it can act as the template for the synthesis of a new DNA strand. All DNA synthesis requires a primer; in this case a primer is provided that is complementary in sequence to the oligonucleotide attached to the template DNA. Four separate mixtures are prepared. Each contains the DNA template, the primer (which has been radiolabeled), DNA polymerase, and the four deoxynucleotides. The mixtures differ in that each also contains a low concentration of one of the four dideoxynucleotides ddATP, ddGTP, ddCTP, or ddTTP. When a molecule of dideoxynucleotide is joined to the newly synthesized chain, DNA synthesis will stop.\\n\\nLet us follow what happens in the tube containing ddTTP in Figure 7.11. The first base that DNA polymerase encounters in the DNA template to be sequenced is an A. Since the tube contains much more dTTP than ddTTP, DNA polymerase will add a dTTP to most of the primer molecules. However, a small fraction of the primers will have a ddTTP added to them instead of dTTP since DNA polymerase can use either nucleotide as a substrate. The next base encountered is a G. DNA polymerase is unable to attach dCTP to the ddTTP since there is no OH group on the 3\\' carbon of the sugar, and so DNA synthesis is terminated. The majority of strands, however, had a dTTP added, and for these DNA polymerase can proceed, building the growing strand. No problems are encountered with the next six bases. However, the eighth base in the template strand is another A, and once again a small fraction of the growing strands will have ddTTP added instead of dTTP. In the same way as before, these strands can grow no further. This process will be repeated each time an A occurs on the template strand. When the reaction is over, the tube will contain a mixture of DNA fragments of different length, each of which ends in a ddTTP. Similarly, each of the other three tubes will contain a mixture of DNA chains of different length, each of which ends in either ddCTP, ddATP, or ddGTP. To determine the sequence of the newly synthesized chains, each of the four samples is loaded onto a polyacrylamide gel. Because the primer used was radiolabeled, all the new DNA chains will carry a radioactive tag so that, after electrophoresis, the pattern of DNA fragments on the gel can be detected by autoradiography. Each terminated reaction will show up as a black band on the X-ray film. The smallest DNA molecules are that fraction in the tube containing ddTTP whose growth was blocked after the first base, T. These move farthest and produce the band at the bottom of the T lane. DNA molecules one nucleotide larger were produced in the tube containing ddCTP—their growth was blocked after the second base, C. These move almost as far, but not quite, producing the band at the bottom of the C lane. Reading bands up from the bottom of the gel therefore tells us the sequence in which bases were added to the unknown strand: T, C, and so on. Because the new chain is complementary in sequence to its template strand, the sequence of the template strand can be inferred.\\n\\nThe Human Genome Project required the process of DNA sequencing to be automated. Instead of using radioactivity and one reaction tube for each dideoxynucleotide, we now use dideoxynucleotides tagged with fluorescent dyes. Each of the four dideoxynucleotides ddATP, ddGTP, ddCTP, and ddTTP is tagged with a different fluorescent dye. This means that all four of the termination reactions can be carried out in a single reaction tube and loaded into the same well on the polyacrylamide gel. As the reaction product drips from the bottom of the gel, the fluorescence intensity of each of the four colors corresponding to the four dideoxynucleotides is monitored, and this information is transferred straight to a computer where the data are analyzed. An example of a DNA trace produced using fluorescently tagged dideoxynucleotides is shown on the CBASC website. Each peak represents a terminated\\n---\\n# USES OF DNA CLONES\\n\\n|DNA sequencing|single-stranded DNA|\\n|---|---|\\n|to be sequenced|known sequence|\\n|by the Sanger method|5′ C T A G G C T T G A C A G T A G C A G 3′|\\n| |G T C A T C G T C|\\n| |plus radioactively labeled primer that pairs with known sequence|\\n| |plus dATP, dCTP, dTTP, dGTP|\\n| |plus DNA polymerase|\\n| |incubated with chain terminating dideoxynucleotides|\\n|ddTTP|ddCTP|\\n|ddATP|ddGTP|\\n| |gel electrophoresis|\\n|ddTTP|ddCTP|\\n|ddATP|ddGTP|\\n|G|3′|\\n|A| |\\n|T| |\\n|C|sequence|\\n|C|read from gel|\\n|G| |\\n|A| |\\n|A| |\\n|C| |\\n| |autoradiogram|\\n|T|5′|\\n\\nFigure 7.11. DNA sequencing and the dideoxy chain termination method.\\n---\\n# RECOMBINANT DNA AND GENETIC ENGINEERING\\n\\nTo determine the base sequence of the entire human genome, the DNA was cut by partial restriction endonuclease digestion (page 140) to give large fragments of about 150,000 bp. These were cloned into a vector such as PAC (Table 7.1). The aim was to create a library of clones that overlapped one another. Each clone was sequenced, as described above, and their base sequences compared. Because the clones overlapped, it was possible, by comparing their sequences, to line up the position of each clone with respect to its neighbors. This required the development of sophisticated computer programs, and the creation of a large database of information to order the 3 x 109 base pairs that comprise the human genome.\\n\\nSouthern Blotting\\n\\nIn 1975 Ed Southern developed an ingenious technique, now known as Southern blotting, which can be used to detect specific genes (Fig. 7.12). Genomic DNA is isolated and digested with one or more restriction endonucleases. The resultant fragments are separated according to size by agarose gel electrophoresis. The gel is soaked in alkali to break hydrogen bonds between the two DNA strands and then transferred to a nylon membrane. This produces an exact replica of the pattern of DNA fragments in the agarose gel. The nylon membrane is incubated with a cloned DNA fragment tagged with a radioactive label (page 143). The gene probe is heated before adding to the nylon membrane to make it single-stranded so it will base pair, or hybridize, to its complementary sequences on the nylon membrane. As the gene probe is radiolabeled, the sequences to which it has hybridized can be detected by autoradiography.\\n\\nMutations that change the pattern of DNA fragments—for instance, by altering a restriction endonuclease recognition site or deleting a large section of the gene—can easily be detected by Southern blotting. This technique is therefore useful in determining whether an individual carries a certain genetic defect or if a fetus is homozygous for a particular disorder. All that is needed is a small DNA sample from white blood cells or, in the case of a fetus, from the amniotic fluid in which it is bathed, or by removing a small amount of tissue from the chorion villus that surrounds the fetus in the early stages of pregnancy.\\n\\nForensic laboratories use Southern blotting to generate DNA fingerprints from samples of blood or semen left at the scene of a crime. A DNA fingerprint is a person-specific Southern blot. The gene probe used in the test is a sequence that is repeated very many times within the human genome—a microsatellite sequence (page 100). Everyone carries a different number of these repeated sequences, and because they lie adjacent to each other on the chromosome they are called VNTRs (variable number tandem repeats). When genomic DNA is digested with a restriction endonuclease and then analyzed by Southern blotting, a DNA pattern of its VNTRs is produced. Unless they are identical twins, it is extremely unlikely that two individuals will have the same DNA fingerprint profile. It has been estimated that if eight restriction endonucleases are used, the probability of two people who are not identical twins generating the same pattern is one in 1030.\\n\\nA special type of Southern blot, called a zoo blot, can be used to reveal genes that are similar in different species. Such genes, which have been conserved through evolution, are likely to code for crucial proteins. A probe generated from a genomic DNA library is used to probe genomic DNA from many different species. A probe that hybridizes with DNA from a number of species is likely to represent all or part of such a conserved gene.\\n---\\n# USES OF DNA CLONES\\n\\nDNA placed in well at the top of the gel\\n\\n|agarose gel|high molecular weight|\\n|---|---|\\n|agarose gel|low molecular weight|\\n\\nDNA fragments separate according to size in an electric field for the preparation of Southern blot.\\n\\n|salt solution|agarose gel|\\n|---|---|\\n|nylon membrane|filter paper acts as a wick|\\n|stack of paper towels| |\\n\\nDNA transfers from the gel to the nylon membrane as the salt solution passes through the gel and membrane and into the paper towels.\\n\\nNylon membrane incubated with radioactive cDNA probe.\\n\\n|agarose gel|( )|\\n|---|---|\\n|nylon membrane| |\\n|nylon membrane exposed to X-ray film| |\\n\\nGene probe hybridizes to complementary sequence in the autoradiogram.\\n\\nFigure 7.12. The technique of Southern blotting.\\n\\n# In situ Hybridization\\n\\nIt is possible, using the technique of in situ hybridization, to identify individual cells expressing a particular mRNA. To do this we need to synthesize an antisense RNA molecule—an RNA that is complementary in sequence to the mRNA of interest. In a test tube, the appropriate strand of the cloned cDNA is copied into the antisense RNA using RNA polymerase. The RNA is then labeled with a modified nucleotide that can subsequently be detected using an antibody and a color reaction. The cDNA must first be cloned into an\\n---\\n# RECOMBINANT DNA AND GENETIC ENGINEERING\\n\\n1. Denature RNA\\n\\n2. Separate RNAs on denaturing agarose gel\\n\\n|size markers|animal|\\n|---|---|\\n|treated animal|From untreated|\\n|From phenobarbital| |\\n\\n3. Blot RNA onto membrane filter\\n\\n4. Incubate membrane with cDNA probe CYP2B1 mRNA\\n\\n5. Autoradiography\\n\\nSpecific mRNA detected by cDNA probe\\n\\nFigure 7.13. (a). The technique of northern blotting. (b). A northern blot reveals that the CYP2B1 gene is upregulated in animals given phenobarbital.\\n\\nExpression vector that contains a promoter sequence to which RNA polymerase can bind. A thin tissue section, attached to a glass microscope slide, is incubated with the antisense RNA. The antisense RNA will hybridize, in the cell, to its complementary mRNA partner. Excess antisense RNA is washed off the slide, leaving only the hybridized probe. The color reaction is now carried out so that the cells expressing the mRNA of interest can be identified by bright-field microscopy (page 3).\\n\\nNorthern Blotting\\n\\nFigure 7.13a shows a blotting technique that can determine the size of an mRNA and tell us about its expression patterns. RNA is denatured by heating to remove any intramolecular double-stranded regions and then electrophoresed on a denaturing agarose gel. The RNA is transferred to a nylon membrane (as described on page 136 for the transfer of DNA). The nylon membrane is incubated with a radiolabeled, single-stranded cDNA probe\\n---\\n# Table 7.2. Blotting Techniques\\n\\n|What Is Probed|Nature of the Probe|Book Page for Description|\\n|---|---|---|\\n|Southern blot|DNA|DNA|146|\\n|Northern blot|RNA|cDNA or RNA|148|\\n|Western blot|Protein|Antibody|166|\\n\\n(page 143), or an antisense RNA probe (page 147). Following hybridization, excess probe\\nis washed off and the nylon membrane exposed to X-ray film. The mRNA is visualized on\\nthe autoradiogram (page 139) because it hybridized to the radioactive probe. By analogy\\nwith Southern blotting, this technique is called northern blotting (Table 7.2). Figure 7.13b\\nshows a northern blot for a cytochrome P450 (page 249) mRNA known as CYP2B1. The\\ngene for this protein is activated in the liver by the barbiturate phenobarbital and hence a\\nlot of CYP2B1 mRNA is made.\\n\\nProduction of Mammalian Proteins in Bacteria\\nThe large-scale production of proteins using cDNA-based expression systems has wide applications for medicine and industry. It is increasingly being used to produce polypeptide-based drugs, vaccines, and antibodies. Such protein products are called recombinant because they are produced from a recombinant plasmid. For a mammalian protein to be synthesized in bacteria its cDNA must be cloned into an expression vector (as described on page 139). Insulin was the first human protein to be expressed from a plasmid introduced into bacterial cells and has now largely replaced insulin from pigs and cattle for the treatment of diabetes. Other products of recombinant DNA technology include growth hormone and factor VIII, a protein used in the treatment of the blood clotting disorder hemophilia. Factor VIII was previously isolated from donor human blood. However, because of the danger of infection from viruses such as HIV, it is much safer to treat hemophiliacs with recombinant factor VIII. It should, in theory, be possible to express any human protein via its cDNA.\\n\\nProtein Engineering\\nThe ability to change the amino acid sequence of a protein by altering the sequence of its cDNA is known as protein engineering. This is achieved through the use of a technique known as site-directed mutagenesis. A new cDNA is created that is identical to the natural one except for changes designed into it by the scientist. This DNA can then be used to generate protein in bacteria, yeast, or other eukaryotic cell lines.\\nThe first use of protein engineering is to study the protein itself. A comparison of the catalytic properties of the normal and mutated form of an enzyme helps to identify amino acid residues important for substrate and cofactor binding sites (Chapter 11). This technique was also used to identify the particular charged amino acid residues responsible for the selectivity of ion channels (page 437). Now scientists are using protein engineering to generate new proteins as tools, not only for scientific research but for wider medical and industrial purposes.\\nSubtilisin is a protease and is one of the enzymes used in biological washing powder. The natural source of this enzyme is Bacillus subtilis, an organism that grows on pig feces. To produce, from this source, the 6000 tons of subtilisin used per year by the soap powder\\n---\\n150 RECOMBINANT DNA AND GENETIC ENGINEERING\\n\\nindustry is a difficult and presumably unpleasant task. The cDNA for subtilisin has been isolated and is now used by industry to synthesize the protein on a large-scale in E. coli. The wild-type (natural) form of subtilisin is, however, prone to oxidation because of a methionine present at position 222 in the protein. Its susceptibility to oxidation makes it an unsuitable enzyme for a washing powder that must have a long shelf life and be robust enough to withstand the rigours of a washing machine with all its temperature cycles. Scientists therefore changed the codon for methionine (AUG) to the codon for alanine (GCG). When the modified cDNA was expressed in E. coli, the resulting enzyme was found to be active and not susceptible to oxidation. This was excellent news for the makers of soap powder. However, it is always necessary to check the kinetic parameters of a \"new\" protein produced from a modified cDNA. For subtilisin (met 222) the KM (page 244) is 1.4 × 10-4 M while for subtilisin (ala 222) the KM is 7.3 × 10-4 M. This means that at micromolar concentrations of dirt the modified enzyme will bind less dirt than the wild-type one, but the dirt concentrations caked onto our clothes are well above micromolar. The product turnover number, kcat (page 242), is 50 s-1 for subtilisin (met 222) and 40 s-1 for subtilisin (ala 222): The mutant enzyme is slightly slower, but not by much. By changing a met to an ala, a new enzyme has been produced that can do a reasonable job and is stable during storage and in our washing machines.\\n\\nGreen fluorescent protein is found naturally in certain jellyfish. Protein engineering has now created a palette of proteins with different colors (blue, cyan, green, and yellow). However, the great advantage of these proteins to biologists is that chimeric proteins (proteins composed of two parts, each derived from a different protein) incorporating a fluorescent protein are intrinsically fluorescent. This means that our protein of interest can be imaged inside a living cell using a fluorescence microscope (page 6). The fluorescent part of the chimeric protein tells us exactly where our protein is targeted in the cell and if this location changes in response to signals.\\n\\nFigure 7.14 illustrates how this approach can be used to determine what concentration of glucocorticoid drug is required to cause the glucocorticoid receptor to move to the nucleus. The plasmid, like many plasmids designed for convenience of use, contains a multiple cloning site (MCS), sometimes called a polylinker, which is a stretch of DNA that contains several restriction endonuclease recognition sites. A convenient restriction endonuclease is used to cut the plasmid (in this case, for green fluorescent protein) and the cDNA for the glucocorticoid receptor is inserted. The plasmid also contains a promoter sequence, derived from a virus, that will drive the expression of the DNA into mRNA in mammalian cells that have been infected with the plasmid (or transfected). The chimeric protein, synthesized from the mRNA, will have one part coded for by the cDNA of interest—the other part being the fluorescent protein. The plasmid is grown up in bacteria and then used to transfect mammalian cells. In the absence of glucocorticoid the protein, and therefore the green fluorescence, is in the cytosol. When enough glucocorticoid is added, it binds to the chimeric protein, which then moves rapidly to the nucleus.\\n\\nPolymerase Chain Reaction\\n\\nA technique called the polymerase chain reaction (PCR) has revolutionized recombinant DNA technology. It can amplify DNA from as little material as a single cell and from very old tissue such as that isolated from Egyptian mummies, a frozen mammoth, and insects trapped in ancient amber. A simple mouth swab can yield enough cheek cell DNA to determine carriers of a particular recessive genetic disorder. PCR is used to amplify DNA\\n---\\nGreen fluorescent protein\\n\\n|CMV promoter|Glucocorticoid receptor|\\n|---|---|\\n|ori|Bacterial gene promoter|\\n|Antibiotic resistance| |\\n|Multiple cloning site|add glucocorticoid|\\n\\nGreen fluorescent protein\\n\\nBacterial promoter\\n\\nA chimera of green fluorescent protein and the glucocorticoid receptor reveals its location in living cells.\\n\\nCytomegalovirus promoter for mammalian expression\\n\\n|(ori)|gene|Antibiotic resistance|\\n|---|---|---|\\n|Origin of replication| | |\\n\\nFigure 7.14.\\n---\\n# RECOMBINANT DNA AND GENETIC ENGINEERING\\n\\nfrom fetal cells or from small amounts of tissue found at the scene of a crime. The tool that makes PCR possible is thermostable DNA polymerase, an enzyme that survives and can even function at extremely high temperatures that would denature (page 206) most enzymes. Thermostable DNA polymerases are isolated from bacteria that live in extremely hot deep-sea volcanic environments.\\n\\nFigure 7.15 shows how PCR uses a thermostable DNA polymerase and two short oligonucleotide DNA sequences called primers. Each primer is complementary in sequence to a short length of one of the two strands of DNA to be amplified. The primers direct DNA polymerase to copy each of the template strands. The DNA duplex is heated to 90°C to separate the two strands (step 1). The mixture is cooled to 60°C to allow the primers to anneal to their complementary sequences (step 2). At 72°C the primers direct the thermostable DNA polymerase to copy each of the template strands (step 3). These three steps, which together constitute one cycle of the PCR, produce twice the number of original templates. The process of template denaturation, primer annealing, and DNA synthesis is repeated many times to yield many thousands of copies of the original target sequence.\\n\\nIdentifying the Gene Responsible for a Disease\\n\\nUntil recently, the starting point for an identification of the gene responsible for a particular inherited disease was a pattern of inheritance in particular families plus a knowledge of the tissues affected. It is very difficult to find the gene responsible for a disease when the identity of the normal protein is unknown. Very often, the first clue is the identification of other genes that are inherited along with the malfunctioning gene and that are therefore likely to lie close on the same chromosome (this is linkage, page 408). In the past, chromosome walking was then used to identify the disease gene. This is a slow and tedious process that involves isolating a series of overlapping clones from a genomic clone library. One starts by isolating a single clone and then uses this as a probe to find the next clone along the chromosome. The second clone is used to find the third, and this is used to find a fourth clone, and so on. Each successive clone is tested to find out if it might include all or part of the gene of interest (e.g., by using northern blotting, page 148, to see if the gene is expressed in a tissue known to be affected in the disease). Once a candidate gene is identified, one can find out if it was the gene of interest by sequencing it in unaffected individuals and in disease sufferers; if it is the disease gene, its sequence will be different in the affected people. With the publication of the entire human genome, chromosome walking to generate a series of overlapping clones for testing is unnecessary, but identifying the gene that is responsible for a particular inherited condition is even now a time-consuming task.\\n\\nReverse Genetics\\n\\nBecause beginning with an inherited defect in function and working toward identification of a gene is even today a time-consuming task, more and more scientists are now working the other way: they take a gene with a known sequence but unknown function and deduce its role. Since we know the complete genome of a number of species, we can sit at a computer and identify genes that look interesting—for example, because their sequence is similar to a gene of known function. The gene of interest can be mutated and reinserted into cells or organisms, and the cells or organisms tested for any altered function. This approach is called reverse genetics.\\n---\\n|step 1|step 2|step 3|\\n|---|---|---|\\n|heat to 90 separate strands|cool to 60 sequences primers anneal complementary|warm to 72 extends primers DNA polymerase|\\n|add primers|primers anneal complementary|extends primers DNA polymerase|\\n|oligonucleotide| | |\\n|target sequence| | |\\n---\\nMedical Microarrays and Cancer Classification\\n\\nRelevance Microarrays or gene chips are tiny glass wafers to which cloned DNAs are attached.\\n\\n7.1 The principle of microarray technology is to isolate mRNA from a particular cell and to hybridize this to the DNAs on the chip. Because the mRNAs are tagged with a fluorescent dye, the DNAs to which they hybridize on the chip can be detected. Excess mRNA is removed, and fluorescent areas are viewed using a special scanner and microscope. Computer algorithms have been written to analyze the hybridization patterns seen for a particular microarray. The number and type of DNAs used to make a microarray is dependent on the question to be answered.\\n\\nOne of the spin-offs of the Human Genome Project is to identify sets of genes involved in disease. Microarrays are being used to type the mRNAs produced by different cancers in the hope that this will lead to better diagnosis and therefore better treatment. Leukemia, a cancer affecting the blood, is not a single type of disease. Microarray analysis is helping to classify different types of leukemia more precisely by cataloging the mRNAs expressed in different patients. Preliminary studies are encouraging. A microarray of 6817 cDNAs was used to compare the blood mRNAs from patients with acute lymphoblastic leukemias or acute myeloid leukemias. The mRNAs of each patient was passed over the microarray and complementary sequences hybridized. Fifty cDNA/mRNA hybrids were found that could be used to refine the classification of lymphoblastic and myeloid leukemias. In addition, the patterns allowed the classification of lymphoblastic leukemias into T-cell or B-cell classes. This separation of lymphoblastic leukemias into two classes is an important distinction when it comes to deciding on the best treatment for a patient. As more patients with leukemia are investigated using microarrays, it should be possible to design smaller chips, with fewer cDNAs, but with greater prognosis value.\\n\\nA recent study has shown that the fate of women with breast cancer can be predicted by microarray analysis of their cancer tissue. Several genes were shown to be important for this prediction. If the women had a certain gene expression pattern, then the disease was likely to recur within a 5-year period. However, if the gene pattern was of a second type, the cancer almost never returned. Women that fall into this second class also show no additional benefit from chemotherapy or radiotherapy. Microarray analysis not only offers a survival prognosis for women with breast cancer but has also shown that unpleasant treatment for the disease is unnecessary for those possessing the second type of gene pattern.\\n\\n|micro array|chip of|12 DNA|samples|\\n|---|---|---|---|\\n|A|B|C| |\\n|A1, A4|B1, B3|C2| |\\n---\\n|gene construct|nucleus derived from the sperm|\\n|---|---|\\n|isolate fertilized eggs|nucleus derived from the egg|\\n|inject \"gene construct\"| |\\n|implant eggs into foster mother| |\\n\\n(a)\\n\\n|Bgl I|mouse metalothionein 5\\'-flanking sequence|Bam HI|\\n|---|---|---|\\n|rat growth hormone structural gene| | |\\n|plasmid digest with Bgl I and Bam HI| | |\\n|heavy metal ion enhancer sequence| | |\\n|inject eggs| | |\\n|implantation| | |\\n|pups fed Zn ++| | |\\n\\n(b)\\n\\n| |+|-|+|-|\\n|---|---|---|---|---|\\n|heavy metal ion enhancer sequence| | | | |\\n|Transgenic mouse carrying a foreign gene| | | | |\\n|the metalothionein gene contains a heavy-metal ion enhancer sequence. The + mice carry the transgene while littermates without the transgene are indicated by -.| | | | |\\n---\\n# RECOMBINANT DNA AND GENETIC ENGINEERING\\n\\n|exon|exon|exon|\\n|---|---|---|\\n|1|2|3|\\n|cross-over region|neo|cross-over region|\\n| | |gene in chromosome|\\n| | |vector genomic clone|\\n\\nHomologous recombination targets transgene to exon 2 on chromosome\\n\\n1\\nneo\\n3\\n\\nstrain 1 (white)\\n\\n1. remove blastocysts from pregnant mouse.\\n2. grow embryonic stem cells\\n3. Isolate embryonic stem cells\\n\\ntransfect with transgene and select for neo-resistant cells\\n\\n1. inject embryonic stem cells into blastocyst of strain 2 (black)\\n\\n1. implant into pups born foster mother\\n\\ntransgenic chimeric mouse\\n\\nFigure 7.17. Knock-out transgenic mice. (a). Transgene is incorporated into genome of embryonic stem cells by homologous recombination. (b). Genetically modified embryonic stem cells are injected into a blastocyst, which is implanted into a foster mother.\\n---\\n# ETHICS OF DNA TESTING FOR INHERITED DISEASE\\n\\nA transgenic animal is produced by introducing a foreign gene into the nucleus of a fertilized egg (Fig. 7.16a). The egg is then implanted into a foster mother and the offspring are tested to determine whether they carry the foreign gene. If they do, a transgenic animal has been produced. The first transgenic mice ever made were used to identify an enhancer sequence that activates the metallothionein gene when an animal is exposed to metal ions in its diet. The 5\\' flanking sequence of the metallothionein gene was fused to the rat growth hormone gene (Fig. 7.16b). This DNA construct, the transgene, was injected into fertilized eggs. When the mice were a few weeks old, they were given drinking water containing zinc. Mice carrying the transgene grew to twice the size of their litter mates because the metallothionein enhancer sequence, stimulated by zinc, had increased growth hormone production.\\n\\nTransgenic farm animals—such as sheep synthesizing human factor VIII in their milk—have been created. This is an alternative to producing human proteins in bacteria.\\n\\nTransgenic mice are increasingly being used to prove a protein’s function. To do this the gene for the protein is knocked out. This is done by either inserting a piece of foreign DNA into the gene, or by deleting the gene from the mouse genome. The consequences of the protein’s absence are then established. Figure 7.17 describes the method called insertional mutagenesis for knocking out a gene’s function. The first step is to isolate a genomic clone containing the gene to be knocked-out. A marker gene, such as the drug resistance gene neo, is then inserted into the genomic clone, usually in exon 2 of the gene. This means that the normal, functional product of the gene cannot be synthesized. This construct is the transgene, which is then introduced into embryonic stem (ES) cells. These are cells derived from the inner mass of a blastocyst—that is, a very early embryo—of a white mouse. Homologous recombination inside the embryonic stem cells will replace the normal gene with the transgene. Cells in which this rare event happens will survive when grown on neomycin while embryonic stem cells not containing the transgene will die. The genetically modified embryonic stem cells are inserted into the blastocyst cavity of a black mouse and the blastocyst implanted into a foster mother. Transgenic mice will be chimeric and have a mixed color coat because the cells derived from the genetically modified embryonic stem cells will give a white-color coat while the cells from the blastocyst will give a black-color coat. Subsequent breeding will produce a pure white mouse that is homozygous for the knocked-out gene. The effect of knocking-out the gene can then be analyzed.\\n\\nThe applications of recombinant DNA technology are exciting and far-reaching. However, the ability to examine the base sequence of an individual raises important ethical questions. Would you want to know that you had inherited a gene that will cause you to die prematurely? Some of you might feel fine about this and decide to live life to the full. We suspect most people would not want to know their fate. But what if you have no choice and DNA testing becomes obligatory should you wish to take out life insurance? In the United Kingdom insurance companies are now able to ask for the results of the test for Huntington’s disease. This is a fatal degenerative brain disorder that strikes people in their forties. From the insurance company’s point of view DNA testing could mean higher premiums according to life expectancy or at worst refusal of insurance cover. There is much ongoing debate on this issue.\\n---\\n# RECOMBINANT DNA AND GENETIC ENGINEERING\\n\\nOn the positive side, we can now test for changes in certain genes that are involved in drug metabolism. Adverse drug reactions are the fourth highest cause of death in the United States. By knowing, for example, which individuals carry mutations in genes coding for cytochromes P450 (page 249), medicines that need to be metabolized by these proteins can be avoided.\\n\\nAnalysis of fetal DNA can inform parents if their child is affected by an inherited disorder. It is now possible, using the PCR, to examine the genotype of a single cell from an eight-cell embryo. If the DNA test is negative, the embryo can be inserted into the mother, and the parents know they have not passed on a defective gene to their child. This combination of DNA testing and in vitro fertilization has given rise to the unfortunate phrase “designer babies”.\\n\\nIn the future it may be possible to correct genetic defects before or after birth by replacing mutated genes by a normal copy. The technology to carry out such experiments is very demanding and important ethical questions are raised. In Chapter 20 we will discuss the potential use of gene therapy for the correction of the genetic disorder cystic fibrosis.\\n\\n# IN DEPTH 7.1 Genetically Modified (GM) Plants—Can They Help to Feed the World?\\n\\nThe arguments about the value of GM crops and the damage they cause the environment will continue for a long time. These have largely concerned plants that produce insecticide and plants that are resistant to herbicide. Almost unnoticed in the maelstrom of claim and counterclaim has been the use of genetic engineering to produce nutritionally enhanced crops. Rice is a staple food in many countries but lacks many vital nutrients. About 800 million children suffer from vitamin A deficiency, which can result in blindness and a weakened immune system. In an attempt to overcome this severe nutritional deficiency, a group of Swiss scientists have engineered the rice endosperm to produce provitamin A (beta-carotene). This is converted in the body to vitamin A. Four genes were introduced into the rice endosperm, three genes from the daffodil and one from the bacterium Erwinia. These genes code for all the proteins needed to make beta-carotene. The transgenic rice is golden in color because of the presence of large amounts of beta-carotene.\\n\\nAlmost 24% of the world’s population, mainly women, are deficient in iron. Rice has been engineered with the gene for ferritin, an iron-binding protein. This rice has higher amounts of iron and has been shown to correct iron deficiency in laboratory animals. The plan is to cross the provitamin-A-enriched, iron-enriched strains with agricultural rice strains to produce a hardy and nutritional crop. Golden rice, enriched in iron, provides a great opportunity to improve the health of billions of people.\\n\\n# SUMMARY\\n\\n1. DNA sequences can be cloned using reverse transcriptase, which copies mRNA into DNA to make a hybrid mRNA:DNA double-stranded molecule. The mRNA strand is then converted into DNA by the enzymes ribonuclease H and DNA polymerase. The new double-stranded DNA molecule is called complementary DNA (cDNA).\\n2. Restriction endonucleases cut DNA at specific sequences. DNA molecules cut with the same enzyme can be joined together. To clone a cDNA, it is joined to a cloning\\n---\\n# REVIEW QUESTIONS\\n\\n1. Plasmids used as cloning vectors\\n\\nA. are circular molecules.\\nB. have an origin of replication.\\nC. carry antibiotic resistance genes.\\nD. have unique restriction endonuclease cutting sites.\\nE. all of pe above.\\n\\n2. Recombinant DNA technology involves:\\n\\n- Making genomic DNA clones by joining fragments of chromosomal DNA to a cloning vector.\\n- Introducing recombinant DNA molecules into bacterial cells through transformation.\\n- Selecting the DNA molecule of interest from a library using an antibody or nucleic acid probe.\\n\\n3. Important uses of DNA clones include:\\n\\n- Determining the base sequence of cloned DNA fragments.\\n- Using in situ hybridization to detect specific cells producing RNA complementary to the clone.\\n- Performing Southern blotting and genetic fingerprinting for DNA analysis.\\n- Synthesizing mammalian proteins in bacteria or eukaryotic cells.\\n- Changing DNA sequences to produce new proteins.\\n- Generating fluorescent protein chimeras for microscopy on live cells.\\n- Utilizing the polymerase chain reaction to produce multiple copies of a DNA molecule.\\n- Producing transgenic animals for studying gene function.\\n\\n# FURTHER READING\\n\\n- McPherson J. D., et al. 2001. A physical map of the human genome. Nature 409: 934–941.\\n- Mullis, K. B. 1990. The unusual origin of the polymerase chain reaction. Sci. Am. 262: 56–65.\\n- Venter, J. C., et al. 2001. The sequence of the human genome. Science 291: 1304–51.\\n- Golden rice: Beyer, P., Al-Babili, S., Ye, X., Lucca, P., Schaub, P., Welsch, R., and Potrykus, I. 2002. Golden Rice: Introducing the beta-carotene biosynthesis pathway into rice endosperm by genetic engineering to defeat vitamin A deficiency J. Nutrition 132: 506S–510S.\\n- Microarrays: Friend, S. H., and Stoughton, R. B. 2002. The magic of microarrays. Sci. Am. 286: 44–49.\\n---\\n# RECOMBINANT DNA AND GENETIC ENGINEERING\\n\\n|2.|A cDNA clone library is a collection of clones representing|\\n|---|---|\\n| |A. the entire genome of the sample tissue.|\\n| |B. all protein-coding gene sequences in the genome of the sample tissue.|\\n| |C. all intronic sequences of genes expressed in the sample tissue.|\\n| |D. the mRNAs expressed in the sample tissue.|\\n| |E. all tRNAs being used in the sample tissue.|\\n|3.|A cDNA clone library can be screened with an antibody if|\\n| |A. the plasmid vector contains a convenient promoter such as that of the lac operon to increase expression.|\\n| |B. the plasmid vector contains a multiple cloning site.|\\n| |C. the nylon “colony lift” membrane is first soaked in alkali to denature DNA.|\\n| |D. the nylon “colony lift” membrane is first treated with a strong protease to break peptide bonds.|\\n| |E. an antibody that recognizes IPTG, and that is linked to an enzyme such as alkaline phosphatase, is available.|\\n|4.|A radioactive gene probe can be synthesized by the method of random priming if the following factors are present in the incubation mixture:|\\n| |A. The DNA template from which the probe will be made, present as single-stranded DNA|\\n| |B. The four deoxynucleotides, dATP, dCTP, dGTP, and dTTP, one of which should carry a radioactive tag|\\n| |C. Primers for DNA synthesis|\\n| |D. DNA polymerase|\\n| |E. All of the above|\\n|5.|To sequence DNA using the automated dideoxy chain termination method, the reaction mixture should contain the DNA to be sequenced plus|\\n| |A. the four deoxynucleotides, dATP, dCTP, dGTP, and dTTP.|\\n| |B. the four dideoxynucleotides, ddATP, ddCTP, ddGTP, and ddTTP, each carrying a different fluorescent tag.|\\n| |C. a DNA primer.|\\n| |D. DNA polymerase.|\\n| |E. all of the above.|\\n|6.|The technique called|\\n| |A. Southern blotting is used to analyze RNA.|\\n| |B. western blotting is used to analyze DNA.|\\n| |C. eastern blotting is used to analyze RNA.|\\n| |D. Southern blotting is used to analyze DNA.|\\n| |E. northern blotting is used to analyze protein.|\\n|7.|Each cycle in the polymerase chain reaction involves the following steps:|\\n| |A. Polymerization then annealing then denaturation|\\n| |B. Denaturation then polymerization then annealing|\\n| |C. Denaturation then annealing then polymerization|\\n| |D. Denaturation then chimera formation then polymerization|\\n| |E. Denaturation then polymerization then sequencing|\\n\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n1. E. All these statements are true. Plasmids used as cloning vectors are circular, have an origin of replication so that they can replicate in the host bacterium, carry antibiotic genes so that transformed\\n---\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n|1.|Cells can be selected for, and have unique restriction enzyme cutting sites to allow genes of interest to be inserted into the plasmid.|\\n|---|---|\\n|2.|D. cDNA means DNA that is complementary to the mRNA found in a tissue sample. cDNAs are generated using a poly-T primer that binds to processed mRNA, but which will not bind to tRNA or rRNA, which have no poly-A sequence.|\\n|3.|A. Addition of the gratuitous inducer lac operon inducer IPTG should ensure good production of the protein, which can then be recognized by the antibody. Considering the other answers: (B) The multiple cloning site makes insertion of the gene of interest easier, but has no implications regarding the method to be used to then screen the library. (C) If one is using an antibody to recognize protein, the presence of DNA is largely irrelevant. (D) The antibody is used to recognize the particular protein coded by the gene of interest. It would therefore be highly counterproductive to destroy the proteins with a protease. (E) There is no reason why one should want to use an antibody that recognizes IPTG.|\\n|4.|E. The primers bind to the template, allowing DNA polymerase to generate the probe using the deoxynucleotides provided. Because one of the deoxynucleotides carries a radioactive tag, the resulting probe is radioactive and can therefore be used, for example, to reveal the position of complementary DNA on a Southern blot.|\\n|5.|E. The primer binds to the DNA to be sequenced. DNA polymerase then begins work using the deoxynucleotides, but each time it instead incorporates a dideoxynucleotide; synthesis of that strand stops so that the DNA of that specific length is flagged with a fluorophore of the color corresponding to the base encountered.|\\n|6.|D. Southern blotting is used to analyze DNA, northern blotting is used to analyze RNA, and western blotting is used to analyze protein (Table 7.2). There is no eastern blotting (although there is a technique called SouthWestern blotting!).|\\n|7.|C. PCR proceeds by cycles of denaturation (separation of the DNA strands), annealing (of primers onto the strands), and polymerization to form double helices.|\\n---\\n# MANUFACTURING PROTEIN\\n\\nThe genetic code (page 75) dictates the sequence of amino acids in a protein molecule. The synthesis of proteins is quite complex, requiring three types of RNA. Messenger RNA (mRNA) contains the code and is the template for protein synthesis. Transfer RNAs (tRNAs) are adapter molecules that carry amino acids to the mRNA. Ribosomal RNAs (rRNAs) form part of the ribosome that brings together all the components necessary for protein synthesis. Several enzymes also help in the construction of new protein molecules. This chapter describes how the nucleotide sequence of an mRNA molecule is translated into the amino acid sequence of a protein.\\n\\nFigure 8.1 shows the basic mechanism of protein synthesis, also called translation. In the first step, free amino acids are attached to tRNA molecules. In the second step, a ribosome assembles on the mRNA strand to initiate synthesis. In the third step, the ribosome travels along the mRNA. At each codon on the RNA a tRNA binds, bringing the amino acid defined by that codon to be added to the growing polypeptide chain. In the last, fourth, step the ribosome encounters a stop codon and protein synthesis is terminated.\\n\\n# ATTACHMENT OF AN AMINO ACID TO ITS tRNA\\n\\nAmino acids are not directly incorporated into protein on a messenger RNA template. An amino acid is carried to the mRNA chain by a tRNA molecule. tRNAs are small, about 70–80 nucleotides in length, and are folded into precise three-dimensional structures because of hydrogen bonding between bases in particular stretches of the molecule. This gives rise\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C2004 by John Wiley & Sons, Inc.\\n---\\n# MANUFACTURING PROTEIN\\n\\n+ = aminoacid (aa)\\n\\nanticodon 1 aa\\n\\ntRNA\\n\\n|ribosome|2|polypeptide|\\n|---|---|---|\\n|5′|3|3|3|4|3′|\\n|mRNA|start|stop|\\n\\n50s 70s\\n\\n30s\\n\\nprotein\\n\\nFigure 8.1. Overview of protein synthesis\\n\\nto four double-stranded regions, and it is these that give tRNA its characteristic cloverleaf\\nstructure when drawn in two dimensions as in Figure 8.2.\\nEach tRNA molecule has an amino acid attachment site at its 3′ end and an anticodon,\\nthree bases that are complementary in sequence to a codon on the mRNA. The tRNA binds\\nto the mRNA molecule because hydrogen bonds form between the anticodon and codon.\\nFor example, the codon for methionine is 5′AUG 3′ which will base pair with the anticodon\\n3′ UAC 5′.\\n\\nTransfer RNA, the Anticodon, and the Wobble\\nAlthough 61 codons specify the 20 different amino acids, there are not 61 tRNAs; instead\\nthe cell economizes. The codons for some amino acids differ only in the third position of the\\ncodon. Figure 4.8 on page 78 shows that when an amino acid is encoded by only two different\\ntriplets the third bases will be either C and U, or A and G. For example aspartate is coded\\nby GAC and GAU and glutamine by CAA and CAG. The wobble hypothesis suggests that\\nthe pairing of the first two bases in the codon and anticodon follows the standard rules—G\\nbonds with C and A bonds with U—but the base pairing in the third position is not as\\nrestricted and can wobble. If there is a pyrimidine, U or C, in the third position of the codon,\\nit can fit with any purine, G or A, in the anticodon, and vice versa. Thus, only one tRNA\\nmolecule is required for two codon sequences. The anticodon of some tRNAs contains\\nthe unusual nucleoside inosine (I), whose base is the purine hypoxanthine (Fig. 2.13, page\\n34). Inosine can base pair with any of U, C, or A in the third position of the codon. Some\\n---\\n# THE RIBOSOME\\n\\nThe ribosome is the cell’s factory for protein synthesis. Each ribosome consists of two subunits, one large and one small, each of which is made up of RNA plus a large number of proteins. The ribosomal subunits and their RNAs are named using a parameter, called the S value. The S value, or Svedberg unit, is a sedimentation rate. It is a measure of how fast a molecule moves in a gravitational field. For example, the bigger a ribosomal subunit, the quicker it will sediment and the larger the S value. Prokaryotic ribosomes, and those found\\n\\nOH     3′ end - amino acid\\n\\nA     attachment site\\n\\nC\\n\\n5′ end    P    C       Hydrogen bonds between complementary bases\\n\\nanticodon\\n\\nFigure 8.2.   Transfer RNA (tRNA).\\n\\ntRNA molecules can therefore base pair with as many as three different codons provided the first two bases of the codon are the same. For example, the tRNA for isoleucine has the anticodon UAI and can therefore base pair with any of AUU, AUC, or AUA.\\n\\nThe attachment of an amino acid to its correct tRNA molecule is illustrated in Figure 8.3. This process occurs in two stages, both catalyzed by the enzyme aminoacyl tRNA synthase. During the first reaction, the amino acid is joined, via its carboxyl group, to an adenosine monophosphate (AMP) and remains bound to the enzyme. All tRNA molecules have at their 3 ′ end the nucleotide sequence CCA. In the second reaction aminoacyl tRNA synthase transfers the amino acid from AMP to the tRNA, forming an ester bond between its carboxyl group and either the 2 - or 3 -hydroxyl group of the ribose of the terminal adenosine (A) on the tRNA to form an aminoacyl tRNA. This step is often referred to as amino acid activation because the energy of the ester bond can be used in the formation of a lower energy peptide bond between two amino acids. A tRNA that is attached to an amino acid is known as a charged tRNA. There are at least 20 aminoacyl tRNA synthases, one for each amino acid and its specific tRNA.\\n---\\n# MANUFACTURING PROTEIN\\n\\nIN DEPTH 8.1 How We Study Proteins in One Dimension\\n\\nThe technique known as SDS–PAGE is widely used to analyze the spectrum of proteins made by a particular tissue, cell type, or organelle. It is also invaluable for assessing the purity of isolated proteins. SDS stands for sodium dodecyl sulfate and PAGE for polyacrylamide gel electrophoresis.\\n\\nThe aim of the technique is to denature the proteins to be analyzed and then to separate them according to their size in an electrical field. To do this we first add a chemical called 2-mercaptoethanol to the protein sample. This will break any disulfide bonds (page 191) within a protein or between protein subunits. Next SDS, which is an anionic detergent, is added and the protein sample boiled. SDS coats each protein chain with negative charge. Each individual polypeptide in the sample becomes covered with an overall net negative charge. This means that when placed in an electrical field the SDS-coated proteins will separate according to their size because the smaller proteins move most quickly toward the positive electrode or anode. Polyacrylamide provides the matrix through which the proteins move during electrophoresis. The monomeric form, acrylamide, is poured into a mold. A solid but porous gel forms as the acrylamide polymerizes. The shape of the mold is such that wells are formed in the gel into which the protein sample can be loaded for electrophoresis.\\n\\nWhen electrophoresis is complete, the proteins are stained by incubating the gel in a solution of Coomassie brilliant blue. Each protein band stains blue and is detectable by eye. However, if the amount of protein is very low, a more sensitive detection system is needed such as a silver stain. Proteins of known molecular mass are also electrophoresed on the gel. By comparison with the standard proteins, the mass of an unknown protein can be determined.\\n\\nIf we want to follow the fate of a single protein in a complex mixture of proteins, we combine SDS–PAGE with a technique called western blotting. The name western blotting is, like northern blotting, a play on the name of Dr. Ed Southern who devised the technique of Southern blotting to analyze DNA (Table 7.2).\\n\\nThe protein mixture is separated by SDS–PAGE. A nylon membrane is then placed up against the polyacrylamide gel and picks up the proteins, so that the pattern of protein spots on the original polyacrylamide gel is preserved on the nylon membrane. The nylon membrane is then incubated with an antibody specific for the protein of interest. This antibody, the primary antibody, will seek out and bind to its partner protein on the nylon membrane. A second antibody is added that will bind to the primary antibody. To be able to detect the specific protein of interest on the membrane, the secondary antibody is attached to an enzyme. In the figure shown, the enzyme used was horseradish peroxidase. A substrate is added and is converted by the enzyme into a colored product. The protein of interest is seen as a colored band on the nylon membrane. The same enzyme-linked secondary antibody can be used in other laboratories or at other times for western blotting of many different proteins because the specificity is determined by the unlabeled primary antibody.\\n\\nPart A of the figure shows the Coomassie brilliant-blue-stained pattern of proteins isolated from the endoplasmic reticulum of liver. The leftmost lane is from a phenobarbital-treated animal while the middle lane is from an untreated control animal. The dark bands indicate the presence of protein. The spectrum of proteins is very similar in the two samples, except that a band with a relative molecular mass (Mr) of about 52,000 is much darker in the sample from the treated animal. This tells us that drug treatment has caused an increase in the production of a protein with this relative molecular mass. Western blotting (part B) using an antiCYP2B1\\n---\\n# SDS-PAGE\\n\\n| |From phenobarbital treated animal|From untreated animal|Molecular weight standards|From phenobarbital treated animal|From untreated animal|\\n|---|---|---|---|---|---|\\n| | | | | | |\\n| | | | | | |\\n|94,000| | | | | |\\n|67,000| | | | | |\\n|52,000| | | | | |\\n|43,000| | | | | |\\n|Proteins separate according to size| | | | | |\\n|30,000| | | | | |\\n| | | | | | |\\n\\nSDS - PAGE\\n\\n(a) Western blot\\n\\n(b)\\n\\nAntibody confirms that the induced protein is the cytochrome P450 protein known as CYP2B1. The CYP2B1 gene is activated by phenobarbital to produce more CYP2B1 protein to metabolize and clear the drug from the body (page 249).\\n\\nWe already showed how northern blotting revealed that transcription of the CYP2B1 gene is increased after phenobarbital treatment (Fig. 7.13 on Page 148).\\n\\nThe western blot shown here demonstrates that, as expected, the amount of CYP2B1 protein is increased as well.\\n\\nSodium dodecyl sulfate (SDS) is a major constituent of hair shampoo, where it is usually called by its alternative name of sodium lauryl sulfate.\\n---\\n# MANUFACTURING PROTEIN\\n\\n|O|amino acid + ATP|\\n|---|---|\\n|O|P|OH|\\n|O|aminoacyl tRNA|\\n|O|P|O|synthase|\\n|O|pyrophosphate|\\n|amino acid|AMP|\\n\\n|tRNA|amino acid|\\n|---|---|\\n| |A|\\n| |C|\\n| |C|\\n\\n|amino acid|AMP|\\n|---|---|\\n|tRNA|aminoacyl tRNA|\\n|AMP|(charged tRNA)|\\n\\nFigure 8.3. Attachment of an amino acid to its tRNA.\\n\\nInside mitochondria and chloroplasts, are 70S when fully assembled and comprise a larger, 50S subunit and a smaller 30S one. Eukaryotic ribosomes are 80S when fully assembled and comprise a larger, 60S subunit and a smaller 40S one.\\n\\nThe formation of a peptide bond (page 38) between two amino acids takes place on the ribosome. The ribosome has binding sites for the mRNA template and for two charged tRNAs (Fig. 8.4). An incoming tRNA with its linked amino acid occupies the aminoacyl site (A site), and the tRNA attached to the growing polypeptide chain occupies the peptidyl site (P site).\\n\\n# BACTERIAL PROTEIN SYNTHESIS\\n\\nRibosome-Binding Site\\n\\nFor protein synthesis to take place, a ribosome must first attach to the mRNA template. AUG is not only the start codon for protein synthesis; it is used to code for all the other methionines in the protein. How does the ribosome recognize the correct AUG at which to\\n---\\n# BACTERIAL PROTEIN SYNTHESIS\\n\\n5′\\nUAG\\nUCG\\nUUC\\nCUG\\n3′\\n\\nFigure 8.4. The P and A sites on the ribosome.\\n\\nbegin protein synthesis? All bacterial mRNAs have at their 5′ end a stretch of nucleotides called the untranslated (or leader) sequence, which do not code for the protein. These nucleotides are nevertheless essential for the correct placing of the ribosome on the mRNA. Bacterial mRNA molecules usually have a nucleotide sequence similar to 5′GGAGG3′ whose center is about 8 to 13 nucleotides upstream of (5′ to) the AUG start codon. This sequence is complementary to a short stretch of sequence, 3′CCUCC5′, found at the 3′ end of the rRNA molecule within the 30S ribosomal subunit (Fig. 8.5). The mRNA and the rRNA interact by complementary base pairing to place the 30S ribosomal subunit in the correct position to start protein synthesis. The sequence on the mRNA molecule is called the ribosome-binding site. This is sometimes referred to as the Shine–Dalgarno sequence after the two scientists who found it.\\n\\nBecause the genetic code is read in triplets of three bases, there are three possible reading frames (page 79). The reading frame that is actually used by the cell is defined by the first AUG that the ribosome encounters downstream of the ribosome-binding site.\\n\\n# Chain Initiation\\n\\nThe first amino acid incorporated into a new bacterial polypeptide is always a modified methionine, formyl methionine (fmet) (Fig. 8.6). Methionine first attaches to a specific tRNA molecule, tRNA fmet, and is then modified by the addition of a formyl group that attaches to its amino group. tRNA fmet has the anticodon sequence 5′CAU3′ that binds to its complementary codon, the universal start codon AUG.\\n---\\n# ribosome binding site start codon\\n\\n|5′ mRNA|GGAGG|AUG|3′|\\n|---|---|---|---|\\n|untranslated sequence| | |+ IF3|\\n\\n|CCUCC|IF3|\\n|---|---|\\n|5′|GGAGG|IF1 AUG|3′|\\n\\n|IF3|\\n|---|\\n|5′ AUG|IF1|3′|\\n\\n|fmet|\\n|---|\\n| |IF2| |GTP|\\n|fmet|\\n|GTP|IF2|IF3|\\n|5′ AUG|IF1|3′|\\n\\n30S initiation complex\\n\\nIF1\\nIF2\\nIF3\\n\\nGDP + Pi fmet PP AA\\n\\n5′ AUG\\n3′\\n\\n70S initiation complex\\n\\nFigure 8.5. Formation of the 70S initiation complex.\\n---\\n# BACTERIAL PROTEIN SYNTHESIS\\n\\nFigure 8.6. Formyl methionine.\\n\\nExample 8.1 The Irritating Formyl Methionine\\n\\nWhite blood cells are strongly attracted by any peptide that begins with formyl methionine: they assume an amoeboid shape (Fig. 1.5 on page 5) and begin to crawl toward the source of the peptide. To a white blood cell, the presence of a peptide beginning with formyl methionine means that there is an infection nearby that needs to be fought. This is because the body’s own proteins do not contain formyl methionine: only prokaryotes begin protein synthesis with this modified amino acid.\\n\\nThe 70S Initiation Complex\\n\\nThe initiation phase of protein synthesis involves the formation of a complex between the ribosomal subunits, an mRNA template and tRNA fmet (Fig. 8.5). A 30S subunit attaches to the ribosome-binding site as described above. tRNA fmet then interacts with the AUG initiation codon, and finally the 50S ribosomal subunit attaches. The ribosome is now complete, and the first tRNA and its amino acid are in place in the P site of the ribosome. A 70S initiation complex has been formed, and protein synthesis can begin. The ribosome is orientated so that it will move along the mRNA in the 5 ′to 3 ′ direction, the direction in which the information encoded in the mRNA molecule is read.\\n\\nThree proteins called initiation factors 1, 2, and 3, together with the nucleotide guanosine triphosphate, are needed to help the 70S initiation complex form. Initiation factors 1 and 3 are attached to the 30S subunit. Initiation factor 3 helps in the recognition of the ribosome-binding site on the mRNA. Initiation factor 2 specifically recognizes tRNA fmet and binds it to the ribosome. When the 50S subunit attaches, the three initiation factors are released and the guanosine triphosphate is hydrolyzed, losing its γ phosphate to become guanosine diphosphate.\\n\\nElongation of the Protein Chain\\n\\nThe synthesis of a protein begins when an aminoacyl tRNA enters the A site of the ribosome (Fig. 8.7). The identity of the incoming aminoacyl tRNA is determined by the codon on\\n---\\n# Figure 8.7. Elongation of the protein chain.\\n\\n| |5′| | | | |3′|\\n|---|---|---|---|---|---|---|\\n| |UAC| |AUG|UUC|CUG| |\\n| | |phe|+|EF-tu|+|GTP|\\n| |Pi|+|EF-tu|GDP| | |\\n| |EF-ts| | | | | |\\n| |GDP|+|EF-tu| | | |\\n| |fmet PP|phe|AA| | | |\\n| |UAC|AAG| | | | |\\n| | | | | | | |\\n| | | | | | | |\\n| |fmet-phe| | | | | |\\n| | |AAG| | | | |\\n| |UUC|CUG| | | | |\\n| | | | | | | |\\n| |GTP|AUG|EF–G| | | |\\n| |GDP|+|Pi| | | |\\n| |EF–G| | | | | |\\n| | | | | | | |\\n| |fmet-phe|PP|AA| | | |\\n| |AAG| | | | | |\\n| | | | | | | |\\n| | | | | | | |\\n| |AUG|UUC|CUG| | | |\\n\\nPi represents the inorganic phosphate ion HPO42−.\\n---\\n# BACTERIAL PROTEIN SYNTHESIS\\n\\nthe mRNA. If, for example, the second codon is 5 UUC 3 , then phenylalanyl tRNA (whose anticodon is 5 GAA 3 ) will occupy the A site. The P site has, of course, already been occupied by tRNA fmet during the formation of the initiation complex. Now that both the A and P sites are occupied, the enzyme peptidyl transferase catalyzes the formation of a peptide bond between the two amino acids (fmet and phe in this example). The dipeptide is attached to the tRNA occupying the A site. Because the tRNA in the P site is no longer attached to an amino acid, it is released from the ribosome and it can be reused. The ribosome now moves along the mRNA to the right so that the tRNA dipeptide now occupies the P site (leaving the A site free for another incoming aminoacyl tRNA). The movement of the ribosome, three nucleotides at a time, relative to the mRNA, is called translocation, a term often used in biology to describe movement that is actively driven and precisely controlled. The process of peptide bond formation is followed by translocation, and the whole process is repeated until the ribosome reaches a stop signal and protein synthesis terminates.\\n\\nProteins are synthesized beginning at their amino or N terminus (page 184). The first amino acid hence has a free (although formylated) amino group. The last amino acid in the chain has a free carboxyl group and is known as the carboxyl or C terminus (page 185).\\n\\nElongation of a polypeptide chain needs the help of three proteins called elongation factors. These proteins are needed to speed up the process of protein synthesis. To bind to the A site, the aminoacyl tRNA must form a complex with elongation factor tu and a molecule of guanosine triphosphate (GTP). On hydrolysis of GTP to guanosine diphosphate (GDP) and inorganic phosphate ion (Pi), the aminoacyl tRNA is able to enter the A site. Elongation factor tu is released from the aminoacyl tRNA by the action of a second protein, elongation factor ts. This removes the GDP bound to elongation factor tu, and the protein is recycled. The movement of the ribosome needs the help of a third protein elongation factor G and the energy supplied by the hydrolysis of a second GTP molecule.\\n\\n# Example 8.2 The Diptheria Bacterium Inhibits Protein Synthesis\\n\\nSome bacteria cause disease because they inhibit eukaryotic protein synthesis. Diphtheria was once a widespread and often fatal disease caused by infection with the bacterium Corynebacterium diptheriae. This organism produces an enzyme (diphtheria toxin) that inactivates eukaryotic elongation factor 2 (the equivalent of the bacterial elongation factor G). Diphtheria toxin splits the bond between ribose and nicotinamide in NAD+ (page 37), releasing free ribose and attaching the remainder, called ADP-ribose, to elongation factor 2, a process known as ADP ribosylation. The protein is now inactive and is unable to assist in the movement of the ribosome along the mRNA template. Protein synthesis therefore stops in the affected human cells. The advantage of this to the bacterium is that all the amino acids that the host was using to make protein are now available for its own use.\\n\\nThe Polyribosome\\n\\nMore than one polypeptide chain is synthesized from an mRNA molecule at any given time. Once a ribosome has begun translocating along the mRNA, the start AUG codon is free, and another ribosome can bind. A second 70S initiation complex forms. Once this ribosome\\n---\\n# MANUFACTURING PROTEIN\\n\\nribosome\\n\\n5′ mRNA\\n3′\\npolypeptide chain\\n\\nFigure 8.8. Many ribosomes reading one mRNA form a polyribosome.\\n\\nhas moved away, a third ribosome can attach to the start codon. This process is repeated until the mRNA is covered with ribosomes. Each of these spans about 80 nucleotides. The resultant structure, the polyribosome or polysome (Fig. 8.8), is visible under the electron microscope. This mechanism allows many protein molecules to be made at the same time on one mRNA.\\n\\nTermination of Protein Synthesis\\n\\nThere are three codons, UAG, UAA, and UGA, that have no corresponding tRNA molecule. These codons are called stop codons. Instead of interacting with tRNAs, the A site occupied by one of these codons is filled by proteins known as chain release factors. In the presence of these factors the newly synthesized polypeptide chain is freed from the ribosome, and the mRNA, tRNA, and the 30S and 50S ribosomal subunits dissociate (Fig. 8.9). Release factor 1 causes polypeptide chain release from UAA and UAG, and release factor 2 terminates chains with UAA and UGA. A third protein, release factor 3, cooperates with the other two to stop protein synthesis. When the A site is occupied by a release factor, the enzyme peptidyl transferase is unable to add an amino acid to the growing polypeptide chain and instead catalyzes the hydrolysis of the bond joining the polypeptide chain to the tRNA. The carboxyl (COOH) end of the protein is therefore freed from the tRNA, and the protein is released.\\n---\\nEUKARYOTIC PROTEIN SYNTHESIS IS A LITTLE MORE COMPLEX\\n\\n|polypeptide chain|release factor 1|\\n|---|---|\\n|5′|UAA|3′|\\n| |peptidyl transferase|\\n|release factor 3| |\\n|5′| |3′|\\n| |release factor 1|\\n|ribosomal subunits dissociate|free polypeptide chain|\\n\\nFigure 8.9. Termination of protein synthesis.\\n\\nThe Ribosome Is Recycled\\nIt would be wasteful if a complex structure such as the ribosome were used only once. When protein synthesis is over, the two ribosomal subunits dissociate and can be reused when needed.\\n\\nEUKARYOTIC PROTEIN SYNTHESIS IS A LITTLE MORE COMPLEX\\n\\nElongation of the polypeptide chain and the termination of protein synthesis in eukaryotes does not differ very much from that described for bacteria. However, the initiation of protein synthesis is more complex in eukaryotes. Their proteins always start with methionine\\n---\\n# MANUFACTURING PROTEIN\\n\\nInstead of the formyl methionine used in bacterial protein synthesis. A special transfer RNA, tRNA met, is used to initiate protein synthesis from the AUG start codon. The methionine is often removed from the protein after synthesis. Eukaryotic mRNAs do not contain the bacterial Shine–Dalgarno sequence for ribosome binding. Eukaryotic mRNAs have at their 5 ′ end a 7-methyl guanosine cap (page 118). A number of proteins are needed to attach the small, 40S, ribosomal subunit to the cap. Once attached, the 40S subunit slides along the leader sequence of the mRNA looking for the start codon. All eukaryotic mRNAs have a sequence very similar to 5 ′ CCACC 3 ′ adjacent to the initiating AUG codon. This sequence (known as the Kozak sequence after the scientist who noted it) tells the ribosome that it has reached the start AUG codon. The recognition of the AUG codon that specifies the start site for translation also requires the help of at least nine proteins. The tRNA met binds to the 40S subunit and then the large 60S ribosomal subunit attaches to form the 80S initiation complex. Protein synthesis can now begin.\\n\\n|Example 8.3|How Hepatitis C Takes Control|\\n|---|---|\\n|The hepatitis C virus shuts down the liver protein synthesis machinery and forces the resources of the infected cell to be diverted into production of viral proteins that will be used to make new virus. It does this by commandeering the cell’s ribosomes to its own mRNAs. Hepatitis C mRNAs do not have a 7-methyl guanosine cap at their 5 ′ ends. Instead they have internal ribosome entry sites (IRES), which are specific sequences to which the ribosomes bind tightly. The clamping of the ribosome to the IRES forces initiation of viral protein synthesis at the expense of host cell protein synthesis.| |\\n|Hepatitis C is a particularly nasty virus because infected people can go on to develop liver cancer, cirrhosis, and other chronic liver disease. In the United States alone, 10,000 people die per year from hepatitis C infection. It is hoped that drugs can be developed that will prevent the host ribosome from binding to the viral IRES and so halt the production of new virus in an infected liver.| |\\n\\nAntibiotics and Protein Synthesis\\n\\nMany antibiotics work by blocking protein synthesis, a property that is extensively exploited in research and medicine. Many antibiotics only inhibit protein synthesis in bacteria and not in eukaryotes. They are therefore extremely useful in the treatment of infections because the invading bacteria will die but protein synthesis in the host organism remains unaffected. Examples are chloramphenicol, which blocks the peptidyl transferase reaction, and tetracycline, which inhibits the binding of an aminoacyl tRNA to the A site of the ribosome. Both of these antibiotics therefore block chain elongation. Streptomycin, on the other hand inhibits the formation of the 70S initiation complex because it prevents tRNA fmet from binding to the P site of the ribosome.\\n\\nPuromycin causes the premature release of polypeptide chains from the ribosome and acts on both bacterial and eukaryotic cells. This antibiotic has been widely used in the study of protein synthesis. Puromycin can occupy the A site of the ribosome because its structure resembles the 3 ′end of an aminoacyl-tRNA (Fig. 8.10). However, puromycin does not bind to the mRNA. Puromycin blocks protein synthesis because peptidyl transferase uses it as a substrate and forms a peptide bond between the growing polypeptide and the antibiotic. Once translocation has occurred, the growing polypeptide has no strong attachment to the mRNA and is therefore released from the ribosome.\\n---\\nEUKARYOTIC PROTEIN SYNTHESIS IS A LITTLE MORE COMPLEX                                     177\\n\\nFigure 8.10.   The antibiotic puromycin inhibits protein synthesis.\\n\\n# IN DEPTH 8.2 Proteomics\\n\\nProteomics is the study of the proteome—the complete protein content of a cell. It is the proteins a cell makes that allow it to carry out its specialized functions. Although, for example, a liver and a kidney cell have many proteins in common, they also each possess a unique subset of proteins, which gives to them their own characteristics. Similarly, a cell will need to make different proteins, and proteins in different amounts, according to its metabolic state. The goal of proteomics is to identify all of the proteins produced by different cells and how a particular disease changes a cell’s protein profile.\\n\\nThe separation of a cell’s protein mixture into individual components is tackled using a technique known as two-dimensional polyacrylamide gel electrophoresis. This produces a pattern of protein spots. These patterns are recorded and serve as templates for the comparison of the proteomes of different cells. Spots that change during a cell’s development, or changes that occur in disease, can be easily identified. A protein spot of interest is excised from the polyacrylamide gel and the protein broken into small, overlapping peptide fragments by proteolytic enzymes.\\n---\\nMANUFACTURING PROTEIN\\n\\n|Separation in first dimension|protein mixture pH 4.0 separates according to charge protein stops migrating when it reaches the pH of its isoelectric point|\\n|---|---|\\n| |pH10.0|\\n|Separation in second dimension|pH4.0|pH10.0|\\n| |proteins separate according to size on an SDS polyacrylamide gel|\\n| |+|\\n\\nThe fragments are fed into a mass spectrometer, and their peptide mass fingerprint determined. The fingerprint identifies the protein. The dream of many scientists, now that we know the base sequence of the human genome, is to define the human proteome and to determine all of its variations. However, the genome projects use a single automated method to sequence the order of the four bases in DNA, and data are obtained at a relatively rapid pace. In contrast it is very labor intensive to identify proteins and much effort is being made to devise ways of speeding things up. The aim of proteomic research centers is to increase their throughput from about 40 to 100 peptide samples an hour, to the daunting number of 1 million peptides a day.\\n\\nSUMMARY\\n\\n1. During protein synthesis, the genetic code in an mRNA is translated into a sequence of amino acids. An amino acid attaches to the 3\\' end of a tRNA to form an aminoacyl tRNA. Protein synthesis takes place on the ribosome, that binds both to mRNA and tRNA. It has two tRNA binding sites, the P site and the A site.\\n2. Initiation of protein synthesis in bacteria involves the binding of the 30S ribosomal subunit to the mRNA. tRNA fmet binds to the initiation codon, and then the 50S ribosomal subunit attaches and the 70S initiation complex is formed.\\n3. Protein synthesis begins when a second aminoacyl tRNA occupies the A site. Each incoming amino acid is specified by the codon on the mRNA. The anticodon on the tRNA hydrogen bonds to the codon, thus positioning the amino acid on the ribosome.\\n---\\n# REVIEW QUESTIONS\\n\\nA peptide bond is formed, by peptidyl transferase, between the amino acids in the P and A sites. The newly synthesized peptide occupies the P site, and another amino acid is brought into the A site. This process of elongation requires a number of proteins (elongation factors); as it continues, the peptide chain grows. When a stop codon is reached, the polypeptide chain is released with the help of proteins known as release factors.\\n\\n4. More than one ribosome can attach to an mRNA. This forms a polyribosome, and many protein molecules can be made simultaneously from the same mRNA.\\n\\n5. Many antibiotics fight disease because they inhibit particular steps in protein synthesis.\\n\\n# FURTHER READING\\n\\nArnez, J. G., and Moras, D. 1997. Structural and functional considerations of the aminoacylation reaction. Trends Biochem. Sci. 22: 211–216.\\n\\nLytle, J. R., Wu, L., and Robertson, H. D. 2001. The ribosome binding site of hepatitis C virus mRNA. J. Virol. 75: 7629–7636.\\n\\nMoore, P. B., and Steitz, T. A. 2002. The involvement of RNA in ribosome function. Nature 418: 229–235.\\n\\nRibas de Pouplana, L., and Schimmel, P. 2001. Aminoacyl-tRNA synthetases: Potential markers of genetic code development. Trends Biochem. Sci. 26: 591–596.\\n\\n# REVIEW QUESTIONS\\n\\nFor each question, choose the ONE BEST answer or completion.\\n\\n1. Which of the following steps is NOT part of translation?\\n- A. Synthesis of mRNA\\n- B. Charging of tRNA\\n- C. Initiation of protein synthesis\\n- D. Elongation of the polypeptide chain\\n- E. Termination of protein synthesis\\n2. In bacterial and eukaryotic mRNAs, the normal start codon for protein synthesis is\\n- A. GAU.\\n- B. UGA.\\n- C. AUG.\\n- D. AGU.\\n- E. UAG.\\n3. The enzyme that attaches the amino acid to its tRNA is\\n- A. aminoacyl mRNA synthase.\\n- B. aminoacyl tRNA synthase.\\n- C. amino acid attachment enzyme.\\n---\\n# MANUFACTURING PROTEIN\\n\\nD. elongation factor tu.\\n\\nE. tRNA polymerase.\\n\\n4. Bacterial mRNAs usually have the nucleotide sequence 5\\' GGAGG 3\\', which bonds to\\n\\nA. tRNA.\\n\\nB. methionine.\\n\\nC. the 30S ribosomal subunit.\\n\\nD. the 40S ribosomal subunit.\\n\\nE. protein initiation factors.\\n\\n5. The enzyme that catalyzes the formation of a peptide bond between two amino acids is\\n\\nA. amino acid transferase.\\n\\nB. peptide joinase.\\n\\nC. peptidyl transferase.\\n\\nD. peptide elongation factor.\\n\\nE. puromycin.\\n\\n6. Which of the following do NOT play a part in the termination of protein synthesis?\\n\\nA. The codons AUG, AGG, and GUA\\n\\nB. The codons UAA, UAG and UGA\\n\\nC. Release factor 1\\n\\nD. Release factor 3\\n\\nE. Peptidyl transferase\\n\\n7. The base hypoxanthine in the nucleotide inosine can hydrogen bond with\\n\\nA. G, U, and C.\\n\\nB. G, U, and A.\\n\\nC. G, A, and C.\\n\\nD. U, C, and A.\\n\\nE. A only.\\n\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n1. A. Synthesis of mRNA is not part of translation. All the others are, in particular, “charging” means attaching an amino acid to a tRNA ready for use in protein synthesis.\\n\\n2. C. AUG, which codes for the amino acid methionine.\\n\\n3. B. The enzyme is called aminoacyl tRNA synthase. Concerning the other answers: (A) Because amino acids are not attached to mRNA, there is no such process in cells. (C) Amino acid attachment enzyme is a plausible name, albeit less specific since it does not say to what the amino acid is attached. However, the name is not used. (D) Elongation factor tu plays a role later in the process of protein synthesis, not during tRNA charging. (E) tRNA polymerase is not a name that is ever used. If it did mean anything, it would mean the enzyme that makes tRNA (there is such an enzyme in eukaryotes: it is called RNA polymerase III), not the one that attaches an amino acid.\\n\\n4. C. The rRNA molecule within the 30S ribosomal subunit has a short stretch of sequence, 3\\'CCUCC 5\\', at its 3\\' end that is complementary in sequence to the 5\\'GGAGG 3\\' of the mRNA. Hydrogen bonding between these two sequences aligns the ribosome for protein synthesis. Concerning answer D: the 40S ribosomal subunit plays a similar role in eukaryotes as the 30S unit does in prokaryotes. However, eukaryotic mRNAs do not have the prokaryotic GGAGG Shine–Dalgarno sequence.\\n---\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n5.\\nC.\\nThe enzyme is called peptidyl transferase. Concerning the other answers: A, B, and D are plausible answers, though D is the least plausible since most enzymes have names ending in ase; however, they are not the names used. Puromycin is an antibiotic, not a protein (and therefore certainly not an enzyme).\\n\\n6.\\nA.\\nThese are codons directing the incorporation of methionine, arginine, and valine, respectively. All the others do play a part in the termination of protein synthesis, in particular, UAA, UAG, and UGA are stop codons, while if the A site is occupied by a release factor, peptidyl transferase catalyzes the hydrolysis of the bond joining the polypeptide chain to the tRNA. The carboxyl (COOH) end of the protein is therefore freed from the tRNA and the completed protein is released.\\n\\n7.\\nD.\\nHypoxanthine can bond with any of U, C, or A.\\n---\\n# PROTEIN STRUCTURE\\n\\nVirtually everything cells do depends upon proteins. We are all, regardless of build, made up of water plus more or less equal amounts of fat and protein. Although the DNA in our cells contains the information necessary to make our bodies, DNA itself is not a significant part of our body mass. Nor is it a chemically interesting molecule, in the sense that one length of DNA is much the same as another in terms of shape and chemical reactivity. The simplicity of DNA arises because it is a polymer made up of only four fairly similar monomers, and this is appropriate because the function of DNA is simply to remain as a record and to be read during transcription. In contrast, proteins made using the instructions in DNA vary enormously in physical characteristics and function and can be considered as constituting the much more complex proteome (page 178). Silk, hair, the lens of an eye, an immunoassay (such as found in a pregnancy test kit), and cottage cheese are all just protein plus more or less water, but they are different because the proteins they contain are different. Proteins carry out almost all of the functions of the living cell including, of course, the synthesis of new DNA. Neither growth nor development would be possible without proteins.\\n\\nMost proteins have functions that depend on their ability to recognize other molecules by binding. This recognition depends on specific three-dimensional binding sites that make multiple interactions with the ligand, the molecule being bound. To do this a protein must itself have a specific three-dimensional structure. Each of the huge number of protein functions demands its own protein structure. Evolution has produced this diversity by using a palette of 20 amino acid monomers, each with its own unique shape and chemical properties, as the building blocks of proteins. Huge numbers of very different structures are therefore possible.\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\n\\nISBN 0-471-26393-1 Copyright ©C 2004 by John Wiley & Sons, Inc.\\n---\\n# PROTEIN STRUCTURE\\n\\nNAMING PROTEINS\\n\\nTo be able to discuss proteins, we need to give them names. Naming conventions vary between different areas of biology. We have already seen how enzymes such as DNA polymerase are named for the reaction they catalyze. Many proteins have names that describe their structure or their role in cells, such as hemoglobin and connexin. However, the pace with which new proteins are being discovered at the moment means that many are not given proper names but are simply referred to by their size: p38 (page 418) and p53 (page 415) have relative molecular masses of about 38,000 and about 53,000, respectively. Clearly this could cause confusion, so we sometimes add the name of the gene as a superscript: p16INK4a is a protein of relative molecular mass (M) of about 16,000 that is the product of the INK4a gene.\\n\\nPOLYMERS OF AMINO ACIDS\\n\\nTranslation produces linear polymers of α-amino acids. If there are fewer than around 50 amino acids in a polymer, we tend to call it a peptide. More and it is a polypeptide. Proteins are polypeptides, and most have dimensions of a few nanometers (nm), although structural proteins like keratin in hair are much bigger. The relative molecular masses of proteins can range from 5000 to hundreds of thousands.\\n\\nThe Amino Acid Building Blocks\\n\\nThe general structure of α-amino acids, the building blocks of polypeptides and proteins, is shown in Figure 9.1. R is the side chain. It is the side chain that gives each amino acid its unique properties. During the process of translation (page 173), peptidyl transferase joins the amino group of one amino acid to the carboxyl of the next to generate a peptide bond. A generalized polypeptide is shown in Figure 9.1. The backbone, a series of peptide bonds separated by the α carbons, is shown in green. At the left-hand end is a free amino group; this is known as the N terminal or amino terminal. At the right-hand end is a free carboxyl group; this is\\n\\n|R|\\n|---|\\n|OOC|CH|NH3|\\n|α - amino acid|\\n\\nFigure 9.1. α-Amino acids and the peptide bond.\\n---\\n# POLYMERS OF AMINO ACIDS\\n\\nknown as the C terminal or carboxy terminal. Peptides are normally written this way, with\\nthe C terminal to the right.\\n\\nThe properties of individual polypeptides are conferred by the side chains of their\\nconstituent amino acids. Many different properties are important—size, electrical charge,\\nthe ability to participate in particular reactions—but the most important is the affinity of the\\nside chain for water. Side chains that interact strongly with water are hydrophilic. Those\\nthat do not are hydrophobic. We have already encountered the 20 amino acids coded for by\\nthe genetic code (page 75), but we will now describe each in turn, beginning with the most\\nhydrophilic and ending with the most hydrophobic. Each amino acid has both a three-letter\\nabbreviation and a one-letter code (Fig. 9.2). In the following section we refer to each amino\\nacid by its full name and give the three- and one-letter abbreviations. This will help you to\\nfamiliarize yourself with the amino acid abbreviations, which are used in other sections of\\nthe book.\\n\\nFigure 9.3 shows four amino acids with hydrophilic side chains. Aspartate (Asp, D) and\\nglutamate (Glu, E) have acidic side chains. At the pH of the cytoplasm, these side chains\\nbear negative charges, which interact strongly with water molecules. As they are usually\\ncharged, we generally name them as their ionized forms: aspartate and glutamate rather\\nthan aspartic acid and glutamic acid. A polypeptide made entirely of these amino acids is\\nvery soluble in water. Lysine (Lys, K) and arginine (Arg, R) have basic side chains. At the\\npH of the cytoplasm, these side chains bear positive charges, which again interact strongly\\nwith water molecules.\\n\\nFigure 9.4 shows histidine (His, H), whose side chain is weakly basic with a pKa of\\n7.0. At neutral pH about half the histidine side chains, therefore, bear a positive charge. The\\nfact that histidine is equally balanced between protonated and unprotonated forms gives it\\nimportant roles in enzyme catalysis. At neutral pH a polypeptide made entirely of histidine\\nwill be very soluble in water.\\n\\nFigure 9.4 also shows cysteine (Cys, C) whose thiol (—SH) group is weakly acidic\\nwith a pKa (page 25) of about 8. At neutral pH most (about 90%) of cysteine side chains\\nwill have their hydrogen attached. Even so the charge on the remaining 10% means that\\na polypeptide made entirely of cysteine will be soluble in water. Cysteine’s thiol group is\\nchemically reactive and has important roles in some enzyme-active sites. Two cysteine thiol\\ngroups can form a disulfide bond (page 191).\\n\\nFigure 9.5 shows the three amino acids serine (Ser, S), threonine (Thr, T), and tyrosine\\n(Tyr, Y). These amino acids have hydroxyl (—OH) groups in their side chains that can\\nhydrogen bond with water molecules. A polypeptide composed of these amino acids is\\nsoluble in water.\\n\\nAsparagine (Asn, N) and glutamine (Gln, Q) (Fig. 9.6) are the amides of aspartate and\\nglutamate. They are hydrophilic.\\n\\nGlycine (Gly, G; Fig. 9.6) has nothing but a hydrogen atom for its side chain. It is\\nrelatively indifferent to its surroundings.\\n\\nThe five amino acids shown in Figure 9.7 have side chains of carbon and hydrogen\\nonly. These are alanine (Ala, A), valine, (Val, V), leucine (Leu, L), isoleucine (Ile, I), and\\nphenylalanine (Phe, F). The side chains cannot interact with water so they are hydrophobic.\\nA polypeptide composed entirely of these amino acids does not dissolve in water but will\\ndissolve in olive oil.\\n\\nTryptophan (Trp, W; Fig. 9.8) is the largest of the amino acids. Its double ring side\\nchain is mainly hydrophobic. Methionine (Met, M; Fig. 9.8) is also hydrophobic: its sulfur\\natom is in the middle of the chain so cannot interact with water. Last comes proline (Pro, P;\\nFig. 9.9). Proline is not really an amino acid at all—it is an imino acid, but biologists give it\\n---\\n# PROTEIN STRUCTURE\\n\\n|alanine (ala)|asparagine (asn)|aspartate (asp)|arginine (arg)|\\n|---|---|---|---|\\n|A|N|D|R|\\n|GCU|AAU|GAU|CGU|\\n|CH3|GCC|NH2|AAG|\\n|GCA|CCC|CCC|CGC|\\n|GCG|CH|CH|CGA|\\n| | |Negatively charged.| |\\n| |2 Site for attachment|2 charged.| |\\n| |of sugars| | |\\n| | |Can be phosphorylated|Positively charged|\\n|cysteine (cys)|glutamine (gln)|glutamate (glu)|glycine (gly)|\\n|C|Q|E|G|\\n|SH|UGU|NH|GAA|\\n|UGC| |CCC|GGC|\\n|CH2| | |GGA|\\n| | | |GGG|\\n|About 10% are deprotonated and hence negatively charged.| |Negatively charged|The smallest side chain|\\n|Forms disulfide bonds| |Can be phosphorylated| |\\n|histidine (his)|isoleucine (ile)|leucine (leu)|lysine (lys)|\\n|H|I|L|K|\\n|HN|CAU|CH|AUU|\\n|CAC|3|AUC|3|\\n|HC|About 50%|CH| |\\n|N|are protonated.| | |\\n|H|pK is 7.0|H3C| |\\n|Can be phosphorylated| | | |\\n|methionine (met)|phenylalanine (phe)|proline (pro)|serine (ser)|\\n|M|F|P|S|\\n|CH|AUG|UUU|AGU|\\n|3| |UUC|AGC|\\n|S| |CCA|UCU|\\n|(CH2)2|CH| |UCC|\\n| |Introduces a kink| |Can be phosphorylated|\\n|threonine (thr)|tryptophan (trp)|tyrosine (tyr)|valine (val)|\\n|T|W|Y|V|\\n|ACU|UGG|UAU|GUU|\\n|ACC| |OH|GUC|\\n|HO| | |GUG|\\n|Can be phosphorylated| |The largest side chain|Can be phosphorylated|\\n|Site for attachment of sugars| |STOP|STOP|\\n| | |UGA|UAA|\\n| | |UAG| |\\n\\nFigure 9.2. The genetic code and the corresponding amino acid side chains, with special properties noted. Hydrophilic side chains are shown in green, hydrophobic side chains in black.\\n\\nhonorary amino acid status. Because the side chain (which is hydrophobic) is connected to the nitrogen atom and therefore to the peptide bond, including proline in the polypeptide chain introduces a kink.\\n\\nHow is it that amino acids with hydrophobic side chains are present in solution in the cytosol and can be picked up by their respective tRNAs? The answer is simple: as free amino acids, they all bear two charges, a positive charge on the amino group and a negative\\n---\\n|Three-letter|One-letter|Amino Acid|Side Chain|\\n|---|---|---|---|\\n|Asp|D|Aspartate|CH2 OOC CH NH3|\\n|Glu|E|Glutamate|CH2 CH2 OOC CH NH3|\\n|Lys|K|Lysine|CH2 CH2 CH2 OOC CH NH3 NH2 C NH2|\\n|Arg|R|Arginine|NH CH2 CH2 CH2 OOC CH NH3|\\n\\n|Three-letter|One-letter|Amino Acid|Side Chain|\\n|---|---|---|---|\\n|His|H|Histidine|HCHNHNHN CH NNN C + H pK = 7.0 NNN C CH2 H CH2 OOC CH NH 3 OOC CH NH 3|\\n|Cys|C|Cysteine|SH CH2 pK = 8.0 CH2 + H OOC CH NH 3 OOC CH NH 3|\\n---\\n# PROTEIN STRUCTURE\\n\\n|Abbreviation|Letter|Name|\\n|---|---|---|\\n|Ser|S|Serine|\\n|Thr|T|Threonine|\\n|Tyr|Y|Tyrosine|\\n\\nFigure 9.5. Amino acids with hydroxl groups in their side chains.\\n\\ncharge on the carboxyl group. Like all small ions they are soluble in water. Formation of\\nthe peptide bond removes these charges. The term residue is used to refer to the amino acid\\nwhen it is part of a peptide chain.\\n\\nThe Unique Properties of Each Amino Acid\\nAlthough we have classified side chains on the basis of their affinity for water, their other\\nproperties are important (Fig. 9.2).\\n\\n|Abbreviation|Letter|Name|\\n|---|---|---|\\n|Asn|N|Asparagine|\\n|Gln|Q|Glutamine|\\n|Gly|G|Glycine|\\n\\nFigure 9.6. Asparagine, glutamine, and glycine.\\n---\\n|Amino Acid|Abbreviation|Name|Side Chain|\\n|---|---|---|---|\\n|Ala|A|Alanine|OOCCH2CHNH3|\\n|Val|V|Valine|CH3CH2OOCCHNH3|\\n|Leu|L|Leucine|CH3CH2CH2OOCCHNH3|\\n|IIe|I|Isoleucine|CH3CH2CHCH2OOCCHNH3|\\n|Phe|F|Phenylalanine|CH2OOCCHNH3|\\n\\nFigure 9.7. Amino acids with simple hydrophobic side chains.\\n\\n|Amino Acid|Abbreviation|Name|Side Chain|\\n|---|---|---|---|\\n|Trp|W|Tryptophan|HNNNCH2OOCCHNH3S|\\n|Met|M|Methionine|CH2CH2OOCCHNH3|\\n\\nFigure 9.8. Amino acids with more complicated hydrophobic side chains.\\n---\\n# PROTEIN STRUCTURE\\n\\nFigure 9.9. Proline, an imino acid.\\n\\nCHARGE. If a side chain is charged it will be hydrophilic. But charge has other effects too. A positively charged residue such as a lysine will attract a negatively charged residue such as glutamate. If the two residues are buried deep within a folded polypeptide, where neither can interact with water, then it will be very difficult to pull them apart. We call such an electrostatic bond inside a protein a salt bridge. Negatively charged residues will attract positively charged ions out of solution, so a pocket on the surface of a protein lined with negatively charged residues will, if it is the right size, form a binding site for a particular positively charged ion like sodium or calcium.\\n\\nPOSTTRANSLATIONAL MODIFICATIONS. After a polypeptide has been synthesized some of the amino acid side chains may be modified. A good example of this is glycosylation, in which chains of sugars may be added to asparagine or threonine side chains (page 225). A very specific posttranslational modification occurs in the connective tissue protein collagen (page 13). After the polypeptide chain has been synthesized, specific prolines and lysines are hydroxylated to hydroxyproline and hydroxylysine. Because hydroxyl groups can hydrogen bond with water, this helps the extracellular matrix to form a hydrated gel (page 13).\\n\\n|Example 9.1|Hydroxyproline, Hydroxylysine, and Scurvy|\\n|---|---|\\n| |Scurvy, a very unpleasant disease in which blood vessels become leaky, is caused by a lack of vitamin C in the diet. As far as we know, the only use of vitamin C in the body is to participate in the post-translational modification of proline and lysine to generate the hydroxyproline and hydroxylysine in collagen molecules. Without vitamin C collagen, necessary for the physical strength of blood vessels, fails to form properly. In 1747 James Lind demonstrated that citrus fruits (which we now know to be rich in vitamin C) prevented scurvy, a finding that allowed explorers to make long-distance sea voyages that had previously been impossible.|\\n\\nNot all posttranslational modifications are so permanent. One very important modification is phosphorylation, the attachment of a phosphate group. We have already seen how sugars can be phosphorylated (page 33), but so can the side chains of the six amino acids serine, threonine, tyrosine, aspartate, glutamate, and histidine (Fig. 9.10). Usually the phosphate group comes from ATP, in which case the enzyme that does the job is called a kinase. Phosphate groups carry two negative charges and can therefore markedly alter the balance of electrical forces within a protein. Another group of enzymes called phosphatases remove the phosphate groups. Since phosphorylation is readily reversible, it is often used to turn proteins off and on. For example, the transcription factor NFAT (page 222) is only\\n---\\nOTHER AMINO ACIDS ARE FOUND IN NATURE\\n\\n|phosphoesters| |O|\\n|---|---|---|\\n|O|P|O|\\n| | |O|\\n|O|P|O|\\n| | |O|\\n|CH3| |O|\\n|CH2| |CH|\\n|C|CH|NH|\\n| | |O|\\n|O| |O|\\n|phosphoserine| |phosphothreonine|\\n|phosphotyrosine| | |\\n|phosphoanhydrides| |phosphoimides|\\n|O|O| |\\n|C|O|P|O|\\n|CH2|O| |\\n|CH2| | |\\n|C|CH|NH|\\n| | |O|\\n|phosphoaspartate| |phosphoglutamate|\\n|phosphohistidine| | |\\n\\nFigure 9.10. Six side chains can be phosphorylated.\\n\\nactive in the dephosphorylated state, while insulin receptor substrate number 1 (page 356) is activated by phosphorylation.\\n\\nDISULFIDE BONDING. Under oxidizing conditions the thiol (—SH) groups of two cysteine residues can link together to form a disulfide bond (or disulfide bridge) (Fig. 9.11). Proteins made for use outside the cell often have disulfide bonds that confer additional rigidity on the protein molecule. If all the peptide bonds in a polypeptide are hydrolyzed, any cysteines that were linked by disulfide bonds remain connected in dimers called—very confusingly—cystine molecules.\\n\\nOTHER AMINO ACIDS ARE FOUND IN NATURE\\n\\nOrnithine and citrulline (page 293) are α-amino acids that play a vital role in the body—they are used as part of the urea cycle to get rid of ammonium ions, which would otherwise poison us (page 293). However, they are not used as building blocks in the synthesis of polypeptides.\\n---\\n# PROTEIN STRUCTURE\\n\\nNH                          NH\\n\\nH    C CH     2  S    S   CH  2   C    H\\n\\nO      C                           C      O\\n\\ndisulfide bond\\n\\nbetween 2 cysteines\\n\\nNH  3                       NH   3\\n\\nH    C CH     2  S    S   CH  2   C    H\\n\\nCOO                         COO\\n\\nthe double amino acid cystine\\n\\nFigure 9.11. Oxidation of adjoining cysteine residues produces a disulfide bond. Proteolysis of the polypeptide releases cystine.\\n\\nThere are many other nonprotein amino acids. Very early in the evolution of life, the palette of amino acids used to make polypeptides became fixed at the 20 that we have described. Almost the entire substance of all living things on earth is either polypeptide, composed of these 20 monomers, or other molecules synthesized by enzymes that are themselves proteins made of these 20 monomers. Natural selection has directed evolution within the constraints imposed by the palette.\\n\\n# THE THREE-DIMENSIONAL STRUCTURES OF PROTEINS\\n\\nProteins are polypeptides folded into specific, complex three-dimensional shapes. Generally, hydrophobic amino acids pack into the interior of the protein while hydrophilic amino acid side chains end up on the surface where they can interact with water. The amino acid side chains pack together tightly in the interior: there are no spaces. It is the three-dimensional structure of a particular protein that allows it to carry out its role in the cell. The shape can be fully defined by stating the position and orientation of each amino acid, and such knowledge lets us produce representations of the protein (Fig. 9.12). However, when discussing protein structure, it is helpful to think in terms of a series of different levels of complexity.\\n\\nThe primary structure of a protein is the sequence of amino acids. Lysozyme is an enzyme that attacks bacterial cell walls. It is found in secretions such as tears and in the white of eggs. Lysozyme has the following primary structure:\\n\\n(NH  2 )KVFGRCELAAAMKRHGLDNYRGYSLGNWVCAAKFESNFNTQATNRNTD\\nGSTDYGILQINSRWWCDNGRTPGSRNLCNIPCSALLSSDITASVNCAKKIVSDGDG\\nMNAWVAWRNRCKGTDVQAWIRGCRL(COOH).\\n\\nNumbering is always from the amino terminal end where synthesis of the protein begins on the ribosome. Lysozyme has four disulfide bonds between four pairs of cysteines. The 129 amino acids of hen egg white lysozyme are shown in linear order in Figure 9.12a\\n---\\n# THE THREE-DIMENSIONAL STRUCTURES OF PROTEINS\\n\\n| |20| | | | | | | | | | | | | | | | | | |\\n|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\\n|H|G|L|D|N|Y|R|G|Y|S|L|G|N|W|V| | | | | |\\n|K|R| | |W|A|Q|V|D|T|G| | | | |C|30| | | |\\n|M| | |IR| | | | | |120| |K|C|SSSSSS|A| | | | | |\\n|A| | | |G|C| | |129| | | |R| | |A| | | | |\\n|10|A|A| | |SSS|R|L| | | |N| | | |L| | | | |\\n| | |L|EC|SSS| | | |COO|−|A|W|R| | |EF| | | | |\\n| | |R| | |W|V|110| | | | | | | |S| | | | |\\n|+|1|V|FG| |N|A| | | | | | | | |NF| | | | |\\n|H3N| |K| |D|GM|SSS|CL|N|R|S|G|P|70| | |NT|Q|40| |\\n| | | | |G| | |N| | | | |T| | | |A| | | |\\n| | | |100|SD| | | |I| | | |GR| | | |TN| | | |\\n| | |V| | |80|PC| |S| | | |DN| | | |R| | | |\\n| | | |KI| | |S| | |S| |S|C| | | |N| | | |\\n| | | |K| | | | |AL| | |W| | | |T| | | | |\\n| | | |A| |S| | | | | |W| | | |D| | | | |\\n| | |C| |SS| | | |L| | |R| | | |G| | | | |\\n| |NV|S|90| |I|D|S|S|60|SNI| | | | | | | |DTS|50|\\n| | |A|T| | | | | | | |(a)| |N|L|I|G|Y| | |\\n\\n(b)\\n\\n|70|\\n|---|\\n|50|60| | | |100|\\n| |80| |90| |20| | | |alpha helix|\\n| | | |110|\\n| |40| | |30| | |10|120|βstrands|\\n\\n(c)\\n\\n(d)\\n\\nFigure 9.12. Lysozyme: (a) linear map, (b) space filling model, (c) backbone representation, and (d) cartoon.\\n---\\n|H|H|H|H|H|\\n|---|---|---|---|---|\\n|N|C|C|N|C|C|N|C|C|N|C|C|N|C|C|N|C|C|\\n|H|R|O|H|R|O|H|R|O|H|R|O|H|R|O|\\n\\n1 2 3 4 5\\n\\nhydrogen bond\\n\\n(a)\\nH\\n\\nN\\n\\nC α\\n\\nC\\n\\nN\\n\\nC α\\n\\nsingle hydrogen bond\\n\\nHN Cα H R NH R\\n\\nN Cα N C\\n\\nH R Cα ON C C α H\\n\\nC α C\\n\\nC\\n\\nN CαN\\n\\nH Cα C O N H Cα H R\\n\\nC αN N C C Cα\\n\\nR C αO HN C H OCNCRα H\\n\\nC Cα\\n\\nH N R O\\n\\nN\\n\\nC\\n\\n(b) C C α\\n\\nH O O C N C Cα H R\\n\\nα-helix showing the position\\n\\nOO\\n\\n(c)\\n\\nα-helix with all\\n\\nhydrogen bonds shown\\n\\n(d)\\n\\nα-helix top view\\n\\nFigure 9.13. The α helix: (a) the nitrogen atom in each peptide bond forms a hydrogen bond with the oxygen of the peptide bond four ahead. (b) An α helix, with one hydrogen bond drawn. (c) An α helix, with all the hydrogen bonds drawn. (d ) An end-on view of an α helix.\\n\\n194\\n---\\n# THE THREE-DIMENSIONAL STRUCTURES OF PROTEINS\\n\\nwith the disulfide bonds indicated. Lysozyme was the first enzyme to have its three-dimensional structure fully determined (in 1965). Figure 9.12b shows that structure, with all of the atoms that form the molecule displayed. We see little except an irregular surface. However, if the amino acid side chains are stripped away and the path of the peptide-bonded backbone drawn (Fig. 9.12c), we see that some regions of the protein backbone are ordered in a repeating pattern. Two types of protein backbone organization are common to many proteins. These are named the α helix and the β sheet. Figure 9.12d redraws the peptide backbone of lysozyme to emphasize these patterns, with the lengths of peptide participating in β sheets represented as arrows. Collectively these repeating patterns are known as secondary structures. There are other regions of the protein that do not have any such ordered pattern.\\n\\nIn an α helix the polypeptide chain twists around in a spiral, each turn of the helix taking 3.6 amino acid residues. This allows the nitrogen atom in each peptide bond to form a hydrogen bond with the oxygen four residues ahead of it in the polypeptide chain (Fig. 9.13). All the peptide bonds in the helix are able to form such hydrogen bonds, producing a rod in which the amino acid side chains point outward. Because it introduces a kink into the polypeptide chain, proline cannot participate in an α helix.\\n\\nIn a β sheet lengths of polypeptide run alongside each other, and hydrogen bonds form between the peptide bonds of the strands. This generates a sheet that has the side chains protruding above and below it (Fig. 9.14). Along a single strand the side chains alternate up then down, up then down. Because the actual geometry prevents them from being completely flat, they are sometimes called β pleated sheets. A polypeptide chain can form two types of β sheet: Either all of the strands in the β sheet are running in the same direction forming a parallel β sheet or they can alternate in direction making an antiparallel β sheet. The polypeptide chains in β sheets are fully extended unlike the chain in an α helix.\\n\\nIn structural proteins like the keratin in hair or the fibroin in silk, the whole polypeptide chain is ordered into one of these secondary structures. Such fibrous proteins have relatively simple repeating shapes and do not have binding sites for other molecules. Most proteins have regions without secondary structures, the precise folding and packing of the amino acids being unique to the protein, side by side with regions of secondary structure. We have seen that secondary structures such as the α helix and β sheet are formed because of hydrogen bonds involving the peptide bonds of the backbone. The three-dimensional structure is held together by various interactions most of which are individually weak but collectively produce stable molecules. These interactions involve the amino acid side chains that interact with one another, with the backbone, with water molecules, and (if there is one) with a prosthetic group or cofactor (page 205). These interactions are hydrogen bonds, electrostatic interactions, van der Waals forces, hydrophobic interactions, and (in some proteins) disulfide bonds.\\n\\n# Hydrogen Bonds\\n\\nIf a hydrogen attached to an oxygen, nitrogen, or sulfur by a covalent bond gets close to a second electron-grabbing atom, then that second atom also grabs a small share of the electrons to form a hydrogen bond. The atom to which the hydrogen is covalently bonded is called the donor because it is losing some of its share of electrons; the other electron-grabbing atom is the acceptor. In order for a hydrogen bond to form, the donor and acceptor must be within a fixed distance of one another (typically 0.3 nm) with the hydrogen on a straight line between them. In an α helix the nitrogen atom within a peptide bond shares its\\n---\\nIN DEPTH 9.1 Chirality and Amino Acids PROTEIN STRUCTURE\\n\\nA chiral structure is one in which the mirror image of the structure cannot be superimposed on the structure. The shape of your whole body is not chiral. Your mirror image could be rotated through 180°, so that it faces into the mirror, then step back and be superimposed on you. Your right hand, however, is chiral: Its mirror image is not a right hand anymore but a left hand. Molecules are often chiral—this arises if a carbon atom has four different groups attached to it and is therefore asymmetric. The α carbon of α-amino acids is asymmetric for all except glycine. It is possible for there to be more than one asymmetric carbon in a molecule. If there is an asymmetric carbon in a molecule, there will be two different ways that the groups on that carbon can be arranged and so there will be two optical isomers. These isomers interact with polarized light differently. Although a different system is used in chemistry, optical isomers of amino acids (and sugars) are denoted L and D. L amino acids are exclusively used in proteins and predominate in the metabolism of amino acids generally. However D amino acids are found in nature. D alanine occurs in bacterial cell walls while some antibiotics such as valinomycin and gramicidin A contain D amino acids. These molecules are synthesized in entirely different ways from proteins. Among sugars it is the D forms that are mainly used by cells.\\n\\n|COO| |H|\\n|---|---|---|\\n|NH3|C|H| | |NH3|C| |COO|\\n|CH3| | | | |CH3|\\n|L-alanine| | | | |D-alanine|\\n|right handed| | | | |left handed|\\n|α helix| | | | |α helix|\\n\\nHelices are chiral too. The α helix found in proteins is right-handed, like a regular screw thread. Reflect a length of right-handed α helix in a mirror, and you will get a left-handed α helix composed of D amino acids. Because of the actual structures, it turns out that L amino acids fit nicely in a right-handed α helix, while D amino acids fit nicely in a left-handed α helix. (If you doubt the chirality of helices, try making wire models of the two shown in the accompanying figure and see if they can be superimposed.)\\n---\\n# THE THREE-DIMENSIONAL STRUCTURES OF PROTEINS\\n\\n|(a)|O|R|H|O|R|H|\\n|---|---|---|---|---|---|---|\\n|H| |C|N|C|C|N|C|C|N|C|C|N|C|\\n|R|H|H|O|R|H|H|O|R|\\n| |O|R|H|O|R|H|\\n|H| |C|C|N|H|C|C|N|H|\\n| | |C|N|H|C|C|N|H|C|C|\\n|R|H|O|R|H|O|R|\\n| |O|R|H|O|R|H|\\n|H| |C|C|N|H|C|C|N|H|\\n| | |C|N|H|C|C|N|H|C|C|\\n|R|H|O|R|H|O|R|\\n|N| | | | | | |\\n|N| | | | | | |\\n|N|parallel β-sheets| | | | | |\\n\\n|(b)|O|R|H|O|R|H|\\n|---|---|---|---|---|---|---|\\n|H|C|C|N|H|C|C|N|H|\\n| |C|N|H|C|C|N|H|C|C|\\n| |R|O|HH|H|R|O|H|H|R|\\n| |C|N|C|C|N|C|C|N|C|C|N|C|C|\\n| |H|RR|O|H|H|R|O|H|\\n| |O|R|H|O|R|H|\\n|H|C|C|N|H|C|C|N|H|\\n| |C|N|H|C|C|N|H|C|C|\\n|R|H|O|R|H|O|R|\\n|N| | | | | | |\\n| |C| | | | | |\\n|N| | | | | | |\\n|antiparallel β-sheets| | | | | | |\\n\\nFigure 9.14. (a) Parallel and (b) antiparallel β sheets.\\n---\\n198\\n\\nPROTEIN STRUCTURE\\nextracellular\\nmedium\\n\\nin this region N amino acid side chains are hydrophobic\\n\\n|ATP|binding|\\n|---|---|\\n|cytosol| |\\n|C| |\\n\\nFigure 9.15. Cartoon representation of the platelet derived growth factor receptor.\\n---\\n# THE THREE-DIMENSIONAL STRUCTURES OF PROTEINS\\n\\nhydrogen atom with the carbonyl group oxygen of the peptide bond four residues ahead of it in the chain.\\n\\nElectrostatic Interactions\\n\\nIf positive and negatively charged amino acid residues are buried in the hydrophobic interior of a protein, where neither can interact with water, then they will attract each other and the force between them will be stronger than it would be in water. Such an electrostatic bond inside a protein is called a salt bridge.\\n\\nPolar groups such as hydroxyl and amide groups are dipoles. They have an excess of electrons at one atom and a corresponding deficiency at another, as we have already seen for the water molecule (page 27). The partial charges of dipoles will be attracted to other dipoles and to fully charged groups.\\n\\nvan der Waals Forces\\n\\nThese are relatively weak close-range interactions between atoms. Imagine two atoms sitting close together. At a given instant more of the electrons on one atom may be on one side, and this exposes the positive charge on the nucleus. This positive charge attracts the electrons of the adjacent atom thus exposing its nuclear charge, which would attract the electrons of another atom and so on. The next instant the electrons will have moved so we have a situation of fluctuating attractions between atoms. These forces are important in the close-packed interiors of proteins and membranes and in the specific binding of a ligand to its binding site.\\n\\nHydrophobic Interactions\\n\\nMolecules that do not interact with water (and that are therefore classed as hydrophobic) force the water around them to become more organized: hydrogen-bonded “cages” form. This organization is thermodynamically undesirable and is minimized by the clustering together of such molecules. This is called the hydrophobic effect. A polypeptide with hydrophilic and hydrophobic residues will spontaneously adopt a configuration in which the hydrophobic residues are not exposed to water. They can achieve this either by sitting in a lipid bilayer (Fig. 9.15) or by adopting a globular shape in which the hydrophobic residues are clustered in the center of the protein.\\n\\nDisulfide Bonds\\n\\nExtracellular proteins often have disulfide bonds between specific cysteine residues. These are strong covalent bonds, and they tend to lock the molecule into its conformation. Although relatively few proteins contain disulfide bonds, those that do are more stable and are therefore easy to purify and study. For this reason many of the first proteins studied in detail, such as the digestive enzymes chymotrypsin and ribonuclease and the bacterial cell wall degrading enzyme lysozyme, have disulfide bonds.\\n---\\n# PROTEIN STRUCTURE\\n\\nThe three-dimensional protein structure often has protrusions, clefts, or grooves on the surface where particular amino acids are positioned to form sites that bind ligands and, in the case of enzymes, catalyze reactions within or between ligands. The whole three-dimensional arrangement of the amino acids in the protein is called the tertiary structure.\\n\\nA tertiary structure is unique to a particular protein. However, common patterns or motifs occur in tertiary structures. For example, many proteins with different functions show a β-barrel structure where β sheet is rolled up to form a tube. Green fluorescent protein (page 150) is one example (Fig. 9.16). Often the tertiary structure is seen to divide into discrete regions. The calcium-binding protein calmodulin shows this clearly (Fig. 9.17). Its single chain is organized into two domains, one shown in green, the other in black, joined by only one strand of the polypeptide chain. In calmodulin the two domains are very similar, and the modern gene probably arose through duplication of an ancestral gene that was half as big. Domains are easier to see than to define. “A separately folded region of a single polypeptide chain” is as good a definition as any.\\n\\nDomains may be similar or different in both structure and function. The catabolite-activating protein (CAP) of Escherichia coli (page 114) binds to a specific sequence of bases on DNA, assisting RNA polymerase to bind to its promoter and initiate transcription of the lac operon. It does this only when it has bound cAMP (which in turn only happens when the intracellular glucose concentration is low). One of the domains of CAP has the job of binding cAMP. Another recognizes DNA sequences using a helix-turn-helix motif (Fig. 9.18). One of these helices fits into the major groove of the DNA where it can make specific interactions with the exposed edges of the bases.\\n\\nProteins that interact with DNA often do so via zinc fingers. The DNA binding domain of the glucocorticoid receptor (page 121) contains two zinc finger motifs. Each finger is formed by the coordination of four cysteine residues with a Zn2+ ion (Fig. 9.19a). The domain contains two α helices (Fig. 9.19b), one in each finger. The first helix, also known as the recognition helix, fits into the major groove of the DNA and makes contacts with specific bases. The second helix is bent at right angles to the first and helps to stabilize the receptor homodimer on DNA by promoting dimerization between the two receptor monomers. This finger can alternatively interact with other proteins that help to activate or repress transcription, such as AP1 subunits (page 124).\\n\\nDomains usually correspond to exons (page 98) in the gene. It is therefore relatively easy for evolution to create new proteins by mixing and matching domains from existing proteins. The cAMP gated channel (page 350) arose in the ancestor of the vertebrates when a mutation occurred that spliced the cAMP-binding domain from cAMP-dependent protein kinase (page 353) onto the end of the voltage-gated sodium channel (page 330).\\n\\nIntegral membrane proteins have special tertiary structures. The polypeptide chain may cross the membrane once or many times, but each time it does so it has a region of hydrophobic side chains that can be in a hydrophobic environment such as the interior of a lipid bilayer. The commonest membrane-spanning structure is a 22-amino-acid α helix. Figure 9.15 shows an integral membrane protein called the platelet-derived growth factor receptor (page 353). Hydrophobic side chains on the transmembrane α helix interact with the hydrophobic interior of the membrane. The protein is held tightly in the membrane because for it to leave would expose these hydrophobic side chains to water.\\n---\\n|Figure 9.16. Green fluorescent protein molecule comprises a ß barrel and a central α helix.|Figure 9.16. Green fluorescent protein molecule comprises a ß barrel and a central α helix.|Figure 9.16. Green fluorescent protein molecule comprises a ß barrel and a central α helix.|\\n|---|\\n| |Ca|++++|\\n|NH2|Ca|CaCa|\\n| | |COOH|\\n| |Ca|++|\\n| |Ca|++|\\n\\nFigure 9.17. Calmodulin is composed of two very similar domains.\\n---\\n# PROTEIN STRUCTURE\\n\\ncyclic AMP binding domain\\n\\nDNA binding domain\\n\\nActive CAP is a dimer of two identical α helix-turn-helix DNA-binding proteins, one shown in green, one in gray.\\n\\nFigure 9.18.\\nActive catabolite activator protein is a dimer.\\n\\nExample 9.2 Recognition by Binding\\n\\nMany of the jobs proteins do depend on recognition and binding. Proteins vary greatly in the specificity with which they bind other substances. Histones are at one extreme. About 25% of their amino acids are lysine or arginine, whose side chains are positively charged. Histones therefore bind to DNA—any DNA—through an electrostatic attraction to the negative charges on the phosphates of the DNA backbone.\\n\\nThe glucocorticoid receptor also binds DNA, but it is at the very other extreme of specificity. After binding glucocorticoid hormone, the hormone–receptor complex enters the nucleus and binds to the base sequence 5\\' AGAACA 3\\'. The protein does this using a zinc finger domain (page 200), which inserts an α helix into the major groove of DNA. The receptor works as a dimer so the sequence 5\\' AGAACA 3\\' must appear twice in the DNA, separated by three nucleotides (page 203). Once bound the receptor binds other proteins that in turn stimulate transcription.\\n---\\n# TERTIARY STRUCTURE: DOMAINS AND MOTIFS\\n\\nFigure 9.19. (a) Two zinc finger motifs in the glucocorticoid receptor. Each circle is one amino acid residue. Amino acid residues that are important in binding Zn2+ or DNA are specified. (b) Cartoon representation of the zinc finger domains of a dimerized pair of glucocorticoid hormone receptors interacting with DNA.\\n\\n| | | | | | | | | | | | |\\n|---|---|---|---|---|---|---|---|---|---|---|---|\\n|H| | |A| | | | | |K|R|R|\\n|DG|C|C|C|T| | | | |I|C|C|\\n|V| |Zn ++|EG| | | |Zn ++|QA| | | |\\n|P|R|I|C| | |CK| |F|F|R|R|\\n|Helix 1| | | | | | | | | | | |\\n| | | | | |Helix 2| | | | | | |\\n\\n(a) R R\\n\\n|5\\'| |3\\'|\\n|---|---|---|\\n|A| |T|\\n|GA| |T|\\n|C|Zn ++| |\\n|G T| |A|\\n|C| | |\\n|T|N| |\\n|NA| | |\\n|N| | |\\n|N| | |\\n|N| | |\\n|G|T T|A|\\n|CZn ++A| | |\\n|ZnZn| | |\\n|Zn ++++++| | |\\n|ZnZn| | |\\n|G|A A|T|\\n|C|T| |\\n|5\\'| |3\\'|\\n\\n(b) to rest of glucocorticoid hormone receptor protein\\n---\\n# IN DEPTH 9.2 Hydropathy Plotting—The PDGF Receptor PROTEIN STRUCTURE\\n\\nOne of the central problems in structural biology is the prediction of the structure of a protein from the primary structure or indeed from the DNA sequence once the gene has been found. Although we have come some way in understanding protein folding, the problem remains unsolved. It is possible, however, to identify proteins that will have similar structures. One of the simpler things that can be looked for is regions of hydrophobic amino acids: if these are 21–22 amino acids long, they are likely to be membrane spanning α helices. The figure shows a hydropathy plot for the platelet-derived growth factor (PDGF) receptor. Each amino acid in the protein has a hydrophobicity allotted to it, so that ionized groups like those on aspartate and arginine get a big negative score while groups like those on phenylalanine and leucine get a big positive score. The plot is a running average of the hydrophobicity along the polypeptide chain. The protein begins with a somewhat hydrophobic region: this is the signal sequence (page 215) that directs the growing protein to the endoplasmic reticulum. The rest of the protein is neutral or somewhat hydrophilic except for a prominent short hydrophobic region in the center. We can therefore predict that this protein will cross the membrane once at this location. Figure 9.15 shows the full predicted structure of this protein.\\n\\n4.0\\n3.0\\n2.0\\n1.0\\n0.0\\n-1.0\\nHydrophobicity\\n-2.0\\n-3.0\\n-4.0 10 Amino acid number 1096\\n\\nQuaternary Structure: Assemblies of Protein Subunits\\n\\nMany proteins associate to form multiple molecular structures that are held together tightly but not covalently by the same interactions that stabilize tertiary structures. For example, gap junction channels (page 55) are formed from six identical monomers. CAP is only active when it dimerizes (Fig. 9.18). Hemoglobin, the protein that carries oxygen in our red blood cells, is formed from four individual polypeptide chains, two α chains and two\\n---\\n# PROSTHETIC GROUPS\\n\\nFigure 9.20. Hemoglobin, a tetrameric protein with heme prosthetic groups. (Illustration: Irving Geis. Rights owned by Howard Hughes Medical Institute. Reproduction by permission only.)\\n\\nβ chains (Fig. 9.20). In all these cases, we call the three-dimensional arrangement of the protein subunits as they fit together to form the multimolecular structure the quaternary structure of the protein.\\n\\n# PROSTHETIC GROUPS\\n\\nEven with the enormous variety of structures available, there are some functions that proteins need help with because the 20 protein amino acid side chains do not cover the properties required. Moving electrons in oxidation and reduction reactions and binding oxygen are good examples. Proteins therefore associate with other chemical species that have the required chemical properties. Hemoglobin uses iron-containing heme groups to carry oxygen molecules (Fig. 9.21). The general name for a nonprotein species that is tightly bound to a protein and helps it perform its function is prosthetic group.\\n\\nSome proteins have tightly bound metal ions that are essential to their function. We have already met the zinc fingers of the glucocorticoid receptor and the iron in hemoglobin. Other proteins use magnesium, manganese, or copper.\\n---\\n# PROTEIN STRUCTURE\\n\\nFigure 9.21. Iron-containing prosthetic group heme in the form in which it is found in oxygenated hemoglobin. The oxygen molecule is at the top. (From D. Voet and J. D. Voet, Biochemistry, 2nd ed., New York: Wiley, 1995, p. 216. c©1995 John Wiley & Sons, Inc. This material is used by permission of John Wiley & Sons, Inc.)\\n\\nTHE PRIMARY STRUCTURE CONTAINS ALL THE INFORMATION NECESSARY TO SPECIFY HIGHER-LEVEL STRUCTURES\\n\\nProtein structures are stable and functional over a small range of environmental conditions. Outside this range the pattern of interactions that stabilizes the tertiary structure is disrupted and the molecule denatures—activity disappears as the molecule loses its structure. Denaturation may be caused by many factors, which include excessive temperature, change of pH, and detergents. Concentrated solutions of urea (8 mol per liter) have long been used by biochemists to denature proteins. Unlike heat and pH, urea does not cause the protein to precipitate. Physicochemical techniques have shown that in urea solution all of the higher levels of structure are lost and that polypeptide chains adopt random, changing conformations. Reagents such as urea that do this are called chaotropic reagents. If the urea is removed (by dialysis or simply by dilution), the protein refolds, regaining its structure and biological activity. This shows that the sequence of amino acids contains all of the information necessary to specify the final structure. The refolding of a urea-denatured protein cannot be random. Even a small protein with 100 amino acids would take some 10^50 years to try out all of the structural conformations available. The fact that refolding does happen and happens on a time scale of seconds tells us that there\\n---\\n# THE PRIMARY STRUCTURE\\n\\nmust be a folding pathway, and the process is not random. Secondary structures may form first and act as folding units. In the cell folding is assisted by proteins called chaperones (page 221).\\n\\n☛ Medical Protein Folding Gone Awry—Mad Cow Disease\\n\\n✟ Relevance\\n\\nSome years ago one of the dogmas of biology was overthrown—it was long believed that diseases could only be transmitted by structures that contained nucleic acids, that is, by viruses or by microorganisms. It is now clear that a group of brain diseases can be transmitted by a protein. These are the diseases called spongiform encephalopathies: bovine spongiform encephalopathy in cows (mad cow disease), scrapie in sheep, and Creutzfeldt–Jacob disease and kuru in humans. These diseases are rare in humans but have recently increased in farm animals.\\n\\nThe infectious agent is called a prion and is a protein. It is coded for by a gene the animals have as part of their genome. In healthy individuals the gene is expressed in the cells of the nervous system and generates an innocuous protein called PrPc (prion-related protein cellular). PrPc has a globular region at the C-terminal end but the N-terminal region seems to be unstructured.\\n\\nSometimes the same polypeptide chain folds up differently with the disordered N-terminal section instead folding into a structure rich in β sheet. This is called PrPsc (prion-related protein scrapie). The disease arises because PrPsc can cause normal PrPc protein molecules to fold up into the abnormal form. This keeps happening and lumps of PrPsc form that damage nerve cells. The evidence is that it spread in cattle herds through food that contained recycled meat from infected animals. It is not yet known how one PrPsc molecule triggers the aberrant folding of a normal PrPc molecule.\\n\\nBefore the problem was fully comprehended, infected animals had been used for human food. There have been a number of cases in people that are thought to have come from eating contaminated beef—these are new variant Creutzfeldt–Jacob disease. It remains to be seen how many people will develop this fatal disease.\\n\\n✠\\n---\\nIN DEPTH 9.3 Curing Mad Mice with Smelly Fish PROTEIN STRUCTURE\\n\\nIf a cell is immersed in a solution more dilute than its own cytosol, it tends to swell, as we have seen in the case of plant cells (page 53). On the other hand, cells immersed in concentrated solutions shrink. The overall strength of a solution is its osmolarity, and the movement of water into or out of cells because of solution strength is called osmosis. Many fish are exposed to changes in the salt concentration of the water around them, either because they swim from salt water to fresh and back (salmon, eels) or because they live in estuaries. The cells of these fish adjust the osmolarity of their cytosol to match the surrounding water by increasing the concentration of various small, easily synthesized molecules, one of which is urea. However, this introduces a further problem because high concentrations of urea are chaotropic—they cause cellular proteins to unfold.\\n\\nThe small molecule trimethylamine N-oxide, in contrast, helps proteins to fold into their correct shape. One of the most dramatic examples of this is that trimethylamine N-oxide, added to mouse nerve cells in culture along with the scrapie agent PrP sc, protects the endogenous PrP c prion protein from being incorrectly refolded to generate PrP sc (page 207). Fish producing urea, therefore, also upregulate a gene encoding a flavin-containing monooxygenase (page 82) and produce trimethylamine N-oxide. A battle of opposing forces then ensues, with urea tending to denature the protein and trimethylamine N-oxide counteracting this to maintain the protein in its correctly folded state. When fish die and rot, bacterial action converts trimethylamine N-oxide to trimethylamine, which gives rotting fish its distinctive smell (page 82).\\n---\\n# SUMMARY\\n\\n1. Polypeptides are linear polymers of α-amino acids linked by peptide bonds. There are 20 amino acids coded for by the genetic code. They differ in the properties of their side chains, which range from hydrophobic groups to charged and uncharged hydrophilic groups.\\n\\n2. In addition to their hydrophilicity, side chains of individual amino acids have specific properties of which the most important are charge, the ability to form disulfide bonds, and the ability to undergo posttranslational modification.\\n\\n3. Phosphorylation (on S, T, Y, D, E, or H) is a posttranslational modification that allows the balance of electrical charges on a protein to be dramatically and reversibly altered.\\n\\n4. Proteins are polypeptides that have a complex three-dimensional shape.\\n\\n5. It is convenient to consider protein structure as having three levels. The primary structure is the linear sequence of the amino acid monomers.\\n\\n6. In some parts of a protein regular, repeated foldings of the polypeptide chain can be seen: these are secondary structures. Secondary structures are held together by hydrogen bonds between the carboxyl oxygens and the hydrogens on the nitrogens of the peptide bonds.\\n\\n7. There are two common types of secondary structure. In the α helix the chain coils upon itself making a spiral with hydrogen bonds running parallel to the length of the spiral. In β sheets the hydrogen bonds are between extended strands of polypeptide that run alongside one another. The hydrogen bonds are at right angles to the strands and in the plane of the sheet. There are two types—parallel and antiparallel.\\n\\n8. The final, complex folding of a protein is its tertiary structure. Interactions between side chains stabilize the tertiary structure.\\n\\n9. Some proteins have a quaternary structure. This is an association of subunits, each of which has a tertiary structure.\\n\\n# FURTHER READING\\n\\n|Branden, C., and Tooze. J. 1999.|Introduction to Protein Structure, 2nd ed.|New York: Garland.|\\n|---|---|---|\\n|Creighton, T. 1993.|Proteins, Structures and Molecular Properties, 2nd ed.|New York: W. H. Freeman.|\\n|Fersht, A. 1999.|Structure and Mechanism in Protein Science.|New York: W. H. Freeman.|\\n|McGee, H. 1987.|On Food and Cooking,|London: Unwin.|\\n|Perutz, M. 1992.|Protein Structure: New Approaches to Disease and Therapy.|New York: W. H. Freeman.|\\n|Tanford, C., and Reynolds, J. 2001.|Nature’s Robots—A History of Proteins.|Oxford: Oxford University Press.|\\n|Voet D., and Voet J. D. 2003.|Biochemistry, 3rd ed.|New York: Wiley.|\\n---\\n# PROTEIN STRUCTURE\\n\\nREVIEW QUESTIONS\\nFor each question, choose pe ONE BEST answer or completion.\\n1. Peptide bonds result from\\nA. pe reaction of amino groups wip water.\\nB. pe reaction of two carboxyl groups and removal of water.\\nC. pe reaction of an amino group wip a carboxyl group and loss of water.\\nD. reactions between amino acid side chains.\\nE. transfer of a phosphate group from ATP to pe amino acid.\\n2. The amino acids found in proteins are called\\nA. α amino acids.\\nB. ß amino acids.\\nC. γ amino acids.\\nD. δ amino acids.\\nE. π amino acids.\\n3. The side chains of pe amino acids serine, asparagine, and glutamine are all\\nA. normally negatively charged at pH 7.\\nB. normally positively charged at pH 7.\\nC. able to interact wip water.\\nD. hydrophobic.\\nE. indifferent to peir environment.\\n4. The amino acids making up an α helix\\nA. have side chains pat point outward—away from pe center of pe helix.\\nB. have side chains pat point inward—making up a hydrophobic core for pe helix.\\nC. have side chains pat do not interact wip amino acids elsewhere in pe protein.\\nD. have side chains pat are always hydrophobic.\\nE. include proline.\\n5. Disulfide bonds\\nA. form between two mepionine residues.\\nB. form between two cystine residues.\\nC. form between two cysteine residues.\\nD. are found in cytoplasmic proteins.\\nE. attach microtubules to organelles.\\n6. Integral membrane proteins are helped to locate across pe lipid bilayer by\\nA. glycosylation.\\nB. formation of disulfide bonds.\\nC. phosphorylation.\\nD. using an α helix made up of amino acids wip hydrophilic side chains.\\nE. using an α helix made up of amino acids wip hydrophobic side chains.\\n7. Proteins cannot usually be denatured by\\nA. concentrated urea solutions.\\nB. oxygen.\\nC. strong acid.\\nD. concentrated epanol solutions.\\nE. heat.\\n---\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n|1.|C. The amino group of one amino acid reacts with the carboxyl group of another with loss of water.|\\n|---|---|\\n|2.|A. They are α-amino acids meaning that the amino group is on the same carbon as the carboxyl group.|\\n|3.|C. Serine, asparagine, and glutamine all have polar side chains that have no overall charge (so A and B are wrong). These are able to hydrogen bond with water and so are hydrophilic.|\\n|4.|A. The side chains point outward away from the center of the helix. Note that because proline introduces a kink into the polypeptide chain, it cannot participate in an α helix, so E is wrong.|\\n|5.|C. Disulfide bonds are formed between two cysteine residues. Concerning answer B: cystine is the name given to the double amino acid found when proteins containing disulfide bonds are hydrolyzed. If we spoke of a cystine residue in a protein we would therefore mean two cysteines that were already joined by a disulfide bond. Two such residues would not be able to form a further disulfide bond.|\\n|6.|E. Most proteins cross membranes by forming an α helix made up of hydrophobic amino acids. The side chains of amino acid residues in an α helix point out, so the residues need to be hydrophobic to interact with the hydrophobic interior of the bilayer. Such a helix is usually 22 amino acids long, which matches the thickness of a membrane bilayer. Formation of disulfide bonds is unlikely to help proteins adopt a transmembrane configuration, while glycosylation and phosphorylation both make proteins more hydrophilic, making a transmembrane configuration less likely.|\\n|7.|B. Concentrated urea solutions, strong acid, concentrated ethanol solutions, and heat can all denature proteins. In contrast, although oxygen may sometimes oxidize amino acids, such an oxidation does not normally result in loss of structure.|\\n---\\n# INTRACELLULAR PROTEIN TRAFFICKING\\n\\nAn essential feature of all eukaryotic cells is their highly compartmentalized organization, which allows the spatial and temporal separation of the synthesis, modification, degradation, secretion, or take up of molecules. Each of the various organelles within cells is specialized for one or more tasks and therefore needs a specialized set of proteins to carry out its function. In this chapter we will examine how a protein, once synthesized on a ribosome, is precisely and actively moved (translocated) to the correct cellular compartment.\\n\\n# THREE MODES OF INTRACELLULAR PROTEIN TRANSPORT\\n\\nNewly synthesized proteins must be delivered to their appropriate site of function within the cell. The mechanisms and machinery that eukaryotic cells use to accomplish this are highly conserved from yeast to humans. There are three possible ways by which the cell accomplishes the task (Fig. 10.1). First, the protein may fold into its final form as it is synthesized and then move through an aqueous medium to its final destination, remaining folded all the way. Delivery of proteins to the nucleus follows this scheme; the proteins are synthesized on cytosolic ribosomes and pass through nuclear pores into the nucleoplasm by a process called gated transport. In the second form of transport, called transmembrane translocation, unfolded polypeptide chains are threaded across one or more membranes to reach their final destination. Proteins destined for the interior of peroxisomes, mitochondria, and chloroplasts are synthesized on cytosolic ribosomes and may fold up completely or\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C 2004 by John Wiley & Sons, Inc.\\n---\\n# INTRACELLULAR PROTEIN TRAFFICKING\\n\\n|mitochondria|transmembrane|translocation|\\n|---|---|---|\\n|(and chloroplasts in plants)| | |\\n|peroxisomes| | |\\n|free ribosomes| | |\\n|gated transport|Nucleus| |\\n| |rough endoplasmic reticulum| |\\n| |cis face| |\\n|vesicular trafficking|Golgi apparatus|trans face|\\n| | |lysosome|\\n|regulated secretion| | |\\n| |constitutive secretion| |\\n|endosome|exocytosis|endocytosis|\\n\\nFigure 10.1. The three modes of protein transport.\\n---\\n# TRANSPORT TO AND FROM THE NUCLEUS\\n\\npartially into their final form but are then unfolded in order to be transported across the appropriate membrane. Proteins that are synthesized on the rough endoplasmic reticulum are threaded across the membrane and into the interior of that organelle while they are being synthesized. In the third form of transport, small closed bags made of membrane, called vesicles, carry newly synthesized protein from the endoplasmic reticulum to the Golgi apparatus and between other compartments in a process called vesicular trafficking. The fate of a protein—whether to remain in the cytosol or be sent along one of these alternative transport paths—is determined by sections of the protein itself that act as sorting signals. When the protein is first made on the ribosome, it is simply a stretch of polypeptide. The initial sorting decisions must therefore be made on the basis of particular amino acid sequences called targeting sequences. For proteins synthesized on the rough endoplasmic reticulum, additional sorting signals such as sugars and phosphate groups can be added by enzymes that modify the chemical structure of the protein. In general, sorting operates by proteins containing a specific sorting signal binding to a receptor protein, which in turn binds to translocation machinery situated in the membrane of the appropriate compartment.\\n\\n# Targeting Sequences\\n\\nTargeting sequences (also known as localization sequences) usually comprise a length of 3–80 amino acids that are recognized in the cytosol by specific receptors that then guide the protein to the correct site and make contact with the appropriate translocation machinery. Once the protein has been imported into the new location the targeting sequence is often removed by enzymes that break the peptide bond between the targeting sequence and the rest of the protein. Some targeting sequences have been characterized better than others. The targeting sequence encoding import into the endoplasmic reticulum consists of about 5–15 mostly hydrophobic amino acids at the N terminus of the protein called the signal sequence. The import signal for mitochondria is a stretch of 20–80 amino acids in which positively charged side chains stick out on one side of the helix and hydrophobic side chains stick out on the other, a so-called amphipathic helix. A cluster of about five positively charged amino acids located within the protein sequence targets a protein to the nucleus, while the best known peroxisomal targeting sequence is the C-terminal tripeptide Ser-Lys-Leu-COOH.\\n\\n# Retention\\n\\nAnother class of sorting signal does not activate the transport of a protein out of its present location but is rather a signal to the cell that the protein has reached its final destination and should not be moved. For example, proteins with the motif Lys-Asp-Glu-Leu-COOH (KDEL) at their C terminus are retained within the endoplasmic reticulum.\\n\\n# TRANSPORT TO AND FROM THE NUCLEUS\\n\\nIn contrast to the situation in prokaryotes, RNA transcription and protein synthesis are separated in space and time in eukaryotes. Exchange of material between the nucleus and the cytoplasm is essential for the basic functioning of these cells and must be tightly controlled. RNA and ribosomal subunits that are assembled in the nucleus have to enter the cytoplasm where they are required for protein synthesis. On the other hand, proteins such\\n---\\n# INTRACELLULAR PROTEIN TRAFFICKING\\n\\nAs histones and transcription factors must enter the nucleus to carry out their functions. The nuclear pore complex mediates trafficking between the nucleus and the cytoplasm. The pore is made from a large number of proteins, and we refer to this type of structure as a multiprotein complex. Transport through the nuclear pore complex is mediated by signals and requires both energy and transporter proteins.\\n\\n|Example 10.1|Holding Calcium Ions in the Endoplasmic Reticulum|\\n|---|---|\\n|One of the functions of the smooth endoplasmic reticulum is to hold calcium ions ready for release into the cytosol when the cell is stimulated. A protein called calreticulin (short for calcium-binding protein of the endoplasmic reticulum) helps hold the calcium ions. Its primary structure is:|(NH2)MLLSVPLLLGLLGLAVAEPAVYFKEQFLDGDGWTSRWIESKHKSDFGKFVLSSGKFYGDEEKDKGLQTSQDARFYALSASFEPFSNKGQTLVVQFTVKHEQNIDCGGGYVKLFPNSLDQTDMHGDSEYNIMFGPDICGPGTKKVHVIFNYKGKNVLINKDIRCKDDEFTHLYTLIVRPDNTYEVKIDNSQVESGSLEDDWDFLPPKKIKDPDASKPEDWDERAKIDDPTDSKPEDWDKPEHIPDPDAKKPEDWDEEMDGEWEPPVIQNPEYKGEWKPRQIDNPDYKGTWIHPEIDNPEYSPDPSIYAYDNFGVLGLDLWQVKSGTIFDNFLITNDEAYAEEFGNETWGVTKAAEKQMKDKQDEEQRLKEEEEDKKRKEEEEAEDKEDDEDKDEDEEDEEDKEEDEEEDVPGQAKDEL(COOH)|\\n|The first 17 amino acids include 14 hydrophobic ones, shown in black: this is the signal sequence that triggers translocation of the protein into the endoplasmic reticulum. The last 4 amino acids, KDEL, ensure that it remains there. In between these two sorting signals is the functional core of the protein.| |\\n\\n# The Nuclear Pore Complex\\n\\nThe nuclear pore complex is embedded in the double membrane of the nuclear envelope (Fig. 10.2). The basic structure of the nuclear pore complex, which has been mainly elucidated by electron microscopy, is very similar in all eukaryotic cells. It consists of over 50 different components, called nucleoporins, which form eight identical subunits arranged in a circle. Seen from the side the nuclear pore complex comprises three rings. The inner ring sends out radial spokes into the center of the pore and includes transmembrane proteins that anchor the complex to the nuclear envelope. A central transporter, also called the plug, is situated in the heart of each nuclear pore complex, forming the gate for macromolecular traffic.\\n\\n# Gated Transport Through the Nuclear Pore\\n\\nIn general, a protein has to display a distinct signal for it to be transported through the nuclear pore. Proteins with a nuclear localization signal are transported in, while proteins with a nuclear export signal are transported out. Mobile transporter proteins, usually mediating either export or import, recognize the appropriate targeting sequence and then interact with structural elements of the nuclear pore. This causes the pore to expand, allowing the active transport of the transporter–protein complex. The precise mechanism by which large molecules move through the nuclear pore is not yet fully understood. However, the discovery of the role played by the GTPase Ran in nuclear transport has given valuable insight into\\n---\\n# TRANSPORT TO AND FROM THE NUCLEUS\\n\\n|outer ring|Plug|\\n|---|---|\\n|Spokes| |\\n\\n|Spokes|Outer ring|\\n|---|---|\\n|Plug|inner ring|\\n\\nribosome\\nnucleolus\\n\\nFigure 10.2. The nuclear pore.\\n---\\n# INTRACELLULAR PROTEIN TRAFFICKING\\n\\n**Figure 10.3.** The GDP/GTP cycle of a GTPase. Pi represents an inorganic phosphate ion.\\n\\nhow directionality of transport into and out of the nucleus is achieved. GTP hydrolysis by\\nRan provides the energy for transport. We will now describe this process.\\n\\n# GTPases and the GDP/GTP Cycle\\n\\nGTPases form a family of proteins that are often involved when cells need to control complex\\nprocesses. They all share the ability to hydrolyze the nucleotide GTP but otherwise differ\\nmarkedly in the processes they control and their mode of operation. We have already met\\none GTPase: the protein EF-tu, a component of the protein synthesis mechanism (page 172).\\nOnce a GTPase has hydrolyzed GTP to GDP it adopts an inactive shape and is unable to\\nactivate its target process (Fig. 10.3). In contrast, if the protein expels the GDP and binds a\\nmolecule of GTP, it then adopts its active form. The cycle between the GDP-bound and GTP-\\nbound state is regulated by effector proteins. GTPase activating proteins or GAPs speed up\\nthe rate at which GTPases hydrolyze GTP, and hence the rate at which they inactivate, while\\nguanine nucleotide exchange factors, or GEFs, assist in the exchange of GDP for GTP and\\ntherefore help GTPases adopt their active configuration.\\n\\n# GTPases in Nuclear Transport\\n\\nIn the case of nuclear pore transport, the GEFs that operate on Ran are found in the nucleus,\\nwhile Ran GAPs are cytosolic (Fig. 10.4). Thus nucleoplasmic Ran is predominantly in the\\nGTP bound state (Ran:GTP), while most cytosolic Ran has GDP bound (Ran:GDP).\\n**Figure 10.5** shows how Ran regulates import of proteins into the nucleus. An import\\ntransporter binds the nuclear localization sequence on the protein. As long as the transporter\\nremains on the cytosolic side of the nuclear envelope, its cargo will remain bound. However,\\n---\\n# TRANSPORT TO AND FROM THE NUCLEUS\\n\\n| |Ran-GEF|GTP|\\n|---|---|---|\\n|GDP| | |\\n|Ran| | |\\n|Pi|Ran-GAP| |\\n| |cytosol| |\\n\\nFigure 10.4. Ran GEF and GAP are localized to the nucleoplasm and cytosol, respectively. Pi represents an inorganic phosphate ion.\\n\\nOnce the transporter finds itself on the nucleoplasmic side, Ran in its active, GTP-bound state binds and causes the cargo to be released. Now, as long as the transporter remains on the nucleoplasmic side, it will have Ran:GTP bound and will be unable to bind cargo. However, once the transporter finds itself on the cytoplasmic side Ran-GAPs attached to the cytosolic face of the nuclear pore complex will cause Ran to hydrolyze its bound GTP to GDP and Ran will dissociate from the transporter, which is then able to bind more cargo. The same principle is used when a protein is to be exported from the nucleus (Fig. 10.6). In this case the export receptor can only bind proteins with an export sequence if they also bind Ran:GTP. As long as the transporter remains on the nucleoplasmic side of the nuclear envelope, its cargo will remain bound. However, once the transporter finds itself on the cytoplasmic side Ran GAPs will cause Ran to hydrolyze its bound GTP to GDP and both Ran and the cargo will dissociate from the transporter, which is then unable to bind more cargo until it moves back into the nucleoplasm where Ran:GTP is available.\\n\\nAlthough Ran can drive both nuclear import and nuclear export, in each case it is the active GTP-bound form of the protein that binds to the transporter. The GDP-bound form is inactive and cannot bind the transporter.\\n\\nSome proteins move back and forth between the cytosol and the nucleus by successively revealing and masking nuclear localization sequences. For example, the glucocorticoid receptor (page 121) only reveals its nuclear localization sequence when it has bound glucocorticoid.\\n---\\n# INTRACELLULAR PROTEIN TRAFFICKING\\n\\n|nuclear localization|nucleoplasm|\\n|---|---|\\n|sequence| |\\n\\n|cargo|nuclear protein|import|transporter|\\n|---|---|---|---|\\n| | |nuclear|GTP|\\n| | |pore|Ran|\\n|Ran-|RanGDP|GAP|GTP|\\n| | |Pi|Ran|\\n|cytosol|imported cargo| | |\\n\\nFigure 10.5. Nuclear import. Pi represents an inorganic phosphate ion.\\n\\n|cytosol|GTP|\\n|---|---|\\n|GDP|GTP|nuclear|Ran-GAP|\\n|nuclear|pore|\\n|exported cargo|Pi|GTP|\\n|GTP| |\\n\\nFigure 10.6. Nuclear export. Pi represents an inorganic phosphate ion.\\n---\\n# TRANSPORT ACROSS MEMBRANES\\n\\nTransport to Mitochondria\\n\\nMitochondria have their own DNA and manufacture a small number of their own proteins. However, the majority of mitochondrial proteins are coded for by nuclear genes. These are synthesized on free ribosomes and only imported into the mitochondrion posttranslationally. For example, proteins destined for the mitochondrial matrix carry a targeting sequence at their N terminus and are recognized by a receptor protein in the outer membrane. The mitochondrial receptor makes contact with a translocation complex, which unfolds the protein and moves it across both outer and inner membrane simultaneously. Once the protein is translocated, the targeting sequence is cleaved off and the protein refolded.\\n\\nChaperones and Protein Folding\\n\\nA correctly addressed protein may fail to be targeted to an organelle if it folds too soon into its final three-dimensional shape. For example, movement of proteins into the mitochondrial matrix requires that a protein must move through a channel through the outer and inner membranes. This channel is just wide enough to allow an unfolded polypeptide to pass through. Our cells have proteins called chaperones, which as the name indicates, “look after” proteins. Chaperones use energy derived from the hydrolysis of ATP to keep newly synthesized proteins destined for the mitochondrial matrix in an unfolded state. As soon as the protein moves through the channels and into the matrix, the matrix targeting sequence is cleaved. The protein now folds into its correct shape. Some small proteins can fold without help. Larger proteins are helped to fold in the mitochondrial matrix by a chaperone protein called chaperonin, which provides a surface on which another protein can fold. Chaperonin itself does not change shape when helping another protein to fold.\\n\\nCertain stresses that cells can experience, such as excessive heat, can cause proteins to denature (page 206). The cell responds by making proteins called heat-shock proteins in large amounts. The heat-shock proteins bind to misfolded proteins, usually to a hydrophobic region exposed by denaturation, and help the protein to refold. Like chaperone proteins, the heat-shock proteins are not themselves changed, but instead form a platform on which the denatured protein can refold itself. Heat-shock proteins are found in all cell compartments and also in bacteria.\\n\\nTransport to Peroxisomes\\n\\nMost organelles that are bound by a single membrane have their proteins made at the rough endoplasmic reticulum and transported to them in vesicles (Fig. 10.1). Peroxisomes are an exception: their proteins are synthesized on free-floating polyribosomes and then transported to their final destination. Peroxisomal targeting sequences on the protein bind to peroxisome import receptors in the cytosol. The complex of cargo and receptor docks onto the peroxisomal membrane and then crosses the membrane to enter the peroxisome. Here, the protein cargo is released and the import receptor is shuttled back into the cytosol.\\n---\\n# INTRACELLULAR PROTEIN TRAFFICKING\\n\\nMedical Relevance\\n\\nBlocking Calcineurin—How Immunosuppressants Work\\n\\nThe drug cyclosporin A is invaluable in modern medicine because it suppresses the immune response that would otherwise cause the rejection of transplanted organs. It does this by blocking a critical stage in the activation of T lymphocytes, one of the cell types in the immune system. T lymphocytes signal to other cells of the immune system by synthesizing and releasing the protein interleukin 2. Transcription of the interleukin 2 gene is activated by a transcription factor called NFAT.\\n\\n|foreign protein|Receptor|\\n|---|---|\\n| | |plasmalemma|cytoplasm|acts via Gq and IP (see Chapter 16)|\\n|active phosphatase| |\\n|inactive calcineurin calmodulin| |\\n|nuclear localization sequence|dephosphorylated NFAT|phosphorylated NFAT|\\n|nucleus| | |\\n|RNA polymerase|NFAT binding site in enhancer region of interleukin 2 gene|mRNA coding for interleukin 2|\\n---\\n# TRANSPORT ACROSS MEMBRANES\\n\\nNFAT has a sorting signal that would normally direct it to the nucleus, but in unstimulated cells this is masked by a phosphate group, so NFAT remains in the cytoplasm and interleukin 2 is not made. However, when foreign proteins, for example, any beginning with f-met (page 171), activate the T lymphocyte, they cause the concentration of calcium ions in the cytosol to increase (an increase of calcium concentration is a common feature of cell stimulation to be described in Chapter 16). Calcium activates a phosphatase called calcineurin, which removes the phosphate group from many substrates including NFAT. NFAT then moves to the nucleus and activates interleukin 2 transcription. The released interleukin 2 activates other immune system cells that attack the foreign body. Cyclosporin blocks this process by inhibiting calcineurin, so that even though calcium rises in the cytoplasm of the T lymphocyte, NFAT remains phosphorylated and does not move to the nucleus.\\n\\n# Medical Relevance\\n\\nHow Protein Mistargeting Can Give You Kidney Stones\\n\\nPrimary hyperoxaluria type 1 is a rare genetic disease in which calcium oxalate \"stones\" accumulate in the kidney. Healthy people convert dietary glyoxylate (from plants, see page 297) to the useful amino acid glycine by the enzyme alanine glyoxylate aminotransferase (AGT) (page 250). AGT is located in peroxisomes in liver cells. If glyoxylate cannot be converted to glycine, it is instead oxidized to oxalate and excreted by the kidney, where it tends to precipitate as hard lumps of calcium oxalate. Two thirds of patients with primary hyperoxaluria type 1 have a mutant form of AGT that simply fails to work. However, the other third have an AGT with a single amino acid change (G170R) that works reasonably well, at least in the test tube. However, this amino acid change is enough to make the mitochondrial import system believe that AGT is a mitochondrial protein and import it inappropriately, so that no AGT is available to be transported to the peroxisomes. For the clinician, the mistargeting of AGT in primary hyperoxaluria type 1 poses an unusual problem, namely, how to explain to a patient that the way to cure their kidney stones is to have a liver transplant!\\n\\n# Synthesis on the Rough Endoplasmic Reticulum\\n\\nSynthesis of proteins destined for import into the endoplasmic reticulum starts on free polyribosomes. When the growing polypeptide chain is about 20 amino acids long, the endoplasmic reticulum signal sequence is recognized by a signal recognition particle that is made up of a small RNA molecule and several proteins (Fig. 10.7). The signal recognition particle brings the ribosome to the endoplasmic reticulum membrane where it interacts with a specific receptor—the signal recognition particle receptor (or the docking protein). This interaction directs the polypeptide chain to a protein translocator. Once this has occurred the signal recognition particle and its receptor are no longer required and are released. Protein synthesis now continues; and, as the polypeptide continues to grow, it threads its way through the membrane via the protein translocator, which acts as a channel allowing hydrophilic stretches of polypeptide chain to cross. Once the polypeptide chain has entered the lumen of the endoplasmic reticulum, the signal sequences may be cleaved off by an enzyme called signal peptidase. Some proteins do not undergo this step but instead retain their signal sequences.\\n\\nThe platelet-derived growth factor receptor is an example of an integral membrane protein. It contains a stretch of 22 hydrophobic amino acids that spans the plasma membrane (Fig. 9.15 on page 198). The first part of the polypeptide to be synthesized is an endoplasmic reticulum signal sequence, so the polypeptide begins to be threaded into the lumen of the endoplasmic reticulum. This section will become the extracellular domain of the receptor.\\n---\\n# INTRACELLULAR PROTEIN TRAFFICKING\\n\\n|1|ribosome|2|\\n|---|---|---|\\n|mRNA|mRNA|mRNA|\\n|ER signal sequence|ER signal sequence|ER signal sequence|\\n|3|signal recognition particle|signal recognition particle receptor|\\n|translocator lumen cytosol|translocator lumen cytosol|translocator lumen cytosol|\\n|endoplasmic reticulum|endoplasmic reticulum|endoplasmic reticulum|\\n| | |4|signal recognition particle and its receptor dissociate|\\n|6|growing polypeptide chain| |\\n|5|polypeptidefree ER signal sequence| |\\n|oligosaccharide|oligosaccharide|oligosaccharide|\\n\\nFigure 10.7. Transport of a growing protein across the membrane of the endoplasmic reticulum.\\n---\\n# TRANSPORT ACROSS MEMBRANES\\n\\nWhen the stretch of hydrophobic residues is synthesized, it is threaded into the translocator in the normal way but cannot leave at the other end because the amino acid residues do not associate with water. As synthesis continues, therefore, the newest length of polypeptide bulges into the cytosol. Once synthesis stops, this section is left as the cytosolic domain.\\n\\nIf a protein contains more than one hydrophobic stretch, then synthesis of the second stretch reinitiates translocation across the membrane, so that the protein ends up crossing the membrane more than once.\\n\\nWe can identify key amino acid sequences that play a role in protein targeting by making use of protein engineering. If we join the stretch of nucleotides that codes for the endoplasmic reticulum signal sequence to the cDNA that codes for a cytosolic protein, we produce a chimeric DNA molecule (page 150). When this cDNA is transfected (page 150) into cells, it will be transcribed into mRNA and then translated into a protein that is unchanged except that it now has, at its N terminus, the endoplasmic reticulum import sequence. The protein is targeted to the endoplasmic reticulum. However, because the protein has no other sorting signal, it is not recognized by any other receptor protein. It gets secreted from the cell by the constitutive route. This result shows that constitutive exocytosis is the default route for proteins synthesized on the rough endoplasmic reticulum.\\n\\n# Glycosylation: The Endoplasmic Reticulum and Golgi System\\n\\nMost polypeptides synthesized on the rough endoplasmic reticulum are glycosylated, that is, they have sugar residues added to them, as soon as the growing polypeptide chain enters the lumen of the endoplasmic reticulum. In a process called N-glycosylation a premade oligosaccharide composed of two N-acetyl glucosamines, then nine mannoses, and then three glucoses is added to an asparagine residue by the enzyme oligosaccharide transferase. The three glucose residues are subsequently removed, marking the protein as ready for export from the endoplasmic reticulum to the Golgi apparatus.\\n\\nThe stacks of the Golgi apparatus have distinct polarity that reflects the passage of proteins through the organelle. The asymmetry of the stack is reflected in the\\n\\n|cis face|\\n|---|\\n|cis-Golgi network|transfer vesicle|\\n|cis cisterna|\\n|trans cisterna|\\n|trans-Golgi network|\\n|secretory vesicle|trans face|\\n\\nFigure 10.8. The Golgi apparatus.\\n---\\n# INTRACELLULAR PROTEIN TRAFFICKING\\n\\nThe morphology of the membranes from which it is formed. The cis cisternae are made of membranes 5.5 nm thick, like those of the rough endoplasmic reticulum, while the trans cisternae are 10 nm thick, like the plasma membrane. Each cisterna is characterized by a central flattened region where the luminal space, as well as the gap between adjacent cisternae, is uniform. The margin of each cisterna is often dilated and is often fenestrated (i.e., has holes through it) as well.\\n\\nSmall, spherical vesicles are always found in association with the Golgi apparatus, especially with the edges of the cis face. These are referred to as transfer vesicles; some of them carry proteins from the rough endoplasmic reticulum to the Golgi stacks, others transfer proteins between the stacks, that is, from cis to middle and from middle to trans. As proteins move through the Golgi apparatus, the oligosaccharides already attached to them are modified, and additional oligosaccharides can be added. As well as having important functions once the protein has reached its final destination, glycosylations play an important role in sorting decisions at the trans-Golgi network.\\n\\n# VESICULAR TRAFFICKING BETWEEN INTRACELLULAR COMPARTMENTS\\n\\nMost of the single-membrane organelles of the eukaryotic cell pass material between themselves by vesicular traffic, in vesicles that bud off from one compartment to fuse with another. In this way the cargo proteins are never in contact with the cytosol. Two main directions of traffic can be identified. The exocytotic pathway runs from the endoplasmic reticulum through the Golgi apparatus to the plasma membrane. The endocytotic pathway runs from the plasma membrane to the lysosome. This is the route by which extracellular macromolecules can be taken up and processed. If vesicles are to be moved over long distances, they are transported along cytoskeletal highways.\\n\\nThe Principle of Fission and Fusion Figure 10.9 illustrates how budding of a vesicle from one organelle, followed by fusion with a second membrane, can transport both soluble proteins and integral membrane proteins to the new compartment. The process retains the \"sidedness\" of the membrane and the compartment it encloses: the side of an integral membrane protein that faced the lumen of the first compartment ends up facing the lumen of the second compartment, while soluble proteins in the lumen of the first compartment do not enter the cytosol but end up in the lumen of the second compartment, or in the extracellular medium if the target membrane is the plasma membrane.\\n\\n# IN DEPTH 10.1 Trafficking Movies\\n\\nChimeric proteins that contain green fluorescent protein and appropriate sorting signals will be processed by the cell\\'s machinery just as proteins coded for on its own genes. Thus protein trafficking can be viewed by fluorescence microscopy in live cells that have been transfected with plasmids coding for such chimeras. Movies illustrating many aspects of membrane transport are available on the World Wide Web (see the CBASC website).\\n---\\n|Fusion|Fusion|\\n|---|\\n|target membrane|Docking|\\n|Fission and fusion.|Fission and fusion.|\\n|Vesicle Transport|Vesicle Transport|\\n|Figure 10.9.|Figure 10.9.|\\n|shape change|protein is conserved|\\n|membrane undergoes orientation of hydrophilic protein|Fission|\\n|hydrophobic|hydrophilic|\\n|LUMEN| |\\n|donor membrane| |\\n---\\n# INTRACELLULAR PROTEIN TRAFFICKING\\n\\n(a)\\n\\n|coatamer|GTP|GTP|GTP|GTP|GTP|GTP|GTP|\\n|---|---|---|---|---|---|---|---|\\n|ARF|GTP| | | | | | |\\n\\n(b)\\n\\nclathrin\\nadaptor protein\\n\\nclathrin\\nreceptor\\n\\nFigure 10.10. Generation of buds by (a) coatamers and (b) clathrin.\\n\\nVesicle Formation\\n\\nVesicle formation is the process during which cargo is captured and the lipid membrane is shaped with the help of cytosolic proteins into a bud, which is then pinched off in a process called fission. The ordered assembly of cytosolic proteins into a coat over the surface of the newly forming vesicle is responsible for forcing the membrane into a curved shape (Fig. 10.10). There are two types of coats that serve this function: coatomer coats and clathrin coats. The coat must be shed before fusion of the vesicle with its target membrane can occur.\\n\\nCoatomer-Coated Vesicles\\n\\nTransport along the default pathway uses coatomer-coated vesicles. This is the mechanism used in trafficking between the endoplasmic reticulum and Golgi, between the individual Golgi stacks, and in budding of constitutive secretory vesicles from the trans Golgi. The coatomer coat consists of seven different proteins that assemble into a complex. The current model for coatomer coat formation at the Golgi is that a guanine nucleotide exchange factor in the donor membrane exchanges GDP for GTP in the GTPase ARF. This causes ARF to adopt its active configuration, in which a fatty acid tail is exposed that anchors ARF into the donor membrane (Fig. 10.10a). Membrane-bound ARF is the initiation site for coatamer assembly and coatomer-coated vesicle formation. The coat is only shed when the vesicle is docking to its target membrane. An ARF-GAP in the target membrane causes the hydrolysis of GTP, and the resulting conformational change causes ARF to retract its hydrophobic tail and become cytosolic and therefore causes the coat to be shed.\\n---\\n# VESICULAR TRAFFICKING BETWEEN INTRACELLULAR COMPARTMENTS\\n\\nClathrin-Coated Vesicles\\n\\nClathrin-coated vesicles mediate selective transport. They are, for example, the means by which protein in the trans Golgi that bears mannose-6-phosphate is collected into a vesicle bound for the lysosome. Clathrin-coated vesicles carry proteins and lipids from the plasma membrane to the endosome, and operate in other places where selective transport is required.\\n\\nFigure 10.10b illustrates how clathrin generates a vesicle. The process starts when the cargo of interest binds to integral proteins of the donor membrane that are selective receptors for that cargo. Clathrin adaptor proteins then bind to cargo-loaded receptors and begin to associate, forming a complex. Lastly, clathrin molecules bind to this complex, forming the coat and bending the membrane into the bud shape.\\n\\nEven though clathrin can force the membrane into a bud shape, it cannot force the bud to leave as an independent vesicle. One of the best-studied membrane fission events is endocytosis. Here a GTPase called dynamin forms a ring around the neck of a budding vesicle. GTP hydrolysis then causes a change in dynamin’s shape that mechanically pinches the vesicle off from its membrane of origin. Unlike coatamer, clathrin coats dissociate as soon as the vesicle is formed, leaving the vesicle ready to fuse with the target membrane.\\n\\n|Example 10.2|Buds in a Test Tube|\\n|---|---|\\n| |If phospholipids are shaken up with aqueous medium, they spontaneously form artificial vesicles called liposomes. If these are then incubated with purified coatomer proteins, ARF, and GTP, bud formation and coated vesicles can be observed by electron microscopy. In contrast neither buds nor vesicles form if liposomes are incubated with clathrin and clathrin adaptor proteins. Clathrin-mediated budding requires ligand to be present in the vesicles, and receptors for that ligand to be present in the membrane of the vesicle. Furthermore even if buds did form, clathrin on its own, unlike coatamer, cannot cause fission. Clathrin-coated buds require another protein such as dynamin to trigger fission.|\\n\\nThe Trans-Golgi Network and Protein Secretion\\n\\nAt its trans face the Golgi apparatus breaks up into a complex system of tubes and sheets called the trans-Golgi network (Fig. 10.8). Although there is some final processing of proteins in the trans-Golgi network, most of the proteins reaching this point have received all the modifications necessary to make them fully functional and to specify their final destination. Rather, the trans-Golgi network is the place where proteins are sorted into the appropriate vesicles and sent down one of three major pathways: constitutive secretion, regulated secretion, or transport to lysosomes.\\n\\nVesicles for constitutive or regulated secretion, though functionally different, look very much alike and are directed to the cell surface (Fig. 10.1). When the vesicle membrane comes into contact with the plasma membrane, it fuses with it. At the point of fusion the membrane is broken through, the contents of the vesicle are expelled to the extracellular space, and the vesicle membrane becomes a part of the plasma membrane. This process, by which the contents of a vesicle are delivered to the plasma membrane and following membrane fusion and breakthrough are released to the outside, is called exocytosis.\\n\\nThe difference between the constitutive and regulated pathways of secretion by exocytosis is that the former is always “on” (vesicles containing secretory proteins are presented\\n---\\n# INTRACELLULAR PROTEIN TRAFFICKING\\n\\nFor exocytosis continuously) while the regulated pathway is an intermittent one in which the vesicles containing the substance to be secreted accumulate in the cytoplasm until they receive a specific signal, usually an increase in the concentration of calcium ions in the surrounding cytosol, whereupon exocytosis proceeds rapidly (page 343). After secretion, vesicular membrane proteins are retrieved from the plasma membrane by endocytosis and transported to the endosomes (Fig. 10.1). From the endosomes vesicles are targeted to the lysosomes, back to the Golgi, or into the pool of regulated secretory vesicles. In a cell that is not growing in size, the amount of membrane area added to the plasma membrane by exocytosis is balanced over a period of minutes by endocytosis of the same area of plasma membrane.\\n\\nNotice that in exocytosis the membrane of the vesicle becomes incorporated into the plasma membrane; consequently the integral proteins and lipids of the vesicle membrane become the integral proteins and lipids of the plasma membrane. This is the principal, if not the only way, that integral proteins made on the rough endoplasmic reticulum are added to the plasma membrane.\\n\\n# Targeting Proteins to the Lysosome\\n\\nOne of the best-understood examples of sorting in the trans-Golgi network is lysosomal targeting (Fig. 10.11). Proteins that are destined for the lysosome are synthesized on the rough endoplasmic reticulum and therefore, like all proteins synthesized here, have a mannose-containing oligosaccharide added. Because they do not have an endoplasmic reticulum retention signal such as KDEL, they are transported to the Golgi apparatus. There proteins\\n\\n|Golgi apparatus|Golgi apparatus|(pre) lysosome|\\n|---|---|\\n|cis face|trans face| |\\n|P|P|clathrin coat|\\n| | |P low pH|\\n| |mannose-6-phosphate receptor|lysosomal protein is released|\\n|addition of a phosphate group to a mannose group on the polypeptide|transport vesicle| |\\n\\nFigure 10.11. Targeting of protein to the lysosome.\\n---\\n# VESICULAR TRAFFICKING BETWEEN INTRACELLULAR COMPARTMENTS\\n\\nDestined for the lysosome are modified by phosphorylation of some of their mannose residues to form mannose-6-phosphate (Fig. 2.11 on page 32). Once the proteins reach the trans-Golgi network, specific receptors for mannose-6-phosphate recognize this sorting signal and cause the proteins to be packaged into vesicles that are transported to the lysosome, where they fuse. In the low pH (5.0) environment of the lysosome, the lysosomal protein can no longer bind to its receptor. The phosphate group is removed by a phosphatase. Vesicles containing the receptor bud off from the lysosome and deliver the mannose-6-phosphate receptors back to the trans-Golgi network.\\n\\nThe function of the lysosome is to degrade unwanted materials. To carry out this function, an inactive or primary lysosome fuses with a vesicle containing the material to be digested. This makes a secondary lysosome. The vesicles with which primary lysosomes fuse may be bringing materials in from outside the cell or they may be vesicles made by condensing a membrane around worn out or unneeded organelles in the cell\\'s own cytoplasm. The latter are sometimes called autophagic vacuoles. Some materials may not be digestible and remain in the lysosome for the lifetime of the cell. These small dense remnant lysosomes are called residual bodies.\\n\\n# Fusion\\n\\nMembrane fusion is the process by which a vesicle membrane incorporates its components into the target membrane and releases its cargo into the lumen of the organelle or, in the case of secretion, into the extracellular medium. Different steps in membrane fusion are distinguished. First, the vesicle and the target membrane mutually identify each other. Then, proteins from both membranes interact with one another to form stable complexes and bring the two membranes into close apposition, resulting in the docking of the vesicle to the target membrane. Finally, considerable energy needs to be supplied to force the membranes to fuse, since the low-energy organization—in which the hydrophobic tails of the phospholipids are kept away from water while the hydrophilic head groups are in an aqueous medium—must be disrupted, even if only briefly, as the vesicle and target membranes distort and then fuse. Each type of vesicle must only dock with and fuse with the correct target membrane, otherwise the protein constituents of all the different organelles would become mixed with each other and with the plasma membrane. Our understanding of the molecular processes leading to membrane fusion is only just beginning to take shape, but our current understanding is that two types of proteins, called SNARES and Rab family GTPases work together to achieve this. SNARES located on the vesicles (v-SNARES) and on the target membranes (t-SNARES) interact to form a stable complex that holds the vesicle very close to the target membrane (Fig. 10.12). Not all vSNARES can interact with all tSNARES, so SNARES provide a first level of specificity. So far, over 50 members of the Rab family have been identified in mammalian cells, and each seems to be found at one particular site where it regulates one specific transport event, thus controlling which vesicle fuses with which target. For example, the recycling of the mannose-6-phosphate receptor back from the lysosome to the trans-Golgi network (Fig. 10.11) requires Rab9, and yellow fluorescent protein-Rab9 chimeras (page 150) locate to the returning vesicles. GTP hydrolysis by Rabs is thought to provide energy for membrane fusion.\\n\\nAnother important factor in membrane fusion is the lipid composition at the fusion site. In particular phosphoinositides, a group of lipids that we will encounter again in Chapter 16 (page 346), seem to play a crucial role.\\n---\\n# INTRACELLULAR PROTEIN TRAFFICKING\\n\\nv-SNARE\\n\\n|t-SNARES|complex formation|fusion and dissociation|\\n|---|---|---|\\n|very stable| | |\\n\\nFigure 10.12. SNAREs and vesicle fusion.\\n\\nExample 10.3 SNARES, Food Poisoning, and Face-Lifts\\n\\nBotulism, food poisoning caused by a toxin released from the anaerobic bacterium Clostridium botulinum, is fortunately rare. Botulinum toxin comprises a number of enzymes that specifically destroy those SNARE proteins required for regulated exocytosis in nerve cells. Without these proteins regulated exocytosis cannot occur, so the nerve cells cannot tell muscle cells to contract. This causes paralysis: most critically, paralysis of the muscles that drive breathing. Death in victims of botulism results from respiratory failure.\\n\\nLow concentrations of botulinum toxin (or \"BoTox\") can be injected close to muscles to paralyze them. For example, in a \"chemical facelift,\" botulinum toxin is used to paralyze facial muscles, producing an effect variously described as \"youthful\" and \"zombie-like.\"\\n\\n# SUMMARY\\n\\n1. The basic mechanisms of intracellular protein trafficking are similar in all eukaryotic cells, from yeast cells to human nerve cells.\\n2. The final destination of a protein is defined by sorting signals that are recognized by specific receptors. The polypeptide chain itself contains targeting sequences while glycosylation and phosphorylation can add additional sorting signals.\\n3. Some sorting signals activate translocation of a protein to a new location, while others such as the endoplasmic reticulum retention signal KDEL cause the protein to be retained at its present location.\\n4. Nuclear proteins are synthesized on free ribosomes and carried through the nuclear pore by Ran-mediated gated transport. Other proteins with a nuclear export signal are carried the other way, again by a Ran-mediated process.\\n5. Peroxisomal proteins, together with the majority of mitochondrial and chloroplast proteins that are not coded for by mitochondrial or chloroplast genes, are synthesized on free ribosomes and then transported across the membrane(s) of the target organelle.\\n---\\n# REVIEW QUESTIONS\\n\\n6. Proteins with an endoplasmic reticulum signal sequence are synthesized on the rough endoplasmic reticulum. The growing polypeptide chain is fed across the membrane as it is synthesized. The signal sequence may then be cleaved off.\\n\\n7. The default pathway for proteins synthesized on the rough endoplasmic reticulum is to pass through the Golgi apparatus and be secreted from the cell via the constitutive pathway.\\n\\n8. Glycosylation in the endoplasmic reticulum and then in the Golgi apparatus has two functions: to produce the final, functional form of the protein and to add further sorting signals such as the lysosomal sorting signal mannose-6-phosphate.\\n\\n9. Vesicles shuttle between most of the single-membrane cellular organelles, the exception being peroxisomes. Budding and then fission of vesicles from the donor membrane can be driven by coatamer family proteins or by clathrin plus dynamin.\\n\\n10. A tight association of vesicular and target SNARE proteins forces vesicles into close association with the target membrane. Fusion requires the action of a Rab family GTPase.\\n\\n# FURTHER READING\\n\\nHammer III, J. A., and Wu, X.S. 2002. Rabs grab motors: Defining the connections between Rab GTPases and motor proteins. Curr. Opin. Cell Biol. 14: 69–75\\n\\nJahn, R., and Sudhof, T. C. 1999. Membrane fusion and exocytosis. Annu. Rev. Biochem. 68: 863–893.\\n\\nLin, R. C., and Scheller, R. H. 2000. Mechanisms of synaptic vesicle exocytosis. Annu. Rev. Cell Dev. Biol. 16: 19–49.\\n\\nSegev, N. 2001. Ypt and Rab GTPases: Insight into functions through novel interactions. Curr. Opin. Cell Biol. 13: 500–511.\\n\\nZerial, M., and McBride, H. 2001. Rab proteins as membrane organizers. Nat. Rev. Mol. Cell Biol. 2: 107–117.\\n\\n# REVIEW QUESTIONS\\n\\nFor each question, choose the ONE BEST answer or completion.\\n\\n1. Cytosolic ribosomes synpesize proteins pat will remain free in pe cytosol plus pose destined for pe interior of\\nA. peroxisomes, endoplasmic reticulum, and pe Golgi apparatus.\\nB. peroxisomes, mitochondria, chloroplasts, and lysosomes.\\nC. peroxisomes, mitochondria, chloroplasts, and pe nucleus.\\nD. mitochondria, chloroplasts, lysosomes, and pe nucleus\\nE. pe Golgi apparatus, lysosomes, and pe nucleus.\\n2. The motif KDEL at pe C terminus of a protein marks it for\\nA. import into pe endoplasmic reticulum and subsequent translocation to pe Golgi apparatus.\\nB. import into pe endoplasmic reticulum and subsequent translocation to pe lysosome.\\n---\\n# INTRACELLULAR PROTEIN TRAFFICKING\\n\\n|3.|Addition of the appropriate GAP to a solution containing a GTPase, GTP, and GDP only will|\\n|---|---|\\n| |A. increase the fraction of the GTPase that is in the GDP-loaded state.|\\n| |B. increase the fraction of the GTPase that is in the GTP-loaded state.|\\n| |C. cause an active cycling process in which the GTPase binds GTP, hydrolyses it to GDP, releases the GDP, and binds another GTP.|\\n| |D. cause an active cycling process in which the GTPase binds GDP, hydrolyses it to GTP, releases the GTP, and binds another GDP.|\\n| |E. have no detectable effect.|\\n\\n|4.|Glycosylation of proteins occurs in the|\\n|---|---|\\n| |A. peroxisome.|\\n| |B. mitochondrion.|\\n| |C. lysosome.|\\n| |D. endoplasmic reticulum.|\\n| |E. all these organelles.|\\n\\n|5.|The lysosomal sorting signal is|\\n|---|---|\\n| |A. N-acetyl-glucosamine.|\\n| |B. asparagine.|\\n| |C. Ran:GTP.|\\n| |D. ribose-6-phosphate.|\\n| |E. mannose-6-phosphate.|\\n\\n|6.|Endocytotic vesicles can only form in the presence of|\\n|---|---|\\n| |A. receptors specific for a ligand that is present in the extracellular medium.|\\n| |B. adaptor proteins.|\\n| |C. clathrin.|\\n| |D. dynamin.|\\n| |E. all of the above.|\\n\\n|7.|Ribosomes that are located on the rough endoplasmic reticulum are there by virtue of|\\n|---|---|\\n| |A. an interaction between a signal recognition particle, a hydrophobic sequence at the N terminus of the protein being synthesized by the ribosome, and a receptor in the membrane of the endoplasmic reticulum.|\\n| |B. a KDEL sequence at the C terminus of the 60S subunit.|\\n| |C. an interaction between the sequence CCUCC on the 16S subunit and a receptor in the membrane of the endoplasmic reticulum.|\\n| |D. mannose-6-phosphate at the 5\\' terminus of the 40S subunit.|\\n| |E. a hydrophobic stretch of RNA at the 5\\' terminus of the 40S subunit.|\\n\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n1. C. See Figure 10.1.\\n\\n2. D. The KDEL motif does not activate a translocation process; rather it holds proteins bearing the sequence in the endoplasmic reticulum if they have already been translocated there.\\n---\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n|3.|A. GAP means GTPase activator protein, so the GTPase will hydrolyze GTP to GDP more rapidly once the GAP has been added. Concerning other answers: (B) A GEF, a guanine nucleotide exchange factor, would do this. (C) This won’t happen because there is no GEF present to cause the GTPase to release its GDP and then bind a GTP. (D) This is nonsense—hydrolysis of GTP yields GDP, not the other way around.|\\n|---|---|\\n|4.|D. Glycosylation occurs in the endoplasmic reticulum and in the Golgi apparatus.|\\n|5.|E. The lysosomal sorting signal is mannose-6-phosphate. Concerning other answers: (B) It is true that the oligosaccharide that contains the mannose-6-phosphate signal is attached to an asparagine residue, but asparagine by itself is not a lysosomal targeting signal. (C) Ran is concerned with nuclear import/export. (D) Ribose is a pentose sugar so it does not have a number 6 carbon!|\\n|6.|E. All of these are required for endocytotic vesicles to form. Receptors specific for a ligand that is present in the extracellular medium, adaptor proteins, and clathrin are required for the generation of a clathrin-coated invagination of the plasma membrane; dynamin is then required to pinch the invagination off to make an endocytotic vesicle.|\\n|7.|A. All the other answers are more or less nonsensical. In particular, it is important to note that ribosomes are found at the endoplasmic reticulum by virtue of a signal sequence on the protein that they are synthesizing, not because the ribosome itself is any different from those that are free in the cytosol, as was implied by answers B through E.|\\n---\\n# HOW PROTEINS WORK\\n\\nThe three-dimensional structures of proteins generate binding sites for other molecules. This reversible binding is central to most of the biological roles of proteins, whether the protein is a gap junction channel that binds a similar channel on another cell (page 55) or a transcription factor that binds to DNA. One special class of proteins, enzymes, have sites that not only bind another molecule but then catalyze a chemical reaction involving that molecule.\\n\\n# How Proteins Bind Other Molecules\\n\\nProteins can bind other protein molecules, DNA or RNA, polysaccharides, lipids, and a very large number of other small molecules and inorganic ions and can even bind dissolved gases such as oxygen, nitrogen, and nitric oxide. Binding sites are usually very specific for a particular ligand, although the degree of specificity can vary widely. Usually the binding is reversible so that there is an equilibrium between the free and bound ligand.\\n\\nA binding site is usually a cleft or pocket in the surface of the protein molecule, which is made up of amino acid side chains appropriately positioned to make specific interactions with the ligand. All of the forces that stabilize tertiary structures of proteins are also used in ligand–protein interaction: Hydrogen bonds, electrostatic interactions, the hydrophobic effect, and van der Waals forces all have their roles. Even covalent bonds may be formed in a few cases—some enzymes form a transient covalent bond with the substrate as part of the mechanism used to effect the reaction.\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright ©C 2004 by John Wiley & Sons, Inc.\\n---\\n# HOW PROTEINS WORK\\n\\nDynamic Protein Structures\\n\\nIt is easy to get the impression that protein structures are fixed and immobile. In fact proteins are always flexing and changing their structure slightly around their lowest energy state. A good term for this is “breathing.” Many proteins have two low-energy states in which they spend most of their time, like a sleeper who, though twisting and turning throughout the night, nevertheless spends most time lying on their back or side. An example is the glucose carrier (Fig. 11.1). This is a transmembrane protein that forms a tube through the membrane. It is stable in one of two configurations. In one the tube is open to the cytosol; in the other the tube is open to the extracellular medium. By switching between the two states, the glucose carrier carries glucose into and out of the cell.\\n\\nAllosteric Effects\\n\\nThe glucose carrier is able to bind a ligand—the glucose molecule—in either of its low-energy conformations. In contrast, the lac repressor (page 113) can only bind its ligand, the operator region of the lac operon, in one conformation. On its own the protein predominantly adopts this conformation so transcription is prevented as it binds to the DNA. When the lac repressor binds allolactose (a signal that lactose is abundant), it is locked into a second, inactive form that cannot bind to the DNA (Fig. 6.8 on page 113). Transcription is no longer repressed, although the cAMP–CAP complex is additionally required if transcription is to proceed at a high rate. This type of interaction, in which the binding of a ligand at one place affects the ability of the protein to bind another ligand at another location, is called allosteric and is usually a property of proteins with a quaternary structure (i.e., with multiple subunits).\\n\\nHemoglobin (Fig. 9.20 on page 205) is an example of a protein where allosteric effects play an important role. Each heme prosthetic group, one on each of the four subunits, can bind an oxygen molecule. We can get an idea of what one subunit on its own can do by looking at myoglobin (Fig. 11.2a), a related molecule that moves oxygen within the cytoplasm. Myoglobin has just one polypeptide chain and one heme. The green line in Figure 11.2b shows the oxygen-binding curve for myoglobin. Starting from zero oxygen, the first small increase in oxygen concentration produces a large amount of binding to myoglobin; the\\n---\\n| |% of binding sites occupied|\\n|---|---|\\n|100| |\\n|80|myoglobin|\\n|60| |\\n|40|hemoglobin|\\n|20| |\\n|0|0|\\n| |2000|\\n| |4000|\\n| |6000|\\n\\nFigure 11.2. (a) The monomeric oxygen-carrying protein myoglobin. (Illustration: Irving Geis. Rights owned by Howard Hughes Medical Institute. Reproduction by permission only.) (b) Oxygen binding of myoglobin (in green) and hemoglobin (in black) as oxygen pressure increases.\\n---\\n# HOW PROTEINS WORK\\n\\nNext increase in oxygen produces a slightly smaller amount of binding, and so on, until myoglobin is fully loaded with oxygen. A curve of this shape is known as hyperbolic. The black line in Figure 11.2b shows the oxygen-binding curve for hemoglobin. Starting from zero oxygen, the first small increase in oxygen concentration produces hardly any binding to hemoglobin. The next increase in oxygen produces much more binding so the curve gets steeper before leveling off again as the hemoglobin becomes fully loaded. This behavior is called cooperative. The explanation for this behavior is that the hemoglobin subunits can exist in one of two states, only one of which has a high affinity for oxygen. The way that the four subunits fit together means that they all must be in one form or the other. When oxygen concentration is low, most of the hemoglobin molecules have their subunits in the low-affinity form. As oxygen increases, it begins to bind to the hemoglobin—little at first as most of the hemoglobin is in the low-affinity form and only a little in the high-affinity form. As oxygen binds, more molecules switch to the high-affinity form as the low- and high-affinity forms are in equilibrium. Eventually virtually all of the molecules have made the switch to the high-affinity form. This produces the curve shown in Figure 11.2b. This cooperative oxygen binding makes hemoglobin an effective transporter as it will load up with oxygen in the lungs but will release it readily in the capillaries of the tissues where the oxygen concentration is low. Myoglobin would release little of its bound oxygen at the oxygen concentrations typical of respiring tissues.\\n\\nSome enzymes show cooperative behavior caused by an allosteric effect that causes binding of one substrate molecule to make it easier for the other substrate to bind. The degree of cooperativity can be altered by the binding of other molecules (called effectors) that act to switch the enzyme on or off.\\n\\n# Chemical Changes That Shift the Preferred Shape of a Protein\\n\\nProteins can change conformation as a result of environmental changes, by binding a ligand or by having a particular group attached covalently to them. Anything that changes the pattern of electrostatic interactions within a protein will alter the relative energy of its states. If, for example, a protein contains histidine residues, merely changing the pH will do this. In solutions with pH greater than 7 most of the histidine residues in a protein will be\\n\\n|histidine|residues|\\n|---|---|\\n|pH8| |\\n|+| |\\n|+| |\\n|pH6| |\\n\\nFigure 11.3. A pH change that alters the charge on histidine will alter the balance of forces within a protein. At pH 6, the structure on the right will predominate.\\n---\\nENZYMES ARE PROTEIN CATALYSTS\\n\\n|Resting state|phosphorylated ATP held non-covalently in crevice|Phosphorylated aspartate residue|state|\\n|---|---|---|---|\\n|aspartate residue|γ phosphate of ATP transferred to aspartate|ADP| |\\n|residue| | | |\\n| | | | |\\n| | | | |\\n|O| |P| |\\n|PO| | | |\\n|OO| | | |\\n|PO| |O| |\\n|O|O| | |\\n|PO| |O| |\\n|O|O| | |\\n|charges repel| | | |\\n\\nFigure 11.4. Phosphorylation changes the charge pattern, and hence the balance of forces within the calcium ATPase, forcing a change of shape.\\n\\nuncharged (page 187). In solutions with a pH less than 7, most of the histidine residues will bear a positive charge. A protein conformation that had two histidine residues close together (Fig. 11.3) would therefore be stable in alkaline conditions, but in acid conditions the two residues would each bear a positive charge and repel, destabilizing the conformation. Another mechanism that is used to change the conformation of proteins is the addition of a negatively charged phosphate group by a class of enzymes called protein kinases. Proteins can be phosphorylated on serine, threonine, tyrosine, aspartate, glutamate, or histidine. The calcium ATPase (Fig. 11.4) is a transmembrane protein that forms a tube that can be open to the cytosol or to the extracellular medium. In its resting state the tube is open to the cytosol, while ATP is held by noncovalent interactions in a crevice in a cytosolic domain. Phosphorylation transfers the γ phosphate of ATP to a nearby aspartate residue. Repulsion between the phosphoaspartate and the remaining ADP means that the lowest energy state of the protein is now one in which the tube is open to the extracellular medium. This mechanism is used to force calcium ions out of the cytoplasm into the extracellular fluid. The calcium ATPase is described in more detail on page 318. The movements produced in a protein by phosphorylation are small, on the order of nanometers or less. When these are repeated many times and amplified by lever systems, they can produce movements of micrometers or even meters. The beating of a flagellum (page 386) or the kicking of a leg are both produced by phosphorylation-induced protein shape changes.\\n\\nENZYMES ARE PROTEIN CATALYSTS\\n\\nLife depends on complex networks of chemical reactions. These are mediated by enzymes. Enzymes are catalysts of enormous power and high specificity. Consider a lump of sugar. It is combustible but quite difficult to set alight. A chemical catalyst would speed up its combustion, and we would end up with heat, a little light, carbon dioxide, and water. Swallowed and digested, the sucrose is broken down in many steps to carbon dioxide and\\n---\\n# HOW PROTEINS WORK\\n\\nwater by the action of at least 22 different enzymes, and the energy released is used to drive other reactions in the body.\\n\\nAt a basic level, a reaction carried out by an enzyme can be expressed as\\n\\n|E + S|ES|EP|E + P|\\n|---|---|---|---|\\n| | | |⇀|⇀|⇀|\\n| | | |↽|↽|↽|\\n\\nwhere E is the enzyme, which binds the substrate S to form the complex ES. The term substrate is used for a ligand that binds to an enzyme and that is then transformed to the product, P. The region of the protein where the substrate binds and the reaction occurs is called the active site. This binding is specific, often highly so. The enzyme β-galactosidase (page 109) is moderately specific and will split not only lactose but also any other disaccharide that has a glycosidic bond to β-galactose. By contrast glycogen phosphorylase kinase (page 305) acts with absolute specificity on a single substrate, another enzyme called glycogen phosphorylase—none of the thousands of other proteins in the cell can substitute. In general, the specificity of an enzyme is conferred by the shape of the active site and by particular amino acid side chains that interact with the substrate. Binding of the substrate produces the enzyme–substrate complex ES; the catalytic function of the protein then converts the substrate to product, still bound to the enzyme in the complex EP. Finally the product dissociates from the enzyme. Chemical engineers use catalysts made of many materials, and within cells there are catalysts called ribozymes that are made of RNA. However, only proteins, with their enormous repertoire of different shapes can produce catalysts of high selectivity.\\n\\nThe catalytic rate constant kcat (also known as the turnover number) of an enzyme gives us an idea of the enormous catalytic power of most enzymes. It is defined as the number of molecules of substrate converted to product per molecule of enzyme per unit time (equally it is moles of substrate converted per mole of enzyme per unit time). Many enzymes have kcat values around 1000 to 10,000 per second. The reciprocal of kcat is the time taken for a single event. Thus if kcat is 10,000 s-1, one substrate molecule will be converted every tenth of a millisecond. Some enzymes achieve very much higher rates. Catalase, an enzyme found in peroxisomes, has a kcat of 4 × 107 s-1, and so it takes only 25 ns to split a molecule of hydrogen peroxide into oxygen and water.\\n\\nThe Initial Velocity of an Enzyme Reaction\\n\\nThe basic experiment in the study of enzyme behavior is measurement of the appearance of product as a function of time (Fig. 11.5). This is often called an enzyme assay. The product appears most rapidly at the very beginning of the reaction. As the reaction progresses, the rate at which the product appears slows down and eventually becomes zero when the system has reached equilibrium. All reactions are in principle reversible; thus the observed overall rate of the reaction is actually the difference between the rate at which the product is being formed and the rate at which the product is being broken down again in the reverse reaction. In many enzyme reactions the equilibrium lies strongly toward the product.\\n\\n# IN DEPTH 11.1 What to Measure in an Enzyme Assay\\n\\nIn an enzyme assay we measure the appearance of product as a function of time. In principle we could do this by starting identical reactions in a series of test tubes and stopping the reaction in each test tube at a different time after the start and then measuring the amount of product in each tube.\\n---\\nENZYMES ARE PROTEIN CATALYSTS\\n\\nHowever, if either the substrate or the product has a property that can be measured while the reaction is proceeding, then the whole assay can be performed in one test tube. Many enzyme assays are done by using changes in the absorption of light when the substrate is converted to product. Other optical properties can be used. In their original work Michaelis and Menten studied an enzyme that breaks the disaccharide sucrose down into glucose and fructose. The mixture of glucose plus fructose rotated polarized light differently from sucrose, and they used this property to follow the course of the reaction. If there is no convenient optical property, then others may be available; for example, one can monitor the progress of a reaction in which the polysaccharide glycogen (page 30) is hydrolyzed by digestive enzymes by measuring the resulting fall in viscosity.\\n\\nWe can simplify the analysis of enzyme reactions if we consider only the start of the reaction, when there is no product present and so the back reaction can be neglected. The rate of reaction at time zero (the initial velocity v0, sometimes called the initial rate) is found by plotting a graph of product concentration as a function of time and measuring the slope at time zero (Fig. 11.5). In practice the slope is measured over the first 5% of the total reaction. Initial velocity, v 0, is conveniently expressed as the rate at which the concentration of the product increases, that is, moles per liter per second. Enzymes must be assayed under controlled conditions because temperature, pH, and other factors alter the activity. Most are such very effective catalysts that they must be assayed under conditions where the concentration of the enzyme is always very much less than the concentration of the substrate. Otherwise the reaction would be over in a fraction of a second.\\n\\nProduct concentration [P] slope = v o\\n\\nFigure 11.5. Definition of v 0, the initial velocity of a chemical reaction.\\n---\\n# HOW PROTEINS WORK\\n\\nEnzymes are studied for many reasons. An understanding of how they achieve their catalytic excellence and specificity is of fundamental interest and has many practical applications as we increasingly use them in industrial processes and we even seek to design enzymes for particular tasks. Measurements of enzyme concentrations and properties allow us to study processes within cells and organisms. The starting point of any study of an enzyme is to determine its activity, measured as kcat, and how tightly it can bind its substrate—its substrate affinity. Substrate affinity is measured by another constant called the Michaelis constant, KM. We will now explain how these constants are measured.\\n\\nEffect of Substrate Concentration on Initial Velocity\\n\\nLet us consider a series of experiments designed to see how the initial velocity of an enzyme reaction varies with the concentration of substrate (which is always much greater than that of the enzyme). Each of the smaller graphs in Figure 11.6 shows the result of one of these experiments. As we increase the substrate concentration, we find that at first the velocity increases with each increase in substrate concentration but that, as the substrate\\n\\n|[P]|[P]|[P]|\\n|---|---|---|\\n|[S] = 1K M|[S] = 2K M|[S] = 5K M|\\n|time|time|time|\\n\\nFigure 11.6. v0 measured in a number of reaction tubes (with [E] constant and always less than [S]) forms a hyperbolic curve when plotted as a function of substrate concentration.\\n---\\n# ENZYMES ARE PROTEIN CATALYSTS\\n\\nconcentration becomes larger, the increases in rate produced get smaller and smaller. We have already met the name of a curve of this type: it is a hyperbola. The initial velocity approaches a maximum value that is never exceeded. Reactions that show this sort of dependence on substrate concentration are said to show saturation kinetics.\\n\\nHow can we explain this? The reaction sequence can be simplified to\\n\\nE + S\\nES → E + P\\n\\n(we are using initial velocities so we can ignore any back reaction). The enzyme and substrate must collide in solution, and the substrate must bind at the enzyme’s active site to form the ES complex. The chemical reaction then takes place within the ES complex, and finally the product is released. In experiments with higher and higher substrate concentrations, there is ever more ES present, and this increasing ES gives an increasing rate of product release. At very high substrate concentrations virtually all of the enzyme is present as ES, and the observed rate is limited by the ES → E + P step. Thus, as the substrate concentration increases, the reaction rate levels off as it approaches a maximal velocity called Vm or Vmax. We can define Vm as the limiting initial velocity obtained as the substrate concentration approaches infinity. It is the product of the catalytic rate constant kcat and the amount of enzyme present, that is, Vm = kcat [E]. Having defined Vm, we now define the Michaelis constant, KM, as that substrate concentration that gives an initial velocity equal to half Vm.\\n\\nThe plot of v0 against [S] gives a hyperbolic curve that is described by the equation\\n\\nv0 = Vm [S]\\nKM + [S]\\n\\nwhere [S] is the substrate concentration. This is called the Michaelis–Menten equation after Maud Menten and Leonor Michaelis who propounded a general theory of enzyme action in 1913. The Michaelis constant KM is the substrate concentration that gives an initial velocity numerically equal to half of Vm. Another way of looking at this is to say that if the enzyme is saturated with substrate, then the rate of reaction will be Vm, while at a substrate concentration giving an initial velocity of 1/2Vm the enzyme is half saturated with substrate. A small value of KM means that the enzyme has a high affinity for the substrate.\\n\\nThe Effect of Enzyme Concentration\\n\\nAs the concentration of an enzyme increases, so does the initial velocity, and this relationship is linear: if you double the enzyme concentration, you double the initial velocity. This fact means that we can measure how much of an enzyme is present by measuring the rate of the reaction it catalyzes.\\n---\\n# IN DEPTH 11.2 Determination of Vm and KM HOW PROTEINS WORK\\n\\nThe initial rate of a reaction depends on Vm and KM according to the Michaelis–Menten equation:\\n\\nv0 = Vm[S] / (KM + [S])\\n\\nThus the information necessary to estimate Vm and KM is contained within data such as that shown in Figure 11.6. However, it is not immediately obvious how one would go about this: The curve flattens off well before Vm is reached so that guessing Vm by eye is near to impossible. Nowadays many computer programs are available that will fit the hyperbolic Michaelis–Menten equation to the raw data. Before this, biologists used mathematical transformations of the Michaelis–Menten equation that yielded straight-line relationships. The best known is called the Lineweaver–Burk plot after its inventors. It simply inverts the Michaelis–Menten equation to get\\n\\n1/v0 = KM + [S] / Vm[S] = KM + [S] / Vm[S]\\n\\nhence 1/v0 = KM × 1 / Vm + 1 / [S]\\n\\nThis function can be recognized as having the form of the equation of a straight line y = mx + c, where y is 1/v0 and x is 1/[S]. Thus if one plots 1/v0 on the vertical axis and 1/[S] on the horizontal axis, one gets a straight line where the y intercept is 1/Vm, the x intercept is −1/KM, and the slope is KM/Vm.\\n\\n|1 υ0|1|VM|\\n|---|---|---|\\n|-1|KM|[S]|\\n---\\n# ENZYME CATALYSIS\\n\\n|Enzyme(s)|K (mol liter-1)|k/K (liter mols-1)|\\n|---|---|---|\\n|Trimeric G protein (page 344)|0.02|6 × 10-7|\\n|Lysozyme (page 192)|0.5|6 × 10-6|\\n|Aminoacyl tRNA synthase (page 165)|7.6|9 × 10-4|\\n|Intestinal ribonuclease (page 199)|7.9 × 102|7.9 × 10-3|\\n|Acetylcholinesterase (page 368)|1.4 × 104|9 × 10-5|\\n|Catalase (page 60)|4 × 101.1|4 × 10-1|\\n\\nThe Specificity Constant\\n\\nThe Michaelis constant KM reflects the affinity of an enzyme for its substrate; kcat reflects\\nthe catalytic ability of an enzyme. The ratio of these, kcat/KM, is the specificity constant,\\nwhich is a measure of how good the enzyme is at its job. A high specificity constant means that a reaction goes fast\\n(kcat is big) and the enzyme does not need a high concentration of substrate (KM is small). When\\nan enzyme has relatively low specificity, that is, it can work on a number of different substrates, the substrate that\\nhas the largest specificity constant is the preferred substrate for the enzyme.\\n\\nA reaction cannot go faster than the rate at which enzyme and substrate actually collide. In some enzymes this rate\\nof collision is the factor limiting the overall rate, and such enzymes are said to be “diffusion limited” and are\\nconsidered to have reached a perfection of biological design. Diffusion-limited values of kcat/KM\\ncan be calculated to be in the range 108–1010 liter per mole per second. Table 11.1 shows kcat,\\nKM, and kcat/KM for a number of enzymes.\\n\\n# Example 11.1 Speed Isn’t Everything\\n\\nIn this chapter we emphasize enzymes as tools for performing a reaction that is required in the cell. For this type\\nof enzyme, the faster the reaction occurs, the better, although in many cases an “optimized” enzyme must be controlled\\nso that it only operates at maximum rate when lots of product is needed.\\n\\nFor other enzymes, slowness is a virtue. We will meet trimeric G proteins in Chapter 16. Like the GTPases that we\\nhave already met (EF-tu, Ran, and Rab), these are active when they have GTP bound, and switch to an inactive state\\nwhen they hydrolyze the GTP to GDP. While they are active, they turn on their target processes. In these enzymes,\\ntherefore, a slow rate of reaction allows them to act as timer switches. They are activated when GDP is ejected and\\nGTP bound. They then remain active until they hydrolyze the GTP. For the typical trimeric G protein listed in Table\\n11.1, kcat is 0.02 per second, so the “timer” is set to 50 s (the reciprocal of 0.02 s-1).\\n\\nENZYME CATALYSIS\\n\\nA catalyst speeds the rate of reaction but does not alter the position of the equilibrium E + S ⇀ E + P. Reaction\\nspeed is increased by lowering the activation energy barrier between the reactants and products. Figure 11.7 illustrates\\nthis effect. The top graph shows the total energy of the system as the reaction proceeds, the horizontal axis representing\\nthe progress\\n---\\n# HOW PROTEINS WORK\\n\\n| |energy|without enzyme|\\n|---|---|---|\\n|transition state| | |\\n|A|reactants|products|\\n|progress of reaction|energy|(a)|with enzyme|\\n\\n| |transition state|\\n|---|---|\\n|A|reactants|products|\\n|progress of reaction|(b)|\\n\\nFigure 11.7. Catalysts act by reducing the activation energy (A) of a reaction.\\n\\nMedical Relevance 11.1\\n\\nMeasurement of Enzyme Concentrations in Blood\\n\\nBecause the initial velocity of a reaction is a linear function of the enzyme concentration, we can measure the concentration of an enzyme by measuring the rate of the reaction it catalyzes. Clinicians make use of this to measure the amount of an enzyme present in the blood. If enzymes that are normally found within cells appear in the blood in more than trace amounts, this indicates that cells are being damaged and their contents are leaking out. The presence of enzymes characteristic of a particular organ tells the clinician which tissue is affected. The digestive enzyme trypsin is stored in an inactive form called trypsinogen in the cells of the pancreas. Trypsinogen is activated once it is secreted into the lumen of the gut. Trypsinogen can be detected in the blood by converting it to the active enzyme and measuring its activity. The presence of trypsinogen in the blood of a fetus is a sign that the cells of the pancreas are dying, and this is in turn an early indicator of cystic fibrosis (page 426).\\n---\\n# COFACTORS AND PROSTHETIC GROUPS\\n\\nof the reaction so that at the left we have the reactants, on the right we have the products, and in the middle we have the transition state that the system has to pass through for the reaction to occur. Because the transition state is of higher energy than the reactants, the reaction will only occur if the reactants have additional energy (e.g., as thermal motion) equal to the activation energy. The bottom graph shows the same reaction in the presence of a catalyst. Although the energy of the reactants and products has not changed, the presence of the catalyst has reduced the energy of the transition state and has therefore reduced the activation energy. A greater proportion of all the reactant molecules in solution will possess this smaller amount of extra energy, so more can undergo the reaction: the catalyst has speeded up the rate at which the reaction occurs. Each enzyme has evolved to reduce the activation energy of a particular reaction. Clearly the binding of the substrate is as central to catalysis as it is to specificity. The active site contains precisely positioned amino acid side chains that promote the reaction. Daniel Koshland proposed that the enzyme’s active site was shaped not so much to fit the substrate but to fit a molecule that was halfway between substrate and product, and therefore to reduce the activation energy of the reaction by stabilizing, and hence reducing the energy of, the transition state. He proposed that the enzyme might actually change shape when it bound the substrate to promote catalysis. This is called induced fit and many enzymes have been shown to do this.\\n\\nEnzymes that need to perform reactions that are outside the repertoire of the 20 amino acid side chains recruit other chemical species to help them do the job. The aminotransferases provide a good example. These enzymes are central to amino acid metabolism (page 292) and catalyze the interconversion of amino acids and oxo-acids by moving an amino group (Fig. 11.8). Pyridoxal phosphate (derived from vitamin B6) is bound by the protein and accepts the amino group from the amino acid donor, which then leaves the enzyme converted to a new oxo-acid. The oxo-acid substrate then binds, accepts the amino group from the pyridoxal phosphate, and leaves having been converted to an amino acid. Pyridoxal phosphate ends up exactly as it started and remains bound to the enzyme ready for another cycle. These helper chemicals are called cofactors when, like pyridoxal phosphate, they are not tightly bound to the enzyme. We already know the term for a molecule that is very tightly bound to a protein and helps it perform its job—it is a prosthetic group (page 205). These terms overlap and usage varies.\\n\\nThe prosthetic group of hemoglobin and myoglobin, heme, (Fig. 9.21 on page 206) binds oxygen. It is a versatile molecule that is also found in electron transfer proteins called cytochromes. Cytochromes found in the inner mitochondrial membrane play important roles in the electron transport chain (page 265). Other cytochromes, the cytochromes P450 that we have met earlier (pages 124, 158, 166), are found in the membranes of the endoplasmic reticulum and mitochondria. There are at least 50 different cytochrome P450s in humans. Some of these proteins carry out very specific metabolic reactions in the process of converting cholesterol into the different steroid hormones our body utilizes. Other cytochromes P450 are less specific in their reactions. Their role is to detoxicate the millions of foreign chemicals to which we are exposed on a daily basis. However, all cytochromes P450 share one property in common: they all insert a single atom of oxygen, derived from molecular oxygen, into a substrate. Subsequent rearrangements often result in the oxygen being in the form of a hydroxyl group in the final reaction product. This transformation converts a\\n---\\n# HOW PROTEINS WORK\\n\\npyridoxal phosphate is the cofactor for aminotransferases during the reaction it picks up the amino group from the amino acid substrate\\n\\npyridoxamine phosphate\\n\\n|Alanine glyoxylate aminotransferase catalyzes the reaction:|\\n|---|\\n|CH3|O|CH3|H|\\n|OOC|CH|NH3| |+|H|C|OOC|C|O|+|H|C|NH3|\\n|alanine| | |C|O| |pyruvate| |C|O| |glycine|\\n| | | |O| | |glyoxylate| | | | | |\\n\\nThe reaction is in two stages:\\n\\n|E-pyridoxal phosphate|+|alanine|E-pyridoxamine phosphate|+|pyruvate|\\n|---|---|---|---|---|---|\\n|E-pyridoxamine phosphate|+|glyoxylate|E-pyridoxal phosphate|+|glycine|\\n\\nFigure 11.8. Aminotransferases use a cofactor that participates in the reaction but ends up unchanged. E represents the enzyme molecule.\\n\\nhydrophobic molecule into one that is more hydrophilic. After hydroxylation other molecules are coupled to the hydroxyl group to make the foreign chemical even more hydrophilic and hence allow it to be rapidly excreted in the urine and feces. Unfortunately, in very rare cases, the addition of an oxygen atom into a molecule can increase its reactivity and, instead of being rendered harmless by the action of a cytochrome P450, it is transformed into a very dangerous chemical that can damage DNA. If the damage causes mutation or strand breakage, then cancer can result. Polycyclic aromatic hydrocarbons, found in tobacco\\n---\\n# ENZYMES CAN BE REGULATED\\n\\nSmoke, and the chemical aflatoxin, found in the mold that grows on peanuts, are converted into chemical carcinogens by the action of certain cytochromes P450.\\n\\n# ENZYMES CAN BE REGULATED\\n\\nIn this chapter we have discussed the catalytic role of enzymes in isolation. In fact enzymes are like other proteins in showing the complex behavior described in Chapter 9 such as multiple states, multiple binding sites, quaternary structure, and phosphorylation. For instance, some important enzymes have quaternary structures and show cooperativity between the active sites. Such enzymes will not follow the Michaelis–Menten equation, but instead the curve of initial velocity against substrate concentration will have the same S-shaped or sigmoid curve seen in the binding of oxygen to hemoglobin (Fig. 11.2).\\n\\nA good example of an allosteric enzyme is to be found in the processes that allow us to use sugars as metabolic fuels. The pathway called glycolysis (page 284) converts glucose and fructose to pyruvate with a net production of two ATP molecules. The pyruvate is then passed to the mitochondria, which use it to produce more ATP.\\n\\nThe first committed step in this pathway is the addition of a second phosphate group to monophosphorylated fructose. The γ phosphate of ATP is transferred to the number 1 carbon of fructose-6-phosphate. Before this step, sugars can be converted to glycogen or metabolized in other ways, but after this step the sugar is committed to be broken down to pyruvate. The step is catalyzed by the enzyme phosphofructokinase (Fig. 11.9). ATP serves as an allosteric inhibitor of this enzyme so that it operates slowly when ATP concentrations are high and there is no need for more ATP to be produced. When there is a drain on ATP, levels of AMP increase (page 259). AMP competes with ATP for the regulatory site and causes the enzyme to switch to the active, high-affinity conformation. Thus as the ATP concentration is reduced and AMP levels increase, the pathway is more and more activated until it is turned on to full speed. We very often see this type of control system, where the product of a pathway or process feeds back to allosterically inhibit an enzyme at the start of that pathway. They are an example of negative feedback (page 303).\\n\\nIt is not only enzymes involved in metabolism that are regulated by ligand binding. Cdk1, the enzyme that triggers mitosis (page 411), is an allosterically regulated enzyme. On its own, in solution, it spends the vast majority of time in an inactive conformation. In order to assume the active conformation, Cdk1 must bind a protein ligand called cyclin B (Fig. 11.10). In addition to this allosteric regulation, Cdk1 is also regulated by phosphorylation: for most of the time the catalytic site is blocked by two phosphate groups within the catalytic domain.\\n\\n|fructose-6-phosphate|ATP|+ve|fructose-1,6-bisphosphate|\\n|---|---|---|---|\\n| |inhibited by ATP| | |\\n\\nFigure 11.9. Phosphofructokinase is regulated by the binding of ATP or AMP at a regulatory site that is separate from the active site. Binding of ATP inhibits while binding of AMP activates.\\n---\\n# HOW PROTEINS WORK\\n\\n|inactive|inactive|active kinase|inactive|\\n|---|---|---|---|\\n|T14|P T14 Y15|T14 Y15|T14|\\n|P P Y15| |Cdk1|P P Y15|\\n| |Cdk1|catalytic site| |\\n\\nFigure 11.10. Control of Cdk1 by cyclin B and by phosphorylation.\\n\\nat threonine 14 and tyrosine 15. In order to become an active enzyme, Cdk1 must lose these\\ntwo phosphates while cyclin B is bound. This complex control means that Cdk1 can act as\\na checkpoint. If the phosphate groups have been removed from T14 and Y15, AND cyclin\\nB is present at a high enough concentration, THEN it is safe to proceed into mitosis.\\nCdk1 is an example of an enzyme that is turned off by phosphorylation. Other enzymes\\nare turned on by phosphorylation. Examples include RNA polymerase II (page 120) and\\nglycogen phosphorylase (page 305).\\n\\n# IN DEPTH 11.3 Rapid Reaction Techniques\\n\\nEnzyme measurements are usually carried out with low concentrations of the\\nenzyme because higher concentrations would give initial velocities too fast to\\nmeasure. Initial velocity measurements are always made long after the enzyme–\\nsubstrate complex has formed. It would be very interesting to be able to observe\\nthe actual formation of enzyme–substrate complexes and indeed, more generally,\\nto observe the formation of protein–ligand complexes. These reactions are rapid,\\noccurring on a millisecond time scale or less.\\nHartridge and Roughton devised the method illustrated in the accompanying\\ndiagram for measuring the rate at which hemoglobin combined with oxygen. A\\nram pushes two syringes, one containing hemoglobin and one containing oxy-\\ngenated solution. The solutions mix and pass down a tube, so that the further one\\nlooks to the right in the tube, the more time the oxygen has had to bind to the\\nhemoglobin. The association of oxygen with hemoglobin can be monitored by an\\noptical change, the same color change that causes arterial blood to be bright red\\nwhile venous blood is a dark bluish red. Simply moving an optical detector from left\\nto right along the tube showed the concentration of oxygen–hemoglobin complex\\nafter different reaction times. However, all the time the measurements are being\\nmade, the hemoglobin solution is running out of the pipe at the right-hand end,\\nso that one can only use this approach when large quantities of the protein are\\navailable.\\nThe second figure shows a less wasteful method called stopped flow. Here two\\nsolutions are mixed and passed into an observation chamber and from it into a sy-\\nringe. The plunger of this “stopping syringe” moves back until it hits a stopping\\n---\\nRAPID REACTION TECHNIQUES\\n\\n|mixer|monochromator|\\n|---|---|\\n|ram|hemoglobin|\\n| |oxygenated solution|\\n| |detector|\\n| |move along pipe to sample reaction at different times after mixing|\\n|monochromator| |\\n|ram|mixer|\\n|light beam|microswitch stop|\\n| |stopping syringe|\\n|filling valves|detector trigger|\\n| |oscilloscope|\\n---\\n# HOW PROTEINS WORK\\n\\n1. Most protein functions arise from peir ability to specifically bind oper molecules in a reversible yet specific fashion. Shape changes, triggered by pe binding of oper molecules, mediate protein movements and function. Allostery is a special case where a shape change induced by a ligand at one site changes pe affinity at anoper site.\\n2. Enzymes are highly specific biological catalysts. The turnover number or catalytic rate constant, kcat, is pe maximum number of substrate molecules pat can be converted to product per molecule of enzyme per unit time.\\n3. The initial velocity (i.e., pe rate at pe start when product is absent) of many enzymes shows a hyperbolic dependence on substrate concentration. At high substrate concentrations pe initial velocity approaches a limiting value Vm, as pe enzyme is saturated wip substrate. The substrate concentration pat gives an initial velocity equal to half Vm, is pe Michaelis constant KM. This indicates pe enzyme’s affinity for pe substrate.\\n4. The initial velocity is related to pe substrate concentration by pe Michaelis–Menten equation:\\nv0 = Vm[S] / (KM + [S])\\n5. The initial velocity is directly proportional to pe enzyme concentration.\\n6. Some enzymes use cofactors to carry out reactions pat require different properties from pose of pe side chains of pe 20 amino acids found in proteins.\\n7. Enzyme activity wipin cells is modulated by a variety of mepods pat include phosphorylation and allosteric effects.\\n\\n# FURTHER READING\\n\\nCornish-Bowden, A. 1995. Fundamentals of Enzyme Kinetics, rev. ed. London: Portland Press.\\nFersht, A. 1999. Structure and Mechanism in Protein Science. New York: W. H. Freeman.\\nVoet, D., Voet, J. D. 2003. Biochemistry, 3rd ed. New York: Wiley.\\n---\\n# REVIEW QUESTIONS\\n\\n1.  Conformational or shape changes in proteins allow\\nA. conversion of chemical energy into kinetic (movement) energy.\\nB. cooperativity between subunits in a quaternary structure.\\nC. allosteric control of enzymes.\\nD. proteins to carry oper molecules across membranes.\\nE. all of pe above.\\n\\n2.  The turnover or catalytic constant (kcat) of an enzyme is\\nA. pe number of molecules of substrate converted to product per mole of enzyme per unit time.\\nB. pe time taken for a single catalytic event.\\nC. pe number of substrate molecules in a mole.\\nD. pe number of moles of substrate converted to product per mole of enzyme per unit time.\\nE. pe initial velocity of pe reaction when half pe enzyme molecules have bound substrate.\\n\\n3.  The oxygen-binding curve for hemoglobin is S-shaped: starting from zero oxygen, pe\\nfirst small increase in oxygen concentration produces hardly any binding to hemoglobin.\\nThe next increase in oxygen produces much more binding so pat pe curve gets steeper\\nbefore leveling off again as pe hemoglobin becomes fully loaded. Hemoglobin can show\\npis behavior because\\nA. it contains heme prospetic groups.\\nB. it contains magnesium, in contrast to pe iron in myoglobin.\\nC. it comprises two α chains of high oxygen affinity and two β chains of low oxygen affinity.\\nD. its four subunits can exist in one of two states, only one of which has a high affinity for oxygen.\\nThe way pat pe four subunits fit togeper means pat pey all must be in one form or pe oper.\\nWhen oxygen concentration is low, most of pe hemoglobin molecules have peir subunits in\\npe low-affinity form. As oxygen binds, more molecules switch to pe high-affinity form.\\nE. its four subunits can exist in one of two states, only one of which has a high affinity for oxygen.\\nThe way pat pe four subunits fit togeper means pat pey all must be in one form or pe oper.\\nWhen oxygen concentration is low, most of pe hemoglobin molecules have peir subunits in\\npe high-affinity form. As oxygen binds, more molecules switch to pe low-affinity form.\\n\\n4.  Which of pe following statements about enzymes is false?\\nA. They bind peir substrate.\\nB. They lower pe activation energy for pe reaction pey catalyze.\\nC. They alter pe equilibrium constant for pe reaction pey catalyze.\\nD. They stabilize intermediates along pe papway of pe reaction pey catalyze.\\nE. They can change shape to promote pe reaction.\\n\\n5.  Protein kinases alter pe conformation of peir target protein by adding\\nA. H  +  ions.\\nB. phosphate groups.\\nC. histidine residues.\\nD. oxygen.\\nE. heme groups.\\n---\\n# HOW PROTEINS WORK\\n\\n|6. A prosthetic group is| |\\n|---|---|\\n|A.|a highly charged domain or motif that alters the shape of a protein.|\\n|B.|a nonprotein molecule that is loosely attached to a protein and helps it to perform its function.|\\n|C.|a nonprotein molecule that is tightly bound to a protein and helps it to perform its function.|\\n|D.|a steroid molecule attached to a histidine residue.|\\n|E.|a region of a protein that can accept a phosphate group.|\\n\\n|7. Phosphofructokinase| |\\n|---|---|\\n|A.|produces a product that is inevitably broken down to pyruvate.|\\n|B.|uses ATP as a substrate in its catalytic site.|\\n|C.|binds ATP at a regulatory site.|\\n|D.|works more slowly if ATP concentrations increase.|\\n|E.|all of the above.|\\n\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n1. E. All these statements about shape changes in proteins are true.\\n\\n2. D. Note that this statement is equivalent to “katis the number of molecules of substrate converted to product per molecule of enzyme per unit time” but is not equivalent to the statement in answer A, which is false.\\n\\n3. D. Because the subunits begin at low affinity but switch to the high-affinity form as oxygen binds, the curve for oxygen binding starts out shallow and then gets steeper. Answer E would produce exactly the opposite effect. Of the other answers: (A) is true, but irrelevant—myoglobin also contains heme prosthetic groups, but does not show cooperative behavior. (B) is false: both myoglobin and hemoglobin contain iron. It is chlorophyll, in plants, that uses magnesium. (C) is false: the oxygen affinities of all four subunits are similar and, in any case, a difference in affinity would not cause the binding curve to be S-shaped.\\n\\n4. C. No catalyst, whether inorganic or biological, can alter the equilibrium constant of a reaction. (If one did, it would allow the generation of a perpetual motion machine.)\\n\\n5. B. Concerning the other answers: (A) there are no enzymes that add H+, those groups that can be protonated (e.g., —NH2, histidine residues) accept H+ rapidly without the requirement for any catalyst. (C) Amino acids are not added to proteins once they have been synthesized on the ribosome, although processing of synthesized proteins can often involve the removal of one or more amino acids, as, for example, when the endoplasmic reticulum signal sequence is removed (page 223). (D) Those proteins that do bind oxygen, such as hemoglobin, do so spontaneously, without the need for a catalyst.\\n\\n6. C. Both cofactors and prosthetic groups are nonprotein molecules that help a protein to perform its function, but cofactors are only loosely attached while prosthetic groups are attached tightly.\\n\\n7. E. ATP is a substrate for phosphofructokinase, but it also acts at an independent regulatory site, reducing the activity of the enzyme.\\n---\\nENERGY TRADING WITHIN THE CELL\\n\\nIn the nonliving world complex things degrade naturally to simpler things: Gradients of temperature or concentration disappear, chemical reactions approach equilibrium, and uniformity triumphs. Living things do not appear to follow these trends. Cells are complex and divide to make other complex cells: A fertilized egg differentiates to make a whole complex organism. Living things must obey the laws of thermodynamics. The escape from the behavior of nonliving systems is allowed because living systems take matter and energy from the environment and use it to grow, to reproduce, and to repair themselves. Living systems are open systems while nonliving systems are closed.\\n\\nA chemical reaction that has reached equilibrium can do no work. A good definition of death is the state at which all of the chemical reactions in a cell/organism have reached equilibrium. In a living organism the concentrations of metabolites are often very far from the equilibrium concentrations and yet are more or less constant: This is said to be a steady state. Cells can do this because they are open systems taking energy and matter from their environment.\\n\\nWe can use an analogy with the world of economics. It is unlikely that people would spontaneously repair our houses, or feed us, or give us this book, but we can drive these otherwise unlikely processes by spending money. In a similar way, cells can drive otherwise unlikely processes by using up one of four energy currencies that are then replaced using energy taken from the outside world.\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C 2004 by John Wiley & Sons, Inc.\\n---\\n# ENERGY TRADING WITHIN THE CELL\\n\\nCELLULAR ENERGY CURRENCIES\\n\\nThe scientific way of saying that a process will proceed (although a catalyst may be necessary to achieve a reasonable reaction speed) is to say that the change in Gibbs free energy, expressed as \\x02G, is negative. One reaction that we will meet again is the hydrolysis of glucose-6-phosphate to yield glucose and a phosphate ion:\\n\\nGlucose-6-phosphate + H2O → Glucose + HPO42−\\n\\x02G = −19 kJmol−1\\n\\nSince this reaction has a negative \\x02G, it can proceed, releasing 19 kJ of energy for every mole of glucose-6-phosphate hydrolyzed. Another reaction we will meet again is the hydrolysis of nucleotides, the building blocks of DNA and RNA (Chapter 4). Simply losing the terminal phosphate from the nucleotide ATP releases 30 kJmol−1:\\n\\nAdenosine triphosphate + H2O → Adenosine diphosphate + HPO42− + H+\\n\\x02G = −30 kJmol−1\\n\\nThe reverse of these reactions will of course not proceed. For instance, cells need to phosphorylate glucose to make glucose-6-phosphate but cannot use the reaction\\n\\nGlucose + HPO42− → Glucose-6-phosphate + H2O\\n\\x02G = +19 kJmol−1\\n\\nThe reaction will not proceed because it has a positive \\x02G. Crucially, though, an unfavorable (positive \\x02G) reaction can occur if it is tightly coupled to a second reaction that has a negative free-energy change (negative \\x02G) so the overall change for the reactions put together is negative. Thus cells phosphorylate glucose by carrying out the following reaction:\\n\\nGlucose + Adenosine triphosphate → Glucose-6-phosphate + Adenosine diphosphate + H+ \\x02G = −11 kJmol−1\\n\\nAdenosine triphosphate, or ATP, has given up the energy of its hydrolysis to drive an otherwise energetically unfavorable reaction forward. We call ATP a cellular currency to draw an analogy with money in human society. Just as we can spend money to cause someone to do something they would not otherwise do, such as give us food or build us a house, the cell can spend its energy currency to cause processes that would otherwise not occur. However, the analogy is not exact because energy currencies are not hoarded. There is a continuous turnover of ATP to ADP and back again. ATP is therefore not an energy store but simply a way of linking reactions. It can be thought of as a truck that carries metabolic energy to where it is needed and that returns empty to be refilled. The number of trucks is small but the amount moved can be large. An average person hydrolyzes about 50 kg of ATP per day but makes exactly the same amount from ADP and inorganic phosphate. We will see how this happens in this chapter. The cell has a number of energy currencies of which four—NADH, ATP, the hydrogen ion gradient across the mitochondrial membrane, and the sodium gradient across the plasma membrane—are the most important. We will now discuss each of these in turn.\\n---\\n# CELLULAR ENERGY CURRENCIES\\n\\nReduced Nicotinamide Adenine Dinucleotide (NADH)\\n\\nThis, the most energy rich of the four currencies, is shown in Figure 12.1a. NADH is a strong reducing agent. It will readily react to allow two hydrogen atoms to be added to molecules, in the general reaction NADH + H+ + X → NAD+ + H2X. NAD+ is shown in Figure 12.1b. Addition of hydrogen atoms to molecules, or the removal of oxygen atoms, is called reduction. The opposite of reduction is oxidation, the addition of oxygen. Because oxygen atoms tend to take more than their fair share of electrons in any bonds they make, addition of oxygen means removal of electrons, and vice versa, so the most general definition of reduction is the addition of electrons, with oxidation being defined as the removal of electrons. We will later see NADH acting to reduce complex cell chemicals like pyruvate and acetoacetate. However, when it is acting as an energy currency, NADH simply passes its two hydrogen atoms to oxygen, making water. This releases a lot of energy: Every mole of NADH that is used in this way releases 206 kJ of energy.\\n\\nMedical NAD+, Pellagra, and Chronic Fatigue Syndrome Relevance\\n\\nA small fraction of our total body NAD+ is lost from the body each day. New NAD+ is synthesized from the vitamin niacin, which in turn can be synthesized from the essential amino acid tryptophan. If we don’t get enough niacin or tryptophan, we develop the disease called pellagra. In pellagra parts of our bodies that use a lot of energy, such as the brain, begin to fail. Pellagra killed about 100,000 Americans in the first half of the twentieth century before Joseph Goldberger of the U.S. Public Heath Service showed that it could be prevented and cured by a varied diet. Niacin is now added to all flour, and pellagra is almost unknown in Western countries. Since chronic fatigue syndrome is manifested as a lack of energy, some people have suggested that NAD+ itself, given as a dietary supplement, might reenergize the patients. There is no scientific evidence for this idea, but nevertheless many chronic fatigue syndrome sufferers buy and take NAD+ in the hope that it may help.\\n\\nNucleoside Triphosphates (ATP plus GTP, CTP, TTP, and UTP)\\n\\nAdenosine triphosphate, the second most energy rich of the four currencies, is shown in Figure 12.2. In earlier chapters we met many chemical processes in the cell that are driven by ATP hydrolysis. When one mole of ATP is hydrolyzed, 30 kJ of energy are released. The γ phosphate is easily transferred between nucleotides in reactions such as this one:\\n\\nATP + GDP\\nADP + GTP\\n\\nSo as far as energy is concerned, we can regard GTP, CTP, TTP, and UTP as equivalent to the most commonly used nucleotide energy currency, ATP. Another easy, reversible reaction is this one:\\n\\n2 ADP\\nAMP + ATP\\n\\nin which one ADP transfers its β phosphate to another, so that it itself is left as AMP while converting the other ADP to ATP. This conversion is at equilibrium in cells and is important in maintaining the supply of ATP when ATP is used in reactions. NADH and ATP take part in so many reactions within the cell that they are often called coenzymes, meaning molecules that act as second substrates for many enzymes as they do.\\n---\\n# nicotinamide\\n\\n|H|ribose|O|C|H|ribose|\\n|---|---|---|---|---|---|\\n|adenine|C|N|C|H|N|C|H|+|\\n|N|C|H|C|OH|H|C|OH|NAD|\\n\\n(b)\\n\\nis a strong reducing agent and energy currency. The oxidized form, a\\n\\n# nicotinamide\\n\\n|H|ribose|OH|C|H|ribose|\\n|---|---|---|---|---|---|\\n|adenine|C|N|C|H|N|C|H|NADH|\\n|C|N|O|H|H|O|(a)|\\n\\n).\\n\\nNicotinamide adenine dinucleotide, NADH, shown in (+, is shown in (Figure 12.1.NAD\\n---\\n# CELLULAR ENERGY CURRENCIES\\n\\nFigure 12.2. Adenosine triphosphate, an energy currency.\\n\\nTheir particular jobs. The term is easy to confuse with cofactor (page 249). A cofactor is a\\nchemical species that is loosely associated with an enzyme, helps it carry out its function,\\nand, although it may undergo reactions, ends up in the same state that it began. The concept\\nof coenzymes is very different. Coenzymes are bona fide substrates that are converted to\\nproducts (e.g., NAD+ and ADP) by the enzyme.\\n\\nThe Hydrogen Ion Gradient Across the Mitochondrial Membrane\\nThe endosymbiotic theory states that mitochondria are derived from bacteria that evolved\\nto live in eukaryotic cells (page 12). The bacterial cytosol is usually about 0.6 pH unit more\\nalkaline than the world outside; that is, H+ ions are four times more concentrated outside\\nthan inside. If they could move freely across the bacterial cell membrane, H+ ions would\\nrush in down this gradient. Furthermore, there is a voltage difference across the membrane:\\nThe inside is about 140 mV more negative than the extracellular medium. Transmembrane\\nvoltages are always referred to in terms of the internal voltage relative to that outside: in this\\ncase, −140 mV. The transmembrane voltage attracts the positively charged H+ ions into the\\nbacterium. Any combination of a concentration gradient and a voltage gradient is called an\\nelectrochemical gradient. For hydrogen ions at the bacterial membrane the electrochemical\\ngradient is large and inward. Should H+ ions be allowed to rush into the bacterium, they\\nwould release energy: about 17 kJ for every mole that enters.\\n\\nFigure 12.3 is a representation of a mitochondrion inside a eukaryotic cell. This is\\nnot an accurate picture of what a mitochondrion looks like (see Fig. 3.6 on page 59) but\\nrather emphasizes the topology and the function. In the center is shown the mitochondrion\\nwith its two membranes. In the very middle is a volume equivalent to bacterial\\ncytosol that is called the mitochondrial matrix. Next comes the inner mitochondrial membrane,\\nthen the outer mitochondrial membrane. The intermembrane space is the small space\\nbetween the two mitochondrial membranes. The green region is the cytosol, bound by\\n---\\n# ENERGY TRADING WITHIN THE CELL\\n\\n| |electron transport chain|ATP synthase|ADP/ATP exchanger|Na+/K+ ATPase|\\n|---|---|---|---|---|\\n|HIGH [H+]|12H+| |10H+| |\\n| |3 ATP + NADH + H+| |3 ADP + 3 Pi| |\\n| |H2O + NAD+| |2 ADP| |\\n| |tricarboxylic acid cycle| | |ATP ADP|\\n| | | |3Na+| |\\n| | | |ATP ADP + Pi|2K+ [Na+]|\\n\\nporin plasma membrane outer mitochondrial membrane inner mitochondrial membrane\\n\\nFigure 12.3. Sites in the cell where the energy currencies are interconverted.\\n\\nThe plasma membrane. An integral membrane protein of the outer mitochondrial membrane, called porin, forms a hole or channel that lets through all ions and molecules of Mr < 10,000, so the ionic composition of the intermembrane space is the same as that of cytosol. However, there is a large electrochemical gradient across the inner mitochondrial membrane. When H+ ions move in from the intermembrane space to the mitochondrial matrix, they release 17 kJ mol−1 of energy, exactly as in the mitochondrion’s proposed bacterial ancestors.\\n\\nThe Sodium Gradient Across the Plasma Membrane\\n\\nUnlike bacteria, most eukaryotic cells do not have an H+ electrochemical gradient across their plasma membranes. Rather, it is sodium ions that are more concentrated outside the cell than inside (Fig. 12.3). Typically, the sodium concentration in the cytosol is about 10 mmol liter−1 while the concentration in the extracellular medium is about 150 mmol liter−1. This chemical gradient is supplemented by a voltage gradient. The cytosol is between 70 and 90 mV more negative than the extracellular medium, that is, the transmembrane voltage of the plasma membrane is between −70 and −90 mV. There is therefore a large inward electrochemical gradient for sodium ions. If sodium ions are allowed to rush down this gradient, they release energy—approximately 15 kJ for every mole of Na+ entering the cytosol.\\n---\\nENERGY CURRENCIES ARE INTERCONVERTIBLE\\n\\nA company that buys raw materials in the United States and Mexico, spending dollars and pesos, and then sells products in Europe and Japan, receiving euros and yen, simply converts from euros and yen to dollars and pesos to pay its bills. In the same way, cells convert from the energy currency in which they are in credit to the energy currency they are using up.\\n\\nExchange Mechanisms Convert Between the Four Energy Currencies\\n\\nThe cell has mechanisms that transfer energy between the four currencies. The conversions are summarized in Figure 12.4. In a typical animal cell oxidation of fuel molecules in the mitochondria (by the Krebs cycle, page 283) tops up the supply of NADH. The cell then converts this energy currency into the other three. All the interconversions are reversible. Figure 12.3 shows where each conversion mechanism is located.\\n\\nConsider what happens if a few sodium ions move out of the extracellular fluid and into the cytosol of a eukaryotic cell—for example, when a nerve cell transmits the electrical signal called an action potential (Chapter 15). The sodium gradient has been slightly depleted: the cell holds less of this energy currency than it did before. However, the cell still has plenty of energy in the form of ATP that it can convert into energy as a sodium gradient. It does this using the sodium/potassium ATPase. This protein is located in the plasma membrane. Its function is to move 3Na+ ions out of the cell and to move 2K+ ions into the cell. For this to happen ATP is hydrolyzed to ADP thus giving up energy that is used to push Na+ out of the cytosol to its higher energy state in the extracellular medium.\\n\\nThe cell has now used some ATP, but it still holds plenty of energy in another currency—the H+ gradient across the inner mitochondrial membrane. The enzyme ATP synthase is located in the inner mitochondrial membrane. The protein interconverts the two energy currencies: as H+ ions move into the mitochondrion they give up energy that is then used by ATP synthase to make ATP from ADP and inorganic phosphate.\\n\\nThe energy of the H+ gradient is now depleted. However, the cell still has plenty of energy in its NADH account. The electron transport chain allows interconversion of energy as NADH to energy as H+ gradient. NADH is used to reduce molecular oxygen to high energy H+outside Na+in extracellular medium\\n\\nEnergy flow between pe currencies in a normal animal cell.\\n\\nKrebs cycle\\n\\nNADH\\n\\nNAD+\\n\\nElectron transport chain\\n\\nATP synpase\\n\\nNa+/K+ ATPase\\n\\nADP\\n\\nNa+ in cytosol\\n\\nH+ inside mitochondrion\\n\\nNa+ in extracellular medium\\n---\\n# ENERGY TRADING WITHIN THE CELL\\n\\nWater, releasing energy that is used to push H+ ions out of the mitochondrion to their higher energy state in the cytosol.\\n\\nEach of the three energy conversion systems is a protein structure called a carrier because it carries solute across the membrane. The sodium/potassium ATPase carries sodium and potassium ions, while both ATP synthase and the electron transport chain carry H+ ions. There are many carriers in the cell with a wide variety of functions, some of which we will meet later in this book. The three that convert between the energy currencies are vital and are evolutionarily ancient.\\n\\nExample 12.1 The Fuel That Makes Bacteria Swim\\n\\nAlthough by no means all bacteria are motile, some, such as Escherichia coli and Salmonella typhimurium, can swim. Their small size makes the process very different from swimming as we experience it. If you, the reader, dive into a swimming pool, your momentum will carry you some distance because your mass is great relative to the density of the water. For a bacterium whose mass is extremely small, water is a highly viscous medium. A bacterium swimming in water experiences roughly what a human swimmer in treacle might feel! Despite this, motile bacteria can achieve speeds of up 100 μm s−1 by the use of a structure called the bacterial flagellum. This consists of a rigid helical filament, 20–40 nm in diameter and up to 10 μm long, composed of a single protein, flagellin. The bacterial flagellum operates like a boat’s propeller, pushing the bacterium along as it is turned by the motor at its base. The flagellar motor consists of a series of rings that allow the motor to rotate within the complex layers of membranes and cell wall that make up the bacterial cell surface. Like ATP synthase, the flagellar motor allows H+ ions to flow in across the plasma membrane. The energy released as the H+ ions flow down their electrochemical gradient is used to turn the rotor of the motor and hence the flagellum at up to 100 Hz (=100 times per second).\\n\\n|flagellum|\\n|---|\\n|cell wall|\\n|H+|H+|plasma membrane|\\n---\\n# ENERGY CURRENCIES ARE INTERCONVERTIBLE\\n\\n|high energy|NADH|H + outside mitochondrion|\\n|---|---|---|\\n|low energy|NAD +|H + inside mitochondrion|\\n\\nFigure 12.5. Currency conversion: the electron transport chain converts between NADH and the H + gradient.\\n\\nElectron Transport Chain\\n\\nThe reason we breathe is to supply oxygen to the electron transport chain, which uses the oxygen to oxidize NADH in the overall chemical reaction:\\n\\nNADH + H+ /O↽ NAD+ H O\\n\\nThis reaction releases energy, 206 kJ for every mole of NADH used. The energy is used to carry about 12H + ions from their low-energy state in the mitochondrial matrix to their high-energy state in the cytosol. Figure 12.5 summarizes the reaction in terms of energy currencies. The circle symbolizes the linkage between the energy released in the conversion of NADH to NAD + and the energy used to drive H + out of the mitochondrial matrix. If the electron transport chain simply allowed NADH to reduce oxygen to water, then the reaction’s energy would be released as heat. Instead, the enzymatic function of the electron transport chain is tightly coupled to its function as a carrier that moves H + ions. The energy of NADH is thus converted to the energy of the H + gradient.\\n\\nThe electron transport chain has six components: four large protein complexes, each containing many polypeptide chains, embedded in the inner membrane, plus two mobile electron carriers. The electron transport process has been the subject of intensive investigation for decades and this has led to some rather complex nomenclature. Some aspects of the process remain unclear.\\n\\nElectrons are mostly carried by prosthetic groups built into the protein complexes. We have already seen the heme prosthetic group in hemoglobin and myoglobin. The heme-containing proteins of the electron transport chain are, like hemoglobin, colored red or brown and are called cytochromes from the Latin for cell and color. The iron in a reduced cytochrome is Fe 2+ while in an oxidized cytochrome it is Fe 3+. The exact colors of the cytochromes vary because the proteins in which they are buried provide different environments. These spectral differences are shown by different letters: cytochromes a, b, c with subscript numbers for more subtle differences: cytochrome c 1, a3. Other prosthetic groups are iron–sulfur clusters that vary in detail but are essentially iron ions coordinated to sulfurs in the thiol groups of cysteine side chains or free sulfides. Flavins are also present as electron-carrying prosthetic groups and the final complex also contains copper.\\n\\nThe first of the mobile electron carriers is called ubiquinone or coenzyme Q. This is chemically a quinone (hence “Q”), which is fairly hydrophobic, and its hydrophobicity is increased by a long hydrocarbon tail (Fig. 12.6). It carries both hydrogen ions and electrons.\\n---\\nENERGY TRADING WITHIN THE CELL\\n\\n|Coenzyme Q|CH3O|O|CH3|\\n|---|---|---|---|\\n| |oxidized form| |(quinone)|\\n| |CH3O|H| |\\n| | |O|CH10|Q|\\n| |1H| |1e|\\n| |OH| | |\\n| |CH3O|CH3| |\\n| |reduced by one electron| |(semiquinone)|\\n| |CH3O|H| |\\n| | |O|CH10|QH|•|\\n| |1H| |1e|\\n| |OH| | |\\n| |CH3O|CH3| |\\n| |reduced form| |CH3O|OH|CH10H|QH2|\\n| |Figure 12.6. Coenzyme Q carries two hydrogen atoms.| | |\\n\\nThe second electron carrier is a small protein found in the mitochondrial intermembrane space called cytochrome c. It has a relative molecular mass of 12,270, which is just large enough to be retained by the mitochondrion (the channel porin in the outer membrane passes solute of Mr = 10,000 or less). Cytochrome c is soluble but associates with the inner mitochondrial membrane.\\n\\nWe have said that oxidation is loss of electrons, and reduction gain of electrons. Electron carriers that cycle between the reduced state and the oxidized state normally move either one\\n---\\nENERGY CURRENCIES ARE INTERCONVERTIBLE\\n\\nOverview of the mitochondrial electron transport chain\\n\\n| | | | | | | | | | | | | | |\\n|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\\n| |4H|+| | |4H|+| | |cyto| |cyto|4H|+|\\n| |I| |Q| |II|Q| |III| |C| |CIV| |\\n|NADH| |NAD|+| |succinate| |fumarate| |O2| |H2O|matrix| |\\n\\nFigure 12.7. The electron transport chain comprises four large multimolecular complexes in the inner mitochondrial membrane, three of which are hydrogen ion carriers.\\n\\nor two electrons. NADH and a cofactor called FADH2 each carry two electrons. Reduced cytochromes carry single electrons as do the iron–sulfur clusters. Flavins and coenzyme Q can carry either one or two electrons.\\n\\nFigure 12.7 shows an overview of the electron transport chain. Complex I (sometimes called NADH dehydrogenase or NADH-Q oxidoreductase) accepts electrons from NADH, thus oxidizing it, and uses these to reduce coenzyme Q (Fig. 12.8): In doing so it moves four hydrogen ions outward from the matrix to the mitochondrial intermembrane space. The reaction can be summarized as\\n\\nNADH + Q + 5H+ → NAD+ + QH2 + 4H+ matrix intermem\\n\\nThe transport of the hydrogen ions is accomplished using coenzyme Q as it picks up a hydrogen ion when it picks up an electron. Complex I has a bound coenzyme Q that collects two hydrogen ions from the matrix when it accepts two electrons from iron–sulfur clusters (which received them from a bound flavin which accepted them from NADH). The bound coenzyme QH2 formed then passes its electrons to iron–sulfur clusters, which finally reduce a mobile coenzyme Q, which again collects two hydrogen ions from the matrix. These are released to the intermembrane space when this mobile carrier is reoxidized. The machinery that produces this directionality is not yet clear.\\n\\nComplex II (also known as succinate-Q reductase complex or succinate dehydrogenase) is the only one of the four complexes that is not a hydrogen ion pump. As we will see in Chapter 13 (page 284) complex II is part of the Krebs cycle—it is the enzyme that oxidizes succinate to fumarate using FAD, which becomes FADH2. It passes its electrons to mobile coenzyme Q. Other mitochondrial enzymes that generate FADH2 share this property of being able to reduce coenzyme Q. Because hydrogen ions are not moved across the membrane in this process one gets less ATP formed from the electrons on FADH2 than from those on NADH.\\n\\nComplex III (Q-cytochrome c oxidoreductase or cytochrome reductase) accepts electrons from reduced coenzyme Q (QH2) and uses them to reduce the other mobile electron\\n---\\n# ENERGY TRADING WITHIN THE CELL\\n\\n|Matrix|NADH|I|4H +|\\n|---|---|---|---|\\n|NAD +| | | |\\n| |Q|QH 2| |\\n|Fumarate|FADH2|Q| |\\n|Succinate|FAD|QH 2|Q|\\n| | | | |\\n| |III|4H +| |\\n|Cyto C (oxid)|Cyto C (reduced)| | |\\n| |IV|4H +| |\\n|O2|H2O| | |\\n\\nFigure 12.8. Overview of the operation of the mitochondrial electron transport chain.\\n\\nCarrier cytochrome c. This process results in the movement of four hydrogen ions from the\\nmatrix to the intermembrane space. Complex II marks a change from two electrons being\\ncarried to one. It contains hemes, bound coenzyme Q, and iron–sulfur clusters.\\n\\nCytochrome c moves the electrons to complex IV (cytochrome c oxidase). This protein\\ncomplex reduces oxygen to water and moves four hydrogen ions from the matrix to the\\nintermembrane space. It contains two types of heme and copper ions. Oxygen binds to\\nFe 2+ in a heme just as it binds to the heme in myoglobin or hemoglobin. The overall\\nreaction is\\n\\n4Cyto c reduced+ 8H + + O2 → 4Cyto c oxidized+ 2H2O + 4H +\\nmatrix intermem\\n\\nThe electron transport chain is very similar in all organisms. In eukaryotes the complexes are found in the inner mitochondrial membrane, while in prokaryotes the complexes are found in the plasma membrane.\\n---\\nENERGY CURRENCIES ARE INTERCONVERTIBLE\\n\\n|high energy|H + outside|ATP|\\n|---|---|---|\\n|low energy|H + inside|ADP|\\n\\nFigure 12.9. Currency conversion: ATP synthase interconverts the H + gradient and ATP.\\n\\nATP Synthase\\n\\nThe energy stored in the hydrogen ion gradient can be used to drive the synthesis of ATP from ADP and inorganic phosphate. The conversion is accomplished by a marvelous “nanomachine” called ATP synthase. At its most basic, the operation is a simple one and is summarized in Figure 12.9. Once again, the circle symbolizes the linkage between the enzyme and carrier functions. ATP synthase is a protein complex that is also known as mitochondrial ATPase (because it can carry out the reverse reaction, hydrolyzing ATP and driving H + out of the mitochondria) or F 1 F 0ATPase (because it can be prepared in two fragments F 1 and F 0) while others call it complex V.\\n\\nIn most mitochondria ATP synthase is the only route by which the large electrochemical gradient for H + generated by the electron transport chain can be relieved. If the supply of ADP runs out, then ATP synthase will stop moving H + in. If this happens, the electron transport chain will stop too because it cannot keep pushing H + out against a greater and greater electrochemical gradient. Another way of stopping ATP synthase is to block its operation with the antibiotic oligomycin; again, if this is done, the electron transport chain will stop too.\\n\\nThe coupling between ATP synthesis and electron transport can be broken if hydrogen ions can be carried back across the membrane by an alternative route. Weak organic acids such as 2,4-dinitrophenol can do this. The dinitrophenol is uncharged in the intermembrane space and in this form is sufficiently hydrophobic to diffuse into the membrane. If it enters the matrix, it will tend to lose its hydrogen ion as the hydrogen ion concentration is much lower. The result is movement of hydrogen ions down their concentration gradient. Such molecules are called “uncouplers.” In the presence of an uncoupler electron transport (and so oxidation of fuels) will carry on regardless of the availability of ADP or a working ATP synthase.\\n---\\n# IN DEPTH 12.1 Brown Fat\\n\\nENERGY TRADING WITHIN THE CELL\\n\\nTriacylglycerols are stored within specialized fat cells in the body. Most fat cells are composed of a droplet of lipid surrounded by a thin layer of cytoplasm with a nucleus and a few mitochondria. The resulting tissue is white in color and simply releases or stores fatty acids in response to the needs of the organism. This is the kind of fat that is typically found around our kidneys and under the skin.\\n\\nA second kind of fat is found in babies. Brown fat cells not only have stored triacylglycerols but are also rich in mitochondria, the cytochromes of the mitochondria giving the brown color. Brown fat is a heat-generating tissue. A channel selective for H+ called thermogenin is found in the inner mitochondrial membrane. As fast as the electron transport chain pushes H+ ions out of the mitochondrial matrix, they flow through thermogenin back down their electrochemical gradient into the matrix. In other cells this flux would only occur through ATP synthase and would be tightly coupled to the production of ATP from ADP. The presence of thermogenin uncouples the phosphorylation of ADP from the flow of electrons to oxygen so that the electron transport chain can work flat out even though there is no ADP available, as long as the cell contains triacylglycerols, which can be oxidized to regenerate NADH (see β oxidation, page 290). This generates a lot of heat and helps the infant maintain body temperature. Large blocks of brown fat are found in animals that hibernate.\\n\\nA similar uncoupling mechanism is used by some plants to generate heat. Some arum lilies rely on carrion-eating flies for pollination. Uncoupled mitochondria at the base of the flower generate sufficient heat to evaporate the evil-smelling odorants used to attract the flies.\\n\\nSodium/Potassium ATPase\\n\\nThis is a single protein in the plasma membrane, with a carrier action that is tightly linked to an enzymatic one. Under normal conditions, it hydrolyzes ATP. The energy released drives sodium ions up their electrochemical gradient out of the cell. The Na+/K+ ATPase also moves potassium ions the other way, into the cytosol. For every ATP hydrolyzed, three Na+ ions are moved out and two K+ ions are moved in. Figure 12.10 summarizes the reaction in terms of energy currencies.\\n\\n|high energy|ATP|Na+ in extracellular medium|\\n|---|---|---|\\n|low energy|ADP|Na+ in cytosol|\\n\\nFigure 12.10. Currency conversion: the sodium–potassium ATPase interconverts ATP and the Na+ gradient.\\n---\\n# ENERGY CURRENCIES ARE INTERCONVERTIBLE\\n\\nADP/ATP Exchanger\\n\\nATP synthase makes ATP inside the mitochondrion. For the ATP to be available to the rest of the cell, there needs to be a mechanism to enable it to leave the mitochondrion for use in the cytosol. This job is performed by another carrier, the ADP/ATP exchanger. This protein has no enzymatic action; it simply moves ADP in one direction across the mitochondrial inner membrane and ATP in the opposite direction. In most eukaryotic cells the carrier operates in the direction shown in Figure 12.3. Carriers such as the Na+ /K+ ATPase together with many synthetic processes use up ATP in the cytosol, producing ADP. Then ADP enters the mitochondria by the ADP/ATP exchanger and is reconverted to ATP by ATP synthase. ATP then leaves the mitochondrion with the help of the ADP/ATP exchanger.\\n\\nExample 12.2\\n\\nChemicals That Interfere with Energy Conversion Are Highly Toxic\\n\\nThe electron transport chain, ATP synthase, and the sodium/potassium ATPase together run the energy currency market and are vital to the cell. Chemicals that interfere with them are very toxic. Rotenone, the most widely used rat poison, blocks complex I of the electron transport chain while cyanide blocks complex IV. Digitalis, from foxgloves, blocks the Na+ /K+ ATPase.\\n\\n|rotenone|cyanide|digitalis|\\n|---|---|---|\\n|block electron transport chain|blocks Na+/K+ ATPase| |\\n|high energy|NADH|H+ outside mitochondrion|ATP|Na+ in extracellular medium|\\n|Krebs cycle| | |\\n|low energy|NAD+|H+ inside mitochondrion|ADP|Na+ in cytosol|\\n|Electron transport chain|ATP synthase|Na+/K+ ATPase|\\n\\nPhotosynthesis\\n\\nThe fuels that our cells oxidize to give NADH and FADH2 must of course come from somewhere! Ultimately that somewhere is the fixation of carbon dioxide into more complex molecules, which is in turn driven by light energy captured by photosynthesis. Photosynthesis also generates the oxygen in the atmosphere. Some 1010 tons of carbon are fixed every year.\\n\\nThe dark reactions that fix carbon will be described in Chapter 13 (page 302). However, the overall effect can be summarized as:\\n\\n6CO2 + 18ATP + 12NADPH + 12H2O → C6H12O6 + 18ADP + 18Pi + 12NADP+ + 6H+\\n---\\n# IN DEPTH 12.2 ATP Synthase, Rotary Motor, and Synthetic Machine ENERGY TRADING WITHIN THE CELL\\n\\nATP synthase can be seen in the electron microscope as tiny beads attached to the matrix face of the inner mitochondrial membrane by a thinner region. We now know that this enzyme is a molecular motor. The beadlike head or F1 fragment is attached to proteins that span the membrane—the F0 part.\\n\\nMost of the bulk of F1 is composed of three α and three β subunits arranged alternately. The β subunits bind ATP or ADP and Pi. They are joined to the membrane-bound F0 part by a central stalk composed of γ and ε subunits. The F0 part has 10–14 subunits (called c subunits) forming a ring embedded in the membrane and an a subunit, which binds to the outside of the ring of c subunits. The head of the F1 part is connected to the base by the b subunit, which connects to the a subunit in the membrane.\\n\\nWe have described how the splitting of ATP can drive a conformational change in the protein that has bound it and that this conformational change can produce movement (page 241). The converse of this is true—a conformational change can be produced by movement and this conformational change can drive a reaction.\\n\\nThe stalk of the F1 part (the γ subunit) is not symmetrical. It fits into the center of the circle of alternating α and β subunits and makes different interactions with each β subunit. The β subunits have three conformations possible—a T conformation, which binds ATP very strongly, an L conformation, which binds ADP and Pi but cannot release them, and an O or open conformation, which can release bound nucleotides. The T conformation has been shown experimentally to catalyze the conversion of ADP and Pi to bound ATP with an equilibium constant near 1 (at equilibrium the concentrations of substrates and products are 1 : 1). Rotation of the stalk (γ subunit) alters the conformation of the β subunits (which do not rotate because the b subunit holds the head stationary).\\n\\nImagine that all three β subunits have bound nucleotide. One is in the T conformation, one has the L conformation, and one is O. The T conformation subunit has ADP+Pi interconverting to ATP, and the O and L conformations will have bound ADP and Pi. As the stalk rotates 120 ◦, it converts the T conformation subunit into O so that the ATP can escape and more ADP + Pi can bind. The same movement converts the previous O state subunit into L, trapping the ADP and Pi, and the subunit in the L state converts into T. One third of a rotation generates one ATP, so a full rotation of the stalk generates three as each of the three β subunits cycles from T to O to L and back to T.\\n\\nThis sounds too good to be true! Experiments using the reverse reaction indicate it is true. Isolated F1 parts were attached to a glass surface so the stalk was sticking upward. The γ subunit (the stalk) was linked to a fluorescently labeled actin filament. This could be seen using a fluorescence microscope, and the movies acquired are published on the Web (see CBASC website). Addition of ATP caused the filament to rotate, and it could be demonstrated that the hydrolysis of a single ATP caused a 120 ◦ rotation!\\n\\nThis has not told us how hydrogen ion flow causes the rotation. This remains speculation, but Howard Berg and George Oster have proposed an appealing hypothesis. In the membrane F0 has a ring of c subunits with the single a subunit attached. The a subunit has two half length hydrogen ion channels. Each c subunit has an aspartate residue in the center that is exposed to the hydrophobic interior of the membrane. If this residue has lost its hydrogen ion, the negative charge is very unfavorable in the hydrophobic environment of the membrane interior. Normally this residue will have its hydrogen attached as the uncharged form can be tolerated in the hydrophobic environment of the membrane interior.\\n---\\n# ATP SYNTHASE\\n\\n|matrix|δ|α|β|α|ATP|\\n|---|---|---|---|---|---|\\n|ADP + Pi| |β|α|β|F1|\\n| |H+|b|b|γ|ε|\\n| | |a|c|F0| |\\n| |H+| | | |intermembrane space|\\n\\nThe a subunit makes close contact with two c subunits so their aspartates are exposed to the two half channels. One channel opens toward the matrix and one toward the intermembrane space. The channels provide a hydrophilic environment in which the aspartic acids can lose their hydrogen ions. Once its aspartate has ionized, a c subunit is stuck—it cannot rotate away from the channel on the a subunit. It is much more likely that an aspartate will pick up a hydrogen ion again if it is in contact with the half channel leading to the intermembrane space as the hydrogen ion concentration is around four times higher than it is on the matrix side, and the large transmembrane voltage acts to pull H+ toward the matrix. Similarly a c subunit contacting the half channel leading to the matrix is much more likely to be ionized. Molecular movements ensure that as soon as a c subunit becomes uncharged, it will be able to move into the hydrophobic membrane. As all of the c subunits are linked in a ring, this movement is a rotation of the ring. The adjacent c subunit, which was in contact with the half channel to the matrix, is very likely to have a charged aspartate. Movement of this into the membrane is unfavorable so the only allowed rotation is movement toward the other channel on the a subunit—the one contacting the hydrogen-ion-rich intermembrane space. It is likely to become uncharged and can rotate into the membrane repeating the process. The ring of c subunits acts to carry hydrogen ions from the intermembrane space back to the matrix. The rotation has to be clockwise because of the design of the a subunit, and the hydrogen ion concentration difference across the membrane. The size of the ring seems to vary with the species, but 10 c subunits are found in ATP synthase from yeast. Ten subunits would use 10 hydrogen ions for one rotation, and one rotation generates 3 ATP so 3.33 hydrogen ions are used per ATP generated.\\n\\nThe experiment described above tends to confirm this theory as the rotation of the stalk of the immobilized F1 part is anticlockwise, and this is driven by the hydrolysis of ATP so the rotation should be clockwise for ATP synthesis.\\n\\nThe understanding of how cells generate and interconvert their energy currencies is one of the triumphs of twentieth-century biological science. In 1997 the Nobel prize for chemistry was awarded to Jens Skou, Paul Boyer, and John Walker for working out the structure and mechanism of ATP synthase.\\n\\nThe F0 component of ATP synthase is evolutionarily related to the motor that drives the bacterial flagellum (Example 12.1 on page 264): both generate rotational force from the H+ gradient. Which came first is a matter for speculation.\\n---\\n# ENERGY TRADING WITHIN THE CELL\\n\\nThe process requires 3ATP and 2NADPH for every carbon dioxide fixed. These are regenerated by the light reactions of photosynthesis. In plants both light and dark reactions occur in the chloroplast, which, like the mitochondrion, is thought to have originated through endosymbiosis (page 12). Figure 3.6 (on page 59) shows the structure of a chloroplast. It has an outer membrane that has porins, and so is permeable to many molecules, and an inner membrane that is impermeable. The infolding of the inner membrane has been carried further in chloroplasts than in mitochondria and has produced discrete membrane structures called thylakoids. Within the thylakoids is the thylakoid space or lumen, and outside the thylakoids is the stroma. The dark reactions take place in the stroma and the light reactions in the thylakoid membranes.\\n\\nWhen a molecule absorbs a photon of light of an appropriate wavelength (determined by the electronic structure of the molecule), an electron jumps to a higher energy level. Normally it drops back to its normal level, and the energy of the photon is lost as heat. In photosynthesis, however, this high-energy electron is captured by another molecule and transferred through a chain of electron carriers rather as we have seen in mitochondrial electron transport. This allows hydrogen ions to be pumped across the thylakoid membrane, setting up a hydrogen ion electrochemical gradient with the inside of the thylakoid space having a positive voltage, and a lower pH, compared to the stroma. An ATP synthase, very similar to that found in mitochondria, then allows hydrogen ions to flow back from the thylakoid space into the stroma, driving the synthesis of ATP.\\n\\nLike electron transport in mitochondria, photosynthetic electron transport takes place in large multiprotein complexes that span the thylakoid membrane and uses mobile carriers to shuttle electrons between the complexes. There are actually two different photosystems called photosystem I and photosystem II, which work together. The fundamental pigment involved is chlorophyll. Chlorophyll is rather like heme (Fig. 9.21 on page 206) but differs in detail and importantly has a magnesium ion rather than iron at the center and has a long hydrocarbon tail called phytol. Two types of chlorophyll are present: chlorophyll a and chlorophyll b. Structural differences give them different colors, which means they absorb light of different wavelengths.\\n\\nMost of the chlorophyll molecules are antenna chlorophylls, which act to harvest light and pass the energy to a pair of reaction center chlorophylls. Antenna chlorophylls are supplemented by other pigments such as carotenoids, which absorb different wavelengths of light to enable efficient use of most of the light energy reaching the plant.\\n\\nLet us start with the reaction center chlorophylls in PSII. These chlorophylls absorb light at 680 nm and so the reaction center is called P680. As soon as an electron in one of these is raised to a higher energy (either by energy transfer from antenna chlorophylls or by direct light absorption), it is immediately captured by an adjacent molecule (called pheophytin). This generates a charge pair: the chlorophyll is +ve and the pheophytin −ve. The complex structure prevents the charges neutralizing by rapidly transferring the electron from pheophytin to a quinone called plastoquinone, which is very similar in structure to ubiquinone (coenzyme Q) in mitochondria (Fig 12.6). This leaves a positively charged reaction center chlorophyll P680+. Also at the reaction center is a cluster of four manganese ions together with bound water. An electron is abstracted from the bound water—this is done four times resulting in oxidation of two molecules of water to give a molecule of O2. The net result is a movement of electrons from water to plastoquinone, which is reduced to plastoquinol (QH2). During this process hydrogen ions are pumped from the stroma into the thylakoid space.\\n---\\n# ENERGY CURRENCIES ARE INTERCONVERTIBLE\\n\\nPlastoquinol passes its electrons to a copper protein called plastocyanin. This process is carried out by a large membrane protein complex called cytochrome bf, which is analogous to complex III in the mitochondrial electron transport chain. The other photosystem now comes into play. Here again we have a pair of reaction center chlorophyll a molecules called P700 (as they absorb light of wavelength 700 nm) at the center of a complex of other chlorophyll molecules and accessory pigments. The electron lifted to a higher energy level is captured by another chlorophyll and immediately passed to plastoquinone. This leaves a +ve charge at the reaction centre (P700+). This charge is neutralized by an electron from plastocyanin, which came from photosystem II.\\n\\nThe captured electron passes from plastoquinone to an iron–sulfur cluster, which in turn passes electrons to a protein called ferredoxin. Ferredoxin can pass electrons to an enzyme called ferredoxin-NADP+ reductase, which reduces NADP+ to NADPH. The process has used light energy to oxidize water to oxygen and to drive electrons through a series of carriers to produce NADPH and the movement of hydrogen ions from the stroma to the thylakoid space. This hydrogen ion gradient is used by ATP synthase to make ATP from ADP and inorganic phosphate. The ATP and NADPH are used in the dark reactions to fix carbon dioxide into complex molecules.\\n\\nIf there is no NADP+ available to accept the electrons from ferredoxin, photosynthesis would have to stop were it not for an alternate route. Ferredoxin can also pass its electrons back to the cytochrome bf complex, resulting in a cyclic flow of electrons. As the electrons pass along the chain of carriers, hydrogen ions are pumped into the thylakoid space. This process, which allows ATP synthesis without production of either NADPH or oxygen, is called cyclic photophosphorylation.\\n\\n# All Carriers Can Change Direction\\n\\nIn a normal animal cell, the primary source of energy is the Krebs cycle (page 283). This regenerates NADH from NAD+, making at the same time a small amount of ATP. Because the NADH currency is always being topped up, while the others are being used, the direction of operation of the energy conversion systems is usually that shown in Figure 12.3. However, all the carriers are reversible.\\n\\nYeast cells in a wine barrel, or muscle cells in the leg of a sprinter, are anaerobic; there is no oxygen available. In this situation cells can make NADH, but the electron transport chain cannot drive H+ out of the mitochondria because there is no molecular oxygen waiting to be reduced by NADH. Instead, the cell’s energy needs are met by anaerobic glycolysis (page 286), which makes ATP. Figure 12.11 shows how the cell maintains the amounts of energy currencies. Any drain on the mitochondrial H+ gradient is counteracted by ATP synthase running in the opposite direction from that shown in Figure 12.3. ATP is hydrolyzed and H+ ions are pushed out of the mitochondrion. The ADP/ATP exchanger also reverses its direction. ATP is regenerated by anaerobic respiration in the cytosol and is used up by ATP synthase in the mitochondrial matrix.\\n\\nChloroplasts do not contain the enzymes of the Krebs cycle, so at night they have no internal source of energy and are dependent on their host cell. ATP from the cytosol (most of it created by mitochondria) enters the chloroplast by the ADP/ATP exchanger, which is carrying ADP and ATP in the opposite direction from that shown in Figure 12.3. ATP synthase hydrolyzes ATP and maintains the H+ gradient.\\n---\\n# ENERGY TRADING WITHIN THE CELL\\n\\nAnaerobic glycolysis\\n\\n|high energy|H+ outside mitochondrion|Na +in extracellular medium|\\n|---|---|---|\\n|low energy|H + inside mitochondrion|ADP Na +in cytosol|\\n|electron transport chain|ATP synthase|Na + /K+ ATPase|\\n\\ndoes not function in absence of molecular oxygen\\n\\nFigure 12.11. Energy flow between the currencies in an anaerobic cell.\\n\\nHealthy cells maintain themselves in a steady state in which none of the energy currencies are allowed to run down. The direction in which energy moves between the four currencies depends on the primary source of energy for that cell.\\n\\n# IN DEPTH 12.3 Can It Happen? The Concept of Free Energy\\n\\nWe know from experience that some chemical reactions give out heat (the burning of organic matter for instance) while others absorb heat (so heat is used to make the reaction happen). The study of how energy affects matter and particularly how it affects chemical reactions is called thermodynamics. A vast amount of research in the late nineteenth and early twentieth centuries led to the development of this complex field. To make a very long story short, a process results in a change in two parameters: heat and entropy. Heat we are familiar with and understand as the random motion of molecules, but entropy may be unfamiliar. Entropy is the degree of disorder in a system: Entropy always tends to increase. The melting of ice is favorable because it results in an increase in entropy as the ordered, crystalline ice becomes more random water.\\n\\nA physical process can absorb or emit heat and/or result in an increase or decrease in entropy. If we wish to know whether a process can occur or not, we need to consider both. The concept of free energy was formulated by the American physical chemist J. Willard Gibbs in 1878. It is now called the Gibbs free energy and given the symbol G in his honour. The change in free energy in a process, \\x02G (\\x02 is the Greek upper-case delta used to denote a change) is\\n\\n\\x02G = \\x02H − T \\x02S\\n\\nWhere \\x02H is the heat change (called enthalpy) in the process, T is the absolute temperature in degrees Kelvin, and \\x02S is the change in entropy. If \\x02G for a process is negative, it will proceed, although the rate may be very slow in the absence of a catalyst. If it is not negative, the process cannot occur.\\n\\nUnfortunately, we can only measure changes and not absolute values for G, H, and S in chemical processes. This difficulty is avoided by defining standard\\n---\\n# ENERGY CURRENCIES ARE INTERCONVERTIBLE\\n\\nstates so we can make comparisons. A free-energy change in a process under standard conditions is denoted \\x02G◦. Biochemists use a slightly different standard state, which is in water at pH 7, and this is shown as \\x02G◦′.\\n\\nThe free-energy change is related to the equilibrium constant for a reaction. Consider the reaction\\n\\nA + B ⇀ C + D\\n↽\\n\\nK= ([C][D])/([A][B])eq\\n\\nAnd\\n\\n\\x02G = \\x02G◦′ + RT ln{([C][D])/([A][B])}\\n\\nwhere R is the gas constant and T the absolute temperature. At equilibrium \\x02G = 0 so\\n\\n\\x02G◦′= −RT ln K′eq\\n\\nwhere K′eq is the equilibrium constant under standard conditions. Conditions in the cell are not usually the standard ones, but if we know \\x02G◦′ for a reaction and the relevant concentrations, we can calculate \\x02G for the reaction. For this book we have calculated \\x02G values under human cellular conditions by using the following reasonable concentration values:\\n\\n[ATP] = 6 mmol liter−1\\n[ADP] = 0.6 mmol liter−1\\n[AMP] = 0.2 mmol liter−1\\n[Pi] = 4.5 mmol liter−1\\nGlucose = 250 μmol liter−1\\nGlucose-6-phosphate = 10 mmol liter−1\\n[NAD +] / [NADH] = 4 in pe mitochondrial matrix\\n[O2] = 21 μ mol liter−1\\nTemperature 37 C◦\\n\\nFor example, we have used the hexokinase reaction (page 258) to illustrate coupling a reaction with a negative free-energy change with one which has a positive free energy change.\\n\\nGlucose + ATP → Glucose-6-phosphate + ADP + H+\\n\\nUnder biochemical standard conditions this reaction has a \\x02G◦ of −36 kJmol−1. However, inside cells the concentration of glucose-6-phosphate is much higher than the concentration of unphosphorylated glucose; when this and other concentration effects are taken into account, the calculated overall free-energy change is −11 kJmol−1: The reaction will certainly still proceed but is not as strongly favored as the \\x02G◦′ would suggest.\\n\\nThe \\x02G values we have given for the four energy currencies are those that apply in an aerobic animal cell, as shown in Figure 12.3, where each of the conversion reactions has a negative \\x02G, as it must if it is to proceed as shown. Under conditions where one or more of the conversion reactions is operating in the opposite directions, the concentrations, and therefore \\x02G values, will of course be different.\\n---\\n# ENERGY TRADING WITHIN THE CELL\\n\\n# SUMMARY\\n\\n1. Reactions with a positive Gibbs free-energy change (ΔG) can be caused to happen in a cell by linking them with a second reaction with a larger, negative ΔG. The second reaction drives the first.\\n\\n2. The majority of such reactions in the cell are driven by one of four energy currencies—NADH, ATP, the hydrogen ion gradient across the inner mitochondrial membrane, and the sodium gradient across the plasma membrane.\\n\\n3. The electron transport chain in the mitochondrial inner membrane converts between energy as NADH and energy in the hydrogen ion gradient.\\n\\n4. ATP synthase in the mitochondrial inner membrane converts between energy in the hydrogen ion gradient and energy as ATP.\\n\\n5. The sodium/potassium ATPase in the plasma membrane converts between energy in ATP and energy in the sodium gradient.\\n\\n6. In a healthy cell, none of the energy currencies are allowed to run down. The direction of energy exchange between the four currencies depends on the primary source of energy available to the cell.\\n\\n# FURTHER READING\\n\\nVoet, D., and Voet, J. D. 2003. Biochemistry, 3rd ed. New York: Wiley\\n\\n# REVIEW QUESTIONS\\n\\nFor each question, choose the ONE BEST answer or completion.\\n\\n1. 1. A reaction that has a positive ΔG will\\n\\nA. proceed, although the rate may be very slow in the absence of an enzyme.\\n\\nB. only proceed in the presence of an enzyme.\\n\\nC. generate water.\\n\\nD. not proceed, even if an enzyme is present.\\n\\nE. be coupled to a carrier.\\n2. 2. Which of the following is not a cellular energy currency?\\n\\nA. The sodium gradient across the plasma membrane\\n\\nB. Glucose\\n\\nC. ATP\\n\\nD. NADH\\n\\nE. The H+ gradient across the mitochondrial inner membrane\\n3. 3. Which of these statements about the proteins responsible for converting between the four energy currencies is not correct?\\n\\nA. They are all reversible.\\n\\nB. They are all located on the mitochondrial inner membrane.\\n---\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n|4.|Hydrogen ions that enter the mitochondrial matrix through ATP synthase release a lot|\\n|---|---|\\n| |of energy because|\\n|A.|the mitochondrial matrix is electrically negative with respect to the cytosol.|\\n|B.|H + is at a higher concentration in the mitochondrial matrix than in the cytosol.|\\n|C.|the mitochondrial inner membrane contains thermogenin.|\\n|D.|H + is at a higher concentration in the cytosol than in the mitochondrial matrix.|\\n|E.|There is a large electrochemical gradient favoring H + entry.|\\n\\n|5.|The overall reaction catalyzed by the electron transport chain is|\\n|---|---|\\n|A.|Glucose + ATP → Glucose-6-phosphate + ADP|\\n|B.|ATP + GDP ⇀ ADP + GTP|\\n|C.|NADH + H + ↽+ 1/2O2 → NAD + + H 2O|\\n|D.|ATP + H 2O → ADP + H 2PO 4−|\\n|E.|NADH + OH − + ATP → NAD + H + + ADP + HPO 4 2−|\\n\\n|6.|Which of the following statements is incorrect?|\\n|---|---|\\n|A.|Mitochondria may have been derived in evolution from a prokaryote.|\\n|B.|Mitochondria have their own DNA.|\\n|C.|Mitochondria have a large negative voltage across their inner membrane.|\\n|D.|Mitochondria increase their generation of ATP in cells starved of oxygen.|\\n|E.|Mitochondria generate NADH.|\\n\\n|7.|The electron transport chain in mitochondria|\\n|---|---|\\n|A.|accepts electrons from NADH, thus oxidizing it to NAD + .|\\n|B.|passes electrons to oxygen, thus reducing it to water.|\\n|C.|uses the hydrophobic molecule ubiquinone to carry electrons between integral membrane protein|\\n| |complexes.|\\n|D.|uses the hydrophilic molecule cytochrome c to carry electrons between integral membrane|\\n| |protein complexes.|\\n|E.|all of the above.|\\n\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n|1.|D. Reactions with negative \\x02G can proceed; those with positive \\x02G cannot proceed. Enzymes|\\n|---|---|\\n| |are catalysts and can therefore speed up the rate of a reaction, but they cannot cause a reaction to|\\n| |proceed if the \\x02G is positive.|\\n\\n|2.|B. Although glucose is the major extracellular fuel supplied by the blood to our cells, it is not|\\n|---|---|\\n| |used as an energy currency inside cells. Rather, on entering cells energy is supplied to glucose to|\\n| |convert it to glucose-6-phosphate.|\\n\\n3.\\nB. The sodium/potassium ATPase is located on the plasma membrane.\\n\\n|4.|E. Answers B and C are false: H + is at a lower concentration in the mitochondrial matrix than|\\n|---|---|\\n| |in the cytosol, and thermogenin (an alternative route for H + entry expressed in a few cell types,|\\n| |especially brown fat) will not act to increase the energy released by H + as it enters. Of answers|\\n| |A, D, and E the last is the best answer because it states the complete source of the energy. It is|\\n| |true that hydrogen ions that enter the mitochondrial matrix through ATP synthase release energy|\\n| |because the mitochondrial matrix is electrically negative with respect to the cytosol, but this is not|\\n---\\n# ENERGY TRADING WITHIN THE CELL\\n\\nThe only source of energy—the concentration gradient also favors H+ entry. Similarly, it is true that hydrogen ions that enter the mitochondrial matrix through ATP synthase release energy because H+ is at a higher concentration in the cytosol than in the mitochondrial matrix, but this is not the only source of energy—the electrical gradient also favors H+ entry.\\n\\n1. Answers A, B, and D are reactions that occur somewhere in the cell, but they are not the reaction catalyzed by the electron transport chain. Answer E is a nonsense equation; in particular, there is no compound NAD as written, and nicotine adenine dinucleotide is always found as NADH or as NAD+. Note that answers A, B, D, and E involve ATP: ATP is not a substrate or product of the electron transport chain.\\n2. Under anaerobic conditions, mitochondria cannot generate ATP and are dependent on the rest of the cell for an ATP supply. In an anaerobic animal or yeast cell, the ATP is generated by glycolysis.\\n3. All of answers A through D are true. In particular, while ubiquinone and cytochrome c both function as electron carriers, ubiquinone is hydrophobic and remains within the lipid bilayer, while cytochrome c is a hydrophilic molecule found in the aqueous intermembrane space.\\n---\\n# METABOLISM\\n\\nIn Chapter 12 we described the energy currencies NADH and ATP. In this chapter we will describe the chemical pathways that regenerate these currencies when their levels are depleted. We will then consider some other important chemical pathways, some that operate in all cells, others that are found only in certain types of organism or in specialized biochemical centers like the liver. Figure 13.1 is an overview of the main metabolic pathways within a cell.\\n\\nAll the processes that occur within a living cell are ultimately driven by energy taken from the outside world. Green plants and some bacteria take energy directly from sunlight. Other organisms take compounds made using sunlight and break them down to release energy, a process called catabolism. The most common way of breaking down these food compounds is to oxidize them, that is, to burn them but in a controlled way. The energy trapped in energy currencies can then be used for the building, repair, and homeostatic processes termed anabolism. The collective term for all of the reactions going on inside a cell is metabolism. All metabolic reactions share some general features:\\n\\n- They are catalyzed by enzymes.\\n- They are universal in that all organisms show remarkable similarity in the main pathways.\\n- They involve relatively few types of chemical reaction.\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright © 2004 by John Wiley & Sons, Inc.\\n---\\n# METABOLISM\\n\\n| |glucose|glycogen|\\n|---|---|---|\\n|amino acids|glycolysis|gluconeogenesis|\\n|alanine|pyruvate| |\\n|only| |fats|\\n| |fatty acid|synthesis|\\n| |acetyl CoA|fatty acids|\\n| | |β-oxidation|\\n|NADH|Krebs cycle|CO 2|\\n|(FADH2)| | |\\n|NAD+| |CO 2|\\n\\nFigure 13.1. Overview of metabolism.\\n\\n- They are controlled, often by modulation of key regulatory enzymes.\\n- They are compartmentalized within cells. In eukaryotes different sets of metabolic reactions are carried out in different organelles. In animals and plants this compartmentalization is carried further, so that in some cases different reactions take place in different body organs. Prokaryotes too show compartmentalization of a simpler sort—some processes are associated with regions on the inner face of the cell membrane.\\n- They usually involve coenzymes, molecules that are second substrates in a number of different reactions.\\n- The pathways that break particular molecules down are different from those used to synthesize them. This allows them to be controlled separately.\\n\\nMolecules that are common second substrates for a number of reactions are called coenzymes. These are either energy currencies themselves (such as NADH and ATP) or carriers of chemical groups such as coenzyme A. ATP is used to drive other reactions by transfer of a phosphate group. The resultant ADP can be rephosphorylated back to ATP. The energy to do this can be derived from NADH passing its electrons to the electron transport chain, which converts the NADH back to its oxidized form NAD+. The total amount of ATP + ADP in a cell is relatively constant, but the proportion that is ATP can vary. Similarly the amount of NADH + NAD+ is constant, but the proportion present as NADH can vary.\\n---\\n# THE KREBS CYCLE: THE CENTRAL SWITCHING YARD OF METABOLISM\\n\\nThe fuels we take in in the diet are mainly fats, proteins, and carbohydrates. At the very center of metabolism is a cycle of reactions that takes place in the mitochondrial matrix. The cycle is named after its discoverer, Hans Krebs, and is also known as the tricarboxylic acid (TCA) cycle or the citric acid cycle. The foods we eat are converted to the two-carbon unit acetate, CH3COO−. The acetate is not free but is carried by a coenzyme called coenzyme A. Acetate bound to coenzyme A—acetyl-CoA for short—is then fed into the Krebs cycle and may be completely oxidized to carbon dioxide and water. In the process the energy currency NADH is produced. The Krebs cycle is central to carbohydrate, fat, and amino acid metabolism.\\n\\nWe shall first describe the Krebs cycle and then look at the other pathways that interact with it. The reactions are shown in Figure 13.2.\\n\\n| |O| |CH3 C CoA| |CoA|\\n|---|---|---|---|---|---|\\n|acetyl CoA| | | | |COO|\\n| | |1| |CH2| |\\n| |HO| |C COO| |CH2|\\n|COO|C O| | | |COO|\\n|NADH| | |citrate| |CH2|\\n|NAD| | | | | |\\n| |HO|C H| | |COO|\\n|malate|CH2| | | | |\\n| |COO|oxaloacetate| | | |\\n| | | | | | |\\n|7| | | | | |\\n| | | | | | |\\n|COO| | | | |CH2|\\n|fumarate|CH| | | | |\\n|HC| |6| | | |\\n|COO| | | | | |\\n| |COO| | | |CoA|\\n|succinate| | | | | |\\n| | | | | | |\\n| | | | | | |\\n| | | | | | |\\n| | | | | | |\\n| | | | | | |\\n| | | | | | |\\n| | | | | | |\\n\\nFigure 13.2. The Krebs cycle.\\n---\\n# Metabolism\\n\\n1. A molecule of acetyl-CoA enters the cycle, and the acetate (two carbons) is combined with the four-carbon molecule oxaloacetate, making citrate (as carboxyl groups are ionized at the pH in the cell, we normally speak of them as ions; so we say “citrate” rather than citric acid).\\n\\n2. Citrate is rearranged to isocitrate.\\n\\n3. In the first oxidation step isocitrate is oxidized to 2-oxoglutarate (sometimes called α-ketoglutarate). A carbon is lost as CO2 and NAD+ is reduced to NADH.\\n\\n4. A second oxidation converts 2-oxoglutarate to succinyl-CoA. A second carbon leaves as CO2 and again NAD+ is reduced to NADH. Note that the product is attached to coenzyme A. This reaction is catalyzed by oxoglutarate dehydrogenase.\\n\\n5. The bond between succinate and CoA is now broken and the energy released used to drive the phosphorylation of a GDP to GTP. γ-phosphate groups can be swapped between nucleotides, so this GTP can be used to regenerate ATP from ADP.\\n\\n6. Succinate is oxidized to fumarate. The oxidant in this case is flavin adenine dinucleotide (FAD) and not NAD+ so an FADH2 is produced. FADH2, the reduced form of FAD, does not carry as much energy as NADH but like NADH is used to drive H+ up its electrochemical gradient out of the mitochondrial matrix. The enzyme involved is succinate dehydrogenase, which is actually part of the electron transport chain.\\n\\n7. Water is added to the double bond in fumarate making malate.\\n\\n8. Malate is oxidized to oxaloacetate in a reaction catalyzed by malate dehydrogenase. One NAD+ is reduced to NADH. The starting compound oxaloacetate has been regenerated and is ready to accept another acetyl group to start another turn of the cycle.\\n\\nThe reactions of the Krebs cycle can be summarized as:\\n\\nCH3CO-CoA + (3 NAD+ + FAD) + GDP + Pi + 3H2O\\n→\\nCoA-H + (3 NADH + FADH2) + GTP + 2CO2 + 3H2\\nwhere Pi represents an inorganic phosphate ion.\\n\\nFrom Glucose to Pyruvate: Glycolysis\\n\\nGlucose is an important fuel for most organisms, and glycolysis is the main pathway that enables it to be used. The word glycolysis simply means the breakdown of glucose. It is an ancient pathway that can function without oxygen and indeed is thought to have evolved before there was much oxygen in the atmosphere. The pathway is present in almost all cells and takes place in the cytosol. It is shown in Figure 13.3.\\n\\n1. Free glucose is phosphorylated on carbon 6 to produce glucose-6-phosphate. As we discussed earlier, this reaction, catalyzed by hexokinase, is driven by the free energy available from ATP.\\n\\n2. Glucose-6-phosphate is isomerized to fructose 6-phosphate.\\n---\\n# FROM GLUCOSE TO PYRUVATE: GLYCOLYSIS\\n\\n|glucose|catalyzed by hexokinase|glucose-6-phosphate|fructose-6-phosphate|\\n|---|---|---|---|\\n|ATP| |ADP| |\\n\\n|triose phosphate|catalyzed by triose phosphate isomerase|glyceraldehyde-3-phosphate|fructose-1,6-biphosphate|\\n|---|---|---|---|\\n|dihydroxyacetone phosphate (DHAP)|catalyzed by glyceraldehyde 3-phosphate dehydrogenase|NADH|PiP6|\\n\\n1,3-bisphosphoglycerate\\n3-phosphoglycerate\\n\\n|pyruvate|phosphoenolpyruvate|2-phosphoglycerate|\\n|---|---|---|\\n|ATP| |ADP| |\\n\\nFigure 13.3. Glycolysis breaks glucose down into pyruvate.\\n\\n3. Fructose-6-phosphate is phosphorylated to produce fructose-1,6-bisphosphate. Again ATP is used and the reaction is carried out by phosphofructokinase. This reaction commits the sugar to being broken down and used to provide energy rather than used for other purposes. Phosphofructokinase is regulated allosterically, being inhibited by ATP.\\n\\n4. The fructose-1,6-bisphosphate is now split into two halves. Each of the halves has a phosphate attached. The two products are dihydroxyacetone phosphate and glyceraldehyde-3-phosphate. The enzyme is aldolase.\\n---\\n# Metabolism\\n\\n5. Triose phosphate isomerase interconverts dihydroxyacetone phosphate and glyceraldehyde-3-phosphate. This allows both halves of the original glucose to be used.\\n\\nThe glucose-6-phosphate has now been converted into two molecules of glyceraldehyde-3-phosphate. Each of the following reactions occurs twice for each molecule of glucose fed into the pathway.\\n\\n6. Glyceraldehyde-3-phosphate is oxidized in a reaction that also attaches one inorganic phosphate ion to give 1,3-bisphosphoglycerate. NAD+ is the oxidant and a NADH is produced. The enzyme is called glyceraldehyde-3-phosphate dehydrogenase.\\n\\n7. One of the phosphate groups from bisphosphoglycerate is transferred to ADP. This is a process called substrate-level phosphorylation. ATP has been made from ADP by a single enzyme, without any involvement of ATP synthase. We are left with 3-phosphoglycerate.\\n\\n8. After a rearrangement to 2-phosphoglycerate, water is removed leaving phosphoenolpyruvate.\\n\\n9. In another substrate-level phosphorylation phosphoenolpyruvate transfers its phosphate group to ADP leaving pyruvate.\\n\\nFigure 13.4 illustrates the various ways that the cell can use pyruvate. If it is to be used to make fatty acids or to be oxidized in the Krebs cycle, it is carried into the mitochondrial matrix. Here it is oxidized and decarboxylated by the complex enzyme pyruvate dehydrogenase (Fig. 13.4a). NAD+ is used as the oxidant, producing NADH. Coenzyme A is also added. The product is therefore acetyl-CoA, which may enter the Krebs cycle or can be converted to fatty acids or some other molecules. Overall glycolysis has used up two ATP molecules but has produced four: a net gain of 2 ATP per glucose.\\n\\nThe Krebs cycle is limited by the availability of oxaloacetate. This may run low as some biosynthetic pathways use one or the other component of the Krebs cycle as their starting material. If this happens new oxaloacetate is made from pyruvate (Fig. 13.4b). Pyruvate may also be converted to the amino acid alanine in a transamination reaction (Fig. 13.4c; Fig. 11.8 on page 250).\\n\\nGlycolysis Without Oxygen\\n\\nThe leg muscles of a sprinter cannot be supplied with oxygen rapidly enough to supply the mitochondria. Muscle cells need to make ATP by a method that does not require oxygen. As we have seen glycolysis itself produces two ATPs per glucose. Can the cell let the pyruvate pile up and use it when oxygen becomes available again? No, because glycolysis as far as pyruvate also converts one NAD+ to NADH. If this was all that happened, the cell would quickly convert all its NAD+ to NADH, and glycolysis would stop as the cell does not have oxygen available for the mitochondria to oxidize the NADH back to NAD+. To solve this problem cells reduce pyruvate to lactate (Fig. 13.4d) and in doing so regenerate the NAD+ needed to allow glycolysis to carry on. The buildup of lactic acid in poorly\\n---\\n# FROM GLUCOSE TO PYRUVATE: GLYCOLYSIS\\n\\n|pyruvate|CO2|oxaloacetate|\\n|---|---|---|\\n|pyruvate|(c)|alanine|\\n|pyruvate| |lactate|\\n|pyruvate|acetaldehyde|ethanol|\\n\\nFigure 13.4. Pyruvate can be used in a number of ways.\\n\\nOxygenated muscles is thought to cause the pain of cramp. The blood carries the lactate\\nto the liver where it is reoxidized to pyruvate. When we stop using the muscle, the blood\\ncan supply more oxygen to the muscle and the need for lactate production abates. Red\\nblood cells lack mitochondria and are entirely dependent on glycolysis for their energy\\nneeds.\\n---\\n# METABOLISM\\n\\nMedical Sleeping Sickness\\n\\nRelevance On page 259 we discussed chronic fatigue syndrome, a problem of unknown cause characteristic of Western countries. In contrast sleeping sickness, caused by a parasite called Trypanosoma brucei, is a serious disease in Africa. Trypanosomes are single-celled eukaryotes with a complex life cycle, part of which is spent in the human bloodstream. Here they reveal a prodigious appetite for glucose. Each tiny parasite can consume its own body weight in glucose every hour. The reason for the parasite’s insatiable demand for glucose is twofold. First, trypanosomes dispense with mitochondrial ATP production and rely on anaerobic respiration to generate their ATP, so they only make 2 ATPs per glucose rather than the 30 or so that they could if their mitochondria were still working. Second, trypanosomes do not have any glycogen or fat stores and are therefore entirely dependent on a continuous supply of glucose in the host blood. The enormous consumption of glucose by the parasites leaves little for the host, who is overcome by extreme langor such that even keeping the eyes open is an unsurmountable effort.\\n\\nSome microorganisms—particularly yeasts—regenerate their NAD+ in a different way. Pyruvate is first decarboxylated to acetaldehyde (also called ethanal) and then reduced to ethanol by alcohol dehydrogenase, which uses NADH and regenerates NAD+. One molecule of CO2 is also produced (Fig. 13.4e).\\n\\nExample 13.1 Anaerobes Good and Bad The fact that yeasts regenerate their NAD+ by making the gas CO2 together with ethanol has been utilized in breadmaking, brewing, and winemaking since prehistoric times. Other microorganisms are like our muscles and generate lactate under anaerobic conditions. These too have their uses in the food industry: yogurts, many cheeses, sauerkraut, and dill pickles all rely on lactic acid released in anaerobic respiration. On the minus side, some food-spoiling bacteria can only function when there is no oxygen. These obligate anaerobes include the deadly Clostridium botulinum (page 232).\\n\\nGlycogen Can Provide Glucose for Glycolysis The polysaccharide glycogen (page 30) is used as a store of glucose particularly in liver and muscle cells. We saw in Chapter 2 how the glycosidic bond can be hydrolyzed with the broken ends of the bond being sealed with groups from a water molecule, so that a hydrogen atom is added to one side of the broken bond and a hydroxyl group is added to the other. The enzyme glycogen phosphorylase specifically breaks the α(1→4) glycosidic bond in glycogen but seals the broken ends with groups from inorganic phosphate, so that a hydrogen atom is added to one side of the broken bond and a phosphate group is added to the freed glucose monomer (Fig. 13.5). The resulting glucose-1-phosphate is readily converted to glucose-6-phosphate for glycolysis. Breaking up glycogen this way is more energy efficient than simply hydrolyzing it (as happens in the intestine) since the ATP that would otherwise be required to make glucose-6-phosphate from free glucose is saved. The occasional α(1→6) links that attach side arms to the glycogen chain are broken by other enzymes.\\n\\nOne of the many important roles of the liver is to maintain the level of glucose in the blood—it is the most important of the circulating fuels and is the primary fuel for red\\n---\\n# FROM GLUCOSE TO PYRUVATE: GLYCOLYSIS\\n\\nGlycogen stores in the liver can provide glucose when none is available from the gut. The glucose carrier (page 238) allows glucose to enter and leave the liver cells but cannot transport phosphorylated sugars, so glucose-6-phosphate must be converted to free glucose for transport out into the extracellular medium and from there to the blood. To do this, liver has the enzyme glucose-6-phosphate phosphatase, which removes the phosphate group. Muscle also has stores of glycogen, but these are for its own use: it does not have glucose-6-phosphate phosphatase so cannot release glucose into the extracellular medium.\\n\\nGlucose May Be Oxidized to Produce Pentose Sugars\\n\\nCells need the five-carbon sugar ribose to manufacture nucleotides. This is made from glucose in an oxidative pathway called the pentose phosphate pathway (Fig. 13.6). This pathway also provides reducing power for biosynthetic reactions in the form of NADPH. Glucose-6-phosphate undergoes two oxidations generating a molecule of a pentose phosphate, CO2, and two molecules of NADPH. Like glycolysis, these reactions occur in the cytoplasm. The pentose phosphate pathway interacts with glycolysis, and this allows it to perform a different function. In a cell that does not need lots of ribose, but does need lots of NADPH for biosynthesis, the pentose is recycled by being combined with glycolytic intermediates. Going around the loop six times converts a glucose entirely to 6CO2, giving 12 NADPH for biosynthesis. Although the pentose phosphate pathway breaks up sugar to give CO2 and the strong reducing agent NADPH, it is not used for energy production: only in the mitochondria, where NADH can be used to power the electron transport chain, can reducing agents be converted to other energy currencies.\\n---\\n# METABOLISM\\n\\n|glucose 6-phosphate|ribulose 5-phosphate + CO2|\\n|---|---|\\n|fructose-6-phosphate|ribose-5-phosphate|\\n|fructose-1,6-bis phosphate|interconversions|\\n|dihydroxy- acetone phosphate|glyceraldehyde-3-phosphate|\\n| |pyruvate|\\n\\nFigure 13.6. The reactions of the pentose phosphate pathway are shown in black with the reactions of glycolysis in green.\\n\\n# FROM FATS TO ACETYL-CoA: β OXIDATION\\n\\nCells must be able to break down fatty acids (page 39) and use the energy released, whether the fatty acids come from the lipolysis of triacylglycerols in fat droplets or triacylglycerols and phospholipids in food. Before they can be broken down, the fatty acids are coupled to coenzyme A to give an acyl-CoA in a reaction driven by the conversion of ATP to AMP. A spiral of reactions in the mitochondrial matrix called β oxidation (Fig. 13.7) then oxidizes the fatty acyl-CoA. Each turn of the spiral shortens the fatty acyl chain by two carbons, releasing an acetyl-CoA and generating an NADH and an FADH2 for each acetyl-CoA. The acetyl-CoA may be oxidized in the Krebs cycle or converted to other molecules.\\n\\nDuring fasting, fat cells supply fuels for other parts of the body. Because fats are insoluble and fatty acids have only a limited solubility, fat cells convert fatty acids into soluble, circulating fuels called ketone bodies (Fig. 13.8) (the word ketone means any chemical containing a carbon atom with single bonds to two other carbons and a double bond to an oxygen). The fundamental ketone body is acetoacetate, which the liver synthesizes from acetyl-CoA. Acetoacetate is then reduced to 3-hydroxybutyrate. These two molecules are important circulating fuels in mammals. Heart muscle, for instance, prefers ketone bodies to glucose as a fuel source.\\n---\\n# FROM FATS TO ACETYL-CoA: β OXIDATION\\n\\n|O|R|CH|CH|CH|C|CoA|fatty acid|\\n|---|---|---|---|---|---|---|---|\\n|2|2|2| | | | | |\\n|O| |R|CH|CH|CH|C|CoA|\\n|2| | | | | | |FAD|\\n| | | | | | | |FADH2 oxidation|\\n|O| |R|CH|CH|CH|C|CoA|\\n|2| | | | | | |H2O hydration|\\n|OH| | | | | | |O|\\n|R|CH|2|CH|CH|2|C|CoA|\\n| | | | | | | |NAD|\\n| | | | | | | |NADH oxidation|\\n|O| | | | | | |O|\\n|R|CH|2|C|CH|2|C|CoA|\\n| | | | | | | |CoA cleavage by CoA|\\n|O| | | | | | |O|\\n|R|CH|2|C|CH|2|C|CoA|\\n| | | | | | | |fatty acid chain acetyl CoA shortened by 2 carbons|\\n\\nFigure 13.7. ß oxidation of fatty acids produces acetyl-CoA.\\n\\n|CH3| |CH3|\\n|---|---|---|\\n|C|O| |NADH| |H|C|OH|\\n|H| |C|H| |NAD|+|H|\\n|COO| | | | |COO|\\n|acetoacetate| | | | |3-hydroxy butyrate|\\n\\nFigure 13.8. Formation of ketone bodies from acetyl-CoA.\\n---\\n# AMINO ACIDS AS ANOTHER SOURCE OF METABOLIC ENERGY\\n\\nProtein forms a considerable part of the animal diet, even in vegetarians. It is broken down to free amino acids during digestion. These amino acids can be used for the biosynthesis of new proteins in the cell, but those in excess of this need can serve as metabolic fuels. To begin this process, the amino groups are removed in a process called transamination by aminotransferases such as alanine glyoxylate aminotransferase (page 249). The resulting carbon skeletons are then converted to intermediates in the Krebs cycle or to acetyl-CoA. Since there are 20 amino acids, there are many different steps, but the overall effect is that the amino groups are passed to oxaloacetate or 2-oxoglutarate to form aspartate and glutamate, respectively. The amino groups on glutamate and aspartate are then converted to urea (see below) for excretion.\\n\\n# IN DEPTH 13.1 The Urea Cycle—The First Metabolic Cycle Discovered\\n\\nDietary amino acids that are not required for protein synthesis have their amino groups passed to oxaloacetate or 2-oxoglutarate to form aspartate and glutamate, respectively. The amino groups on each of these amino acids are then used in the liver to make urea, which is excreted in the urine. The first step is the release of the amino group as a free ammonium ion.\\n\\n| |NADH|H|\\n|---|---|---|\\n| | |glutamate|2-oxoglutarate|\\n| |COO|COO|\\n| |CH2|CH2|\\n| |CH2|CH2|\\n|OOC|CH|NH3| |OOC|C|O|\\n| |+| | | |+| |\\n|H2O| | | |H|N|H|\\n| | | | | |H| |\\n| |NAD|(a)| | |ammonium|\\n\\nThis ammonium ion, plus the amino groups on aspartate, is then used to make urea in the urea cycle, which operates only in the liver. First, the ammonium ion is converted to carbamoyl phosphate. This is then combined with the α-amino acid ornithine to produce citrulline. In turn, citrulline is joined with aspartate, which carries a second nitrogen into the cycle, producing argininosuccinate. Argininosuccinate is now cleaved to release fumarate and arginine. Urea is removed from arginine to regenerate ornithine, which is now ready for another cycle. Urea can then be excreted in the urine, eliminating two nitrogen atoms per urea molecule.\\n---\\nAMINO ACIDS AS ANOTHER SOURCE OF METABOLIC ENERGY\\n\\nThe overall cycle converts two ATP to ADP and one ATP to AMP. This is energetically equivalent to a total of four ATP to ADP conversions, since a fourp ATP is required to turn pe AMP into ADP in pe reaction:\\nATP + AMP ← 2ADP→\\n\\nThe fumarate can enter the Krebs cycle and be converted to malate, which is oxidized to oxaloacetate, which can (among other things) be transaminated back to aspartate to carry in another nitrogen.\\n\\n| |COO| |COO| |COO|\\n|---|---|---|---|---|---|\\n|H|2ATP 2ADP Pi|aspartate|CH2|fumarate HC| |\\n|H|N| |NH2|OOC|CH NH3|\\n|ammonium|H| |C O| |C NH2|\\n|+|O| |O| |H N|\\n| |O P O|NH2|CH2 arginino-| | |\\n|carbon C O| |O| | | |\\n|dioxide| | | | | |\\n|O|carbamoyl| | | | |\\n|phosphate| | | | | |\\n|NH3| | | | | |\\n|CH2|ATP| | | | |\\n| | |AMP| | | |\\n|ornithine|CH2| | | | |\\n| |OOC|CH NH3| | | |\\n| | | |citrulline| | |\\n|OOC|CH NH3| | | | |\\n| | |arginine| | | |\\n| | | | | | |\\n| | | | | | |\\n|urea|NH2| | | | |\\n| |O C NH2| | | | |\\n|(b)| | | | | |\\n\\nOrnithine plays the same role in the urea cycle as oxaloacetate in the Krebs cycle: it accepts the incoming molecule, undergoes a series of interconversions, and is regenerated allowing the cycle to begin again. The availability of ornithine determines the rate at which the cycle can operate. Arginine is a protein amino acid and is normally present in the diet as a source of ornithine. Vice versa the urea cycle can make arginine so this is not normally considered to be an essential amino acid for adults. Arginine is, however, essential in the diet of growing children as the net protein synthesis of growth needs more of it than can be supplied from the urea cycle without draining too much and so impeding the cycle.\\n\\nSince the urea cycle only occurs in the liver, liver failure results in a buildup of ammonium ions in the blood. This is toxic to nerve cells, producing first mental confusion and finally coma and death.\\n---\\n# METABOLISM\\n\\nIt is interesting to note that of the three major energy sources in the diet—carbohydrate, fat, and protein—only protein is not used as an energy store in our bodies. We make specific proteins when we need them but never simply as a way of storing amino acids. Of course, if one eats a lot of protein, one puts on weight: once the amino acids have been converted to carbon skeletons or to acetyl-CoA, these can then be used to make glucose and hence glycogen (page 298) or to make fat (page 300).\\n\\n✎ Medical Phenylketonuria Relevance\\n\\nAll the phenylalanine in the diet that is not required for protein synthesis is converted to tyrosine by the enzyme phenylalanine hydroxylase. If this enzyme is missing or defective through mutation, then there is a serious problem. The standard way the body deals with amino acids is for aminotransferases (page 249) to transfer the NH3+ group to aspartate or glutamate, which are then processed by the urea cycle (In Depth 13.1 on page 292). However, the product formed when phenylalanine has its NH3+ group transferred away, a phenylketone called phenylpyruvate, cannot be further metabolized. Both phenylalanine and phenylpyruvate therefore accumulate in the body. About 1 in 20,000 new-born babies have this defect, which is called phenylketonuria because the phenylpyruvate appears in the urine.\\n\\nPHENYLALANINE HYDROXYLASE\\n\\n|Phenylalanine|Tyrosine|\\n|---|---|\\n|OOC CH NH3|OOC CH NH3|\\n\\nTRANSAMINATION\\n\\nPhenylpyruvate\\n---\\n# MAKING GLUCOSE: GLUCONEOGENESIS\\n\\nThe disease is a devastating one, and, if left untreated, babies of only a few weeks old begin to suffer severe neurological damage. All newborn babies in the United States and in most other developed countries are therefore tested for phenylketonuria within the first week of life. This represents the largest genetic screening program carried out by the medical profession. The test, often called the heel-prick test or more correctly the Guthrie test after the scientist who developed it, is relatively simple. A drop of blood taken from a baby’s heel is dried onto a small filter paper disk. Disks from hundreds of infants can be tested at the same time by placing their disks onto an agar plate containing bacteria that require phenylalanine for growth. If the bacteria grow, then the baby is at risk from phenylketonuria and blood from the baby will be retested a few days later to ensure the infant does indeed suffer from phenylketonuria. The Guthrie test is simple and cost-effective and can test for phenylketonuria irrespective of the mutation in DNA that has caused the problem.\\n\\nIf detected in the first weeks of life the prognosis for the patient is good. Affected infants are fed a strict diet that provides just enough phenylalanine for protein synthesis but no more. This treatment, carried on to maturity, is very effective and individuals develop normally.\\n\\n# MAKING GLUCOSE: GLUCONEOGENESIS\\n\\nSuch is the importance of glucose that there is a pathway for its synthesis from other molecules. Gluconeogenesis enables animals to maintain their glucose levels even during starvation. It makes use of some of the enzymes of glycolysis but uses different enzymes to bypass the steps that are not freely reversible. Figure 13.9 shows the glycolytic pathway in green. The new reactions that allow the entire pathway to run in reverse are shown in black.\\n\\nNew step 1: Pyruvate is carboxylated to make oxaloacetate (a reaction that also serves to “top-up” levels of oxaloacetate for the Krebs cycle). The oxaloacetate is moved from the mitochondria into the cytosol where it is converted to phosphoenolpyruvate and CO2. The source of the phosphate group is GTP, which is converted to GDP.\\n\\nFrom phosphoenolpyruvate all of the reactions are reversible until fructose bisphosphate is reached. However, the reaction catalyzed by phosphofructokinase is not reversible and this step is avoided by:\\n\\nNew step 2: One of the phosphoester bonds on fructose bisphosphate is hydrolyzed by a phosphatase.\\n\\nThe interconversion of fructose-6-phosphate and glucose-6-phosphate is easily reversible. The final production of free glucose is accomplished by:\\n\\nNew step 3: The other phosphoester bond is hydrolyzed by glucose-6-phosphate phosphatase.\\n\\nGluconeogenesis is an expensive process: The conversion of 2 pyruvates to a glucose molecule uses 4 ATP, 2 GTP, and 2 NADH.\\n\\nDifferent compounds can be fed into the gluconeogenesis pathway as appropriate. Glutamate is converted to 2-oxoglutarate (page 292), fed into the Krebs cycle, and tapped off as oxaloacetate to feed gluconeogenesis. Lactate and alanine are converted to pyruvate and hence oxaloacetate. Glycerol, released from lipids by hydrolysis, is phosphorylated and then oxidized to generate dihydroxyacetone phosphate. However, animals cannot make glucose from fats. Figures 13.1 and 13.2 (pages 282 and 283) might suggest that they could: use β oxidation to make acetyl CoA, feed the acetyl-CoA into the Krebs cycle, and tap off\\n---\\n# METABOLISM\\n\\n|CH OH|O|P|\\n|---|---|---|\\n|2| | |\\n|HO OH|OOH OH|ATP|ADP|\\n|glucose|+ H O|glucose-6-phosphate|fructose-6-phosphate|\\n|3| |2|Pi|ATP|\\n|P2P|Pi|P2|ADP|\\n\\nH2 O + PP OCH CH O PP 2 O 2\\n\\nO = fructose-1,6-biphosphate\\n\\nCH O PP H O 2 CC\\n\\nC O + HCC OH\\n\\nCH OH CH O PP\\n\\n2\\n\\ndihydroxyacetone glyceraldehyde\\n\\nphosphate (DHAP) -3-phosphate\\n\\nPi NAD + NADH\\n\\n|O CC O|O CC O|ATP ADP|O CC O PP|\\n|---|---|---|---|\\n|HC|HCC OH| | |\\n|C O PP| | | |\\n|CH OH|CH 2O PP|ATP ADP| |\\n|2-phosphoglycerate|3-phosphoglycerate|1,3-bisphosphoglycerate| |\\n\\nH O 2\\n\\nO O ADP ATP O O ATP ADP CCOO\\n\\nCC CC CC\\n\\nCC O PP CC O P1 + CO 2 CH 2\\n\\nCH 2 CH 3 COO\\n\\nphosphoenolpyruvate pyruvate oxaloacetate\\n\\n1 P\\n\\nCO 2 GDP GTP\\n\\nFigure 13.9. Gluconeogenesis allows glucose to be made from pyruvate.\\n---\\nMAKING GLUCOSE: GLUCONEOGENESIS 297\\n\\noxaloacetate to feed gluconeogenesis. However, in passing around the Krebs cycle from citrate to oxaloacetate, the two carbons delivered by the acetyl-CoA have already been lost as CO2.\\n\\nIN DEPTH 13.2 The Glyoxylate Shunt\\n\\nMany plants store triacylglycerol oils in their seeds. We have emphasized that mammals cannot make glucose from acetyl-CoA as two carbon dioxides are lost for every two carbons on acetyl-CoA that enter the Krebs cycle. Plants, however, can convert acetyl-CoA into glucose. Special organelles called glyoxysomes carry out a series of reactions called the glyoxylate pathway or glyoxylate shunt, which effectively short-circuits the Krebs cycle.\\n\\nAs in the Krebs cycle acetyl-CoA combines with oxaloacetate to give citrate, which is converted to isocitrate. However, rather than undergoing oxidation and loss of CO2, the isocitrate is split to give succinate and the two carbon glyoxylate. Glyoxylate is then combined with another acetyl-CoA to yield malate, which is oxidized to oxaloacetate. In parallel the succinate is also converted to oxaloacetate using the standard Krebs cycle reactions. The overall effect of one complete cycle is therefore to generate one oxaloacetate that can enter gluconeogenesis (page 295).\\n\\n|oxaloacetate|CH3 C CoA CoA COO|citrate|\\n|---|---|---|\\n| |COO CH2|COO CH2|\\n| |COO COO CH2|COO H C COO|\\n| |HO C H CH3 C CoA HO C H isocitrate|malate CH2 CoA COO|\\n| |COO H C O| |\\n|fumarate|CH COO glyoxylate|succinate|\\n| |HC CH2| |\\n| |COO CH2| |\\n---\\n# METABOLISM\\n\\nMany bacteria can also carry out these reactions. Indeed, some bacteria can grow on acetate as the sole carbon source because they are able to combine free acetate with coenzyme A to form acetyl-CoA, which is then fed into the glyoxylate pathway as required.\\n\\n# MAKING GLYCOGEN: GLYCOGENESIS\\n\\nGlucose is stored as the polymer glycogen. Glucose polymerization, as a stand-alone reaction, has a positive ΔG and will not occur. The synthesis of glycogen is therefore driven by the hydrolysis of nucleoside triphosphates—not only ATP but also UTP. If phosphorylated glucose is not available, then it is made by hexokinase using ATP (page 284). Glucose-1-phosphate then reacts with UTP to make UDP-glucose. Glycogen synthase then transfers glucose from the UDP to the growing glycogen chain. Other enzymes insert the α(1→6) branches at intervals.\\n\\nCH2OHO OH O OO OO O O N HO O P O + O PP OO PP O P O CH2 glucose-1-OH O O O O O uridine phosphate triphosphate\\nCH2OH HO OH O O HP2O7 H2O 2Pi OHO OP O HN O O OP OCH2OO N UDP-glucose\\n\\nFigure 13.10. Uridine diphosphate glucose is synthesized from UTP and glucose-1-phosphate.\\n---\\nMAKING GLYCOGEN: GLYCOGENESIS\\n\\n|CH2OH| |CH2OH| |CH2OH|\\n|---|---|---|---|---|\\n| |H|C|O|H| | |H|C|O|H| | |H|C|O|H|\\n| | |H| | | | |H| | | | | | |H| | |\\n|C|OH|C|H|CC|OO|OO|C|OH|H|C|O|C|OH|H|C|O|\\n|OH|C|C|OO|PP|OO|PP|uridineuridine|OH|C|C| | |C| |C|\\n| |H|OH|O|O| | | |H|OH| | | |H|OH|\\n| |UDP-glucose| | | | | |glycogen (n residues)|\\n\\n|CH2OH| |CH2OH| |CH2OH|\\n|---|---|---|---|---|\\n| |O|O| | | |H|C|O|H| | | |H|C|O|H|\\n|O|P|O|P|O|uridine|C|H| |C| |C|H| |C|\\n| | | |O| | | | | | | | | | | |\\n| |O|O| | | |H|C|O|H| | | |H|C|O|\\n| | | |OH|H| | | |OH| | | | |OH| | |\\n| |UDP| | | | | | |glycogen (n +1 residues)|\\n\\nGlycogen synthesized from UDP-glucose monomers.\\n\\nFigure 13.11. Glycogen is synthesized from UDP-glucose monomers.\\n\\nExample 13.2 Diabetes, Starvation, Ketone Bodies, and the Odor of Sanctity\\n\\nDuring starvation the only source of glucose is gluconeogenesis, which must use body protein amino acids as a source of precursors. It is clearly far better to use fat reserves than to degrade body protein, so those tissues that can shift to using ketone bodies as their main fuel source do so. Even the brain changes over to get three quarters of its energy needs from ketone bodies. Fat stores can keep a starving human being going for weeks.\\n\\nDiabetes mellitus is characterized by a similar shift to using ketone bodies. Diabetes arises when either no insulin is produced (Type 1) or cells are unable to respond to this hormone (Type 2). In both cases the body switches to a starvation type metabolism and the ketone bodies 3-hydroxybutyrate and acetoacetate are made to excess. Acetoacetate is chemically unstable and slowly loses carbon dioxide to form acetone. When ketone bodies are at a high concentration, sufficient acetone is present to give the breath the fruity smell of acetone. This pathological condition is described as ketosis and is characteristic of untreated diabetes.\\n\\nMedieval saints were given to mortification of the body by voluntary starvation. Perhaps the so-called odor of sanctity was simply acetone on the breath of starving and thus ketotic saints?\\n\\n|CH3| |CH3|\\n|---|---|---|\\n|C|O|H| |C|O|acetone|\\n|H|C|H|\\n|COO| | |\\n|acetoacetate| | |\\n| |CO2| |\\n---\\n# MAKING FATTY ACIDS AND GLYCERIDES\\n\\nAll cells need fatty acids for membrane lipids. Fat cells make large amounts of fat (triacylglycerols) in times of plenty. The basic machinery is a multienzyme complex (in bacteria) or a multidomain protein (in eukaryotes) that uses the substrate acetyl-CoA. In both cases the growing fatty acid chain is not released: it swivels from enzyme to enzyme or domain to domain in the array, adding two carbons for each complete cycle until the limiting length of 16 carbons is reached: The product, palmitic acid, is then released. Although the reactions look similar (Fig. 13.12), the process is not a reversal of β oxidation (page 290). It uses entirely different enzymes, takes place in the cytosol rather than in the mitochondria, and is separately regulated. Like much of biosynthesis it is reductive, and the reducing power comes not from NADH but from the closely related dinucleotide NADPH.\\n\\nInitially acetyl-CoA is carboxylated to malonyl-CoA. From here on, however, fatty acid synthesis does not use free coenzyme A to carry the growing chain but instead uses a protein called acyl carrier protein (ACP). The malonyl residue is transferred to ACP from malonyl-CoA. This condenses with a molecule of acetyl-ACP (made from an acetyl-CoA) to give a four-carbon molecule with the release of ACP and CO2. The four-carbon acetoacetyl-ACP is next reduced to hydroxybutyryl-ACP. The next enzyme (molecule or domain) removes water, leaving a double bond, which is again reduced to give butyryl-CoA. Another malonyl-CoA is condensed with this and the cycle continues. Finally a chain 16 carbons long has been made (palmitic acid). At this point it is hydrolyzed from the ACP. Overall 14 NADPH molecules, 1 acetyl-CoA, and 7 malonyl-CoA molecules have been used to make palmitic acid. Palmitic acid is then used by enzymes on the endoplasmic reticulum that extend the chains and that can introduce double bonds. Mammals cannot, however, synthesize all the different kinds of fatty acids that they need for their membranes and must obtain essential fatty acids in food (page 42).\\n\\nThe main use of fatty acids is to make glycerides, both triacylglycerols for storage in fat globules and phospholipids for membranes. The process uses glycerol phosphate, which is usually generated by reduction of dihydroxyacetone phosphate. The fatty acid is then swopped in, replacing the phosphate group, which leaves as inorganic phosphate.\\n\\nExample 13.3\\nEating Well, Getting Fat\\n\\nGlycerol phosphate, the building block from which lipids and phospholipids are made, is obtained in most cells by reducing dihydroxyacetone phosphate. This in turn is generated in the glycolytic pathway. This means that adipose cells can only make fats when glucose is abundant—when we are eating lots of sugar or carbohydrate. In contrast, liver cells have an enzyme called glycerol kinase that phosphorylates glycerol to make glycerol phosphate directly, so that some lipids and phospholipids can be made even during times of fasting.\\n\\n# SYNTHESIS OF AMINO ACIDS\\n\\nNitrogen is an important constituent of proteins and nucleic acids and many other molecules important in cells. Although nitrogen gas is plentiful, making up 80% of the atmosphere, it is inert. It is chemically a very difficult task to break the triple bond and reduce nitrogen gas to ammonia that can be used for incorporation into biomolecules. Chemical fertilizers are made using the Haber process, which fixes nitrogen by the use of pressures of around\\n---\\n# SYNTHESIS OF AMINO ACIDS\\n\\n|O| |O| |O|\\n|---|---|---|---|---|\\n|CH3|C|CoA| |+| |HCO3|\\n|acetyl CoA| |bicarbonate| |ATP| |ADP| |O| |malonyl CoA|\\n\\n|O| | | |O|\\n|---|---|---|---|---|\\n|CH3|C|CoA| |+| |ACP|\\n|acetyl CoA| | | |acyl carrier| | | |CH3|C|ACP| |+| |CoA|\\n|O| | | |O|\\n|C|CH2|C|CoA| |+| |ACP|\\n|O| |malonyl CoA| | | |acyl carrier|\\n| | | | |protein| | | |O| | | |O|\\n\\n|O| | | |O|\\n|---|---|---|---|---|\\n|CH3|C|ACP| |+| |C|CH2|C|ACP|\\n|acetyl ACP| | | |O| |malonyl ACP|\\n|CO2| | | |ACP| |condensation|\\n\\n|O| | | |O|\\n|---|---|---|---|---|\\n|CH| |CH| |C|CH|C|ACP|\\n|acetyl ACP| | | |O| |malonyl ACP|\\n| | | | |NADPH| |reduction|\\n| | | | |NADP+|\\n\\n|OH| | | |O|\\n|---|---|---|---|---|\\n|CH| |CH| |CH|CH|CH|C|ACP|\\n|acetyl ACP| | | |O| |dehydration|\\n| | | | |H2O|\\n\\n|O| | | |O|\\n|---|---|---|---|---|\\n|CH3|CH2|CH2|CH|C|ACP|\\n| | | | |NADPH| |reduction|\\n| | | | |NADP+|\\n\\nCH3CH2CH2CACP\\n\\nprocess repeats\\n\\nto produce a chain\\n\\n+ CCH2CAC\\n\\n16 carbons long\\n\\nFigure 13.12. Synthesis of fatty acids.\\n---\\n# METABOLISM\\n\\n300 atmospheres and temperatures of 500 °C. Some nitrogen is fixed naturally by lightning, but most is fixed by various types of prokaryotes, which possess the complex enzyme called nitrogenase. Some of these nitrogen-fixing organisms are free-living while others form symbiotic relationships with plants: the root nodules of legumes are a good example of plants generating a special environment for their nitrogen-fixing symbionts.\\n\\nNitrogenase consists of two protein complexes: a reductase and an iron–molybdenum protein. Both proteins have iron–sulfur clusters. The reductase accepts electrons from donors such as ferredoxin (page 275), which can get its electrons through photosynthetic electron transport or from other reactions. The reductase passes electrons to the iron–molybdenum protein (also called dinitrogenase). Nitrogen gas is bound to the iron–molybdenum cofactor where it is reduced to ammonia. Ammonia in water forms ammonium ions NH4+, which can be incorporated into the amino acid glutamine and from there into other amino acids and other molecules.\\n\\nThe overall reaction is\\n\\n|N|+ 8e− + 8 H+ + 16 ATP → 2NH3 + H2 + 16 ADP + 16 Pi|\\n|---|---|\\n| |where e− represents an electron.|\\n\\nNitrogenase is inhibited by oxygen. Plants with nitrogen-fixing symbiotic bacteria have evolved methods of restricting the oxygen concentration in the vicinity of the bacteria. For example, legumes (the pea family) surround the bacteria with cells that produce a molecule called leghaemoglobin, which is very similar to myoglobin (page 239). The leghaemoglobin binds oxygen, preventing it from reaching the nitrogenase.\\n\\nGiven ammonium ions, plants and bacteria can synthesize all 20 amino acids. Animals are more limited and must obtain some amino acids from their diet. Amino transferases (page 249) allow animals to move amino groups from an amino acid to an oxo-acid to generate a new amino acid, but there are some carbon skeletons that cannot be synthesized. Adult humans require histidine, isoleucine, leucine, lysine, methionine, phenylalanine, threonine, tryptophan, and valine in their diet—these are the essential amino acids. Adult humans are in nitrogen balance—we excrete the same amount of nitrogen as we take in. Growing infants, however, have a net uptake of nitrogen: they take in more nitrogen than they excrete. Growth clearly demands more amino acids, and in this case the limited ability we have to synthesize arginine is insufficient and so it must be present in the diet as well.\\n\\nOther molecules, such as the nucleic acid bases, are synthesized from amino acid starting materials.\\n\\n# CARBON FIXATION IN PLANTS\\n\\nWe have already described how chloroplasts use the energy of light to oxidize H2O to give O2 and to generate ATP and NADPH (page 271). A series of so-called dark reactions in the chloroplast stroma uses ATP and NADPH in a process that grabs carbon dioxide from the air and builds it into larger molecules, a process called carbon fixation. These reactions are often called the Calvin cycle in honor of their discoverer (Fig. 13.13). The initial step is carried out by the enzyme ribulose bisphosphate carboxylase, said to be the most abundant protein on earth. CO2 is combined with the five-carbon sugar ribulose bisphosphate to give a six-carbon intermediate that immediately splits into two molecules of 3-phosphoglycerate. Further reactions use NADPH and ATP to convert these three-carbon\\n---\\n# CONTROL OF ENERGY PRODUCTION\\n\\n|ribulose-1,5-bisphosphate|2ATP|\\n|---|---|\\n|ATP|2NADPH|\\n|ribulose-5-phosphate|2 × glyceraldehyde-3-phosphate|\\n|fructose-6-phosphate|complex sugar intraconversions|\\n\\nFigure 13.13. The dark reactions of photosynthesis capture carbon dioxide from the air.\\n\\nIt is interesting that many of these reactions are similar to those of the pentose phosphate pathway (page 289).\\n\\n# CONTROL OF ENERGY PRODUCTION\\n\\nFeedback and Feedforward\\nWe have seen how an energy currency that runs low is topped up by conversion from another currency. This is not enough to ensure a constant energy supply, however. Therefore the cell has more mechanisms that ensure that the supply of cellular energy is accelerated or slowed as appropriate. These mechanisms are of two types called feedforward and feedback. We will introduce the terms by analogy with real money. Consider a bank teller. During the day people deposit checks but draw out cash to spend. As time passes the stock of banknotes and change in the till gets low. The teller signals to the supervisor, who opens the bank vault, takes out more cash, and refills the teller’s till. This is an example of negative feedback. In general negative feedback is said to occur when a change in some parameter activates a mechanism that reverses the change in that parameter. We have already met an analogous negative feedback system in the control of tryptophan biosynthesis (page 116). A downward change in the concentration of tryptophan in the bacterial cell activates the mechanism that causes the cell to make more tryptophan. Positive feedback is less common in both biology.\\n---\\n# Metabolism\\n\\nPhosphofructokinase is regulated by the binding of ATP or AMP at a regulatory site that is separate from the active site. Binding of ATP inhibits while binding of AMP activates.\\n\\nPositive feedback is said to occur when a change in some parameter activates a mechanism that accelerates the change. In banking, this occurs when a rumor starts that a bank is about to fail. Biological examples of positive feedback are unusual. Quorum sensing in luminescent bacteria (page 116) and the action potential (page 325) are two examples of positive feedback that we describe in this book.\\n\\nFeedforward control also exists, where preparations are made for a future event. An example in banking is stocking up cash before the lunchtime rush. This concept is also present in biological systems.\\n\\n# Negative Feedback Control of Glycolysis\\n\\nPhosphofructokinase catalyzes the first irreversible step in glycolysis and is allosterically regulated by ATP. High ATP concentrations lock the enzyme into an inactive conformation, while reduced ATP levels lead to AMP binding, activating the enzyme. This process is a negative feedback mechanism.\\n\\n# Feedforward Control in Muscle Cells\\n\\nWhen a signal is sent from the brain to the muscles, calcium ions are released into the cytosol, activating processes that cause muscle contraction using ATP energy. This is an example of feedforward control in muscle cells.\\n---\\n# CONTROL OF ENERGY PRODUCTION\\n\\n|cAMP|active|\\n|---|---|\\n|inactive protein kinase A|protein kinase A|\\n|calmodulin|inactive glycogen|\\n|Ca ++|phosphorylase kinase|\\n| |phosphorylation of glycogen|\\n| |phosphorylase|\\n| |phosphorylase kinase|\\n|active glycogen phosphorylase kinase|inactive glycogen phosphorylase|\\n|phosphorylation of glycogen phosphorylase| |\\n| |active glycogen phosphorylase|\\n| |glucose|\\n| |glucose + O−|\\n|glucose|to glycolysis and|\\n|glucose|therefore ATP production|\\n|glucose| |\\n\\nFigure 13.15. Calcium and cyclic-AMP both activate glycogen breakdown in muscle and liver.\\n\\nAt the same time, other calcium ions pass through a channel into the mitochondrial matrix, attracted by the large negative voltage of the mitochondrion interior. Once there, calcium activates three key enzymes: pyruvate dehydrogenase, oxoglutarate dehydrogenase, and malate dehydrogenase (pages 286, 284). The cell has not waited until the ATP concentration starts to decrease before activating the Krebs cycle in the mitochondria, so this is feedforward control. Meanwhile, in the cytosol, calcium ions bind to the protein calmodulin (page 201), which in turn binds to and activates an enzyme called glycogen phosphorylase kinase (Fig. 13.15). Glycogen phosphorylase kinase is only able to phosphorylate one target, glycogen phosphorylase. Glycogen phosphorylase is activated by phosphorylation and proceeds to break down glycogen to release glucose-1-phosphate, which is fed into the glycolytic pathway. The cell has not waited until glucose concentration falls before activating glycogen breakdown, so this is feedforward control.\\n\\nIn fact, muscles can begin to break down glycogen even before the message goes out from the brain to tell them to contract. When the brain realizes that we are in a dangerous situation and might be going to have to run, it causes the release of the hormone adrenaline from the adrenal glands above the kidneys. Adrenaline binds to an integral membrane protein of the skeletal muscle cells called the β-adrenergic receptor. This causes the production of the intracellular messenger cyclic AMP (cAMP) within the cytosol of the muscle cell. This topic is dealt with in more detail in Chapter 16. cAMP then activates cAMP-dependent protein kinase, which is given the short name of protein kinase A (Fig. 13.15). Glycogen phosphorylase kinase, the enzyme that phosphorylates glycogen phosphorylase, is itself phosphorylated by protein kinase A, and is then active even when cytosolic calcium is low.\\n---\\n# METABOLISM\\n\\nThus even before we know for sure that we need to run, the muscles are breaking down glycogen and making the glucose they will use if running becomes necessary.\\n\\n# SUMMARY\\n\\n- Metabolism is the collective term for all of the reactions going on inside a cell. These reactions are divided into catabolic—those that break down chemical compounds to provide energy—and anabolic—those that build up complex molecules from simpler ones. Catabolic reactions are oxidative and anabolic reactions are reductive.\\n- The Krebs cycle is at the center of the cell’s metabolism. It can act to oxidize two carbon units derived from carbohydrates, fats, or amino acids and acts as a central switching yard for the molecules in metabolism.\\n- Glycolysis converts glucose to pyruvate. If the pyruvate is reduced to lactate, glycolysis can continue in the absence of oxygen as this reduction regenerates the NAD+ needed for glycolysis.\\n- Glycogen can act as a reserve of glucose for glycolysis.\\n- Glucose-6-phosphate can be converted to a pentose sugar for nucleotide manufacture with production of NADPH for biosynthesis.\\n- Fats (triacylglycerols) are concentrated fuel stores. Their fatty acid components are oxidized to two-carbon units by β oxidation.\\n- Amino groups must be removed before excess dietary amino acids can be used as fuels. This is done by transferring them to make aspartate or glutamate, and thence to urea for excretion.\\n- Gluconeogenesis allows the synthesis of glucose from noncarbohydrate precursors (but mammals cannot make glucose from fatty acids).\\n- Biosynthetic pathways for molecules follow different routes from the catabolic pathways. Good examples are fatty acid synthesis and breakdown (β oxidation) and glycogen synthesis and breakdown.\\n- Plants use light energy to generate ATP and reducing power as NADPH and hence to fix carbon dioxide into sugars.\\n- Metabolic reactions are controlled by feedforward and feedback mechanisms, which make use of allosteric control and covalent modification of key enzymes.\\n\\n# FURTHER READING\\n\\n- Bender, D. A. 2002. An Introduction to Nutrition and Metabolism, 3rd ed. London: Taylor and Francis.\\n- Buchanan, B. B., Gruissem, W., and Jones, R. L. 2000. Biochemistry and Physiology of Plants. Rockville, Maryland: American Society of Plant Physiologists.\\n- Devlin, T. M. 2002. Textbook of Biochemistry with Clinical Correlations, 5th ed. New York: Wiley.\\n- Voet, D., and Voet, J. D. 2003. Biochemistry, 3rd ed. New York: Wiley.\\n---\\n# REVIEW QUESTIONS\\n\\n|1. Metabolism is|B. the sum of all the anabolic and catabolic reactions in a cell.|\\n|---|---|\\n| | |\\n|2. The Krebs cycle|B. oxidizes two-carbon units to produce carbon dioxide and reduced coenzymes.|\\n| | |\\n|3. Which of the following statements about the glycolysis pathway in the cytosol is incorrect?|B. It makes acetyl-CoA.|\\n| | |\\n|4. β oxidation|D. oxidizes fatty acids to give acetyl-CoA, NADH, and FADH2.|\\n| | |\\n|5. Gluconeogenesis is not capable of making glucose from|D. palmitate.|\\n| | |\\n|6. Which of the following statements is incorrect concerning photosynthesis in green plants?|C. Its enzymes are found in the mitochondria.|\\n| | |\\n|7. Positive feedback is operating|B. when cAMP activates transcription of the lac operon.|\\n---\\n# METABOLISM\\n\\nC. when tryptophan inhibits transcription of the trp operon.\\n\\nD. when N -acyl-HSL promotes transcription of the lux operon.\\n\\nE. in all the examples above.\\n\\n8. ATP\\n\\nA. is converted to ADP as H+ enters the mitochondrial matrix.\\n\\nB. is a coenzyme.\\n\\nC. passes through the outer mitochondrial membrane by simple diffusion.\\n\\nD. cannot be made in the absence of oxygen.\\n\\nE. concentrations fall to a low level in resting cells.\\n\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n1. B. It is the sum of all of the catabolic and anabolic reactions in a cell.\\n\\n2. B. The Krebs cycle oxidizes acetyl-CoA and two carbon dioxides are lost for each turn of the cycle. The cycle does not use pyruvate, rather, pyruvate is first converted to acetyl-CoA by pyruvate dehydrogenase, and it is acetyl-CoA that is fed into the cycle. Concerning answer D: the electron transport chain drives H+ out of the mitochondrial matrix. ATP synthase can also (by running in reverse) perform this function, but the enzymes of the Krebs cycle certainly do not. Concerning answer E: the Krebs cycle does not use either GTP or ATP to supply phosphate groups, though it does generate one GTP from GDP (step 5 in Fig. 13.2).\\n\\n3. B. Glycolysis generates pyruvate. Pyruvate can then be converted to acetyl-CoA, but this takes place in mitochondria, not in the cytosol, and is not regarded as being part of the glycolysis pathway.\\n\\n4. D. β oxidation produces acetyl-CoA from fatty acids and also produces NADH and FADH2.\\n\\n5. D. Glucose cannot be produced from any fatty acid, including palmitate.\\n\\n6. C. Photosynthesis takes place in chloroplasts.\\n\\n7. D. One of the genes within the lux operon is luxI, which codes for the enzyme that makes N -acyl-HSL. This is therefore an example of positive feedback. Concerning the other examples (A and C): these are examples of negative feedback. Activation of phosphofructokinase will cause more fuel to be sent to the mitochondria and will therefore help the mitochondria convert AMP and ADP to ATP. Inhibiting transcription of the trp operon will reduce the rate at which the cell synthesizes tryptophan. (B) This is a sensible control mechanism, causing the cell to use glucose when it is available in preference to lactose. However, there is no feedback involved: the products of the lac operon are concerned with lactose metabolism, while cAMP levels in bacteria are affected by the glucose concentration.\\n\\n8. B. A coenzyme is a molecule that acts as a second substrate for many enzymes; ATP is a substrate for a wide range of enzymes; some we have already met are polynucleotide kinase (page 139), hexokinase (page 284), phosphofructokinase (page 285), glycogen phosphorylase kinase (page 305), and cAMP-dependent protein kinase (page 305). Answer A is false: As H+ enters the mitochondria ADP is converted to ATP, not the other way around. Answer C is false: ATP is a highly charged ion and will certainly not pass through membranes by simple diffusion; it crosses the outer mitochondrial membrane by passing through porin. Answer D is false: in the absence of oxygen ATP can be made by anaerobic glycolysis. Answer E is false: in a healthy cell none of the energy currencies are allowed to run down.\\n---\\n# IONS AND VOLTAGES\\n\\nWe described in Chapter 2 how membranes are composed of phospholipids arranged so that their hydrophobic tails are directed toward the center of the membrane, while the polar hydrophilic head groups face out. Membranes are a barrier to the movement of many solutes. In particular, small hydrophilic solutes such as ions and sugars cannot pass through membranes easily because, to do so, they would have to lose the cloud of water molecules that forms their hydration shell (page 21). Two consequences follow from the fact that membranes are barriers. First, the composition of the liquid on one side of a membrane can be different from the composition of the liquid on the other side. Indeed, by allowing cells to retain proteins, sugars, ATP, and many other solutes, the barrier property of the cell membrane makes life possible. Table 14.1 shows how five important ions have different concentrations in cytosol and extracellular medium. Second, the cell must make proteins called channels and carriers whose job it is to help hydrophilic solutes across the membrane. This chapter describes the properties of membranes, with particular emphasis on their role in energy storage.\\n\\n# THE POTASSIUM GRADIENT AND THE RESTING VOLTAGE\\n\\nIons are electrically charged. This fact has two consequences for membranes. First, the movement of ions across a membrane will tend to change the voltage across that membrane. If positive ions leave the cytosol, they will leave the cytosol with a negative voltage, and vice versa. Second, a voltage across a membrane will exert a force on all the ions present. If the cytosol has a negative voltage, then positive ions such as sodium and potassium will\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C2004 by John Wiley & Sons, Inc.\\n---\\n# IONS AND VOLTAGES\\n\\n|Ion|Cytosol|Extracellular Medium|\\n|---|---|---|\\n|Sodium Na+|10 mmol liter−1|150 mmol liter−1|\\n|Potassium K+|140 mmol liter−1|5 mmol liter−1|\\n|Calcium Ca2+|100 nmol liter−1|1 mmol liter−1|\\n|Chloride Cl−|5 mmol liter−1|100 mmol liter−1|\\n|Hydrogen ion H+ (really H3O+)|63 nmol liter−1 or pH 7.2|40 nmol liter or pH 7.4|\\n\\nNote: the units n for nano is one million times smaller than m for milli.\\n\\nIn this chapter, we will begin to address the question of how ions and voltages interact by considering the effect of potassium movements on the voltage across the plasma membrane.\\n\\nPotassium Channels Make the Plasma Membrane Permeable to Potassium Ions\\n\\nThe potassium channel is a protein found in the plasma membrane of almost all cells. It is a tube that links the cytosol with the extracellular fluid. Potassium ions, which cannot pass through the lipid bilayer of the plasma membrane, pass through the potassium channel.\\n\\nFigure 14.1. The positively charged potassium ion cannot cross the lipid bilayer but passes easily through a water-filled tube in the potassium channel.\\n---\\n# THE POTASSIUM GRADIENT AND THE RESTING VOLTAGE\\n\\nchannel easily. Other ions cannot go through. The precise shape of the tube, and the position\\nof charged amino acid side chains within the tube, blocks their movement. The channel is\\nselective for potassium.\\nWe saw earlier that the Na+/K+ ATPase (page 270) uses the energy of ATP hydrolysis\\nto drive sodium ions out of the cell and, at the same time, brings potassium ions into the\\ncell. This ensures that potassium is much more concentrated in the cytosol than outside—\\ntypically 140 mmol liter−1 in the cytosol but only 5 mmol liter−1 in the extracellular medium.\\nThere is an apparent paradox here. If potassium can pass through the potassium channel,\\nwhy is this ion much more concentrated inside the cell than outside? Why doesn’t all the\\npotassium rush out? To explain why, we must think about the effects of ion movement on\\ntransmembrane voltage. First a word on nomenclature. All cells have a voltage across their\\nmembrane when they are not being stimulated. This resting voltage is about −80 mV in\\na relaxed skeletal muscle cell. As soon as the muscle is stimulated to contract, there is a\\nsudden sharp change in the transmembrane voltage called the action potential (a process\\ndescribed in more detail in Chapter 15). Many cells never change their transmembrane\\nresting voltage.\\n\\nConcentration Gradients and Electrical Voltage Can Balance\\nA few potassium ions do escape from the cell through the potassium channel. As they do\\nso, they carry out their positive charge and leave the cytosol with a negative voltage that\\nattracts positively charged ions like potassium. There is still a tendency for potassium ions\\nto leave the cell down the concentration gradient, but there is now an electrical force pulling\\nthe positively charged potassium ions back inside. We have met a similar situation before,\\nwhere we described how a concentration gradient and an electrical force combine to form\\nan electrochemical gradient down which H+ ions will rush into bacteria or mitochondria\\n(page 261). However, in the case of potassium ions at the plasma membrane, the concentration\\ngradient and the electrical force act in opposite directions. As potassium ions continue\\nto leave the cell, carrying out their positive charge and leaving the cytosol at a more and\\nmore negative voltage, the electrical force pulling them back in gets increasingly strong.\\nSoon, the opposing electrical and concentration gradients are equal, and the overall electrochemical gradient for potassium is zero. Potassium ions then stop leaving the cell, even\\nthough they are much more concentrated inside than outside.\\nFor every ion present on both sides of a membrane, it is possible to calculate the\\ntransmembrane voltage that will exactly balance the concentration gradient. This voltage is\\ncalled the equilibrium voltage for that ion at that membrane. For potassium at the plasma\\nmembrane of a normal animal cell, the equilibrium voltage is about −90 mV.\\nThe departure of potassium ions through the potassium channels, leaving negative\\ncharge behind, produces the resting voltage across the plasma membrane. Because the\\npotassium channels are the major pathway by which ions can cross the plasma membrane\\nof an unstimulated cell, the resting voltage has a value close to the potassium equilibrium\\nvoltage. In some cells, such as white blood cells, potassium channels are the only channels in\\nthe plasma membrane, and potassium ions move out until the transmembrane voltage tending\\nto pull them back in exactly balances their tendency to move out down their concentration\\ngradient. The resting voltage of white blood cells therefore has a value equal to the potassium\\nequilibrium voltage, about −90 mV. In other cells the situation is more complicated. In\\nmuscle cells, for instance, the resting voltage is −80 mV; in nerve cells it is −70 mV. Even\\nin these cells, though, the major influence on the resting voltage is potassium movement\\n---\\n# IONS AND VOLTAGES\\n\\nThrough its channels, so the resting voltage does not deviate very far from the potassium equilibrium voltage. The resting voltage set up by potassium movement turns the action of the Na+/K+ ATPase into an energetically asymmetrical one. Consider one conversion cycle: one molecule of ATP is hydrolyzed, three sodium ions are pushed out of the cell, and two potassium ions move in. Very little of the energy of ATP hydrolysis is used up in moving the two potassium ions into the cell because the electrochemical gradient for this ion is close to zero. Although potassium is being moved up a concentration gradient, it is also being pulled in by the negative voltage of the cytosol, and the two forces cancel. In contrast, pushing the three sodium ions out of the cell requires more energy than would be required if the cytosol were at the same voltage as the extracellular fluid. The sodium ions are positively charged, so they are attracted by the negative voltage inside the cell, which combines with the concentration gradient to form a large inward electrochemical gradient. All the energy released by ATP hydrolysis is needed to push the three sodium ions up this large electrochemical gradient. The presence of the potassium channels, and the resting voltage that they set up, means that almost all the energy of ATP hydrolysis by the Na+/K+ ATPase is stored in the sodium gradient, while potassium ions are close to equilibrium.\\n\\n# IN DEPTH 14.1 The Nernst Equation\\n\\nAn ion that can pass across a membrane is acted on by two forces. The first derives from the concentration gradient. The ion tends to diffuse from a region where it is at high concentration to one where it is at low concentration. The second force derives from the transmembrane voltage. In the case of positively charged ions such as Na+ and K+, the ions tend to move toward a negative voltage. Negatively charged ions such as Cl- tend to move toward a positive voltage. For each ion there is a value of the transmembrane voltage for which these forces balance, and the ion will not move. The ion is said to be at equilibrium, and this value of the transmembrane voltage is called the equilibrium voltage for that ion at that membrane.\\n\\n|Ion I at concentration [I] outside|Ion I at concentration [I] inside cytosol at voltage V|\\n|---|---|\\n|When the forces balance, then ions that move in will neither gain nor lose energy. This way of describing equilibrium is useful because it allows us to set equivalent the effects of the two very different gradients, concentration and voltage. For concentration, the free energy possessed by a mole of ions I by virtue of its concentration is| |\\n|G = G0 + RT log e[I] joules| |\\n|where G0 is the standard free energy, R is the gas constant (8.3 J mol-1 degree-1), and T is the absolute temperature.| |\\n---\\n# THE NERNST EQUATION\\n\\nA mole of I passing in therefore moves from a region where it had a free energy of\\n\\n|G|= G|+ RT log [I]|joules|\\n|---|---|---|---|\\n|outside 0 eoutside| | | |\\n\\nto one where its free energy is\\n\\n|G|= G|+ RT log [I]|joules|\\n|---|---|---|---|\\n|inside 0 einside| | | |\\n\\nOne mole of ions I moving inward therefore gains by virtue of the concentration gradient free energy equal to\\n\\n|RT log [I]|− RT log [I]|joules|\\n|---|---|---|\\n|einside eoutside| | |\\n\\nNow consider the electrical force. The definition of a volt means that one coulomb of charge moving across a membrane with a transmembrane voltage of V volts gains V joules of free energy. However, we are working in moles, not coulombs. One mole of ions has a charge of zF coulombs, where z is the charge on the ion in elementary units. For Na+ and K+ z is 1; for Ca2+ z is 2; and for Cl− z is −1. The term F is a number that relates the coulomb to the mole. It has the value 96,500. One mole of ions I moving inward gains by virtue of the transmembrane voltage free energy equal to\\n\\nzFV joules\\n\\nThis does not mean that an ion always gains energy from the transmembrane voltage when it moves inward: the term zFV can just as easily be negative as positive. When the effects of concentration and voltage just balance, a mole of ions moving inward neither gains nor loses free energy. Hence, at equilibrium\\n\\n|RT log [I]|− RT log [I]|zFV|= 0|\\n|---|---|---|---|\\n|e inside|e outside|eq| |\\n\\nThis can be simplified to Veq = RT log e ([Ioutside]) volts\\n\\nzF\\n\\nThis is the Nernst equation. At a typical mammalian body temperature of 37°C, the Nernst equation can be written in the more convenient form:\\n\\nVeq = 62 log10 ([Ioutside]) mvolts\\n\\nz[I]\\ninside\\n\\nNote the change in the type of logarithm and the units.\\n\\n“In” and “out” can refer to any two solutions separated by a membrane. At the plasma membrane in is the cytosol and out is the extracellular medium, but when considering equilibria across the inner mitochondrial membrane in is the mitochondrial matrix and out is the intermembrane space.\\n---\\n# IONS AND VOLTAGES\\n\\nTHE CHLORIDE GRADIENT\\n\\nChloride ions (Fig. 14.2) are at a lower concentration in the cytosol than in the extracellular medium. Typically, their concentration in the cytosol is 5 mmol liter−1 compared with 100 mmol liter−1 in the extracellular fluid. This is because of the resting voltage set up by the potassium channels. Chloride ions are repelled by the negative voltage of the cytosol. They leave until their tendency to enter the cell down their concentration gradient exactly matches their tendency to be repelled by the negative voltage of the cytosol.\\n\\n|[Cl−] = 100 mmol liter−1|chloride would .....the negatively charged move down its concentration cytosol pushes the negative chloride ions but.................. outward|\\n|---|---|\\n|[Cl−] = 5 mmol liter−1|cytosol -80 mV 0 mV|\\n\\nFigure 14.2. Chloride is close to equilibrium across most plasma membranes.\\n\\nGENERAL PROPERTIES OF CHANNELS\\n\\nChannels are integral membrane proteins that form water-filled tubes through the membrane. We have already introduced three: the gap junction channel (page 55), porin (page 262), and the potassium channel. Channels that, like the potassium channel, are selective for particular ions can set up transmembrane voltages. The gap junction channel (page 55) is much less selective than the potassium channel. It forms a tube, 1.5 nm in diameter, through which any solute of Mr ≤ 1000 can pass. The gap junction channel is not always open. It opens only when it connects with a second gap junction channel on another cell, forming a tube through which solutes can pass from the cytosol of one cell to the cytosol of the other. Channels that are sometimes open and sometimes shut are said to be gated. When a gap junction channel contacts another on another cell, its gate opens and solute can pass through; at other times the gate is shut. The usefulness of gating is obvious: if the gap junction channels not contacting others were open, many solutes, including ATP and sodium, would leak out into the extracellular fluid exhausting the cell’s energy currencies.\\n\\nPorin in the outer mitochondrial membrane plays an important role in energy conversion. It forms a very large diameter tube that allows all solutes of Mr ≤ 10,000 to pass and seems to spend a large fraction of time open under most circumstances. This is why the outer mitochondrial membrane is permeable to most solutes and ions. Appendix 1, at the end of this book, lists all the different types of channels described in this book. It represents only a small fraction of the total number known.\\n---\\n# GENERAL PROPERTIES OF CHANNELS\\n\\n|Example 14.1|Cytochrome c—Vital But Deadly|\\n|---|---|\\n|We have described how the electron carrier cytochrome c resides in the intermembrane space between the outer and inner mitochondrial membranes and helps the electron transport chain to convert energy as NADH to energy as the hydrogen ion electrochemical gradient across the mitochondrial inner membrane (page 266). Although cytochrome c is a soluble protein of relative molecular mass 12,270, it cannot escape from the intermembrane space into the cytosol because porin, the channel of the outer mitochondrial membrane, only allows solutes of Mr ≤ 10,000 to pass. Although cytochrome c is essential for mitochondrial function, it has another, deadly role. If cytochrome c comes into contact with a class of cytosolic enzymes called caspases, it activates them, turning on the process of cell suicide called apoptosis (page 417). Under certain conditions, porin can associate with other proteins to form a channel of larger diameter; when this happens, cytochrome c can leak out and the cell dies by apoptosis. This process seems to occur in hearts during heart attacks, and in the brain during a stroke: there is therefore a considerable research effort aimed at preventing this from occurring.| |\\n\\n|cytochrome c|Mr = 12,270|\\n|---|---|\\n|Normal|porin : cutoff Mr= 10,000|\\n|outer mitochondrial membrane| |\\n|inner mitochondrial membrane| |\\n|After stress|Apoptosis|\\n|inactive caspase 9|active caspase 9|\\n|cytochrome c|porin plus other proteins|\\n---\\n# IONS AND VOLTAGES\\n\\nGENERAL PROPERTIES OF CARRIERS\\n\\nWe have already met the three carriers that interconvert the four energy currencies of the cell (page 263). Carriers are like channels in that they are integral membrane proteins that allow solute to cross the membrane and, like channels, they form a tube across the membrane. However, there is a critical difference. In carriers the tube is never open all the way through; it is always closed at one or other end. Solutes can move into the tube through the open end. When the carrier changes shape, so that the end that was closed is open, the solute can move into the solution on the other side of the membrane.\\n\\nThe Glucose Carrier\\n\\nOne of the simplest carriers is the glucose carrier (Fig. 14.3). It switches freely between a form that is open to the cytosol and a form that is open to the extracellular medium. Inside the tube is a site to which a glucose molecule can bind. On the left, a glucose molecule is entering\\n\\n|extracellular medium|cytosol|\\n|---|---|\\n|glucose|glucose|\\n|glucose carrier| |\\n\\n|3Na +|1Ca 2+|\\n|---|---|\\n|Na +/Ca 2+ exchanger| |\\n\\n1Ca 2+\\n3Na +\\n\\nFigure 14.3. The glucose carrier and the sodium/calcium exchanger are sometimes open to the cytosol and sometimes open to the extracellular medium.\\n---\\n# GENERAL PROPERTIES OF CARRIERS\\n\\nThe tube and binding to the site. Sometimes glucose leaves the binding site before the carrier switches shape. In Figure 14.3 we see the other possibility: the carrier switches shape before the glucose has left. The binding site is now open to the cytosol, and the glucose can escape into the cytosol. It has been carried across the plasma membrane. Unlike channels, carriers never form open tubes all the way across the membrane. Instead, they bind one or more molecules or ions, then change shape to carry the molecules or ions across the membrane.\\n\\nExample 14.2 The Glucose Carrier Is Essential\\n\\nThe cells of our bodies are bathed in a glucose-rich solution. However, glucose cannot cross the plasma membrane by simple diffusion because it is strongly hydrophilic: It can only get in via the glucose carrier. Some cells have glucose carriers in their membranes all the time, but others such as muscle and fat cells translocate the glucose carrier to the plasma membrane only in the presence of insulin. Insulin-dependent diabetics cannot produce their own insulin, and unless they inject synthetic insulin, their muscles and fat cells cannot take up glucose and therefore run out of energy, even though the concentration of glucose in the blood becomes very high. This is why muscular weakness is one symptom of diabetes.\\n\\nThe glucose carrier is very simple, whereas other carriers are more complex. We will next consider two carriers, the sodium/calcium exchanger and the calcium ATPase, which do much the same job—they push calcium ions up their concentration gradient out of the cell—but which take their energy from different currencies. The sodium/calcium exchanger uses the sodium concentration gradient while the calcium ATPase uses ATP.\\n\\n# The Sodium–Calcium Exchanger\\n\\nFigure 14.3 shows that like the glucose carrier, the sodium/calcium exchanger can exist in two shapes, one open to the extracellular medium and one open to the cytosol. Inside the tube are three sites that can bind sodium ions and one site that can bind a calcium ion. The sodium/calcium exchanger is not free to switch between its two shapes at any time. Instead, it switches only if either all the sodium sites are filled and the calcium site is empty or if the calcium site is filled and all the sodium sites are empty.\\n\\nOn the left, the carrier is open to the extracellular medium. It can switch its shape so that it is open to the cytosol if one of two things happen. It could, as shown, bind three sodium ions (keeping the calcium site empty) or it could bind one calcium and keep the sodium sites vacant. This second option does not often happen, since sodium is at high concentration in the extracellular fluid, and one or more of the sodium sites is usually occupied. Therefore nearly all the switches from open-to-outside to open-to-inside are of the type shown. Once the tube has opened to the low sodium environment of the cytosol, the sodium ions tend to leave.\\n\\nOnce the carrier is open to the cytosol, it can switch back to the open-to-outside form by binding either one calcium or three sodium ions. Since sodium is scarce in the cytosol, the latter event is unlikely. More frequently a calcium ion will bind and will be carried out. The carrier is now ready to bind sodium again and perform another cycle.\\n\\nThe overall effect of one cycle is to carry three sodium ions into the cell down their electrochemical gradient, and one calcium ion out of the cell up its electrochemical gradient. A simple rule about when the carrier can switch shape has produced a machine that uses the energy currency of the sodium gradient to do work in pushing calcium ions out of the cell. Figure 14.4 shows a simple way of representing what is happening. The circle represents one cycle of operation, from open to extracellular medium back to open to extracellular medium again.\\n---\\n# IONS AND VOLTAGES\\n\\nFigure 14.4. Action of the sodium/calcium exchanger.\\n\\nCarriers with an Enzymatic Action: The Calcium ATPase\\n\\nThe electron transport chain, ATP synthase, and the Na+/K+ ATPase are carriers with an additional level of complexity in that they carry out an enzymatic action as well as their carrier function. Figure 14.5 illustrates another carrier with a linked enzymatic function: the Ca2+ ATPase. Like the sodium/calcium exchanger, this is found in the plasma membrane. The transmembrane part of this carrier forms a tube that can be open to the cytosol or to the extracellular medium. Two -COO- groups in the tube can form a binding site for a calcium ion. Other domains of the protein lie in the cytosol and can hydrolyze ATP. Changes in the shape of the cytosolic region are transmitted to the transmembrane region and force it between the open-to-cytosol and open-to-outside shapes. Figure 14.5 shows our present understanding of how this might happen. (1) is the relaxed shape of the protein. ATP is held by noncovalent interactions in a crevice in one of the cytosolic domains. One of the COO- groups in the tube is protonated at this time. (2) A calcium ion moving in from the cytosol pushes the tube open and binds to both binding sites inside the tube. The positively charged calcium ion displaces an H+ ion and is then able to interact with both COO- groups. The distortion in protein shape caused by the calcium ion pushing the tube open is transmitted to the cytosolic region and brings the γ phosphate of ATP close to an aspartate residue on a neighboring domain. (3) An intrinsic kinase activity of the two domains transfers the γ-phosphate group from ATP to the aspartate residue. As the insert shows, this means that a phosphate group, with two negative charges, is sitting very close to ADP, with three negative charges. (4) The repulsion between the phosphorylated aspartate and the ADP forces the two domains apart. The shape change is transmitted to the transmembrane region, which is forced into a wide open-to-outside shape. The calcium ion, which can no longer bind to the COO- groups on both sides of the tube, is held only weakly and tends to escape into the extracellular medium. An H+ ion moves in to protonate one of the COO- groups. (5) A third cytosolic domain, which has phosphatase catalytic ability, swings in and dephosphorylates the aspartate residue. Meanwhile, ATP moves in and displaces ADP at the binding crevice. Once the aspartate has been dephosphorylated, the protein is no longer held in the open-to-outside shape by electrical repulsion and relaxes back to the resting shape shown in (1). It is now ready to accept another calcium ion from the cytosol and repeat the process.\\n\\nFigure 14.6 summarizes what is happening. The circle represents one cycle of operation. One molecule of ATP is hydrolyzed to ADP and inorganic phosphate, one calcium ion moves out of the cell, and one H+ moves in. The energy released by ATP hydrolysis has been used to drive one calcium ion up its electrochemical gradient out of the cell.\\n---\\n|O|O|O|\\n|---|---|---|\\n| | |O|P|O|O|P|O|\\n| | |O|O|\\n| | |C|O|\\n| | |H|O|\\n| | |O|\\n| | |C|O|\\n|ADP|O|5|\\n|O|P|OH|\\n| | |O|\\n|O|O|+|\\n| | |O|P|O|P|O|\\n| | |O|O|\\n| | |H|\\n| | |O|P|\\n|O|OC|O|\\n| | |O|\\n| | |C|O|\\n| | |O|O|\\n|O|P|O|\\n| | |O|O|4|++|\\n|O|P|O|\\n|O|PO|O|\\n|O|PO|O|\\n| | |O|O|\\n| | |O|CO|\\n| | |++|\\n| | |O|CO|\\n| | |3|\\n| |++H| |\\n|O|OOPO| |\\n|O|P|O|\\n| | |PO|O|\\n|O|O|OO|\\n|O|PO|POO|\\n| | |O|O|\\n| | |O|CO|\\n| | |++|\\n| | |O|CO|\\n| | |2|\\n|ATP|crevice| |\\n|binding| |OO|\\n| | |O|PO|\\n|O|P|O|\\n| | |PO|O|\\n|O|O|CO|\\n| | |HO|\\n| | |O|CO|\\n|residue|1|Calcium ATPase undergoes a cycle of phosphorylation and dephosphorylation. These drive shape changes that in turn push calcium|\\n|aspartate| | |\\n| |site| |\\n| |catalytic| |\\n|cytosol| | |\\n| |phosphatase| |\\n| | |Figure 14.5.ions out of the cell.|\\n---\\n# IONS AND VOLTAGES\\n\\nMedical Poisoned Hearts Are Stronger\\n\\nRelevance\\nDigitalis is used to treat heart failure. Digitalis inhibits the sodium/potassium ATPase and is extremely toxic. Nevertheless, a small dose, which inhibits the sodium/potassium pump just a little, causes the heart muscle to beat more strongly. The reason is that inhibiting the sodium/potassium pump just a little causes a small increase of cytosolic sodium concentration. Because the sodium/calcium exchanger has three binding sites for sodium, its activity is extremely sensitive to sodium concentration, and even a small increase of cytosolic sodium reduces its activity significantly. The calcium concentration in the cytosol therefore rises. The mechanical motor that drives heart contraction (Chapter 18) is controlled by calcium, so that a small increase of cytosolic calcium makes the heart beat more strongly.\\nDigitalis\\n\\n|extracellular medium|K+ Na+|\\n|---|---|\\n|Cytosol|Na+ Ca++|\\n| |Na+/K+ ATPase Na+/Ca++ Exchanger Force of contraction|\\n\\nAll the cells of our bodies have one of these two calcium pumps—the sodium/calcium exchanger or the calcium ATPase—and many have both. Because of the action of these carriers, the calcium concentration in the cytosol is much less than the concentration in the extracellular medium: usually about 100 nmol liter−1 compared with 1 mmol liter−1. Because the resting voltage is attracting the positively charged calcium ions inward, the overall result is a large electrochemical gradient favoring calcium entry into cells.\\n\\nIN DEPTH 14.2 Measuring the Transmembrane Voltage\\n\\nIn 1949 Gilbert Ling and Ralph Gerard discovered that when a fine glass micropipette filled with an electrically conducting solution impaled a cell, the plasma membrane sealed to the glass, so that the transmembrane voltage was not discharged. The voltage difference between a wire inserted into the micropipette and an electrode in the extracellular fluid could then be measured. By passing current through the micropipette, the transmembrane voltage could be altered. Twenty-five years later Erwin Neher and Bert Sakmann showed that the micropipette did not have to impale the cell. If it just touched the cell, a slight suction caused the plasma membrane to seal to the glass. The technique, called\\n---\\nMEASURING THE TRANSMEMBRANE VOLTAGE\\n\\nglass micropipette tip diameter less than 1μm electrode in extracellular medium (a)\\nchannel ammeter, to measure the current flowing (b)\\nvoltmeter (c)\\ncell-attached patch clamping, can measure currents through the few channels present in the tiny patch of membrane within the pipette. Stronger suction bursts the membrane within the pipette. The transmembrane voltage can now be measured. Alternatively, current can be passed through the micropipette to change the transmembrane voltage—this is the whole cell patch clamp technique. In 1991, Neher and Sakmann received the Nobel prize for medicine.\\n---\\n# IONS AND VOLTAGES\\n\\n|ADP + PiATP|Ca++|H+|\\n|---|---|---|\\n|cytosol|extracellular medium| |\\n\\nFigure 14.6. Action of the calcium ATPase.\\n\\nSUMMARY\\n\\n1. Channels are membrane proteins with a central water-filled hole through which hydrophilic solutes, including ions, can pass from one side of the membrane to the other. Changes in protein structure may act to gate the channel but are not required for movement of the solute from one side to the other.\\n2. The presence of potassium channels in the plasma membrane, and the resting voltage that they set up, means that almost all the energy of ATP hydrolysis by the Na+/K+ ATPase is stored in the sodium gradient, while potassium ions are close to equilibrium.\\n3. The resting voltage repels chloride ions from the cell interior.\\n4. Carriers, like channels, form a tube across the membrane, but the tube is always closed at one end. Solutes can move into the tube through the open end. When the carrier changes shape, so that the end that was closed is open, the solute can leave to the solution on the other side of the membrane.\\n5. The glucose carrier is present in the plasma membrane of all human cells.\\n6. The sodium/calcium exchanger is a carrier that uses the energy of the sodium gradient to push calcium ions out of the cell.\\n7. The electron transport chain, ATP synthase, the sodium/potassium ATPase, and calcium ATPase are both carriers and enzymes, the two actions being tightly linked.\\n\\nFURTHER READING\\n\\n- Ashcroft, F. M. 2000. Ion Channels and Disease. San Diego: Academic Press.\\n- Levitan, I. B., and Kaczmarek, L. K. 2002. The Neuron. New York: Oxford University Press.\\n\\nREVIEW QUESTIONS\\n\\nFor each question, choose the ONE BEST answer or completion.\\n\\n1. Which of the following statements about potassium ions is false?\\n- A. Potassium ions are more concentrated in the extracellular medium than in cytosol.\\n---\\n# REVIEW QUESTIONS\\n\\n1. Which of the following statements is false?\\n\\nA. Potassium ions are moved into cells by pe Na+/K+ ATPase.\\nB. Most plasma membranes contain potassium-selective channels.\\nC. Potassium would flow out of most cells rapidly but for pe effect of pe negative resting voltage.\\nD. Answers A pough D are false.\\n\\n2. Which of the following statements is false?\\n\\nA. The potassium channel allows potassium ions to pass from one side of pe plasma membrane to pe oper.\\nB. Porin allows chloride ions to pass from one side of pe inner mitochondrial membrane to pe oper.\\nC. The glucose carrier allows glucose to pass from one side of pe plasma membrane to pe oper.\\nD. The gap junction channel allows potassium ions to pass from pe cytosol of one cell to pe cytosol of its neighbor.\\nE. Thermogenin allows H+ to pass from one side of pe inner mitochondrial membrane to pe oper.\\n\\n3. The resting voltage\\n\\nA. is between −70 and −90 mV in most cells.\\nB. attracts chloride ions in from pe extracellular medium.\\nC. is mainly determined by pe chloride concentration gradient across pe plasma membrane.\\nD. helps retain ATP and glucose wipin cells.\\nE. helps keep pe sodium concentration low in pe cytosol.\\n\\n4. Solute movement through which channels is mainly responsible for the resting voltage?\\n\\nA. Porin\\nB. Gap junction channels\\nC. Glucose\\nD. Potassium\\nE. All of pe above contribute equally to pe resting voltage.\\n\\n5. Which of the following statements is false?\\n\\nA. Channels form water-filled tubes prough pe membrane.\\nB. Channels are often gated.\\nC. Channels are only found in pe plasma membrane.\\nD. Channels cannot transport solute up an electrochemical gradient.\\nE. Channels are often selective for a particular solute or type of solute.\\n\\n6. Carrier proteins\\n\\nA. never form a tube open all pe way prough a membrane.\\nB. can transport solute up an electrochemical gradient.\\nC. can often carry out an enzymatic function in addition to peir carrier function.\\nD. are found in pe plasma membrane and in intracellular membranes.\\nE. all of pe above.\\n\\n7. An ion X+ is at equilibrium across the plasma membrane of a cell. The cell is voltage clamped, that is, the value of the transmembrane voltage is set by the experimenter, not by the properties of the membrane. Which of the following would cause X+ to no longer be at equilibrium?\\n\\nA. A change of transmembrane voltage\\nB. A change in pe concentration of X+ in pe extracellular medium\\nC. Eiper answer A or B\\nD. Closure of pe channels pat allow X+ to cross pe membrane\\nE. A change in pe selectivity of pe channel pat allows X+ to cross pe membrane so pat Y+ can cross as well\\nF. Eiper answer D or E\\n---\\n# IONS AND VOLTAGES\\n\\n8.   Which of pe following is not a carrier?\\nA. The electron transport chain\\nB. Hexokinase\\nC. ATP synpase\\nD. The sodium/potassium ATPase\\nE. Calcium ATPase\\nF. The sodium/calcium exchanger\\nG. The ADP/ATP exchanger\\n\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n1.  A.   Potassium ions are more concentrated in the cytosol due to the action of the Na+/K+ ATPase.\\n\\n2.  B. Although porin will certainly allow chloride ions to pass, it is present in the outer mitochondrial membrane, not the inner. All the other statements are true, although, regarding answer D, note that the gap junction channel will allow many different solutes to pass; it is not in any way selective for potassium.\\n\\n3.  A. All the others are false. In particular, (B) chloride ions are repelled from the negative cytosol; (C) the resting voltage is mainly determined by the potassium concentration gradient across the plasma membrane; (D) ATP is negatively charged and is therefore not attracted by a negatively charged cytosol, it is retained by the cytosol because it is not able to cross the plasma membrane. Glucose is uncharged and is therefore unaffected by the transmembrane voltage; it is retained within cells because it is rapidly phosphorylated to glucose-6-phosphate, which is not able to cross the plasma membrane. (E) Sodium is attracted into cells by the negatively charged cytosol; the reason it has a low concentration in the cytosol is because it is pushed out of cells by the Na+/K+ ATPase.\\n\\n4.  D. Because most cells express potassium channels in their plasma membranes, potassium is the ion that can most easily cross the plasma membrane and therefore is mainly responsible for the resting voltage. Concerning answer C, note that there is no “glucose channel,” although large channels, including porin and the gap junction channel, can carry glucose along with many other solutes.\\n\\n5.  C.   Channels are found in all intracellular membranes. We have already discussed porin, a channel of the outer mitochondrial membrane. All the other statements are true. In particular, channels cannot transport solute up an electrochemical gradient; they simply make it easier for solute to pass down its electrochemical gradient.\\n\\n6.  E.   Answers A through D are true. In particular, carrier proteins can transport solute up an electrochemical gradient by linking the movement with another process with negative ΔG. In most cases this is either the hydrolysis of ATP to ADP and inorganic phosphate or the movement of another solute down its electrochemical gradient. The electron transport chain links the movement of H+ up its electrochemical gradient with the reaction\\n\\n7.  C.    If X+ is at equilibrium across the plasma membrane, its tendency to move one way down its concentration gradient is matched by its tendency to move the other way down the voltage gradient. Changing the amplitude of either gradient will cause these two effects to no longer balance. Neither answer D nor E is true. If X+ is at equilibrium, then it will remain at equilibrium irrespective of how easily it or other ions can cross the membrane.\\n\\n8.  B. Hexokinase is a soluble protein of the cytosol; it catalyzes the vital metabolic step of phosphorylating glucose to glucose-6-phosphate but has no carrier action. All the others are carriers.\\n---\\n# THE ACTION POTENTIAL\\n\\nAn action potential is an explosive change in the voltage across the plasma membrane. The most sophisticated electrically excitable cells—cells that can produce action potentials—are the nerve cells that allow our brains to carry out the complex electrical data processing called thought. However, we will begin with a much simpler electrically excitable cell: the sea urchin egg.\\n\\n# THE CALCIUM ACTION POTENTIAL IN SEA URCHIN EGGS\\n\\nIn most animals, eggs that are fertilized by more than one sperm fail to develop. A number of mechanisms exist to prevent such “polyspermy,” one of which is based on action potentials.\\n\\nEffect of Egg Transmembrane Voltage on Sperm Fusion An experiment first carried out in 1976 by Laurinda Jaffe is illustrated in Figure 15.1a. She impaled a sea urchin egg with a micropipette (page 320). Instead of just measuring the natural resting transmembrane voltage (around −70 mV), she depolarized the plasma membrane by passing positive current through the pipette. We use the word depolarization to mean any positive shift in the transmembrane voltage, whatever its size or cause. In this case an electronic feedback circuit was used to alter the value of the current passed though the electrode so that the transmembrane voltage was depolarized to +10 mV and held at\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C 2004 by John Wiley & Sons, Inc.\\n---\\n# THE ACTION POTENTIAL\\n\\n(a)\\n\\npass whatever electrical current is necessary to keep the transmembrane voltage at +10 mV\\npass whatever electrical current is necessary to keep the transmembrane voltage at −70 mV\\n\\nmany sperm fuse-cell division is abnormal and zygote cannot develop\\n\\n(b)\\n\\nrecord transmembrane voltage but do not alter it\\nfirst sperm successfully fuses\\nsubsequent sperm cannot fuse-cell division is normal and zygote will develop\\n\\nFigure 15.1. In sea urchin eggs transmembrane voltage controls sperm fusion.\\n\\nJaffe observed that sperm would not fuse with eggs in which the transmembrane voltage was clamped at +10 mV, and the eggs remained unfertilized (Fig. 15.1a). However, an egg clamped at −70 mV (Fig. 15.1b) was fertilized by multiple sperm, leading to a defective embryo. The transmembrane voltage is able to control the fusion of sperm and egg.\\n\\nWhen no current is passed through the micropipette, the transmembrane voltage is set by the channels in the egg plasma membrane (Fig. 15.1c). The resting voltage of the unfertilized egg is set by potassium movement through potassium channels. The first sperm that arrives can fuse because the transmembrane voltage is −70 mV. But within 100 ms, the transmembrane voltage depolarizes to +10 mV and further sperm fusion is blocked. Thus transmembrane voltage changes to prevent multiple sperm fusion. The sea urchin\\n---\\n# THE CALCIUM ACTION POTENTIAL IN SEA URCHIN EGGS\\n\\nThe egg, which when unfertilized has a transmembrane voltage of −70 mV and is receptive to fertilization, depolarizes its membrane following successful fertilization to protect itself from subsequent fertilization. The result is shown on the right of Figure 15.1c. Only one sperm has fused, so the fertilized egg has the correct number of chromosomes and can go on to develop into an embryo and then into a mature sea urchin.\\n\\n# The Voltage-Gated Calcium Channel\\n\\nThe rapid depolarization that occurs upon fertilization of the sea urchin egg is an example of an action potential. Like many cells, urchin eggs can generate action potentials because their plasma membranes contain, together with potassium channels, an ion-conducting channel called the voltage-gated calcium channel.\\n\\nFigure 15.2 shows how the voltage-gated calcium channel works. On the left is the shape of the protein when the transmembrane voltage is −70 mV, as in an unfertilized egg. The protein forms a tube, but it is not open all the way through the membrane. If the membrane depolarizes so that the cytosol is less negative or even becomes positive, positively charged amino acid residues on the protein are repelled, popping the tube into an open state. Calcium ions can now pass through the channel. If the membrane is then repolarized, the channel quickly recloses because the positive charges are pulled back toward the inside of the cell.\\n\\nA protein domain in the cytosol called the inactivation plug is constantly jiggling about at the end of a flexible link and can bind to the inside of the open channel. The resulting blockage is called inactivation, and it occurs after about 100 ms. As long as the plasma membrane remains depolarized, the voltage-gated calcium channel will remain inactivated. When the plasma membrane is repolarized, the positive charges are attracted back toward the inside of the cell, squeezing the plug out. In summary:\\n\\n1. When the transmembrane voltage is −70 mV, the voltage-gated calcium channel is gated shut.\\n2. When the plasma membrane is depolarized, the channel opens rapidly and after about 100 ms inactivates.\\n\\n|Figure 15.2. The voltage-gated calcium channel.|Figure 15.2. The voltage-gated calcium channel.|\\n|---|\\n|depolarization repolarization inactivation plug|spontaneous after 100 ms Ca ++ repolarization|\\n|extracellular medium|extracellular medium|\\n|Cytosol|Cytosol|\\n---\\n# THE ACTION POTENTIAL\\n\\n3. After the channel has gone through this cycle, it must spend at least 100 ms with the transmembrane voltage at the resting voltage before it can be opened by a second depolarization.\\n\\nThe voltage-gated calcium channel goes through this process whether or not calcium ions are present. To understand the action potential, we must now think about the effect that movements of calcium ions through this channel have on the transmembrane voltage.\\n\\nThe Calcium Action Potential\\n\\nAt point A in Figure 15.3 the egg is unfertilized. The potassium channels are open and set the transmembrane voltage to −70 mV. This negative voltage closes the voltage-gated calcium channels.\\n\\nAt point B we show what happens if an experimenter passes enough current down a micropipette to depolarize the plasma membrane by 10 mV, to −60 mV, and then stops passing current and once again allows the ion channels in the plasma membrane to determine the transmembrane voltage. This small depolarization opens only a very few voltage-gated calcium channels, so what happens next depends almost entirely on the movement of ions through the potassium channels. The slight decrease in the negative voltage of the cytosol reduces its attraction to potassium ions, and they move out of the cell down their concentration gradient. In so doing, they carry positive charge away from the cytosol, and the transmembrane voltage returns to its resting value of −70 mV.\\n\\nAt point C, the experiment is repeated, but this time the membrane is fleetingly depolarized by 20 to −50 mV. Again, most of the voltage-gated calcium channels stay shut, but some open, and calcium ions rush into the cell down their electrochemical gradient. The flow of positive charge inward on the calcium ions outweighs the outward flow of positive charge on potassium ions, so the cytosol is gaining positive charge and its voltage is moving in the positive direction. As a result more voltage-gated calcium channels open, and this allows more calcium ions to flood in. This is an example of positive feedback (page 303) because every calcium ion that moves in makes the cytosol more positive, causing more calcium channels to open, and therefore making it easier for the next calcium ion to move in. At point D, all the calcium channels are open, the inward calcium current is much greater than the outward potassium current, and the plasma membrane is rapidly depolarizing.\\n\\nHowever, after 100 ms, the voltage-gated calcium channels begin to inactivate and the inward flow of calcium ions stops. The plasma membrane stops depolarizing and returns to −70 mV as potassium leaves through its channels.\\n\\nThis experiment shows the critical feature of an action potential: it is all or nothing. The depolarization at point B was too small to generate an action potential, and the transmembrane voltage simply fell back to its resting state. At point C the depolarization was big enough to start the process, and the action potential took off in an explosive, self-amplifying way until all the voltage-gated calcium channels were open and the plasma membrane had greatly depolarized. In all excitable cells there is a threshold for initiating an action potential; in the sea urchin egg it lies between −60 and −50 mV. Depolarizations to below the threshold elicit nothing; depolarizations to voltages more positive than the threshold elicit the complete action potential. At the heart of the action potential is a positive feedback loop. Depolarization causes voltage-gated calcium channels to open, and open calcium channels cause depolarization.\\n---\\n# THE CALCIUM ACTION POTENTIAL IN SEA URCHIN EGGS\\n\\n| |10|time|\\n|---|---|---|\\n| |-10|1 s|\\n| |-30|D|\\n| |-50|C|E|\\n|transmembrane voltage|B| |\\n| |-70|A|\\n\\n|voltage-gated|Ca 2+|calcium channels|\\n|---|---|---|\\n| |-70 mV|-60 mV|-50 mV|\\n| |A|B|C|\\n\\n|potassium|channels|K +|K+|K +|K+|\\n|---|---|---|---|---|---|\\n| |Ca 2+|Ca 2+|Ca 2+|membrane| |\\n| |-30 mV|-50 mV|of sperm| | |\\n| |D|E| | | |\\n\\nK +\\nK+\\nK +\\nK +\\n\\nFigure 15.3. The calcium action potential in a sea urchin egg.\\n\\nIn nature the initial depolarization that sets off an action potential is produced by the sperm. This is illustrated at point E. The membrane of the sperm contains an unusual type of calcium channel that is open even at the resting voltage. When the membrane of the sperm is added to that of the egg, the inward current through the sperm’s calcium channels depolarizes the plasma membrane of the egg by 20 mV to -50 mV, enough to cause the\\n---\\n# THE ACTION POTENTIAL\\n\\nThe critical number of the egg’s own voltage-gated calcium channels to open. The mechanism of the action potential takes over and the plasma membrane depolarizes rapidly to +10 mV. The graph has been cut off at the right-hand edge (where the voltage-gated calcium channels inactivate) where other mechanisms, not described here, take over and maintain a positive transmembrane voltage. The action potential has served its purpose in preventing second and subsequent sperm fusions in the 100 ms following the first fertilization.\\n\\n# THE VOLTAGE-GATED SODIUM CHANNEL IN NERVE CELLS\\n\\nThe voltage-gated calcium channel appeared early in evolution and is found in all eukaryotes and in a wide variety of cell types. Later on in evolution, when multicellular organisms arose, there was a need for a system that could send electrical signals rapidly over long distances within the body. This system uses sodium rather than calcium ions.\\n\\nThe Voltage-Gated Sodium Channel\\n\\nThe cells in multicellular animals that are specialized for rapid conduction have a second, voltage-gated channel that is selective for sodium instead of calcium ions. It operates like the voltage-gated calcium channel, but both opening and inactivation are faster.\\n\\nIn summary:\\n\\n1. When the transmembrane voltage is −70 mV, the voltage-gated sodium channel is gated shut.\\n2. When the plasma membrane is depolarized, the channel opens rapidly and then, after about 1 ms, inactivates.\\n3. After the channel has gone through this cycle, it must spend at least 1 ms with the transmembrane voltage at the resting voltage before it can be opened by a second depolarization.\\n\\n|Figure 15.4. The voltage-gated sodium channel.|Figure 15.4. The voltage-gated sodium channel.|\\n|---|\\n|depolarization|extracellular medium|\\n|repolarization|spontaneous after 1 ms|\\n|inactivation plug|Na+ Cytosol|\\n|repolarization| |\\n---\\n# THE VOLTAGE-GATED SODIUM CHANNEL IN NERVE CELLS\\n\\nExample 15.1 Chewing off the Inactivation Plug\\n\\nProteases are enzymes that hydrolyze the peptide bonds within proteins. In 1976 Emilio Rojas and Bernardo Rudy investigated the effect of introducing protease into squid axons. The membrane of squid axons contains voltage-gated sodium channels that, like ours, normally inactivate about 1 ms after they are opened by a depolarization. However, after introduction of the protease, depolarization caused the voltage-gated sodium channels to open and remain open indefinitely, although the channels would close if the membrane was repolarized. Back in 1976 Rojas and Rudy had no idea why this should be so. Now we can understand the result—the protease cuts the linker between the main part of the voltage-gated sodium channel and the inactivation plug. The plug then floats off and is not available to block the open channel.\\n\\nLike calcium, sodium is at a much higher concentration outside the cell than inside. In a mammal, the sodium concentration in the blood is 150 mmol liter-1, whereas in the cytosol it is 10 mmol liter-1. When voltage-gated sodium channels open, sodium ions rush into the cell carrying positive charge and depolarizing the plasma membrane. This then favors the opening of more voltage-gated sodium channels, and so on. This positive feedback produces a depolarization to +30 mV called the sodium action potential. Because the voltage-gated sodium channels inactivate so quickly, the sodium-based action potential lasts only for 1 ms.\\n\\nMedical Relevance: A Sodium “Channelopathy”\\n\\nEpilepsy, in which nerve cells in the brain fire action potentials in an uncontrolled fashion, is not uncommon in young children but often clears up as the child gets older. However, one form of childhood-onset epilepsy, termed GEFS+ for “generalized epilepsy with febrile seizures that extend beyond six years of age,” does not improve with age. Robyn Wallace and co-workers in Australia showed that this condition could be caused by a mutation that slows the rate at which the inactivation plug moves to block the voltage-gated sodium channel. The slow rate of inactivation makes it easier for stimuli to trigger action potentials in the nerve cells in the brain.\\n\\nAlthough enough sodium ions move into the cell to dramatically change the transmembrane voltage, the concentration of sodium ions inside the cell is increased only very slightly. The amount depends on the electrical capacitance of the cell membrane and the cell volume, but, for example, it can be calculated that a single sodium-based action potential increases the sodium concentration in the nerve cell axon by only 250 nmol liter-1.\\n---\\n# THE ACTION POTENTIAL\\n\\nElectrical Transmission down a Nerve Cell Axon Figures 15.5 and 15.6 show how one nerve cell, a pain receptor, uses sodium-based action potentials to signal to the central nervous system. The cell body is close to the spinal cord and extends an axon that branches to the spinal cord and out to the body. The particular pain receptor illustrated in Figure 15.5 sends its axon for almost a meter to the tip of a finger, an extraordinary distance for a single cell. Potentially damaging events are detected at the finger, and the message is passed on to another nerve cell (a pain relay cell) in the spinal cord. We will now explain how this function is performed.\\n\\nFigure 15.5 shows the transmembrane voltage of the pain receptor at the two axon terminals, one near the tip of the finger (green line) and one at the opposite end of the hand in warm air.\\n\\n| |10|-10|0|30|60 ms|\\n|---|---|---|---|---|---|\\n|-30| | | | | |\\n|-50| | | | |warm air|\\n|-70|transmembrane voltage| | | | |\\n\\nFigure 15.5. Mild heat stimulus is not felt as pain because the pain receptor is not depolarized to threshold.\\n---\\nTHE VOLTAGE-GATED SODIUM CHANNEL IN NERVE CELLS\\n\\n| |hand in|hot air|\\n|---|---|---|\\n|30| | |\\n|10| | |\\n|-10|0|30|60 ms|\\n|-30| | | |\\n|-50|transmembrane voltage| | |\\n|-70| | | |\\n\\nrepresents transmembrane voltage measured at proximal axon terminal\\n\\ncell body of pain receptor\\n\\npain relay nerve cell\\n\\nproximal axon terminal\\n\\naxon of pain receptor\\n\\nspinal cord\\n\\nhot air represents transmembrane voltage measured at distal axon terminal\\n\\nFigure 15.6. Stronger heat stimulus evokes a propagating action potential.\\n\\ncell, in the spinal cord (dashed black line). The axon terminal in the skin is the distal, or far away, terminal and that in the spinal cord is the proximal one. When the finger is passed through the warm air from a hand dryer, the heat causes the plasma membrane of the distal terminal to depolarize by 10 mV (continuous green line). This depolarization is not enough to open more than a very few voltage-gated sodium channels. Even in the warm air, the transmembrane voltage of the distal axon terminal is below threshold, and as soon as the finger leaves the warm air jet it sags back to -70 mV under the influence of the potassium channels. The axon is so long that the small voltage change at the distal terminal does not affect the voltage at the proximal terminal at all (dashed black line). No signal passes to the pain relay cell, and the subject feels no sensation of pain.\\n---\\n# THE ACTION POTENTIAL\\n\\nFigure 15.6 shows what happens if we pass our finger through hotter air. The depolarization caused by the heat is greater and this causes more voltage-gated sodium channels to open (continuous green line). Just as in the sea urchin egg at fertilization, the plasma membrane then depolarizes rapidly (in this case as more and more sodium enters) until the transmembrane voltage is +30 mV. After only 1 ms, the voltage-gated sodium channels inactivate, and the transmembrane voltage returns to −70 mV, due to the action of the potassium channels. At the proximal terminal (dashed black line), a meter away, there is also an action potential after a short delay. The signal then passes on to the brain and the person notices the hot air in time to pull the hand away before too much damage is caused. The pain receptor is one example of how we are quickly made aware of changes in our environment by sodium action potentials traveling rapidly along nerve cell axons.\\n\\nMyelination and Rapid Action Potential Transmission\\n\\nThe axon transmits the action potential so rapidly because for most of its length it is insulated by a fatty sheath called myelin made by glial cells (the glial cells outside the brain and spinal cord are often called Schwann cells). Figure 15.7 shows part of the axon of the pain receptor and its associated glial cells, each of which wraps around the axon to form an electrically insulating sheath. Only at short gaps called nodes is the membrane of the nerve cell exposed to extracellular fluid. Note that the vertical and horizontal scales of this diagram are completely different. The axon together with its myelin sheath is only 3 μm across, but the section of axon between nodes is 1 mm long, a large distance by normal cell standards. Between nodes the axon is an insulated electrical cable: just as voltage changes at one end of an insulated metal wire have an almost instantaneous effect on the voltage at the other end, so a change in transmembrane voltage at one node has an almost instantaneous effect at the next node along. The plasma membrane at the nodes has potassium and voltage-gated sodium channels and can therefore generate action potentials.\\n\\nExample 15.2 Local Anaesthetic, Overall Well-being\\n\\nNerve conduction is exploited when a dentist uses local anaesthetic. A patient feels pain from the drill because pain receptors in the tooth are depolarized and transmit action potentials toward the brain. The site at which they are being depolarized is inside the tooth and therefore inaccessible to drugs until the drill has made a hole. However, the axons of the pain receptors run through the gum. Local anaesthetics injected into the gum close to the axon bind to the nerve cell membrane at the node and prevent the opening of voltage-gated sodium channels. Drilling into the tooth still depolarizes the pain receptor membrane, and action potentials begin their journey toward the brain, but they cannot pass the injection site because the nodes there cannot generate an action potential. The patient therefore feels no pain.\\n\\nFigure 15.7 represents a snapshot of one brief instant in the operation of the nerve cell. The node on the left is generating an action potential. Voltage-gated sodium channels are open, sodium ions are flooding in, and the transmembrane voltage is +30 mV. Some of the charge flowing in is carried out again by potassium ions flowing through the potassium channels. However, some positive charge flows axially up and down the cytosol of the nerve\\n---\\nm\\nμ\\n3\\n\\n1 mm\\n\\nm\\nμ\\nnode3\\n\\nsodium channel\\n\\npotassium channel\\n\\naxial current flow\\n\\n|+K|30 mV|+|\\n|---|---|---|\\n|+| |+Na|\\n|Na| |Na|\\n\\ncytoplasm of nerve cell\\n\\nfatty sheath of myelin created by glial cell\\n\\nnucleus of glial cell\\n\\nFigure 15.7.\\n---\\n# THE ACTION POTENTIAL\\n\\ncell. Because myelin is an electrical insulator, this charge cannot leave the cytosol until the next node. At the node on the right the incoming charge depolarizes the plasma membrane to threshold, and voltage-gated sodium channels at this node open as well. Sodium rushes in, and this node generates an action potential. The action potential has jumped from node to node in the extremely short time of 50 μs. Now sodium ions will rush into the node on the right, current will pass axially up the interior of the axon to the next node up, and in a further 50 μs the action potential will have jumped to the next node, and so on, all the way up to the proximal terminal at an overall speed of 20 m s-1. This process, in which the action potential jumps from node to node, is named after the Latin for “to jump,” saltere, so it is saltatory conduction.\\n\\nThe combination of axon plus myelin sheath is called a nerve fiber. The nerve fiber shown in Figure 15.7 has an overall diameter of 3 μm. The human body contains myelinated nerve fibers with diameters ranging from 1 μm, conducting at 6 m s-1, to 10 μm, conducting at 60 m s-1. A number of axons are not myelinated and generally conduct more slowly than myelinated fibers. The axons of scent-sensitive nerve cells (page 350) are an example. Getting scent information from the nose to the brain as fast as possible is apparently not a significant advantage, given that a scent can take many seconds to waft from its source to our nose.\\n\\n# IN DEPTH 15.1 Frequency Coding in the Nervous System\\n\\nAt the distal axon terminal of the pain receptor, the intensity of the stimulus is represented by the amount of depolarization produced. The hotter the air blowing on the skin, the more positive the transmembrane voltage of the terminal becomes. Other sensory cells behave in the same way. For instance, in Chapter 16 we will meet the scent-sensitive nerve cells in the nose that detect smell chemicals. The higher the concentration of the smell chemical, the more these cells depolarize.\\n\\nHowever, the signal that passes toward the brain is an all-or-none action potential. Either the stimulus produces a depolarization that is below threshold, in which case no signal passes to the brain, or the membrane depolarizes beyond threshold, in which case an action potential is generated. How then is the brain told about the intensity of the stimulus?\\n\\nAction potentials continue to be produced and travel up toward the brain as long as the stimulus is maintained. The diagrams represent what happens when the hand is being held in a constant airstream. If the stimulus is strong, the membrane of the pain receptor depolarizes rapidly, so that after each action potential the threshold voltage is soon reached once more. Thus time between successive action potentials is short. The stronger the stimulus, the higher the frequency of action potentials generated.\\n\\nAmplitude modulation (AM) and frequency modulation (FM) are familiar terms from radio. Here we are seeing exactly the same two coding strategies in the nervous system. AM is used in the distal terminals of the pain receptor. The strength of stimulus is coded for by the amplitude of the depolarization. FM is used in almost all axons, including those of the pain receptor. The strength of stimulus is coded for by the frequency of action potentials, each of which has the same amplitude.\\n\\nMan-made systems use exactly the same strategy. On the control unit of a games console, for instance, the force with which the player presses on a button is coded in terms of the amplitude of a voltage change. This is then recoded in all-or-none digital pulses for transmission to the main console.\\n---\\n# SUMMARY\\n\\n| |0|1 second|0|1 second|\\n|---|---|---|---|---|\\n|mV| | | | |\\n|Transmembrane voltage (mV)|−70| |−70| |\\n\\nweak stimulus\\n\\n| |0|1 second|0|1 second|\\n|---|---|---|---|---|\\n|mV| | | | |\\n|Transmembrane voltage (mV)|−70| |−70| |\\n\\nstimulus just over threshold\\n\\n| |0|1 second|0|1 second|\\n|---|---|---|---|---|\\n|mV| | | | |\\n|Transmembrane voltage (mV)|−70| |−70| |\\n\\nstrong stimulus\\n\\n| |0|1 second|0|1 second|\\n|---|---|---|---|---|\\n|mV| | | | |\\n|Transmembrane voltage (mV)|−70| |−70| |\\n\\n# SUMMARY\\n\\n1. The voltage-gated calcium channel is shut at the resting voltage but opens upon depolarization. After about 100 ms the channel inactivates.\\n2. While the calcium channel is open, calcium ions pour into the cell down their electrochemical gradient. The mutual effect of current through the channel on transmembrane voltage, and of transmembrane voltage on current through the channel, constitutes a positive feedback system. If the membrane is initially depolarized to threshold, the positive feedback of the calcium current system ensures that depolarization continues in an all-or-nothing fashion. Repolarization occurs when the\\n---\\n# THE ACTION POTENTIAL\\n\\n1. The action potential is a rapid change in membrane potential caused by the opening and closing of voltage-gated ion channels.\\n\\n2. In cells expressing voltage-gated calcium channels, depolarization occurs when the channels open, allowing calcium ions to enter the cell. Repolarization happens when the calcium channel inactivates. The entire cycle of depolarization and repolarization is called an action potential.\\n\\n3. The same process occurs in cells expressing voltage-gated sodium channels. However, since these channels inactivate within about 1 ms, the action potential lasts only this long. Axons can transmit action potentials at speeds up to 60 m per second due to the insulation provided by myelin.\\n\\n4. Some axons are not myelinated, conducting action potentials more slowly than myelinated axons where high conduction speed is not required.\\n\\n# FURTHER READING\\n\\nAshcroft, F. M. 2000. Ion Channels and Disease. San Diego: Academic Press.\\n\\nLevitan, I. B., and Kaczmarek, L. K. 2002. The Neuron. New York: Oxford University Press.\\n\\n# REVIEW QUESTIONS\\n\\nFor each question, choose the ONE BEST answer or completion.\\n\\n1. A sea urchin egg is used in a voltage clamp experiment; pat is, pe value of pe transmembrane voltage is set by pe experimenter, not by pe properties of pe membrane. The transmembrane voltage is held at +10 mV for one minute. At pe end of pat time\\nA. pe voltage-gated calcium channels are all open.\\nB. pe voltage-gated calcium channels are all inactivated.\\nC. pe voltage-gated calcium channels have recovered from inactivation.\\nD. pe voltage-gated calcium channels are going prough cycles of opening and inactivation as pe cell generates repetitive action potentials.\\nE. any of answer A pough D above could occur, depending on pe density of potassium channels.\\n\\n2. In a calcium action potential\\nA. pe influx of calcium ions depolarizes pe plasma membrane.\\nB. depolarization of pe plasma membrane opens voltage-gated calcium channels.\\nC. open voltage-gated calcium channels carry an inward current of calcium ions.\\nD. an inward current of calcium ions adds positive charge to pe cytosol.\\nE. all of pe above occur.\\n\\n3. Action potentials in nerve cell axons last a much shorter time pan action potentials in sea urchin eggs because\\nA. sodium has only a single positive charge while calcium has two.\\nB. voltage-gated sodium channels are found only at pe nodes, while voltage-gated calcium channels are found over pe entire surface of pe egg cell.\\nC. sodium ions are removed rapidly from pe cytosol by pe Na+/K+ ATPase.\\nD. voltage-gated sodium channels inactivate more rapidly pan do voltage-gated calcium channels.\\nE. calcium has a higher atomic weight pan does sodium and perefore diffuses more slowly.\\n---\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n4. An action potential is being generated at pe node of a myelinated axon. Which of pe following statements is false?\\nA. Sodium is entering pe axon interior prough voltage-gated sodium channels.\\nB. Potassium is leaving pe axon interior prough potassium channels.\\nC. Calcium is entering pe axon interior prough voltage-gated calcium channels.\\nD. A net inward current is flowing across pe plasma membrane of pe node.\\nE. Current is leaving pe region of pe node axially, prough pe cytosol of pe nerve cell.\\n\\n5. The nodes of a myelinated axon are spaced a distance apart of approximately\\nA. 1 nm.\\nB. 1 μm.\\nC. 1 mm.\\nD. 1 cm.\\nE. 1 m.\\n\\n6. A sperm can trigger an action potential in a sea urchin egg because\\nA. its cytosol contains a lot of calcium ions pat, upon fertilization, rush into pe egg.\\nB. its membrane contains ion channels pat, upon fertilization, pass an inward current pat depolarizes pe egg plasma membrane.\\nC. its membrane contains ion channels pat, upon fertilization, pass an outward current pat repolarizes pe egg plasma membrane.\\nD. its cytosol contains a chemical pat tends to pop pe voltage-gated calcium channels into an open state.\\nE. its cytosol contains a protease pat modifies pe behavior of voltage-gated calcium channels in pe plasma membrane of pe egg.\\n\\n7. When a nerve cell axon is depolarized to preshold, pe flow of potassium prough potassium channels in pe plasma membrane is\\nA. outward, and of a greater absolute amplitude pan at pe resting voltage.\\nB. inward, and of a greater absolute amplitude pan at pe resting voltage.\\nC. outward, but of a smaller absolute amplitude pan at pe resting voltage.\\nD. inward, but of a smaller absolute amplitude pan at pe resting voltage.\\nE. zero.\\n\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n1. B. They are all inactivated. Upon depolarization they opened and inactivated, and since the transmembrane voltage was never returned to the resting value, the channels have not recovered from inactivation. Regarding answer D, note that the cell cannot generate repetitive action potentials because it is voltage clamped: The value of the transmembrane voltage is set by the experimenter, not by the properties of the membrane. For this reason the density of potassium channels (answer E) is irrelevant—the density of potassium channels will affect the current through those channels, but since the cell is voltage clamped, the current through potassium channels will have no effect on transmembrane voltage and therefore no effect on the behavior of the voltage-gated calcium channels.\\n\\n2. E. Answers A through D are true. The influx of calcium acts, through the voltage change it causes, to increase the influx of calcium—this is why the generation of an action potential is an example of positive feedback.\\n\\n3. D. Statements A, B, C, and E are all true in themselves but are not reasons why action potentials in nerve cell axons are shorter than those in sea urchin eggs. Concerning answer C, the sodium ions\\n---\\n# THE ACTION POTENTIAL\\n\\nThe very small amounts of sodium that do enter (see page 331) are removed over a timescale of seconds by the Na+/K+ ATPase. If the Na+/K+ ATPase is blocked by the toxin digitalis (page 271), a nerve cell can continue to generate action potentials as normal. Only after many thousands of action potentials have occurred in a digitalis-poisoned nerve cell will the cytosolic sodium concentration rise to a concentration sufficient to significantly affect the threshold for action potential generation.\\n\\n4. C. There are no voltage-gated calcium channels at the node. All the other statements are true: although some charge is being carried out of the axon at the node by potassium, the current carried by the inward movement of sodium is larger, so there is a net inward current. The excess charge is carried out of the node axially.\\n\\n5. C.\\n\\n6. B. Statements A, D, and E are untrue. Statement C is true in itself in that, like most cells, the sperm plasma membrane will contain some potassium channels that will pass an outward current, but as the answer states, the action of these will be to repolarize the egg membrane, not depolarize it. The depolarizing effect of the calcium influx through the sperm calcium channels, which are open even at the resting voltage, is greater.\\n\\n7. A. Potassium ions are usually attracted to the negative interior of the cell, and this force opposes the tendency of potassium ions to leave down their concentration gradient. Depolarization reduces the inward electrical force, so potassium ions leave at a greater rate than at rest. At the same time, voltage-gated sodium channels are carrying an inward current, but that movement is not relevant to the question.\\n---\\n# INTRACELLULAR SIGNALING\\n\\nThe behavior of cells is not constant. Cells need to be able to alter their behavior in response to internal changes or to external events, and the internal signaling mechanisms that allow them to do so are varied and complex. We will begin by continuing the story begun in Chapter 15 and explain how calcium ions carry a signal from the plasma membrane of nerve cells to vesicles deep within the cytosol. The rest of the chapter will describe some other very different methods of intracellular signaling.\\n\\n# CALCIUM\\n\\nCalcium ions are present at a very low concentration (about 100 nmol liter−1) in the cytosol of a resting cell. An enormous number of processes in many types of cells are activated when the concentration of calcium rises. Calcium can move into the cytosol from two sources: the extracellular medium or the endoplasmic reticulum.\\n\\nCalcium Can Enter from the Extracellular Medium\\n\\nIn Chapter 15, we saw how heating the hand causes action potentials to travel from the hand to the spinal cord along the axons of pain receptors. When the action potential reaches the proximal axon terminal, the amino acid glutamate is released onto the surface of another nerve cell, the pain relay cell. Glutamate is a transmitter that stimulates the pain relay nerve cell, so that the message that the finger is being damaged is passed toward the brain.\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright © 2004 by John Wiley & Sons, Inc.\\n---\\n# INTRACELLULAR SIGNALING\\n\\n|cell body of pain receptor|spinal cord pain relay nerve cell|\\n|---|---|\\n|to skin|proximal axon terminal|\\n\\n|potassium channels|voltage-gated sodium channels|\\n|---|---|\\n|voltage-gated calcium channels| |\\n\\n|[Ca 2+] = 100 nmol liter−1|100 nm gap|pain relay nerve cell|\\n|---|---|---|\\n|−70 mV| | |\\n|vesicles containing 100 mmol liter−1 glutamate| | |\\n\\n[Ca 2+] = 2 mmol liter−1\\n\\n|[Ca 2+] = 1 μmol liter−1|100 nm gap|pain relay nerve cell|\\n|---|---|---|\\n|+30 mV| | |\\n\\nexocytosis of glutamate\\n\\n[Ca 2+] = 2 mmol liter−1\\n\\nFigure 16.1. Calcium ions entering the cytosol from the extracellular fluid activate regulated exocytosis in the proximal axon terminal of the pain receptor nerve cell.\\n---\\nCALCIUM 343\\n\\nThe proximal axon terminal of the nerve cell is unmyelinated, so that the plasma membrane is exposed to the extracellular medium (Fig. 16.1). The membrane contains not only potassium channels and voltage-gated sodium channels, but also voltage-gated calcium channels. These are closed in a resting cell, but when an action potential travels in from the skin and depolarizes the plasma membrane of the proximal axon terminal, the voltage-gated calcium channels open. Calcium ions pour in, increasing their concentration in the cytosol by 10 times, from the normal concentration of 100 nmol liter -1 to 1 μmol liter -1. At the proximal axon terminal the cytosol of the nerve cell contains regulated exocytotic vesicles (page 229). In the case of the pain receptor, these vesicles are filled with sodium glutamate. In response to the increased concentration of cytosolic calcium the regulated exocytotic vesicles move to the plasma membrane and fuse with it, releasing their contents into the extracellular medium. The glutamate then diffuses across the gap to the pain relay cell, stimulating it (we will describe how on page 372). As the action potential in the axon terminal of the pain receptor cell is over in 1 ms, the voltage-gated calcium channels do not have time to inactivate. They simply return to the ready-to-open state and can be reopened immediately by the next action potential that arrives.\\n\\n✄✄✄ ▯▯▯\\n\\n✂ Example 16.1 ✁ Visualizing Calcium Signals\\n\\n✂✂ ✁✁\\n\\n(a)\\n\\n(b)\\n\\nCalcium signals in cells can be visualized by using dyes that fluoresce brightly when they bind calcium. The images in the figure show a cell body of a pain receptor cell that has been filled with a calcium indicator dye. In the first image, at rest, the cell is dim. The second image was acquired after 100 ms of depolarization. In this the edges of the cell are bright because calcium ions have entered through voltage-gated calcium channels. The bright object in the lower part of the cell is the nucleus.\\n\\nIn the process that we have described, calcium ions act as a link between the depolarization of the plasma membrane and the regulated exocytotic vesicles within the cytosol.\\n---\\n# INTRACELLULAR SIGNALING\\n\\nCalcium ions are intracellular messengers. Regulated exocytotic vesicles that are triggered by an increase of cytosolic calcium concentration were first discovered in nerve cells but are now known to be a feature of almost all cells.\\n\\nBefore we continue with the topic of intracellular messengers, we will review two general points raised by our description of the pain receptor proximal axon terminal. The first point is that different nerve cells release different transmitters. Glutamate is in the exocytotic vesicles of many nerve cells, but other nerve cells release other transmitters. The second point concerns nomenclature. Many nerve cells, like the pain receptor, have their axon terminals close to a second cell and release their transmitter onto it. In such cases the complete unit of axon terminal, gap, and the part of the second cell that receives the transmitter is called a synapse. The part of the axon terminal that releases transmitter is called a presynaptic terminal, and the cell upon which the transmitter is released is called the postsynaptic cell. Many nerve cells do not come close enough to a specific second cell to form a synapse but simply release transmitter from their axon terminals into the extracellular medium.\\n\\nCalcium Can Be Released from the Endoplasmic Reticulum\\n\\nCells may show an increase of cytosolic calcium not because of an action potential but because of the appearance of a transmitter or other chemical in the extracellular medium. The presence of the chemical is detected by integral membrane proteins, each one a receptor that recognizes a particular chemical with high affinity. These receptors then participate in a more general mechanism, the end result of which is the release of calcium ions from the smooth endoplasmic reticulum into the cytosol. This mechanism will be illustrated with the particular example of blood platelets. We will then discuss how much of the mechanism is general to a wider range of cells.\\n\\nPlatelets are common in the blood. They are small fragments of cells and contain no nucleus, but they do have a plasma membrane and some endoplasmic reticulum. Blood platelets use the release of calcium from the endoplasmic reticulum as one step in the mechanism of blood clotting. Two new mechanisms are involved in calcium release from the endoplasmic reticulum. The inositol trisphosphate-gated calcium channel, like the voltage-gated calcium channel, allows only calcium ions to pass. Most of the time its gate is shut, and no ions flow. It is not opened by a change in the voltage across the endoplasmic reticulum membrane. Instead, when the intracellular solute inositol trisphosphate (IP3 for short) binds to the cytosolic face of the channel, the channel changes to an open shape.\\n\\nThe second new mechanism that we must describe is the one that makes IP3. If a blood vessel is cut, cytosol from damaged cells at the edge of the cut can leak into the blood. The appearance of solutes that normally are found only inside cells is a sure sign that damage has occurred. Adenosine diphosphate (ADP) is one such solute, and it acts to stimulate platelets, causing them to begin a blood clot to help plug the damaged vessel. The plasma membrane of the platelet contains a protein receptor that binds ADP, that is, ADP is its ligand. When the ADP has bound, the receptor, which is free to move in the plasma membrane, becomes a guanine nucleotide exchange factor for a trimeric G protein called Gq. Like the GTPases Ran, ARF, and Rab that we have met earlier, trimeric G proteins are GTPases that activate target proteins when they have GTP bound, but turn themselves off by hydrolyzing the GTP to GDP. Trimeric G proteins have a slight additional complication in that, as the name indicates.\\n---\\n# inositoltrisphosphate\\n\\n|DAG| | |\\n|---|---|---|\\n|OH| | |\\n\\nin platelets.\\n\\nβ\\n\\n|2| |\\n|---|---|\\n|PIP| | |\\n|O| | | |\\n|O| | | |\\n|O| | |\\n\\nblood platelets and hence phospholipase C\\n\\nq\\n\\nADP released from damaged cells in vessel wall\\n\\nα GTP γ phospholipase\\n\\nβ GDP\\n\\nADP from damaged cells activates G q\\n\\nGDPα G β γ trimeric G protein\\n\\nGTP ADP receptor\\n\\nFigure 16.2.\\n\\n345\\n---\\n# INTRACELLULAR SIGNALING\\n\\nThe inositol trisphosphate-gated calcium channel receptor is a calcium-selective channel in the membrane of the endoplasmic reticulum. An increase of cytosolic calcium concentration in platelets makes them sticky, initiating blood clotting.\\n\\nThey are composed of three subunits. The α subunit is homologous to the GTPases we have met before, while the β and γ subunits dissociate from the α subunit when it has GTP bound and only reassociate when the GTP has been hydrolyzed. GTP-loaded Gq activates the β isoform of an enzyme called phosphoinositide phospholipase C, which we will call phospholipase C or just PLC. (The capital C refers to the bond hydrolyzed by the enzyme; A, B, and D phospholipases also exist, but we will not meet them in this book). PLC specifically hydrolyzes phosphatidylinositol bisphosphate (PIP2 for short), a phospholipid in the plasma membrane that has phosphorylated inositol as its polar head group. Hydrolysis releases the IP3 to diffuse freely in the cytosol, leaving behind the glycerol backbone with its attached fatty acids: this residual lipid is called diacylglycerol, or DAG. As it diffuses through the cytosol, IP3 reaches the endoplasmic reticulum and binds to the inositol trisphosphate-gated calcium channels. The inositol trisphosphate-gated calcium\\n---\\nglucose\\n\\nadenylatecyclase\\n\\ncAMP\\n\\n|ATP|activate production of glucose ready for use by glycolysis and hence mitochondria|\\n|---|---|\\n|GTP|glycogen muscle cell|\\n\\n-adrenergic receptor β adrenaline\\n\\nactivate enzymes of Krebs cycle in mitochondria\\n\\nendoplasmic reticulum Ca\\n\\nADP\\n\\nATP porin Calcium and cyclic-AMP activate distinct but overlapping sets of target processes in skeletal muscle cells.\\n\\nmitochondrial calcium channel Figure 16.4.\\n\\ncytoskeleton from nerve stimulus activate force generation by\\n---\\n# INTRACELLULAR SIGNALING\\n\\nchannels open, and calcium ions pour out of the endoplasmic reticulum into the cytosol. The increase of calcium concentration causes the platelet to change shape and to become very sticky, so platelets begin to clump together in a clot.\\n\\nPhosphatidylinositol bisphosphate, Gq, PLCβ, and the inositol trisphosphate-gated calcium channel are found in almost all eukaryotic cells, but the distribution of the ADP receptor is more restricted. Only cells that express the ADP receptor will show an increase of cytosolic calcium concentration in response to ADP. Other cells respond to other chemicals. Each produces a receptor specific for that chemical, which then activates Gq. Over 100 such receptors are known. We will meet two more in the next chapter.\\n\\nProcesses Activated by Cytosolic Calcium Are Extremely Diverse\\n\\nThe targets activated by an increase of cytosolic calcium concentration differ between different cells. An increase of calcium is a crude signal that says “do it” but contains no information about what the cell should do. This depends on what the cell was designed to do. Cells and cell regions designed for regulated exocytosis (e.g., salivary gland cells and axon terminals) exocytose when cytosolic calcium increases. Cells designed to contract (e.g., muscle cells) contract when calcium increases, and so on. In each case, the calcium ions bind to a calcium-binding protein; the calcium ion–calcium-binding protein complex activates the target process.\\n\\nIn skeletal muscle cells (page 14) a transmitter released from nerve axon terminals leads to the escape of calcium from the endoplasmic reticulum (Fig. 16.4). This activates several processes. First, calcium ions bind to a protein called troponin that is attached to the cytoskeleton. This causes the cytoskeleton to contract, using the energy released by ATP hydrolysis to do mechanical work (page 393). Second, the calcium binds to the protein calmodulin, which in turn activates glycogen phosphorylase kinase and hence glycogen breakdown as part of the feedforward control of energy metabolism (page 305).\\n\\nMitochondria are also affected by calcium, which passes down its electrochemical gradient from the cytosol into the mitochondrial matrix through a calcium channel. Once there, calcium stimulates the mitochondria to increase the production of NADH and ATP (page 305).\\n\\nIN DEPTH 16.1 Ryanodine Receptors\\n\\nMany cells contain a second type of calcium channel in the membrane of the endoplasmic reticulum. This channel was initially distinguished from the IP3 receptor by the fact that it bound the plant alkaloid ryanodine, so it was called the ryanodine receptor. Ryanodine receptors were first identified in skeletal muscle. In these cells they are physically linked to voltage-gated calcium channels in the plasma membrane (left-hand side of diagram). When the voltage-gated calcium channels switch to their open configuration, they induce the ryanodine receptors below them to switch into an open configuration, allowing calcium to flow out of the smooth endoplasmic reticulum into the cytosol.\\n\\nNo such direct linkage between ryanodine receptors and plasma membrane channels is found in other cell types. Instead, ryanodine receptors are opened when the concentration of calcium ions in the cytosol rises above a critical level. For example, in heart muscle (right-hand side of diagram) depolarization causes voltage-gated calcium channels in the plasma membrane to open. The calcium ions that enter through this channel bind to the cytosolic aspect of the ryanodine receptor, causing it to open too, allowing calcium to flow out of the smooth endoplasmic reticulum into the cytosol.\\n---\\n|cardiac muscle|depolarization|repolarization|\\n|---|---|---|\\n|ryanodine receptor|voltage gated calcium channel| |\\n|plasmalemma|reticulum|endoplasmic|\\n|extracellular medium|membrane of Ca|membrane of Ca|\\n|skeletal muscle cell|depolarization|repolarization|\\n|ryanodine receptor|voltage gated calcium channel| |\\n|lumen of endoplasmic reticulum|lumen of endoplasmic reticulum|lumen of endoplasmic reticulum|\\n---\\n350\\n\\n# INTRACELLULAR SIGNALING\\n\\nThe same simple intracellular messenger, calcium, has many different actions inside the skeletal muscle cell. Under its influence the cytoskeleton begins to use ATP, glycogen phosphorylase releases more glucose, and the mitochondria produce more ATP. The skeletal muscle cell is an example of how diverse mechanisms inside a cell can be integrated by the action of intracellular messengers.\\n\\nExample 16.2 Training for Anaerobic Respiration\\n\\nMuscle cells that are contracted for long periods, cutting off the blood supply—such as those in the arms of rock climbers—have a much enhanced capacity for anaerobic respiration compared to those that are used briefly and then relaxed (such as those in the arms of baseball pitchers). This occurs because the calcium signals that activate muscle contraction (page 393) also act to increase the transcription of the genes coding for glycolysis (via calcineurin and NFAT, page 222).\\n\\n# Return of Calcium to Resting Levels\\n\\nAs soon as a stimulus, be it depolarization or extracellular chemical, disappears, cytosolic calcium concentration falls again. Calcium ions are pumped up their electrochemical gradients from the cytosol into the extracellular medium or into the endoplasmic reticulum by two carriers. The Ca2+ ATPase (page 318) uses the energy released by ATP hydrolysis to move calcium ions up their electrochemical gradient out of the cytosol, while the Na+/Ca2+ exchanger (page 317) uses the energy released by sodium ion movement into the cytosol to do the same job.\\n\\nCYCLIC ADENOSINE MONOPHOSPHATE\\n\\nWe have already met the nucleotide cyclic adenosine monophosphate, or cAMP (Fig. 16.5), in the context of the regulation of the lac operon (page 114). (It is worth mentioning that cAMP is always pronounced “cy-clic A-M-P,” with five syllables.) In eukaryotes, cAMP is also important and acts as an intracellular messenger in a great many cells, including the scent-sensitive nerve cells in our nose (Fig. 16.6). These cells have their cell bodies in the skin of the air passages in the nose. Each cell sends an axon into the brain, and shorter processes, called dendrites, into the mucus lining the air passages. Scent-sensitive nerve cells are stimulated by scents in the air. Particular chemicals in the air stimulate these cells because the cells have protein receptors that specifically bind the scent (Fig. 16.7). When the scent binds, the receptor becomes a guanine nucleotide exchange factor for a trimeric G protein called Gs. GTP-loaded Gs activates an enzyme called adenylate cyclase that converts ATP to cAMP. The next stage in the detection of scents is a channel in the plasma membrane called the cAMP-gated channel. Like the inositol trisphosphate-gated calcium channel, the cAMP-gated channel is opened by a cytosolic solute, in this case cAMP. When the channel is open, it allows sodium and potassium ions to pass through. The electrochemical gradient pushing sodium ions into the cell is much greater than the electrochemical gradient pushing potassium ions out of the cell. Thus, when the cAMP-gated channel opens, sodium ions pour in, carrying their positive charge and depolarizing the plasma membrane. The plasma membrane also contains voltage-gated sodium channels. Thus, when enough cAMP-gated channels open, the transmembrane voltage reaches threshold for generation of sodium action potentials. These then propagate along the axon to the brain, and the person becomes aware of the particular scent for which that scent-sensitive nerve cell was specific.\\n---\\nFigure 16.5. Cyclic adenosine monophosphate, also called cyclic-AMP or just cAMP.\\n\\n|H| | |N|C| | |C| |N| |C| | |H|\\n|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\\n| |N| |C| |N| |C| |N| |C| | | |\\n| | | | | | | | | | | | | | | |\\n| | | |5′| | |O| |CH2| |O| | | | |\\n| | | |C| | |H| | | |H| |C| | |\\n| | | |H| | |C|3′| |C| |H| | | |\\n|O| | |P| | |O| | | |OH| | | | |\\n| | | | | | | | | | | | | | | |\\n\\nFigure 16.6. Scent-sensitive nerve cells send axons to the brain.\\n---\\n# INTRACELLULAR SIGNALING\\n\\n|cAMP binding site|\\n|---|\\n|dendrite of scent-sensitive nerve cell|\\n|scent chemical|\\n|extracellular fluid|\\n|chemical in air|\\n|Na+|\\n|cAMP|Na+|cAMP binds Na+|\\n|ATP|Na+|\\n|cell body of scent-sensitive nerve cell|\\n|scent chemical|\\n|receptor for scent chemical|\\n|GTP|GTP α|cAMP|\\n|β|GDP|\\n|γ α|β γ|adenylate cyclase ATP|\\n|trimeric G protein Gs|\\n\\nFigure 16.7. Scent chemicals activate Gs and hence adenylate cyclase in scent-sensitive nerve cells. cAMP then opens a nonselective cation channel in the plasma membrane.\\n\\nGs and adenylate cyclase are found in many cells, but only scent-sensitive nerve cells are sensitive to scents, because only they have specific scent receptors in their plasma membranes. Other cells that use cAMP as an intracellular messenger are sensitive to other specific chemicals because each makes a receptor that binds the chemical, whatever it may be, and then activates Gs. Once the stimulating chemical has gone, the cAMP concentration\\n---\\n# BIOCHEMICAL SIGNALING\\n\\nreturns to resting levels. The enzyme cAMP phosphodiesterase hydrolyzes cAMP to AMP, which is inactive at cAMP-gated channels and other cAMP-binding proteins.\\n\\nMost of the symptoms of the deadly disease cholera are caused by a toxin released by the gut bacterium Vibrio cholera. The toxin is an enzyme that enters the cytosol of the cells lining the gut and attaches an ADP ribosyl group to the catalytic domain of G , preventing it from hydrolyzing GTP. G is therefore locked in the active state, and activates adenylate cyclase nonstop. The cAMP concentration in the cytosol then shoots up. Ion channels in the plasma membrane are opened by the increase of cAMP, allowing ions to leak from the cells into the gut contents. If untreated this loss of ions, and of the water that accompanies them, leads to death from dehydration.\\n\\n# CYCLIC GUANOSINE MONOPHOSPHATE\\n\\nAnother nucleotide that acts as a intracellular messenger is cGMP. In light-sensitive nerve cells called photoreceptors, cGMP plays a role like that of cAMP in scent-sensitive nerve cells. In the dark, the enzyme guanylate cyclase makes cGMP from GTP. cGMP binds to and opens a channel that allows sodium and potassium ions to pass, so that in the dark the photoreceptor is depolarized to about −40 mV because of the constant influx of sodium ions through the cGMP-gated channel. In the light the concentration of cGMP in the cytosol falls, and the transmembrane voltage changes to the more typical resting voltage of −70 mV. The changing transmembrane voltage is transmitted to other nerve cells, making us aware of the pattern of light and dark.\\n\\n# MULTIPLE MESSENGERS\\n\\nMany cells use more than one intracellular messenger at once. Skeletal muscle cells are a good example (Fig. 16.4). In the excitement before a race, the runner’s adrenal glands release adrenaline into the blood. This binds to a receptor on skeletal muscle cells called the β-adrenergic receptor. The complex of adrenaline plus β-adrenergic receptor can now activate G s and hence adenylate cyclase. This in turn generates cAMP, which activates cAMP-dependent protein kinase. This enzyme, which is given the short name of protein kinase A, phosphorylates glycogen phosphorylase kinase and turns on the latter enzyme even when cytosolic calcium is low (see Fig. 13.15 on page 305). The end result is that even before the runner begins to run, the muscles break down glycogen and make the glucose-6-phosphate (page 288) they will need once the race begins.\\n\\n# BIOCHEMICAL SIGNALING\\n\\nReceptor Tyrosine Kinases and the MAP Kinase Cascade When platelets are stimulated, they not only help the blood to clot, but also trigger the damaged blood vessel to repair itself. They do this by regulated exocytosis of a protein called platelet-derived growth factor (PDGF) (Fig. 16.8). The plasma membranes of the cells of the blood vessel contain receptors for this protein. PDGF receptors (Fig. 9.15 on page 198) comprise an extracellular domain, which binds PDGF, a single polypeptide chain that crosses the plasma membrane, and a cytosolic domain with tyrosine kinase activity. The PDGF receptor is therefore able to phosphorylate tyrosine residues on other proteins.\\n---\\n# Figure 16.8\\n\\n|Ras|MAPKK|MAPK|\\n|---|---|---|\\n| |GTP|MAPKKK|\\n|PDGF|SOS|+|=|GDP|MAPKK|MAPK|\\n|mutual phosphorylation|MAPK factors| |\\n| |transcription|phosphorylates|\\n|kinase site|Grb2| |\\n| |SH2 domain| |\\n\\nThe PDGF receptor, like other growth factor receptors, activates the GTPase Ras and therefore the MAP kinase cascade. mRNA for cell division proteins required.\\n\\nPDGF receptor residue tyrosine\\n---\\n# BIOCHEMICAL SIGNALING\\n\\nIn the absence of PDGF, the PDGF receptor does not come in contact with proteins to phosphorylate. PDGF can bind to two receptor molecules, drawing them close to each other and allowing them to phosphorylate each other on tyrosine residues.\\n\\nA number of cytoplasmic proteins have a domain called SH2 that is just the right shape to stick to phosphorylated tyrosine. At the bottom of a deep pocket in the protein surface is a positively charged arginine. Although proteins can be phosphorylated on amino acids other than tyrosine, only tyrosine is long enough to insert down the pocket so that the negative phosphate can stick to the positive arginine. Proteins with SH2 domains therefore stick to dimerized PDGF receptors. In contrast, SH2 domains do not stick to solitary PDGF receptors whose tyrosines do not carry the negatively charged phosphate group.\\n\\nOne protein that has an SH2 domain is called growth factor receptor binding protein number 2 (Grb2). Grb2 has no catalytic function but recruits a second protein called SOS, and SOS is a guanine nucleotide exchange factor for a GTPase called Ras, allowing it to discard GDP and bind GTP. Active Ras turns on a chain of protein kinases, each of which phosphorylates and hence activates the next, culminating in a kinase called mitogen-associated protein kinase (MAP kinase or MAPK). The kinase that phosphorylates MAP kinase is called MAP kinase kinase (MAPKK) while the one that phosphorylates MAPKK is called MAP kinase kinase kinase (MAPKKK). MAPKKK is activated by Ras.\\n\\nThe word mitogen means a chemical that tends to cause mitosis, reflecting the fact that MAP kinase is activated by transmitters such as PDGF that turn on cell division. When MAP kinase is phosphorylated, it moves to the nucleus and phosphorylates transcription factors that in turn stimulate the transcription of the genes for cyclin D (page 413) and other proteins that are required for DNA synthesis and cell division.\\n\\nPlatelet-derived growth factor is one of many growth factors. All work in much the same way: Their receptors are tyrosine kinases that are triggered to dimerize and phosphorylate their partners upon tyrosine when the growth factor binds. The general term for this type of receptor is receptor tyrosine kinase. Phosphorylated tyrosine then recruits proteins with SH2 domains including Grb2, which in turn allows activation of Ras and the MAP kinase pathway, leading to DNA synthesis and cell division.\\n\\n|Medical Relevance|Blocking Growth Factor Receptors|\\n|---|---|\\n|Active growth factor receptors cause cell division, by activating the MAP kinase pathway, and keep cells alive, by activating PKB (page 417). Turning off growth factor receptors therefore tends both to stop cells dividing and to kill them. The drug trastuzumab is effective in slowing down the progression of breast cancer because it prevents a growth factor, called epidermal growth factor (EGF), from binding to its receptor tyrosine kinase. This slows cell division and promotes cell death in the cancer cells.| |\\n\\n|Medical Relevance|R115777 Blocks Ras|\\n|---|---|\\n|Like other GTPases, Ras turns itself off by hydrolyzing its bound GTP. Mutant forms of Ras without GTPase activity are therefore constitutively active: they are always in the on state and will therefore be activating the pathway that terminates in MAP kinase and cell division at all times, even in the absence of growth factor. Such mutant forms of Ras are found in about 20% of all human cancers. The drug R115777 prevents Ras from activating MAPKKK and should therefore help stop the uncontrolled cell division that is characteristic of cancer cells.| |\\n---\\nINTRACELLULAR SIGNALING\\n\\n| |=| |=+ +| |=| |+ +| |P−| |P=|P =| |P =|P =| |P =|P =|P =|\\n|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\\n|phosphorylation|of PLCγ| | | | |Active PLCγ| | | | | | | | | | | | | |\\n| | | | | | |hydrolyses PIP|2| | | | | | | | | | | | |\\n\\nFigure 16.9. The PDGF receptor, like other growth factor receptors, phosphorylates and hence activates phospholipase Cγ.\\n\\nGrowth Factors Can Trigger a Calcium Signal\\n\\nA second protein that contains an SH2 domain and that is therefore recruited to phosphorylated receptor tyrosine kinases is another isoform of phosphoinositide phospholipase C, PLCγ. Binding to the phosphorylated tyrosine holds PLCγ at the growth factor receptor long enough for it itself to be phosphorylated (Fig. 16.9), and this activates its enzymatic action. Active PLCγ hydrolyzes PIP 2 into diacylglycerol and inositol trisphosphate, and in turn inositol trisphosphate triggers the release of calcium from the smooth endoplasmic reticulum.\\n\\nProtein Kinase B and the Glucose Transporter: How Insulin Works\\n\\nFigure 16.10 shows the insulin receptor. Like other receptor tyrosine kinases, the insulin receptor has an extracellular domain that can bind the transmitter, in this case the protein insulin, a single polypeptide chain that crosses the plasma membrane, and a cytosolic domain with tyrosine kinase activity. Unlike growth factor receptors, the insulin receptor exists as a dimer even in the absence of its ligand, insulin. When insulin binds, the shape and orientation of the individual insulin receptors change a little, and this allows each receptor to phosphorylate its partner upon tyrosine. An associated protein called the insulin receptor substrate number 1 (IRS-1) is also phosphorylated on tyrosine. Proteins with SH2 domains are therefore recruited, either to the phosphorylated tyrosines on the insulin receptors themselves or to the phosphorylated tyrosines on IRS-1. The most important protein to be recruited is phosphoinositide 3-kinase (PI 3-kinase). Binding to the phosphorylated tyrosine holds PI 3-kinase close to the insulin receptor long enough for it to\\n---\\n# BIOCHEMICAL SIGNALING\\n\\ninsulin\\n\\ninsulin\\n\\ninsulin receptor\\n\\n|kinase site|kinase site|P −|P −|\\n|---|---|---|\\n|P=|=|IRS-1|PIP 3|\\n|phosphorylation of|=|PI-3 kinase|Active PI-3 kinase|\\n|PI-3 kinase| | |phosphorylates PIP 2|\\n\\nFigure 16.10. The insulin receptor phosphorylates and hence activates PI 3-kinase.\\n\\nbecome phosphorylated, and this activates its enzymatic action. PI 3-kinase operates on\\nthe same substrate, PIP 2, that PLC does, but instead of hydrolyzing it, PI 3-kinase adds\\nanother phosphate group to the inositol head group, generating the intensely charged lipid\\nphosphatidylinositol trisphosphate (PIP 3).\\n\\nJust as phosphotyrosine is bound by a wide variety of proteins that contain SH2 domains,\\nso highly phosphorylated inositols, such as those in PIP 3, are bound by a domain called the\\nPH domain found on many proteins. Most important among these is a protein kinase called\\nprotein kinase B (PKB). PKB is itself activated by phosphorylation, but the kinase that does\\nthis is located at the plasma membrane and therefore only gets a chance to phosphorylate\\nPKB when PKB is held at the plasma membrane through its binding to PIP 3 (Fig. 16.11).\\n\\nIn many cell types, particularly fat cells and muscle cells, the final stage in trafficking\\n(page 226) of the glucose carrier (page 316) from the Golgi apparatus to the plasma mem-\\nbrane requires active PKB. At the same time, glucose carriers are being endocytosed and\\nare only returned to the plasma membrane if PKB is active. When we eat a large meal,\\ninsulin concentrations in the blood increase. Activation of the insulin receptor therefore\\ncauses an increase in the activation of PKB and hence a translocation of glucose carriers to\\nthe plasma membrane. This allows muscle and fat cells to take up large amounts of glucose\\nfrom the extracellular medium. Muscle cells convert the glucose to glycogen (page 298);\\nfat cells convert the glucose to fat (page 300). The action of PKB on a protein called BAD\\nthat is shown in Figure 16.11 will be described in Chapter 19.\\n\\n# Crosstalk—Signaling Pathways or Signaling Webs?\\n\\nCell biologists often talk of signaling pathways in which a transmitter activates a chain of\\nevents culminating in an effect. The vertical arrows in Figure 16.12 show four signaling\\npathways that we have discussed in this chapter. We have already seen one example of\\n---\\n# INTRACELLULAR SIGNALING\\n\\n|serine and threonine|residues|\\n|---|---|\\n|pH|=|=| | |kinase specific|\\n|domain|P|P|=|P|for PKB|\\n\\n|Protein kinase B|=|=| | |+|=|+| | |+| | |+|\\n|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\\n|Active| | | | | |PKB| | | | | |+| |\\n| | | | | | | | | | | | |14-3-3| |\\n| | | | | | | | | | | | |glucose| |\\n| | | | | | | | | | | | |protein| |\\n| | | | | | | | | | | | |carrier| |\\n| | | | | | | | | | | | |cytochrome C| |\\n\\nMany other cellular events\\n\\nFigure 16.11. PIP3 recruits protein kinase B (PKB) to the plasma membrane, where it is activated. Active PKB has many effects including exocytosis of vesicles containing glucose carriers and inactivation of BAD.\\n\\nHow these pathways can interact: growth factor receptors can trigger a calcium signal by activating phospholipase Cγ. In fact, the pathways can interact in many ways and at many levels. The green arrows in Figure 16.12 show some of the most important. PI 3-kinase can bind to phosphotyrosine on growth factor receptors via its SH2 domain and be activated by phosphorylation, thus growth factors, as well as insulin, will activate protein kinase B (arrow X in Fig. 16.12). In Chapter 19 we will see that this process is literally vital for our cells—if it stops, they die (page 417).\\n\\nMany of the actions of cAMP occur through the phosphorylation of other proteins by protein kinase A. In particular, cAMP can activate transcription of many eukaryotic genes (arrow Y in Fig. 16.12), but it does so by causing the phosphorylation of transcription factors, and hence activating them. This is in sharp contrast to the cAMP-CAP system in Escherichia coli (page 114).\\n\\nCalcium ions can activate a number of protein kinases, notably protein kinase C and calcium-calmodulin–activated protein kinase. These kinases phosphorylate many of the same targets as protein kinase A and hence can activate the same downstream events (arrows Z in Fig. 16.12). Paradoxically, calcium-calmodulin also activates the phosphatase calcineurin, which dephosphorylates and hence activates the transcription factor NFAT (page 222). The more we know about intracellular signaling systems, the more they come to resemble a web of interactions rather than a set of discrete pathways.\\n---\\n# SUMMARY\\n\\n|agonist|agonist|\\n|---|---|\\n|insulin receptor|growth factor receptor|\\n|insulin receptor|growth factor receptor|\\n|X| |\\n|PI3 kinase|Ras|Adenylate cyclase|Phospholipase C γ|Phospholipase C β|\\n|cAMP|Ca ++|\\n|Protein kinase B|MAP kinase|Protein kinase A|Ca ++ activated protein kinases and phosphatases|\\n|targets 1 (e.g. glucose carrier)|targets 2 (e.g. gene transcription)|targets 3 (e.g. ion channels)|targets 4 (e.g. regulated exocytosis)|\\n\\nFigure 16.12. Interactions of signaling pathways.\\n\\n# SUMMARY\\n\\n1. An intracellular messenger is an intracellular solute whose concentration changes in response to cell stimulation; it activates or modulates a variety of cellular processes. The most important intracellular messengers are calcium ions, cyclic adenosine monophosphate (cAMP), and cyclic guanosine monophosphate (cGMP).\\n\\n2. The increased cytosolic calcium may be derived from two sources: the extracellular fluid and the smooth endoplasmic reticulum.\\n\\n3. A complex cascade of reactions causes release of calcium from the smooth endoplasmic reticulum in response to stimulation of the cell. Binding of extracellular chemical to a cell surface receptor activates the trimeric G protein G q, which in turn activates the enzyme phospholipase Cβ, which cleaves the hydrophilic head group inositol trisphosphate (IP 3) from the lipid phosphatidylinositol bisphosphate. Inositol trisphosphate then binds to and opens a calcium channel in the membrane of the endoplasmic reticulum.\\n---\\n# INTRACELLULAR SIGNALING\\n\\n4. A different set of receptors activates Gs and hence adenylate cyclase, which makes cyclic AMP from ATP. Many of the actions of cAMP are mediated by cAMP-dependent protein kinase (protein kinase A).\\n\\n5. The majority of transmitters are released from cells by exocytosis induced by an increase of cytosolic calcium concentration.\\n\\n6. Many nerve cells have their axon terminals close to a second cell, and they release their transmitter onto it. The complete unit of axon terminal, gap, and the part of the cell that receives the transmitter is called a synapse.\\n\\n7. Receptor tyrosine kinases are caused to dimerize when their ligand binds. This allows each partner to phosphorylate the other upon tyrosine; this in turn recruits proteins containing SH2 domains.\\n\\n8. Grb2 is an SH2-domain-containing protein that serves to bring together SOS and Ras, allowing activation of Ras and therefore activation of the MAP kinase pathway. This culminates in the transcription of genes necessary for DNA synthesis and cell division.\\n\\n9. Phospholipase Cγ and phosphoinositide 3-kinase, two enzymes that act on the membrane lipid phosphatidylinositol bisphosphate (PIP2), both have SH2 domains and are both recruited to, and phosphorylated by, receptor tyrosine kinases: this activates them.\\n\\n10. Protein kinase B is recruited to the plasma membrane by PIP3; this allows it to be phosphorylated, which activates it.\\n\\n11. In many cells including fat and skeletal muscle, glucose transporters only appear at the plasma membrane when PKB is kept active through the activation of the insulin receptor.\\n\\n# FURTHER READING\\n\\nDownward, J. 1998. Mechanisms and consequences of activation of protein kinase B/Akt. Curr. Opin. Cell. Biol. 10: 262–267.\\n\\nMarshall, C. J. 1996. Ras effectors. Curr. Opin. Cell Biol. 8: 197–204.\\n\\nToescu E. C., and Verkhratsky, A. 1998. Integrative Aspects of Calcium Signalling. New York: Plenum Press.\\n\\n# REVIEW QUESTIONS\\n\\nFor each question, choose the ONE BEST answer or completion.\\n\\n1. Which of the following does not exist?\\n\\n- A. Protein kinase A\\n- B. Protein kinase B\\n- C. Phospholipase Cα\\n---\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n|1.|C. While protein kinase A, protein kinase B, phospholipase Cβ, and phospholipase Cγ are enzymes with important roles in signaling, there is no enzyme with the name phospholipase Cα.|\\n|---|---|\\n|2.|E. Protein kinases transfer the γ -phosphate group of ATP to a protein. Thus all protein kinases (including the PDGF receptor and the insulin receptor, both of which are protein kinases) are|\\n\\n# 1.\\n\\n|A.|Protein kinase A|\\n|---|---|\\n|B.|Protein kinase B|\\n|C.|Phospholipase Cα|\\n|D.|Phospholipase Cβ|\\n|E.|Phospholipase Cγ|\\n\\n# 2.\\n\\nWhich of the following is an enzyme, one of whose substrates is a nucleotide triphosphate?\\n\\n|A.|The PDGF receptor|\\n|---|---|\\n|B.|The insulin receptor|\\n|C.|Ras|\\n|D.|Protein kinase A|\\n|E.|All of the above|\\n\\n# 3.\\n\\nArrival of a scent chemical at a scent-sensitive nerve cell causes that cell to\\n\\n|A.|activate adenylate cyclase.|\\n|---|---|\\n|B.|open cAMP-gated channels in the plasma membrane.|\\n|C.|open voltage-gated sodium channels in the plasma membrane.|\\n|D.|generate action potentials.|\\n|E.|all of the above.|\\n\\n# 4.\\n\\nWhich of the following statements is false?\\n\\n|A.|An increase of cytosolic cAMP concentration can activate a protein kinase.|\\n|---|---|\\n|B.|An increase of cytosolic calcium concentration can activate a protein kinase.|\\n|C.|Dimerization of growth factor receptors can activate a protein kinase.|\\n|D.|Appearance of PIP3 at the plasma membrane can activate a protein kinase.|\\n|E.|Hydrolysis of GTP by Ras can activate a protein kinase.|\\n\\n# 5.\\n\\nDepolarization of axon terminals causes calcium ions to flow into the cytosol through\\n\\n|A.|voltage-gated calcium channels.|\\n|---|---|\\n|B.|inositol trisphosphate-gated calcium channels.|\\n|C.|ryanodine receptors.|\\n|D.|calcium ATPases.|\\n|E.|cAMP-gated calcium channels.|\\n\\n# 6.\\n\\nInositol trisphosphate-gated calcium channels are located in\\n\\n|A.|the plasma membrane.|\\n|---|---|\\n|B.|the smooth endoplasmic reticulum.|\\n|C.|the inner mitochondrial membrane.|\\n|D.|regulated exocytotic vesicles.|\\n|E.|all of the above.|\\n\\n# 7.\\n\\nPhosphatidylinositol bisphosphate is used in two different signaling pathways. These convert it, respectively, into\\n\\n|A.|IP3 and PIP3.|\\n|---|---|\\n|B.|IP3 and cAMP.|\\n|C.|PIP2 and PIP3.|\\n|D.|PIP2 and cAMP.|\\n|E.|cAMP and cGMP.|\\n---\\n# INTRACELLULAR SIGNALING\\n\\nenzymes that operate on ATP, a nucleotide triphosphate. Ras, in contrast, hydrolyzes the nucleotide triphosphate GTP.\\n\\n# 3. E. These statements constitute a summary of signaling in scent-sensitive nerve cells.\\n\\n# 4. E. Hydrolysis of GTP by Ras turns Ras off, so that it is no longer able to activate MAPKK kinase. The other statements are true. In particular (A) cAMP activates protein kinase A, (B) calcium activates a number of protein kinases, (C) dimerization activates growth factor receptors, and (D) PIP 3 recruits PKB to the plasma membrane, where it is phosphorylated and hence activated.\\n\\n# 5. A. Statements B, D, and E are completely false, in particular (B) inositol trisphosphate-gated calcium channels are opened by inositol trisphosphate, which is not generated by depolarization. (D) Calcium ATPases push calcium ions out of the cytosol—they do not allow it in. (E) cAMP-gated calcium channels are opened by cAMP, which is not generated by depolarization. Some scientists think that (C) may operate in a restricted set of axon terminals, that is, depolarization of the axon terminal allows calcium to flow in through voltage-gated calcium channels, and this calcium then opens ryanodine receptors (page 348). However, this is certainly not a general phenomenon.\\n\\n# 6. B. Answers C through E are completely false. Some rare cells (the best studied is a sensory cell in lobsters) do express inositol trisphosphate-gated calcium channels in their plasma membranes (answer A), but this is certainly not typical.\\n\\n# 7. A. Phospholipase C hydrolyzes phosphatidylinositol bisphosphate (PIP 2), releasing the head group as IP 3, while phosphatidylinositol 3-kinase phosphorylates the inositol head group of PIP 2, generating PIP 3. Each of these products has an important role in cell signaling. IP 3 acts to cause release of calcium from the endoplasmic reticulum, while PIP 3 causes activation of protein kinase B.\\n---\\n# INTERCELLULAR COMMUNICATION\\n\\nThe millions of cells that make up a multicellular organism can work together only because they continually exchange the chemical messages called transmitters. Here we describe how systems of cells use transmitters to cooperate for the good of the organism. Most of the many transmitters known are found in all animals and probably evolved with ancestral multicellular organisms more than a billion years ago. We will introduce the general principles of intercellular communication and then illustrate how transmitters operate in a single tissue, the gastrocnemius muscle.\\n\\n# CLASSIFYING TRANSMITTERS AND RECEPTORS\\n\\nTransmitter mechanisms can be classified in two ways. The first depends on their lifetime in the extracellular fluid. A transmitter that is rapidly broken down or taken up into cells acts only near its release site. One that is broken down slowly can diffuse a long way and may act on cells a long way away. The shortest lived transmitters of all are those released at synapses (page 344), where the distance from release site to receptor is only 100 nm. At the other extreme are transmitters that last many minutes or, sometimes, even longer. Hormones are long-lived transmitters that are released into the blood and travel around the body before being broken down. Most are released by specialized groups of secretory cells that form a structure called an endocrine gland. Paracrine transmitters also last many\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C 2004 by John Wiley & Sons, Inc.\\n---\\nINTERCELLULAR COMMUNICATION\\n\\n|acetylcholine binding site|\\n|---|\\n|Na +|\\n|extracellular medium|cytosol|\\n|acetylcholine binding opens the channel|\\n|acetylcholine binds|\\n|Na +|\\n|Na +|\\n\\nFigure 17.1. The nicotinic acetylcholine receptor is an ionotropic cell surface receptor that opens when acetylcholine in the extracellular medium binds.\\n\\nminutes before being broken down, but they are released into specific tissues rather than into the blood, and only diffuse within the tissue before they are destroyed. The second way of classifying transmitter mechanisms depends on the location and action of the receptor on the target cell. There are three types of receptors: ionotropic cell surface receptors, metabotropic cell surface receptors, and intracellular receptors.\\n\\nIonotropic Cell Surface Receptors Ionotropic cell surface receptors are channels that open when a specific chemical binds to the extracellular face of the channel protein. The nicotinic acetylcholine receptor (Fig. 17.1), so named because the drug nicotine binds to it, is one example. In the absence of a transmitter called acetylcholine, the channel is closed. When acetylcholine binds, the channel opens and allows sodium and potassium ions to pass through. The electrochemical gradient pushing sodium ions into the cell is much greater than that pushing potassium ions out of the cell, so when the channel opens, sodium ions pour in carrying positive charge and depolarizing the\\n---\\n# INTERCELLULAR COMMUNICATION IN ACTION: THE GASTROCNEMIUS MUSCLE\\n\\nThe gastrocnemius muscle illustrates how transmitters and receptors operate in a single tissue. This is the calf muscle at the back of the lower leg. When it contracts, it pulls on the Achilles tendon so that the toes push down on the ground. Most of the bulk of the muscle is made up of one type of cell, skeletal muscle cells.\\n\\nplasma membrane. This mechanism is similar to that of two channels we met in Chapter 16, the inositol trisphosphate-gated calcium channel and the cAMP-gated channel. There is, however, a major difference: Those two channels were opened by cytosolic solutes, while ionotropic cell surface receptors are opened by extracellular solutes.\\n\\n# Metabotropic Cell Surface Receptors\\n\\nMetabotropic cell surface receptors are linked to enzymes. We have already met a number of metabotropic cell surface receptors in Chapter 16. When the ADP receptor (page 344) binds extracellular ADP, it activates Gq and hence phospholipase Cβ. The receptor for smell chemicals (page 350) and the β-adrenergic receptor (page 353) are linked to Gs and hence adenylate cyclase so ligand binding increases cytosolic cAMP. Receptor tyrosine kinases are themselves protein kinases activated when their ligand binds.\\n\\nThe α-adrenergic receptor (Fig. 17.2) is another receptor that causes cytosolic calcium concentration to increase. The α- and β-adrenergic receptors are distinct proteins that bind the same transmitters, adrenaline and noradrenaline. To simplify the issue somewhat, we can say that noradrenaline acts mainly on α receptors and adrenaline acts mainly on β receptors. Because the α and β receptors are distinct proteins, it is possible to design drugs (α and β blockers) that interfere with one or the other.\\n\\n# Intracellular Receptors\\n\\nIntracellular receptors lie within the cell (in the cytosol or in the nucleus) and bind transmitters that diffuse through the plasma membrane. They always exert their effects by activating enzymes. The receptors for nitric oxide and steroid hormones are two examples.\\n\\nNitric oxide, or NO, is a transmitter in many tissues. It is not stored ready to be released but is made from arginine at the time it is needed. NO diffuses easily through the plasma membrane and binds to various cytosolic proteins that are NO receptors. One particularly important NO receptor is the enzyme guanylate cyclase, which in the presence of NO converts the nucleotide GTP to the intracellular messenger cyclic guanosine monophosphate, or cGMP.\\n\\nSteroid hormones have intracellular receptors such as the glucocorticoid receptor (page 121). In the absence of hormone this receptor remains in the cytosol and is inactive because it is bound to an inhibitor protein. However, when the glucocorticoid hormone binds to its receptor, the inhibitor protein is displaced. The complex of the glucocorticoid receptor with its attached hormone now moves into the nucleus. Here two molecules of the complex bind to a 15-bp sequence known as the hormone response element (HRE), which lies upstream of the TATA box (page 122). The HRE is a transcriptional enhancer sequence. The binding of the glucocorticoid hormone receptor to the HRE stimulates transcription.\\n---\\n# Figure 17.2.\\n\\n|noradrenaline|GTP|\\n|---|---|\\n|GDP|α|\\n| |β|\\n| |γ|\\n| |Gq|\\n| |trimeric|\\n| |G protein|\\n| |α adrenergic receptor|\\n---\\nINTERCELLULAR COMMUNICATION IN ACTION: THE GASTROCNEMIUS MUSCLE\\n\\n|motoneurone|axon of motoneurone|\\n|---|---|\\n|gastrocnemius muscle| |\\n|axon presynaptic terminal| |\\n|synaptic vesicle containing acetylcholine|plasmalemma of muscle cell|\\n|voltage-gated calcium channels| |\\n|voltage-gated sodium channels|skeletal muscle cell|\\n|potassium channels|acetylcholine receptors of nicotinic type|\\n|action potential produced by voltage-gated sodium channels| |\\n\\n0               5 ms       time\\n\\n−50                           threshold\\n\\n|skeletal muscle cell (mV)|acetylcholine|\\n|---|---|\\n|−80|transmembrane voltage of depolarization due to sodium influx through acetylcholine receptor channel|\\n\\nextracellular medium                                 Na +    cytosol\\n\\nFigure 17.3.     Motoneurones release the transmitter acetylcholine that binds to nicotinic receptors on the muscle cells. The plasma membrane of the muscle cell is depolarized to threshold and fires an action potential.\\n\\nTelling the Muscle to Contract: The Action of Motoneurones\\n\\nAll skeletal muscle cells are relaxed until they receive a command to contract from nerve cells called motoneurones. Motoneurone cell bodies are in the spinal cord while the myelinated axons run to the muscle. The axon terminal of the motoneurone that controls the gastrocnemius muscle contains vesicles of the transmitter acetylcholine (Fig. 17.3). To\\n---\\n# INTERCELLULAR COMMUNICATION\\n\\nWhen a person presses the foot down, action potentials travel from the spinal cord down the motoneurone axon to its terminal, opening voltage-gated calcium channels in the plasma membrane. Calcium ions pour in, raising their cytosolic concentration from 100 nmol liter−1 to 1 μmol liter−1. The calcium torrent causes the synaptic vesicles containing acetylcholine to fuse with the plasma membrane, releasing the transmitter into the extracellular fluid. Acetylcholine survives only 0.2 ms in the extracellular fluid since it is quickly hydrolyzed into choline and acetate by the enzyme acetylcholinesterase. However, its target—the skeletal muscle cell—is only 100 nm away. It hits its mark before it is broken down and binds to the nicotinic acetylcholine receptors present in the plasma membrane of the skeletal muscle cell. These open, allowing both potassium and sodium ions to pass. There is no large electrochemical gradient for potassium, but sodium is at a high concentration in the extracellular fluid and rushes in, depolarizing the plasma membrane of the skeletal muscle cell. The plasma membrane of the skeletal muscle cell also contains voltage-gated sodium channels. When the flow of sodium ions through the nicotinic acetylcholine receptors depolarizes the plasma membrane to threshold, the skeletal muscle cell fires an action potential, which in turn causes release of calcium from the endoplasmic reticulum (page 348). The resulting increase of calcium concentration in the cytosol of the muscle cell causes it to contract (page 393).\\n\\n# Controlling the Blood Supply: Paracrine Transmitters\\n\\nFigure 17.4 is a cut-away drawing of a blood vessel in the muscle. Lining the tube is a thin layer of endothelial cells. Wrapped around these are muscle cells of a different type, smooth muscle cells. Both endothelial cells and smooth muscle cells are much smaller than skeletal muscle cells. A small blood vessel may be only as large as a single skeletal muscle cell. Smooth muscle cells have no input from motoneurones. Instead, like many other internal organs, they are supplied by a distinct class of nerve cells that are called autonomic. Autonomic nerve cells usually use one of two transmitters: acetylcholine or noradrenaline. Two other transmitters, adrenaline and nitric oxide, also help to adjust the blood flow on a timescale of seconds to minutes. The nerve cells that release noradrenaline onto the smooth muscle of blood vessels are called vasoconstrictors because they cause blood vessels to contract. The smooth muscle cells have α-adrenergic receptors in their plasma membranes. Binding of noradrenaline to α-adrenergic receptors activates PLCβ, which generates IP3, which in turn releases calcium from the endoplasmic reticulum into the cytosol. The increase of cytosolic calcium concentration causes the smooth muscle cells to contract, constricting the blood vessel and reducing the flow. Vasoconstrictor nerves are used to restrict the flow of blood to muscles and other organs that are not in heavy use. The hormone adrenaline is chemically related to noradrenaline but is more stable, lasting a minute or so in the extracellular fluid before being broken down. It is released from an endocrine gland (the adrenal gland) during times of stress and spreads around the body in the blood. Adrenaline that diffuses to the skeletal muscle cells stimulates them to begin breaking down glycogen to make glucose-6-phosphate (page 305). The smooth muscle cells of blood vessels within skeletal muscles also have β-adrenergic receptors connected to adenylate cyclase. However, they do not contain glycogen, and cAMP has another effect in these cells. When cAMP rises, it activates cAMP-dependent protein kinase. This in turn phosphorylates proteins that relax the smooth muscle cell. The action of adrenaline is\\n---\\n# INTERCELLULAR COMMUNICATION IN ACTION: THE GASTROCNEMIUS MUSCLE\\n\\n| |axon of autonomic nerve cell| |\\n|---|---|---|\\n|endothelial cells| |blood vessel|\\n| |BLOOD| |\\n|skeletal muscle cell| |smooth muscle cells|\\n|FGF released from active skeletal muscle cells stimulates cell division in endothelial cells| |skeletal muscle cell|\\n|adrenaline diffusing from the blood relaxes smooth muscle and triggers glycogen breakdown in the skeletal muscle| |endothelial cell|\\n|NO diffuses easily through many plasma membranes and relaxes smooth muscle cells|blood|smooth muscle cell|\\n|acetylcholine released from vasodilator nerve cells causes endothelial cells to make NO| |axon of autonomic nerve cell|\\n|noradrenaline released from vasoconstrictor nerve cells causes smooth muscle to contract| | |\\n\\nFigure 17.4. Transmitters regulate the blood supply to muscles.\\n---\\n# INTERCELLULAR COMMUNICATION\\n\\nTherefore to increase the blood supply to all the muscles of the body in preparation for flight or fight. If we are very frightened and too much adrenaline is released, so much blood is diverted from the brain to the muscles that we faint.\\n\\nJust as autonomic vasoconstrictor nerve cells are used to shut down blood flow to particular regions, autonomic vasodilator nerves are used to dilate blood vessels in muscles that are about to be used. They release acetylcholine, but neither smooth muscle cells nor endothelial cells have nicotinic acetylcholine receptors. Endothelial cells have, instead, a different receptor called the muscarinic acetylcholine receptor. This is named for a chemical that binds it: muscarine, from the poisonous fly agaric mushroom Amanita muscaria (page 376). Like the α-adrenergic receptor, the muscarinic acetylcholine receptor is linked to Gq and therefore to phospholipase Cβ, so that binding of acetylcholine causes an increase of cytosolic calcium concentration. Endothelial cells do not contract in response to a calcium increase. However, they have an enzyme that makes nitric oxide, and this enzyme is activated by calcium. When endothelial cells are stimulated with acetylcholine, they make nitric oxide. This intercellular transmitter easily passes through the plasma membranes of both endothelial and smooth muscle cells and reaches its receptor within the smooth muscle cells. Here it activates guanylate cyclase, causing an increase of cGMP concentration.\\n\\nJust as cAMP exerts many of its effects through cAMP-dependent protein kinase, called protein kinase A for short, so cGMP exerts many of its effects through cGMP-dependent protein kinase, also called protein kinase G. One of the targets of protein kinase G is calcium ATPase (page 318). When this is phosphorylated by protein kinase G it works harder, reducing the concentration of calcium ions in the cytosol. This has the effect of relaxing the smooth muscle cells. Nitric oxide lasts for only about 4 s before being broken down. It is therefore a paracrine transmitter, able to diffuse through and relax all the smooth muscle cells surrounding the endothelial cells, but without lasting long enough to pass into more remote tissues.\\n\\n|Example 17.1|Nitroglycerine Relieves Angina|\\n|---|---|\\n|The discovery in 1987 that nitric oxide was a transmitter explained why nitroglycerine (more familiar as an explosive than as a medicine) relieved angina pectoris. Angina is a pain felt in an overworked heart. Nitroglycerine spreads throughout the body via the bloodstream and slowly breaks down, releasing nitric oxide that then dilates blood vessels. The heart no longer has to work so hard to drive the blood around the body.| |\\n\\n|Example 17.2|Viagra|\\n|---|---|\\n|Just as cAMP phosphodiesterase hydrolyzes cAMP to AMP and hence terminates its action as an intracellular messenger, so cGMP is inactivated by cGMP phosphodiesterase. There are a number of isoforms of this enzyme in different human tissues. The drug sildenafil, sold as Viagra, inhibits the form of the enzyme found in the penis. If cGMP is not being made, this has little effect on blood flow to the region. However, when cGMP is made in response to a local production of NO, its concentration in blood vessel smooth muscle increases much more than would otherwise occur because its hydrolysis to GMP is blocked. This in turn causes a greater activation of protein kinase G, a greater activation of the calcium ATPase, a lower cytosolic calcium concentration, a greater relaxation of blood vessel smooth muscle, and therefore greater blood flow.| |\\n---\\n# New Blood Vessels in Growing Muscle\\n\\nAll the phenomena we have discussed so far occur on a timescale of seconds. However, when a muscle is repeatedly exercised over many days, it becomes stronger: the individual skeletal muscle cells enlarge. This is because high cytosolic calcium acts via NFAT (page 222) to stimulate the transcription of genes coding for structural proteins. Furthermore, new blood vessels sprout and grow into the enlarging muscle. A growth factor called FGF is released by stimulated muscle. The receptor tyrosine kinase for FGF, like that for PDGF (page 353) is found on endothelial and smooth muscle cells and like the PDGF receptor signals via Ras and MAP kinase (page 355) to trigger cell division and hence the growth of new blood vessels. (FGF stands for fibroblast growth factor, but FGF is effective on a vast range of cell types including both endothelial and smooth muscle cells).\\n\\nIn our study of the gastrocnemius muscle and its blood supply, we have seen examples of all types of transmitter mechanism. Acetylcholine acts as a synaptic transmitter at the axon terminal of the motoneurone. Adrenaline is a hormone. The other transmitters are paracrine. The nicotinic acetylcholine receptor is an ionotropic cell surface receptor. The nitric oxide receptor is intracellular. The other receptors are metabotropic cell surface receptors. There is a wide variety of timescales of action. The acetylcholine released from the axon terminal of the motoneurone causes a contraction of the skeletal muscle cell within 5 ms, by which time the acetylcholine has already been destroyed. Adrenaline lasts 1 min and dilates blood vessels for all this time. FGF lasts 10 min, but its effects are much longer lasting. FGF triggers the synthesis of proteins, which then act to cause cell proliferation that lasts for\\n\\n|presynaptic cell = pain receptor nerve cell|presynaptic cell = pain receptor nerve cell|postsynaptic cell = pain relay nerve cell|postsynaptic cell = pain relay nerve cell|\\n|---|---|\\n|one action potential|no signal|glutamate| |\\n|0|time|0|time|\\n|-50| |-50| |\\n|-70| |-70| |\\n|depolarization due to sodium ions entering pain relay cell through glutamate receptor channels|depolarization due to sodium ions entering pain relay cell through glutamate receptor channels|depolarization due to sodium ions entering pain relay cell through glutamate receptor channels|depolarization due to sodium ions entering pain relay cell through glutamate receptor channels|\\n\\nFigure 17.5. At most synapses, one presynaptic action potential is insufficient to depolarize the postsynaptic nerve cell to threshold.\\n---\\nINTERCELLULAR COMMUNICATION\\n\\ndays. A similar pattern of intercellular communication, using some of the same transmitters and receptors but also many others, is found in every tissue of the body.\\n\\nSYNAPSES BETWEEN NEURONS\\n\\nThe synapse between a motoneurone and a skeletal muscle cell is unusual in that a single action potential in the presynaptic cell releases enough transmitter to depolarize the postsynaptic cell to threshold and hence produce an action potential in the postsynaptic cell. The synapse between a pain receptor and a pain relay nerve cell (page 341) is more typical (Fig. 17.5). An action potential in the axon terminal of the pain receptor raises cytosolic calcium and releases the transmitter glutamate. This diffuses to the pain relay cell where it binds to glutamate receptors, which are ionotropic and allow both potassium and sodium ions to pass. The postsynaptic nerve cell depolarizes as sodium ions move in through the glutamate receptor channels. However, the depolarization is not enough to take the transmembrane voltage to threshold. As soon as glutamate is removed from the extracellular medium, the transmembrane voltage returns to the resting level. In the axon of the relay cell, some distance from the synapse, the transmembrane voltage does not change, and no message passes on to the brain. The subject does not feel pain.\\n\\nExample 17.3 A Toxic Glutamate Analogue\\n\\nThe grasspea is a protein-rich crop that has been cultivated since ancient times and is still an important source of calories and protein in India, Africa, and China. The peas contain the amino acid β-N-oxalyl- L-α-β-diaminopropionic acid. If untreated peas are eaten, the toxin binds to and opens the glutamate receptor on nerve cells. The resulting long-lasting depolarization damages and finally kills the nerve cells. Boiling the peas during cooking destroys the toxin; but in times of famine, when fuel is scarce, many people are poisoned, and the resulting brain damage is irreversible.\\n\\n|COO|COO|C|O|\\n|---|---|---|---|\\n|CH2| |NH| |\\n|CH2| |CH2| |\\n|OOC|CH|NH3| |OOC|CH|NH3|\\n|glutamate| |β-N-oxalyl-L-α-β-diaminopropionic acid| |\\n\\nA subject does feel pain when they move their finger into the hot air from a hair dryer (Fig. 17.6). This is because the hot air heats a large area and causes many pain receptors to fire action potentials. Many of the pain receptors synapse onto one relay cell, which\\n---\\n# SYNAPSES BETWEEN NEURONS\\n\\naction potential\\n\\nin each of many\\n\\npain receptor\\n\\nnerve cells postsynaptic cell = pain relay sodium glutamate nerve cell\\n\\n| |action potential|\\n|---|---|\\n| |action potential|\\n|0|time|\\n|-50|postsynaptic cell (mV)|\\n|-70|transmembrane voltage of depolarization due to sodium ions entering pain relay cell through glutamate receptor channels|\\n\\nFigure 17.6. Spatial summation at a synapse.\\n\\ntherefore receives many doses of glutamate. Because of this spatial summation, enough glutamate receptor channels open to depolarize the relay cell to threshold, and an action potential travels along the axon of the relay cell to the brain. The subject feels pain. An intense stimulus to a small area is also painful. When the subject is jabbed with a needle, only one pain receptor is activated, but that receptor is intensely stimulated and fires a rapid barrage of action potentials, each of which causes the release of glutamate and an extra depolarization of the pain relay cell (Fig. 17.7). Soon the transmembrane voltage of the relay cell reaches threshold, and an action potential travels along its axon toward the brain and the subject feels pain. Such temporal summation only occurs if the presynaptic action potentials are frequent enough to ensure that the depolarizations produced in the postsynaptic cell add. Another amino acid, γ -amino butyric acid (GABA) (page 37), is also a synaptic transmitter. Its receptor is an ionotropic cell surface receptor that forms a channel selective\\n---\\n# INTERCELLULAR COMMUNICATION\\n\\npresynaptic cell = pain receptor &emsp; postsynaptic cell = pain relay\\n\\nnerve cell &emsp; sodium glutamate &emsp; nerve cell\\n\\nmany action potentials &emsp; action potential\\n\\n&emsp; time &emsp; 0 &emsp; action potential &emsp; time\\n\\nof presynaptic cell &emsp; -50 &emsp; of presynaptic cell (mV)\\n\\ntransmembrane voltage &emsp; -70\\n\\ndepolarizations in postsynaptic cell add in time\\n\\nFigure 17.7.\\nTemporal summation at a synapse.\\nfor chloride ions. Figure 17.8 illustrates a single nerve cell bearing both glutamate and γ-amino butyric acid receptors. At A an action potential in the GABA-secreting axon releases GABA onto the surface of the postsynaptic cell, causing the GABA receptor channels to open. Although chloride ions could now move into or out of the cell, they do not, because their tendency to travel into the cell down their concentration gradient is balanced by the repulsive effect of the negative voltage of the cytosol: chloride ions are at equilibrium (page 314). The opening of GABA channels therefore causes no ion movements and therefore does not alter the transmembrane voltage. At B, action potentials occur simultaneously in six glutamate-secreting axons. In this example, this activity provides enough glutamate to depolarize the postsynaptic cell to threshold, and the postsynaptic nerve cell fires an action potential. At C, action potentials occur simultaneously in six glutamate-secreting axons and also in the GABA-secreting axon. The same number of glutamate receptor channels open as before, and the same number of sodium ions flow into the postsynaptic nerve cell, tending to depolarize it. However, as soon as the transmembrane voltage of the postsynaptic cell deviates from the resting value, chloride ions start to enter through the GABA receptor channels because the cytosol is no longer negative enough to prevent them from entering the cell down their concentration gradient. The inward movement of negatively charged chloride ions neutralizes some of the positive charge carried in by sodium ions moving in through the glutamate receptor channels. The postsynaptic nerve cell therefore does not\\n---\\n# SYNAPSES BETWEEN NEURONS\\n\\nsix axons with terminals that release glutamate\\n\\npostsynaptic nerve cell\\n\\none axon with a terminal that releases GABA\\n\\n0 time\\n\\n-50\\n\\npostsynaptic cell (mV)\\n\\n-70 transmembrane voltage of\\n\\nA\\nB\\nC\\n\\nFigure 17.8. Inhibition by a GABAergic synapse.\\n\\ndepolarize as much and does not reach threshold. No action potential is generated in the postsynaptic nerve cell. Antianxiety drugs such as Valium act on the GABA receptor and increase the chance of its channel opening. Nerve cells exposed to the drug are less likely to depolarize to threshold. Valium therefore reduces action potential activity in the brain, calming the patient.\\n---\\n# INTERCELLULAR COMMUNICATION\\n\\nMedical Cigarettes, Mushrooms, and Insecticides Relevance\\n\\nNicotine is one of the most addictive substances known. The reason for this is still unclear. The muscle weakness that new smokers experience is due to nicotine binding to and blocking receptors on the muscle cells, but we are still not sure how nicotine acts in the brain to cause a pleasurable experience.\\n\\nThe fly agaric mushroom Amanita muscaria is seldom taken recreationally. Its popular use is more prosaic, in killing flies, hence its English and Latin names. However, Robert Graves has argued that it was used by the classical Greeks and by the Vikings to produce a state of berserk strength with hallucinations. Any such use would be highly dangerous: muscarine acts by stimulating the muscarinic acetylcholine receptor, and the difference between a dose of muscarine large enough to cause pharmacological effects and a lethal one is very small.\\n\\nAmanita muscaria has an even more dangerous relative, A. phalloides, the death cap mushroom. The toxin it produces, phalloidin, does not affect acetylcholine signaling. Instead, it interferes with the actin cytoskeleton (page 390).\\n\\nFarmers and others who suffer insecticide toxicity show similar symptoms to those of muscarine poisoning. This is because these insecticides work by blocking the enzyme acetylcholinesterase, which breaks down acetylcholine. With the enzyme blocked, acetylcholine hangs around in the extracellular medium, stimulating the receptor for longer than it should. Nerve gases developed to kill people (like Sarin, released in the Tokyo subway by members of the Aum cult in 1995) work in the same way. Paradoxically, sufferers are treated with another toxin, atropine, from the deadly nightshade plant. Atropine turns off the muscarinic acetylcholine receptor.\\n\\nSUMMARY\\n\\n1. Transmitter molecules can be classified in two ways. One depends on their lifetime and the extracellular medium they are found in, the other on the location and action of the receptor on the target cell.\\n2. Synaptic transmitters are extremely short lived. Paracrine transmitters and hormones have a longer lifetime and are found in specific tissues and in the blood, respectively.\\n3. Receptors can be divided into ionotropic cell surface receptors, metabotropic cell surface receptors, and intracellular receptors. Ionotropic cell surface receptors are ion channels that open in response to ligand binding. The effect upon the target cell is electrical. Metabotropic cell surface receptors are linked to enzymes. The effect upon the target cell is mediated through a biochemical process. Intracellular receptors lie within the target cell and bind transmitters that are able to cross the cell membrane by simple diffusion.\\n4. The gastrocnemius muscle provides examples of all types of intercellular signaling operating in concert to fit the operation of the tissue to the requirements of the organism.\\n5. Synapses between nerve cells do not generally mediate a 1-to-1 transmission of the action potential. The presynaptic signal must show summation in time or space to elicit a postsynaptic action potential.\\n6. Stimulation of ionotropic cell surface receptors that pass chloride ions makes it more difficult for a nerve cell to be depolarized to threshold.\\n---\\n# FURTHER READING\\n\\nSeeley, R. R., Stephens, T. D., and Tate, P. 1995. Anatomy and Physiology. St. Louis: Mosby.\\n\\nSiegel, G. J., Agranoff, B. W., Albers, R. W., Fisher, S. K., and Uhler, M. D. 1999. Basic Neurochemistry. Philadelphia: Lippincott-Raven.\\n\\n# REVIEW QUESTIONS\\n\\n1. Ionotropic cell surface receptors\\nA. are ion channels permeable to bop sodium and potassium.\\nB. allow pe transmembrane voltage of pe cell possessing pem to change in response to transmitter.\\nC. are coupled to, or are pemselves, enzymes.\\nD. are only found on nerve cells.\\nE. open when pe concentration of acetylcholine in pe extracellular medium increases.\\n\\n2. Metabotropic cell surface receptors\\nA. are ion channels.\\nB. exert peir effects by activating enzymes.\\nC. are tyrosine kinases.\\nD. cause pe concentration of cAMP in pe cytosol to increase.\\nE. make it more difficult for a postsynaptic cell to be depolarized to preshold.\\n\\n3. Application of adrenaline to a tissue tends to increase blood flow prough it, while noradrenaline has pe opposite effect. This is because\\nA. noradrenaline, but not adrenaline, is able to diffuse prough plasma membranes.\\nB. pe receptor at which noradrenaline is most effective is a cAMP-gated channel while pe receptor at which adrenaline is most effective is a calcium channel.\\nC. adrenaline is a hormone, while noradrenaline is released close to pe target tissue.\\nD. pe electrochemical gradient for noradrenaline is inward, so it tends to make pe cytosol more positive, while pe electrochemical gradient for adrenaline is outward.\\nE. pe receptor at which noradrenaline is most effective is coupled to phospholipase C while pe receptor at which adrenaline is most effective is coupled to adenylate cyclase.\\n\\n4. Nitric oxide\\nA. is generated in endopelial cells when pe cytosolic calcium concentration increases.\\nB. easily diffuses across plasma membranes.\\nC. acts at intracellular receptors.\\nD. causes an increase of cGMP in pe target cell.\\nE. All of pe above.\\n\\n5. When seven action potentials arrive in close succession at a particular axon terminal in pe brain, pe postsynaptic cell is depolarized to just negative of preshold. If, one minute later, pe same train of seven action potentials arrives again, pe postsynaptic cell is more likely to fire an action potential if\\nA. simultaneously, anoper nerve cell fires an action potential and releases glutamate onto pe same postsynaptic cell.\\n---\\n# INTERCELLULAR COMMUNICATION\\n\\n6. Which of the following statements about the gastrocnemius muscle and its blood supply is untrue?\\n\\n- A. Acetylcholine released from motoneurons causes skeletal muscle cells to contract.\\n- B. Acetylcholine released from sympathetic neurons causes smooth muscle cells to contract.\\n- C. NO released from endothelial cells causes smooth muscle cells to relax.\\n- D. PDGF released from platelets causes endothelial cells to enter the cell cycle.\\n- E. FGF released from skeletal muscle cells causes endothelial cells to enter the cell cycle.\\n\\n7. The cells in the adrenal gland that secrete adrenaline have, in their plasma membranes, nicotinic acetylcholine receptors, muscarinic acetylcholine receptors, and voltage-gated calcium channels. Application of acetylcholine will therefore cause\\n\\n- A. an increase of cytosolic calcium concentration, the vast majority of which is due to influx of calcium from the extracellular medium.\\n- B. an increase of cytosolic calcium concentration, the vast majority of which is due to release of calcium from the smooth endoplasmic reticulum.\\n- C. an increase of cytosolic calcium concentration, due to a combination of influx of calcium from the extracellular medium and release of calcium from the smooth endoplasmic reticulum.\\n- D. a simultaneous increase of cytosolic calcium concentration and a shift in the transmembrane voltage to a more negative value.\\n- E. a reduction in the cytosolic calcium concentration.\\n\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n1. B. Ionotropic cell surface receptors pass current when they open and therefore change the transmembrane voltage. Considering the other answers: (A) Some ionotropic cell surface receptors are ion channels permeable to both sodium and potassium, but others (such as the GABA receptor) are not. (C) Metabotropic, not ionotropic, receptors are coupled to, or are themselves, enzymes. (D) Many other cells have ionotropic cell surface receptors. Skeletal muscle cells are one example. (E) One class of ionotropic cell surface receptors (nicotinic acetylcholine receptors) respond to acetylcholine, but others respond to other transmitters.\\n2. B. This is the definition of a metabotropic cell surface receptor. Of the other answers: (A) Metabotropic cell surface receptors are not ion channels. (C) Some metabotropic cell surface receptors are tyrosine kinases, but many are not, including those that signal to Gq or Gs. (D) The subset of metabotropic cell surface receptors that signal to Gs will increase the concentration of cAMP, but many others will not. (E) Metabotropic receptors affect many aspects of cell function, so some will indeed make it more difficult for a postsynaptic cell to be depolarized to threshold, but this is not a general rule.\\n3. E. Both receptors are on smooth muscle cells. The receptor at which noradrenaline is most effective is coupled to phospholipase C and therefore causes an increase of cytosolic calcium concentration, leading to contraction and hence a closing of the blood vessel. The receptor at which adrenaline is most effective is coupled to adenylate cyclase and therefore causes an increase of cAMP concentration, leading to phosphorylation and therefore relaxation of the contractile machinery and\\n---\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n|4.|E.|These statements constitute a summary of the role of nitric oxide in blood vessels.|\\n|---|---|---|\\n|5.|A.|If more glutamate is released onto the postsynaptic cell, more glutamate receptors will open, more sodium will flow in, and so the cell will depolarize further. Of the other answers: (B) Opening of GABA receptors, allowing chloride to enter the postsynaptic cell, will make it harder for the cell to depolarize to threshold. (C) Noradrenaline acts at metabotropic receptors, so if it does have an effect, it will be slow and therefore will not affect the ability of the seven other action potentials to trigger an action potential. (D) If the action potentials are spaced further apart, the transmembrane voltage has more time to relax back to the resting voltage between each one, so the maximum voltage during each action potential will be less, that is, further from threshold. (E) Nerve cells do not usually have PDGF receptors. If they did, the effect of this metabotropic receptor would be slow and therefore would not affect the ability of the seven other action potentials to trigger an action potential.|\\n|6.|B.|The smooth muscle cells in blood vessels do not have receptors for acetylcholine. Rather, acetylcholine acts indirectly, via endothelial cells and NO, to relax the smooth muscle. All the other statements are true.|\\n|7.|C.|Binding of acetylcholine to nicotinic acetylcholine receptors will depolarize the cells, opening voltage-gated calcium channels and allowing calcium ions to flow in from the extracellular medium. Simultaneously, binding of acetylcholine to muscarinic acetylcholine receptors will activate phospholipase Cβ and therefore cause release of calcium from the smooth endoplasmic reticulum. Answers A and B are therefore wrong since neither influx nor release from endoplasmic reticulum can be said to constitute the vast majority of the calcium entering the cytosol. Answers D and E are completely wrong: the transmembrane voltage becomes more positive and the cytosolic calcium concentration increases.|\\n---\\n# MECHANICAL MOLECULES\\n\\nTHE CYTOSKELETON IS BOTH STRONG AND MOTILE\\n\\nEukaryotic cells are supported by a network of struts and cables called the cytoskeleton. In animal cells, which lack a rigid cell wall, it is the cytoskeleton that determines cell shape. In addition the cytoskeleton is responsible for cell locomotion, for moving components from place to place within the cell, and for cell division. The cytoskeleton is composed of three cytoplasmic filament networks: microtubules, microfilaments, and intermediate filaments (Fig. 18.1). Although all three are physically strong, microtubules and microfilaments have the additional role of organizing movement, both of entire cells and of structures within them. The term cytoskeleton implies a rigid set of “bones” within the cell. Nothing could be further from the truth, and all three filament systems are highly dynamic, altering their organization in response to the needs of the cell. The molecules that make up the three filament systems have been highly conserved throughout evolution; the cytoskeletal proteins present in the cells of complex organisms such as humans are much the same as those in a simple organism such as a yeast. Although the individual molecules making up the cytoskeleton are below the limit of resolution of the light microscope, the cytoskeleton itself can be readily observed within the cell by using fluorescence microscopy (e.g., page 6).\\n\\nMICROTUBULES\\n\\nMicrotubules possess a combination of physical properties that allows them to participate in multiple cellular functions. They can form bundles of rigid fibers that make excellent\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C2004 by John Wiley & Sons, Inc.\\n---\\n# MECHANICAL MOLECULES\\n\\ncentrosome nucleus\\n\\nmicrotubules\\nmicrofilaments\\nintermediate filaments (stress fibers)\\n\\nFigure 18.1. Typical spatial organization of microtubules, stress fibers (one form of microfilaments), and intermediate filaments.\\n\\nstructural scaffolds and hence serve an important role in the determination of cell shape. They have an inherent structural polarity that defines a polarity (front vs. back) to the cell. They provide a system of intracellular highways that supports a two-way traffic of organelles and small vesicles powered by enzymes that interact with the microtubule surface. They can be rapidly formed and broken down, a property that allows the cell to respond to subtle environmental changes. Finally, they play a role in one of the most exquisite and precise of all movements within the cell, the segregation of chromosomes at mitosis and meiosis (Chapter 19).\\n\\nAnimal cells contain a network of several thousand microtubules, each 25 nm in diameter. All the cell’s microtubules can be traced back to a single structure called the centrosome, which is tightly attached to the surface of the nucleus at the cell centre (Fig. 18.1). The centrosome is the microtubule organizing center of the cell and consists of amorphous material enclosing a pair of centrioles (Fig. 18.2). Centrioles have a characteristic nine-way pattern that we will meet again in cilia and flagella (page 386). Microtubules in plant cells have a quite different organization, lying immediately beneath the cell membrane, oriented at right angles to the direction of cell expansion. The role of microtubules in plants is to direct the deposition of cellulose fibers on the outside of the cell membrane. Cellulose synthase, a multiprotein enzyme complex that spans the cell membrane, is thought to move along microtubules forming new parallel tracks of cellulose microfibrils on the cell surface (Fig. 18.3). As a consequence of the position of the microtubules beneath the membrane, cellulose is laid down in hoops that encase the plant cell in a rigid corset and allow it to expand only in one direction. Plant cells do not contain centrioles, and it remains unclear whether their microtubules arise from a defined organizing center.\\n\\nMicrotubules are composed of a protein called tubulin that consists of two dissimilar subunits designated α and β. In the human genome there are about five α-tubulin genes and roughly the same number for β tubulin. There is a third member of the tubulin superfamily, γ tubulin, which does not itself contribute to microtubule structure but which is found at the centrosome and plays a role in initiating microtubule assembly. α-Tubulin/β-tubulin dimers assemble into chains called protofilaments, 13 of which make up the microtubule wall (Fig. 18.4). Within each protofilament the tubulin dimers are arranged in a “head-to-tail” manner, α β α β and so on. This gives the microtubule an built-in molecular polarity that is reflected in the way it grows. Tubulin subunits are added to, and lost from, one end much more rapidly than the other. By convention, the fast growing end is referred to as the (+) end and the slow growing end as the (−) end.\\n---\\n# cross section\\n\\nmaterial pericentriolar centrioles microtubules\\n\\nThe microtubule organizing center or centrosome consists of amorphous material enclosing a pair of centrioles.\\n\\nFigure 18.2.\\n---\\n# MECHANICAL MOLECULES\\n\\n(a)\\n\\nDirection of cell expansion\\ncellulose microfibril\\n\\n(b)\\n\\ncellulose microfibril\\nplasma membrane\\nmicrotubule\\ncellulose synpase\\n\\nFigure 18.3. Cellulose synthase runs on microtubule tracks.\\n\\nIn cells, the minus end of each microtubule is embedded in the centrosome so that only the plus ends are free to grow or shrink. This process is surprisingly complex. Individual microtubules undergo periods of slow growth followed by rapid shrinkage, sometimes disappearing completely. This phenomenon is referred to as dynamic instability. By chance, the growing ends of certain microtubules may be captured by sites at the cell membrane and stabilized, so that they are protected from shrinkage. Their further growth influences the shape of the cell (Fig. 18.5). Groups of microtubules having a common orientation make an excellent structural framework. Because microtubules are dynamic, the framework can be continually remodeled as the needs of the cell change.\\n\\nOne of the most important tools in establishing microtubule function in cells has been the plant alkaloid colchicine. Extracted from the corms of the autumn crocus, Colchicum\\n---\\n# MICROTUBULES\\n\\n|(+)|(−)|\\n|---|---|\\n|α-tubulin/β-tubulin dimer|microtubule protofilament|\\n\\nFigure 18.4. Microtubule structure.\\n\\nautumnale, colchicine has been used since Roman times as a treatment for gout. Cells exposed to colchicine lose their shape, and the movement of organelles within the cytoplasm ceases. When the drug is washed away, microtubules reassemble from the centrosome and normal functions are resumed (Fig. 18.6). Another drug, taxol, obtained from the bark of the Pacific yew, Taxus brevifolia, has the opposite effect, causing large numbers of very stable microtubules to form in the cell, an effect that is difficult to reverse.\\n\\nUnlike animal cells, plant cells do not use their cytoskeletons to define their cell shape, but instead use their extracellular cell walls. The direction in which a growing plant cell extends is determined in the first instance by the direction in which the cellulose microfilaments run. However, these are laid down by cellulose synthase following microtubule tracks so in fact the cytoskeleton has defined the direction of cell growth, but indirectly rather than by itself acting as a load-bearing component.\\n\\nthe ends of these two microtubules have been \"captured\" and stabilized. Now, they do not shrink but only grow, altering the cell shape\\n\\ncentrosome\\n\\nnucleus\\n\\ndynamic instability, microtubules grow and shrink, \"sensing\" their environment\\n\\nFigure 18.5. Microtubules show dynamic instability.\\n---\\n# MECHANICAL MOLECULES\\n\\ntaxol colchicine\\n\\nFigure 18.6.\\nEffects of taxol and colchicine on microtubules.\\n\\n# MICROTUBULE-BASED MOTILITY\\n\\nCells move for a variety of reasons. Human spermatozoa in their millions swim frantically toward an ovum; the soil amoeba, Acanthamoeba (said to be the most abundant organism on Earth), crawls over and between soil particles, engulfing bacteria and small organic particles as it does so. Cells in the early human embryo show similar crawling movement as they reorganize to form tissues and organs; the invasive properties of some cancer cells is due to their reverting to this highly motile embryonic state. Of course, not all cells show these obvious forms of motility, but careful observation of even the most sedentary eukaryotic cells often reveals a remarkable repertoire of intracellular movements. Both microtubules and actin filaments play important roles in cell motility. We will describe microtubule-based motility first; actin-based motility will be covered following the section on microfilaments.\\n\\nCilia and Flagella\\n\\nCilia and flagella appeared very early in the evolution of eukaryotic cells and have remained essentially unchanged to the present day. The terms cilium (meaning an eyelash) and flagellum (meaning a whip) are often used arbitrarily. Generally, cilia are shorter than flagella (<10 μm compared to >40 μm) and are present on the surface of the cell in much greater numbers (ciliated cells often have hundreds of cilia but flagellated cells usually have a single flagellum). The real difference, however, lies in the nature of their movement (Fig. 18.7). Cilia row like oars. The movement is biphasic, consisting of an effective stroke in which the cilium is held rigid and bends only at its base and a recovery stroke in which the bend formed at the base passes out to the tip. Flagella wriggle like eels. They generate waves that pass along their length, usually from base to tip at constant amplitude. Thus the movement of water by a flagellum is parallel to its axis while a cilium moves water perpendicular to its axis and, hence, perpendicular to the surface of the cell.\\n\\nCilia are such a conspicuous feature of some protists that they are called ciliates. The swimming of a paramecium (Fig. 18.8), for example, is generated by the coordinated motion of several thousand cilia on the cell surface. Cilia also play a number of important roles in the human body. The respiratory tract, for example, is lined with about 0.5 m 2 of ciliated epithelium, bearing something like 10 12 cilia. The beating of these cilia moves a belt of mucus containing inhaled particles and microorganisms away from the lungs. This activity is paralyzed by cigarette smoke, causing mucus to accumulate in the smoker’s lung and causing the typical cough.\\n---\\n| | |cilium| |\\n|---|---|---|---|\\n| | | |medium pushed this way|\\n| |8|7|6|\\n|4| |1|5|\\n| |cell pushed this way| |basal body|\\n| | |flagellum| |\\n| | | | |\\n| | | | |\\n| | | |medium pushed this way|\\n| | | |basal body|\\n| | | |cell pushed this way|\\n\\nFigure 18.7. How cilia and flagella bend.\\n---\\n# MECHANICAL MOLECULES\\n\\n|Paramecium (cilia)|sea urchin spermatozoon|\\n|---|---|\\n|(flagellum)| |\\n\\ncross section\\n\\n|B microtubule|A microtubule|\\n|---|---|\\n|interdoublet link|outer dynein arm|\\n|inner dynein arm|membrane|\\n|central pair projections|tubulin|\\n|radial spoke|subunit|\\n|central pair microtubules| |\\n\\nFigure 18.8. Cilia and flagella have identical structure.\\n\\nDespite their different pattern of beating, cilia and flagella are indistinguishable structurally (Fig. 18.8). Both consist of a regular arrangement of microtubules and associated proteins with the nine-way pattern we have also seen in centrioles. Unlike centrioles, cilia and flagella have a central pair of microtubules, so that the overall structure is called the 9 + 2 axoneme. The axoneme is enclosed by an extension of the cell membrane. Attached to the nine outer doublet microtubules are projections, or arms, composed of a motor protein called dynein. Dynein is an ATPase that converts the energy released by ATP hydrolysis into the mechanical work of ciliary and flagellar beating. Using ATP produced by mitochondria near the base of the cilium or flagellum as fuel, the dynein arms push on the adjacent outer doublets, forcing a sliding movement to occur between adjacent outer doublets. Because the arms are activated in a strict sequence both around and along the axoneme and because the amount of sliding is restricted by the radial spokes and interdoublet links, sliding is converted into bending.\\n\\nBacterial flagella (page 264) use a fundamentally different mechanism. Like the propellor of a boat, the motion of the bacterial flagellum is entirely driven by the rotary motor at its base. The bacterial flagellum itself is a specialized piece of extracellular cell wall.\\n---\\n# MICROTUBULE-BASED MOTILITY\\n\\nMade of one protein (flagellin) that has no similarity to tubulin or dynein. Cilia and flagella are full of cytosol all the way to their tips, and use the ATP in that cytosol to generate force all the way along their length.\\n\\n# IN DEPTH 18.1 Calcium and Paramecium Swimming\\n\\nParamecium, although it is a single-celled organism, is a fast-swimming predator. Its surface is covered with hundreds of cilia that are arranged in rows that lie obliquely to the cell axis. Cilia beat with their effective stroke oriented toward the rear, propelling the cell forward in a spiral motion. The beating of the individual cilia is coordinated to produce “metachronal waves” that pass from the front of the animal to the rear, rather like the wind passing across a corn field. Paramecium needs millisecond-to-millisecond control of its cilia as it swims through pondweed in search of prey. One reflex, the “avoiding reaction,” was first described almost a century ago. When a Paramecium is swimming forward, the internal calcium concentration is kept low, at about 100 nmol liter−1. When the front of the animal bumps into an object, the plasma membrane depolarizes, opening voltage-gated calcium channels in the cilia all the way down the animal as a calcium action potential is generated. The influx of calcium ions raises the intracellular calcium concentration, and this in turn causes the cilia to swing through 180° so that the effective stroke is now directed toward the anterior. This causes the Paramecium to reverse a few body lengths, taking it out of contact from the object with which it collided. Calcium is now pumped out of the cell and the internal calcium concentration falls back to 100 nmol liter−1. The cilia now swing back to their original position and forward swimming is resumed.\\n\\n# Medical Relevance\\n\\nA Sinister Complaint\\n\\nA small number of men seeking the help of fertility clinics turn out to have not only nonswimming sperm but a failure of all the cilia and flagella in their bodies. On questioning, they are often found to suffer from chronic respiratory disease because the cilia lining their airways are also immobile. Most surprising is the fact that half of these individuals also exhibit Kartegener’s syndrome, a condition in which the internal organs have the reversed orientation from normal, that is, the heart is on the right, the appendix on the left, and so on. It turns out that cilia set up a flow from right to left across the early embryo. Transmitters carried in this flow tell the left-hand side to develop its characteristic organs, and vice versa. If a mutation renders the cilia immobile, the left–right gradient of transmitter does not get set up, and the body organs are positioned by chance and not design.\\n\\n# Intracellular Transport\\n\\nThe beating of cilia and flagella is an obvious manifestation of movement generated by a microtubule-based structure having a defined and geometric shape. However, motility is a general property of microtubules within cells. It is seen particularly clearly in some specialized cell types such as the pigment cells called chromatophores in the skin of fish and amphibia. The inward and outward movement of pigment granules along radial arrays of microtubules underlies the remarkable color changes such animals are able to display. But this is just an exaggerated example of a process that occurs less.\\n---\\n# MECHANICAL MOLECULES\\n\\nFigure 18.9.\\nPigment migration in fish chromatophores.\\n\\nspectacularly in all cells. For instance, nerve cell axons extend up to 1 m from the cell\\nbody. Organelles, small vesicles, and even mRNA are transported in both directions in a\\nphenomenon referred to as axonal transport (Fig. 18.10). This is subdivided into outward\\nor anterograde transport and inward or retrograde transport. Both are dependent upon\\nmicrotubules that are abundant in nerve cells.\\n\\nThe two forms of axonal transport are dependent upon different molecular motors.\\nDynein, sometimes referred to as cytoplasmic dynein to distinguish it from its relative in\\ncilia and flagella, moves vesicles and organelles in the retrograde direction while another\\nprotein, kinesin, is the motor for movement along microtubules in the anterograde direction.\\nTheir directionality is specified by the polarity of the microtubules. Thus, dynein moves\\nalong a microtubule in a plus to minus direction while kinesin works in the opposite direction.\\nBoth proteins consist of a tail that binds to the cargo to be transported and two (kinesin) or\\nthree (dynein) globular heads that interact with the surface of the microtubule, generating\\nmovement. Specificity is imparted to this process by having multiple dyneins and kinesins\\nin a single cell, each responsible for transporting a specific type of cargo. This multiplicity\\nof dyneins and kinesins is created by alternative splicing of a smaller number of genes\\n(page 118).\\n\\n# MICROFILAMENTS\\n\\nMicrofilaments are fine fibers, about 7 nm in diameter, that are made up of subunits of\\nthe protein actin. Because it is a globular protein, the actin monomer is designated G-\\nactin while the filament that forms from it is referred to as F-actin. Each actin filament is\\ncomposed of two chains of actin monomers twisted around one another like two strands of\\nbeads (Fig. 18.11). In animal cells, actin is particularly associated with the cell periphery.\\nWhen they are grown in plastic cell culture dishes, nonmotile cells have two main types of\\nmicrofilaments: bundles of actin filaments called stress fibers run across the cell and help to\\nanchor it to the dish (Fig. 18.1) while under the cell membrane can be seen loose meshwork\\nof filaments that give the edges of the cell structural strength. In actively moving cells,\\n---\\n# MICROFILAMENTS\\n\\n| |axon terminal| |\\n|---|---|---|\\n| |axon| |\\n|microtubule|+|vesicle|\\n| |vesicle|plus|\\n| |kinesin|molecules|\\n| | |anterograde transport|\\n| | |retrograde transport|\\n| | | |\\n| |nucleus|cell body|\\n\\nFigure 18.10. Axonal transport.\\n\\nthe stress fibers disappear and actin becomes concentrated at the leading edge. Projections\\nfrom the cell surface such as microvilli (page 13) are maintained by rigid bundles of actin\\nfilaments.\\n\\nThe equilibrium between G- and F-actin is affected by many factors including actin-binding proteins (Fig. 18.11). Filament growth is prevented by profilin, which binds\\nG-actin monomers and prevents their polymerization, and by capping proteins that bind\\nto F-actin filament ends. In contrast, filament nucleation proteins such as actin-related\\nprotein (Arp) act as a base on which new filaments can form. Crosslinking proteins stick\\nto two existing filaments, forming a mechanically strong lattice. Of these villin, found in\\n---\\n# MECHANICAL MOLECULES\\n\\nGrowing filaments push membrane forward\\n\\n|Arp|capping protein|\\n|---|---|\\n|capped filaments|cannot elongate|\\n| |actin filament|\\n|G-actin|profilin|\\n|profilin : actin dimer| |\\n\\nFigure 18.11. Actin polymerization is regulated by actin-binding proteins.\\n\\nmicrovilli, generates parallel bundles, while related proteins bind criss-crossing filaments together to form a viscous, three-dimensional cytoplasmic gel.\\n\\nExample 18.1 Spearing Eggs\\n\\nUpon contacting an egg the sperm of the marine organism Thyone suddenly extends a needlelike process called an acrosome from its front end. This 90-μm-long needle helps the sperm to penetrate the egg. Prior to activation, a vesicle at the sperm front end contains G-actin at concentrations way above that which would normally result in spontaneous polymerization into F-actin filaments. The reason it does not do so is that all the actin is bound to profilin. Upon contact with the egg, salts from seawater flow into the vacuole and cause dissociation of the profilin–actin dimers. The actin then immediately polymerizes to form the long microfilaments of the acrosome.\\n\\nCells are anchored to the extracellular matrix through transmembrane proteins such as integrins. These are dimeric proteins that have an extracellular domain that binds to collagen and other extracellular matrix proteins and an intracellular domain that attaches to actin microfilaments on the inside (Fig. 18.12).\\n---\\n# MICROFILAMENTS\\n\\nFigure 18.12. Integrins anchor the actin cytoskeleton to the extracellular matrix.\\n\\nMedical Relevance Some Bacteria Highjack the Cytoskeleton for Their Own Purposes\\n\\nWe are all familiar with the ways that bacteria and viruses spread from person to person by various forms of social or antisocial contact, but how does a bacterium spread through the cells of a human body? A number of bacteria including the important pathogens Listeria (responsible for sepsis and meningitis in immunocompromised patients and for infections of the fetus during pregnancy) and Shigella (which causes dysentery), avoid contact with the human body’s antibodies and white blood cells by remaining hidden within the cytoplasm of our cells as they spread through our tissues. The bacteria use actin to power their journey from cell to cell. Listeria has a protein called ActA at one end of its rod-shaped body. Like Arp, ActA promotes the polymerization of actin filaments. This generates enough force to drive the bacterium through the cytoplasm in the opposite direction. Movement is random, but occasionally the bacterium will bump into the cell membrane causing the formation of a fingerlike projection from the cell surface. The membrane of the projection fuses with the membrane of the neighboring cell, transferring the bacterium in a membrane sac from which it quickly escapes to repeat the procedure. The really clever thing (from the bacterium’s point of view) about such a transfer mechanism is that the bacterium never leaves the safety of the cytoplasm.\\n\\nMuscle Contraction\\n\\nStriated muscle, the kind of muscle found attached to bones or in our hearts, gains that appearance because passing along the cell one encounters in turn regions of parallel microfilaments (which are relatively transparent) then regions of thick filaments of a second protein called myosin. The complete repeating unit is called a sarcomere and is delineated by the Z disc, which holds the microfilaments in a regular pattern, so that striated muscle has an almost crystalline appearance in transverse section. The myosin molecule has a distinctive structure, consisting of a tail and two globular heads. The thick filament is formed from a large number of myosin molecules arranged in a tail-to-tail manner. This design means that the thick filament is a bipolar structure with myosin heads at both ends. In a resting striated muscle, the myosin heads are unable to operate on the actin microfilaments.\\n---\\n# MECHANICAL MOLECULES\\n\\n|Z disc|sliding|sliding|Z disc|\\n|---|---|---|---|\\n|enlarged areaenlarged areaenlarged area|enlarged areaenlarged areaenlarged area|enlarged areaenlarged areaenlarged area|enlarged areaenlarged areaenlarged area|\\n|contractioncontracti contraction longitudinal section|contractioncontracti contraction longitudinal section|contractioncontracti contraction longitudinal section|contractioncontracti contraction longitudinal section|\\n|thick filament (myosin)| | | |\\n|thin filament (actin)| | | |\\n|transverse section|transverse section|transverse section|transverse section|\\n|enlarged area|enlarged area|enlarged area|enlarged area|\\n|actin monomer| | | |\\n|myosin heads| | | |\\n|myosin tail| | | |\\n\\nFigure 18.13.    Muscle contraction.\\n\\nWhen the cytosolic calcium concentration increases, calcium binds to a protein called troponin that changes shape to allow myosin access to the actin. The myosin heads now crawl along the actin (thin) filaments and generate sliding that is powered by the hydrolysis of ATP. Because of the geometry of the system, the two Z discs are pulled toward each other and the cell shortens (Fig. 18.13).\\n\\nSeveral types of myosin are found in nonmuscle cells. One of them, called myosin II, is very similar to muscle myosin but does not assemble into filaments to the same extent.\\n---\\nMICROFILAMENTS\\n\\n|Low calcium, actin depolymerization (gel-sol)|High calcium, actin polymerization (sol-gel)|\\n|---|---|\\n|endoplasm (sol)|ectoplasm (gel)|\\n|myosin contraction| |\\n\\nTail\\n\\nPseudopodium plasma membrane focal contacts Surface\\n\\nDirection of movement\\n\\nFigure 18.14. Amoeboid movement resembles the progression of a military tank.\\n\\nprobably because the levels of force required within nonmuscle cells is relatively small. The primary role of myosin II is in cell division (page 404). Myosin V is also two-headed and is responsible for carrying cargo (vesicles or organelles) along actin filaments. Unlike the microtubule-associated motors dynein and kinesin, which can make long journeys, myosin V can only make short excursions along an actin filament before falling off.\\n\\nCell Locomotion For a cell such as an amoeba to crawl across a surface it must generate traction. Less than 1% of the ventral surface of a crawling cell is in contact with the surface over which it is moving, so that the cell does not slither along on its belly but rather “tiptoes” delicately across the surface. The transient points of attachment with the surface are called focal contacts. At the leading edge actin polymerization (Fig. 18.11) causes the cell to extend projections or pseudopodia in the direction of progress. The overall increase in actin polymerization and crosslinking in this region causes the cytoplasm to form a viscous ectoplasm (Fig. 18.14). In contrast, raised cytosolic calcium concentrations in the tail activate actin-severing proteins such as gelsolin. Calcium also activates myosin II, the resulting contraction driving the now fluid endoplasm forward and displacing the existing actin cytoskeleton backward. Since actin is connected via the focal contacts to the surface, the backward movement of the ectoplasm drives the cell forward, operating rather like the tracks of a military tank.\\n\\nCytoplasmic Streaming Even though the cells themselves do not move, the cytoplasm of many plant cells also shows active cytoplasmic streaming. In the epidermal cells of the leaves of the water plant, Elodea, a belt of moving cytosol carries the chloroplasts and other organelles in a continuous, unidirectional stream around the central vacuole. In the giant cells of the alga Nitella, the chloroplasts are embedded in a layer of viscous ectoplasm while other organelles are carried along in the flowing endoplasm. Cytoplasmic streaming is driven by the interaction of myosin molecules attached to the moving organelles with cables of actin filaments in the ectoplasm.\\n---\\n# MECHANICAL MOLECULES\\n\\nIntermediate filaments were so named originally because their diameter, 10 nm, lay between that of the thin and thick muscle filaments. They are the most stable of the cytoplasmic filament systems. Although intermediate filaments from different mammalian tissues look much the same in the electron microscope, they are in fact composed of different protein monomers (and can therefore be distinguished by immunofluorescence, page 6). Nerve cells contain neurofilaments, muscle cells contain desmin filaments, fibroblasts contain vimentin filaments, and epithelial cells contain filaments composed of keratin, the protein that gives our skin its protective coating, forms our hair and finger nails, and makes the horns and hooves of our domestic animals and pets. These different proteins can generate a common structure because all share the same basic design. This consists of a central, α-helical rod and a nonhelical head and tail. Most of the variation between intermediate filament proteins is in the head and tail, and these regions probably confer subtly different properties on different intermediate filament classes. The basic building block of intermediate filaments is a tetramer of pairs of subunit proteins joined by their central region to form coiled coils.\\n\\nIn the cell, intermediate filaments tend to form wavy bundles that extend from the nucleus to the cell surface (Fig. 18.1). It appears that the nucleus is suspended by intermediate filaments stretching to the plasma membrane, rather like a sailor in a hammock. Indeed intermediate filaments share a high degree of amino acid sequence homology with the nuclear lamins that reinforce the nuclear envelope on the inside (page 57).\\n\\n# Anchoring Cell Junctions\\n\\nThe cells that form tissues in multicellular organisms are attached together with anchoring junctions (Fig. 18.15). Integral membrane proteins called cell adhesion molecules, of which cadherin is an example, extend out from each cell and bind tightly together, while their cytosolic domains attach to the cytoskeleton. There are two basic types of anchoring junction. In adherens junctions the cell adhesion molecules are linked to actin microfilaments by linking proteins such as catenin. In desmosomes the cell adhesion molecules are linked to intermediate filaments. Tissues that need to be mechanically strong, such as the epithelial cells of the gut (page 14) and cardiac muscle, have many anchoring junctions linking the cytoskeletons of the individual cells. Anchoring junctions are one of the three types of cell junctions, the others being tight junctions (page 55) and gap junctions (page 55).\\n\\nExample 18.2\\nProtected by the Dead\\n\\nThe epidermis of the skin is made up of a layer of living cells called keratinocytes covered by a protective layer of their dead bodies. Dead keratinocytes form a good protective layer because while alive they generate a dense internal cytoskeleton of the intermediate filament keratin, with adjacent cells being linked by desmosomes. When the cells die, the keratin fibers remain because intermediate filaments are stable. Since the intermediate filaments were joined by desmosomes, the resulting protective fibers do not stop at the edge of the now dead cell, but are strongly connected with the fibers in the next cell, and the next, and so on, forming an extremely strong network of fibers.\\n---\\n|cadherin|\\n|---|\\n|adherens junction|catenin|\\n|cadherin|actin filament|\\n|desmosome|cadherin|\\n|intermediate filament|\\n|microfilament|\\n|extracellular medium|cytosol|\\n|cadherin|catenin|\\n|actin monomer|\\n\\nFigure 18.15. Anchoring junctions attach the cytoskeletons of adjacent cells.\\n\\n397\\n---\\n# MECHANICAL MOLECULES\\n\\n# SUMMARY\\n\\n1. The cytoskeleton is made up of microtubules, microfilaments, and intermediate filaments.\\n\\n2. Microtubules are composed of equal amounts of α and β tubulin. In animal cells their minus (−) end is stabilized at the centrosome while their plus (+) end shows dynamic instability.\\n\\n3. Cilia and flagella contain a 9+2 axoneme of microtubules plus the motor protein dynein.\\n\\n4. Dynein and kinesin transport cargo along microtubules.\\n\\n5. Microfilaments are composed of actin.\\n\\n6. Actin-binding proteins control actin polymerization and determine the organization of the actin lattice.\\n\\n7. The motor protein myosin operates in all cell types but is particularly prominent in muscle cells. It operates on actin microfilaments when cytosolic calcium concentration rises.\\n\\n8. Cell locomotion is driven by spatially distinct zones of actin polymerization and depolymerization, aided by myosin. A similar cycle underlies cytoplasmic streaming in plants.\\n\\n9. The proteins that form intermediate filaments differ in different cells. Intermediate filaments are more stable than microtubules and microfilaments and serve a structural role only.\\n\\n10. Anchoring cell junctions attach the cytoskeletons of adjacent cells together. They are divided into adherens junctions, which link to actin, and desmosomes, which link to intermediate filaments.\\n\\n# FURTHER READING\\n\\nHirokawa N. 1998. Kinesin and dynein superfamily proteins and the mechanism of organelle transport. Science 279: 519–526.\\n\\nPreston, T. M., King, C. A., and Hyams, J. S. 1990. The Cytoskeleton and Cell Motility. Glasgow: Blackie.\\n\\n# REVIEW QUESTIONS\\n\\nFor each question, choose the ONE BEST answer or completion.\\n\\n1. Which of the following are not components of the cytoskeleton?\\n\\nA. Actin filaments\\n---\\n# REVIEW QUESTIONS\\n\\n|1.|Which of the following cytoskeletal elements is involved in cell motility?|\\n|---|---|\\n| |A. Microtubules|\\n| |B. Collagen fibers|\\n| |C. Microtubules|\\n| |D. Microfilaments|\\n| |E. Intermediate filaments|\\n\\n|2.|Which of the following statements are incorrect?|\\n|---|---|\\n| |A. Dynein mediates retrograde axonal transport.|\\n| |B. Myosin molecules move along actin filaments.|\\n| |C. Muscle thin filaments are composed of actin.|\\n| |D. Bacterial flagella have a 9+2 structure.|\\n| |E. Cilia are abundant in the human respiratory tract.|\\n\\n|3.|Arp, profilin, and villin are all|\\n|---|---|\\n| |A. cell adhesion molecules.|\\n| |B. molecular motors.|\\n| |C. actin-binding proteins.|\\n| |D. intermediate filament proteins.|\\n| |E. nonmuscle myosins.|\\n\\n|4.|Plant cells lack|\\n|---|---|\\n| |A. microtubules.|\\n| |B. centrioles.|\\n| |C. actin.|\\n| |D. gelsolin.|\\n| |E. cellulose synthase.|\\n\\n|5.|In cross section cilia and flagella have nine equally spaced sets of microtubule structures. This organization—nine equally spaced units—is also shown by|\\n|---|---|\\n| |A. nuclear lamins.|\\n| |B. the nuclear pore.|\\n| |C. bacterial flagellin.|\\n| |D. centrosomes.|\\n| |E. connexins.|\\n\\n|6.|Motor proteins acting on actin microfilaments include|\\n|---|---|\\n| |A. dynamin.|\\n| |B. myosin.|\\n| |C. kinesin.|\\n| |D. dynein.|\\n| |E. all of the above.|\\n\\n|7.|Desmosomes differ from adherens junctions in that|\\n|---|---|\\n| |A. desmosomes are structurally strong, while adherens junctions block the movement of extracellular medium.|\\n| |B. desmosomes block the movement of extracellular medium, while adherens junctions are structurally strong.|\\n| |C. desmosomes connect to intermediate filaments while adherens junctions connect to microtubules.|\\n| |D. desmosomes connect to intermediate filaments while adherens junctions connect to microfilaments.|\\n| |E. desmosomes anchor the nucleus in position within the cell while adherens junctions connect cells together.|\\n---\\n# MECHANICAL MOLECULES\\n\\nANSWERS TO REVIEW QUESTIONS\\n1.  B. Collagen is a component of pe extracellular matrix.\\n2.  D. The bacterial flagellum is made of pe protein called flagellin; it does not have a 9 + 2 structure. Tubulin and dynein, pe components of pe 9 + 2 axoneme, are only found in eukaryotes.\\n3.  C.\\n4.  B. Plant cells do not contain centrioles, and it remains unclear wheper peir microtubules arise from a defined organizing center.\\n5.  D. Centrosomes are made of tubulin and in cross section have an organization similar to pat of cilia and flagella. Bacterial flagellin has no obvious subdivision; nuclear lamins, pe nuclear pore, and connexins are formed of subunits arranged in patterns of 4, 8, and 6, respectively.\\n6.  B. Myosin acts on actin. In contrast kinesin and dynein act on microtubules, and dynamin causes pe pinching off of claprin-coated vesicles.\\n7.  D. Bop desmosomes and adherens junctions are types of anchoring cell junction, desmosomes connecting to intermediate filaments while adherens junctions connect to microfilaments.\\n---\\nCELL CYCLE AND CONTROL OF CELL NUMBER\\n\\nAll cells arise by the division of an existing cell. The life of a cell from the time it is generated by the division of a progenitor cell to the time it in turn divides is called the cell division cycle or just the cell cycle for short. The duration of the cell cycle varies from 2 to 3 h in a single-celled organism like the budding yeast Saccharomyces cerevisiae to around 24 h in a human cell grown in a culture dish. During this period the cell doubles in mass, duplicates its genome and organelles, and partitions these between two new progeny cells. These events have to be carried out with great precision and in the correct order, and cells have established sophisticated control processes to ensure that the cell cycle proceeds with the required accuracy. In humans the cells of some tissues, such as the skin, the lining of the gut, and the bone marrow continue to divide throughout life. Others, such as the light-sensitive cells of the eye and skeletal muscle cells, show almost no replacement. The latter, laid down in infancy, must last a lifetime. Not only must cell division be precisely controlled in terms of which cells divide when, the whole process has to know when to stop. A human being is bigger than a mouse and smaller than an elephant. Allowing for some minor differences in basic design, this is because humans are made up of more cells than a mouse (and, hence, are the result of more cell cycles) and less than an elephant. How are these differences achieved? Why don’t humans grow to be the size of elephants or whales? What has become increasingly clear over recent years is that cells contain what have become known as “cell cycle control genes.” The proper functioning of such genes not only determines how big we are, it also prevents cell division becoming “out of control” and leading to cancer.\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C 2004 by John Wiley & Sons, Inc.\\n---\\n# CELL CYCLE AND CONTROL OF CELL NUMBER\\n\\nUnder the microscope it is possible to distinguish two elements of the cell cycle. Interphase occupies about 90% of the cell cycle and is a period of synthesis and growth, during which the cell roughly doubles in mass but without displaying obvious morphological changes. Once interphase is complete, the cell enters mitosis, which is a brief period of profound structural changes. The focal point of mitosis is the behavior of the chromosomes, and it is this that we will deal with first.\\n\\n# STAGES OF MITOSIS\\n\\nMitosis is designed to produce two progeny cells each containing an identical set of the progenitor cell’s chromosomes. To achieve this, the chromosomes execute a precisely choreographed sequence of movements that were first described well over a century ago. Classically, mitosis is divided into five stages, each of which is characterized by changes in the appearance of the chromosomes and their organization with respect to a cellular structure, called the mitotic spindle, that is responsible for their segregation. The stages of mitosis are shown diagramatically in Figure 19.1.\\n\\nOne copy of our genome is encoded in 23 chromosomes and is made up of 3 × 109 base pairs. Human cells contain 46 chromosomes, 23 inherited from each parent. This state, in which each cell has two complete sets of chromosomes, is called diploid. For simplicity\\n\\n|microtubule|centrosome|nuclear envelope|(a) Interphase|(b) Prophase|\\n|---|---|---|---|---|\\n|chromatids|kinetochore| |Prometaphase|(c)|\\n| | | |Anaphase|Metaphase|\\n| | | |(d)| |\\n| | | | |spindle|\\n\\nFigure 19.1. Stages of mitosis. One pair of chromosomes is shown, one chromosome originating from the father (light green kinetochores), and the matching one from the mother (dark green kinetochores).\\n---\\n# STAGES OF MITOSIS\\n\\nFigure 19.1 shows just one pair of chromosomes, one that came from the father (shown with light green kinetochores), and the matching one from the mother (dark green kinetochores).\\n\\n# Prophase\\n\\nThe first evidence of mitosis in most cells is the compaction of the threads of chromatin that existed through interphase into compact chromosomes that are visible in the light microscope. As the chromosomes compact, each can be seen to be paired structures composed of two chromatids. This is the visible effect of the DNA molecules having been replicated (page 87) in interphase. Chromosome condensation reduces the chance of long DNA molecules becoming tangled and broken. Each chromosome has a constriction called the kinetochore, a structure that forms around a region rich in satellite DNA (page 100) called the centromere. The kinetochore is the point of attachment of the chromosome to the spindle. At the same time as the chromosomes are condensing within the nucleus, the centrosomes, which lie on the cytoplasmic side of the nuclear envelope, begin to separate to establish the mitotic spindle.\\n\\n# Prometaphase\\n\\nAt the breakdown of the nuclear envelope, the chromosomes become free to interact with the forming spindle. Microtubule assembly from the centrosomes is random and dynamic. The growing ends of individual microtubules make chance contact with and are captured by the kinetochores. Because of the random nature of these events, the kinetochores of chromatid pairs are initially associated with different numbers of microtubules, and the forces acting upon each chromosome are unbalanced. Initially, therefore, the spindle is highly unstable and chromosomes make frequent excursions toward and away from the poles. Gradually, a balance of forces is established and the chromosomes become aligned at the equator, with the kinetochores of each member of a chromatid pair oriented toward opposite poles.\\n\\n# Example 19.1 Counting Chromosomes\\n\\nThe DNA molecules of an interphase cell can only be observed using an electron microscope (page 8). However, at mitosis, when they condense, individual chromosomes can be made out easily using a standard light microscope (page 76). Mitotic cells can then be screened for chromosome abnormalities such as the presence of three copies of chromosome 21, a condition that causes Down syndrome, and the “Philadelphia chromosome,” a rearrangement between chromosomes 9 and 21 (which results in a form of leukemia).\\n\\n# Metaphase\\n\\nMetaphase is the most stable period of mitosis. The system can be regarded as being at steady state with the chromosomes lined up rather like athletes at the start of a race (albeit half facing in one direction and half the other). The metaphase spindle consists of two major groups of microtubules: those connecting the chromatids to the poles and a second group extending from each pole toward the other. The second group overlap at the spindle equator. If a single chromatid becomes detached from the spindle (Fig. 19.2), it sends a signal that tells the other chromosomes not to start anaphase without it. The other chromosomes dutifully wait until the missing chromatid reattaches, whereupon all of the chromosomes enter anaphase together.\\n\\n# Anaphase\\n\\nThe trigger for the separation of the paired chromatids and the start of their journey to the spindle poles is the degradation of the protein cohesin, which acts as the glue holding the pairs of chromatids together. In anaphase A the microtubules holding the chromosomes shorten, pulling the chromosomes to the spindle poles. The chromosomes move as a “V” with the kinetochores, at which the force for chromosome movement is\\n---\\n# CELL CYCLE AND CONTROL OF CELL NUMBER\\n\\nFigure 19.2. Anaphase only begins when all chromatids are correctly attached to spindle microtubules.\\n\\nDuring cell division, cytoskeletal microtubules break down and the tubulin monomers reform as the mitotic spindle. Taxol (page 385) stabilizes existing microtubules, making it impossible for the cell to form spindles and therefore preventing cell division. For this reason taxol, usually referred to by its generic name palitaxel, is a valuable anti-cancer drug.\\n\\nTelophase. This stage sees the reversal of many of the events of prophase; the chromosomes decondense, the spindle disassembles, the nuclear envelope reforms, Golgi apparatus and endoplasmic reticulum reform, and the nucleolus reappears. Each progeny nucleus now contains one complete copy of the genome from the father and one copy from the mother. Since in Figure 19.1 we show only one pair of chromosomes, our diagram shows each nucleus containing two chromosomes, one that originated from the father plus one that originated from the mother.\\n\\nCytokinesis. During the last stages of telophase, the cell itself divides. In animal cells, a cleavage furrow made of actin and its motor protein myosin II constricts the middle of the cell; in plants, a structure called the phragmoplast forms at the equator of the spindles where it directs the formation of a new cell wall.\\n\\nMEIOSIS AND FERTILIZATION\\n\\nIn sexually reproducing organisms, the germ cells that give rise to the eggs and sperm arise by a different type of cell division from the somatic cells that make up most of the body. In regular cell division DNA synthesis is followed by mitosis so that the two progeny end\\n---\\n# MEIOSIS AND FERTILIZATION\\n\\nTo produce germ cells, a single round of DNA synthesis is followed by two nuclear divisions. The result is that an individual sperm or egg cell is haploid; that is, it contains only one copy of each chromosome. Fusion of an egg and sperm at fertilization creates a diploid cell.\\n\\n# Meiosis\\n\\nEach meiotic division (known as meiosis I and II), involves the formation of a meiotic spindle and the same sequence of prophase, prometaphase, metaphase, anaphase, and telophase. The steps are illustrated in Figure 19.3. Once again one pair of chromosomes is shown with green kinetochores, with the chromosome that came from the father shown with a light green kinetochore and the matching one from the mother with a dark green kinetochore. We also show a second chromosome, light gray and dark gray kinetochores again indicating paternal and maternal origin. Mitosis is usually a rather brief process, but meiosis is often extended.\\n\\n|Metaphase I|Anaphase I|\\n|---|---|\\n|Metaphase II|Anaphase II|\\n|Telophase I/Cytokinesis I|Telophase I/Cytokinesis I|\\n|Telophase II/Cytokinesis II|Telophase II/Cytokinesis II|\\n\\nFigure 19.3. Stages of meiosis. Two pairs of chromosomes are shown; two paternal chromosomes (light green and light gray kinetochores) and two matching maternal ones (dark green and dark gray kinetochores).\\n---\\n# CELL CYCLE AND CONTROL OF CELL NUMBER\\n\\nand, in different organisms can last for months or even years. Most of this is occupied by a lengthy prophase of meiosis I, prophase I. Prophase I differs from mitotic prophase in that the paternal and maternal chromosomes (which, because DNA synthesis has occurred in the preceding interphase, are each composed of two chromatids) line up side by side. As the cell progresses to metaphase I, the maternal/paternal chromosome pairs line up along the metaphase plate. At anaphase I, the homologous pairs separate but, unlike the situation in mitosis, the paired chromatids remain attached and journey together to the pole. The two progeny nuclei formed during telophase I almost immediately enter meiosis II. Prophase II is often so brief as to be undetectable. Metaphase II and anaphase II resemble their mitotic counterparts, the chromatids finally separating to give haploid gametes (sperm or eggs). For each part of the genome, each gamete contains only one copy, derived from either the father or the mother. Since in Figure 19.3 we show only two pairs of chromosomes, one with green and one with gray centromeres, the final panel in the figure shows each gamete containing two chromosomes.\\n\\nAlthough male and female meiosis in animals follow roughly the same lines, there are some important distinctions. In males meiosis produces four equal sized haploid cells called spermatids, each subsequently developing into a spermatozoon. In females, both meiotic divisions are asymmetric, resulting in one large cell that survives, the oocyte, and three small cells, called polar bodies, that are discarded.\\n\\n# Fertilization and Inheritance\\n\\nAt fertilization the sperm cell fuses with the much larger egg cell. The nuclei that originated in sperm and egg, now called pronuclei, move together and eventually fuse to restore the normal somatic cell chromosome number. This diploid cell then divides many times by mitosis to generate the multicellular organism.\\n\\nMany of the attributes that make each human being distinct are determined by the pattern of genes we inherit. The simplest pattern of inheritance is seen when one form of a gene fails to make a functional protein. Brown eye color is an example. If you inherit a functional BEY (brown eye) gene, then you can make brown pigment and your eyes will be brown. The gene is found on chromosome 15. If you inherit a chromosome from one parent with a defective BEY gene, and a chromosome from the other parent with a functional BEY gene, then you can make enough of the BEY protein to make brown pigment and your eyes will be brown. Brown eye color is said to be dominant.\\n\\nIf both of your BEY genes are defective, a different gene, on chromosome 19, becomes important. The GEY (green eye) gene codes for a protein that makes green pigment. If you have a functional BEY gene, then the GEY gene is largely irrelevant, because the brown pigment drowns out the more subtle green pigment. However, if you have two nonfunctional BEY genes and at least one functional GEY gene, you have green eyes. If both BEY genes and both GEY genes are nonfunctional, you don’t make brown pigment or green pigment and your eyes are blue. Green eye color is said to be recessive to brown because an individual can only have green eyes if they lack any brown eye genes. Similarly, blue eye color is recessive to brown and green because an individual can only have blue eyes if he or she lacks any functional brown and green eye genes.\\n\\nClearly most of us are not unhappy with green or blue eyes. However, when a more vital protein doesn’t work, the results are much less benign. Phenylketonuria is a recessive genetic disease that leads to severe mental retardation (page 294). If an individual inherits a chromosome from one parent with a defective phenylalanine hydroxylase gene, and a\\n---\\n# MEIOSIS AND FERTILIZATION\\n\\nchromosome from the other parent with a functional phenylalanine hydroxylase gene, then they can make enough phenylalanine hydroxylase to convert phenylalanine to tyrosine and are perfectly normal. However, when a baby inherits a nonfunctional phenylalanine hydroxylase gene from each parent, then they have the potentially disastrous condition called phenylketonuria (page 294). People with one functional gene and one nonfunctional one are called carriers. If two carriers of the same defective gene have a child, that child has a one in four chance of getting a defective gene from both parents and exhibiting the recessive condition, be it nonbrown eyes or phenylketonuria.\\n\\nMedical Relevance\\n\\nWhy Marrying Your Cousin Is Dangerous\\n\\nMarriage between first cousins is illegal in 30 states of the United States. The scientific rationale behind these laws is that cousin marriage provides an easy mechanism for a single carrier of a recessive genetic disease to generate great-grandchildren with the disease. The accompanying figure shows how this could happen in an imaginary family tree. Harry and Ingrid are first cousins: they share a grandmother, Alice, who was a carrier for phenylketonuria. We indicate this in the letters below Alice’s name: P indicates the gene coding for a functional phenylalanine hydroxylase while p indicates the gene coding for a defective protein. Both Alice’s children, Constance and David, inherited her defective gene and were therefore carriers. Of their four children, Harry, Ingrid, and Jill inherited the defective gene and are carriers. The children of Harry and Ingrid therefore each had a one-in-four chance of inheriting one defective gene from Harry plus one defective gene from Ingrid. One of their children, Maria, does so and has phenylketonuria.\\n\\nIn the case we have drawn we know that Constance, David, Harry, and Ingrid are carriers. If we did not know this, what is the probability that a child of first-cousin parents will have phenylketonuria, given that one shared grandparent is a carrier? This is easy to calculate: each child of a carrier has a 1-in-2 chance of inheriting the defective gene, while each child of two carriers has a 1-in-4 chance of having the disease. The overall probability is therefore 1/2 × 1/2 × 1/2 × 1/2 or 1/4, under 1.6% per child. The extent to which this risk is significant enough to require legislation, given the low frequency of cousin marriage in those countries and states where it is allowed, is debatable.\\n\\n| |Alice|Brad|\\n|---|---|---|\\n|Edward|PP|Pp|\\n|Constance|Pp|PP|\\n|David|Pp|PP|\\n|Frances|Pp|PP|\\n|Geoff|PP|Pp|\\n|Harry|Pp|Pp|\\n|Ingrid|Pp|Pp|\\n|Jill|Pp|Pp|\\n|Kyle|PP|Pp|\\n|Lucia|Pp|Pp|\\n|Maria|pp|pp|\\n---\\n# CELL CYCLE AND CONTROL OF CELL NUMBER\\n\\nDominant Genetic Disease\\n\\nMore unusually, a defective gene may cause a problem even if the individual has one good copy, that is, the defective gene is dominant over the normal one. One example is familial Creutzfeldt–Jacob disease. As we described earlier (page 207), the brain damage that occurs in Creutzfeldt–Jacob disease results when an alternatively folded form of PrP protein called PrP sc causes all the PrP protein around it to fold up in the alternative way. Sufferers from familial Creutzfeldt–Jacob disease have a mutant form of the PrP gene that generates protein that spontaneously folds into the PrP sc form. The PrP sc then triggers the normal protein, generated by the second, normal copy of the gene, to fold into the PrP sc shape. Thus one copy of the defective gene is sufficient to cause the disease.\\n\\nCrossing Over and Linkage\\n\\nProphase I lasts such a long time because the chromosomes deliberately tie themselves in knots and then untangle (using topoisomerase II, page 74). Figure 19.4 shows what happens. At the top we see the paternal and maternal chromosomes, each composed of two chromatids, lined up side by side. As before we show the chromosome that originated from the father in light green and the maternal one in dark green. During prophase 1 the chromosomes are cut and resealed at points called chiasmata (singular chiasma) so that lengths of paternal chromosome are transferred to a maternal one and vice versa. The rest of meiosis I proceeds, followed by meiosis II, and the end result is that some gametes contain chromosomes that are neither completely paternal nor completely maternal but are a recombination of the two.\\n\\nThe biological advantage of sexual reproduction is that it allows organisms to possess a random selection of the genes from their ancestors. Those individuals with a complement of genes that makes them better suited to their environment tend to do better, allowing evolution by natural selection of the individuals possessing the better genes. Without crossing over this could not happen: Those genes that are located on the same chromosome would remain linked down the generations, greatly reducing the number of gene permutations possible at each generation. Crossing over allows a child to inherit, for example, his grandmother’s green eyes without also inheriting her defective sodium channel gene (page 331), although both genes are on chromosome 19. Even with crossing over, genes on the same chromosome are inherited together more than they would be if they were on different chromosomes. The closer the genes, the less likely it is that a chiasma will form between them, and therefore the greater the probability that they will be inherited together. This phenomenon is used to help identify the genes responsible for specific diseases such as cystic fibrosis (page 426).\\n\\nCONTROL OF THE CELL DIVISION CYCLE\\n\\nAlthough mitosis is the most dramatic period of the cell cycle, it is the culmination of a sequence of biochemical and structural events that occur during interphase. The most important of these is the duplication of the genome in S phase (the “S” stands for DNA synthesis). In this cell cycle shorthand, mitosis is referred to as M phase. S phase and M phase must occur (a) in the correct order and (b) only once per cell cycle. To make sure this is the case, S phase and M phase do not follow immediately after one another but are separated by “gaps” that allow the cell to check that everything is in order before going.\\n---\\n# CONTROL OF THE CELL DIVISION CYCLE\\n\\nChiasmata allow crossing over of genetic material during prophase I of meiosis.\\n\\nOn to the next stage. The gap between M phase and S phase is called G1 (gap 1), and the gap between S phase and M phase is called G2 (gap 2). These four phases, G1, S, G2, and M make up the classic cell cycle “clock” (Fig. 19.5). Nondividing or quiescent cells are said to be in the G0 (gap 0) phase of the cell cycle. Cells can only enter G0 from G1. G0 cells can remain viable for months or even years, and most of the cells in the human body are in fact in this nondividing state. Cells are said to be terminally differentiated if they are unable to return to the cell division cycle. Nerve cells are one such example. In contrast\\n---\\n# CELL CYCLE AND CONTROL OF CELL NUMBER\\n\\nAmount of DNA in non-dividing cell is doubled during DNA replication in the nucleus.\\n\\n|Non-Dividing Cell|DNA Replication|\\n|---|---|\\n|G0|S Phase|\\n|G1|G2|\\n|Mitosis|Cell Division|\\n\\nFigure 19.5. Cell division cycle.\\n\\nOther differentiated cells, such as glial cells (page 13), can return to the cycle if they receive the correct signals from their neighbors.\\n\\nFor dividing cells, two points in the cell cycle are particularly critical. The first is at the G1/S boundary at which point the cell is committed to DNA replication. The second is at the G2/M transition when it is committed to mitosis. These are the major control points of the cell cycle, and before crossing them the cell must be sure that conditions are such that S phase and M phase can be executed successfully. At G1/S, the cell must decide whether it is big enough and whether nutritional conditions are appropriate to begin the crucial process of replicating its genome. In G2 the primary concern is that its DNA is in perfect condition before entering mitosis. There are sensitive mechanisms for detecting the presence of unreplicated or damaged DNA, and cells will not commit themselves to mitosis until any defects have been attended to.\\n\\n# Molecular Regulation of the G2/M (Interphase/Mitosis) Cell Cycle Control Point\\n\\nAs cells enter mitosis, they undergo a remarkable sequence of structural changes. Not only do the chromosomes condense as we have seen above but, in addition, the nuclear envelope breaks down, the nucleolus disperses, the membranes of the Golgi apparatus and endoplasmic reticulum fragment, and the cytoskeleton undergoes remodeling to form a completely new cellular structure, the mitotic spindle. How are so many and diverse\\n---\\n# CONTROL OF THE CELL DIVISION CYCLE\\n\\nFigure 19.6. Cell fusion experiments reveal how constituents of the cytoplasm control the behavior of the nucleus. Mitotic cells are identified by the presence of the mitotic spindle. Nuclei in S, G1, and G2 phases are represented by different color shades.\\n\\n|S + G1|(a)|\\n|---|---|\\n|S + G2|(b)|\\n|G1 + G2|(c)|\\n|G2 + M|(d)|\\n\\nStructural changes coordinated in a narrow window of the cell cycle, often lasting only a few minutes? During the 1960s some remarkable experiments started the hunt that led to the identification of a cytoplasmic master switch that activated all these changes. These involved fusing together cultured mammalian cells at different stages of the cell cycle (Fig. 19.6). These experiments demonstrated that cell cycle progress was under the control of cytoplasmic factors and that cells had to complete certain cell cycle steps before entering subsequent ones. In particular, row (d) shows how fusion of an M-phase cell with a G2 cell causes the G2 nucleus to enter mitosis. Similar results were obtained when cytoplasm from unfertilized Xenopus oocytes (which are naturally arrested in meiotic M phase) was injected into interphase oocytes (Fig. 19.7). The implication of these experiments was that the cytoplasm of dividing cells contained a factor capable of controlling entry into mitosis. This activity was given the name M-phase promoting factor, or MPF. Defining MPF in biochemical terms took almost 20 years and culminated in the award of the 2001 Nobel prize in physiology or medicine to Paul Nurse, Tim Hunt, and Leland Hartwell for their pioneering work on the identification of the molecules that drive the cell division cycle.\\n\\nRemarkably, given the variety of the changes it induces, MPF is a complex of only two proteins. These are an enzyme called cyclin-dependent kinase 1 (Cdk1) and its regulatory subunit called cyclin B. Cdk1 can act as a cell cycle switch because it is a protein kinase, an\\n---\\n# CELL CYCLE AND CONTROL OF CELL NUMBER\\n\\nFigure 19.7. Injection of cytosol from a mitotic cell into an interphase one triggers mitosis in the interphase nucleus.\\n\\nThe substrates of Cdk1 are proteins associated with the chromosomes, the nuclear envelope, nucleolus, centrosomes, and so on and phosphorylation of such proteins sets in train the structural changes that culminate in mitosis. However, this activity must be activated only at the G2/M boundary and then must be quickly inactivated if the cell is not to undergo multiple rounds of mitosis. Figure 19.8 shows how Cdk1 activity is regulated through the cell cycle. Without cyclin B Cdk1 is very slow at phosphorylating its target but, to ensure that it remains completely inactive through interphase, it is itself phosphorylated by another protein kinase called Wee1 (because mutants, with unphosphorylated Cdk1, divide too early, while they are still “wee”—Scottish for small!). Wee1 adds two phosphates to Cdk1, one on a threonine residue at amino acid number 14, the other on a tyrosine residue at amino acid number 15. These two amino acids lie within the ATP-binding site of Cdk1, and the presence of two phosphates prevents ATP binding, the first step of the phosphorylation reaction. Immediately following mitosis, the concentration of cyclin B in the cell is vanishingly small. Cyclin B is made steadily so its concentration gradually increases, and so more and more Cdk1 comes to have cyclin B attached (Fig. 19.8b). However, this heterodimer is inactive because it is phosphorylated on T14 and Y15. At the G2/M boundary the phosphates are removed by a second enzyme called Cdc25. MPF is now active and can phosphorylate its target proteins. Once mitosis is initiated, it is essential that Cdk1 be deactivated for the reasons outlined above. This is achieved by the destruction of cyclin B and the rephosphorylation of Cdk1 by Wee1.\\n\\nWhat About the G1/S Control Point?\\n\\nThe decision to begin DNA synthesis is arguably an even more critical decision than entry into M phase because once a cell begins, it is committed to eventually dividing. (Cells that replicate their DNA but then fail to undergo mitosis in a reasonable time commit suicide by apoptosis (page 415) rather than hang around in a polyploid state). Cell fusion experiments again indicated that factors in the cytoplasm controlled advance to the next stage in the cell\\n---\\n# CONTROL OF THE CELL DIVISION CYCLE\\n\\n| |Active|Active|\\n|---|---|---|\\n| |cyclin B| |\\n| |Cdk1| |\\n| |Inactive|Inactive|\\n|G1|S|G2|M|G1|S|G2|M|\\n|Protein concentration| | | | | | | | |\\n| |(a)| |\\n|inactive|inactive|active kinase|inactive|\\n|T14| |P T14 Y15| |T14 Y15| |T14 Y15| |\\n|P P Y15| |P Cdk1| |active Cdk1| |P P Y15| |\\n| |Cdk1| | | |catalytic site| |Cdk1| |\\n|synthesis|cyclin B| | |cyclin B| |cyclin B|destroyed|\\n| |(b)| |\\n\\nFigure 19.8. Control of Cdk1 by cyclin B and by phosphorylation.\\n\\ncycle. In particular, fusion of an S-phase cell with a G1 cell caused the G1 nucleus to begin DNA synthesis (Fig. 19.6a). We now know that entry into S phase is regulated not by one but by three cyclin-dependent kinases called Cdk2, Cdk4, and Cdk6, which, like Cdk1, are active at high efficiency only when coupled to a cyclin partner, in this case cyclin D for Cdks 4 and 6 and cyclin E for Cdk2. The function of Cdk4 is most clearly understood and is illustrated in Figure 19.9. DNA replication requires a specific set of proteins not used in quiescent cells (page 87), and transcription of the genes coding for these proteins is activated by a transcription factor called E2F-1. In quiescent cells E2F-1 is prevented from activating transcription by being bound to RB, the product of a gene that is mutated in the cancer retinoblastoma. The main function of Cdk4 seems to be to phosphorylate RB, preventing its association with E2F-1, which is therefore freed to bind to enhancer sequences, activate transcription, and hence trigger DNA synthesis. Growth factors trigger cell division in part through activating transcription of the cyclin-D gene and hence allowing\\n---\\n# factor\\n\\n|Growth receptors|RB|\\n|---|---|\\n|P|1 - E2F|\\n\\nRas - DNA replication\\n\\nP - PKB - MAP Kinase - cyclin D synthesis\\n\\napoptosis - Cdk-4 : cyclin D\\n\\nRB - E2F\\n\\nsynthesis of BAX\\n\\nCKIs\\n\\nUV - DNA damage - p53 concentration rises\\n\\nDNA repair - activation of\\n\\nRetinoblastoma protein RB sequesters E2F-1, the critical transcription factor controlling entry into S phase.\\n\\nFigure 19.9.\\n---\\n# APOPTOSIS\\n\\nCdk4 to operate. Once S phase has begun, RB is dephosphorylated to prevent further rounds of DNA replication.\\n\\nThe G1 Cdks have an extra level of regulation above that described for Cdk1–cyclin B because they are inhibited by an additional class of proteins called CKIs for cyclin-dependent kinase inhibitor proteins. There are many CKIs mediating a number of pathways all with the same outcome: stopping cell division. One of these pathways mediates contact inhibition. Cells in a culture dish, or at the edge of a wound, divide until they touch each other. When they contact neighboring cells, they stop dividing because contact causes production of two CKIs called p16 INK4a and p27 KIP1. These inhibit the G1 Cdks and therefore prevent DNA synthesis. Loss of contact inhibition is one of the first changes seen in the transformation of normal cells into cancer cells.\\n\\nAnother CKI called p21 CIP1 is produced in response to the transcription factor p53. p53 is produced all the time in cells but is broken down rapidly, so its concentration is usually low. However, if the DNA is damaged, for example, by ultraviolet light (page 94), the destruction of p53 is halted. As p53 concentrations increase, a number of processes are activated. The systems that repair DNA are activated (page 94). p21 CIP1 is produced, inhibiting the G1 CDKs and therefore preventing replication of the defective DNA. Lastly, if the DNA cannot be repaired within a set time (usually a few hours), the cell is triggered to self-destruct in the process called apoptosis.\\n\\n# APOPTOSIS\\n\\nAn adult human is made up of about 30 trillion (3 × 10 13) cells, all of which originate from a single fertilized egg. If this first cell divides into 2, the 2 progeny cells into 4, and so on, it would take only about 45 rounds of division to produce the number of cells required to make an adult human. In fact, cell division occurs constantly through our lifetimes, such that we generate a new complete set of 3 × 10 13 cells every 2 weeks. The reason that multicellular organisms do not become infinitely large is because the proliferation of cells is balanced by cell death. Cells die for two quite different reasons. One is accidental, the result of mechanical trauma or exposure to some kind of toxic agent, and often referred to as necrosis. This is the only type of death seen in unicellular organisms. The other type of death is deliberate, the result of an built-in suicide mechanism known as apoptosis or programmed cell death. The two types of cell death are quite distinct. In cells that are injured, ATP concentrations fall so low that the Na + /K + ATPase can no longer operate, and therefore ion concentrations are no longer controlled. This causes the cells to swell and then burst. The cell contents then leak out, causing the surrounding tissues to become inflamed. Cells that die by suicide on the other hand shrink, and their cell contents are packaged into small membrane-bound packets called blebs. The nuclear DNA becomes chopped up into small fragments, each of which becomes enclosed in a portion of the nuclear envelope. The dying cell modifies its plasma membrane, signaling to macrophages (page 13), which respond by engulfing the blebs and the remaining cell fragments and by secreting cytokines that inhibit inflammation. The changes that occur during apoptosis are the result of hydrolysis of cellular proteins by a family of proteases called caspases (short for cysteine-containing, cleaving at aspartate). All the cells of our body contain caspases, but they are normally locked in an inactive form by an integral inhibitory domain of the protein. Proteolysis cleaves the inhibitory domain off, releasing the active caspases. The advantage\\n---\\n# CELL CYCLE AND CONTROL OF CELL NUMBER\\n\\n|Receptor|P tyrosine kinases|Death domain receptors|\\n|---|---|---|\\n|Ras|Protein Kinase B|inactive caspase 8|\\n|Cell division|B|active caspase 8|\\n| |p53 concentration rises|inactive caspase 9|\\n|14-3-3 protein|inactive BAD|active BAD|\\n\\nFigure 19.10. Pathways controlling apoptosis.\\n\\nThe advantage to the cell of this strategy is that no protein synthesis is required to activate the apoptotic pathway—all the components are already present. Thus, for example, if a virus infects a cell and takes over all protein synthesis, the cell can still commit suicide and hence prevent viral replication. Figure 19.10 summarizes the complex control systems that regulate the decision to die or survive. Cells activate apoptosis in response to three types of event.\\n\\nInstructed Death: Death Domain Receptors\\n\\nCells are instructed to die when a ligand binds to one of a family of death domain receptors. This occurs, for example, in the case described above: If a cell is infected by a virus, white blood cells recognize viral proteins on the cell surface and activate Fas, a death domain receptor on the surface of the unlucky cell. On binding its ligand, a death domain receptor causes caspase 8 to activate. In turn, caspase 8 can hydrolyze and hence activate the effector caspases that begin the processes of cell destruction.\\n\\nDefault Death: Absence of Growth Factors\\n\\nCells that are not required by the organism die. To make sure that this occurs, death is the default option for the cells of a multicellular organism: Only if a cell receives growth factors from other cells will it survive. We have already described the first part of this pathway.\\n---\\n# APOPTOSIS\\n\\n(page 356). Active receptor tyrosine kinases recruit and in turn activate phosphatidylinositol 3-kinase, which generates the highly charged membrane lipid phosphoinositide trisphosphate. This causes protein kinase B to visit the membrane, where it is itself phosphorylated and hence activated. Protein kinase B phosphorylates one of a family of bcl-2 family proteins called bcl-2 associated death promoter, or BAD. Phosphorylated BAD is inactive and is sequestered by binding to a protein called 14-3-3. However, if PKB is ever allowed to stop working, BAD loses its phosphates and is then active. Active BAD makes holes in the outer mitochondrial membrane (how it does this is unclear) allowing cytochrome c to escape into the cytosol. Although cytochrome c plays a vital role in allowing mitochondria to generate the H+ gradient energy currency (page 266), once in the cytosol it is deadly. It activates another caspase, number 9, which then hydrolyzes and hence activates the effector caspases.\\n\\n✄✄✄\\n▯▯▯\\n\\nExample 19.3 Neurotrophin Trafficking\\n\\nDuring fetal development motoneurons die unless they succeed in growing their axons all the way to an appropriate muscle. Those that do find the target are bathed in neurotrophin 3, a growth factor released by the muscle cells. Neurotrophin 3 binds to its receptor tyrosine kinase, called Trk C, on the nerve cell surface. However, the activated kinase then has to activate protein kinase B in the cell body in order to prevent apoptosis, and the cell body may be many millimeters away. To achieve this, the neurotrophin–receptor complex is endocytosed using the clathrin mechanism (page 229). The endocytotic vesicles are then transported back to the cell body by dynein (page 390). Once in the cell body, protein kinase B is activated and apoptosis prevented.\\n\\nThe nerve that contains motoneurons also contains pain receptor axons. Instead of Trk C these express a related receptor, Trk A, which requires a different neurotrophin (number 1): Trk A cannot be activated by neurotrophin 3. However, both motoneurons and pain receptors express a second, unrelated receptor called the p75 neurotrophin receptor, which binds both neurotrophin 1 and neurotrophin 3. p75 is a death domain receptor. The overall result is that when an axon arrives at a target, it will automatically receive a signal to die. However, if it has arrived at the correct location, it will receive a countermanding signal to survive.\\n\\nThe Sick Are Left to Die: Stress-Activated Apoptosis\\n\\nIf a unicellular organism is damaged, it will try to repair itself since the alternative is the death of the organism. However, if a cell of a multicellular organism is stressed or damaged, it may be more efficient to allow the cell to undergo a quick and polite suicide and replace it by cell division of a healthy neighbor. There are therefore a number of mechanisms that trigger apoptosis in response to cell stress. One mechanism operates directly on mitochondria. When mitochondria are stressed, they can spontaneously release cytochrome c: This appears to happen when porin, a channel of the outer mitochondrial membrane, associates with other proteins to form a channel of large enough diameter to allow cytochrome c to leak out. This is a major medical problem because it can occur in tissues that are unable to rebuild by cell division: in hearts during heart attacks and in brain nerve cells during a stroke.\\n\\nA second mechanism operates through the transcription factor p53, which we have already met in the context of cell cycle control (page 415). p53 concentrations increase when DNA is damaged. DNA repair mechanisms are activated, but so is transcription of the gene for another bcl-2 family protein called BAX. Like BAD, BAX allows cytochrome\\n---\\n# CELL CYCLE AND CONTROL OF CELL NUMBER\\n\\nc to escape from mitochondria, so if the DNA is not repaired in time, BAX concentrations\\nincrease, cytochrome        c escapes, and apoptosis results. Without this mechanism, cancers,\\nwhich result from changes in the DNA of our body cells, would be far more abundant.\\n\\nExample 19.4 Sunburn, Skin Cancer, and Cell Death\\n\\nWithout knowing it, we have all observed apoptosis in action. All of us at some time or other have\\nbeen out in the sun without proper protection. The ultraviolet (UV) light causes damage to the DNA\\nof the skin cells and activates p53. If the DNA damage is minor, p53 simply arrests the cell cycle\\nuntil the DNA repair machinery has had time to repair the lesions. However, if the damage was more\\nsevere, the skin cells activate the apoptoptic pathway—and the sunburned skin dies and sloughs off.\\n\\nA third mechanism (not shown in Fig. 19.10) operates through a protein kinase called\\np38. Although p38 is related to MAP kinase, it is activated not by growth factors but rather\\nby cell stresses such as swelling, shrinkage, or radiation. The effects of p38 and another\\nstress-activated protein kinase, called JNK, are complex, and we will consider only two.\\np38 phosphorylates p53, which protects it from the rapid breakdown that normally occurs.\\nAn increase of p53 concentration then triggers the usual consequences, including synthesis\\nof BAX. p38 also has a direct effect on the founder member of the bcl-2 family, bcl-2\\nitself. Unlike BAD and BAX, bcl-2 cannot trigger cytochrome c release from mitochondria.\\nIndeed, bcl-2 is antiapoptotic because it can bind to BAD and BAX and prevent them from\\nreleasing cytochrome c. p38 phosphorylates bcl-2, preventing it from associating with BAD\\nand BAX. When BAD and BAX are freed, they can trigger cytochrome c release.\\nSince caspases can self-activate by mutual proteolysis, there is a steady slow rate of\\nactivation even in a healthy cell. To prevent this triggering apoptosis, cells produce apoptosis\\ninhibitor proteins, which block caspase action. If the extent of caspase activation exceeds\\nthe capacity of the inhibitory proteins, death results.\\n\\n# IN DEPTH 19.1 A Worm’s Eye View of Cell Death\\n\\nOur understanding of the mechanisms and importance of cell death has been\\ngreatly influenced by work on the nematode worm, Caenorhabditis elegans. This\\ntiny creature (about 1 mm long) is a true animal with nervous, digestive, and\\nmuscular systems but is small enough to be grown in Petri dishes (where it is fed\\na diet of Escherichia coli ). Crucially, every C. elegans worm is made up of a precise\\nnumber of cells. In the newly hatched worm this number is 558, rising to 959\\nsomatic cells and a variable number of germ cells in the adult. Because the worm\\nis transparent, it has been possible to trace the lineage of every somatic cell back\\nto the original fertilized egg. This pattern is identical in every worm examined.\\nStrikingly, exactly 131 cells die during the developmental process as the result of\\nprogrammed cell death.\\nWorkers trying to understand this process isolated ced (= cell death defective)\\nmutants in which various organs contained abnormally high numbers or abnor-\\nmally low numbers of cells because the cell death program was abnormal. When\\nthese genes were cloned, it became clear that similar genes were present in other\\nanimals including humans. For example, ced-3 was found to code for a proteolytic\\nenzyme. Screening the human genome for similar enzymes revealed the family of\\napoptotic proteases we now know as caspases.\\n---\\nSUMMARY\\n\\nThe contribution of pe simple nematode to biology was recognized by pe award of pe 2002 Nobel prize in physiology or medicine to pe pioneers of nematode research, Sydney Brenner, Robert Horvitz, and John Sulston.\\n\\nSUMMARY\\n\\n1. Haploid cells contain only one copy of each chromosome. Diploid cells contain two copies, one from the organism’s father, one from the mother.\\n2. The cell cycle comprises S, G1, M, and G2 phases. G2, S, and G1 phases together constitute interphase. DNA is replicated in S phase, so that each chromosome becomes a pair of identical chromatids. The cell physically divides in mitosis, or M phase.\\n3. Mitosis comprises prophase, prometaphase, metaphase, anaphase, telophase, and cytokinesis. The end result of mitosis is two diploid cells whose chromosome complement is the same as that of the original cell before it underwent S phase.\\n4. Meiosis generates haploid germ cells: in vertebrates, eggs and sperm. Like mitosis, it follows an S phase but comprises two cycles of cell division, so that the end result is four cells whose chromosome complement is only half that of the original cell before it underwent S phase.\\n5. During meiosis I homologous chromosomes undergo recombination, a physical resplicing of homologous chromosomes that allows information on chromosomes originating from father and mother to be mixed.\\n6. Recessive genes are usually those that fail to make functional protein. Examples are the gene for blue eyes and the gene for phenylketonuria. The functional gene is called dominant because an individual need inherit only one copy to be able to make functional protein.\\n7. In a few cases a defective gene, for example, the gene for familial Creutzfeldt–Jacob disease, is dominant over a normal one.\\n8. Cells enter mitosis when cyclin-dependent kinase 1 is active. This in turn requires that cyclin B be present in high enough concentration and that cyclin-dependent kinase 1 be dephosphorylated by Cdc25. Once mitosis is initiated, cyclin-dependent kinase 1 is rapidly turned off through phosphorylation by WEE1 in parallel with the proteolytic destruction of cyclin B.\\n9. The entry of cells into S phase is a more complex decision involving cyclin-dependent kinases 2, 4, and 6. The main effect of active cyclin-dependent kinase 4 is to phosphorylate RB, causing it to release the transcription factor E2F-1 and hence allowing the synthesis of proteins required for DNA synthesis. Important components of the decision are a raised concentration of cyclin D as a result of MAP kinase activity and a low concentration of CKIs. Cell–cell contact upregulates CKIs so that when an organ has filled the space available it stops growing.\\n---\\n# CELL CYCLE AND CONTROL OF CELL NUMBER\\n\\n10. p53 is continually produced in cells but is as quickly destroyed. An increase in the concentration of p53 follows DNA damage or other cell stress and has three main effects: (a) activation of DNA repair mechanisms, (b) synthesis of CKIs, preventing cell division, and (c) activation of apoptosis.\\n\\n11. In contrast to necrosis, which causes inflammation, apoptosis is a regulated mechanism of cell suicide that has little effect on the surrounding tissue. The final effectors of apoptosis are a family of proteases called caspases.\\n\\n12. Apoptosis can be triggered in three ways: (a) binding of ligand to death domain receptors, (b) denial of growth factors, and (c) cell stress.\\n\\n# FURTHER READING\\n\\nHyams, J. S. 2002. The cell cycle and mitosis. Biol. Sci. Rev. 14: 37–41.\\nJacobson, M. D., Weil, M., and Raff, M. C. 1997. Programmed cell deap in animal development. Cell 88: 347–354.\\nMichael D., Jacobson, M. D., and McCarpy, N. (eds.) 2002. Apoptosis: The Molecular Biology of Programmed Cell Deap. Oxford: Oxford University Press.\\nMitchison T. J., and Salmon E. D. 2001. Mitosis: A history of division. Nature Cell Biol. 3: E17–E21.\\nNicklas R. B. 1997. How cells get pe right chromosomes. Science 275: 632–637.\\nSharp, D. J., Rogers, G. C., and Scholey, J. M. 2000. Microtubule motors in mitosis. Nature 407: 41–47.\\nVaux D. L., and Korsmeyer, S. J. 1999. Cell deap in development. Cell Jan 96: 245–254.\\n\\n# REVIEW QUESTIONS\\n\\nFor each question, choose the ONE BEST answer or completion.\\n\\n1. 1. Mitosis proceeds in the order\\n- A. cytokinesis, prophase, prometaphase, telophase, metaphase, and anaphase.\\n- B. telophase, anaphase, prophase, prometaphase, metaphase, and cytokinesis.\\n- C. prophase, prometaphase, metaphase, anaphase, telophase, and cytokinesis.\\n- D. prophase, prometaphase, metaphase, cytokinesis, anaphase, and telophase.\\n- E. prophase, prometaphase, cytokinesis, metaphase, anaphase, and telophase.\\n2. 2. Paired chromatids separate and begin to move toward the spindle poles in mitotic\\n- A. prophase.\\n- B. prometaphase.\\n- C. metaphase.\\n- D. anaphase.\\n- E. telophase.\\n3. 3. Meiosis differs from mitosis in that\\n- A. the centrioles do not separate during the first meiotic division.\\n- B. meiotic divisions are always asymmetric.\\n---\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n|4.|Cdk1 can only be fully active when|\\n|---|---|\\n| |A. it is phosphorylated on threonine 14.|\\n| |B. it is phosphorylated on tyrosine 15.|\\n| |C. it is bound to cyclin A.|\\n| |D. it is dephosphorylated by Cdc25.|\\n| |E. All of the above.|\\n\\n|5.|The main target of cdk4 is|\\n|---|---|\\n| |A. RB.|\\n| |B. E2F-1.|\\n| |C. P53.|\\n| |D. nuclear lamins.|\\n| |E. Wee1.|\\n\\n|6.|Caspases are activated by|\\n|---|---|\\n| |A. phosphorylation.|\\n| |B. dimerization.|\\n| |C. trimerization.|\\n| |D. proteolysis.|\\n| |E. binding to bcl-2 family members.|\\n\\n|7.|Phenylketonuria is said to be a recessive inherited disease because|\\n|---|---|\\n| |A. the fraction of babies with the disease falls in successive generations.|\\n| |B. the disease is not observed in individuals with one mutant gene and one normal one.|\\n| |C. phenylketonuria accumulates at the back of the foot.|\\n| |D. myelin recedes from axons, causing failure of action potential transmission.|\\n| |E. the condition only appears following meiotic recombination.|\\n\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n1. C. Mitosis proceeds in the order prophase, prometaphase, metaphase, anaphase, telophase, and cytokinesis.\\n2. D. Anaphase begins when the protein cohesin, the glue holding the pairs of chromatids together, is degraded, allowing the chromosomes to begin traveling toward the spindle poles.\\n3. C. Like mitosis, meiosis is preceded by one round of DNA synthesis but generates four progeny cells rather than two. Concerning the other answers: (A) The centrioles always separate in cell division, becoming the spindle poles on either side of the nucleus. If you answered answer A you may have been thinking of the fact that in the first meiotic division the centromeres of paired chromatids do not separate, rather, homologous pairs of chromosomes separate. (B) Although meiosis in females is asymmetric, producing one large egg cell and three small polar bodies, in males meiosis is symmetrical, producing four equally sized spermatids. (D) Meiosis often takes much longer than mitosis, but this is because of a lengthy prophase I rather than any change in the duration of anaphase.\\n4. D. For Cdk1 to be fully active, it must be dephosphorylated (not phosphorylated) on threonine 14 and tyrosine 15 by Cdc25 and must be bound to cyclin B (not cyclin A, which does play a role in the cell division cycle but which is not described in this book).\\n---\\n# CELL CYCLE AND CONTROL OF CELL NUMBER\\n\\n|5.|A. Cdk4 phosphorylates RB, which is then unable to sequester E2F-1. None of the others are targets of Cdk4, although nuclear lamin is certainly a target of Cdk1.|\\n|---|---|\\n|6.|D. As generated at the ribosome, caspases are locked into an inactive form by an integral inhibitory domain. Proteolysis removes this inhibitory domain.|\\n|7.|B. Recessive diseases are so called because the defective gene is overruled by the dominant normal gene if the latter is present.|\\n---\\nCASE STUDY: CYSTIC FIBROSIS\\n\\nINTRODUCTION\\n\\nIn the final Chapter of the book we describe the biochemical basis of the disease cystic fibrosis (CF), the search for the CF gene, the prospects for CF gene therapy, and prenatal diagnosis. We have selected this disease as a case study to show how the combined efforts of biochemistry, genetics, molecular and cell biology, and physiology were needed to find the cause of CF and the function of the protein encoded by the CF gene. The basic principles of many of the techniques used in achieving this knowledge are described in Chapters 1–19.\\n\\nCYSTIC FIBROSIS IS A SEVERE GENETIC DISEASE\\n\\nAmong Caucasians (white, non-Jewish) about 1 child in 2500 is born with CF. Inheritance is simple: when both parents are carriers, there is a 1 in 4 chance that a pregnancy will result in a child with the disease. The disease is a distressing one. Most of its symptoms arise from faults in the way the body moves liquids, leading in particular to a build up of inadequately hydrated, sticky mucus in various parts of the body. In the lungs this leads to difficulty in breathing, a persistent cough, and a greatly increased risk of infection. A bacterium known as Pseudomonas aerigunosa thrives in the lung mucus and is particularly recalcitrant to antibiotic treatment. The pancreas—which provides a digestive secretion that flows to the intestine—is also affected and may be badly damaged (which explains\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C 2004 by John Wiley & Sons, Inc.\\n---\\n# CASE STUDY: CYSTIC FIBROSIS\\n\\nThe disease’s full name, cystic fibrosis of the pancreas). Often, there are digestive problems because the damaged pancreas is not producing enzymes. The reproductive system is also harmed and most males who survive to adolescence are infertile. Even though there are so many varied symptoms, the simple Mendelian pattern of inheritance led researchers to believe that the cause was an abnormality or absence of a single protein.\\n\\nUntil recently babies with CF did not survive to their first birthday. Today, the life expectancy is 22 years. This remarkable improvement is due to intensive therapy designed to reverse individual symptoms. Digestive enzymes are taken by mouth to replace those proteins the pancreas fails to produce. Physiotherapy—helping patients to cough up the mucus in their lungs by slapping their backs—reduces the severity of the lung disease. Yet, however remarkable the increase in survival, each life that extends only to early adulthood is still a tragedy.\\n\\n# THE FUNDAMENTAL LESION IN CYSTIC FIBROSIS LIES IN CHLORIDE TRANSPORT\\n\\nAccording to the Almanac of Children’s Songs and Games from Switzerland, “the child will soon die whose brow tastes salty when kissed.” Cystic fibrosis has long been recognized as a fatal disease of children. The link between a salty brow and a disease known for its effects on the pancreas and lungs was not, however, made until 1951. During a heat wave in New York that summer, Paul Di Sant’ Agnese and co-workers noticed that babies with cystic fibrosis were more likely than others to suffer from heat prostration. This prompted them to test their sweat, and they found that it contained much more salt than the sweat of unaffected babies.\\n\\nSweat glands have two regions that perform different jobs (Fig. 20.1). The secretory region deep in the skin produces a fluid that has an ionic composition similar to that of extracellular medium; that is, it is rich in sodium and chloride. If the sweat glands were simply to pour this liquid onto the surface of the skin, they would do a good job of cooling, but the body would lose large amounts of sodium and chloride. A reabsorptive region closer to the surface removes ions from the sweat, leaving the water (plus a small amount of sodium chloride) to flow out of the end of the gland. CF patients produce normal volumes\\n\\n| |watery sweat|salty sweat|\\n|---|---|---|\\n|H2O| | |\\n|Na+| | |\\n|Cl-|reabsorbtive region|secretory region|\\n|Na+| | |\\n|H2O| | |\\n\\nFigure 20.1. Sodium, chloride, and water transport in the sweat gland.\\n---\\nHOMING IN ON THE CF GENE\\n\\n|Input|Common|Input|Common|\\n|---|---|---|---|\\n| |normal sweat gland|sweat gland of CF patient| |\\n\\nFigure 20.2. Measuring the transepithelial voltage of human sweat glands in situ. The voltmeter measures the voltage of the input terminal with respect to the common terminal. In this case a reading of −30 mV indicates that the droplet of sweat has a voltage 30 mV negative to the extracellular medium.\\n\\nof sweat, but this contains lots of sodium and chloride, implying that the secretory region is working fine but that the reabsorptive region has failed. The pathways by which sodium and chloride ions are removed are distinct. Which one has failed in CF? A simple electrical test gave the answer. A normal sweat gland has a small voltage difference across the gland epithelium (Fig. 20.2). In CF patients the inside of the gland is much more negative than usual. This result tells us at once that it is the chloride transport (and not sodium transport) that has failed. In CF sweat glands the reabsorptive region has the transport systems to move sodium, but the chloride ions remain behind in the sweat, giving it a negative voltage. The sodium transport system cannot continue to move sodium ions out of the sweat in the face of this larger electrical force pulling them in, so sodium movement stops too, and sodium chloride is lost in the sweat, which then tastes salty. Every one of the symptoms suffered by CF patients is caused by a failure of chloride transport.\\n\\nHOMING IN ON THE CF GENE\\n\\nThe first family studies were carried out more than 50 years ago, and family pedigrees showing CF as a classic case of recessive inheritance were published in 1946. However, there was no obvious way to identify the millions of people who carry a single copy of the defective CF gene. There were many attempts to find a test for CF. For a time it seemed that a simple stain might do the job; but this was abandoned. Other tests were more eccentric. For example, there was a claim that an extract from the blood of CF patients, and perhaps even from that of their parents, who must be carriers, slowed the beating of the cilia of oysters.\\n---\\nMost other claims that particular gene products were peculiar to CF carriers also failed to stand up or proved to be symptoms rather than cause. For instance, trypsinogen, a precursor of the digestive enzyme trypsin, is found in fluid bathing fetuses homozygous for CF, and not in others—but this proves to be because the pancreas is already dying at this time. In the 1980s there was hope that this result could be used to diagnose affected pregnancies, but the approach was quickly overtaken by developments in studying the gene itself. As for most inborn diseases, the big problem was that nobody had any real idea what the faulty gene did; looking for a defect in an unknown protein is a task that requires hard work and good ideas. After much time-consuming work, in 1985 the research group led by Lap-Chee Tsui published data indicating that the gene for CF lay on chromosome 7. Researchers then used linkage measurements (page 408) to try to pinpoint the region of chromosome 7 containing the CF gene. An international scientific collaboration analyzed DNA from 200 families. The results indicated that the CF gene must lie between two marker regions on chromosome 7, named met and D7S8. The distance between the markers was 2 million base pairs. It was now possible to think about isolating the CF gene.\\n\\nSheer hard work won the day when it came to cloning the CF gene. The scientists knew that the gene was on chromosome 7, somewhere between the met and D7S8 markers. Their strategy was to “walk” along the chromosome (page 152), away from each marker, and hopefully approach the CF gene from either side. In the 1980s, the only genomic vectors available were cosmids (insert size 40,000 bp) and those based on the bacteriophage lambda (insert size 20,000 bp) (page 140). So each genomic clone covered only a tiny part of chromosome 7. Clever tricks had to be devised to speed up the process, and finally a series of cosmid clones, thought to possibly contain regions of the CF gene, were isolated. But how could they tell which clones were the right ones? Since CF affects so many organs, the gene is likely to be a critical one conserved through mammalian evolution. A zoo blot (page 146) was therefore used in the hope of identifying the gene. In turn, each of the human cosmid clones was radiolabeled and used to probe a zoo blot. Researchers found three clones that hybridized to the DNA of all the test mammals. The next step was to check whether these clones contained sequences that code for mRNAs expressed in normal tissues that are affected by CF. Each clone was used to probe a northern blot (page 148) of sweat gland mRNA. One sequence passed the test—with luck, it would contain part of the CF gene. With hindsight we now know that the scientists’ luck had almost run out: The one positive sequence did contain part of the gene, but only a tiny part—113 bp, less than 1% of the total gene. Things moved swiftly from then on. The 113-bp section was used to screen cDNA libraries (page 134) made from sweat gland mRNA. This gave scientists cDNA for the entire CF mRNA, which could then be sequenced. Meanwhile, the cDNA was used to screen a genomic library to find the rest of the CF gene. The gene is 220,000 bp long and has 24 exons.\\n\\nOnce CF cDNA had been sequenced, it could be read, revealing that CF codes for a protein of 1480 amino acids. Although we knew that CF was caused by a defect in chloride\\n\\nOnce CF cDNA had been sequenced, it could be read, revealing that CF codes for a protein of 1480 amino acids. Although we knew that CF was caused by a defect in chloride\\n---\\n# GENE THERAPY FOR CF\\n\\nPhenylalanine at position 508\\n\\nFigure 20.3. The CFTR protein forms a chloride ion channel in the plasma membrane of epithelial cells.\\n\\nion transport, it was not immediately clear that the product of the CF gene was itself a chloride channel. Although hydropathy plots (page 204) indicated that the CF protein was a transmembrane protein, the investigators had to consider the possibility that it was, for example, a cell surface receptor whose activation triggered expression of the proteins mediating chloride transport. The protein was therefore given the catch-all name “cystic fibrosis transmembrane regulator” (CFTR) and its gene was renamed as the CFTR gene. However, for once things were simple: CFTR is indeed itself the chloride ion channel in the plasma membrane (Fig. 20.3). We know this because if purified CFTR is introduced into lipid bilayers, chloride currents can be measured (see In Depth 20.1). In resting cells the channel mouth is blocked by a plug called the regulatory domain. When the regulatory domain is phosphorylated by cAMP-dependent protein kinase (page 353), the plug leaves the channel, allowing chloride ions to flow through.\\n\\nComparing the sequence of the normal CFTR gene to the one in people with CF revealed the mutations that caused the disorder. In 70% of CF patients, three nucleotides are missing, generating a protein that is missing the amino acid phenylalanine at position 508 (Fig. 20.4). The protein is a functional chloride channel, but the mutation affects its ability to fold properly (page 192), and so the mutant protein fails to be inserted into the plasma membrane. In the remaining 30% of patients more than 300 CFTR gene mutations have been documented. These include missense, nonsense, and frameshift mutations (page 80). In some people mutations that cause incorrect exon–intron splicing (page 118) occur. The severity of CF symptoms is dependent on the particular mutation. Some cause a complete failure to express the protein and produce severe symptoms. Other mutations produce a protein that is correctly trafficked to the plasma membrane and that works, but not as well as the normal protein.\\n\\n# GENE THERAPY FOR CF\\n\\nIt is an exciting prospect to think that we might one day treat disease by introducing a normal gene into the cells of patients and correct a defect. However, gene therapy is a\\n---\\n# CASE STUDY: CYSTIC FIBROSIS\\n\\ndifficult procedure, still in its infancy, and one that is not without risk to the patient. At present, permission has only been granted to carry out gene therapy on somatic cells. In this method the defect is corrected in specific cells or tissues rather than in the germ line cells that will give rise to eggs and sperm. Before trying out gene therapy on CF patients, an animal study was conducted. A knock-out mouse (page 157), lacking the CFTR gene, was created (Fig. 20.5). The mouse showed the symptoms of CF. A plasmid containing the CFTR cDNA was used to prepare an aerosol. When the mice inhaled the aerosol droplets, the results were spectacular. The lung symptoms disappeared for many days. The CFTR cDNA had been expressed into protein and the protein targeted to the plasma membrane. Could this approach work in humans?\\n\\n# IN DEPTH 20.1 Lipid Bilayer Voltage Clamp\\n\\nIn 1992 Christine Bear and co-workers published a definitive demonstration that the CFTR is a chloride ion channel. They used a technique called the lipid bilayer voltage clamp, which is illustrated in the accompanying figure. A lipid bilayer is constructed from phospholipid and used to plug a hole between two chambers filled with salt solution. Since a lipid bilayer is a barrier to ion movement, no current is recorded when a voltage is imposed across the bilayer. However, when artificial vesicles containing integral membrane proteins are added to one bath, the vesicles fuse with the artificial membrane so that the proteins now span the barrier bilayer. When Bear and her co-workers did this with purified CFTR, they recorded currents with the direction and amplitude expected of a chloride-selective channel. In particular, no current flowed when the voltage difference across the membrane was equal to the calculated chloride equilibrium voltage (page 312). No other integral membrane proteins were present in this experiment, so the CFTR must itself be a chloride channel.\\n\\n| |ammeter|0|time|\\n|---|---|---|---|\\n|artificial bilayer| |current| |\\n| |-50mV|0mV| |\\n|salt solution containing cAMP dependent protein kinase| |0|time|\\n|artificial vesicles containing purified CFTR| |current| |\\n| |-50mV| | |\\n---\\n# GENE THERAPY FOR CF\\n\\n|sequence of bases in the normal CFTR gene| | | | | |\\n|---|---|---|---|---|---|\\n|A T C|A T C|T T T|G G T|G T T| |\\n|amino acid sequence of CFTR protein| | | | | |\\n|ile|ile|phe|gly|val| |\\n|506|507|508|509|510| |\\n|sequence of bases in mutated CFTR gene| | | | | |\\n|A T C|A T T|G G T|G T T| | |\\n|amino acid sequence of mutant CFTR protein| | | | | |\\n|ile|ile|gly|val| | |\\n|506|507|508|509| | |\\n\\nFigure 20.4. The mutation seen in 70% of CF patients: a deletion of three nucleotides from the CFTR gene has removed a phenlyalanine at position 508.\\n\\nIn 1992 the National Institutes of Health in the United States granted permission for a CFTR gene therapy trial. Because viruses are so good at infecting our cells, they are useful tools for transferring DNA into somatic cells. Viral vectors for gene therapy have been engineered to remove their harmful genes. The gene to be transferred into the patient is inserted into the modified viral genome. A modified adenovirus vector containing the CFTR cDNA was introduced into the lung cells of four patients. Each of the patients made CFTR protein, but only for a short time. Some patients showed adverse reactions to the therapy, probably due to the adenovirus itself. Subsequent trials have delivered an adenovirus, containing the CFTR cDNA, to the nasal epithelium of CF patients. However, again adverse reactions to the treatment developed.\\n\\nOn a more positive note, a method called liposomal-mediated gene transfer shows some promise. This technique has the advantage that it avoids the use of viruses. The plasmid containing the CFTR cDNA is coated with a synthetic lipid bilayer to form a liposome (page 229). The liposome can fuse with the epithelial cell membrane and deliver its DNA contents into the cell. In the case of CF this approach seems to work when the liposomes are delivered to the lungs in the form of an aerosol spray. The CFTR cDNA is expressed and\\n---\\n|lung|enlarged area|\\n|---|---|\\n| |Cl−|\\n|CFTR|chromosomechromosomechromosome|\\n|protein|mRNA|\\n|(a) normal mouse|Cl−|\\n| |(b) CFTR knockout|\\n|droplets|containing|\\n| |Cl−|\\n|CFTR|plasmidplasmidplasmid|\\n|protein|mRNA|\\n| |(c) gene therapy|\\n\\nFigure 20.5. The CFTR knock-out mouse is a model that can be used to test gene therapy strategies.\\n---\\n# DIAGNOSTIC TESTS FOR CF\\n\\nBecause 70% of CF patients carry the same mutation, the deletion of phe at position 508, a diagnostic test was developed. The test, based on PCR (page 150), requires a few cells from the inside of the mouth and two DNA primers that will amplify a stretch of DNA that will include the region encoding amino acid 508 (Fig. 20.6). The PCR products are dotted onto two membrane filters. One membrane is incubated with an oligonucleotide containing the sequence encoding phe508 and the other is incubated with an oligonucleotide lacking the codon for phe508. By comparing the patterns of hybridization, carriers of CF can be detected. Other mutations that cause CF can also be tested for. However, because there are so many CFTR mutations, many families may require a new predictive test to be designed to match their version of the defective gene.\\n\\n# PRENATAL IMPLANTATION DIAGNOSIS FOR CF\\n\\nIn 1992 the first preimplantation diagnosis for CF was carried out. Three sets of parents took part in the trial. Each parent was a carrier for the phe508 mutation. Oocytes were removed from the women\\n\\n| |1|2|3|4|\\n|---|---|---|---|---|\\n|add \\'normal\\' oligonucleotide| | | | |\\n|add \\'mutant\\' oligonucleotide| | | | |\\n| |1|2|3|4|\\n\\nIndividual 1: CF carrier\\n\\nIndividual 2: both copies of gene are normal\\n\\nIndividual 3: both copies of gene are mutant: CF patient\\n\\nIndividual 4: CF carrier\\n\\nFigure 20.6. A diagnostic test for the most common CFTR gene mutation.\\n---\\n# CASE STUDY: CYSTIC FIBROSIS\\n\\nand fertilized with sperm from their partners. Success was reported for only one couple. Six of their oocytes developed into fertilized embryos. A single cell was tested from each embryo, using PCR, for the presence of the defective CFTR gene. Five embryos were characterized. Two embryos each had two copies of the normal CFTR gene, one on each of their chromosome 7’s. Two embryos each had two copies of the phe508 deletion. One embryo tested positive for both the normal gene and the phe508 deletion and was therefore a carrier. One normal/normal embryo and the carrier embryo were implanted into the mother. A baby girl was born. Her DNA showed that both copies of her CFTR gene were normal.\\n\\n# THE FUTURE\\n\\nGene therapy strategies for CF have progressed, but much needs to be improved before this type of treatment can offer real hope for patients. The best gene therapy can offer is an alleviation of some of the symptoms. A cure for CF does not seem a likely prospect. Detection of CF carriers and prenatal diagnosis together with genetic counseling informs parents of the risks and gives them the choice of terminating an affected pregnancy. Preimplantation diagnosis raises ethical questions about selecting a specific embryo. In the future it may be possible to carry out germ line therapy in the egg or sperm of carriers. Most countries regard such interference as unethical.\\n\\n|Medical Relevance|Success and a Setback with SCID|\\n|---|---|\\n|Conditions|Children with severe combined immunodeficiency disease (SCID) show an almost complete failure to fight infection and die soon after birth if not kept in sterile conditions. SCID is more widely known as “bubble boy disease” because confinement to a germ-free plastic bubble was the way in which patients were kept alive for a number of years. One rare form of SCID, caused by a failure to make an enzyme called adenosine deaminase, can be treated by regular injections of recombinant protein (page 149). In 1990 the first trial of a genetic therapy began at the National Institutes of Health in Washington. White blood cells from patients were transfected with a plasmid encoding adenosine deaminase, which inserted into random positions in the genome and began to be transcribed. Although the therapy does seem to have helped the patients, they still need to be periodically injected with recombinant adenosine deaminase. The commonest form of the disease, X-SCID, affects only boys and occurs in about 1 in every 150,000 births. X-SCID is caused by a defective receptor for interleukin 2 (see page 222 for more on interleukin 2). In March 1999 a team at INSERM in Paris initiated a trial in which white blood cells from X-SCID patients were transfected with a viral vector containing the normal, interleukin-2 receptor gene, which inserted randomly into the genome. So far 10 patients have been treated in Paris and one at University College London. Of these, 9 are now essentially cured; they are at home living a normal life and are unlikely to need any further treatment. Tragically, however, one of the children has developed leukemia—one of the transfected cells began dividing out of control. Apparently, the plasmid inserted into the gene for a transcription factor called LMO-2 and the viral promoter caused the transfected cells to make lots of LMO-2 as well as lots of the desired interleukin-2 receptor. Expression of LMO-2 causes white blood cells to divide uncontrollably. At the time of writing, it is uncertain whether this method of treatment for X-SCID will continue. In the long term, this tragic accident will increase the drive toward treating genetic disorders by homologous recombination (page 157) so that only the defective gene is changed.|\\n---\\n# SUMMARY\\n\\n1. Cystic fibrosis is the most common serious single-gene inherited disease in the Western world. Many organs are affected. Sticky mucus builds up in the reproductive tract and in the lungs. The pancreas is always affected and usually fails completely.\\n\\n2. Electrical measurements show that the basic problem is in chloride transport.\\n\\n3. CF is a classic recessive disorder. The gene is on chromosome 7.\\n\\n4. A combination of hard work and novel techniques helped isolate a small part of the CFTR gene. This sequence helped to isolate the normal CFTR cDNA from a sweat gland clone library.\\n\\n5. From the cDNA it was possible to identify the gene and infer its amino acid sequence in both normal and mutated forms. The gene codes for a chloride channel protein called CFTR. Hundreds of different CFTR mutations have now been found.\\n\\n6. For the time being gene therapy only holds out hope of alleviating the lung problems, leaving other less accessible organs untreated. Tests have been devised for prenatal diagnosis and to detect carriers of CF.\\n\\n# FURTHER READING\\n\\nBragonzi, A., and Conese, M. 2002. Non-viral approach toward gene perapy of cystic fibrosis lung disease. Curr. Gene Therapy 2: 295–305.\\nHandyside, A. H., Lesko, J. G., Tarin, J. J., Winston, R. M., and Hughes, M. R. 1992. Birp of a normal girl after in vitro fertilization and preimplantation diagnostic testing for cystic fibrosis. N. Engl. J Med. 327: 905–909.\\nKerem, B., Rommens, J. M., Buchanan, J. A., Markiewicz, D., Cox, T. K., Chakravarti, A., Buchwald, M., and Tsui, L. C. 1989. Identification of pe cystic fibrosis gene: genetic analysis. Science 245: 1073–1080.\\nRich, D. P., Anderson, M. P., Gregory, R. J., Cheng, S. H., Paul, S., Jefferson, D. M., McCann, J. D., Klinger, K. W., Smip, A. E., and Welsh, M. J. 1990. Expression of cystic fibrosis transmembrane conductance regulator corrects defective chloride channel regulation in cystic fibrosis airway epipelial cells. Nature 347: 358–363.\\nRiordan, J. R., Rommens, J. M., Kerem, B., Alon, N., Rozmahel, R., Grzelczak, Z., Zielenski, J., Lok, S., Plavsic, N., Chou, J. L., Drumm, M. L., Iannuzzi, M. C., Collins, F. S., and Tsui, L-C. 1989. Identification of pe cystic fibrosis gene: Cloning and characterization of complementary DNA. Science 245: 1066–1073.\\nWelsh, M. J., and Smip, A. E. 1995. Cystic fibrosis. Sci. Am. 273: 52–59.\\nBear, C. E., Li, C. H., Kartner, N., Bridges, R. J., Jensen, T. J., Ramjeesingh, M. and Riordan, J. R. 1992. Purification and functional reconstitution of pe cystic fibrosis transmembrane conductance regulator (CFTR). Cell 68: 809–818.\\nBenjamini, E., Coico, R. and Sunshine, G. 2000. Immunology, a Short Course, 4p ed. New York: Wiley-Liss, pp. 348–353.\\nCheck, E. 2002. A tragic setback. Nature 420: 116–118.\\n---\\n# CASE STUDY: CYSTIC FIBROSIS\\n\\n# REVIEW QUESTIONS\\n\\n1. A moper and a faper, bop CF carriers, have two children pat do not suffer from CF. The chance of a pird pregnancy producing a child wip pe disease is\\nA. zero.\\nB. 1 in 4.\\nC. 1 in 3.\\nD. 1 in 2.\\nE. very high—pe child will almost certainly have CF.\\n\\n2. Cystic fibrosis results from a defect in a protein pat forms\\nA. a plasma membrane sodium ion channel.\\nB. a plasma membrane potassium ion channel.\\nC. a plasma membrane calcium ion channel.\\nD. a plasma membrane chloride ion channel.\\nE. a plasma membrane phenylalanine carrier.\\n\\n3. A zoo blot helps to detect DNA sequences pat\\nA. are mutating at a fast rate.\\nB. are conserved between species.\\nC. show no similarity between species.\\nD. are lost due to species extinction.\\nE. are processed pseudogenes.\\n\\n4. Somatic gene perapy attempts to correct a gene defect by introducing pe normal gene into\\nA. pe egg.\\nB. pe sperm.\\nC. pe fertilized egg.\\nD. cultured embryonic stem (ES) cells.\\nE. cells in pe patient’s body oper pan eggs or sperm.\\n\\n5. The most common mutation pat causes CF\\nA. is a nonsense mutation at codon 508 pat causes premature chain termination of pe CFTR protein.\\nB. is a frameshift mutation such pat all amino acids following phe at 508 are different from pe normal protein.\\nC. is a deletion of pree nucleotides pat causes phe508 to be absent in pe mutant protein.\\nD. is a nonsense mutation causing failure to express a receptor tyrosine kinase whose activation triggers expression of pe proteins mediating chloride transport.\\nE. only causes disease in boys.\\n\\n6. Cystic fibrosis is said to be “linked” to anoper gene if\\nA. CFTR is part of an operon pat includes pe oper gene.\\nB. CFTR and pe oper gene are located near each oper on pe same chromosome.\\nC. CFTR is translated as a chimera wip pe oper protein.\\nD. bop CFTR and pe protein product of pe oper gene are activated when phosphorylated by cAMP-dependent protein kinase.\\nE. pe function of pe oper gene requires chloride transport.\\n---\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n|7.|CFTR and the voltage-gated calcium channel are similar in that|\\n|---|---|\\n| |A. both pass calcium ions.|\\n| |B. both are opened by depolarization.|\\n| |C. both are caused to open by an increase in the concentration of an intracellular messenger.|\\n| |D. both are closed when a cytosolic plug enters the channel lumen and blocks it.|\\n| |E. both play a critical role in brain cells.|\\n\\n# ANSWERS TO REVIEW QUESTIONS\\n\\n|1.|B. Each pregnancy has a 1-in-4 chance of producing a child with CF. The presence or absence of|\\n|---|---|\\n| |the disease in earlier children is irrelevant.|\\n|2.|D. CF is characterized by a defect in chloride ion transport.|\\n|3.|B. Zoo blots detect protein-coding sequences that have been conserved through evolution and are|\\n| |therefore present in many species.|\\n|4.|E. In somatic gene therapy the normal gene is introduced into a somatic cell where expression of|\\n| |the new gene will alleviate or cure the disease. This is any cell other than the gametes, the egg,|\\n| |and sperm.|\\n|5.|C. Seventy percent of individuals with CF lack three nucleotides in their CFTR gene. This causes|\\n| |the phe at 508 in the normal protein to be absent from the mutant protein.|\\n|6.|B. Concerning the other answers: (A) This would certainly produce genetic linkage, but operons|\\n| |are not found in eukaryotes. (C) Chimeras, which are characterized by the physical linkage on|\\n| |the same polypeptide chain of two proteins that are normally independent, can be produced by|\\n| |naturally ocurring mutations, but this is not what we mean by linkage. Answers D and E are not|\\n| |relevant to the concept of linkage.|\\n|7.|D. The voltage-gated calcium channel is inactivated when a cytosolic plug enters the channel|\\n| |lumen and blocks it, while the CFTR is opened when phosphorylation of its cytosolic plug causes|\\n| |the plug to vacate the channel lumen. Of the other answers: (A) CFTR does not pass calcium ions.|\\n| |(B) CFTR is not opened by depolarization. (C) This answer is not completely wrong because while|\\n| |the CFTR opens when phosphorylated by cAMP-dependent protein kinase, phosphorylation by|\\n| |cAMP-dependent protein kinase does also make it easier for depolarization to open the voltage-|\\n| |gated calcium channel. However, phosphorylation alone cannot open the voltage-gated calcium|\\n| |channel, so this answer is not as good as answer D. (E) As far as we know, CFTR plays no role in|\\n| |the brain. CF patients are cognitively normal.|\\n---\\n|Name|Page for Details|Location|Selective for|Opened by|Comments|\\n|---|---|---|---|---|---|\\n|Gap junction channel|55|Plasma membrane of many cells|Any solute of Mr < 1000|Contact with second gap junction channel|Allows solutes to pass from cytosol to cytosol|\\n|Porin|262|Outer mitochondrial membrane|Any solute of Mr < 10,000|Probably spends a large fraction of time open under most circumstances|Stress can cause switch to a larger form that allows cytochrome c to pass|\\n|Potassium channels| | | | | |\\n|Potassium channel|310|Plasma membrane of all cells|Potassium ions|Some isoforms are opened by depolarization; some are open all the time|Responsible for the resting voltage|\\n---\\n# APPENDIX: CHANNELS AND CARRIERS\\n\\n|Name|Details|Location|Selective for|Opened by|Comments|\\n|---|---|---|---|---|---|\\n|Calcium channels|Voltage-gated|327|Plasma membrane of sea urchin egg, axon terminal, muscle, and other cells|Calcium ions|Depolarization of plasma membrane|The inward movement of calcium ions changes the transmembrane voltage and raises the calcium concentration in the cytosol|\\n|Inositol trisphosphate-gated calcium channel|344|Membrane of endoplasmic reticulum|Calcium ions|Binding of cytosolic inositol trisphosphate|Part of the system that allows an extracellular solute to raise cytosolic calcium|\\n|Ryanodine receptor; skeletal muscle isoform|348|Membrane of endoplasmic reticulum|Calcium ions|Direct physical link to voltage-gated calcium channels in the plasma membrane|Allows rapid increase of cytosolic calcium concentration in response to depolarization of plasma membrane|\\n|Ryanodine receptor; standard isoforms|348|Membrane of endoplasmic reticulum|Calcium ions|Binding of cytosolic calcium|Allows a small increase of cytosolic calcium concentration to trigger a larger increase|\\n|Mitochondrial calcium channel|305|Inner mitochondrial membrane|Calcium ions|Not easily studied, probably always open|Allows an increase of cytosolic calcium concentration to activate mitochondria|\\n|Sodium and unselective cation channels|Voltage-gated|330|Plasma membrane of nerve and muscle cells|Sodium ions|Depolarization of plasma membrane|Produces brief action potentials|\\n---\\n# APPENDIX: CHANNELS AND CARRIERS\\n\\n|Name|Details|Location|Selective for|Opened by|Comments|\\n|---|---|---|---|---|---|\\n|Nicotinic acetylcholine receptor channel|364|Plasma membrane of skeletal muscle cells and some nerve cells|Sodium and potassium|Extracellular acetylcholine|An ionotropic cell surface receptor that causes the plasma membrane to depolarize|\\n|Glutamate receptor channel|372|Plasma membrane of some nerve cells|Sodium, potassium, some isoforms also pass calcium|Extracellular glutamate|An ionotropic cell surface receptor that causes the plasma membrane to depolarize|\\n|cAMP-gated channel|350|Plasma membranes of scent-sensitive nerve cells|Sodium and potassium|cAMP in the cytosol|Depolarizes the plasma membrane so that the cell produces action potentials|\\n|cGMP-gated channel|353|Plasma membranes of photoreceptor cells|Sodium and potassium|cGMP in the cytosol|Depolarizes the plasma membrane|\\n|Chloride channels| |Plasma membranes of various cells| | | |\\n|GABA receptor|373|Plasma membrane of some nerve cells|Chloride ions|Extracellular GABA|When this channel is open, it is more difficult to depolarize the cell to threshold|\\n|CFTR|426|Plasma membrane of sweat gland cells, airway epithelial cells, and of many other cells|Chloride ions|Phosphorylation by protein kinase A|Required for several transport processes in glands|\\n---\\n|Name|Details|Location|Mode of Action|Comments|\\n|---|---|---|---|---|\\n|Glucose carrier|316|Plasma membrane of all cells|Required by all human cells|Glucose|\\n|ADP/ATP exchanger|271|Inner mitochondrial membrane|Gets ATP and ADP across inner mitochondrial membrane| |\\n|Na + /Ca2+ exchanger|317|Plasma membrane of many cells|Pushes calcium ions out of the cytosol| |\\n|β-Galactoside permease|111|Bacterial plasma membrane|Brings lactose into the cell. One of the products of the lac operon|Lactose|\\n|ATP synthase|272|Inner mitochondrial membrane|Converts between energy in the H+ gradient and energy as ATP| |\\n|Ca2+ ATPase|318|Plasma membrane of many cells|Pushes calcium ions out of the cytosol. All cells have either this or the Na + /Ca2+ exchanger; many have both| |\\n|Na + /K + ATPase|270|Plasma membrane of all cells|Converts between energy as ATP and energy in the Na + gradient| |\\n|Electron transport chain|265|Inner mitochondrial membrane|Converts between energy as NADH and energy in the H+ gradient| |\\n---\\n# GLOSSARY\\n\\n|14-3-3|protein that binds phosphorylated BAD, hence preventing it from triggering apoptosis.|\\n|---|---|\\n|7-methyl guanosine cap|modified guanosine found at the 5\\' terminus of eukaryotic mRNA. A guanosine is attached to the mRNA by a 5\\' - 5\\'-phosphodiester link and is subsequently methylated on atom number 7 of the guanine.|\\n|9 + 2 axoneme|structure of cilium or flagellum; describes the arrangement of nine peripheral microtubules surrounding two central microtubules.|\\n|A (adenine)|one of the bases present in DNA and RNA; adenine is a purine.|\\n|α-amino acid|amino acid in which the carboxyl and amino groups are attached to the same carbon.|\\n|α adrenergic receptor|a receptor for the related chemicals adrenaline and noradrenaline. The α adrenergic receptor activates Gq and therefore generates a calcium signal. To a first approximation, α receptors respond to noradrenaline, while adrenaline acts mainly on β adrenergic receptors.|\\n|α helix|a common secondary structure in proteins, in which the polypeptide chain is coiled, each turn of the helix taking 3.6 amino acid residues. The nitrogen atom in each peptide bond forms a hydrogen bond with the oxygen four residues ahead of it in the polypeptide chain.|\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C2004 by John Wiley & Sons, Inc.\\n\\n441\\n---\\n# GLOSSARY\\n\\n|A site|(aminoacyl site)|site on a ribosome occupied by an incoming tRNA and its linked amino acid.|\\n|---|---|---|\\n|acceptor|in a hydrogen bond|the atom (oxygen, nitrogen, or sulfur) that is not covalently bonded to the hydrogen but that nevertheless accepts a small share of the electrons.|\\n|acetone|CH3—CO—CH3|A chemical with a fruity smell produced from acetoacetate during the ketosis that can occur in diabetes.|\\n|acetylcholine|CH3COCH2CH2NCH3|A transmitter released by various nerve cells including motoneurones and autonomic vasodilator neurones.|\\n|acetylcholine receptor|integral membrane protein that binds acetylcholine.|There are two types: the nicotinic acetylcholine receptor is ionotropic while the muscarinic acetylcholine receptor is metabotropic, linked via Gq to phospholipase Cβ.|\\n|acetylcholinesterase|enzyme that hydrolyzes acetylcholine|terminating its action at synapses.|\\n|acid|molecule that readily gives H+|to water. Most organic acids are compounds containing the group —COOH, although —SH is also weakly acidic.|\\n|acrosome|actin-rich structure that helps the sperm to penetrate the egg at fertilization.| |\\n|ActA|protein on the surface of the bacterium Listeria|from which actin filaments polymerize.|\\n|actin-binding proteins|proteins that bind to and modulate the function of G-actin or F-actin.| |\\n|actin-related protein (Arp)|an actin nucleation protein.|New actin filaments can grow out from an Arp base.|\\n|actin-severing protein|one of a group of enzymes that cuts actin microfilaments.|Gelsolin is an example.|\\n|action potential|explosive depolarization of the plasma membrane.| |\\n|activation energy|height of the energy barrier between the reactants and the products of a chemical reaction.| |\\n|acyl group|group having the general formula CnHmOC|Acyl groups are formed when fatty acids are attached to other compounds by ester bonds.|\\n|adenine (A)|one of the bases present in DNA and RNA; adenine is a purine.| |\\n---\\n# GLOSSARY\\n\\n|adenosine|adenine linked to the sugar ribose. Adenosine is a nucleoside.|\\n|---|---|\\n|adenosine diphosphate (ADP)|adenosine with two phosphates attached to the 5\\' carbon of ribose.|\\n|adenosine monophosphate (AMP)|adenosine with one phosphate attached to the 5\\' carbon of ribose.|\\n|adenosine triphosphate (ATP)|adenosine with three phosphates attached to the 5\\' carbon of ribose. ATP is a coenzyme and one of the cell’s energy currencies.|\\n|adenylate cyclase|enzyme that converts ATP to the intracellular messenger cyclic AMP (cAMP). Also called adenyl cyclase.|\\n|adenyl cyclase|enzyme that converts ATP to the intracellular messenger cyclic AMP (cAMP). Also called adenylate cyclase.|\\n|adherens junctions|type of anchoring junction in which the cell adhesion molecules are linked to actin microfilaments.|\\n|adipocytes|cells that store fats (triacylglycerols).|\\n|adipose tissue|type of fatty connective tissue.|\\n|ADP|adenosine diphosphate; adenosine with two phosphates attached to the 5\\' carbon of ribose.|\\n|ADP/ATP exchanger|carrier in the inner mitochondrial membrane. ADP is moved in one direction and ATP in the other.|\\n|adrenaline|hormone released into the blood when an individual is under stress. Adrenaline acts at β-adrenergic receptors to activate Gs and hence adenylate cyclase.|\\n|adrenergic receptor|receptor for the related chemicals adrenaline and noradrenaline. There are two isoforms, α and β. To a first approximation, noradrenaline acts mainly on α receptors linked to Gq and therefore generates a calcium signal, while adrenaline acts mainly on β receptors linked to Gs and therefore generates a cAMP signal.|\\n|α helix|common secondary structure in proteins in which the polypeptide chain is coiled, each turn of the helix taking 3.6 amino acid residues. The nitrogen atom in each peptide bond forms a hydrogen bond with the oxygen four residues ahead of it in the polypeptide chain.|\\n|Akt|earlier name for protein kinase B, a protein kinase that is activated when it is itself phosphorylated; this in turn only occurs when Akt is recruited to the plasma membrane by phosphatidylinositol trisphosphate (PIP3). Akt phosphorylates proteins on serine and threonine residues (e.g., the bcl-2 family protein BAD).|\\n|alkali|strong base (that will take H+ from water) such as sodium hydroxide or potassium hydroxide.|\\n|alkaline|of a solution: one with a low concentration of H+ (really H3O+) so that the pH is greater than 7.0.|\\n|allolactose|disaccharide sugar. Lactose is converted to allolactose by the enzyme β-galactosidase. Allolactose is an inducer of lac operon transcription.|\\n---\\n# GLOSSARY\\n\\nall or nothing: phenomenon that, once initiated, proceeds to completion even if the stimulus is removed. A process that involves positive feedback, such as an action potential, will tend to be all or nothing.\\n\\nallosteric; allostery: when the binding of a ligand to one site on a protein affects the binding at another site. These interactions can be between sites for the same ligand or different ligands. Molecules that are allosteric almost always have a quaternary structure.\\n\\nalternative splicing: phenomenon in which a single eukaryotic primary mRNA transcript can be processed to yield a number of different processed mRNAs and can therefore generate a number of different proteins.\\n\\nα-amino acid: amino acid in which the carboxyl and amino groups are attached to the same carbon.\\n\\namino acid: chemical that has both a —COOH group and an —NH2 group. In an α-amino acid, both the —COOH and —NH2 groups are attached to the same carbon atom. All proteins are generated using the genetically encoded palette of 19 α-amino acids plus proline.\\n\\naminoacyl site (A site): site on a ribosome occupied by an incoming tRNA and its linked amino acid.\\n\\naminoacyl tRNA: tRNA attached to an amino acid via an ester bond.\\n\\naminoacyl tRNA synthases: family of enzymes, each of which attaches an amino acid to the appropriate tRNA.\\n\\nγ-amino butyric acid: a γ -amino acid that acts to open chloride channels in the plasma membrane of sensitive nerve cells. Usually called γ -amino butyric acid rather than γ -amino butyrate, even though the latter (−OOC—CH2—CH2—CH2—NH3+) is the form in which it is found at neutral pH.\\n\\namino terminal: end of a peptide or polypeptide that has a free α-amino group.\\n\\namorphous: without form. Not a specifically scientific word.\\n\\nAMP: adenosine monophosphate; adenosine with one phosphate attached to the 5′ carbon of ribose.\\n\\namphipathic: “hating both”—a molecule with a hydrophobic region and a hydrophilic region is said to be amphipathic.\\n\\nanabolism: those metabolic reactions that build up molecules: biosynthesis.\\n\\nanaerobic: without air. Obligate anaerobes are poisoned by oxygen and therefore can only function anaerobically. Other cells, such as yeast and skeletal muscle, can switch to using anaerobic respiration when they are denied oxygen.\\n\\nanaerobic respiration: partial breakdown of sugars and other cellular fuels that can be accomplished in the absence of oxygen.\\n\\nanaphase: period of mitosis or meiosis during which sister chromatids or homologous chromosome pairs separate; consists of anaphase A and anaphase B.\\n\\nanaphase A: part of anaphase in which the chromosomes move to the spindle poles.\\n---\\n# GLOSSARY\\n\\n|anaphase B|part of anaphase in which the spindle poles are separated.|\\n|---|---|\\n|anaphase I|anaphase of the first meiotic division (meiosis I).|\\n|anaphase II|anaphase of the second meiotic division (meiosis II).|\\n|anchoring junction|class of cell junction that attaches the cytoskeleton of one cell to the cytoskeleton of its neighbor, forming a physically strong connection. There are two types: desmosomes, which connect to intermediate filaments, and adherens junctions, which connect to actin microfilaments.|\\n|anion|negatively charged ion, e.g., chloride, Cl-, or phosphate, HPO42-.|\\n|anode|positively charged electrode, e.g., in a gel electrophoresis apparatus used for SDS–PAGE.|\\n|antenna chlorophyll|chlorophyll molecules that capture light and pass the energy to reaction center chlorophylls.|\\n|anterograde|forward movement; when applied to axonal transport it means away from the cell body.|\\n|anthrax|lethal bacterial disease. When food supplies are exhausted, anthrax bacteria produce spores that can survive for long periods in the soil or in (illegal) stocks of biological weapons.|\\n|antibiotic|chemical that is produced by one type of organism and kills others—often by inhibiting protein synthesis. The most useful antibiotics to humans are those that are selective for prokaryotes.|\\n|antibody|protein formed by the immune system that binds to and helps eliminate another chemical. Antibodies can be extremely selective for their ligand and are useful in many aspects of cell biology such as immunofluorescence microscopy and western blotting.|\\n|anticodon|three bases on a tRNA molecule that hydrogen-bond to the codon on an mRNA molecule.|\\n|antiparallel β sheet|β sheet in which alternate parallel polypeptide chains run in opposite directions.|\\n|AP endonuclease|DNA repair enzyme that cleaves the phosphodiester links on either side of a depurinated or depyrimidinated sugar residue. AP stands for apurinic/apyrimidinic.|\\n|apoptosis|process in which a cell actively promotes its own destruction, as distinct from necrosis. Apoptosis is important in vertebrate development, where tissues and organs are shaped by the death of certain cell lineages.|\\n|apoptosis inhibitor protein|proteins that block the action of caspases and hence help prevent apoptosis.|\\n|aporepressor|protein that binds to an operator region and represses transcription only when it is complexed with another molecule.|\\n|aqueous|watery.|\\n|ARF|GTPase that plays a critical role in the formation of coated vesicles.|\\n---\\n# GLOSSARY\\n\\n|Arp (actin-related protein)|actin nucleation protein. New actin filaments can grow out from an Arp base.|\\n|---|---|\\n|A site (aminoacyl site)|site on a ribosome occupied by an incoming tRNA and its linked amino acid.|\\n|assay|term for a chemical measurement, e.g., one in which the activity of an enzyme reaction is measured.|\\n\\nATP: adenosine triphosphate; adenosine with three phosphates attached to the 5\\' carbon of ribose. ATP is a coenzyme and one of the cell’s energy currencies.\\n\\nATP/ADP exchanger: carrier in the inner mitochondrial membrane. ADP is moved in one direction and ATP in the other.\\n\\nATP synthase: carrier of the inner mitochondrial membrane that is built around a rotary motor. Ten H+ enter the mitochondrial matrix for every three ATP made.\\n\\nATP synthetase: another name for ATP synthase; a carrier of the inner mitochondrial membrane that is built around a rotary motor. Ten H+ enter the mitochondrial matrix for every three ATP made.\\n\\nautoinduction: process of self-induction that occurs when a product of a reaction stimulates the production of more of itself. An example is the activation of the lux operon by the small molecule N-acyl-HSL; transcription of the lux operon then causes the production of more N-acyl-HSL.\\n\\nautonomic nerves: nerves that control processes of which we are usually not consciously aware, such as blood vessel size.\\n\\nautoradiography: process that detects a radioactive molecule. For example, in a Southern blot experiment, the membrane that has been hybridized to a radioactive gene probe is placed in direct contact with a sheet of X-ray film. Radioactive decay activates the silver grains on the emulsion of the X-ray film. When the film is developed, areas that have been in contact with radioactivity will show as black.\\n\\naxon: long process of a nerve cell, specialized for the rapid conduction of action potentials.\\n\\naxonal transport: movement of material along microtubules within a nerve cell process; can be outward (anterograde) or inward (retrograde).\\n\\nβ adrenergic receptor: a receptor for the related chemicals adrenaline and noradrenaline. The β adrenergic receptor activates Gs and therefore generates a cAMP signal. To a first approximation, β receptors respond to adrenaline, while noradrenaline acts mainly on α adrenergic receptors.\\n\\nβ oxidation: process by which fatty acids are broken down into individual two-carbon units coupled to CoA to form acetyl-CoA. The process, which takes place in the mitochondrial matrix, generates both NADH and FADH2.\\n\\nβ sheet: common secondary structure in proteins in which lengths of fully extended polypeptide run alongside each other, hydrogen bonds forming between the peptide bonds of the adjoining strands.\\n\\nB-DNA: right-handed DNA double helix.\\n---\\n# GLOSSARY\\n\\nBAC (bacterial artificial chromosome): cloning vector used to propagate DNAs of about 300,000 bp in bacterial cells.\\n\\nbacteriophage: (sometimes shortened to phage) a virus that infects bacterial cells.\\n\\nBAD: a bcl-2 family protein that causes the release of cytochrome c from mitochondria, triggering apoptosis. BAD is phosphorylated and thereby inactivated by protein kinase B.\\n\\nbasal body: structure from which cilia and flagella arise; has the same structure as the centriole.\\n\\nbase: there are two meanings in cell biology: (1) A chemical that will accept an H+. Many organic bases contain the group —NH2, which accepts an H+ to become —NH3+. (2) One of a group of ring-containing nitrogenous compounds that combine with a sugar to create a nucleoside; the members include adenine, guanine, hypoxanthine, cytosine, thymine, uracil, and nicotinamide.\\n\\nbase excision repair: process that repairs DNA double helices that have lost a purine (depurination) or in which a cytosine has been deaminated to uracil (U). The entire damaged monomer is removed and a correct deoxyribonucleotide inserted in its place.\\n\\nbasement membrane: thin layer of extracellular fibers that supports epithelial cells.\\n\\nbase pair: The Watson–Crick model of DNA showed that guanine in one DNA strand would fit nicely with cytosine in another strand while adenine would fit nicely with thymine. The two hydrogen-bonded bases are called a base pair. RNA can also participate in base pairing: instead of thymine it is uracil that now pairs with adenine. The rare base inosine, found in some tRNAs, can base pair with any of uracil, cytosine, or adenine.\\n\\nBAX: bcl-2 family protein that causes the release of cytochrome c from mitochondria, triggering apoptosis.\\n\\nbcl-2: antiapoptotic protein. By binding to BAD and BAX, it prevents them causing release of cytochrome c from mitochondria.\\n\\nbcl-2 family: family of related proteins that regulate cytochrome c release from mitochondria.\\n\\nB-DNA: right-handed DNA double helix.\\n\\nβ-galactosidase: enzyme that cleaves the disaccharide lactose to produce glucose and galactose and that also catalyzes the interconversion of lactose and allolactose. β Galactosidase is a product of the lac operon.\\n\\nbinding site: region of a protein that specifically binds a ligand. A property of the protein’s tertiary structure.\\n\\nbioluminesence: production of light by a living organism.\\n\\nbisphosphate: of a compound, bearing two independent phosphate groups (as opposed to a diphosphate, which bears a chain of two phosphates in a line). Fructose-1,6-bisphosphate is an example.\\n\\nblastocyst: early embryo.\\n---\\n# GLOSSARY\\n\\n|Bloom’s syndrome|disease resulting from a deficiency in helicase. Affected individuals cannot repair their DNA and are susceptible to developing skin cancer and other cancers.|\\n|---|---|\\n|blue-green algae|old name for the photosynthetic prokaryotes now known as cyanobacteria.|\\n|blunt ends|ends of a DNA molecule produced by an enzyme that cuts the two DNA strands at sites directly opposite one another.|\\n|β oxidation|process by which fatty acids are broken down into individual two-carbon units coupled to CoA to form acetyl-CoA. The process, which takes place in the mitochondrial matrix, generates both NADH and FADH2.|\\n|bright-field microscopy|most basic form of light microscopy. The specimen appears against a bright background and appears darker than the background because of the light it has absorbed or scattered.|\\n|brown fat|tissue specialized for generating heat. Thermogenin in the inner mitochondrial membrane allows H+ to enter down its electrochemical gradient so that the rate at which the electron transport chain works is not limited by the availability of ADP.|\\n|β sheet|common secondary structure in proteins in which lengths of fully extended polypeptide run alongside each other, hydrogen bonds forming between the peptide bonds of the adjoining strands.|\\n|bulky lesion|distortion of the DNA helix caused by a thymine dimer.|\\n|C (cytosine)|one of the bases present in DNA and RNA; cytosine is a pyrimidine.|\\n|Ca2+ ATPase|carrier that uses the energy released by ATP hydrolysis to move calcium ions up their concentration gradient out of the cytosol. Different isoforms of calcium ATPase are located at the plasma membrane and in the membrane of the endoplasmic reticulum.|\\n|cadherin|cell adhesion molecule that helps form adherens junctions.|\\n|calcineurin|calcium-calmodulin–activated phosphatase, that is, an enzyme that removes phosphate groups from proteins, opposing the effects of kinases. Calcineurin is inhibited by the immunosuppressant drug cyclosporin.|\\n|calcium action potential|action potential driven by the opening of voltage-gated calcium channels and the resulting calcium influx.|\\n|calcium ATPase|carrier that uses the energy released by ATP hydrolysis to move calcium ions up their concentration gradient out of the cytosol. Different isoforms of calcium ATPase are located at the plasma membrane and in the membrane of the endoplasmic reticulum.|\\n|calcium-binding protein|any protein that binds calcium. Calmodulin, troponin and calreticulin are examples found in the cytosol, attached to actin filaments in striated muscle, and in the endoplasmic reticulum respectively.|\\n|calcium-calmodulin–activated protein kinase|important regulatory enzyme, activated when calcium-loaded calmodulin binds, which phosphorylates target proteins on serine and threonine residues.|\\n---\\n# GLOSSARY\\n\\n|Term|Definition|\\n|---|---|\\n|calcium-induced calcium release|process in which a rise of calcium concentration in the cytoplasm triggers the release of more calcium from the endoplasmic reticulum. The best understood mechanism of calcium-induced calcium release is via ryanodine receptors.|\\n|calcium pump|carrier that moves calcium ions up their electrochemical gradient out of the cytosol into the extracellular medium or into the endoplasmic reticulum. There are two important calcium pumps: the sodium/calcium exchanger is found on the plasma membrane, while different isoforms of the calcium ATPase are found on the plasma membrane and on the membrane of the endoplasmic reticulum.|\\n|calmodulin|calcium-binding protein found in many cells. When calmodulin binds calcium, it can then activate other proteins such as the enzymes calcineurin and glycogen phosphorylase kinase.|\\n|Calvin cycle|series of reactions in the chloroplast that act to fix atmospheric carbon dioxide and build it into larger molecules.|\\n|CaM-kinase|another name for calcium-calmodulin–activated protein kinase; an important regulatory enzyme, activated when calcium-loaded calmodulin binds, that phosphorylates target proteins on serine and threonine residues.|\\n|cAMP (cyclic adenosine monophosphate)|nucleotide produced from ATP by the action of the enzyme adenylate cyclase. cAMP is an intracellular messenger in many cells.|\\n|cAMP-dependent protein kinase (protein kinase A; PKA)|protein kinase that is activated by the intracellular messenger cyclic-AMP. PKA phosphorylates proteins (e.g., glycogen phosphorylase kinase) on serine and threonine residues.|\\n|cAMP-gated channel|channel found in the plasma membrane of scent-sensitive nerve cells. The channel opens when cAMP binds to its cytoplasmic face and allows sodium and potassium ions to pass.|\\n|cAMP phosphodiesterase|enzyme that hydrolyzes cyclic AMP, producing AMP and hence turning off signaling through the cAMP system.|\\n|cap|methylated guanine added to the 5\\' end of a eukaryotic mRNA.|\\n|CAP (catabolite activator protein)|protein found in prokaryotes that binds to cAMP. The CAP–cAMP complex then binds within the promoter region of some bacterial operons and helps RNA polymerase to bind to the promoter.|\\n|carbohydrates|monosaccharides and all compounds made from monosaccharide monomers.|\\n|carbon fixation|process of reducing atmospheric carbon dioxide and building it into larger molecules.|\\n|carboxylation|introduction of a carboxyl group (—COOH).|\\n|carboxyl group|—COOH group. Carboxyl groups give up hydrogen ions to form the deprotonated group —COO−, so molecules that bear carboxyl groups are usually acids.|\\n|carboxyl terminal|end of a peptide or polypeptide that has a free α-carboxyl group. This end is made last at the ribosome.|\\n|cardiac muscle|form of striated muscle that is found in the heart.|\\n---\\n# GLOSSARY\\n\\n|carrier|there are two meanings used in this book: (1) An integral membrane protein that forms a tube through the membrane that is never open all the way through. Solutes can move into the tube through the open end. When the channel changes shape, so that the end that was closed is open, the solute can leave on the other side of the membrane. (2) A person who has one nonfunctional or mutant copy of a gene, but who shows no effects because the other copy produces sufficient functional protein.|\\n|---|---|\\n|caspase|cysteine-containing protease that cleaves at aspartate residues. Caspases are responsible for the degradative processes that occur during apoptosis.|\\n|catabolism|metabolic reactions that break down molecules to derive chemical energy.|\\n|catabolite activator protein (CAP)|protein that binds to cAMP. The CAP–cAMP complex then binds within the promoter region of some bacterial operons and helps RNA polymerase to bind to the promoter.|\\n|catalyst|chemical or substance that reduces the activation energy of a reaction, allowing it to proceed more quickly. Many biological reactions would proceed at an infinitesimal rate without the aid of enzymes, which are protein catalysts.|\\n|catalytic rate constant (kcat)|proportionality constant that relates the maximal initial velocity (Vm) of an enzyme-catalyzed reaction to the enzyme concentration. kcat = Vm / [E]. The units are reciprocal time. See also Turnover number and Maximal velocity.|\\n|cation|positively charged ion, for example, Na+, K+, and Ca2+.|\\n|cdc|cell division cycle. The acronym is usually used to denote genes, especially of the yeasts Saccharomyces cerevisiae and Schizosaccharomyces pombe, which, when mutated, cause the cell division cycle to be abnormal.|\\n|Cdc25|protein phosphatase involved in the regulation of CDK1.|\\n|Cdk1 (cyclin-dependent kinase 1)|protein kinase involved in the regulation of the G2/M transition of the cell cycle. Associates with cyclin B to form MPF.|\\n|Cdk2 (cyclin-dependent kinase 2)|protein kinase involved in the regulation of the G1 phase of the cell cycle. Associates with cyclin E.|\\n|Cdk4 (cyclin-dependent kinase 2)|protein kinase involved in the regulation of the G1 phase of the cell cycle. Associates with cyclin D.|\\n|Cdk6 (cyclin-dependent kinase 2)|protein kinase involved in the regulation of the G1 phase of the cell cycle. Associates with cyclin D.|\\n|cDNA (or complementary DNA)|DNA copy of an mRNA molecule.|\\n|cDNA library|collection of bacterial cells each of which contains a different foreign cDNA molecule.|\\n|cell|fundamental unit of life. A membrane-bound collection of protein, nucleic acid, and other components that is capable of self-replication using simpler building blocks.|\\n|cell adhesion molecule|integral membrane protein responsible for sticking cells together. The extracellular domain binds a cell adhesion molecule on another cell while the intracellular domain binds to the cytoskeleton, either directly or via a linker protein.|\\n---\\n# GLOSSARY\\n\\n|cell center|point immediately adjacent to the nucleus of eukaryotes where the centrosome and Golgi apparatus are located.|\\n|---|---|\\n|cell cycle (cell division cycle)|ordered sequence of events that must occur for successful cell division; consists of G1, S, G2, and M phases.|\\n|cell division cycle|another name for the cell cycle: the ordered sequence of events that must occur for successful cell division; consists of G1, S, G2, and M phases.|\\n|cell division cycle (CDC) mutants|mutants, especially of the yeasts Saccharomyces cerevisiae and Schizosaccharomyces pombe, that are temperature-sensitive for cell division.|\\n|cell junctions|points of cell–cell interaction in tissues; includes tight junctions; anchoring junctions and gap junctions.|\\n|cell membrane|membrane that surrounds the cell; also known as the plasmalemma or plasma membrane.|\\n|cell surface membrane|another name for the plasma membrane.|\\n|cellulose|major structural polysaccharide of the plant cell wall; a β1 → 4 polymer of glucose.|\\n|cellulose synthase|multiprotein complex of the plant plasma membrane that makes cellulose.|\\n|cell wall|rigid case that encloses plant and fungal cells and many prokaryotes. The cell wall lies outside the plasma membrane. Plant cell walls are composed of cellulose plus other polysaccharide molecules such as hemicellulose and pectin.|\\n|central dogma (of molecular biology)|“DNA makes RNA makes protein”—the concept that the sequence of bases on DNA defines the sequence of bases on RNA, and the sequence of bases on RNA then defines the sequence of amino acids on protein.|\\n|centriole|structure found at the centrosome (= microtubule organizing center) of animal cells; composed of microtubules.|\\n|centromere|region of the chromosome at which the kinetochore (where the microtubules of the mitotic or meiotic spindle attach) is formed.|\\n|centrosome (microtubule organizing center)|structure from which cytoplasmic microtubules arise.|\\n|CF (cystic fibrosis)|inherited disease characterized by failure of the pancreas and by thick sticky mucus in the lungs leading to fatal lung infection unless treated. Cystic fibrosis is caused by failure to make, or properly target, plasma membrane chloride channels.|\\n|cGMP-dependent protein kinase (PKG; protein kinase G)|protein kinase that is activated by the intracellular messenger cyclic GMP. PKG phosphorylates proteins (e.g., the calcium ATPase) on serine and threonine residues.|\\n|chain release factors|proteins that occupy the A site in the ribosome when a stop codon is encountered and that act to trigger termination of polypeptide synthesis.|\\n|channel|integral membrane protein that forms a continuous water-filled hole through the membrane.|\\n---\\n# GLOSSARY\\n\\n|chaotropic reagents|reagents such as urea, which cause proteins to lose all their higher levels of structure and adopt random, changing conformations.|\\n|---|---|\\n|chaperone|protein that helps other proteins to remain unfolded for correct protein targeting, or to fold into their correct three-dimensional structure.|\\n|charge|(1) excess or deficit of electrons giving a negative or positive charge, respectively; (2) transfer RNA is said to be charged when it has an amino acid attached.|\\n|charged tRNA|tRNA attached to an amino acid.|\\n|chiasmata (singular chiasma)|structures formed during crossing over between the chromatids of homologous chromosomes during meiosis; the physical manifestation of genetic recombination.|\\n|chimera|structure formed from two different parts. Chimeric proteins are generated by joining together all or part of the protein coding sections of two distinct genes. Chimeric organisms are formed by mixing two or more distinct clones of cells.|\\n|chimeric protein|proteins generated by joining together all or part of the protein coding sections of two distinct genes, for example, GFP and a protein of interest.|\\n|chiral|structure whose mirror image cannot be superimposed on it. Organic molecules will be chiral if a carbon atom has four different groups attached to it and is therefore asymmetric.|\\n|chlorophyll|major photosynthetic pigment of plants and algae.|\\n|chloroplast|photosynthetic organelle of plant cells.|\\n|chromatid|complete DNA double helix plus accessory proteins subsequent to DNA replication in eukaryotes. At mitosis, the chromosome is seen to be composed of two chromatids; these then separate to form the chromosomes of the two progeny cells.|\\n|chromatin|complex of DNA and certain DNA-binding proteins such as histones.|\\n|chromatophore|pigment cell in the skin of fish and amphibia.|\\n|chromosome|single, enormously long molecule of DNA, together with its accessory proteins. Chromosomes are the units of organization of the nuclear chromatin and carry many genes. In eukaryotes chromosomes are linear; in prokaryotes they are circular.|\\n|chromosome walking|investigating a chromosome bit by overlapping bit, each bit being used to clone the next.|\\n|cilium (plural cilia)|locomotory appendage of some epithelial cells and protozoa.|\\n|CIP1|gene encoding a cyclin-dependent kinase inhibitor protein that prevents cells entering the S phase of the cell cycle. p21 CIP1 is upregulated by cell–cell contact.|\\n|cis|side to which material is added. Of the Golgi complex, the surface that receives vesicles from the endoplasmic reticulum.|\\n|cisternae|flattened membrane-bound sacs, e.g., those that make up the Golgi apparatus.|\\n|CKI (cyclin-dependent kinase inhibitor)|type of cell cycle regulatory protein. Binds to and inactivates CDKs.|\\n---\\n# GLOSSARY\\n\\n|clathrin|protein that functions to cause vesicle budding in response to binding of specific ligand.|\\n|---|---|\\n|clathrin adaptor protein|protein that binds to specific transmembrane receptors and which in turn recruits clathrin to form a coated vesicle. The vesicle therefore contains the molecule for which the receptor is specific.|\\n|cleavage furrow|in animal cells, the structure that constricts the middle of the cell during cytokinesis.|\\n|clone|a number of genetically identical individuals.|\\n|clone library|collection of bacterial clones where each clone contains a different foreign DNA molecule.|\\n|cloning|strictly, the creation of a number of genetically identical organisms. In molecular genetics, the term is used to mean the multiplication of particular sequences of DNA by an asexual process such as bacterial cell division.|\\n|cloning vector|DNA molecule that carries genes of interest, can be inserted into cells, and which will then be replicated inside the cells. Cloning vectors range in size from plasmids to entire artificial chromosomes.|\\n|closed promoter complex|structure formed when RNA polymerase binds to a promoter sequence prior to the start of transcription.|\\n|coatamer|protein complex that encapsulates one class of coated vesicle. Formation of the coatamer coat on a previously flat membrane forces the membrane into a curved shape and therefore drives vesicle formation.|\\n|coated vesicle|cytoplasmic vesicle encapsulated by a protein coat. There are two types of coated vesicles, coated by coatamer and clathrin, respectively.|\\n|codon|sequence of three bases in an mRNA molecule that specifies a particular amino acid.|\\n|coenzyme|molecule that acts as a second substrate for a group of enzymes. ATP/ADP, NADH/NAD+, and Acyl-coenzyme A/coenzyme A are all coenzymes.|\\n|cofactor|nonprotein molecule or an ion necessary for the activity of a protein. Cofactors are associated tightly with the protein but can be removed. Examples are pyridoxal phosphate in aminotransferases and zinc in zinc finger proteins.|\\n|cohesin|protein that holds two chromatids together. Degradation of cohesin allows the two chromatids to separate at the start of anaphase in mitosis and in meiosis II.|\\n|colchicine|plant alkaloid from the autumn crocus, Colchicum autumnale; binds to tubulin.|\\n|collagen|major structural protein of the extracellular matrix.|\\n|columnar|taller than it is broad. Used as a description of some types of epithelial cells.|\\n|competent|in molecular genetics this refers to a bacterial culture treated with a solution such as calcium chloride so that uptake of foreign DNA is enhanced.|\\n---\\n# GLOSSARY\\n\\n|complementary|two structures are said to be complementary when they fit into or associate with each other. The two strands of the DNA double helix are complementary, as are the anticodon and codon of transfer and messenger RNA.|\\n|---|---|\\n|complementary DNA (cDNA)|DNA copy of an mRNA molecule.|\\n|complex (as a noun)|association of molecules that is held together by noncovalent interactions and often can be readily dissociated.|\\n|complex I, complex II, complex III, complex IV|large multimolecular complexes that make up the electron transport chain in the inner mitochondrial membrane.|\\n|condenser lens|lens of light and electron microscopes that focuses light (or electrons) onto the specimen.|\\n|connective tissue|tissue that contains relatively few cells within a large volume of extracellular matrix.|\\n|connexon (gap junction channel)|channel in the plasma membrane with a central hole about 1.5 nm in diameter. Gap junction channels only open when they contact a second channel on another cell, in this case they open and form a water-filled tube that runs all the way through the plasma membrane of the first cell, across the small gap between the cells, and through the plasma membrane of the second cell, so allowing passage of solute from the cytosol of one cell to the cytosol of the other.|\\n|constitutive|operating all the time without obvious regulation. Housekeeping genes, expressed all the time, are sometimes called constitutive genes. Proteins that are secreted all the time are said to use the constitutive route.|\\n|constitutively active|mutant or modified protein that is always in the “on” state. A constitutively active enzyme is active in the absence of its normal regulators. A constitutively active GTPase activates its downstream targets in the absence of GTP exchange factors. Paradoxically, the easiest way to generate a constitutively active GTPase is to eliminate its enzymatic activity, so that it does not hydrolyze GTP and therefore remains in the active GTP-bound state.|\\n|constitutive secretion|secretion that continues all the time, without the need for a signal such as an increase of cytosolic calcium concentration.|\\n|contact inhibition|inhibition of cell division by cell–cell contact. Contact inhibition allows cells to proliferate to fill a gap and then stop dividing.|\\n|contigs|overlapping lengths of DNA that can together be used to build up a map of the genome.|\\n|—COOH|carboxyl group. Carboxyl groups give up hydrogen ions to form the deprotonated group —COO−, so molecules that bear carboxyl groups are usually acids.|\\n|corepressor|chemical species that binds to a repressor protein, the complex then binding to an operator region on DNA to prevent transcription. Tryptophan is a corepressor at the trp operon.|\\n|cortex|outer part of any organ or structure. For instance, the tissue outside the vascular tissue of plants, and the tissue that forms the outer region of the brain, are called cortex.|\\n---\\n# GLOSSARY\\n\\n|covalent bond|strong bond between two atoms in which electrons are shared.|\\n|---|---|\\n|cristae|name given to the folds of the inner membrane of mitochondria.|\\n|crossing-over|physical exchange of material that takes place between homologous chromosomes during recombination and is manifest in the formation of chiasmata.|\\n|crosstalk|two messenger systems show crosstalk when one messenger can produce some or all of the effects of the other.|\\n|C-terminal (carboxyl terminal)|end of a peptide or polypeptide that has a free α-carboxyl group. This end is made last on the ribosome.|\\n|cyanobacteria|photosynthetic prokaryotes that were formerly known as blue-green algae.|\\n|cyclic adenosine monophosphate (cAMP)|nucleotide produced from ATP by the action of the enzyme adenylate cyclase. cAMP is an intracellular messenger in many cells and exerts many of its actions by activating protein kinase A.|\\n|cyclic guanosine monophosphate (cGMP)|nucleotide produced from GTP by the action of the enzyme guanylate cyclase. cGMP is an intracellular messenger in many cells and exerts many of its actions by activating protein kinase G.|\\n|cyclin|one of a family of proteins whose level oscillates (cycles) through the cell division cycle. Cyclins associate with and activate cyclin-dependent kinases and hence allow progression through cell cycle control points.|\\n|cyclin B|one of the two proteins that make up maturation promoting factor (MPF), the other being CDK1; one of a family of proteins whose level oscillates (cycles) through the cell division cycle.|\\n|cyclin D|protein that binds to CDK4 and 6 to form a complex that is active at the G1 control point of the cell division cycle; one of a family of proteins whose level oscillates (cycles) through the cell division cycle.|\\n|cyclin-dependent kinase (CDKs)|one of the family of protein kinases that regulate the cell cycle. Cyclin-dependent kinases are only active when bound to one of the family of cyclin proteins. For example, CDK1 associates with cyclin B and regulates the G2/M transition while CDK2 associates with cyclin E and regulates the G1/S transition.|\\n|cyclin E|protein that binds to CDK2 to form a complex that is active at the G1 control point of the cell division cycle; one of a family of proteins whose level oscillates (cycles) through the cell division cycle.|\\n|cystic fibrosis (CF)|inherited disease characterized by failure of the pancreas and by thick sticky mucus in the lungs leading to fatal lung infection unless treated. Cystic fibrosis is caused by failure to make, or properly target, plasma membrane chloride channels.|\\n|cystine|double amino acid formed by two cysteine molecules joined by a disulfide bond.|\\n|cytochemistry|use of chemical compounds to stain specific cell structures and organelles.|\\n|cytochrome|proteins with a heme prosthetic group that are able to transfer electrons. Cytochromes form a critical part of the electron transport chain of mitochondria and also form part of the cytochrome P450 detoxification system in the liver.|\\n---\\n# GLOSSARY\\n\\n|cytochrome c|soluble protein of the mitochondrial intermembrane space, often found loosely associated with the inner mitochondrial membrane. Cytochrome c transports electrons between components of the electron transport chain. If it is allowed to escape from mitochondria cytochrome c activates caspase 9 and hence triggers apoptosis.|\\n|---|---|\\n|cytokinesis|process by which a cell divides in two; part of the M phase of the cell division cycle.|\\n|cytology|study of cell structure by light microscopy.|\\n|cytoplasm|semiviscous ground substance of the cell. All the volume outside the nucleus and inside the plasma membrane is cytoplasm.|\\n|cytoplasmic dynein|motor protein that moves organelles along microtubules in a retrograde direction.|\\n|cytoplasmic streaming|movement of cytoplasm that is commonly seen in plant cells and in amoebae; generated by actin and myosin.|\\n|cytosine|one of the bases present in DNA and RNA; cytosine is a pyrimidine.|\\n|cytoskeleton|cytoplasmic filament system consisting of microtubules, microfilaments, and intermediate filaments.|\\n|cytosol|viscous, aqueous medium in which the organelles and the cytoskeleton are bathed.|\\n|DAG (diacylglycerol)|two acyl groups (fatty acid chains) joined by ester bonds to a glycerol backbone. Diacylglycerol is produced by the action of phospholipase C on phospholipid and helps to activate protein kinase C.|\\n|dark reactions|reactions occurring in plants that use ATP and reducing power provided by NADPH to fix atmospheric carbon dioxide and use it to make sugar.|\\n|deamination|removal of an amino group. Deamination of cytosine to form uracil is a form of DNA damage.|\\n|death domain|domain found on proteins concerned with regulating apoptosis, such as Fas and the p75 neurotrophin receptor. When activated, death domain proteins turn on caspase 8 and hence initiate apoptosis.|\\n|denature|to cause to lose three-dimensional structure by breaking noncovalent intramolecular bonds.|\\n|dendrite|branching cell process. The term is commonly used to name those processes of nerve cells that are too short to be called axons.|\\n|deoxyribonucleic acid (DNA)|polymer of deoxyribonucleotides. DNA specifies the inherited instructions of a cell.|\\n|deoxyribonucleotide|building block of DNA made up of a nitrogenous base and the sugar deoxyribose to which a phosphate group is attached.|\\n|deoxyribose|ribose that instead of —OH has only —H on carbon 2. Deoxyribose is the sugar used in the nucleotides that make up DNA.|\\n|depolarization|any positive shift in the transmembrane voltage, whatever its size or cause.|\\n---\\n# GLOSSARY\\n\\n|depth of focus|distance toward and away from an object over which components of the object remain in clear focus.|\\n|---|---|\\n|depurination|removal of either of the purine bases, adenine and guanine, from a DNA molecule. Depurination is a form of DNA damage.|\\n|desmin|protein that makes up the intermediate filaments in muscle cells.|\\n|desmosome|type of anchoring junction that joins the intermediate filaments of neighboring cells. Desmosomes are common in tissues such as skin.|\\n|diabetes|The word diabetes simply refers to conditions in which a patient produces lots of urine. The only form of diabetes we discuss in this book is diabetes mellitus in which the patient produces large volumes of urine containing sugar. Diabetes mellitus is caused by either a failure in the endocrine gland that produces insulin or a failure of the tissues to respond adequately to insulin.|\\n|diacylglycerol (DAG)|two acyl groups (fatty acid chains) joined by ester bonds to a glycerol backbone. Diacylglycerol is produced by the action of phospholipase C on phospholipid and helps to activate protein kinase C.|\\n|dideoxyribonucleotide|man-made molecule similar to a deoxyribonucleotide but lacking a 3\\'-hydroxyl group on its sugar residue. Used in DNA sequencing.|\\n|differentiation|process whereby a cell becomes specialized for a particular function.|\\n|diffusion|movement of a substance that results from the individual small random thermal movements of its molecules.|\\n|dimeric|formed of two parts. Of a molecule, formed of two parts that are not covalently linked.|\\n|diphosphate|of a compound, bearing a chain of two phosphates in a line (as opposed to a bisphosphate, which bears two independent phosphate groups). Adenosine diphosphate is an example.|\\n|diploid|containing two sets of chromosomes, in humans, this means two sets of 23, one from the father, one from the mother. Most of the cells of the body (the somatic cells) are diploid.|\\n|dipolar bond|unusual form of covalent bond found in trimethylamine N-oxide.|\\n|disaccharide|dimer of two monosaccharides. Examples are lactose (galactose β1 → 4 glucose) and sucrose (glucose α1 → β2 fructose).|\\n|distal|far from the center.|\\n|disulfide bond (bridge)|covalent bond between two sulfur atoms. In proteins disulfide bonds form by oxidation of two thiol (—SH) groups of cysteine residues. Found chiefly in extracellular proteins.|\\n|DNA (deoxyribonucleic acid)|polymer of deoxyribonucleotides; DNA specifies the inherited instructions of a cell.|\\n|DNAa|DNA binding protein that causes the two strands of the double helix to separate in the first stages of DNA replication.|\\n---\\n# GLOSSARY\\n\\n|DNAb|helicase that moves along a DNA strand, breaking hydrogen bonds, and in the process unwinding the helix.|\\n|---|---|\\n|DNAc|DNA-binding protein that serves to bring DNAb to the DNA strands.|\\n|DNA chip|tiny glass wafer to which cloned DNAs are attached. Also known as gene chips or microarrays.|\\n|DNA excision|process of cutting out damaged DNA prior to repair.|\\n|DNA fingerprint|individual pattern of DNA fragments determined by the number and position of specific repeated sequences.|\\n|DNA ligase|enzyme that joins two DNA molecules by catalyzing the formation of a phosphodiester bond.|\\n|DNA polymerase|enzyme that synthesizes DNA by catalyzing the formation of a phosphodiester bond. DNA is always synthesized in the 5\\' to 3\\' direction.|\\n|DNA repair enzymes|enzymes that detect and repair altered DNA.|\\n|DNA replication|process in which the two strands of the double helix unwind and each acts as a template for the synthesis of a new strand of DNA.|\\n|DNA sequencing|determining the order of bases on the DNA strand.|\\n|docking protein (signal recognition particle receptor)|receptor on the endoplasmic reticulum to which the signal recognition particle binds during the process of polypeptide chain synthesis and import into the endoplasmic reticulum.|\\n|dogma|belief. The “central dogma” of molecular biology is that “DNA makes RNA makes protein”—the concept that the sequence of bases on DNA defines the sequence of bases on RNA, and the sequence of bases on RNA then defines the sequence of amino acids on protein.|\\n|domain|separately folded segment of the polypeptide chain of a protein.|\\n|dominant|gene that exerts its effect even when only one copy is present. Most dominant genes are dominant because they produce functional protein while the recessive gene does not. However, some mutant genes are dominant because even having 50% of one’s protein as the mutant form is enough to cause an effect; the gene causing familial Creutzfeldt–Jacob disease is an example.|\\n|donor|that which gives. In a hydrogen bond, the donor is the atom (oxygen, nitrogen, or sulfur) to which the hydrogen is covalently bonded and that gives up some of its share of electrons to a second electron-grabbing atom.|\\n|double helix|structure formed when two filaments wind about each other, most commonly applied to DNA, but also applicable to, e.g., F-actin.|\\n|downstream|general term meaning the direction in which things move. When applied to the DNA within and adjacent to a gene, it means lying on the side of the transcription start site that is transcribed into RNA. When applied to signaling pathways it means in the direction in which the signal travels, thus MAP kinase is downstream of Ras.|\\n|duplication|doubling-up of a particular sequence of the genetic material.|\\n---\\n# GLOSSARY\\n\\n|dynamic instability|term that describes the behavior of microtubules in which they switch from phases of growth to phases of shortening.|\\n|---|---|\\n|dynamin|GTPase that plays a critical role in the formation of clathrin-coated vesicles.|\\n|dynein|motor protein. Cytoplasmic dynein moves vesicles along microtubules while dynein arms power ciliary and flagellar beating by generating sliding between adjacent outer doublet microtubules.|\\n|E2F-1|transcription factor required for DNA synthesis. In quiescent cells E2F-1 is prevented from activating transcription by being bound to RB, the product of the retinoblastoma gene. E2F-1 is released when RB is phosphorylated by CDK4.|\\n|ectoplasm|viscous, gel-like outer layer of cytoplasm.|\\n|effective stroke|part of the beat cycle of a cilium that pushes on the extracellular medium.|\\n|effector caspase|caspase that digests cellular components in the process of apoptosis. Effector caspases are differentiated from caspases 8 and 9, which do not themselves digest cellular components but which activate effector caspases by hydrolyzing particular peptide bonds in them.|\\n|electrically excitable|able to produce action potentials.|\\n|electrochemical gradient|free energy gradient for an ion in solution. The arithmetical sum of gradients due to concentration and voltage.|\\n|electron gun|source of electrons in electron microscopes.|\\n|electron microscope|microscope in which the image is formed by passage of electrons through, or scattering of electrons by, the object.|\\n|electron transport chain|series of electron acceptor/donator molecules found in the inner mitochondrial membrane which transport electrons from the reduced carriers NADH and FADH2 (thus reoxidizing them) to oxygen. The entire complex forms a carrier that uses the energy of NADH and FADH2 oxidation to transport hydrogen ions up their electrochemical gradient out of the mitochondrion.|\\n|electrophoresis|method of separating charged molecules by drawing them through a filtering gel material using an electrical field.|\\n|electrostatic bond|a strong attraction between ions or charged groups of opposite charge.|\\n|electrostatic interaction|attraction or repulsion between ions or charged groups.|\\n|elongation factors|proteins that speed up the process of protein synthesis at the ribosome. Elongation factor tu (EF-tu) is a GTPase.|\\n|embryonic stem cell|cells derived from an early embryo. Embryonic stem cells have the capability to divide indefinitely and to become any cell type in the body.|\\n|endocytosis|inward budding of plasma membrane to form vesicles. Endocytosis is the process by which cells retrieve plasma membrane and take up material from their surroundings.|\\n|endonuclease|enzyme that digests nucleic acids by cleaving internal phosphodiester bonds.|\\n|endoplasm|fluid, inner layer of cytoplasm that streams during cytoplasmic streaming.|\\n---\\n# GLOSSARY\\n\\n|endoplasmic reticulum (ER)|network (reticulum) of membrane-delimited tubes and sacs that extends from the outer membrane of the nuclear envelope almost to the plasma membrane. There are two types of ER, rough endoplasmic reticulum (RER) with a surface coating of ribosomes and smooth endoplasmic reticulum (SER).|\\n|---|---|\\n|endosome|organelle to which newly formed endocytotic vesicles are translocated and with which they fuse.|\\n|endosymbiotic theory|proposal that some of the organelles of eukaryotic cells originated as free-living prokaryotes.|\\n|endothelial cells|cells that line blood vessels and other body cavities that do not open to the outside.|\\n|endothelium|layer of cells that lines blood vessels and other body cavities that do not open to the outside.|\\n|energy currency|source of energy that the cell can use to drive processes that would otherwise not occur because their ΔG is positive. Energy currencies can be coenzymes such as ATP and NADH, which give up energy on conversion to, respectively, ADP and NAD+ , or electrochemical ion gradients such as the hydrogen ion gradient across the mitochondrial membrane and the sodium gradient across the plasma membrane.|\\n|enhancer sequence|specific DNA sequence to which a protein binds to increase the rate of transcription of a gene.|\\n|enthalpy|thermodynamic description of heat.|\\n|entropy|degree of disorder in a system.|\\n|envelope|closed sheet enclosing a volume. The term is used to describe, among other things, the double membrane layer enclosing certain organelles, and the outer membrane of certain viruses.|\\n|enzyme|catalyst made mainly of protein. Like all catalysts, enzymes work by reducing the activation energy of the reaction.|\\n|epidermis|protective outer cell layer of an organism.|\\n|epithelial cells|cells that make up an epithelium.|\\n|epithelium|sheet of cells covering the surface of the human body and those internal cavities such as the lungs and gut that open to the outside. In contrast endothelium lines internal spaces such as blood vessels.|\\n|equilibrium|total balance of opposing forces. A process or object is in equilibrium if the tendency to go in one direction is exactly equal to the tendency to go in the other direction. For an ion this condition is equivalent to saying that the electrochemical gradient for that ion across the membrane is zero. For a chemical reaction, equilibrium occurs when the rate of the forward reaction is equal to the rate of the reverse reaction, e.g., 2H2O ←→OH- + H3O+.|\\n|equilibrium voltage|transmembrane voltage that will exactly balance the concentration gradient of a particular ion.|\\n---\\n# GLOSSARY\\n\\n|ER (endoplasmic reticulum):|network (reticulum) of membrane channels that extends from the outer membrane of the nuclear envelope almost to the plasma membrane; there are two types of ER, rough endoplasmic reticulum (RER) with a surface coating of ribosomes and smooth endoplasmic reticulum (SER).|\\n|---|---|\\n|ES (embryonic stem) cell:|cell derived from a very early embryo. ES cells have not yet determined their developmental fate and can therefore, depending on the conditions, generate the entire range of tissue types.|\\n|essential amino acid:|amino acid that the organism needs but cannot synthesize. Histidine, isoleucine, leucine, lysine, methionine, phenylalanine, threonine, tryptophan, and valine are essential for humans.|\\n|essential fatty acid:|fatty acid that the organism needs but cannot synthesize. Linoleic and linolenic acids are essential for humans.|\\n|ester bond:|bond formed between the hydrogen of an alcohol group and the hydroxyl of a carboxyl group by the elimination of water.|\\n|euchromatin:|that portion of the nuclear chromatin that is not tightly packed. Euchromatin contains genes that code for proteins that are being actively transcribed.|\\n|eukaryotic:|organism whose cells contain distinct nuclei and other organelles; includes all known organisms except prokaryotes (bacteria and cyanobacteria).|\\n|exocytosis:|fusion of a vesicle with the plasma membrane. Exocytosis causes the soluble contents of the vesicle to be released to the extracellular medium, while integral membrane proteins of the vesicle become integral membrane proteins of the plasma membrane.|\\n|exon:|in a eukaryotic gene, exons are those parts that after RNA processing leave the nucleus. In contrast introns are spliced out before the RNA leaves the nucleus.|\\n|exonuclease:|enzyme that digests nucleic acids by cleaving phosphodiester links successively from one end of the molecule.|\\n|expression (of a gene):|appearance of the protein which the gene encodes.|\\n|expression vector:|cloning vector containing a promoter sequence recognized by the host cell, thus enabling a foreign DNA insert to be transcribed into mRNA.|\\n|extracellular matrix:|meshwork of filaments and fibers that surrounds and supports mammalian cells. The major protein of the extracellular matrix is collagen.|\\n|extracellular medium:|aqueous medium outside cells. For a unicellular organism the extracellular medium is the outside world. For a multicellular organism such as a human being the extracellular medium is the fluid between the cells.|\\n|extragenic DNA:|DNA that can neither be identified as coding for protein or RNA nor as being promoters or enhancers regulating transcription.|\\n|Fas:|metabotropic cell surface receptor that activates caspase 8 and hence triggers apoptosis.|\\n|fat:|triacylglycerol (triglyceride) that is solid at room temperature. In contrast, oils are liquid at room temperature.|\\n|fat cells (adipocytes):|cells that store fats (triacylglycerols).|\\n---\\n# GLOSSARY\\n\\n|fatty acid|carboxyl group attached to a long chain of carbon atoms with attached hydrogens, that is, a chemical of the general form C nH m C OH O|\\n|---|---|\\n|feedback|process in which the result of a process modifies the mechanisms carrying out that process to increase or decrease their rate. In negative feedback a change in some parameter activates a mechanism that reverses the change in that parameter; an example is the effect of tryptophan on expression of the trp operon. In positive feedback a change in some parameter activates a mechanism that accelerates the change; an example is the effect of depolarization on the opening of voltage-gated sodium channels.|\\n|feedforward|control process in which a change in a parameter is predicted, and mechanisms initiated that will act to reduce the change in that parameter.|\\n|ferredoxin|protein component of the electron transport system of chloroplasts.|\\n|fibroblast|cell found in connective tissue. Fibroblasts synthesize collagen and other components of the extracellular matrix.|\\n|fibroblast growth factor|paracrine transmitter that opposes apoptosis and promotes cell division in target cells. Although named for its effect on fibroblasts, FGF triggers proliferation of many tissues and plays critical roles in determining cell fate during development.|\\n|fission|breakage into two parts. The word is used for prokaryote replication. In addition, it describes the division of a single membrane-bound organelle into two and the process whereby a vesicle breaks away from a membrane.|\\n|flagellin|protein that is the bacterial flagellum.|\\n|flagellum|swimming appendage. In eukaryotes flagella are extensions of the cell that use a dynein/microtubule motor system. In prokaryotes flagella are extracellular proteins rotated by a motor at their base.|\\n|fluid mosaic model|generally accepted hypothesis of how cell membranes are formed from a lipid bilayer plus protein.|\\n|fmet (formyl methionine)|methionine modified by the attachment of a formyl group; fmet is the first amino acid in all newly made bacterial polypeptides.|\\n|focal contact|points at which a locomoting cell makes contact with its substrate.|\\n|formyl methionine (fmet)|methionine modified by the attachment of a formyl group; fmet is the first amino acid in all newly made bacterial polypeptides.|\\n|fourteen-three-three (14-3-3)|protein that binds phosphorylated BAD, hence preventing it from triggering apoptosis.|\\n|frameshift mutation|mutation that changes the mRNA reading frame, caused by the insertion or deletion of a nucleotide.|\\n|G (guanosine)|guanine linked to the sugar ribose. Guanosine is a nucleoside.|\\n---\\n# GLOSSARY\\n\\n|G0|describes the quiescent state of cells that have left the cell division cycle.|\\n|---|---|\\n|G1 (gap 1)|period of the cell division cycle that separates mitosis from the following S phase.|\\n|G2 (gap 2)|period of the cell division cycle between the completion of S phase and the start of cell division or M phase.|\\n|GABA (γ-amino butyric acid)|γ -amino acid that acts to open chloride channels in the plasma membrane of sensitive nerve cells. Usually called γ -amino butyric acid rather than γ -amino butyrate, even though the latter (− OOC—CH2—CH2—CH2—NH3+) is the form in which it is found at neutral pH.|\\n|G-actin|globular, subunit form of actin.|\\n|gamete|sperm or egg. Gametes are haploid; that is, they contain just one set of chromosomes (23 in humans).|\\n|gamma-amino butyric acid (γ-amino butyric acid, GABA)|γ -amino acid that acts to open chloride channels in the plasma membrane of sensitive nerve cells. Usually called γ -amino butyric acid rather than γ -amino butyrate, even though the latter (− OOC—CH2—CH2—CH2—NH3+) is the form in which it is found at neutral pH.|\\n|GAP (GTPase activating protein)|protein that speeds up the rate at which GTPases hydrolyze GTP and therefore switch from the active to the inactive state.|\\n|gap 0 (G0)|describes the quiescent state of cells that have left the cell division cycle.|\\n|gap 1 (G1)|period of the cell division cycle that separates mitosis from the following S phase.|\\n|gap 2 (G2)|period of the cell division cycle between the completion of S phase and the start of cell division or M phase.|\\n|gap junction|type of cell junction that allows solute to pass from the cytosol of one cell to the cytosol of its neighbor without passing through the extracellular medium. Gap junctions consist of many paired gap junction channels or connexons.|\\n|gap junction channel (connexon)|channel in the plasma membrane with a central hole about 1.5 nm in diameter. Gap junction channels only open when they contact a second channel on another cell, in this case they open and form a water-filled tube that runs all the way through the plasma membrane of the first cell, across the small gap between the cells, and through the plasma membrane of the second cell, so allowing passage of solute from the cytosol of one cell to the cytosol of the other.|\\n|gastrocnemius muscle|muscle at the back of the shin. When it contracts the toes move down.|\\n|gated, gating|a channel is gated if it can switch to a shape in which the tube through the membrane is closed.|\\n|gated transport|transport of fully folded proteins through intracellular pores that open to allow their passage.|\\n|GDP|guanosine diphosphate; guanosine with two phosphates attached to the 5\\' carbon of ribose.|\\n---\\n# GLOSSARY\\n\\nGEF (guanine nucleotide exchange factor): protein that accelerates the rate at which GDP leaves a GTPase to be replaced by GTP, thus switching the GTPase from its inactive state to its active state.\\n\\nGEFS (generalized epilepsy with febrile seizures): relatively common form of childhood epilepsy, usually clearing up spontaneously. GEFS + is the less common condition in which the seizures still occur past the age of 6 years.\\n\\ngelsolin: type of actin-binding protein that binds to and fragments actin filaments.\\n\\ngene: fundamental unit of heredity. In many cases a gene contains the information needed to code for a single polypeptide.\\n\\ngene chip: tiny glass wafer to which cloned DNAs are attached. Also known as microarrays or DNA chips.\\n\\ngene family: group of genes that share sequence similarity and usually code for proteins with a similar function.\\n\\ngene probe: cDNA or genomic DNA fragment used to detect a specific DNA sequence to which it is complementary in sequence. The probe is tagged in some way to make it easy to detect. The tag could be, for example, a radioactive isotope or a fluorescent dye.\\n\\ngene therapy: correction or alleviation of a genetic disorder by the introduction of a normal gene copy into an affected individual.\\n\\ngenetically modified (GM): organism with a genome that has been modified by modern molecular techniques, usually by the addition of novel gene(s) or by swopping in new DNA to replace existing gene(s).\\n\\ngenetic code: relationship between the sequence of the four bases in DNA and the amino acid sequence of proteins.\\n\\ngenome: complete set of genes in an organism.\\n\\ngenomic DNA library: collection of bacterial clones each of which contains a different fragment of foreign genomic DNA.\\n\\ngerm cells: cells that give rise to the eggs and sperm.\\n\\nGibbs free energy: if the change of Gibbs free energy during a reaction is negative, the reaction is favored and will proceed if the activation energy is sufficiently low.\\n\\nglial cells: electrically inexcitable cells found in the nervous system.\\n\\nglucocorticoid: steroid hormone produced by the adrenal cortex that forms part of the system controlling blood sugar levels.\\n\\nglucocorticoid receptor: intracellular receptor to which glucocorticoid hormone binds.\\n\\ngluconeogenesis: synthesis of glucose from noncarbohydrate precursors such as amino acids and lactate.\\n\\nglucose: hexose monosaccharide. Glucose is the commonest sugar in the blood and is the dominant cellular fuel in animals, being used in glycolysis to generate ATP and pyruvate, the latter fueling the Krebs cycle.\\n---\\n# GLOSSARY\\n\\n|glucose carrier|plasma membrane protein that carries glucose into or out of cells. Some cells, such as skeletal muscle cells, will only translocate glucose carriers to their membranes when protein kinase B is active.|\\n|---|---|\\n|glyceride|compound formed by attaching units to a glycerol backbone. Triacylglycerols (previously called triglycerides) and phospholipids are glycerides.|\\n|glycerol|CH2OH—CHOH—CH2OH. The backbone to which acyl groups (fatty acid chains) are attached to make triacylglycerols and phospholipids.|\\n|glycogen|glucose polymer that can be quickly hydrolyzed to yield glucose; an α1 → 4 polymer of glucose with α1 → 6 branches.|\\n|glycogen phosphorylase|enzyme that releases glucose-1-phosphate monomers from glycogen. The glucose-1-phosphate is then converted to glucose-6-phosphate, which can be used in respiration or dephosphorylated to glucose for release into the blood.|\\n|glycogen phosphorylase kinase (phosphorylase kinase)|kinase that is activated by the calcium–calmodulin complex and phosphorylates glycogen phosphorylase, activating the latter enzyme.|\\n|glycolysis|breakdown of glucose to pyruvate.|\\n|glycosidic bond|bond linking monosaccharide residues in which the carbon backbones are linked through oxygen and a water molecule is lost.|\\n|glycosylation|addition of sugar residues to a molecule. Both proteins and lipids can be glycosylated.|\\n|glyoxisome|type of peroxisome found in plant cells that is concerned with the conversion of fatty acids to carbohydrate.|\\n|glyoxylate|CHO—COO−; an intermediate in various metabolic pathways including the glyoxylate shunt.|\\n|glyoxylate shunt|pathway found in plants and bacteria that allows acetyl-CoA to be converted into glucose.|\\n|GM (genetically modified)|organism with a genome that has been modified by modern molecular techniques, usually by the addition of novel gene(s) or by swopping in new DNA to replace existing gene(s).|\\n|Golgi apparatus|system of flattened cisternae concerned with glycosylation and other modifications of proteins.|\\n|G protein (trimeric G protein)|protein that links a class of metabotropic cell surface receptors with downstream targets. Trimeric G proteins comprise an α subunit that binds and hydrolyzes GTP and a βγ subunit that dissociates from the α subunit while the latter is in the GTP-bound state. Important G proteins are Gq, which activates phospholipase Cβ, and Gs, which activates adenylate cyclase.|\\n|Gq|isoform of trimeric G protein that activates phospholipase Cβ and therefore generates a calcium signal.|\\n|grana|distinctive structures within the chloroplast formed by the stacking of the thylakoid membranes.|\\n---\\n# GLOSSARY\\n\\n|gratuitous inducer|inducer of transcription that is not itself metabolized by the resulting enzymes.|\\n|---|---|\\n|Grb2 (growth factor receptor binding protein number 2)|linker protein that has an SH2 domain and is therefore recruited to phosphotyrosine, e.g., on receptor tyrosine kinases. Grb2 in turn recruits SOS, bringing SOS to the plasma membrane where it can act as a guanine nucleotide exchange protein (GEF) for Ras.|\\n|green fluorescent protein|fluorescent protein made by the jellyfish Victoria victoria. Unlike other colored or fluorescent proteins, it contains no prosthetic groups and therefore will fluoresce when expressed by any cell in which the gene is successfully inserted and expressed.|\\n|growth factor|paracrine transmitter that modifies the developmental pathway of the target cell, often by causing cell division.|\\n|growth factor receptor binding protein number 2 (Grb2)|linker protein that has an SH2 domain and is therefore recruited to phosphotyrosine, e.g., on receptor tyrosine kinases. Grb2 in turn recruits SOS, bringing SOS to the plasma membrane where it can act as a guanine nucleotide exchange protein (GEF) for Ras.|\\n|G|isoform of trimeric G protein that activates adenylate cyclase and therefore causes an increase of cAMP concentration.|\\n|GTP|guanosine triphosphate; guanosine with three phosphates attached to the 5\\' carbon of ribose.|\\n|GTPase|enzyme that hydrolyzes GTP. The name is usually restricted to that family of proteins that bind GTP and adopt a new shape that can then activate target proteins. Once they hydrolyze the GTP they have bound, they switch back to the original form. Examples are EF-tu, Arf, Ran, dynamin, Gq, and Gs.|\\n|GTPase activating protein (GAP)|protein that speeds up the rate at which GTPases hydrolyze GTP and therefore switch from the active to the inactive state.|\\n|guanine|one of the bases found in DNA and RNA; guanine is a purine.|\\n|guanine nucleotide exchange factor (GEF)|protein that accelerates the rate at which GDP leaves a GTPase to be replaced by GTP, thus switching the GTPase from its inactive state to its active state.|\\n|guanosine|guanine linked to the sugar ribose; guanosine is a nucleoside.|\\n|guanosine diphosphate (GDP)|guanosine with two phosphates attached to the 5\\' carbon of ribose.|\\n|guanosine triphosphate (GTP)|guanosine with three phosphates attached to the 5\\' carbon of ribose.|\\n|guanylate cyclase|An enzyme that converts GTP to the intracellular messenger cyclic GMP (cGMP). One isoform of guanylate cyclase is activated by NO.|\\n|Guthrie test|test for phenylketonuria. Newborn babies’ blood is tested for the presence of phenylalanine at unusually high concentration.|\\n---\\n# GLOSSARY\\n\\nhairpin loop: loop in which a linear object folds back on itself. Used to describe the loop formed in a RNA molecule due to complementary base pairing.\\n\\nhaploid: containing a single copy of each chromosome, in humans, this means 23 chromosomes. Sperm and eggs are haploid while somatic cells contain one set of 23 from each parent and are referred to as being diploid.\\n\\nhead group: hydrophilic group found in phospholipids. The head group is attached to the glycerol backbone by a phosphodiester link. Examples are choline and inositol.\\n\\nheel prick test (Guthrie test): test for phenylketonuria. Newborn babies’ blood is tested for the presence of phenylalanine at unusually high concentration.\\n\\nhelicase: enzyme that helps unwind the DNA double helix during replication.\\n\\nα helix: a common secondary structure in proteins, in which the polypeptide chain is coiled, each turn of the helix taking 3.6 amino acid residues. The nitrogen atom in each peptide bond forms a hydrogen bond with the oxygen four residues ahead of it in the polypeptide chain.\\n\\nhemicellulose: polysaccharide component of the plant cell wall that links cellulose fibrils together.\\n\\nhemoglobin: oxygen-carrying, iron-containing protein of the blood.\\n\\nheterochromatin: that portion of the nuclear chromatin that is tightly packed. Much of the heterochromatin is repetitive DNA with no coding function.\\n\\nheterozygote: individual whose two copies of a gene differ, e.g., one may be mutant. We have not used the term in this book but have used the simpler word carrier for the more restricted case where an individual is a heterozygote for a recessive gene.\\n\\nhexokinase: enzyme that phosphorylates glucose on the number 6 carbon.\\n\\nhistone: positively charged protein that binds to negatively charged DNA and helps to fold DNA into chromatin.\\n\\nhistone octamer: two molecules each of histones H2A, H2B, H3, and H4, the whole forming a nucleosome.\\n\\nhomologous recombination: process in which a length of DNA with ends that are homologous to a section of chromosome swops in, replacing the existing length of DNA in the chromosome. Homologous recombination occurs naturally at chiasmata during crossing over during meiosis. It can also occur in some cells when they transfected with the appropriate exogenous DNA.\\n\\nhomozygote: individual whose two copies of a gene are identical. Most of us are homozygous for most of our genes.\\n\\nhormone: long-lived transmitter that is released into the blood and travels around the body before being broken down.\\n\\nhormone response element (HRE): specific DNA sequence to which a steroid hormone receptor binds.\\n---\\n# GLOSSARY\\n\\n|housekeeping gene|gene that is transcribed into mRNA in nearly all the cells of a eukaryotic organism; in bacterial cells a housekeeping gene is one that is always being transcribed.|\\n|---|---|\\n|HRE (hormone response element)|specific DNA sequence to which a steroid hormone receptor binds.|\\n|hybridization|association of unlike things. In molecular genetics, the association of two nucleic acid strands (either RNA or DNA) by complementary base pairing.|\\n|hydration shell|cloud of water molecules that surrounds an ion in solution.|\\n|hydrocarbon tail|long chain of carbon atoms with attached hydrogens found in phospholipids and triacylglycerols. The tail represents all of a molecule of fatty acid except the carboxyl group.|\\n|hydrogen bond|relatively weak bond formed between a hydrogen atom and two electronegative atoms (such as nitrogen or oxygen) where the hydrogen is shared between the other atoms.|\\n|hydrogen ion gradient|energy currency. Hydrogen ions are more concentrated outside bacteria and mitochondria than inside, and this chemical gradient is supplemented by a voltage gradient pulling hydrogen ions in. If hydrogen ions are allowed to rush in down their electrochemical gradient, they release 17,000 J/mol.|\\n|hydrolysis|breakage of a covalent bond by the addition of water. —H is added to one side, —OH to the other.|\\n|hydropathy plot|running average of side chain hydrophobicity along a polypeptide chain. From the hydropathy plot one can predict, e.g., membrane-spanning domains in integral membrane proteins.|\\n|hydrophilic|molecule or part of a molecule that can interact with water.|\\n|hydrophobic|a molecule or part of a molecule that will associate with other hydrophobic molecules in preference to water.|\\n|hydrophobic effect|tendency of hydrophobic molecules or parts of molecules to cluster together away from water, such as hydrophobic amino acid residues in the center of a protein, or the fatty acid chains in lipid bilayers.|\\n|hydroxyl group|—OH group. The term is specifically not used for an —OH that forms part of a carboxyl group —COOH.|\\n|hypoxanthine|purine that is used to make the nucleotide inosine. Inosine can pair with any of uracil, cytosine, or adenine.|\\n|imino acid|organic molecule containing both carboxyl (—COOH) and imino (—NH—) groups. Proline is an imino acid although it is usually called an amino acid.|\\n|immunofluorescence|use of fluorescently labeled antibodies to reveal the location of specific chemicals, e.g., in fluorescence microscopy or in western blotting.|\\n|inactivation (of voltage gated channels)|long term closure, often because of blockage of the open channel with a plug that is attached to the cytosolic face of the protein.|\\n---\\n# GLOSSARY\\n\\n|indirect immunofluorescence microscopy|form of light microscopy in which a fluorescent secondary antibody is used to label a preapplied primary antibody specific for a particular protein or subcellular structure. Also called secondary immunofluorescence microscopy.|\\n|---|---|\\n|inducible operon|operon that is transcribed only when a specific substance is present.|\\n|initial velocity (of a reaction)|rate at which an enzyme converts substrate to product in the absence of product, that is, at the onset of a reaction.|\\n|inositol|cyclic polyalcohol, (CHOH)6, that forms the headgroup of the phospholipid phosphatidylinositol. Phosphorylation of inositol yields inositol trisphosphate.|\\n|inositol trisphosphate (IP3, InsP3)|small phosphorylated cyclic polyalcohol that is released into the cytosol by the action of phospholipase C on the membrane lipid phosphatidylinositol bisphosphate and that acts to cause release of calcium ions from the endoplasmic reticulum.|\\n|inositol trisphosphate-gated calcium channel|channel found in the endoplasmic reticulum of many cells. The channel opens when inositol trisphosphate binds to its cytoplasmic aspect. It allows only calcium ions to pass.|\\n|insertional mutagenesis|insertion of additional nucleotides into a stretch of DNA.|\\n|in situ hybridization|binding of a particular labeled sequence of RNA (or DNA) to its matching sequence in the genome as a way of searching for the location of that sequence in a tissue sample.|\\n|InsP3 (inositol trisphosphate, IP3)|small phosphorylated cyclic polyalcohol that is released into the cytosol by the action of phospholipase C on the membrane lipid phosphatidylinositol bisphosphate and that acts to cause release of calcium ions from the endoplasmic reticulum.|\\n|insulin|hormone produced by endocrine cells in the pancreas. It activates its own receptor tyrosine kinase, which in turn acts mainly through activation of PI-3-kinase and hence protein kinase B.|\\n|insulin receptor|receptor tyrosine kinase specific for insulin, which acts mainly through activation of PI-3-kinase and hence protein kinase B.|\\n|insulin receptor substrate number 1 (IRS-1)|protein phosphorylated on tyrosine by the insulin receptor. Once phosphorylated, it recruits PI-3-kinase, which can then be phosphorylated and hence activated.|\\n|integral protein (of a membrane)|class of protein that is tightly associated with a membrane, usually because its polypeptide chain crosses the membrane at least once. Integral membrane proteins can only be isolated by destroying the membrane, e.g., with detergent. In contrast, peripheral membrane proteins are more loosely associated.|\\n|integrin|dimeric proteins with an extracellular domain that binds to extracellular matrix proteins and an intracellular domain that attaches to actin microfilaments.|\\n|intermediate filament|one of the filaments that makes up the cytoskeleton; composed of various subunit proteins.|\\n---\\n# GLOSSARY\\n\\nintermembrane space: in organelles such as mitochondria, chloroplasts, and nuclei that are bound by two membranes, the intermembrane space is the aqueous space between the inner and outer membranes. The intermembrane space of nuclei is contiguous with the lumen of the ER. The intermembrane space of mitochondria has the ionic composition of cytosol because porin in the outer mitochondrial membrane allows solutes of Mr < 10,000 to pass.\\n\\ninterphase: period of synthesis and growth that separates one cell division from the next; consists of the G1, S, and G2 phases of the cell division cycle.\\n\\nintracellular messenger: cytosolic solute that changes in concentration in response to external stimuli or internal events, and which acts on intracellular targets to change their behavior. Calcium ions, cyclic AMP and cyclic GMP are the three common intracellular messengers.\\n\\nintracellular receptors: receptors that are not on the plasma membrane but that lie within the cell and bind transmitters that diffuse through the plasma membrane.\\n\\nintron: in a eukaryotic gene, introns are those parts that are spliced out before the RNA leaves the nucleus. In contrast exons are those parts that after RNA processing leave the nucleus.\\n\\nion: charged chemical species. A single atom that has more or less electrons than are required to exactly neutralize the charge on the nucleus is an ion (e.g., Na+, Cl−). A molecule with one or more charged regions is also an ion (e.g., phosphate, HPO42−; leucine, −OOC—CH(CH2.CH(CH3)2)—NH3+.\\n\\nionotropic cell surface receptors: channels that open when a specific chemical binds to the extracellular face of the channel protein.\\n\\nIP3 (inositol trisphosphate, InsP3): small phosphorylated cyclic polyalcohol that is released into the cytosol by the action of phospholipase C on the membrane lipid phosphatidylinositol bisphosphate and that acts to cause release of calcium ions from the endoplasmic reticulum.\\n\\nIP3 receptor: calcium channel in the membrane of the endoplasmic reticulum that opens when inositol trisphosphate binds to its cytosolic aspect.\\n\\nIRS-1: protein phosphorylated on tyrosine by the insulin receptor. Once phosphorylated, it recruits PI-3-kinase, which can then be phosphorylated and hence activated.\\n\\nisoelectric point: pH at which a protein or other molecule has no net charge.\\n\\nisoforms: related proteins that are the products of different genes or of differential splicing of mRNAs from one gene.\\n\\nisomers: different compounds with the same molecular formula. For example, glucose and mannose are both C6H12O6 and are therefore isomers.\\n\\nJNK: stress-activated protein kinase that stimulates cell repair but can also trigger apoptosis.\\n\\nK+/Na+ ATPase (sodium/potassium ATPase): plasma membrane carrier. For every ATP hydrolyzed, three Na+ ions are moved out of the cytosol and two K+ ions are moved in.\\n---\\n# GLOSSARY\\n\\n|Kartegener’s syndrome|human condition in which the heart is on the wrong side. Male subjects may also be sterile. Caused by mutations in components of cilia and flagella.|\\n|---|---|\\n|keratin|protein that makes up the intermediate filaments in epithelial cells.|\\n|ketone|any chemical containing a carbon atom with single bonds to two other carbons and a double bond to an oxygen. Acetone (CH3—CO—CH3) and acetoacetate (CH3—CO—CH2—COO−) are ketones.|\\n|ketone bodies|acetoacetate (CH3—CO—CH2—COO−) and 3-hydroxybutyrate (CH3—CHOH—CH2—COO−); circulating fuels formed from fats and used to fuel body tissues in times of starvation. Acetoacetate slowly loses carbon dioxide to form acetone.|\\n|ketosis|overproduction of ketone bodies that is seen in extreme starvation and in diabetes.|\\n|kinase|enzyme that phosphorylates a molecule by transferring a phosphate group from ATP to the molecule.|\\n|kinesin|molecular motor protein responsible for movement along microtubules in the anterograde direction.|\\n|kinetochore|point of attachment of the chromosome to the spindle. The kinetochore forms around the centromere.|\\n|K M (Michaelis constant)|substrate concentration at which one measures an initial velocity that is half as fast as the maximal velocity (V m) of an enzyme reaction.|\\n|Krebs cycle (tricarboxylic acid cycle)|series of reactions in the mitochondrial matrix in which acetate is completely oxidized to CO2 with the attendant reduction of NAD+ to NADH and FAD to FADH2.|\\n|lac (lactose) operon|cluster of three bacterial genes that encode enzymes involved in metabolism of lactose.|\\n|lactose|disaccharide comprising galactose linked to glucose by a β1 → 4 glycosidic bond.|\\n|lactose intolerant|unable to hydrolyze dietary lactose.|\\n|lactose (lac) operon|cluster of three bacterial genes that encode enzymes involved in metabolism of lactose.|\\n|lagging strand|strand of DNA that grows discontinuously during replication; in contrast to the leading strand, which is synthesized continuously.|\\n|lamins|proteins that make up the nuclear lamina. Lamins are chemically related to intermediate filaments.|\\n|leading strand|strand of DNA that grows continuously in the 5\\' to 3\\' direction by the addition of deoxyribonucleotides.|\\n|ligand|when two molecules bind together, one (often the smaller one) is called the ligand, and the other (often the bigger one) is called the receptor.|\\n|light reactions|reactions in photosynthesis which use the energy of light to split water to release oxygen, to produce NADPH, and to move hydrogen ions across the thylakoid membrane setting up a gradient which is then used to drive the formation of ATP from ADP. The NADPH and ATP are used in the dark reactions.|\\n---\\n# GLOSSARY\\n\\n|linkage, linked (of genes)|physical association of genes on the same chromosome. Linked genes tend to be inherited together.|\\n|---|---|\\n|linker DNA|stretch of DNA that separates two nucleosomes.|\\n|lipid bilayer|two layers of lipid molecules that form a membrane.|\\n|localization sequence (targeting sequence)|stretch of polypeptide that determines the cellular compartment to which a synthesized protein is sent.|\\n|lumen|inside of a closed structure or tube.|\\n|lysosome|membrane-bound organelle containing digestive enzymes.|\\n|M-phase|the period of the cell division cycle during which the cell divides; M phase consists of mitosis and cytokinesis.|\\n|M-phase promoting factor (MPF)|the complex of CDK1 and cyclin B that regulates the G2/M phase transition of the cell division cycle.|\\n|macromolecule|large molecule.|\\n|macrophage|phagocytic housekeeping cell that engulfs and digests bacteria and dead cells.|\\n|major groove (of DNA)|larger of the two grooves along the surface of the DNA double helix.|\\n|mannose|hexose monosaccharide.|\\n|mannose-6-phosphate|mannose that is phosphorylated on carbon number 6. Mannose-6-phosphate is the sorting signal that identifies lysosomal proteins.|\\n|MAPK, MAP kinase (mitogen-associated protein kinase)|enzyme that phosphorylates numerous targets, including transcription factors that trigger transcription of the cyclin-D gene.|\\n|MAPKK (MAP kinase kinase)|enzyme that phosphorylates and activates MAP kinase, it itself being phosphorylated and hence activated by MAPKK kinase.|\\n|MAPKKK (MAP kinase kinase kinase)|enzyme that phosphorylates and activates MAPK kinase. The most important isoform of MAPKKK is also called Raf. Raf is activated by the G protein Ras.|\\n|matrix|very vague term meaning a more or less closed location, often but not exclusively one that is a solid basis on which things can grow or attach. The term is used of the extracellular matrix in animals, formed of collagen and other fibers, and of the aqueous volume inside various organelles.|\\n|maximal velocity (of an enzyme-catalyzed reaction)|limiting value of the initial velocity of a reaction as the substrate concentration is increased at constant enzyme concentration. Occurs when the enzyme is saturated with substrate.|\\n|meiosis|form of cell division that produces gametes, each with half the genetic material of the cells that produce them.|\\n|meiosis I and II|first and second meiotic divisions.|\\n---\\n# GLOSSARY\\n\\nmeiotic spindle: bipolar, microtubule-based structure on which chromosome segregation occurs during meiosis I and II.\\n\\nmembrane: bilayer formed from lipid plus protein.\\n\\nmessenger RNA (mRNA): RNA molecule that carries the genetic code. The order of bases on mRNA specifies the amino acid sequence of a polypeptide chain. In eukaryotes, the mRNA leaves the nucleus and is translated into protein in the cytoplasm.\\n\\nmetabolism: all of the reactions going on inside a cell.\\n\\nmetabotropic cell surface receptors: receptors that are linked to, and activate, enzymes.\\n\\nmetaphase: period of mitosis or meiosis at which the chromosomes align prior to separation at anaphase.\\n\\nmetaphase plate: equator of the mitotic or meiotic spindle; the point at which the chromosomes congregate at metaphase of mitosis or meiosis.\\n\\n7-methyl guanosine cap: modified guanosine found at the 5\\' terminus of eukaryotic mRNA. A guanosine is attached to the mRNA by a 5\\'-5\\'-phosphodiester link and is subsequently methylated on atom number 7 of the guanine.\\n\\nMichaelis constant (KM): substrate concentration at which one measures an initial velocity that is half as fast as the maximal velocity (Vm) of an enzyme reaction.\\n\\nMichaelis–Menten equation: equation that defines the effect of substrate concentration on the initial velocity of an enzyme-catalyzed reaction:\\n\\n|v0 =|Vm[S]|\\n|---|---|\\n|KM + [S]| |\\n\\nwhere Vm is the limiting initial velocity obtained as the substrate concentration approaches infinity, [S] is the substrate concentration, and KM is a constant called the Michaelis constant, equal to the substrate concentration at which one measures an initial velocity that is half as fast as the maximal velocity.\\n\\nmicroarray: tiny glass wafer to which cloned DNAs are attached. Also known as gene chips or DNA chips.\\n\\nmicrofilament: one of the major filaments of the cytoskeleton; also known as actin filament or F-actin; synonymous with the thin filament of striated muscle.\\n\\nmicrosatellite DNA: repetitious DNA of unknown function comprising many repeats of a unit of four or fewer base pairs.\\n\\nmicrotubular molecular motor: protein that moves vesicles along microtubules; examples are cytoplasmic dynein and kinesin.\\n\\nmicrotubule: tubular cytoplasmic filament composed of tubulin. Tubulin is the major component of 9 + 2 cilia and flagella and of the mitotic and meiotic spindle.\\n\\nmicrotubule organizing center (MTOC): structure from which cytoplasmic microtubules arise; synonymous with the centrosome.\\n---\\n# GLOSSARY\\n\\n|microvilli (sing. microvillus)|projections from the surface of epithelial cells that increase the absorptive surface; contain actin filaments.|\\n|---|---|\\n|minisatellite DNA|repetitious DNA of unknown function comprising up to 20,000 repeats of a unit of about 25 base pairs.|\\n|minor groove (of DNA)|smaller of the two grooves along the surface of the DNA double helix.|\\n|missense mutation|base change in a DNA molecule that changes a codon so that it now specifies a different amino acid.|\\n|mitochondrial inner membrane|inner membrane of mitochondria that is elaborated into cristae. The electron transport chain, and ATP synthase, are integral membrane proteins of the mitochondrial inner membrane.|\\n|mitochondrial matrix|aqueous space inside the mitochondrial inner membrane, where the enzymes of the Krebs cycle are located.|\\n|mitochondrial outer membrane|outer membrane of mitochondria, permeable to solutes of M r < 10,000 because of the presence of the channel porin.|\\n|mitochondrion|cell organelle concerned with aerobic respiration.|\\n|mitogen|anything that promotes cell division. FGF and PDGF (fibroblast growth factor and platelet-derived growth factor) are potent mitogens for endothelial and smooth muscle cells.|\\n|mitogen-associated protein kinase (MAP kinase, MAPK)|enzyme that phosphorylates numerous targets, including transcription factors that trigger transcription of the cyclin-D gene.|\\n|mitosis|type of cell division found in somatic cells, in which each daughter cell receives the full complement of genetic material present in the original cell.|\\n|mitotic spindle|microtubule-based structure upon which chromosomes are arranged and translocated during mitosis.|\\n|mole|amount of substance comprising 6.023 × 10^23 (Avogadro’s number) molecules. One mole has a mass equal to the value of the relative molecular mass expressed in grams.|\\n|monomer|single unit, usually used to refer to a single building block of a larger molecule. Thus DNA is formed of nucleotide monomers. By analogy, the word is sometimes used to describe proteins that act as a single unit to distinguish them from related proteins that act as larger units, so myoglobin is said to be monomeric by comparison with hemoglobin, which has four subunits. GTPases such as Ran, Arf, and Ras are sometimes called “monomeric G proteins” to distinguish them from trimeric G proteins such as Gq and Gs.|\\n|monosaccharide|sweet-tasting chemical with many hydroxyl groups that can adopt a form in which an oxygen atom completes a ring of carbons. All the monosaccharides in this book have the general formula Cn(H2O)n where n = 5 (pentoses) or 6 (hexoses).|\\n|motif|recognizable conserved sequence of bases (in DNA) or amino acids (in a polypeptide). A motif in DNA may bind one transcription factor, e.g., 5\\'AGAACA3\\' binds the|\\n---\\n# GLOSSARY\\n\\nglucocorticoid receptor. A motif in a polypeptide may, e.g., fold in a particular way (e.g., a helix-turn-helix motif), or bind to a target (e.g., KDEL).\\n\\nmotoneuron (motoneurone): nerve cell that carries action potentials from the spine to the muscles. It releases the transmitter acetylcholine onto the muscle cells, causing them to depolarize and hence contract.\\n\\nM phase: period of the cell division cycle during which the cell divides; consists of mitosis and cytokinesis.\\n\\nMPF (M-phase promoting factor): complex of CDK1 and cyclin B that regulates the G2/M phase transition of the cell division cycle.\\n\\nM-phase promoting factor (MPF): complex of CDK1 and cyclin B that regulates the G2/M phase transition of the cell division cycle.\\n\\nM r: Relative Molecular Mass (sometimes called “molecular weight” and abbreviated M r or RMM) is the mass of one mole of a substance compared to the mass of one mole of hydrogen. It is dimensionless but is sometimes expressed in Daltons.\\n\\nmRNA (messenger RNA): RNA molecule that carries the genetic code. The order of bases on mRNA specifies the amino acid sequence of a polypeptide chain.\\n\\nMTOC (microtubule organizing center): structure from which cytoplasmic microtubules arise; synonymous with the centrosome.\\n\\nmuscarinic acetylcholine receptor: integral membrane protein that binds acetylcholine and then becomes a GEF for the trimeric G protein G q, hence leading to the activation of phospholipase Cβ which in turn leads to release of calcium ions from the endoplasmic reticulum.\\n\\nmuscle: tissue specialized for generating contractile force.\\n\\nmutation: inherited change in the structure of a gene or chromosome.\\n\\nmyelin: fatty substance that is wrapped around nerve cell axons by glial cells.\\n\\nmyosin: type of motor protein that moves along, or pulls on, actin filaments. The thick filaments in skeletal muscle are formed of myosin II molecules.\\n\\nmyosin II: class of myosin that forms the large filaments seen in striated muscle but that is also found in other cell types.\\n\\nmyosin V: class of myosin that carries cargo along actin filaments.\\n\\nN-terminal (amino terminal): the end of a peptide or polypeptide that has a free α-amino group. This end is made first on the ribosome.\\n\\nNADH (nicotinamide adenine dinucleotide, reduced form): combination of two nucleotides that is a strong reducing agent and one of the cell’s energy currencies.\\n\\nNADPH (nicotinamide adenine dinucleotide phosphate, reduced form): phosphorylated NADH; like NADH it is a strong reducing agent, but NADPH is not an energy currency, rather it is used in synthetic reactions in the cytoplasm.\\n\\nNa + /K + ATPase: plasma membrane carrier. For every ATP hydrolyzed, three Na + ions are moved out of the cytosol and two K + ions are moved in.\\n---\\n# GLOSSARY\\n\\n|necrosis|cell death that is due to damage so severe that the cell cannot maintain the level of its energy currencies and therefore falls apart—distinct from apoptosis.|\\n|---|---|\\n|negative feedback|control system in which a change in some parameter activates a mechanism that reverses the change in that parameter; an example is the effect of tryptophan on expression of the trp operon.|\\n|negative regulation (of transcription)|inhibition of transcription due to the presence of a particular substance, which is often the end product of a metabolic pathway.|\\n|Nernst equation|equation that allows the equilibrium voltage of an ion across a membrane to be calculated. The general form is Veq = (RT/zF) loge([Ioutside]/[Iinside]) volts where R is the gas constant (8.3 joules per degree per mole), T is the absolute temperature, z is the charge on the ion in elementary units, and F is 96,500 coulombs in a mole of monovalent ions. At 37°C the equation approximates to Veq = (62/z) log10([Ioutside]/[Iinside]) millivolts|\\n|nerve cell|electrically excitable cell with a long axon specialized for transmission of (usually sodium) action potentials.|\\n|nerve growth factor (NGF)|older name for neurotrophin 1.|\\n|nervous tissue|tissue formed of nerve and glial cells that carries out electrical data processing.|\\n|neurofilament|type of intermediate filament found in nerve cells.|\\n|neuron|another name for a nerve cell.|\\n|neurone|yet another name for a nerve cell.|\\n|neurotrophin|one of a family of paracrine transmitters that act upon nerve cells to prevent apoptosis and trigger differentiation. Neurotrophin 1 used to be called nerve growth factor or NGF.|\\n|NFAT|transcription factor that only translocates to the nucleus when dephosphorylated by the calcium–calmodulin activated phosphatase calcineurin.|\\n|NGF (nerve growth factor)|older name for neurotrophin 1.|\\n|nicotinamide|base used in the double nucleotides NADH and NADPH.|\\n|nicotinamide adenine dinucleotide (reduced form) (NADH)|combination of two nucleotides that is a strong reducing agent and one of the cell’s energy currencies.|\\n|nicotinamide adenine dinucleotide phosphate (reduced form) (NADPH)|phosphorylated NADH; like NADH it is a strong reducing agent, but NADPH is not an energy currency, rather it is used in synthetic reactions in the cytoplasm.|\\n---\\n# GLOSSARY\\n\\nnicotinic acetylcholine receptor: integral membrane protein of the plasma membrane that forms an ion channel that allows both sodium and potassium ions to pass. When it binds acetylcholine it opens, which in turn leads to electrical depolarization.\\n\\nnine plus two (9 + 2) axoneme: structure of cilium or flagellum; describes the arrangement of nine peripheral microtubules surrounding two central microtubules.\\n\\nnitric oxide (NO): paracrine transmitter that acts on intracellular receptors, the most important of which is guanylate cyclase.\\n\\nnode (of a myelinated axon): gap between adjacent lengths of myelin sheath, where the nerve cell plasma membrane directly contacts the extracellular medium. Also called node of Ranvier, after its discoverer, Louis Ranvier.\\n\\nnonpolar: covalent bonds in nonpolar molecules have electrons shared equally so that the constituent atoms do not carry a charge.\\n\\nnonsense mutation: base change that causes a codon that coded for an amino acid to become a stop codon.\\n\\nnorthern blotting: blotting technique in which RNAs, separated by size, are probed using a single-stranded cDNA probe or an antisense RNA probe.\\n\\nN-terminal (amino terminal): end of a peptide or polypeptide that has a free α-amino group. This end is made first on the ribosome.\\n\\nNTR: neurotrophin receptor. Although the Trk family are receptor tyrosine kinases for neurotrophins, the name NTR is usually reserved for the death domain receptor called p75 NTR.\\n\\nnuclear envelope: double-membrane system enclosing the nucleus; it contains nuclear pores and is continuous with the endoplasmic reticulum.\\n\\nnuclear lamina: meshwork of proteins lining the inner face of the nuclear envelope.\\n\\nnuclear pores: holes running through the nuclear envelope that regulate traffic of proteins and nucleic acids between the nucleus and the cytoplasm.\\n\\nnuclear pore complex: multiprotein complex that lies within and around the nuclear pore and that regulates import into and export from the nucleus in a process called gated transport.\\n\\nnuclease: enzyme that degrades nucleic acids.\\n\\nnucleic acid: polymer of nucleotides joined together by phosphodiester bonds; DNA and RNA are nucleic acids.\\n\\nnucleoid: region of a bacterial cell that contains the chromosome.\\n\\nnucleolar organizer regions: region of one or more chromosomes at which the nucleolus is formed.\\n\\nnucleolus: region(s) of the nucleus concerned with the production of ribosomes.\\n\\nnucleoside: purine, pyrimidine, or nicotinamide attached to either ribose or deoxyribose.\\n\\nnucleosome: beadlike structure formed by a stretch of DNA wrapped around a histone octamer.\\n---\\n# GLOSSARY\\n\\n|nucleotide|building block of nucleic acids; a nucleoside that is phosphorylated on its 5\\' carbon atom.|\\n|---|---|\\n|nucleotide excision repair|process that removes a thymine dimer, together with some 30 surrounding nucleotides, from DNA. The gap is then repaired by the actions of DNA polymerase I and DNA ligase.|\\n|nucleus|cell organelle housing the chromosomes; enclosed within a nuclear envelope.|\\n|objective lens|lens of a light or electron microscope that forms a magnified image of a specimen.|\\n|obligate anaerobe|organism that is poisoned by oxygen so that it can only function anaerobically.|\\n|oil|triacylglycerol (triglyceride) that is liquid at room temperature. In contrast, fats are solid at room temperature.|\\n|Okasaki fragment|series of short fragments that are joined together to form the lagging strand during DNA replication.|\\n|oleic acid|C17H33—COOH; a fatty acid with a single double bond. The dominant acyl group in olive oil is derived from oleic acid.|\\n|olfactory neuron|nerve cell found in the nose, whose dendrites are sensitive to smell chemicals.|\\n|oligonucleotide|short fragment of DNA or RNA.|\\n|oligosaccharide|chain of up to 100 or so monosaccharides linked by glycosidic bonds.|\\n|oligosaccharide transferase|enzyme that adds an oligosaccharide group onto a protein.|\\n|oocyte|cell that undergoes meiosis to give rise to an egg.|\\n|open complex|complex between DNA and protein that causes the two strands of DNA to separate, e.g., during DNA replication and transcription.|\\n|open promoter complex|structure formed when the two strands of the double helix separate so that transcription can commence.|\\n|operator|DNA sequence to which a repressor protein binds to prevent transcription from an adjacent promoter.|\\n|operon|cluster of genes that encode proteins involved in the same metabolic pathway and that are transcribed as one length of mRNA under the control of one promoter. As far as we know, operons are only found in prokaryotes.|\\n|optical isomers|two molecules that differ only in that one is the mirror image of the other.|\\n|organelle|membrane-bound, intracellular structure such as a mitochondrion, chloroplast, lysosome, etc.|\\n|organic|in chemistry, an organic compound is one that contains carbon atoms. When applied to farming and food, organic means the avoidance of high-intensity agricultural techniques including genetic engineering.|\\n---\\n# GLOSSARY\\n\\n|organism|cell or clone of cells that functions as a discrete and integrated whole to maintain and replicate itself.|\\n|---|---|\\n|origin of replication|site on a chromosome at which DNA replication can commence.|\\n|osmolarity|overall strength of a solution; the more solute that is dissolved in a solution, the higher the osmolarity.|\\n|osmosis|movement of water down its concentration gradient.|\\n|outer doublet microtubules|paired microtubules that make the 9 of the 9 + 2 axoneme in cilia and flagella.|\\n|oxidation|removal of electrons from a molecule, e.g., by adding oxygen atoms, which tend to take more than their fair share of electrons in any bonds they make. Removal of hydrogen atoms from a molecule oxidizes it. (Don’t confuse with deprotonation, which is the loss of H + ).|\\n|β oxidation|process by which fatty acids are broken down into individual two-carbon units coupled to CoA to form acetyl-CoA. The process, which takes place in the mitochondrial matrix, generates both NADH and FADH2.|\\n|oxidizing agent|agent that will act to lessen the share of electrons that the atoms of a molecule have. Oxidizing agents often work by adding oxygen atoms or by removing hydrogen atoms (Don’t confuse with a base, which will accept H+, not H). Examples are oxygen itself, NAD+, and FAD.|\\n|P site (peptidyl site)|the site on a ribosome that is occupied by the growing polypeptide chain.|\\n|p21 CIP1|inhibitor of cyclin-dependent kinases. p21 CIP1, which is upregulated by cell–cell contact, prevents cells entering the S phase of the cell cycle.|\\n|p38|a stress-activated protein kinase related to mitogen associated protein kinase (MAP kinase) but with a very different role: p38 is activated by cell stress and stimulates both cell repair and apoptosis.|\\n|p53|transcription factor that stimulates cell repair but also apoptosis. Cancer cells are frequently found to have mutated, nonfunctional p53 genes.|\\n|p75 neurotrophin receptor|death domain receptor for neurotrophins. Upon binding neurotrophins p75 activates caspase 8 and hence initiates apoptosis unless a countermanding signal to survive is present.|\\n|PAC (P1 artificial chromosome)|cloning vector, derived from the bacteriophage P1, that is used to propagate, in E. coli, DNAs of about 150,000 bp.|\\n|pain receptor|nerve cell whose distal axon terminal is depolarized by potentially damaging events such as heat or stretching.|\\n|pain relay cell|this nerve cell, upon which a pain receptor synapses, carries the message on toward the brain.|\\n|paracrine transmitters|agonists released by cells into the extracellular fluid that can last many minutes and can therefore diffuse widely within the tissue before they are destroyed.|\\n---\\n# GLOSSARY\\n\\n|parallel β sheet|β sheet in which all the parallel polypeptide chains run in the same direction.|\\n|---|---|\\n|parenchyma cells|unspecialized plant cells whose cell walls are usually thin and bendable; the major site of metabolic activity and photosynthesis in leaves and green shoots.|\\n|patch clamp|technique in which a glass micropipette is sealed to the surface of a cell to allow electrical recording of cell properties.|\\n|PCR (polymerase chain reaction)|method for making many copies of a DNA sequence where the sequence, at least at each end, is known.|\\n|PDGF (platelet-derived growth factor)|paracrine transmitter that opposes apoptosis and promotes cell division in target cells such as endothelial cells and smooth muscle.|\\n|pectin|polysaccharide component of the plant cell wall; the amount of pectin in the cell wall determines its thickness.|\\n|pentose|monosaccharide with five carbon atoms.|\\n|pentose phosphate pathway|pathway that makes pentoses and generates reducing power in the form of NADPH by oxidizing glucose-6-phosphate.|\\n|peptide|short linear polymer of amino acids.|\\n|peptide bond|bond between amino acids. The bond is formed between the carboxyl group of one amino acid and the amino group of the next.|\\n|peptidyl site (P site)|site on a ribosome that is occupied by the growing polypeptide chain.|\\n|peptidyl transferase|enzyme that catalyzes the formation of a peptide bond between two amino acids.|\\n|peripheral membrane protein|class of protein that is easily detached from a cell membrane, unlike integral membrane proteins, which can only be isolated by destroying the membrane, e.g., with detergent.|\\n|peroxisome|class of cell organelles of diverse function. Peroxisomes frequently contain the enzyme catalase, which breaks down hydrogen peroxide into oxygen and water.|\\n|pH|measure of the acidity of a solution, equal to minus the logarithm to base 10 of the hydrogen ion concentration in moles liter-1. The smaller the pH value, the more acid the solution. A neutral solution has a pH of 7, that is, the H+ concentration is 10-7 mol liter-1 or 100 nmol liter-1.|\\n|PH domain|protein domain that binds phosphorylated inositols. The PH group on protein kinase B is recruited to the membrane by the intensely charged lipid phosphatidylinositol trisphosphate.|\\n|phage|short for bacteriophage, i.e., a virus that infects bacterial cells.|\\n|phase-contrast microscopy|type of light microscopy in which differences in the refractive index of a specimen are converted into differences in contrast.|\\n|PH domain|protein domain that binds phosphorylated inositols. The PH group on protein kinase B is recruited to the membrane by the intensely charged lipid phosphatidylinositol trisphosphate.|\\n---\\n# GLOSSARY\\n\\n|phenylketonuria:|inherited disease in which the enzyme phenylalanine hydroxylase is missing or defective. Unless intake of phenylalanine is drastically curtailed, phenylalanine and its transamination product phenylpyruvate build up in the body and cause brain damage. The heel prick or Guthrie test, performed on all newborn babies, tests for this condition.|\\n|---|---|\\n|phloem:|part of the plant vascular system; conveys the products of photosynthesis to their sites of use or storage.|\\n|phosphatase:|enzyme that removes phosphate groups from substrates. Glucose-6-phosphate phosphatase and calcineurin are phosphatases operating on a sugar and on proteins, respectively.|\\n|phosphate:|properly, a name for the ions H2PO4−, HPO42−, and PO43−. The word is also very commonly used to mean the group O−P−O− and we have used this convention in this book. If one wants to specifically describe the group, not the ions, one can refer to a phosphoryl group (a term that does not include the leftmost oxygen in the diagram above).|\\n|phosphatidylinositol bisphosphate (PIP 2):|membrane lipid. PIP 2 releases inositol trisphosphate into the cytosol upon hydrolysis by phospholipase C. Alternatively, PIP 2 can be further phosphorylated to yield phosphatidylinositol trisphosphate.|\\n|phosphatidylinositol trisphosphate (PIP 3):|a membrane lipid. Its intensely charged head-group can recruit proteins that contain PH domains to the plasma membrane.|\\n|phosphodiester link:|link between two parts of a molecule in which a phosphate is attached through an oxygen atom to each of the two parts. In phospholipids, the headgroup is attached to glycerol by a phosphodiester link. In DNA and RNA, successive nucleotides are joined together by phosphodiester links.|\\n|phosphofructokinase:|enzyme that phosphorylates fructose-6-phosphate to generate fructose-1,6-bisphosphate.|\\n|phosphoinositide 3-kinase (PI-3-kinase):|enzyme that phosphorylates phosphatidylinositol bisphosphate on the 3 position of the inositol ring to generate the intensely charged lipid phosphatidylinositol trisphosphate (PIP 3). PIP 3 can then recruit proteins containing PH domains to the plasma membrane.|\\n|phospholipase C:|enzyme that hydrolyzes the membrane lipid phosphatidylinositol bisphosphate (PIP 2) to release inositol trisphosphate into the cytosol. The β isoform is activated by the trimeric G protein G q, while the γ isoform is activated by tyrosine phosphorylation.|\\n---\\n# GLOSSARY\\n\\n|phosphoryl group:|O –|\\n|---|---|\\n| |P – O|\\n| |O|\\n|phosphorylase:|enzyme that cleaves a glycosidic bond by adding phosphate.|\\n|phosphorylase kinase (glycogen phosphorylase kinase):|kinase that is activated by the calcium–calmodulin complex and which phosphorylates glycogen phosphorylase, activating the latter enzyme.|\\n|phosphorylated:|having had a phosphate group added. The phosphate groups are usually substituted into a hydroxyl group to form a phosphoester bond but are sometimes substituted into an acid to form a phosphoanhydride or attached to a nitrogen atom to form a phosphoimide.|\\n|phosphorylation:|addition of a phosphate group to a molecule. Usually substituted into a hydroxyl group to form a phosphoester bond. Sometimes substituted into an acid to form a phosphoanhydride, or attached to a nitrogen atom to form a phosphoimide.|\\n|photosynthesis:|synthesis of complex organic molecules, and oxidation of water to release oxygen, that is driven by the energy of light.|\\n|phragmoplast:|structure associated with the formation of a new cell wall during cytokinesis in plant cells.|\\n|PI 3-kinase (phosphoinositide 3-kinase):|enzyme that phosphorylates phosphatidylinositol bisphosphate on the 3 position of the inositol ring to generate the intensely charged lipid phosphatidylinositol trisphosphate (PIP 3). PIP 3 can then recruit proteins containing PH domains to the plasma membrane.|\\n|PIP 2 (phosphatidylinositol bisphosphate):|membrane lipid that releases inositol trisphosphate into the cytosol upon hydrolysis by phospholipase C.|\\n|pKa:|parameter equal to −log 10 Ka and representing the pH at which the concentration of the dissociated acid is equal to the concentration of the undissociated acid.|\\n|PKA (protein kinase A; cAMP-dependent protein kinase):|protein kinase that is activated by the intracellular messenger cyclic AMP. PKA phosphorylates proteins (e.g., glycogen phosphorylase kinase) on serine and threonine residues.|\\n|PKB (protein kinase B):|protein kinase that is activated when it is itself phosphorylated; this in turn only occurs when PKB is recruited to the plasma membrane by phosphatidylinositol trisphosphate (PIP 3). PKB phosphorylates proteins on serine and threonine residues (e.g., the bcl-2 family protein BAD). An older name for PKB is Akt.|\\n|PKC (protein kinase C):|protein kinase that is activated by a rise of cytosolic calcium concentration; it is also activated by diacylglycerol. PKC phosphorylates proteins on serine and threonine residues.|\\n---\\n# GLOSSARY\\n\\n|PKG (protein kinase G; cGMP-dependent protein kinase)|protein kinase that is activated by the intracellular messenger cyclic GMP. PKG phosphorylates proteins (e.g., calcium ATPase) on serine and threonine residues.|\\n|---|---|\\n|plaque|area of dead bacteria in a lawn of live bacteria that is caused by infection of bacterial cells by bacteriophages.|\\n|plasmalemma|membrane that surrounds the cell. Also called the plasma membrane or the cell membrane.|\\n|plasma membrane|membrane that surrounds the cell. Also called the plasmalemma or the cell membrane.|\\n|plasmid|circular DNA molecule that is replicated independently of the host chromosome in bacterial cells.|\\n|plasmodesmata (singular, plasmodesma)|type of cell junction unique to plant cells, which provides a much bigger hole for passage of substances between the cytoplasm of the two cells than gap junctions do.|\\n|plastoquinone|quinone electron carrier found in chloroplasts.|\\n|platelet|small fragment of cell that contains no nucleus but that has a plasma membrane and some endoplasmic reticulum. Platelets are critical in the process of blood clotting, and they also release platelet-derived growth factor.|\\n|platelet-derived growth factor (PDGF)|paracrine transmitter that opposes apoptosis and promotes cell division in target cells such as endothelial cells and smooth muscle.|\\n|PLC (phospholipase C)|enzyme that hydrolyzes the membrane lipid phosphatidylinositol bisphosphate (PIP2) to release inositol trisphosphate into the cytosol. The β isoform is activated by the trimeric G protein Gq, while the γ isoform is activated by tyrosine phosphorylation.|\\n|polar|having covalent bonds in which the electrons are unequally shared, so that atoms have partial charges. Polar molecules can interact with water by electrostatic interactions and by hydrogen bonding.|\\n|poly adenosine tail|string of adenine residues added to the 3′ end of a eukaryotic mRNA.|\\n|polyadenylation|process whereby a poly-A tail is added to the 3′ end of a eukaryotic mRNA.|\\n|poly-A tail|string of adenine residues added to the 3′end of a eukaryotic mRNA.|\\n|polycistronic mRNA|mRNA that, when translated, yields more than one polypeptide.|\\n|polymer|chemical composed of a long chain of identical or similar subunits.|\\n|polymerase|enzyme that makes polymers, i.e., long chains of identical or very similar subunits. DNA and RNA polymerase, respectively, are involved in making DNA and RNA.|\\n|polymerase chain reaction (PCR)|method for making many copies of a DNA sequence when the sequence, at least at each end, is known.|\\n|polynucleotide kinase|enzyme that adds a phosphate group to the 5′end of DNA or RNA.|\\n---\\n# GLOSSARY\\n\\n|polypeptide|polymer of more than 50 amino acids joined by peptide bonds.|\\n|---|---|\\n|polyploid|having three or more sets of chromosomes.|\\n|polyribosome (or polysome)|chain of ribosomes attached to an mRNA molecule.|\\n|polysaccharide|chain of more than 100 or so monosaccharides linked by glycosidic bonds.|\\n|polysome (or polyribosome)|chain of ribosomes attached to an mRNA molecule.|\\n|polyunsaturated|having several double bonds. Usually applied to fatty acids; triacylglycerols with polyunsaturated acyl (fatty acid) groups are liquid even at low temperatures.|\\n|porin|channel found in the mitochondrial outer membrane. It spends most of its time open and allows all solutes of Mr < 10,000 to pass.|\\n|positive feedback|process in which the consequences of a change act to increase the magnitude of that change, so that a small initial change tends to get bigger and bigger.|\\n|positive regulation (of transcription)|process whereby transcription is activated in the presence of a particular substance.|\\n|postsynaptic cell|cell upon which a nerve cell releases its transmitter at a synapse.|\\n|potassium channel|channel in the plasma membrane of many cells that allows potassium ions to pass.|\\n|potassium/sodium ATPase (Na+ /K+ ATPase)|plasma membrane carrier. For every ATP hydrolyzed, three Na+ ions are moved out of the cytosol and two K+ ions are moved in.|\\n|preinitiation complex|complex formed between transcription factors and RNA polymerase at the promoter of a eukaryotic gene.|\\n|presynaptic terminal|region of an axon terminal that is specialized for exocytosis of transmitter.|\\n|primary antibody|first antibody applied to the preparation. In many cases the primary antibody is not itself labeled but must be revealed by applying a labeled secondary antibody.|\\n|primary cell wall|first layer of the plant cell wall.|\\n|primary endosome|acidic cell compartment with which coated vesicles fuse.|\\n|primary immunofluorescence|technique in which the location or presence of a chemical is revealed by treating the sample with a dye-labeled antibody.|\\n|primary structure (of a protein)|sequence of amino acids held together by peptide bonds making up a polypeptide.|\\n|primase|enzyme that synthesizes the RNA primers needed for the initiation of synthesis of the leading and lagging DNA strands.|\\n|primer|short sequence of nucleic acid (RNA or DNA) that acts as the start point at which a polymerase can initiate synthesis of a longer nucleic acid chain.|\\n|primosome|protein complex, including the enzyme primase, that is involved in the synthesis of RNA primers for DNA replication.|\\n|prion|protein that is itself an infective agent for a disease.|\\n---\\n# GLOSSARY\\n\\nprofilin: type of actin-binding protein that regulates the assembly of actin filaments.\\n\\nprogenitor: ancestor; an individual from which others are descended. In this book we use the term progenitor cell to mean the cell that divides to give rise to two other cells (in mitosis) or four others (in meiosis).\\n\\nprogeny: children, offspring. In this book we use the term progeny cells to mean the products of cell division.\\n\\nprogrammed cell death (apoptosis): process in which a cell actively promotes its own destruction, as distinct from necrosis. Apoptosis is important in vertebrate development, where tissues and organs are shaped by the death of certain cell lineages.\\n\\nprojector lens: lens of light or electron microscope that carries the image to the eye; more commonly known as the “eyepiece.”\\n\\nprokaryotic: type of cellular organization found in bacteria in which the cells lack a distinct nucleus and other organelles.\\n\\nprometaphase: period of mitosis or meiosis that sees the breakdown of the nuclear envelope and the attachment of the chromosomes to the mitotic spindle.\\n\\npromoter: region of DNA to which RNA polymerase binds to initiate transcription.\\n\\npronuclei: nuclei of the egg and sperm prior to fusion.\\n\\nprophase: period of mitosis or meiosis in which the chromosomes condense.\\n\\nprophase I: first prophase of meiosis.\\n\\nprophase II: second prophase of meiosis.\\n\\nprosthetic group: nonprotein molecule necessary for the activity of a protein. The concept overlaps with the concept of cofactor. The difference is just how tightly they are bound: a prosthetic group is very tightly bound and cannot be removed without at least partial unfolding of the protein. Examples are the heme groups of myoglobin, hemoglobin, and the cytochromes.\\n\\nprotein: polypeptide (a polymer of α-amino acids) that has a preferred way of folding.\\n\\nprotein engineering: designing a novel protein and then causing it to be built by altering the sequence of DNA.\\n\\nprotein kinase: enzyme that phosphorylates a protein by transferring a phosphate group from ATP to the molecule.\\n\\nprotein kinase A: protein kinase that is activated by the intracellular messenger cAMP. Protein kinase A phosphorylates proteins (e.g., glycogen phosphorylase kinase) on serine and threonine residues.\\n\\nprotein kinase B (PKB): protein kinase that is activated when it is itself phosphorylated; this in turn only occurs when PKB is recruited to the plasma membrane by phosphatidylinositol trisphosphate (PIP3). PKB phosphorylates proteins on serine and threonine residues (e.g., the bcl-2 family protein BAD). An older name for PKB is Akt.\\n---\\n# GLOSSARY\\n\\n|protein kinase C (PKC):|protein kinase that is activated by a rise of cytosolic calcium concentration; it is also activated by diacylglycerol. PKC phosphorylates proteins on serine and threonine residues.|\\n|---|---|\\n|protein kinase G (PKG; cGMP-dependent protein kinase):|protein kinase that is activated by the intracellular messenger cyclic GMP. PKG phosphorylates proteins (e.g., calcium ATPase) on serine and threonine residues.|\\n|protein phosphatase:|enzyme that removes phosphate groups from proteins.|\\n|protein phosphorylation:|addition of a phosphate group to a protein. The addition of the charge on the phosphoryl group can markedly alter the tertiary structure and therefore function of a protein.|\\n|protein targeting:|delivery of proteins to their correct cellular location.|\\n|protein translocator:|channel-like protein of the endoplasmic reticulum that allows polypeptide chains to cross the membrane as they are synthesized.|\\n|proteoglycans:|heavily glycosylated proteins that contribute to the extracellular matrix.|\\n|proteome:|complete protein content of the cell.|\\n|proteomics:|study of the proteome. For example, one might compare the protein profiles of cells in two tissues.|\\n|protofilaments:|chains of subunits that make up the wall of a microtubule.|\\n|protonated:|having accepted an H+. For example, the lactate ion, CH3CH(OH)COO−, will become protonated in acid solutions to become lactic acid, CH3CH(OH)COOH. The word proton is being used as shorthand for hydrogen atom nucleus since, for the commonest isotope of hydrogen, the nucleus is a single proton.|\\n|proximal:|close to the center.|\\n|pseudogene:|gene that has mutated such that it no longer codes for a protein.|\\n|pseudopodium:|projection extended by an amoeba or other crawling cell in the direction of movement.|\\n|P site (peptidyl site):|site on the ribosome occupied by the growing polypeptide chain.|\\n|purine:|nitrogenous base found in nucleotides and nucleosides; adenine, guanine and hypoxanthine are purines.|\\n|pyrimidine:|nitrogenous base found in nucleotides and nucleosides; cytosine, thymine, and uracil are pyrimidines.|\\n|pyrophosphate:||\\n---\\n# GLOSSARY\\n\\nthe pyrophosphate ion is stable if kept away from biological tissue but is rapidly hydrolyzed\\nto two phosphate ions by intracellular and extracellular enzymes.\\n\\nquaternary structure: structure in which subunits of a protein, each of which has a tertiary\\nstructure, associate to form a more complex molecule. The subunits associate tightly, but\\nnot covalently and may be the same or different.\\n\\nquiescent: at rest. Nondividing cells are often said to be quiescent.\\n\\nRab family: family of GTPases that mediate fusion of vesicles with other membranes.\\n\\nRaf: isoform of MAPKKK, the enzyme at the top of the mitogen-associated protein kinase\\ncascade.\\n\\nRas: GTPase that activates the MAP kinase pathway and hence promotes cell division. Ras\\nis activated by SOS, its GTP exchange factor (GEF), which is in turn recruited to the plasma\\nmembrane via receptor tyrosine kinases and the adaptor protein Grb2.\\n\\nRB: protein that binds to the transcription factor E2F-1 and therefore prevents transcription\\nof proteins required for DNA synthesis. Phosphorylation of RB by CDK4 causes it to release\\nE2F-1, allowing entry to S phase. Mutations in RB lead to the formation of the eye cancer\\ncalled retinoblastoma.\\n\\nreading frame: reading of the genetic code in blocks of three bases—there are three possible\\nreading frames for each mRNA only one of which will produce the correct protein.\\n\\nreceptor: protein that specifically binds a particular solute. Receptors can be transmem-\\nbrane, cytosolic, or nuclear proteins. Particular receptor proteins perform additional func-\\ntions (ion channel, enzyme, activator of endocytosis, transcription factor . . . ) that are acti-\\nvated by the binding of the solute.\\n\\nreceptor-mediated endocytosis: process in which ligands bind to specific receptors in the\\nplasma membrane and trigger clathrin-mediated vesicle budding.\\n\\nreceptor tyrosine kinase: integral membrane protein with a binding site for transmitter on\\nthe extracellular aspect and tyrosine kinase catalytic ability on the cytosolic aspect. Binding\\nof transmitter causes self-phosphorylation on tyrosine and hence recruitment of proteins\\nwith SH2 domains such as Grb2, phospholipase Cγ , and phosphoinositide 3-kinase, which\\nmay then themselves be phosphorylated. The PDGF receptor, the FGF receptor, the Trk\\nfamily of receptors, and the insulin receptor are all receptor tyrosine kinases.\\n\\nrecessive: gene is recessive if its effects are hidden when the organism posesses a second,\\ndominant, version. Recessive genes usually code for nonfunctional proteins, so that if the\\nindividual can make the protein using the other, working gene, no effects are seen.\\n\\nrecombinant DNA: artificial DNA molecule formed of DNA from two or more different\\nsources.\\n\\nrecombinant plasmid: plasmid into which a foreign DNA sequence has been inserted.\\n\\nrecombinant protein: protein expressed from the foreign DNA inserted into a recombinant\\nplasmid or other cloning vector. Recombinant proteins are often expressed in bacteria, yeast,\\ninsect, or mammalian cells.\\n---\\n# GLOSSARY\\n\\n|recombination|“cut-and-paste” of DNA. Recombination occurs at chiasmata during meiosis and also in embryonic stem cells, allowing insertional mutagenesis.|\\n|---|---|\\n|recovery stroke|part of the beat cycle of a cilium in which the cilium is moved back into a position where it can push again; used in contrast to the effective stroke that generates the force.|\\n|reduction|addition of electrons to a compound, e.g., by the addition of hydrogen atoms or the removal of oxygen atoms.|\\n|refractive index|measure of the capacity of any material to slow down the passage of light.|\\n|regulated secretion|secretion that only occurs in response to a signal, such as a rise in the cytosolic concentration of calcium ions.|\\n|Relative Molecular Mass|mass of one mole of a substance compared to the mass of one mole of hydrogen. It is abbreviated Mr or RMM and is sometimes called “molecular weight.” It is dimensionless but is sometimes expressed in Daltons.|\\n|repetitious DNA|DNA sequence that is repeated many times within the genome.|\\n|replication (of DNA)|The process whereby two DNA molecules are made from one.|\\n|replication fork|Y-shaped structure formed when the two strands of the double helix separate during replication.|\\n|repressible operon|operon whose transcription is repressed in the presence of a particular substance; often the final product of the metabolic pathway.|\\n|RER (rough endoplasmic reticulum)|portion of the endoplasmic reticulum associated with ribosomes and concerned with the synthesis of secreted proteins. Proteins destined to remain within the majority of single-membrane-bound organelles (Golgi, lysosomes, . . . ) are also made on the rough ER, as are integral proteins of these organelles and of the plasma membrane.|\\n|residue|that which is left over. In chemistry, when a molecule is built into a larger molecule with the loss of some part, the part that remains and forms part of the larger molecule is called a residue. For example, an amino acid that has been built into a polypeptide, with the loss of the elements of water, is said to be an amino acid residue.|\\n|resolving power|measure that defines the smallest object that can be distinguished using a microscope.|\\n|resting voltage|voltage across the plasma membrane of an unstimulated cell, typically −70 to −90 mV.|\\n|restriction enzyme endonuclease|enzyme that cleaves phosphodiester bonds within a specific sequence in a DNA molecule.|\\n|retinoblastoma|cancer of the eye, usually caused by a mutation in the RB gene.|\\n|retrograde|backward movement; when applied to axonal transport it means toward the cell body.|\\n|retrovirus|virus whose genetic information is stored in RNA.|\\n---\\n# GLOSSARY\\n\\n|reverse genetics|research that begins with a gene of known sequence but unknown effect and works toward deducing its effects and therefore function.|\\n|---|---|\\n|reverse transcriptase|enzyme of some viruses that copies RNA into DNA.|\\n|reverse transcription|process whereby RNA is copied into DNA.|\\n|ribonuclease|enzyme that cleaves phosphodiester links in RNA.|\\n|ribonuclease H|enzyme that cleaves phosphodiester links in an RNA molecule that is joined to a DNA molecule by complementary base pairing.|\\n|ribonucleic acid (RNA)|polymer of ribonucleoside monophosphates; see mRNA, tRNA, and rRNA.|\\n|ribonucleic acid polymerase|enzyme that synthesizes RNA.|\\n|ribonucleic acid primer|short length of RNA, complementary in sequence to a DNA strand, that allows DNA polymerase III to attach and begin DNA synthesis.|\\n|ribonucleoside monophosphate|nitrogenous base attached to the sugar ribose that has one phosphate group on its 5\\' carbon atom; also known as a ribonucleotide.|\\n|ribose|pentose sugar used to make the nucleotides that form RNA.|\\n|ribosomal RNA (rRNA)|RNA component of ribosomes. rRNA forms a major part of the ribosome and participates fully in the translation process.|\\n|ribozyme|RNA molecule with enzyme-like catalytic activity.|\\n|RNA (ribonucleic acid)|polymer of ribonucleoside monophosphates. Cells contain three types of RNA: messenger RNA, ribosomal RNA, and transfer RNA.|\\n|RNA polymerase|enzyme that synthesizes RNA.|\\n|RNA primer|short length of RNA, complementary in sequence to a DNA strand that allows DNA polymerase III to attach and begin DNA synthesis.|\\n|RNA splicing|removal of introns from an RNA molecule and the joining together of exons to form the final RNA product.|\\n|rough endoplasmic reticulum (RER)|portion of the endoplasmic reticulum associated with ribosomes and concerned with the synthesis of secreted proteins. Proteins destined to remain within the majority of single-membrane-bound organelles (Golgi, lysosomes, . . . ) are also made on the rough ER, as are integral proteins of these organelles and of the plasma membrane.|\\n|rRNA (ribosomal RNA)|RNA component of ribosomes. rRNA forms a major part of the ribosome and participates fully in the translation process.|\\n|ryanodine|plant toxin that binds to the ryanodine receptor, with complex effects on gating of the channel.|\\n|ryanodine receptor|calcium channel found in the membrane of the endoplasmic reticulum. In most cells it opens in response to a rise of calcium concentration in the cytosol. In skeletal muscle ryanodine receptors are directly linked to voltage-gated calcium channels in the plasma membrane and open when the latter open.|\\n---\\n# GLOSSARY\\n\\nσ factor (sigma factor): subunit of bacterial RNA polymerase that recognises the promoter sequence.\\n\\nS phase: period of the cell division cycle during which DNA replication occurs.\\n\\nS value (sedimentation coefficient): a value that describes how fast macromolecules and organelles sediment in a centrifuge.\\n\\nsaltatory conduction: jumping of an action potential from node to node down a myelinated axon.\\n\\nsalt bridge (in protein structure): interaction between a positively charged amino acid residue (such as arginine) and a negatively charged residue (such as aspartate).\\n\\nsarcomere: contractile unit of striated muscle.\\n\\nsarcoplasmic reticulum: type of smooth endoplasmic reticulum found in striated muscle; concerned with the regulation of the concentration of calcium ions.\\n\\nsatellite DNA: DNA sequence that is tandemly repeated many times.\\n\\nsaturated (of fatty acids): containing no carbon–carbon double bonds.\\n\\nsaturation kinetics: are said to occur when the rate of a reaction approaches a maximum, limiting value as the concentration of reactant increases. The rate becomes limited by the availability of binding sites on the catalyst for the reactant.\\n\\nscanning electron microscope (SEM): type of electron microscope in which the image is formed from electrons that are reflected back from the surface of a specimen as the electron beam scans rapidly back and forth over it. The scanning electron microscope is particularly useful for providing topographical information about the surfaces of cells or tissues.\\n\\nSchwann cell: glial cell of the peripheral (outside the brain and spinal cord) nervous system.\\n\\nSDS–PAGE (sodium dodecyl sulfate–polyacrylamide gel electrophoresis): technique for separating proteins by their relative molecular mass.\\n\\nsecondary antibody: in immunofluorescence, a labeled antibody that is used to reveal the location of an unlabeled primary antibody. Secondary antibodies are produced by one species of animal in response to the injection of another animal’s antibodies, thus a “goat antirabbit” secondary will bind to any primary antibody that has been generated by a rabbit. Labeled secondary antibodies can be used to study many different questions because the specificity in the final image or western blot is provided by the primary antibody, not by the secondary.\\n\\nsecondary cell wall: layers of a plant cell wall formed external to the primary cell wall.\\n\\nsecondary immunofluorescence microscopy: form of light microscopy in which a fluorescent secondary antibody is used to label a preapplied primary antibody specific for a particular protein or subcellular structure. Also called indirect immunofluorescence microscopy.\\n\\nsecondary structure: regular, repeated folding of the backbone of a polypeptide. The side chains of the amino acids have an influence but are not directly involved. There are two common types of secondary structure: the α helix and the β sheet.\\n---\\n# GLOSSARY\\n\\nsecond messenger: another name for intracellular messenger (the “first messenger” being the extracellular transmitter chemical). A cytosolic solute that changes in concentration in response to external stimuli or internal events and that acts on intracellular targets to change their behavior. Calcium ions, cyclic AMP, and cyclic GMP are the three common intracellular messengers.\\n\\nsecretion: synthesis and release from the cell of a chemical.\\n\\nsecretory vesicles: vesicles derived from the Golgi apparatus that transport secreted proteins to the cell membrane with which they fuse.\\n\\nsedimentation coefficient (S value): value that describes how fast macromolecules and organelles sediment in a centrifuge.\\n\\nsemiconservative replication: mode of replication used for DNA; both strands of the double helix serve as templates for the synthesis of new daughter strands.\\n\\nserine–threonine kinase: enzyme that phosphorylates proteins on serine or threonine residues, by transferring the terminal phosphate of ATP to the amino acid side chain. With a few exceptions, protein kinases are either serine–threonine kinases (which can phosphorylate on serine or on threonine) or tyrosine kinases (which only phosphorylate on tyrosine).\\n\\nseven-methyl guanosine cap: modified guanosine found at the 5\\' terminus of eukaryotic mRNA. A guanosine is attached to the mRNA by a 5\\'-5\\'-phosphodiester link and is subsequently methylated on atom number 7 of the guanine.\\n\\nSH2 domain: domain found in a number of proteins that binds to phosphorylated tyrosine. Many proteins with SH2 domains are recruited to receptor tyrosine kinases when the latter phosphorylate themselves in response to ligand binding. Important proteins with SH2 domains are Grb2, PI-3-kinase, and phospholipase Cγ.\\n\\nβ sheet: common secondary structure in proteins, in which lengths of fully extended polypeptide run alongside each other, hydrogen bonds forming between the peptide bonds of the adjoining strands.\\n\\nShine–Dalgarno sequence: sequence on a bacterial mRNA molecule to which the ribosome binds.\\n\\nside chain (of amino acids): group attached to the α carbon of an amino acid.\\n\\nsigma factor (σ factor): subunit of bacterial RNA polymerase that recognizes the promoter sequence.\\n\\nsignal peptidase: enzyme that cleaves the signal sequence from a polypeptide as it enters the lumen of the endoplasmic reticulum.\\n\\nsignal recognition particle: ribonucleoprotein particle that recognizes and binds to the signal sequence at the N terminus of a polypeptide.\\n\\nsignal recognition particle receptor: receptor on the endoplasmic reticulum to which the signal recognition particle binds during the process of polypeptide chain synthesis and import into the endoplasmic reticulum. Also called the “docking protein.”\\n\\nsignal sequence: short stretch of amino acids found at the N terminus of polypeptides that targets them to the endoplasmic reticulum.\\n---\\n# GLOSSARY\\n\\n|simple diffusion|not a specific term: we often say “simple diffusion” to emphasize that a solute movement is passive, down a concentration gradient, and does not require a carrier or channel.|\\n|---|---|\\n|single-strand binding protein|protein that binds to the separated DNA strands to keep them in an extended form during replication, thus preventing the double helix from reforming.|\\n|site-directed mutagenesis|technique used to change the base sequence of a DNA molecule at a specific site.|\\n|skeletal muscle cells|large multinucleate muscle cells that are attached to bone. Most cuts of meat are mainly skeletal muscle.|\\n|small nuclear RNAs (snRNAs)|small RNA molecules found in the nucleus that play a role in RNA splicing.|\\n|smooth endoplasmic reticulum (SER)|portion of the endoplasmic reticulum without attached ribosomes, among its functions are the synthesis of lipids and the storage and stimulated release of calcium ions.|\\n|smooth muscle|nonstriated muscle, found in many places in the body including blood vessels and the intestine.|\\n|smooth muscle cells|small muscle cells that lack the characteristic striations seen in skeletal and cardiac muscle.|\\n|SNARES|proteins that mediate fusion of vesicles with other membranes.|\\n|snRNAs (small nuclear RNAs)|small RNA molecules found in the nucleus that play a role in RNA splicing.|\\n|sodium action potential|action potential driven by the opening of sodium channels and the resulting sodium influx.|\\n|sodium/calcium exchanger|carrier in the plasma membrane. Three sodium ions move into the cell down their electrochemical gradient and one calcium is moved out up its concentration gradient.|\\n|sodium gradient|energy currency. Sodium ions are more concentrated outside the cell than inside, and this chemical gradient is usually supplemented by a voltage gradient pulling sodium ions in. If sodium ions are allowed to enter down their electrochemical gradient, they release 15,000 J/mol.|\\n|sodium/potassium ATPase|plasma membrane carrier. For every ATP hydrolyzed, three Na+ ions are moved out of the cytosol and two K+ ions are moved in.|\\n|sodium pump (sodium/potassium ATPase)|plasma membrane carrier. For every ATP hydrolyzed, three Na+ ions are moved out of the cytosol and two K+ ions are moved in.|\\n|solute|substance that is dissolved in a liquid.|\\n|somatic cells|cells that make up all the normal tissues of the human body; distinct from the germ cells that form the gametes (eggs and sperm).|\\n|sorting signal|section of a protein that causes the cell to direct the protein to a specific cell compartment such as the nucleus or mitochondrion. Sorting signals can be lengths of peptide|\\n---\\n# GLOSSARY\\n\\n|sorting vesicle|vesicle that carries proteins from one membrane compartment to another.|\\n|---|---|\\n|SOS|guanine nucleotide exchange factor for a GTPase called Ras. SOS is recruited to the plasma membrane by binding to growth factor receptor binding protein number 2, which in turn binds activated receptor tyrosine kinases.|\\n|Southern blotting|blotting technique in which DNAs, separated by size, are probed using a single-stranded cDNA probe.|\\n|spatial summation|the process in which simultaneous release of synaptic transmitter by a number of presynaptic nerve cells depolarizes the postsynaptic neuron to threshold.|\\n|specific activity|ratio between activity and amount of material. For an enzyme it is the ratio between measured activity and mass of protein. It can be used as an indication of the purity of an enzyme preparation.|\\n|specificity constant|ratio between the catalytic rate constant (kcat) and the Michaelis constant (KM) for an enzyme. It is a rate constant with dimensions of liters mol−1 s−1 and is used to compare different substrates for the same enzyme or to compare the effectiveness of one enzyme with another.|\\n|spermatid|cell formed by meiosis that differentiates to form a spermatozoon.|\\n|spermatozoon|motile male gamete.|\\n|S phase|period of the cell division cycle during which DNA replication occurs.|\\n|splicesome|complex of proteins and small RNA molecules involved in RNA splicing.|\\n|squamous|flat. A term used of epithelial cells.|\\n|standard state|defined set of conditions under which to state the thermodynamics of a reaction. For example, ΔG°\\' is determined for aqueous solutions at pH 7.0|\\n|start signal|start signal for protein synthesis is the codon AUG, specifying the incorporation of methionine.|\\n|stearic acid|C17H35—COOH; a fatty acid with no double bonds; i.e., it is fully saturated. The acyl group derived from stearic acid is common in animal fats and phospholipids.|\\n|stereo isomers|isomers (i.e., different compounds with the same molecular formula) in which the atoms have the same connectivity but that differ in the way the atoms are arranged in space.|\\n|steroid hormones|these transmitters act on intracellular receptors to activate transcription of particular genes. Glucocorticoids are one type of steroid hormone; the sex hormones (oestradiol, testosterone, progesterone) are another.|\\n|steroid hormone receptor|transcription factors that, upon binding their appropriate steroid hormone, activate transcription of particular genes.|\\n|sticky ends (of DNA)|short single-stranded ends produced by cleavage of the two strands of a DNA molecule at sites that are not opposite to one another.|\\n---\\n# GLOSSARY\\n\\n|stop codon|codons UAA, UAG, and UGA are codons that signal protein synthesis to stop. Also known as termination codons.|\\n|---|---|\\n|stop signal|signal, to stop protein synthesis, given to the ribosome by a stop codon on mRNA.|\\n|stratified epithelium|type of epithelium consisting of several layers, such as the skin.|\\n|stress-activated protein kinase|kinase that stimulates cell repair but can also trigger apoptosis. p38 and JNK are stress-activated protein kinases.|\\n|stress fiber|bundle of actin filaments commonly seen in cultured, nonmotile animal cells.|\\n|striated muscle|striped muscle; includes skeletal and cardiac muscle.|\\n|stroma|volume within the chloroplast inner membrane but outside the thylakoids.|\\n|substrate|in normal English, a substrate is a solid base. In biology, the term is used to mean (1) a reactant in a reaction catalyzed by an enzyme, e.g., lactose is a substrate for β-galactosidase; (2) a base that cells grow and move on, e.g., collagen is a good substrate for cell attachment.|\\n|sucrose|disaccharide comprising glucose linked to fructose by an α1↔ β2 glycosidic bond.|\\n|summation (at synapses)|additive effects of more than one presynaptic action potential upon the postsynaptic voltage.|\\n|supercoiling|organization of a linear structure into coils at more than one spatial scale.|\\n|S value (sedimentation coefficient)|value that describes how fast macromolecules and organelles sediment in a centrifuge.|\\n|Svedberg unit (S value)|value that describes how fast macromolecules and organelles sediment in a centrifuge.|\\n|synapse|structure formed from the axon terminal of a nerve cell and the adjacent region of the postsynaptic cell. Transmitter released by the axon terminal diffuses across the synapse gap and acts upon the postsynaptic cell.|\\n|T (thymine)|one of the four bases found in DNA; thymine is a pyrimidine.|\\n|tandem repeats|many copies of the same DNA sequence that lie adjacent to each other on the chromosome.|\\n|targeting sequence|stretch of polypeptide that determines the cellular compartment to which a synthesized protein is sent.|\\n|TATA box|sequence found about 20 bases upstream of the beginning of many eukaryotic genes that forms part of the promoter sequence and is involved in positioning RNA polymerase for correct initiation of transcription.|\\n|taxol|compound obtained from the bark of the Pacific yew, Taxus brevifolia; binds to tubulin. Taxol is a powerful anticancer drug.|\\n|telomeres|specialized regions at the ends of eukaryotic chromosomes. Telomeres are rich in minisatellite DNA.|\\n---\\n# GLOSSARY\\n\\n|telophase|final period of mitosis or meiosis in which the chromosomes decondense and the nuclear envelope reforms.|\\n|---|---|\\n|telophase I|telophase of the first meiotic division (meiosis I).|\\n|telophase II|telophase of the second meiotic division (meiosis II).|\\n|temporal summation|the process in which action potentials in a presynaptic nerve cell occur at a high enough frequency that the depolarizations that they evoke in the postsynaptic cell do not have time to decay back to the resting voltage but rather ride on each other and depolarize the postsynaptic cell to threshold.|\\n|terminally differentiated|term that describes a cell that cannot return to the cell division cycle. Nerve cells are terminally differentiated; glial cells are not.|\\n|termination codon|codons UAA, UAG, and UGA are codons that signal protein synthesis to stop. Also known as stop codons.|\\n|terminator (of transcription)|DNA sequence that, when transcribed into mRNA, causes transcription to terminate.|\\n|tertiary structure|three-dimensional folding of a polypeptide chain into a biologically active protein molecule. It usually includes regions of secondary structure. Interactions of the amino acid side chains are central in its formation.|\\n|thermodynamics|study of how energy affects matter and chemical reactions.|\\n|thermogenin|H+ channel found in the inner mitochondrial membrane of brown fat cells. Its presence uncouples NADH oxidation from ATP synthesis, that is, the electron transport chain can operate, pumping H+ out of the mitochondrial matrix, even when ATP synthase is not allowing H+ back in.|\\n|thick filament|one of the two filaments that form the cytoskeleton of striated muscle; composed of the motor protein myosin II.|\\n|thin filament|one of the two filaments that form the cytoskeleton of striated muscle; composed of actin.|\\n|thiol group|—SH group.|\\n|threshold (voltage)|plasma membrane transmembrane voltage at which enough calcium or sodium channels open to initiate an action potential.|\\n|thylakoid|folded inner membrane of the chloroplast; site of the light reaction of photosynthesis. Thylakoids are arranged in stacks called grana.|\\n|thymine|one of the four bases found in DNA; thymine is a pyrimidine.|\\n|tight junctions|type of cell junction in which a tight seal is formed between adjacent cells occluding the extracellular space.|\\n|tissue|group of cells having a common function.|\\n|topoisomerases|enzymes that cut and rejoin DNA strands. Topoisomerase I relieves simple torsional stress by cutting one strand, allowing rotation about the phosphodiester link of the other strand, then rejoining the cut strand. Topoisomerase II cuts both strands of a double helix and passes another complete double helix through the gap, keeping hold of the ends.|\\n---\\n# GLOSSARY\\n\\nand rejoining them when the other strand has passed through. Topoisomerases are essential during DNA replication.\\n\\ntoxin: poison.\\n\\ntranscription: synthesis of an RNA molecule from a DNA template.\\n\\ntranscription bubble: structure formed when two strands of DNA separate and one acts as the template for synthesis of an RNA molecule.\\n\\ntranscription complex: A complex of RNA polymerase and various transcription factors.\\n\\ntranscription factor: protein (other than RNA polymerase) that is required for gene transcription.\\n\\ntrans: side from which material is removed. Of the Golgi apparatus, the surface from which vesicles bud to pass to the plasma membrane and to lysosomes.\\n\\ntransfected, transfection: cell, prokaryotic or eukaryotic, that has been infected by a foreign DNA molecule(s) is said to be transfected. The process of DNA infection is called transfection.\\n\\ntransfer RNA (tRNA): RNA molecule that carries an amino acid to an mRNA template.\\n\\ntransform, transformation: in addition to its common English meaning, this is used in molecular genetics to mean introduction of foreign DNA into a cell.\\n\\ntransgenic animal: animal carrying a gene from another organism; the foreign gene is usually injected into the nucleus of a fertilized egg.\\n\\ntrans-Golgi network: complex network of tubes and sheets that comprise the trans face of the Golgi apparatus. It is in the trans-Golgi network that proteins made on the rough endoplasmic reticulum are sorted as to their final destination.\\n\\ntranslation: synthesis of a protein molecule from an mRNA template.\\n\\ntranslocation: movement. When used of the ribosome, it means the movement, three nucleotides at a time, of the ribosome on the mRNA molecule.\\n\\ntransmembrane proteins: class of proteins that span the plasma membrane.\\n\\ntransmembrane translocation: form of protein transport in which unfolded polypeptide chains are threaded across one or more membranes as a simple polypeptide chain and then (re)folded at their final destination.\\n\\ntransmembrane voltage: voltage difference between one side of a membrane and the other. It is usually stated as the voltage inside with respect to outside.\\n\\ntransmission electron microscope: type of microscope in which the image is formed by electrons that are transmitted through the specimen.\\n\\ntransmitter: chemical that is released by one cell and that changes the behavior of another cell.\\n\\ntranspiration: loss of water from plant leaves.\\n\\ntransport vesicle: membrane vesicle that transports proteins from one membrane compartment to another.\\n---\\n# GLOSSARY\\n\\n|triacylglycerol (triglyceride):|three acyl (i.e., fatty acid) groups attached to a glycerol backbone by ester bonds. If the compound is liquid at room temperature, it is called an oil; if it is solid, it is called a fat.|\\n|---|---|\\n|tricarboxylic acid cycle (Krebs cycle):|series of reactions in the mitochondrial matrix in which acetate is completely oxidized to CO2 with the attendant reduction of NAD+ to NADH and FAD to FADH2.|\\n|triglyceride (triacylglycerol):|three acyl (i.e., fatty acid) groups attached to a glycerol backbone by ester bonds. If the compound is liquid at room temperature, it is called an oil; if it is solid, it is called a fat.|\\n|trimeric:|formed of three subunits.|\\n|trimeric G protein:|protein with three subunits, α, β and γ , where the α subunit is a GTPase that dissociates from the βγ units when it is in its GTP bound state. Both the α subunit, in its GTP bound state, and the now independent βγ subunit, can activate target proteins. Examples are Gs, which activates adenylate cyclase, and Gq, which activates phospholipase Cβ.|\\n|trimethylamine:|substance that gives rotting fish its characteristic smell. A base; when dissolved in water, trimethylamine will accept an H+. Produced by the action of intestinal bacteria on trimethylamine N-oxide (from dietary fish), choline (from dietary phospholipid), or other chemicals containing a trimethylamine group. Trimethylamine is oxidized in mammalian liver by the action of flavin-containing monooxygenase. A mutation in the FMO3 gene causes the disorder trimethylaminuria. Affected individuals excrete large amounts of trimethylamine in their urine, sweat, and breath and consequently have a fishy body odor. The structures of trimethylamine and protonated trimethylamine are CH3CH3, CH3NCH3CH3, CH3N+CH3H.|\\n|trimethylamine N-oxide:|The compound CH3CHN3CH3O. The arrow indicates a so-called dipolar bond between nitrogen and oxygen. This bond can also be represented as N+—O−. Trimethylamine N-oxide is generated from trimethylamine by the action of flavin-containing monooxygenase. A mutation in the FMO3 gene causes the disorder trimethylaminuria. Affected individuals excrete large amounts of trimethylamine in their urine, sweat, and breath and consequently have a fishy body odor.|\\n|trisphosphate:|having three phosphate groups attached at three different points on the molecule. Inositol trisphosphate and phosphatidylinositol trisphosphate are examples.|\\n|Trk:|a family of receptor tyrosine kinases that bind growth factors of the neurotrophin class.|\\n---\\n# GLOSSARY\\n\\n|tRNA (transfer RNA)|RNA molecule that carries an amino acid to an mRNA template.|\\n|---|---|\\n|troponin|calcium-binding protein found in muscle cells.|\\n|tryptophan (trp) operon|cluster of five bacterial genes involved in the synthesis of the amino acid tryptophan.|\\n|tubulin|protein that forms microtubules; exists as α, β, and γ subunits.|\\n|tumour|proliferative cell mass associated with many cancers.|\\n|turnover number|number of moles of substrate converted to product per mole of enzyme per unit time. Another term for catalytic rate constant, k cat.|\\n|tyrosine kinase|enzyme that phosphorylates on tyrosine residues in proteins by transferring a phosphate group from ATP. In this book all the tyrosine kinases described are receptor tyrosine kinases.|\\n|U (uracil)|one of the four bases found in RNA; uracil is a pyrimidine.|\\n|ultramicrotome|machine for cutting thin sections (<100 nm) for electron microscopy.|\\n|ultrastructure|fine structure of the cell and its organelles revealed by electron microscopy.|\\n|uncouples, uncoupler, uncoupled (of mitochondria)|mitochondria are uncoupled when the tight link between the respiratory chain and ATP synthesis is broken. Since the link is the gradient of H+ across the inner mitochondrial membrane, any chemical that allows H+ ions to cross the inner mitochondrial membrane is an uncoupler. If the chemical is present in more than a small fraction of the cells of the body, it will kill, but expression of the protein uncoupler thermogenin in brown fat allows that tissue to generate heat.|\\n|unsaturated (of fatty acids)|containing carbon–carbon double bonds.|\\n|untranslated sequence|sequence of bases in an mRNA molecule that does not code for protein. Untranslated regions are found at the 5′ and 3′ ends of an mRNA.|\\n|upstream|general term meaning the direction from which things have come. When applied to the DNA within and adjacent to a gene, it means lying on the side of the transcription start site that is not transcribed into RNA. When applied to signaling pathways, it means opposite to the direction in which the signal travels; thus the insulin receptor is upstream of protein kinase B.|\\n|uracil|one of the four bases found in RNA; uracil is a pyrimidine.|\\n|uracil–DNA glycosidase|DNA repair enzyme that recognizes and removes uracil from DNA molecules.|\\n|urea|H2N—(CO)—NH2, compound made in the liver that contains two nitrogen atoms but is less toxic than ammonia. Urea is chaotropic: at high concentration it reversibly denatures proteins.|\\n|urea cycle|metabolic cycle in the liver that makes urea that is then excreted in the urine.|\\n|vacuole|large membrane-bound compartment. Plant cells often contain a large vacuole filled with sugars and pigments.|\\n---\\n# GLOSSARY\\n\\n|Valium|antianxiety drug that increases the chance that the GABA receptor channel will open, allowing chloride ions to pass.|\\n|---|---|\\n|van der Waals force|weak close-range attraction between atoms.|\\n|vascular tissue|blood vessels. The term is also used to describe the water transporting and support tissue of plants that is composed of xylem and phloem.|\\n|vasoconstrictor|substance that constricts blood vessels.|\\n|vasodilator|substance that dilates (widens) blood vessels.|\\n|vector|Something that carries something else. The term is often used to describe a plasmid or bacteriophage that carries a foreign DNA molecule and is capable of independent replication within a bacterial cell.|\\n|vesicle|small closed bags made of membrane.|\\n|vesicular trafficking|precisely controlled movement of vesicles between different organelles and/or the plasma membrane.|\\n|villin|type of actin-binding protein that crosslinks actin filaments.|\\n|vilus (plural villi)|fingerlike extension of an epithelial surface that increases the surface area.|\\n|vimentin|protein that makes up the intermediate filaments in cells of mesenchymal origin such as fibroblasts.|\\n|virus|packaged fragment of DNA or RNA that uses the synthetic machinery of a host cell to replicate its component parts.|\\n|Vm|maximal velocity of an enzyme-catalyzed reaction: the limiting initial velocity obtained as the substrate concentration approaches infinity. Vm is also often used to mean transmembrane voltage.|\\n|VNTRs (variable number tandem repeats)|DNA sequences that occur many times within the human genome. Each person carries a different number of these repeats.|\\n|voltage clamp|technique in which the experimenter passes current to one side of a membrane to artificially set the value of the transmembrane voltage to a desired level.|\\n|voltage-gated calcium channel|channel that is selective for calcium ions and that opens upon depolarization. Found in the plasma membrane of many cells.|\\n|voltage-gated sodium channel|channel that is selective for sodium ions and that opens upon depolarization. Found in the plasma membrane of nerve and muscle cells.|\\n|Wee1|protein kinase that phosphorylates and hence inactivates CDK1.|\\n|wobble (in tRNA binding)|flexibility in the base pairing between the 5\\' position of the anticodon and the 3\\' position of the codon.|\\n|xeroderma pigmentosum|inherited human disease caused by defective DNA repair enzymes; affected individuals are sensitive to ultraviolet light and contract skin cancer when exposed to sunlight.|\\n|xylem|part of the plant vascular system; transports water from the roots to the leaves.|\\n---\\n# GLOSSARY\\n\\n|YAC (yeast artificial chromosome):|cloning vector used to propagate DNAs, of about 500,000 bp, in yeast cells.|\\n|---|---|\\n|Z disc:|disc that is set within and at right angles to the actin microfilaments in striated muscle, holding them in a regularly spaced array.|\\n|Z-DNA:|left-handed helical form of DNA.|\\n|Zellweger’s syndrome:|inherited human disease resulting from aberrant targeting of proteins to the peroxisome.|\\n|zinc fingers:|structural motif in some families of DNA binding proteins in which a zinc ion coordinated by cysteines and/or histidines stabilizes protruding regions that touch the edges of the base pairs exposed in the major groove of DNA.|\\n---\\n# INDEX\\n\\nUnderlined page numbers indicate a glossary entry, often a good place to begin reading about a topic.\\n\\n|-10 box, 108|Acetylcholinesterase, 442|\\n|---|---|\\n|-35 box, 108|at synapse, 368|\\n|1,3-bisphosphoglycerate, 286|inhibition, 376|\\n|2-oxoglutarate|specificity constant, 247|\\n|in Krebs cycle, 284| |\\n|in nitrogen excretion, 292| |\\n|2-phosphoglycerate, 286| |\\n|3-dimensional structure of proteins, 192–208| |\\n|specification by primary structure, 206| |\\n|3-hydroxybutyrate, 290| |\\n|3-phosphoglycerate, 286, 302| |\\n|7-methyl guanosine cap, 118, 176, 441| |\\n|9 + 2 axoneme, 388, 441| |\\n|14-3-3 protein, 358, 417, 441| |\\n|30S subunit, of prokaryote ribosome, 168–175| |\\n|40S subunit, of eukaryote ribosome, 168, 176| |\\n|50S subunit, of prokaryote ribosome, 168–175| |\\n|60S subunit, of eukaryote ribosome, 168, 176| |\\n|70S initiation complex, 171, 176| |\\n|80S initiation complex, 176| |\\n|α-adrenergic receptor, 365, 368, 441| |\\n|α-amino acid, 37–39, 184, 441| |\\n|α-blocker, 365| |\\n|α form, of monosaccharide, 27–28| |\\n|α-globin gene family cluster, 100| |\\n|α helix, 194–196, 441| |\\n|A site, 168, 441| |\\n|Acanthamoeba, 386| |\\n|Acceptor atom in hydrogen bond, 26, 442| |\\n|Acetate, 39| |\\n|breakdown product of acetylcholine, 368| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n|Acetylcholine, 364, 442| |\\n|at motoneurone:muscle synapse, 367–368| |\\n|in control of blood supply, 370| |\\n|Acetic acid, 39| |\\n|Acetoacetate, 290| |\\n|Acetoacetyl-ACP, 300| |\\n|Acetone, on breath, 299, 442| |\\n\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams, Elizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann ISBN 0-471-26393-1 Copyright ©C 2004 by John Wiley & Sons, Inc.\\n\\n501\\n---\\n# INDEX\\n\\n|Adenosine diphosphate (ADP)|258–261, 263, 269, 272–273, 443|\\n|---|---|\\n|ADP/ATP exchanger|271|\\n|ADP-ribosylation|173|\\n|receptor|344|\\n|sign of cell damage|344|\\n|Adenosine monophosphate (AMP)|259, 277, 443|\\n|cAMP|56, 114, 200, 305, 350–353, 358|\\n|regulation of phosphofructokinase|251, 304|\\n|Adenosine triphosphate (ATP)|37, 259, 277, 443|\\n|anaerobic generation by glycolysis|286–288|\\n|ATP/ADP exchanger|271, 440|\\n|ATPase, calcium|318, 350, 370, 440|\\n|ATPase, sodium/potassium|270, 271, 311, 312, 318, 320, 440|\\n|coenzyme/energy source|90, 221, 270, 282, 290, 293, 295, 298, 302, 318, 388, 394|\\n|DNA helix unwinding|90|\\n|feedforward control of ATP production|304–306|\\n|necrosis|415|\\n|phosphate source for kinase|190, 251, 258, 282, 284, 285, 293, 318, 412|\\n|radioactive|139|\\n|regulation of phosphofructokinase|251, 304|\\n|RNA synthesis|105|\\n|substrate for adenylate cyclase|350|\\n|synthase|269, 272–273, 274, 275|\\n|Adenylate cyclase|350, 352, 353, 443|\\n|Adherens junction|396, 443|\\n|Adipose tissue and cells|13, 300, 443|\\n|ADP|258–261, 263, 269, 272–273, 443|\\n|ADP/ATP exchanger|271, 443|\\n|ADP-ribosylation|173|\\n|receptor|344|\\n|sign of cell damage|344|\\n|Adrenal gland|60, 121, 305, 368|\\n|Adrenaline|353, 368–371, 443|\\n|action on blood vessels|368|\\n|acts mainly at β receptor|365|\\n|promotion of glycogen breakdown|305|\\n|Adrenergic receptor|365, 443|\\n|α adrenergic receptor|365, 368|\\n|β adrenergic receptor|305, 368|\\n|Adrenoleukodystrophy|60|\\n|Adverse drug reactions|158|\\n|Aerobic|277|\\n|Aflatoxin|251|\\n|Agarose gel|146, 148|\\n|AGT (alanine glyoxylate aminotransferase)|223, 250, 292|\\n|Akt (protein kinase B)|356–358, 417, 443|\\n|Alanine|185|\\n|aminotransferase (AGT)|223, 250, 292|\\n|conversion to pyruvate|250|\\n\\n# Algae\\n\\nblue-green algae, 12\\n\\n# Alkaline\\n\\nalkaline phosphatase, in DNA cloning, 139\\n\\n# Allolactose\\n\\n113\\n\\n# All or nothing feature\\n\\nof action potential, 328, 444\\n\\n# Allosteric\\n\\nenzymes, 251\\n\\n# Alpha-adrenergic receptor\\n\\n365, 368\\n\\n# Alpha-amino acid\\n\\n37–39, 184\\n\\n# Alpha-blocker\\n\\n365\\n\\n# Alpha form\\n\\nof monosaccharide, 27–28\\n\\n# Alpha-globin gene family cluster\\n\\n100\\n\\n# Alpha helix\\n\\n194–196\\n\\n# Alternative splicing\\n\\n68, 118, 444\\n\\n# Alu repeat\\n\\n101\\n\\n# Amanita muscaria\\n\\n370, 376\\n\\n# Amanita phalloides\\n\\n376\\n\\n# Ambiguity\\n\\nin codes, 79\\n\\n# Amino acids\\n\\n37–39, 46, 75–76, 444\\n\\n# Aminoacyl site (A site)\\n\\n168, 444\\n\\n# Aminoacyl tRNA\\n\\n165, 171, 173, 178, 444\\n\\n# Amino terminal\\n\\n184, 444\\n\\n# Aminotransferases\\n\\n249\\n\\n# Amoeba\\n\\n1, 386, 395\\n\\n# Amorphous ground substance\\n\\n13\\n---\\n# INDEX\\n\\n|AMP|259, 277, 444|\\n|---|---|\\n| |in histones, 73|\\n| |and regulation of phosphofructokinase, 251, 304|\\n| |NO synthesis, 365|\\n| |SH2 domain, 355|\\n| |synthesis, 293, 302|\\n|Amphipathic|39, 444|\\n|Amplitude modulation (AM)|336|\\n|Anabolism|281, 444|\\n|Anaerobic|275, 350, 444|\\n| |direction of energy flow, 275|\\n| |glycolysis, 286–288|\\n| |mode of life (anaerobes), 288|\\n|Anaesthetic, local|334|\\n|Anaphase|403–404, 444|\\n|Anchoring cell junction|54, 56, 396, 445|\\n|Anemia, sickle cell|81, 82|\\n|Angina|370|\\n|Animal cell, summary|10|\\n|Anion|21, 445|\\n|Anode|166, 445|\\n|Antenna chlorophylls|274, 445|\\n|Anterograde transport|390|\\n|Anthocyanin|11|\\n|Anthrax|89, 445|\\n|Antianxiety drugs|375|\\n|Antibiotics|445|\\n| |blocking ATP synthase, 269|\\n| |blocking protein synthesis, 176|\\n| |blocking topoisomerase I, 89|\\n| |Cipro, 89|\\n| |contain D amino acids, 196|\\n| |in cystic fibrosis, 423|\\n| |in DNA cloning, 130, 134|\\n| |oligomycin, 269|\\n| |puromycin, 176|\\n| |resistance, 74, 130, 131, 157|\\n|Antibodies|7, 139, 445|\\n| |manufacture in bacteria, 149|\\n|Anticodon|164–165, 178, 445|\\n|Antiparallel|β sheet, 195, 445|\\n| |DNA structure, 69|\\n|Antisense RNA|147–148|\\n| |probe, 149|\\n|Anxiety|375|\\n|AP1 transcription factor|124, 200|\\n|AP endonuclease|95, 101, 445|\\n|Apoptosis|315, 412, 415–418, 420, 445|\\n| |inhibitor proteins, 418, 445|\\n| |p53 and DNA damage, 417|\\n| |pathways controlling, 416|\\n| |stress-activated, 417–418|\\n|Aporepressor|117, 445|\\n|Aqueous liquid|21|\\n|ARF GTPase|228, 445|\\n|Arginine|185|\\n| |essential in infants, 302|\\n| |genetic code, 78|\\n|β adrenergic receptor|305, 368, 446|\\n|β barrel structure, of proteins|200|\\n---\\n# INDEX\\n\\n|β blocker|365|\\n|---|---|\\n|Binding, binding site|447|\\n|β form, of monosaccharide|27–28|\\n|β-galactosides and galactosidase|109–111|\\n|in pBluescript vector|132|\\n|β-galactoside permease|111, 440|\\n|β oxidation of fatty acid|290, 446|\\n|β pleated sheet|195|\\n|β sheet|195, 446|\\n|B-form, of DNA|71, 446|\\n|Bacillus subtilis|97|\\n|Back reaction|243|\\n|Bacteria, bacterial|9|\\n|artificial chromosome (BAC)|140|\\n|cytoskeleton hijacking by|393|\\n|flagellum|388–389|\\n|introducing foreign DNA molecules into|130–133|\\n|introducing recombinant plasmids into|134|\\n|motility of|264|\\n|producing mammalian proteins in|149|\\n|protein synthesis in|168–175, 178|\\n|RNA synthesis|106–118|\\n|Bacteriophage|12, 74, 97, 133, 140, 447|\\n|BAD protein|358, 417, 447|\\n|Basal body|387, 447|\\n|BAM H1|133|\\n|Base|25, 35, 447|\\n|amino acid side chain|185|\\n|as opposed to acid|25|\\n|excision repair|94–95, 447|\\n|in nucleic acids|35, 94|\\n|mutation|94|\\n|pair|27, 67, 69, 447|\\n|Basement membrane|13, 447|\\n|BAX protein|417–418, 447|\\n|Bcl-2|418, 447|\\n|associated death promoter (BAD)|417|\\n|family proteins|417–418, 447|\\n|B-DNA|71, 447|\\n|Bear, Christine|428|\\n|Berg, Howard|272|\\n|Berserk|376|\\n|Beta-adrenergic receptor|305, 368|\\n|Beta barrel structure, of proteins|200|\\n|Beta form, of monosaccharide|27–28|\\n|Beta-galactosides and galactosidase|109–111, 447|\\n|in pBluescript vector|132|\\n|Beta-galactoside permease|111, 440|\\n|Beta oxidation of fatty acid|290|\\n|Beta pleated sheet|195|\\n|Beta sheet|195|\\n|Bicycle tire|53|\\n|Bilayer|42, 52|\\n|bilayer voltage clamp|428|\\n\\n# Biochemical signaling\\n\\n353–359\\n\\n# Biochemistry\\n\\n19–46, 237–306\\n\\n# Bioluminescence\\n\\n116, 447\\n\\n# Bisphosphoglycerate\\n\\n286\\n\\n# Biochemical signaling\\n\\n353–359\\n\\n# Biochemistry\\n\\n19–46, 237–306\\n\\n# Bioluminescence\\n\\n116, 447\\n\\n# Bisphosphoglycerate\\n\\n286\\n\\n# Blobel, Gunter\\n\\n60\\n\\n# Blood\\n\\n53\\n\\nammonium\\n\\n293\\n\\ncells\\n\\n5, 81, 171, 287, 311, 416, 432\\n\\nclotting\\n\\n149, 344–348\\n\\nenzyme concentrations in\\n\\n248\\n\\nflow regulation\\n\\n368–370\\n\\nfuels for cells\\n\\n288, 290, 299, 317\\n\\nhemoglobin\\n\\n204–205, 238–240, 252\\n\\nhormones\\n\\n363\\n\\nleukemia\\n\\n154, 432\\n\\nparasites\\n\\n288\\n\\nplatelets\\n\\n149, 344–348\\n\\nscurvy\\n\\n190\\n\\nsickle cell anaemia\\n\\n81\\n\\ntest for cystic fibrosis\\n\\n425\\n\\ntest for phenylketonuria\\n\\n295\\n\\nvessel structure and growth\\n\\n190, 368–372\\n\\n# Bloom’s syndrome\\n\\n98, 448\\n\\n# Blotting techniques\\n\\n149\\n\\n# Blue eyes\\n\\n406\\n\\n# Blue-green algae\\n\\n12, 448\\n\\n# Blunt ends\\n\\n134, 448\\n\\n# Bond\\n\\ncovalent\\n\\n19\\n\\ndisulfide\\n\\n191, 192, 199\\n\\ndouble bonds in acyl groups\\n\\n42, 300\\n\\nelectrostatic\\n\\n190, 199\\n\\nester\\n\\n39\\n\\nglycosidic\\n\\n28–31\\n\\nhydrogen\\n\\n25–27, 69, 88, 90, 108, 109, 163, 164, 195\\n\\nhydrolysed by phospholipases\\n\\n346\\n\\npeptide\\n\\n38, 168, 173, 184\\n\\ntriple, in nitrogen\\n\\n300\\n\\n# Borage\\n\\n42\\n\\n# BoTox\\n\\n232\\n\\n# Botulism, botulinum toxin\\n\\n232, 288\\n\\n# -10 Box\\n\\n108\\n\\n# -35 Box\\n\\n108\\n\\n# β oxidation\\n\\n290–291, 448\\n\\n# Boyer, Paul\\n\\n273\\n\\n# Breast cancer\\n\\n154, 355\\n\\n# Breathing, of proteins\\n\\n238\\n\\n# Brenner, Sydney\\n\\n76\\n\\n# Bright-field microscopy\\n\\n3–5, 448\\n---\\n# INDEX\\n\\n|Brown|Calreticulin, 216|\\n|---|---|\\n|eyes|406|\\n|fat|270, 448|\\n|β sheet|195, 448|\\n|Bubble| |\\n|boy disease|432|\\n|transcription|108|\\n|Bud formation|229|\\n|Budding yeast|68, 401|\\n|Bulky lesion|94, 448|\\n|C terminal|173, 185|\\n|Cadherin|396, 448|\\n|Caenorhabditis elegans|68, 99, 418–419|\\n|Calcineurin|222–223, 350, 358, 448|\\n|Calcium| |\\n|action potential|325–330, 448|\\n|activation of NO synthesis|370|\\n|ATPase|318, 370, 448|\\n|binding protein|200, 216, 448|\\n|binding sites|190, 318|\\n|calcineurin|222–223, 358|\\n|cell locomotion and mobility|389, 394, 395|\\n|channel|IP 3 gated, 344, 438|\\n|channel|mitochondrial, 305, 348, 438|\\n|channel|ryanodine receptor, 348, 438|\\n|channel|voltage gated, 327, 389, 438|\\n|chloride|in DNA cloning, 134|\\n|concentration|310|\\n|control of glycogen breakdown|305|\\n|control of mitochondria|305, 348|\\n|endoplasmic reticulum|60, 216, 344, 348–349|\\n|gelsolin|395|\\n|indicator dye|343|\\n|influx from extracellular medium|341–343|\\n|intracellular messenger|341–350, 356, 368–370|\\n|kinases|calcium activated, 305, 358|\\n|muscle cells|304–306, 348–350, 394|\\n|nerve cells|341–344|\\n|Paramecium|swimming control, 389|\\n|phosphatase|calcium-activated, 222–223, 358|\\n|release from endoplasmic reticulum|344–348|\\n|removal from cytosol|350|\\n|sea urchin eggs|327–330|\\n|stimulation of gene transcription|222, 350, 358|\\n|trigger for exocytosis|230, 343, 348|\\n|Calf muscle|365|\\n|Calming|by Valium, 375|\\n|Calmodulin| |\\n|calcineurin activation|222–223, 358|\\n|regulator of glycogen phosphorylase kinase|305|\\n|structure|200|\\n|study by stopped flow|253|\\n---\\n# INDEX\\n\\n|Catenin|396|\\n|---|---|\\n|Chain initiation in protein synthesis| |\\n|Cation|21, 450|\\n|Cdc25 enzyme|412, 419, 450|\\n|Cdk1 enzyme|251–252, 411–412, 450|\\n|Cdk2 enzyme|413, 450|\\n|Cdk4 enzyme|413, 450|\\n|Cdk6 enzyme|413, 450|\\n|cDNA|130, 450|\\n|Cell|1, 450|\\n|Cell surface membrane (plasma membrane)|51|\\n|Cella|1|\\n|Cellulose|30–31, 53, 382, 451|\\n|Central dogma|39, 451|\\n|Central transporter|216|\\n|Centriole|382, 451|\\n|Centromere|100, 403, 451|\\n|Centrosome|11, 382, 383, 451|\\n|CF (cystic fibrosis)|423–432, 451|\\n|CF gene, CFTR|425–427|\\n|cGMP (cyclic guanosine monophosphate)|353, 451|\\n\\n# Chain initiation in protein synthesis\\n\\nin eukaryotes, 175–176\\n\\nin prokaryotes, 169\\n\\n# Chain release factors\\n\\n174, 451\\n\\n# Channel\\n\\n314, 437–439, 451\\n\\ncalcium, IP 3 gated, 344, 438\\n\\ncalcium, mitochondrial, 305, 348, 438\\n\\ncalcium, ryanodine receptor, 348, 438\\n\\ncalcium, voltage gated, 327, 389, 438\\n\\ncAMP gated, 200, 350, 439\\n\\nCFTR (cystic fibrosis...), 426–428, 439\\n\\ncGMP gated, 353, 439\\n\\nchloride, 373–375, 426–428, 439\\n\\ncurrents through, measuring, 320\\n\\ngap junction, 55–56, 314, 437\\n\\nglutamate receptors, 372–374, 439\\n\\nhydrogen ion channels, in ATP synthase, 272\\n\\nmitochondrial calcium, 305, 348, 438\\n\\nmitochondrial protein import, 221\\n\\nnicotinic acetylcholine receptor, 364, 439\\n\\nporin, 262, 266, 314, 315, 417–418, 437\\n\\npotassium, 310–312, 437\\n\\nprotein translocator, 223\\n\\nryanodine receptor, 348, 438\\n\\nsodium, 200, 330–336, 408, 438\\n\\nsolute-gated, 344, 350, 364, 438–439\\n\\nthermogenin, 270\\n\\n# Chaotropic reagents\\n\\n206, 452\\n\\n# Chaperones\\n\\n207, 221, 452\\n\\n# Chaperonin\\n\\n221\\n\\n# Chargaff, Erwin\\n\\n71\\n\\n# Charge\\n\\n21, 452\\n\\non amino acid side chain, 190\\n\\n# Charged tRNA\\n\\n165, 452\\n\\n# Chemical bond\\n\\n19–21\\n\\n# Chemistry\\n\\n19–46, 237–306\\n\\n# Chiasma, chiasmata\\n\\n408, 409, 452\\n\\n# Chilies\\n\\n39\\n\\n# Chimeric proteins\\n\\n150, 452\\n\\n# Chiral structure, of amino acids\\n\\n196, 452\\n\\n# Chitin\\n\\n33\\n\\n# Chloramphenicol\\n\\n176\\n\\n# Chloride\\n\\n21\\n\\ncalcium, in DNA cloning, 134\\n\\nchannel, 373–375, 426–428\\n\\n# Chlorine\\n\\n20\\n\\n# Chlorophyll\\n\\n274, 452\\n\\n# Chloroplast\\n\\n58, 452\\n\\nappearance, 58\\n\\ncarbon fixation, 302–303\\n\\ncytoplasmic streaming, 395\\n\\nDNA, 12\\n\\nenergy flux, 271–275\\n\\norigin as cyanobacteria, 12\\n---\\n# INDEX\\n\\n|Cholera|353|\\n|---|---|\\n|Choline|45|\\n|breakdown product of acetylcholine|368|\\n|in phospholipid structure|40, 82|\\n|fish odor syndrome|82–83|\\n|trimethylaminuria|82–83|\\n|Chromatid|73, 403, 406, 408, 452|\\n|Chromatin|57, 73, 403, 452|\\n|euchromatin|57, 73|\\n|heterochromatin|57, 73|\\n|Chromatophore|389, 452|\\n|Chromosome|58, 71, 452|\\n|changes at mitosis|403|\\n|counting|403|\\n|eukaryotic|71–73|\\n|human|76, 402|\\n|packaging of DNA molecules into|71–75|\\n|prokaryotic|73|\\n|walking|152, 452|\\n|Chronic fatigue syndrome|259|\\n|Cigarettes|13, 94, 250, 376, 386|\\n|Ciliates (cilated protists)|2, 386|\\n|Cilium|13, 386–389, 398, 452|\\n|CIP1|415|\\n|Cipro|89|\\n|cis cisternae|226|\\n|cis face of Golgi|226|\\n|Cisternae|60, 452|\\n|Citrate|284, 297|\\n|Citric acid cycle|283|\\n|Citrulline|191|\\n|Citrus fruit|190|\\n|CKIs (cyclin dependent kinase inhibitors)|415, 452|\\n|Classification of transmitters|363|\\n|Clathrin|228–229, 453|\\n|Claude, Albert|60|\\n|Cleavage furrow|404, 453|\\n|Cleavage sites, of restriction endonucleases|133|\\n|Clone|129, 453|\\n|library|134, 453|\\n|Cloning, DNA|129–130, 453|\\n|Cloning vectors|130–132, 453|\\n|joining foreign DNAs to|133–134|\\n|Closed promoter complex|108, 453|\\n|Clostridium botulinum|232|\\n|Clotting of blood|149, 344–348|\\n|Coatamer|228, 453|\\n|Coated vesicle|228–229, 453|\\n|Code, genetic|75–80|\\n|Coding, by action potential frequency|336–337|\\n|Codons|75–76, 453|\\n|start and stop|79–80|\\n|Coenzyme|259–261, 282, 453|\\n|coenzyme A|283–284, 286, 290–291, 297|\\n|coenzyme Q|265–268|\\n|Cofactors|249, 254, 453|\\n|Cyanide|271|\\n---\\n|INDEX|\\n|---|\\n|Cyanobacteria|12, 455|\\n|Cyclic adenosine monophosphate (cAMP)|350–353, 455|\\n| |control of glycogen breakdown, 305|\\n| |dependent protein kinase, 200, 305, 353, 358, 427|\\n| |gated channel, 200, 350|\\n| |gene expression control, 114, 200, 358|\\n| |intracellular messenger, 350–353|\\n| |muscle cells, 305|\\n| |oocyte, 56|\\n| |phosphodiesterase, 353|\\n|Cyclic guanosine monophosphate (cGMP)|353, 455|\\n| |dependent protein kinase, 370|\\n| |gated channel, 353|\\n| |phosphodiesterase, 370|\\n| |photoreceptors, role in, 353|\\n| |smooth muscle cells, role in, 370|\\n| |synthesis in response to NO, 365, 370|\\n|Cyclic photophosphorylation|275|\\n|Cyclin B|251–252, 411–412, 419, 455|\\n|Cyclin D|355, 413, 455|\\n|Cyclin-dependent kinase 1 (Cdk1)|411–412, 419, 455|\\n|Cyclin-dependent kinase inhibitor proteins (CKIs)|415, 419|\\n|Cyclin E|413, 455|\\n|Cyclosporin A|222–223|\\n|CYP (cytochrome P450)|124, 149, 158, 166–167, 249–251|\\n|Cysteine|185|\\n| |caspase, 415|\\n| |disulfide bond, 191, 199|\\n| |genetic code, 78|\\n| |iron binding, 265|\\n| |synthesis, 302|\\n| |zinc binding, 200|\\n|Cystic fibrosis|423–432, 455|\\n|Cystine|191, 455|\\n|Cytidine triphosphate (CTP) as energy currency|259|\\n| |dCTP, building block of DNA, 65–67|\\n|Cytochemistry|5, 455|\\n|Cytochrome|249, 265, 455|\\n| |c, 266, 268, 315, 417–418, 456|\\n| |c oxidase, 268|\\n| |electron transport chain, 265–268|\\n|Cytokinesis|404, 456|\\n|Cytology|2, 456|\\n|Cytoplasm|19, 456|\\n| |cytoplasmic dynein, 390, 456|\\n| |cytoplasmic streaming, 395, 456|\\n|Cytosine (C)|34, 35, 46, 69, 456|\\n| |base pair with guanine, 69|\\n---\\n# INDEX\\n\\n|sugar-phosphate backbone|69|\\n|---|---|\\n|synthesis|87–93, 130|\\n|testing|157–158|\\n\\nDeoxyribonucleoside triphosphates, 90\\n\\nDeoxyribonucleotides, 92, 456\\n\\nDeoxyribose, 31–33, 65, 456\\n\\nDeoxythymidine-5 -triphosphate (dTTP), 65–67\\n\\nDephosphorylation, 190–191\\n\\nof proteins, 190–191, 252, 318, 412\\n\\nof sugars, 295\\n\\nDepolarization, 325–327, 328–330, 456\\n\\nDeprotonation, 22–25\\n\\nDepurination, 94, 457\\n\\nDesigner babies, 158\\n\\nDesmin, 396, 457\\n\\nDesmosomes, 396, 457\\n\\nDetoxication, by cytochromes P450, 249\\n\\ndGTP (deoxyguanosine-5 -triphosphate), 65–67\\n\\nDiabetes, 299, 317, 457\\n\\nDiacylglycerol (DAG), 346, 457\\n\\nDideoxyadenosine triphosphate (ddATP), 143–145\\n\\nDideoxy chain termination method, 143–146\\n\\nDideoxycytidine triphosphate (ddCTP), 143–145\\n\\nDideoxyguanosine triphosphate (ddGTP), 143–145\\n\\nDideoxynucleotides, 143–145\\n\\nDideoxythymidine triphosphate (ddTTP), 143–145\\n\\nDiet\\n\\nessential amino acids, 302\\n\\nessential fatty acids, 42\\n\\nfish odor syndrome, 82–83\\n\\nglyoxylate and kidney stones, 223\\n\\nmetabolism of food chemicals, 44–46, 109, 283–295\\n\\nphenylalanine and phenylketonuria, 294–295\\n\\nniacin and pellagra, 259\\n\\nstarvation and ketone bodies, 290, 299\\n\\ntrimethylaminuria, 82–83\\n\\nvitamin C and scurvy, 190\\n\\nDifferentiation, 119, 457\\n\\nDiffusion, 457\\n\\nacross membranes, 53\\n\\nlimit to enzyme specificity constant, 247\\n\\nthrough gap junctions, 55–56\\n\\nthrough connective tissue, 13\\n\\nDigitalis, 271, 320\\n\\nDihydroxyacetone phosphate, 285–286, 295, 300\\n\\nDiphtheria bacterium, protein synthesis and, 173\\n\\nDiplococcus pneumoniae, transformation, 66\\n\\nDiploid, 402, 405, 457\\n\\nDipole, 199\\n\\nDisaccharide, 28, 457\\n\\nDi Sant’ Agnese, Paul, 424\\n\\nDrinking water, filtering, 10\\n\\nDissolve, 21, 42\\n\\nDistal axon terminal, 333, 336\\n\\nDisulfide bond (bridge), 185, 191, 199, 457\\n\\nSDS-PAGE and, 166\\n\\nDNA (deoxyribonucleic acid), 65, 457\\n\\namount, in cells, 67, 68, 71–73\\n\\nbases found in, 65–67\\n\\nblotting, 146\\n\\nclasses, 99–101\\n\\ncloning, 129–143\\n\\ndamage, 94, 415, 417–418\\n\\ndata storage medium, 87–103\\n\\ndouble helix, 68–69\\n\\nexcision enzymes, 94\\n\\nfingerprint, 146, 458\\n\\nforms of, 71\\n\\nhelicase, 90, 98\\n\\nintroduction into bacteria, 130–133\\n\\nligase, 90, 92, 97, 130, 134, 458\\n\\nmicrosatellite, 100, 146\\n\\nminisatellite, 100\\n\\nnotation, 106\\n\\npackaging, 71–73\\n\\npolymerase I, 92, 95, 97\\n\\npolymerase III, 90–92\\n\\npolymerase, thermostable, 152\\n\\nproof that it’s the genetic material, 66\\n\\npurification, 129\\n\\npurine and pyrimidine content of, 71\\n\\nrecombinant, 129–158\\n\\nrepair, 94–98\\n\\nrepetitious, 100\\n\\nreplication, 87–93, 458\\n\\nsatellite, 100, 403\\n\\nsequencing, 143–146, 458\\n\\nstaining for microscopy, 6–7\\n\\nstructure, 65–71\\n\\nsugar-phosphate backbone, 69\\n\\nsynthesis, 87–93, 130\\n\\ntesting, 157–158\\n\\nDnaA protein, 88–89, 457\\n\\nDnaB protein, 90, 458\\n\\nDnaC protein, 90, 458\\n\\nDocking protein, 223, 458\\n\\nDomain, 99, 200, 458\\n\\nDominant, 458\\n\\ngenes, 406\\n\\ngenetic diseases, 408\\n\\nDonor atom, 26, 458\\n\\nDouble bond in acyl group (fatty acids), 42–43, 300\\n\\nDouble helix structure, 68–69, 458\\n\\nopening during replication, 88–90\\n\\nDouble-membrane envelopes, 56–58, 61\\n\\nDown syndrome, 403\\n\\nDownstream sequences, 106, 458\\n---\\n# INDEX\\n\\n|Driving energetically unfavourable reactions, protein synthesis on|223–225|\\n|---|---|\\n| |258|\\n|Drosophila|68|\\n|dTTP (deoxythymidine-5 -triphosphate)|65–67|\\n|Duplication, of genes|100, 458|\\n|Dynamic instability|384, 459|\\n|Dynamic protein structures|238|\\n|Dynamin|229, 459|\\n|Dynein|118, 388, 398, 390, 459|\\n|E2F-1 transcription factor|413, 414, 459|\\n|Eco RI|133|\\n|Ectoplasm|395, 459|\\n|Endoplasm|395, 459|\\n|Effective stroke|386, 459|\\n|Effector caspases|416, 459|\\n|Egg dimensions|2|\\n| |oocytes and gap junctions, 56|\\n| |transmembrane voltage and sperm fusion, 325–327|\\n| |white lysozyme, 192|\\n|Electrical electrically excitable cells|325–338, 459|\\n| |force, on ions at a membrane, 311–314|\\n| |signalling, via gap junctions, 15, 55–56|\\n| |summation, 372–375|\\n| |transmission, down a nerve cell axon, 332–334|\\n| |voltage, 311–312|\\n|Electrochemical gradient|261, 459|\\n|Electron gun|8, 459|\\n|Electron microscope|2–3, 4, 8–9, 16, 459|\\n|Electrons, sharing|19–21|\\n|Electron transport chain|263, 265–268, 278, 459|\\n| |photosynthetic, 274|\\n|Electrophoresis|166, 177, 459|\\n|Electrostatic interactions|199, 459|\\n|Elodea|395|\\n|Elongation factors|173, 459|\\n| |of protein chain at ribosome, 171–174|\\n|Embedding for microscopy|8|\\n|Embryonic stem (ES) cells|157, 459|\\n|Endocrine gland|363|\\n|Endocytosis|229, 230, 459|\\n| |endocytotic pathway, 226|\\n| |endocytotic vesicle, 229, 417|\\n|Endonuclease|459|\\n| |AP endonuclease, 95|\\n| |restriction endonuclease, 133|\\n|Endoplasmic reticulum|57, 60, 61, 460|\\n| |calcium storage and release, 216, 344–348|\\n| |changes at mitosis, 410|\\n| |fat synthesis, 300|\\n| |glycosylation and, 225–226|\\n| |protein synthesis on, 223–225|\\n| |rough, 60, 223–225|\\n| |smooth, 60|\\n|ES cell|157, 461|\\n|ES complex|242, 245|\\n|Escherichia coli| |\\n| |chromosome size, 67|\\n| |Meselson–Stahl experiment and, 92–93|\\n| |replication in, 88–89|\\n| |RNA synthesis in, 106–109|\\n| |transcription termination in, 110|\\n|Essential amino acids|259, 302, 461|\\n|Essential fatty acids|42, 461|\\n---\\n# INDEX\\n\\n|Ester bond|39, 461|\\n|---|---|\\n|Esterase, specific for acetylcholine|368|\\n|Ethics| |\\n|of DNA testing for inherited disease|157–158|\\n|of gene therapy|432|\\n|Euchromatin|57, 461|\\n|Eukaryotes, eukaryotic|9, 10, 16, 461|\\n|chromosomes|71–73|\\n|differences from prokaryotic cells|11|\\n|DNA, major classes of|99–101|\\n|gene, complexity in|98–99|\\n|gene expression, control of|119–124|\\n|gene structure and organization in|98–101|\\n|mRNAs|125, 176|\\n|organization of|213|\\n|origin of|12|\\n|protein-coding genes|118|\\n|protein synthesis|175–178|\\n|ribosomes, S value of|168|\\n|RNA synthesis|118–119|\\n|Euprymna scolopes|116|\\n|Evening primrose oil|42|\\n|Evolution| |\\n|crossing over (chiasmata)|408|\\n|of cAMP gated channel|200|\\n|of eukaryotic cell|12|\\n|Exchange mechanisms for energy currencies|263–264|\\n|Excision repair|94–98|\\n|Exocytosis|229, 230, 343, 461|\\n|Exocytotic pathway|226|\\n|Exon|98, 101, 118, 461|\\n|Exonuclease|92, 461|\\n|Explosive, nitroglycerine|370|\\n|Export, from nucleus|219|\\n|Expression, gene|105, 461|\\n|Expression vector|139, 149, 461|\\n|Extracellular matrix|13, 53–54, 61, 461|\\n|Extracellular medium|2, 19, 461|\\n|calcium entry from|341–344|\\n|Extragenic DNA|100, 461|\\n|Eye color|406|\\n|Eyepiece|3|\\n|F-actin|390–391|\\n|σ factor|106, 108|\\n|Factor VIII|149, 157|\\n|FAD (flavin adenine dinucleotide)|267, 271, 284, 290|\\n|Fainting|370|\\n|Fallopian tube|56|\\n|Fas receptor|416, 461|\\n|Fat|39–43, 461|\\n|adipose tissue|13|\\n|brown fat|270|\\n|cell|39, 461|\\n|energy store|294|\\n|Frequency modulation (FM)|336|\\n---\\n# INDEX\\n\\n|Fructose|27, 29, 243, 303|\\n|---|---|\\n|6-phosphate and 1,6-bisphosphate|284–285, 295, 303|\\n|in gluconeogenesis|295|\\n|in glycolytic pathway|284–285|\\n|Fruit fly|68|\\n|Fumarate|267, 283–284, 292–293|\\n|Fusion, membrane|231|\\n|γ -amino butyrate (GABA)|37–38, 373–375|\\n|γ -linolenic acid|42–43|\\n|G (Gibbs free energy)|258, 276–277|\\n|of solute at concentration [I]|312|\\n|G0 (gap 0) phase|409, 463|\\n|G1 (gap 1) phase|409, 463|\\n|G1/S cell cycle control point|412–415|\\n|G2 (gap 2) phase|409, 463|\\n|G2/M cell cycle control point, molecular regulation of|410–412|\\n|G q|344–346|\\n|G s|350, 353|\\n|GABA (γ -amino butyrate)|37–38, 373–375, 463|\\n|Galactose|27, 29|\\n|lac operon|109–115|\\n|G-actin|390–391, 463|\\n|β-Galactosidase|109–113, 132|\\n|β-Galactoside|111–112, 115|\\n|Gamete|406, 408, 463|\\n|GAP (GTPase activating protein)|218, 463|\\n|Gap junction|55–56, 463|\\n|Gastrocnemius muscle|363, 365–372, 376, 463|\\n|Gated, gating, of channels|314, 463|\\n|Gated transport|213, 214, 463|\\n|through the nuclear pore|216–218|\\n|GC box|120|\\n|GDP (guanosine diphosphate)|173, 259, 463|\\n|GEF (guanine nucleotide exchange factor)|218–219, 228, 344, 350, 355, 464|\\n|GEFS (generalized epilepsy with febrile seizures)|331, 464|\\n|Gelsolin|395, 464|\\n|Gene|464|\\n|chip|154 , 464|\\n|detection|146–147|\\n|expression|105, 109–124|\\n|family|100, 464|\\n|knockout|157|\\n|nomenclature|101|\\n|notation|106|\\n|number|67–68|\\n|probe|143, 464|\\n|Glutamate|185|\\n|charge on side chain|185, 190|\\n|conversion to 2-oxoglutarate|292|\\n|genetic code|78|\\n|phosphorylation of side chain|190–191|\\n|mutation, in sickle cell anaemia|81|\\n---\\n# INDEX\\n\\n|nitrogen metabolism and urea cycle|292–293|\\n|---|---|\\n|receptor binding protein number 2 (Grb2)|355, 466|\\n|receptor|140–141, 372, 439|\\n|synthesis|302|\\n|toxic analogue|372|\\n|transmitter action|341–343, 372|\\n|Glutamine|185|\\n|genetic code and wobble|78, 80, 164|\\n|nitrogen fixation|302|\\n|synthesis|302|\\n|Glutaraldehyde|8|\\n|Glyceraldehyde 3-phosphate|285–286|\\n|Glycerides|39, 465|\\n|synthesis|300|\\n|Glycerol|39, 40, 300, 465|\\n|Glycine|185, 186, 188|\\n|Glycogen|30, 243, 465|\\n|breakdown|288–289, 305|\\n|glucose source|288–289|\\n|phosphorylase|288, 305, 465|\\n|phosphorylase kinase|242, 305, 465|\\n|synthesis|298–299|\\n|Glycolysis|251, 284–290, 306, 465|\\n|glycogen and|288|\\n|negative feedback control of|304|\\n|without oxygen|286–288|\\n|Glycosidic bond|28, 465|\\n|Glycosylation|53, 190, 225–226, 465|\\n|Glyoxylate|465|\\n|shunt|297–298, 465|\\n|GM (genetically modified)|158, 465|\\n|Goldberger, Joseph|259|\\n|Golgi apparatus|60–61, 215, 225–231, 357, 465|\\n|cell center|61|\\n|changes at mitosis|410|\\n|glycosylation|226|\\n|Golgi, Camillo|60|\\n|Gordian knot|74|\\n|Gout|385|\\n|G protein (trimeric)|247, 344–346, 350, 359, 465|\\n|G q|344–346, 465|\\n|Gramicidin A|196|\\n|Granum, grana|58, 465|\\n|Gratuitous inducer|114, 466|\\n|Graves, Robert|376|\\n|Grb2|355, 466|\\n|Green eyes|406|\\n|Green fluorescent protein|150, 466|\\n|β barrel structure|200|\\n|chimeric proteins|150|\\n|protein trafficking|226|\\n|Griffith, Fred|66|\\n|Ground substance|13|\\n|Growth factor(s)|466|\\n|apoptosis inhibition|416–417|\\n|cell division promotion|371, 413|\\n|mechanism and pathways|353–356, 358–359|\\n|Gyrase|89|\\n|H +|21–25|\\n|concentrations|261, 310|\\n|gradient|driving bacterial flagellum, 264|\\n|gradient|energy currency, 261–262|\\n|protonation|25, 185|\\n|transport by ATP synthase|269, 272–273|\\n|transport by electron transport chain|265–268|\\n|transport by thermogenin|270|\\n|H2O2|59–60|\\n|H3O+|22|\\n|Hair|183, 195|\\n|dryer and pain receptors|372|\\n|keratin|184, 195, 396|\\n|root hair|16|\\n|shampoo|167|\\n|Hairpin loop|in transcription termination, 109, 467|\\n---\\n|Index|\\n|---|\\n|Haploid cells, 405, 419, 467|Hormone, 363, 467|\\n|Hartridge, Hamilton, 252|response element (HRE), 122–123, 365, 467|\\n|Hartwell, Leland, 411|steroid, 53, 60, 121–124, 249, 365|\\n|Head group, 42, 467|Horns, 396|\\n|Heart, 14–15, 56, 290, 315, 320, 348, 370, 393, 396, 417|Horseradish peroxidase, 166|\\n|Housekeeping gene, 120, 468|HRE (hormone response element), 122–123, 365, 468|\\n|Hsp90, 122|Human Genome Project, 68|\\n|disease genes and, 154|DNA sequencing and, 144–146|\\n|Human immunodeficiency virus (HIV), 12|Human proteome, 178|\\n|Hunt, Tim, 411|Huntington’s disease, 157|\\n|Hybridization, 139, 468|Hydrated gel, 190|\\n|Hydration shell, 21, 468|Hydrocarbon tail, 39, 468|\\n|Hydrogen bond, 25–26, 195, 468|between bases in DNA, 26, 67–71|\\n|in α helix, 195|in β sheet, 195|\\n|in protein structure, 195|in water, 27|\\n|Hydrogen ion, 21–25|concentrations, 261, 310|\\n|gradient, driving bacterial flagellum, 264|gradient, energy currency, 261–262, 468|\\n|protonation, 25, 185|transport by ATP synthase, 269, 272–273|\\n|transport by electron transport chain, 265–268|transport by thermogenin, 270|\\n|See also pH|Hydrogen peroxide, 59–60|\\n|Hydrolysis, 44–46, 47, 468|Hydropathy plot, 204, 468|\\n|Hydrophilic, 42, 468|amino acid side chains, 80–81, 185, 186, 192, 204|\\n|action of cytochromes P450, 249–251|head group, of phospholipids, 42|\\n|solute, transport across membranes, 309–320|Hydrophobic, 39, 468|\\n|acyl groups, 39|amino acid side chains, 80–81, 185–186, 192, 199, 204|\\n|effect, 199|interaction, 199|\\n|lipid bilayer interior, 53, 309|signal sequence, 215, 216, 223–225|\\n|solvent, 39|tail, of ARF, 228|\\n|tail, of chlorophyll, 274|tail, of ubiquinone/coenzyme Q, 265|\\n---\\n# INDEX\\n\\n|tail, of fatty acid|39|\\n|---|---|\\n|receptor|356–357, 469|\\n|transmembrane helix|200, 204, 223–225|\\n|receptor substrate number 1 (IRS-1)|356, 469|\\n|Hydrostatic pressure|11, 54|\\n|3-hydroxybutyrate|290|\\n|Hydroxyl groups|27, 28, 468|\\n|in side chains|188|\\n|Hydroxylysine|190|\\n|Hydroxyproline|190|\\n|Hyperforin|124|\\n|Hypoxanthine|34, 35, 46, 164, 468|\\n|Imino acids|185–186, 468|\\n|Immune system|7, 222, 432|\\n|Immunofluorescence|7, 468|\\n|Immunoglobulin gene family|100|\\n|Immunosuppressants|222–223|\\n|Import| |\\n|ER|223–225|\\n|nuclear|218–219|\\n|mitochondrial|215|\\n|Inactivation|327, 330, 331, 468|\\n|Induced fit|249|\\n|Inducers, gratuitous|114|\\n|Inducible operon|111–115, 469|\\n|Inheritance|406–407|\\n|Inhibition| |\\n|acetylcholine receptor by atropine|376|\\n|acetylcholinesterase by Sarin etc.|376|\\n|contact|415|\\n|GABA at synapse|374|\\n|phosphofructokinase by ATP|251, 304|\\n|Initial rate|243|\\n|Initial velocity (v o)|242–244, 254, 469|\\n|effect of substrate concentration on|244–245|\\n|measurements of|252|\\n|Initiation (of)| |\\n|coatamer assembly, by ARF|228|\\n|complex, on DNA, transcriptional|120, 122|\\n|complex, 70S, protein synthesis|171, 176|\\n|complex, 80S, protein synthesis|176|\\n|factors|171|\\n|protein synthesis|168–171, 175–176|\\n|transcription|106–109, 120|\\n|INK4a|184, 415|\\n|Inosine|164|\\n|Inositol|469|\\n|Inositol trisphosphate (IP 3)|37, 344–348, 359, 469|\\n|gated calcium channel|344, 346–348, 469|\\n|Input-output relations at a synapse|372–375|\\n|Insecticide toxicity|376|\\n|Insertional mutagenesis|157, 469|\\n|In situ hybridization|147–148, 469|\\n|Insulation, of axons|334|\\n|Insulin|469|\\n|diabetes|299, 317|\\n|mechanism and pathways|317, 356–357|\\n|Isocitrate|284, 297|\\n|Isoelectric point|25, 178, 470|\\n|Isoforms|100, 470|\\n---\\n# INDEX\\n\\n|Isoleucine|185|\\n|---|---|\\n|essential amino acid|302|\\n|genetic code|78|\\n|inosine in tRNA|165|\\n|Isomers|27, 196, 470|\\n|Isopropylthio-β-D-galactoside (IPTG)|114, 139|\\n|Jaffe, Laurinda|325–327|\\n|Jellyfish|150|\\n|JNK protein kinase|418, 470|\\n|Junctions|54–56, 396|\\n|adherens|396|\\n|anchoring|396|\\n|desmosome|396|\\n|gap|15, 55–56, 314, 437|\\n|tight|55|\\n|Ka|24–25|\\n|kcat|242, 244, 245, 247|\\n|Keq|277|\\n|Kartegener’s syndrome|389, 471|\\n|KDEL|215, 216|\\n|Keratin|184, 195, 396, 471|\\n|Keratinocytes|396|\\n|Ketone bodies|290, 299, 471|\\n|Khorana, Gobind|77|\\n|Kidney stones|223|\\n|Kinase|190, 471|\\n|A|200, 305, 353, 368, 427|\\n|B|357, 358, 417|\\n|C|358|\\n|G|370|\\n|calcium-calmodulin-activated|358|\\n|cAMP-dependent|200, 305, 353, 368, 427|\\n|cGMP-dependent|370|\\n|cyclin dependent|411–415|\\n|glycerol|300|\\n|glycogen phosphorylase kinase|242, 305, 348, 353|\\n|hexokinase|277, 284|\\n|intrinsic to Ca++ ATPase|318|\\n|MAP|353–355, 371|\\n|phosphofructokinase|251, 285, 304|\\n|phosphoinositide 3-kinase|356, 358, 417|\\n|polynucleotide|139|\\n|protein|190, 241|\\n|protein kinase B|357, 358, 417|\\n|protein kinase C|358|\\n|receptor tyrosine|353–357, 365, 371, 417|\\n|stress activated|418|\\n|Kinesin|390, 398, 471|\\n|Kinetics|of chemical reaction, 242–248|\\n|saturation kinetics|245|\\n|Kinetochore|403, 471|\\n|Kinks|in fatty acid chains, 42–43|\\n|KIP1|415|\\n|Lactose|30, 471|\\n|intolerance|44, 471|\\n|lac operon|111–115, 471|\\n|lac Z gene|132|\\n|Lagging strand|90, 471|\\n|Lambda (bacteriophage)|140, 426|\\n|L Amino acids|196|\\n|Lamins|396, 471|\\n|Lead citrate|8|\\n|Leader sequence|169, 176|\\n|Leading strand|90, 471|\\n|Lens|3, 8, 183|\\n|Leucine|185|\\n|essential amino acid|302|\\n|genetic code|77, 78|\\n|Leukemia|154|\\n| |Library|\\n|cDNA|134|\\n|genomic DNA|140|\\n|in cloning CF gene|426|\\n|screening|136–139|\\n|Ligand|183, 471|\\n|allosteric effects|238, 251|\\n|binding site on protein|237|\\n|clathrin-mediated vesicle formation|229|\\n|rapid reaction techniques|252–253|\\n|regulation of protein activity|238, 251|\\n| |Light|\\n|absorption, in assays|243, 252–253|\\n|DNA, Meselson-Stahl experiment|92–93|\\n|emission, squid|116|\\n|fluorescence|6–7|\\n|polarized|196, 243|\\n|microscope|3–7|\\n|reactions of photosynthesis|274–275, 471|\\n|sensitive-nerve cells|353, 401|\\n|ultraviolet, DNA damage|94, 98, 415, 418|\\n|Lignin|54|\\n| |Limit|\\n|of resolution in microscopy|2–3|\\n|to enzyme-catalyzed reactions|245, 247|\\n|Lind, James|190|\\n|LINEs (long interspersed nuclear elements)|100|\\n|Lineweaver–Burk plot|246|\\n|Ling, Gilbert|320|\\n---\\n# INDEX\\n\\n|Link, linker|Lux operon, 116|\\n|---|---|\\n| |DNA, 73|\\n| |phosphodiester, 42, 67, 89, 90, 92, 95, 97, 105, 109, 118|\\n| |polylinker, 150|\\n| |inactivation plug, 327, 330, 331|\\n|Linkage|408, 472|\\n|Linoleic acid|42–43|\\n|Linolenic acid|42, 43|\\n|Lipid|39–43|\\n| |β oxidation, 290|\\n| |bilayer, 42, 52–53, 309, 472|\\n| |bilayer voltage clamp, 428|\\n| |diacylglycerol (DAG), 346|\\n| |fish odor syndrome, 82–83|\\n| |hydrolysis, 45|\\n| |liposomes, 229|\\n| |liposomal-mediated gene transfer, 429|\\n| |peroxisomes, 60|\\n| |phosphatidylinositides (PIP 2, PIP 3 etc), 231, 346, 357, 417|\\n| |phospholipid, 40–45|\\n| |synthesis, 300|\\n| |translocation to plasma membrane, 230|\\n| |trimethylaminuria, 82–83|\\n|Lipolysis|40|\\n|Listeria|393|\\n|Liver (cell)| |\\n| |alanine glyoxylate aminotransferase and kidney stones, 223|\\n| |detoxication (cytochromes P450), 60, 124, 149, 166–167|\\n| |failure, 293|\\n| |fish odor syndrome, 82–83|\\n| |glucose export, 289|\\n| |glycogen, 30, 288–289|\\n| |hepatitis C, 176|\\n| |identity and gene expression, 119|\\n| |ketone bodies, 290|\\n| |lactate oxidation, 287|\\n| |lipid synthesis, 300|\\n| |specialized biochemical center, 281|\\n| |trimethylaminuria, 82–83|\\n| |urea cycle, 292–293|\\n|Local anaesthetic|334|\\n|Localization| |\\n| |nuclear localization signal, 216, 218–219|\\n| |sequence, 215, 472|\\n|Long interspersed nuclear elements (LINEs)|100|\\n|Luciferase|116|\\n|Lung| |\\n| |diffusion of gases, 53|\\n| |disease in cystic fibrosis, 423, 428–431|\\n| |epithelium, 12–13|\\n| |hemoglobin oxygen loading, 238–240|\\n| |mucus clearance and cilia, 13, 386|\\n---\\n# INDEX\\n\\n|Membrane (Continued)|light, 3–7|\\n|---|---|\\n| |cracking, 42|\\n| |diffusion across, 53, 121, 365|\\n| |double-membrane envelopes, 56–58|\\n| |fish odor syndrome, 82–83|\\n| |fission and fusion, 226–231|\\n| |fluidity, 42|\\n| |integral membrane proteins, 52–53, 202, 223–225, 226, 230|\\n| |intermembrane space, 56–58, 261–262, 266|\\n| |nylon, in genetic engineering, 136, 139, 143, 146, 148–149|\\n| |nylon, in western blotting, 166–167|\\n| |plasma, 1, 51|\\n| |proteins, tertiary structures in, 200|\\n| |single-membrane envelopes, 58–61|\\n| |solidifying, 42|\\n| |structure, 39–43, 52|\\n| |transmembrane translocation, 213, 221–225|\\n| |trimethylaminuria, 82–83|\\n| |voltage, 261, 311–314, 320–321, 325–336, 372–375|\\n|Menten, Maud|245|\\n|Meselson, Matthew|92|\\n| |Meselson–Stahl experiment, 92–93|\\n|Messenger RNA|75–76, 105–120|\\n| |cDNA library, 130|\\n| |central dogma, 39|\\n| |northern blotting, 148–149|\\n| |polycistronic, 109|\\n| |processing after transcription, 68, 98–99, 118–119|\\n| |splicing, 68, 98–99, 118–119|\\n| |translation, 163–176|\\n|Messenger, intracellular|341–353, 368–370|\\n|Metabolism|281–308|\\n|Metabotropic cell surface receptors|365|\\n|Metalothionein gene|155|\\n|Metaphase|403|\\n|Methionine|185|\\n| |essential amino acid, 302|\\n| |formyl, 5, 169, 171|\\n| |genetic code, 78, 79|\\n| |start signal, 79, 168, 175–176|\\n| |subtilisin, engineering methionine out, 150|\\n|7-Methyl guanosine cap|118, 176|\\n|Michaelis, Leonor|245|\\n| |constant (KM), 244–247|\\n|Michaelis–Menten equation|245, 246|\\n|Microarray analysis|154|\\n|Microfilament|390–392, 398|\\n|Micropipette|320–321, 325–326|\\n|Microsatellite DNA|100, 146|\\n|Microscope, microscopy|2–9|\\n| |bright-field, 3–5|\\n| |electron, 2–3, 4, 8–9|\\n| |fluorescence, 6–7|\\n---\\n# INDEX\\n\\n|northern blotting|148–149|\\n|---|---|\\n|polycistronic|109|\\n|processing after transcription|68, 98–99, 118–119|\\n|splicing|68, 98–99, 118–119|\\n|translation|163–176|\\n\\nMTOC (microtubule organizing center), 382, 475\\n\\nMultiple cloning site (MCS), 132, 150\\n\\nMuscarine, 376\\n\\nMuscarinic acetylcholine receptor, 370, 475\\n\\nMuscle, 14, 475\\n\\n|activation|348–349, 367–368|\\n|---|---|\\n|anaerobic|275, 286–287, 350|\\n|anchoring junctions|396|\\n|blood supply|368–372|\\n|calcium effects|348–350, 394|\\n|calf|365–372|\\n|cAMP effects|353|\\n|cGMP effects|370|\\n|cardiac|14, 55–56|\\n|cramp|21, 286–287|\\n|desmin|396|\\n|digitalis|320|\\n|feedforward control|304–306|\\n|fiber|15|\\n|gap junctions|56|\\n|gastrocnemius|365–372|\\n|glucose carrier, translocation to plasma membrane|317, 357|\\n|glycogen storage and breakdown|30, 288, 304–306|\\n|intermediate filament|396|\\n|ketone bodies as fuel|290|\\n|mechanism of contraction|393–394|\\n|mitochondria|58|\\n|neurotrophin synthesis|417|\\n|paralysis (BoTox)|232|\\n|resting voltage|311|\\n|ryanodine receptors|348–349|\\n|skeletal|14, 365|\\n|smooth|14, 368|\\n|striated|14|\\n|Viagra|370|\\n\\nMushroom toxicity, 376\\n\\nMutagenesis\\n\\n|insertional|157|\\n|---|---|\\n|site-directed|149|\\n\\nMutation, 87, 475\\n\\n|detection by Southern blotting|146|\\n|---|---|\\n|FMO3 gene|82|\\n|frameshift|80, 82|\\n|gene families, generation of|100|\\n|missense|80–81, 82|\\n|nonsense|80, 82|\\n|pseudogenes, generation of|100|\\n---\\n# INDEX\\n\\n|Nerve, nerve cell (Continued)|Nicotine, 376|\\n|---|---|\\n|resting voltage, 311|Nicotinic acetylcholine receptor, 364, 368, 376, 439, 477|\\n|saltatory conduction, 336|Nightshade, 376|\\n|scent-sensitive, 350–353|Nirenberg, Marshall, 77|\\n|stroke, 417|Nitella, 395|\\n|synapse, 341–344, 367–368, 372–375|Nitric oxide, 53, 365, 370, 477|\\n|terminal, 332–334, 341–344, 367, 371, 372–375|receptor, 365|\\n|terminally differentiated, 409|Nitrogen, 25–26, 27, 35, 82, 195, 237, 292–293|\\n|urea toxicity, 293|excretion, 292–293|\\n|voltage-gated sodium channel, 330–336, 350, 368, 438|fixation, 300–302|\\n|Nitrogenase, 302|Nitroglycerine, 370|\\n|NO (nitric oxide), 53, 365, 370|receptor, 365|\\n|Node, 334, 335, 477|Nomenclature, gene, 101|\\n|Nondividing cells, 409|Nonmuscle myosin, 394–395|\\n|Nonpolar, 20, 21, 477|Nonsense mutation, 80, 82, 477|\\n|Nonvirulent bacteria, 66|Noradrenaline, 365, 368|\\n|Northern blot, 148–149, 477|Nose and scent receptors, 350–353|\\n|N terminal/terminus, 173, 184, 477|Nucleoid, 73, 477|\\n|Nucleolus, 57, 477|changes at mitosis, 58, 410, 412|\\n|nucleolar organizer region, 57, 477|Nucleoporins, 216|\\n|Nucleoside, 35–37, 477|triphosphates, 259–261|\\n|Nucleosome, 73, 477|Nucleotide, 37, 478|\\n|bases found in, 34|as energy currencies, 258, 259–261|\\n|excision–repair, 97, 478|in DNA, 65–67|\\n|in RNA, 105–106|Nucleus, nuclear, 56–58, 478|\\n|changes at mitosis, 57–58, 402–404, 410, 412|envelope, 56, 477|\\n|import/export, 216–220|lamina, lamins, 57, 396, 477|\\n|pore, 216–220, 477|Numbering|\\n|amino acids in polypeptides, 192|bases in DNA, 106|\\n|carbons in sugars, 35|Nurse, Paul, 411|\\n|Objective lens, 3, 478|Obliate anaerobes, 288, 478|\\n|Octane, 21|Odor of sanctity, 299|\\n|-OH (hydroxyl group), 27–28| |\\n---\\n|INDEX| |\\n|---|---|\\n|Oil, 39, 478|Osmosis, 53, 208, 479|\\n|Okasaki, Reiji, 90|Oster, George, 272|\\n|fragments, 90, 478|Outer doublet microtubules, 388, 479|\\n|Oleate, oleic acid, 39, 42–43, 478|Ovulation, 56|\\n|Olfactory neurone (scent-sensitive nerve cell), 350, 478|Oxaloacetate de novo generation from pyruvate, 286 in gluconeogenesis, 295 in glyoxylate shunt, 297 in Krebs cycle, 284 in nitrogen excretion/urea cycle, 292–293|\\n|Oligomycin, 269|Oxidation, 259, 479|\\n|Oligonucleotide, 134, 478|β oxidation, 290–291, 306, 479|\\n|primer, for PCR, 152, 431|2-oxoglutarate in Krebs cycle, 284 in nitrogen excretion, 292|\\n|probe for library screening, 136|Oxoglutarate dehydrogenase, 284 activation by calcium, 305|\\n|Oligosaccharide, 30, 478|Oxygen addition, by cytochromes P450, 249 anaerobic, 275, 286–288 cooperative binding, 240 diffusion across membranes, 53 electron-grabbing, 19 generation by photosynthesis, 274 heme group, 206 hydrogen bond, 25–27, 195|\\n|glycosylation in ER and Golgi, 225–226, 230|loading, myo- and hemoglobin, 238–240, 252–253 monosaccharide ring, 27 removal, by leghemoglobin, 302 oxidation, 259 why we breathe, 265|\\n|transferase, 225, 478|P site, 168, 479|\\n|Olive oil, 39|P1 artificial chromosomes (PACs), 139, 479|\\n|Oocyte, 406, 478|p16 INK4a, 184, 415|\\n|cAMP, gap junctions, 56|p21 CIP1, 415, 479|\\n|cell cycle control, 411|p27 KIP1, 415|\\n|gene cloning, 140–141|p38 protein kinase, 418, 479|\\n|generation by meiosis, 406|p53 transcription factor, 415, 417, 418, 420, 479|\\n|MPF, 411|p75 neurotrophin receptor, 417, 479|\\n|pre-implantation diagnosis for CF, 431|PAC, 140|\\n|Open complex, 89, 108, 478|Pain receptors, 334, 336, 344, 372–375, 479|\\n|DNA replication, 89|Pain relay cell, 332, 341, 372–375, 479|\\n|promoter, 108, 478|Palade, George, 60|\\n|Operator, in transcriptional control, 111–117, 478|Palmitic acid, 300|\\n|Operon, 109, 111, 125, 478|Paracrine transmitters, 363–364, 368–370, 479|\\n|inducible, 111–115|Parallel β sheets, 195, 197, 480|\\n|repressible, 116–118|Paramecium, 386, 389|\\n|Optical isomers, 196, 478|Parenchyma cells, 16, 480|\\n|Organelle, 9–10, 51, 56–61, 478|Patch clamp, 320–321, 480|\\n|cytoplasmic streaming, 395|glutamate receptor cloning, 140–141|\\n|definition, 51|pBluescript cloning vector, 132|\\n|double-membrane bounded, 56–58|PBS2 bacteriophage, 97|\\n|endosymbiotic theory, 12|PCR (polymerase chain reaction), 150–152, 158, 431, 432, 480|\\n|single-membrane bounded, 58–61|PDGF (platelet derived growth factor), 353, 480|\\n|trafficking, to specific organelles, 213–225, 230–231| |\\n|transport on actin microfilaments, 395| |\\n|transport on microtubules, 389–390| |\\n|Organic molecules, 36, 478| |\\n|Organisms, gene number and complexity in, 68| |\\n|Orientation| |\\n|antiparallel in DNA, 69| |\\n|cellulose, 54, 382| |\\n|chromatids, at mitosis, 403| |\\n|ciliary beat, Paramecium, 389| |\\n|microscope, 5| |\\n|organs, Kartegener’s syndrome, 389| |\\n|plant microtubules, 382| |\\n|ribosome on mRNA, 171| |\\n|water in hydration shell, 21| |\\n|Ori, origin of replication, 88, 479| |\\n|in cloning vectors, 130, 132, 140| |\\n|Ornithine, 191, 293| |\\n|Osmium tetroxide, 8| |\\n|Osmolarity, 208, 479| |\\n|Osmosis, 53, 208, 479| |\\n|Oster, George, 272| |\\n|Outer doublet microtubules, 388, 479| |\\n|Ovulation, 56| |\\n|Oxaloacetate de novo generation from pyruvate, 286 in gluconeogenesis, 295 in glyoxylate shunt, 297 in Krebs cycle, 284 in nitrogen excretion/urea cycle, 292–293| |\\n|Oxidation, 259, 479| |\\n|β oxidation, 290–291, 306, 479| |\\n|2-oxoglutarate in Krebs cycle, 284 in nitrogen excretion, 292| |\\n|Oxoglutarate dehydrogenase, 284 activation by calcium, 305| |\\n|Oxygen addition, by cytochromes P450, 249 anaerobic, 275, 286–288 cooperative binding, 240 diffusion across membranes, 53 electron-grabbing, 19 generation by photosynthesis, 274 heme group, 206 hydrogen bond, 25–27, 195| |\\n|loading, myo- and hemoglobin, 238–240, 252–253 monosaccharide ring, 27 removal, by leghemoglobin, 302 oxidation, 259 why we breathe, 265| |\\n|transferase, 225, 478| |\\n|Olive oil, 39| |\\n|Oocyte, 406, 478| |\\n|cAMP, gap junctions, 56| |\\n|cell cycle control, 411| |\\n|gene cloning, 140–141| |\\n|generation by meiosis, 406| |\\n|MPF, 411| |\\n|pre-implantation diagnosis for CF, 431| |\\n|Open complex, 89, 108, 478| |\\n|DNA replication, 89| |\\n|promoter, 108, 478| |\\n|Operator, in transcriptional control, 111–117, 478| |\\n|Operon, 109, 111, 125, 478| |\\n|inducible, 111–115| |\\n|repressible, 116–118| |\\n|Optical isomers, 196, 478| |\\n|Organelle, 9–10, 51, 56–61, 478| |\\n|cytoplasmic streaming, 395| |\\n|definition, 51| |\\n|double-membrane bounded, 56–58| |\\n|endosymbiotic theory, 12| |\\n|single-membrane bounded, 58–61| |\\n|trafficking, to specific organelles, 213–225, 230–231| |\\n|transport on actin microfilaments, 395| |\\n|transport on microtubules, 389–390| |\\n|Organic molecules, 36, 478| |\\n|Organisms, gene number and complexity in, 68| |\\n|Orientation| |\\n|antiparallel in DNA, 69| |\\n|cellulose, 54, 382| |\\n|chromatids, at mitosis, 403| |\\n|ciliary beat, Paramecium, 389| |\\n|microscope, 5| |\\n|organs, Kartegener’s syndrome, 389| |\\n|plant microtubules, 382| |\\n|ribosome on mRNA, 171| |\\n|water in hydration shell, 21| |\\n|Ori, origin of replication, 88, 479| |\\n|in cloning vectors, 130, 132, 140| |\\n|Ornithine, 191, 293| |\\n|Osmium tetroxide, 8| |\\n|Osmolarity, 208, 479| |\\n|Osmosis, 53, 208, 479| |\\n|Oster, George, 272| |\\n|Outer doublet microtubules, 388, 479| |\\n|Ovulation, 56| |\\n|Oxaloacetate de novo generation from pyruvate, 286 in gluconeogenesis, 295 in glyoxylate shunt, 297 in Krebs cycle, 284 in nitrogen excretion/urea cycle, 292–293| |\\n|Oxidation, 259, 479| |\\n|β oxidation, 290–291, 306, 479| |\\n|2-oxoglutarate in Krebs cycle, 284 in nitrogen excretion, 292| |\\n|Oxoglutarate dehydrogenase, 284 activation by calcium, 305| |\\n|Oxygen addition, by cytochromes P450, 249 anaerobic, 275, 286–288 cooperative binding, 240 diffusion across membranes, 53 electron-grabbing, 19 generation by photosynthesis, 274 heme group, 206 hydrogen bond, 25–27, 195| |\\n|loading, myo- and hemoglobin, 238–240, 252–253 monosaccharide ring, 27 removal, by leghemoglobin, 302 oxidation, 259 why we breathe, 265| |\\n|transferase, 225, 478| |\\n|Olive oil, 39| |\\n|Oocyte, 406, 478| |\\n|cAMP, gap junctions, 56| |\\n|cell cycle control, 411| |\\n|gene cloning, 140–141| |\\n|generation by meiosis, 406| |\\n|MPF, 411| |\\n|pre-implantation diagnosis for CF, 431| |\\n|Open complex, 89, 108, 478| |\\n|DNA replication, 89| |\\n|promoter, 108, 478| |\\n|Operator, in transcriptional control, 111–117, 478| |\\n|Operon, 109, 111, 125, 478| |\\n|inducible, 111–115| |\\n|repressible, 116–118| |\\n|Optical isomers, 196, 480| |\\n|Paramecium, 386, 389| |\\n|Parenchyma cells, 16, 480| |\\n|Patch clamp, 320–321, 480| |\\n|glutamate receptor cloning, 140–141| |\\n|pBluescript cloning vector, 132| |\\n|PBS2 bacteriophage, 97| |\\n|PCR (polymerase chain reaction), 150–152, 158, 431, 432, 480| |\\n|PDGF (platelet derived growth factor), 353, 480| |\\n---\\n# INDEX\\n\\n|PDGF receptor|200, 204, 223–225, 353–355|\\n|---|---|\\n|Pectin|53, 480|\\n|Pellagra|259|\\n|Pentose|27, 480|\\n|pentose phosphate pathway|289, 480|\\n|Peptide(s)|38, 184, 480|\\n|Peptidyl site (P site)|168, 480|\\n|Peptidyl transferase|173, 184, 480|\\n|Pericentriolar material|383|\\n|Peripheral (membrane) protein|52–53, 480|\\n|Peroxisome|59–60, 480|\\n|pH|24, 480|\\n|Phage (bacteriophage)|12, 74, 97, 133, 140, 480|\\n|Phalloidin|376|\\n|Phase-contrast microscopy|5, 480|\\n|PH domain|357, 480|\\n|Phenobarbital|149, 167|\\n|Phenylalanine|185|\\n|Phenylketonuria|294–295, 406–407, 481|\\n|Phenylpyruvate|294|\\n|Philadelphia chromosome|403|\\n|Phloem|16, 481|\\n|Phosphatase|190, 223, 481|\\n|Phosphate|33, 36, 481|\\n|Phosphatidylcholine|40, 41|\\n|Phosphatidylinositides (PIP 2, PIP 3 etc)|231, 346, 356–357|\\n|Phosphoanhydrides|191|\\n\\nPhosphoaspartate, 191\\n\\nPhosphodiester link, 42, 67, 105, 481\\n\\n5\\'-phosphodiester link, 118\\n\\nDNA, 67, 89, 90, 92, 95, 97\\n\\nphospholipid, 42\\n\\nRNA, 105, 109, 118\\n\\nPhosphoenolpyruvate, 286, 295\\n\\nPhosphoesters, 191\\n\\nPhosphofructokinase, 285, 481\\n\\n2-Phosphoglycerate, 286\\n\\n3-Phosphoglycerate, 286, 302\\n\\n1,3-bisphosphoglycerate, 286\\n\\nPhosphoglutamate, 191\\n\\nPhosphohistidine, 191\\n\\nPhosphoinositide 3-kinase (PI 3-kinase), 356, 481\\n\\nPhosphoinositide phospholipase C (PLC), 346\\n\\nβ isoform, 346\\n\\nγ isoform, 356\\n\\nPhospholipase C (PLC), 346, 481\\n\\nβ isoform, 346\\n\\nγ isoform, 356\\n\\nPhospholipid, 40–43, 52\\n\\nartificial bilayer, 428\\n\\nfish odor syndrome, 82–83\\n\\nhydrolysis, 45–46\\n\\nsynthesis, 300\\n\\ntrimethylaminuria, 82–83\\n\\nPhosphoryl group, 482\\n\\nPhosphorylase kinase (glycogen phosphorylase kinase), 242, 305, 348, 353, 482\\n\\nPhosphorylation, 190–191, 482\\n\\namino acid side group, 190–191\\n\\nCFTR, 427\\n\\ncontrol of gene expression, 358\\n\\nMPF and cell cycle, 411–412\\n\\neffect on protein shape, 241\\n\\nenzyme regulation, 251–252, 305, 356, 357\\n\\nenergy cost, 258\\n\\ngenerator of motile force, 241\\n\\nmannose and protein targeting, 230–231\\n\\nsubstrate level, 286\\n\\nPhosphoserine, 191\\n\\nPhosphothreonine, 191\\n\\nPhosphotyrosine, 191\\n\\nPhotoreceptor, 353\\n\\nPhotosynthesis, 11, 58, 271–275, 302–303, 482\\n\\nPhragmoplast, 404, 482\\n\\nPi (inorganic phosphate), 36\\n\\nPigment cell (chromatophore), 389\\n\\nPIP 2 (phosphatidylinositol bisphosphate), 346, 357, 482\\n\\nPIP 3 (phosphatidylinositol trisphosphate), 357\\n\\nPipette, for electrical measurements, 320–321\\n\\npKa, 25, 482\\n\\nof histidine and cysteine, 185\\n---\\n# INDEX\\n\\n|PKA (protein kinase A)|305, 353, 358, 427, 482|molecules, 20|\\n|---|---|---|\\n|PKB (protein kinase B)|356–357, 417, 482|solvents, 21|\\n|PKC (protein kinase C)|358, 482| |\\n|PKG (protein kinase G)|370, 483| |\\n|Plant(s)|11| |\\n|carbon fixation in|302–303| |\\n|cells|11| |\\n|genetically modified|15| |\\n|glyoxylate shunt in|297–298| |\\n|tissues|15–16| |\\n|Plaque|143, 483| |\\n|Plasmalemma (plasma membrane)|51, 483| |\\n|action potential|325–337| |\\n|apoptosis, modification in|415| |\\n|crossing by polypeptide|200, 204| |\\n|diffusion across|53, 121, 365| |\\n|endocytosis|229, 230| |\\n|patch clamping|320–321| |\\n|permeability to potassium ions|310–311| |\\n|resting voltage|311–312| |\\n|sodium gradient, as energy currency| |262|\\n|source of its protein and lipid|60, 230| |\\n|Plasma membrane|1, 51, 483| |\\n|action potential|325–337| |\\n|apoptosis, modification in|415| |\\n|crossing by polypeptide|200, 204| |\\n|diffusion across|53, 121, 365| |\\n|endocytosis|229, 230| |\\n|patch clamping|320–321| |\\n|permeability to potassium ions|310–311| |\\n|resting voltage|311–312| |\\n|sodium gradient, as energy currency| |262|\\n|source of its protein and lipid|60, 230| |\\n|Plasmid|74, 483| |\\n|cloning vectors|130, 132, 150| |\\n|recombinant|134| |\\n|in gene therapy|429, 432| |\\n|Plasmodesma, plasmodesmata|55, 483| |\\n|Plastoquinone|274–275, 483| |\\n|Platelet|344, 483| |\\n|calcium signalling|344–348| |\\n|derived growth factor (PDGF)|353| |\\n|derived growth factor receptor|200, 204, 223–225, 353–355, 483| |\\n|PLC (phospholipase C)|346, 483| |\\n|β isoform|346| |\\n|γ isoform|356| |\\n|Plug| | |\\n|nuclear pore|216| |\\n|inactivation|327, 330, 331| |\\n|CFTR regulatory domain|427| |\\n|Pneumococcus transformation experiment| |66|\\n|Pneumonia|46| |\\n|Polar|483| |\\n|body|406| |\\n|chemical groups|199| |\\n|head group|42, 346| |\\n---\\n# INDEX\\n\\n|Primase|90, 484|\\n|---|---|\\n|Primer|152, 484|\\n|Prion|207, 484|\\n|Processed pseudogene|100|\\n|Processing, of mRNA|118|\\n|Product, of reaction|242|\\n|Profilin|391, 485|\\n|Progenitor|73, 129, 401, 485|\\n|Programmed cell death|415–418, 485|\\n|Projector lens|3, 485|\\n|Prokaryote, prokaryotic|9, 10, 16, 485|\\n|Proline|185|\\n|Prometaphase|403, 485|\\n|Promoter|106, 108, 111, 117, 120, 485|\\n|Pronucleus|406, 485|\\n|Proof-reading|91, 92|\\n|Prophase|403, 485|\\n|Prosthetic group|205–206, 249–251, 265, 485|\\n|Proteases|331|\\n|Protein|183, 485|\\n\\n# Prokaryote\\n\\n- differences from eukaryotes, 11\\n- chromosomes, 73\\n- ribosomes, 165, 168\\n\\n# Proline\\n\\n- genetic code, 77, 78\\n- hydroxylation in collagen, 190\\n- steric (shape) effects, 185, 195\\n\\n# Prophase\\n\\n- I, 406, 408, 485\\n- II, 406, 485\\n\\n# Proton\\n\\n- Protonation, 25, 185, 486\\n\\n# Pseudogene\\n\\n- 100, 101, 486\\n\\n# Puromycin\\n\\n- 176\\n\\n# Pyridoxamine\\n\\n- pyridoxal phosphate, 249\\n\\n# Pyrimidines\\n\\n- 35, 65, 71, 486\\n\\n# Pyrophosphate\\n\\n- 44–46, 486\\n\\n# Pyruvate\\n\\n- 285\\n\\n# Quaternary structure\\n\\n- 204–205, 487\\n\\n# Quiescent cells\\n\\n- 409, 487\\n\\n# Quorum sensing\\n\\n- 116\\n\\n# R bacteria\\n\\n- 66\\n\\n# R115777\\n\\n- 355\\n\\n# Rab family GTPases\\n\\n- 231, 233, 487\\n\\n# Radioactive\\n\\n- radiation\\n\\n# Raf\\n\\n- 487\\n\\n# Ran GTPase\\n\\n- 216–219, 220\\n\\n# Random priming method\\n\\n- 143\\n---\\n|INDEX| |\\n|---|---|\\n|Ran-GAP|218, 219, 220|\\n|plasmid|134, 487|\\n|Rapid reaction techniques|252–253|\\n|Ras GTPase|355, 371, 487|\\n|RB (retinoblastoma)|413, 414, 487|\\n|Reading frame|79–80, 169, 487|\\n|Receptor|487|\\n|α-adrenergic|365, 368|\\n|ADP|344, 348|\\n|β-adrenergic|305, 353, 365, 368|\\n|classification|363–365|\\n|cloning|140–141|\\n|death domain|416, 417|\\n|endocytosis|229, 417, 487|\\n|fibroblast growth factor|371|\\n|fluorescent chimera|150|\\n|glucocorticoid|121–124, 150, 200, 202, 205, 365|\\n|glutamate|372–375|\\n|growth factor|353–356, 358|\\n|insulin|356–357|\\n|interleukin-2|432|\\n|intracellular|365|\\n|ionotropic cell surface|364–365|\\n|mannose-6-phosphate|231|\\n|metabotropic cell surface|365|\\n|muscarinic acetylcholine|370, 376|\\n|neurotrophin|417|\\n|nicotinic acetylcholine|364, 368, 376, 439|\\n|nitric oxide|365|\\n|p75|417|\\n|pain|332, 334, 336, 341–344, 372–375|\\n|photoreceptor|353|\\n|platelet derived growth factor|200, 204, 223|\\n|pregnane X|124|\\n|ryanodine|348–349|\\n|scent|350, 352|\\n|signal recognition particle|223|\\n|sorting signal|215, 219, 221|\\n|steroid|121–124, 365|\\n|Trk|417|\\n|tyrosine kinase|353–358, 417, 487|\\n|vesicle budding|229|\\n|Recessive gene|406–407, 425, 487|\\n|Recognition| |\\n|helix|200|\\n|of AUG codon|176|\\n|of ligand|183, 202|\\n|of ribosome-binding site|171|\\n|motif|123|\\n|signal recognition particle|223|\\n|site|of restriction enzyme, 133, 146, 150|\\n|Recombinant, recombination|488|\\n|bacteriphage|140|\\n|DNA|129–158, 487|\\n|genetic|408|\\n|homologous|157|\\n---\\n|INDEX|\\n|---|\\n|Ribonucleic acid (RNA), 105, 118, 489|Saints, odor of sanctity, 299|\\n|antisense, 132, 147|Saint John’s wort, 124|\\n|catalytic molecule, 242|Sakmann, Bert, 320|\\n|instability, 118|Salad dressing, 39|\\n|messenger (mRNA), 105–124|Salmonella, 264|\\n|northern blot, 148|Saltatory conduction, 336, 490|\\n|polymerase, 106–109, 118, 120, 489|Salt, 21, 39|\\n|primer, in DNA synthesis, 90, 92, 489|sweat, in CF, 424–425|\\n|ribosomal (rRNA), 57, 99, 105, 163, 165|Sanctity, odor of, 299|\\n|ribozymes, 242|Sanger, Frederick, 143|\\n|splicing, 68, 98, 118|Sarcomere, 393, 490|\\n|synthesis, 73, 106–123|Satellite DNA, 100, 403, 490|\\n|transfer (tRNA), 99, 163–165, 169–176|Saturation, saturated|\\n|types, 105, 163|enzyme, 245|\\n|viral, 71, 74|fatty acids, 42, 490|\\n|Ribose, 27, 29, 31, 65, 105, 489|kinetics, 245, 490|\\n|Ribosome, ribosomal, 57, 60, 165–176, 223–225|Scanning electron microscope, 9, 490|\\n|binding site, on mRNA, 168, 176|Scent-sensitive nerve cells, 336, 350, 351, 352|\\n|recycling, 175|Schleiden, Matthias, 1|\\n|Ribozyme, 242, 489|Schwann, Theodor, 1|\\n|Ribulose, 27, 29|Schwann cells, 334, 490|\\n|bisphosphate, 302|SCID (severe combined immunodeficiency disease), 432|\\n|RNA (ribonucleic acid), 105, 118, 489|Scrapie, 207, 208|\\n|antisense, 132, 147|Screening a library, 136|\\n|catalytic molecule, 242|Scurvy, 190|\\n|instability, 118|SDS–PAGE, 166–167, 490|\\n|messenger (mRNA), 105–124|Sea urchin egg, 325–330|\\n|northern blot, 148|Secondary|\\n|polymerase, 106–109, 118, 120, 489|antibody, 7, 490|\\n|primer, in DNA synthesis, 90, 92|cell wall, 54, 490|\\n|ribosomal (rRNA), 57, 99, 105, 163, 165|lysosome, 231|\\n|ribozymes, 242|immunofluorescence, 7, 490|\\n|splicing, 68, 98, 118|structure, 195, 490|\\n|synthesis, 73, 106–123|Secretory vesicle, 228, 230, 491|\\n|transfer (tRNA), 99, 163–165, 169–176|Sedimentation coefficient (S value), 165–168, 491|\\n|types, 105, 163|Selection, DNA clones, 134–139|\\n|viral, 71, 74|Self-correcting enzyme, 92|\\n|Rock climbers, 350|Semi-conservative replication, 92–93, 491|\\n|Root, 15–16|Sequencing of DNA, 143–146|\\n|Rotary motion, 264, 272–273, 388|Serine, 185|\\n|Rotenone, 271|genetic code, 77, 78|\\n|Rough endoplasmic reticulum (RER), 57, 60, 223–225, 489|phosphorylation of side chain, 120, 190, 241|\\n|changes at mitosis, 410|70S initiation complex, formation of, 170, 171|\\n|glycosylation and, 225–226|Severe combined immunodeficiency disease (SCID), 432|\\n|protein synthesis on, 223–225|SH2 domain, 355, 358, 491|\\n|Roughton, Francis, 252|β sheet, 195, 491|\\n|rRNA, 57, 99, 105, 163, 165, 489|Shigella, 393|\\n|Ruska, Ernst, 9|Shine–Dalgarno sequence, 169, 491|\\n|Ryanodine receptor, 348, 489|Shoot, of plant, 15–16|\\n|σ factor, 106, 108, 490|Short interspersed nuclear elements (SINEs), 100–101|\\n|S bacteria, 66|Sickle cell anemia, 81, 82|\\n|S phase, 408–410, 412–415, 490|S value, 165–168, 490|\\n|Saccharomyces, 68, 401| |\\n---\\n# INDEX\\n\\n|Side chains|38, 78, 491|\\n|---|---|\\n|hydrophilic and hydrophobic|185, 186, 187, 189|\\n|hydroxyl groups in|188|\\n|posttranslational modifications in|190|\\n|Sigma (σ ) factor|106, 108, 491|\\n|Signal| |\\n|death|417|\\n|ER retention|215, 216|\\n|mitochondrial import|215|\\n|nuclear localization|215–219, 222|\\n|peptidase|223, 491|\\n|recognition particle|223, 491|\\n|sequence|204, 215, 216, 223–225, 491|\\n|sorting|215, 230–231|\\n|start|79|\\n|stop|78–80|\\n|Signalling| |\\n|Biochemical|353–358|\\n|Electrical|325–337|\\n|Intracellular|341–359|\\n|Pathways|357–359|\\n|webs|357–359|\\n|Sildenafil|370|\\n|Silk|183, 195|\\n|Silver stain|166|\\n|SINEs (short interspersed nuclear elements)|100–101|\\n|Single| |\\n|celled organisms|1|\\n|copy genes|99|\\n|membrane envelopes|58–61|\\n|strand binding proteins|90, 492|\\n|Site-directed mutagenesis|149, 492|\\n|Skeletal muscle|14, 492|\\n|activation|348, 367–368|\\n|calcium signals|348|\\n|control of energy production|305, 348, 350, 353|\\n|enlargement|371|\\n|mechanism of contraction|393–394|\\n|resting voltage|311|\\n|ryanodine receptor|348–349|\\n|Skin cancer|418|\\n|Skou, Jens|273|\\n|Sleeping sickness|288|\\n|Sma H1|133|\\n|Small nuclear RNA (snRNA)|118, 492|\\n|Smell detection|350–353|\\n|Smoking|13, 94, 376, 386|\\n|Smooth| |\\n|endoplasmic reticulum|60, 216, 300, 492|\\n|muscle|14, 368–371, 492|\\n|SNARE proteins|231, 232, 492|\\n|snRNA (small nuclear RNA)|118, 492|\\n|Sodium| |\\n|action potential|332–337, 492|\\n---\\n# INDEX\\n\\n|Sticky ends|133, 493|\\n|---|---|\\n|St. John’s wort|124|\\n|Stoma, stomata|16|\\n|Stomach, cell|61|\\n|Stop codons|79–80, 174, 494|\\n|Stop signals|79–80, 109, 173–174, 494|\\n|Stopped flow|252–253|\\n|Stratified epithelium|13, 494|\\n|Streptomycin|176|\\n|Stress| |\\n|-activated apoptosis|417–418|\\n|-activated protein kinase|418, 494|\\n|adrenaline in|368|\\n|fiber|390–391, 494|\\n|heat shock proteins|221|\\n|rotational, in DNA|89|\\n|Striated muscle|14, 393, 494|\\n|activation|348–349, 367–368|\\n|anaerobic|275, 286–287, 350|\\n|blood supply|368–372|\\n|calcium effects|348–350, 394|\\n|calf|365–372|\\n|cAMP effects|353|\\n|desmin|396|\\n|digitalis|320|\\n|feedforward control|304–306|\\n|fiber|15|\\n|gastrocnemius|365|\\n|glucose carrier translocation|317, 357|\\n|glycogen storage and breakdown|30, 288, 304–306|\\n|intermediate filament|396|\\n|ketone bodies as fuel|290|\\n|mechanism of contraction|393–394|\\n|mitochondria|58|\\n|neurotrophin synthesis|417|\\n|paralysis (BoTox)|232|\\n|resting voltage|311|\\n|ryanodine receptors|348–349|\\n|Stroma|274, 494|\\n|Substrate|241–242, 494|\\n|Subtilisin|149–150|\\n|Succinate|267, 284, 297|\\n|succinate-Q reductase|267|\\n|succinate dehydrogenase|267, 284|\\n|Sucrose|243, 494|\\n|Sugar|27–33|\\n|-phosphate backbone|68|\\n|Suicide, cell|417|\\n|Sulfur|25–26, 185, 191, 195, 265–256, 275|\\n|Summation at synapse|372–375, 494|\\n|Sunburn|418|\\n|Supercoiling|73, 494|\\n|S value|165–168, 494|\\n|Svedberg unit (S value)|165–168, 494|\\n|Sweat gland|424–425|\\n|Synapse|344, 372–375, 376, 494|\\n---\\n# INDEX\\n\\n|glycosylation|190|\\n|---|---|\\n|phosphorylation of side group|190|\\n|Threshold, of action potential|328, 372–375, 495|\\n|Thylakoid|58, 274, 495|\\n|Thymidine triphosphate (TTP)| |\\n|dTTP|building block of DNA, 65–67|\\n|dTTP|energy currency, 259|\\n|Thymine (T)|34, 35, 495|\\n|bonding with adenine|70|\\n|dimer|94, 97|\\n|Thyone|392|\\n|Tight junctions|54, 55, 61, 495|\\n|Tire|11, 53|\\n|Tissue|12, 495|\\n|epithelium|12|\\n|connective|13|\\n|muscle|14–15|\\n|nervous|13|\\n|plant|15–16|\\n|T lymphocyte|222|\\n|Tobacco|94, 250, 376|\\n|Topoisomerase I|89, 495|\\n|Topoisomerase II|74–75, 89, 495|\\n|Trafficking|213–232, 357|\\n|trans cisternae|226|\\n|trans Golgi network|226, 228, 229–231, 496|\\n|Transamination|292, 294|\\n|Transcription|105–124, 496|\\n|bubble|108–109, 496|\\n|control|109–118, 119–124|\\n|factor|120–124, 190, 216, 222–223, 355, 358, 413, 415, 417, 432, 496|\\n|preinitiation complex|120|\\n|tissue-specific|122|\\n|Transepithelial voltage|425|\\n|Transfection|150, 496|\\n|Transfer| |\\n|RNA (tRNA)|99, 163–165, 169–176, 496|\\n|vesicle|226|\\n|Transformation|66, 134, 415, 496|\\n|Transgenic| |\\n|animal|157, 496|\\n|plant|158|\\n|Transition state|249|\\n|Translation|76, 163–176, 496|\\n|Translocation|173, 496|\\n|protein|213–232, 317, 357|\\n|ribosome on mRNA|173|\\n|Transmembrane| |\\n|helix|200, 204|\\n|translocation|213, 221, 223–225, 496|\\n|voltage|261–262, 311–314, 320–321, 325–337, 372–375, 496|\\n|Transmission electron microscope|8, 496|\\n|Transmitter|123, 341, 344, 357, 363–375, 496|\\n|Transpiration|16, 496|\\n---\\n# INDEX\\n\\n|Untangling chromosomes|74|\\n|---|---|\\n|Untranslated sequence|169, 498|\\n|Unwinding of DNA|87–90, 108|\\n|Upstream|106, 498|\\n|Uracil (U)|34, 35, 46, 94, 498|\\n|DNA glycosidase|95, 498|\\n|Uranyl acetate|8|\\n|Urea|498|\\n|Uridine| |\\n|diphosphate-glucose|298|\\n|dUTP, in DNA|97|\\n|triphosphate|105, 259|\\n|UV (ultraviolet) light|6|\\n|DNA damage|94, 98, 415, 418|\\n|fluorescence|6|\\n|thymine dimers|94|\\n|Vaccines|manufacture in bacteria, 149|\\n|Vacuole|11, 498|\\n|Valine|185|\\n|Valinomycin|196|\\n|Valium|375, 499|\\n|van der Waals forces|199, 237, 499|\\n|van Leeuwenhoek, Anton|1|\\n|Variable number tandem repeats (VNTRs)|146|\\n|Vascular tissue, plant|15–16|\\n|Vasoconstrictor|368, 499|\\n|Vasodilator|370, 499|\\n|Vector|499|\\n|cloning|130–132|\\n|expression|139, 149|\\n|genomic DNA|139–140|\\n|Vermeer, Johannes|1|\\n|Vesicle|215, 233, 499|\\n|clathrin-coated|229|\\n|coatomer-coated|228|\\n|formation of|228|\\n|Golgi apparatus and|226|\\n|transfer|226|\\n|Vesicular trafficking|214, 215, 499|\\n|between intracellular compartments|226–232|\\n|Viagra|370|\\n|Vibrio| |\\n|cholera|353|\\n|fischeri|116|\\n|Vikings|376|\\n\\n# Villin\\n\\n391–392, 499\\n\\n# Villus, villi\\n\\n13, 499\\n\\n# Vimentin\\n\\n396, 499\\n\\n# Vinegar\\n\\n39\\n\\n# Viral vectors\\n\\nfor gene therapy, 429\\n\\n# Virulent bacteria\\n\\n66\\n\\n# Virus\\n\\n11–12, 71, 74, 97, 176, 416, 429, 499\\n\\n# Vitamin\\n\\nA, 158\\n\\nB6, 249\\n\\nC, 190\\n\\nniacin (B3), 259\\n\\n# VNTRs (variable number tandem repeats)\\n\\n146, 499\\n\\n# vo (initial velocity)\\n\\n243–246\\n\\n# Voltage\\n\\nclamp, 326, 428, 499\\n\\nequilibrium, 311–313\\n\\n-gated calcium channel, 327–328, 343, 348–349, 389, 499\\n\\n-gated sodium channel, 200, 330–331, 499\\n\\nmeasurement, 320–321\\n\\nmitochondrial, 261\\n\\nresting, 311–312\\n\\nsperm fusion, 325–327\\n\\nsummation, 372–375\\n\\nsweat gland, 425\\n\\nthylakoid, 274\\n\\ntransmembrane, 261–262, 311–314, 320–321, 325–337, 372–375\\n\\ntransmission down axons, 332–336\\n\\ntransmission through gap junctions, 55–56\\n\\n# vSNARES\\n\\n231, 232\\n\\n# Walker, John\\n\\n273\\n\\n# Washing powder\\n\\n150\\n\\n# Water\\n\\n19\\n\\nabundance, 19, 183\\n\\ndiffusion across membranes, 53, 208\\n\\nfiltration, 10\\n\\nH + donor and acceptor, 22–25\\n\\nhydrogen bonding, 25–27\\n\\nhydrolysis, 44–46\\n\\noxidation in chloroplast, 274\\n\\nsolvent, 21, 39\\n\\nviscosity, 264\\n\\n# Watson, James\\n\\n68\\n\\n# Watson–Crick model\\n\\n69\\n\\n# Wee1\\n\\n412, 499\\n\\n# Western blotting\\n\\n166–167\\n\\n# White blood cell\\n\\ncalcineurin and immunosuppressants, 222\\n\\nFas activation, 416\\n\\nfmet-leu-phe activation, 5, 171\\n---\\n# INDEX\\n\\n|resting voltage|311|\\n|---|---|\\n|severe combined immunodeficiency disease (SCID)|432|\\n|Wilting|53|\\n|Wobble hypothesis|164–165, 499|\\n|Xenopus oocyte|140–141, 411|\\n|Xeroderma pigmentosum|98, 499|\\n|X-linked adrenoleukodystrophy|60|\\n|X-ray|68|\\n|X-SCID|432|\\n|Xylem|16, 499|\\n|Yeast|1, 68, 275, 288, 401|\\n|artificial chromosome (YAC) vectors|139–140|\\n|Yew|385|\\n|Z disc|393–394, 500|\\n|Zebrafish genome|99|\\n|Zellweger’s syndrome|500|\\n|Zernike, Frits|9|\\n|Z-DNA|71, 500|\\n|Zinc|157, 200, 202, 205|\\n|finger|200, 500|\\n|Zoo blot|146, 426|', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n')]\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "documents2 = SimpleDirectoryReader(input_files=['/content/data/cellbiology.pdf']).load_data()\n",
        "print(documents2)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "-uSK1LgjZDMQ",
        "outputId": "9320027b-74ab-439a-8ece-7a6dfc1d7d3a"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[Document(id_='937a36df-22e6-45e0-ae56-1b7d6d2fb46f', embedding=None, metadata={'page_label': '1', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GDZ\\nWY001-Bolsover-FM WY001-Bolsover-v3.cls October 22, 2003 14:59\\nCELL BIOLOGY\\nSECOND EDITION\\ni', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='5ed8779d-1998-43a8-bd2e-c53f3428defe', embedding=None, metadata={'page_label': '2', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GDZ\\nWY001-Bolsover-FM WY001-Bolsover-v3.cls October 22, 2003 14:59\\nCELL BIOLOGY\\nA Short Course\\nSECOND EDITION\\nStephen R. Bolsover\\nDepartment of Physiology\\nUniversity College London\\nJeremy S. Hyams\\nDepartment of Biology\\nUniversity College London\\nElizabeth A. Shephard\\nDepartment of Biochemistry and Molecular Biology\\nUniversity College London\\nHugh A. White\\nDepartment of Biochemistry and Molecular Biology\\nUniversity College London\\nClaudia G. Wiedemann\\nDepartment of Physiology\\nUniversity College London\\nA JOHN WILEY & SONS, INC., PUBLICATION\\niii', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3b0bd49d-f0a0-4c30-af89-838cfc4b794e', embedding=None, metadata={'page_label': '3', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GDZ\\nWY001-Bolsover-FM WY001-Bolsover-v3.cls October 22, 2003 14:59\\nCopyright C⃝2004 by John Wiley & Sons, Inc. All rights reserved.\\nPublished by John Wiley & Sons, Inc., Hoboken, New Jersey.\\nPublished simultaneously in Canada.No part of this publication may be reproduced, stored in a retrieval system, or transmitted in any form or by any\\nmeans, electronic, mechanical, photocopying, recording, scanning, or otherwise, except as permitted underSection 107 or 108 of the 1976 United States Copyright Act, without either the prior written permission of thePublisher, or authorization through payment of the appropriate per-copy fee to the Copyright Clearance Center,Inc., 222 Rosewood Drive, Danvers, MA 01923, 978-750-8400, fax 978-646-8600, or on the web atwww.copyright.com. Requests to the Publisher for permission should be addressed to the PermissionsDepartment, John Wiley & Sons, Inc., 111 River Street, Hoboken, NJ 07030, (201) 748-6011,fax (201) 748-6008.\\nLimit of Liability/Disclaimer of Warranty: While the publisher and author have used their best efforts in\\npreparing this book, they make no representations or warranties with respect to the accuracy or completeness ofthe contents of this book and speciﬁcally disclaim any implied warranties of merchantability or ﬁtness for aparticular purpose. No warranty may be created or extended by sales representatives or written sales materials.The advice and strategies contained herein may not be suitable for your situation. You should consult with aprofessional where appropriate. Neither the publisher nor author shall be liable for any loss of proﬁt or any othercommercial damages, including but not limited to special, incidental, consequential, or other damages.\\nFor general information on our other products and services please contact our Customer Care Department within\\nthe U.S. at 877-762-2974, outside the U.S. at 317-572-3993 or fax 317-572-4002.\\nWiley also publishes its books in a variety of electronic formats. Some content that appears in print, however,\\nmay not be available in electronic format.\\nLibrary of Congress Cataloging-in-Publication Data:\\nCell biolog y : a short course / Stephen R. Bolsover ...[et al.].—2nd ed.\\np. cm.\\nIncludes bibliographical references and index.\\nISBN 0-471-26393-1 (Paper)1. Cytology. I. Bolsover, Stephen R., 1954–\\nQH581.2.C425 2003571.6—dc21 2003000577\\nPrinted in the United States of America\\n1 0987654321\\niv', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b02c734f-1ef6-41f3-aa00-9b8c67671af0', embedding=None, metadata={'page_label': '4', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GDZ\\nWY001-Bolsover-FM WY001-Bolsover-v3.cls October 22, 2003 14:59\\nCONTENTS IN BRIEF\\n1CELLS AND TISSUES 1\\n2FROM WATER TO DNA: THE CHEMISTRY OF LIFE 19\\n3MEMBRANES AND ORGANELLES 51\\n4DNA STRUCTURE AND THE GENETIC CODE 65\\n5DNA AS A DATA STORAGE MEDIUM 87\\n6TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION 105\\n7RECOMBINANT DNA AND GENETIC ENGINEERING 129\\n8MANUFACTURING PROTEIN 163\\n9PROTEIN STRUCTURE 183\\n10 INTRACELLULAR PROTEIN TRAFFICKING 213\\n11 HOW PROTEINS WORK 237\\n12 ENERGY TRADING WITHIN THE CELL 257\\n13 METABOLISM 281\\n14 IONS AND VOLTAGES 309\\n15 THE ACTION POTENTIAL 325\\n16 INTRACELLULAR SIGNALING 341\\n17 INTERCELLULAR COMMUNICATION 363\\n18 MECHANICAL MOLECULES 381\\n19 CELL CYCLE AND CONTROL OF CELL NUMBER 401\\n20 CASE STUDY: CYSTIC FIBROSIS 423\\nv', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1ccc1bab-fcf3-4871-a6da-1900603e7fab', embedding=None, metadata={'page_label': '5', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GDZ\\nWY001-Bolsover-FM WY001-Bolsover-v3.cls October 22, 2003 14:59\\nCONTENTS\\nPREFACE, xv\\nACKNOWLEDGMENTS, xviiINSTRUCTOR NOTES, xix\\n1CELLS AND TISSUES, 1\\nPrinciples of Microscopy, 2\\nThe Light Microscope, 3\\nThe Electron Microscope, 8The Scanning Electron Microscope, 9\\nOnly Two Types of Cell, 9\\nSpecial Properties of Plant Cells, 11\\nViruses, 11Origin of Eukaryotic Cells, 12Cell Specialization, 12\\nEpithelia, 12Connective Tissue, 13Nervous Tissue, 13Muscle, 14Plants, 15\\nSummary, 16Review Questions, 16Answers to Review Questions, 17\\n2FROM WATER TO DNA:\\nTHE CHEMISTRY OF LIFE, 19\\nThe Chemical Bond: Sharing\\nElectrons, 19\\nInteractions with Water: Solutions, 21\\nIonic Compounds Will Dissolve Only in\\nPolar Solvents, 21\\nAcids Are Molecules That Give H+to\\nWater, 21Bases Are Molecules That Take H+from\\nWater, 25\\nIsoelectric Point, 25A Hydrogen Bond Forms When a\\nHydrogen Atom Is Shared, 25\\nBiological Macromolecules, 27Carbohydrates: Candy and Canes, 27\\nAn Assortment of Sweets, 27Disaccharides, 28Out of the Sweet Comes Forth\\nStrength, 30\\nModi ﬁed Sugars, 31\\nNucleosides, Phosphate, and\\nNucleotides, 35\\nAmino Acids, Polypeptides, and Proteins, 37Lipids, 39Hydrolysis, 44Summary, 46Further Reading, 47Review Questions, 47Answers to Review Questions, 48\\n3MEMBRANES AND\\nORGANELLES, 51\\nBasic Properties of Cell Membranes, 51\\nStraight Through the Membrane:\\nDiffusion Through the Bilayer, 53\\nBeyond the Cell Membrane:\\nThe Extracellular Matrix, 53\\nCell Junctions, 54\\nOrganelles Bounded by Double-Membrane\\nEnvelopes, 56The Nucleus, 56Mitochondria and Chloroplasts, 58\\nvii', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3c874ed3-cb07-43a0-844d-53aa54533651', embedding=None, metadata={'page_label': '6', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GDZ\\nWY001-Bolsover-FM WY001-Bolsover-v3.cls October 22, 2003 14:59\\nviii CONTENTS\\nOrganelles Bounded by Single-Membrane\\nEnvelopes, 58\\nPeroxisomes, 59Endoplasmic Reticulum, 60Golgi Apparatus, 60Lysosomes, 61\\nSummary, 61Review Questions, 62Answers to Review Questions, 63\\n4DNA STRUCTURE AND THE GENETIC\\nCODE, 65\\nIntroduction, 65The Structure of DNA, 65\\nThe DNA Molecule Is a Double Helix, 68The Two DNA Chains Are\\nComplementary, 69\\nDifferent Forms of DNA, 71\\nDNA as the Genetic Material, 71Packaging of DNA Molecules into\\nChromosomes, 71Eukaryotic Chromosomes and Chromatin\\nStructure, 71\\nProkaryotic Chromosomes, 73Plasmids, 74Viruses, 74\\nThe Genetic Code, 75\\nAmino Acid Names Are Abbreviated, 79The Code Is Degenerate But\\nUnambiguous, 79\\nStart and Stop Codons and the Reading\\nFrame, 79\\nThe Code Is Nearly Universal, 80Missense Mutations, 80\\nSummary, 81Further Reading, 84Review Questions, 84Answers to Review Questions, 85\\n5DNA AS A DATA STORAGE\\nMEDIUM, 87\\nIntroduction, 87DNA Replication, 87\\nThe DNA Replication Fork, 88\\nProteins Open up the DNA Double Helix\\nDuring Replication, 88DnaA Protein, 88DnaB and DnaC Proteins, 90Single-Strand Binding Proteins, 90\\nBiochemistry of DNA Replication, 90\\nDNA Synthesis Requires an RNA\\nPrimer, 90\\nRNA Primers Are Removed, 92The Self-Correcting DNA\\nPolymerase, 92\\nDNA Repair, 94\\nSpontaneous and Chemically Induced\\nBase Changes, 94\\nRepair Processes, 94\\nGene Structure and Organization in\\nEukaryotes, 98Introns and Exons —Additional\\nComplexity in Eukaryotic Genes, 98\\nThe Major Classes of Eukaryotic\\nDNA, 99\\nGene Nomenclature, 101Summary, 101Further Reading, 102Review Questions, 102Answers to Review Questions, 103\\n6TRANSCRIPTION AND THE\\nCONTROL OF GENE EXPRESSION, 105\\nStructure of RNA, 105RNA Polymerase, 106Gene Notation, 106Bacterial RNA Synthesis, 106Control of Bacterial Gene Expression, 109\\nlac, an Inducible Operon, 111\\ntrp, a Repressible Operon, 116\\nEukaryotic RNA Synthesis, 118Messenger RNA Processing, 118\\nControl of Eukaryotic Gene\\nExpression, 119', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='baea60c1-cb36-4173-b3e3-e3a40ea9e7a0', embedding=None, metadata={'page_label': '7', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GDZ\\nWY001-Bolsover-FM WY001-Bolsover-v3.cls October 22, 2003 14:59\\nCONTENTS ix\\nGlucocorticoids Cross the Cell Membrane\\nto Activate Transcription, 121\\nSummary, 125\\nFurther Reading, 125Review Questions, 126Answers to Review Questions, 127\\n7RECOMBINANT DNA AND GENETIC\\nENGINEERING, 129\\nDNA Cloning, 129Creating the Clone, 130\\nIntroduction of Foreign DNA Molecules\\ninto Bacteria, 130\\nSelection of cDNA Clones, 134Genomic DNA Clones, 139\\nUses of DNA Clones, 143\\nDNA Sequencing, 143Southern Blotting, 146In situ Hybridization, 147Northern Blotting, 148Production of Mammalian Proteins in\\nBacteria, 149\\nProtein Engineering, 149Polymerase Chain Reaction, 150Identifying the Gene Responsible for a\\nDisease, 152\\nReverse Genetics, 152Transgenic Animals, 157\\nEthics of DNA Testing for Inherited\\nDisease, 157\\nSummary, 158Further Reading, 159Review Questions, 159Answers to Review Questions, 160\\n8MANUFACTURING PROTEIN, 163\\nAttachment of an Amino Acid to Its\\ntRNA, 163Transfer RNA, the Anticodon, and the\\nWobble, 164\\nThe Ribosome, 165Bacterial Protein Synthesis, 168Ribosome-Binding Site, 168Chain Initiation, 169The 70S Initiation Complex, 171Elongation of the Protein Chain, 171The Polyribosome, 173Termination of Protein Synthesis , 174The Ribosome Is Recycled, 175\\nEukaryotic Protein Synthesis Is a Little\\nMore Complex, 175Antibiotics and Protein Synthesis, 176\\nSummary, 178Further Reading, 179Review Questions, 179Answers to Review Questions, 180\\n9PROTEIN STRUCTURE, 183\\nNaming Proteins, 184Polymers of Amino Acids, 184\\nThe Amino Acid Building Blocks, 184The Unique Properties of Each Amino\\nAcid, 188\\nOther Amino Acids Are Found in\\nNature, 191\\nThe Three-Dimensional Structures of\\nProteins, 192Hydrogen Bonds, 195Electrostatic Interactions, 199van der Waals Forces, 199Hydrophobic Interactions, 199Disul ﬁde Bonds, 199\\nTertiary Structure: Domains and\\nMotifs, 200\\nQuaternary Structure: Assemblies of Protein\\nSubunits, 204\\nProsthetic Groups, 205The Primary Structure Contains all the\\nInformation Necessary to Specify\\nHigher-Level Structures, 206\\nSummary, 209\\nFurther Reading, 209Review Questions, 210Answers to Review Questions, 211', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='8976b70a-6ed1-4a44-8871-98c4680faff5', embedding=None, metadata={'page_label': '8', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GDZ\\nWY001-Bolsover-FM WY001-Bolsover-v3.cls October 22, 2003 14:59\\nx CONTENTS\\n10 INTRACELLULAR PROTEIN\\nTRAFFICKING, 213\\nThree Modes of Intracellular Protein\\nTransport, 213\\nTargeting Sequences, 215Retention, 215\\nTransport to and from the Nucleus, 215\\nThe Nuclear Pore Complex, 216Gated Transport Through the Nuclear\\nPore, 216\\nGTPases and the GDP/GTP Cycle, 218GTPases in Nuclear Transport, 218\\nTransport Across Membranes, 221\\nTransport to Mitochondria, 221Chaperones and Protein Folding, 221Transport to Peroxisomes, 221Synthesis on the Rough Endoplasmic\\nReticulum, 223\\nGlycosylation: The Endoplasmic\\nReticulum and Golgi System, 225\\nVesicular Traf ﬁcking Between\\nIntracellular Compartments, 226The Principle of Fission and Fusion, 226Vesicle Formation, 228Coatomer-Coated Vesicles, 228Clathrin-Coated Vesicles, 229The Trans-Golgi Network and Protein\\nSecretion, 229\\nTargeting Proteins to the Lysosome, 230Fusion, 231\\nSummary, 232Further Reading, 233Review Questions, 233Answers to Review Questions, 234\\n11 HOW PROTEINS WORK, 237\\nHow Proteins Bind Other Molecules, 237Dynamic Protein Structures, 238Allosteric Effects, 238Chemical Changes That Shift the\\nPreferred Shape of a Protein, 240\\nEnzymes Are Protein Catalysts, 241The Initial Velocity of an Enzyme\\nReaction, 242\\nEffect of Substrate Concentration on\\nInitial Velocity, 244\\nThe Effect of Enzyme Concentration, 245The Speci ﬁcity Constant, 247\\nEnzyme Catalysis, 247Cofactors and Prosthetic Groups, 249Enzymes Can Be Regulated, 251Summary, 254Further Reading, 254Review Questions, 255Answers to Review Questions, 256\\n12 ENERGY TRADING WITHIN THE\\nCELL, 257\\nCellular Energy Currencies, 258\\nReduced Nicotinamide Adenine\\nDinucleotide (NADH), 259\\nNucleoside Triphosphates (ATP plus\\nGTP, CTP, TTP, and UTP), 259\\nThe Hydrogen Ion Gradient Across the\\nMitochondrial Membrane, 261\\nThe Sodium Gradient Across the Plasma\\nMembrane, 262\\nEnergy Currencies Are Interconvertible, 263\\nExchange Mechanisms Convert Between\\nthe Four Energy Currencies, 263\\nElectron Transport Chain, 265ATP Synthase, 269Sodium/Potassium ATPase, 270ADP/ATP Exchanger, 271Photosynthesis, 271All Carriers Can Change Direction, 275\\nSummary, 278Further Reading, 278Review Questions, 278Answers to Review Questions, 279\\n13 METABOLISM, 281\\nThe Krebs Cycle: The Central Switching\\nYard of Metabolism, 283', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3e81e0e9-4d33-426b-8f6f-769694567c4f', embedding=None, metadata={'page_label': '9', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GDZ\\nWY001-Bolsover-FM WY001-Bolsover-v3.cls October 22, 2003 14:59\\nCONTENTS xi\\nFrom Glucose to Pyruvate: Glycolysis, 284\\nGlycolysis Without Oxygen, 286\\nGlycogen Can Provide Glucose for\\nGlycolysis, 288\\nGlucose May Be Oxidized to Produce\\nPentose Sugars, 289\\nFrom Fats to Acetyl-CoA: βOxidation, 290\\nAmino Acids as Another Source of\\nMetabolic Energy, 292\\nMaking Glucose: Gluconeogenesis, 295Making Glycogen: Glycogenesis, 298Making Fatty Acids and Glycerides, 300Synthesis of Amino Acids, 300Carbon Fixation in Plants, 302Control of Energy Production, 303\\nFeedback and Feedforward, 303Negative Feedback Control of\\nGlycolysis, 304\\nFeedforward Control in Muscle\\nCells, 304\\nSummary, 306Further Reading, 306Review Questions, 307Answers to Review Questions, 308\\n14 IONS AND VOLTAGES, 309\\nThe Potassium Gradient and the Resting\\nV oltage, 309Potassium Channels Make the Plasma\\nMembrane Permeable to Potassium\\nIons, 310\\nConcentration Gradients and Electrical\\nV oltage Can Balance, 311\\nThe Chloride Gradient, 314\\nGeneral Properties of Channels, 314General Properties of Carriers, 316\\nThe Glucose Carrier, 316The Sodium –Calcium Exchanger, 317\\nCarriers with an Enzymatic Action:\\nThe Calcium ATPase, 318\\nSummary, 322Further Reading, 322Review Questions, 322Answers to Review Questions, 324\\n15 THE ACTION POTENTIAL, 325\\nThe Calcium Action Potential in Sea Urchin\\nEggs, 325Effect of Egg Transmembrane V oltage on\\nSperm Fusion, 325\\nThe V oltage-Gated Calcium\\nChannel, 327\\nThe Calcium Action Potential, 328\\nThe Voltage-Gated Sodium Channel in\\nNerve Cells, 330The V oltage-Gated Sodium Channel, 330Electrical Transmission down a Nerve\\nCell Axon, 332\\nMyelination and Rapid Action Potential\\nTransmission, 334\\nSummary, 337Further Reading, 338Review Questions, 338Answers to Review Questions, 339\\n16 INTRACELLULAR SIGNALING, 341\\nCalcium, 341\\nCalcium Can Enter from the Extracellular\\nMedium, 341\\nCalcium Can Be Released from the\\nEndoplasmic Reticulum, 344\\nProcesses Activated by Cytosolic\\nCalcium Are Extremely Diverse, 348\\nReturn of Calcium to Resting\\nLevels, 350\\nCyclic Adenosine Monophosphate, 350Cyclic Guanosine Monophosphate, 353Multiple Messengers, 353Biochemical Signaling, 353\\nReceptor Tyrosine Kinases and the MAP\\nKinase Cascade, 353\\nGrowth Factors Can Trigger a Calcium\\nSignal, 356\\nProtein Kinase B and the Glucose\\nTransporter: How Insulin Works, 356', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b147c9b9-1b95-438b-81db-42b472ca4c04', embedding=None, metadata={'page_label': '10', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GDZ\\nWY001-Bolsover-FM WY001-Bolsover-v3.cls October 22, 2003 14:59\\nxii CONTENTS\\nCrosstalk —Signaling Pathways or\\nSignaling Webs?, 357\\nSummary, 359\\nFurther Reading, 360Review Questions, 360Answers to Review Questions, 361\\n17 INTERCELLULAR\\nCOMMUNICATION, 363\\nClassifying Transmitters and Receptors, 363\\nIonotropic Cell Surface Receptors, 364Metabotropic Cell Surface\\nReceptors, 365\\nIntracellular Receptors, 365\\nIntercellular Communication in Action:\\nThe Gastrocnemius Muscle, 365Telling the Muscle to Contract:\\nThe Action of Motoneurones, 367\\nControlling the Blood Supply: Paracrine\\nTransmitters, 368\\nNew Blood Vessels in Growing\\nMuscle, 371\\nSynapses Between Neurons, 372Summary, 376Further Reading, 377Review Questions, 377Answers to Review Questions, 378\\n18 MECHANICAL MOLECULES, 381\\nThe Cytoskeleton is Both Strong and\\nMotile, 381\\nMicrotubules, 381Microtubule-Based Motility, 386\\nCilia and Flagella, 386Intracellular Transport, 389\\nMicro ﬁlaments, 390\\nMuscle Contraction, 393Cell Locomotion, 395Cytoplasmic Streaming, 395\\nIntermediate Filaments, 396\\nAnchoring Cell Junctions, 396\\nSummary, 398Further Reading, 398Review Questions, 398Answers to Review Questions, 400\\n19 CELL CYCLE AND CONTROL OF\\nCELL NUMBER, 401\\nStages of Mitosis, 402Meiosis and Fertilization, 404\\nMeiosis, 405Fertilization and Inheritance, 406Dominant Genetic Disease, 408Crossing Over and Linkage, 408\\nControl of the Cell Division Cycle, 408\\nMolecular Regulation of the G2/M\\n(Interphase/Mitosis) Cell Cycle\\nControl Point, 410\\nWhat About the G1/S Control Point?, 412\\nApoptosis, 415\\nInstructed Death: Death Domain\\nReceptors, 416\\nDefault Death: Absence of Growth\\nFactors, 416\\nThe Sick Are Left to Die:\\nStress-Activated Apoptosis, 417\\nSummary, 419\\nFurther Reading, 420Review Questions, 420Answers to Review Questions, 421\\n20 CASE STUDY: CYSTIC\\nFIBROSIS, 423\\nIntroduction, 423Cystic Fibrosis is a Severe Genetic\\nDisease, 423\\nThe Fundamental Lesion in Cystic Fibrosis\\nLies in Chloride Transport, 424\\nHoming in on the CFGene, 425\\nCloning the Gene for CF, 426TheCFTR Gene Codes for a Chloride Ion\\nChannel, 426\\nGene Therapy for CF, 427Diagnostic Tests for CF, 431', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4271a900-cf42-410f-981e-21461e1f6e10', embedding=None, metadata={'page_label': '11', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GDZ\\nWY001-Bolsover-FM WY001-Bolsover-v3.cls October 22, 2003 14:59\\nCONTENTS xiii\\nThe Future, 432\\nSummary, 433Further Reading, 433Review Questions, 434Answers to Review Questions, 435APPENDIX: CHANNELS AND\\nCARRIERS, 437\\nGLOSSARY, 441INDEX, 501', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='08c6de8b-4802-4008-aa44-196b61e629d1', embedding=None, metadata={'page_label': '12', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GDZ\\nWY001-Bolsover-FM WY001-Bolsover-v3.cls October 22, 2003 14:59\\nPREFACE\\nCell Biology, A Short Course aims to cover a wide area of cell biology in a form especially\\nsuitable for ﬁrst year undergraduates. We have deliberately kept the book to a manageable\\nsize so that neither the cost, the content, nor the weight is too daunting for the student.\\nThe overall theme for the book is the cell as the unit of life. We begin (Chapters 1 –3)\\nby describing the components of the cell as seen under the microscope. We then (Chapters\\n4–8) turn to the central dogma of molecular biology and describe how DNA is used to make\\nRNA which in turn is used to make protein. The next section (Chapters 9 –11) describes\\nhow proteins are delivered to the appropriate location inside or outside the cell, and howproteins perform their many functions. We then (Chapters 12 –14) turn to cell energetics and\\nmetabolism. Signaling within and between cells is covered in Chapters 15 through 17. Toconclude the book, Chapter 18 describes the composition and function of the cytoskeleton,Chapter 19 covers cell birth and cell death, while Chapter 20 uses the example of the commonand severe genetic disease cystic ﬁbrosis to illustrate many of the themes discussed earlier\\nin the book.\\nBoxed material throughout the book is divided into examples to illustrate the topics\\ncovered in the main text, explanations of the medical relevance of the material, and in\\ndepth sections that extend the coverage beyond the content of the main text. Questions are\\nprovided at the end of each chapter to help the reader assess how well they have assimilatedand understood the material.\\nAs well as giving references to printed material, we reference material available on\\nthe internet in many places in the book. Rather than give detailed addresses, we provide\\nlinks to all these sites and many others from the book ’s homepage at\\nhttp://www.physiol.ucl.ac.uk/sbolsover/teaching/cbasc/cbasc.html.\\nxv', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e5911ab2-c03f-4a5f-85cd-033b32531f8c', embedding=None, metadata={'page_label': '13', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GDZ\\nWY001-Bolsover-FM WY001-Bolsover-v3.cls October 22, 2003 14:59\\nACKNOWLEDGMENTS\\nWe thank all the students, colleagues and family members who read the initial versions of\\nthe book and whose suggestions and constructive criticism helped enormously.\\nxvii', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='be55675e-ea76-454b-9fd0-bd2a3fdaeb4f', embedding=None, metadata={'page_label': '14', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GDZ\\nWY001-Bolsover-FM WY001-Bolsover-v3.cls October 22, 2003 14:59\\nINSTRUCTOR NOTES\\nMolecular cell biology courses now form a foundation for many subsequent specializations\\nin areas outside cell biology. We therefore cover molecular genetics, metabolic pathwaysand electrophysiology in suf ﬁcient detail to make Cell Biology a suitable course book for\\nﬁrst year students who will later specialize in genetics, biochemistry, pharmacology or\\nphysiology.\\nEach chapter comprises:\\nrThemain text, with ﬁgures and tables.\\nrA numbered summary .\\nrReview questions with answers for student self-assessment. These questions concern\\nthe main text only; no knowledge of the boxed material is required.\\nrExample boxes that illustrate the points made in the main text.\\nrMedical relevance boxes to show how basic cell biological knowledge illuminates\\nmedical problems or has provided solutions.\\nrIn Depth boxes that extend the content.\\nSelf-assessment questions can form the basics for tutorials, with students asked to defendthe correct answer. They are also easily modi ﬁed to generate new questions for student\\nassessment. Instructors are encouraged to submit new questions for inclusion on the CBASCwebsite.\\nInstructors may wish to specify parts of Cell Biology as core material for courses\\ntargeted to particular specialties. The parts chosen can be customized to the particularspecialty in two ways:\\n1. By selecting from the complete set of twenty chapters. The following sections could\\nbe used to support particular teaching modules:\\nChapters 4 through 7 DNA, RNA and genetic engineering.\\nChapters 8 through 10 Protein synthesis, structure and traf ﬁcking.\\nChapters 11 through 13 Metabolism and cellular energetics.Chapters 14 through 17 Electrophysiology and cell signaling.Chapter 18 The cytoskeleton and cell motility.Chapter 19 Cell division and apoptosis.\\nChapters 16, 17 and 19 might in contrast be selected in a module concentrating\\non the control of development, since these describe how growth factors and otherextracellular chemicals regulate cell division and cell death.\\nxix', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='816fa358-5364-4432-8484-3898939837eb', embedding=None, metadata={'page_label': '15', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GDZ\\nWY001-Bolsover-FM WY001-Bolsover-v3.cls October 22, 2003 14:59\\nxx INSTRUCTOR NOTES\\n2. By including In Depth boxes. The following boxes are especially to be noted:\\nIn Depth 1.2: Fluorescence Microscopy\\nIn Depth 8.1: How We Study Proteins in One Dimension\\ndescribes SDS-PAGE\\nIn Depth 9.1: Chirality and Amino AcidsIn Depth 9.2: Hydropathy Plotting —The PDGF Receptor\\nIn Depth 9.3: Curing Mad Mice with Smelly Fish\\nintroduces the concept of osmolarity and osmosis and extendsthe coverage of chaotropic and structure stabilizing agents\\nIn Depth 11.1: What to Measure in an Enzyme AssayIn Depth 11.2: Determination of V\\nmand K M\\nthe Lineweaver-Burk plot\\nIn Depth 12.2: ATP Synthase, Rotary Motor, and Synthetic MachineIn Depth 12.3: Can It Happen? The Concept of Free EnergyIn Depth 13.1: The Urea Cycle —The First Metabolic Cycle Discovered\\nIn Depth 13.2: The Glyoxylate ShuntIn Depth 14.1: The Nernst EquationIn Depth 14.2: Measuring the Transmembrane V oltageIn Depth 15.1: Frequency Coding in the Nervous SystemIn Depth 16.1: Ryanodine ReceptorsIn Depth 19.1: A Worm ’s Eye View of Cell Death\\nIn Depth 20.1: Lipid Bilayer V oltage Clamp\\nFor example, a course emphasizing protein structure would include In Depth 8.1,\\n9.1, 9.2 and 9.3, while a course concentrating on metabolic pathways would includeIn Depth 13.1 and 13.2.\\nThe CBASC website is maintained by the authors. As well as providing over one hundredlinks to sites with information that extends or illustrates the material in the book, we will usethe site to post typological or other errors, comments and test questions sent to us by read-ers. The full address is http://www.physiol.ucl.ac.uk/sbolsover/teaching/cbasc/cbasc.htmlor simply type ‘CBASC ’into a search engine.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='574353c9-5b3b-44b9-afad-0d5bc23b35f0', embedding=None, metadata={'page_label': '16', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\n1\\nCELLS AND TISSUES\\nThe cellis the basic unit of life. Microorganisms such as bacteria, yeast, and amoebae\\nexist as single cells. By contrast, the adult human is made up of about 30 trillion cells\\n(1 trillion =1012) which are mostly organized into collectives called tissues. Cells are,\\nwith a few notable exceptions, small (Fig. 1.1) with lengths measured in micrometers ( µm,\\nwhere 1000 µm=1 mm) and their discovery stemmed from the conviction of a small group\\nof seventeenth-century microscope makers that a new and undiscovered world lay beyondthe limits of the human eye. These pioneers set in motion a science and an industry thatcontinues to the present day.\\nThe ﬁrst person to observe and record cells was Robert Hooke (1635–1703) who\\ndescribed the cella (open spaces) of plant tissues. But the colossus of this era of discovery\\nwas a Dutchman, Anton van Leeuwenhoek (1632–1723), a man with no university educationbut with unrivaled talents as both a microscope maker and as an observer and recorder ofthe microscopic living world. van Leeuwenhoek was a contemporary and friend of the Delftartist Johannes Vermeer (1632–1675) who pioneered the use of light and shade in art at thesame time that van Leeuwenhoek was exploring the use of light to discover the microscopicworld. Sadly, none of van Leeuwenhoek’s microscopes have survived to the present day.Despite van Leeuwenhoek’s Herculean efforts, it was to be another 150 years before, in 1838,the botanist Matthias Schleiden and the zoologist Theodor Schwann formally proposed thatall living organisms are composed of cells. Their “cell theory,” which nowadays seemsso obvious, was a milestone in the development of modern biology. Nevertheless generalacceptance took many years, in large part because the plasma membrane, the membrane\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n1', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3fb80d8d-cbb2-4f1a-8125-b9658c56e120', embedding=None, metadata={'page_label': '17', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\n2 CELLS AND TISSUES\\nyeast 5 µm human cell \\n20 µmmost \\neukaryotic cells are in the range5 µm–100 µm\\nfrog egg 1 mmsome exceptional cells can be seen with the naked eye\\nciliated protists\\n0.25 mmprokaryotesare the smallesttrue cells. Most are 1–2 µmviruses aresubmicroscopicparticles50–100 nm\\nFigure 1.1. Dimensions of some example cells. 1 mm =10−3m; 1µm=10−6m; 1 nm =10−9m.\\nsurrounding the cell that divides the living inside from the nonliving extracellular medium\\n(Fig. 1.2) is too thin to be seen using a light microscope.\\nPRINCIPLES OF MICROSCOPY\\nMicroscopes make small objects appear bigger. A light microscope will magnify an image\\nup to 1500 times its original size. Electron microscopes can achieve magni ﬁcations up to\\n1 million times. However, bigger is only better when more details are revealed. The ﬁneness\\nof detail that a microscope can reveal is its resolving power. This is de ﬁned as the smallest\\ndistance that two objects can approach one another yet still be recognized as being separate.The resolution that a microscope achieves is mainly a function of the wavelength of the illu-mination source it employs. The smaller the wavelength, the smaller the object that will causediffraction, and the better the resolving power. The light microscope, because it uses visiblelight of wavelength around 500 nanometers (nm, where 1000 nm =1µm), can distinguish\\nobjects as small as about half this: 250 nm. It can therefore be used to visualize the small-est cells and the major intracellular structures or organelles. The microscopic study of cellstructure organization is known as cytology. An electron microscope is required to reveal the\\nultrastructure (theﬁne detail) of the organelles and other cytoplasmic structures (Fig. 1.2).\\nThe wavelength of an electron beam is about 100,000 times less than that of white\\nlight. In theory, this should lead to a corresponding increase in resolution. In practice, the', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='bdb91681-665f-43aa-8dee-fae07a4f8bb4', embedding=None, metadata={'page_label': '18', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\nPRINCIPLES OF MICROSCOPY 3\\nlight microscope image\\nnucleolus\\nnucleus\\nmitochondrionelectron microscope image\\nmitochondrionmitochondrial \\nribosomeGolgi \\napparatus\\nnuclear envelopenuclear porecentrosome\\nmicrotubule\\nrough endoplasmic reticulumvesicle\\nsmooth \\nendoplasmic reticulumcytoplasmheterochromatin\\nplasma membrane\\n10 µm\\n10 µminternal\\nmembranes\\nperoxisomelysosome\\ncytoplasmic ribosome\\nFigure 1.2. Cell structure as seen through the light and transmission electron microscopes.\\nelectron microscope can distinguish structures about 1000 times smaller than is possible in\\nthe light microscope, that is, down to about 0.2 nm in size.\\nThe Light Microscope\\nA light microscope (Figs. 1. 3 aand 1.4) consists of a light source, which may be the sun or\\nan arti ﬁcial light, plus three glass lenses: a condenser lens to focus light on the specimen,\\nanobjective lens to form the magni ﬁed image, and a projector lens, usually called the\\neyepiece, to convey the magni ﬁed image to the eye. Depending on the focal length of the\\nvarious lenses and their arrangement, a given magni ﬁcation is achieved. In bright-ﬁeld', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='7c09d79d-9ea3-429f-9eb2-76b745d365d4', embedding=None, metadata={'page_label': '19', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\n4 CELLS AND TISSUES\\n(a) (b)condenser lens\\nlightlightcameraelectronselectron\\nsource\\nobjective lens\\nprojector lens\\ndisplay monitorimage\\nprocessingcomputersystemimageprojected onfluorescentscreen\\ntransmissionelectronmicroscopespecimenlight\\nsource\\ncondenser lens\\nobjective lenslight\\nmirror\\nintroduced\\nfor direct\\nviewing\\nby eye\\ndisplay monitorimageprocessingcomputersystemprojectorlensimagevieweddirectlyspecimen\\ncamera\\nlight\\nmicroscopedetectorlens\\nreflected\\nelectronselectronselectron\\nsource\\nlensbeam scanner\\nscanningelectronmicroscopespecimen\\n(c)display monitorimageprocessingcomputersystem\\nFigure 1.3. Basic design of light and electron microscopes.\\nProjector lenses\\n(eyepieces)\\nObjective lens\\nSpecimen holder\\nCondenser lens\\nLight sourceFocussing system\\nFigure 1.4. Simple upright light microscope.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='942175c8-172a-4c7e-9f77-9963553164a5', embedding=None, metadata={'page_label': '20', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\nPRINCIPLES OF MICROSCOPY 5\\nBright field microscopy Bright field microscopy\\nFormyl-met-leu-phe added\\nPhase contrast microscopy Phase contrast microscopy\\nFormyl-met-leu-phe added\\nFigure 1.5. Human blood cells viewed by bright-ﬁeld and phase-contrast light microscopy. Arrow\\nindicates a white blood cell. Formyl-met-leu-phe (page 171) causes the white blood cell to spread\\nout and become very thin. It becomes almost invisible by bright-ﬁeld microscopy but can still bedetected by phase-contrast microscopy.\\nmicroscopy, the image that reaches the eye consists of the colors of white light less that\\nabsorbed by the cell. Most living cells have little color (plant cells are an obvious exception)\\nand are therefore largely transparent to transmitted light. This problem can be overcomebycytochemistry, the use of colored stains to selectively highlight particular structures\\nand organelles. However, many of these compounds are highly toxic and to be effectivethey often require that the cell or tissue is ﬁrst subjected to a series of harsh chemical\\ntreatments.\\nA different approach, and one that can be applied to living cells, is the use of phase-\\ncontrast microscopy. This relies on the fact that light travels at different speeds through\\nregions of the cell that differ in composition. The phase-contrast microscope converts thesedifferences in refractive index into differences in contrast, and considerably more detail isrevealed (Fig. 1.5). Light microscopes come in a number of physical orientations (upright,inverted, etc.) but whatever the orientation of the microscope the optical principles are thesame.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='438fe798-5662-47a4-889d-7ff00bac477a', embedding=None, metadata={'page_label': '21', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\n6 CELLS AND TISSUES\\nf\\nIN DEPTH 1.1 Fluorescence Microscopy\\nFluorescent molecules emit light when they are illuminated with light of a shorter\\nwavelength. Familiar examples are the hidden signature in bank passbooks, whichis written in ﬂuorescent ink that glows blue (wavelength about 450 nm) when illu-\\nminated with ultraviolet light (UV) (wavelength about 360 nm), and the whitenerin fabric detergents that causes your white shirt to glow blue when illuminatedby the ultraviolet light in a club. The ﬂuorescent dye Hoechst 33342 has a similar\\nwavelength dependence: It is excited by UV light and emits blue light. However, itdiffers from the dyes used in ink or detergent in that it binds tightly to the DNA inthe nucleus and only ﬂuoresces when so bound. Diagram a shows the optical path\\nthrough a microscope set up so as to look at a preparation stained with Hoechst.White light from an arc lamp passes through an excitation ﬁlter that allows only\\nUV light to pass. This light then strikes the heart of the ﬂuorescent microscope:\\na special mirror called a dichroic mirror that re ﬂects light of wavelengths shorter\\nthan a designed cutoff but transmits light of longer wavelength. To view Hoechst,we use a dichroic mirror of cutoff wavelength 400 nm, which therefore re ﬂects\\nthe UV excitation light up through the objective lens and onto the specimen. AnyHoechst bound to DNA in the preparation will emit blue light. Some of this will be\\nobjective lens\\narc\\nlamp\\nprojector\\nlens\\ncamera\\ndisplay monitorexcitation\\nfilterdichroic\\nmirror\\nimage\\nprocessingcomputersystemspecimen\\nemission\\nfilter\\nmirror\\nintroduced\\nfor direct\\nviewing\\nby eyeimagevieweddirectly(a)\\n(b)\\n', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='7d7d8c72-783f-4d59-b32d-cde3ca4aa24e', embedding=None, metadata={'page_label': '22', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\nPRINCIPLES OF MICROSCOPY 7\\ncaptured by the objective lens and, because its wavelength is greater than 400 nm,\\nwill not be re ﬂected by the dichroic mirror but will instead pass through. An emis-\\nsion ﬁlter, set to pass only blue light, cuts out any scattered UV light. The blue light\\nnow passes to the eye or camera in the usual way. Image b shows a ﬁeld of cells\\ncultured from rat brain (gift of Dr. Charles Krieger, Simon Fraser University) afterstaining with Hoechst. Only the nuclei are seen, as bright ovals.\\nAlthough some of the structures and chemicals found in cells can be selectively\\nstained by speci ﬁcﬂuorescent dyes, others are most conveniently revealed by using\\nantibodies. In this technique an animal (usually a mouse, rabbit, or goat) is injected\\nwith a protein or other chemical of interest. The animal ’s immune system recog-\\nnizes the chemical as foreign and generates antibodies that bind to (and thereforehelp neutralize) the chemical. Some blood is then taken from the animal and theantibodies puri ﬁed. The antibodies can then be labeled by attaching a ﬂuorescent\\ndye. Images c and d show the same ﬁeld of brain cells but with the excitation ﬁlter,\\ndichroic mirror, and emission ﬁlter changed so as to reveal in c a protein called\\nELAV that is found only in nerve cells; then in d an intermediate ﬁlament protein\\n(page 000) found only in glial cells. The antibody that binds to ELAV is labeled withaﬂuorescent dye that is excited by blue light and emits green light. The antibody\\nthat binds to the glial ﬁlaments is labeled with a dye that is excited by green light\\nand emits red light. Because these wavelength characteristics are different, thelocation of the three chemicals —DNA, ELAV, and intermediate ﬁlament —can be\\nrevealed independently in the same specimen. See the CBASC website for an imageof all three signals in color and superimposed.\\n(c)\\n (d)\\nThe technique just described is called primary immunoﬂuorescence and re-\\nquires that the antibody to the chemical of interest be labeled with a dye. Onlyantibodies to chemicals that many laboratories study are so labeled. In order toreveal other chemicals, scientists use secondary immunoﬂuorescence. In this ap-\\nproach, a commercial company injects an animal (e.g., a goat) with an antibodyfrom another animal (e.g., a rabbit). The goat then makes “goat anti rabbit ”anti-\\nbody. This, called the secondary antibody, is puri ﬁed and labeled with a dye. All the\\nscientist has to do is make or buy a rabbit antibody that binds to the chemical ofinterest. No further modi ﬁcation of this specialized, primary antibody is necessary.\\nOnce the primary antibody has bound to the specimen and excess antibody rinsedoff, the specimen is then exposed to the secondary antibody that binds selectivelyto the primary antibody. Viewing the stained preparation in a ﬂuorescence micro-\\nscope then reveals the location of the chemical of interest. The same dye-labeledsecondary antibody can be used in other laboratories or at other times to revealthe location of many different chemicals because the speci ﬁcity is determined by\\nthe unlabeled primary antibody.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='70ead98a-b2f2-4322-8069-1fe6274a2760', embedding=None, metadata={'page_label': '23', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\n8 CELLS AND TISSUES\\nThe Electron Microscope\\nThe most commonly used type of electron microscope in biology is called the transmis-\\nsion electron microscope because electrons are transmitted through the specimen to the\\nobserver. The transmission electron microscope has essentially the same design as a light\\nmicroscope, but the lenses, rather than being glass, are electromagnets that bend beamsof electrons (Fig. 1.3 b). An electron gun generates a beam of electrons by heating a thin,\\nV-shaped piece of tungsten wire to 3000\\n◦C. A large voltage accelerates the beam down\\nthe microscope column, which is under vacuum because the electrons would be slowedand scattered if they collided with air molecules. The magni ﬁed image can be viewed on a\\nﬂuorescent screen that emits light when struck by electrons. While the electron microscope\\noffers great improvements in resolution, electron beams are potentially highly destructive,and biological material must be subjected to a complex processing schedule before it canbe examined. The preparation of cells for electron microscopy is summarized in Figure 1.6.\\nA small piece of tissue (~1 mm3) is immersed in glutaraldehyde \\nand osmium tetroxide. These chemicals bind all the component \\nparts of the cells together; the tissue is said to be fixed . It is \\nthen washed thoroughly.\\nThe tissue is dehydrated  by soaking in acetone or ethanol.\\nThe tissue is embedded  in resin which is then baked hard.\\nSections (thin slices less than 100 nm thick) are \\ncut with a machine called an ultramicrotome.\\nThe sections are placed on a small copper grid  and \\nstained with uranyl acetate and lead citrate. When \\nviewed in the electron microscope, regions that have bound lots of uranium and lead will appear dark because they are a barrier to the electron beam.\\nFigure 1.6. Preparation of tissue for electron microscopy.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='0bdbc1d3-fac4-499d-ace8-2aa545099bb5', embedding=None, metadata={'page_label': '24', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\nONLY TWO TYPES OF CELL 9\\nThe transmission electron microscope produces a detailed image but one that is static,\\ntwo-dimensional, and highly processed. Often, only a small region of what was once a\\ndynamic, living, three-dimensional cell is revealed. Moreover, the picture revealed is essen-tially a snapshot taken at the particular instant that the cell was killed. Clearly, such imagesmust be interpreted with great care. Electron microscopes are large and require a skilledoperator. Nevertheless, they are the main source of information on the structure of the cellat the nanometer scale, called the ultrastructure.\\nThe Scanning Electron Microscope\\nWhereas the image in a transmission electron microscope is formed by electrons transmittedthrough the specimen, in the scanning electron microscope it is formed from electrons thatare re ﬂected back from the surface of a specimen as the electron beam scans rapidly back\\nand forth over it (Fig. 1.3 c). These re ﬂected electrons are processed to generate a picture on\\na display monitor. The scanning electron microscope operates over a wide magni ﬁcation\\nrange, from 10 ×to 100,000 ×. Its greatest advantage, however, is a large depth of focus\\nthat gives a three-dimensional image. The scanning electron microscope is particularlyuseful for providing topographical information on the surfaces of cells or tissues. Moderninstruments have a resolution of about 1 nm.f\\nIN DEPTH 1.2 Microscopy Rewarded\\nSuch has been the importance of microscopy to developments in biology that two\\nscientists have been awarded the Nobel prize for their contributions to microscopy.Frits Zernike was awarded the Nobel prize for physics in 1953 for the developmentof phase-contrast microscopy and Ernst Ruska the same award in 1986 for theinvention of the transmission electron microscope. Ruska ’s prize marks one of the\\nlongest gaps between a discovery (in the 1930s in the research labs of the SiemensCorporation in Berlin) and the award of a Nobel prize. Anton van Leeuwenhoekdied almost two centuries before the Nobel prizes were introduced in 1901 andthe prize is not awarded posthumously.\\nONLY TWO TYPES OF CELL\\nSuper ﬁcially at least, cells exhibit a staggering diversity. Some lead a solitary existence;\\nothers live in communities; some have de ﬁned, geometric shapes; others have ﬂexible\\nboundaries; some swim, some crawl, and some are sedentary; many are green (some are\\neven red, blue, or purple); others have no obvious coloration. Given these differences, it isperhaps surprising that there are only two types of cell (Fig. 1.7). Bacterial cells are said tobeprokaryotic (Greek for “before nucleus ”) because they have very little visible internal\\norganization so that, for instance, the genetic material is free within the cell. They are alsosmall, the vast majority being 1 –2µm in length.\\nThe cells of all other organisms, from protists to mammals to fungi to plants, are eu-\\nkaryotic (Greek for “with a nucleus ”). These are generally larger (5 –100µm, although some\\neukaryotic cells are large enough to be seen with the naked eye; Fig. 1.1) and structurallymore complex. Eukaryotic cells contain a variety of specialized structures known collec-tively as organelles, surrounded by a viscous substance called cytosol. The largest organelle,\\nthenucleus, contains the genetic information stored in the molecule deoxyribonucleic', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1eddd0f4-c5ac-41a0-be19-064ee7641807', embedding=None, metadata={'page_label': '25', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\n10 CELLS AND TISSUES\\nnucleoluschromatin\\nmitochondrion\\nrough \\nendoplasmic \\nreticulum\\nvacuole\\nvacuolenucleus\\nchloroplastcell wallGolgi apparatusprokaryotic \\nribosome\\nDNA\\nbacterial cell\\neukaryotic 1 µm\\nanimal cell\\nplant cellplasmodesma\\n10 µmplasma membrane\\nplasma\\nmembrane\\nFigure 1.7. Organization of prokaryotic and eukaryotic cells.\\nacid (DNA). The structure and function of organelles will be described in detail in subse-\\nquent chapters. Table 1.1 provides a brief glossary of the major organelles and summarizesthe differences between prokaryotic and eukaryotic cells.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example Box 1.1 Sterilization by Filtration\\nBecause even the smallest cells are larger than 1 µm, harmful bacteria and protists can be removed\\nfrom drinking water by passing through a ﬁlter with 200-nm-diameter holes. Filters can vary in\\nsize from huge, such as those used in various commercial processes, to small enough to be easily\\ntransportable by backpackers. Filtering drinking water greatly reduces the chances of bringing backan unwanted souvenir from your camping trip!', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='5a819ab6-94d6-4b03-a5ed-d3c0d79fddd0', embedding=None, metadata={'page_label': '26', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\nVIRUSES 11\\nTable 1.1. Differences Between Prokaryotic and Eukaryotic Cells\\nProkaryotes Eukaryotes\\nSize Usually 1 –2µm Usually 5 –100µm\\nNucleus Absent Present, bounded by nuclear envelope\\nDNA Usually a single circular\\nmolecule ( =chromosome)Multiple molecules ( =chromosomes),\\nlinear, associated with protein.a\\nCell division Simple ﬁssion Mitosis or meiosis\\nInternal membranes Rare Complex (nuclear envelope, Golgi\\napparatus, endoplasmic reticulum,\\netc.—Fig. 1.2)\\nRibosomes 70Sb80S (70S in mitochondria\\nand chloroplasts)\\nCytoskeleton Absent Microtubules, micro ﬁlaments,\\nintermediate ﬁlaments\\nMotility Rotary motor (drives bacterial\\nﬂagellum)Dynein (drives cilia and eukaryote\\nﬂagellum); kinesin, myosin\\nFirst appeared 3 .5×109years ago 1 .5×109years ago\\naThe tiny chromosomes of mitochondria and chloroplasts are exceptions; like prokaryotic chromosomes they are\\noften circular.\\nbThe S value, or Svedberg unit, is a sedimentation rate. It is a measure of how fast a molecule moves in a\\ngravitational ﬁeld, and therefore in an ultracentrifuge.\\nSpecial Properties of Plant Cells\\nAmong eukaryotic cells the most striking difference is between those of animals and plants\\n(Fig. 1.7). Plants have evolved a sedentary lifestyle and a mode of nutrition that meansthey must support a leaf canopy. Their cells are enclosed within a rigid cell wall that givesshape to the cell and structural rigidity to the organism (page 53). This is in contrast totheﬂexible boundaries of animal cells. Plant cells frequently contain one or more vacuoles\\nthat can occupy up to 75% of the cell volume. Vacuoles accumulate a high concentrationof sugars and other soluble compounds. Water enters the vacuole to dilute these sugars,generating hydrostatic pressure that is counterbalanced by the rigid wall. In this way thecells of the plant become stiff or turgid, in the same way that when an inner tube is in ﬂated\\ninside a bicycle tire the combination becomes stiff. Vacuoles are often pigmented, andthe spectacular colors of petals and fruit re ﬂect the presence of compounds such as the\\npurple anthocyanins in the vacuole. Cells of photosynthetic plant tissues contain a specialorganelle, the chloroplast, that houses the light-harvesting and carbohydrate-generating\\nsystems of photosynthesis (page 271). Plant cells lack centrosomes (page 382) (Fig. 1.2)\\nalthough these are found in many algae.\\nVIRUSES\\nViruses occupy a unique space between the living and nonliving worlds. On one hand theyare made of the same molecules as living cells. On the other hand they are incapable ofindependent existence, being completely dependent on a host cell to reproduce. Almostall living organisms have viruses that infect them. Human viruses include polio, in ﬂuenza,\\nherpes, rabies, ebola, smallpox, chickenpox, and the AIDS (acquired immunode ﬁciency', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1f64fe72-333a-4b47-a64b-49061d3a5621', embedding=None, metadata={'page_label': '27', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\n12 CELLS AND TISSUES\\nsyndrome) virus HIV (human immunode ﬁciency virus). Viruses are submicroscopic parti-\\ncles consisting of a core of genetic material enclosed within a protein coat called the capsid.\\nSome viruses have an extra membrane layer called the envelope. Viruses are metabolicallyinert until they enter a host cell, whereupon the viral genetic material directs the host cellmachinery to produce viral protein and viral genetic material. Viruses often insert theirgenome into that of the host, an ability that is widely made use of in molecular genet-ics (Chapter 7). Bacterial viruses, called bacteriophages (page 74), are used by scientists totransfer genes between bacterial strains. Human viruses are used as vehicles for gene therapy.By exploiting the natural infection cycle of a virus such as adenovirus, it is possible to in-troduce a functional copy of a human gene into a patient suffering from a genetic diseasesuch as cystic ﬁbrosis (Chapter 20).\\nORIGIN OF EUKARYOTIC CELLS\\nProkaryotic cells are simpler and more primitive in their organization than eukaryotic cells.According to the fossil record, prokaryotic organisms antedate, by at least 2 billion years,theﬁrst eukaryotes that appeared some 1.5 billion years ago. It seems highly likely that\\neukaryotes evolved from prokaryotes, and the most likely explanation of this process is theendosymbiotic theory. The basis of this hypothesis is that some eukaryotic organelles orig-\\ninated as free-living prokaryotes that were engulfed by larger cells in which they establisheda mutually bene ﬁcial relationship. For example, mitochondria would have originated as\\nfree-living aerobic bacteria and chloroplasts as cyanobacteria, photosynthetic prokaryotes\\nformerly known as blue-green algae.\\nThe endosymbiotic theory provides an attractive explanation for the fact that both mito-\\nchondria and chloroplasts contain DNA and ribosomes of the prokaryotic type (Table 1.1).The case for the origin of other eukaryotic organelles is less persuasive. While it is clearlynot perfect, most biologists are now prepared to accept that the endosymbiotic theory pro-vides at least a partial explanation for the evolution of the eukaryotic cell from a prokaryoticancestor. Unfortunately, living forms having a cellular organization intermediate betweenprokaryotes and eukaryotes are rare. Some primitive protists possess a nucleus but lackmitochondria and other typical eukaryotic organelles. They also have the prokaryotic typeof ribosomes. These organisms are all intracellular parasites and they include Microspora,\\nan organism that infects AIDS patients.\\nCELL SPECIALIZATION\\nAll the body cells that comprise a single organism share the same set of genetic instructionsin their nuclei. Nevertheless, the cells are not all identical. Rather, plants and animals arecomposed of different tissues, groups of cells that are specialized to carry out a common\\nfunction. This specialization occurs because different cell types read out different parts of theDNA blueprint and therefore make different proteins, as we will see in Chapter 6. In animalsthere are four major tissue types: epithelium, connective tissue, nervous tissue, and muscle.\\nEpithelia\\nEpithelia are sheets of cells that cover the surface of the body and line its internal cavities\\nsuch as the lungs and intestine. The cells may be columnar, taller than they are broad, or', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='829538cc-3e81-4777-9677-dd2d93132293', embedding=None, metadata={'page_label': '28', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\nCELL SPECIALIZATION 13\\nsquamous, meaning ﬂat. In the intestine, the single layer of columnar cells lining the inside,\\norlumen, has an absorptive function that is increased by the folding of the surface into villi\\n(Fig. 1.8). The luminal surfaces of these cells have microvilli that increase the surface area\\neven further. The basal surface sits on a supporting layer of extracellular ﬁbers called the\\nbasement membrane. Many of the epithelial cells of the airways, for instance, those lining\\nthe trachea and bronchioles, have cilia on their surfaces. These are hairlike appendages that\\nactively beat back and forth, moving a layer of mucus away from the lungs (Chapter 18).\\nParticles and bacteria are trapped in the mucus layer, preventing them from reaching thedelicate air exchange membranes in the lung. In the case of the skin, the epithelium is saidto be strati ﬁedbecause it is composed of several layers.\\n✄\\n✂\\x00\\n✁Medical\\nRelevance\\n1.1Smoking Is Bad for Your Cilia\\nThe cilia on the epithelial cells that line our airways move a belt of mucus that\\ncarries trapped particles and microorganisms away from the lungs. These cilia are\\nparalyzed by cigarette smoke. As a consequence, smokers have severely reduced mucociliaryclearance with the result that they have to cough to expel the mucus that continuously accumulatesin their lungs. Because of this impaired ciliary function, smokers are much more susceptible torespiratory diseases such as emphysema and asthma. The cilia of passive smokers, people whobreathe smoky air, are also affected.\\nConnective Tissue\\nConnective tissues provide essential support for the other tissues of the body. They include\\nbone, cartilage, and adipose (fat) tissue. Unlike other tissues, connective tissue contains\\nrelatively few cells within a large volume of extracellular matrix that consists of different\\ntypes of ﬁber embedded in amorphous ground substance (Fig. 1.8). The most abundant of\\ntheﬁbers is collagen, a protein with the tensile properties of steel that accounts for about a\\nthird of the protein of the human body. Other ﬁbers have elastic properties that permit the\\nsupported tissues to be displaced and then to return to their original position. The amorphousground substance absorbs large quantities of water, facilitating the diffusion of metabolites,oxygen, and carbon dioxide to and from the cells in other tissues and organs. Of the manycell types found in connective tissue, two of the most important are ﬁbroblasts, which\\nsecrete the ground substance and ﬁbers, and macrophages, which remove foreign, dead,\\nand defective material from it. A number of inherited diseases are associated with defectsin connective tissue. Marfan ’s syndrome, for example, is characterized by long arms, legs,\\nand torso and by a weakness of the cardiovascular system and eyes. These characteristicsresult from a defect in the organization of the collagen ﬁbers.\\nNervous Tissue\\nNervous tissue is a highly modi ﬁed epithelium that is composed of several cell types.\\nPrincipal among these are the nerve cells, also called neurons, along with a variety of\\nsupporting cells that help maintain them. Neurons extend processes called axons, which\\ncan be over a meter in length. Neurons constantly monitor what is occurring inside andoutside the body. They integrate and summarize this information and mount appropriateresponses to it (Chapters 15 –17). Another type of cell called gliahas other roles in nervous\\ntissue including forming the electrical insulation around axons.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e64cf8f1-e6f8-4a17-b44c-770dd9335849', embedding=None, metadata={'page_label': '29', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\n14 CELLS AND TISSUES\\nbrush border\\ngoblet\\ncellone villus\\nred blood cellcapillary\\nsmooth muscle cells \\norientated circularly \\naround intestine\\nsmooth muscle cells \\norientated longitudinally \\nalong intestine (cut in \\ncross section)brush \\nborderone \\nmicrovillus\\nnucleus\\nbasement \\nmembraneepithelial\\ncell\\nepithelial cell\\nconnective\\ntissuefibroblast\\ncollagen\\nfibers\\nextracellular\\nmatrixtight junction\\nFigure 1.8. Tissues and structures of the intestine wall.\\nMuscle\\nMuscle tissue can be of two types, smooth orstriated. Smooth muscle cells are long\\nand slender and are usually found in the walls of tubular organs such as the intestine and\\nmany blood vessels. In general, smooth muscle cells contract slowly and can maintainthe contracted state for a long period of time. There are two classes of striated muscle:cardiac andskeletal. Cardiac muscle cells make up the walls of the heart chambers. These', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='8bdf1c81-9b4f-411c-b08a-e31abdf2c3e7', embedding=None, metadata={'page_label': '30', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\nCELL SPECIALIZATION 15\\nAA\\nC\\nCD\\nDB B\\nstomacuticle\\nepidermis\\nparenchyma \\ncells\\ncortexroot hair\\nvascular\\ntissuevascular\\ntissue\\nair \\nspace\\nvascular\\ntissue\\nepidermal cellparenchymacell cortexupper surfaceof leaf\\nunder surface\\nof leafvascular\\ntissue\\nFigure 1.9. Plants are composed of several tissues.\\nare branched cells that are connected electrically by gap junctions (page 55), and their\\nautomatic rhythmical contraction powers the beating of the heart. Each skeletal muscle isa bundle of hundreds to thousands of ﬁbers, each ﬁber being a giant single cell with many\\nnuclei. This rather unusual situation is the result of an event that occurs in the embryo whenthe cells that give rise to the ﬁbers fuse together, pooling their nuclei in a common cytoplasm.\\nThe mechanism of contraction of skeletal muscle will be described in Chapter 18.\\nPlants\\nPlant cells are also organized into tissues (Fig. 1.9). The basic organization of a shoot or rootis into an outer protective layer, or epidermis, avascular tissue that provides support and', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e13444b1-be1a-412a-bf1c-6539645baddf', embedding=None, metadata={'page_label': '31', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\n16 CELLS AND TISSUES\\ntransport, and a cortex thatﬁlls the space between the two. The epidermis consists of one or\\nmore layers of closely packed cells. Above the ground these cells secrete a waxy layer, the\\ncuticle, which helps the plant retain water. The cuticle is perforated by pores called stomatathat allow gas exchange between the air and the photosynthetic cells and also constitute themajor route for water loss by a process called transpiration. Below ground, the epidermal\\ncells give rise to root hairs that are important in the absorption of water and minerals. Thevascular tissue is composed of xylem, which transports water and its dissolved solutes\\nfrom the roots, and phloem, which conveys the products of photosynthesis, predominantly\\nsugars, to their site of use or storage. The cortex consists primarily of parenchyma cells,\\nunspecialized cells whose cell walls are usually thin and bendable. They are the major siteof metabolic activity and photosynthesis in leaves and green shoots.\\nSUMMARY\\n1.All living organisms are made of cells.\\n2.Our understanding of cell structure and function has gone hand in hand with devel-opments in microscopy and its associated techniques.\\n3.Light microscopy revealed the diversity of cell types and the existence of the majororganelles: nucleus, mitochondrion and, in plants, the vacuole and chloroplast.\\n4.The electron microscope revealed the detailed structure of the larger organelles andresolved the cell ultrastructure, the ﬁne detail at the nanometer scale.\\n5.There are two types of cells, prokaryotes and eukaryotes.\\n6.Prokaryotic cells have very little visible internal organization. They usually measure1–2µm across.\\n7.Eukaryotic cells usually measure 5 –100µm across. They contain a variety of spe-\\ncialized internal organelles, the largest of which, the nucleus, contains the geneticmaterial.\\n8.The endosymbiotic theory proposes that some eukaryotic organelles, such as mito-chondria and chloroplasts, originated as free-living prokaryotes.\\n9.The cells of plants and animals are organized into tissues. In animals there are fourtissue types: epithelium, connective tissue, nervous tissue, and muscle. Plants areformed of epidermis, cortex, and vascular tissues.\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.Eukaryotic cells usually contain\\nA. a nucleus.\\nB. mitochondria.C. ribosomes.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='033e27cb-8203-4c2e-a4ea-e2634ae70ade', embedding=None, metadata={'page_label': '32', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\nANSWERS TO REVIEW QUESTIONS 17\\nD. microtubules.\\nE. all of the above.\\n2.The transmission electron microscope\\nA. is ideal for looking at living cells.B. has a resolution of 0.2 nm.C. uses lenses made of glass.D. was invented by Anton van Leeuwenhoek.E. is easily transportable.\\n3.Which of the following statements about tissues is incorrect?\\nA. Microvilli are typically associated with nervous tissue.B. Columnar and squamous are types of epithelial tissue.C. Collagen is a component of the extracellular matrix.D. Glial cells form the insulation around nerve cells.E. Epidermis, vascular tissue, and cortex are types of plant tissue.\\n4.1,000,000 nm is equal to\\nA. 1µm.\\nB. 10µm.\\nC. 1 mm.D. 10 mm.E .1m .\\n5.Unlike eukaryotes, prokaryotes lack\\nA. a plasma membrane.B. DNA.C. ribosomes.D. nuclei.E. molecular motors.\\n6.Light passes through a light microscope in the following order:\\nA. condenser, specimen, objective, projector.B. condenser, objective, specimen, projector.C. condenser, projector, specimen, objective.D. objective, specimen, projector, condenser.E. objective, projector, specimen, condenser.\\n7.Which of the following is not a type of muscle cell?\\nA. Smooth.B. Ciliated.C. Striated.D. CardiacE. Skeletal.\\nANSWERS TO REVIEW QUESTIONS\\n1.E.These are all typical features of eukaryotic cells.\\n2.B.None of the others are true. In particular the fact that an electron beam is used means that the\\nlenses are electromagnets, not glass, and means that the interior of an electron microscope is heldat a vacuum, which means in turn that it is not a suitable environment for living tissue and tendsto mean that electron microscopes are bulky, heavy pieces of equipment.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='871d4127-feaf-4d1f-a287-7831513590ff', embedding=None, metadata={'page_label': '33', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-01 WY001-Bolsover-v2.cls September 16, 2003 10:24\\n18 CELLS AND TISSUES\\n3.A.Microvilli are found on epithelia, for example, the epithelium lining the intestine.\\n4.C.The international system pre ﬁxes used in this book are m for milli, meaning 10−3,µfor micro,\\nmeaning 10−6, and n for nano meaning 10−9. Thus 1,000,000 times 10−9is 10−3so 1,000,000 nm\\nis equal to 1 mm.\\n5.D.Eukaryotes have all of these, but prokaryotes do not have nuclei. Rather, the DNA is free in the\\ncytoplasm.\\n6.A\\n7.B.No muscle cells are ciliated; as we will see in Chapter 18, the molecular motor systems used\\nin cilia and in muscles are different. Muscle cells are divided into smooth and striated; the striated\\ntype is then further subdivided into cardiac and skeletal.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='286e66b0-5048-450a-bd14-c9eea189879d', embedding=None, metadata={'page_label': '34', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\n2\\nFROM WATER TO DNA: THE\\nCHEMISTRY OF LIFE\\nOrganisms are made up of a lot of different chemicals. These vary in size, from small\\nmolecules like water to large molecules like DNA, and interact and associate in manydifferent ways to generate the processes of life. In this chapter we will introduce some basicconcepts of how these chemicals are made and interact. We will then describe the mostimportant of these chemicals: water, carbohydrates, nucleotides, amino acids, and lipids.\\nTHE CHEMICAL BOND: SHARING ELECTRONS\\nWater is the most abundant substance in organisms. Cells are rich in water. Cytoplasmconsists of organelles ﬂoating in a watery medium called cytosol that also contains proteins.\\nThe situation is not so different outside our cells. Although we are land animals living inair, most of our cells are bathed in a watery ﬂuid called extracellular medium . We will\\ntherefore start by considering water itself.\\nFigure 2.1 ashows a molecule of water, consisting of one atom of oxygen and two\\nhydrogen atoms, joined to form an open V shape. The lines represent covalent bondsformed when atoms share electrons, each seeking the most stable structure. Oxygen has agreater afﬁnity for electrons than does hydrogen so the electrons are not distributed equally.The oxygen grabs a greater share of the available negative charge than do the hydrogenatoms. The molecule of water is polarized, with partial negative charge on the oxygen andpartial positive charges on the two hydrogens. We write the charge on each hydrogen as δ+\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n19', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2a01c7a6-2eb3-4023-b557-3d2830895d8c', embedding=None, metadata={'page_label': '35', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\n20 FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\n(a) (b)δ+\\n2δ−H Cl ClH\\n Oδ+\\nFigure 2.1. Water is a polar molecule while the chlorine molecule is nonpolar.\\nto indicate that it is smaller than the charge on a single hydrogen nucleus. The oxygen atom\\nhas the small net negative charge 2 δ−. Molecules that, like water, have positive regions\\nsticking out one side and negative regions sticking out the other are called polar .\\nFigure 2.1 bshows a molecule of chlorine gas. It consists of two chlorine atoms, each of\\nwhich consists of a positively charged nucleus surrounded by negatively charged electrons.Like oxygen, chlorine atoms tend to accept electrons when they become available, but thebattle is equal in the chlorine molecule: The two atoms share their electrons equally and themolecule is nonpolar .\\n− + − + −\\n+ − + − +\\n− + − + −\\n+ − + − +Cl Cl−e−\\nCl Cl−\\nNa Na+e−\\n(a)\\n(b)Na+Na\\nFigure 2.2. Formation of sodium chloride, an ionic compound.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='f9ad380e-aa9d-45f0-9690-5ca40cde9e92', embedding=None, metadata={'page_label': '36', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\nINTERACTIONS WITH WATER: SOLUTIONS 21\\nFigure 2.2 ashows what happens when a chlorine molecule is allowed to react with\\nthe metal sodium. Each atom of chlorine takes over one electron from a sodium atom. This\\nleaves the sodium atoms with a single positive charge because there is now one more positivecharge on the sodium nucleus than negatively charged surrounding electrons. Similarly,each chlorine atom now has a single negative charge because it now has one more electronthan there are positive charges in its nucleus. Chemical species that have either gained orlost electrons, and that therefore bear an overall charge, are called ions. The reaction of\\nchlorine and sodium has produced sodium ions and chloride ions. Positively charged ionslike sodium are called cations while negatively charged ones like chloride are called anions .\\nThe positively charged sodium ions and the negatively charged chloride ions now attracteach other strongly. If there are no other chemicals around, the ions will arrange themselvesto minimize the distance between sodium and chloride, and the resulting well-packed arrayof ions is a crystal of sodium chloride, shown in Figure 2.2 b.\\nINTERACTIONS WITH WATER: SOLUTIONS\\nIonic Compounds Will Dissolve Only in Polar Solvents\\nFigure 2.3 ashows one molecule of octane, the main constituent of gasoline. Octane is an\\nexample of a nonpolar solvent. Electrons are shared equally between carbon and hydrogen,and the component atoms do not bear a net charge.\\nFigure 2.3 bshows a small crystal of sodium chloride immersed in octane. At the edge\\nof the crystal, positively charged sodium ions are being pulled in toward the center ofthe crystal by the negative charge on chloride ions, and negatively charged chloride ionsare being pulled in toward the center of the crystal by the positive charge on sodium ions.The sodium and chloride ions will not leave the crystal. Sodium chloride is insoluble inoctane. However, sodium chloride will dissolve in water, and Figure 2.4 ashows why. The\\nchloride ion at the top left is being pulled into the crystal by the positive charge on its sodiumion neighbors, but at the same time it is being pulled out of the crystal by the positive chargeon the hydrogen atoms of nearby water molecules. Similarly, the sodium ion at the bottomleft is being pulled into the crystal by the negative charge on its chloride ion neighbors, butat the same time it is being pulled out of the crystal by the negative charge on the oxygenatoms of nearby water molecules. The ions are not held in the crystal so tightly and canleave. Once the ions have left the crystal, they become surrounded by a hydration shell of\\nwater molecules, all oriented in the appropriate direction (Fig. 2.4 b)—oxygen inward for\\na positive ion like sodium, hydrogen inward for a negative ion like chloride. A chemicalspecies in solution, whether in water or in any other solvent, is called a solute . Liquids\\nwhose main constituent is water are called aqueous .\\nAcids Are Molecules That Give H+to Water\\nWhen we exercise, our muscle cells can become acid, and this is what creates the painof cramping muscles and the heart pain of angina. Acidity is important in all areas ofbiology, from the acidity gradient that drives our mitochondria (page 261) to the ecologicalconsequences of acid rain.\\nAcid solutions contain a high concentration of hydrogen ions. The hydrogen atom is\\nunusual in that it only has one electron while, in its most common isotope, its nucleus', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='15c3d8ae-c580-4748-bbee-18f294f82a9a', embedding=None, metadata={'page_label': '37', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\n22 FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\nC\\nCH\\nH\\nHC\\nCH\\nH\\nHHC\\nCH\\nH\\nHHC\\nC\\nCHH\\nH\\nH\\nHHH\\n(a)\\n− + − + −\\n+ − + − +\\n− + − + −\\n+ − + − +octanesodium chloride\\n(b)\\nFigure 2.3. (a) Structure of the nonpolar compound octane. ( b) Ionic compounds are insoluble in\\nnonpolar solvents.\\ncomprises a single proton. In gasses at very low pressure it is possible for bare protons\\nto exist alone and be manipulated, for example, in linear accelerators. However, in waterprotons never exist alone but always associate with another molecule, for example, withwater to create the H\\n3O+ion. Acid solutions are those with an H 3O+concentration higher\\nthan 100 nmol liter−1.\\nSour cream contains lactic acid. Pure lactic acid has the structure shown at the left of\\nFigure 2.5 a. The —COOH part in the box is called a carboxyl group . Both oxygens have a\\ntendency to pull electrons away from the hydrogen and, in aqueous solution, the hydrogenis donated with a full positive charge to a molecule of water. The electron is left behind onthe now negatively charged lactate ion.\\nCH\\n3CH(OH)COOH +H2O→CH 3CH(OH)COO−+H3O+', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d6ce51e1-e61a-4d00-bde7-630f6a78d8fd', embedding=None, metadata={'page_label': '38', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\nINTERACTIONS WITH WATER: SOLUTIONS 23\\n− + − + −\\n+ − + − +\\n− + − + −\\n+ − + − +water\\nδ+δ+2δ−\\n(a)\\n(b)Na\\nCl\\nH\\nHO\\nHH\\nOHH\\nOH H\\nO\\nH H\\nO\\nH\\nHOH\\nHO\\nH\\nHO\\nCl\\nH\\nHO\\nHH\\nOH\\nHOH\\nHOH H\\nOH H\\nO\\nH\\nHOH\\nHOH\\nHOH\\nHO\\nNaH\\nHOH\\nHOH H\\nOH H\\nO\\nH\\nHOH\\nHOH\\nHOH\\nHO\\nHH\\nOH H\\nO\\nH H\\nO\\nH\\nHOH\\nHO\\nH\\nHO\\nH H\\nOHH\\nOH\\nHOH\\nH OHH\\nOH\\nHO\\nH\\nHOHH\\nOH\\nHO\\nH\\nH\\nO H\\nHOH\\nHOHH\\nO H\\nHOH\\nH\\nO\\nH\\nHO\\nH\\nHOH HOHH\\nOH H\\nOH H\\nO H\\nHOH\\nHO\\nHH\\nOH\\nHOHH\\nOH\\nH O\\nHH\\nO\\nH\\nHOH\\nHO\\nH\\nHOH\\nH O\\nH\\nHOH\\nHOH H\\nOH H\\nO\\nsodium chloride\\nFigure 2.4. Ionic compounds dissolve readily in water.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='31c45c53-08fd-4b83-9818-bc297b13413b', embedding=None, metadata={'page_label': '39', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\n24 FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\nH3O+H2O\\nCO C\\nHOH\\nH\\nH\\nlactic acid lactate ionH\\nCOH\\nCO C\\nHOH\\nH\\nHH\\nCO\\nCH3N\\nCH3\\ntrimethylamineCH3\\nCH3H\\nCH3\\nprotonated\\ntrimethylamine ionCH3H3O+H2O(a)\\n(b)N\\nFigure 2.5. Acids and bases, respectively, give up and accept H+when dissolved in water.\\nFor convenience we often write this as\\nCH 3CH(OH)COOH →CH 3CH(OH)COO−+H+\\nHere we are using H+as a convenient symbol to denote H 3O+. We do not mean that there\\nare real H+ions, that is, bare hydrogen nuclei, in aqueous solutions.\\nThe equilibrium constant, Kafor the dissociation of lactic acid, is de ﬁned as\\nKa=[CH 3CH(OH)COO−]e[H+]e\\n[CH 3CH(OH)COOH]e\\nwhere the square brackets refer, by convention, to concentrations and the subscripts, e,\\ndenote that these are the concentrations of each species at equilibrium.\\nDissolving lots of lactic acid in water produces an acid solution, that is, one with a high\\nconcentration of H+(really H 3O+) ions. For historical reasons, the acidity of a solution is\\ngiven as the pH,deﬁned thus:\\npH=− log10([H+]) where [H+] is measured in moles per liter\\nPure water has a pH of 7, corresponding to [H+]=100 nmol liter−1—this is said to be\\nneutral as regards pH. If the pH is lower than this, then there is more H+about and the\\nsolution is acid. Cytosol has a pH that lies very slightly on the alkaline side of neutrality, at\\nabout 7.2.\\nThe pH of a solution determines the ratio of lactate to undissociated lactic acid. As\\nthe H+concentration rises and pH falls, the equilibrium is pushed over from lactate toward\\nlactic acid. At pH 3.9, the concentrations of lactate and lactic acid become equal. Looking at', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='955b4775-167b-4e34-b785-d071f01d517c', embedding=None, metadata={'page_label': '40', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\nINTERACTIONS WITH WATER: SOLUTIONS 25\\nthe equation above, we can therefore see that when this happens Ka=[H+]. Just as acidity\\nis given on the logarithmic pH scale so is the scale of strengths of different acids, p Kabeing\\ndeﬁned as −log10Ka. The p Kais the pH at which the concentration of the dissociated acid\\nis equal to the concentration of the undissociated acid. For an acid that is weaker than lactic\\nacid, the p Kais higher than 3.9, meaning that the acid is less readily dissociated and needs\\na lower concentration of protons before it will give up its H+. For an acid that is stronger\\nthan lactic acid, the p Kais less than 3.9: Such an acid is more readily dissociated and needs\\na higher H+concentration before it will accept H+and form undissociated acid.\\nBases Are Molecules That Take H+from Water\\nTrimethylamine is the compound that gives rotting ﬁsh its unpleasant smell. Pure trimethy-\\nlamine has the structure shown at the left of Figure 2.5 b. When trimethylamine is dissolved\\nin water, it accepts an H+to become the positively charged trimethylamine ion shown on the\\nright of the ﬁgure. We refer to molecules that have accepted H+ions as protonated, using\\n“proton ”as a short way of saying “hydrogen nucleus. ”Dissolving lots of trimethylamine\\nin water produces an alkaline solution, that is, one with a low concentration of H+ions and\\nhence a pH greater than 7. The solution never runs out of H+completely because new H+\\nare formed from water:\\nH2O→OH−+H+\\nThus if we keep adding trimethylamine to water and using up H+, we end up with a low\\nconcentration of H+, but lots of OH−.\\nThe pH of a solution determines the position of equilibrium between protonated and\\ndeprotonated trimethylamine, and as before we de ﬁne the p Kaas the pH at which the\\nconcentration of protonated and deprotonated base are the same. The p Kaof trimethylamine\\nis 9.74, meaning that the concentration of H+must fall to the low level of 10−9.74mol liter−1,\\nthat is, 0.2 nmol liter−1, before half of the trimethylamines will give up their H+s.\\nIsoelectric Point\\nThe large molecules called proteins (page 183) have many acidic and basic sites that willgive up or accept an H\\n+as the pH changes. In alkaline solutions proteins will tend to have\\nan overall negative charge because the acidic sites have lost an H+and bear a negative\\ncharge. As the pH falls, the acidic sites accept an H+and become uncharged, and basic\\nsites also accept an H+to gain a positive charge. Thus as the pH falls from an initial high\\nvalue, the overall charge on the protein becomes less and less negative and then more andmore positive. The pH at which the protein has no overall charge is called the isoelectric\\npoint . The isoelectric points of different proteins are different, and this property is useful\\nin separating them during analysis (page 178). The majority of intracellular proteins havean isoelectric point that is less than 7.2, so that at normal intracellular pH they bear a netnegative charge.\\nA Hydrogen Bond Forms When a Hydrogen Atom Is Shared\\nWe have seen how oxygen tends to grab electrons from hydrogen, forming a polar bond.Nitrogen and sulfur are similarly electron-grabbing. If a hydrogen attached to an oxygen,', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6d2e183b-0075-43ab-816b-d1b169832151', embedding=None, metadata={'page_label': '41', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\n26 FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\nHH\\nN\\nCC\\nN C\\nN C\\nHN\\nC\\nN\\ndeoxyriboseHH\\nH\\nCC\\nCC\\nH C\\ndeoxyriboseOO\\nNHH\\nH\\nNH\\nthymine (T) adenine (A)a hydrogen bond \\ncannot form here because the N H N atoms are not in a straight lineO\\nOH\\nHHH\\nO\\n(a)\\n(b)hydrogen bond\\nFigure 2.6. The hydrogen bond.\\nnitrogen or sulfur by a covalent bond gets close to a second electron-grabbing atom; then that\\nsecond atom also grabs a small share of the electrons to form what is known as a hydrogen\\nbond . The atom to which the hydrogen is covalently bonded is called the donor because it\\nis losing some of its share of electrons; the other electron-grabbing atom is the acceptor .\\nFor a hydrogen bond to form, the donor and acceptor must be within a ﬁxed distance of one\\nanother (typically 0.3 nm) with the hydrogen on a straight line between them.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='48cbc859-ed64-4ca7-8806-7017808351a0', embedding=None, metadata={'page_label': '42', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\nCARBOHYDRATES: CANDY AND CANES 27\\nLiquid water is so stable because the individual molecules can hydrogen bond, as\\nillustrated in Figure 2.6 a. Hydrogen bonding also plays a critical role in allowing DNA to\\nstore and replicate genetic information. Figure 2.6 bshows how the base pairs (page 69)\\nof DNA form hydrogen bonds in which hydrogen atoms are shared between nitrogen and\\noxygen and between nitrogen and nitrogen.\\nBIOLOGICAL MACROMOLECULES\\nVery large molecules, or macromolecules, are central to the working of cells. Large bi-\\nological molecules are polymers: they are assembled by joining together small, simpler\\nmolecules, which are therefore called monomers . Chemical technology has mimicked\\nnature by producing many important polymers —polyethylene is a polymer of ethylene\\nmonomers. Cells make a number of macromolecules that we will introduce, together withtheir monomer building blocks, in this chapter.\\nCARBOHYDRATES: CANDY AND CANES\\nCarbohydrates —sugars and the macromolecules built from them —have many different\\nroles in cells and organisms.\\nAn Assortment of Sweets\\nAll carbohydrates are formed from the simple sugars called monosaccharides . Figure 2.7\\nshows the monosaccharide called glucose. The form shown at the top has ﬁve carbons\\njoined each to the other with an oxygen atom completing the ring. As well as the oxygenin the ring, glucose has ﬁve other oxygens, each in an −OH ( hydroxyl ) group. Glucose\\neasily switches between the three forms, or isomers, illustrated in Figure 2.7. The two\\nring structures are stereo isomers: although they comprise the same atoms connected by\\nthe same bonds, they represent two different ways of arranging the atoms in space. Thetwo stereo isomers, named αandβ, continually interconvert in solution via the open-chain\\nform.\\nFigure 2.8 shows ﬁve other monosaccharides that we will meet again in this book.\\nLike glucose, each of these monosaccharides can adopt an open-chain form and a numberof ring structures. In Figure 2.8 we show each sugar in a form that it adopts quite often.These sugars share with glucose the two characteristics of monosaccharides: they can adopta form in which an oxygen atom completes a ring of carbons, and they have many hydroxylgroups. The generic names for monosaccharides are derived from the Greek for the numberof their carbon atoms, so glucose, galactose, mannose, and fructose are hexoses (6 carbons)while ribose and ribulose are pentoses (5 carbons). Classically, a monosaccharide has thegeneral formula C\\nn(H2O)n, hence the name carbohydrate. All the monosaccharides shown\\nin Figures 2.7 and 2.8 ﬁt this rule —the four hexoses can be written as C 6(H2O)6and the\\ntwo pentoses as C 5(H2O)5.\\nIt is worth noting that although both sugars and organic acids contain an —OH group,\\nthe behavior of an —OH group that forms part of a carboxyl group is very different from\\none that is not next to a double-bonded oxygen. In general the —OH group in a carboxyl\\ngroup will readily give up an H+to water or other acceptors; this is not true of —OH groups', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ab87c63c-62b6-43d1-80c6-327246db8a29', embedding=None, metadata={'page_label': '43', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\n28 FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\nα-glucoseCCH2OH\\nHO\\nC\\nCOHH\\nOH\\nHCH\\nOHOHCH\\nβ-glucoseCCH2OH\\nHO\\nC\\nCOHH\\nOH\\nHCH\\nOHC\\nHOHCHO\\nCO HH\\nCH HO\\nCO HH\\nCO H H\\nCH2OHglucose in open\\nchain form5\\n4\\n32\\n1\\n2\\n3\\n4\\n5\\n6\\n1\\n6\\n5\\n4\\n3216\\nFigure 2.7. Glucose, a monosaccharide, easily switches between three isomers.\\nin general. We therefore reserve the term hydroxyl group for—OH groups on carbon atoms\\nthat are not also double-bonded to oxygen.\\nDisaccharides\\nMonosaccharides can be easily joined by glycosidic bonds in which the carbon backbones\\nare linked through oxygen and a water molecule is lost. The bond is identi ﬁed by the', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4143ccb4-f6bf-4398-af46-2f1562a56b3f', embedding=None, metadata={'page_label': '44', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\nCARBOHYDRATES: CANDY AND CANES 29\\nβ-riboseO\\nC\\nCHCH2OH\\nH\\nOHOH\\nCH\\nOHC\\nHOH5\\n4\\n321α-fructoseO\\nC\\nCHCH2OH\\nH\\nCOH\\nHC\\nOHCH2OH6\\n5\\n4321\\nα-ribuloseO\\nC\\nCHH\\nH\\nOHCH\\nOHC\\nOHCH2OH\\n5\\n4321β-galactoseCCH2OH\\nHO\\nC\\nCOH\\nHCH\\nOHC\\nH HOH OH\\n4\\n21\\nα-mannoseCCH2OH\\nHO\\nC\\nCOH\\nHCOH\\nHC\\nOH OHH H6\\n55\\n4\\n33\\n216\\nFigure 2.8. Some monosaccharides, each shown as a common isomer.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='09df890f-e2e4-450c-b83a-bdad667c458a', embedding=None, metadata={'page_label': '45', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\n30 FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\nCCH2OH\\nHO\\nC\\nCOHH\\nO\\nHCH\\nOHOHCH\\nCCH2OH\\nHO\\nC\\nCOHOH\\nH\\nHCH\\nOHC\\nHglucose residue\\ngalactose residue6\\n5\\n4\\n321\\n56\\n4\\n321\\nFigure 2.9. The disaccharide lactose.\\ncarbons linked. For example, Figure 2.9 shows lactose, a sugar found in milk that is formed\\nwhen galactose and glucose are linked by a (1 →4) glycosidic bond. Sugars formed from\\ntwo monosaccharide monomers are called disaccharides . Whenever a more complicated\\nmolecule is made from simpler building blocks, we call the remnant of the individualbuilding blocks residues . Thus lactose is made of one galactose residue and one glucose\\nresidue.\\nA complication arises here. Although free monosaccharides can easily switch between\\ntheαandβforms, formation of the glycosidic bond locks the shape. Thus, although\\ngalactose in solution spends equal time in the αandβforms, the galactose residue in\\nlactose is locked into the βform, so the full speci ﬁcation of the bond in lactose is β1→4.\\nOut of the Sweet Comes Forth Strength\\nFormation of glycosidic bonds can continue almost inde ﬁnitely. A chain of up to 100 or so\\nmonosaccharides is called an oligosaccharide, from the Greek word oligo meaning few.\\nLonger polymers are called polysaccharides and have characteristics very different from\\nthose of their monosaccharide building blocks. Figure 2.10 ashows part of a molecule of\\nglycogen, a polymer made exclusively of glucose monomers organized in long chains with\\nα1→4 links. The glycogen chain branches at intervals, each branch being an α(1→4) linked\\nchain of glucose linked to the main chain with an α(1→6) bond. Solid lumps of glycogen\\nare found in the cytoplasm of muscle, liver, and some other cells. These glycogen granulesare 10 –40 nm in diameter with up to 120,000 glucose residues. Glycogen is broken down\\nto release glucose when the cell needs energy (Chapter 13).\\nCellulose makes up the cell wall of plants and is the world ’s most abundant macro-\\nmolecule. Like glycogen, cellulose is a polymer of glucose, but this time the links areβ1→4. It is shown in Figure 2.10 b. Drawn on the ﬂat page, it looks very like glycogen.\\nHowever, the difference in bond type is critical. Glucoses linked by α(1→4) links arrange\\nthemselves in a ﬂoppy helix, while glucoses linked by β(1→4) links form extended chains,', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2c2af335-141b-43da-8e47-eb7d3f4db627', embedding=None, metadata={'page_label': '46', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\nCARBOHYDRATES: CANDY AND CANES 31\\nCCH2OH\\nHO\\nOOC\\nCOHH\\nHCH\\nOHCH CO\\nCH2\\nHO\\nOC\\nCOHH\\nHCH\\nOHCH CCH2OH\\nHO\\nOC\\nCOHH\\nHCH\\nOHCH6\\n5\\n4\\n3216\\n5\\n4\\n3216\\n5\\n4\\n321OO\\nO\\nCHCOH\\nH\\nCH\\nOHCHO\\nCCHCH\\nCCH2OH\\nH6\\n55\\n12346\\n1234\\n(a) glycogen\\nCCH2OH\\nHO\\nOO\\nC\\nCOHH\\nHC\\nHC\\nH CH\\nOHCH\\nOH\\nO\\nCH\\nCH2OHOCH CCH2OH\\nHO\\nO\\nC\\nCOHH\\nHC\\nH CH\\nOH6\\n5\\n4\\n3213COH\\nH2\\n4\\n5\\n616\\n5\\n4\\n321\\n(b) celluloseCOH\\nH\\nCH\\nOHCCH2OH\\nH\\nFigure 2.10. The polysaccharides glycogen and cellulose.\\nideal for building the rigid plant cell wall. Animals have enzymes (protein catalysts) that\\ncan break down the α(1→4) bond in glycogen, but only certain bacteria and fungi can break\\ntheβ(1→4) link in cellulose. All animals that eat plants rely on bacteria in their intestines\\nto provide the enzymes to digest cellulose.\\nModiﬁed Sugars\\nA number of chemically modi ﬁed sugars are important in biology. Figure 2.11 shows\\nthree. Deoxyribose is a ribose that is missing the —OH group on carbon number two. Its', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='9a59f43b-3601-4797-bb15-16751293b6cd', embedding=None, metadata={'page_label': '47', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\n32 FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\nO\\nC\\nCHCH2OH\\nH\\nOHCH\\nHC\\nHOH\\n4\\n321\\nCC\\nHO\\nO\\nC\\nCOH\\nHCOH\\nHC\\nOH OHH H6\\n5\\n4\\n321deoxyribose\\nN-acetyl glucosamine\\nmannose-6-phosphateCCH2OH\\nHO\\nC\\nCOH\\nHCH\\nNH\\nC\\nCH3\\nP O\\nH HOC\\nH OHOH H\\n4\\n216\\n5\\n3\\nO\\nO5\\nFigure 2.11. Deoxyribose, N-acetyl glucosamine and mannose-6-phosphate are modiﬁed sugars.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='5530556e-0ab4-42de-98b6-1d518036bdc5', embedding=None, metadata={'page_label': '48', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\nCARBOHYDRATES: CANDY AND CANES 33\\nmain use is in making deoxyribonucleic acid —DNA. N-acetyl glucosamine is a glucose in\\nwhich an —NHCOCH 3group replaces the —OH on carbon two. It is used in a number of\\noligosaccharides and polysaccharides, for example, in chitin, which forms the hard parts of\\ninsects. Mannose-6-phosphate is a mannose that has been phosphorylated, that is, had\\naphosphate group attached, on carbon number six. We will meet more phosphorylated\\nsugars in the next section.\\nadenine\\nN\\nCC\\nN C\\nN C\\nHN\\nC\\nNHH\\nH\\nCH2HO\\nriboseO\\nC\\nC CC HH\\nOHH H\\nOH\\nadenine\\nN\\nCC\\nN C\\nN C\\nHN\\nC\\nNHH\\nH\\nriboseO\\nC\\nC CC HH\\nOHH H\\nOHtriphosphate\\nCH2O P\\nOO P\\nOO P\\nO\\nγ\\nphosphate β\\nphosphateα\\nphosphateγ\\nphosphate β\\nphosphateα\\nphosphateO O O\\nO(a) adenosine, a nucleoside\\n(b) adenosine triphosphate, a nucleotide5′\\n4′ 1′\\n1′\\n2′ 3′4′5′2′ 3′\\nFigure 2.12. Adenosine, a nucleoside, and adenosine triphosphate, a nucleotide.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='f5d2b11d-71e0-4162-9c2f-8380e8dc5604', embedding=None, metadata={'page_label': '49', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\n34 FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\nN\\nCC\\nN C\\nN C\\nHN\\nC\\nN\\nHHH\\nH\\nCC\\nCC\\nH C\\nH OO\\nNHH\\nH\\nNHO\\nCC\\nH N C\\nN C\\nNN\\nC\\nN\\nHH\\nH\\nHH\\nCC\\nH C\\nC N\\nH ONH\\nH\\nN\\ncytosine (C)\\nthymine (T)\\nuracil (U)CH\\nCC\\nH C\\nH OO\\nN\\nNHguanine (G)\\nhypoxanthineO\\nCC\\nH N C\\nN C\\nHN\\nC\\nN\\nHHadenine (A)\\nnicotinamideN\\nHH\\nCC\\nNC\\nCCO\\nCH\\nH\\nH Hpurines pyrimidines\\nFigure 2.13. Seven bases found in nucleotides.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6ae7412e-9a35-4357-a2d8-992bafcd0456', embedding=None, metadata={'page_label': '50', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\nNUCLEOSIDES, PHOSPHATE, AND NUCLEOTIDES 35\\nNUCLEOSIDES, PHOSPHATE, AND NUCLEOTIDES\\nFigure 2.12 ashows a nucleoside called adenosine . It is composed of ribose coupled to a\\nnitrogen-rich compound called adenine . The numbers on the sugar —1′,2′; and so forth —are\\nthe same numbering system we have seen before; the′symbol is pronounced “prime ”and is\\nthere to indicate that we are identifying the atoms of the sugar, not the atoms of the adenine.\\nThe name nucleoside re ﬂects the fact that phosphorylated nucleosides (see below) are the\\nbuilding blocks of the nucleic acids that form the genetic material in the nucleus. However,\\nnucleosides also play important roles in other places inside and outside the cell. Sevendifferent compounds can be used to generate nucleosides (Fig. 2.13). All seven containmany nitrogen atoms and one or more ring structures. They are the three purines called\\nadenine, guanine andhypoxanthine, the three pyrimidines called cytosine, thymine, and\\nuracil, and an odd man out called nicotinamide . These ring compounds are called bases .\\nHistorically, the name arose because the compounds are indeed bases in the sense usedearlier in this chapter —they will exchange an H\\n+with water. The roots of the name are\\nnow forgotten by most molecular biologists, who now use the word base to mean a purine,\\nP O H O\\nO\\n(a)\\nP OH2O HO\\nO\\nO OP OO\\nO C C OH\\n(b)\\nP O HO\\nO\\nO OP OO\\nO C H\\nH2O\\nOP OO\\nO C PO\\nO\\nO\\n(c)O\\nFigure 2.14. Phosphate groups can attach to COH groups or to other phosphate groups.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='21acd9ff-4617-42e2-b4f5-dbd4e2a83770', embedding=None, metadata={'page_label': '51', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\n36 FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\nCH2O Pnicotinamideadenine\\nN\\nCC\\nN C\\nN C\\nHN\\nC\\nNHH\\nH\\nCH2O P\\nOriboseO\\nOOH\\nH\\nHHC\\nNO\\nC\\nC CC HH\\nOHH H\\nOH\\nriboseO\\nC\\nC CC HH\\nOHH H\\nOHN\\nHHO\\nO\\nFigure 2.15. Nicotine adenine dinucleotide is formed of two nucleotides joined via their phos-\\nphate groups.\\na pyrimidine, or nicotinamide. In general, a nucleoside is formed by attaching one of these\\nbases to the 1′-carbon atom of ribose.\\nPhosphorous, although by weight a relatively minor fraction of the whole cell, plays a\\nnumber of critical roles. In solution phosphorous is mainly found as a phosphate ion with a\\nsingle hydrogen atom still attached, HPO 42−(Fig. 2.14 a). We often indicate phosphate ions\\nwith the symbol Pi, meaning inorganic phosphate. Where phosphate becomes important,however, is when it is attached to an organic, that is, carbon-containing, molecule. Phosphate\\ncan substitute into any C —OH group with the loss of a water molecule (Fig. 2.14 b). The\\nequilibrium in the reaction shown lies far to the left, but cells have other strategies forattaching phosphate groups to organic molecules. Once one phosphate group has beenadded, more can be added to form a chain (Fig. 2.14 c). Once again, the equilibrium in the\\nreaction shown lies far to the left, but cells can achieve this result using other strategies.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='0b3a10ee-6983-49ac-a747-d2151d232052', embedding=None, metadata={'page_label': '52', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\nAMINO ACIDS, POLYPEPTIDES, AND PROTEINS 37\\nOH\\nHHH\\nOH\\nOH\\nHOH\\nHO\\n4216\\n5\\n3\\nOP O OP O OO\\nOP O OO\\nFigure 2.16. Inositol trisphosphate, a multiply phosphorylated polyalcohol.\\nFigure 2.12 bshows adenosine with a chain of three phosphate groups attached to the\\n5′carbon of the ribose. This important molecule is adenosine triphosphate (ATP ), and we\\nwill meet it many times in the course of this book. The three phosphate groups are denoted\\nby the Greek letters α,β, andγ. Phosphorylated nucleosides are called nucleotides .\\nFigure 2.15 shows another important molecule formed from adenosine. Here adenosine\\nand a nicotinamide nucleotide are joined through their phosphate groups. The resultingmolecule, called nicotinamide adenine dinucleotide (NAD\\n+), plays a critical role in the\\ncellular energy budget.\\nMolecules with several OH groups can become multiply phosphorylated. Figure 2.16\\nshows inositol trisphosphate (IP3), an important messenger molecule we will meet again\\nin Chapter 16. Both ATP and IP 3have three phosphate groups, but to indicate the fact that\\nin ATP these are arranged in a chain, while in IP 3they are attached to different carbons,\\nwe use the pre ﬁxtriin adenosine triphosphate and the pre ﬁxtrisin inositol trisphosphate.\\nSimilarly, a compound with two phosphates in a chain is called a diphosphate, while one\\nwith one phosphate on each of two different carbons is called a bisphosphate.\\nAMINO ACIDS, POLYPEPTIDES, AND PROTEINS\\nAmino acids contain both a COOH group, which readily gives an H+to water and is therefore\\nacidic, and a basic NH 2group, which readily accepts H+to become NH 3+. Figure 2.17 a\\nshows two amino acids, leucine and γ-amino butyric acid (GABA ), in the form in which\\nthey are found at normal pH: the COOH groups have each lost an H+and the NH 2groups\\nhave each gained one, so that the molecules bear both a negative and a positive charge.\\nWe name organic acids by labeling the carbon that bears the carboxyl group α, the next\\noneβ, and so on. When we add an amino group, making an amino acid, we state the letter\\nof the carbon to which the amino group is attached. Hence leucine is an α-amino acid', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2dfb76fa-d1ca-4bda-8677-92bcd51770ae', embedding=None, metadata={'page_label': '53', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\n38 FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\nγCH\\nβCH2\\nαCH NH3OOCCH3\\nleucine\\nR\\nCH NH3OOC\\nα-amino acid general formγCH2\\nβCH2\\nαCH2OOCNH3\\nγ-amino butyric acid\\nR1\\nCH C\\nOR3\\nCH C\\nOH\\nNR2\\nCH C\\nOH\\nNRn\\nCH COOH\\nN NH3\\npeptide\\nbond\\ngeneralized peptide- - - - - - -\\npeptide\\nbond(a)\\n(b)\\n(c)δCH3\\nFigure 2.17. Amino acids and the peptide bond.\\nwhile GABA stands for gamma -amino butyric acid. α-Amino acids are the building blocks\\nof proteins. They have the general structure shown in Figure 2.17 bwhere R is called the\\nside chain . Leucine has a simple side chain of carbon and hydrogen. Other amino acids\\nhave different side chains and so have different properties. It is the diversity of amino side\\nchains that give proteins their characteristic properties (page 185).\\nα-Amino acids can link together to form long chains through the formation of a peptide\\nbond between the —COO−group of one amino acid and the —NH 3+group of the next.\\nFigure 2.17 cshows the generalized stucture of such a chain of α-amino acids. If there are\\nfewer than about 50 amino acids in a polymer we tend to call it a peptide . More and it\\nis apolypeptide . Polypeptides that fold into a speci ﬁc shape are called proteins . Most of\\nthe components of the machinery of the cell —the chemicals that carry out the processes\\nof life —are proteins. Peptides and polypeptides are formed inside cells by adding speci ﬁc\\namino acids to the end of a growing polymer. As we will see in Chapter 4, the particularamino acid to be added at each step is de ﬁned by the instructions on a molecule called', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='033015d1-728d-4946-b253-e0e1e234b69c', embedding=None, metadata={'page_label': '54', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\nLIPIDS 39\\nmessenger ribonucleic acid (RNA), which in turn contains a copy of the information\\nrecorded on the cell ’s DNA that is stored in the nucleus. This is the central dogma of\\nmolecular biology: DNA makes RNA makes protein.\\nLIPIDS\\nFigure 2.18 ashows the fatty acid oleic acid. It comprises a carboxyl group plus a long\\ntail of carbons and hydrogens. At neutral pH oleic acid gives up its H+to become the\\noleate ion (Fig. 2.18 b). The two ends of the oleate ion are very different. The carboxyl\\ngroup is negatively charged and hence will associate readily with water molecules. The\\nhydrocarbon tail is nonpolar and does not readily associate with water. The molecule is\\nsaid to be amphipathic, from the Greek for “hating both, ”meaning that one half does not\\nlike to be in water while the other half does not like to be in a nonpolar environment likeoctane.\\nFigure 2.18 cshows the small molecule glycerol . Like sugars, glycerol has many hy-\\ndroxyl groups and, like sugars, tastes sweet, as its name suggests. However, it is not a sugarbecause it cannot adopt an oxygen-containing ring structure. Rather, because compoundscontaining hydroxyl groups are called alcohols, glycerol is a polyalcohol.\\nAs we shall see in Chapter 13, cells can join fatty acids and glycerol to make glycerides .\\nThe bond is formed by removing the elements of water between the carboxyl group of thefatty acid and a hydroxyl group of glycerol. Any bond of this type, between a carboxylgroup and a hydroxyl group, is called an ester bond . Figure 2.19 shows two glycerides. In\\nFigure 2.19 ais trioleoylglycerol, the main component of olive oil. It is a triacylglycerol\\n(ortriglyceride ) formed from three molecules of oleic acid and one molecule of glycerol.\\nThe fatty acid residues are called acyl groups . Formation of the ester bonds has removed\\nthe charged carboxyl groups that rendered the oleate amphipathic. Almost all of the tria-cylglycerol molecule is simple hydrocarbon chain that cannot hydrogen bond with watermolecules. For this reason, olive oil and other triacylglycerols do not mix with water. Theyare therefore said to be hydrophobic . Triacylglycerols that are liquid at room tempera-\\nture are called oils, those that are solid are called fats, but they are all the same sort ofmolecule.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 2.1 Salad Dressing Is a Mixture of Solvents\\nVinegar is a dilute solution of acetic acid in water. Acetic acid gives up an H+to water to leave the\\nnegatively charged acetate ion. The simplest salad dressing is a shaken-up mixture of olive oil and\\nvinegar. Olive oil is hydrophobic and so does not dissolve in the water, the two liquids remain asseparate droplets and soon separate again after shaking. However much one shakes, all the acetatewill remain in the water because it is an ion. If salt (Na\\n+Cl−) is added to the dressing, it also dissolves\\nin the water, with none dissolving in the oil. In contrast if you add chilies to the salad dressing, theactive chemical component, capsacin, will dissolve in the oil because it is nonpolar. Only by shakingup the mixture can you get all the tastes together.\\nCells contain droplets of triacylglycerol within their cytoplasm. These are usually small,\\nbut in fat cells, which are specialized as fat stores, the droplets coalesce into a singlelarge globule so that the cell ’s cytoplasm is squeezed into a thin layer surrounding the fat', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='285bbc8b-6008-4949-be06-81817aad2376', embedding=None, metadata={'page_label': '55', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\n40 FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\nHHH\\nO\\nC\\nC\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nH C\\nH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHO\\nHHO\\nC\\nC\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nH C\\nH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHO\\nH\\nHC C\\nOH\\nCH CH\\nO O\\nH H H(a) oleic acid (b) oleate\\n(c) glycerol\\nFigure 2.18. Oleic acid and glycerol.\\nglobule. During fasting triacylglycerols are broken down into free fatty acids and glycerol, a\\nprocess called lipolysis. The fatty acids and the glycerol then enter the circulation for use bytissues.\\nFigure 2.19 bshows phosphatidylcholine, an example of the phospholipids that make\\nthe plasma membrane and other cell membranes. Like triacylglycerols, phospholipids have', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='c02e4869-18c8-43e5-a515-bc4175b16e70', embedding=None, metadata={'page_label': '56', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\ntrioleoylglycerol, a triacylglycerolglycerol\\nacyl\\ngroupacylgroup\\nphosphatidylcholine, a phospholipidC HH\\nHHH\\nO\\nC\\nC\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nH C\\nH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHOC H\\nHHH\\nOOO\\nC\\nC\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nH\\nHOC\\nHHH\\nC\\nC\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nH C\\nHC\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHOCH\\nO\\nC\\nC\\nCOC\\nHHH\\nOOOCHO–CH3CH3\\nO\\nOOCH2CH2CH3\\nN+\\nP\\nH\\nC\\nC\\nHH C\\nHH C\\nH C\\nH C\\nHH C\\nH C\\nH C\\nHH C\\nH C\\nH C\\nHH C\\nH C\\nH C\\nHH C\\nHH C\\nHH\\nHH C\\nHH CCOC\\nHHH\\nO\\nC\\nC\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nH C\\nH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHOglycerol\\nesterbondschargedheadgroup\\nphospho-diesterlink\\naqueous mediumaqueous medium\\nhydrophilic\\nlipidheadgroupshydrophobiclipid tailslipid bilayer(a)\\n(c)(b)\\nFigure 2.19. Lipids.\\n41', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2d7074c0-c048-4eaf-ac8e-ba0f5a49d8d7', embedding=None, metadata={'page_label': '57', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\n42 FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\nfatty acids attached to glycerol, but only two of them. In place of the third fatty acid is\\na polar, often electrically charged head group . The head group is joined to the glycerol\\nthrough phosphate in a structure called a phosphodiester link . The combination of head\\ngroup and negatively charged phosphate is able to associate strongly with water —it is said\\nto be hydrophilic . The two fatty acids, on the other hand, form a tail that, like olive oil,\\nis hydrophobic. Phospholipids can therefore neither dissolve in water (because of theirhydrophobic tails) nor remain completely separate, like olive oil (because then the headgroup could not associate with water). Phospholipid molecules therefore spontaneouslyform lipid bilayers between 5 and 10 nm thick (Fig. 2.19 c) in which each part of the\\nmolecule is in its preferred environment. Cell membranes are lipid bilayers plus someadded protein.\\n✎\\n✍\\x0c\\n✌Medical\\nRelevance\\n2.1The Kinks Have It: Double Bonds, Membrane Fluidity,\\nand Evening Primroses\\nOne of the fatty acids commonly found as a component of animal fat is stearic acid,\\nillustrated as (a) opposite. Its 18 carbon atoms are joined by single bonds, making a long straightmolecule. In contrast, oleic acid (b) has a double bond between the ninth and tenth carbons. Thisintroduces a kink in the chain. A fatty acid with kinks is less able to solidify because it is lessable to pack in a regular fashion. The more double bonds in a fatty acid, the lower its meltingpoint. Thus stearic acid melts at 69.6\\noC, while oleic acid melts at 13.4◦C and is therefore liquid at\\nroom temperature. Fatty acids containing double bonds between the carbon atoms are said to beunsaturated .Polyunsaturated fatty acids have more than one double bond, more than one bend,\\nand therefore even lower melting points. Linoleic acid (c) has two, and melts at −9\\n◦C; linolenic\\nacid (d) has three and a melting point of −17◦C.\\nFor the triacylglycerols inside our cells, and the phospholipids in our membranes, the same\\nrule applies —the more double bonds in the acyl groups, the lower the melting point. The large\\nmasses of triacylglycerols containing stearic acid in animal bodies are liquid at body temperaturebut form solid lumps of fat at room temperature, while trioleoylglycerol is liquid at room temper-ature (but should not be kept in the fridge!). Membranes must not solidify —if they did, then they\\nwould crack and the cell contents would leak out each time the cell was ﬂexed. Unsaturated fatty\\nacids play an essential role in maintaining membrane liquidity.\\nMammals are unable to introduce double bonds beyond carbon 9 in the fatty acid chain. This\\nmeans that linoleic and linolenic acids must be present in the diet. They are known as essential\\nfatty acids . Fortunately, the biochemical abilities of plants are not so restricted, and plant oils\\nform a valuable source of unsaturated fatty acids.\\nThe normal form of linolenic acid, shown in (d), is α-linolenic acid, which has its double\\nbonds between carbons 9 and 10, 12 and 13, and 15 and 16. Some plant seed oils contain anisomer of linolenic acid with double bonds between carbons 6 and 7, 9 and 10, and 12 and13. This is called γ-linolenic acid (e). The attractive, yellow- ﬂowered garden plant called the\\nevening primrose ( Oenethera perennis ) has seeds that contain an oil with γ-linolenic in its\\ntriacyglycerols. The 6 to 7 double bond introduces a kink closer to the glycerol. This is thoughtto increase ﬂuidity when incorporated into membrane lipids. No one really knows if it really has\\nthe marvelous health effects that some claim for it, or if it does, how it works. This ignorancehas not stopped people from making lots of money from evening primrose oil. It is included incosmetics and in alternative medicines for internal and external application. γ-Linolenic acid\\noccurs in other plants —the seeds of borage ( Borago ofﬁcinalis ) are one of the richest sources. It\\nis even found in some fungi. However, these lack the romance of the evening primrose!', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='036ca58f-0db8-4cad-9f1f-f88c1c4e54b6', embedding=None, metadata={'page_label': '58', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\nLIPIDS 43\\n✎\\n✍\\x0c\\n✌1\\n912\\n1\\n91215\\n691\\n12Stearic acid, C18 no double bonds, melts at 69.6 °\\n(a)\\nOleic acid, C18 one double bond, melts at 13.4 °\\n(b)\\nLinoleic acid, C18 two double bonds, melts at −9°\\n(c)\\nLinolenic acid, C18 three double bonds, melts at −17°\\n(d)\\nγ-Linolenic acid, C18 three double bonds\\n(e)= C\\nOHO', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='9f6416d0-abca-40cf-86b7-61fca6b21c93', embedding=None, metadata={'page_label': '59', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\n44 FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\nHYDROLYSIS\\nMany of the macromolecules of which cells are made are generated from their individual\\nbuilding blocks by the removal of the elements of water. Equally, macromolecules can bebroken into their individual building blocks by hydrolysis —breakage by the addition of\\nwater. Figure 2.20 shows four examples. Lactose is hydrolyzed to the monosaccharidesgalactose and glucose by the addition of one water molecule. Next, we show a dipeptide be-ing broken into individual amino acids by hydrolysis. These two hydrolysis reactions occurall the time in our intestines, though some people lose the ability to hydrolyze lactose as theyget older: they are lactose intolerant . Third, we show a dimeric (formed of two parts) inor-\\nganic phosphate ion called pyrophosphate being hydrolyzed to regular phosphate ions. The\\nhydrolysis of pyrophosphate is catalyzed by enzymes found throughout the body, both insidecells and out. Later in the book we will meet a number of instances where a reaction creates\\nserinepyrophosphate\\nleucineH3NC CH CCH3\\nCH2\\nNH\\nOCH\\nCH COOCH2OHCH3\\nO\\nO PO H\\nOO\\nO PO\\nOHO\\nPO\\nO\\nHOO\\nPO\\nO\\nCHCH3\\nCH2CHCH3OOC\\nOOCCH NH3CH2OH\\nH NH3H2O\\nH2O\\nglucoseCCH2OH\\nHO\\nC\\nCOHH\\nHO\\nHCH\\nOHOHCH\\ngalactoseCCH2OH\\nHO\\nC\\nCOHOH\\nH\\nHCH\\nOHC\\nHOH6\\n5\\n4\\n321\\n6\\n5\\n4\\n321CCH2OH\\nHO\\nC\\nCOHH\\nO\\nHCH\\nOHOHCH\\nCCH2OH\\nHO\\nC\\nCOHOH\\nH\\nHCH\\nOHC\\nHlactose\\n5\\n4\\n321\\n56\\n4\\n321\\nH2O\\n(a) (b) (c)6\\nFigure 2.20. Hydrolysis is the breakage of covalent bonds by the addition of the elements\\nof water.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='c5044b86-329c-4e03-bb20-6a63cc56c4e8', embedding=None, metadata={'page_label': '60', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\nstearate arachidonate \\nO\\nO PO H\\nOO\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nHCOO\\nH CH\\nO\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nHCOH CHH\\nH CH\\nH CH\\nCH\\nCHCCH\\nCOO\\nP\\nO\\nO\\nH CH\\nCH\\nCH\\nH CH\\nCH\\nH CH\\nCH\\nCH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nHCHOH\\nO\\nH CH\\nH CH\\nH CH\\nCH\\nCHC\\nH CH\\nCH\\nCH\\nH CH\\nCH\\nH CH\\nCH\\nCH\\nH CH\\nH CH\\nH CH\\nH CH\\nH CH\\nHCHOCH2CH2CH3CH3N\\nglycerolCH2CHCH2OH\\nOH\\nOH\\ncholineCH2\\nOHCH2CH3\\nCH3CH3N3H\\n(d)4H2OCH3\\nFigure 2.20. Continued\\n45', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6df97851-b38d-41bc-9a63-d7c918cfb5b0', embedding=None, metadata={'page_label': '61', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\n46 FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\npyrophosphate, but as soon as the pyrophosphate is produced it will be hydrolyzed to regu-\\nlar phosphate. Lastly, we show the complete hydrolysis of the phospholipid phosphatidyl-choline. Five molecules result: one phosphate ion together with choline, glycerol, and twofatty acids. This hydrolysis also occurs in our intestines, where it is a multistage process.\\nHydrolysis reactions usually proceed rapidly in living tissue when the appropriate\\ncatalyst is present, because water is present at high concentration. In contrast the generationof macromolecules from their constituent building blocks, often by the elimination of theelements of water between constituent building blocks, usually requires the expenditure ofenergy by the cell. We will see how these reactions are performed in later chapters.\\nSUMMARY\\n1.When two atoms interact, electrons may be shared between the two to form acovalent bond, or may pass completely from one atom to another, forming ions.\\n2.In water the electrons are not shared equally but are displaced toward the oxygenatom, so that the molecule is polar.\\n3.Ionic compounds will only dissolve in polar solvents.\\n4.Acids are molecules that give an H\\n+to water, forming H 3O+. Dissolving an acid\\nin water produces a solution of pH less than 7.\\n5.Bases are molecules that accept an H+from water, leaving OH−. Dissolving a base\\nin water produces a solution of pH greater than 7.\\n6.Pure water has the neutral pH of 7.0.\\n7.A hydrogen bond can form when a hydrogen atom takes up a position between twoelectron-grabbing atoms (oxygen, nitrogen, or sulfur), the three forming a straightline.\\n8.Monosaccharides are compounds with a central skeleton of carbon to which areattached many OH groups. They can switch between two oxygen-containing ringstructures via an open-chain con ﬁguration.\\n9.Monosaccharides can join together to form disaccharides such as lactose and toform long chains such as glycogen and cellulose.\\n10.Adenine, guanine, hypoxanthine, cytosine, thymine, uracil, and nicotinamide arenitrogen-rich ring-shaped molecules called bases. Bases combine with ribose ordeoxyribose to form nucleosides. Phosphorylation of a nucleoside on the 5\\n′carbon\\nproduces a nucleotide.\\n11.Amino acids are compounds with a —COOH acidic group and an —NH 2basic\\ngroup. At neutral pH these groups will, respectively, lose and gain an H+to become\\n—COO−and—NH 3+.\\n12.Polypeptides are polymers of α-amino acids linked by peptide bonds. The sequence\\nof amino acids is determined by instructions on the cell ’s DNA. Proteins are polypep-\\ntides that fold into a speci ﬁc shape.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='64016eff-9f36-4f1e-914d-900e978506b8', embedding=None, metadata={'page_label': '62', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\nREVIEW QUESTIONS 47\\n13.Lipids are formed by the attachment of hydrophobic long-chain fatty acids to a glyc-\\nerol backbone. Phospholipids also have a hydrophilic head group and spontaneouslyform lipid bilayers.\\n14.Hydrolysis is the breakage of covalent bonds by the addition of the elements ofwater.\\nFURTHER READING\\nV oet, D., and V oet, J. 2003. Biochemistry . New York: Wiley.\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.Water\\nA. can give up an H+, becoming OH−.\\nB. can accept an H+, becoming H 3O+.\\nC. can form hydrogen bonds.\\nD. is a good solute for ionic substances.E. all of the above.\\n2.The amino acid leucine has an acidic COOH group and a basic NH 2group. The reaction\\n—COOH⇄—COO−+H+h a sap Kaof 2.3, while the reaction —NH 3+⇄—NH 2+\\nH+h a sap Kaof 9.7. At pH 1.0\\nA. there will be more−OOC —CH(CH 2.CH(CH 3)2)—NH 2than either\\n−OOC —CH(CH 2.CH(CH 3)2)—NH 3+or HOOC —CH(CH 2.CH(CH 3)2)—NH 3+.\\nB. there will be equal amounts of−OOC —CH(CH 2.CH(CH 3)2)—NH 2and\\n−OOC —CH(CH 2.CH(CH 3)2)—NH 3+, but not much HOOC —CH(CH 2.CH(CH 3)2)—NH 3+.\\nC. there will be more−OOC —CH(CH 2.CH(CH 3)2)—NH 3+than either\\nHOOC —CH(CH 2.CH(CH 3)2)—NH 3+or−OOC —CH(CH 2.CH(CH 3)2)—NH 2.\\nD. there will be equal amounts of HOOC —CH(CH 2.CH(CH 3)2)—NH 3+and\\n−OOC —CH(CH 2.CH(CH 3)2)—NH 3+, but not much−OOC —CH(CH 2.CH(CH 3)2)—NH 2.\\nE. there will be more HOOC —CH(CH 2.CH(CH 3)2)—NH 3+than either\\n−OOC —CH(CH 2.CH(CH 3)2)—NH 3+or−OOC —CH(CH 2.CH(CH 3)2)—NH 2.\\n3.Which of the following statements is incorrect?\\nA. Glycogen is formed from units that include glucose.B. Cellulose is formed from units that include glucose.C. Lactose is formed from units that include ribose.D. Adenosine is formed from units that include ribose.E. Nicotinamide adenine dinucleotide is formed from units that include ribose.\\n4.Phospholipids\\nA. have hydrophilic fatty acid tails.B. have charged, hydrophobic head groups.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='c951af8b-2b57-431f-b2c5-31b6d88e783d', embedding=None, metadata={'page_label': '63', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\n48 FROM WATER TO DNA: THE CHEMISTRY OF LIFE\\nC. spontaneously form lipid bilayers in an aqueous environment.\\nD. form droplets within the cytoplasm.E. all of the above.\\n5.When an acid is added to pure water\\nA. it will dissolve easily.B. it will give up an H\\n+to form a negatively charged ion.\\nC. it will form a solution with a high concentration of H 3O+ions.\\nD. it will form a solution with a low pH.E. all of the above.\\n6.Hydrogen bonds cannot form between\\nA. water and glucose.B. water and water.C. water and phosphate.D. phosphate and octane.E. phosphate and glucose.\\n7.Which of these statements about the polypeptides in cells is wrong?\\nA. They are a polymer of γ-amino acids linked by peptide bonds.\\nB. The sequence of amino acids is determined by instructions on the cell ’sD N A .\\nC. They have an —NH 3+group at one end of the polymer.\\nD. They have a —COO−group at one end of the polymer.\\nE. They are called proteins if they fold into a speci ﬁc shape.\\nANSWERS TO REVIEW QUESTIONS\\n1.E.All these statements about water are true.\\n2.E.A pH of 1.0 is very acid (an H+concentration of 100 mmol liter−1), more acid even than 2.3, the\\npH at which the COOH group is half-protonated. Both the —COOH and NH 2groups on leucine\\nare therefore protonated at pH 1.0.\\nIf you are still confused, consider the situation at pH 11. At this pH, the concentration of H+\\nis extremely low (10−11mol liter−1or 10 pmol liter−1). Leucine therefore loses all its hydrogen\\nions and has the form−OOC —CH(CH 2.CH(CH 3)2)—NH 2. Now consider adding H+so that the\\npH falls. The NH 2group is basic and accepts H+readily, even when H+is present at quite a low\\nconcentration. At pH 9.7 (an H+concentration of 200 pmol liter−1) half the NH 2groups have\\naccepted an H+to become NH 3+, and at pH values less than 9.7 a majority of the NH 2groups are\\nprotonated. This is the situation at neutral pH: the NH 2groups are protonated while the COOH\\ngroups are deprotonated, so leucine has the form−OOC —CH(CH 2.CH(CH 3)2)—NH 3+.I fw e\\nkeep adding H+, we eventually approach a pH of 2.3, where the H+concentration has the high\\nvalue of 5 mmol liter−1. Even though the COOH groups like to give up H+, they are forced to\\naccept H+when the concentration of H+in solution gets this high. At pH 2.3 half the COOH\\ngroups are protonated; half are still COO−. At a pH less than 2.3, that is, at H+concentrations\\nhigher than 5 mmol liter−1, the majority of the COOH groups are protonated and leucine has the\\nform HOOC —CH(CH 2.CH(CH 3)2)—NH 3+.\\n3.C.Lactose is a disaccharide formed from glucose and galactose.\\n4.C.All the other statements are false. In particular: (A) The fatty acid tails are hydrophobic. (B) The\\nhead groups are hydrophilic because they are polar and often are charged. (D) Phospholipids will notform droplets because this would place the hydrophilic head group in a hydrophobic environment.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='916a2e38-b3fa-48a4-96e6-01001746ebd5', embedding=None, metadata={'page_label': '64', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-02 WY001-Bolsover-v2.cls August 21, 2003 18:22\\nANSWERS TO REVIEW QUESTIONS 49\\n5.E.All these statements are true. In particular, the de ﬁnition of pH means that a solution with a\\nhigh concentration of H 3O+ions is one with a low pH.\\n6.D.Octane is formed of carbon and hydrogen only. It does not contain any of the electron-grabbing\\natoms that, together with hydrogen, form a hydrogen bond. Octane therefore cannot form hydrogen\\nbonds with other chemicals.\\n7.A.Cellular polypeptides are polymers of α-amino acids, not γ-amino acids. Changing γtoαin\\nthis question generates an accurate summary of the biology of polypeptides.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='198024c4-96b0-4f6e-bcd2-59358c3880b4', embedding=None, metadata={'page_label': '65', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-03 WY001-Bolsover-v2.cls September 15, 2003 21:3\\n3\\nMEMBRANES AND ORGANELLES\\nThe vast majority of the reactions that cells carry out take place in water. Eukaryotic cells\\nare, at any one time, carrying out an enormous range of such chemical manipulations:collectively, these reactions are called metabolism (Chapter 13). In much the same way thatour homes are divided into rooms that are adapted for particular activities, so eukaryotic cellscontain distinct compartments or organelles to house speciﬁc functions. The term organelle\\nis used rather loosely. At one extreme some scientists use it to mean any distinct cellularstructure that has a more or less well-deﬁned job to do; at the other are scientists who wouldreserve the name organelle for those cellular compartments that contain their own DNAand have some limited genetic autonomy. In this book, we will deﬁne organelles as thosecellular components whose limits, like those of the cell itself, are deﬁned by membranes.It is ﬁrst necessary to consider some of the fundamental properties of cell membranes.\\nBASIC PROPERTIES OF CELL MEMBRANES\\nIt is difﬁcult to overstate the importance of membranes to living cells; without them lifecould not exist. The plasma membrane, also known as the cell surface membrane or\\nplasmalemma, deﬁnes the boundary of the cell. It regulates the movement of materials\\ninto and out of the cell and facilitates electrical signaling between cells. Other membranesdeﬁne the boundaries of organelles and provide a matrix upon which complex chemicalreactions can occur. Some of these themes will be developed in subsequent chapters. In thefollowing section the basic structure of the cell membrane will be outlined.\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n51', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='c646308a-b798-474c-b60f-e33b3719982e', embedding=None, metadata={'page_label': '66', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-03 WY001-Bolsover-v2.cls September 15, 2003 21:3\\n52 MEMBRANES AND ORGANELLES\\nintegral membrane protein\\nperipheral membrane proteinsugar residues\\ncytosolextracellular\\nmedium\\nhydrophiliclipidheadgroupshydrophobiclipid tailslipid bilayer\\nFigure 3.1. Membranes comprise a lipid bilayer plus integral and peripheral proteins.\\nThe basic structure of a biological membrane is shown in Figure 3.1. Approximately half\\nthe mass is phospholipid, which spontaneously organizes to form a lipid bilayer (page 42).\\nAll the membranes of the cell, including the plasma membrane, also contain proteins. Thesemay be tightly associated with the membrane and extracted from it only with great dif ﬁculty,\\nin which case they are called integral proteins (e.g., the gap junction channel, page 55);or they may be separated with relative ease, in which case they are termed peripheral\\nO2\\nNa+K+Ca++GlucoseCO2NO H2O\\nSteroid hormones\\nFigure 3.2. Small uncharged molecules can pass through membranes by simple diffusion, but ions cannot.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='044d65ca-d2a3-4713-9e53-d7dbfdbbd301', embedding=None, metadata={'page_label': '67', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-03 WY001-Bolsover-v2.cls September 15, 2003 21:3\\nBASIC PROPERTIES OF CELL MEMBRANES 53\\nproteins (e.g., clathrin adaptor protein, page 229). Membrane proteins are free to move later-\\nally, within the plane of the membrane. Integral membrane proteins are often glycosylated —\\nthat is, have sugar residues attached —on the side facing the extracellular medium.\\nStraight Through the Membrane: Diffusion Through the Bilayer\\nMolecules of oxygen are uncharged. Although they dissolve readily enough in water, theyare also able to dissolve in the hydrophobic interior of lipid bilayers. Oxygen molecules cantherefore pass from the extracellular ﬂuid into the interior of the plasma membrane, and\\nfrom there pass on into the cytoplasm, in a simple diffusion process (Fig. 3.2). Three othersmall molecules with important roles in biology —carbon dioxide, nitric oxide, and water\\nitself —also pass across the plasma membrane by simple diffusion, as do the uncharged\\nhormones of the steroid family. In contrast, charged ions cannot dissolve in hydrophobicregions (page 21) and therefore cannot cross membranes by simple diffusion.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 3.1 Rapid Diffusion in the Lungs\\nBlood is composed of cells, red and white, and plasma, a solution of sodium chloride, other ions,\\nvarious organic molecules and proteins. Red blood cells are very simple. They have no nucleus andtheir plasma membrane encloses a cytosol packed with the oxygen-carrying protein hemoglobin,bathed in a salt solution. The sodium concentration in the red blood cytosol is much lower than thesodium concentration in plasma. It is important that the sodium concentration in the cytosol remainlow: if it increases, the red blood cells will swell and then burst as water rushes in. As red bloodcells pass through the lungs, they quickly gain oxygen because oxygen molecules can pass rapidlyacross the plasma membrane by simple diffusion. The cells do not, however, gain sodium ions fromthe plasma as they pass around the body because the lipid bilayer is impermeable to ions. Red bloodcells therefore remain the right size to pass though even the tiniest blood vessels in our bodies.\\nBeyond the Cell Membrane: The Extracellular Matrix\\nIn connective tissue (page 13), the spaces between cells are ﬁlled by an extracellular matrix\\nconsisting of polysaccharides and proteins such as collagen together with combinations of\\nthe two called proteoglycans. The proteins provide tensile strength and elasticity while the\\npolysaccharides form a hydrated gel that expands to ﬁll the extracellular space.\\nThe sugar-based extracellular matrix reaches its highest form of expression in the cell\\nwalls of plants (Fig. 3.3). The plant cell wall consists primarily of cellulose micro ﬁbrils\\nlinked together by other polysaccharide molecules such as hemicellulose andpectin. The\\nthickness of the wall is determined largely by its pectin content. In healthy plant cells theplasma membrane is constantly pressing against the cell wall.\\nIn plants the extracellular medium is a much more dilute solution than the intracel-\\nlular solution. If no other forces operated, water would diffuse into plant cells down itsconcentration gradient, a process called osmosis. However, plant cells are prevented from\\nexpanding by the inextensible cellulose micro ﬁbrils in their cell walls. As water diffuses\\nin, the hydrostatic pressure in the cells rises, making the combination of cell plus cell wallstiff. The same effect can be seen when a bicycle inner tube is blown up inside the tire, andis called turgor. If plants are starved of water, water will leave the cell interior, hydrostaticpressure drops, and the cells lose their turgor pressure, like a tire when the tube inside isdeﬂated. As a result the nonwoody parts of the plant wilt.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='c24a3a38-3ecd-40e8-9de5-f6ee7136b1fe', embedding=None, metadata={'page_label': '68', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-03 WY001-Bolsover-v2.cls September 15, 2003 21:3\\n54 MEMBRANES AND ORGANELLES\\ncellulose\\nmicro fibrilPrimary\\ncell\\nwall\\n(a)direction ofexpansionvacuole\\ncytoplasm\\nnucleus\\nprimary\\ncell wall\\nsecondary\\ncell wall\\n(b)\\nFigure 3.3. The cell wall directs plant cell growth.\\nNewly generated plant cells ﬁrst lay down a primary cell wall of cellulose micro ﬁbrils\\nthat are orientated in one direction around the plant cell (Fig. 3.3 a). Hydrostatic pressure\\ntherefore tends to cause a growing plant cell to elongate in one direction, perpendicular to\\nthe axis of the micro ﬁbrils. Once the growth of the cell is complete, more layers of cellulose\\nand/or other compounds are added, most notably the polyphenolic compound lignin, toform the secondary cell wall (Fig. 3.3 b).\\nCell Junctions\\nIn multicellular organisms, and particularly in epithelia, it is often necessary for neighboringcells within a tissue to be connected together. This function is provided by cell junctions. Inanimal cells there are three types of junctions. Those that form a tight seal between adjacentcells are known as tight junctions; those that allow communication between cells are\\nknown as gap junctions. A third class of cell junction that anchors cells together, allowing\\nthe tissue to be stretched without tearing, are called anchoring junctions. Plant cells do', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='342a2cdb-ca8d-46c7-808d-2c9abbdd9d4a', embedding=None, metadata={'page_label': '69', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-03 WY001-Bolsover-v2.cls September 15, 2003 21:3\\nBASIC PROPERTIES OF CELL MEMBRANES 55\\nnot have tight junctions, gap junctions, or anchoring junctions but do contain a unique class\\nof communicating junction known as plasmodesmata.\\nTight junctions are found wherever ﬂow of extracellular medium is to be restricted\\nand are particularly common in epithelial cells such as those lining the small intestine. Theplasma membranes of adjacent cells are pressed together so tightly that no intercellular spaceexists between them (Fig. 1.8 on page 14). Tight junctions between the epithelial cells of theintestine ensure that the only way that molecules can get from the lumen of the intestine tothe blood supply that lies beneath is by passing through the cells, a route that can be selective.\\nGap junctions are specialized structures that allow cell-to-cell communication in an-\\nimals (Fig. 3.4). When two cells form a gap junction, ions and small molecules can passdirectly from the cytosol of one cell to the cytosol of the other cell without going into theextracellular ﬂuid. Since ions can move through the junction, changes in electrical voltage\\nGap junction\\ncytosol of right\\nhand cell\\ngapjunctionchannelplasma\\nmembranecytosolof lefthandcell\\nplasma\\nmembranehydrophilic\\nsolute\\nFigure 3.4. Gap junctions allow solute and electrical current to pass from the cytosol of one cell\\nto the cytosol of its neighbor.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d92e8c7f-b5bb-4c87-9c46-7ff42e42fb99', embedding=None, metadata={'page_label': '70', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-03 WY001-Bolsover-v2.cls September 15, 2003 21:3\\n56 MEMBRANES AND ORGANELLES\\nare also rapidly transmitted from cell to cell by this route. The structure that makes this\\npossible is the gap junction channel. Channels, as we will see in Chapter 14, are water-\\nﬁlled holes through membranes. When two gap junction channels or connexons meet, they\\nform a water- ﬁlled tube that runs all the way through the plasma membrane of the ﬁrst cell,\\nacross the small gap between the cells, and through the plasma membrane of the second cell.In the middle of the channel is a continuous hole about 1.5 nm in diameter. This hole is largeenough to allow small ions through (and therefore to pass electrical current) together withamino acids and nucleotides, but it is too small for proteins or nucleic acids. Gap junctionsare especially important in the heart, where they allow an electrical signal to pass rapidlybetween all the cardiac muscle cells, ensuring that they all contract at the proper time. Eachgap junction channel is composed of six protein subunits that can twist against each other toopen and close the central channel in a process called gating (page 314) that allows the cellto control the degree to which it shares solute with its neighbor. The plasmodesmata thatperforate the cell walls of many plant tissues (Fig. 1.7 on page 10) serve much the samepurpose as the gap junctions of animal cells but are much bigger and cannot shut quickly.Some plant viruses use plasmodesmata to spread from cell to cell.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 3.2 Gap Junctions Keep Eggs Ready But Waiting\\nIn the days leading up to ovulation, oocytes (cells that undergo meiosis to give rise to eggs) that\\nare to be released from the follicle are kept in a state of suspended development by a chemical intheir cytosol called cyclic AMP (page 350). The oocytes themselves do not make cAMP. Rather, thefollicle cells that surround them in the ovary make cAMP, which then passes through gap junctionsinto the oocyte.\\nOnly when the oocyte is released and begins its passage down the Fallopian tube does it prepare\\nfor fertilization. In particular, the process of meiosis (page 404) is completed so as to remove halfthe chromosomes, so that when the sperm adds its complement of chromosomes the fertilized eggwill have the correct number.\\nAnchoring junctions bind cells tightly together and are found in tissues such as the\\nskin and heart that are subjected to mechanical stress. These junctions are described later\\n(page 396).\\nORGANELLES BOUNDED BY\\nDOUBLE-MEMBRANE ENVELOPES\\nThree of the major cell organelles, the nucleus, mitochondrion, and, in plant cells, the chloro-\\nplast, share two distinctive features. They are all enclosed within an envelope consisting oftwo parallel membranes and they all contain the genetic material DNA.\\nThe Nucleus\\nThe nucleus is often the most prominent cell organelle. It contains the genome, the cell ’s\\ndatabase, which is encoded in molecules of the nucleic acid, DNA. The nucleus is boundedby a nuclear envelope composed of two membranes separated by an intermembrane space(Fig. 3.5). The inner membrane of the nuclear envelope is lined by a meshwork of proteins', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='330e28aa-1de8-4126-b266-f98385d51b46', embedding=None, metadata={'page_label': '71', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-03 WY001-Bolsover-v2.cls September 15, 2003 21:3\\nORGANELLES BOUNDED BY DOUBLE-MEMBRANE ENVELOPES 57\\nribosome\\nnucleolus\\ncytosolribosomeeuchromatin\\nheterochromatin\\nnuclear pore\\ninner nuclear membrane\\nintermembrane space\\nouter nuclear membrane\\nnuclear lamina\\nrough endoplasmic\\nreticulum\\nFigure 3.5. The nucleus and the relationship of its membranes to those of the endoplasmic\\nreticulum.\\ncalled the nuclear lamina which provides rigidity to the nucleus. A two-way traf ﬁc of proteins\\nand nucleic acids between the nucleus and the cytoplasm passes through holes in the nuclear\\nenvelope called nuclear pores. The nucleus of a cell that is synthesizing proteins at a lowlevel will have few nuclear pores. In cells that are undergoing active protein synthesis,however, virtually the whole nuclear surface is perforated.\\nWithin the nucleus it is usually possible to recognize discrete areas. Most of it is\\noccupied by chromatin, a complex of DNA and certain DNA-binding proteins such ashistones (page 73). In most cells it is possible to recognize two types of chromatin; lightly\\nstaining euchromatin is that portion of the cell ’s DNA that is being actively transcribed into\\nRNA; heterochromatin, on the other hand, represents the inactive portion of the genome\\nwhere no RNA synthesis is occurring. The DNA in heterochromatin is tightly coiled thensupercoiled (see Fig. 4.5 on page 72), leading to its dense appearance in both the light andelectron microscopes.\\nUnlike DNA, RNA is not con ﬁned within the nucleus and other organelles but is also\\nfound within the cytoplasm associated with particles called ribosomes whose function is tomake proteins. Ribosomes are made in the nucleus, in specialized regions called nucleolithat form at speci ﬁc sites on the DNA called nucleolar organizer regions. These contain\\nblocks of genes that code for the ribosomal RNA.\\nIt should be stressed that the appearance of the nucleus we have described thus far\\nrelates to the cell in interphase, the period between successive rounds of cell division. As\\nthe cell enters mitosis (Chapter 19) the organization of the nucleus changes dramatically. The', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='0725367c-cb3c-45b8-b066-6e8dd2c9f395', embedding=None, metadata={'page_label': '72', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-03 WY001-Bolsover-v2.cls September 15, 2003 21:3\\n58 MEMBRANES AND ORGANELLES\\nDNA becomes more and more tightly packed and is revealed as a number of separate rods\\ncalled chromosomes —46 in human cells. The nucleolus disperses, and the nuclear envelope\\nfragments. Upon completion of mitosis, these structural rearrangements are reversed andthe nucleus resumes its typical interphase organization.\\nMitochondria and Chloroplasts\\nAlthough most of the genetic information of a eukaryotic cell resides in nuclear DNA, someof the information necessary to make both chloroplasts and mitochondria is stored withinthese organelles themselves. This information resides on small circular DNA molecules thatare similar to the chromosomes of bacteria (Table 1.1 on page 11) but very different from thelong linear DNA molecules in the nucleus. This is strong evidence for the endosymbiotictheory of the origin of mitochondria and chloroplasts (page 12), which proposes that thesmall circular DNA molecules found in these organelles is all that is left of the chromosomesof the original symbiotic bacteria. Chloroplasts and mitochondria both contain ribosomes(again, more like those of bacteria than the ribosomes in the cytoplasm of their own cell,Table 1.1 on page 11), and synthesize a small subset of their own proteins, although the greatmajority of proteins that form chloroplasts and mitochondria are encoded by nuclear genesand synthesized in the cytoplasm. Perhaps the most distinctive feature of these organelles,however, is that the inner of their two membranes is markedly elaborated and folded toincrease its surface area.\\nMitochondria are among the most easily recognizable organelles due to the extensive\\nfolding of their inner membrane to form shel ﬂike projections named cristae (Fig. 3.6). The\\nnumber of cristae, like the number of mitochondria themselves, depends upon the energybudget of the cell in which they are found. In striated muscle cells, which must contractand relax repeatedly over long periods of time, there are many mitochondria that containnumerous cristae; in fat cells, which generate little energy, there are few mitochondria andtheir cristae are less well developed. This gives a clue as to the function of mitochondria: theyare the cell ’s power stations. Mitochondria produce the molecule adenosine triphosphate\\n(ATP), one of the cell ’s energy currencies that provide the energy to drive a host of cellular\\nreactions and mechanisms (Chapter 12). The double-membrane structure provides fourdistinct domains: the outer membrane, the inner membrane, the intermembrane space, andthe matrix. Each domain houses a distinct set of functions, details of which are given inChapters 12 and 13.\\nChloroplasts are found only in photosynthetic protists and plant cells. In addition to\\nthe two bounding membranes, they contain internal membranes called thylakoids which,in plants, form stacks called grana (Fig. 3.6). The thylakoids contain the proteins and othermolecules responsible for light capture. The dark reactions of photosynthesis (page 302),on the other hand, takes place in the matrix, called the stroma, which also contains the DNAand ribosomes.\\nORGANELLES BOUNDED BY\\nSINGLE-MEMBRANE ENVELOPES\\nEukaryotic cells contain many sacs and tubes bounded by a single membrane. Although\\nthese are often rather similar in appearance, they can be subdivided into different typesspecialized to carry out distinct functions.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3c596888-6b35-4808-91ec-001604274bc9', embedding=None, metadata={'page_label': '73', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-03 WY001-Bolsover-v2.cls September 15, 2003 21:3\\nORGANELLES BOUNDED BY SINGLE-MEMBRANE ENVELOPES 59\\nouter membraneinner membranecristae ribosome\\nmatrix\\nthylakoid \\nmembraneouter membraneintermembrane spaceintermembrane space\\ninner membranegranumstromaribosomemitochondrion\\nchloroplastthylakoidspace\\nFigure 3.6. Mitochondrion and chloroplast.\\nPeroxisomes\\nMitochondria and chloroplasts are frequently found close to another membrane-bound\\norganelle, the peroxisome. In human cells peroxisomes have a diameter of about 500 nm,and their dense matrix contains a heterogeneous collection of proteins concerned with avariety of metabolic functions, some of which are only now beginning to be understood.Peroxisomes are so named because they are frequently responsible for the conversion ofthe highly reactive molecule hydrogen peroxide (H\\n2O2), which is formed as a by-product', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='aa4b1249-e551-4c36-b3c1-6908d419bf78', embedding=None, metadata={'page_label': '74', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-03 WY001-Bolsover-v2.cls September 15, 2003 21:3\\n60 MEMBRANES AND ORGANELLES\\nof the reactions in the mitochondrion, into water:\\n2H2O2→2H2O+O2\\nThis reaction is carried out by a protein called catalase, which sometimes forms an obvious\\ncrystal within the peroxisome. Catalase is an enzyme —a protein catalyst that increases\\nthe rate of a chemical reaction (page 241). In fact, it was one of the ﬁrst enzymes to\\nbe discovered. In humans, peroxisomes are primarily associated with lipid metabolism.Understanding peroxisome function is important for a number of inherited human diseasessuch as X-linked adrenoleukodystrophy where peroxisome malfunction and the consequentinability to metabolise lipid properly typically leads to death in childhood or early adulthoodunless dietary lipid is extremely restricted.f\\nIN DEPTH 3.1 Nobel Prizes for Organelle Researchers\\nIn addition to Camillo Golgi’s Nobel prize in 1906, modern studies of cell organelles\\nhave also been recognized by the Nobel committee. George Palade, Albert Claude,and Christian de Duve shared the 1974 Nobel prize in physiology or medicine fortheir work on the identiﬁcation and isolation of cell organelles. G ¨unter Blobel, a\\nformer student of Palade’s, was awarded the same prize in 1999 for the discoverythat proteins carry so-called targeting sequences that determine where in the cellthey should reside (page 215).\\nEndoplasmic Reticulum\\nThe endoplasmic reticulum is a network of membrane enclosed channels that run throughout\\nthe cell, forming a continuous network whose lumen (inside) is at all points separatedfrom the cytosol by a single membrane. The membrane of the endoplasmic reticulum iscontinuous with the outer nuclear membrane (Fig. 3.5). Two regions can be recognizedin most cells, known as smooth endoplasmic reticulum and rough endoplasmic reticulum(Fig. 1.2 on page 3). The basic difference is that the rough endoplasmic reticulum is coveredin ribosomes, which give it its rough appearance in the electron microscope.\\nThe function of the smooth endoplasmic reticulum varies from tissue to tissue. In the\\novaries, testes, and the adrenal gland it is where steroid hormones are made; in the liver it isthe site of detoxication of foreign chemicals including drugs. Probably the most universalrole of the smooth endoplasmic reticulum is the storage and sudden release of calciumions. Calcium ions are pumped from the cytosol into the lumen of the smooth endoplasmicreticulum to more than 100 times the concentration found in the cytosol. Many stimulican cause this calcium to be released back into the cytosol, where it activates many cellprocesses (Chapter 16).\\nThe rough endoplasmic reticulum is where the cells make the proteins that will end up\\nas integral membrane proteins in the plasma membrane, and proteins that the cell will exportto the extracellular medium (such as the proteins of the extracellular matrix, page 13).\\nGolgi Apparatus\\nThe Golgi apparatus, named after its discoverer, 1906 Nobel prize winner Camillo Golgi, isa distinctive stack of ﬂattened sacks called cisternae. The Golgi apparatus is the distribution', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='607ab342-37ce-43f8-89c1-056808368390', embedding=None, metadata={'page_label': '75', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-03 WY001-Bolsover-v2.cls September 15, 2003 21:3\\nSUMMARY 61\\npoint of the cell where proteins made within the rough endoplasmic reticulum are further\\nprocessed and then directed to their ﬁnal destination.\\nAppropriately, given this central role, the Golgi apparatus is situated at the so-called cell\\ncenter, a point immediately adjacent to the nucleus that is also occupied by a structure called\\nthe centrosome (Fig. 1.2 on page 3). The centrosome helps to organize the cytoskeleton(Chapter 18).\\nLysosomes\\nLysosomes are sometimes called “cell stomachs ”because they contain enzymes that digest\\ncellular components. They are particularly plentiful in cells that digest and destroy othercells, such as macrophages. Lysosomes are roughly spherical and usually 250 –500 nm in\\ndiameter.\\nSUMMARY\\n1.Membranes are made of phospholipids and protein.\\n2.Cells are bounded by membranes while cell functions are compartmentalized intomembrane-bound organelles.\\n3.Solutes with a signi ﬁcant solubility in hydrophobic solvents can pass across bio-\\nlogical membranes by simple diffusion.\\n4.The extracellular matrix is found on the outside of animal cells. A cellulose-basedcell wall is found outside plant cells.\\n5.Turgor pressure gives nonligni ﬁed plant tissue its stiffness.\\n6.Tight junctions prevent the passage of extracellular water or solute between the cellsof an epithelium.\\n7.Gap junctions allow solute and electrical current to pass from the cytosol of one cellto the cytosol of its neighbor.\\n8.The nucleus, mitochondrion, and chloroplast are bounded by double-membraneenvelopes. In the case of the nucleus, this is perforated by nuclear pores. All threeorganelles contain DNA.\\n9.Mitochondria produce the energy currency adenosine triphosphate (ATP).\\n10.Chloroplasts capture light energy for use by plant cells.\\n11.Peroxisomes carry out a number of reactions including the destruction of hydrogenperoxide.\\n12.The endoplasmic reticulum is the site of much protein synthesis. Cell stimulationcan often cause calcium ions stored in the endoplasmic reticulum to be released intothe cytosol.\\n13.The Golgi apparatus is concerned with the modi ﬁcation of proteins after they have\\nbeen synthesized.\\n14.Lysosomes contain powerful degradative enzymes.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b36b3e5c-8c57-4633-a616-17986439624f', embedding=None, metadata={'page_label': '76', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-03 WY001-Bolsover-v2.cls September 15, 2003 21:3\\n62 MEMBRANES AND ORGANELLES\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.Biological membranes\\nA. contain phospholipid.\\nB. contain protein.C. are relatively impermeable to ionic solutes.D. are relatively permeable to small uncharged solutes.E. all of the above.\\n2.The extracellular matrix\\nA. can contain collagen.B. can contain phospholipid.C. can contain RNA.D. is relatively impermeable to ionic solutes.E. all of the above.\\n3.Which of the following statements about gap junctions is incorrect?\\nA. They are formed from six subunits.B. They are found in both plants and animals.C. They allow small solute molecules to pass between the cytosol of one cell and the cytosol of\\nits neighbor.\\nD. They allow electrical current to pass between the cytosol of one cell and the cytosol of its\\nneighbor.\\nE. They can switch (gate) to a closed state.\\n4.A cell with a high density of nuclear pores is likely to be\\nA. digesting and destroying other cells.B. using lots of the energy currency ATP.C. synthesizing lots of protein.D. communicating electrically with its neighbors.E. undergoing or soon to enter mitosis.\\n5.In diffusing from the mitochondrial matrix of one animal cell to the interior of the Golgi\\napparatus of another cell 50 µm away in the same animal a molecule of CO 2would cross\\na minimum of\\nA. two membranes.\\nB. three membranes.C. four membranes.D.ﬁve membranes.\\nE. six membranes.\\n6.One organelle often found at the cell center is the\\nA. nucleus.B. peroxisome.C. endoplasmic reticulum.D. Golgi apparatus.E. lysosome.\\n7.A stimulated cell will often release calcium ions from its\\nA. nucleus.B. endoplasmic reticulum.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2be3be0c-2f4a-4db2-88e6-2254907807a4', embedding=None, metadata={'page_label': '77', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-03 WY001-Bolsover-v2.cls September 15, 2003 21:3\\nANSWERS TO REVIEW QUESTIONS 63\\nC. mitochondria.\\nD. lysosomes.E. peroxisomes.\\n8.In addition to being found free in the cytoplasm, ribosomes are found on the outer surface\\nof the\\nA. nucleus.\\nB. smooth endoplasmic reticulum.C. mitochondrion.D. chloroplast.E. plasma membrane.\\nANSWERS TO REVIEW QUESTIONS\\n1.E.All these statements about biological membranes are true.\\n2.A.Collagen is a major constituent of the extracellular matrix. Considering the other answers: B\\nand C are wrong because phospholipid is found in membranes while RNA is intracellular. AnswerD is wrong because the extracellular matrix is an open meshwork through which the aqueousextracellular medium and its dissolved solutes can easily pass.\\n3.B.Plants do not have gap junctions, indeed the presence of the cellulose cell wall means that\\neven if the cells produced the protein subunits neighboring cells never approach each other closelyenough to form gap junctions.\\n4.C.Proteins are synthesized using the information on RNA that must be exported from the nucleus\\nto the cytoplasm through nuclear pores. None of the other choices is correct. In particular, thenuclear envelope with all its nuclear pores breaks down in cells undergoing mitosis.\\n5.D.The CO\\n2must diffuse across the inner mitochondrial membrane, the outer mitochondrial\\nmembrane, the plasma membrane of the ﬁrst cell, the plasma membrane of the second cell, and\\nthe membrane of the Golgi apparatus.\\n6.D.The cell center is occupied by the centrioles (which are not organelles because they are not\\nmembrane-bound) and by the Golgi apparatus.\\n7.B.The vast majority of cells are able to release calcium ions from the endoplasmic reticulum when\\nstimulated. The nuclear envelope, mitochondria, and lysosomes may be able to release calciumions in certain more limited circumstances.\\n8.A.Most ribosomes that are not free in the cytoplasm are found on the rough endoplasmic reticulum,\\nbut some are found on the outer nuclear membrane, which is continuous with the membrane of theendoplasmic reticulum (see Fig. 3.5). Mitochondria and chloroplasts have their own ribosomes,but these are found inside these organelles, not on the outer surface.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a3c440e0-f1e7-4e93-aed7-5417135993ef', embedding=None, metadata={'page_label': '78', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\n4\\nDNA STRUCTURE AND\\nTHE GENETIC CODE\\nINTRODUCTION\\nOur genes are made of deoxyribonucleic acid (DNA ). This remarkable molecule contains\\nall the information necessary to make a cell, and DNA is able to pass on this information\\nwhen a cell divides. This chapter describes the structure and properties of DNA molecules,the way in which our DNA is packaged into chromosomes, and how the information storedwithin DNA is retrieved via the genetic code.\\nTHE STRUCTURE OF DNA\\nDeoxyribonucleic acid is an extremely long polymer made from units called deoxyribonu-cleotides, which are often simply called nucleotides. These nucleotides differ from thosedescribed in Chapter 2 in one respect: the sugar is deoxyribose, not ribose. Figure 4.1shows one deoxyribonucleotide, deoxyadenosine triphosphate. Note that deoxyribose, un-like ribose (page 29), has no OH group on its 2\\n′carbon. Four bases are found in DNA;\\nthey are the two purines adenine (A) and guanine (G) and the two pyrimidines cytosine(C) and thymine (T) (Fig. 4.2). The combined base and sugar is known as a nucleosideto distinguish it from the phosphorylated form, which is called a nucleotide. Four differ-ent nucleotides join to make DNA. They are 2\\n′-deoxyadenosine-5′-triphosphate (dATP),\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n65', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='18a2669f-a8ac-47fa-9e6f-03b6dd84d74d', embedding=None, metadata={'page_label': '79', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\n66 DNA STRUCTURE AND THE GENETIC CODE\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 4.1 Gift from the Dead\\nDeoxyribonucleic acid was known to contain genetic information several years before its structure\\nwas ﬁnally determined. As early as 1928 Fred Grifﬁth carried out the now famous pneumococ-cus transformation experiment. There are several strains of the bacterium Diplococcus pneumoniae.\\nSome strains cause pneumonia and are said to be virulent; others do not and are nonvirulent. The vir-ulent bacteria possess a polysaccharide coat and, when grown on an agar plate, create colonies witha smooth appearance—these are S bacteria. Nonvirulent bacteria do not have this polysaccharidecoat, and their colonies are rough in appearance—these are R bacteria. Mice injected with S bacteriadeveloped pneumonia and died, whereas those injected with R bacteria were unaffected. When S bac-teria were killed by heat treatment before injection, the mice remained healthy. However, those miceinjected with a mixture of live R bacteria and heat-killed S bacteria died of pneumonia. This observa-tion meant that something in the heat-killed S bacteria carried the information that enabled bacteriato make the polysaccharide coat and therefore to change—transform—the R bacteria into a virulentstrain. Later experiments in the mid-1940s by Oswald Avery, Maclyn McCarty, and Colin MacLeodclearly demonstrated that the transforming factor was DNA. They made extracts of S bacteria andtreated them with enzymes that destroy either DNA, RNA, or protein. These extracts were then mixedwith R bacteria. Only the extract rich in DNA was able to transform R bacteria into S bacteria.\\nmouse injected with \\nS bacteria diesS\\nSSS\\nmouse injected with R bacteria stays healthyR\\nR R\\nR\\nmouse injected with heat-killed S bacteria stays healthyS\\nSSS\\nS\\nS\\nSS\\nRRRR mouse injected with live R bacteria + heat-killed S \\nbacteria dies\\n5′\\n4′\\n3′ 2′1′\\ntriphosphateadenine (base)\\nN\\nCC\\nN C\\nN C\\nHN\\nC\\nNHH\\nH\\nO\\nC\\nC CC H\\nOHH\\nHH HCH2O P\\nOO P\\nOO P\\nOOOOO\\ndeoxyribosea deoxynucleoside \\ntriphosphate (dATP)\\nFigure 4.1. 2′-deoxyadenosine-5′-triphosphate.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1cf4e472-26b5-43fa-80fa-39b6fea02912', embedding=None, metadata={'page_label': '80', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\nTHE STRUCTURE OF DNA 67\\nCC\\nCC\\nH C\\nH OO\\nNHH\\nH\\nNHH\\nCC\\nH C\\nC N\\nH ONH\\nH\\nNpyrimidines\\ncytosine (C)\\nthymine (T)N\\nCC\\nN C\\nN C\\nHN\\nC\\nN\\nHHH\\nHpurines\\nO\\nCC\\nH N C\\nN C\\nNN\\nC\\nN\\nHH\\nH\\nH\\nguanine (G)adenine (A)\\nFigure 4.2. The Four bases found in DNA.\\n2′-deoxyguanosine-5′-triphosphate (dGTP), 2′-deoxycytidine-5′-triphosphate (dCTP), and\\n2′-deoxythymidine-5′-triphosphate (dTTP).\\nDNA molecules are very large. The single chromosome of the bacterium Escherichia\\ncoliis made up of two strands of DNA that are hydrogen-bonded together to form a single\\ncircular molecule comprising 9 million nucleotides. Humans have 46 DNA molecules in\\neach cell, each forming one chromosome. We inherit 23 chromosomes from each parent.Each set of 23 chromosomes encodes a complete copy of our genome and is made up of\\n6×10\\n9nucleotides (or 3 ×109base pairs —see below). We do not yet know the exact\\nnumber of genes that encode messenger RNA and therefore proteins in the human genome.The current estimate is in the range of 30,000. Table 4.1 compares the number of predictedmessenger RNA genes in the genomes of different organisms. In each organism, there arealso approximately 100 genes that code for ribosomal RNAs and transfer RNAs. The rolethese three types of RNA play in protein synthesis is described in Chapter 8.\\nFigure 4.3 illustrates the structure of the DNA chain. As nucleotides are added to the\\nchain by the enzyme DNA polymerase (Chapter 5), they lose two phosphate groups. The last(theαphosphate) remains and forms a phosphodiester link between successive deoxyribose\\nresidues. The bond forms between the hydroxyl group on the 3\\n′carbon of the deoxyribose of\\none nucleotide and the α-phosphate group attached to the 5′carbon of the next nucleotide.\\nAdjacent nucleotides are hence joined by a 3′–5′phosphodiester link. The linkage gives rise', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='80a97876-5e01-4bec-bd06-c32022064a48', embedding=None, metadata={'page_label': '81', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\n68 DNA STRUCTURE AND THE GENETIC CODE\\nTable 4.1. Numbers of Predicted Genes in Various Organisms\\nOrganism Number of Predicted Genes\\nBacterium —Haemophilus inﬂuenzae 1,709\\nYeast —Saccharomyces cerevisiae 6,241\\nFruit ﬂy—Drosophila melannogaster 13,601\\nWorm —Caenorhabdites elegans 18,424\\nPlant —Arabidopsis thaliana 25,498\\nHuman —Homo sapiens ∼30,000\\nto the sugar –phosphate backbone of a DNA molecule. A DNA chain has polarity because its\\ntwo ends are different. In the ﬁrst nucleotide in the chain, the 5′carbon of the deoxyribose is\\nphosphorylated but otherwise free. This is called the 5′end of the DNA chain. At the other\\nend is a deoxyribose with a free hydroxyl group on its 3′carbon. This is called the 3′end.\\nThe DNA Molecule Is a Double Helix\\nIn 1953 Rosalind Franklin used X-ray diffraction to show that DNA was a helical (i.e.,\\ntwisted) polymer. James Watson and Francis Crick demonstrated, by building three-dimensional models, that the molecule is a double helix (Fig. 4.4). Two hydrophilicf\\nIN DEPTH 4.1 Gene Number and Complexity\\nAs the genomes of more and more organisms were sequenced, the most sur-\\nprising feature to emerge was just how few genes supposedly “complex” organ-isms possess. The ﬁrst eukaryotic genome to be sequenced was that of the lowlybudding yeast, Saccharomyces cerevisiae, the simple unicellular fungus that we\\nuse to make bread and beer. S. cerevisiae has about 6000 genes. The fruit ﬂy,\\nDrosophila melanogaster, a much more complex organism with a brain, nervous\\nand digestive systems, and the ability to ﬂy and navigate, on the other hand, has13,600 genes, or roughly twice the number in a yeast. Even more surprising was thecase of the human genome. Prior to the completion of the Human Genome Projectpredictions of the number of human genes were in the order of 100,000. Surelythis complex vertebrate that could send a spaceship to Mars and write War and\\nPeace would need vastly more genes than the fruit ﬂy. In the event, the number\\nof human genes turned out to be much lower than expected, about 30,000, onlytwice as many genes as in Drosophila.\\nIf complex biological and social achievements are not the result of having more\\ngenes, where do they come from? Two factors help the human genome generatea more complex organism. First, having more DNA per gene means that moreDNA can be used in enhancer sequences (page 120), allowing more subtle control\\nof where, when, and to what extent a gene is expressed. Second, there is not astraightforward one-to-one relationship between genes and proteins. Alternative\\nsplicing (page 118) allows the cell to “cut and paste” a messenger RNA molecule in\\ndifferent ways to produce many different proteins from the same gene. Estimatesare that something like 50% of human genes show alternative splicing with thepattern of splicing (the range of proteins produced) varying from tissue to tissue.Drosophila genes also show alternative splicing but those of yeast, which contain\\nfewintrons (page 98), do not.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='9493b998-ad35-41c0-bf53-30985d4383f0', embedding=None, metadata={'page_label': '82', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\nTHE STRUCTURE OF DNA 69\\nphosphodiester link5′\\n3′\\n5′\\n3′\\n5′\\n3′O\\nC\\nC CC HH\\nHHCH2O P\\nOO P\\nOO P\\nOOOOO\\nO\\nC\\nC CC HH\\nHHCH2O P\\nOOO\\nO\\nC\\nC CC H\\nOHH\\nHHCH2O P\\nOOOH\\nH\\nHbase\\nbase\\n3′ end5′ endbase\\nsugar-phosphate backbone of DNAmany repeating units\\nFigure 4.3. The Phosphodiester link and the sugar –phosphate backbone of DNA.\\nsugar –phosphate backbones lie on the outside of the molecule, and the purines and pyrim-\\nidines lie on the inside of the molecule. There is just enough space for one purine and\\none pyrimidine in the center of the double helix. The Watson –Crick model showed that\\nthe purine guanine (G) would ﬁt nicely with the pyrimidine cytosine (C), forming three\\nhydrogen bonds. The purine adenine (A) would ﬁt nicely with the pyrimidine thymine (T),\\nforming two hydrogen bonds. Thus A always pairs with T, and G always pairs with C.The three hydrogen bonds formed between G and C produce a relatively strong base pair.Because only two hydrogen bonds are formed between A and T, this weaker base pair ismore easily broken. The difference in strengths between a G –C and an A –T base pair is\\nimportant in the initiation and termination of RNA synthesis (page 108). The two chainsof DNA are said to be antiparallel because they lie in the opposite orientation with respectto one another, with the 3\\n′-hydroxyl terminus of one strand opposite the 5′-phosphate ter-\\nminus of the second strand. The sugar –phosphate backbones do not completely conceal\\nthe bases inside. There are two grooves along the surface of the DNA molecule. One iswide and deep —the major groove —and the other is narrow and shallow —the minor groove\\n(Fig. 4.4). Proteins can use the grooves to gain access to the bases (page 123).\\nThe Two DNA Chains Are Complementary\\nA consequence of the base pairs formed between the two strands of DNA is that if the basesequence of one strand is known, then that of its partner can be inferred. A Gin one strand', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='7b467bba-ca7a-476a-995c-acb90d60e50d', embedding=None, metadata={'page_label': '83', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\n70 DNA STRUCTURE AND THE GENETIC CODE\\nN\\nCC\\nN C\\nN C\\nHN\\nC\\nN\\ndeoxyriboseHH\\nH\\nCC\\nCC\\nH C\\ndeoxyriboseOO\\nNHH\\nH\\nNHO\\nCC\\nH N C\\nN C\\nNN\\nC\\nN\\ndeoxyriboseH\\nH\\nH\\nthymine-adenine \\nbase pair (T    A)\\nT\\nTACG\\nAT\\nA\\nGA T\\nG C\\nATGH\\nCC\\nH C\\nC N\\ndeoxyribose ONH\\nH\\nN\\ncytosine (C)\\nthymine (T)guanine (G)\\nadenine (A)hydrogen bondcytosine-guanine base pair (C    G)\\nschematic representation of the double- helical structure of DNA\\nsugar-phosphate backbone\\nbase pairsminor groove\\nmajor grooveG C\\nA\\nT\\nFigure 4.4. The DNA double helix is held together by hydrogen bonds.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='033c676e-e033-426e-a272-fa61561e5aef', embedding=None, metadata={'page_label': '84', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\nPACKAGING OF DNA MOLECULES INTO CHROMOSOMES 71\\nwill always be paired with a Cin the other. Similarly an Awill always pair with a T.The\\ntwo strands are therefore said to be complementary.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 4.2 Erwin Chargaff’s Puzzling Data\\nIn a key discovery of the 1950s, Erwin Chargaff analyzed the purine and pyrimidine content of DNA\\nisolated from many different organisms and found that the amounts of A and T were always thesame, as were the amounts of G and C. Such an identity was inexplicable at the time, but helpedJames Watson and Francis Crick build their double-helix model in which every A on one strand ofthe DNA helix has a matching T on the other strand, and every G on one strand has a matching Con the other.\\nDifferent Forms of DNA\\nThe original Watson –Crick model of DNA is now called the B-form. In this form, the two\\nstrands of DNA form a right-handed helix. If viewed from either end, it turns in a clockwise\\ndirection. B-DNA is the predominant form in which DNA is found. Our genome, however,also contains several variations of the B-form double helix. One of these, Z-DNA, so-calledbecause its backbone has a zig-zag shape, forms a left-handed helix and occurs when theDNA sequence is made of alternating purines and pyrimidines. Thus the structure adoptedby DNA is a function of its base sequence.\\nDNA AS THE GENETIC MATERIAL\\nDeoxyribonucleic acid carries the genetic information encoded in the sequence of the fourbases —adenine, guanine, cytosine, and thymine. The information in DNA is transferred\\nto its daughter molecules through replication (the duplication of DNA molecules) andsubsequent cell division. DNA directs the synthesis of proteins through the intermediarymolecule RNA. The DNA code is transferred to RNA by a process known as transcription(Chapter 6). The RNA code is then translated into a sequence of amino acids during proteinsynthesis (Chapter 8). This is the central dogma of molecular biology: DNA makes RNAmakes protein.\\nRetroviruses such as human immunode ﬁciency virus, the cause of AIDS, are an ex-\\nception to this rule. As their name suggests, they reverse the normal order of data transfer.Inside the virus coat is a molecule of RNA plus an enzyme that can make DNA from anRNA template by the process known as reverse transcription.\\nPACKAGING OF DNA MOLECULES INTO CHROMOSOMES\\nEukaryotic Chromosomes and Chromatin Structure\\nA human cell contains 46 chromosomes (23 pairs), each of which is a single DNA moleculebundled up with various proteins. On average, each human chromosome contains about1.3×10\\n8base pairs (bp) of DNA. If the DNA in a human chromosome were stretched as', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='10b2fdf4-769b-4ce4-ab8a-2d0cd5c5d80b', embedding=None, metadata={'page_label': '85', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\n72 DNA STRUCTURE AND THE GENETIC CODE\\nDNA has to be highly \\ncompacted in order to fit into the cell1400 nm\\nhistone octamer30nmsolenoid\\n30nm\\nnucleosomes\\n(\"beads on a string\")2 nmCG\\nAT\\nA\\nTA\\nTG\\nC\\nT\\nFigure 4.5. How DNA is packaged into chromosomes.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='68207dc0-3d72-4f93-986f-9b9aba58492d', embedding=None, metadata={'page_label': '86', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\nPACKAGING OF DNA MOLECULES INTO CHROMOSOMES 73\\nfar as it would go without breaking it would be about 5 cm long, so the 46 chromosomes in\\nall represent about 2 m of DNA. The nucleus in which this DNA must be contained has adiameter of only about 10 µm, so large amounts of DNA must be packaged into a small space.\\nThis represents a formidable problem that is dealt with by binding the DNA to proteins toform chromatin. As shown in Figure 4.5, the DNA double helix is packaged at both smalland larger scales. In the ﬁrst stage, shown on the right of the ﬁgure, the DNA double helix\\nwith a diameter of 2 nm is bound to proteins known as histones. Histones are positively\\ncharged because they contain high amounts of the amino acids arginine and lysine (page185) and bind tightly to the negatively charged phosphates on DNA. A 146 bp length ofDNA is wound around a protein complex composed of two molecules each of four differenthistones —H2A, H2B, H3, and H4 —to form a nucleosome. Because each nucleosome\\nis separated from its neighbor by about 50 bp of linker DNA, this unfolded chromatinstate looks like beads on a string when viewed in an electron microscope. Nucleosomesundergo further packaging. A ﬁfth type of histone, H1, binds to the linker DNA and pulls\\nthe nucleosomes together helping to further coil the DNA into chromatin ﬁbers 30 nm in\\ndiameter, which are referred to as 30-nm solenoids. The ﬁbers then form loops with the\\nhelp of a class of proteins known as nonhistones, and this further condenses the DNA(panels on left-hand side of Fig. 4.5) into a higher order set of coils in a process calledsupercoiling.\\nIn a normal interphase cell about 10% of the chromatin is highly compacted and visible\\nunder the light microscope (page 57). This form of chromatin is called heterochromatin andis the portion of the genome where no RNA synthesis is occurring. The remaining interphasechromatin is less compacted and is known as euchromatin.\\nChromatin is in its most compacted form when the cell is preparing for mitosis, as shown\\nat the top left of Figure 4.5. The chromatin folds and condenses further to form the 1400-nm-wide chromosomes we see under the light microscope. Because the cell is to divide,the DNA has been replicated, so that each chromosome is now formed by two chromatids,each one a DNA double helix. This means the progeny cell, produced by division of theprogenitor cell, will receive a full set of 46 chromosomes. Figure 4.6 is a photograph ofhuman chromosomes as they appear at cell division.\\nProkaryotic Chromosomes\\nThe chromosome of the bacterium E. coli is a single circular DNA molecule of about\\n4.5×106base pairs. It has a circumference of 1 mm, yet must ﬁt into the 1- µm cell, so\\nlike eukaryotic chromosomes it is coiled, supercoiled, and packaged with basic proteinsthat are similar to eukaryotic histones. However, an ordered nucleosome structure similar tothe“beads on a string ”seen in eukaryotic cells is not observed in prokaryotes. Prokaryotes\\ndo not have nuclear envelopes so the condensed chromosome together with its associatedproteins lies free in the cytoplasm, forming a mass that is called the nucleoid to emphasize\\nits functional equivalence to the eukaryotic nucleus.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ce1bfae6-2d4f-4ba1-99d7-f66e94f9fe72', embedding=None, metadata={'page_label': '87', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\n74 DNA STRUCTURE AND THE GENETIC CODE\\nPlasmids\\nPlasmids are small circular minichromosomes found in bacteria and some eukaryotes. They\\nare several thousand base pairs long and are probably tightly coiled and supercoiled inside\\nthe cell. Plasmids often code for proteins that confer resistance to a particular antibiotic.In Chapter 7 we describe how plasmids are used by scientists and genetic engineers toartiﬁcially introduce foreign DNA molecules into bacterial cells.\\nViruses\\nViruses (page 11) rely on the host cell to make more virus. Once viruses have entered cells,the cells ’machinery is used to copy the viral genome. Depending on the virus type, the\\ngenome may be single- or double-stranded DNA, or even RNA. A viral genome is packagedwithin a protective protein coat. Viruses that infect bacteria are called bacteriophages. One\\nof these, called lambda, has a ﬁxed-size DNA molecule of 4 .5×10\\n4base pairs. In contrast,\\nthe bacteriophage M13 can change its chromosome size, its protein coat expanding inparallel to accommodate the chromosome. This makes M13 useful in genetic engineering(Chapter 7).f\\nIN DEPTH 4.2 DNA —A Gordian Knot\\nAt the start of his career Alexander the Great was shown the Gordian Knot, a tan-\\ngled ball of knotted rope, and told that whoever untied the knot would conquerAsia. Alexander cut through the knot with his sword. A similar problem occurs inthe nucleus, where the 46 chromosomes form 2 m of tangled, knotted DNA. Howdoes the DNA ever untangle at mitosis? The cell adopts Alexander ’s solution —it\\ncuts the rope. At any place where the DNA helix is under strain, for instance, wheretwo chromosomes press against each other, an enzyme called topoisomerase II cuts\\none chromosome double helix so that the other can pass through the gap. Then,surpassing Alexander, the enzyme rejoins the cut ends. Topoisomerases are activeall the time in the nucleus, relieving any strain that develops in the tangled massof DNA.\\nchromosome\\nunder tension\\ntopoisomerase II moves tothe region of most strain', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ea84d2af-d680-4f6a-9f0c-5c0fbe2d4b56', embedding=None, metadata={'page_label': '88', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\nTHE GENETIC CODE 75\\nADP + PiATPtopoisomerase II1st chromosome \\nis bound to the enzyme topoisomerase II\\n2nd chromosome \\nis bound to the enzyme topoisomerase II\\n1st chromosome is \\ncut and the cut ends are separated\\n2nd chromosome \\npasses through the gap in the 1st chromosome \\n2nd chromosome is released by the enzyme\\nDNA of 1st chromosomeis resealed\\nTHE GENETIC CODE\\nProteins are linear polymers of individual building blocks called amino acids (page 37).\\nThe sequence of bases along the DNA strand determines the sequence of the amino acids inproteins. There are 20 different amino acids in proteins but only 4 different bases in DNA(A, T, C, and G). Each amino acid is speci ﬁed by a codon, a group of three bases. Because', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e2287470-a2cd-4ad3-b7b2-7586210f9774', embedding=None, metadata={'page_label': '89', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\n76 DNA STRUCTURE AND THE GENETIC CODE\\nthere are 4 bases in DNA, a three-letter code gives 64 (4 ×4×4) possible codons. These\\n64 codons form the genetic code —the set of instructions that tells a cell the order in which\\namino acids are to be joined to form a protein. Despite the fact that the sequence of codons\\non DNA determines the sequence of amino acids in proteins, the DNA helix does not itselfplay a role in protein synthesis. The translation of the sequence from codons into amino\\nacids occurs through the intervention of members of a third class of molecule —messenger\\nRNAs (mRNA) (Fig. 4.7). Messenger RNA acts as a template, guiding the assembly ofamino acids into a polypeptide chain. Messenger RNA uses the same code as the one usedin DNA with one difference: In mRNA the base uracil (U) (Fig. 2.13 on page 34) is usedin place of thymine (T). When we write the genetic code, we usually use the RNA format,that is, we use U instead of T.\\nThe code is read in sequential groups of three, codon by codon. Adjacent codons do\\nnot overlap, and each triplet of bases speci ﬁes one particular amino acid. This discovery\\nFigure 4.6. A spread of human chromosomes (at metaphase —see page 403). The green signal re-\\nveals the gene called FMO3, which when mutated causes trimethylaminuria ( ﬁsh odor syndrome)\\n(Medical Relevance 4.1). There are two copies of the gene, one inherited from each parent, in-\\ndicated by the arrows. The FMO3 gene is located on the long arm of chromosome 1, the longest\\nhuman chromosome.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='dd740de0-d524-4f64-9d9c-be84fd7e7343', embedding=None, metadata={'page_label': '90', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\nTHE GENETIC CODE 77\\nDNA 5′ 3′\\ntranscription5′ 3′\\n\\uf8f5AUG \\uf8f5GGC \\uf8f5UAC \\uf8f5CCC \\uf8f5UGC \\uf8f5CUG \\uf8f5\\ntranslation3′ 5′ mRNA\\nprotein N-terminus met gly tyr pro cys leu C-terminus\\nThe 3 ′ to 5′ DNA strand is transcribed into an mRNA molecule that is translated,\\naccording to the genetic code, into protein. Note that mRNA is made in the 5 ′\\nto 3′ direction and proteins are synthesized from the N terminus. The 3-and\\n1- letter amino acid codes are shown.MG  Y P C LATG GGC TAC CCC TGC CTG\\nTAC CCG ATG GGG ACG GAC\\nFigure 4.7. The central dogma of molecular biology.\\nwas made by Sydney Brenner, Francis Crick, and their colleagues by studying the effect of\\nvarious mutations (changes in the DNA sequence) on the bacteriophage T4, which infectsthe common bacterium E. coli. If a mutation caused either one or two nucleotides to be\\nadded or deleted, from one end of the T4 DNA, then a defective polypeptide was produced,with a completely different sequence of amino acids. However, if three bases were addedor deleted, then the protein made often retained its normal function. These proteins werefound to be identical to the original protein, except for the addition or loss of one aminoacid.\\nThe identi ﬁcation of the triplets encoding each amino acid began in 1961. This was made\\npossible by using a cell-free protein synthesis system prepared by breaking open E. coli\\ncells. Synthetic RNA polymers, of known sequence, were added to the cell-free systemtogether with the 20 amino acids. When the RNA template contained only uridine residues(poly-U) the polypeptide produced contained only phenylalanine. The codon specifyingthis amino acid must therefore be UUU. A poly-A template produced a polypeptide oflysine, and poly-C one of proline: AAA and CCC must therefore specify lysine and proline,respectively. Synthetic RNA polymers containing all possible combinations of the bases A,C, G, and U, were added to the cell-free system to determine the codons for the other aminoacids. A template made of the repeating unit CU gave a polypeptide with the alternatingsequence leucine –serine. Because the ﬁrst amino acid in the chain was found to be leucine,\\nits codon must be CUC and that for serine UCU. Although much of the genetic codewas read in this way, the amino acids de ﬁned by some codons were particularly hard\\nto determine. Only when speci ﬁctransfer RNA molecules (page 163) were used was it\\npossible to demonstrate that GUU codes for valine. The genetic code was ﬁnally solved\\nby the combined efforts of several research teams. The leaders of two of these, MarshallNirenberg and Gobind Khorana, received the Nobel prize in 1968 for their part in crackingthe code.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='cef93832-891d-4696-a11e-dca4e3c40329', embedding=None, metadata={'page_label': '91', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\n78 DNA STRUCTURE AND THE GENETIC CODE\\nalanine (ala)       A\\ncysteine (cys)    C\\nhistidine (his)   H\\nmethionine (met)   M\\nthreonine (thr)   Tasparagine (asn)   N The genetic code and the corresponding amino acid side chains.\\nglutamine (gln)     Q\\nisoleucine (ile)   I\\nphenylalanine (phe)  F\\ntryptophan (trp)   Waspartate (asp)    D\\nglutamate (glu)   E\\nleucine (leu)   L\\nproline (pro)   P\\ntyrosine (tyr)   Yarginine (arg)    R\\nglycine (gly)    G\\nlysine (lys)     K\\nserine (ser)   S\\nvaline (val)   VCH2CH2\\nC\\nHNCH2\\nCH2HNCH3NH2\\nNH2CH2\\n(CH2)3NH2\\nHNH2\\nNHC\\nNH3\\n(CH2)4\\nOH\\nCH2O O\\nO–O \\nCCC\\nCH(CH2)2CH2SH\\nSCH3\\nCH HOCH3(CH2)2CH2\\nCH2\\nCH2\\nCH2OHCH H3CH3CCH3CH3\\nCHH3CC H3CCC\\nCH2O\\nCCC\\nO\\nCCC\\n(CH2)2\\nCH2HC\\nN\\nHCCH HNGCU\\nGCCGCAGCGAAUAAGGAUGACCGUCGCCGACGGAGAAGG\\nUGUUGCCAACAGGAAGAGGGUGGCGGAGGG\\nCAUCACAUUAUCAUAUUAUUGCUUCUCCUACUGAAAAAG\\nAUG UUU\\nUUCCCUCCCCCACCGAGUAGCUCUUCCUCAUCG\\nACUACCACGACAUGG UAU\\nUAC\\nUGA UAA\\nUAGGUUGUCGUGGUA\\nSTOP STOP\\nFigure 4.8. The genetic code and the corresponding amino acid side chains. The three- and one-\\nletter amino acid abbreviations are shown. Hydrophilic side chains are shown in green, hydropho-\\nbic side chains in black. The signi ﬁcance of this distinction is discussed in Chapter 9.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='460fdedb-1659-4bfa-87ba-0ec0eb0aaf7b', embedding=None, metadata={'page_label': '92', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\nTHE GENETIC CODE 79\\nAmino Acid Names Are Abbreviated\\nTo save time we usually write an amino acid as either a three-letter abbreviation, for example,\\nglycine is written as gly and leucine as leu, or as a one-letter code, for example, glycine is Gand leucine is L. Figure 4.8 shows the full name, and the three- and one-letter abbreviations,used for each of the 20 amino acids found in proteins.\\nThe Code Is Degenerate But Unambiguous\\nTo introduce the terms degenerate andambiguous, consider the English language. English\\nshows considerable degeneracy, meaning that the same concept can be indicated usinga number of different words —think, for example, of lockup, cell, pen, pound, brig, and\\ndungeon. English also shows ambiguity, so that it is only by context that one can tell\\nwhether cellmeans a lockup orthe basic unit of life. Like the English language the genetic\\ncode shows degeneracy, but unlike language the code is unambiguous.\\nThe 64 codons of the genetic code are shown in Figure 4.8 together with the side\\nchains of the amino acids for which each codes. Amino acids with hydrophilic side chainsare shown in green while those with hydrophobic side chains are in black. The importanceof this distinction will be discussed in Chapter 9. Sixty-one codons specify an amino acid,and the remaining three act as stop signals for protein synthesis. Methionine and tryptophan\\nare the only amino acids coded for by single codons. The other 18 amino acids are encodedby either 2, 3, 4, or 6 codons and so the code is degenerate. No triplet codes for more thanone amino acid and so the code is unambiguous. Notice that when two or more codonsspecify the same amino acid, they usually only differ in the third base of the triplet. Thusmutations can arise in this position of the codon without altering amino acid sequences.Perhaps degeneracy evolved in the triplet system to avoid a situation in which 20 codonseach meant one amino acid, and 44 speci ﬁed none. If this were the case, then most mutations\\nwould stop protein synthesis dead.\\nStart and Stop Codons and the Reading Frame\\nThe order of the codons in DNA and the amino acid sequence of a protein are colinear.The start signal for protein synthesis is the codon AUG specifying the incorporation of\\nmethionine. Because the genetic code is read in blocks of three, there are three potentialreading frames in any mRNA. Figure 4.9 shows that only one of these results in the\\nsynthesis of the correct protein. When we look at a sequence of bases, it is not obviouswhich of the reading frames should be used to code for protein. As we shall see later\\nA  U  G \\uf8f5C  U  A \\uf8f5G  A  A \\uf8f5U  A  C ... reading frame 1met his glu tyr\\nA\\uf8f5U  G  C \\uf8f5U  A  G \\uf8f5A  A  U \\uf8f5A  C ... reading frame 2cys stop asn\\nA  U \\uf8f5G  C  U \\uf8f5A  G  A \\uf8f5A  U  A \\uf8f5C ... reading frame 3ala arg ile\\nFigure 4.9. Reading frames —the genetic code is read in blocks of three.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='47f390ec-eae8-495f-a689-9abc6391a4e3', embedding=None, metadata={'page_label': '93', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\n80 DNA STRUCTURE AND THE GENETIC CODE\\nAUGmet\\nCAAgln\\nUG Gtrp\\nGUCval\\nGAGglu\\nA.....\\nU deleted\\nAUGmet\\nCAAgln\\nGGGgly\\nUCGser\\nAGAarg\\n......frameshift \\nmutation\\nAUGmet\\nCAAgln\\nUGGtrp\\nGUCval\\nGAGglu\\n......\\nchanged to A\\nAUGmet\\nCAAgln\\nUGASTO\\nGUCval\\nGAGglu\\n......nonsense mutation\\nAUGmet\\nCAAgln\\nUGGtrp\\nGUCval\\nGAGglu\\n......\\nchanged to U\\nAUGmet\\nCAUhis\\nUGGtrp\\nGUCval\\nGAGglu\\n......missense mutationnormal\\nnormal\\nnormalmutant\\nmutant\\nmutant\\nFigure 4.10. Mutations that alter the sequence of bases.\\n(page 168), the ribosome scans along the mRNA until it encounters an AUG. This both\\ndeﬁnes the ﬁrst amino acid of the protein and the reading frame used from that point on.\\nA mutation that inserts or deletes a nucleotide will change the normal reading frame and iscalled a frameshift mutation (Fig. 4.10).\\nThe codons UAA, UAG, and UGA are stop signals for protein synthesis. A base\\nchange that causes an amino acid codon to become a stop codon is known as a nonsense\\nmutation (Fig. 4.10). If, for example, the codon for tryptophan UGG changes to UGA,\\nthen a premature stop signal will have been introduced into the messenger RNA template.A shortened protein, usually without function, is produced.\\nThe Code Is Nearly Universal\\nThe code shown in Figure 4.8 is the one used by organisms as diverse as E. coli and humans\\nfor their nuclear-encoded proteins .It was originally thought that the code would be universal.\\nHowever, several mitochondrial genes use UGA to mean tryptophan rather than stop. The\\nnuclear code for some protists uses UAA and UAG to code for glutamine rather than stop.\\nMissense Mutations\\nA mutation that changes the codon from one amino acid to that for another is a missense\\nmutation (Fig. 4.10). As shown in Figure 4.8, the second base of each codon shows the most\\nconsistency with the chemical nature of the amino acid it encodes. Amino acids with charged,', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='22d1090c-c9de-4e15-99b4-6747786753ab', embedding=None, metadata={'page_label': '94', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\nSUMMARY 81\\nhydrophilic side chains usually have A or G —a purine —in the second position. Those with\\nhydrophobic side chains usually have C or U —a pyrimidine —in that position. This has\\nimplications for mutations of the second base. Substitution of a purine for a pyrimidine\\nis very likely to change the chemical nature of the amino acid side chain signi ﬁcantly\\nand can therefore seriously affect the protein. Sickle cell anemia is an example of such amutation. At position 6 in the β-globin chain of hemoglobin the mutation in DNA changes\\na glutamate residue encoded by G AG to a valine residue encoded by G TG( G UGi nR N A ) .\\nThe shorthand notation for this mutation is E6V , meaning that the glutamate (E) at position6 of the protein becomes a valine (V). This change in amino acid alters the overall charge ofthe chain, and the hemoglobin tends to precipitate in the red blood cells of those affected.The cells adopt a sickle shape and therefore tend to block blood vessels, causing painfulcramplike symptoms and progressive damage to vital organs.\\nSUMMARY\\n1.DNA, the cell ’s database, contains the genetic information necessary to encode RNA\\nand protein.\\n2.The information is stored in the sequence of four bases. These are the purines, adenineand guanine, and the pyrimidines, cytosine and thymine. Each base is attached tothe 1\\n′-carbon atom of the sugar deoxyribose. A phosphate group is attached to the\\n5′-carbon atom of the sugar. The base +sugar +phosphate is called a nucleo-\\ntide.\\n3.The enzyme DNA polymerase joins nucleotides together by forming a phosphodiesterlink between the hydroxyl group on the 3\\n′carbon of deoxyribose of one nucleotide and\\nthe 5′-phosphate group of another. This gives rise to the sugar –phosphate backbone\\nstructure of DNA.\\n4.The two strands of DNA are held together in a double-helical structure becauseguanine hydrogen bonds with cytosine and adenine hydrogen bonds with thymine.This means that if the sequence of one strand is known, that of the other can beinferred. The two strands are complementary in sequence.\\n5.DNA binds to histone and nonhistone proteins to form chromatin. DNA is wrappedaround histones to form a nucleosome structure. This is then folded again andagain. This packaging compresses the DNA molecule to a size that ﬁts into the\\ncell.\\n6.The genetic code speci ﬁes the sequence of amino acids in a polypeptide. The code\\nis transferred from DNA to mRNA and is read in groups of three bases (a codon)during protein synthesis. There are 64 codons; 61 specify an amino acid and 3 arethe stop signals for protein synthesis.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d31874ae-7735-4728-ac18-26a8e0d787ec', embedding=None, metadata={'page_label': '95', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\n82 DNA STRUCTURE AND THE GENETIC CODE\\n✎\\n✍\\x0c\\n✌Medical\\nRelevance\\n4.1Trimethylaminuria\\nTrimethylaminuria is an inherited disorder characterized by an unpleasant body odor\\nsimilar to rotting ﬁsh. Mutations in the gene FMO3, located on human chromosome\\n1 (Fig. 4.6) produce a defective version of a protein called ﬂavin-containing monooxygenase 3\\n(FMO3). One of the functions of this protein, which is found in the liver, is to convert a chemicalcalled trimethylamine (very smelly) to trimethylamine N-oxide (nonsmelly). FMO3 carries out this\\nreaction by adding an oxygen onto the nitrogen of trimethylamine. Defective FMO3 protein cannotcarry out this reaction. Affected individuals therefore excrete large amounts of trimethylamine intheir breath, sweat, and urine. Trimethylamine is the chemical that gives rotting ﬁsh its distinctive\\nand unpleasant smell, and this gives rise to the alternative name for trimethylaminuria —ﬁsh odor\\nsyndrome.\\nTrimethylamine is derived from our diet in two ways. Fish contain lots of trimethylamine\\nN-oxide (page 208). The bacteria that live in our intestines remove the oxygen from the nitrogen\\nof the N-oxide to produce trimethylamine. However, one cannot avoid trimethylamine simply by\\navoiding ﬁsh. The phospholipids in the cell membranes of food include phosphatidylcholine, and\\nthis is hydrolyzed by our digestive enzymes to release free choline. Gut bacteria then break downthe choline further, producing trimethylamine together with acetaldehyde (also called ethanal).\\nTrimethylaminuria is a dif ﬁcult disorder to live with. Because of their body odor, sufferers\\noften experience rejection and social isolation. A diet very low in choline, and other chemicalsthat contain a trimethylamine group, can help to reduce the problem.\\nAnalysis of the faulty genes in different families has revealed different mutations in the FMO3\\ngene. Some families carry the missense mutation P153L in which a CCC that codes for a prolineas the 153rd amino acid in the normal protein is mutated to CTC (leu), a change that turns out to becritical for the operation of the protein. Other families carry a nonsense mutation in which the GGA(gly) at amino acid 148 in the normal protein, is mutated to TGA, a stop codon. Another familycarries a frameshift mutation in which the sequence . . . GGA AAG CGT GT CC T GG..., which\\nspeci ﬁes amino acids 188gly lys arg val leu192 in the normal protein, is mutated, by the deletion\\nof the highlighted (bold) G, to . . . GGA AAG CGT TCC TG G..., thus generating the amino acid\\nsequence 188gly lys arg ser trp 192. The frameshift mutation changes not only the amino acid\\nsequence, but also introduces a stop codon further down the mRNA. Because different mutationscan give rise to trimethylaminuria, the development of a general diagnostic test for carriers of thefaulty gene (who also have one good copy and are therefore carriers who do not themselves havethe disorder; page 407) is not possible.\\nIn contrast, a disorder such as sickle cell anemia is caused, in all individuals, by the same\\nmutation (page 81). One test will diagnose the defective gene and prospective parents can beadvised and counseled.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='824dd6f3-ab7b-4d32-9493-e448698faff0', embedding=None, metadata={'page_label': '96', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\nTRIMETHYLAMINURIA 83\\n✎\\n✍\\x0c\\n✌phosphatidyl cholinehuman\\ndigestive\\nenzymescholine\\nC H\\nHHH\\nOOOCH HH\\nOHCH\\nOH\\nglycenoltrimethylamine acetaldehyde\\n(ethanal)smelly\\nCH\\nOH\\nC\\nC\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nH\\nHOC\\nHHH\\nOOOCHOCH3CH3\\nO\\nOOCH2CH2CHCH3CH3\\nN\\nP\\nH\\nC\\nC\\nHH C\\nHH C\\nH C\\nH C\\nHH C\\nH C\\nH C\\nHH C\\nH C\\nH C\\nHH C\\nH C\\nH C\\nHH C\\nHH C\\nHH\\nHH C\\nHH CCO\\nstearate\\narachidonateHHO\\nC\\nC\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nHH C\\nH\\nHO\\nHHO\\nC\\nC\\nHH C\\nHH C\\nH C\\nH C\\nHH C\\nH C\\nH C\\nHH C\\nH C\\nH C\\nHH C\\nH C\\nH C\\nHH C\\nHH C\\nHH\\nHH C\\nHH CCOCH3CH3\\nOHCH2CH2CH3\\nN\\nO\\nOOH\\nPenzymebacterialCH3CH3\\nHOCH3\\nN\\nCH3CH3\\nOCH3\\nN H\\ntrimethylamine\\nN-oxidebacterial\\nenzyme\\nnon smellyhuman\\nFM03\\n/H110014H2O3H/H11001\\n/H11001    O2/H11002    O2OP', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6a05b12c-786f-421a-9cae-0b57e82f9817', embedding=None, metadata={'page_label': '97', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\n84 DNA STRUCTURE AND THE GENETIC CODE\\nFURTHER READING\\nDiGuilo, M. 1997. The origin of the genetic code. Trends Biochem. Sci. 22: 49 –50.\\nDolphin, C. T., Janmohamed, A., Smith, R. L., Shephard, E. A., and Phillips, I. R. 1997. Missense\\nmutation in ﬂavin-containing monooxygenase 3 gene, FMO3, underlies ﬁsh-odour syndrome.\\nNature Genetics 17: 491 –494.\\nMaddox, B. 2002. Rosalind Franklin: The Dark Lady of DNA . New York: Harper Collins.\\nRoca, J. 1995. The mechanisms of DNA topoisomerases. Trends Biochem. Sci. 20: 156 –160\\nWatson, J. D., and Crick, F. H. C. 1953. A structure for deoxyribose nucleic acid. Nature 171:\\n737.\\nWoodcock, C. L., and Horowitz, R. A. 1995. Chromatin organization re-viewed. Trends Cell Biol .\\n5: 272 –277.\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.Which base is not found in the DNA of most organisms?\\nA. Adenine\\nB. GuanineC. CytosineD. UracilE. Thymine\\n2.The bonds that join the nucleotides to form a single strand of DNA are\\nA. peptide bonds.B. hydrogen bonds.C. phosphodiester links.D. van der Waals bonds.E. disul ﬁde bonds.\\n3.DNA is packaged into chromatin with the help of the protein called\\nA. DNA polymerase I.B. histone.C. DNA ligase.D. topoisomerase I.E. RNA.\\n4.Histone proteins are rich in the amino acids\\nA. arginine and asparagine.B. leucine and isoleucine.C. aspartate and glutamate.D. arginine and lysine.E. histidine and glutamine.\\n5.The 20 amino acids found in proteins are coded for by\\nA. 59 codons.B. 60 codons.C. 61 codons.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e43d21e3-b4a2-4bbf-bd44-67a05b499c85', embedding=None, metadata={'page_label': '98', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FHB\\nWY001-04 WY001-Bolsover-v2.cls September 16, 2003 10:49\\nANSWERS TO REVIEW QUESTIONS 85\\nD. 63 codons.\\nE. 64 codons.\\n6.Methionine is coded for by\\nA. one codon only.B. two different codons.C. three different codons.D. four different codons.E. six different codons.\\n7.A missense mutation is one that\\nA. creates a stop codon in the mRNA.B. changes the codon of one amino acid to that of another.C. changes the reading frame of the mRNA.D. creates a break in the DNA strand.E. has no effect on the amino acid sequence.\\nANSWERS TO REVIEW QUESTIONS\\n1.D.Uracil, which hydrogen bonds with adenine, is present in RNA but is absent from the DNA of\\nmost organisms.\\n2.C.The deoxyribose residues of adjacent nucleotides are joined by phosphodiester links.\\n3.B.Histones help to package DNA into chromatin.\\n4.D.Histones bind to negatively charged DNA because they are rich in the positively charged amino\\nacids arginine and lysine.\\n5.C.Of the 64 possible permutations of a triplet of four types of base, 61 code for the 20 amino\\nacids, leaving 3 (UGA, UAG, and UAA) to code for stop.\\n6.A.Only AUG codes for methionine, which is the ﬁrst amino acid found in all newly synthesized\\nproteins of most species.\\n7.B.A missense mutation changes the codon of one amino acid to that of another. A nonsense\\nmutation is one that creates a stop codon and a frameshift mutation is one that changes the readingframe of the mRNA.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e59ddcac-1451-464b-b103-93fa090741d2', embedding=None, metadata={'page_label': '99', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\n5\\nDNA AS A DATA STORAGE MEDIUM\\nINTRODUCTION\\nThe genetic material DNA must be faithfully replicated every time a cell divides to ensure\\nthat the information encoded in it is passed unaltered to the progeny cells. DNA moleculeshave to last a long time compared to RNA and protein. The sugar–phosphate backbone ofDNA is a very stable structure because there are no free hydroxyl groups on the sugar—they are all used up in bonds, either to the base or to phosphate. The bases themselvesare protected from chemical attack because they are hidden within the DNA double helix.Nevertheless, chemical changes— mutations —do occur in the DNA molecule, and cells\\nhave had to evolve mechanisms to ensure that mutation is kept to a minimum. Repairsystems are essential for both cell survival and to ensure that the correct DNA sequenceis passed on to daughter cells. This chapter describes how new DNA molecules are madeduring chromosome duplication and how the cell acts to correct base changes in DNA.\\nDNA REPLICATION\\nDuring replication the two strands of the double helix unwind. Each then acts as a templatefor the synthesis of a new strand. This process generates two double-stranded daughter DNAmolecules, each of which is identical to the parent molecule. The base sequences of thenew strands are complementary in sequence to the template strands upon which they werebuilt. This means that G, A, C, and T in the old strand cause C, T, G, and A, respectively,to be placed in the new strand.\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n87', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='49145498-83eb-4510-ba78-44bd475677c7', embedding=None, metadata={'page_label': '100', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\n88 DNA AS A DATA STORAGE MEDIUM\\nsingle-strand\\nbinding protein5′\\n5′3′\\n3′5′3′5′\\n5′\\n3′\\n3′ 5′DnaB  helicase\\nDnaB  helicaseRNA primer\\nleading strand\\nreplication forkDNA polymeraseOkazaki\\nfragmentlagging strand\\nRNA\\nprimaseDNA\\npolymerase\\nFigure 5.1. DNA replication. The helicases, and the replication fork, are moving to the left.\\nThe DNA Replication Fork\\nReplication of a new DNA strand starts at speci ﬁc sequences known as origins of repli-\\ncation . The small circular chromosome of Escherichia coli has only one of these, whereas\\neukaryotic chromosomes, which are much larger, have many. At each origin of replication,\\nthe parental strands of DNA untwist to give rise to a structure known as the replication fork(Fig. 5.1). This unwinding permits each parental strand to act as a template for the synthesisof a new strand. The structure of the double helix and the nature of DNA replication posea mechanical problem. How do the two strands unwind and how do they stay unwound sothat each can act as a template for a new strand?\\nPROTEINS OPEN UP THE DNA DOUBLE HELIX\\nDURING REPLICATION\\nThe DNA molecule must be opened up before replication can proceed. The helix is a very\\nstable structure, and in a test tube the two strands separate only when the temperaturereaches about 90\\n◦C. In the cell the combined actions of several proteins help to separate the\\ntwo strands. Much of our knowledge of replication comes from studying E. coli, but similar\\nsystems operate in all organisms, prokaryote and eukaryote. The proteins this bacteriumuses to open up the double helix during replication include DnaA, DnaB, DnaC, and single-strand binding proteins.\\nDnaA Protein\\nSeveral copies of the protein DnaA bind to four sequences of nine base pairs within theE. coli origin of replication ( ori C ). This causes the two strands to begin to separate (or\\n“melt”) because the hydrogen bonds in DNA are broken near to where the DnaA protein', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='18f0b88b-c467-4c1e-ad54-feaa914426ea', embedding=None, metadata={'page_label': '101', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\nPROTEINS OPEN UP THE DNA DOUBLE HELIX DURING REPLICATION 89\\nbinds. The DNA is now in the open complex formation and has been prepared for the next\\nstage in replication, which is to open up the helix even further.\\n✎\\n✍\\x0c\\n✌Medical\\nRelevance\\n5.1Anthrax, Cipro, and the Need to Unwind\\nThe DNA helix must be able to rotate if it is to relieve the rotational stress caused\\nby the formation of the replication fork. This is achieved by an enzyme called\\ntopoisomerase I, which breaks a phosphodiester link in one strand of the helix. The DNA helixcan now rotate around the remaining unbroken strand to relieve the stress. The phosphodiesterlink is then reformed. The repeated operation of topoisomerase 1 allows progressive unwindingof the two template strands as replication proceeds.\\nTopoisomerase I should not be confused with topoisomerase II (page 74), which cuts both\\nstrands of a DNA double helix to allow a second chromosome to pass through the gap. Topoiso-merases II play an essential role during DNA replication in prokaryotes. When the chromosome\\nTopoisomerase I cleaves aphosphodiester bond inone of the DNA strands\\nThe intact strand passes\\nthrough the nicked strandand the phosphodiesterbond is reformedDNA helix now rotates around the\\nremaining unbroken strand\\nunwinds to form the replication fork, the region of DNA in front of the fork overwinds; the twistscannot dissipate because the prokaryotic chromosome is circular. Topoisomerase II relieves thisstress and allows DNA replication to proceed by performing the same action as described onpage 000, except that both the DNA double helix that is cut and then resealed, and the doublehelix that is passed through the gap, belong to the same circular chromosome.\\nConcerns that a terrorist organization might release large amounts of anthrax spores have\\ncaused several governments to stockpile large amounts of the antibiotic Cipro. This works by in-hibiting the prokaryotic form of topoisomerase II (sometimes called gyrase) and hence preventingcell replication.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='c130421e-1202-4f9a-a0ab-89d79a1ae872', embedding=None, metadata={'page_label': '102', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\n90 DNA AS A DATA STORAGE MEDIUM\\nDnaB and DnaC Proteins\\nDnaB is a helicase . It moves along a DNA strand, breaking hydrogen bonds, and in the\\nprocess unwinds the helix (Fig. 5.1). Two molecules of DnaB are needed, one for each\\nstrand of DNA. One DnaB attaches to one of the template strands and moves in the 5′→3′\\ndirection; the second DnaB attaches to the other strand and moves in the 3′→5′direction.\\nThe unwinding of the DNA double helix by DnaB is an ATP-dependent process. DnaB isescorted to the DNA strands by another protein DnaC. However, having delivered DnaB toits destination, DnaC plays no further role in replication.\\nSingle-Strand Binding Proteins\\nAs soon as DnaB unwinds the two parental strands, they are engulfed by single-strandbinding proteins. These proteins bind to adjacent groups of 32 nucleotides. DNA coveredby single-strand binding proteins is rigid, without bends or kinks. It is therefore an excellenttemplate for DNA synthesis (Fig. 5.1). Single-strand binding proteins are sometimes calledhelix-destabilizing proteins.\\nBIOCHEMISTRY OF DNA REPLICATION\\nThe synthesis of a new DNA molecule is catalyzed by DNA polymerase III. Its substratesare the four deoxyribonucleoside triphosphates, dATP, dCTP, dGTP, and dTTP. DNA poly-merase III catalyzes the formation of a phosphodiester link (Fig. 4.3 on page 69) betweenthe 3\\n′-hydroxyl group of one sugar residue and the 5′-phosphate group of a second sugar\\nresidue (Fig. 5.2 a). The base sequence of a newly synthesized DNA strand is dictated by the\\nbase sequence of its parental strand. If the sequence of the template strand is 3′CATCGA\\n5′, then that of the daughter strand is 5′GTAGCT 3′.\\nDNA polymerase III can only add a nucleotide to a free 3′-hydroxyl group and therefore\\nsynthesizes DNA in the 5′to 3′direction. The template strand is read in the 3′to 5′direction.\\nHowever, the two strands of the double helix are antiparallel. They cannot be synthesizedin the same direction because only one has a free 3\\n′-hydroxyl group, the other has a free 5′-\\nphosphate group. No DNA polymerase has been found that can synthesize DNA in the 3′to\\n5′direction, that is, by attaching a nucleotide to a 5′phosphate, so the synthesis of the two\\ndaughter strands must differ. One strand, the leading strand , is synthesized continuously\\nwhile the other, the lagging strand , is synthesized discontinuously. DNA polymerase III can\\nsynthesize both daughter strands, but must make the lagging strand as a series of short 5′to\\n3′sections. The fragments of DNA, called Okasaki fragments after Reiji Okasaki who\\ndiscovered them in 1968, are then joined together by DNA polymerase and ligase (Fig. 5.1).\\nDNA Synthesis Requires an RNA Primer\\nDNA polymerase III cannot itself initiate the synthesis of DNA. An enzyme called primase\\nis needed to catalyze the formation of a short stretch of RNA complementary in sequenceto the DNA template strand (Fig. 5.1). This RNA chain, the primer ,is needed to prime (or\\nstart) the synthesis of the new DNA strand. DNA polymerase III catalyzes the formation of aphosphodiester link between the 3\\n′-hydroxyl group of the RNA primer and the 5′-phosphate\\ngroup of the appropriate deoxyribonucleotide. Several RNA primers are made along thelength of the lagging strand template. Each is extended in the 5\\n′to 3′direction by DNA', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='337aadd5-ffa7-4923-bdf9-c0c26bcdf913', embedding=None, metadata={'page_label': '103', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\nBIOCHEMISTRY OF DNA REPLICATION 91\\nT G\\nA CA\\nTG\\nCGOH\\nA3′\\n5′\\nDNA polymerase\\nadds a Cto the growingchain5′\\n3′\\nT G\\nA CA\\nT CGG COH\\nA3′\\n5′\\nReplication continues5′\\n3′\\nT G\\nA CA\\nT CGG COH\\nA3′\\n5′\\nReplication continuesReplication continues5′\\n3′T G\\nA CA\\nTG\\nCGOH\\nA3′\\n5′\\nDNA polymerase\\nmakes a mistakeand adds an A to thegrowing chain5′\\n3′\\nT G\\nA CA\\nT CGGAOH\\nA3′\\n5′5′\\n3′\\nDNA polymerase IIIuses itsactivity to remove theincorrect deoxynucleotide.This is known as‘proof-reading’.\\nT G\\nA CA\\nT CGGOH\\nA3′\\n5′5′\\n3′\\nDNA polymerase IIIuses itsactivity to insert thecorrect deoxynucleotide3′    5′ exonuclease\\n5′    3′ polymerase(a)\\n(b)\\nFigure 5.2. DNA polymerase III can correct its own mistakes.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='86d96f4a-9f5e-45ef-ae0c-fc75e0829e60', embedding=None, metadata={'page_label': '104', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\n92 DNA AS A DATA STORAGE MEDIUM\\npolymerase III until it reaches the 5′end of the next RNA primer (Fig. 5.1). In prokaryotes\\nthe lagging strand is primed about every 1000 nucleotides whereas in eukaryotes this takes\\nplace each 200 nucleotides.\\nRNA Primers Are Removed\\nOnce the synthesis of the DNA fragment is complete, the RNA primers must be replacedby deoxyribonucleotides. In prokaryotes the enzyme DNA polymerase I removes ribonu-\\ncleotides using its 5\\n′→3′exonuclease activity and then uses its 5′→3′polymerizing\\nactivity to incorporate deoxyribonucleotides. In this way, the entire RNA primer gets re-placed by DNA. Synthesis of the lagging strand is completed by the enzyme DNA ligase ,\\nwhich joins the DNA fragments together by catalyzing formation of phosphodiester linksbetween adjacent fragments.\\nEukaryotic organisms probably use an enzyme called ribonuclease H to remove their\\nRNA primers. This enzyme breaks phosphodiester links in an RNA strand that is hydrogen-bonded to a DNA strand.\\nThe Self-Correcting DNA Polymerase\\nThe human genome consists of about 3 ×109base pairs of DNA. DNA polymerase III\\nmakes a mistake about every 1 in 104bases and joins an incorrect deoxyribonucleotide\\nto the growing chain. If unchecked, these mistakes would lead to a catastrophic mutationrate. Fortunately, DNA polymerase III has an built-in proofreading mechanism that correctsits own errors. If an incorrect base is inserted into the newly synthesized daughter strand,the enzyme recognizes the change in shape of the double-stranded molecule, which arisesthrough incorrect base pairing, and DNA synthesis stops (Fig. 5.2 b). DNA polymerase III\\nthen uses its 3\\n′to 5′exonuclease activity to remove the incorrect deoxyribonucleotide and\\nreplace it with the correct one. DNA synthesis then proceeds. DNA polymerase III hencefunctions as a self-correcting enzyme.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 5.1 The Meselson–Stahl Experiment\\nIn 1958 Matthew Meselson and Franklin Stahl designed an ingenious experiment to test whether\\neach strand of the double helix does indeed act as a template for the synthesis of a new strand. Theygrew the bacterium Escherichia coli in a medium containing the heavy isotope\\n15N that could be\\nincorporated into new DNA molecules. After several cell divisions they transferred the bacteria, nowcontaining “heavy ”DNA, to a medium containing only the lighter, normal, isotope\\n14N. Any newly\\nsynthesized DNA molecules would therefore be lighter than the original parent DNA moleculescontaining\\n15N. The difference in density between the heavy and light DNAs allows their separation\\nusing very high speed centrifugation. The results of this experiment are illustrated in the ﬁgure. DNA\\nisolated from cells grown in the15N medium had the highest density and migrated the furthest during\\ncentrifugation. The lightest DNA was found in cells grown in the14N medium for two generations,\\nwhereas DNA from bacteria grown for only one generation in the lighter14N medium had a density\\nhalf way between these two. This is exactly the pattern expected if each strand of the double helixacts as a template for the synthesis of a new strand. The two heavy parental strands separated duringreplication, with each acting as a template for a newly synthesized light strand, which remainedbound to the heavy strand in a double helix. The resulting DNA was therefore of intermediatedensity. Only in the second round of DNA replication, when the light strands created during the ﬁrst\\nround of replication were allowed to act as templates for the construction of complementary lightstrands, did DNA double helices composed entirely of\\n14N-containing building blocks appear.\\nThe two scientists were awarded the Nobel prize for this discovery that DNA replication is\\n“semiconservative, ”meaning that the results are not completely new but are half new and half old.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='68330406-c41c-4dd9-8c36-618199702c55', embedding=None, metadata={'page_label': '105', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\nBIOCHEMISTRY OF DNA REPLICATION 93\\nbacteria\\ncontinue togrowin mediumcontaining\\n14Nbacteriatransferredto mediumcontaining\\n14Nbacteria grownin mediumenriched withthe heavyisotope \\n15N\\n1st cell\\ndivision2nd cell\\ndivision\\nthe density of the DNA isolated from the bacteria was\\nanalyzed by high speed centrifugation\\nleast\\ndense\\nmost\\ndense\\nThis experiment demonstrated that the two strands of the\\nparental DNA unwind and each acts as template for a new\\ndaughter strand\\nnew\\nold newold', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a1c57e3b-0505-423e-bb45-5748173afba6', embedding=None, metadata={'page_label': '106', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\n94 DNA AS A DATA STORAGE MEDIUM\\nDNA REPAIR\\nDeoxyribonucleic Acid can be damaged by a number of agents, which include oxygen,\\nwater, naturally occurring chemicals in our diet, and radiation. Because damage to DNAcan change the sequence of bases, a cell must be able to repair alterations in the DNA codeif it is to survive and pass on the DNA database unaltered to its progeny cells.\\nSpontaneous and Chemically Induced Base Changes\\nThe most common damage suffered by a DNA molecule is depurination —the loss of an\\nadenine or guanine because the bond between the purine base and the deoxyribose sugarto which it is attached is hydrolyzed by water (Fig. 5.3). Within each human cell about5000 –10,000 depurinations occur every day.\\nDeamination is a less frequent event; it happens about 100 times a day in every\\nhuman cell. Collision of water molecules with the bond linking the amine group to cy-tosine sets off a spontaneous deamination that produces uracil (Fig. 5.3). Cytosine basepairs with guanine, whereas uracil pairs with adenine. If this change were not corrected,then a CG base pair would mutate to a UA base pair the next time the DNA strand wasreplicated.\\nUltraviolet light or chemical carcinogens such as benzopyrene, found in cigarette\\nsmoke, can also disrupt the structure of DNA. The absorption of ultraviolet light can causetwo adjacent thymine residues to link and form a thymine dimer (Fig. 5.4). If uncorrected,thymine dimers create a distortion in the DNA helix known as a bulky lesion . This inhibits\\nnormal base pairing between the two strands of the double helix and blocks the replicationprocess. Ultraviolet light has a powerful germicidal action and is widely used to sterilizeequipment. One of the reasons why bacteria are killed by this treatment is because theformation of large numbers of thymine dimers prevents replication.\\nRepair Processes\\nIf there were no way to correct altered DNA, the rate of mutation would be intolerable.DNA excision andDNA repair enzymes have evolved to detect and to repair altered\\nDNA. The role of the repair enzymes is to cut out (excise) the damaged portion of DNAand then to repair the base sequence. Much of our knowledge on DNA repair has beenderived from studies on E. coli, but the general principles apply to other organisms such as\\nourselves. Repair is possible because DNA comprises two complementary strands. If therepair mechanisms can identify which of the two strands is the damaged one, it can then berepaired as good as new by rebuilding it to be complementary to the undamaged one.\\nTwo types of excision repair are described in this section: base excision repair and\\nnucleotide excision repair. The common themes of each of these repair mechanisms are:(1) An enzyme recognizes the damaged DNA, (2) the damaged portion is removed, (3) DNApolymerase inserts the correct nucleotide(s) into position (according to the base sequenceof the second DNA strand), and (4) DNA ligase joins the newly repaired section to theremainder of the DNA strand.\\nBase excision –repair is needed to repair DNAs that have lost a purine (depurination), or\\nwhere a cytosine has been deaminated to uracil (U). Although uracil is a normal constituentof RNA, it does not form part of undamaged DNA and is recognized and removed by the', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='358197b8-5c97-409b-8d8a-d9041771d5a1', embedding=None, metadata={'page_label': '107', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\nDNA REPAIR 95\\nREPAIR\\n(Figure 5.5)depurinated sugar residueO\\nC C\\nH N C\\nN C\\nNN\\nC\\nNH\\nH\\nHO\\nC\\nC CCHH\\nHHCH2O\\nOH\\nH2O\\nO\\nC\\nC CCHH\\nHHCH2O\\nOHOHdepurination\\n5′\\n5′\\n3′3′\\nguanineguanineNH2\\nNH4+\\nH\\nCC\\nHH C\\nC N\\nOO\\nN\\nO\\nC\\nC CCHH\\nHHCH2O\\nOH\\nOH\\nO\\nC\\nC CCHH\\nHHCH2O\\nOH\\ndepyrimidated sugar residueH\\nCC\\nH C\\nC N\\nON\\nO\\nC\\nC CCHH\\nHHCH2O\\nOH\\nH3O+deamination\\n3′5′\\n3′5′\\nremoval of uracil by the\\nenzyme uracil-DNA-glycosidase\\n3′5′uracilcytosine\\nFigure 5.3. Spontaneous reactions corrupt the DNA database.\\nrepair enzyme uracil–DNA glycosidase (Fig. 5.3). This leaves a gap in the DNA where the\\nbase had been attached to deoxyribose. There is no enzyme that can simply reattach a C into\\nthe vacant space on the sugar. Instead, an enzyme called AP endonuclease recognizes thegap and removes the sugar by breaking the phosphodiester links on either side (Fig. 5.5).When DNA has been damaged by the loss of a purine (Fig. 5.3), AP endonuclease alsoremoves the sugar that has lost its base. The AP in the enzyme ’s name means apyrimidinic\\n(without a pyrimidine) or apurinic (without a purine).\\nThe repair process for reinserting a purine, or a pyrimidine, into DNA is now the\\nsame (Fig. 5.5). DNA polymerase I replaces the appropriate deoxyribonucleotide into', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='cfef40bc-5e80-430a-913d-e063a03ef258', embedding=None, metadata={'page_label': '108', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\n96 DNA AS A DATA STORAGE MEDIUM\\nNthymine dimerC\\nCO\\nC\\nC CCHH\\nHHCH2 O\\nO\\nC\\nC CCHH\\nHHCH2 O P\\nOOO\\nOH\\nHNC\\nO C\\nHO\\nC\\nNH\\nCH3C\\nO\\nHH\\nCO\\nC\\nNCH35′\\n5′3′\\n3′C\\nCO\\nC\\nC CCHH\\nHHCH2A O\\nO\\nC\\nC CCHH\\nHHCH2CH3\\nO P\\nOOO\\nOH\\nHNC\\nO C\\nHO\\nC\\nNH\\nH\\nNC\\nO C\\nHO\\nC\\nNCH3\\nultraviolet light\\n5′\\n3′\\n5′\\n3′\\nFigure 5.4. Formation of a thymine dimer in DNA.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='61a5e3f6-c7d4-4150-afcb-5d0453881934', embedding=None, metadata={'page_label': '109', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\nDNA REPAIR 97\\nStage 1 Stage 2\\nHydrolysis of the phosphodiester\\nlinks surrounding the sugarresidue from which the base hasbeen removed\\nO\\nC\\nC CCHH\\nHHCH2O\\nO\\nC\\nC CCHH\\nHHCH2O P\\nOO\\nO\\nC\\nC CCH\\nOH\\nHHCH2O P\\nOOH\\nH\\nHbasebaseDNA pol I inserts the correct\\ndeoxyribonucleotide and DNA ligasecatalyzes the formation of thephosphodiester links\\nbase5′\\n3\\n′\\n5\\n′\\n3\\n′\\n5\\n′\\n3\\n′ 5 ′ \\n3\\n′\\n5\\n′\\n3\\n′O\\nC\\nC CCHH\\nHHCH2O\\nO\\nO\\nC\\nC CCH\\nOH\\nHHCH2O P\\nOOH\\nHbasebase\\nphosphodiester links are\\ncleaved by a specific APendonucleaseO\\nC\\nC CCHH\\nHHCH2O P\\nOO\\nOHOHO\\nOO\\nFigure 5.5. DNA repair.\\nposition. DNA ligase then seals the strand by catalyzing the reformation of a phosphodiester\\nlink.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 5.2 A Curious Code\\nA bacteriophage called PBS2, which infects the bacterium Bacillus subtilis, is unusual in that it uses\\nuracil in place of thymine in its DNA. This would be expected to have the unfortunate result —for\\nthe virus —that as soon as the bacteriophage DNA entered the bacterial cytoplasm, bacterial repair\\nenzymes would begin cutting the uracils out. PBS2 gets around this problem by quickly making a\\nprotein called uracil glycosylase inhibitor, which binds irreversibly to the bacterium ’s uracil DNA\\nglycosylase and inactivates it. Once this DNA repair enzyme is inactivated, the next problem isto persuade the bacterium to make lots of uracil nucleotides rather than thymine ones. Thymineis normally made from uracil, so all PBS2 does is to produce a second protein that inhibits thisconversion, ensuring a good supply of dUTP for replication of its DNA.\\nNucleotide excision –repair is required to correct a thymine dimer. The thymine dimer,\\ntogether with some 30 surrounding nucleotides, is excised from the DNA. Repairing damage\\nof this bulky type requires several proteins because the exposed, undamaged, DNA strandmust be protected from nuclease attack while the damaged strand is repaired by the actionsof DNA polymerase I and DNA ligase.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='40201955-7dcf-4679-be2a-ff000a96e554', embedding=None, metadata={'page_label': '110', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\n98 DNA AS A DATA STORAGE MEDIUM\\n✄\\n✂\\x00\\n✁Medical\\nRelevance\\n5.2Bloom ’s Syndrome and Xeroderma Pigmentosum\\nDNA helicases are essential proteins required to open up the DNA helix during repli-\\ncation. In Bloom ’s syndrome, mutations give rise to a defective helicase. The result is\\nexcessive chromosome breakage, and affected people are predisposed to many different types ofcancers when they are young.\\nPeople who suffer from the genetic disorder known as xeroderma pigmentosum are de ﬁcient\\nin one of the enzymes for excision repair. As a result, they are very sensitive to ultraviolet light.They contract skin cancer even when they have been exposed to sunlight for very short periodsbecause thymine dimers produced by ultraviolet light are not excised from their genomes.\\nGENE STRUCTURE AND ORGANIZATION IN EUKARYOTES\\nIntrons and Exons —Additional Complexity in Eukaryotic Genes\\nGenes that code for proteins should be simple things: DNA makes RNA makes protein, and\\na gene codes for the amino acids of a protein by the three-base genetic code. In prokaryotes,indeed, a gene is a continuous series of bases that, read in threes, code for the protein.This simple and apparently sensible system does not apply in eukaryotes. Instead, theprotein-coding regions of almost all eukaryotic genes are organized as a series of separatebits interspersed with noncoding regions. The protein-coding regions of the split genes arecalled exons . The regions between are called introns , short for intervening sequences. In\\nFigure 5.6 we show the structure of the β-globin gene, which contains three exons and\\ntwo introns. Introns are often very long compared to exons. As happens in prokaryotes,messenger RNA complementary to the DNA is synthesized, but then the introns are splicedout before the mRNA leaves the nucleus (page 118). This means that a gene is much longer\\nα-globin gene cluster - chromosome 16\\nembryo fetus & adult\\nβ-globin gene cluster - chromosome 11\\nembryo fetus adult\\nexon 1 intron 1 intron 2 exon 2 exon 3ζψ ψ α2α1\\nεγG\\nPromoter\\nand\\nenhancersγA β ψδ\\nFigure 5.6. The human α- and β-globin gene family clusters. Adults only express α,β, and δ, and\\nof these the expression of δis very low. The exon/intron boundaries of the β-globin gene are\\nindicated at the bottom.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='f9f8317e-3ebe-4555-956d-ab4e996ced94', embedding=None, metadata={'page_label': '111', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\nGENE STRUCTURE AND ORGANIZATION IN EUKARYOTES 99\\nthan the mRNA that ultimately codes for the protein. The name exon derives from the fact\\nthat these are the regions of the gene that, when transcribed into mRNA, exit from thenucleus.\\nIn fact there is an evolutionary rationale to this apparently perverse arrangement. As\\nwe will see (page 200), a single protein is often composed of a series of domains, with\\neach domain performing a different role. The breaks between exons usually correspond todomain boundaries. During evolution, reordering of exons has created new genes that havesome of the exons of one gene, and some of the exons of another, and hence generates novelproteins composed of new arrangements of domains, each of which still does its job.\\nThe Major Classes of Eukaryotic DNA\\nWe do not yet fully understand the construction of our nuclear genome. Only about 1.5%of the human genome codes for exonic sequences (i.e., makes protein) with about 23.5%coding for introns, tRNA genes, and ribosomal RNA (rRNA) genes. Most protein-codinggenes occur only once in the genome and are called single-copy genes.f\\nIN DEPTH 5.1 Genome Projects\\nA major goal of biology in the latter decades of the twentieth century was to de-\\ntermine the base sequence of the human genome. This was published in 2001. Thegoal of biology in the twenty-ﬁrst century is to determine what all the genes do.This is a difﬁcult task, which is made easier by a comparative genomics approach.Humans share many genes with other organisms. Genes such as those encodingproteins important in metabolism, development, and the control of cell devisionare especially conserved through evolution.\\nThe genome of the pufferﬁsh, Fugu rubripes rupribes, has about the same\\nnumber of genes as the human genome, but is only one seventh of the size becauseit has less extragenic DNA. The pufferﬁsh genome has been sequenced and nowcan be compared with the human genome to help identify human protein-codinggenes.\\nThe genomes of other organisms are also being sequenced. The mouse is a very\\ngood model to compare to the human genome. The mouse is a mammal but has arapid generation time. Mutant mouse models are proving very useful in deﬁningthe function of human genes. Caenorhabditis elegans is a tiny nematode worm\\nwith just 959 body cells. It is also transparent. Many mutant strains of C. elegans\\nnow exist, and it is a good model for analyzing genes important in developmentand aging. The zebraﬁsh ( Danio rerio ) is a vertebrate whose embryo is transparent.\\nIt is thus easy to spot developmental mutants. A comparison of the mutant genewith that of the human genome will help to identify genes important in humandevelopment.\\nSophisticated databases have been created to store and analyze base sequence\\ninformation from the genome projects. Computer programs analyze the data forexon sequences and compare the sequence of one genome to that of another.In this way sequences encoding related proteins (proteins that share stretchesof similar amino acids) can be identiﬁed. Some important programs, which canbe easily accessed through the Internet (see CBASC website) are BLASTN forthe comparison of a nucleotide sequence to other sequences stored in a nucleotidedatabase and BLASTP, which compares an amino acid sequence to protein sequencedatabases.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='bddbddae-7f37-4b8d-837f-e0e6ee76da43', embedding=None, metadata={'page_label': '112', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\n100 DNA AS A DATA STORAGE MEDIUM\\nMany genes have been duplicated at some time during their evolution. Mutation over\\nthe succeeding generations causes the initially identical copies to diverge in sequence and\\nproduce what is known as a gene family. Members of a gene family usually have a re-lated function —the immunoglobulin gene family, for example, makes antibodies. Different\\nmembers of a family sometimes encode proteins that carry out the same specialized functionbut at different times during development. These genes then generate related proteins orisoforms, which are often distinguished by placing a Greek letter after the protein name,\\nfor example, hemoglobin αand hemoglobin β. Figure 5.6 shows the α- andβ-globin gene\\nfamilies. The β-globin gene cluster is on human chromosome 11, and the α-globin gene\\ncluster is on human chromosome 16. Hemoglobin is composed of two αglobins and two\\nβglobins (page 204). The gene clusters encode proteins produced at speci ﬁc times during\\ndevelopment: from embryo to fetus to adult. The different globin proteins are produced atdifferent stages of gestation to cope with the different oxygen transport requirements duringdevelopment. The duplication of genes and their subsequent divergence allows the expan-sion of the gene repertoire, the production of new protein molecules, and the elaboration ofever more specialized gene functions during evolution.\\nSome sections of DNA are very similar in sequence to other members of their gene\\nfamily but do not produce mRNA. These are known as pseudogenes . There are two in\\ntheα-globin gene cluster ( ψin Fig. 5.6). Pseudogenes may be former genes that have\\nmutated to such an extent that they can no longer be transcribed into RNA. Some pseu-dogenes have arisen because an mRNA molecule has inadvertently been integrated intothe genome, copied back into mRNA by an enzyme called reverse transcriptase found in\\nsome viruses (page 71). Such pseudogenes are immediately recognizable because some orall of their introns were spliced out before the integration occured. Some have the poly-Atail characteristic of intact mRNA (page 118). These are called processed pseudogenes.\\nSometimes DNA that encodes RNA is repeated as a series of copies that follow one\\nafter the other along the chromosome. Such genes are said to be tandemly repeated and\\ninclude the genes that code for ribosomal RNAs (about 250 copies/cell), transfer RNAs(50 copies/cell), and histone proteins (20 –50 copies/cell). The products of these genes are\\nrequired in large amounts.\\nThis still leaves about 75% of our genome with no very clearly understood function.\\nA large proportion of this so-called extragenic DNA is made up of repetitious DNA\\nsequences that are repeated many times in the genome. Some sequences are repeated morethan a million times and are called satellite DNA . The repeating unit is usually several\\nhundred base pairs long, and many copies are often lined up next to each other in tandemrepeats. Most of the satellite DNA is found in a region called the centromere , which plays\\na role in the physical movement of the chromosomes that occurs at cell division (page 403),and one theory is that it has a structural function.\\nOur genome also contains minisatellite DNA where the tandem repeat is about 25 bp\\nlong. Minisatellite DNA stretches can be up to 20,000 bp in length and are often found nearthe ends of chromosomes, a region called the telomere .Microsatellite DNA has an even\\nsmaller repeat unit of about 4 bp or less. Again the function of these repeated sequencesis unknown, but microsatellites, because their number varies between different individuals,have proved very useful in DNA testing (page 146).\\nOther extragenic sequences, known as LINEs (long interspersed nuclear elements) and\\nSINEs (short interspersed nuclear elements) occur in our genome. There are about 50,000copies of LINEs in a mammalian genome. SINEs probably arose by the insertion of small', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='83e66115-14ab-480e-891d-86b99d1da7ac', embedding=None, metadata={'page_label': '113', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\nSUMMARY 101\\nRNAs into the genome. The most abundant SINE in the human genome is known as the\\nAlu repeat. It occurs about one million times.\\nGENE NOMENCLATURE\\nOne of the great dif ﬁculties that has arisen out of genome-sequencing projects is how to\\nname the genes and the proteins they encode. This has not been easy and a number ofcommittees have been set up to deal with this problem. In general, each gene is designatedby an abbreviation, written in capitalized italics. For example, the human gene for ﬂavin-\\ncontaining monooxygenase is designated FMO. Because there is more than one FMO gene,\\nwe assign a number to identify the speci ﬁc gene to which we are referring. The gene\\nwhich when mutated gives rise to trimethylaminuria (page 82) is called FMO3 . The protein\\nencoded by the FMO3 gene is written in normal capitals, as FMO3. Similarly, cytochrome\\nP-450 genes (page 249) are abbreviated to CYP.CYP3A4 is a gene that belongs to the CYP3\\nfamily. This family has several members so we must include additional information in thegene name to specify precisely the member of the CYP3 gene family we are referring to.\\nThe protein name is written as CYP3A4.\\nSUMMARY\\n1.During replication each parent DNA strand acts as the template for the synthesisof a new daughter strand. The base sequence of the newly synthesized strand iscomplementary to that of the template strand.\\n2.Replication starts at speci ﬁc sequences called origins of replication. The two strands\\nuntwist and form the replication fork. Helicase enzymes unwind the double helix, andsingle-strand binding proteins keep it unwound during replication. DNA polymeraseIII synthesizes the leading strand continuously in the 5\\n′to 3′direction. The lagging\\nstrand is made discontinuously in short pieces in the 5′to 3′direction. These are\\njoined together by DNA ligase.\\n3.DNA polymerase is a self-correcting enzyme. It can remove an incorrect base usingits 3\\n′-t o5′-exonuclease activity and then replace it.\\n4.DNA repair enzymes can correct mutations. Uracil in DNA, resulting from the sponta-neous deamination of cytosine, is removed by uracil –DNA glycosidase. The depyrim-\\nidinated sugar is cleaved from the sugar –phosphate backbone by AP endonuclease,\\nand DNA polymerase then inserts the correct nucleotide. The phosphodiester link isreformed by DNA ligase.\\n5.Protein-coding genes are split into exons and introns. Only exons code for protein.The human genome has a large amount of DNA whose function is not obvious. Thisincludes much repetitious DNA, whose sequence is multiplied many times.\\n6.Protein-coding genes may be found in repeated groups of slightly diverging structurecalled gene families, either close together or scattered over the genome. Some of thefamily members have lost the ability to operate —they are pseudogenes.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6bc39130-2710-49ba-8b18-428172219696', embedding=None, metadata={'page_label': '114', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\n102 DNA AS A DATA STORAGE MEDIUM\\nFURTHER READING\\nBrenner, S., Elgar, G., Sandford, R., Macrae, A., Venkatesh, B., and Aparicio, S. 1993. Characterization\\nof the puffer ﬁsh (Fugu) genome as a compact model vertebrate genome. Nature 366: 265 –268.\\nFriedberg, E. C. 2001. How nucleotide excision repair protects against cancer. Nature Rev. Cancer 1:\\n22–33.\\nRadman, M., and Wagner, R. 1988. The high ﬁdelity of DNA duplication. Sci. Am. 259: 40 –46.2.\\nScharer, O. D., and Jiricny, J. 2001. Recent progress in the biology, chemistry and structural biology\\nof DNA glycosylases. Bioessays 23: 270 –281.\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.When a DNA strand with the sequence 3′GACTAACGATGC 5′is used as a template\\non which a daughter DNA sequence is formed, the daughter strand has the sequence\\nA. 3′CTGATTGCTACG 5′\\nB. 5′GCATCGTTAGTC 3′\\nC. 5′CTGATTGCTACG 3′\\nD. 3′CGTAGCAATCAG 5′\\nE. 3′CGUAGCAAUCAG 5′\\n2.Okazaki fragments are\\nA. short peptide sequences.\\nB. RNA primers.C. newly synthesized stretches of DNA produced during replication of the lagging template strand.D. degraded fragments of DNA.E. the lengths of DNA between origins of replication.\\n3.Following a spontaneous DNA deamination of cytosine that generates uracil, repair\\nproceeds in the following order:\\nA. Removal of the uracil from its ribose, removal of the sugar to leave a gap, addition of an\\nappropriate new deoxyribonucleotide, reformation of the phosphodiester links.\\nB. Removal of the sugar from the sugar –phosphate backbone, removal of the hydrogen-bonded\\nuracil, addition of an appropriate new deoxyribonucleotide, reformation of the phosphodiester\\nlinks.\\nC. Removal of the uracil from its ribose, insertion of cytosine which hydrogen bonds to guanosine,\\nreformation of the bond between cytosine and ribose.\\nD. Addition of an appropriate new deoxyribonucleotide to the opposite side of the DNA helix,\\nrotation of the helix by topisomerase 1, reformation of the phosphodiester links.\\nE. Transamination of uracil to regenerate cytosine, checking of the conversion by self-correcting\\nDNA polymerase.\\n4.Which of the following do not play a role in replication?\\nA. RNA primersB. DeoxynucleotidesC. HelicaseD. Stop codonsE. DNA polymerase', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='0bdc6ecb-c592-41ae-99ae-be98df708d63', embedding=None, metadata={'page_label': '115', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-05 WY001-Bolsover-v2.cls September 15, 2003 21:4\\nANSWERS TO REVIEW QUESTIONS 103\\n5.During DNA replication, the new DNA is formed on the lagging strand in a series of\\nindependent short sections because\\nA. DNA polymerase III can synthesize DNA in both the 3′to 5′and 5′to 3′directions but makes\\nmore mistakes when working in the 5′to 3′direction.\\nB. DNA polymerase III can only operate in the 5′to 3′direction.\\nC. Thymine dimers are excised from the growing strand, leaving it cut into short fragments.\\nD. The hydrogen bonding between adenine and thymine is relatively weak.E. DnaA causes the DNA strands to melt.\\n6.The human genome comprises\\nA. protein-coding genes.B. genes encoding mRNAs.C. sequences that do not encode genes.D. genes for ribosomal RNAs and transfer RNAs.E. all of the above.\\nANSWERS TO REVIEW QUESTIONS\\n1.C.5′CTGATTGCTACG 3′—DNA is always synthesized in the 5′to 3′direction, and the sequence\\nof the newly synthesized strand is complementary to that of the template strand.\\n2.C.Okazaki fragments are newly synthesized stretches of DNA produced during replication of the\\nlagging template strand.\\n3.A.Answers B, C, and E are more or less topologically and chemically plausible but are not the\\nway repair proceeds. Answer D is nonsense.\\n4.D.Stop codons are used to halt protein synthesis during translation of mRNA. None of the DNA\\ntranscription machinery takes any special notice of bases that will generate a stop codon in mRNA;they are copied in exactly the same way as the rest of the genome.\\n5.B.Because DNA polymerase can only proceed in the 5\\n′to 3′direction, the lagging strand must be\\nreplicated as a series of short sections called Okazaki fragments.\\n6.E.All are correct statements; the genome represents all the information on the DNA.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='191fb43a-fe33-4cd2-8c46-6f044c6303a5', embedding=None, metadata={'page_label': '116', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\n6\\nTRANSCRIPTION AND THE CONTROL\\nOF GENE EXPRESSION\\nTranscription (or RNA synthesis) is the process whereby the information held in the nu-\\ncleotide sequence of DNA is transferred to RNA. The three major classes of RNA areribosomal RNA (rRNA), transfer RNA (tRNA), and messenger RNA (mRNA). All playkey roles in protein synthesis. Genes encoding mRNAs are known as protein-coding genes.A gene is said to be expressed when its genetic information is transferred to mRNA and then\\nto protein. Two important questions are addressed in this chapter: how is RNA synthesized,and what factors control how much is made?\\nSTRUCTURE OF RNA\\nRibonucleic acid is a polymer made up of monomeric nucleotide units. RNA has a chemicalstructure similar to that of DNA, but there are two major differences. First, the sugar in RNAis ribose instead of deoxyribose (Fig. 6.1). Second, although RNA contains the two purinebases adenine and guanine and the pyrimidine cytosine, the fourth base is different. Thepyrimidine uracil (U) replaces thymine (Fig. 6.1). The building blocks of RNA are thereforethe four ribonucleoside triphosphates adenosine 5\\n′-triphosphate, guanosine 5′-triphosphate,\\ncytidine 5′-triphosphate, and uridine 5′-triphosphate. These four nucleotides are joined\\ntogether by phosphodiester links (Fig. 6.2). Like DNA the RNA chain has direction. In theﬁrst nucleotide in the chain, the 5\\n′carbon of the ribose is phosphorylated and is available\\nfor formation of phosphodiester links. This is called the 5′end of the RNA chain. At the\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n105', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='7518cc1d-5769-4b1d-a5c4-86893f5e3bb5', embedding=None, metadata={'page_label': '117', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\n106 TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\n3′ 2′1′ 4′5′\\nO\\nC\\nC CCH\\nOHH\\nOHHHOCH2\\nHCN\\nNC\\nCC CO\\nOH\\nHH\\nO H\\nribose uracil (U)\\nFigure 6.1. RNA contains the sugar ribose and the base uracil in place of deoxyribose and thymine.\\nother end is a ribose with a free hydroxyl group on its 3′carbon. This is called the 3′end.\\nRNA molecules are single-stranded along much of their length, although they often contain\\nregions that are double stranded due to intramolecular base pairing.\\nRNA POLYMERASE\\nIn any gene only one DNA strand acts as the template for transcription. The sequence ofnucleotides in RNA depends on their sequence in the DNA template. The bases T, A, G, andC in the DNA template will specify the bases A, U, C, and G, respectively, in RNA. DNAis transcribed into RNA by the enzyme RNA polymerase. Transcription requires that thisenzyme recognize the beginning of the gene to be transcribed and catalyze the formation ofphosphodiester links between nucleotides that have been selected according to the sequencewithin the DNA template (Fig. 6.2).\\nGENE NOTATION\\nFigure 6.3 shows the notation used in describing the positions of nucleotides within andadjacent to a gene. The nucleotide in the template strand at which transcription begins isdesignated with the number +1. Transcription proceeds in the downstream direction, and\\nnucleotides in the transcribed DNA are given successive positive numbers. Downstreamsequences are drawn, by convention, to the right of the transcription start site. Nucleotidesthat lie to the left of this site are called the upstream sequences and are identi ﬁed by negative\\nnumbers.\\nBACTERIAL RNA SYNTHESIS\\nEscherichia coli genes are all transcribed by the same RNA polymerase .This enzyme is\\nmade up of ﬁve subunits (polypeptide chains). The subunits are named α(there are two of\\nthese), β,β′, andσ. Each of the subunits has its own job to do in transcription. The role of\\nthe sigma ( σ) factor is to recognize a speci ﬁc DNA sequence called the promoter, which\\nlies just upstream of the gene to be transcribed (Fig. 6.3). E. coli promoters contain two\\nimportant regions. One centered around nucleotide −10 usually has the sequence TATATT.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='db0bd670-bd54-49ec-9d1e-4c13a5c609d5', embedding=None, metadata={'page_label': '118', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\nγ β α5′\\n3′O\\nC\\nC CCH\\nOHH\\nOHHCH2O P\\nOO P\\nOO P\\nOO\\nO\\nO\\nOH\\nO O OOO O\\nOO\\nO\\nO\\nOO O\\nP\\nOO P\\nOHbase  U\\nγβ α5′\\n3′O\\nC\\nC CCH\\nOHH\\nOHHCH2O P\\nOO P\\nOO P\\nO\\nH\\nγβ α5′\\n3′ O\\nC\\nC CCH\\nOH\\nOHHCH2O P\\nOO P\\nOO P\\nO\\nHbase  U\\nα5′\\n3′O\\nC\\nC CCH\\nOHH\\nOHHCH2O P\\nO\\nHbase  CA\\nG\\nphosphodiester link+\\npyrophosphate\\nDNA           \\ntemplate3′RNA polymerasebase  C\\n5′\\nFigure 6.2. Synthesis of an RNA strand.\\nTTGACA TATATT –1  +1DNA\\ndirection of \\nRNA synthesisupstream downstream\\n–35 box –10 box\\npromoter\\nFigure 6.3. Numbering of a DNA sequence.\\n107', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='7bb49977-55e1-4d72-9a6f-ae6b3006e2de', embedding=None, metadata={'page_label': '119', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\n108 TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\ntranscription−35 box −10 box\\nUnwinding of \\nDNA molecule\\nRewinding \\nof DNA \\nmoleculeRNA\\n3′\\n5′5′\\n3′DNA \\ntemplate\\nDNA \\ntemplate\\ntranscription bubble+1\\n5′\\n3′(a)\\n(b)RNA polymerase\\nRNA polymeraseσ factor\\nFigure 6.4. (a) RNA polymerase binds to the promoter. ( b) DNA helix unwinds and RNA polymerase\\nsynthesizes an RNA molecule.\\nThis sequence is called the −10 box (or the Pribnow box). The second, centered near\\nnucleotide −35 often has the sequence TTGACA. This is the −35 box.\\nOn binding to the promoter sequence (Fig. 6.4 a), theσfactor brings the other subunits\\n(two of αplus one each of βandβ′) of RNA polymerase into contact with the DNA to be\\ntranscribed. This forms the closed promoter complex. For transcription to begin, the two\\nstrands of DNA must separate, enabling one strand to act as the template for the synthesis\\nof an RNA molecule. This formation is called the open promoter complex. The separation\\nof the two DNA strands is helped by the AT-rich sequence of the −10 box. There are\\nonly two hydrogen bonds between the bases adenine and thymine; thus it is relatively easyto separate the two strands at this point. DNA unwinds and rewinds as RNA polymeraseadvances along the double helix, synthesizing an RNA chain as it goes. This produces atranscription bubble (Fig. 6.4 b). The RNA chain grows in the 5\\n′to 3′direction, and the\\ntemplate strand is read in the 3′to 5′direction (Fig. 6.4).\\nWhen the RNA chain is about 10 bases long, the σfactor is released from RNA\\npolymerase and plays no further role in transcription. The βsubunit of RNA polymerase', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='9db2d5d5-c857-44ed-8c0d-369609ebb77f', embedding=None, metadata={'page_label': '120', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\nCONTROL OF BACTERIAL GENE EXPRESSION 109\\nbinds ribonucleotides and joins them together by catalyzing the formation of phosphodiester\\nlinks as it moves along the DNA template. The β′subunit helps to keep the RNA polymerase\\nattached to DNA. The two αsubunits are important as they help RNA polymerase to\\nassemble on the promoter (see discussion of the lacoperon below).\\nRNA polymerase has to know when it has reached the end of a gene. Escherichia coli has\\nspeci ﬁc sequences, called terminators, at the ends of its genes that cause RNA polymerase\\nto stop transcribing DNA. A terminator sequence consists of two regions rich in the basesG and C that are separated by about 10 bp. This sequence is followed by a stretch of Abases. Figure 6.5 shows how the terminator halts transcription. When the GC-rich regionsare transcribed, a hairpin loop forms in the RNA with the ﬁrst and second GC-rich regions\\naligning and pairing up. Formation of this structure within the RNA molecule causes thetranscription bubble to shrink because where the template DNA strand can no longer bindto the RNA molecule it reconnects to its sister DNA strand. The remaining interactionsbetween the adenines in the DNA template and the uracils in the RNA chain have onlytwo hydrogen bonds per base pair and are therefore too weak to maintain the transcriptionbubble. The RNA molecule is then released, transcription terminates, and the double helixreforms. This type of transcription termination is known as rho-independent termination.\\nSome E. coli genes contain different terminator sites. These are recognized by a protein,\\nknown as rho, which frees the RNA from the DNA. In this case transcription is terminatedby a process known as rho-dependent termination.\\nCONTROL OF BACTERIAL GENE EXPRESSION\\nMany bacterial proteins are always present in the cell in a constant amount. However, theamount of other proteins is regulated by the presence or absence of a particular nutrient. Togrow and divide and make the most ef ﬁcient use of the available nutrients, bacteria have to\\nadjust quickly to changes in their environment. They do this by regulating the production ofproteins required for either breakdown or synthesis of a particular compound. Gene expres-sion in bacteria is controlled mainly at the level of transcription. This is because bacterialcells have no nuclear envelope, and RNA synthesis and protein synthesis are not separate butoccur simultaneously. This is one reason why bacteria lack the more sophisticated controlmechanisms that regulate gene expression in eukaryotes.\\nEach bacterial promoter usually controls the transcription of a cluster of genes coding\\nfor proteins that work together on a particular task. This collection of related genes is calledanoperon and is transcribed as a single mRNA molecule called a polycistronic mRNA.\\nAs shown in Figure 6.6, translation of this mRNA produces the required proteins becausethere are several start and stop codons for protein synthesis along its length. Each start andstop codon (page 79) speci ﬁes a region of RNA that will be translated into one particular\\nprotein. The organization of genes into operons ensures that all the proteins necessary tometabolize a particular compound are made at the same time and hence helps bacteria torespond quickly to environmental changes.\\nThe three major factors involved in regulating how much RNA is made are (1) nucleotide\\nsequences within or ﬂanking a gene, (2) proteins that bind to these sequences, and (3)\\nthe environment. The human intestine contains many millions of E. coli cells that must\\nrespond very quickly to the sudden appearance of a particular nutrient. For instance, mostfoods do not contain the disaccharide lactose (Fig. 6.7), but milk contains large amounts.Within minutes of our drinking a glass of milk, E. coli in our intestines start to produce the\\nenzyme β-galactosidase that cleaves lactose to glucose and galactose (Fig. 6.7). In general,', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d2289116-463d-4121-ace8-aa22db3976e7', embedding=None, metadata={'page_label': '121', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\n110 TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\nA\\nUAUAUAUAU\\n3′5′\\n3′3′\\n5′5′\\nAUAUAUAU\\n3′5′\\n3′3′\\n5′5′ C  G\\nC  GG  CG  CG  C\\nUAAUGAGA\\nC  G\\nC  GG  CG  CG  C\\nU AAUGAG UUUU AU\\nAA\\nUU\\nAA\\nU\\n5′\\n3′3′\\n5′RNA chain \\nreleasedC\\nGCGGCGC\\n(a)\\n(b)\\n(c)UAUAGCCGGAGAGUAA\\ntranscription bubble shortensRNA hairpin loop\\nDNA double\\nhelix reformstranscription bubble\\nFigure 6.5. Transcription termination in Escherichia coli .', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='c3cefbf3-250c-498c-b0d5-acef20f3012e', embedding=None, metadata={'page_label': '122', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\nCONTROL OF BACTERIAL GENE EXPRESSION 111\\npromoter gene A gene B gene C\\ntranscription\\npolycistronic mRNA\\nprotein A protein B protein Ctranslation \\nproduces three proteinsstart stop start stop start stopAUG AUG AUG\\nFigure 6.6. A bacterial operon is transcribed into a polycistronic mRNA.\\nthe substrates of β-galactosidase are compounds like lactose that contain a β-galactoside\\nlinkage and are therefore called β-galactosides.\\nlac, an Inducible Operon\\nβ-Galactosidase is encoded by one of the genes that make up the lactose (lac) operon, which\\nis shown in Figure 6.8. The operon contains three protein-coding genes called lac z ,lac\\ny, and lac a. β-Galactosidase is encoded by the lac z gene. As noted before, gene names\\nare always italicized, while the protein product is always in standard type. Lac y encodes\\nβ-galactoside permease, a carrier (page 316) that helps lactose get into the cell. The lac\\nagene codes for transacetylase. This protein is thought to remove compounds that have a\\nstructure similar to lactose but that are not useful to the cell.\\nIn the absence of β-galactoside compounds like lactose, there is no need for E. coli\\nto produce β-galactosidase or β-galactoside permease, and the cell contains only a few\\nmolecules of these proteins. The lacoperon is said to be inducible because the rate of\\ntranscription into RNA increases greatly when a β-galactoside is present. How is the tran-\\nscription of the lac z, lac y, andlac a genes switched on and off? A repressor protein (the\\nproduct of the lac i gene) binds to a sequence in the lacoperon known as the operator. The\\noperator lies next to the promoter so that, when the repressor is bound, RNA polymerase is\\nunable to bind to the promoter. In the absence of a β-galactoside, the lacoperon spends most\\nof its time in the state shown in Figure 6.8 a. The repressor is bound to the operator, RNA\\npolymerase cannot bind, and no transcription occurs. Only for the small fraction of time', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='503cc63f-9905-479f-a0f1-5f0233f00d54', embedding=None, metadata={'page_label': '123', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\n112 TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\nCCH2OH\\nHO O\\nC\\nCOHOH\\nH\\nHCH\\nOHC\\nH\\nC CH2\\nHO\\nC\\nCOHH\\nOH\\nHCH\\nOHC\\nOHH\\nCCH2OH\\nHO\\nC\\nCOHOH\\nH\\nHCH\\nOHCO\\nHCCH2OH\\nO\\nC\\nCOHH\\nHCH\\nOHC\\nOHH\\nCCH2OH\\nHO\\nC\\nCOHH\\nHCH\\nOHC\\nOH OHHβ 1→ 6 galactoside linkage\\nβ 1→ 4 galactoside linkagelactose\\n6\\n4\\n35\\n1\\n2\\n6 6\\n41 411\\n35\\n5\\n23 246\\n3256\\n4\\n35\\n1\\n265\\n3\\n421\\ngalactose glucoseCCH2OH\\nHO\\nC\\nCOHOH\\nH\\nHCH\\nOHC\\nHOHallolactose\\n+H\\nFigure 6.7. Reactions catalyzed by β-galactosidase.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='8162124b-8c89-494a-b6c1-aaff61500d09', embedding=None, metadata={'page_label': '124', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\nCONTROL OF BACTERIAL GENE EXPRESSION 113\\noperator lac z lac y lac a promoter\\nno transcriptionlac i\\nactive\\nrepressorproteinrepressor\\n(a) no β-galactoside sugars present —operon repressed\\noperator lac z lac y lac a promoter\\ntranscriptionlac i\\ninactiverepressorprotein\\ninducer\\n(e.g. allolactose)RNA \\npolymerase\\nβ-galactosidase\\npermease\\ntransacetylasetranslation\\n(b) β-galactoside sugars present —operon derepressedribosome\\nFigure 6.8. Transcription of the lacoperon requires the presence of an inducer.\\nthat the operator is unoccupied by the repressor can RNA polymerase bind and generate\\nmRNA. Thus in the absence of a β-galactoside, only very small amounts of β-galactosidase,\\nβ-galactoside permease, and transacetylase are synthesized.\\nIf lactose appears, it is converted to an isomer called allolactose. This conversion is car-\\nried out by β-galactosidase (Fig. 6.7); as we have seen, a small amount of β-galactosidase\\nis made even when β-galactoside is absent. The repressor protein has a binding site for allo-\\nlactose and undergoes a conformational change when bound to this compound (Fig. 6.8 b).\\nThis means that the repressor is no longer able to bind to the operator. The way is thenclear for RNA polymerase to bind to the promoter and to transcribe the operon. Thus in', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='fce1ea37-3cbc-4222-a69a-8a3a719126f1', embedding=None, metadata={'page_label': '125', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\n114 TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\na short time the bacteria produce the proteins necessary for utilizing the new food source.\\nThe concentration of the substrate (lactose in this case) determines whether or not mRNA issynthesized. The lacoperon is said to be under negative regulation by the repressor protein.\\nThe transcription of the lacoperon is controlled not only by the repressor protein but\\nalso by another protein, the catabolite activator protein (CAP ). If both glucose and lactose\\nare present, it is more ef ﬁcient for the cell to use glucose as the carbon source because the\\nutilization of glucose requires no new RNA and protein synthesis, all the proteins necessarybeing already present in the cell. Only in the absence of glucose, therefore, does E. coli\\ntranscribe the lacoperon at a high rate. This control operates through an intracellular\\nmessenger molecule called cyclic adenosine monophosphate (cyclic AMP) (Fig. 6.9).\\nWhen glucose concentrations are low, the concentration of cyclic AMP increases. CyclicAMP binds to CAP, and the complex then binds to a sequence upstream of the lacoperon\\npromoter (Fig. 6.10) where it has a remarkable effect. The DNA surrounding CAP bendsby about 90\\n◦. Both αsubunits of RNA polymerase are now able to make contact with CAP\\nat the same time so that the af ﬁnity of RNA polymerase for the lacpromoter is increased.\\nThe result is that now the lac z, lac y, andlac a genes can be transcribed very ef ﬁciently.\\nThelacoperon is said to be under positive regulation by the CAP –cAMP complex.\\nTo recap, the control of the lacoperon is not simple. Several requirements need to be\\nmet before it can be transcribed. The repressor must not be bound to the operator, and theCAP –cyclic AMP complex and RNA polymerase must be bound to their respective DNA\\nbinding sites. These requirements are only met when glucose is absent and a β-galactoside,\\nsuch as the sugar lactose, is present.\\nOther compounds such as isopropylthio- β-\\nD-galactoside (IPTG) (Fig. 6.11) can bind\\nto the repressor but are not metabolized. These gratuitous inducers are very useful in DNA\\nresearch and in biotechnology. Chapter 7 deals with this and with some of the industrialapplications of the lacoperon.\\nN\\nCC\\nN C\\nN C\\nHN\\nC\\nNHH\\nH\\nCH2O\\nO\\nC\\nC CC HH\\nOHH H\\nO O\\nOP5′\\n3′\\nFigure 6.9. Cyclic adenosine monophosphate, also called cyclic AMP or just cAMP.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='816ba67f-f834-45ca-9510-5274228a65e9', embedding=None, metadata={'page_label': '126', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\noperator lac z lac y lac a promoter\\ntranscriptionlac i\\ninactive\\nrepressorprotein\\ninducer\\n(e.g. allolactose)RNA \\npolymerase\\nβ-galactosidase\\npermease\\ntransacetylasetranslationCAPcAMPATPlow glucose\\nFigure 6.10. For efﬁcient transcription of the lacoperon, both cAMP and a β-galactoside sugar\\nmust be present.\\nIPTG (isopropyl thio- β-D-galactoside)CCH2OH\\nHO\\nC\\nCOHOH\\nH\\nHCH\\nOHC\\nHS C CH3CH3\\nH\\nFigure 6.11. Isopropylthio- β-D-galactoside (IPTG), which can bind to the lacrepressor protein but\\nwhich is not metabolized.\\n115', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1a67a715-1311-4c41-92ae-3c4115413eb0', embedding=None, metadata={'page_label': '127', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\n116 TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 6.1 Quorum Sensing: Squids That Glow in the Dark\\nThe bacterium Vibrio ﬁscheri lives free in seawater but is also found at high densities in the light-\\nemitting organs of the nocturnal squid Euprymna scolopes, where it synthesizes an enzyme called\\nluciferase that generates light. This phenomenon is called bioluminescence. When living free in\\nseawater V . ﬁscheri synthesizes almost no luciferase; indeed, there would be little point in doing so\\nbecause the light emitted by a single bacterium would be too dim for anything to see. V . ﬁscheri\\nonly begins making lots of luciferase when the density of bacteria is high —just as it is in the squid\\nlight organs. The word quorum is deﬁned in Webster ’s Dictionary as “the number of . . . members of\\na body that when duly assembled is legally competent to transact business. ”Thus quorum sensing\\nis a good description of V . ﬁscheri’s behavior. How does it work?\\nLuciferase is a product of the lux A andlux B genes in a bacterial operon called the luxoperon. A\\nregion in the luxoperon promoter binds a transcription factor, LuxR, which is only active when it has\\nbound a small, uncharged molecule called N-acyl-HSL (also called V AI for V . ﬁscheri autoinducer).\\nN-acyl-HSL in turn is made by an enzyme called N-acyl-HSL synthase (or V AI synthase) that is\\nencoded by the gene lux I, which is part of the luxoperon. In free-living V . ﬁscheri theluxoperon is\\ntranscribed at a low level. Small amounts of N-acyl-HSL are made, which immediately leak out of\\nthe cell into the open sea without binding to luxR. When the bacterium is concentrated in the squid ’s\\nlight organs, then some N-acyl-HSL binds to LuxR, increasing transcription of the luxoperon. This\\nmakes more luciferase —but it also makes more N-acyl-HSL synthase. The concentration of N-acyl-\\nHSL therefore rises —so transcription of the luxoperon increases further. This means that the genes\\nforN-acyl-HSL synthase, luciferase, and the enzymes that produce the substrate for luciferase are\\nnow transcribed at a high rate. This autoinduction of the luxoperon by N-acyl-HSL is a form of\\npositive feedback (page 303). When luciferase carries out its reaction, the squid will luminesce at\\nthe intensity of moonlight so that as it glides at night over the coral reefs of Hawaii it does not casta dark shadow; thus its prey are less likely to notice its presence.\\nlux R operator promoter lux I lux A lux B\\nRNA\\npolymerasetranscription\\ninactive\\ntranscription\\nfactor N-acyl-HSL\\nN-acyl HSL\\nsynthaseLuciferase\\nlight\\ntrp, a Repressible Operon\\nOperons that code for proteins that synthesize amino acids are regulated in a different way\\nfrom the lacoperon. These operons are only transcribed if the amino acid is not present,\\nand transcription is switched off if there is already enough of the amino acid around. In this', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='146b2515-aa6d-4824-8d9e-5097f2755ac8', embedding=None, metadata={'page_label': '128', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\nCONTROL OF BACTERIAL GENE EXPRESSION 117\\noperatortrp E promoter\\ntranscriptiontrp R\\naporepressor\\naporepressorrepressortrp D trp C trp B trp A\\nRNA \\npolymerase\\noperatortrp E promoter\\nno transcriptiontrp R trp D trp C trp B trp A\\ntryptophan\\ncorepressor+(a) no tryptophan; operon transcribed\\n(b) tryptophan present; operon repressed\\nFigure 6.12. Transcription of the trpoperon is controlled by the concentration of the amino acid\\ntryptophan. ( a) No tryptophan, operon transcribed. ( b) Tryptophan present, operon repressed.\\nway the cell carefully controls the concentration of free amino acids. The tryptophan ( trp)\\noperon is made up of ﬁve structural genes encoding enzymes that synthesize the amino\\nacid tryptophan (Fig. 6.12). This is a repressible operon. The cell regulates the amount\\nof tryptophan produced by preventing transcription of the trpoperon mRNA when there is\\nsufﬁcient tryptophan about. As with the lacoperon, the transcription of the trpoperon is\\ncontrolled by a regulatory protein. The gene trp R encodes an inactive repressor protein that\\nis called an aporepressor. Tryptophan binds to this to produce an active repressor complex\\nand is therefore called the corepressor. The active repressor complex binds to the operator\\nsequence of the trpoperon and prevents the attachment of RNA polymerase to the trp\\npromoter sequence. Therefore, when the concentration of tryptophan in the cell is high,the active repressor complex will form, and transcription of the trpoperon is prevented.\\nHowever, when the amount of tryptophan in the cell decreases, the active repressor complexcannot be formed. RNA polymerase binds to the promoter, transcription of the trpoperon', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6ff7464e-2484-427e-906f-3599992745be', embedding=None, metadata={'page_label': '129', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\n118 TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\nproceeds, and the enzymes needed to synthesize tryptophan are produced. This is an example\\nof negative feedback (page 303).\\nMany other operons are regulated by similar mechanisms in which speci ﬁc regulatory\\nproteins interact with speci ﬁc small molecules.\\nEUKARYOTIC RNA SYNTHESIS\\nEukaryotes have three types of RNA polymerase. RNA polymerase I transcribes the\\ngenes that code for most of the ribosomal RNAs. All messenger RNAs are synthesizedusing RNA polymerase II. Transfer RNA genes are transcribed by RNA polymerase\\nIII.This last enzyme also catalyzes the synthesis of several small RNAs including the\\n5S ribosomal RNA. The chemical reaction catalyzed by these three RNA polymerases,the formation of phosphodiester links between nucleotides, is the same in eukaryotes andbacteria.\\nMessenger RNA Processing\\nA newly synthesized eukaryotic mRNA undergoes several modi ﬁcations before it leaves the\\nnucleus (Fig. 6.13). The ﬁrst is known as capping. Very early in transcription the 5′-terminal\\ntriphosphate group is modi ﬁed by the addition of a guanosine via a 5′-5′-phosphodiester\\nlink. The guanosine is subsequently methylated to form the 7-methyl guanosine cap. The\\n3′ends of nearly all eukaryotic mRNAs are modi ﬁed by the addition of a long stretch of\\nadenosine residues, the poly-A tail (Fig. 6.13). A sequence AAUAAA is found in most\\neukaryotic mRNAs about 20 bases from where the poly-A tail is added and is probably asignal for the enzyme poly-A polymerase to bind and to begin the polyadenylation process.The length of the poly-A tail varies, it can be as long as 250 nucleotides. Unlike DNA,RNA is an unstable molecule, and the capping of eukaryotic mRNAs at their 5\\n′ends and\\nthe addition of a poly-A tail to their 3′end increases the lifetime of mRNA molecules by\\nprotecting them from digestion by nucleases.\\nMany eukaryotic protein-coding genes are split into exon and intron sequences. Both\\nthe exons and introns are transcribed into mRNA. The introns have to be removed and theexons joined together by a process known as RNA splicing before the mRNA can be usedto make protein. Removal of introns takes place within the nucleus. Splicing is complexand not yet fully understood. It has, however, certain rules. Within an mRNA the ﬁrst\\ntwo bases following an exon are always GU and the last two bases of the intron are AG.Several small nuclear RNAs (snRNAs) are involved in splicing. These are complexed witha number of proteins to form a structure known as the spliceosome. One of the snRNAs iscomplementary in sequence to either end of the intron sequence. It is thought that bindingof this snRNA to the intron, by complementary base pairing, brings the two exon sequencestogether, which causes the intron to loop out (Fig. 6.13). The proteins in the spliceosomeremove the intron and join the exons together. Splicing is the ﬁnal modi ﬁcation made to\\nthe mRNA in the nucleus. The mRNA is now transported to the cytoplasm for proteinsynthesis.\\nAs well as removing introns, splicing can sometimes remove exons in a process called\\nalternative splicing. This allows the same gene to give rise to different proteins at different\\ntimes or in different cells. For example, alternative splicing of the gene for the molecularmotor dynein produces motors that transport different types of cargo (page 390).', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='bca9959e-4e4f-4161-8e92-bc2ce4224cd0', embedding=None, metadata={'page_label': '130', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text=\"P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\nCONTROL OF EUKARYOTIC GENE EXPRESSION 119\\ntranscriptionexon 1 intron 1 exon 2\\npolyadenylation\\nAAUAAA\\npoly A \\ntail\\npolyadenylation “signal ”snRNAs\\nG AGspliceosome formation\\nspliceosomeintron loops out\\nprocessed mRNAexon 1 exon 2\\ncytoplasmprecursor \\nmRNA\\n5′  m7Gppp\\n5′  m7Gppp\\n7-methyl guanosine cap\\nnucleus5′  m7Gppp\\n5′  m7Gppp5′ 3′\\n3′\\nAAAA  3'\\nAAAA  3'\\nAAAA  3'U\\nFigure 6.13. mRNA processing in eukaryotes.\\nCONTROL OF EUKARYOTIC GENE EXPRESSION\\nSince most eukaryotes are multicellular organisms with many cell types, gene expression\\nmust be controlled so that different cell lineages develop differently and remain different.A brain cell is quite different from a liver cell because it contains different proteins eventhough the DNA in the two cell types is identical. During development and differentiation,different sets of genes are switched on and off. Hemoglobin, for example, is only expressedin developing red blood cells even though the globin genes are present in all types of cell.Genetic engineering technology (Chapter 7) has made the isolation and manipulation ofeukaryotic genes possible. This has given us some insight into the extraordinarily complex\", start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='c51b9470-a356-4195-881c-9c3c54cbb981', embedding=None, metadata={'page_label': '131', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\n120 TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\nprocesses that regulate transcription of eukaryotic genes and allow a fertilized egg to develop\\ninto a multicellular, multitissue adult.\\nUnlike the situation in bacteria, the eukaryotic cell is divided by the nuclear envelope\\ninto nucleus and cytoplasm. Transcription and translation are therefore separated in spaceand in time. This means that the expression of eukaryotic genes can be regulated at morethan one place in the cell. Although gene expression in eukaryotes is controlled primarilyby regulating transcription in the nucleus, there are many instances in which expression iscontrolled at the level of translation in the cytoplasm or by altering the way in which theprimary mRNA transcript is processed.\\nThe interaction of RNA polymerase with its promoter is far more complex in eukaryotes\\nthan it is in bacteria. This section describes how the transcription of a gene, encodingmRNA, is transcribed by RNA polymerase II. In contrast to bacterial RNA polymerase,RNA polymerase II cannot recognize a promoter sequence. Instead, other proteins knownas transcription factors bind to the promoter and guide RNA polymerase II to the beginningof the gene to be transcribed.\\nThe promoter sequence of most eukaryotic genes encoding mRNAs contains an AT-rich\\nregion about 25 bp upstream of the transcription start site. This sequence, called the TATAbox, binds a protein called the transcription factor IID (TFIID), one of whose subunits iscalled the TATA-binding protein, or TBP (Fig. 6.14 a). Several other transcription factors\\n(TFIIA, TFIIB, TFIIE, TFIIF, and TFIIH) then bind to TFIID and to the promoter region(Fig. 6.14 b). TFIIF is the protein that guides RNA polymerase II to the beginning of the gene\\nto be transcribed. The complex formed between the TATA box, TFIID, the other transcriptionfactors, and RNA polymerase is known as the transcription preinitiation complex. Note thatthe proteins with the pre ﬁx TFII are so named because they are transcription factors that\\nhelp RNA polymerase II to bind to promoter sequences.\\nAlthough many gene promoters contain a TATA box, some do not. These TATA-less\\ngenes usually encode proteins that are needed in every cell and are hence called housekeepinggenes. The promoters of these genes contain the sequence GGGGCGGGGC, called the GCbox. A protein called Sp1 binds to the GC box and is then able to recruit TATA-bindingprotein to the DNA even though there is no TATA box for the latter to bind to. TATA-bindingprotein then recruits the rest of the transcription preinitiation complex so that transcriptioncan proceeed.\\nIn either case, transcription begins when the carboxy-terminal domain of RNA poly-\\nmerase II is phosphorylated. This region is rich in the amino acids serine and threonine eachof which contains an OH group in their side chain. When these OH groups are phospho-rylated (pages 190, 252), RNA polymerase II breaks away from the preinitiation complexand proceeds to transcribe DNA into mRNA (Fig. 6.14 c).\\nAlthough the formation of the initiation protein complex is sometimes enough to pro-\\nduce a few molecules of RNA, the binding of other proteins to sequences next to thegene greatly increases the rate of transcription producing much more mRNA. These pro-teins are also called transcription factors, and the DNA sequences to which they bind arecalled enhancers, so named because their presence enhances transcription. Enhancer se-\\nquences often lie upstream of a promoter, but they have also been found downstream.\\nEnhancer sequences and the proteins that bind to them play an important role in deter-mining whether a particular gene is to be transcribed. Some transcription factors bind toa gene to ensure that it is transcribed at the right stage of development or in the righttissue. Figure 6.15 shows how one gene can be transcribed in skeletal muscle but not', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='fab385c0-860c-4e5a-a000-5c91cfd40a59', embedding=None, metadata={'page_label': '132', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\nCONTROL OF EUKARYOTIC GENE EXPRESSION 121\\n(a)TATA\\n(b)TATA\\ncarboxy terminal domain\\nD\\nAE\\nBH\\nRNA pol II\\n(c)TATA\\nD\\nAE\\nBH\\nRNA pol II\\ntranscriptionP\\nPPD ATBP\\nF\\nFTBP\\nTBP\\nFigure 6.14. RNA polymerase is guided to the promoter by TFII accessory proteins.\\nin the liver simply because of the presence or absence of proteins that bind to enhancer\\nsequences.\\nGlucocorticoids Cross the Cell Membrane to Activate Transcription\\nGlucocorticoids are steroid hormones produced by the adrenal cortex that increase the tran-scription of several genes important in carbohydrate and protein metabolism. Because theyare uncharged and relatively nonpolar, steroid hormones can pass through the plasma mem-brane by simple diffusion to enter the cytosol. Here they encounter a class of transcriptionfactors that have a binding site for steroid hormones and are therefore called steroid hor-\\nmone receptors (Fig. 6.16). In the absence of glucocorticoid hormone, its receptor remains', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='416530a8-eec0-4219-984d-c653c5109477', embedding=None, metadata={'page_label': '133', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\n122 TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\nskeletal\\nmuscleNFAT, atranscriptionfactormyosin IIa gene\\ntranscription\\nliverenhancersequencepromoter\\nno transcription\\nFigure 6.15. Tissue-speci ﬁc transcription.\\nin the cytosol and is inactive because it is complexed to two molecules of an inhibitor\\nprotein known as Hsp90. However, when the glucocorticoid hormone enters the cell andbinds to its receptor, the Hsp90 protein is displaced. The targeting sequence (page 215) thattargets the receptor to the nucleus is uncovered, and the glucocorticoid receptor –hormone\\ncomplex can now move into the nucleus. Here, two molecules of the complex bind to a\\ncytoplasmplasma membrane\\nnucleus\\nHRE\\nRNA \\npolymerase II transcription followsglucocorticoid \\nhormoneglucocorticoid hormone receptor\\ninhibitory protein (hsp 90)\\nreceptor protein is now able to bind to DNAinhibitory protein\\nmRNATATA  boxTFIIDreceptor hormone\\ncomplex - inhibitoryprotein released- targeting sequenceexposed\\nFigure 6.16. The steroid hormone receptor acts to increase gene transcription in the presence\\nof hormone.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='bf6e4251-e573-4819-89c7-867694751f70', embedding=None, metadata={'page_label': '134', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\nCONTROL OF EUKARYOTIC GENE EXPRESSION 123\\nZn++A5′ 3′\\n T \\nG\\nA\\nT\\nT\\nA\\nT\\nN\\nNN\\nNN\\nT\\nG\\nC\\nT\\nT\\nA\\nG\\nC\\nA\\nT\\nAA\\nN\\nA\\nG\\nC\\nC\\nZn++ZnZn++++ZnZn++++\\nZn++Zn++to rest of glucocorticoid\\nhormone receptor protein\\n3′ 5′to rest of glucocorticoidhormone receptor protein\\nFigure 6.17. The palindromic HRE binds to dimerized steroid hormone receptor.\\n15-bp sequence known as the hormone response element (HRE) that lies upstream of the\\nTATA box. The HRE is an enhancer sequence. The glucocorticoid receptor –hormone com-\\nplex interacts with the preinitiation complex bound to the TATA box, and the rate at whichRNA polymerase transcribes genes containing the HRE is increased.\\nFigure 6.17 shows why the glucorticoid hormone receptor binds to DNA as a dimer. The\\nHRE is a palindrome —the sequence on both the top and bottom strands is the same when\\nread in the 5\\n′to 3′direction. Each strand of the HRE has the 6-bp sequence 5′AGAACA 3′\\nthat is known as the core recognition motif. This is the sequence to which a single receptormolecule binds. Because the HRE contains two recognition motifs, it binds two moleculesof glucocorticoid receptor. The two 6-bp sequences are separated by 3 base pairs, whichpresumably are there to provide suf ﬁcient space for the receptor homodimer to ﬁt snugly\\non the double helix. The glucocorticoid receptor is unaffected by the identity of these threebase pairs, so the nucleotides are simply indicated as N in Fig. 6.17.\\nChemicals that are released from one cell and that alter the behavior of other cells\\nare called transmitters. Glucocorticoids are an example of transmitters that alter gene\\ntranscription. The cells of multicellular organisms turn the transcription of genes on andoff in response to many extracellular chemicals. Unlike steroid hormones, most of thesetransmitters cannot enter the cell and must activate intracellular messenger systems inside\\nthe cell that in turn carry the signal onward from the plasma membrane to the nucleus(Chapter 16).', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ed6c10d8-8145-4d23-89ba-ff352287428e', embedding=None, metadata={'page_label': '135', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\n124 TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 6.2 Glucocorticoid Hormones Can Repress Transcription:\\nRheumatoid Arthritis\\nGlucocorticoid treatment has been shown to give some relief to individuals who suffer from the\\ndebilitating disorder rheumatoid arthritis. Collagenase, an enzyme that digests collagen, is generatedin the joints of these patients, causing destruction of the extracellular matrix and therefore chronicinﬂammation. Transcription of the collagenase gene is controlled by an enhancer sequence called the\\nAP1 site. For transcription to occur, the enhancer must be occupied by a transcription factor calledAP1, which, like the active glucocorticoid receptor, is a dimer. Glucocorticoid hormones inhibittranscription of the collagenase gene by an ingenious mechanism. The glucocorticoid hormone, onentering the cell interacts with the glucocorticoid receptor as shown in Figure 6.16. The receptor –\\nhormone complex moves to the nucleus and binds to the proteins that would otherwise dimerize toform AP1. The heterodimer is unable to activate transcription of the collagenase gene. By depletingthe pool of AP1 subunits, the glucocorticoid receptor –hormone complex prevents transcription of\\nthe collagenase gene.\\n☛\\n✡✟\\n✠Medical\\nRelevance\\n6.1St. John’s Wort and the Pill\\nWhen we ingest plant products and other chemicals foreign to us, the body re-sponds by increasing the amounts of a group of proteins known as the cytochromes\\nP450 (CYPs). This remarkable response, found largely in the liver, is our built-in detoxicationsystem. The foreign chemical signals the appropriate CYP gene to activate, transcription takes\\nplace, and more CYP protein is produced. The CYP protein then metabolizes the foreign chemicalso that it will be cleared from the body quickly and ef ﬁciently through the urine or feces. To the\\nliver, a medicinal drug is just another foreign chemical, and the cytochrome P450 system is usedto destroy it.\\nSt. John ’s wort is a herbal remedy, taken by many as a natural antidepressant. One of the active\\ncomponents is the chemical hyperforin, which has been shown to decrease the effectiveness ofseveral therapeutic drugs including the oral contraceptive pill, the HIV protease inhibitor indinavir,and immunosuppressants such as cyclosporin. This is because each of these drugs is metabolizedby a cytochrome P450 called CYP3A4, and the gene for CYP3A4 is activated by hyperforin.Because the activation of the CYP3A4 gene results in more CYP3A4 protein, the prescribed drug,\\nin a person also taking St. John ’s wort, is metabolized more rapidly, cleared from the body quickly,\\nand is therefore less effective.\\nThe transcription of the CYP3A4 gene is increased because hyperforin passes to the nucleus\\nand binds to a receptor called the pregnane X receptor (PXR). The receptor –hyperforin complex is\\nthen able to dimerize with another receptor called RXR. The dimer binds to an enhancer sequenceof the CYP3A4 gene, interacts with the transcription preinitiation complex, and triggers RNA\\npolymerase II to begin transcription of CYP3A4 mRNA.\\nThis activation of the CYP3A4 gene by St. John ’s wort means that this herbal remedy should\\nbe taken with caution if a patient has been prescribed a therapeutic drug. CYP3A4 is responsiblefor the metabolism of about 80% of currently prescribed drugs.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='5566bed1-828d-4a24-a2ce-3333b0f0e6b3', embedding=None, metadata={'page_label': '136', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\nSUMMARY 125\\nSUMMARY\\n1.DNA is transcribed into RNA by the enzyme RNA polymerase. The three types\\nof RNA are ribosomal RNA (rRNA), transfer RNA (tRNA), and messenger RNA(mRNA). Uracil, adenine, cytosine, and guanine are the four bases in RNA.\\n2.In bacteria, RNA polymerase binds to the promoter sequence just upstream of the startsite of transcription. The enzyme moves down the DNA template and synthesizes anRNA molecule. RNA synthesis stops once the enzyme has transcribed a terminatorsequence.\\n3.Bacterial genes encoding proteins for the same metabolic pathway are often clusteredinto operons. Some operons are induced in the presence of the substrate of theirpathway, for example, the lacoperon. Others are repressed in the presence of the\\nproduct of the pathway, for example, the trpoperon.\\n4.Eukaryotic mRNAs are modi ﬁed by the addition of a 7-methyl-guanosine cap at their\\n5\\n′end. A poly-A tail is added to their 3′end. Intron sequences are removed, and the\\nexon sequences joined together by the process known as splicing. The fully processedmRNA is then ready for transport to the cytoplasm and protein synthesis.\\n5.In eukaryotes, there are three RNA polymerases —RNA polymerases I, II, and III.\\nRNA polymerase II needs the help of the TATA-binding protein and other transcrip-tion factors to become bound to a promoter. This group of proteins is called thetranscription preinitiation complex, and this is suf ﬁcient to make a small number of\\nRNA molecules. However, to make a lot of RNA in response to a signal, such as ahormone, other proteins bind to sequences called enhancers. These proteins interactwith the initiation complex and increase the rate of RNA synthesis.\\nFURTHER READING\\nLatchman, D. 2002. Gene Regulation, A Eukaryotic Perspective. 4th ed. Cheltenham: Nelson\\nThornes Ltd.\\nLloyd, G., Landini, P., and Busby, S. 2001. Activation and repression of transcription initiation in\\nbacteria. Essays Biochem. 37: 17 –31.\\nRoberts, G. C., and Smith, C. W. 2000. Alternative splicing: Combinatorial output from the genome.\\nCurr. Opin. Chem. Biol. 6: 375 –383.\\nTjian, R. 1995. Molecular machines that control genes. Sci. Am. 272: 38 –45.\\nQuorum sensing: Hastings, J. W., and Greenberg, E. P. 1999. Quorum sensing: The explanation of\\na curious phenomenon reveals a common characteristic of bacteria. J. Bacteriol. 188: 2667 –2668.\\nSt. John’s wort: Moore, L. B., Goodwin, B., Jones, S. A., Wisely, G. B., Serabjit-Singh, C. J., Willson,\\nT. M., Collins, and Kliewer, J. L., 2000. S. A. St. John ’s wort induces hepatic drug metabolism\\nthrough activation of the pregnane X receptor. Proc Natl Acad Sc iUSA 13: 7500 –7502.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2ce5bca1-a17b-484d-98fd-f1de2d2f4e64', embedding=None, metadata={'page_label': '137', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\n126 TRANSCRIPTION AND THE CONTROL OF GENE EXPRESSION\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.A DNA strand of sequence 3′GTCAAGGATATTCGAT 5′would be transcribed in the\\nnucleus to form a polymer of sequence\\nA. 3′GTCAAGGATATTCGAT 5′.\\nB. 5′GUCAAGGAUAUUCGAU 3′.\\nC. 5′CAGUUCCUAUAAGCUA 3′\\nD. 3′AUCGAAUAUCCUUGAC 5′\\nE. NH+\\n3-Gln-Phe-Leu-COOH\\n2.RNA polymerase from E. coli is a multisubunit protein composed of α,β,β′, andσ.\\nThe promoter sequence is recognized and bound by\\nA.α.\\nB.β.\\nC.β′.\\nD. aβ:β′dimer.\\nE.σ.\\n3.A rho-independent transcription terminator sequence in prokaryotic DNA has\\nA. a GC-rich region followed by a string of uracils.\\nB. a GC-rich region followed by a string of adenines.C. several stop codons.D. the sequences TTGACA and TATATT separated by about 17 bases.E. the sequence AAUAAA and a poly-A tail up to 250 nucleotides long, separated by about 20\\nbases.\\n4.Thelacoperon is transcribed when\\nA. lactose is present and glucose is absent.B. cAMP concentrations in the cell are high.C. The cAMP –CAP protein is bound to the lacpromoter region.\\nD. the lacrepressor is bound to allolactose or a similar shaped molecule.\\nE. all of the above.\\n5.Thetrpoperon is transcribed when\\nA. tryptophan concentrations in the cell are high.B. the trprepressor is bound to tryptophan or a similar shaped molecule.\\nC. tryptophan is bound to its aporepressor.D. the appropriate corepressor is absent.E. all of the above.\\n6.Before a newly synthesized eukaryotic mRNA attaches to the ribosome, it undergoes\\nA. capping.B. polyadenylation.C. splicing.D. transport to the cytosol.E. all of the above.\\n7.The preinitiation complex of proteins forms at a eukaryotic protein-coding gene promoter\\nin order to\\nA. align RNA polymerase I.\\nB. align RNA polymerase II.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a23bf926-74ac-4913-89c9-41c04bad4d72', embedding=None, metadata={'page_label': '138', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-06 WY001-Bolsover-v2.cls September 4, 2003 13:14\\nANSWERS TO REVIEW QUESTIONS 127\\nC. align RNA polymerase III.\\nD. align the helicase DnaB.E. inhibit transcription.\\nANSWERS TO REVIEW QUESTIONS\\n1.C.5′CAGUUCCUAUAAGCUA 3′. Transcription generates RNA (not protein, which is made\\nat the next step, translation, outside the nucleus in eukaryotes). RNA is synthesized in the 5′to\\n3′direction, and the sequence of the newly synthesized strand is complementary to that of the\\ntemplate DNA strand.\\n2.E.It is the σfactor that binds the promoter sequence and then brings the other subunits onto the\\nDNA.\\n3.B.The terminator sequence in DNA comprises a GC-rich region followed by a string of adenines.\\nConcerning other answers: (C) stop codons exert their effect during translation, not duringtranscription. (D) The sequences TTGACA and TATATT, the −35 and −10 boxes, are found\\nin the promoter region of prokaryotic genes, not at the end. (E) Processed eukaryotic RNA ter-minates with the sequence AAUAAA and a poly-A tail up to 250 nucleotides long, separated byabout 20 bases. The presence of uracil should have been enough to tell you that this was not DNA!\\n4.E.Thelacoperon is transcribed when lactose is present, generating allolactose, which binds to the\\nrepressor, preventing its binding to the promoter, AND if glucose is absent, causing an increase inthe cAMP concentration and therefore allowing the cAMP –CAP protein to bind to the lacpromoter\\nregion.\\n5.D.The corepressor is tryptophan itself, and the trpoperon is only transcribed when tryptophan is\\nabsent and therefore needs to be made by the cell. When tryptophan is present at a suf ﬁciently high\\nconcentration (viz. A), then it binds to its aporepressor (B, C), which then binds to the promoter,preventing transcription.\\n6.E.All these processes occur before eukaryotic mRNA is translated at the ribosome.\\n7.B.RNA polymerase II transcribes protein-coding genes in eukaryotes. DnaB is part of the machin-\\nery of DNA replication that assembles, at the appropriate time, at the origin of replication —not at\\na promoter.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='f6200dd3-e80c-44e5-87f9-c341801ab569', embedding=None, metadata={'page_label': '139', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n7\\nRECOMBINANT DNA AND\\nGENETIC ENGINEERING\\nDeoxyribonucleic acid is the cell’s database. Within the base sequence is all the information\\nnecessary to encode RNA and protein. A number of biological and chemical methods nowgive us the ability to isolate DNA molecules and to determine their base sequence. Oncewe have the DNA and know the sequence, many possibilities open up. We can identifymutations that cause disease, make a human vaccine in a bacterial cell, or alter a sequenceand hence the protein it encodes. The knowledge of the entire base sequence of the humangenome, and of the genomes of many other organisms, such as bacteria that cause disease, isset to revolutionize medicine and biology. In future years the power of genetic engineeringis likely to impact ever more strongly on industry and on the way we live. This chapterdescribes some of the important methods involved in recombinant DNA technology at theheart of which is DNA cloning.\\nDNA CLONING\\nSince DNA molecules are composed of only four nucleotides, their physical and chemicalproperties are very similar. Hence it is extremely difﬁcult to purify individual species of DNAby classical biochemical techniques similar to those used successfully for the puriﬁcationof proteins. However, we can use DNA cloning to help us to separate DNA molecules fromeach other. A clone is a population of cells that arose from one original cell (the progenitor )\\nand, in the absence of mutation, all members of a clone will be genetically identical. If aforeign gene or gene fragment is introduced into a cell and the cell then grows and divides\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n129', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ab6bd2d8-9257-4aee-9c37-a60d46b9a775', embedding=None, metadata={'page_label': '140', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n130 RECOMBINANT DNA AND GENETIC ENGINEERING\\nrepeatedly, many copies of the foreign gene can be produced, and the gene is then said to\\nhave been cloned. A DNA fragment can be cloned from any organism. The basic approachto cloning a gene is to take the genetic material from the cell of interest, which in theexamples we will describe is a human cell, and to introduce this DNA into bacterial cells.Clones of bacteria are then generated, each of which contains and replicates one fragmentof the human genetic material. The clones that contain the gene we are interested in are thenidenti ﬁed and grown separately. We therefore use a biological approach to isolate DNA\\nmolecules rather than physical or chemical techniques.\\nCREATING THE CLONE\\nHow do we clone a human DNA sequence? The human genome has 3 ×109base pairs of\\nDNA, and the DNA content of each cell (other than eggs and sperm) is identical. However,each cell expresses only a fraction of its genes. Different types of cells express differentsets of genes and thus their mRNA content is not the same. In addition, processed mRNAis shorter than its parent DNA sequence and contains no introns (page 98). Consequently,it is much easier to isolate a DNA sequence by starting with its mRNA. We therefore startthe cloning process by isolating mRNA from the cells of interest. The mRNA is then copiedinto DNA by an enzyme called reverse transcriptase that is found in some viruses. As the\\nnewly synthesized DNA is complementary in sequence to the mRNA template, it is knownascomplementary DNA, orcDNA. The sample of cDNAs, produced from the mRNA,\\nwill represent the product of many different genes.\\nThe way in which a cDNA molecule is synthesized from mRNA is shown in Figure 7.1.\\nMost eukaryotic mRNA molecules have a string of A ’s at their 3\\n′end, the poly-A tail\\n(page 118). A short run of T residues can therefore be used to prime the synthesis ofDNA from an mRNA template using reverse transcriptase. The resulting double-strandedmolecule is a hybrid containing one strand of DNA and one of RNA. The RNA strand isremoved by digestion with the enzyme ribonuclease H. This enzyme cleaves phosphodiesterlinks in the RNA strand of the paired RNA-DNA complex, making a series of nicks downthe length of the RNA. DNA polymerase (page 90) is then added. This homes in on thenicks and then moves along replacing ribonucleotides with deoxyribonucleotides. Lastly,DNA ligase (page 92) is used to reform any missing phosphodiester links. In this way adouble-stranded DNA molecule is generated by the replacement of the RNA strand with aDNA strand. If the starting point had been mRNA isolated from liver cells, then a collectionof cDNA molecules representative of all the mRNA molecules within the liver will havebeen produced. These DNA molecules have now to be introduced into bacteria.\\nIntroduction of Foreign DNA Molecules into Bacteria\\nCloning Vectors. To ensure the survival and propagation of foreign DNAs, they\\nmust be inserted into a vector that can replicate inside bacterial cells and be passed on tosubsequent generations of the bacteria. The vectors used for cloning are derived from natu-rally occurring bacterial plasmids or bacteriophages. Plasmids (page 74) are small circularDNA molecules found within bacteria. Each contains an origin of replication (page 88) andthus can replicate independently of the bacterial chromosome and produce many copies ofitself. Plasmids often carry genes that confer antibiotic resistance on the host bacterium. Theadvantage of this to the scientist is that bacteria containing the plasmid can be selected forin a population of other bacteria simply by applying the antibiotic. Those bacteria with theantibiotic resistance gene will survive, whereas those without it will die. Figure 7.2 shows', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a5ab54c2-cf00-48c7-ad91-dfc86ac9b5dd', embedding=None, metadata={'page_label': '141', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\nCREATING THE CLONE 131\\nmRNA  5 ′ 3′\\nmRNA  5 ′ 3′ AAAAAAAA\\nTTTT DNA  3 ′ 5′\\nTTTT DNA  3 ′ 5′\\nTTTT DNA  3 ′ 5′Reverse transcriptase using\\ndATP, dCTP, dGTP,dTTP, (T)\\nn primer\\nRibonuclease H cleavesphosphodiester links inthe mRNA molecule\\nchopped mRNA  5 ′ 3′ AAAA\\nDNA with breaks  5 ′ 3′ AAAA\\nDNA  5 ′ 3′ AAAA\\nDNA  3 ′ 5′ TTTTDNA polymerasereplaces RNA strandwith a DNA strand\\nDNA ligase catalyzes\\nformation ofphosphodiester links\\nDouble-stranded DNA molecule\\nFigure 7.1. Synthesis of a double-stranded DNA molecule.\\nrestrictionendonuclease site\\norigin of replication antibiotic\\nresistance\\ngene\\nFigure 7.2. A plasmid cloning vector.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='96589958-3607-4433-995a-c23a9b126787', embedding=None, metadata={'page_label': '142', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n132 RECOMBINANT DNA AND GENETIC ENGINEERING\\nthe basic components of a typical plasmid cloning vector: an antibiotic resistance gene, a\\nrestriction endonuclease site (see next section) at which foreign DNA can be inserted, andan origin of replication so the plasmid can copy itself many times inside the bacterial cell.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 7.1 The Cloning Vector pBluescript\\nThe plasmid pBluescript is based on a naturally occurring plasmid that has been engineered to include\\nseveral valuable features. pBluescript has an origin of replication, and the ampicillin resistance genefor selecting bacterial cells that have taken up the plasmid.The multiple cloning site (MCS) allowsthe scientist to cut the plasmid with the most appropriate restriction endonuclease for the task at hand.\\nThe MCS lies within the lac Z gene, which codes for the enzyme β-galactosidase (page 112).\\nβ-Galactosidase converts a substrate known as X-gal to a bright blue product. Cells containing\\npBluescript without a foreign DNA in the MCS will produce blue colonies when grown on agarplates. However, when the lac Z gene is disrupted by insertion of a foreign DNA in the MCS, the cells\\ncontaining the recombinant plasmids will grow to produce a colony with the normal color of white.This is because the function of the lac Z gene is destroyed and β-galactosidase is not produced.\\nThis is the basis of a test, called the blue/white assay, to identify colonies containing recombinantplasmids.\\nAnother important feature of pBluescript is the presence of the bacteriophage T7 and T3\\npromoter sequences, which ﬂank the MCS. These promoter sequences are used to transcribe mRNA\\nfrom a cDNA cloned into one of the sites within the MCS. By selecting the promoter sequence,and adding the appropriate RNA polymerase (T3 or T7 RNA polymerase), either the sense or theantisense mRNA can be synthesized. Antisense RNAs are used in a number of techniques, forexample, in situ hybridization (page 147) to detect cells producing a speci ﬁc mRNA.\\npBluescript amp Γlac Z\\nMCS\\norigin ofreplication\\nTranscription fromthe T3 promoterwill transcribethe bottomstrand into mRNA\\nT3\\nT7transcription\\nfrom T7 promoter\\ntranscription\\nfrom T3 promoter5′\\n3′3′3′\\n5′5′\\nMCS\\ntranscription fromthe T7 promoterwill transcribe thetop strand into mRNA\\nrestriction enzyme\\nrecognition sites5′ 3′↑\\n↑\\nT3T7', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2a956dec-9bf6-4213-b85c-c93efd745147', embedding=None, metadata={'page_label': '143', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\nCREATING THE CLONE 133\\nBacterial species/strain Enzyme nameRecognition sites of some common restriction endonucleases\\nRecognition sequences\\nand cleavage sites\\nBam H1 Bacillus amyloliquefaciens  HG\\nCG\\nCA\\nTT\\nAC\\nGC\\nG\\nG\\nCA\\nTA\\nTT\\nAT\\nAC\\nG\\nC\\nGT\\nAG\\nCC\\nGA\\nTG\\nC\\nG\\nCT\\nAA\\nTC\\nGC\\nGC\\nGC\\nGG\\nCG\\nCG\\nCEscherichia coli Ry13\\nProvidencia stuartii  164\\nSerratia marcescens  SB\\nRhodopseudomonas sphaeroidesEco R1\\nPst 1\\nSma H1\\nRsa 1\\nFigure 7.3. Recognition sites of some common restriction endonucleases.\\nBacteriophages (page 74) are viruses that infect bacteria and utilize the host cell ’s\\ncomponents for their own replication. The bacteriophage genome is, like a plasmid, circular,\\nalthough many viruses use RNA rather than DNA as their genetic material. If human DNAis inserted into a bacteriophage, the bacteriophage will do the job of introducing it into abacterium.\\nJoining Foreign DNAs to a Cloning Vector. Enzymes known as restriction en-\\ndonucleases are used to insert foreign DNA into a cloning vector. Each restriction endonu-clease recognizes a particular DNA sequence of (usually) 4 or 6 bp. The enzyme binds tothis sequence and then cuts both strands of the double helix. Many restriction endonucle-ases have been isolated from bacteria. The names and recognition sequences of a few of thecommon ones are shown in Figure 7.3. Restriction endonuclease names are conventionallywritten in italics because they are derived from the Latin name for the bacterium in whichthe protein occurs.\\nSome enzymes such as Bam HI,EcoRI, and PstI make staggered cuts on each strand.\\nThe resultant DNA molecules are said to have sticky ends (Fig. 7.4) because such fragments\\ncan associate by complementary base pairing to any other fragment of DNA generated bythe same enzyme. Other enzymes such as Sma HI cleave the DNA smoothly to produce', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ee2634f3-20c0-4f26-b964-c084a97eff36', embedding=None, metadata={'page_label': '144', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n134 RECOMBINANT DNA AND GENETIC ENGINEERING\\na sticky end\\na blunt end\\nFigure 7.4. Restriction endonulceases generate two types of cut ends in double-stranded DNA.\\nblunt ends (Fig. 7.4). DNA fragments produced in this way can be joined to any other\\nblunt-ended fragment.\\nFigure 7.5 illustrates how human DNA is inserted into a plasmid that contains a Bam\\nH1 restriction endonuclease site. A short length of synthetic DNA (an oligonucleotide) that\\nincludes a Bam H1 recognition site is added to each end of the human DNA fragment. Both\\nthe human DNA and the cloning vector are cut with Bam H1. The cut ends are now com-\\nplementary and will anneal together. DNA ligase then catalyzes the formation of phospho-diester links between the vector and the human DNA. The resultant molecule is known as arecombinant plasmid. If our starting material was mRNA from a sample of liver, we would\\nnow have a collection of plasmids each carrying a cDNA from one of the genes that wasbeing transcribed in this organ.\\nIntroduction of Recombinant Plasmids into Bacteria. Figure 7.6 summarizes\\nhow recombinant plasmids are introduced into bacteria such as Escherichia coli. Bacteria\\nareﬁrst treated with concentrated calcium chloride to make the cell wall more permeable\\nto DNA. DNA can now enter these cells, which are said to have been made competent.\\nCells that take up DNA in this way are said to be transformed. The transformation process\\nis very inef ﬁcient, and only a small percentage of cells actually take up the recombinant\\nmolecules. This means that it is extremely unlikely that any one bacterium has taken uptwo plasmids. The presence of an antibiotic resistance gene in the cloning vector makesit possible to select those bacteria that have taken up a molecule of foreign DNA, sinceonly the transformed cells can survive in the presence of the antibiotic. The collection ofbacterial colonies produced after this selection process is a clone library. All the cells of asingle colony harbor identical recombinant molecules that began as one mRNA molecule inthe original cell sample. Other colonies in the same clone library contain plasmids carryingdifferent DNA inserts. Isolating individual bacterial colonies will produce different clonesof foreign DNA. In the example we have described, where the starting DNA material usedto produce these clones was a population of cDNA molecules, the collection of clones iscalled a cDNA library.\\nSelection of cDNA Clones\\nHaving constructed a cDNA library —which may contain many thousands of different\\nclones —the next step is to identify the clones that contain the cDNA of interest. There', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='75c1d1cc-2902-4070-8cc9-09ebe402c91f', embedding=None, metadata={'page_label': '145', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\nCREATING THE CLONE 135\\nGGATCCCCTAGG\\nrecombinant\\nplasmidDNA ligase\\nrepairs thephosphodiesterbondsCCTAGG\\nGGATCCvectorDNACCTAGGGGATCC\\nBam  H1oligonucleotide\\nwith restriction\\nendonuclease\\nsiteoligonucleotidewith restrictionendonuclease site\\nGGAT CC\\nCCTAGGGGAT CC\\nCCTAGG\\nGATCC\\nGG\\nCCTAGGATCC\\nCCTAG\\nGG\\nDNA strands anneal\\nvia complementary\\nbase pairingBam  H1human DNAfragment\\nFigure 7.5. Generation of a recombinant plasmid.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='434710a4-3052-432a-8baa-8c4971c54fab', embedding=None, metadata={'page_label': '146', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n136 RECOMBINANT DNA AND GENETIC ENGINEERING\\nsome bacterial\\ncells take up therecombinantplasmids\\nE. coli  containing\\nthe plasmid isable to grow anddivideE. coli  with no\\nplasmid will dieand not produceany coloniesselection of\\ncells\\ncontaining\\nplasmidchromosome\\nmedium containing the\\nantibiotic ampicillincompetent \\nE. coli  cells recombinant plasmids +\\nFigure 7.6. Introduction of recombinant plasmids into bacteria.\\nare many ingenious ways of doing this. We will describe two ways of selecting cDNA\\nclones from a library. One method simply detects the presence of the foreign DNA attachedto the plasmid vector, and the second detects the protein encoded for by the foreign DNA.We call the process of selecting speci ﬁc clones “screening the library ”.\\nPreparation of the cDNA Library for Screening. Bacterial colonies are plated\\nonto agar plates, and the colonies are then replica-plated onto a nylon membrane, which isthen treated with detergent to burst (or lyse) the bound cells. If the clone is to be selectedby virtue of its DNA sequence (Fig. 7.7), the nylon membrane is processed with sodiumhydroxide. This is necessary to break all hydrogen bonds between the DNA strands boundto the nylon membrane and ensures that the DNA is single-stranded. The processed nylonmembrane is an exact replica of the DNA contained within each bacterial colony on the agarplate. If the clone is to be selected from the library by detecting the protein encoded by theforeign DNA (Fig. 7.8), then colonies are again replica-plated on to a nylon membrane. Thistime, however, the nylon membrane is processed to produce an exact copy of the proteinssynthesized by each bacterial colony.\\nOligonucleotide Probes for cDNA Clones. If some of the amino acid sequence is\\nknown for the protein whose cDNA is to be cloned, an oligonucleotide can be synthesizedin vitro (i.e., in a machine rather than in a cell) that has a sequence complementary to', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1ffa142c-9839-4516-8a4a-19c9453cb719', embedding=None, metadata={'page_label': '147', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\nCREATING THE CLONE 137\\nmembrane lift incubated with\\nradio-labelled oligonucleotides\\nclone hybridized\\nto oligonucleotideX-rayfilmAUG CAA\\nG AAAG UUU\\nC AAU\\nCall possible mRNA sequences\\nsynthesize 16\\nprimers\\nlabel each oligonucleotide\\nwith polynucleotide kinase+ γ\\n32 PATPDNA oligonucleotides ATG CAA\\nG AAA\\nG TTT\\nC AAT\\nCmet gln lys phe asn known protein sequence\\nFigure 7.7. Selection of a cDNA clone with a radioactive oligonucleotide probe.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d7e0026d-0f4a-4573-9067-995811cb636f', embedding=None, metadata={'page_label': '148', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n138 RECOMBINANT DNA AND GENETIC ENGINEERING\\nHuman cDNA\\nsequence\\nremove\\nmembranemembraneplaced onsurface ofagar platehumanproteinsexpressed inbacterial cells\\n1. incubate filter with primary antibody \\n2. incubate filter with secondary antibody\\nlinked to alkaline phosphataseproteins bind\\nto membrane\\nadd substrate for alkaline\\nphosphatase—a coloredproduct is produced\\nspecific bacterialcolony identifiedtransform\\nbacterial cells\\nFigure 7.8. Selection of a cDNA clone with an antibody probe.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='f17f005d-a01b-4cc8-8ab5-f56e38d2b895', embedding=None, metadata={'page_label': '149', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\nCREATING THE CLONE 139\\none of the strands of the cDNA. The ﬁrst step is to use the genetic code to predict all\\npossible DNA sequences that could code for a short stretch of amino acids within the\\nprotein. This strategy is shown in Figure 7.7. The sequence —met gln lys phe asn —can be\\ncoded for by 16 possible sequences, because of the redundancy of the genetic code. All 16oligonucleotide sequences are synthesized. One of the 16 sequences will be complementaryto the cDNA we want to select from the library. The oligonucleotides are tagged with aradioactive phosphate group (\\n32P) at their 5′ends using the enzyme polynucleotide kinase\\n(PNK) and the substrate [ γ-32P]ATP, that is, ATP whose γphosphate is the radioactive\\nisotope32P. PNK removes the 5′-phosphate group from each oligonucleotide, leaving a 5′-\\nhydroxyl group. The enzyme then transfers the γ(32P) phosphate group of [ γ-32P]ATP to the\\n5′-hydroxyl.\\nThe nylon membrane to which the library DNA is attached is incubated together with\\nthe mixture of radiolabeled oligonucleotides. This process is called hybridization, a word\\nused whenever two nucleic acid strands associate together by hydrogen bonding. In thiscase the oligonucleotide complementary in sequence to the clone we want to select willhydrogen bond to the single-stranded DNA on the nylon membrane. Once hybridizationis complete, excess oligonucleotide is washed from the nylon membrane, which is nowcovered with a sheet of X-ray ﬁlm and placed in a light-tight cassette. The radioactivity in\\nthe oligonucleotide will darken the silver grains on the X-ray ﬁlm—a process known as\\nautoradiography. A positive clone will show up as a black spot on the ﬁlm. Superimposing\\nthe X-ray ﬁlm back onto the original bacterial plate will identify the living bacterial colony\\nthat contains the desired foreign DNA clone.\\nAntibody Probes for cDNA Clones. This method makes use of speci ﬁc antibodies\\nto detect bacteria expressing the protein product of the DNA to be cloned. For this to work,the foreign DNA must be expressed in the bacterial cells; that is to say, its informationmust be copied ﬁrst into mRNA and then into protein. To ensure ef ﬁcient expression, the\\nplasmid vector contains a bacterial promoter sequence that is used to control transcriptionof foreign DNA. Such cloning vectors are known as expression vectors. The promoter ofthelacoperon is commonly used in this way. The clone library is plated onto agar plates\\ncontaining an inducer of the lacoperon such as IPTG (page 114) to ensure that lots of\\nmRNA and in turn lots of protein is synthesized. Figure 7.8 shows how an antibody, linkedto an enzyme (usually alkaline phosphatase), can detect a positive clone by generating acolored product. The pattern of colored spots on the nylon membrane is used to identify thebacterial clones of interest on the original agar plate.\\nGenomic DNA Clones\\nThe approach described in the previous section permits the isolation of cDNA clones.Complementary DNA clones have many important uses, some of which are described below.However, as a cDNA is a copy of mRNA only, when we want to isolate a gene to investigateits structure and function, we need to create genomic DNA clones. Genes contain exons andintrons and have regulatory sequences at their 5\\n′and 3′ends and are therefore much larger\\nthan cDNAs. The vectors used to clone genes must therefore be able to hold long stretchesof DNA. Plasmids used for cDNA cloning cannot do this. A selection of vectors used toclone genes is shown in Table 7.1. Vectors such as the P1 arti ﬁcial chromosomes (PACs) —\\nbased on the bacteriophage P1 —can hold about 150,000 bp of DNA. PACs have been used\\nvery successfully in the Human Genome Project (page 144). Yeast arti ﬁcial chromosome', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='76c59cae-e8d2-483a-931d-a9652eb988dc', embedding=None, metadata={'page_label': '150', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n140 RECOMBINANT DNA AND GENETIC ENGINEERING\\nTable 7.1. Vectors Used for Cloning Genomic DNA\\nGenomic DNA Cloning Vector Size of Insert (kb)\\nBacteriophage λ 9–23\\nCosmid 30 –44\\nPAC (P1 arti ﬁcial chromosome) 130 –150\\nBAC (bacterial arti ﬁcial chromosome) Up to 300\\nYAC (yeast arti ﬁcial chromosome) 200 –600\\n(YAC) vectors can hold between 200,000 and 600,000 million base pairs of foreign DNA.\\nThe choice of genomic vector is governed by the size of DNA the scientist needs to clone.PAC, Bacterial arti ﬁcial chromosome (BAC), and YAC vectors are needed if an entire gene\\nsequence is to be represented in a single clone. PAC, BAC, and YAC vectors contain allthe sequences needed to produce a minichromosome in the appropriate cell type: PACs andBACs in bacterial cells and YACs, as their name implies, to replicate foreign sequences ina yeast cell. The YAC vector therefore contains sequences that will allow the DNA to beprocessed by the host yeast cell as if it were a normal chromosome and to allow replicationalongside the other chromosomes in the cell. Thus a YAC vector has sequences that specifythe yeast centromere (page 403), telomeres (the ends of the new chromosome), and theyeast origin of replication. A YAC vector also contains selectable marker genes so that onlythose cells that have been transformed by a correctly constructed YAC chromosome willsurvive.\\nTo generate the large DNA fragments needed for genomic cloning, the chromosomal\\nDNA is incubated with a restriction endonuclease for a very short time. Not all the recog-nition sites for that enzyme are cleaved, and large fragments of DNA are hence producedby partial digestion. Genomic DNA fragments are joined to genomic cloning vectors in thesame way we join cDNAs to cDNA cloning vectors. In the example shown in Figure 7.9,human DNA has been introduced into the genome of a bacteriophage known as lambda ( λ).\\nThis particular vector can accommodate up to 23,000 bp of foreign DNA in its genome.Bacteria are then infected, generating a collection of bacteria called a genomic DNA library.\\nTo select the genomic DNA sequence of interest, the library is plated onto a layer (or\\nlawn) of cultured bacteria so that many copies of the recombinant bacteriophage can beproduced. A single λbacteriophage infects a single E. coli. The recombinant bacteriophages\\nthen multiply inside the host cells. The cells die and lyse, and the bacteriophages spread tothe surrounding layer of bacteria and infect them. These cells lyse, in turn, and the process\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 7.2 Cloning a Receptor Protein cDNA\\nGlutamate is one of the most important transmitters in the brain. The gene coding for the glutamate\\nreceptor, the protein on the surface of nerve cells that, upon binding glutamate, allows an in ﬂux of\\nsodium ions (page 372), remained uncloned for a number of years. Success came with the use of avery clever cloning strategy, based on the function of the receptor. mRNA was isolated from braincells and used as the template for the production of cDNA molecules. These were inserted into aplasmid expression vector. Following the introduction of these cDNAs into bacteria, a cDNA libraryrepresentative of all the mRNAs in the brain was produced. The many thousands of cDNA clonesin the library were then divided into pools. Each of the many pools was then injected into a frog egg(Xenopus oocyte), which transcribed the cDNAs into RNA and translated the RNA into protein. To\\nsee which of the oocytes had been injected with the cDNA for the glutamate receptor, these cellswere whole cell patch clamped (page 321). Glutamate was applied to the oocytes, and the oocyte', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='589fc5dd-5106-423b-873e-8ca27b468bd1', embedding=None, metadata={'page_label': '151', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\nCLONING A RECEPTOR PROTEIN 141\\nwhose injected pool had included the cDNA for the glutamate receptor responded with an inward\\ncurrent of sodium ions indicating the presence of glutamate receptors in the plasma membrane.\\nThe pool of cDNAs giving this response was further divided into smaller pools. Each of these\\nwas rescreened for the presence of glutamate receptor activity. This was followed by several roundsof rescreening. For each round a further subdivision was made of the cDNAs into pools containingfewer and fewer cDNA molecules. Eventually each pool contained only a single cDNA so that thecDNA for the glutamate receptor could be identi ﬁed. A number of other receptors have now been\\nisolated using the same strategy in which a functional assay is used to identify the cDNA encodingthe receptor.\\n1. mRNA from brain used to make cDNA\\n3.\\n5.4.2. cDNA clones\\nextract cDNAs and inject into oocytes \\nNa+\\nNa+pool 1 pool 4 pool 3 pool 2subdivide into pools\\nglutamateno receptor activity\\noocytes producing an inward sodium current\\non glutamate application identify the pool containingthe cDNA coding for the glutamate receptor\\nrepeat steps 4 and 5 on pool 1 cDNAs, each\\ntime using smaller cDNA subpools until asingle cDNA giving a positive response inthe receptor assay has been identified', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4ee2ae32-78bf-4aa3-a59f-401bb56f7f71', embedding=None, metadata={'page_label': '152', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n142 RECOMBINANT DNA AND GENETIC ENGINEERING\\nprobe will hybridize to\\ncomplementarygene sequenceswash nylon\\nmembrane and\\nexpose to X-ray filmbacteriophage λ vector\\nthe nonessential regionof the λ genome can be\\nreplaced by a pieceof  foreign DNA\\npackaged to produce bacteriophage particlesPartially digest with\\nEco Rl (E) to generatelarge fragmentsHuman genomic\\nDNA\\nλ genome containing a fragment of human DNAE E\\ninfected cells replicate therecombinant bacteriophage andplaques developinfect a lawn of \\nE. coli  growing on an agar medium\\nnylon membrane placedon surface of agar plate\\nremove nylonmembrane andtreat with alkali\\nincubate nylon\\nmembrane with radiolabeled\\ncDNA probe\\nspecificplaqueidentified\\nX-ray film\\nautoradiogramdigest with Eco Rl\\nleft λ arm right λ armEE E\\nEE E E\\nE EDNA\\nligase\\nFigure 7.9. Generation and selection of genomic DNA clones.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a6354ba5-0d73-4fe0-8364-bb58380f34b9', embedding=None, metadata={'page_label': '153', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\nUSES OF DNA CLONES 143\\nis repeated. The dead cells give rise to a clear area on the bacterial lawn called a plaque.\\nEach plaque contains many copies of a recombinant bacteriophage that can be transferredto a nylon membrane (Fig. 7.9). Speci ﬁc DNA clones are selected by incubating the nylon\\nmembrane with a radiolabeled cDNA probe complementary to the genomic sequence beingsearched for. This produces a radioactive area on the nylon membrane that is identi ﬁed by\\nautoradiography. The use of a cDNA sequence as a gene probe makes the task of isolating\\nthe corresponding genomic sequence much easier.\\nA radioactive cDNA probe can be synthesized using the method called random priming.\\nThe cDNA clone is heated so that the two strands will separate. Each strand will act asthe template for the synthesis of a new DNA strand. A mixture of random hexamers, sixnucleotide sequences, containing all possible combinations of the four bases (A, T, C, G) isadded to the denatured cDNA along with DNA polymerase and the four deoxynucleotidesdATP, dTTP, dCTP, and dGTP. The hexamers will hydrogen-bond to their correspondingsequences on the cDNA templates and prime the synthesis of new DNA strands. If, forexample, radiolabeled [ α-\\n32P]dATP is included in the reaction, the newly synthesized DNA\\nstrands will be radioactive.\\nUSES OF DNA CLONES\\nDNA Sequencing\\nThe ability to determine the order of the bases within a DNA molecule has been one of thegreatest technical contributions to molecular biology. DNA is made by the polymerizationof the four deoxynucleotides dATP, dGTP, dCTP, and dTTP. These are joined togetherwhen DNA polymerase catalyzes the formation of a phosphodiester link between a free 3\\n′-\\nhydroxyl on the deoxyribose sugar moiety of one nucleotide and a free 5′-phosphate group\\non the sugar residue of a second nucleotide. However, the arti ﬁcial dideoxynucleotides\\nddATP, ddGTP, ddCTP, and ddTTP have no 3′-hydroxyl on their sugar residue (Fig. 7.10),\\nand so if they are incorporated into a growing DNA chain, synthesis will stop. This is thebasis of the dideoxy chain termination DNA sequencing technique devised by FrederickSanger and for which he was awarded a Nobel prize in 1980.\\nO\\nC\\nC CCH\\nHH\\nHH HCH2 O O OP\\nOOO\\nP\\nOO\\nP\\nOO\\ncannot form a phosphodiester bond with\\nan incoming deoxynucleotidegeneralstructure of adideoxynucleotidebase\\ntriphosphateγβ α5′\\n4′\\n3′ 2′1′\\nFigure 7.10. A dideoxynucleotide.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4d92867e-31ed-45fa-bd32-19f1e9247170', embedding=None, metadata={'page_label': '154', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n144 RECOMBINANT DNA AND GENETIC ENGINEERING\\nThis technique is illustrated in Figure 7.11. A cloned piece of DNA of unknown se-\\nquence is ﬁrst joined to a short oligonucleotide whose sequence is known. The DNA is\\nthen made single-stranded so that it can act as the template for the synthesis of a new\\nDNA strand. All DNA synthesis requires a primer; in this case a primer is provided thatis complementary in sequence to the oligonucleotide attached to the template DNA. Fourseparate mixtures are prepared. Each contains the DNA template, the primer (which hasbeen radiolabeled), DNA polymerase, and the four deoxynucleotides. The mixtures differin that each also contains a low concentration of one of the four dideoxynucleotides ddATP,ddGTP, ddCTP, or ddTTP. When a molecule of dideoxynucleotide is joined to the newlysynthesized chain, DNA synthesis will stop.\\nLet us follow what happens in the tube containing ddTTP in Figure 7.11. The ﬁrst base\\nthat DNA polymerase encounters in the DNA template to be sequenced is an A. Since thetube contains much more dTTP than ddTTP, DNA polymerase will add a dTTP to mostof the primer molecules. However, a small fraction of the primers will have a ddTTP addedto them instead of dTTP since DNA polymerase can use either nucleotide as a substrate. Thenext base encountered is a G. DNA polymerase is unable to attach dCTP to the ddTTP sincethere is no OH group on the 3\\n′carbon of the sugar, and so DNA synthesis is terminated.\\nThe majority of strands, however, had a dTTP added, and for these DNA polymerase canproceed, building the growing strand. No problems are encountered with the next six bases.However, the eighth base in the template strand is another A, and once again a smallfraction of the growing strands will have ddTTP added instead of dTTP. In the same wayas before, these strands can grow no further. This process will be repeated each time an Aoccurs on the template strand. When the reaction is over, the tube will contain a mixtureof DNA fragments of different length, each of which ends in a ddTTP. Similarly, each ofthe other three tubes will contain a mixture of DNA chains of different length, each ofwhich ends in either ddCTP, ddATP, or ddGTP. To determine the sequence of the newlysynthesized chains, each of the four samples is loaded onto a polyacrylamide gel. Becausethe primer used was radiolabeled, all the new DNA chains will carry a radioactive tag sothat, after electrophoresis, the pattern of DNA fragments on the gel can be detected byautoradiography. Each terminated reaction will show up as a black band on the X-ray ﬁlm.\\nThe smallest DNA molecules are that fraction in the tube containing ddTTP whose growthwas blocked after the ﬁrst base, T. These move farthest and produce the band at the bottom\\nof the T lane. DNA molecules one nucleotide larger were produced in the tube containingddCTP —their growth was blocked after the second base, C. These move almost as far, but\\nnot quite, producing the band at the bottom of the C lane. Reading bands up from the bottomof the gel therefore tells us the sequence in which bases were added to the unknown strand:T, C, and so on. Because the new chain is complementary in sequence to its template strand,the sequence of the template strand can be inferred.\\nThe Human Genome Project required the process of DNA sequencing to be automated.\\nInstead of using radioactivity and one reaction tube for each dideoxynucleotide, we nowuse dideoxynucleotides tagged with ﬂuorescent dyes. Each of the four dideoxynucleotides\\nddATP, ddGTP, ddCTP, and ddTTP is tagged with a different ﬂuorescent dye. This means\\nthat all four of the termination reactions can be carried out in a single reaction tube and loadedinto the same well on the polyacrylamide gel. As the reaction product drips from the bottomof the gel, the ﬂuorescence intensity of each of the four colors corresponding to the four\\ndideoxynucleotides is monitored, and this information is transferred straight to a computerwhere the data are analyzed. An example of a DNA trace produced using ﬂuorescently tagged\\ndideoxynucleotides is shown on the CBASC website. Each peak represents a terminated', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='67cd9f80-0de1-41e9-9ed4-0c7eddfacf42', embedding=None, metadata={'page_label': '155', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\nUSES OF DNA CLONES 145\\nDNA\\nsequencingby theSangermethodsingle-stranded DNA\\nto be sequenced known sequence\\nGTCATCGTCCAGTAGCAG CTAGGCTTGA\\nplus radioactively labeledprimer that pairs withknown sequence\\nplus dATP, dCTP,\\ndTTP, dGTP\\nplus DNA polymerase\\nsequence\\nread fromgelG\\nA\\nT\\nC\\nC\\nG\\nAA\\nC\\nT\\nautoradiogram3′\\n5′ddTTP ddCTP ddATP ddGTP5′ 3′\\nincubated with chain\\nterminating\\ndideoxynucleotides\\nddTTP ddCTP ddATP ddGTP\\ngel electrophoresis\\nFigure 7.11. DNA sequencing and the dideoxy chain termination method.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='f1998eb4-7f40-4c15-a7d9-238deaea9746', embedding=None, metadata={'page_label': '156', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n146 RECOMBINANT DNA AND GENETIC ENGINEERING\\nDNA product; so by reading the order of the peaks, the DNA sequence is determined. In\\nthis example G is yellow, A is green, T is red, and C is blue.\\nTo determine the base sequence of the entire human genome, the DNA was cut by partial\\nrestriction endonuclease digestion (page 140) to give large fragments of about 150,000 bp.These were cloned into a vector such as PAC (Table 7.1). The aim was to create a libraryof clones that overlapped one another. Each clone was sequenced, as described above,and their base sequences compared. Because the clones overlapped, it was possible, bycomparing their sequences, to line up the position of each clone with respect to its neighbors.This required the development of sophisticated computer programs, and the creation of alarge database of information to order the 3 x 10\\n9base pairs that comprise the human\\ngenome.\\nSouthern Blotting\\nIn 1975 Ed Southern developed an ingenious technique, now known as Southern blotting,which can be used to detect speci ﬁc genes (Fig. 7.12). Genomic DNA is isolated and digested\\nwith one or more restriction endonucleases. The resultant fragments are separated accordingto size by agarose gel electrophoresis. The gel is soaked in alkali to break hydrogen bondsbetween the two DNA strands and then transferred to a nylon membrane. This producesan exact replica of the pattern of DNA fragments in the agarose gel. The nylon membraneis incubated with a cloned DNA fragment tagged with a radioactive label (page 143). Thegene probe is heated before adding to the nylon membrane to make it single-stranded so itwill base pair, or hybridize, to its complementary sequences on the nylon membrane. Asthe gene probe is radiolabeled, the sequences to which it has hybridized can be detected byautoradiography.\\nMutations that change the pattern of DNA fragments —for instance, by altering a re-\\nstriction endonuclease recognition site or deleting a large section of the gene —can easily\\nbe detected by Southern blotting. This technique is therefore useful in determining whetheran individual carries a certain genetic defect or if a fetus is homozygous for a particulardisorder. All that is needed is a small DNA sample from white blood cells or, in the caseof a fetus, from the amniotic ﬂuid in which it is bathed, or by removing a small amount of\\ntissue from the chorion villus that surrounds the fetus in the early stages of pregnancy.\\nForensic laboratories use Southern blotting to generate DNA ﬁngerprints from samples\\nof blood or semen left at the scene of a crime. A DNA ﬁngerprint is a person-speci ﬁc\\nSouthern blot. The gene probe used in the test is a sequence that is repeated very manytimes within the human genome —a microsatellite sequence (page 100). Everyone carries\\na different number of these repeated sequences, and because they lie adjacent to eachother on the chromosome they are called VNTRs (variable number tandem repeats). Whengenomic DNA is digested with a restriction endonuclease and then analyzed by Southernblotting, a DNA pattern of its VNTRs is produced. Unless they are identical twins, it isextremely unlikely that two individuals will have the same DNA ﬁngerprint pro ﬁle. It has\\nbeen estimated that if eight restriction endonucleases are used, the probability of two peoplewho are not identical twins generating the same pattern is one in 10\\n30.\\nA special type of Southern blot, called a zoo blot, can be used to reveal genes that are\\nsimilar in different species. Such genes, which have been conserved through evolution, arelikely to code for crucial proteins. A probe generated from a genomic DNA library is usedto probe genomic DNA from many different species. A probe that hybridizes with DNAfrom a number of species is likely to represent all or part of such a conserved gene.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='c3825b46-269f-4b2f-a0dc-5bc826c1f663', embedding=None, metadata={'page_label': '157', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\nUSES OF DNA CLONES 147\\nnylon membrane\\nincubated withradioactive cDNAprobe (       )agarose gel\\nsalt solution+−\\nagarose gel\\nagarose gel\\nwash nylonmembrane\\nautoradiogramnylon membrane\\nnylon membranestack of paper towelsDNA fragments separate\\naccording to size in an\\nelectric fieldhigh molecular weight\\nlow molecular weight\\npreparation ofSouthern blot\\nDNA transfers from the gel tothe nylon membrane as the saltsolution passes through the geland membrane and into thepaper towelsfilter paper actsas a wick\\nnylon membrane\\nexposed to\\nX-ray film\\ngene probe\\nhybridizes to\\ncomplementary\\nsequenceDNA placed in\\nwell at the topof the gel\\nFigure 7.12. The technique of Southern blotting.\\nIn situ Hybridization\\nIt is possible, using the technique of in situ hybridization, to identify individual cells ex-\\npressing a particular mRNA. To do this we need to synthesize an antisense RNA molecule —\\nan RNA that is complementary in sequence to the mRNA of interest. In a test tube, the\\nappropriate strand of the cloned cDNA is copied into the antisense RNA using RNA poly-merase. The RNA is then labeled with a modi ﬁed nucleotide that can subsequently be\\ndetected using an antibody and a color reaction. The cDNA must ﬁrst be cloned into an', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a231d410-a713-4a2c-a97c-2aadaf7d0a8a', embedding=None, metadata={'page_label': '158', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n148 RECOMBINANT DNA AND GENETIC ENGINEERING\\n1. Denature RNA\\n2. Separate RNAs on denaturing\\n3. Blot RNA onto membrane filter\\n4. incubate membrane with cDNA probe\\n5. Autoradiography\\nspecific mRNA\\ndetected bycDNA probeagarose gel\\nsizemarkers\\nFrom phenobarbital\\ntreated animal\\nFrom untreated\\nanimal\\nCYP2B1\\nmRNA\\n(a) (b)\\nFigure 7.13. (a). The technique of northern blotting. ( b). A northern blot reveals that the CYP2B1\\ngene is upregulated in animals given phenobarbital.\\nexpression vector that contains a promoter sequence to which RNA polymerase can bind.\\nA thin tissue section, attached to a glass microscope slide, is incubated with the antisenseRNA. The antisense RNA will hybridize, in the cell, to its complementary mRNA partner.Excess antisense RNA is washed off the slide, leaving only the hybridized probe. The colorreaction is now carried out so that the cells expressing the mRNA of interest can be identi ﬁed\\nby bright- ﬁeld microscopy (page 3).\\nNorthern Blotting\\nFigure 7.13 ashows a blotting technique that can determine the size of an mRNA and tell\\nus about its expression patterns. RNA is denatured by heating to remove any intramolec-ular double-stranded regions and then electrophoresed on a denaturing agarose gel. TheRNA is transferred to a nylon membrane (as described on page 136 for the transfer ofDNA). The nylon membrane is incubated with a radiolabeled, single-stranded cDNA probe', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='06aa5cf3-e36a-4aea-8e83-66aaa1fe1f2b', embedding=None, metadata={'page_label': '159', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\nUSES OF DNA CLONES 149\\nTable 7.2. Blotting Techniques\\nWhat Is Probed Nature of the Probe Book Page for Description\\nSouthern blot DNA DNA 146\\nNorthern blot RNA cDNA or RNA 148Western blot Protein Antibody 166\\n(page 143), or an antisense RNA probe (page 147). Following hybridization, excess probe\\nis washed off and the nylon membrane exposed to X-ray ﬁlm. The mRNA is visualized on\\nthe autoradiogram (page 139) because it hybridized to the radioactive probe. By analogywith Southern blotting, this technique is called northern blotting (Table 7.2). Figure 7.13 b\\nshows a northern blot for a cytochrome P450 (page 249) mRNA known as CYP2B1. Thegene for this protein is activated in the liver by the barbiturate phenobarbital and hence alot of CYP2B1 mRNA is made.\\nProduction of Mammalian Proteins in Bacteria\\nThe large-scale production of proteins using cDNA-based expression systems has wide ap-plications for medicine and industry. It is increasingly being used to produce polypeptide-based drugs, vaccines, and antibodies. Such protein products are called recombinant be-\\ncause they are produced from a recombinant plasmid. For a mammalian protein to besynthesized in bacteria its cDNA must be cloned into an expression vector (as described onpage 139). Insulin was the ﬁrst human protein to be expressed from a plasmid introduced\\ninto bacterial cells and has now largely replaced insulin from pigs and cattle for the treat-ment of diabetes. Other products of recombinant DNA technology include growth hormoneand factor VIII, a protein used in the treatment of the blood clotting disorder hemophilia.Factor VIII was previously isolated from donor human blood. However, because of thedanger of infection from viruses such as HIV , it is much safer to treat hemophiliacs withrecombinant factor VIII. It should, in theory, be possible to express any human protein viaits cDNA.\\nProtein Engineering\\nThe ability to change the amino acid sequence of a protein by altering the sequence of itscDNA is known as protein engineering. This is achieved through the use of a techniqueknown as site-directed mutagenesis. A new cDNA is created that is identical to the naturalone except for changes designed into it by the scientist. This DNA can then be used togenerate protein in bacteria, yeast, or other eukaryotic cell lines.\\nTheﬁrst use of protein engineering is to study the protein itself. A comparison of the cat-\\nalytic properties of the normal and mutated form of an enzyme helps to identify amino acidresidues important for substrate and cofactor binding sites (Chapter 11). This technique wasalso used to identify the particular charged amino acid residues responsible for the selectivityof ion channels (page 437). Now scientists are using protein engineering to generate new pro-teins as tools, not only for scienti ﬁc research but for wider medical and industrial purposes.\\nSubtilisin is a protease and is one of the enzymes used in biological washing powder.\\nThe natural source of this enzyme is Bacillus subtilis, an organism that grows on pig feces.\\nTo produce, from this source, the 6000 tons of subtilisin used per year by the soap powder', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='48eecce1-3208-4270-82e7-76c304ba3732', embedding=None, metadata={'page_label': '160', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n150 RECOMBINANT DNA AND GENETIC ENGINEERING\\nindustry is a dif ﬁcult and presumably unpleasant task. The cDNA for subtilisin has been\\nisolated and is now used by industry to synthesize the protein on a large-scale in E. coli. The\\nwild-type (natural) form of subtilisin is, however, prone to oxidation because of a methionine\\npresent at position 222 in the protein. Its susceptibility to oxidation makes it an unsuitableenzyme for a washing powder that must have a long shelf life and be robust enough to with-stand the rigours of a washing machine with all its temperature cycles. Scientists thereforechanged the codon for methionine (AUG) to the codon for alanine (GCG). When the mod-iﬁed cDNA was expressed in E. coli, the resulting enzyme was found to be active and not\\nsusceptible to oxidation. This was excellent news for the makers of soap powder. However,it is always necessary to check the kinetic parameters of a “new”protein produced from\\na modi ﬁed cDNA. For subtilisin (met\\n222) the KM(page 244) is 1 .4×10−4M while for\\nsubtilisin (ala 222) the KMis 7.3×10−4M. This means that at micromolar concentrations\\nof dirt the modi ﬁed enzyme will bind less dirt than the wild-type one, but the dirt concentra-\\ntions caked onto our clothes are well above micromolar. The product turnover number, kcat\\n(page 242), is 50 s−1for subtilisin (met 222) and 40 s−1for subtilisin (ala 222): The mutant\\nenzyme is slightly slower, but not by much. By changing a met to an ala, a new enzyme hasbeen produced that can do a reasonable job and is stable during storage and in our washingmachines.\\nGreen ﬂuorescent protein is found naturally in certain jelly ﬁsh. Protein engineering\\nhas now created a palette of proteins with different colors (blue, cyan, green, and yellow).However, the great advantage of these proteins to biologists is that chimeric proteins\\n(proteins composed of two parts, each derived from a different protein) incorporating aﬂuorescent protein are intrinsically ﬂuorescent. This means that our protein of interest can\\nbe imaged inside a living cell using a ﬂuorescence microscope (page 6). The ﬂuorescent\\npart of the chimeric protein tells us exactly where our protein is targeted in the cell and ifthis location changes in response to signals.\\nFigure 7.14 illustrates how this approach can be used to determine what concentration of\\nglucocorticoid drug is required to cause the glucocorticoid receptor to move to the nucleus.The plasmid, like many plasmids designed for convenience of use, contains a multiplecloning site (MCS), sometimes called a polylinker, which is a stretch of DNA that containsseveral restriction endonuclease recognition sites. A convenient restriction endonucleaseis used to cut the plasmid (in this case, for green ﬂuorescent protein) and the cDNA for\\nthe glucocorticoid receptor is inserted. The plasmid also contains a promoter sequence,derived from a virus, that will drive the expression of the DNA into mRNA in mammaliancells that have been infected with the plasmid (or transfected ). The chimeric protein,\\nsynthesized from the mRNA, will have one part coded for by the cDNA of interest —the\\nother part being the ﬂuorescent protein. The plasmid is grown up in bacteria and then used\\nto transfect mammalian cells. In the absence of glucocorticoid the protein, and thereforethe green ﬂuorescence, is in the cytosol. When enough glucocorticoid is added, it binds to\\nthe chimeric protein, which then moves rapidly to the nucleus.\\nPolymerase Chain Reaction\\nA technique called the polymerase chain reaction (PCR) has revolutionized recombinantDNA technology. It can amplify DNA from as little material as a single cell and fromvery old tissue such as that isolated from Egyptian mummies, a frozen mammoth, andinsects trapped in ancient amber. A simple mouth swab can yield enough cheek cell DNAto determine carriers of a particular recessive genetic disorder. PCR is used to amplify DNA', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='63c378b2-2682-41a4-8306-75c0ab1363e6', embedding=None, metadata={'page_label': '161', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\nCytomegalovirus promoter\\nfor mammalian expression\\nGreenfluorescentproteinGreenfluorescentprotein\\nOrigin of\\nreplication\\n(ori)\\nAntibiotic\\nresistance\\ngeneAntibiotic\\nresistance\\ngene\\nBacterial\\npromoterBacterial\\npromoter\\ntransfect cellsMultiplecloning\\nsiteoriCMV promoter\\nGlucocorticoid\\nreceptor\\nadd\\nglucocorticoid\\nFigure 7.14. A chimera of green ﬂuorescent protein and the glucocorticoid receptor reveals its location in living cells.\\n151', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='22b49e1b-6c3f-4e29-90cc-f6ec6087d633', embedding=None, metadata={'page_label': '162', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n152 RECOMBINANT DNA AND GENETIC ENGINEERING\\nfrom fetal cells or from small amounts of tissue found at the scene of a crime. The tool\\nthat makes PCR possible is thermostable DNA polymerase, an enzyme that survives andcan even function at extremely high temperatures that would denature (page 206) mostenzymes. Thermostable DNA polymerases are isolated from bacteria that live in extremelyhot deep-sea volcanic environments.\\nFigure 7.15 shows how PCR uses a thermostable DNA polymerase and two short\\noligonucleotide DNA sequences called primers. Each primer is complementary in sequenceto a short length of one of the two strands of DNA to be ampli ﬁed. The primers direct DNA\\npolymerase to copy each of the template strands. The DNA duplex is heated to 90\\n◦Ct o\\nseparate the two strands (step 1). The mixture is cooled to 60◦C to allow the primers to anneal\\nto their complementary sequences (step 2). At 72◦C the primers direct the thermostable\\nDNA polymerase to copy each of the template strands (step 3). These three steps, whichtogether constitute one cycle of the PCR, produce twice the number of original templates.The process of template denaturation, primer annealing, and DNA synthesis is repeatedmany times to yield many thousands of copies of the original target sequence.\\nIdentifying the Gene Responsible for a Disease\\nUntil recently, the starting point for an identi ﬁcation of the gene responsible for a particular\\ninherited disease was a pattern of inheritance in particular families plus a knowledge of thetissues affected. It is very dif ﬁcult to ﬁnd the gene responsible for a disease when the identity\\nof the normal protein is unknown. Very often, the ﬁrst clue is the identi ﬁcation of other\\ngenes that are inherited along with the malfunctioning gene and that are therefore likely tolie close on the same chromosome (this is linkage, page 408). In the past, chromosome\\nwalking was then used to identify the disease gene. This is a slow and tedious process that\\ninvolves isolating a series of overlapping clones from a genomic clone library. One startsby isolating a single clone and then uses this as a probe to ﬁnd the next clone along the\\nchromosome. The second clone is used to ﬁnd the third, and this is used to ﬁnd a fourth\\nclone, and so on. Each successive clone is tested to ﬁnd out if it might include all or part of\\nthe gene of interest (e.g., by using northern blotting, page 148, to see if the gene is expressedin a tissue known to be affected in the disease). Once a candidate gene is identi ﬁed, one\\ncanﬁnd out if it was the gene of interest by sequencing it in unaffected individuals and\\nin disease sufferers; if it is the disease gene, its sequence will be different in the affectedpeople. With the publication of the entire human genome, chromosome walking to generatea series of overlapping clones for testing is unnecessary, but identifying the gene that isresponsible for a particular inherited condition is even now a time-consuming task.\\nReverse Genetics\\nBecause beginning with an inherited defect in function and working toward identi ﬁcation\\nof a gene is even today a time-consuming task, more and more scientists are now workingthe other way: they take a gene with a known sequence but unknown function and deduce itsrole. Since we know the complete genome of a number of species, we can sit at a computerand identify genes that look interesting —for example, because their sequence is similar to\\na gene of known function. The gene of interest can be mutated and reinserted into cellsor organisms, and the cells or organisms tested for any altered function. This approach iscalled reverse genetics.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='07dcd3ef-103b-4bdc-adc6-ed940c3a0c2a', embedding=None, metadata={'page_label': '163', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n5′\\n3′3′5′5′\\n3′ 5′3′\\n5′ 3′\\n3′ 5′target sequence\\nadd\\noligonucleotide\\nprimerscycle 1\\ncycle 2\\ncycle 3\\nand so on.....step 1\\nheat to 90°C to\\nseparate strands\\nstep 1\\nheat to 90°C to\\nseparate strands\\nstep 1\\nheat to 90°C to\\nseparate strandsstep 2\\ncool to 60°C\\nprimers anneal\\nto their\\ncomplementary\\nsequencesstep 3\\nwarm to 72°C\\nthermostable\\nDNA polymerase\\nextends primers\\nstep 3\\nwarm to 72°C\\nthermostable\\nDNA polymerase\\nextends primers\\nstep 3\\nwarm to 72°C\\nthermostable\\nDNA polymerase\\nextends primersstep 2\\ncool to 60°C\\nprimers anneal\\nto their\\ncomplementary\\nsequencesstep 2\\ncool to 60°C\\nprimers anneal\\nto their\\ncomplementary\\nsequences\\nFigure 7.15. Ampliﬁcation of a DNA sequence using the polymerase chain reaction.\\n153', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='9446fd14-18a3-46b9-9752-29a33ec9ae86', embedding=None, metadata={'page_label': '164', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n✎\\n✍\\x0c\\n✌Medical\\nRelevance\\n7.1Microarrays and Cancer Classiﬁcation\\nMicroarrays or gene chips are tiny glass wafers to which cloned DNAs are attached.\\nThe principle of microarray technology is to isolate mRNA from a particular cell\\nand to hybridize this to the DNAs on the chip. Because the mRNAs are tagged with a ﬂuorescent\\ndye, the DNAs to which they hybridize on the chip can be detected. Excess mRNA is removed, andﬂuorescent areas are viewed using a special scanner and microscope. Computer algorithms have\\nbeen written to analyze the hybridization patterns seen for a particular microarray. The numberand type of DNAs used to make a microarray is dependent on the question to be answered.\\nOne of the spin-offs of the Human Genome Project is to identify sets of genes involved in\\ndisease. Microarrays are being used to type the mRNAs produced by different cancers in the hopethat this will lead to better diagnosis and therefore better treatment. Leukemia, a cancer affectingthe blood, is not a single type of disease. Microarray analysis is helping to classify different typesof leukemia more precisely by cataloging the mRNAs expressed in different patients. Preliminarystudies are encouraging. A microarray of 6817 cDNAs was used to compare the blood mRNAsfrom patients with acute lymphoblastic leukemias or acute myeloid leukemias. The mRNAs ofeach patient was passed over the microarray and complementary sequences hybridized. FiftycDNA/mRNA hybrids were found that could be used to re ﬁne the classi ﬁcation of lymphoblas-\\ntic and myeloid leukemias. In addition, the patterns allowed the classi ﬁcation of lymphoblastic\\nleukemias into T-cell or B-cell classes. This separation of lymphoblastic leukemias into two classesis an important distinction when it comes to deciding on the best treatment for a patient. As morepatients with leukemia are investigated using microarrays, it should be possible to design smallerchips, with fewer cDNAs, but with greater prognosis value.\\nA recent study has shown that the fate of women with breast cancer can be predicted by\\nmicroarray analysis of their cancer tissue. Several genes were shown to be important for thisprediction. If the women had a certain gene expression pattern, then the disease was likely to recurwithin a 5-year period. However, if the gene pattern was of a second type, the cancer almost neverreturned. Women that fall into this second class also show no additional bene ﬁt from chemotherapy\\nor radiotherapy. Microarray analysis not only offers a survival prognosis for women with breastcancer but has also shown that unpleasant treatment for the disease is unnecessary for thosepossessing the second type of gene pattern.\\nB\\nCA\\n1\\nBCA234hybridize to\\nmRNAs\\nmRNAs hybridize to DNAs inA1, A4, B1, B3, C2micro arraychip of12 DNAsamples 1234\\n154', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='8788201e-89f8-4ecb-a75a-477c376745a8', embedding=None, metadata={'page_label': '165', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\nnucleus derived\\nfrom the sperm\\nnucleus derivedfrom the egg\\ninject “gene\\nconstruct ”isolate fertilized\\neggs\\nimplant eggs\\ninto foster mothertest fortransgene\\npups\\nrat growthhormonestructural gene\\nplasmid\\ndigest with Bgl I and Bam HImouse\\nmetalothionein\\n5′-flanking\\nsequenceBgl I\\nBam HI\\nBam HIBgI I\\ninject eggs\\nimplantation\\nfed Zn++heavy metal\\nion enhancer\\nsequence\\npups(a)\\n(b)+− +−gene\\nconstruct\\nFigure 7.16. (a). Transgenic mouse carrying a foreign gene. ( b). the metalothionein gene contains\\na heavy-metal ion enhancer sequence. The +mice carry the transgene while littermates without\\nthe transgene are indicated by −.\\n155', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='dd6fa2f8-af34-4bec-977e-06536efc0ee3', embedding=None, metadata={'page_label': '166', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n156 RECOMBINANT DNA AND GENETIC ENGINEERING\\n13cross-over region cross-over region\\nhomologous recombination\\ntargets transgeneto exon 2 on chromosome\\ntransfect withtransgene and selectfor neo-resistant cellsvectorgenomic clonegene inchromosome\\nneo\\nneo\\n(a)\\n(b)strain 1 (white)\\nremove blastocysts\\nfrom pregnant mouse.\\nIsolate embryonic stem cellsgrow embryonic\\nstem cells1\\n42\\ninject embryonic stemcells into blastocyst ofstrain 2 (black)\\npups born3\\n5\\ntransgenic\\nchimeric\\nmouseimplant into\\nfoster motherexon\\n1exon\\n3exon\\n2\\nFigure 7.17. Knock-out transgenic mice. ( a). Transgene is incorporated into genome of embry-\\nonic stem cells by homologous recombination. ( b). Genetically modi ﬁed embryonic stem cells are\\ninjected into a blastocyst, which is implanted into a foster mother.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1017123d-5183-4676-9af1-2a157293e7ea', embedding=None, metadata={'page_label': '167', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\nETHICS OF DNA TESTING FOR INHERITED DISEASE 157\\nTransgenic Animals\\nA transgenic animal is produced by introducing a foreign gene into the nucleus of a fertilized\\negg (Fig. 7.16 a). The egg is then implanted into a foster mother and the offspring are tested\\nto determine whether they carry the foreign gene. If they do, a transgenic animal has beenproduced. The ﬁrst transgenic mice ever made were used to identify an enhancer sequence\\nthat activates the metallothionein gene when an animal is exposed to metal ions in its diet.The 5\\n′ﬂanking sequence of the metallothionein gene was fused to the rat growth hormone\\ngene (Fig. 7.16 b). This DNA construct, the transgene, was injected into fertilized eggs.\\nWhen the mice were a few weeks old, they were given drinking water containing zinc. Micecarrying the transgene grew to twice the size of their litter mates because the metallothioneinenhancer sequence, stimulated by zinc, had increased growth hormone production.\\nTransgenic farm animals —such as sheep synthesizing human factor VIII in their milk —\\nhave been created. This is an alternative to producing human proteins in bacteria.\\nTransgenic mice are increasingly being used to prove a protein ’s function. To do this the\\ngene for the protein is knocked out. This is done by either inserting a piece of foreign DNAinto the gene, or by deleting the gene from the mouse genome. The consequences of theprotein ’s absence are then established. Figure 7.17 describes the method called insertional\\nmutagenesis for knocking out a gene ’s function. The ﬁrst step is to isolate a genomic clone\\ncontaining the gene to be knocked-out. A marker gene, such as the drug resistance geneneo, is then inserted into the genomic clone, usually in exon 2 of the gene. This means\\nthat the normal, functional product of the gene cannot be synthesized. This construct is thetransgene, which is then introduced into embryonic stem (ES) cells. These are cells derivedfrom the inner mass of a blastocyst —that is, a very early embryo —of a white mouse.\\nHomologous recombination inside the embryonic stem cells will replace the normal gene\\nwith the transgene. Cells in which this rare event happens will survive when grown onneomycin while embryonic stem cells not containing the transgene will die. The geneticallymodi ﬁed embryonic stem cells are inserted into the blastocyst cavity of a black mouse and\\nthe blastocyst implanted into a foster mother. Transgenic mice will be chimeric and have amixed color coat because the cells derived from the genetically modi ﬁed embryonic stem\\ncells will give a white-color coat while the cells from the blastocyst will give a black-colorcoat. Subsequent breeding will produce a pure white mouse that is homozygous for theknocked-out gene. The effect of knocking-out the gene can then be analyzed.\\nETHICS OF DNA TESTING FOR INHERITED DISEASE\\nThe applications of recombinant DNA technology are exciting and far-reaching. However,the ability to examine the base sequence of an individual raises important ethical questions.Would you want to know that you had inherited a gene that will cause you to die prematurely?Some of you might feel ﬁne about this and decide to live life to the full. We suspect most\\npeople would not want to know their fate. But what if you have no choice and DNA testingbecomes obligatory should you wish to take out life insurance? In the United Kingdominsurance companies are now able to ask for the results of the test for Huntington ’s disease.\\nThis is a fatal degenerative brain disorder that strikes people in their forties. From the insur-ance company ’s point of view DNA testing could mean higher premiums according to life\\nexpectancy or at worst refusal of insurance cover. There is much ongoing debate on this issue.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='8085b0f4-1af1-40e8-8d02-965cc0a3b2ae', embedding=None, metadata={'page_label': '168', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n158 RECOMBINANT DNA AND GENETIC ENGINEERING\\nOn the positive side, we can now test for changes in certain genes that are involved in\\ndrug metabolism. Adverse drug reactions are the fourth highest cause of death in the United\\nStates. By knowing, for example, which individuals carry mutations in genes coding forcytochromes P450 (page 249), medicines that need to be metabolized by these proteins canbe avoided.\\nAnalysis of fetal DNA can inform parents if their child is affected by an inherited\\ndisorder. It is now possible, using the PCR, to examine the genotype of a single cell froman eight-cell embryo. If the DNA test is negative, the embryo can be inserted into themother, and the parents know they have not passed on a defective gene to their child. Thiscombination of DNA testing and in vitro fertilization has given rise to the unfortunate phrase“designer babies ”.\\nIn the future it may be possible to correct genetic defects before or after birth by\\nreplacing mutated genes by a normal copy. The technology to carry out such experiments isvery demanding and important ethical questions are raised. In Chapter 20 we will discussthe potential use of gene therapy for the correction of the genetic disorder cystic ﬁbrosis.\\nf\\nIN DEPTH 7.1 Genetically Modi ﬁed (GM) Plants —Can They Help to Feed the\\nWorld?\\nThe arguments about the value of GM crops and the damage they cause the en-\\nvironment will continue for a long time. These have largely concerned plants thatproduce insecticide and plants that are resistant to herbicide. Almost unnoticed inthe maelstrom of claim and counterclaim has been the use of genetic engineeringto produce nutritionally enhanced crops. Rice is a staple food in many countriesbut lacks many vital nutrients. About 800 million children suffer from vitamin Adeﬁciency, which can result in blindness and a weakened immune system. In an\\nattempt to overcome this severe nutritional de ﬁciency, a group of Swiss scientists\\nhave engineered the rice endosperm to produce provitamin A (beta-carotene). Thisis converted in the body to vitamin A. Four genes were introduced into the rice en-dosperm, three genes from the daffodil and one from the bacterium Erwinia. These\\ngenes code for all the proteins needed to make beta-carotene. The transgenic riceis golden in color because of the presence of large amounts of beta-carotene.\\nAlmost 24% of the world ’s population, mainly women, are de ﬁcient in iron.\\nRice has been engineered with the gene for ferritin, an iron-binding protein. Thisrice has higher amounts of iron and has been shown to correct iron de ﬁciency in\\nlaboratory animals. The plan is to cross the provitamin-A-enriched, iron-enrichedstrains with agricultural rice strains to produce a hardy and nutritional crop. Goldenrice, enriched in iron, provides a great opportunity to improve the health of billionsof people.\\nSUMMARY\\n1.DNA sequences can be cloned using reverse transcriptase, which copies mRNA into\\nDNA to make a hybrid mRNA:DNA double-stranded molecule. The mRNA strandis then converted into DNA by the enzymes ribonuclease H and DNA polymerase.The new double-stranded DNA molecule is called complementary DNA (cDNA).\\n2.Restriction endonucleases cut DNA at speci ﬁc sequences. DNA molecules cut with\\nthe same enzyme can be joined together. To clone a cDNA, it is joined to a cloning', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='dfa93524-9da5-4a33-835f-d7e30adc9ff3', embedding=None, metadata={'page_label': '169', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\nREVIEW QUESTIONS 159\\nvector —a plasmid or a bacteriophage. Genomic DNA clones are made by joining\\nfragments of chromosomal DNA to a cloning vector. When the foreign DNA fragment\\nhas been inserted into the cloning vector, a recombinant molecule is formed.\\n3.Recombinant DNA molecules are introduced into bacterial cells by the process oftransformation. This produces a collection of bacteria (a library) each of which con-tains a different DNA molecule. The DNA molecule of interest is then selected fromthe library using either an antibody or a nucleic acid probe.\\n4.There are many important medical, forensic and industrial uses for DNA clones.These include:\\nrDetermination of the base sequence of the cloned DNA fragment.rIn situ hybridization to detect speci ﬁc cells making RNA complementary to the\\nclone.rSouthern blotting and genetic ﬁngerprinting to analyze an individual ’s DNA pat-\\ntern.rSynthesis of mammalian proteins in bacteria or eukaryotic cells.rChanging the DNA sequence to produce a new protein.rGeneration of ﬂuorescent protein chimeras for subsequent microscopy on live cells.rThe polymerase chain reaction, which lets us produce many copies of a DNAmolecule in a test tube.\\nrProduction of transgenic animals to study gene function.\\nFURTHER READING\\nMcPherson J. D., et al. 2001. A physical map of the human genome. Nature 409: 934 –941.\\nMullis, K. B. 1990. The unusual origin of the polymerase chain reaction. Sci. Am. 262: 56 –65.\\nVenter, J. C., et al. 2001. The sequence of the human genome. Science 291: 1304 –51.\\nGolden rice: Beyer, P., Al-Babili, S., Ye, X., Lucca, P., Schaub, P., Welsch, R., and Potrykus, I. 2002.\\nGolden Rice: Introducing the beta-carotene biosynthesis pathway into rice endosperm by genetic\\nengineering to defeat vitamin A de ﬁciency J. Nutrition 132: 506S –510S.\\nMicroarrays: Friend, S. H., and Stoughton, R. B. 2002. The magic of microarrays. Sci. Am. 286:\\n44–49.\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.Plasmids used as cloning vectors\\nA. are circular molecules.\\nB. have an origin of replication.C. carry antibiotic resistance genes.D. have unique restriction endonuclease cutting sites.E. all of the above.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='914b2e1a-2f4d-4ec6-8820-1ce963619672', embedding=None, metadata={'page_label': '170', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\n160 RECOMBINANT DNA AND GENETIC ENGINEERING\\n2.A cDNA clone library is a collection of clones representing\\nA. the entire genome of the sample tissue.\\nB. all protein-coding gene sequences in the genome of the sample tissue.C. all intronic sequences of genes expressed in the sample tissue.D. the mRNAs expressed in the sample tissue.E. all tRNAs being used in the sample tissue.\\n3.A cDNA clone library can be screened with an antibody if\\nA. the plasmid vector contains a convenient promoter such as that of the lacoperon to increase\\nexpression.\\nB. the plasmid vector contains a multiple cloning site.C. the nylon “colony lift ”membrane is ﬁrst soaked in alkali to denature DNA.\\nD. the nylon “colony lift ”membrane is ﬁrst treated with a strong protease to break peptide bonds.\\nE. an antibody that recognizes IPTG, and that is linked to an enzyme such as alkaline phosphatase,\\nis available.\\n4.A radioactive gene probe can be synthesized by the method of random priming if the\\nfollowing factors are present in the incubation mixture:\\nA. The DNA template from which the probe will be made, present as single-stranded DNA\\nB. The four deoxynucleotides, dATP, dCTP, dGTP, and dTTP, one of which should carry a ra-\\ndioactive tag\\nC. Primers for DNA synthesisD. DNA polymeraseE. All of the above\\n5.To sequence DNA using the automated dideoxy chain termination method, the reaction\\nmixture should contain the DNA to be sequenced plus\\nA. the four deoxynucleotides, dATP, dCTP, dGTP, and dTTP.\\nB. the four dideoxynucleotides, ddATP, ddCTP, ddGTP, and ddTTP, each carrying a different\\nﬂuorescent tag.\\nC. a DNA primer.D. DNA polymerase.E. all of the above.\\n6.The technique called\\nA. Southern blotting is used to analyze RNA.B. western blotting is used to analyze DNA.C. eastern blotting is used to analyze RNA.D. Southern blotting is used to analyze DNA.E. northern blotting is used to analyze protein.\\n7.Each cycle in the polymerase chain reaction involves the following steps:\\nA. Polymerization then annealing then denaturationB. Denaturation then polymerization then annealingC. Denaturation then annealing then polymerizationD. Denaturation then chimera formation then polymerizationE. Denaturation then polymerization then sequencing\\nANSWERS TO REVIEW QUESTIONS\\n1.E.All these statements are true. Plasmids used as cloning vectors are circular, have an origin of\\nreplication so that they can replicate in the host bacterium, carry antibiotic genes so that transformed', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='044c7402-c0ab-43e9-95a1-04fa720a70fa', embedding=None, metadata={'page_label': '171', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-07 WY001-Bolsover-v2.cls September 4, 2003 13:46\\nANSWERS TO REVIEW QUESTIONS 161\\ncells can be selected for, and have unique restriction enzyme cutting sites to allow genes of interest\\nto be inserted into the plasmid.\\n2.D.cDNA means DNA that is complementary to the mRNA found in a tissue sample. cDNAs are\\ngenerated using a poly-T primer that binds to processed mRNA, but which will not bind to tRNAor rRNA, which have no poly-A sequence.\\n3.A.Addition of the gratuitous inducer lacoperon inducer IPTG should ensure good production\\nof the protein, which can then be recognized by the antibody. Considering the other answers:(B) The multiple cloning site makes insertion of the gene of interest easier, but has no implicationsregarding the method to be used to then screen the library. (C) If one is using an antibody torecognize protein, the presence of DNA is largely irrelevant. (D) The antibody is used to recognizethe particular protein coded by the gene of interest. It would therefore be highly counterproductiveto destroy the proteins with a protease. (E) There is no reason why one should want to use anantibody that recognizes IPTG.\\n4.E.The primers bind to the template, allowing DNA polymerase to generate the probe using the\\ndeoxynucleotides provided. Because one of the deoxynucleotides carries a radioactive tag, theresulting probe is radioactive and can therefore be used, for example, to reveal the position ofcomplementary DNA on a Southern blot.\\n5.E.The primer binds to the DNA to be sequenced. DNA polymerase then begins work using the\\ndeoxynucleotides, but each time it instead incorporates a dideoxynucleotide; synthesis of thatstrand stops so that the DNA of that speci ﬁc length is ﬂagged with a ﬂuorophore of the color\\ncorresponding to the base encountered.\\n6.D.Southern blotting is used to analyze DNA, northern blotting is used to analyze RNA, and western\\nblotting is used to analyze protein (Table 7.2). There is no eastern blotting (although there is atechnique called SouthWestern blotting!).\\n7.C.PCR proceeds by cycles of denaturation (separation of the DNA strands), annealing (of primers\\nonto the strands), and polymerization to form double helices.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='04874ec3-45c9-41fa-8663-0b1d3304b106', embedding=None, metadata={'page_label': '172', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\n8\\nMANUFACTURING PROTEIN\\nThe genetic code (page 75) dictates the sequence of amino acids in a protein molecule.\\nThe synthesis of proteins is quite complex, requiring three types of RNA. Messenger RNA(mRNA) contains the code and is the template for protein synthesis. Transfer RNAs (tRNAs)are adapter molecules that carry amino acids to the mRNA. Ribosomal RNAs (rRNAs) formpart of the ribosome that brings together all the components necessary for protein synthesis.Several enzymes also help in the construction of new protein molecules. This chapterdescribes how the nucleotide sequence of an mRNA molecule is translated into the aminoacid sequence of a protein.\\nFigure 8.1 shows the basic mechanism of protein synthesis, also called translation.\\nIn the ﬁrst step, free amino acids are attached to tRNA molecules. In the second step, aribosome assembles on the mRNA strand to initiate synthesis. In the third step, the ribosometravels along the mRNA. At each codon on the RNA a tRNA binds, bringing the amino aciddeﬁned by that codon to be added to the growing polypeptide chain. In the last, fourth, stepthe ribosome encounters a stop codon and protein synthesis is terminated.\\nATTACHMENT OF AN AMINO ACID TO ITS tRNA\\nAmino acids are not directly incorporated into protein on a messenger RNA template. Anamino acid is carried to the mRNA chain by a tRNA molecule. tRNAs are small, about 70–80 nucleotides in length, and are folded into precise three-dimensional structures becauseof hydrogen bonding between bases in particular stretches of the molecule. This gives rise\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n163', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='fc5cecf7-6fce-4736-aa8c-32164ab42cf7', embedding=None, metadata={'page_label': '173', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\n164 MANUFACTURING PROTEIN\\n= aminoacid (aa)\\nanticodon\\ntRNA\\nmRNAstartpolypeptide\\nproteinribosome1\\n50s\\n30s70sstop2\\n3 3 3\\n4\\n3′ 5′aa+\\nFigure 8.1. Overview of protein synthesis\\nto four double-stranded regions, and it is these that give tRNA its characteristic cloverleaf\\nstructure when drawn in two dimensions as in Figure 8.2.\\nEach tRNA molecule has an amino acid attachment site at its 3′end and an anticodon,\\nthree bases that are complementary in sequence to a codon on the mRNA. The tRNA bindsto the mRNA molecule because hydrogen bonds form between the anticodon and codon.For example, the codon for methionine is 5\\n′AUG 3′which will base pair with the anticodon\\n3′UAC 5′.\\nTransfer RNA, the Anticodon, and the Wobble\\nAlthough 61 codons specify the 20 different amino acids, there are not 61 tRNAs; insteadthe cell economizes. The codons for some amino acids differ only in the third position of thecodon. Figure 4.8 on page 78 shows that when an amino acid is encoded by only two differ-ent triplets the third bases will be either C and U, or A and G. For example aspartate is codedby GAC and GAU and glutamine by CAA and CAG. The wobble hypothesis suggests that\\nthe pairing of the ﬁrst two bases in the codon and anticodon follows the standard rules —G\\nbonds with C and A bonds with U —but the base pairing in the third position is not as\\nrestricted and can wobble. If there is a pyrimidine, U or C, in the third position of the codon,it can ﬁt with any purine, G or A, in the anticodon, and vice versa. Thus, only one tRNA\\nmolecule is required for two codon sequences. The anticodon of some tRNAs containsthe unusual nucleoside inosine (I), whose base is the purine hypoxanthine (Fig. 2.13, page34). Inosine can base pair with any of U, C, or A in the third position of the codon. Some', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d3b64725-ae7d-47fa-b2e3-efe6a83cc623', embedding=None, metadata={'page_label': '174', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\nTHE RIBOSOME 165\\nHydrogen bonds between\\ncomplementary basesA\\nCC\\nanticodon5′ end P3′ end - amino acid\\nattachment siteOH\\nFigure 8.2. Transfer RNA (tRNA).\\ntRNA molecules can therefore base pair with as many as three different codons provided\\ntheﬁrst two bases of the codon are the same. For example, the tRNA for isoleucine has the\\nanticodon UAI and can therefore base pair with any of AUU, AUC, or AUA.\\nThe attachment of an amino acid to its correct tRNA molecule is illustrated in Figure 8.3.\\nThis process occurs in two stages, both catalyzed by the enzyme aminoacyl tRNA synthase.\\nDuring the ﬁrst reaction, the amino acid is joined, via its carboxyl group, to an adenosine\\nmonophosphate (AMP) and remains bound to the enzyme. All tRNA molecules have attheir 3\\n′end the nucleotide sequence CCA. In the second reaction aminoacyl tRNA synthase\\ntransfers the amino acid from AMP to the tRNA, forming an ester bond between its carboxylgroup and either the 2\\n′-o r3′-hydroxyl group of the ribose of the terminal adenosine (A)\\non the tRNA to form an aminoacyl tRNA. This step is often referred to as amino acid\\nactivation because the energy of the ester bond can be used in the formation of a lowerenergy peptide bond between two amino acids. A tRNA that is attached to an amino acid isknown as a charged tRNA. There are at least 20 aminoacyl tRNA synthases, one for each\\namino acid and its speci ﬁc tRNA.\\nTHE RIBOSOME\\nThe ribosome is the cell ’s factory for protein synthesis. Each ribosome consists of two\\nsubunits, one large and one small, each of which is made up of RNA plus a large number ofproteins. The ribosomal subunits and their RNAs are named using a parameter, called theS value. The S value, or Svedberg unit, is a sedimentation rate. It is a measure of how fast\\na molecule moves in a gravitational ﬁeld. For example, the bigger a ribosomal subunit, the\\nquicker it will sediment and the larger the S value. Prokaryotic ribosomes, and those found', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='169b1193-9591-4a52-a9ee-40e832928967', embedding=None, metadata={'page_label': '175', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\n166 MANUFACTURING PROTEIN\\nf\\nIN DEPTH 8.1 How We Study Proteins in One Dimension\\nThe technique known as SDS–PAGE is widely used to analyze the spectrum of\\nproteins made by a particular tissue, cell type, or organelle. It is also invaluable\\nfor assessing the purity of isolated proteins. SDS stands for sodium dodecyl sulfateand PAGE for polyacrylamide gel electrophoresis.\\nThe aim of the technique is to denature the proteins to be analyzed and then\\nto separate them according to their size in an electrical ﬁeld. To do this we ﬁrstadd a chemical called 2-mercaptoethanol to the protein sample. This will breakanydisulﬁde bonds (page 191) within a protein or between protein subunits. Next\\nSDS, which is an anionic detergent, is added and the protein sample boiled. SDScoats each protein chain with negative charge. Each individual polypeptide in thesample becomes covered with an overall net negative charge. This means thatwhen placed in an electrical ﬁeld the SDS-coated proteins will separate accordingto their size because the smaller proteins move most quickly toward the positiveelectrode or anode. Polyacrylamide provides the matrix through which the proteins\\nmove during electrophoresis. The monomeric form, acrylamide, is poured into amold. A solid but porous gel forms as the acrylamide polymerizes. The shape ofthe mold is such that wells are formed in the gel into which the protein samplecan be loaded for electrophoresis.\\nWhen electrophoresis is complete, the proteins are stained by incubating the\\ngel in a solution of Coomassie brilliant blue. Each protein band stains blue and isdetectable by eye. However, if the amount of protein is very low, a more sensitivedetection system is needed such as a silver stain. Proteins of known molecular massare also electrophoresed on the gel. By comparison with the standard proteins, themass of an unknown protein can be determined.\\nIf we want to follow the fate of a single protein in a complex mixture of pro-\\nteins, we combine SDS–PAGE with a technique called western blotting. The namewestern blotting is, like northern blotting, a play on the name of Dr. Ed Southernwho devised the technique of Southern blotting to analyze DNA (Table 7.2).\\nThe protein mixture is separated by SDS–PAGE. A nylon membrane is then\\nplaced up against the polyacrylamide gel and picks up the proteins, so that thepattern of protein spots on the original polyacrylamide gel is preserved on thenylon membrane. The nylon membrane is then incubated with an antibody speciﬁcfor the protein of interest. This antibody, the primary antibody, will seek out andbind to its partner protein on the nylon membrane. A second antibody is addedthat will bind to the primary antibody. To be able to detect the speciﬁc proteinof interest on the membrane, the secondary antibody is attached to an enzyme.In the ﬁgure shown, the enzyme used was horseradish peroxidase. A substrateis added and is converted by the enzyme into a colored product. The protein ofinterest is seen as a colored band on the nylon membrane. The same enzyme-linkedsecondary antibody can be used in other laboratories or at other times for westernblotting of many different proteins because the speciﬁcity is determined by theunlabeled primary antibody.\\nPart A of the ﬁgure shows the Coomassie brilliant-blue-stained pattern of pro-\\nteins isolated from the endoplasmic reticulum of liver. The leftmost lane is from aphenobarbital-treated animal while the middle lane is from an untreated controlanimal. The dark bands indicate the presence of protein. The spectrum of proteinsis very similar in the two samples, except that a band with a relative molecular mass(M\\nr) of about 52,000 is much darker in the sample from the treated animal. This\\ntells us that drug treatment has caused an increase in the production of a proteinwith this relative molecular mass. Western blotting (part B) using an antiCYP2B1', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='82bafcb8-6b76-4bd6-8bdf-e63795a92e79', embedding=None, metadata={'page_label': '176', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\nSDS-PAGE 167\\nve\\nve94,000\\n67,000\\n43,000\\n30,000\\nSDS - PAGE Western blot\\n(a) (b)52,000From phenobarbital\\ntreated animalFrom untreatedanimalMolecular weightstandardsFrom phenobarbitaltreated animalFrom untreatedanimalProteins separate according to size\\nantibody con ﬁrms that the induced protein is the cytochrome P450 protein known\\nas CYP2B1. The CYP2B1 gene is activated by phenobarbital to produce more\\nCYP2B1 protein to metabolize and clear the drug from the body (page 249).\\nWe already showed how northern blotting revealed that transcription of the\\nCYP2B1 gene is increased after phenobarbital treatment (Fig. 7.13 on Page 148).\\nThe western blot shown here demonstrates that, as expected, the amount of\\nCYP2B1 protein is increased as well.\\nSodium dodecyl sulfate (SDS) is a major constituent of hair shampoo, where it\\nis usually called by its alternative name of sodium lauryl sulfate.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='44aa3706-8ec0-475b-96c8-ebfcd064eb25', embedding=None, metadata={'page_label': '177', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\n168 MANUFACTURING PROTEIN\\namino acid + ATP\\naminoacyl tRNA \\nsynthase\\namino acid\\nAMP\\namino acid\\nAMPaminoacyl tRNA\\n(charged tRNA)tRNApyrophosphate\\nAMPPOH\\nOO\\nP\\nO OO\\nO\\namino acid\\nACC\\nFigure 8.3. Attachment of an amino acid to its tRNA.\\ninside mitochondria and chloroplasts, are 70S when fully assembled and comprise a larger,\\n50S subunit and a smaller 30S one. Eukaryotic ribosomes are 80S when fully assembledand comprise a larger, 60S subunit and a smaller 40S one.\\nThe formation of a peptide bond (page 38) between two amino acids takes place on\\nthe ribosome. The ribosome has binding sites for the mRNA template and for two chargedtRNAs (Fig. 8.4). An incoming tRNA with its linked amino acid occupies the aminoacyl\\nsite (A site), and the tRNA attached to the growing polypeptide chain occupies the peptidyl\\nsite (P site).\\nBACTERIAL PROTEIN SYNTHESIS\\nRibosome-Binding Site\\nFor protein synthesis to take place, a ribosome must ﬁrst attach to the mRNA template.\\nAUG is not only the start codon for protein synthesis; it is used to code for all the othermethionines in the protein. How does the ribosome recognize the correct AUG at which to', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4d822772-538b-4110-a11a-c1fed83f0afd', embedding=None, metadata={'page_label': '178', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\nBACTERIAL PROTEIN SYNTHESIS 169\\n5′ 3′ UCGAGCCUG UUCAAGAA PP\\nsergluhislystyrvalgly\\nphefmet\\nFigure 8.4. The P and A sites on the ribosome.\\nbegin protein synthesis? All bacterial mRNAs have at their 5′end a stretch of nucleotides\\ncalled the untranslated (or leader) sequence, which do not code for the protein. These\\nnucleotides are nevertheless essential for the correct placing of the ribosome on the mRNA.Bacterial mRNA molecules usually have a nucleotide sequence similar to 5\\n′GGAGG 3′\\nwhose center is about 8 to 13 nucleotides upstream of (5′to) the AUG start codon. This\\nsequence is complementary to a short stretch of sequence, 3′CCUCC 5′, found at the 3′\\nend of the rRNA molecule within the 30S ribosomal subunit (Fig. 8.5). The mRNA and therRNA interact by complementary base pairing to place the 30S ribosomal subunit in thecorrect position to start protein synthesis. The sequence on the mRNA molecule is calledthe ribosome-binding site. This is sometimes referred to as the Shine –Dalgarno sequence\\nafter the two scientists who found it.\\nBecause the genetic code is read in triplets of three bases, there are three possible\\nreading frames (page 79). The reading frame that is actually used by the cell is de ﬁned by\\ntheﬁrst AUG that the ribosome encounters downstream of the ribosome-binding site.\\nChain Initiation\\nThe ﬁrst amino acid incorporated into a new bacterial polypeptide is always a modi ﬁed\\nmethionine, formyl methionine (fmet) (Fig. 8.6). Methionine ﬁrst attaches to a speci ﬁc\\ntRNA molecule, tRNAfmet, and is then modi ﬁed by the addition of a formyl group that\\nattaches to its amino group. tRNAfmethas the anticodon sequence 5′CAU 3′that binds to\\nits complementary codon, the universal start codon AUG.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='7a11d607-2d1f-4f16-9db9-4686a63d9b48', embedding=None, metadata={'page_label': '179', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\nmRNA\\n30S ribosomal \\nsubunit\\n30S initiation complex5′ribosome \\nbinding sitestart \\ncodon\\nAUG GGAGG 3′\\n5′ 3′IF3\\n5′ 3′fmet\\nGTP\\nGDP + Pi\\n70S initiation complex5′ 3′50S ribosomal \\nsubunituntranslated \\nsequence\\n+\\nAUG\\nIF2\\nAUGPP AAIF3IF2IF1\\nfmet5′ 3′\\nfmet\\n+ IF2 +GTP\\nAUGAUGGGAGGCCUCCIF1\\nIF1\\nIF1IF1IF3\\nIF3\\nIF3\\nFigure 8.5. Formation of the 70S initiation complex.\\n170', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='14089edd-960b-45d2-aed9-fe0be9386ea6', embedding=None, metadata={'page_label': '180', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\nBACTERIAL PROTEIN SYNTHESIS 171\\nCH3\\nS\\nNCH2\\nH\\nC\\nHOCH2\\nCH OOC\\nformyl\\ngroup\\nFigure 8.6. Formyl methionine.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 8.1 The Irritating Formyl Methionine\\nWhite blood cells are strongly attracted by any peptide that begins with formyl methionine: theyassume an amoeboid shape (Fig. 1.5 on page 5) and begin to crawl toward the source of the peptide.To a white blood cell, the presence of a peptide beginning with formyl methionine means thatthere is an infection nearby that needs to be fought. This is because the body ’s own proteins do\\nnot contain formyl methionine: only prokaryotes begin protein synthesis with this modi ﬁed amino\\nacid.\\nThe 70S Initiation Complex\\nThe initiation phase of protein synthesis involves the formation of a complex between the\\nribosomal subunits, an mRNA template and tRNAfmet(Fig. 8.5). A 30S subunit attaches\\nto the ribosome-binding site as described above. tRNAfmetthen interacts with the AUG\\ninitiation codon, and ﬁnally the 50S ribosomal subunit attaches. The ribosome is now\\ncomplete, and the ﬁrst tRNA and its amino acid are in place in the P site of the ribosome.\\nA 70S initiation complex has been formed, and protein synthesis can begin. The ribosomeis orientated so that it will move along the mRNA in the 5\\n′to 3′direction, the direction in\\nwhich the information encoded in the mRNA molecule is read.\\nThree proteins called initiation factors 1, 2, and 3, together with the nucleotide guano-\\nsine triphosphate, are needed to help the 70S initiation complex form. Initiation factors 1and 3 are attached to the 30S subunit. Initiation factor 3 helps in the recognition of theribosome-binding site on the mRNA. Initiation factor 2 speci ﬁcally recognizes tRNA\\nfmet\\nand binds it to the ribosome. When the 50S subunit attaches, the three initiation factors arereleased and the guanosine triphosphate is hydrolyzed, losing its γphosphate to become\\nguanosine diphosphate.\\nElongation of the Protein Chain\\nThe synthesis of a protein begins when an aminoacyl tRNA enters the A site of the ribosome(Fig. 8.7). The identity of the incoming aminoacyl tRNA is determined by the codon on', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='628d9826-9ca7-41eb-959a-70503b9016b1', embedding=None, metadata={'page_label': '181', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\n5′ 3′ AUGUAC\\nUUC CUGP A PA\\n+EF-tu +GTP\\n+ GDP EF-tu Pi\\nEF-ts\\nEF-tu+ GDP\\npeptidyl transferase\\nEF–G\\nEF–GGTP\\nPi+ GDPphefmet\\n5′ 3′ AUGUAC\\nUUCAAG\\nCUGfmet\\ntRNAphe\\n5′ 3′ AUG UUC CUGfmet-phe\\nAAG\\n5′ 3′ UUCAAG\\nCUG AUGfmet-pheAA PP\\nAA PP\\nAA PP\\nFigure 8.7. Elongation of the protein chain. Pi represents the inorganic phosphate ion HPO2−\\n4.\\n172', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='9d6e0af5-c9e0-4a8e-bfc4-e8f5465a2cda', embedding=None, metadata={'page_label': '182', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\nBACTERIAL PROTEIN SYNTHESIS 173\\nthe mRNA. If, for example, the second codon is 5′UUC 3′, then phenylalanyl tRNAphe\\n(whose anticodon is 5′GAA 3′) will occupy the A site. The P site has, of course, already\\nbeen occupied by tRNAfmetduring the formation of the initiation complex. Now that both\\nthe A and P sites are occupied, the enzyme peptidyl transferase catalyzes the formation of\\na peptide bond between the two amino acids (fmet and phe in this example). The dipeptide\\nis attached to the tRNA occupying the A site. Because the tRNA in the P site is no longerattached to an amino acid, it is released from the ribosome and it can be reused. Theribosome now moves along the mRNA to the right so that the tRNA dipeptide now occupiesthe P site (leaving the A site free for another incoming aminoacyl tRNA). The movementof the ribosome, three nucleotides at a time, relative to the mRNA, is called translocation,\\na term often used in biology to describe movement that is actively driven and preciselycontrolled. The process of peptide bond formation is followed by translocation, and thewhole process is repeated until the ribosome reaches a stop signal and protein synthesisterminates.\\nProteins are synthesized beginning at their amino or N terminus (page 184). The\\nﬁrst amino acid hence has a free (although formylated) amino group. The last amino\\nacid in the chain has a free carboxyl group and is known as the carboxyl or C terminus(page 185).\\nElongation of a polypeptide chain needs the help of three proteins called elongation\\nfactors. These proteins are needed to speed up the process of protein synthesis. To bind\\nto the A site, the aminoacyl tRNA must form a complex with elongation factor tu and amolecule of guanosine triphosphate (GTP). On hydrolysis of GTP to guanosine diphosphate(GDP) and inorganic phosphate ion (Pi), the aminoacyl tRNA is able to enter the A site.Elongation factor tu is released from the aminoacyl tRNA by the action of a second protein,elongation factor ts. This removes the GDP bound to elongation factor tu, and the protein isrecycled. The movement of the ribosome needs the help of a third protein elongation factorG and the energy supplied by the hydrolysis of a second GTP molecule.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 8.2 The Diptheria Bacterium Inhibits Protein Synthesis\\nSome bacteria cause disease because they inhibit eukaryotic protein synthesis. Diphtheria was once\\na widespread and often fatal disease caused by infection with the bacterium Corynebacterium\\ndiptheriae. This organism produces an enzyme (diphtheria toxin) that inactivates eukaryotic elon-\\ngation factor 2 (the equivalent of the bacterial elongation factor G). Diphtheria toxin splits thebond between ribose and nicotinamide in NAD\\n+(page 37), releasing free ribose and attaching the\\nremainder, called ADP-ribose, to elongation factor 2, a process known as ADP ribosylation. Theprotein is now inactive and is unable to assist in the movement of the ribosome along the mRNAtemplate. Protein synthesis therefore stops in the affected human cells. The advantage of this to thebacterium is that all the amino acids that the host was using to make protein are now available for itsown use.\\nThe Polyribosome\\nMore than one polypeptide chain is synthesized from an mRNA molecule at any given time.\\nOnce a ribosome has begun translocating along the mRNA, the start AUG codon is free,and another ribosome can bind. A second 70S initiation complex forms. Once this ribosome', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d5017c24-323a-4dd6-ab59-c787038d6cad', embedding=None, metadata={'page_label': '183', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\n174 MANUFACTURING PROTEIN\\n5′mRNA\\n3′\\npolypeptide chainribosome\\nFigure 8.8. Many ribosomes reading one mRNA form a polyribosome.\\nhas moved away, a third ribosome can attach to the start codon. This process is repeated\\nuntil the mRNA is covered with ribosomes. Each of these spans about 80 nucleotides. Theresultant structure, the polyribosome or polysome (Fig. 8.8), is visible under the electronmicroscope. This mechanism allows many protein molecules to be made at the same timeon one mRNA.\\nTermination of Protein Synthesis\\nThere are three codons, UAG, UAA, and UGA, that have no corresponding tRNA molecule.These codons are called stop codons. Instead of interacting with tRNAs, the A site occupied\\nby one of these codons is ﬁlled by proteins known as chain release factors. In the presence\\nof these factors the newly synthesized polypeptide chain is freed from the ribosome, and themRNA, tRNA, and the 30S and 50S ribosomal subunits dissociate (Fig. 8.9). Release factor1 causes polypeptide chain release from UAA and UAG, and release factor 2 terminateschains with UAA and UGA. A third protein, release factor 3, cooperates with the othertwo to stop protein synthesis. When the A site is occupied by a release factor, the enzymepeptidyl transferase is unable to add an amino acid to the growing polypeptide chain andinstead catalyzes the hydrolysis of the bond joining the polypeptide chain to the tRNA. Thecarboxyl (COOH) end of the protein is therefore freed from the tRNA, and the protein isreleased.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e879f0e6-ce9c-4018-83e4-2f6b1373f6a8', embedding=None, metadata={'page_label': '184', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\nEUKARYOTIC PROTEIN SYNTHESIS IS A LITTLE MORE COMPLEX 175\\n5′ 3′ UAA\\n5′ 3′peptidyl \\ntransferase\\nribosomal subunits dissociate\\nfree polypeptide chainpolypeptide chain\\nreleasefactor 1\\nrelease\\nfactor 3\\nrelease\\nfactor 1\\nFigure 8.9. Termination of protein synthesis.\\nThe Ribosome Is Recycled\\nIt would be wasteful if a complex structure such as the ribosome were used only once.\\nWhen protein synthesis is over, the two ribosomal subunits dissociate and can be reusedwhen needed.\\nEUKARYOTIC PROTEIN SYNTHESIS IS A LITTLE\\nMORE COMPLEX\\nElongation of the polypeptide chain and the termination of protein synthesis in eukaryotes\\ndoes not differ very much from that described for bacteria. However, the initiation ofprotein synthesis is more complex in eukaryotes. Their proteins always start with methionine', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6fea0d19-3979-49ad-8b90-7d9fcca234b8', embedding=None, metadata={'page_label': '185', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\n176 MANUFACTURING PROTEIN\\ninstead of the formyl methionine used in bacterial protein synthesis. A special transfer RNA,\\ntRNAmet\\ni, is used to initiate protein synthesis from the AUG start codon. The methionine\\nis often removed from the protein after synthesis. Eukaryotic mRNAs do not contain the\\nbacterial Shine –Dalgarno sequence for ribosome binding. Eukaryotic mRNAs have at their\\n5′end a 7-methyl guanosine cap (page 118). A number of proteins are needed to attach the\\nsmall, 40S, ribosomal subunit to the cap. Once attached, the 40S subunit slides along theleader sequence of the mRNA looking for the start codon. All eukaryotic mRNAs have asequence very similar to 5\\n′CCACC 3′adjacent to the initiating AUG codon. This sequence\\n(known as the Kozak sequence after the scientist who noted it) tells the ribosome that ithas reached the start AUG codon. The recognition of the AUG codon that speci ﬁes the start\\nsite for translation also requires the help of at least nine proteins. The tRNA\\nmet\\nibinds to the\\n40S subunit and then the large 60S ribosomal subunit attaches to form the 80S initiation\\ncomplex. Protein synthesis can now begin.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 8.3 How Hepatitis C Takes Control\\nThe hepatitis C virus shuts down the liver protein synthesis machinery and forces the resources\\nof the infected cell to be diverted into production of viral proteins that will be used to make newvirus. It does this by commandeering the cell ’s ribosomes to its own mRNAs. Hepatitis C mRNAs\\ndo not have a 7-methyl guanosine cap at their 5\\n′ends. Instead they have internal ribosome entry\\nsites (IRES), which are speci ﬁc sequences to which the ribosomes bind tightly. The clamping of the\\nribosome to the IRES forces initiation of viral protein synthesis at the expense of host cell proteinsynthesis.\\nHepatitis C is a particularly nasty virus because infected people can go on to develop liver\\ncancer, cirrhosis, and other chronic liver disease. In the United States alone, 10,000 people die peryear from hepatitis C infection. It is hoped that drugs can be developed that will prevent the hostribosome from binding to the viral IRES and so halt the production of new virus in an infectedliver.\\nAntibiotics and Protein Synthesis\\nMany antibiotics work by blocking protein synthesis, a property that is extensively ex-\\nploited in research and medicine. Many antibiotics only inhibit protein synthesis in bacteriaand not in eukaryotes. They are therefore extremely useful in the treatment of infectionsbecause the invading bacteria will die but protein synthesis in the host organism remainsunaffected. Examples are chloramphenicol, which blocks the peptidyl transferase reaction,and tetracycline, which inhibits the binding of an aminoacyl tRNA to the A site of theribosome. Both of these antibiotics therefore block chain elongation. Streptomycin, on theother hand inhibits the formation of the 70S initiation complex because it prevents tRNA\\nfmet\\nfrom binding to the P site of the ribosome.\\nPuromycin causes the premature release of polypeptide chains from the ribosome and\\nacts on both bacterial and eukaryotic cells. This antibiotic has been widely used in the studyof protein synthesis. Puromycin can occupy the A site of the ribosome because its structureresembles the 3\\n′end of an aminoacyl-tRNA (Fig. 8.10). However, puromycin does not bind\\nto the mRNA. Puromycin blocks protein synthesis because peptidyl transferase uses it asa substrate and forms a peptide bond between the growing polypeptide and the antibiotic.Once translocation has occurred, the growing polypeptide has no strong attachment to themRNA and is therefore released from the ribosome.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d4d28ea1-f5b6-4c9c-9fae-8f7545108c25', embedding=None, metadata={'page_label': '186', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\nEUKARYOTIC PROTEIN SYNTHESIS IS A LITTLE MORE COMPLEX 177\\npeptidyl \\ntransferaseOCH3\\nCH2\\nCH H2N\\nCO\\nNH\\nCH2\\nOH\\nN\\nNNN\\nN\\nCH3H3CR\\nCH HN\\nC O\\nO OH\\nO\\nN\\nN NN\\nNH2growing polypeptide\\npeptidyl tRNA puromycin binds \\nto the A sitetRNAtRNAtRNAOH\\nO\\nFigure 8.10. The antibiotic puromycin inhibits protein synthesis.\\nf\\nIN DEPTH 8.2 Proteomics\\nProteomics is the study of the proteome —the complete protein content of a cell.\\nIt is the proteins a cell makes that allow it to carry out its specialized functions.\\nAlthough, for example, a liver and a kidney cell have many proteins in common,they also each possess a unique subset of proteins, which gives to them their owncharacteristics. Similarly, a cell will need to make different proteins, and proteinsin different amounts, according to its metabolic state. The goal of proteomics is toidentify all of the proteins produced by different cells and how a particular diseasechanges a cell ’s protein pro ﬁle.\\nThe separation of a cell ’s protein mixture into individual components is tackled\\nusing a technique known as two-dimensional polyacrylamide gel electrophoresis.This produces a pattern of protein spots. These patterns are recorded and serveas templates for the comparison of the proteomes of different cells. Spots thatchange during a cell ’s development, or changes that occur in disease, can be easily\\nidenti ﬁed. A protein spot of interest is excised from the polyacrylamide gel and the\\nprotein broken into small, overlapping peptide fragments by proteolytic enzymes.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='8cda3816-867c-412e-b0d9-bc4e0eab8f18', embedding=None, metadata={'page_label': '187', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\n178 MANUFACTURING PROTEIN\\nSeparation\\nin seconddimensionSeparation in\\nfirst dimensionprotein\\nmixture\\nseparates\\naccording\\nto charge\\npH4.0 pH10.0pH10.0pH 4.0\\nproteins separateaccording to sizeon an SDS polyacrylamindegelprotein stops migratingwhen it reaches the pHof its isoelectric point\\n−\\n+\\nThe fragments are fed into a mass spectrometer, and their peptide mass ﬁngerprint\\ndetermined. The ﬁngerprint identi ﬁes the protein.\\nThe dream of many scientists, now that we know the base sequence of the\\nhuman genome, is to de ﬁne the human proteome and to determine all of its vari-\\nations. However, the genome projects use a single automated method to sequence\\nthe order of the four bases in DNA, and data are obtained at a relatively rapid pace.In contrast it is very labor intensive to identify proteins and much effort is beingmade to devise ways of speeding things up. The aim of proteomic research centersis to increase their throughput from about 40 to 100 peptide samples an hour, tothe daunting number of 1 million peptides a day.\\nSUMMARY\\n1.During protein synthesis, the genetic code in an mRNA is translated into a sequence\\nof amino acids. An amino acid attaches to the 3′end of a tRNA to form an aminoacyl\\ntRNA. Protein synthesis takes place on the ribosome, that binds both to mRNA andtRNA. It has two tRNA binding sites, the P site and the A site.\\n2.Initiation of protein synthesis in bacteria involves the binding of the 30S ribosomalsubunit to the mRNA. tRNA\\nfmetbinds to the initiation codon, and then the 50S\\nribosomal subunit attaches and the 70S initiation complex is formed.\\n3.Protein synthesis begins when a second aminoacyl tRNA occupies the A site. Eachincoming amino acid is speci ﬁed by the codon on the mRNA. The anticodon on the\\ntRNA hydrogen bonds to the codon, thus positioning the amino acid on the ribosome.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='fa687b17-4147-4213-9f78-160226a55f76', embedding=None, metadata={'page_label': '188', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\nREVIEW QUESTIONS 179\\nA peptide bond is formed, by peptidyl transferase, between the amino acids in the P\\nand A sites. The newly synthesized peptide occupies the P site, and another aminoacid is brought into the A site. This process of elongation requires a number ofproteins (elongation factors); as it continues, the peptide chain grows. When a stopcodon is reached, the polypeptide chain is released with the help of proteins knownas release factors.\\n4.More than one ribosome can attach to an mRNA. This forms a polyribosome, andmany protein molecules can be made simultaneously from the same mRNA.\\n5.Many antibiotics ﬁght disease because they inhibit particular steps in protein\\nsynthesis.\\nFURTHER READING\\nArnez, J. G., and Moras, D. 1997. Structural and functional considerations of the aminoacylation\\nreaction. Trends Biochem. Sci. 22: 211 –216.\\nLytle, J. R., Wu, L., and Robertson, H. D. 2001. The ribosome binding site of hepatitis C virus mRNA.\\nJ. Virol. 75: 7629 –7636.\\nMoore, P. B., and Steitz, T. A. 2002. The involvement of RNA in ribosome function. Nature 418:\\n229–235.\\nRibas de Pouplana, L., and Schimmel, P. 2001. Aminoacyl-tRNA synthetases: Potential markers of\\ngenetic code development. Trends Biochem. Sci. 26: 591 –596.\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.Which of the following steps is NOT part of translation?\\nA. Synthesis of mRNA\\nB. Charging of tRNAC. Initiation of protein synthesisD. Elongation of the polypeptide chainE. Termination of protein synthesis\\n2.In bacterial and eukaryotic mRNAs, the normal start codon for protein synthesis is\\nA. GAU.B. UGA.C. AUG.D. AGU.E. UAG.\\n3.The enzyme that attaches the amino acid to its tRNA is\\nA. aminoacyl mRNA synthase.B. aminoacyl tRNA synthase.C. amino acid attachment enzyme.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='f481fa63-da65-4bc9-8f20-19ec71e45f39', embedding=None, metadata={'page_label': '189', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\n180 MANUFACTURING PROTEIN\\nD. elongation factor tu.\\nE. tRNA polymerase.\\n4.Bacterial mRNAs usually have the nucleotide sequence 5′GGAGG 3′, which bonds to\\nA. tRNA.B. methionine.C. the 30S ribosomal subunit.D. the 40S ribosomal subunit.E. protein initiation factors.\\n5.The enzyme that catalyzes the formation of a peptide bond between two amino acids is\\nA. amino acid transferase.B. peptide joinase.C. peptidyl transferase.D. peptide elongation factor.E. puromycin.\\n6.Which of the following do NOT play a part in the termination of protein synthesis?\\nA. The codons AUG, AGG, and GUAB. The codons UAA, UAG and UGAC. Release factor 1D. Release factor 3E. Peptidyl transferase\\n7.The base hypoxanthine in the nucleotide inosine can hydrogen bond with\\nA. G, U, and C.B. G, U, and A.C. G, A, and C.D. U, C, and A.E. A only.\\nANSWERS TO REVIEW QUESTIONS\\n1.A.Synthesis of mRNA is not part of translation. All the others are, in particular, “charging ”means\\nattaching an amino acid to a tRNA ready for use in protein synthesis.\\n2.C.AUG, which codes for the amino acid methionine.\\n3.B.The enzyme is called aminoacyl tRNA synthase. Concerning the other answers: (A) Because\\namino acids are not attached to mRNA, there is no such process in cells. (C) Amino acid attachmentenzyme is a plausible name, albeit less speci ﬁc since it does not say to what the amino acid is\\nattached. However, the name is not used. (D) Elongation factor tu plays a role later in the processof protein synthesis, not during tRNA charging. (E) tRNA polymerase is not a name that is everused. If it did mean anything, it would mean the enzyme that makes tRNA (there is such an enzymein eukaryotes: it is called RNA polymerase III), not the one that attaches an amino acid.\\n4.C.The rRNA molecule within the 30S ribosomal subunit has a short stretch of sequence, 3\\n′CCUCC\\n5′, at its 3′end that is complementary in sequence to the 5′GGAGG 3′of the mRNA. Hydro-\\ngen bonding between these two sequences aligns the ribosome for protein synthesis. Concerninganswer D: the 40S ribosomal subunit plays a similar role in eukaryotes as the 30S unit does inprokaryotes. However, eukaryotic mRNAs do not have the prokaryotic GGAGG Shine –Dalgarno\\nsequence.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='9d830edf-c68a-4909-a7f8-ec29441e6a91', embedding=None, metadata={'page_label': '190', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-08 WY001-Bolsover-v2.cls September 24, 2003 20:32\\nANSWERS TO REVIEW QUESTIONS 181\\n5.C.The enzyme is called peptidyl transferase. Concerning the other answers: A, B, and D are\\nplausible answers, though D is the least plausible since most enzymes have names ending in ase;\\nhowever, they are not the names used. Puromycin is an antibiotic, not a protein (and therefore\\ncertainly not an enzyme).\\n6.A.These are codons directing the incorporation of methionine, arginine, and valine, respectively.\\nAll the others do play a part in the termination of protein synthesis, in particular, UAA, UAG,and UGA are stop codons, while if the A site is occupied by a release factor, peptidyl transferasecatalyzes the hydrolysis of the bond joining the polypeptide chain to the tRNA. The carboxyl(COOH) end of the protein is therefore freed from the tRNA and the completed protein is released.\\n7.D.Hypoxanthine can bond with any of U, C, or A.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='54f9cbc0-9a82-4add-9649-9b68f0263307', embedding=None, metadata={'page_label': '191', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\n9\\nPROTEIN STRUCTURE\\nVirtually everything cells do depends upon proteins. We are all, regardless of build, made up\\nof water plus more or less equal amounts of fat and protein. Although the DNA in our cellscontains the information necessary to make our bodies, DNA itself is not a signiﬁcant partof our body mass. Nor is it a chemically interesting molecule, in the sense that one length ofDNA is much the same as another in terms of shape and chemical reactivity. The simplicityof DNA arises because it is a polymer made up of only four fairly similar monomers, andthis is appropriate because the function of DNA is simply to remain as a record and to beread during transcription. In contrast, proteins made using the instructions in DNA varyenormously in physical characteristics and function and can be considered as constitutingthe much more complex proteome (page 178). Silk, hair, the lens of an eye, an immunoassay(such as found in a pregnancy test kit), and cottage cheese are all just protein plus more orless water, but they are different because the proteins they contain are different. Proteinscarry out almost all of the functions of the living cell including, of course, the synthesis ofnew DNA. Neither growth nor development would be possible without proteins.\\nMost proteins have functions that depend on their ability to recognize other molecules\\nby binding. This recognition depends on speciﬁc three-dimensional binding sites that makemultiple interactions with the ligand, the molecule being bound. To do this a protein must\\nitself have a speciﬁc three-dimensional structure. Each of the huge number of proteinfunctions demands its own protein structure. Evolution has produced this diversity by using apalette of 20 amino acid monomers, each with its own unique shape and chemical properties,as the building blocks of proteins. Huge numbers of very different structures are thereforepossible.\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n183', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='285d1a65-1e5b-423e-af3f-3fbc53451c6c', embedding=None, metadata={'page_label': '192', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\n184 PROTEIN STRUCTURE\\nNAMING PROTEINS\\nTo be able to discuss proteins, we need to give them names. Naming conventions vary\\nbetween different areas of biology. We have already seen how enzymes such as DNApolymerase are named for the reaction they catalyze. Many proteins have names that describetheir structure or their role in cells, such as hemoglobin and connexin. However, the pacewith which new proteins are being discovered at the moment means that many are not givenproper names but are simply referred to by their size: p38 (page 418) and p53 (page 415) haverelative molecular masses of about 38,000 and about 53,000, respectively. Clearly this couldcause confusion, so we sometimes add the name of the gene as a superscript: p16\\nINK4ais a\\nprotein of relative molecular mass (M r) of about 16,000 that is the product of the INK4a gene.\\nPOLYMERS OF AMINO ACIDS\\nTranslation produces linear polymers of α-amino acids. If there are fewer than around 50\\namino acids in a polymer, we tend to call it a peptide. More and it is a polypeptide. Proteins\\nare polypeptides, and most have dimensions of a few nanometers (nm), although structuralproteins like keratin in hair are much bigger. The relative molecular masses of proteins canrange from 5000 to hundreds of thousands.\\nThe Amino Acid Building Blocks\\nThe general structure of α-amino acids, the building blocks of polypeptides and proteins,\\nis shown in Figure 9.1. R is the side chain. It is the side chain that gives each amino acid itsunique properties.\\nDuring the process of translation (page 173), peptidyl transferase joins the amino group\\nof one amino acid to the carboxyl of the next to generate a peptide bond. A generalizedpolypeptide is shown in Figure 9.1. The backbone, a series of peptide bonds separated bytheαcarbons, is shown in green. At the left-hand end is a free amino group; this is known\\nas the N terminal oramino terminal. At the right-hand end is a free carboxyl group; this is\\nR1R2\\nNH3CHHH\\nNC\\nOCHR3\\nNCOCHHRn\\nNC H COO CONH3R\\nOOC CH\\nα - amino acid\\npolypeptide\\nFigure 9.1. α-Amino acids and the peptide bond.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3d7c97e8-fd71-40dc-9989-28de30356c5b', embedding=None, metadata={'page_label': '193', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\nPOLYMERS OF AMINO ACIDS 185\\nknown as the C terminal orcarboxy terminal. Peptides are normally written this way, with\\nthe C terminal to the right.\\nThe properties of individual polypeptides are conferred by the side chains of their\\nconstituent amino acids. Many different properties are important —size, electrical charge,\\nthe ability to participate in particular reactions —but the most important is the af ﬁnity of the\\nside chain for water. Side chains that interact strongly with water are hydrophilic. Those\\nthat do not are hydrophobic. We have already encountered the 20 amino acids coded for bythe genetic code (page 75), but we will now describe each in turn, beginning with the mosthydrophilic and ending with the most hydrophobic. Each amino acid has both a three-letterabbreviation and a one-letter code (Fig. 9.2). In the following section we refer to each aminoacid by its full name and give the three- and one-letter abbreviations. This will help you tofamiliarize yourself with the amino acid abbreviations, which are used in other sections ofthe book.\\nFigure 9.3 shows four amino acids with hydrophilic side chains. Aspartate (Asp, D) and\\nglutamate (Glu, E) have acidic side chains. At the pH of the cytoplasm, these side chainsbear negative charges, which interact strongly with water molecules. As they are usuallycharged, we generally name them as their ionized forms: aspartate and glutamate ratherthan aspartic acid and glutamic acid. A polypeptide made entirely of these amino acids isvery soluble in water. Lysine (Lys, K) and arginine (Arg, R) have basic side chains. At thepH of the cytoplasm, these side chains bear positive charges, which again interact stronglywith water molecules.\\nFigure 9.4 shows histidine (His, H), whose side chain is weakly basic with a p K\\naof\\n7.0. At neutral pH about half the histidine side chains, therefore, bear a positive charge. Thefact that histidine is equally balanced between protonated and unprotonated forms gives itimportant roles in enzyme catalysis. At neutral pH a polypeptide made entirely of histidinewill be very soluble in water.\\nFigure 9.4 also shows cysteine (Cys, C) whose thiol ( —SH) group is weakly acidic\\nwith a p K\\na(page 25) of about 8. At neutral pH most (about 90%) of cysteine side chains\\nwill have their hydrogen attached. Even so the charge on the remaining 10% means thata polypeptide made entirely of cysteine will be soluble in water. Cysteine ’s thiol group is\\nchemically reactive and has important roles in some enzyme-active sites. Two cysteine thiolgroups can form a disulﬁde bond (page 191).\\nFigure 9.5 shows the three amino acids serine (Ser, S), threonine (Thr, T), and tyrosine\\n(Tyr, Y). These amino acids have hydroxyl ( —OH) groups in their side chains that can\\nhydrogen bond with water molecules. A polypeptide composed of these amino acids issoluble in water.\\nAsparagine (Asn, N) and glutamine (Gln, Q) (Fig. 9.6) are the amides of aspartate and\\nglutamate. They are hydrophilic.\\nGlycine (Gly, G; Fig. 9.6) has nothing but a hydrogen atom for its side chain. It is\\nrelatively indifferent to its surroundings.\\nThe ﬁve amino acids shown in Figure 9.7 have side chains of carbon and hydrogen\\nonly. These are alanine (Ala, A), valine, (Val, V), leucine (Leu, L), isoleucine (Ile, I), andphenylalanine (Phe, F). The side chains cannot interact with water so they are hydrophobic.A polypeptide composed entirely of these amino acids does not dissolve in water but willdissolve in olive oil.\\nTryptophan (Trp, W; Fig. 9.8) is the largest of the amino acids. Its double ring side\\nchain is mainly hydrophobic. Methionine (Met, M; Fig. 9.8) is also hydrophobic: its sulfuratom is in the middle of the chain so cannot interact with water. Last comes proline (Pro, P;Fig. 9.9). Proline is not really an amino acid at all —it is an imino acid, but biologists give it', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='8557697c-fbea-4ac8-b14a-f7d1ba3beb45', embedding=None, metadata={'page_label': '194', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\n186 PROTEIN STRUCTURE\\nalanine (ala)       A\\ncysteine (cys)    C\\nhistidine (his)   H\\nmethionine (met)   M\\nthreonine (thr)   Tasparagine (asn)   N The genetic code and the corresponding amino acid side chains.\\nglutamine (gln)     Q\\nisoleucine (ile)   I\\nphenylalanine (phe)  F\\ntryptophan (trp)   Waspartate (asp)    D\\nglutamate (glu)   E\\nleucine (leu)   L\\nproline (pro)   P\\ntyrosine (tyr)   Yarginine (arg)    R\\nglycine (gly)    G\\nlysine (lys)     K\\nserine (ser)   S\\nvaline (val)   VCH2CH2\\nC\\nHNCH2\\nCH2HNCH3NH2\\nNH2CH2\\n(CH2)3NH2\\nHNH2\\nNHC\\nNH3\\n(CH2)4\\nOH\\nCH2O O\\nO–O \\nCCC\\nCH(CH2)2CH2SH\\nSCH3\\nCH HOCH3(CH2)2CH2\\nCH2\\nCH2\\nCH2OHCH H3CH3CCH3CH3\\nCHH3CC H3CCC\\nCH2O\\nCCC\\nSite for\\nattachmentof sugars\\nAbout 10% aredeprotonated and hencenegatively charged.Forms disulfide bonds\\nIntroduces a kinkin the polypeptidechain\\nCan be phosphorylated.Site for attachmentof sugarsThe largestside chainCan bephosphorylatedCan bephosphorylatedAbout 50%are protonated.pK is 7.0\\nCan be phosphorylated Positively chargedNegatively chargedCan be phosphorylatedThe smallestside chainNegativelycharged.\\nCan be\\nphosphorylated Positivelycharged\\nO\\nCCC\\n(CH2)2\\nCH2HC\\nN\\nHCCH HNGCU\\nGCCGCAGCGAAUAAGGAUGACCGUCGCCGACGGAGAAGG\\nUGUUGCCAACAGGAAGAGGGUGGCGGAGGG\\nCAUCACAUUAUCAUAUUAUUGCUUCUCCUACUGAAAAAG\\nAUG UUU\\nUUCCCUCCCCCACCGAGUAGCUCUUCCUCAUCG\\nACUACCACGACAUGG UAU\\nUAC\\nUGA UAA\\nUAGGUUGUCGUGGUA\\nSTOP STOP\\nFigure 9.2. The genetic code and the corresponding amino acid side chains, with special properties\\nnoted. Hydrophilic side chains are shown in green, hydrophobic side chains in black.\\nhonorary amino acid status. Because the side chain (which is hydrophobic) is connected to\\nthe nitrogen atom and therefore to the peptide bond, including proline in the polypeptidechain introduces a kink.\\nHow is it that amino acids with hydrophobic side chains are present in solution in the\\ncytosol and can be picked up by their respective tRNAs? The answer is simple: as freeamino acids, they all bear two charges, a positive charge on the amino group and a negative', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='7173160f-d651-4e97-a751-59bc8ea9ba0c', embedding=None, metadata={'page_label': '195', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\nCH NH3 OOCAspartate D Asp\\nGlutamate E Glu\\nLysine K Lys\\nArginine R ArgCH2COO\\nCHCH2\\nCH2\\nCHNH\\nCH2\\nCH2\\nCH2CNH2CHCH2\\nCH2\\nCH2\\nCH2NH3\\nNH3\\nNH3\\nNH2COO\\nOOC\\nOOC\\nOOC NH3\\nFigure 9.3. Amino acids with hydrophilic side chains.\\nCHCH2SH\\nCHCH2SHis H Histidine\\nCys C CysteineCHCH2\\nOOC NH3HpK = 7.0\\nNNNHNHNHN\\n+\\nH +pK = 8.0HCCH\\nC\\nCHCH2\\nNH3NNN\\nHHNHNHN\\nHCCH\\nC\\nOOC\\nNH3 OOC NH3 OOC\\nFigure 9.4. Histidine and cysteine.\\n187', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='de192b5d-be4d-4ac3-b27f-056378065df5', embedding=None, metadata={'page_label': '196', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\n188 PROTEIN STRUCTURE\\nCH NH3 OOCCH2OH\\nCH NH3 OOCCH2OHCH NH3 OOCCH OHSer S Serine\\nThr T Threonine\\nTyr Y TyrosineCH3\\nFigure 9.5. Amino acids with hydroxl groups in their side chains.\\ncharge on the carboxyl group. Like all small ions they are soluble in water. Formation of\\nthe peptide bond removes these charges. The term residue is used to refer to the amino acid\\nwhen it is part of a peptide chain.\\nThe Unique Properties of Each Amino Acid\\nAlthough we have classi ﬁed side chains on the basis of their af ﬁnity for water, their other\\nproperties are important (Fig. 9.2).\\nAsn N Asparagine\\nGln Q Glutamine\\nGly G Glycine\\nCH NH3 OOCHCH NH3 OOCNH2\\nCH2CO\\nCH NH3 OOCNH2\\nCH2CH2CO\\nFigure 9.6. Asparagine, glutamine, and glycine.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='8a8acf6e-cf82-48a5-91e7-bc6fe9bb6fcb', embedding=None, metadata={'page_label': '197', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\nPOLYMERS OF AMINO ACIDS 189\\nCH NH3 OOCCH2CH3\\nCH CH3\\nCH NH3 OOCCH2\\nCH CH3CH3CH NH3 OOCCH3\\nCH3\\nCH CH3\\nCH NH3 OOC\\nCH NH3 OOCCH2Ala A Alanine\\nVal V Valine\\nLeu L Leucine\\nPhe F PhenylalanineIIe I Isoleucine\\nFigure 9.7. Amino acids with simple hydrophobic side chains.\\nCH NH3 OOCCH2S\\nCH2CH3CHCH2\\nOOC NH3\\nNNHNTrp W Tryptophan\\nMet M Methionine\\nFigure 9.8. Amino acids with more complicated hydrophobic side chains.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2eef79cd-3b8c-4c80-89f7-b8c8d9ef1fa5', embedding=None, metadata={'page_label': '198', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\n190 PROTEIN STRUCTURE\\nCH2 CH2\\nC\\nHCH2\\nOOC\\nPro P ProlineNH2\\nFigure 9.9. Proline, an imino acid.\\nCHARGE. If a side chain is charged it will be hydrophilic. But charge has other\\neffects too. A positively charged residue such as a lysine will attract a negatively charged\\nresidue such as glutamate. If the two residues are buried deep within a folded polypeptide,where neither can interact with water, then it will be very dif ﬁcult to pull them apart. We\\ncall such an electrostatic bond inside a protein a salt bridge. Negatively charged residues\\nwill attract positively charged ions out of solution, so a pocket on the surface of a proteinlined with negatively charged residues will, if it is the right size, form a binding site for aparticular positively charged ion like sodium or calcium.\\nPOSTTRANSLATIONAL MODIFICATIONS. After a polypeptide has been synthe-\\nsized some of the amino acid side chains may be modi ﬁed. A good example of this is\\nglycosylation, in which chains of sugars may be added to asparagine or threonine sidechains (page 225). A very speci ﬁc posttranslational modi ﬁcation occurs in the connective\\ntissue protein collagen (page 13). After the polypeptide chain has been synthesized, speci ﬁc\\nprolines and lysines are hydroxylated to hydroxyproline and hydroxylysine. Because hy-droxyl groups can hydrogen bond with water, this helps the extracellular matrix to form ahydrated gel (page 13).\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 9.1 Hydroxyproline, Hydroxylysine, and Scurvy\\nScurvy, a very unpleasant disease in which blood vessels become leaky, is caused by a lack of vitamin\\nC in the diet. As far as we know, the only use of vitamin C in the body is to participate in the post-translational modi ﬁcation of proline and lysine to generate the hydroxyproline and hydroxylysine in\\ncollagen molecules. Without vitamin C collagen, necessary for the physical strength of blood vessels,fails to form properly. In 1747 James Lind demonstrated that citrus fruits (which we now know tobe rich in vitamin C) prevented scurvy, a ﬁnding that allowed explorers to make long-distance sea\\nvoyages that had previously been impossible.\\nNot all posttransational modi ﬁcations are so permanent. One very important modi-\\nﬁcation is phosphorylation, the attachment of a phosphate group. We have already seen\\nhow sugars can be phosphorylated (page 33), but so can the side chains of the six amino\\nacids serine, threonine, tyrosine, aspartate, glutamate, and histidine (Fig. 9.10). Usually thephosphate group comes from ATP, in which case the enzyme that does the job is called akinase. Phosphate groups carry two negative charges and can therefore markedly alter the\\nbalance of electrical forces within a protein. Another group of enzymes called phosphatases\\nremove the phosphate groups. Since phosphorylation is readily reversible, it is often usedto turn proteins off and on. For example, the transcription factor NFAT (page 222) is only', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4134dbc1-9d0e-4014-b157-3f0b2492556e', embedding=None, metadata={'page_label': '199', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\nOTHER AMINO ACIDS ARE FOUND IN NATURE 191\\nphosphoesters\\nphosphoanhydridesCH NH C\\nphosphoserineO PO\\nOO\\nCH2O\\nPOCHCHCH3\\nNH C\\nphosphothreonineO PO\\nOOO\\nCHCH2CO\\nNH C\\nphosphoaspartateOOPO\\nCHCH2CO\\nNH C\\nphosphoglutamate phosphohistidinephosphoimides\\nCH2\\nO OOCHCH2\\nNH C\\nphosphotyrosinePO\\nOO\\nOOO O\\nOPO\\nO N\\nOO\\nCHCH2\\nC NHCCCCHCHCHN\\nCH\\nFigure 9.10. Six side chains can be phosphorylated.\\nactive in the dephosphorylated state, while insulin receptor substrate number 1 (page 356)\\nis activated by phosphorylation.\\nDISULFIDE BONDING. Under oxidizing conditions the thiol ( —SH) groups of two\\ncysteine residues can link together to form a disulﬁde bond (ordisulﬁde bridge ) (Fig. 9.11).\\nProteins made for use outside the cell often have disul ﬁde bonds that confer additional\\nrigidity on the protein molecule. If all the peptide bonds in a polypeptide are hydrolyzed,any cysteines that were linked by disul ﬁde bonds remain connected in dimers called —very\\nconfusingly —cystine molecules.\\nOTHER AMINO ACIDS ARE FOUND IN NATURE\\nOrnithine and citrulline (page 293) are α-amino acids that play a vital role in the body —they\\nare used as part of the urea cycle to get rid of ammonium ions, which would otherwise poisonus (page 293). However, they are not used as building blocks in the synthesis of polypeptides.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='28fe8cfb-a140-4b5e-9472-710bc2c9a335', embedding=None, metadata={'page_label': '200', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\n192 PROTEIN STRUCTURE\\nH\\nOH C C CH2NH\\nCCH2SSNH\\nO C\\ndisulfide bond\\nbetween 2 cysteines\\nHHC C CH2NH3\\nCOOCH2SSNH3\\nCOO\\nthe double amino acid cystine\\nFigure 9.11. Oxidation of adjoining cysteine residues produces a disulﬁde bond. Proteolysis of the\\npolypeptide releases cystine.\\nThere are many other nonprotein amino acids. Very early in the evolution of life, the palette\\nof amino acids used to make polypeptides became ﬁxed at the 20 that we have described.\\nAlmost the entire substance of all living things on earth is either polypeptide, composedof these 20 monomers, or other molecules synthesized by enzymes that are themselvesproteins made of these 20 monomers. Natural selection has directed evolution within theconstraints imposed by the palette.\\nTHE THREE-DIMENSIONAL STRUCTURES OF PROTEINS\\nProteins are polypeptides folded into speci ﬁc, complex three-dimensional shapes. Gener-\\nally, hydrophobic amino acids pack into the interior of the protein while hydrophilic aminoacid side chains end up on the surface where they can interact with water. The amino acid sidechains pack together tightly in the interior: there are no spaces. It is the three-dimensionalstructure of a particular protein that allows it to carry out its role in the cell. The shape can befully de ﬁned by stating the position and orientation of each amino acid, and such knowledge\\nlets us produce representations of the protein (Fig. 9.12). However, when discussing proteinstructure, it is helpful to think in terms of a series of different levels of complexity.\\nTheprimary structure of a protein is the sequence of amino acids. Lysozyme is an\\nenzyme that attacks bacterial cell walls. It is found in secretions such as tears and in thewhite of eggs. Lysozyme has the following primary structure:\\n(NH\\n2)KVFGRCELAAAMKRHGLDNYRGYSLGNWVCAAKFESNFNTQATNRNTD\\nGSTDYGILQINSRWWCDNGRTPGSRNLCNIPCSALLSSDITASVNCAKKIVSDGDG\\nMNAWV AWRNRCKGTDVQAWIRGCRL(COOH).\\nNumbering is always from the amino terminal end where synthesis of the protein begins\\non the ribosome. Lysozyme has four disul ﬁde bonds between four pairs of cysteines. The\\n129 amino acids of hen egg white lysozyme are shown in linear order in Figure 9.12 a', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='dd1db97e-dae3-4ec9-9c72-3840396bb1ee', embedding=None, metadata={'page_label': '201', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\nTHE THREE-DIMENSIONAL STRUCTURES OF PROTEINS 193\\n(b)YRGYSGNW\\nV\\nC\\nA\\nA\\nE\\nS\\nN\\nF\\nN\\nT\\nQ\\nA\\nT\\nN\\nR\\nN\\nT\\nD\\nG\\nS\\nT\\nD\\nYGIINSRWWCDNGRTPGSRN\\nN\\nI\\nP\\nC\\nS\\nA\\nS\\nDITASVNCAKKIVSDGDGMNAWVAWRNRCKGTDVQAW\\nI\\nRGCR\\nCOO−\\nH3N+SS\\nSSS\\nSND\\nH\\nR\\nK\\nM\\nA\\nAA\\nE\\nC\\nR\\nG\\nFVK20\\n120\\n10\\n112930\\n40\\n50 6070\\n80\\n90100110SSSSSS\\nSSS\\nSS\\n(a)S\\n(c)8090100\\n4011020\\n30\\n10 12070\\n5060\\n(d)alpha helix\\nβ strands\\nLG\\nLLL\\nL\\nF\\nL\\nLNC\\nL\\nSLS\\nS\\nFigure 9.12. Lysozyme: ( a) linear map, ( b) space ﬁlling model, ( c) backbone representation, and\\n(d) cartoon.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='0810171c-9622-4601-a850-a7eee680a7ed', embedding=None, metadata={'page_label': '202', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\n(d)\\nα-helix top view(b)\\nα-helix showing the position\\nof a single hydrogen bond\\n(c)\\nα-helix with all\\nhydrogen bonds shown(a)\\nα-helix position of a\\nsingle hydrogen bondhydrogen bondC C N\\nHH\\nR1OC C N\\nHH\\nR2OC C N\\nHH\\nR3OC C N\\nHH\\nR4OC C N\\nHH\\nR5O\\nN\\nCCα\\nCN\\nCCαNCα\\nCαN\\nN\\nCαCCCN\\nCα\\nCN\\nCα\\nCα\\nN\\nCN\\nCα\\nC\\nOH\\nR\\nH\\nHH\\nOO\\nR\\nH\\nR\\nO H\\nO R\\nR\\nHH\\nH\\nOCαCαCαN\\nNCCα\\nC\\nCN\\nN\\nCα\\nN\\nCα\\nC\\nCN\\nN\\nCα\\nCN\\nCα\\nC\\nN\\nCα\\nCR\\nOOHOH\\nHRHHOR\\nHHRH\\nFigure 9.13. Theαhelix: ( a) the nitrogen atom in each peptide bond forms a hydrogen bond with\\nthe oxygen of the peptide bond four ahead. ( b)A nαhelix, with one hydrogen bond drawn. ( c)A n\\nαhelix, with all the hydrogen bonds drawn. ( d) An end-on view of an αhelix.\\n194', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='53060e3f-680f-4396-ad90-3d363ba6afdd', embedding=None, metadata={'page_label': '203', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\nTHE THREE-DIMENSIONAL STRUCTURES OF PROTEINS 195\\nwith the disul ﬁde bonds indicated. Lysozyme was the ﬁrst enzyme to have its three-\\ndimensional structure fully determined (in 1965). Figure 9.12 bshows that structure, with\\nall of the atoms that form the molecule displayed. We see little except an irregular surface.\\nHowever, if the amino acid side chains are stripped away and the path of the peptide-bondedbackbone drawn (Fig. 9.12 c), we see that some regions of the protein backbone are ordered\\nin a repeating pattern. Two types of protein backbone organization are common to many pro-teins. These are named the αhelix and the βsheet. Figure 9.12 dredraws the peptide back-\\nbone of lysozyme to emphasize these patterns, with the lengths of peptide participating in β\\nsheets represented as arrows. Collectively these repeating patterns are known as secondary\\nstructures. There are other regions of the protein that do not have any such ordered pattern.\\nIn anαhelix the polypeptide chain twists around in a spiral, each turn of the helix\\ntaking 3.6 amino acid residues. This allows the nitrogen atom in each peptide bond toform a hydrogen bond with the oxygen four residues ahead of it in the polypeptide chain(Fig. 9.13). All the peptide bonds in the helix are able to form such hydrogen bonds,producing a rod in which the amino acid side chains point outward. Because it introducesa kink into the polypeptide chain, proline cannot participate in an αhelix.\\nIn aβsheet lengths of polypeptide run alongside each other, and hydrogen bonds\\nform between the peptide bonds of the strands. This generates a sheet that has the sidechains protruding above and below it (Fig. 9.14). Along a single strand the side chainsalternate up then down, up then down. Because the actual geometry prevents them from beingcompletely ﬂat, they are sometimes called βpleated sheets. A polypeptide chain can form\\ntwo types of βsheet: Either all of the strands in the βsheet are running in the same direction\\n(Fig. 9.14 a) forming a parallel βsheet or they can alternate in direction (Fig. 9.14 b)\\nmaking an antiparallel βsheet. The polypeptide chains in βsheets are fully extended unlike\\nthe chain in an αhelix.\\nIn structural proteins like the keratin in hair or the ﬁbroin in silk, the whole polypeptide\\nchain is ordered into one of these secondary structures. Such ﬁbrous proteins have relatively\\nsimple repeating shapes and do not have binding sites for other molecules. Most proteinshave regions without secondary structures, the precise folding and packing of the aminoacids being unique to the protein, side by side with regions of secondary structure. Wehave seen that secondary structures such as the αhelix and βsheet are formed because\\nof hydrogen bonds involving the peptide bonds of the backbone. The three-dimensionalstructure is held together by various interactions most of which are individually weak butcollectively produce stable molecules. These interactions involve the amino acid side chainsthat interact with one another, with the backbone, with water molecules, and (if there isone) with a prosthetic group or cofactor (page 205). These interactions are hydrogen bonds,electrostatic interactions, van der Waals forces, hydrophobic interactions, and (in someproteins) disul ﬁde bonds.\\nHydrogen Bonds\\nIf a hydrogen attached to an oxygen, nitrogen, or sulfur by a covalent bond gets close toa second electron-grabbing atom, then that second atom also grabs a small share of theelectrons to form a hydrogen bond. The atom to which the hydrogen is covalently bondedis called the donor because it is losing some of its share of electrons; the other electron-grabbing atom is the acceptor. In order for a hydrogen bond to form, the donor and acceptormust be within a ﬁxed distance of one another (typically 0.3 nm) with the hydrogen on a\\nstraight line between them. In an αhelix the nitrogen atom within a peptide bond shares its', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='adb9b382-5eb4-41f3-bd04-595da429e76f', embedding=None, metadata={'page_label': '204', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\n196 PROTEIN STRUCTURE\\nf\\nIN DEPTH 9.1 Chirality and Amino Acids\\nA chiral structure is one in which the mirror image of the structure cannot be super-\\nimposed on the structure. The shape of your whole body is not chiral. Your mirrorimage could be rotated through 180\\n◦, so that it faces into the mirror, then step\\nback and be superimposed on you. Your right hand, however, is chiral: Its mirrorimage is not a right hand any more but a left hand. Molecules are often chiral —this\\narises if a carbon atom has four different groups attached to it and is thereforeasymmetric. The αcarbon of α-amino acids is asymmetric for all except glycine. It is\\npossible for there to be more than one asymmetric carbon in a molecule. If thereis an asymmetric carbon in a molecule, there will be two different ways that thegroups on that carbon can be arranged and so there will be two optical isomers.\\nThese isomers interact with polarized light differently. Although a different systemis used in chemistry, optical isomers of amino acids (and sugars) are denoted\\nLand\\nD.Lamino acids are exclusively used in proteins and predominate in the metabolism\\nof amino acids generally. However Damino acids are found in nature. Dalanine\\noccurs in bacterial cell walls while some antibiotics such as valinomycin and grami-cidin A contain\\nDamino acids. These molecules are synthesized in entirely different\\nways from proteins. Among sugars it is the Dforms that are mainly used by cells.\\nH\\nCH3COO\\nC\\nNH3COO\\nCH3H\\nC\\nNH3\\nL-alanine D-alanine\\nright handed\\nα helixleft handed\\nα helix\\nHelices are chiral too. The αhelix found in proteins is right handed, like a\\nregular screw thread. Re ﬂect a length of right-handed αhelix in a mirror, and you\\nwill get a left-handed αhelix composed of Damino acids. Because of the actual\\nstructures, it turns out that Lamino acids ﬁt nicely in a right-handed αhelix, while D\\namino acids ﬁt nicely in a left-handed αhelix. (If you doubt the chirality of helices,\\ntry making wire models of the two shown in the accompanying ﬁgure and see if\\nthey can be superimposed.)', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='cf94a693-6a7d-42c7-84cd-c3d8f319c941', embedding=None, metadata={'page_label': '205', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\nTHE THREE-DIMENSIONAL STRUCTURES OF PROTEINS 197\\nHH H O\\nHO R\\nHH\\nRR\\nHH\\nN C C N CC C N C C N\\nRH\\nR O OC C\\nHH H O\\nHO R\\nHH\\nRR\\nHCH\\nN C C N CC C N C C N\\nRH\\nR O OC\\nHH H O\\nHO R\\nHH\\nRR\\nHCH\\nN C C N CC C N C C N\\nRH\\nR O OC\\nNC\\nNC\\nNC\\nparallel β-sheets\\nantiparallel β-sheetsNC\\nCN\\nNCH\\nHO H\\nHO R\\nHH\\nRR\\nHCH\\nN C C NC C N C C N\\nRH\\nRC\\nO OC\\nH\\nHO H\\nHO R\\nHH\\nRR\\nHCH\\nN C C NC C N C C N\\nRH\\nRC\\nO O\\nH\\nHO H\\nHO R\\nHH\\nRR\\nHCH\\nN C C NC C N C C N\\nRH\\nRC\\nO OCCR\\nH(b)(a)\\nFigure 9.14. (a) Parallel and ( b) antiparallel βsheets.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d917b0cf-5767-418d-9fdc-dc76ccd62e60', embedding=None, metadata={'page_label': '206', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\n198 PROTEIN STRUCTURE\\nCN\\nATP\\nbindingsite cytosolin this region\\namino acid\\nside chains are\\nhydrophobicextracellular\\nmedium\\nFigure 9.15. Cartoon representation of the platelet derived growth factor receptor.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6d25911d-3753-4f30-b23b-e6b93015acf9', embedding=None, metadata={'page_label': '207', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\nTHE THREE-DIMENSIONAL STRUCTURES OF PROTEINS 199\\nhydrogen atom with the carbonyl group oxygen of the peptide bond four residues ahead of\\nit in the chain.\\nElectrostatic Interactions\\nIf positive and negatively charged amino acid residues are buried in the hydrophobic interiorof a protein, where neither can interact with water, then they will attract each other and theforce between them will be stronger than it would be in water. Such an electrostatic bondinside a protein is called a salt bridge.\\nPolar groups such as hydroxyl and amide groups are dipoles. They have an excess of\\nelectrons at one atom and a corresponding de ﬁciency at another, as we have already seen\\nfor the water molecule (page 27). The partial charges of dipoles will be attracted to otherdipoles and to fully charged groups.\\nvan der Waals Forces\\nThese are relatively weak close-range interactions between atoms. Imagine two atoms sittingclose together. At a given instant more of the electrons on one atom may be on one side, andthis exposes the positive charge on the nucleus. This positive charge attracts the electronsof the adjacent atom thus exposing its nuclear charge, which would attract the electronsof another atom and so on. The next instant the electrons will have moved so we have asituation of ﬂuctuating attractions between atoms. These forces are important in the close-\\npacked interiors of proteins and membranes and in the speci ﬁc binding of a ligand to its\\nbinding site.\\nHydrophobic Interactions\\nMolecules that do not interact with water (and that are therefore classed as hydrophobic)force the water around them to become more organized: hydrogen-bonded “cages ”form.\\nThis organization is thermodynamically undesirable and is minimized by the clusteringtogether of such molecules. This is called the hydrophobic effect. A polypeptide with\\nhydrophilic and hydrophobic residues will spontaneously adopt a con ﬁguration in which\\nthe hydrophobic residues are not exposed to water. They can achieve this either by sitting ina lipid bilayer (Fig. 9.15) or by adopting a globular shape in which the hydrophobic residuesare clustered in the center of the protein.\\nDisulﬁde Bonds\\nExtracellular proteins often have disul ﬁde bonds between speci ﬁc cysteine residues. These\\nare strong covalent bonds, and they tend to lock the molecule into its conformation. Althoughrelatively few proteins contain disul ﬁde bonds, those that do are more stable and are therefore\\neasy to purify and study. For this reason many of the ﬁrst proteins studied in detail, such as\\nthe digestive enzymes chymotrypsin and ribonuclease and the bacterial cell wall degradingenzyme lysozyme, have disul ﬁde bonds.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2570a4a3-899d-4793-8ab2-c9c65bd6bced', embedding=None, metadata={'page_label': '208', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\n200 PROTEIN STRUCTURE\\nTERTIARY STRUCTURE: DOMAINS AND MOTIFS\\nThe three-dimensional protein structure often has protrusions, clefts, or grooves on the\\nsurface where particular amino acids are positioned to form sites that bind ligands and,in the case of enzymes, catalyze reactions within or between ligands. The whole three-dimensional arrangement of the amino acids in the protein is called the tertiary structure.\\nA tertiary structure is unique to a particular protein. However, common patterns or\\nmotifs occur in tertiary structures. For example, many proteins with different functions\\nshow a β-barrel structure where βsheet is rolled up to form a tube. Green ﬂuorescent\\nprotein (page 150) is one example (Fig. 9.16). Often the tertiary structure is seen todivide into discrete regions. The calcium-binding protein calmodulin shows this clearly\\n(Fig. 9.17). Its single chain is organized into two domains, one shown in green, the other in\\nblack, joined by only one strand of the polypeptide chain. In calmodulin the two domainsare very similar, and the modern gene probably arose through duplication of an ancestralgene that was half as big. Domains are easier to see than to de ﬁne.“A separately folded\\nregion of a single polypeptide chain ”is as good a de ﬁnition as any.\\nDomains may be similar or different in both structure and function. The catabolite-\\nactivating protein (CAP) of Escherichia coli (page 114) binds to a speci ﬁc sequence of\\nbases on DNA, assisting RNA polymerase to bind to its promoter and initiate transcriptionof the lacoperon. It does this only when it has bound cAMP (which in turn only happens\\nwhen the intracellular glucose concentration is low). One of the domains of CAP has thejob of binding cAMP. Another recognizes DNA sequences using a helix-turn-helix motif\\n(Fig. 9.18). One of these helices ﬁts into the major groove of the DNA where it can make\\nspeci ﬁc interactions with the exposed edges of the bases.\\nProteins that interact with DNA often do so viazinc ﬁngers. The DNA binding domain\\nof the glucocorticoid receptor (page 121) contains two zinc ﬁnger motifs. Each ﬁnger is\\nformed by the coordination of four cysteine residues with a Zn\\n2+ion (Fig. 9.19 a). The\\ndomain contains two αhelices (Fig. 9.19 b), one in each ﬁnger. The ﬁrst helix, also known\\nas the recognition helix, ﬁts into the major groove of the DNA and makes contacts with\\nspeci ﬁc bases. The second helix is bent at right angles to the ﬁrst and helps to stabilize\\nthe receptor homodimer on DNA by promoting dimerization between the two receptormonomers. This ﬁnger can alternatively interact with other proteins that help to activate or\\nrepress transcription, such as AP1 subunits (page 124).\\nDomains usually correspond to exons (page 98) in the gene. It is therefore relatively\\neasy for evolution to create new proteins by mixing and matching domains from exist-ing proteins. The cAMP gated channel (page 350) arose in the ancestor of the verte-brates when a mutation occurred that spliced the cAMP-binding domain from cAMP-dependent protein kinase (page 353) onto the end of the voltage-gated sodium channel(page 330).\\nIntegral membrane proteins have special tertiary structures. The polypeptide chain\\nmay cross the membrane once or many times, but each time it does so it has a region ofhydrophobic side chains that can be in a hydrophobic environment such as the interior ofa lipid bilayer. The commonest membrane-spanning structure is a 22-amino-acid αhelix.\\nFigure 9.15 shows an integral membrane protein called the platelet-derived growth factorreceptor (page 353). Hydrophobic side chains on the transmembrane αhelix interact with\\nthe hydrophobic interior of the membrane. The protein is held tightly in the membranebecause for it to leave would expose these hydrophobic side chains to water.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='026cd93d-7ea4-4061-9c75-4f11f3294387', embedding=None, metadata={'page_label': '209', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\nFigure 9.16. Green ﬂuorescent protein molecule comprises a ßbarrel and a central αhelix.\\nCa++Ca++Ca++Ca++\\nCa++\\nCa++NH2\\nCOOH\\nFigure 9.17. Calmodulin is composed of two very similar domains.\\n201', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='0e85afdd-2bc5-439d-bd0e-ad81264e9917', embedding=None, metadata={'page_label': '210', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\n202 PROTEIN STRUCTURE\\ncyclic\\nAMPbindingdomain\\nDNAbindingdomain\\nActive CAPis a dimer oftwo identicalα helix-\\nturn-helixDNA-bindingproteins, one shownin green,one in gray.\\nFigure 9.18. Active catabolite activator protein is a dimer.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 9.2 Recognition by Binding\\nMany of the jobs proteins do depend on recognition and binding. Proteins vary greatly in the\\nspeci ﬁcity with which they bind other substances. Histones are at one extreme. About 25% of their\\namino acids are lysine or arginine, whose side chains are positively charged. Histones therefore bindto DNA —any DNA —through an electrostatic attraction to the negative charges on the phosphates\\nof the DNA backbone.\\nThe glucocorticoid receptor also binds DNA, but it is at the very other extreme of speci ﬁcity.\\nAfter binding glucocorticoid hormone, the hormone –receptor complex enters the nucleus and binds\\nto the base sequence 5\\n′AGAACA 3′. The protein does this using a zinc ﬁnger domain (page 200),\\nwhich inserts an αhelix into the major groove of DNA. The receptor works as a dimer so the\\nsequence 5′AGAACA 3′must appear twice in the DNA, separated by three nucleotides (page 203).\\nOnce bound the receptor binds other proteins that in turn stimulate transcription.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='c1fb7ac2-a9b2-44da-ab9a-fc0d4d8effc7', embedding=None, metadata={'page_label': '211', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\nTERTIARY STRUCTURE: DOMAINS AND MOTIFS 203\\nPRICVCGDAG H\\nT\\nC\\nE\\nHelix 1\\n(a)\\n(b)Helix 2G\\nC\\nKZn++\\nFFRRCCIK R\\nR\\nC\\nQ\\nA\\nC\\nR\\nKCGMMK\\nR\\nRZn++\\nZn++A\\n5′ 3′5′ 3′\\nT\\nG\\nAT\\nTA\\nT\\nN\\nNN\\nNN\\nT\\nGC\\nT\\nT A\\nGC\\nATAANAGCC\\nZn++ZnZn++++ZnZn++++\\nZn++Zn++\\nto rest of glucocorticoid\\nhormone receptor proteinto rest of glucocorticoid\\nhormone receptor protein\\nFigure 9.19. (a) Two zinc ﬁnger motifs in the glucocorticoid receptor. Each circle is one amino acid\\nresidue. Amino acid residues that are important in binding Zn2+or DNA are speci ﬁed. (b) Cartoon\\nrepresentation of the zinc ﬁnger domains of a dimerized pair of glucocorticoid hormone receptors\\ninteracting with DNA.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1ab38c15-8d63-4ef9-b3b4-e253e85647e4', embedding=None, metadata={'page_label': '212', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\n204 PROTEIN STRUCTURE\\nf\\nIN DEPTH 9.2 Hydropathy Plotting —The PDGF Receptor\\nOne of the central problems in structural biology is the prediction of the struc-\\nture of a protein from the primary structure or indeed from the DNA sequenceonce the gene has been found. Although we have come some way in under-standing protein folding, the problem remains unsolved. It is possible, however,to identify proteins that will have similar structures. One of the simpler thingsthat can be looked for is regions of hydrophobic amino acids: if these are 21 –22\\namino acids long, they are likely to be membrane spanning αhelices. The ﬁgure\\nshows a hydropathy plot for the platelet-derived growth factor (PDGF) receptor.\\nEach amino acid in the protein has a hydrophobicity allotted to it, so that ion-ized groups like those on aspartate and arginine get a big negative score whilegroups like those on phenylalanine and leucine get a big positive score. The plot isa running average of the hydrophobicity along the polypeptide chain. The pro-tein begins with a somewhat hydrophobic region: this is the signal sequence(page 215) that directs the growing protein to the endoplasmic reticulum. The rest\\n4.0\\n3.02.01.00.0\\n−1.0\\n−2.0\\n−3.0\\n−4.010\\nAmino acid numberHydrophobicity\\n1096\\nof the protein is neutral or somewhat hydrophilic except for a prominent short\\nhydrophobic region in the center. We can therefore predict that this protein willcross the membrane once at this location. Figure 9.15 shows the full predictedstructure of this protein.\\nQUATERNARY STRUCTURE: ASSEMBLIES\\nOF PROTEIN SUBUNITS\\nMany proteins associate to form multiple molecular structures that are held together tightly\\nbut not covalently by the same interactions that stabilize tertiary structures. For example,gap junction channels (page 55) are formed from six identical monomers. CAP is onlyactive when it dimerizes (Fig. 9.18). Hemoglobin, the protein that carries oxygen in ourred blood cells, is formed from four individual polypeptide chains, two αchains and two', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='48870b20-94ec-4535-80e6-dc50244168aa', embedding=None, metadata={'page_label': '213', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\nPROSTHETIC GROUPS 205\\nFigure 9.20. Hemoglobin, a tetrameric protein with heme prosthetic groups. (Illustration: Irving\\nGeis. Rights owned by Howard Hughes Medical Institute. Reproduction by permission only.)\\nβchains (Fig. 9.20). In all these cases, we call the three-dimensional arrangement of the\\nprotein subunits as they ﬁt together to form the multimolecular structure the quaternary\\nstructure of the protein.\\nPROSTHETIC GROUPS\\nEven with the enormous variety of structures available, there are some functions that proteins\\nneed help with because the 20 protein amino acid side chains do not cover the propertiesrequired. Moving electrons in oxidation and reduction reactions and binding oxygen aregood examples. Proteins therefore associate with other chemical species that have the re-quired chemical properties. Hemoglobin uses iron-containing heme groups to carry oxygenmolecules (Fig. 9.21). The general name for a nonprotein species that is tightly bound to aprotein and helps it perform its function is prosthetic group.\\nSome proteins have tightly bound metal ions that are essential to their function. We\\nhave already met the zinc ﬁngers of the glucocorticoid receptor and the iron in hemoglobin.\\nOther proteins use magnesium, manganese, or copper.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='341f941b-054b-4474-90e7-2f3b2fd949bb', embedding=None, metadata={'page_label': '214', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\n206 PROTEIN STRUCTURE\\nH2C\\nOOC COO CH2CH2\\nHNN\\nCH2\\nN\\nHC\\nCO\\nHCH2CH3CH\\nN\\nNN\\nNCH2CH3OO\\nCH2H2C\\nH3CCH\\nFe(II)\\nFigure 9.21. Iron-containing prosthetic group heme in the form in which it is found in oxygenated\\nhemoglobin. The oxygen molecule is at the top. (From D. Voet and J. D. Voet, Biochemistry, 2nd ed.,\\nNew York: Wiley, 1995, p. 216. c⃝1995 John Wiley & Sons, Inc. This material is used by permission\\nof John Wiley & Sons, Inc.)\\nTHE PRIMARY STRUCTURE CONTAINS ALL\\nTHE INFORMATION NECESSARY TO SPECIFYHIGHER-LEVEL STRUCTURES\\nProtein structures are stable and functional over a small range of environmental condi-\\ntions. Outside this range the pattern of interactions that stabilizes the tertiary structureis disrupted and the molecule denatures —activity disappears as the molecule loses its\\nstructure. Denaturation may be caused by many factors, which include excessive tem-perature, change of pH, and detergents. Concentrated solutions of urea (8 mol per liter)have long been used by biochemists to denature proteins. Unlike heat and pH, urea doesnot cause the protein to precipitate. Physicochemical techniques have shown that in ureasolution all of the higher levels of structure are lost and that polypeptide chains adopt ran-dom, changing conformations. Reagents such as urea that do this are called chaotropic\\nreagents. If the urea is removed (by dialysis or simply by dilution), the protein refolds,\\nregaining its structure and biological activity. This shows that the sequence of amino acidscontains all of the information necessary to specify the ﬁnal structure. The refolding of\\na urea-denatured protein cannot be random. Even a small protein with 100 amino acidswould take some 10\\n50years to try out all of the structural conformations available. The\\nfact that refolding does happen and happens on a time scale of seconds tells us that there', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='cade527e-9a73-436d-bf48-7289ac19000e', embedding=None, metadata={'page_label': '215', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\nTHE PRIMARY STRUCTURE 207\\nmust be a folding pathway, and the process is not random. Secondary structures may form\\nﬁrst and act as folding units. In the cell folding is assisted by proteins called chaperones\\n(page 221).\\n☛\\n✡✟\\n✠Medical\\nRelevance\\n9.1Protein Folding Gone Awry —Mad Cow Disease\\nSome years ago one of the dogmas of biology was overthrown —it was long be-\\nlieved that diseases could only be transmitted by structures that contained nucleic\\nacids, that is, by viruses or by microorganisms. It is now clear that a group of brain diseases can\\nbe transmitted by a protein. These are the diseases called spongiform encephalopathies: bovinespongiform encephalopathy in cows (mad cow disease), scrapie in sheep, and Creutzfeldt –Jacob\\ndisease and kuru in humans. These diseases are rare in humans but have recently increased in farmanimals.\\nThe infectious agent is called a prion and is a protein. It is coded for by a gene the animals\\nhave as part of their genome. In healthy individuals the gene is expressed in the cells of the nervoussystem and generates an innocuous protein called PrP\\nc(prion-related protein cellular). PrPchas a\\nglobular region at the C-terminal end but the N-terminal region seems to be unstructured.\\nSometimes the same polypeptide chain folds up differently with the disordered N-terminal\\nsection instead folding into a structure rich in βsheet. This is called PrPsc(prion-related protein\\nscrapie). The disease arises because PrPsccan cause normal PrPcprotein molecules to fold up into\\nthe abnormal form. This keeps happening and lumps of PrPscform that damage nerve cells. The\\nevidence is that it spread in cattle herds through food that contained recycled meat from infectedanimals. It is not yet known how one PrP\\nscmolecule triggers the aberrant folding of a normal PrPc\\nmolecule.\\nBefore the problem was fully comprehended, infected animals had been used for human\\nfood. There have been a number of cases in people that are thought to have come from eatingcontaminated beef —these are new variant Creutzfeldt –Jacob disease. It remains to be seen how\\nmany people will develop this fatal disease.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='5c9faea3-c891-497e-99c4-2aa9bb285d8b', embedding=None, metadata={'page_label': '216', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\n208 PROTEIN STRUCTURE\\nf\\nIN DEPTH 9.3 Curing Mad Mice with Smelly Fish\\nIf a cell is immersed in a solution more dilute than its own cytosol, it tends to\\nswell, as we have seen in the case of plant cells (page 53). On the other hand, cellsimmersed in concentrated solutions shrink. The overall strength of a solution is itsosmolarity, and the movement of water into or out of cells because of solution\\nstrength is called osmosis. Many ﬁsh are exposed to changes in the salt concentra-\\ntion of the water around them, either because they swim from salt water to freshand back (salmon, eels) or because they live in estuaries. The cells of these ﬁsh ad-\\njust the osmolarity of their cytosol to match the surrounding water by increasingthe concentration of various small, easily synthesized molecules, one of which isurea. However, this introduces a further problem because high concentrations ofurea are chaotropic —they cause cellular proteins to unfold.\\nO\\nTrimethylamine N-oxideCH3CH3CH3\\nN\\nThe small molecule trimethylamine N-oxide, in contrast, helps proteins to\\nfold into their correct shape. One of the most dramatic examples of this isthat trimethylamine N-oxide, added to mouse nerve cells in culture along with\\nthe scrapie agent PrP\\nsc, protects the endogenous PrPcprion protein from be-\\ning incorrectly refolded to generate PrPsc(page 207). Fish producing urea,\\ntherefore, also upregulate a gene encoding a ﬂavin-containing monooxygenase\\n(page 82) and produce trimethylamine N-oxide. A battle of opposing forces\\nthen ensues, with urea tending to denature the protein and trimethylamineN-oxide counteracting this to maintain the protein in its correctly folded state.\\nWhen ﬁsh die and rot, bacterial action converts trimethylamine N-oxide to\\ntrimethylamine, which gives rotting ﬁsh its distinctive smell (page 82).', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1810c7a0-0273-40d9-b8fd-88ce1b03c27f', embedding=None, metadata={'page_label': '217', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\nSUMMARY 209\\nSUMMARY\\n1.Polypeptides are linear polymers of α-amino acids linked by peptide bonds. There\\nare 20 amino acids coded for by the genetic code. They differ in the properties of\\ntheir side chains, which range from hydrophobic groups to charged and unchargedhydrophilic groups.\\n2.In addition to their hydrophilicity, side chains of individual amino acids have speci ﬁc\\nproperties of which the most important are charge, the ability to form disul ﬁde bonds,\\nand the ability to undergo posttranslational modi ﬁcation.\\n3.Phosphorylation (on S, T, Y , D, E, or H) is a posttranslational modi ﬁcation that\\nallows the balance of electrical charges on a protein to be dramatically and reversiblyaltered.\\n4.Proteins are polypeptides that have a complex three-dimensional shape.\\n5.It is convenient to consider protein structure as having three levels. The primarystructure is the linear sequence of the amino acid monomers.\\n6.In some parts of a protein regular, repeated foldings of the polypeptide chain canbe seen: these are secondary structures. Secondary structures are held together byhydrogen bonds between the carboxyl oxygens and the hydrogens on the nitrogensof the peptide bonds.\\n7.There are two common types of secondary structure. In the αhelix the chain coils\\nupon itself making a spiral with hydrogen bonds running parallel to the length of thespiral. In βsheets the hydrogen bonds are between extended strands of polypeptide\\nthat run alongside one another. The hydrogen bonds are at right angles to the strandsand in the plane of the sheet. There are two types —parallel and antiparallel.\\n8.Theﬁnal, complex folding of a protein is its tertiary structure. Interactions between\\nside chains stabilize the tertiary structure.\\n9.Some proteins have a quaternary structure. This is an association of subunits, eachof which has a tertiary structure.\\nFURTHER READING\\nBranden, C., and Tooze. J. 1999. Introduction to Protein Structure, 2nd ed. New York: Garland.\\nCreighton, T. 1993. Proteins, Structures and Molecular Properties, 2nd ed. New York: W. H. Freeman.\\nFersht, A. 1999. Structure and Mechanism in Protein Science. New York: W. H. Freeman.\\nMcGee, H. 1987. On Food and Cooking, London: Unwin.\\nPerutz, M. 1992. Protein Structure: New Approaches to Disease and Therapy. New York: W. H.\\nFreeman.\\nTanford, C., and Reynolds, J. 2001. Nature ’s Robots —A History of Proteins. Oxford: Oxford\\nUniversity Press.\\nV oet D., and V oet J. D. 2003. Biochemistry, 3rd ed. New York: Wiley.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='75370750-84f1-4032-a769-e15b70b9f431', embedding=None, metadata={'page_label': '218', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\n210 PROTEIN STRUCTURE\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.Peptide bonds result from\\nA. the reaction of amino groups with water.\\nB. the reaction of two carboxyl groups and removal of water.C. the reaction of an amino group with a carboxyl group and loss of water.D. reactions between amino acid side chains.E. transfer of a phosphate group from ATP to the amino acid.\\n2.The amino acids found in proteins are called\\nA.αamino acids.\\nB.ßamino acids.\\nC.γamino acids.\\nD.δamino acids.\\nE.πamino acids.\\n3.The side chains of the amino acids serine, asparagine, and glutamine are all\\nA. normally negatively charged at pH 7.B. normally positively charged at pH 7.C. able to interact with water.D. hydrophobic.E. indifferent to their environment.\\n4.The amino acids making up an αhelix\\nA. have side chains that point outward —away from the center of the helix.\\nB. have side chains that point inward —making up a hydrophobic core for the helix.\\nC. have side chains that do not interact with amino acids elsewhere in the protein.D. have side chains that are always hydrophobic.E. include proline.\\n5.Disul ﬁde bonds\\nA. form between two methionine residues.B. form between two cystine residues.C. form between two cysteine residues.D. are found in cytoplasmic proteins.E. attach microtubules to organelles.\\n6.Integral membrane proteins are helped to locate across the lipid bilayer by\\nA. glycosylation.B. formation of disul ﬁde bonds.\\nC. phosphorylation.D. using an αhelix made up of amino acids with hydrophilic side chains.\\nE. using an αhelix made up of amino acids with hydrophobic side chains.\\n7.Proteins cannot usually be denatured by\\nA. concentrated urea solutions.B. oxygen.C. strong acid.D. concentrated ethanol solutions.E. heat.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2ebf905f-f20a-4c54-bb3b-f0bc185550ca', embedding=None, metadata={'page_label': '219', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-09 WY001-Bolsover-v2.cls September 1, 2003 11:45\\nANSWERS TO REVIEW QUESTIONS 211\\nANSWERS TO REVIEW QUESTIONS\\n1.C.The amino group of one amino acid reacts with the carboxyl group of another with loss of water.\\n2.A.They are α-amino acids meaning that the amino group is on the same carbon as the carboxyl\\ngroup.\\n3.C.Serine, asparagine, and glutamine all have polar side chains that have no overall charge (so A\\nand B are wrong). These are able to hydrogen bond with water and so are hydrophilic.\\n4.A.The side chains point outward away from the center of the helix. Note that because proline\\nintroduces a kink into the polypeptide chain, it cannot participate in an αhelix, so E is wrong.\\n5.C.Disul ﬁde bonds are formed between two cysteine residues. Concerning answer B: cystine is\\nthe name given to the double amino acid found when proteins containing disul ﬁde bonds are\\nhydrolyzed. If we spoke of a cystine residue in a protein we would therefore mean two cysteines\\nthat were already joined by a disul ﬁde bond. Two such residues would not be able to form a further\\ndisul ﬁde bond.\\n6.E.Most proteins cross membranes by forming an αhelix made up of hydrophobic amino acids. The\\nside chains of amino acid residues in an αhelix point out, so the residues need to be hydrophobic\\nto interact with the hydrophobic interior of the bilayer. Such a helix is usually 22 amino acids long,which matches the thickness of a membrane bilayer. Formation of disul ﬁde bonds is unlikely to\\nhelp proteins adopt a transmembrane con ﬁguration, while glycosylation and phosphorylation both\\nmake proteins more hydrophilic, making a transmembrane con ﬁguration less likely.\\n7.B.Concentrated urea solutions, strong acid, concentrated ethanol solutions, and heat can all dena-\\nture proteins. In contrast, although oxygen may sometimes oxidize amino acids, such an oxidationdoes not normally result in loss of structure.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='90b1aee5-a284-4d37-bf52-367d0edee7ba', embedding=None, metadata={'page_label': '220', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\n10\\nINTRACELLULAR PROTEIN\\nTRAFFICKING\\nAn essential feature of all eukaryotic cells is their highly compartmentalized organization,\\nwhich allows the spatial and temporal separation of the synthesis, modiﬁcation, degradation,secretion, or take up of molecules. Each of the various organelles within cells is specializedfor one or more tasks and therefore needs a specialized set of proteins to carry out itsfunction. In this chapter we will examine how a protein, once synthesized on a ribosome,is precisely and actively moved ( translocated ) to the correct cellular compartment.\\nTHREE MODES OF INTRACELLULAR PROTEIN TRANSPORT\\nNewly synthesized proteins must be delivered to their appropriate site of function withinthe cell. The mechanisms and machinery that eukaryotic cells use to accomplish this arehighly conserved from yeast to humans. There are three possible ways by which the cellaccomplishes the task (Fig. 10.1). First, the protein may fold into its ﬁnal form as it is\\nsynthesized and then move through an aqueous medium to its ﬁnal destination, remainingfolded all the way. Delivery of proteins to the nucleus follows this scheme; the proteins aresynthesized on cytosolic ribosomes and pass through nuclear pores into the nucleoplasm bya process called gated transport. In the second form of transport, called transmembrane\\ntranslocation, unfolded polypeptide chains are threaded across one or more membranes to\\nreach their ﬁnal destination. Proteins destined for the interior of peroxisomes, mitochondria,and chloroplasts are synthesized on cytosolic ribosomes and may fold up completely or\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n213', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='bf196f4e-3c03-4d1f-8275-c994d694d2b5', embedding=None, metadata={'page_label': '221', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\n214 INTRACELLULAR PROTEIN TRAFFICKING\\nmitochondria\\n(and chloroplasts\\nin plants)\\nfree\\nribosomes\\nNucleusperoxisomes\\ngated\\ntransport\\nrough endoplasmic reticulum\\ncis face\\nGolgi apparatus\\ntrans face\\nlysosomevesicular\\ntrafficking\\nregulated\\nsecretion\\nexocytosisconstitutive\\nsecretion endosome\\nendocytosistransmembrane\\ntranslocation\\nFigure 10.1. The three modes of protein transport.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e0309efa-2563-4af8-abad-ffd0573f2e29', embedding=None, metadata={'page_label': '222', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\nTRANSPORT TO AND FROM THE NUCLEUS 215\\npartially into their ﬁnal form but are then unfolded in order to be transported across the\\nappropriate membrane. Proteins that are synthesized on the rough endoplasmic reticulum\\nare threaded across the membrane and into the interior of that organelle while they are beingsynthesized. In the third form of transport, small closed bags made of membrane, called\\nvesicles, carry newly synthesized protein from the endoplasmic reticulum to the Golgi\\napparatus and between other compartments in a process called vesicular trafﬁcking. The\\nfate of a protein —whether to remain in the cytosol or be sent along one of these alternative\\ntransport paths —is determined by sections of the protein itself that act as sorting signals.\\nWhen the protein is ﬁrst made on the ribosome, it is simply a stretch of polypeptide.\\nThe initial sorting decisions must therefore be made on the basis of particular amino acidsequences called targeting sequences. For proteins synthesized on the rough endoplasmic\\nreticulum, additional sorting signals such as sugars and phosphate groups can be added byenzymes that modify the chemical structure of the protein. In general, sorting operates byproteins containing a speci ﬁc sorting signal binding to a receptor protein, which in turn\\nbinds to translocation machinery situated in the membrane of the appropriate compartment.\\nTargeting Sequences\\nTargeting sequences (also known as localization sequences ) usually comprise a length of\\n3–80 amino acids that are recognized in the cytosol by speci ﬁc receptors that then guide the\\nprotein to the correct site and make contact with the appropriate translocation machinery.Once the protein has been imported into the new location the targeting sequence is oftenremoved by enzymes that break the peptide bond between the targeting sequence and therest of the protein. Some targeting sequences have been characterized better than others. Thetargeting sequence encoding import into the endoplasmic reticulum consists of about 5 –15\\nmostly hydrophobic amino acids at the N terminus of the protein called the signal sequence.\\nThe import signal for mitochondria is a stretch of 20 –80 amino acids in which positively\\ncharged side chains stick out on one side of the helix and hydrophobic side chains stick outon the other, a so-called amphipathic helix. A cluster of about ﬁve positively charged amino\\nacids located within the protein sequence targets a protein to the nucleus, while the bestknown peroxisomal targeting sequence is the C-terminal tripeptide Ser-Lys-Leu-COOH.\\nRetention\\nAnother class of sorting signal does not activate the transport of a protein out of its presentlocation but is rather a signal to the cell that the protein has reached its ﬁnal destination\\nand should not be moved. For example, proteins with the motif Lys-Asp-Glu-Leu-COOH(KDEL) at their C terminus are retained within the endoplasmic reticulum.\\nTRANSPORT TO AND FROM THE NUCLEUS\\nIn contrast to the situation in prokaryotes, RNA transcription and protein synthesis areseparated in space and time in eukaryotes. Exchange of material between the nucleusand the cytoplasm is essential for the basic functioning of these cells and must be tightlycontrolled. RNA and ribosomal subunits that are assembled in the nucleus have to enter thecytoplasm where they are required for protein synthesis. On the other hand, proteins such', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='357b7062-dda1-4b09-a5ac-932e28296f58', embedding=None, metadata={'page_label': '223', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\n216 INTRACELLULAR PROTEIN TRAFFICKING\\nas histones and transcription factors must enter the nucleus to carry out their functions.\\nThe nuclear pore complex mediates traf ﬁcking between the nucleus and the cytoplasm. The\\npore is made from a large number of proteins, and we refer to this type of structure as amultiprotein complex. Transport through the nuclear pore complex is mediated by signalsand requires both energy and transporter proteins.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 10.1 Holding Calcium Ions in the Endoplasmic Reticulum\\nOne of the functions of the smooth endoplasmic reticulum is to hold calcium ions ready for release\\ninto the cytosol when the cell is stimulated. A protein called calreticulin (short for calcium-bindingprotein of the endoplasmic reticulum) helps hold the calcium ions. Its primary structure is\\n(NH\\n2)MLL SVPLLL GLLGL AVA EPAVYFKEQ FLDGDG WTSRWIESKHKSD FGKFVL SSGK F\\nYGDEEKDKG LQTSQD ARFYALSASFEPFSNKGQT LVV QFTVKHEQN IDCGGGY VKLFP\\nNSLDQTD MHGDSEYN IMFGPDICGPGTKK VHVIFNYKGKN VLINKD IRCKDDE FTHLYT\\nLIVRPDNTYE VKIDNSQ VESGS LEDD WDFLPP KKIKDPDASKPEDWDER AKIDDPTDSK P\\nEDWDKPEHIPDPDAKKPEDWDEE MDGE WEPPVI QNPEYKGE WKPRQIDNPDYKGT WIH\\nPEIDNPEYS PDPSIYAYDN FGVLGLDLWQVKSGT IFDNFLITNDE AYAEEFGNET WGVTK\\nAAEKQ MKDKQDEEQR LKEEEEDKKRKEEEE AEDKEDDEDKDEDEEDEEDKEEDEEED\\nVPGQAKDE L(COOH)\\nTheﬁrst 17 amino acids include 14 hydrophobic ones, shown in black: this is the signal sequence\\nthat triggers translocation of the protein into the endoplasmic reticulum. The last 4 amino acids,\\nKDEL, ensure that it remains there. In between these two sorting signals is the functional core ofthe protein.\\nThe Nuclear Pore Complex\\nThe nuclear pore complex is embedded in the double membrane of the nuclear envelope\\n(Fig. 10.2). The basic structure of the nuclear pore complex, which has been mainly eluci-dated by electron microscopy, is very similar in all eukaryotic cells. It consists of over 50different components, called nucleoporins, which form eight identical subunits arranged ina circle. Seen from the side the nuclear pore complex comprises three rings. The inner ringsends out radial spokes into the center of the pore and includes transmembrane proteinsthat anchor the complex to the nuclear envelope. A central transporter, also called the plug,\\nis situated in the heart of each nuclear pore complex, forming the gate for macromoleculartrafﬁc.\\nGated Transport Through the Nuclear Pore\\nIn general, a protein has to display a distinct signal for it to be transported through the nuclearpore. Proteins with a nuclear localization signal are transported in, while proteins with anuclear export signal are transported out. Mobile transporter proteins, usually mediatingeither export or import, recognize the appropriate targeting sequence and then interactwith structural elements of the nuclear pore. This causes the pore to expand, allowing theactive transport of the transporter –protein complex. The precise mechanism by which large\\nmolecules move through the nuclear pore is not yet fully understood. However, the discoveryof the role played by the GTPase Ran in nuclear transport has given valuable insight into', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e1cccdc5-e7c3-43fb-b840-fda78a6c99a0', embedding=None, metadata={'page_label': '224', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\nTRANSPORT TO AND FROM THE NUCLEUS 217\\nribosome\\nnucleolus\\ncytosoleuchromatinOuter ring\\nintermembrane\\nspace\\nheterochromatin\\nnuclear pore\\ninner nuclear membrane\\nintermembrane spaceouter nuclear membranenuclear lamina\\nrough endoplasmic\\nreticuluminner ringPlug\\nnuclear poreSpokesPlug\\nnuclear\\nenvelopeouterring\\nSpokes\\nFigure 10.2. The nuclear pore.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='eab7c29f-52a8-4259-a215-7ca487bb0d15', embedding=None, metadata={'page_label': '225', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\n218 INTRACELLULAR PROTEIN TRAFFICKING\\nGDP\\nGTPaseGTP\\nGTPaseinactiveguanine nucleotide\\nexchange factor\\n(GEF)\\nGTPase activating protein\\n(GAP)\\nPiactive\\nGDP bound stateGTP GDP\\nGTP bound\\nstate\\nFigure 10.3. The GDP/GTP cycle of a GTPase. Pi represents an inorganic phosphate ion.\\nhow directionality of transport into and out of the nucleus is achieved. GTP hydrolysis by\\nRan provides the energy for transport. We will now describe this process.\\nGTPases and the GDP/GTP Cycle\\nGTPases form a family of proteins that are often involved when cells need to control complexprocesses. They all share the ability to hydrolyze the nucleotide GTP but otherwise differmarkedly in the processes they control and their mode of operation. We have already metone GTPase: the protein EF-tu, a component of the protein synthesis mechanism (page 172).Once a GTPase has hydrolyzed GTP to GDP it adopts an inactive shape and is unable toactivate its target process (Fig. 10.3). In contrast, if the protein expels the GDP and binds amolecule of GTP, it then adopts its active form. The cycle between the GDP-bound and GTP-bound state is regulated by effector proteins. GTPase activating proteins or GAPs speed upthe rate at which GTPases hydrolyze GTP, and hence the rate at which they inactivate, whileguanine nucleotide exchange factors, or GEFs, assist in the exchange of GDP for GTP andtherefore help GTPases adopt their active con ﬁguration.\\nGTPases in Nuclear Transport\\nIn the case of nuclear pore transport, the GEFs that operate on Ran are found in the nucleus,while Ran GAPs are cytosolic (Fig. 10.4). Thus nucleoplasmic Ran is predominantly in theGTP bound state (Ran:GTP), while most cytosolic Ran has GDP bound (Ran:GDP).\\nFigure 10.5 shows how Ran regulates import of proteins into the nucleus. An import\\ntransporter binds the nuclear localization sequence on the protein. As long as the transporterremains on the cytosolic side of the nuclear envelope, its cargo will remain bound. However,', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='5a9eb57f-a5ca-4175-a4c6-c1ddf2e9eb33', embedding=None, metadata={'page_label': '226', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\nTRANSPORT TO AND FROM THE NUCLEUS 219\\nGDP\\nRanRan-GEF\\nRan-GAPGDP\\ncytosolGTP\\nPinucleoplasm\\nGTP\\nRan\\nFigure 10.4. Ran GEF and GAP are localized to the nucleoplasm and cytosol, respectively. Pi rep-\\nresents an inorganic phosphate ion.\\nonce the transporter ﬁnds itself on the nucleoplasmic side, Ran in its active, GTP-bound\\nstate binds and causes the cargo to be released. Now, as long as the transporter remains\\non the nucleoplasmic side, it will have Ran:GTP bound and will be unable to bind cargo.However, once the transporter ﬁnds itself on the cytoplasmic side Ran-GAPs attached to\\nthe cytosolic face of the nuclear pore complex will cause Ran to hydrolyze its bound GTPto GDP and Ran will dissociate from the transporter, which is then able to bind morecargo.\\nThe same principle is used when a protein is to be exported from the nucleus (Fig. 10.6).\\nIn this case the export receptor can only bind proteins with an export sequence if they alsobind Ran:GTP. As long as the transporter remains on the nucleoplasmic side of the nuclearenvelope, its cargo will remain bound. However, once the transporter ﬁnds itself on the\\ncytoplasmic side Ran GAPs will cause Ran to hydrolyze its bound GTP to GDP and bothRan and the cargo will dissociate from the transporter, which is then unable to bind morecargo until it moves back into the nucleoplasm where Ran:GTP is available.\\nAlthough Ran can drive both nuclear import and nuclear export, in each case it is the\\nactive GTP-bound form of the protein that binds to the transporter. The GDP-bound formis inactive and cannot bind the transporter.\\nSome proteins move back and forth between the cytosol and the nucleus by succes-\\nsively revealing and masking nuclear localization sequences. For example, the glucocorti-coid receptor (page 121) only reveals its nuclear localization sequence when it has boundglucocorticoid.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='70c919a7-0f45-4a39-b9e9-8f9c1a7be3fb', embedding=None, metadata={'page_label': '227', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\n220 INTRACELLULAR PROTEIN TRAFFICKING\\nnuclear\\nimporttransporternuclearporenuclearlocalizationsequence\\ncargoprotein\\nimportedcargonucleoplasm\\ncytosolPiRan\\nGDPGTP\\nRanGTP\\nRan\\nGTP\\nRanRan-GAP\\nFigure 10.5. Nuclear import. Pi represents an inorganic phosphate ion.\\nnuclearexportsequencenuclear\\nexporttransporter\\nnuclear\\nporeGDP\\nGTP\\nGTP GTP PiGTP\\nexported\\ncargocytosolnucleoplasm\\nRan-GAP\\ncargo\\nprotein\\nFigure 10.6. Nuclear export. Pi represents an inorganic phosphate ion.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='09082261-d374-4d64-85ea-f5584923aa44', embedding=None, metadata={'page_label': '228', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\nTRANSPORT ACROSS MEMBRANES 221\\nTRANSPORT ACROSS MEMBRANES\\nTransport to Mitochondria\\nMitochondria have their own DNA and manufacture a small number of their own proteins.\\nHowever, the majority of mitochondrial proteins are coded for by nuclear genes. Theseare synthesized on free ribosomes and only imported into the mitochondrion posttrans-lationally. For example, proteins destined for the mitochondrial matrix carry a targetingsequence at their N terminus and are recognized by a receptor protein in the outer mem-brane. The mitochondrial receptor makes contact with a translocation complex, whichunfolds the protein and moves it across both outer and inner membrane simultaneously.Once the protein is translocated, the targeting sequence is cleaved off and the proteinrefolded.\\nChaperones and Protein Folding\\nA correctly addressed protein may fail to be targeted to an organelle if it folds too soon into itsﬁnal three-dimensional shape. For example, movement of proteins into the mitochondrial\\nmatrix requires that a protein must move through a channel through the outer and innermembranes. This channel is just wide enough to allow an unfolded polypeptide to passthrough. Our cells have proteins called chaperones, which as the name indicates, “look\\nafter”proteins. Chaperones use energy derived from the hydrolysis of ATP to keep newly\\nsynthesized proteins destined for the mitochondrial matrix in an unfolded state. As soon asthe protein moves through the channels and into the matrix, the matrix targeting sequence iscleaved. The protein now folds into its correct shape. Some small proteins can fold withouthelp. Larger proteins are helped to fold in the mitochondrial matrix by a chaperone proteincalled chaperonin, which provides a surface on which another protein can fold. Chaperoninitself does not change shape when helping another protein to fold.\\nCertain stresses that cells can experience, such as excessive heat, can cause proteins to\\ndenature (page 206). The cell responds by making proteins called heat-shock proteins inlarge amounts. The heat-shock proteins bind to misfolded proteins, usually to a hydrophobicregion exposed by denaturation, and help the protein to refold. Like chaperone proteins, theheat-shock proteins are not themselves changed, but instead form a platform on which thedenatured protein can refold itself. Heat-shock proteins are found in all cell compartmentsand also in bacteria.\\nTransport to Peroxisomes\\nMost organelles that are bound by a single membrane have their proteins made at therough endoplasmic reticulum and transported to them in vesicles (Fig. 10.1). Peroxi-somes are an exception: their proteins are synthesized on free- ﬂoating polyribosomes and\\nthen transported to their ﬁnal destination. Peroxisomal targeting sequences on the protein\\nbind to peroxisome import receptors in the cytosol. The complex of cargo and receptordocks onto the peroxisomal membrane and then crosses the membrane to enter the peroxi-some. Here, the protein cargo is released and the import receptor is shuttled back into thecytosol.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='7b4cc194-71d9-433f-b229-739c50776f8f', embedding=None, metadata={'page_label': '229', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\n222 INTRACELLULAR PROTEIN TRAFFICKING\\n✎\\n✍\\x0c\\n✌Medical\\nRelevance\\n10.1Blocking Calcineurin—How Immunosuppressants Work\\nThe drug cyclosporin A is invaluable in modern medicine because it suppresses the\\nimmune response that would otherwise cause the rejection of transplanted organs.\\nIt does this by blocking a critical stage in the activation of T lymphocytes, one of the cell typesin the immune system. T lymphocytes signal to other cells of the immune system by synthesizingand releasing the protein interleukin 2. Transcription of the interleukin 2 gene is activated by atranscription factor called NFAT.\\ncytoplasmplasmalemma\\nnucleus\\nRNA polymerase\\ntranscription[Ca++]\\nCa++Receptor\\nmRNA coding for \\ninterleukin 2foreign protein\\ninactive\\ncalcineurincalmodulin\\nphosphorylated NFAT dephosphorylated NFATnuclear localization\\nsequence\\nNFAT binding\\nsite in enhancer\\nregion of\\ninterleukin 2 geneactive\\nphosphatase\\ncyclosporinacts via Gq and IP3 (see Chapter 16)\\nP', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b970413c-d4a7-4f19-a10e-28462c6fbc11', embedding=None, metadata={'page_label': '230', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\nTRANSPORT ACROSS MEMBRANES 223\\n✞\\n✝✌\\n✆NFAT has a sorting signal that would normally direct it to the nucleus, but in unstimulated\\ncells this is masked by a phosphate group, so NFAT remains in the cytoplasm and interleukin 2 is\\nnot made. However, when foreign proteins, for example, any beginning with f-met (page 171),activate the T lymphocyte, they cause the concentration of calcium ions in the cytosol to increase(an increase of calcium concentration is a common feature of cell stimulation to be described inChapter 16). Calcium activates a phosphatase called calcineurin, which removes the phosphate\\ngroup from many substrates including NFAT. NFAT then moves to the nucleus and activatesinterleukin 2 transcription. The released interleukin 2 activates other immune system cells thatattack the foreign body. Cyclosporin blocks this process by inhibiting calcineurin, so that eventhough calcium rises in the cytoplasm of the T lymphocyte, NFAT remains phosphorylated anddoes not move to the nucleus.\\n✞\\n✝✌\\n✆Medical\\nRelevance\\n10.3How Protein Mistargeting Can Give You Kidney Stones\\nPrimary hyperoxaluria type 1 is a rare genetic disease in which calcium oxalate“stones ”accumulate in the kidney. Healthy people convert dietary glyoxylate (from\\nplants, see page 297) to the useful amino acid glycine by the enzyme alanine glyoxylate amino-transferase (AGT) (page 250). AGT is located in peroxisomes in liver cells. If glyoxylate cannotbe converted to glycine, it is instead oxidized to oxalate and excreted by the kidney, where it tendsto precipitate as hard lumps of calcium oxalate. Two thirds of patients with primary hyperoxaluriatype 1 have a mutant form of AGT that simply fails to work. However, the other third have anAGT with a single amino acid change (G170R) that works reasonably well, at least in the test tube.However, this amino acid change is enough to make the mitochondrial import system believe thatAGT is a mitochondrial protein and import it inappropriately, so that no AGT is available to betransported to the peroxisomes. For the clinician, the mistargeting of AGT in primary hyperox-aluria type 1 poses an unusual problem, namely, how to explain to a patient that the way to curetheir kidney stones is to have a liver transplant!\\nSynthesis on the Rough Endoplasmic Reticulum\\nSynthesis of proteins destined for import into the endoplasmic reticulum starts on free\\npolyribosomes. When the growing polypeptide chain is about 20 amino acids long, theendoplasmic reticulum signal sequence is recognized by a signal recognition particle that\\nis made up of a small RNA molecule and several proteins (Fig. 10.7). The signal recognitionparticle brings the ribosome to the endoplasmic reticulum membrane where it interacts witha speci ﬁc receptor —thesignal recognition particle receptor (or the docking protein ). This\\ninteraction directs the polypeptide chain to a protein translocator. Once this has occurred\\nthe signal recognition particle and its receptor are no longer required and are released.Protein synthesis now continues; and, as the polypeptide continues to grow, it threads itsway through the membrane via the protein translocator, which acts as a channel allowinghydrophilic stretches of polypeptide chain to cross. Once the polypeptide chain has enteredthe lumen of the endoplasmic reticulum, the signal sequences may be cleaved off by anenzyme called signal peptidase. Some proteins do not undergo this step but instead retaintheir signal sequences.\\nThe platelet-derived growth factor receptor is an example of an integral membrane\\nprotein. It contains a stretch of 22 hydrophobic amino acids that spans the plasma membrane(Fig. 9.15 on page 198). The ﬁrst part of the polypeptide to be synthesized is an endoplasmic\\nreticulum signal sequence, so the polypeptide begins to be threaded into the lumen of theendoplasmic reticulum. This section will become the extracellular domain of the receptor.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e65e322b-4ef2-4aa1-9d44-9bd8f81544e4', embedding=None, metadata={'page_label': '231', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\n224 INTRACELLULAR PROTEIN TRAFFICKING\\nER signal\\nsequence\\noligosaccharidegrowingpolypeptidechainsignal recognition particle\\nand its receptor dissociateendoplasmicreticulumlumen cytosoltranslocatorsignal recognitionparticle ER signal\\nsequenceribosome\\nmRNA1\\n2\\n3\\n4\\n56\\nfree\\npolypeptidesignal recognitionparticle receptor\\nFigure 10.7. Transport of a growing protein across the membrane of the endoplasmic reticulum.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='85b5675b-bc6c-4d74-8088-6598b44cdf90', embedding=None, metadata={'page_label': '232', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\nTRANSPORT ACROSS MEMBRANES 225\\nWhen the stretch of hydrophobic residues is synthesized, it is threaded into the translocator\\nin the normal way but cannot leave at the other end because the amino acid residues do notassociate with water. As synthesis continues, therefore, the newest length of polypeptidebulges into the cytosol. Once synthesis stops, this section is left as the cytosolic domain.\\nIf a protein contains more than one hydrophobic stretch, then synthesis of the second\\nstretch reinitiates translocation across the membrane, so that the protein ends up crossingthe membrane more than once.\\nWe can identify key amino acid sequences that play a role in protein targeting by making\\nuse of protein engineering. If we join the stretch of nucleotides that codes for the endoplasmicreticulum signal sequence to the cDNA that codes for a cytosolic protein, we produce achimeric DNA molecule (page 150). When this cDNA is transfected (page 150) into cells,it will be transcribed into mRNA and then translated into a protein that is unchanged exceptthat it now has, at its N terminus, the endoplasmic reticulum import sequence. The proteinis targeted to the endoplasmic reticulum. However, because the protein has no other sortingsignal, it is not recognized by any other receptor protein. It gets secreted from the cell bythe constitutive route. This result shows that constitutive exocytosis is the default route forproteins synthesized on the rough endoplasmic reticulum.\\nGlycosylation: The Endoplasmic Reticulum and Golgi System\\nMost polypeptides synthesized on the rough endoplasmic reticulum are glycosylated, thatis, they have sugar residues added to them, as soon as the growing polypeptide chain entersthe lumen of the endoplasmic reticulum. In a process called N-glycosylation a premadeoligosaccharide composed of two N-acetyl glucosamines, then nine mannoses, and then\\nthree glucoses is added to an asparagine residue by the enzyme oligosaccharide transferase.The three glucose residues are subsequently removed, marking the protein as ready forexport from the endoplasmic reticulum to the Golgi apparatus.\\nThe stacks of the Golgi apparatus (Fig. 10.8) have distinct polarity that re ﬂects the\\npassage of proteins through the organelle. The asymmetry of the stack is re ﬂected in the\\ntrans cisternacis face\\ntrans facetransfer\\nvesiclecis-\\nGolginetwork\\ntrans -\\nGolginetwork\\nsecetory\\nvesiclecis cisterna\\nFigure 10.8. The Golgi apparatus.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='dfb14b9c-bab2-4452-b350-9122b433a781', embedding=None, metadata={'page_label': '233', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\n226 INTRACELLULAR PROTEIN TRAFFICKING\\nmorphology of the membranes from which it is formed. The ciscisternae are made of\\nmembranes 5.5 nm thick, like those of the rough endoplasmic reticulum, while the trans\\ncisternae are 10 nm thick, like the plasma membrane. Each cisterna is characterized by\\na central ﬂattened region where the luminal space, as well as the gap between adjacent\\ncisternae, is uniform. The margin of each cisterna is often dilated and is often fenestrated(i.e., has holes through it) as well.\\nSmall, spherical vesicles are always found in association with the Golgi apparatus,\\nespecially with the edges of the cis face. These are referred to as transfer vesicles; someof them carry proteins from the rough endoplasmic reticulum to the Golgi stacks, otherstransfer proteins between the stacks, that is, from cisto middle and from middle to trans .A s\\nproteins move through the Golgi apparatus, the oligosaccharides already attached to themare modi ﬁed, and additional oligosaccharides can be added. As well as having important\\nfunctions once the protein has reached its ﬁnal destination, glycosylations play an important\\nrole in sorting decisions at the trans -Golgi network.\\nVESICULAR TRAFFICKING BETWEEN\\nINTRACELLULAR COMPARTMENTS\\nMost of the single-membrane organelles of the eukaryotic cell pass material between them-\\nselves by vesicular traf ﬁc, in vesicles that bud off from one compartment to fuse with\\nanother (Fig. 10.9). In this way the cargo proteins are never in contact with the cytosol.Two main directions of traf ﬁc can be identi ﬁed (Fig. 10.1). The exocytotic pathway runs\\nfrom the endoplasmic reticulum through the Golgi apparatus to the plasma membrane. Theendocytotic pathway runs from the plasma membrane to the lysosome. This is the route bywhich extracellular macromolecules can be taken up and processed. If vesicles are to bemoved over long distances, they are transported along cytoskeletal highways (page 389).\\nThe Principle of Fission and Fusion\\nFigure 10.9 illustrates how budding of a vesicle from one organelle, followed by fusion witha second membrane, can transport both soluble proteins and integral membrane proteinsto the new compartment. The process retains the “sidedness ”of the membrane and the\\ncompartment it encloses: the side of an integral membrane protein that faced the lumen oftheﬁrst compartment ends up facing the lumen of the second compartment, while soluble\\nproteins in the lumen of the ﬁrst compartment do not enter the cytosol but end up in the\\nlumen of the second compartment, or in the extracellular medium if the target membraneis the plasma membrane.f\\nIN DEPTH 10.1 Traf ﬁcking Movies\\nChimeric proteins that contain green ﬂuorescent protein (page 150) and appropri-\\nate sorting signals will be processed by the cells machinery just as proteins codedfor on its own genes. Thus protein trafﬁcking can be viewed by ﬂuorescence mi-croscopy in live cells that have been transfected with plasmids coding for suchchimeras. Movies illustrating many aspects of membrane transport are availableon the World Wide Web (see the CBASC website).', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d48319b6-a274-4861-9bdc-72178f4eaac9', embedding=None, metadata={'page_label': '234', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\norientation of\\nprotein is\\nconservedmembrane undergoes\\nshape change\\nLUMENproteinhydrophilichydrophobic\\nTransport\\nVesicletarget\\nmembrane\\nhydrophilic\\nFissionTransport\\nVesicleDocking Fusiondonor\\nmembrane\\nFigure 10.9. Fission and fusion.\\n227', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b2348be3-22f6-482c-bf7d-27904a572792', embedding=None, metadata={'page_label': '235', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\n228 INTRACELLULAR PROTEIN TRAFFICKING\\nGTP\\nGTPGTPGTP GTPGTP\\nGTP\\nGTPGTP\\nARFcoatamer(a)\\nligandreceptorclathrin\\nadaptor\\nproteinclathrin(b)\\nFigure 10.10. Generation of buds by ( a) coatamers and ( b) clathrin.\\nVesicle Formation\\nVesicle formation is the process during which cargo is captured and the lipid membrane is\\nshaped with the help of cytosolic proteins into a bud, which is then pinched off in a processcalled ﬁssion. The ordered assembly of cytosolic proteins into a coat over the surface of\\nthe newly forming vesicle is responsible for forcing the membrane into a curved shape(Fig. 10.10). There are two types of coats that serve this function: coatomer coats and\\nclathrin coats. The coat must be shed before fusion of the vesicle with its target membrane\\ncan occur.\\nCoatomer-Coated Vesicles\\nTransport along the default pathway uses coatomer-coated vesicles. This is the mechanismused in traf ﬁcking between the endoplasmic reticulum and Golgi, between the individual\\nGolgi stacks, and in budding of constitutive secretory vesicles from the trans Golgi. The\\ncoatomer coat consists of seven different proteins that assemble into a complex. The currentmodel for coatomer coat formation at the Golgi is that a guanine nucleotide exchange factorin the donor membrane exchanges GDP for GTP in the GTPase ARF. This causes ARF to\\nadopt its active con ﬁguration, in which a fatty acid tail is exposed that anchors ARF into the\\ndonor membrane (Fig. 10.10 a). Membrane-bound ARF is the initiation site for coatamer\\nassembly and coatomer-coated vesicle formation. The coat is only shed when the vesicle isdocking to its target membrane. An ARF-GAP in the target membrane causes the hydrolysisof GTP, and the resulting conformational change causes ARF to retract its hydrophobic tailand become cytosolic and therefore causes the coat to be shed.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='84aa279a-4384-433a-bc76-30b0cdde66f5', embedding=None, metadata={'page_label': '236', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\nVESICULAR TRAFFICKING BETWEEN INTRACELLULAR COMPARTMENTS 229\\nClathrin-Coated Vesicles\\nClathrin-coated vesicles mediate selective transport. They are, for example, the means by\\nwhich protein in the trans Golgi that bears mannose-6-phosphate is collected into a vesiclebound for the lysosome. Clathrin-coated vesicles carry proteins and lipids from the plasmamembrane to the endosome, and operate in other places where selective transport is required.\\nFigure 10.10 billustrates how clathrin generates a vesicle. The process starts when the\\ncargo of interest binds to integral proteins of the donor membrane that are selective receptorsfor that cargo. Clathrin adaptor proteins then bind to cargo-loaded receptors and begin\\nto associate, forming a complex. Lastly, clathrin molecules bind to this complex, formingthe coat and bending the membrane into the bud shape.\\nEven though clathrin can force the membrane into a bud shape, it cannot force the\\nbud to leave as an independent vesicle. One of the best-studied membrane ﬁssion events\\nis endocytosis. Here a GTPase called dynamin forms a ring around the neck of a budding\\nvesicle. GTP hydrolysis then causes a change in dynamin ’s shape that mechanically pinches\\nthe vesicle off from its membrane of origin. Unlike coatamer, clathrin coats dissociate assoon as the vesicle is formed, leaving the vesicle ready to fuse with the target membrane.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 10.2 Buds in a Test Tube\\nIf phospholipids are shaken up with aqueous medium, they spontaneously form arti ﬁcial vesicles\\ncalled liposomes. If these are then incubated with puri ﬁed coatomer proteins, ARF, and GTP, bud\\nformation and coated vesicles can be observed by electron microscopy. In contrast neither buds\\nnor vesicles form if liposomes are incubated with clathrin and clathrin adaptor proteins. Clathrin-mediated budding requires ligand to be present in the vesicles, and receptors for that ligand tobe present in the membrane of the vesicle. Furthermore even if buds did form, clathrin on its own,unlike coatamer, cannot cause ﬁssion. Clathrin-coated buds require another protein such as dynamin\\nto trigger ﬁssion.\\nThe Trans-Golgi Network and Protein Secretion\\nAt its trans face the Golgi apparatus breaks up into a complex system of tubes and sheets\\ncalled the trans -Golgi network (Fig. 10.8). Although there is some ﬁnal processing of\\nproteins in the trans -Golgi network, most of the proteins reaching this point have received\\nall the modi ﬁcations necessary to make them fully functional and to specify their ﬁnal\\ndestination. Rather, the trans -Golgi network is the place where proteins are sorted into the\\nappropriate vesicles and sent down one of three major pathways: constitutive secretion,\\nregulated secretion, or transport to lysosomes.\\nVesicles for constitutive or regulated secretion, though functionally different, look very\\nmuch alike and are directed to the cell surface (Fig. 10.1). When the vesicle membrane comes\\ninto contact with the plasma membrane, it fuses with it. At the point of fusion the membraneis broken through, the contents of the vesicle are expelled to the extracellular space, andthe vesicle membrane becomes a part of the plasma membrane. This process, by which thecontents of a vesicle are delivered to the plasma membrane and following membrane fusionand breakthrough are released to the outside, is called exocytosis.\\nThe difference between the constitutive and regulated pathways of secretion by exocy-\\ntosis is that the former is always “on”(vesicles containing secretory proteins are presented', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3c9ef7ec-fb76-4268-958d-37122853a2e4', embedding=None, metadata={'page_label': '237', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\n230 INTRACELLULAR PROTEIN TRAFFICKING\\nfor exocytosis continuously) while the regulated pathway is an intermittent one in which\\nthe vesicles containing the substance to be secreted accumulate in the cytoplasm until theyreceive a speci ﬁc signal, usually an increase in the concentration of calcium ions in the\\nsurrounding cytosol, whereupon exocytosis proceeds rapidly (page 343). After secretion,vesicular membrane proteins are retrieved from the plasma membrane by endocytosis and\\ntransported to the endosomes (Fig. 10.1). From the endosomes vesicles are targeted to the\\nlysosomes, back to the Golgi, or into the pool of regulated secretory vesicles. In a cell thatis not growing in size, the amount of membrane area added to the plasma membrane byexocytosis is balanced over a period of minutes by endocytosis of the same area of plasmamembrane.\\nNotice that in exocytosis the membrane of the vesicle becomes incorporated into the\\nplasma membrane; consequently the integral proteins and lipids of the vesicle membranebecome the integral proteins and lipids of the plasma membrane. This is the principal, if notthe only way, that integral proteins made on the rough endoplasmic reticulum are added tothe plasma membrane.\\nTargeting Proteins to the Lysosome\\nOne of the best-understood examples of sorting in the trans -Golgi network is lysosomal tar-\\ngeting (Fig. 10.11). Proteins that are destined for the lysosome are synthesized on the roughendoplasmic reticulum and therefore, like all proteins synthesized here, have a mannose-containing oligosaccharide added. Because they do not have an endoplasmic reticulumretention signal such as KDEL, they are transported to the Golgi apparatus. There proteins\\nGolgi apparatus\\nPP\\nP\\nPPtrans\\nfacemembrane and\\nreceptor decycling\\naddition of a phosphategroup to a mannose groupon the polypeptidemannose-6-phosphatereceptortransport\\nvesiclelysosomal protein\\nis releasedlow pHcisface(pre) lysosome\\nclathrincoat\\nFigure 10.11. Targeting of protein to the lysosome.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='754a0ef3-7061-4133-ad7b-9900ffd519af', embedding=None, metadata={'page_label': '238', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\nVESICULAR TRAFFICKING BETWEEN INTRACELLULAR COMPARTMENTS 231\\ndestined for the lysosome are modi ﬁed by phosphorylation of some of their mannose\\nresidues to form mannose-6-phosphate (Fig. 2.11 on page 32). Once the proteins reach\\nthe trans-Golgi network, speci ﬁc receptors for mannose-6-phosphate recognize this sort-\\ning signal and cause the proteins to be packaged into vesicles that are transported to thelysosome, where they fuse. In the low pH (5.0) environment of the lysosome, the lyso-somal protein can no longer bind to its receptor. The phosphate group is removed by aphosphatase. Vesicles containing the receptor bud off from the lysosome and deliver themannose-6-phosphate receptors back to the trans-Golgi network.\\nThe function of the lysosome is to degrade unwanted materials. To carry out this\\nfunction, an inactive or primary lysosome fuses with a vesicle containing the material to bedigested. This makes a secondary lysosome. The vesicles with which primary lysosomesfuse may be bringing materials in from outside the cell or they may be vesicles made bycondensing a membrane around worn out or unneeded organelles in the cells own cytoplasm.The latter are sometimes called autophagic vacuoles. Some materials may not be digestibleand remain in the lysosome for the lifetime of the cell. These small dense remnant lysosomesare called residual bodies.\\nFusion\\nMembrane fusion is the process by which a vesicle membrane incorporates its componentsinto the target membrane and releases its cargo into the lumen of the organelle or, in thecase of secretion, into the extracellular medium. Different steps in membrane fusion aredistinguished. First, the vesicle and the target membrane mutually identify each other. Then,proteins from both membranes interact with one another to form stable complexes and bringthe two membranes into close apposition, resulting in the docking of the vesicle to the targetmembrane. Finally, considerable energy needs to be supplied to force the membranes to fuse,since the low-energy organization —in which the hydrophobic tails of the phospholipids are\\nkept away from water while the hydrophilic head groups are in an aqueous medium —must\\nbe disrupted, even if only brie ﬂy, as the vesicle and target membranes distort and then\\nfuse.\\nEach type of vesicle must only dock with and fuse with the correct target membrane,\\notherwise the protein constituents of all the different organelles would become mixed witheach other and with the plasma membrane. Our understanding of the molecular processesleading to membrane fusion is only just beginning to take shape, but our current under-standing is that two types of proteins, called SNARES andRab family GTPases work\\ntogether to achieve this. SNARES located on the vesicles (v-SNARES) and on the targetmembranes (t-SNARES) interact to form a stable complex that holds the vesicle very closeto the target membrane (Fig. 10.12). Not all vSNARES can interact with all tSNARES, soSNARES provide a ﬁrst level of speci ﬁcity. So far, over 50 members of the Rab family\\nhave been identi ﬁed in mammalian cells, and each seems to be found at one particular site\\nwhere it regulates one speci ﬁc transport event, thus controlling which vesicle fuses with\\nwhich target. For example, the recycling of the mannose-6-phosphate receptor back fromthe lysosome to the trans-Golgi network (Fig. 10.11) requires Rab9, and yellow ﬂuorescent\\nprotein-Rab9 chimeras (page 150) locate to the returning vesicles. GTP hydrolysis by Rabsis thought to provide energy for membrane fusion.\\nAnother important factor in membrane fusion is the lipid composition at the fusion site.\\nIn particular phosphoinositides, a group of lipids that we will encounter again in Chapter 16(page 346), seem to play a crucial role.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='edd4a81b-8b4e-4f16-ac63-f11a1ffb751b', embedding=None, metadata={'page_label': '239', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\n232 INTRACELLULAR PROTEIN TRAFFICKING\\nv-SNARE\\nt-SNAREScomplex formation\\nvery stablefusion and dissociation\\nFigure 10.12. SNAREs and vesicle fusion.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 10.3 SNARES, Food Poisoning, and Face-Lifts\\nBotulism, food poisoning caused by a toxin released from the anaerobic bacterium Clostridium\\nbotulinum, is fortunately rare. Botulinum toxin comprises a number of enzymes that speci ﬁcally\\ndestroy those SNARE proteins required for regulated exocytosis in nerve cells. Without these proteins\\nregulated exocytosis cannot occur, so the nerve cells cannot tell muscle cells to contract. This causesparalysis: most critically, paralysis of the muscles that drive breathing. Death in victims of botulismresults from respiratory failure.\\nLow concentrations of botulinum toxin (or “BoTox ”) can be injected close to muscles to\\nparalyze them. For example, in a “chemical facelift, ”botulinum toxin is used to paralyze facial\\nmuscles, producing an effect variously described as “youthful ”and“zombie-like. ”\\nSUMMARY\\n1.The basic mechanisms of intracellular protein traf ﬁcking are similar in all eukaryotic\\ncells, from yeast cells to human nerve cells.\\n2.Theﬁnal destination of a protein is de ﬁned by sorting signals that are recognized by\\nspeci ﬁc receptors. The polypeptide chain itself contains targeting sequences while\\nglycosylation and phosphorylation can add additional sorting signals.\\n3.Some sorting signals activate translocation of a protein to a new location, while\\nothers such as the endoplasmic reticulum retention signal KDEL cause the proteinto be retained at its present location.\\n4.Nuclear proteins are synthesized on free ribosomes and carried through the nuclearpore by Ran-mediated gated transport. Other proteins with a nuclear export signalare carried the other way, again by a Ran-mediated process.\\n5.Peroxisomal proteins, together with the majority of mitochondrial and chloroplastproteins that are not coded for by mitochondrial or chloroplast genes, are synthe-sized on free ribosomes and then transported across the membrane(s) of the targetorganelle.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='9363e721-448e-4db1-8ab7-e6c9571be16f', embedding=None, metadata={'page_label': '240', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\nREVIEW QUESTIONS 233\\n6.Proteins with an endoplasmic reticulum signal sequence are synthesized on the rough\\nendoplasmic reticulum. The growing polypeptide chain is fed across the membraneas it is synthesized. The signal sequence may then be cleaved off.\\n7.The default pathway for proteins synthesized on the rough endoplasmic reticulum isto pass through the Golgi apparatus and be secreted from the cell via the constitutivepathway.\\n8.Glycosylation in the endoplasmic reticulum and then in the Golgi apparatus hastwo functions: to produce the ﬁnal, functional form of the protein and to add further\\nsorting signals such as the lysosomal sorting signal mannose-6-phosphate.\\n9.Vesicles shuttle between most of the single-membrane cellular organelles, the ex-ception being peroxisomes. Budding and then ﬁssion of vesicles from the donor\\nmembrane can be driven by coatamer family proteins or by clathrin plus dynamin.\\n10.A tight association of vesicular and target SNARE proteins forces vesicles into closeassociation with the target membrane. Fusion requires the action of a Rab familyGTPase.\\nFURTHER READING\\nHammer III, J. A., and Wu, X.S. 2002. Rabs grab motors: De ﬁning the connections between Rab\\nGTPases and motor proteins. Curr. Opin. Cell Biol. 14: 69 –75\\nJahn, R., and S ¨udhof, T. C. 1999. Membrane fusion and exocytosis. Annu. Rev. Biochem . 68: 863 –893.\\nLin, R. C., and Scheller, R. H. 2000. Mechanisms of synaptic vesicle exocytosis. Annu. Rev. Cell Dev.\\nBiol. 16: 19 –49.\\nSegev, N. 2001. Ypt and Rab GTPases: Insight into functions through novel interactions. Curr. Opin.\\nCell Biol. 13: 500 –511.\\nZerial, M., and McBride, H. 2001. Rab proteins as membrane organizers. Nat. Rev. Mol. Cell Biol .2 :\\n107–117.\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.Cytosolic ribosomes synthesize proteins that will remain free in the cytosol plus those\\ndestined for the interior of\\nA. peroxisomes, endoplasmic reticulum, and the Golgi apparatus.\\nB. peroxisomes, mitochondria, chloroplasts, and lysosomes.C. peroxisomes, mitochondria, chloroplasts, and the nucleus.D. mitochondria, chloroplasts, lysosomes, and the nucleusE. the Golgi apparatus, lysosomes, and the nucleus.\\n2.The motif KDEL at the C terminus of a protein marks it for\\nA. import into the endoplasmic reticulum and subsequent translocation to the Golgi apparatus.B. import into the endoplasmic reticulum and subsequent translocation to the lysosome.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='bb765678-0133-44b1-b045-ad097a7593e5', embedding=None, metadata={'page_label': '241', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\n234 INTRACELLULAR PROTEIN TRAFFICKING\\nC. import into mitochondria and subsequent translocation to the mitochondrial intermembrane\\nspace.\\nD. retention within the endoplasmic reticulum.\\nE. retention within the trans-Golgi network.\\n3.Addition of the appropriate GAP to a solution containing a GTPase, GTP, and GDP only\\nwill\\nA. increase the fraction of the GTPase that is in the GDP-loaded state.\\nB. increase the fraction of the GTPase that is in the GTP-loaded state.C. cause an active cycling process in which the GTPase binds GTP, hydrolyses it to GDP, releases\\nthe GDP, and binds another GTP.\\nD. cause an active cycling process in which the GTPase binds GDP, hydrolyses it to GTP, releases\\nthe GTP, and binds another GDP.\\nE. have no detectable effect.\\n4.Glycosylation of proteins occurs in the\\nA. peroxisome.B. mitochondrion.C. lysosome.D. endoplasmic reticulum.E. all these organelles.\\n5.The lysosomal sorting signal is\\nA.N-acetyl-glucosamine.\\nB. asparagine.C. Ran:GTP.D. ribose-6-phosphate.E. mannose-6-phosphate.\\n6.Endocytotic vesicles can only form in the presence of\\nA. receptors speci ﬁc for a ligand that is present in the extracellular medium.\\nB. adaptor proteins.C. clathrin.D. dynamin.E. all of the above.\\n7.Ribosomes that are located on the rough endoplasmic reticulum are there by virtue of\\nA. an interaction between a signal recognition particle, a hydrophobic sequence at the N terminus\\nof the protein being synthesized by the ribosome, and a receptor in the membrane of theendoplasmic reticulum.\\nB. a KDEL sequence at the C terminus of the 60S subunit.C. an interaction between the sequence CCUCC on the 16S subunit and a receptor in the membrane\\nof the endoplasmic reticulum.\\nD. mannose-6-phosphate at the 5\\n′terminus of the 40S subunit.\\nE. a hydrophobic stretch of RNA at the 5′terminus of the 40S subunit.\\nANSWERS TO REVIEW QUESTIONS\\n1.C.See Figure 10.1.\\n2.D.The KDEL motif does not activate a translocation process; rather it holds proteins bearing the\\nsequence in the endoplasmic reticulum if they have already been translocated there.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6d0da327-a3b7-4147-bca0-67a27e246e75', embedding=None, metadata={'page_label': '242', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-10 WY001-Bolsover-v2.cls October 23, 2003 15:21\\nANSWERS TO REVIEW QUESTIONS 235\\n3.A.GAP means GTPase activator protein, so the GTPase will hydrolyze GTP to GDP more rapidly\\nonce the GAP has been added. Concerning other answers: (B) A GEF, a guanine nucleotide\\nexchange factor, would do this. (C) This won ’t happen because there is no GEF present to cause\\nthe GTPase to release its GDP and then bind a GTP. (D) This is nonsense —hydrolysis of GTP\\nyields GDP, not the other way around.\\n4.D.Glycosylation occurs in the endoplasmic reticulum and in the Golgi apparatus.\\n5.E.The lysosomal sorting signal is mannose-6-phosphate. Concerning other answers: (B) It is true\\nthat the oligosaccharide that contains the mannose-6-phosphate signal is attached to an asparagineresidue, but asparagine by itself is not a lysosomal targeting signal. (C) Ran is concerned withnuclear import/export. (D) Ribose is a pentose sugar so it does not have a number 6 carbon!\\n6.E.All of these are required for endocytotic vesicles to form. Receptors speci ﬁc for a ligand that is\\npresent in the extracellular medium, adaptor proteins, and clathrin are required for the generationof a clathrin-coated invagination of the plasma membrane; dynamin is then required to pinch theinvagination off to make an endocytotic vesicle.\\n7.A.All the other answers are more or less nonsensical. In particular, it is important to note that\\nribosomes are found at the endoplasmic reticulum by virtue of a signal sequence on the proteinthat they are synthesizing, notbecause the ribosome itself is any different from those that are free\\nin the cytosol, as was implied by answers B through E.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3979c35d-073c-4f49-885c-43da86c90cb9', embedding=None, metadata={'page_label': '243', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\n11\\nHOW PROTEINS WORK\\nThe three-dimensional structures of proteins generate binding sites for other molecules.\\nThis reversible binding is central to most of the biological roles of proteins, whether theprotein is a gap junction channel that binds a similar channel on another cell (page 55) ora transcription factor that binds to DNA. One special class of proteins, enzymes, have sitesthat not only bind another molecule but then catalyze a chemical reaction involving thatmolecule.\\nHow Proteins Bind Other Molecules\\nProteins can bind other protein molecules, DNA or RNA, polysaccharides, lipids, and avery large number of other small molecules and inorganic ions and can even bind dissolvedgases such as oxygen, nitrogen, and nitric oxide. Binding sites are usually very speciﬁc fora particular ligand, although the degree of speciﬁcity can vary widely. Usually the bindingis reversible so that there is an equilibrium between the free and bound ligand.\\nA binding site is usually a cleft or pocket in the surface of the protein molecule, which\\nis made up of amino acid side chains appropriately positioned to make speciﬁc interactionswith the ligand. All of the forces that stabilize tertiary structures of proteins are also usedin ligand–protein interaction: Hydrogen bonds, electrostatic interactions, the hydrophobiceffect, and van der Waals forces all have their roles. Even covalent bonds may be formedin a few cases—some enzymes form a transient covalent bond with the substrate as part ofthe mechanism used to effect the reaction.\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n237', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='dc886cb3-1776-4d62-939b-8e8e4f94c73f', embedding=None, metadata={'page_label': '244', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\n238 HOW PROTEINS WORK\\nglucose\\nglucose\\nFigure 11.1. The glucose carrier switches easily between two shapes.\\nDynamic Protein Structures\\nIt is easy to get the impression that protein structures are ﬁxed and immobile. In fact proteins\\nare always ﬂexing and changing their structure slightly around their lowest energy state.\\nA good term for this is “breathing. ”Many proteins have two low-energy states in which\\nthey spend most of their time, like a sleeper who, though twisting and turning throughout\\nthe night, nevertheless spends most time lying on their back or side. An example is theglucose carrier (Fig. 11.1). This is a transmembrane protein that forms a tube through the\\nmembrane. It is stable in one of two con ﬁgurations. In one the tube is open to the cytosol;\\nin the other the tube is open to the extracellular medium. By switching between the twostates, the glucose carrier carries glucose into and out of the cell.\\nAllosteric Effects\\nThe glucose carrier is able to bind a ligand —the glucose molecule —in either of its low-\\nenergy conformations. In contrast, the lacrepressor (page 113) can only bind its ligand, the\\noperator region of the lacoperon, in one conformation. On its own the protein predominantly\\nadopts this conformation so transcription is prevented as it binds to the DNA. When thelacrepressor binds allolactose (a signal that lactose is abundant), it is locked into a second,\\ninactive form that cannot bind to the DNA (Fig. 6.8 on page 113). Transcription is no longerrepressed, although the cAMP –CAP complex is additionally required if transcription is to\\nproceed at a high rate. This type of interaction, in which the binding of a ligand at oneplace affects the ability of the protein to bind another ligand at another location, is calledallosteric and is usually a property of proteins with a quaternary structure (i.e., with multiple\\nsubunits).\\nHemoglobin (Fig. 9.20 on page 205) is an example of a protein where allosteric effects\\nplay an important role. Each heme prosthetic group, one on each of the four subunits, canbind an oxygen molecule. We can get an idea of what one subunit on its own can do by lookingat myoglobin (Fig. 11.2 a), a related molecule that moves oxygen within the cytoplasm.\\nMyoglobin has just one polypeptide chain and one heme. The green line in Figure 11.2 b\\nshows the oxygen-binding curve for myoglobin. Starting from zero oxygen, the ﬁrst small\\nincrease in oxygen concentration produces a large amount of binding to myoglobin; the', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='923c8852-7c4c-4400-9dde-fc63084b29db', embedding=None, metadata={'page_label': '245', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\nO2 pressure in pascal2000 0020406080100(a)\\n(b)% of binding\\nsites occupied\\n4000 6000hemoglobinmyoglobin\\nFigure 11.2. (a) The monomeric oxygen-carrying protein myoglobin. (Illustration: Irving Geis.\\nRights owned by Howard Hughes Medical Institute. Reproduction by permission only.) ( b) Oxy-\\ngen binding of myoglobin (in green) and hemoglobin (in black) as oxygen pressure increases.\\n239', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='46c3bfad-6696-4108-bbe5-a2036fa53ce3', embedding=None, metadata={'page_label': '246', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\n240 HOW PROTEINS WORK\\nnext increase in oxygen produces a slightly smaller amount of binding, and so on, until\\nmyoglobin is fully loaded with oxygen. A curve of this shape is known as hyperbolic. Theblack line in Figure 11.2 bshows the oxygen-binding curve for hemoglobin. Starting from\\nzero oxygen, the ﬁrst small increase in oxygen concentration produces hardly any binding\\nto hemoglobin. The next increase in oxygen produces much more binding so the curve getssteeper before leveling off again as the hemoglobin becomes fully loaded. This behavior iscalled cooperative. The explanation for this behavior is that the hemoglobin subunits canexist in one of two states, only one of which has a high af ﬁnity for oxygen. The way that\\nthe four subunits ﬁt together means that they all must be in one form or the other. When\\noxygen concentration is low, most of the hemoglobin molecules have their subunits in thelow-af ﬁnity form. As oxygen increases, it begins to bind to the hemoglobin —little at ﬁrst\\nas most of the hemoglobin is in the low-af ﬁnity form and only a little in the high-af ﬁnity\\nform. As oxygen binds, more molecules switch to the high-af ﬁnity form as the low- and\\nhigh-af ﬁnity forms are in equilibrium. Eventually virtually all of the molecules have made\\nthe switch to the high-af ﬁnity form. This produces the curve shown in Figure 11.2 b. This\\ncooperative oxygen binding makes hemoglobin an effective transporter as it will load upwith oxygen in the lungs but will release it readily in the capillaries of the tissues wherethe oxygen concentration is low. Myoglobin would release little of its bound oxygen at theoxygen concentrations typical of respiring tissues.\\nSome enzymes show cooperative behavior caused by an allosteric effect that causes\\nbinding of one substrate molecule to make it easier for the other substrate to bind. Thedegree of cooperativity can be altered by the binding of other molecules (called effectors)that act to switch the enzyme on or off.\\nChemical Changes That Shift the Preferred Shape of a Protein\\nProteins can change conformation as a result of environmental changes, by binding a ligandor by having a particular group attached covalently to them. Anything that changes thepattern of electrostatic interactions within a protein will alter the relative energy of itsstates. If, for example, a protein contains histidine residues, merely changing the pH will dothis. In solutions with pH greater than 7 most of the histidine residues in a protein will be\\nhistidine\\nresidues\\npH8\\npH6++++\\nFigure 11.3. A pH change that alters the charge on histidine will alter the balance of forces within\\na protein. At pH 6, the structure on the right will predominate.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='0f9cec7e-7bcd-4da2-abdf-2183c7d4927a', embedding=None, metadata={'page_label': '247', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\nENZYMES ARE PROTEIN CATALYSTS 241\\naspartate\\nresidueATP held\\nnon-covalently\\nin creviceResting statephosphorylated\\naspartate\\nresiduePhosphorylated\\nstate\\nADP\\ncharges\\nrepelγ phosphate of ATP\\ntransferred to aspartate\\nresidue\\n++C C\\nO OO O\\nCC\\nO OOOOP\\nOOP\\nOOP\\nOOO\\nOOOP OOP OO\\nOO\\nOPO O\\nFigure 11.4. Phosphorylation changes the charge pattern, and hence the balance of forces within\\nthe calcium ATPase, forcing a change of shape.\\nuncharged (page 187). In solutions with a pH less than 7, most of the histidine residues will\\nbear a positive charge. A protein conformation that had two histidine residues close together(Fig. 11.3) would therefore be stable in alkaline conditions, but in acid conditions the tworesidues would each bear a positive charge and repel, destabilizing the conformation.\\nAnother mechanism that is used to change the conformation of proteins is the addition of\\na negatively charged phosphate group by a class of enzymes called protein kinases. Proteinscan be phosphorylated on serine, threonine, tyrosine, aspartate, glutamate, or histidine. Thecalcium ATPase (Fig. 11.4) is a transmembrane protein that forms a tube that can be open\\nto the cytosol or to the extracellular medium. In its resting state the tube is open to thecytosol, while ATP is held by noncovalent interactions in a crevice in a cytosolic domain.Phosphorylation transfers the γphosphate of ATP to a nearby aspartate residue. Repulsion\\nbetween the phosphoaspartate and the remaining ADP means that the lowest energy stateof the protein is now one in which the tube is open to the extracellular medium. Thismechanism is used to force calcium ions out of the cytoplasm into the extracellular ﬂuid.\\nThe calcium ATPase is described in more detail on page 318.\\nThe movements produced in a protein by phosphorylation are small, on the order of\\nnanometers or less. When these are repeated many times and ampli ﬁed by lever systems,\\nthey can produce movements of micrometers or even meters. The beating of a ﬂagellum\\n(page 386) or the kicking of a leg are both produced by phosphorylation-induced proteinshape changes.\\nENZYMES ARE PROTEIN CATALYSTS\\nLife depends on complex networks of chemical reactions. These are mediated by enzymes.Enzymes are catalysts of enormous power and high speci ﬁcity. Consider a lump of sugar.\\nIt is combustible but quite dif ﬁcult to set alight. A chemical catalyst would speed up its\\ncombustion, and we would end up with heat, a little light, carbon dioxide, and water.Swallowed and digested, the sucrose is broken down in many steps to carbon dioxide and', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='75072b3b-d0c0-4748-9790-e526b0fcb2d8', embedding=None, metadata={'page_label': '248', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\n242 HOW PROTEINS WORK\\nwater by the action of at least 22 different enzymes, and the energy released is used to drive\\nother reactions in the body.\\nAt a basic level, a reaction carried out by an enzyme can be expressed as\\nE+S⇀↽ES⇀↽EP⇀↽E+P\\nwhere E is the enzyme, which binds the substrate S to form the complex ES. The term\\nsubstrate is used for a ligand that binds to an enzyme and that is then transformed to the\\nproduct, P. The region of the protein where the substrate binds and the reaction occurs is\\ncalled the active site. This binding is speci ﬁc, often highly so. The enzyme β-galactosidase\\n(page 109) is moderately speci ﬁc and will split not only lactose but also any other dis-\\naccharide that has a glycosidic bond to β-galactose. By contrast glycogen phosphorylase\\nkinase (page 305) acts with absolute speci ﬁcity on a single substrate, another enzyme\\ncalled glycogen phosphorylase —none of the thousands of other proteins in the cell can\\nsubstitute. In general, the speci ﬁcity of an enzyme is conferred by the shape of the active\\nsite and by particular amino acid side chains that interact with the substrate. Binding of thesubstrate produces the enzyme –substrate complex ES; the catalytic function of the protein\\nthen converts the substrate to product, still bound to the enzyme in the complex EP. Finallythe product dissociates from the enzyme. Chemical engineers use catalysts made of manymaterials, and within cells there are catalysts called ribozymes that are made of RNA.However, only proteins, with their enormous repertoire of different shapes can producecatalysts of high selectivity.\\nThecatalytic rate constant k\\ncat(also known as the turnover number ) of an enzyme\\ngives us an idea of the enormous catalytic power of most enzymes. It is de ﬁned as the\\nnumber of molecules of substrate converted to product per molecule of enzyme per unittime (equally it is moles of substrate converted per mole of enzyme per unit time). Manyenzymes have k\\ncatvalues around 1000 to 10,000 per second. The reciprocal of kcatis the time\\ntaken for a single event. Thus if kcatis 10,000 s−1, one substrate molecule will be converted\\nevery tenth of a millisecond. Some enzymes achieve very much higher rates. Catalase, anenzyme found in peroxisomes, has a k\\ncatof 4×107s−1, and so it takes only 25 ns to split a\\nmolecule of hydrogen peroxide into oxygen and water.\\nThe Initial Velocity of an Enzyme Reaction\\nThe basic experiment in the study of enzyme behavior is measurement of the appearance ofproduct as a function of time (Fig. 11.5). This is often called an enzyme assay. The productappears most rapidly at the very beginning of the reaction. As the reaction progresses, therate at which the product appears slows down and eventually becomes zero when the systemhas reached equilibrium. All reactions are in principle reversible; thus the observed overallrate of the reaction is actually the difference between the rate at which the product is beingformed and the rate at which the product is being broken down again in the reverse reaction.In many enzyme reactions the equilibrium lies strongly toward the product.f\\nIN DEPTH 11.1 What to Measure in an Enzyme Assay\\nIn an enzyme assay we measure the appearance of product as a function of time.\\nIn principle we could do this by starting identical reactions in a series of test tubesand stopping the reaction in each test tube at a different time after the start andthen measuring the amount of product in each tube.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3d9b1bd2-475e-4bd3-ba3b-562d0605bd5b', embedding=None, metadata={'page_label': '249', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\nENZYMES ARE PROTEIN CATALYSTS 243\\nHowever, if either the substrate or the product has a property that can be\\nmeasured while the reaction is proceeding, then the whole assay can be performed\\nin one test tube. Many enzyme assays are done by using changes in the absorptionof light when the substrate is converted to product. Other optical properties can beused. In their original work Michaelis and Menten studied an enzyme that breaksthe disaccharide sucrose down into glucose and fructose. The mixture of glucoseplus fructose rotated polarized light differently from sucrose, and they used thisproperty to follow the course of the reaction. If there is no convenient opticalproperty, then others may be available; for example, one can monitor the progressof a reaction in which the polysaccharide glycogen (page 30) is hydrolyzed bydigestive enzymes by measuring the resulting fall in viscosity.\\nWe can simplify the analysis of enzyme reactions if we consider only the start of the\\nreaction, when there is no product present and so the back reaction can be neglected. The\\nrate of reaction at time zero (the initial velocity v0, sometimes called the initial rate) is\\nfound by plotting a graph of product concentration as a function of time and measuring theslope at time zero (Fig. 11.5). In practice the slope is measured over the ﬁrst 5% of the total\\nreaction. Initial velocity, v\\n0, is conveniently expressed as the rate at which the concentration\\nof the product increases, that is, moles per liter per second.\\nEnzymes must be assayed under controlled conditions because temperature, pH, and\\nother factors alter the activity. Most are such very effective catalysts that they must beassayed under conditions where the concentration of the enzyme is always very much lessthan the concentration of the substrate. Otherwise the reaction would be over in a fractionof a second.\\nProduct\\nconcentration\\n[P]slope = vo\\ntime\\nFigure 11.5. Deﬁnition of v0, the initial velocity of a chemical reaction.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='cb0e1c5e-7609-4e72-a19d-8b3063fccd77', embedding=None, metadata={'page_label': '250', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\n244 HOW PROTEINS WORK\\nEnzymes are studied for many reasons. An understanding of how they achieve their\\ncatalytic excellence and speci ﬁcity is of fundamental interest and has many practical ap-\\nplications as we increasingly use them in industrial processes and we even seek to design\\nenzymes for particular tasks. Measurements of enzyme concentrations and properties al-low us to study processes within cells and organisms. The starting point of any study ofan enzyme is to determine its activity, measured as k\\ncat, and how tightly it can bind its\\nsubstrate —its substrate af ﬁnity. Substrate af ﬁnity is measured by another constant called\\ntheMichaelis constant, KM. We will now explain how these constants are measured.\\nEffect of Substrate Concentration on Initial Velocity\\nLet us consider a series of experiments designed to see how the initial velocity of anenzyme reaction varies with the concentration of substrate (which is always much greaterthan that of the enzyme). Each of the smaller graphs in Figure 11.6 shows the result of oneof these experiments. As we increase the substrate concentration, we ﬁnd that at ﬁrst the\\nvelocity increases with each increase in substrate concentration but that, as the substrate\\n[S] = 0.5KM\\n[S] = 0.1KMv0\\n[S] 5 4 3 2 KMVm\\n[S] = 0.2KM[P] [P] [P]\\ntime time time\\ntime\\ntime\\ntime[P]\\n[P]\\n[P][S] = 1KM\\n[S] = 2KM[S] = 5KM\\nFigure 11.6. v0measured in a number of reaction tubes (with [E] constant and always less than\\n[S]) forms a hyperbolic curve when plotted as a function of substrate concentration.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='f8791382-aec0-40e8-8502-6db8859c15fb', embedding=None, metadata={'page_label': '251', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\nENZYMES ARE PROTEIN CATALYSTS 245\\nconcentration becomes larger, the increases in rate produced get smaller and smaller. We\\nhave already met the name of a curve of this type: it is a hyperbola. The initial velocityapproaches a maximum value that is never exceeded. Reactions that show this sort ofdependence on substrate concentration are said to show saturation kinetics .\\nHow can we explain this? The reaction sequence can be simpli ﬁed to\\nE+S⇀↽ES→E+P\\n(we are using initial velocities so we can ignore any back reaction). The enzyme and substrate\\nmust collide in solution, and the substrate must bind at the enzyme ’s active site to form the\\nES complex. The chemical reaction then takes place within the ES complex, and ﬁnally the\\nproduct is released. In experiments with higher and higher substrate concentrations, thereis ever more ES present, and this increasing ES gives an increasing rate of product release.At very high substrate concentrations virtually all of the enzyme is present as ES, andthe observed rate is limited by the ES →E+P step. Thus, as the substrate concentration\\nincreases, the reaction rate levels off as it approaches a maximal velocity called V\\nmorVmax.\\nWe can de ﬁneVmas the limiting initial velocity obtained as the substrate concentration\\napproaches in ﬁnity. It is the product of the catalytic rate constant kcatand the amount of\\nenzyme present, that is, Vm=kcat[Etotal]. Having de ﬁnedVm,w en o wd e ﬁne the Michaelis\\nconstant, KM, as that substrate concentration that gives an initial velocity equal to half Vm.\\nThe plot of v0against [S] gives a hyperbolic curve that is described by the equation\\nv0=Vm[S]\\nKM+[S]\\nwhere [S] is the substrate concentration. This is called the Michaelis–Menten equation\\nafter Maud Menten and Leonor Michaelis who propounded a general theory of enzymeaction in 1913. The Michaelis constant K\\nMis the substrate concentration that gives an\\ninitial velocity numerically equal to half of Vm. Another way of looking at this is to say that\\nif the enzyme is saturated with substrate, then the rate of reaction will be Vm, while at a\\nsubstrate concentration giving an initial velocity of1/2Vmthe enzyme is half saturated with\\nsubstrate. A small value of KMmeans that the enzyme has a high af ﬁnity for the substrate.\\nThe Effect of Enzyme Concentration\\nAs the concentration of an enzyme increases, so does the initial velocity, and this relationshipis linear: if you double the enzyme concentration, you double the initial velocity. This factmeans that we can measure how much of an enzyme is present by measuring the rate of thereaction it catalyzes.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='969c7012-28e0-4571-8968-bddd05efb693', embedding=None, metadata={'page_label': '252', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\n246 HOW PROTEINS WORK\\nf\\nIN DEPTH 11.2 Determination of VmandKM\\nThe initial rate of a reaction depends on Vmand KMaccording to the Michaelis –\\nMenten equation:\\nv0=Vm[S]\\nKM+[S]\\nThus the information necessary to estimate Vmand KMis contained within data\\nsuch as that shown in Figure 11.6. However, it is not immediately obvious how\\none would go about this: The curve ﬂattens off well before Vmis reached so that\\nguessing Vmby eye is near to impossible. Nowadays many computer programs\\nare available that will ﬁt the hyperbolic Michaelis –Menten equation to the raw\\ndata. Before this, biologists used mathematical transformations of the Michaelis –\\nMenten equation that yielded straight-line relationships. The best known is calledthe Lineweaver –Burk plot after its inventors. It simply inverts the Michaelis –Menten\\nequation to get\\n1\\nv0=KM+[S]\\nVm[S]=KM\\nVm[S]+[S]\\nVm[S]\\nhence\\n1\\nv0=KM\\nVm×1\\n[S]+1\\nVm\\nThis function can be recognized as having the form of the equation of a straightliney=mx+c, where yis 1/v\\n0andxis 1/[S]. Thus if one plots 1 /v0on the vertical\\naxis and 1/[S] on the horizontal axis, one gets a straight line where the yintercept\\nis 1/Vm, the xintercept is −1/KM, and the slope is KM/Vm.\\n1\\n−1\\nKMυ0\\n1\\nVM\\n1\\n[S]', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='5a1062cf-90a7-445a-8bc6-ee98d39649bf', embedding=None, metadata={'page_label': '253', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\nENZYME CATALYSIS 247\\nTable 11.1. Values of kcat/KMfor a Number of Enzymes\\nEnzyme kcat(s−1) KM(mol liter−1)kcat/KM(liter mol−1s−1)\\nTrimeric G protein (page 344) 0.02 6 ×10−73.3×104\\nLysozyme (page 192) 0.5 6 ×10−68.3×103\\nAminoacyl tRNA synthase (page 165) 7.6 9 ×10−48.4×103\\nIntestinal ribonuclease (page 199) 7 .9×1027.9×10−31×105\\nAcetylcholinesterase (page 368) 1 .4×1049×10−51.6×108\\nCatalase (page 60) 4 ×1071.1 4 ×107\\nThe Speciﬁcity Constant\\nThe Michaelis constant KMreﬂects the af ﬁnity of an enzyme for its substrate; kcatreﬂects\\nthe catalytic ability of an enzyme. The ratio of these, kcat/KM, is the speci ﬁcity constant,\\nwhich is a measure of how good the enzyme is at its job. A high speci ﬁcity constant means\\nthat a reaction goes fast ( kcatis big) and the enzyme does not need a high concentration of\\nsubstrate ( KMis small). When an enzyme has relatively low speci ﬁcity, that is, it can work\\non a number of different substrates, the substrate that has the largest speci ﬁcity constant is\\nthe preferred substrate for the enzyme.\\nA reaction cannot go faster than the rate at which enzyme and substrate actually collide.\\nIn some enzymes this rate of collision is the factor limiting the overall rate, and such\\nenzymes are said to be “diffusion limited ”and are considered to have reached a perfection\\nof biological design. Diffusion-limited values of kcat/KMcan be calculated to be in the\\nrange 108–1010liter per mole per second. Table 11.1 shows kcat,KM, and kcat/KMfor a\\nnumber of enzymes.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 11.1 Speed Isn’t Everything\\nIn this chapter we emphasize enzymes as tools for performing a reaction that is required in the\\ncell. For this type of enzyme, the faster the reaction occurs, the better, although in many casesan“optimized ”enzyme must be controlled so that it only operates at maximum rate when lots of\\nproduct is needed.\\nFor other enzymes, slowness is a virtue. We will meet trimeric G proteins in Chapter 16. Like\\nthe GTPases that we have already met (EF-tu, Ran, and Rab), these are active when they have GTPbound, and switch to an inactive state when they hydrolyze the GTP to GDP. While they are active,they turn on their target processes. In these enzymes, therefore, a slow rate of reaction allows themto act as timer switches. They are activated when GDP is ejected and GTP bound. They then remainactive until they hydrolyze the GTP. For the typical trimeric G protein listed in Table 11.1, k\\ncatis\\n0.02 per second, so the “timer ”is set to 50 s (the reciprocal of 0.02 s−1).\\nENZYME CATALYSIS\\nA catalyst speeds the rate of reaction but does not alter the position of the equilibrium E +\\nS⇀↽E+P. Reaction speed is increased by lowering the activation energy barrier between\\nthe reactants and products. Figure 11.7 illustrates this effect. The top graph shows the total\\nenergy of the system as the reaction proceeds, the horizontal axis representing the progress', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e0dd0cce-d2e2-4d7e-bd32-a2d021392172', embedding=None, metadata={'page_label': '254', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\n248 HOW PROTEINS WORK\\nenergy\\n(a)progress of reactionA\\nproductsreactants\\nenergywithout\\nenzyme\\nwith\\nenzyme\\n(b)progress of reactionA\\nproductstransition\\nstatetransition\\nstate\\nreactants\\nFigure 11.7. Catalysts act by reducing the activation energy (A) of a reaction.\\n✞\\n✝✆\\n✆Medical\\nRelevance\\n11.1Measurement of Enzyme Concentrations in Blood\\nBecause the initial velocity of a reaction is a linear function of the enzyme concen-\\ntration, we can measure the concentration of an enzyme by measuring the rate of the\\nreaction it catalyzes. Clinicians make use of this to measure the amount of an enzyme presentin the blood. If enzymes that are normally found within cells appear in the blood in more thantrace amounts, this indicates that cells are being damaged and their contents are leaking out. Thepresence of enzymes characteristic of a particular organ tells the clinician which tissue is affected.The digestive enzyme trypsin is stored in an inactive form called trypsinogen in the cells of thepancreas. Trypsinogen is activated once it is secreted into the lumen of the gut. Trypsinogen canbe detected in the blood by converting it to the active enzyme and measuring its activity. Thepresence of trypsinogen in the blood of a fetus is a sign that the cells of the pancreas are dying,and this is in turn an early indicator of cystic ﬁbrosis (page 426).', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6af13cff-d939-471d-a555-17b101f42607', embedding=None, metadata={'page_label': '255', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\nCOFACTORS AND PROSTHETIC GROUPS 249\\nof the reaction so that at the left we have the reactants, on the right we have the products,\\nand in the middle we have the transition state that the system has to pass through for thereaction to occur. Because the transition state is of higher energy than the reactants, thereaction will only occur if the reactants have additional energy (e.g., as thermal motion)equal to the activation energy. The bottom graph shows the same reaction in the presence of\\na catalyst. Although the energy of the reactants and products has not changed, the presenceof the catalyst has reduced the energy of the transition state and has therefore reduced theactivation energy. A greater proportion of all the reactant molecules in solution will possessthis smaller amount of extra energy, so more can undergo the reaction: the catalyst hasspeeded up the rate at which the reaction occurs. Each enzyme has evolved to reduce theactivation energy of a particular reaction. Clearly the binding of the substrate is as centralto catalysis as it is to speci ﬁcity. The active site contains precisely positioned amino acid\\nside chains that promote the reaction. Daniel Koshland proposed that the enzyme ’s active\\nsite was shaped not so much to ﬁt the substrate but to ﬁt a molecule that was halfway\\nbetween substrate and product, and therefore to reduce the activation energy of the reactionby stabilizing, and hence reducing the energy of, the transition state. He proposed that theenzyme might actually change shape when it bound the substrate to promote catalysis. Thisis called induced ﬁt and many enzymes have been shown to do this.\\nCOFACTORS AND PROSTHETIC GROUPS\\nEnzymes that need to perform reactions that are outside the repertoire of the 20 amino acidside chains recruit other chemical species to help them do the job. The aminotransferasesprovide a good example. These enzymes are central to amino acid metabolism (page 292)and catalyze the interconversion of amino acids and oxo-acids by moving an amino group(Fig. 11.8). Pyridoxal phosphate (derived from vitamin B\\n6) is bound by the protein and\\naccepts the amino group from the amino acid donor, which then leaves the enzyme con-verted to a new oxo-acid. The oxo-acid substrate then binds, accepts the amino group fromthe pyridoxal phosphate, and leaves having been converted to an amino acid. Pyridoxalphosphate ends up exactly as it started and remains bound to the enzyme ready for anothercycle. These helper chemicals are called cofactors when, like pyridoxal phosphate, theyare not tightly bound to the enzyme. We already know the term for a molecule that is verytightly bound to a protein and helps it perform its job —it is a prosthetic group (page 205).\\nThese terms overlap and usage varies.\\nThe prosthetic group of hemoglobin and myoglobin, heme, (Fig. 9.21 on page 206)\\nbinds oxygen. It is a versatile molecule that is also found in electron transfer proteins calledcytochromes. Cytochromes found in the inner mitochondrial membrane play important rolesin the electron transport chain (page 265). Other cytochromes, the cytochromes P450 thatwe have met earlier (pages 124, 158, 166), are found in the membranes of the endoplasmicreticulum and mitochondria. There are at least 50 different cytochrome P450s in humans.Some of these proteins carry out very speci ﬁc metabolic reactions in the process of convert-\\ning cholesterol into the different steroid hormones our body utilizes. Other cytochromesP450 are less speci ﬁc in their reactions. Their role is to detoxicate the millions of foreign\\nchemicals to which we are exposed on a daily basis. However, all cytochromes P450 shareone property in common: they all insert a single atom of oxygen, derived from molecularoxygen, into a substrate. Subsequent rearrangements often result in the oxygen being in theform of a hydroxyl group in the ﬁnal reaction product. This transformation converts a', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='21cf0f72-2c0b-4763-99b0-7caa0535eb12', embedding=None, metadata={'page_label': '256', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\n250 HOW PROTEINS WORK\\nCH2CH\\nO\\nCH3OH\\nOO\\nOO\\nOP\\nthe reaction is in two stages:\\nE-pyridoxal phosphate E-pyridoxamine phosphate pyruvate alanine ++\\nE-pyridoxamine phosphate E-pyridoxal phosphate glycine glyoxylate++alanine\\nglyoxylateOCO CH3\\nNH3 OOC CH\\nO CH\\nOCH\\nO CH\\npyruvate\\nglycineAlanine glyoxylate aminotransferase catalyzes the reaction:\\nCH3\\nOOC C++\\nNN\\nHHN\\nH\\nCH2CH2NH3\\nCH3OHO\\nOO\\nOP\\n++\\nNN\\nHHN\\nHduring the reaction it picks up the \\namino group from the amino acid substrate pyridoxal phosphate is the cofactor for aminotransferases\\npyridoxamine phosphate\\nNH\\n3 ++\\nFigure 11.8. Aminotransferases use a cofactor that participates in the reaction but ends up un-\\nchanged. E represents the enzyme molecule.\\nhydrophobic molecule into one that is more hydrophilic. After hydroxylation other\\nmolecules are coupled to the hydroxyl group to make the foreign chemical even morehydrophilic and hence allow it to be rapidly excreted in the urine and feces. Unfortunately,in very rare cases, the addition of an oxygen atom into a molecule can increase its reactivityand, instead of being rendered harmless by the action of a cytochrome P450, it is transformedinto a very dangerous chemical that can damage DNA. If the damage causes mutation orstrand breakage, then cancer can result. Polycyclic aromatic hydrocarbons, found in tobacco', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4ba185ab-8987-499f-a447-1f82694ade80', embedding=None, metadata={'page_label': '257', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\nENZYMES CAN BE REGULATED 251\\nsmoke, and the chemical a ﬂatoxin, found in the mold that grows on peanuts, are converted\\ninto chemical carcinogens by the action of certain cytochromes P450.\\nENZYMES CAN BE REGULATED\\nIn this chapter we have discussed the catalytic role of enzymes in isolation. In fact enzymes\\nare like other proteins in showing the complex behavior described in Chapter 9 such as mul-tiple states, multiple binding sites, quaternary structure, and phosphorylation. For instance,some important enzymes have quaternary structures and show cooperativity between theactive sites. Such enzymes will not follow the Michaelis –Menten equation, but instead the\\ncurve of initial velocity against substrate concentration will have the same S-shaped orsigmoid curve seen in the binding of oxygen to hemoglobin (Fig. 11.2).\\nA good example of an allosteric enzyme is to be found in the processes that allow us to\\nuse sugars as metabolic fuels. The pathway called glycolysis (page 284) converts glucose\\nand fructose to pyruvate with a net production of two ATP molecules. The pyruvate is thenpassed to the mitochondria, which use it to produce more ATP.\\nTheﬁrst committed step in this pathway is the addition of a second phosphate group\\nto monophosphorylated fructose. The γphosphate of ATP is transferred to the number 1\\ncarbon of fructose-6-phosphate. Before this step, sugars can be converted to glycogen ormetabolized in other ways, but after this step the sugar is committed to be broken down topyruvate. The step is catalyzed by the enzyme phosphofructokinase (Fig. 11.9). ATP serves\\nas an allosteric inhibitor of this enzyme so that it operates slowly when ATP concentrationsare high and there is no need for more ATP to be produced. When there is a drain on ATP,levels of AMP increase (page 259). AMP competes with ATP for the regulatory site andcauses the enzyme to switch to the active, high-af ﬁnity conformation. Thus as the ATP\\nconcentration is reduced and AMP levels increase, the pathway is more and more activateduntil it is turned on to full speed. We very often see this type of control system, where theproduct of a pathway or process feeds back to allosterically inhibit an enzyme at the startof that pathway. They are an example of negative feedback (page 303).\\nIt is not only enzymes involved in metabolism that are regulated by ligand binding.\\nCdk1, the enzyme that triggers mitosis (page 411), is an allosterically regulated enzyme. Onits own, in solution, it spends the vast majority of time in an inactive conformation. In order toassume the active conformation, Cdk1 must bind a protein ligand called cyclin B (Fig. 11.10).In addition to this allosteric regulation, Cdk1 is also regulated by phosphorylation: for mostof the time the catalytic site is blocked by two phosphate groups within the catalytic domain,\\nfructose-6-phosphate fructose-1,6-bisphosphate\\ninhibited by ATPactivated by AMP\\nATP+\\n−ve+ve\\nFigure 11.9. Phosphofructokinase is regulated by the binding of ATP or AMP at a regulatory site\\nthat is separate from the active site. Binding of ATP inhibits while binding of AMP activates.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2556841e-a5c7-4e3f-a4cc-137c50192205', embedding=None, metadata={'page_label': '258', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\n252 HOW PROTEINS WORK\\nP\\nPP\\nPT14 T14\\nactive\\ncatalytic\\nsiteY15 Y15\\nCdk1\\nCdk1Cdk1\\ncyclin Binactive inactive active kinase inactive\\ncyclin BT14\\nY15 P\\nPT14\\nY15\\nCdk1\\nFigure 11.10. Control of Cdk1 by cyclin B and by phosphorylation.\\nat threonine 14 and tyrosine 15. In order to become an active enzyme, Cdk1 must lose these\\ntwo phosphates while cyclin B is bound. This complex control means that Cdk1 can act asa checkpoint. If the phosphate groups have been removed from T14 and Y15, AND cyclinB is present at a high enough concentration, THEN it is safe to proceed into mitosis.\\nCdk1 is an example of an enzyme that is turned off by phosphorylation. Other enzymes\\nare turned on by phosphorylation. Examples include RNA polymerase II (page 120) andglycogen phosphorylase (page 305).f\\nIN DEPTH 11.3 Rapid Reaction Techniques\\nEnzyme measurements are usually carried out with low concentrations of the\\nenzyme because higher concentrations would give initial velocities too fast tomeasure. Initial velocity measurements are always made long after the enzyme –\\nsubstrate complex has formed. It would be very interesting to be able to observethe actual formation of enzyme –substrate complexes and indeed, more generally,\\nto observe the formation of protein –ligand complexes. These reactions are rapid,\\noccurring on a millisecond time scale or less.\\nHartridge and Roughton devised the method illustrated in the accompanying\\ndiagram for measuring the rate at which hemoglobin combined with oxygen. Aram pushes two syringes, one containing hemoglobin and one containing oxy-genated solution. The solutions mix and pass down a tube, so that the further onelooks to the right in the tube, the more time the oxygen has had to bind to thehemoglobin. The association of oxygen with hemoglobin can be monitored by anoptical change, the same color change that causes arterial blood to be bright redwhile venous blood is a dark bluish red. Simply moving an optical detector from leftto right along the tube showed the concentration of oxygen –hemoglobin complex\\nafter different reaction times. However, all the time the measurements are beingmade, the hemoglobin solution is running out of the pipe at the right-hand end,so that one can only use this approach when large quantities of the protein areavailable.\\nThe second ﬁgure shows a less wasteful method called stopped ﬂow. Here two\\nsolutions are mixed and passed into an observation chamber and from it into a sy-ringe. The plunger of this “stopping syringe ”moves back until it hits a stopping', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1d5eaee5-cdfe-4144-8559-76b5b443a16d', embedding=None, metadata={'page_label': '259', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\nRAPID REACTION TECHNIQUES 253\\nplate and the ﬂow stops. Observations are made on the solution as it ages in the\\nobservation chamber, using a high-speed recording device that is triggered by the\\nstopping syringe as it hits the plate. Stopped ﬂow allows observations of reactions\\ndown to about 0.1 ms after mixing. For example, the rate at which calmodulin(page 200) binds calcium can be measured by ﬁlling one syringe with calmodulin,\\nthe other with a calcium solution. In this case the reaction can be followed be-cause one tyrosine residue on calmodulin (Tyr138) increases its ﬂuorescence when\\ncalmodulin binds calcium.\\nmixer\\nhemoglobin\\noxygenated\\nsolutionmonochromator\\ndetector\\nmove along pipe \\nto sample \\nreaction at \\ndifferent times \\nafter mixingmeterram\\nstopping syringemonochromator\\nstop\\nmicroswitch\\ndetector\\ntriggermixer\\nfilling valves\\noscilloscoperamlight beam', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='c28b8a6e-e9ef-4495-b133-287e0affed67', embedding=None, metadata={'page_label': '260', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\n254 HOW PROTEINS WORK\\nSUMMARY\\n1.Most protein functions arise from their ability to speci ﬁcally bind other molecules\\nin a reversible yet speci ﬁc fashion. Shape changes, triggered by the binding of other\\nmolecules, mediate protein movements and function. Allostery is a special case where\\na shape change induced by a ligand at one site changes the af ﬁnity at another site.\\n2.Enzymes are highly speci ﬁc biological catalysts. The turnover number or catalytic rate\\nconstant, kcat, is the maximum number of substrate molecules that can be converted\\nto product per molecule of enzyme per unit time.\\n3.The initial velocity (i.e., the rate at the start when product is absent) of many en-zymes shows a hyperbolic dependence on substrate concentration. At high substrateconcentrations the initial velocity approaches a limiting value V\\nm, as the enzyme\\nis saturated with substrate. The substrate concentration that gives an initial velocityequal to half V\\nm, is the Michaelis constant KM. This indicates the enzyme ’sa fﬁnity\\nfor the substrate.\\n4.The initial velocity is related to the substrate concentration by the Michaelis –Menten\\nequation:\\nv0=Vm[S]\\nKM+[S]\\n5.The initial velocity is directly proportional to the enzyme concentration.\\n6.Some enzymes use cofactors to carry out reactions that require different propertiesfrom those of the side chains of the 20 amino acids found in proteins.\\n7.Enzyme activity within cells is modulated by a variety of methods that include phos-phorylation and allosteric effects.\\nFURTHER READING\\nCornish-Bowden, A. 1995. Fundamentals of Enzyme Kinetics, rev. ed. London: Portland Press.\\nFersht, A. 1999. Structure and Mechanism in Protein Science. New York: W. H. Freeman.\\nV oet, D., V oet, J. D. 2003. Biochemistry, 3rd ed. New York: Wiley.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='5f04ded1-8ce9-4df5-a15e-f2ada4f864de', embedding=None, metadata={'page_label': '261', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\nREVIEW QUESTIONS 255\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.Conformational or shape changes in proteins allow\\nA. conversion of chemical energy into kinetic (movement) energy.\\nB. cooperativity between subunits in a quaternary structure.C. allosteric control of enzymes.D. proteins to carry other molecules across membranes.E. all of the above.\\n2.The turnover or catalytic constant ( kcat) of an enzyme is\\nA. the number of molecules of substrate converted to product per mole of enzyme per unit time.B. the time taken for a single catalytic event.C. the number of substrate molecules in a mole.D. the number of moles of substrate converted to product per mole of enzyme per unit time.E. the initial velocity of the reaction when half the enzyme molecules have bound substrate.\\n3.The oxygen-binding curve for hemoglobin is S-shaped: starting from zero oxygen, the\\nﬁrst small increase in oxygen concentration produces hardly any binding to hemoglobin.\\nThe next increase in oxygen produces much more binding so that the curve gets steeperbefore leveling off again as the hemoglobin becomes fully loaded. Hemoglobin can showthis behavior because\\nA. it contains heme prosthetic groups.\\nB. it contains magnesium, in contrast to the iron in myoglobin.C. it comprises two αchains of high oxygen af ﬁnity and two βchains of low oxygen af ﬁnity.\\nD. its four subunits can exist in one of two states, only one of which has a high af ﬁnity for oxygen.\\nThe way that the four subunits ﬁt together means that they all must be in one form or the other.\\nWhen oxygen concentration is low, most of the hemoglobin molecules have their subunits inthe low-af ﬁnity form. As oxygen binds, more molecules switch to the high-af ﬁnity form.\\nE. its four subunits can exist in one of two states, only one of which has a high af ﬁnity for oxygen.\\nThe way that the four subunits ﬁt together means that they all must be in one form or the other.\\nWhen oxygen concentration is low, most of the hemoglobin molecules have their subunits inthe high-af ﬁnity form. As oxygen binds, more molecules switch to the low-af ﬁnity form.\\n4.Which of the following statements about enzymes is false?\\nA. They bind their substrate.B. They lower the activation energy for the reaction they catalyze.C. They alter the equilibrium constant for the reaction they catalyze.D. They stabilize intermediates along the pathway of the reaction they catalyze.E. They can change shape to promote the reaction.\\n5.Protein kinases alter the conformation of their target protein by adding\\nA. H+ions.\\nB. phosphate groups.C. histidine residues.D. oxygen.E. heme groups.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3ed0bfc7-4b2a-4bc5-a15d-c2be1d3986d9', embedding=None, metadata={'page_label': '262', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-11 WY001-Bolsover-v2.cls September 15, 2003 21:31\\n256 HOW PROTEINS WORK\\n6.A prosthetic group is\\nA. a highly charged domain or motif that alters the shape of a protein.\\nB. a nonprotein molecule that is loosely attached to a protein and helps it to perform its function.C. a nonprotein molecule that is tightly bound to a protein and helps it to perform its function.D. a steroid molecule attached to a histidine residue.E. a region of a protein that can accept a phosphate group.\\n7.Phosphofructokinase\\nA. produces a product that is inevitably broken down to pyruvate.B. uses ATP as a substrate in its catalytic site.C. binds ATP at a regulatory site.D. works more slowly if ATP concentrations increase.E. all of the above.\\nANSWERS TO REVIEW QUESTIONS\\n1.E.All these statements about shape changes in proteins are true.\\n2.D.Note that this statement is equivalent to “kcatis the number of molecules of substrate converted\\nto product per molecule of enzyme per unit time ”but is not equivalent to the statement in answer\\nA, which is false.\\n3.D.Because the subunits begin at low af ﬁnity but switch to the high-af ﬁnity form as oxygen\\nbinds, the curve for oxygen binding starts out shallow and then gets steeper. Answer E wouldproduce exactly the opposite effect. Of the other answers: (A) is true, but irrelevant —myoglobin\\nalso contains heme prosthetic groups, but does not show cooperative behavior. (B) is false: bothmyoglobin and hemoglobin contain iron. It is chlorophyll, in plants, that uses magnesium. (C) isfalse: the oxygen af ﬁnities of all four subunits are similar and, in any case, a difference in af ﬁnity\\nwould not cause the binding curve to be S-shaped.\\n4.C.No catalyst, whether inorganic or biological, can alter the equilibrium constant of a reaction.\\n(If one did, it would allow the generation of a perpetual motion machine.)\\n5.B.Concerning the other answers: (A) there are no enzymes that add H\\n+, those groups that can\\nbe protonated (e.g., —NH 2, histidine residues) accept H+rapidly without the requirement for\\nany catalyst. (C) Amino acids are not added to proteins once they have been synthesized on theribosome, although processing of synthesized proteins can often involve the removal of one ormore amino acids, as, for example, when the endoplasmic reticulum signal sequence is removed(page 223). (D) Those proteins that do bind oxygen, such as hemoglobin, do so spontaneously,without the need for a catalyst.\\n6.C.Both cofactors and prosthetic groups are nonprotein molecules that help a protein to perform\\nits function, but cofactors are only loosely attached while prosthetic groups are attached tightly.\\n7.E.ATP is a substrate for phosphofructokinase, but it also acts at an independent regulatory site,\\nreducing the activity of the enzyme.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='156127f3-21b8-4d66-bd7f-3ecb49110515', embedding=None, metadata={'page_label': '263', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\n12\\nENERGY TRADING WITHIN THE CELL\\nIn the nonliving world complex things degrade naturally to simpler things: Gradients of\\ntemperature or concentration disappear, chemical reactions approach equilibrium, and uni-formity triumphs. Living things do not appear to follow these trends. Cells are complexand divide to make other complex cells: A fertilized egg differentiates to make a wholecomplex organism. Living things must obey the laws of thermodynamics. The escape fromthe behavior of nonliving systems is allowed because living systems take matter and energyfrom the environment and use it to grow, to reproduce, and to repair themselves. Livingsystems are open systems while nonliving systems are closed.\\nA chemical reaction that has reached equilibrium can do no work. A good deﬁnition\\nof death is the state at which all of the chemical reactions in a cell/organism have reachedequilibrium. In a living organism the concentrations of metabolites are often very far fromthe equilibrium concentrations and yet are more or less constant: This is said to be a steady\\nstate. Cells can do this because they are open systems taking energy and matter from their\\nenvironment.\\nWe can use an analogy with the world of economics. It is unlikely that people would\\nspontaneously repair our houses, or feed us, or give us this book, but we can drive theseotherwise unlikely processes by spending money. In a similar way, cells can drive otherwiseunlikely processes by using up one of four energy currencies that are then replaced using\\nenergy taken from the outside world.\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n257', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='50e3f53e-2c14-4b23-bdda-4da6f245e340', embedding=None, metadata={'page_label': '264', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\n258 ENERGY TRADING WITHIN THE CELL\\nCELLULAR ENERGY CURRENCIES\\nThe scienti ﬁc way of saying that a process will proceed (although a catalyst may be necessary\\nto achieve a reasonable reaction speed) is to say that the change in Gibbs free energy,\\nexpressed as /Delta1G, is negative. One reaction that we will meet again is the hydrolysis of\\nglucose-6-phosphate to yield glucose and a phosphate ion:\\nGlucose-6-phosphate +H2O→Glucose +HPO 42−/Delta1G=−19 kJmol−1\\nSince this reaction has a negative /Delta1G, it can proceed, releasing 19 kJ of energy for every mole\\nof glucose-6-phosphate hydrolyzed. Another reaction we will meet again is the hydrolysisof nucleotides, the building blocks of DNA and RNA (Chapter 4). Simply losing the terminalphosphate from the nucleotide ATP releases 30 kJmol\\n−1:\\nAdenosine triphosphate +H2O→Adenosine diphosphate +HPO42−+H+\\n/Delta1G=−30 kJmol−1\\nThe reverse of these reactions will of course not proceed. For instance, cells need to phos-phorylate glucose to make glucose-6-phosphate but cannot use the reaction\\nGlucose +HPO\\n42−→Glucose-6-phosphate +H2O /Delta1G=+19 kJmol−1\\nThe reaction will not proceed because it has a positive /Delta1G. Crucially, though, an unfavorable\\n(positive /Delta1G) reaction can occur if it is tightly coupled to a second reaction that has\\na negative free-energy change (negative /Delta1G) so the overall change for the reactions put\\ntogether is negative. Thus cells phosphorylate glucose by carrying out the following reaction:\\nGlucose +Adenosine triphosphate →Glucose-6-phosphate\\n+Adenosine diphosphate +H+/Delta1G=−11 kJmol−1\\nAdenosine triphosphate, or ATP, has given up the energy of its hydrolysis to drive anotherwise energetically unfavorable reaction forward. We call ATP a cellular currency todraw an analogy with money in human society. Just as we can spend money to causesomeone to do something they would not otherwise do, such as give us food or build us ahouse, the cell can spend its energy currency to cause processes that would otherwise notoccur. However, the analogy is not exact because energy currencies are not hoarded. Thereis a continuous turnover of ATP to ADP and back again. ATP is therefore not an energy store\\nbut simply a way of linking reactions. It can be thought of as a truck that carries metabolicenergy to where it is needed and that returns empty to be re ﬁlled. The number of trucks is\\nsmall but the amount moved can be large. An average person hydrolyzes about 50 kg ofATP per day but makes exactly the same amount from ADP and inorganic phosphate. Wewill see how this happens in this chapter. The cell has a number of energy currencies ofwhich four —NADH, ATP, the hydrogen ion gradient across the mitochondrial membrane,\\nand the sodium gradient across the plasma membrane —are the most important. We will\\nnow discuss each of these in turn.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='09926d3c-6b45-4d23-960a-81c8587cd2ae', embedding=None, metadata={'page_label': '265', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\nCELLULAR ENERGY CURRENCIES 259\\nReduced Nicotinamide Adenine Dinucleotide (NADH)\\nThis, the most energy rich of the four currencies, is shown in Figure 12.1 a. NADH is a\\nstrong reducing agent. It will readily react to allow two hydrogen atoms to be added to\\nmolecules, in the general reaction NADH +H++X→NAD++H2X. NAD+is shown\\nin Figure 12.1 b. Addition of hydrogen atoms to molecules, or the removal of oxygen\\natoms, is called reduction. The opposite of reduction is oxidation, the addition of oxygen.\\nBecause oxygen atoms tend to take more than their fair share of electrons in any bonds theymake, addition of oxygen means removal of electrons, and vice versa, so the most generaldeﬁnition of reduction is the addition of electrons, with oxidation being de ﬁned as the\\nremoval of electrons. We will later see NADH acting to reduce complex cell chemicals likepyruvate and acetoacetate. However, when it is acting as an energy currency, NADH simplypasses its two hydrogen atoms to oxygen, making water. This releases a lot of energy: Everymole of NADH that is used in this way releases 206 kJ of energy.\\n✞\\n✝✆\\n✆Medical\\nRelevance\\n12.1NAD+, Pellagra, and Chronic Fatigue Syndrome\\nA small fraction of our total body NAD+is lost from the body each day. New NAD+\\nis synthesized from the vitamin niacin, which in turn can be synthesized from the\\nessential amino acid tryptophan. If we don ’t get enough niacin or tryptophan, we develop the\\ndisease called pellagra. In pellagra parts of our bodies that use a lot of energy, such as the brain,\\nbegin to fail. Pellagra killed about 100,000 Americans in the ﬁrst half of the twentieth century\\nbefore Joseph Goldberger of the U.S. Public Heath Service showed that it could be prevented andcured by a varied diet. Niacin is now added to all ﬂour, and pellagra is almost unknown in Western\\ncountries.\\nSince chronic fatigue syndrome is manifested as a lack of energy, some people have suggested\\nthat NAD\\n+itself, given as a dietary supplement, might reenergize the patients. There is no scienti ﬁc\\nevidence for this idea, but nevertheless many chronic fatigue syndrome sufferers buy and takeNAD\\n+in the hope that it may help.\\nNucleoside Triphosphates (ATP plus GTP , CTP , TTP , and UTP)\\nAdenosine triphosphate, the second most energy rich of the four currencies, is shown in\\nFigure 12.2. In earlier chapters we met many chemical processes in the cell that are drivenby ATP hydrolysis. When one mole of ATP is hydrolyzed, 30 kJ of energy are released.Theγphosphate is easily transferred between nucleotides in reactions such as this one:\\nATP+GDP⇀↽ADP+GTP\\nSo as far as energy is concerned, we can regard GTP, CTP, TTP, and UTP as equivalent to the\\nmost commonly used nucleotide energy currency, ATP. Another easy, reversible reaction isthis one:\\n2 ADP ⇀↽AMP+ATP\\nin which one ADP transfers its βphosphate to another, so that it itself is left as AMP while\\nconverting the other ADP to ATP. This conversion is at equilibrium in cells and is important\\nin maintaining the supply of ATP when ATP is used in reactions.\\nNADH and ATP take part in so many reactions within the cell that they are often called\\ncoenzymes, meaning molecules that act as second substrates for many enzymes as they do', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='98df0dd1-aa7a-44ff-8b18-5b80c29c2c5e', embedding=None, metadata={'page_label': '266', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\nO Pnicotinamideadenine\\nN\\nCC\\nN C\\nN C\\nHN\\nC\\nNHH\\nH\\nCH2\\nCH2O P\\nOO\\nriboseO\\nOH H\\nHH\\nH\\nHHCN\\nNO\\nOC\\nC CCHH\\nOHH H\\nOH\\nriboseO\\nC\\nC CCHH\\nOHH H\\nOHO CH2 O Pnicotinamideadenine\\nN\\nCC\\nN C\\nN C\\nHN\\nC\\nNHH\\nH\\nCH2 O P\\nOO\\nriboseO\\nOH\\nH\\nHH NO\\nC\\nC CCHH\\nOHH H\\nOH\\nriboseO\\nC\\nC CCHH\\nOHH H\\nOHO\\n(a) NADHH\\nHCNO\\n(b) NAD+\\nFigure 12.1. Nicotinamide adenine dinucleotide, NADH, shown in (a), is a strong reducing agent and energy currency . The oxidized form,\\nNAD+, is shown in (b).\\n260', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='f518978b-a334-4d73-87b3-0c9f288d6d2e', embedding=None, metadata={'page_label': '267', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\nCELLULAR ENERGY CURRENCIES 261\\nadenine\\nN\\nCC\\nN C\\nN C\\nHN\\nC\\nNHH\\nH\\nriboseO\\nC\\nC CCHH\\nOHH H\\nOHtriphosphate\\nCH2 O P\\nOO P\\nOO P\\nOOO\\nγ\\nphosphate β\\nphosphateα\\nphosphateγ\\nphosphate β\\nphosphateα\\nphosphateO O\\nFigure 12.2. Adenosine triphosphate, an energy currency.\\ntheir particular jobs. The term is easy to confuse with cofactor (page 249). A cofactor is a\\nchemical species that is loosely associated with an enzyme, helps it carry out its function,and, although it may undergo reactions, ends up in the same state that it began. The conceptof coenzymes is very different. Coenzymes are bona ﬁde substrates that are converted to\\nproducts (e.g., NAD\\n+and ADP) by the enzyme.\\nThe Hydrogen Ion Gradient Across the Mitochondrial Membrane\\nThe endosymbiotic theory states that mitochondria are derived from bacteria that evolvedto live in eukaryotic cells (page 12). The bacterial cytosol is usually about 0.6 pH unit morealkaline than the world outside; that is, H\\n+ions are four times more concentrated outside\\nthan inside. If they could move freely across the bacterial cell membrane, H+ions would\\nrush in down this gradient. Furthermore, there is a voltage difference across the membrane:The inside is about 140 mV more negative than the extracellular medium. Transmembranevoltages are always referred to in terms of the internal voltage relative to that outside: in thiscase,−140 mV . The transmembrane voltage attracts the positively charged H\\n+ions into the\\nbacterium. Any combination of a concentration gradient and a voltage gradient is called anelectrochemical gradient. For hydrogen ions at the bacterial membrane the electrochemicalgradient is large and inward. Should H\\n+ions be allowed to rush into the bacterium, they\\nwould release energy: about 17 kJ for every mole that enters.\\nFigure 12.3 is a representation of a mitochondrion inside a eukaryotic cell. This is\\nnot an accurate picture of what a mitochondrion looks like (see Fig. 3.6 on page 59) butrather emphasizes the topology and the function. In the center is shown the mitochon-drion with its two membranes. In the very middle is a volume equivalent to bacterialcytosol that is called the mitochondrial matrix. Next comes the inner mitochondrial mem-brane, then the outer mitochondrial membrane. The intermembrane space is the small spacebetween the two mitochondrial membranes. The green region is the cytosol, bound by', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='52d2b803-5f6c-4127-b0e4-a1369704188d', embedding=None, metadata={'page_label': '268', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\n262 ENERGY TRADING WITHIN THE CELL\\nelectron \\ntransport \\nchainATP \\nsynthaseADP/ATP \\nexchangerNa+/K+\\nATPase\\nNADH + H+ + 1/2O23 ATP\\n3 ADP +\\n 3 Pi\\nouter mitochondrial \\nmembraneplasmamembraneinner mitochondrial membraneADP\\nADP + Pi3Na+\\nATP\\n2K+H2O + NAD+12H+\\nporinATP10H+\\ntricarboxylic \\nacid cycleHIGH [H   ]\\nLOW [H   ]\\nLOW [Na  ]HIGH \\n[Na  ]\\nFigure 12.3. Sites in the cell where the energy currencies are interconverted.\\nthe plasma membrane. An integral membrane protein of the outer mitochondrial mem-\\nbrane, called porin, forms a hole or channel that lets through all ions and molecules of\\nMr<10,000, so the ionic composition of the intermembrane space is the same as that of\\ncytosol. However, there is a large electrochemical gradient across the inner mitochondrialmembrane. When H\\n+ions move in from the intermembrane space to the mitochondrial ma-\\ntrix, they release 17 kJ mol−1of energy, exactly as in the mitochondrion ’s proposed bacterial\\nancestors.\\nThe Sodium Gradient Across the Plasma Membrane\\nUnlike bacteria, most eukaryotic cells do not have an H+electrochemical gradient across\\ntheir plasma membranes. Rather, it is sodium ions that are more concentrated outside thecell than inside (Fig. 12.3). Typically, the sodium concentration in the cytosol is about 10mmol liter\\n−1while the concentration in the extracellular medium is about 150 mmol liter−1.\\nThis chemical gradient is supplemented by a voltage gradient. The cytosol is between 70and 90 mV more negative than the extracellular medium, that is, the transmembrane voltageof the plasma membrane is between −70 and −90 mV . There is therefore a large inward\\nelectrochemical gradient for sodium ions. If sodium ions are allowed to rush down thisgradient, they release energy —approximately 15 kJ for every mole of Na\\n+entering the\\ncytosol.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a85bcd16-0f29-46b4-aba3-34ee79c9e73d', embedding=None, metadata={'page_label': '269', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\nENERGY CURRENCIES ARE INTERCONVERTIBLE 263\\nENERGY CURRENCIES ARE INTERCONVERTIBLE\\nA company that buys raw materials in the United States and Mexico, spending dollars\\nand pesos, and then sells products in Europe and Japan, receiving euros and yen, simplyconverts from euros and yen to dollars and pesos to pay its bills. In the same way, cellsconvert from the energy currency in which they are in credit to the energy currency they areusing up.\\nExchange Mechanisms Convert Between the Four Energy Currencies\\nThe cell has mechanisms that transfer energy between the four currencies. The conversionsare summarized in Figure 12.4. In a typical animal cell oxidation of fuel molecules in themitochondria (by the Krebs cycle, page 283) tops up the supply of NADH. The cell thenconverts this energy currency into the other three. All the interconversions are reversible.Figure 12.3 shows where each conversion mechanism is located.\\nConsider what happens if a few sodium ions move out of the extracellular ﬂuid and\\ninto the cytosol of a eukaryotic cell —for example, when a nerve cell transmits the electri-\\ncal signal called an action potential (Chapter 15). The sodium gradient has been slightlydepleted: the cell holds less of this energy currency than it did before. However, the cellstill has plenty of energy in the form of ATP that it can convert into energy as a sodiumgradient. It does this using the sodium/potassium ATPase. This protein is located in the\\nplasma membrane. Its function is to move 3Na\\n+ions out of the cell and to move 2K+\\nions into the cell. For this to happen ATP is hydrolyzed to ADP thus giving up energythat is used to push Na\\n+out of the cytosol to its higher energy state in the extracellular\\nmedium.\\nThe cell has now used some ATP, but it still holds plenty of energy in another currency —\\nthe H+gradient across the inner mitochondrial membrane. The enzyme ATP synthase is\\nlocated in the inner mitochondrial membrane. The protein interconverts the two energycurrencies: as H\\n+ions move into the mitochondrion they give up energy that is then used\\nby ATP synthase to make ATP from ADP and inorganic phosphate.\\nThe energy of the H+gradient is now depleted. However, the cell still has plenty\\nof energy in its NADH account. The electron transport chain allows interconversion of\\nenergy as NADH to energy as H+gradient. NADH is used to reduce molecular oxygen to\\nATP\\nADPhigh energy\\nlow energyNADH\\nNAD+Na+ in extracellular \\nmedium\\nNa+ in cytosolKrebs cycle\\nElectron transport chainNa+/ K+ \\nATPaseATP synthaseH+ inside\\nmitochondrionH+ outside\\nmitochondrion\\nFigure 12.4. Energy ﬂow between the currencies in a normal animal cell.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='04f0c085-7243-4693-a623-b6d78a641ec7', embedding=None, metadata={'page_label': '270', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\n264 ENERGY TRADING WITHIN THE CELL\\nwater, releasing energy that is used to push H+ions out of the mitochondrion to their higher\\nenergy state in the cytosol.\\nEach of the three energy conversion systems is a protein structure called a carrier\\nbecause it carries solute across the membrane. The sodium/potassium ATPase carries sodium\\nand potassium ions, while both ATP synthase and the electron transport chain carry H+ions.\\nThere are many carriers in the cell with a wide variety of functions, some of which we willmeet later in this book. The three that convert between the energy currencies are vital andare evolutionarily ancient.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 12.1 The Fuel That Makes Bacteria Swim\\nAlthough by no means all bacteria are motile, some, such as Escherichia coli andSalmonella\\ntyphimurium, can swim. Their small size makes the process very different from swimming as we\\nexperience it. If you, the reader, dive into a swimming pool, your momentum will carry you some\\ndistance because your mass is great relative to the density of the water. For a bacterium whose massis extremely small, water is a highly viscous medium. A bacterium swimming in water experiencesroughly what a human swimmer in treacle might feel! Despite this, motile bacteria can achievespeeds of up 100 µms\\n−1by the use of a structure called the bacterial ﬂagellum. This consists of a\\nrigid helical ﬁlament, 20 –40 nm in diameter and up to 10 µm long, composed of a single protein,\\nﬂagellin. The bacterial ﬂagellum operates like a boat ’s propeller, pushing the bacterium along as it\\nis turned by the motor at its base.\\nTheﬂagellar motor consists of a series of rings that allow the motor to rotate within the complex\\nlayers of membranes and cell wall that make up the bacterial cell surface. Like ATP synthase, theﬂagellar motor allows H\\n+ions to ﬂow in across the plasma membrane. The energy released as the\\nH+ions ﬂow down their electrochemical gradient is used to turn the rotor of the motor and hence\\ntheﬂagellum at up to 100 Hz ( =100 times per second).\\nH+H+cell wallflagellum\\nplasma membrane', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='82b99ec1-dc66-4c5e-858b-64564ba5e166', embedding=None, metadata={'page_label': '271', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\nENERGY CURRENCIES ARE INTERCONVERTIBLE 265\\nH+ inside \\nmitochondrionH+ outside \\nmitochondrion\\nNAD+NADHhigh\\nenergy\\nlow\\nenergy\\nFigure 12.5. Currency conversion: the electron transport chain converts between NADH and the\\nH+gradient.\\nElectron Transport Chain\\nThe reason we breathe is to supply oxygen to the electron transport chain, which uses the\\noxygen to oxidize NADH in the overall chemical reaction:\\nNADH +H++1/2O2⇀↽NAD++H2O\\nThis reaction releases energy, 206 kJ for every mole of NADH used. The energy is used\\nto carry about 12H+ions from their low-energy state in the mitochondrial matrix to their\\nhigh-energy state in the cytosol. Figure 12.5 summarizes the reaction in terms of energycurrencies. The circle symbolizes the linkage between the energy released in the conversionof NADH to NAD\\n+and the energy used to drive H+out of the mitochondrial matrix. If\\nthe electron transport chain simply allowed NADH to reduce oxygen to water, then thereaction ’s energy would be released as heat. Instead, the enzymatic function of the electron\\ntransport chain is tightly coupled to its function as a carrier that moves H\\n+ions. The energy\\nof NADH is thus converted to the energy of the H+gradient.\\nThe electron transport chain has six components: four large protein complexes, each\\ncontaining many polypeptide chains, embedded in the inner membrane, plus two mobileelectron carriers. The electron transport process has been the subject of intensive investi-gation for decades and this has led to some rather complex nomenclature. Some aspects ofthe process remain unclear.\\nElectrons are mostly carried by prosthetic groups built into the protein complexes. We\\nhave already seen the heme prosthetic group in hemoglobin and myoglobin. The heme-containing proteins of the electron transport chain are, like hemoglobin, colored red orbrown and are called cytochromes from the Latin for cell and color. The iron in a reduced\\ncytochrome is Fe\\n2+while in an oxidized cytochrome it is Fe3+. The exact colors of the\\ncytochromes vary because the proteins in which they are buried provide different environ-ments. These spectral differences are shown by different letters: cytochromes a,b,cwith\\nsubscript numbers for more subtle differences: cytochrome c\\n1,a3. Other prosthetic groups\\nare iron –sulfur clusters that vary in detail but are essentially iron ions coordinated to sul-\\nfurs in the thiol groups of cysteine side chains or free sul ﬁdes. Flavins are also present as\\nelectron-carrying prosthetic groups and the ﬁnal complex also contains copper.\\nTheﬁrst of the mobile electron carriers is called ubiquinone or coenzyme Q. This is\\nchemically a quinone (hence “Q”), which is fairly hydrophobic, and its hydrophobicity is\\nincreased by a long hydrocarbon tail (Fig. 12.6). It carries both hydrogen ions and electrons.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='8d4b32b8-b537-4ea0-a85f-58329428f2cb', embedding=None, metadata={'page_label': '272', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\n266 ENERGY TRADING WITHIN THE CELL\\nO\\nOCH3OCH3OC H3\\n1H 1eCH310HCoenzyme Q\\nCH3OCH3OC H3OH\\nO•\\n1H 1eCH310H\\nCH3OCH3OC H3OH\\nOH CH310Hreduced by one electron\\n(semiquinone)\\nQH •oxidized form\\n(quinone)\\nQ\\nreduced form\\nQH2\\nFigure 12.6. Coenzyme Q carries two hydrogen atoms.\\nThe second electron carrier is a small protein found in the mitochondrial intermem-\\nbrane space called cytochrome c. It has a relative molecular mass of 12,270, which is just\\nlarge enough to be retained by the mitochondrion (the channel porin in the outer membrane\\npasses solute of Mr=10,000 or less). Cytochrome cis soluble but associates with the\\ninner mitochondrial membrane.\\nWe have said that oxidation is loss of electrons, and reduction gain of electrons. Electron\\ncarriers that cycle between the reduced state and the oxidized state normally move either one', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e8d6ba95-bfa2-4a23-85e3-7e8ebe2b61a8', embedding=None, metadata={'page_label': '273', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\nENERGY CURRENCIES ARE INTERCONVERTIBLE 267\\nNAD+NADH fumarate succinate4H+\\nI\\nIIIIIIV4H+4H+\\nO2H2O\\nmatrixintermembrane\\nspaceOverview of the mitochondrial electron transport chain\\nQQcyto\\nCcyto\\nC\\nFigure 12.7. The electron transport chain comprises four large multimolecular complexes in the\\ninner mitochondrial membrane, three of which are hydrogen ion carriers.\\nor two electrons. NADH and a cofactor called FADH 2each carry two electrons. Reduced\\ncytochromes carry single electrons as do the iron –sulfur clusters. Flavins and coenzyme Q\\ncan carry either one or two electrons.\\nFigure 12.7 shows an overview of the electron transport chain. Complex I (sometimes\\ncalled NADH dehydrogenase or NADH-Q oxidoreductase) accepts electrons from NADH,\\nthus oxidizing it, and uses these to reduce coenzyme Q (Fig. 12.8): In doing so it movesfour hydrogen ions outward from the matrix to the mitochondrial intermembrane space.The reaction can be summarized as\\nNADH +Q+5H\\n+\\nmatrix→NAD++QH2+4H+intermem\\nThe transport of the hydrogen ions is accomplished using coenzyme Q as it picks up a\\nhydogen ion when it picks up an electron. Complex I has a bound coenzyme Q that collectstwo hydrogen ions from the matrix when it accepts two electrons from iron –sulfur clusters\\n(which received them from a bound ﬂavin which accepted them from NADH). The bound\\ncoenzyme QH\\n2formed then passes its electrons to iron –sulfur clusters, which ﬁnally reduce\\na mobile coenzyme Q, which again collects two hydrogen ions from the matrix. These arereleased to the intermembrane space when this mobile carrier is reoxidized. The machinerythat produces this directionality is not yet clear.\\nComplex II (also known as succinate-Q reductase complex or succinate dehydroge-\\nnase) is the only one of the four complexes that is not a hydrogen ion pump. As we willsee in Chapter 13 (page 284) complex II is part of the Krebs cycle —it is the enzyme that\\noxidizes succinate to fumarate using FAD, which becomes FADH\\n2. It passes its electrons\\nto mobile coenzyme Q. Other mitochondrial enzymes that generate FADH 2share this prop-\\nerty of being able to reduce coenzyme Q. Because hydrogen ions are not moved across themembrane in this process one gets less ATP formed from the electrons on FADH\\n2than from\\nthose on NADH.\\nComplex III (Q-cytochrome coxidoreductase or cytochrome reductase) accepts elec-\\ntrons from reduced coenzyme Q (QH 2) and uses them to reduce the other mobile electron', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d205c422-7bef-4da4-ba5d-d4550840c254', embedding=None, metadata={'page_label': '274', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\n268 ENERGY TRADING WITHIN THE CELL\\nNAD+\\nFADFADH2NADH\\nfumarate\\nsuccinate4H+\\nQH2\\nQH2QH2\\nH2OQ\\n4H+\\n4H+\\nO2cyto C\\n(oxid)cyto C\\n(reduced)QQmatrix\\nI\\nII\\nIII\\nIV\\nFigure 12.8. Overview of the operation of the mitochondrial electron transport chain.\\ncarrier cytochrome c. This process results in the movement of four hydrogen ions from the\\nmatrix to the intermembrane space. Complex II marks a change from two electrons being\\ncarried to one. It contains hemes, bound coenzyme Q, and iron –sulfur clusters.\\nCytochrome cmoves the electrons to complex IV (cytochrome coxidase). This protein\\ncomplex reduces oxygen to water and moves four hydrogen ions from the matrix to theintermembrane space. It contains two types of heme and copper ions. Oxygen binds toFe\\n2+in a heme just as it binds to the heme in myoglobin or hemoglobin. The overall\\nreaction is\\n4Cyto creduced+8H+\\nmatrix+O2→4Cyto coxidized +2H2O+4H+intermem\\nThe electron transport chain is very similar in all organisms. In eukaryotes the com-\\nplexes are found in the inner mitochondrial membrane, while in prokaryotes the complexes\\nare found in the plasma membrane.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3efed420-7deb-43e1-b8b2-2183a04114b8', embedding=None, metadata={'page_label': '275', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\nENERGY CURRENCIES ARE INTERCONVERTIBLE 269\\nATP\\nADPH+ inside \\nmitochondrionH+ outside \\nmitochondrion high energy\\nlow energy\\nFigure 12.9. Currency conversion: ATP synthase interconverts the H+gradient and ATP.\\nATP Synthase\\nThe energy stored in the hydrogen ion gradient can be used to drive the synthesis of ATP from\\nADP and inorganic phosphate. The conversion is accomplished by a marvelous “nanoma-\\nchine ”called ATP synthase. At its most basic, the operation is a simple one and is summa-\\nrized in Figure 12.9. Once again, the circle symbolizes the linkage between the enzyme andcarrier functions. ATP synthase is a protein complex that is also known as mitochondrialATPase (because it can carry out the reverse reaction, hydrolyzing ATP and driving H\\n+out\\nof the mitochondria) or F 1F0ATPase (because it can be prepared in two fragments F 1and\\nF0) while others call it complex V .\\nIn most mitochondria ATP synthase is the only route by which the large electrochemical\\ngradient for H+generated by the electron transport chain can be relieved. If the supply of\\nADP runs out, then ATP synthase will stop moving H+in. If this happens, the electron\\ntransport chain will stop too because it cannot keep pushing H+out against a greater and\\ngreater electrochemical gradient. Another way of stopping ATP synthase is to block itsoperation with the antibiotic oligomycin; again, if this is done, the electron transport chainwill stop too.\\nThe coupling between ATP synthesis and electron transport can be broken if hydrogen\\nions can be carried back across the membrane by an alternative route. Weak organic acidssuch as 2,4-dinitrophenol can do this. The dinitrophenol is uncharged in the intermembranespace and in this form is suf ﬁciently hydrophobic to diffuse into the membrane. If it enters\\nthe matrix, it will tend to lose its hydrogen ion as the hydrogen ion concentration is muchlower. The result is movement of hydrogen ions down their concentration gradient. Suchmolecules are called “uncouplers. ”In the presence of an uncoupler electron transport (and\\nso oxidation of fuels) will carry on regardless of the availability of ADP or a working ATPsynthase.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='107d45a4-56ef-464b-8d3d-ea9d61f9862c', embedding=None, metadata={'page_label': '276', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\n270 ENERGY TRADING WITHIN THE CELL\\nf\\nIN DEPTH 12.1 Brown Fat\\nTriacylglycerols are stored within specialized fat cells in the body. Most fat cells\\nare composed of a droplet of lipid surrounded by a thin layer of cytoplasm with anucleus and a few mitochondria. The resulting tissue is white in color and simplyreleases or stores fatty acids in response to the needs of the organism. This is thekind of fat that is typically found around our kidneys and under the skin.\\nA second kind of fat is found in babies. Brown fat cells not only have stored tria-\\ncylglycerols but are also rich in mitochondria, the cytochromes of the mitochondriagiving the brown color. Brown fat is a heat-generating tissue. A channel selectivefor H\\n+called thermogenin is found in the inner mitochondrial membrane. As fast\\nas the electron transport chain pushes H+ions out of the mitochondrial matrix,\\nthey ﬂow through thermogenin back down their electrochemical gradient into\\nthe matrix. In other cells this ﬂux would only occur through ATP synthase and\\nwould be tightly coupled to the production of ATP from ADP. The presence ofthermogenin uncouples the phosphorylation of ADP from the ﬂow of electrons to\\noxygen so that the electron transport chain can work ﬂat out even though there is\\nno ADP available, as long as the cell contains triacylglycerols, which can be oxidizedto regenerate NADH (see βoxidation, page 290). This generates a lot of heat and\\nhelps the infant maintain body temperature. Large blocks of brown fat are foundin animals that hibernate.\\nA similar uncoupling mechanism is used by some plants to generate heat. Some\\narum lilies rely on carrion-eating ﬂies for pollination. Uncoupled mitochondria at\\nthe base of the ﬂower generate suf ﬁcient heat to evaporate the evil-smelling\\nodorants used to attract the ﬂies.\\nSodium/Potassium ATPase\\nThis is a single protein in the plasma membrane, with a carrier action that is tightly linked to\\nan enzymatic one. Under normal conditions, it hydrolyzes ATP. The energy released drivessodium ions up their electrochemical gradient out of the cell. The Na\\n+/K+ATPase also\\nmoves potassium ions the other way, into the cytosol. For every ATP hydrolyzed, three Na+\\nions are moved out and two K+ions are moved in. Figure 12.10 summarizes the reaction\\nin terms of energy currencies.\\nNa+ in extracellular\\nmedium\\nNa+ in cytosol ADPATP\\nhigh\\nenergy\\nlow\\nenergy\\nFigure 12.10. Currency conversion: the sodium –potassium ATPase interconverts ATP and the Na+\\ngradient.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ea2551b7-fc88-4094-a4bb-22e271e626a7', embedding=None, metadata={'page_label': '277', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\nENERGY CURRENCIES ARE INTERCONVERTIBLE 271\\nADP/ATP Exchanger\\nATP synthase makes ATP inside the mitochondrion. For the ATP to be available to the rest of\\nthe cell, there needs to be a mechanism to enable it to leave the mitochondrion for use in thecytosol. This job is performed by another carrier, the ADP/ATP exchanger. This protein hasno enzymatic action; it simply moves ADP in one direction across the mitochondrial innermembrane and ATP in the opposite direction. In most eukaryotic cells the carrier operatesin the direction shown in Figure 12.3. Carriers such as the Na\\n+/K+ATPase together with\\nmany synthetic processes use up ATP in the cytosol, producing ADP. Then ADP enters themitochondria by the ADP/ATP exchanger and is reconverted to ATP by ATP synthase. ATPthen leaves the mitochondrion with the help of the ADP/ATP exchanger.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 12.2 Chemicals That Interfere with Energy Conversion Are Highly\\nToxic\\nThe electron transport chain, ATP synthase, and the sodium/potassium ATPase together run the\\nenergy currency market and are vital to the cell. Chemicals that interfere with them are very toxic.Rotenone, the most widely used rat poison, blocks complex I of the electron transport chain whilecyanide blocks complex IV . Digitalis, from foxgloves, blocks the Na\\n+/K+ATPase.\\nATP\\nADPH+ inside \\nmitochondrionH+ outside \\nmitochondrion high energy\\nlow energyKrebs cycle\\nElectron transport chain ATP synthase Na+/K+ \\nATPaseNa+ in cytosolblock electron\\ntransport chaincyanide\\nblocks Na+/K+\\nATPasedigitalis\\nNa+ in extracellular\\nmedium\\nNAD+NADHrotenone\\nPhotosynthesis\\nThe fuels that our cells oxidize to give NADH and FADH 2must of course come from some-\\nwhere! Ultimately that somewhere is the ﬁxation of carbon dioxide into more complex\\nmolecules, which is in turn driven by light energy captured by photosynthesis. Photosyn-\\nthesis also generates the oxygen in the atmosphere. Some 1010tons of carbon are ﬁxed\\nevery year.\\nThedark reactions thatﬁx carbon will be described in Chapter 13 (page 302). However,\\nthe overall effect can be summarized as:\\n6CO 2+18ATP +12NADPH +12H 2O→C6H12O6+18ADP +18Pi+12NADP++6H+', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='27c4f985-6e8c-4b8e-9801-51c2f9f22172', embedding=None, metadata={'page_label': '278', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\n272 ENERGY TRADING WITHIN THE CELL\\nf\\nIN DEPTH 12.2 ATP Synthase, Rotary Motor, and Synthetic Machine\\nATP synthase can be seen in the electron microscope as tiny beads attached to the\\nthe matrix face of the inner mitochondrial membrane by a thinner region. We nowknow that this enzyme is a molecular motor. The beadlike head or F\\n1fragment is\\nattached to proteins that span the membrane —the F 0part.\\nMost of the bulk of F 1is composed of three αand three βsubunits arranged al-\\nternately. The βsubunits bind ATP or ADP and Pi. They are joined to the membrane-\\nbound F 0part by a central stalk composed of γandεsubunits. The F 0part has 10 –14\\nsubunits (called c subunits) forming a ring embedded in the membrane and an asubunit, which binds to the outside of the ring of c subunits. The head of the F\\n1\\npart is connected to the base by the b subunit, which connects to the a subunit inthe membrane.\\nWe have described how the splitting of ATP can drive a conformational change\\nin the protein that has bound it and that this conformational change can producemovement (page 241). The converse of this is true —a conformational change can\\nbe produced by movement and this conformational change can drive a reaction.\\nThe stalk of the F\\n1part (the γsubunit) is not symmetrical. It ﬁts into the center\\nof the circle of alternating αandβsubunits and makes different interactions with\\neachβsubunit. The βsubunits have three conformations possible —a T conforma-\\ntion, which binds ATP very strongly, an L conformation, which binds ADP and Pi butcannot release them, and an O or open conformation, which can release boundnucleotides. The T conformation has been shown experimentally to catalyze theconversion of ADP and Pi to bound ATP with an equilibium constant near 1 (atequilibrium the concentrations of substrates and products are 1 : 1). Rotation ofthe stalk ( γsubunit) alters the conformation of the βsubunits (which do not rotate\\nbecause the b subunit holds the head stationary).\\nImagine that all three βsubunits have bound nucleotide. One is in the T con-\\nformation, one has the L conformation, and one is O. The T conformation subunithas ADP +Pi interconverting to ATP, and the O and L conformations will have bound\\nADP and Pi. As the stalk rotates 120\\n◦, it converts the T conformation subunit into\\nO so that the ATP can escape and more ADP +Pi can bind. The same movement\\nconverts the previous O state subunit into L, trapping the ADP and Pi, and thesubunit in the L state converts into T. One third of a rotation generates one ATP,so a full rotation of the stalk generates three as each of the three βsubunits cycles\\nfrom T to O to L and back to T.\\nThis sounds too good to be true! Experiments using the reverse reaction indi-\\ncate it is true. Isolated F\\n1parts were attached to a glass surface so the stalk was\\nsticking upward. The γsubunit (the stalk) was linked to a ﬂuorescently labeled\\nactin ﬁlament. This could be seen using a ﬂuorescence microscope, and the movies\\nacquired are published on the Web (see CBASC website). Addition of ATP causedtheﬁlament to rotate, and it could be demonstrated that the hydrolysis of a single\\nATP caused a 120\\n◦rotation!\\nThis has not told us how hydrogen ion ﬂow causes the rotation. This remains\\nspeculation, but Howard Berg and George Oster have proposed an appealing hy-pothesis. In the membrane F\\n0has a ring of c subunits with the single a subunit\\nattached. The a subunit has two half length hydrogen ion channels. Each c subunithas an aspartate residue in the center that is exposed to the hydrophobic interiorof the membrane. If this residue has lost its hydrogen ion, the negative charge isvery unfavorable in the hydrophobic environment of the membrane interior. Nor-mally this residue will have its hydrogen attached as the uncharged form can betolerated in the hydrophobic environment of the membrane interior.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a93c39f4-441e-4a9e-ac4e-e781ff601345', embedding=None, metadata={'page_label': '279', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\nATP SYNTHASE 273\\nATP\\nADP + PiF1matrix\\nintermembrane spaceH+\\nH+F0 abb\\ncγεδαα\\nββ\\nβ α\\nThe a subunit makes close contact with two c subunits so their aspartates are\\nexposed to the two half channels. One channel opens toward the matrix and one\\ntoward the intermembrane space. The channels provide a hydrophilic environmentin which the aspartic acids can lose their hydrogen ions. Once its aspartate has ion-ized, a c subunit is stuck —it cannot rotate away from the channel on the a subunit.\\nIt is much more likely that an aspartate will pick up a hydrogen ion again if it isin contact with the half channel leading to the intermembrane space as the hy-drogen ion concentration is around four times higher than it is on the matrix side,and the large transmembrane voltage acts to pull H\\n+toward the matrix. Similarly\\na c subunit contacting the half channel leading to the matrix is much more likelyto be ionized. Molecular movements ensure that as soon as a c subunit becomesuncharged, it will be able to move into the hydrophobic membrane. As all of thec subunits are linked in a ring, this movement is a rotation of the ring. The adja-cent c subunit, which was in contact with the half channel to the matrix, is verylikely to have a charged aspartate. Movement of this into the membrane is unfa-vorable so the only allowed rotation is movement toward the other channel onthe a subunit —the one contacting the hydrogen-ion-rich intermembrane space. It\\nis likely to become uncharged and can rotate into the membrane repeating theprocess. The ring of c subunits acts to carry hydrogen ions from the intermembranespace back to the matrix. The rotation has to be clockwise because of the designof the a subunit, and the hydrogen ion concentration difference across the mem-brane. The size of the ring seems to vary with the species, but 10 c subunits arefound in ATP synthase from yeast. Ten subunits would use 10 hydrogen ions forone rotation, and one rotation generates 3 ATP so 3.33 hydrogen ions are used perATP generated.\\nThe experiment described above tends to con ﬁrm this theory as the rotation\\nof the stalk of the immobilized F\\n1part is anticlockwise, and this is driven by the\\nhydrolysis of ATP so the rotation should be clockwise for ATP synthesis.\\nThe understanding of how cells generate and interconvert their energy cur-\\nrencies is one of the triumphs of twentieth-century biological science. In 1997 theNobel prize for chemistry was awarded to Jens Skou, Paul Boyer, and John Walkerfor working out the structure and mechanism of ATP synthase.\\nThe F\\n0component of ATP synthase is evolutionarily related to the motor that\\ndrives the bacterial ﬂagellum (Example 12.1 on page 264): both generate rotational\\nforce from the H+gradient. Which came ﬁrst is a matter for speculation.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='21f567f0-cfc4-46e2-a279-89e2ff1f311e', embedding=None, metadata={'page_label': '280', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\n274 ENERGY TRADING WITHIN THE CELL\\nThe process requires 3ATP and 2NADPH for every carbon dioxide ﬁxed. These are regener-\\nated by the light reactions of photosynthesis. In plants both light and dark reactions occur\\nin the chloroplast, which, like the mitochondrion, is thought to have originated through\\nendosymbiosis (page 12). Figure 3.6 (on page 59) shows the structure of a chloroplast. Ithas an outer membrane that has porins, and so is permeable to many molecules, and aninner membrane that is impermeable. The infolding of the inner membrane has been carriedfurther in chloroplasts than in mitochondria and has produced discrete membrane structurescalled thylakoids. Within the thylakoids is the thylakoid space or lumen, and outside the\\nthylakoids is the stroma. The dark reactions take place in the stroma and the light reactions\\nin the thylakoid membranes.\\nWhen a molecule absorbs a photon of light of an appropriate wavelength (determined\\nby the electronic structure of the molecule), an electron jumps to a higher energy level.Normally it drops back to its normal level, and the energy of the photon is lost as heat.In photosynthesis, however, this high-energy electron is captured by another moleculeand transferred through a chain of electron carriers rather as we have seen in mitochondrialelectron transport. This allows hydrogen ions to be pumped across the thylakoid membrane,setting up a hydrogen ion electrochemical gradient with the inside of the thylakoid spacehaving a positive voltage, and a lower pH, compared to the stroma. An ATP synthase, verysimilar to that found in mitochondria, then allows hydrogen ions to ﬂow back from the\\nthylakoid space into the stroma, driving the synthesis of ATP.\\nLike electron transport in mitochondria, photosynthetic electron transport takes place\\nin large multiprotein complexes that span the thylakoid membrane and uses mobile carriersto shuttle electrons between the complexes. There are actually two different photosystemscalled photosystem I and photosystem II, which work together. The fundamental pigmentinvolved is chlorophyll. Chlorophyll is rather like heme (Fig. 9.21 on page 206) but differs\\nin detail and importantly has a magnesium ion rather than iron at the center and has a longhydrocarbon tail called phytol. Two types of chlorophyll are present: chlorophyll a andchlorophyll b. Structural differences give them different colors, which means they absorblight of different wavelengths.\\nMost of the chlorophyll molecules are antenna chlorophylls, which act to harvest\\nlight and pass the energy to a pair of reaction center chlorophylls. Antenna chlorophylls aresupplemented by other pigments such as carotenoids, which absorb different wavelengthsof light to enable ef ﬁcient use of most of the light energy reaching the plant.\\nLet us start with the reaction center chlorophylls in PSII. These chlorophylls absorb\\nlight at 680 nm and so the reaction center is called P680. As soon as an electron in oneof these is raised to a higher energy (either by energy transfer from antenna chlorophyllsor by direct light absorbtion), it is immediately captured by an adjacent molecule (calledpheophytin). This generates a charge pair: the chlorophyll is +ve and the pheophytin −ve.\\nThe complex structure prevents the charges neutralizing by rapidly transferring the electronfrom pheophytin to a quinone called plastoquinone, which is very similar in structure\\nto ubiquinone (coenzyme Q) in mitochondria (Fig 12.6). This leaves a positively chargedreaction center chlorophyll P680\\n+. Also at the reaction center is a cluster of four manganese\\nions together with bound water. An electron is abstracted from the bound water —this is\\ndone four times resulting in oxidation of two molecules of water to give a molecule of O 2.\\nThe net result is a movement of electrons from water to plastoquinone, which is reducedto plastoquinol (QH\\n2). During this process hydrogen ions are pumped from the stroma into\\nthe thylakoid space.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='7e5a55b3-23a3-482a-8efc-cdeb0095c0f5', embedding=None, metadata={'page_label': '281', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\nENERGY CURRENCIES ARE INTERCONVERTIBLE 275\\nPlastoquinol passes its electrons to a copper protein called plastocyanin. This process is\\ncarried out by a large membrane protein complex called cytochrome bf,which is analogous\\nto complex III in the mitochondrial electron transport chain. The other photosystem now\\ncomes into play. Here again we have a pair of reaction center chlorophyll a moleculescalled P700 (as they absorb light of wavelength 700 nm) at the center of a complex of otherchlorophyll molecules and accessory pigments. The electron lifted to a higher energy levelis captured by another chlorophyll and immediately passed to plastoquinone. This leaves a+ve charge at the reaction centre (P700\\n+). This charge is neutralized by an electron from\\nplastocyanin, which came from photosystem II.\\nThe captured electron passes from plastoquinone to an iron –sulfur cluster, which in turn\\npasses electrons to a protein called ferredoxin. Ferredoxin can pass electrons to an enzyme\\ncalled ferredoxin-NADP+reductase, which reduces NADP+to NADPH. The process has\\nused light energy to oxidize water to oxygen and to drive electrons through a series ofcarriers to produce NADPH and the movement of hydrogen ions from the stroma to thethylakoid space. This hydrogen ion gradient is used by ATP synthase to make ATP fromADP and inorganic phosphate. The ATP and NADPH are used in the dark reactions to ﬁx\\ncarbon dioxide into complex molecules.\\nIf there is no NADP\\n+available to accept the electrons from ferredoxin, photosynthesis\\nwould have to stop were it not for an alternate route. Ferredoxin can also pass its electronsback to the cytochrome bfcomplex, resulting in a cyclic ﬂow of electrons. As the electrons\\npass along the chain of carriers, hydrogen ions are pumped into the thylakoid space. Thisprocess, which allows ATP synthesis without production of either NADPH or oxygen, iscalled cyclic photophosphorylation.\\nAll Carriers Can Change Direction\\nIn a normal animal cell, the primary source of energy is the Krebs cycle (page 283). Thisregenerates NADH from NAD\\n+, making at the same time a small amount of ATP. Because\\nthe NADH currency is always being topped up, while the others are being used, the directionof operation of the energy conversion systems is usually that shown in Figure 12.3. However,all the carriers are reversible.\\nYeast cells in a wine barrel, or muscle cells in the leg of a sprinter, are anaerobic; there\\nis no oxygen available. In this situation cells can make NADH, but the electron transportchain cannot drive H\\n+out of the mitochondria because there is no molecular oxygen waiting\\nto be reduced by NADH. Instead, the cell ’s energy needs are met by anaerobic glycolysis\\n(page 286), which makes ATP. Figure 12.11 shows how the cell maintains the amountsof energy currencies. Any drain on the mitochondrial H\\n+gradient is counteracted by ATP\\nsynthase running in the opposite direction from that shown in Figure 12.3. ATP is hydrolyzedand H\\n+ions are pushed out of the mitochondrion. The ADP/ATP exchanger also reverses\\nits direction. ATP is regenerated by anaerobic respiration in the cytosol and is used up byATP synthase in the mitochondrial matrix.\\nChloroplasts do not contain the enzymes of the Krebs cycle, so at night they have no\\ninternal source of energy and are dependent on their host cell. ATP from the cytosol (mostof it created by mitochondria) enters the chloroplast by the ADP/ATP exchanger, whichis carrying ADP and ATP in the opposite direction from that shown in Figure 12.3. ATPsynthase hydrolyzes ATP and maintains the H\\n+gradient.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='384106ee-66c2-4f99-99ef-61bb2396c39a', embedding=None, metadata={'page_label': '282', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\n276 ENERGY TRADING WITHIN THE CELL\\nATP\\nADPhigh energy\\nlow energyNa+ in cytosol\\nATP synthase \\ndoes not function in absence\\nof molecular oxygenNa+ in extracellular\\nmedium\\nH+ inside\\nmitochondrionH+ outside\\nmitochondrion\\nNa+/K+\\nATPaseelectron transport chainAnaerobic glycolysis\\nFigure 12.11. Energy ﬂow between the currencies in an anaerobic cell.\\nHealthy cells maintain themselves in a steady state in which none of the energy cur-\\nrencies are allowed to run down. The direction in which energy moves between the four\\ncurrencies depends on the primary source of energy for that cell.f\\nIN DEPTH 12.3 Can It Happen? The Concept of Free Energy\\nWe know from experience that some chemical reactions give out heat (the burning\\nof organic matter for instance) while others absorb heat (so heat is used to makethe reaction happen). The study of how energy affects matter and particularly howit affects chemical reactions is called thermodynamics. A vast amount of research\\nin the late nineteenth and early twentieth centuries led to the development ofthis complex ﬁeld. To make a very long story short, a process results in a change in\\ntwo parameters: heat and entropy. Heat we are familiar with and understand as\\nthe random motion of molecules, but entropy may be unfamiliar. Entropy is thedegree of disorder in a system: Entropy always tends to increase. The melting of iceis favorable because it results in an increase in entropy as the ordered, crystallineice becomes more random water.\\nA physical process can absorb or emit heat and/or result in an increase or\\ndecrease in entropy. If we wish to know whether a process can occur or not, weneed to consider both. The concept of free energy was formulated by the Americanphysical chemist J. Willard Gibbs in 1878. It is now called the Gibbs free energy andgiven the symbol Gin his honour. The change in free energy in a process, /Delta1G(/Delta1is\\nthe Greek upper-case delta used to denote a change) is\\n/Delta1G=/Delta1H−T/Delta1S\\nWhere /Delta1His the heat change (called enthalpy ) in the process, Tis the absolute\\ntemperature in degrees Kelvin, and /Delta1Sis the change in entropy. If /Delta1Gfor a process\\nis negative, it will proceed, although the rate may be very slow in the absence ofa catalyst. If it is not negative, the process cannot occur.\\nUnfortunately, we can only measure changes and not absolute values for G,\\nH,and Sin chemical processes. This dif ﬁculty is avoided by de ﬁning standard', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a47b4185-5d65-4626-9218-4d16cedf3ac7', embedding=None, metadata={'page_label': '283', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\nENERGY CURRENCIES ARE INTERCONVERTIBLE 277\\nstates so we can make comparisons. A free-energy change in a process under\\nstandard conditions is denoted /Delta1G◦. Biochemists use a slightly different standard\\nstate, which is in water at pH 7, and this is shown as /Delta1G◦′.\\nThe free-energy change is related to the equilibrium constant for a reaction.\\nConsider the reaction\\nA+B⇀↽C+D\\nKeq=([C][D]) /([A][B])\\nAnd\\n/Delta1G=/Delta1G◦′+RTln{([C][D]) /([A][B]) }\\nwhere Ris the gas constant and Tthe absolute temperature.\\nAt equilibrium /Delta1G=0s o\\n/Delta1G◦′=−RTlnK′\\neq\\nwhere K′\\neqis the equilibrium constant under standard conditions.\\nConditions in the cell are not usually the standard ones, but if we know /Delta1G◦′for\\na reaction and the relevant concentrations, we can calculate /Delta1Gfor the reaction.\\nFor this book we have calculated /Delta1Gvalues under human cellular conditions by\\nusing the following reasonable concentration values:\\n[ATP] =6 mmol liter−1\\n[ADP] =0.6 mmol liter−1\\n[AMP] =0.2 mmol liter−1\\n[Pi]=4.5 mmol liter−1\\nGlucose =250µmol liter−1\\nGlucose-6-phosphate =10 mmol liter−1\\n[NAD+] / [NADH] =4 in the mitochondrial matrix\\n[O2]=21µmol liter−1\\nTemperature 37◦C\\nFor example, we have used the hexokinase reaction (page 258) to illustrate cou-\\npling a reaction with a negative free-energy change with one which has a positivefree energy change.\\nGlucose +ATP→Glucose-6-phosphate +ADP+H\\n+\\nUnder biochemical standard conditions this reaction has a /Delta1G◦′of−36 kJmol−1.\\nHowever, inside cells the concentration of glucose-6-phosphate is much higher thanthe concentration of unphosphorylated glucose; when this and other concentra-tion effects are taken into account, the calculated overall free-energy change is−11 kJmol\\n−1: The reaction will certainly still proceed but is not as strongly favored\\nas the /Delta1G◦′would suggest.\\nThe/Delta1Gvalues we have given for the four energy currencies are those that\\napply in an aerobic animal cell, as shown in Figure 12.3, where each of the con-version reactions has a negative /Delta1G, as it must if it is to proceed as shown. Under\\nconditions where one or more of the conversion reactions is operating in the op-posite directions, the concentrations, and therefore /Delta1Gvalues, will of course be\\ndifferent.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3d26f83d-bc8c-4b0b-b38b-bd92362e1f32', embedding=None, metadata={'page_label': '284', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\n278 ENERGY TRADING WITHIN THE CELL\\nSUMMARY\\n1.Reactions with a positive Gibbs free-energy change ( /Delta1G) can be caused to happen in\\na cell by linking them with a second reaction with a larger, negative /Delta1G. The second\\nreaction drives the ﬁrst.\\n2.The majority of such reactions in the cell are driven by one of four energy currencies —\\nNADH, ATP, the hydrogen ion gradient across the inner mitochondrial membrane,\\nand the sodium gradient across the plasma membrane.\\n3.The electron transport chain in the mitochondrial inner membrane converts betweenenergy as NADH and energy in the hydrogen ion gradient.\\n4.ATP synthase in the mitochondrial inner membrane converts between energy in thehydrogen ion gradient and energy as ATP.\\n5.The sodium/potassium ATPase in the plasma membrane converts between energy inATP and energy in the sodium gradient.\\n6.In a healthy cell, none of the energy currencies are allowed to run down. The directionof energy exchange between the four currencies depends on the primary source ofenergy available to the cell.\\nFURTHER READING\\nV oet, D., and V oet, J. D. 2003. Biochemistry, 3rd ed. New York: Wiley\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.A reaction that has a positive /Delta1Gwill\\nA. proceed, although the rate may be very slow in the absence of an enzyme.\\nB. only proceed in the presence of an enzyme.C. generate water.D. not proceed, even if an enzyme is present.E. be coupled to a carrier.\\n2.Which of the following is nota cellular energy currency?\\nA. The sodium gradient across the plasma membraneB. GlucoseC. ATPD. NADHE. The H\\n+gradient across the mitochondrial inner membrane\\n3.Which of these statements about the proteins responsible for converting between the four\\nenergy currencies is notcorrect?\\nA. They are all reversible.\\nB. They are all located on the mitochondrial inner membrane.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='92c57479-18bb-4d6f-bd7e-4c315eb6a171', embedding=None, metadata={'page_label': '285', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\nANSWERS TO REVIEW QUESTIONS 279\\nC. Drugs that block them are highly toxic.\\nD. They are all carriers.E. None of the above are true.\\n4.Hydrogen ions that enter the mitochondrial matrix through ATP synthase release a lot\\nof energy because\\nA. the mitochondrial matrix is electrically negative with respect to the cytosol.\\nB. H+is at a higher concentration in the mitochondrial matrix than in the cytosol.\\nC. the mitochondrial inner membrane contains thermogenin.D. H\\n+is at a higher concentration in the cytosol than in the mitochondrial matrix.\\nE. There is a large electrochemical gradient favoring H+entry.\\n5.The overall reaction catalyzed by the electron transport chain is\\nA. Glucose +ATP→Glucose-6-phosphate +ADP\\nB. ATP +GDP⇀↽ADP+GTP\\nC. NADH +H++1/2O2→NAD++H2O\\nD. ATP +H2O→ADP+H2PO 4−\\nE. NADH +OH−+ATP→NAD+H++ADP+HPO 42−\\n6.Which of the following statements is incorrect?\\nA. Mitochondria may have been derived in evolution from a prokaryote.\\nB. Mitochondria have their own DNA.C. Mitochondria have a large negative voltage across their inner membrane.D. Mitochondria increase their generation of ATP in cells starved of oxygen.E. Mitochondria generate NADH.\\n7.The electron transport chain in mitochondria\\nA. accepts electrons from NADH, thus oxidizing it to NAD+.\\nB. passes electrons to oxygen, thus reducing it to water.C. uses the hydrophobic molecule ubiquinone to carry electrons between integral membrane protein\\ncomplexes.\\nD. uses the hydrophilic molecule cytochrome cto carry electrons between integral membrane\\nprotein complexes.\\nE. all of the above.\\nANSWERS TO REVIEW QUESTIONS\\n1.D.Reactions with negative /Delta1Gcan proceed; those with positive /Delta1Gcannot proceed. Enzymes\\nare catalysts and can therefore speed up the rate of a reaction, but they cannot cause a reaction toproceed if the /Delta1Gis positive.\\n2.B.Although glucose is the major extracellular fuel supplied by the blood to our cells, it is not\\nused as an energy currency inside cells. Rather, on entering cells energy is supplied toglucose to\\nconvert it to glucose-6-phosphate.\\n3.B.The sodium/potassium ATPase is located on the plasma membrane.\\n4.E.Answers B and C are false: H\\n+is at a lower concentration in the mitochondrial matrix than\\nin the cytosol, and thermogenin (an alternative route for H+entry expressed in a few cell types,\\nespecially brown fat) will not act to increase the energy released by H+as it enters. Of answers\\nA, D, and E the last is the best answer because it states the complete source of the energy. It is\\ntrue that hydrogen ions that enter the mitochondrial matrix through ATP synthase release energybecause the mitochondrial matrix is electrically negative with respect to the cytosol, but this is not', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='9d8e338a-af98-4efb-bd35-8aa8809fd130', embedding=None, metadata={'page_label': '286', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-12 WY001-Bolsover-v2.cls September 15, 2003 21:34\\n280 ENERGY TRADING WITHIN THE CELL\\nthe only source of energy —the concentration gradient also favors H+entry. Similarly it is true that\\nhydrogen ions that enter the mitochondrial matrix through ATP synthase release energy because\\nH+is at a higher concentration in the cytosol than in the mitochondrial matrix, but this is not the\\nonly source of energy —the electrical gradient also favors H+entry.\\n5.C.Answers A, B, and D are reactions that occur somewhere in the cell, but they are not the reaction\\ncatalyzed by the electron transport chain. Answer E is a nonsense equation; in particular, there isno compound NAD as written, and nicotine adenine dinucleotide is always found as NADH or asNAD\\n+. Note that answers A, B, D, and E involve ATP: ATP is nota substrate or product of the\\nelectron transport chain.\\n6.D.Under anaerobic conditions mitochondria cannot generate ATP and are dependent on the rest of\\nthe cell for an ATP supply. In an anaerobic animal or yeast cell, the ATP is generated by glycolysis.\\n7.E.All of answers A through D are true. In particular, while ubiquinone and cytochrome cboth\\nfunction as electron carriers, ubiquinone is hydrophobic and remains within the lipid bilayer, whilecytochrome cis a hydrophilic molecule found in the aqueous intermembrane space.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='c14206b1-f6a0-4771-a5a3-966574bfe31c', embedding=None, metadata={'page_label': '287', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\n13\\nMETABOLISM\\nIn Chapter 12 we described the energy currencies NADH and ATP. In this chapter we\\nwill describe the chemical pathways that regenerate these currencies when their levelsare depleted. We will then consider some other important chemical pathways, some thatoperate in all cells, others that are found only in certain types of organism or in specializedbiochemical centers like the liver. Figure 13.1 is an overview of the main metabolic pathwayswithin a cell.\\nAll the processes that occur within a living cell are ultimately driven by energy taken\\nfrom the outside world. Green plants and some bacteria take energy directly from sun-light. Other organisms take compounds made using sunlight and break them down torelease energy, a process called catabolism. The most common way of breaking down\\nthese food compounds is to oxidize them, that is, to burn them but in a controlledway. The energy trapped in energy currencies can then be used for the building, repair,and homeostatic processes termed anabolism. The collective term for all of the reac-\\ntions going on inside a cell is metabolism. All metabolic reactions share some general\\nfeatures:\\nrThey are catalyzed by enzymes.\\nrThey are universal in that all organisms show remarkable similarity in the mainpathways.\\nrThey involve relatively few types of chemical reaction.\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n281', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='f26150bf-7730-4ef6-8689-1b1d5e998982', embedding=None, metadata={'page_label': '288', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\n282 METABOLISM\\nglucose\\npyruvateamino\\nalanine\\nonly\\nacetyl CoA\\nKrebs cycle\\nNADH\\n(FADH2)\\nNAD+\\n(FAD)CO2\\nCO2glycogen\\ngluconeogenesis glycolysis\\nfatty acids\\nβ-oxidationfatty acid\\nsynthesisfatsacids\\nFigure 13.1. Overview of metabolism.\\nrThey are controlled, often by modulation of key regulatory enzymes.\\nrThey are compartmentalized within cells. In eukaryotes different sets of metabolic\\nreactions are carried out in different organelles. In animals and plants this com-partmentalization is carried further, so that in some cases different reactions takeplace in different body organs. Prokaryotes too show compartmentalization of asimpler sort—some processes are associated with regions on the inner face of the cellmembrane.\\nrThey usually involve coenzymes, molecules that are second substrates in a numberof different reactions.\\nrThe pathways that break particular molecules down are different from those used tosynthesize them. This allows them to be controlled separately.\\nMolecules that are common second substrates for a number of reactions are called coen-\\nzymes. These are either energy currencies themselves (such as NADH and ATP) or carriersof chemical groups such as coenzyme A. ATP is used to drive other reactions by transfer ofa phosphate group. The resultant ADP can be rephosphorylated back to ATP. The energyto do this can be derived from NADH passing its electrons to the electron transport chain,which converts the NADH back to its oxidized form NAD\\n+. The total amount of ATP +\\nADP in a cell is relatively constant, but the proportion that is ATP can vary. Similarlythe amount of NADH +NAD\\n+is constant, but the proportion present as NADH can\\nvary.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='64cde811-0394-4623-b9bc-558da70f320c', embedding=None, metadata={'page_label': '289', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\nTHE KREBS CYCLE: THE CENTRAL SWITCHING YARD OF METABOLISM 283\\nTHE KREBS CYCLE: THE CENTRAL SWITCHING\\nYARD OF METABOLISM\\nThe fuels we take in in the diet are mainly fats, proteins, and carbohydrates. At the very\\ncenter of metabolism is a cycle of reactions that takes place in the mitochondrial matrix. Thecycle is named after its discoverer, Hans Krebs, and is also known as the tricarboxylic acid(TCA) cycle or the citric acid cycle. The foods we eat are converted to the two-carbon unitacetate, CH\\n3COO−. The acetate is not free but is carried by a coenzyme called coenzyme\\nA. Acetate bound to coenzyme A —acetyl-CoA for short —is then fed into the Krebs cycle\\nand may be completely oxidized to carbon dioxide and water. In the process the energycurrency NADH is produced. The Krebs cycle is central to carbohydrate, fat, and aminoacid metabolism.\\nWe shall ﬁrst describe the Krebs cycle and then look at the other pathways that interact\\nwith it. The reactions are shown in Figure 13.2.\\nCOO\\nC\\nCH2\\nCOO\\noxaloacetateOCOOCoA\\nCH2\\nC COO HO\\nCH2\\nCOO\\ncitrate\\nH HOCOO\\nCH2\\nC COOH\\nC\\nCOOisocitrate\\nOCOO\\nCH2\\nCH2\\nC\\nCOO2-oxoglutarate\\nO\\nC CoA\\nCoACoA\\nNADHNADHNAD\\nGTPGDP   P i\\nCH2\\nCH2\\nCOO\\nsuccinyl CoACOO\\nCH2\\nCH2\\nCOO\\nsuccinateCOO\\nCH\\nHC\\nCOOfumarateCCOONADH\\nFADH2\\nFADH HO\\nCH2 \\nCOOmalate\\nCO2\\nCO2CH3C CoAO\\nacetyl CoA\\n1\\n2\\n3\\n4\\n5678\\nNADNAD\\nFigure 13.2. The Krebs cycle.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b5bd4267-4d33-43de-8f44-94f1e30b735b', embedding=None, metadata={'page_label': '290', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\n284 METABOLISM\\n1. A molecule of acetyl-CoA enters the cycle, and the acetate (two carbons) is combined\\nwith the four-carbon molecule oxaloacetate, making citrate (as carboxyl groups are\\nionized at the pH in the cell, we normally speak of them as ions; so we say “citrate ”\\nrather than citric acid).\\n2. Citrate is rearranged to isocitrate.\\n3. In the ﬁrst oxidation step isocitrate is oxidized to 2-oxoglutarate (sometimes called\\nα-ketoglutarate). A carbon is lost as CO 2and NAD+is reduced to NADH.\\n4. A second oxidation converts 2-oxoglutarate to succinyl-CoA. A second carbon\\nl e a v e sa sC O 2and again NAD+is reduced to NADH. Note that the product is\\nattached to coenzyme A. This reaction is catalyzed by oxoglutarate dehydrogenase.\\n5. The bond between succinate and CoA is now broken and the energy released\\nused to drive the phosphorylation of a GDP to GTP. γ-phosphate groups can be\\nswapped between nucleotides, so this GTP can be used to regenerate ATP from\\nADP.\\n6. Succinate is oxidized to fumarate. The oxidant in this case is ﬂavin adenine dinu-\\ncleotide (FAD) and not NAD+so an FADH 2is produced. FADH 2, the reduced form\\nof FAD, does not carry as much energy as NADH but like NADH is used to driveH\\n+up its electrochemical gradient out of the mitochondrial matrix (page 265). The\\nenzyme involved is succinate dehydrogenase, which is actually part of the electrontransport chain (page 267).\\n7. Water is added to the double bond in fumarate making malate.\\n8. Malate is oxidized to oxaloacetate in a reaction catalyzed by malate dehydrogenase.\\nOne NAD\\n+is reduced to NADH. The starting compound oxaloacetate has been\\nregenerated and is ready to accept another acetyl group to start another turn of the\\ncycle.\\nThe reactions of the Krebs cycle can be summarized as\\nCH 3CO-CoA +(3 NAD++FAD)+GDP+Pi+3H2O→\\nCoA-H +(3 NADH +FADH 2)+GTP+2CO 2+3H+\\nwhere Pi represents an inorganic phosphate ion.\\nFROM GLUCOSE TO PYRUVATE: GLYCOLYSIS\\nGlucose is an important fuel for most organisms, and glycolysis is the main pathway thatenables it to be used. The word glycolysis simply means the breakdown of glucose. It is an\\nancient pathway that can function without oxygen and indeed is thought to have evolvedbefore there was much oxygen in the atmosphere. The pathway is present in almost all cellsand takes place in the cytosol. It is shown in Figure 13.3.\\n1. Free glucose is phosphorylated on carbon 6 to produce glucose-6-phosphate. As we\\ndiscussed earlier (page 258), this reaction, catalyzed by hexokinase, is driven by thefree energy available from ATP.\\n2. Glucose-6-phosphate is isomerized to fructose 6-phosphate.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='f3fe55a3-698b-457e-8682-9d07aed27669', embedding=None, metadata={'page_label': '291', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\nFROM GLUCOSE TO PYRUVATE: GLYCOLYSIS 285\\nOHOHOCH2\\nOCH2OH\\nCH2OH\\nOH\\nOHOH HO\\nglucose glucose-6-phosphatecatalyzed by\\nhexokinase\\ncatalyzed by\\nphosphofructokinase\\ncatalyzedby aldolasecatalyzed bytriosephosphateisomerase\\ncatalyzed byglyceraldehyde3-phosphatedehydrogenasefructose-6-phosphateATP\\nATPADP\\nADPP\\nOCH\\nOH\\nOHOH HOHOOO P\\nPP\\nP\\nO\\nOO P P =OHOHOCH2\\nCH2OOO HCH2O\\nfructose-1,6-biphosphate glyceraldehyde-3-phosphate\\n1,3-bisphosphoglycerate 3-phosphoglyceratedihydroxyacetone\\nphosphate (DHAP)+HOPP PP1 P2P2\\nP4P4\\nP7\\nP8P5\\nP6P3\\nCH2O\\nCH2OHOO H\\nPiPP\\nPP\\nCH2OPPPPCHC C C C\\nC\\nO O\\nOHCHCCC\\nCH2OPPO O\\nOHCHCCC\\n2-phosphoglycerateCH2OHH2O\\nPPO O\\nOCHCCC\\nphosphoenolpyruvateCH2PPO O\\nOCCC\\nC\\npyruvateCH3O–O\\nOCCCCATPNADH\\nADP\\nATP ADP\\nP9NAD\\nFigure 13.3. Glycolysis breaks glucose down into pyruvate.\\n3. Fructose-6-phosphate is phosphorylated to produce fructose-1,6-bisphosphate.\\nAgain ATP is used and the reaction is carried out by phosphofructokinase. This\\nreaction commits the sugar to being broken down and used to provide energy ratherthan used for other purposes. Phosphofructokinase is regulated allosterically, beinginhibited by ATP.\\n4. The fructose-1,6-bisphosphate is now split into two halves. Each of the halves\\nhas a phosphate attached. The two products are dihydroxyacetone phosphate andglyceraldehyde-3-phosphate. The enzyme is aldolase.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='194e367c-85f8-46b1-97dd-6856cc631744', embedding=None, metadata={'page_label': '292', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\n286 METABOLISM\\n5. Triose phosphate isomerase interconverts dihydroxyacetone phosphate and\\nglyceraldehyde-3-phosphate. This allows both halves of the original glucose to be\\nused.\\nThe glucose-6-phosphate has now been converted into two molecules of glyceraldehyde-3-phosphate. Each of the following reactions occurs twice for each molecule of glucose fedinto the pathway.\\n6. Glyceraldehyde-3-phosphate is oxidized in a reaction that also attaches one inor-\\nganic phosphate ion to give 1,3-bisphosphoglycerate. NAD\\n+is the oxidant and a\\nNADH is produced. The enzyme is called glyceraldehyde-3-phosphate dehydroge-nase.\\n7. One of the phosphate groups from bisphosphoglycerate is transferred to ADP. This\\nis a process called substrate-level phosphorylation. ATP has been made from ADPby a single enzyme, without any involvement of ATP synthase. We are left with3-phosphoglycerate.\\n8. After a rearrangment to 2-phosphoglycerate, water is removed leaving phospho-\\nenolpyruvate.\\n9. In another substrate-level phosphorylation phosphoenolpyruvate transfers its phos-\\nphate group to ADP leaving pyruvate.\\nFigure 13.4 illustrates the various ways that the cell can use pyruvate. If it is to be used\\nto make fatty acids or to be oxidized in the Krebs cycle, it is carried into the mitochondrialmatrix. Here it is oxidized and decarboxylated by the complex enzyme pyruvate dehydro-genase (Fig. 13.4 a). NAD\\n+is used as the oxidant, producing NADH. Coenzyme A is also\\nadded. The product is therefore acetyl-CoA, which may enter the Krebs cycle or can beconverted to fatty acids or some other molecules. Overall glycolysis has used up two ATPmolecules but has produced four: a net gain of 2 ATP per glucose.\\nThe Krebs cycle is limited by the availability of oxaloacetate. This may run low as some\\nbiosynthetic pathways use one or the other component of the Krebs cycle as their startingmaterial. If this happens new oxaloacetate is made from pyruvate (Fig. 13.4 b). Pyruvate\\nmay also be converted to the amino acid alanine in a transamination reaction (Fig. 13.4 c;\\nFig. 11.8 on page 250).\\nGlycolysis Without Oxygen\\nThe leg muscles of a sprinter cannot be supplied with oxygen rapidly enough to supply themitochondria. Muscle cells need to make ATP by a method that does not require oxygen.As we have seen glycolysis itself produces two ATPs per glucose. Can the cell let thepyruvate pile up and use it when oxygen becomes available again? No, because glycolysisas far as pyruvate also converts one NAD\\n+to NADH. If this was all that happened, the cell\\nwould quickly convert all its NAD+to NADH, and glycolysis would stop as the cell does\\nnot have oxygen available for the mitochondria to oxidize the NADH back to NAD+.T o\\nsolve this problem cells reduce pyruvate to lactate (Fig. 13.4 d) and in doing so regenerate\\nthe NAD+needed to allow glycolysis to carry on. The buildup of lactic acid in poorly', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='87813071-72ae-466f-ab74-803999bcd4f5', embedding=None, metadata={'page_label': '293', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\nFROM GLUCOSE TO PYRUVATE: GLYCOLYSIS 287\\npyruvatein eukaryotes\\nin microorganisms(a)\\n(b)\\n(c)\\n(d)acetyl CoA\\npyruvate oxaloacetate\\npyruvate\\nlactate pyruvatealaninefatty acids\\nother lipids\\noxidation in \\nKrebs cycleNAD+\\nCoANADH\\nNAD+NADHNAD+NADH\\nNADH NAD+ATP ADPCO2CO2\\n(e)pyruvate acetaldehyde ethanol\\nCO2\\nFigure 13.4. Pyruvate can be used in a number of ways.\\noxygenated muscles is thought to cause the pain of cramp. The blood carries the lactate\\nto the liver where it is reoxidized to pyruvate. When we stop using the muscle, the bloodcan supply more oxygen to the muscle and the need for lactate production abates. Redblood cells lack mitochondria and are entirely dependent on glycolysis for their energyneeds.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='10dafb3f-2eb3-452c-9f4d-3ec4768e4fc4', embedding=None, metadata={'page_label': '294', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\n288 METABOLISM\\n✞\\n✝\\x04\\n✆Medical\\nRelevance\\n13.1Sleeping Sickness\\nOn page 259 we discussed chronic fatigue syndrome, a problem of un-\\nknown cause characteristic of Western countries. In contrast sleeping sickness,\\ncaused by a parasite called Trypanosoma brucei, is a serious disease in Africa. Trypanosomes are\\nsingle-celled eukaryotes with a complex life cycle, part of which is spent in the human bloodstream.Here they reveal a prodigious appetite for glucose. Each tiny parasite can consume its own bodyweight in glucose every hour. The reason for the parasite ’s insatiable demand for glucose is\\ntwofold. First, trypanosomes dispense with mitochondrial ATP production and rely on anaerobicrespiration to generate their ATP, so they only make 2 ATPs per glucose rather than the 30 or sothat they could if their mitochondria were still working. Second, trypanosomes do not have anyglycogen or fat stores and are therefore entirely dependent on a continuous supply of glucose inthe host blood. The enormous consumption of glucose by the parasites leaves little for the host,who is overcome by extreme langor such that even keeping the eyes open is an unsurmountableeffort.\\nSome microorganisms —particularly yeasts —regenerate their NAD+in a different way.\\nPyruvate is ﬁrst decarboxylated to acetaldehyde (also called ethanal) and then reduced to\\nethanol by alcohol dehydrogenase, which uses NADH and regenerates NAD+. One molecule\\nof CO 2is also produced (Fig. 13.4 e).\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 13.1 Anaerobes Good and Bad\\nThe fact that yeasts regenerate their NAD+by making the gas CO 2together with ethanol has been\\nutilized in breadmaking, brewing, and winemaking since prehistoric times. Other microorganismsare like our muscles and generate lactate under anaerobic conditions. These too have their usesin the food industry: yogurts, many cheeses, sauerkraut, and dill pickles all rely on lactic acidreleased in anaerobic respiration. On the minus side, some food-spoiling bacteria can only functionwhen there is no oxygen. These obligate anaerobes include the deadly Clostridium botulinum\\n(page 232).\\nGlycogen Can Provide Glucose for Glycolysis\\nThe polysaccharide glycogen (page 30) is used as a store of glucose particularly in liver\\nand muscle cells. We saw in Chapter 2 how the glycosidic bond can be hydrolyzed with thebroken ends of the bond being sealed with groups from a water molecule, so that a hydrogenatom is added to one side of the broken bond and a hydroxyl group is added to the other(page 44). The enzyme glycogen phosphorylase speci ﬁcally breaks the α(1→4) glycosidic\\nbond in glycogen but seals the broken ends with groups from inorganic phosphate, so thata hydrogen atom is added to one side of the broken bond and a phosphate group is addedto the freed glucose monomer (Fig. 13.5). The resulting glucose-1-phosphate is readilyconverted to glucose-6-phosphate for glycolysis. Breaking up glycogen this way is moreenergy ef ﬁcient than simply hydrolyzing it (as happens in the intestine) since the ATP that\\nwould otherwise be required to make glucose-6-phosphate from free glucose is saved. Theoccasional α(1→6) links that attach side arms to the glycogen chain are broken by other\\nenzymes.\\nOne of the many important roles of the liver is to maintain the level of glucose in\\nthe blood —it is the most important of the circulating fuels and is the primary fuel for red', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='f1325923-f7bd-46c9-826f-f393395638f3', embedding=None, metadata={'page_label': '295', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\nFROM GLUCOSE TO PYRUVATE: GLYCOLYSIS 289\\nCCH2OH\\nHO\\nOC\\nCOH\\nOHH\\nHCH\\nOHCH CCH2OH\\nHO\\nOC\\nCOHH\\nHCH\\nOHCH CCH2OH\\nHO\\nOC\\nCOHH\\nHCH\\nOHCH\\nglycogen\\nchaincontinuesP HOO\\nCCH2OH\\nHO\\nC\\nCOH\\nOHH\\nOH\\nHCH\\nOHOCH CCH2OH\\nHO\\nOC\\nCOHH\\nHCH\\nOHCH CCH2OH\\nHO\\nOC\\nCOHH\\nHCH\\nOHCH\\nglycogenchaincontinues\\nP OO\\nglucose-1-phosphateOO\\nO+\\n+\\nFigure 13.5. Glycogen phosphorylase cleaves a glucose monomer off glycogen and phosphory-\\nlates it.\\nblood cells and the brain. Glycogen stores in the liver can provide glucose when none is\\navailable from the gut. The glucose carrier (page 238) allows glucose to enter and leavethe liver cells but cannot transport phosphorylated sugars, so glucose-6-phosphate must beconverted to free glucose for transport out into the extracellular medium and from thereto the blood. To do this, liver has the enzyme glucose-6-phosphate phosphatase, whichremoves the phosphate group. Muscle also has stores of glycogen, but these are for its ownuse: it does not have glucose-6-phosphate phosphatase so cannot release glucose into theextracellular medium.\\nGlucose May Be Oxidized to Produce Pentose Sugars\\nCells need the ﬁve-carbon sugar ribose to manufacture nucleotides. This is made from\\nglucose in an oxidative pathway called the pentose phosphate pathway (Fig. 13.6). Thispathway also provides reducing power for biosynthetic reactions in the form of NADPH.Glucose-6-phosphate undergoes two oxidations generating a molecule of a pentose phos-phate, CO\\n2, and two molecules of NADPH. Like glycolysis, these reactions occur in the\\ncytoplasm. The pentose phosphate pathway interacts with glycolysis, and this allows it toperform a different function. In a cell that does not need lots of ribose, but does need lotsof NADPH for biosynthesis, the pentose is recycled by being combined with glycolyticintermediates. Going around the loop six times converts a glucose entirely to 6CO\\n2, giving\\n12 NADPH for biosynthesis. Although the pentose phosphate pathway breaks up sugar togive CO\\n2and the strong reducing agent NADPH, it is not used for energy production: only\\nin the mitochondria, where NADH can be used to power the electron transport chain, canreducing agents be converted to other energy currencies.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='9b51fcf8-beb5-4214-8999-0004b907b58d', embedding=None, metadata={'page_label': '296', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\n290 METABOLISM\\nglucose 6-phosphate ribulose 5-phosphate  +  CO2\\nribose-5-phosphatefructose-6-phosphate\\nfructose-1,6-bis phosphate\\npyruvateglyceraldehyde-3-phosphate dihydroxy-\\n       acetone phosphateNADPH NADPH\\nSugarinterconversions\\nFigure 13.6. The reactions of the pentose phosphate pathway are shown in black with the reac-\\ntions of glycolysis in green.\\nFROM FATS TO ACETYL-CoA: βOXIDATION\\nCells must be able to break down fatty acids (page 39) and use the energy released, whether\\nthe fatty acids come from the lipolysis of triacylglycerols in fat droplets or triacylglycerolsand phospholipids in food. Before they can be broken down, the fatty acids are coupled tocoenzyme A to give an acyl-CoA in a reaction driven by the conversion of ATP to AMP. Aspiral of reactions in the mitochondrial matrix called βoxidation (Fig. 13.7) then oxidizes\\nthe fatty acyl-CoA. Each turn of the spiral shortens the fatty acyl chain by two carbons,releasing an acetyl-CoA and generating an NADH and an FADH\\n2for each acetyl-CoA. The\\nacetyl-CoA may be oxidized in the Krebs cycle or converted to other molecules.\\nDuring fasting, fat cells supply fuels for other parts of the body. Because fats are\\ninsoluble and fatty acids have only a limited solubility, fat cells convert fatty acids intosoluble, circulating fuels called ketone bodies (Fig. 13.8) (the word ketone means any\\nchemical containing a carbon atom with single bonds to two other carbons and a double bondto an oxygen). The fundamental ketone body is acetoacetate, which the liver synthesizesfrom acetyl-CoA. Acetoacetate is then reduced to 3-hydroxybutyrate. These two moleculesare important circulating fuels in mammals. Heart muscle, for instance, prefers ketonebodies to glucose as a fuel source.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2ced5b71-77e0-425e-af34-c8868246e620', embedding=None, metadata={'page_label': '297', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\nFROM FATS TO ACETYL-CoA: βOXIDATION 291\\nRC H2CH2CH2C CoAO\\nFAD\\nFADH 2\\nRC H2CH CH C CoAO\\nRC H2CH2CH C CoAO OH\\nRC H2CH2C C CoAO O\\nRC H2CH3CoAC C CoAO OH2Ooxidation\\nNAD\\nNADHoxidationhydration\\nCoA\\ncleavage by CoA\\nfatty acid chain\\nshortened by\\n2 carbonsfatty acid\\nacetyl CoA\\nFigure 13.7. ß oxidation of fatty acids produces acetyl-CoA.\\nCH3\\nCO HH\\nCHH\\nCOO\\n3-hydroxy\\nbutyrateCH3\\nCO\\nCHH\\nCOO\\nacetoacetate2 Acetyl - CoA\\nNAD+NADH\\nFigure 13.8. Formation of ketone bodies from acetyl-CoA.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='60550ef6-c15d-44ec-b094-648931ff20fd', embedding=None, metadata={'page_label': '298', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\n292 METABOLISM\\nAMINO ACIDS AS ANOTHER SOURCE\\nOF METABOLIC ENERGY\\nProtein forms a considerable part of the animal diet, even in vegetarians. It is broken down\\nto free amino acids during digestion. These amino acids can be used for the biosynthesisof new proteins in the cell, but those in excess of this need can serve as metabolic fuels.To begin this process, the amino groups are removed in a process called transamination byaminotransferases such as alanine glyoxylate aminotransferase (page 249). The resultingcarbon skeletons are then converted to intermediates in the Krebs cycle or to acetyl-CoA.Since there are 20 amino acids, there are many different steps, but the overall effect isthat the amino groups are passed to oxaloacetate or 2-oxoglutarate to form aspartate andglutamate, respectively. The amino groups on glutamate and aspartate are then convertedto urea (see below) for excretion.f\\nIN DEPTH 13.1 The Urea Cycle—The First Metabolic Cycle Discovered\\nDietary amino acids that are not required for protein synthesis have their amino\\ngroups passed to oxaloacetate or 2-oxoglutarate to form aspartate and glutamate,respectively. The amino groups on each of these amino acids are then used in theliver to make urea, which is excreted in the urine. The ﬁrst step is the release of\\nthe amino group as a free ammonium ion.\\n(a)NADNADH    H\\nN  \\nHH\\nH HCH2\\nOOCCH2\\nC OCOO2-oxoglutarate\\nammonium+CH2\\nH2OOOCCH2\\nCH NH3COOglutamate\\n+\\nThis ammonium ion, plus the amino groups on aspartate, is then used to make\\nurea in the urea cycle, which operates only in the liver. First, the ammonium ion is\\nconverted to carbamoyl phosphate. This is then combined with the α-amino acid\\nornithine to produce citrulline. In turn, citrulline is joined with aspartate, whichcarries a second nitogen into the cycle, producing argininosuccinate. Argininosuc-cinate is now cleaved to release fumarate and arginine. Urea is removed fromarginine to regenerate ornithine, which is now ready for another cycle. Urea canthen be excreted in the urine, eliminating two nitrogen atoms per urea molecule.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='23422b98-7a8c-4966-825b-7f2c0bc3f8a9', embedding=None, metadata={'page_label': '299', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\nAMINO ACIDS AS ANOTHER SOURCE OF METABOLIC ENERGY 293\\nThe overall cycle converts two ATP to ADP and one ATP to AMP. This is energetically\\nequivalent to a total of four ATP to ADP conversions, since a fourth ATP is requiredto turn the AMP into ADP in the reaction:\\nATP+AMP→←2ADP\\nThe fumarate can enter the Krebs cycle and be converted to malate, which is\\noxidized to oxaloacetate, which can (among other things) be transaminated backto aspartate to carry in another nitrogen.\\nammonium\\ncarbamoyl\\nphosphate\\ncitrullinepyrophosphatearginino-\\nsuccinateaspartate\\nornithinecarbon\\ndioxideHH\\nN H\\nH\\nC O\\nO2ATP\\n2ADP    Pi\\nAMPOONH2\\nP O\\nOC O\\nCH2NH3\\nCH2\\nOOCCH2\\nCHNH3\\nNH3\\nONH2\\nC NH2CH2CH2\\nOOCCH2\\nCHH NNH2\\nNH3C OCOO\\nOOCCH2\\nCH NH3fumarateCOO\\nOOCHC\\nCH\\nCH2CH2\\nOOCCH2\\nCHH NN\\nNH3C NH2COO\\nOOCCH2\\nCH\\nATP\\narginineCH2CH2\\nOOCCH2\\nCHH N\\nNH3C NH2NH2\\n(b)urea+\\nOrnithine plays the same role in the urea cycle as oxaloacetate in the Krebs\\ncycle: it accepts the incoming molecule, undergoes a series of interconversions,\\nand is regenerated allowing the cycle to begin again. The availability of ornithinedetermines the rate at which the cycle can operate. Arginine is a protein aminoacid and is normally present in the diet as a source of ornithine. Vice versa the ureacycle can make arginine so this is not normally considered to be an essential aminoacid for adults. Arginine is, however, essential in the diet of growing children asthe net protein synthesis of growth needs more of it than can be supplied fromthe urea cycle without draining too much and so impeding the cycle.\\nSince the urea cycle only occurs in the liver, liver failure results in a buildup\\nof ammonium ions in the blood. This is toxic to nerve cells, producing ﬁrst mental\\nconfusion and ﬁnally coma and death.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='43efe592-0572-470d-89af-90753215228a', embedding=None, metadata={'page_label': '300', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\n294 METABOLISM\\nIt is interesting to note that of the three major energy sources in the diet —carbohydrate,\\nfat, and protein —only protein is not used as an energy store in our bodies. We make speci ﬁc\\nproteins when we need them but never simply as a way of storing amino acids. Of course,\\nif one eats a lot of protein, one puts on weight: once the amino acids have been convertedto carbon skeletons or to acetyl-CoA, these can then be used to make glucose and henceglycogen (page 298) or to make fat (page 300).\\n✎\\n✍\\x0c\\n✌Medical\\nRelevance\\n13.2Phenylketonuria\\nAll the phenylalanine in the diet that is not required for protein synthesis is con-\\nverted to tyrosine by the enzyme phenylalanine hydroxylase. If this enzyme is\\nmissing or defective through mutation, then there is a serious problem. The standard waythe body deals with amino acids is for aminotransferases (page 249) to transfer the NH\\n3+\\ngroup to aspartate or glutamate, which are then processed by the urea cycle (In Depth 13.1 onpage 292). However, the product formed when phenylalanine has its NH\\n3+group transferred\\naway, a phenylketone called phenylpyruvate, cannot be further metabolized. Both phenylalanineand phenylpyruvate therefore accumulate in the body. About 1 in 20,000 new-born babieshave this defect, which is called phenylketonuria because the phenylpyruvate appears in theurine.\\nPhenylalanine\\nPhenylpyruvateTRANSAMINATIONTyrosinePHENYLALANINE\\nHYDROXYLASE\\nCH2\\nCH2\\nCO\\nCOOCH NH3CH2OH\\nCH OOC NH3OOC', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='dbc387fb-78b4-4289-b012-71fa3002c1a4', embedding=None, metadata={'page_label': '301', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\nMAKING GLUCOSE: GLUCONEOGENESIS 295\\n✞\\n✝\\x04\\n✆The disease is a devastating one, and, if left untreated, babies of only a few weeks old begin\\nto suffer severe neurological damage. All newborn babies in the United States and in most other\\ndeveloped countries are therefore tested for phenylketonuria within the ﬁrst week of life. This\\nrepresents the largest genetic screening program carried out by the medical profession. The test,often called the heel-prick test or more correctly the Guthrie test after the scientist who developedit, is relatively simple. A drop of blood taken from a baby ’s heel is dried onto a small ﬁlter paper\\ndisk. Disks from hundreds of infants can be tested at the same time by placing their disks ontoan agar plate containing bacteria that require phenylalanine for growth. If the bacteria grow, thenthe baby is at risk from phenylketonuria and blood from the baby will be retested a few dayslater to ensure the infant does indeed suffer from phenylketonuria. The Guthrie test is simple andcost-effective and can test for phenylketonuria irrespective of the mutation in DNA that has causedthe problem.\\nIf detected in the ﬁrst weeks of life the prognosis for the patient is good. Affected infants are\\nfed a strict diet that provides just enough phenylalanine for protein synthesis but no more. Thistreatment, carried on to maturity, is very effective and individuals develop normally.\\nMAKING GLUCOSE: GLUCONEOGENESIS\\nSuch is the importance of glucose that there is a pathway for its synthesis from other\\nmolecules. Gluconeogenesis enables animals to maintain their glucose levels even during\\nstarvation. It makes use of some of the enzymes of glycolysis but uses different enzymesto bypass the steps that are not freely reversible. Figure 13.9 shows the glycolytic pathwayin green. The new reactions that allow the entire pathway to run in reverse are shown inblack.\\nNew step 1: Pyruvate is carboxylated to make oxaloacetate (a reaction that also serves\\nto“top-up ”levels of oxaloacetate for the Krebs cycle). The oxaloacetate is moved from the\\nmitochondria into the cytosol where it is converted to phosphoenolpyruvate and CO\\n2. The\\nsource of the phosphate group is GTP, which is converted to GDP.\\nFrom phosphoenolpyruvate all of the reactions are reversible until fructose bisphosphate\\nis reached. However, the reaction catalyzed by phosphofructokinase is not reversible andthis step is avoided by:\\nNew step 2: One of the phosphoester bonds on fructose bisphosphate is hydrolyzed by\\na phosphatase.\\nThe interconversion of fructose-6-phosphate and glucose-6-phosphate is easily re-\\nversible. The ﬁnal production of free glucose is accomplished by:\\nNew step 3: The other phosphoester bond is hydrolyzed by glucose-6-phosphate phos-\\nphatase.\\nGluconeogenesis is an expensive process: The conversion of 2 pyruvates to a glucose\\nmolecule uses 4 ATP, 2 GTP, and 2 NADH.\\nDifferent compounds can be fed into the gluconeogenesis pathway as appropriate.\\nGlutamate is converted to 2-oxoglutarate (page 292), fed into the Krebs cycle, and tappedoff as oxaloacetate to feed gluconeogenesis. Lactate and alanine are converted to pyruvateand hence oxaloacetate. Glycerol, released from lipids by hydrolysis, is phosphorylatedand then oxidized to generate dihydroxyacetone phosphate. However, animals cannot makeglucose from fats. Figures 13.1 and 13.2 (pages 282 and 283) might suggest that they could:useβoxidation to make acetyl CoA, feed the acetyl-CoA into the Krebs cycle, and tap off', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ba41721b-aa9b-4988-918a-53dcd842b8fa', embedding=None, metadata={'page_label': '302', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\n296 METABOLISM\\nOHOHOCH2\\nOCH2OH\\nCH2OH\\nOH\\nOHOH HO\\nglucose glucose-6-phosphate\\n+ H2O\\nH2O +fructose-6-phosphateATP\\nATPADP\\nADPP\\nOCH\\nOH\\nOHOH HOHOOO P\\nPP\\nP2P3\\nP2\\nP1P1OHOHOCH2\\nOCH2O\\nfructose-1,6-biphosphateHOPP PP\\nCH2OO H\\nglyceraldehyde\\n-3-phosphatedihydroxyacetone\\nphosphate (DHAP)+CH2O\\nCH2OHOO HPP\\nPPCHCCC\\n1,3-bisphosphoglyceratePi\\nCH2OPP PPPPO O\\nOHCHCCCPi\\nPi\\nO\\nO P P =\\n3-phosphoglycerateCH2OO O\\nOHCHCCCADP\\nADPATP\\nATP\\n2-phosphoglycerateCH2OH\\nH2OPPO O\\nOCCC\\nHC\\nphosphoenolpyruvateCH2PPO O\\nOCCCC\\npyruvateCH3O O\\nOCCCCADP ATP\\noxaloacetateCOOCH2OCCCCOOADP\\nGTP GDP+ CO2\\nCO2ATPO\\nC\\nNADHNAD+\\nFigure 13.9. Gluconeogenesis allows glucose to be made from pyruvate.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b9a96682-0690-4b5e-96f4-4e6b24fbeb77', embedding=None, metadata={'page_label': '303', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\nMAKING GLUCOSE: GLUCONEOGENESIS 297\\noxaloacetate to feed gluconeogenesis. However, in passing around the Krebs cycle from\\ncitrate to oxaloacetate, the two carbons delivered by the acetyl-CoA have already been lostas CO\\n2.\\nf\\nIN DEPTH 13.2 The Glyoxylate Shunt\\nMany plants store triacylglycerol oils in their seeds. We have emphasized that mam-\\nmals cannot make glucose from acetyl-CoA as two carbon dioxides are lost for everytwo carbons on acetyl-CoA that enter the Krebs cycle. Plants, however, can convertacetyl-CoA into glucose. Special organelles called glyoxysomes carry out a seriesof reactions called the glyoxylate pathway or glyoxylate shunt, which effectivelyshort-circuits the Krebs cycle.\\nAs in the Krebs cycle acetyl-CoA combines with oxaloacetate to give citrate,\\nwhich is converted to isocitrate. However, rather than undergoing oxidation andloss of CO\\n2, the isocitrate is split to give succinate and the two carbon glyoxylate.\\nGlyoxylate is then combined with another acetyl-CoA to yield malate, which isoxidized to oxaloacetate. In parallel the succinate is also converted to oxaloac-etate using the standard Krebs cycle reactions. The overall effect of one completecycle is therefore to generate one oxaloacetate that can enter gluconeogenesis(page 295).\\nCoA\\nCoACH3O\\nC CoA\\nCH3O\\nC CoACOO\\nCOOC\\nCH2Ooxaloacetate\\nmalateisocitrate\\nsuccinateCOO\\nCOOCOOCH2\\nC\\nCH2HO\\nCOO\\nCOOH C\\nCH2HOCOO\\nCOOCOOCH2\\nC H\\nH C HO\\nglyoxylateCOOO C H\\nCOO\\nCOOCH2\\nCH2COO\\nCOOCH\\nHCcitrate\\nfumarate', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='044705ae-e355-4c6f-aa6e-c879fff0dd17', embedding=None, metadata={'page_label': '304', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\n298 METABOLISM\\nMany bacteria can also carry out these reactions. Indeed, some bacteria can\\ngrow on acetate as the sole carbon source because they are able to combine free\\nacetate with coenzyme A to form acetyl-CoA, which is then fed into the glyoxylatepathway as required.\\nMAKING GLYCOGEN: GLYCOGENESIS\\nGlucose is stored as the polymer glycogen. Glucose polymerization, as a stand-alone reac-\\ntion, has a positive /Delta1Gand will not occur. The synthesis of glycogen is therefore driven by\\nthe hydrolysis of nucleoside triphosphates —not only ATP but also UTP. If phosphorylated\\nglucose is not available, then it is made by hexokinase using ATP (page 284). Glucose-1-phosphate then reacts with UTP to make UDP-glucose (Fig. 13.10). Glycogen synthasethen transfers glucose from the UDP to the growing glycogen chain (Fig. 13.11). Otherenzymes insert the α(1→6) branches at intervals.\\nCH2OH\\nOH\\nOH\\nOH OHHO\\nglucose-1-\\nphosphateuridine\\ntriphosphate\\nH2O2Pi(pyrophosphate)\\nUDP-glucoseOO\\nOOO\\nOP O P OO\\nOPO CH2\\nOO P OO\\nCH2OH\\nOH\\nHOHO O\\nOO P OOO\\nOH OHCH2\\nOO O\\nP O P +OO\\nNHN\\nOO\\nNHNHP2O73−O\\nOO P\\nFigure 13.10. Uridine diphosphate glucose is synthesized from UTP and glucose-1-phosphate.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ec65148b-0725-4611-af14-0b4887b0b3dd', embedding=None, metadata={'page_label': '305', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\nMAKING GLYCOGEN: GLYCOGENESIS 299\\nCCH2OH\\nHO\\nC\\nCOHH\\nOH\\nHCHH\\nO\\nO OHCO\\nOP\\nUDP-glucoseCCH2OH\\nHO\\nC\\nCOHH\\nHCHH\\nOHCCCH2OH\\nHO\\nC\\nCOHH\\nOH\\nHCHH\\nOHC\\nO O\\nglycogen ( n residues)\\nCCH2OH\\nHO\\nC\\nCOHH\\nHCHH\\nOHCCCH2OH\\nHO\\nC\\nCOHH\\nHCHH\\nOHC\\nO OCCH2OH\\nHO\\nC\\nCOHH\\nOH\\nHCHH\\nOHC\\nO\\nglycogen ( n +1 residues)\\nGlycogen synthesized from UDP-glucose monomers.UDPO P O uridineOCO\\nOP O P O uridine\\nO O\\nP O P O uridineO\\nO\\nO O\\nFigure 13.11. Glycogen is synthesized from UDP-glucose monomers.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 13.2 Diabetes, Starvation, Ketone Bodies, and the Odor\\nof Sanctity\\nDuring starvation the only source of glucose is gluconeogenesis, which must use body protein amino\\nacids as a source of precursors. It is clearly far better to use fat reserves than to degrade body protein,so those tissues that can shift to using ketone bodies as their main fuel source do so. Even the brainchanges over to get three quarters of its energy needs from ketone bodies. Fat stores can keep astarving human being going for weeks.\\nDiabetes mellitus is characterized by a similar shift to using ketone bodies. Diabetes arises when\\neither no insulin is produced (Type 1) or cells are unable to respond to this hormone (Type 2). In bothcases the body switches to a starvation type metabolism and the ketone bodies 3-hydroxybutyrateand acetoacetate are made to excess. Acetoacetate is chemically unstable and slowly loses carbondioxide to form acetone. When ketone bodies are at a high concentration, suf ﬁcient acetone is present\\nto give the breath the fruity smell of acetone. This pathological condition is described as ketosis andis characteristic of untreated diabetes.\\nMedieval saints were given to morti ﬁcation of the body by voluntary starvation. Perhaps\\nthe so-called odor of sanctity was simply acetone on the breath of starving and thus ketoticsaints?\\nCH3\\nCH3\\nCO2CH3\\nCO CO\\nCHH\\nCOO\\nacetoacetateacetoneH', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='56852e33-4486-4b7e-a448-48767f46f4b9', embedding=None, metadata={'page_label': '306', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\n300 METABOLISM\\nMAKING FATTY ACIDS AND GLYCERIDES\\nAll cells need fatty acids for membrane lipids. Fat cells make large amounts of fat (triacyl-\\nglycerols) in times of plenty. The basic machinery is a multienzyme complex (in bacteria)or a multidomain protein (in eukaryotes) that uses the substrate acetyl-CoA. In both casesthe growing fatty acid chain is not released: it swivels from enzyme to enzyme or domain todomain in the array, adding two carbons for each complete cycle until the limiting length of16 carbons is reached: The product, palmitic acid, is then released. Although the reactionslook similar (Fig. 13.12), the process is not a reversal of βoxidation (page 290). It uses\\nentirely different enzymes, takes place in the cytosol rather than in the mitochondria, andis separately regulated. Like much of biosynthesis it is reductive, and the reducing powercomes not from NADH but from the closely related dinucleotide NADPH.\\nInitially acetyl-CoA is carboxylated to malonyl-CoA. From here on, however, fatty acid\\nsynthesis does not use free coenzyme A to carry the growing chain but instead uses a proteincalled acyl carrier protein (ACP). The malonyl residue is transferred to ACP from malonyl-CoA. This condenses with a molecule of acetyl-ACP (made from an acetyl-CoA) to give afour-carbon molecule with the release of ACP and CO\\n2. The four-carbon acetoacetyl-ACP\\nis next reduced to hydroxybutyryl-ACP. The next enzyme (molecule or domain) removeswater, leaving a double bond, which is again reduced to give butyryl-CoA. Another malonyl-CoA is condensed with this and the cycle continues. Finally a chain 16 carbons long hasbeen made (palmitic acid). At this point it is hydrolyzed from the ACP. Overall 14 NADPHmolecules, 1 acetyl-CoA, and 7 malonyl-CoA molecules have been used to make palmiticacid. Palmitic acid is then used by enzymes on the endoplasmic reticulum that extend thechains and that can introduce double bonds. Mammals cannot, however, synthesize all thedifferent kinds of fatty acids that they need for their membranes and must obtain essentialfatty acids in food (page 42).\\nThe main use of fatty acids is to make glycerides, both triacylglycerols for storage in\\nfat globules and phospholipids for membranes. The process uses glycerol phosphate, whichis usually generated by reduction of dihydroxyacetone phosphate. The fatty acid is thenswopped in, replacing the phosphate group, which leaves as inorganic phosphate.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 13.3 Eating Well, Getting Fat\\nGlycerol phosphate, the building block from which lipids and phospholipids are made, is obtained\\nin most cells by reducing dihydroxyacetone phosphate. This in turn is generated in the glycolyticpathway. This means that adipose cells can only make fats when glucose is abundant —when we\\nare eating lots of sugar or carbohydrate. In contrast, liver cells have an enzyme called glycerolkinase that phosphorylates glycerol to make glycerol phosphate directly, so that some lipids andphospholipids can be made even during times of fasting.\\nSYNTHESIS OF AMINO ACIDS\\nNitrogen is an important constituent of proteins and nucleic acids and many other molecules\\nimportant in cells. Although nitrogen gas is plentiful, making up 80% of the atmosphere,it is inert. It is chemically a very dif ﬁcult task to break the triple bond and reduce nitrogen\\ngas to ammonia that can be used for incorporation into biomolecules. Chemical fertilizersare made using the Haber process, which ﬁxes nitrogen by the use of pressures of around', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='c816e534-6618-4f6d-a55f-eb93ef26c5c0', embedding=None, metadata={'page_label': '307', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\nSYNTHESIS OF AMINO ACIDS 301\\nCH2C ACPO\\nCH3CO\\nCH2C ACPO\\nCH3CHOH\\nC ACPO\\nCH3CH CH\\nC ACPO\\nCH3CH2CH2ACP\\nH2Ocondensation\\nreduction\\nreductiondehydrationACP\\nNADPH\\nNADP+\\nNADPH\\n+CO2acyl carrier\\nprotein++ CH3C CoAO\\nacetyl CoA\\nCH3C ACPO\\nacetyl ACPCH3C ACP CoAO\\nacetyl ACP\\nACP\\nacyl carrier\\nprotein+\\n++CoAHCO3\\nATP ADPbicarbonate+ CH3C CoAO\\nacetyl CoACH2C CoAO\\nCO\\nO malonyl CoA\\nCH2C CoAO\\nCO\\nmalonyl CoA\\nCH2C ACPO\\nCO\\nmalonyl ACP\\nCH2C ACPO\\nCO\\nmalonyl ACPprocess repeats\\nto produce a chain16 carbons longCH2C ACPO\\nCO\\nmalonyl ACP O O\\nO\\nONADP+\\nFigure 13.12. Synthesis of fatty acids.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ddfc33be-0f41-4628-99fe-c254030bbe8a', embedding=None, metadata={'page_label': '308', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\n302 METABOLISM\\n300 atmospheres and temperatures of 500◦C. Some nitrogen is ﬁxed naturally by lightning,\\nbut most is ﬁxed by various types of prokaryotes, which possess the complex enzyme\\ncalled nitrogenase. Some of these nitrogen- ﬁxing organisms are free-living while others\\nform symbiotic relationships with plants: the root nodules of legumes are a good example\\nof plants generating a special environment for their nitrogen- ﬁxing symbionts.\\nNitrogenase consists of two protein complexes: a reductase and an iron –molybdenum\\nprotein. Both proteins have iron –sulfur clusters. The reductase accepts electrons from donors\\nsuch as ferredoxin (page 275), which can get its electrons through photosynthetic electrontransport or from other reactions. The reductase passes electrons to the iron –molybdenum\\nprotein (also called dinitrogenase). Nitrogen gas is bound to the iron –molybdenum cofactor\\nwhere it is reduced to ammonia. Ammonia in water forms ammonium ions NH\\n4+, which\\ncan be incorporated into the amino acid glutamine and from there into other amino acidsand other molecules.\\nThe overall reaction is\\nN\\n2+8e−+8H++16 ATP →2NH 3+H2+16 ADP +16 Pi\\nwhere e−represents an electron.\\nNitrogenase is inhibited by oxygen. Plants with nitrogen- ﬁxing symbiotic bacteria have\\nevolved methods of restricting the oxygen concentration in the vicinity of the bacteria. Forexample, legumes (the pea family) surround the bacteria with cells that produce a moleculecalled leghaemoglobin, which is very similar to myoglobin (page 239). The leghaemoglobinbinds oxygen, preventing it from reaching the nitrogenase.\\nGiven ammonium ions, plants and bacteria can synthesise all 20 amino acids. Animals\\nare more limited and must obtain some amino acids from their diet. Amino transferases(page 249) allow animals to move amino groups from an amino acid to an oxo-acid to gener-ate a new amino acid, but there are some carbon skeletons that cannot be synthesized. Adulthumans require histidine, isoleucine, leucine, lysine, methionine, phenylalanine, threonine,tryptophan, and valine in their diet —these are the essential amino acids. Adult humans\\nare in nitrogen balance —we excrete the same amount of nitrogen as we take in. Growing\\ninfants, however, have a net uptake of nitrogen: they take in more nitrogen than they excrete.Growth clearly demands more amino acids, and in this case the limited ability we have tosynthesize arginine is insuf ﬁcient and so it must be present in the diet as well.\\nOther molecules, such as the nucleic acid bases, are synthesized from amino acid\\nstarting materials.\\nCARBON FIXATION IN PLANTS\\nWe have already described how chloroplasts use the energy of light to oxidize H 2O\\nto give O 2and to generate ATP and NADPH (page 271). A series of so-called dark reac-\\ntions in the chloroplast stroma uses ATP and NADPH in a process that grabs carbon dioxide\\nfrom the air and builds it into larger molecules, a process called carbon ﬁxation. These re-\\nactions are often called the Calvin cycle in honor of their discoverer (Fig. 13.13). Theinitial step is carried out by the enzyme ribulose bisphosphate carboxylase, said to be themost abundant protein on earth. CO\\n2is combined with the ﬁve-carbon sugar ribulose bis-\\nphosphate to give a six-carbon intermediate that immediately splits into two molecules of3-phosphoglycerate. Further reactions use NADPH and ATP to convert these three-carbon', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6fef9bbe-67b9-4bd3-b0dc-157d227fdbe2', embedding=None, metadata={'page_label': '309', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\nCONTROL OF ENERGY PRODUCTION 303\\nribulose-1,5-bisphosphate2 × 3-phosphoglycerate\\n2 × 1,3 bisphosphoglycerate\\n2 × glyceraldehyde-3-phosphate\\nfructose-6-phosphate\\ncomplex sugar\\nintraconversions\\nATP2ATP\\n2NADPHCO2\\nribulose-5-phosphate\\nFigure 13.13. The dark reactions of photosynthesis capture carbon dioxide from the air.\\nunits to fructose-6-phosphate. It is interesting that many of these reactions are similar to\\nthose of the pentose phosphate pathway (page 289).\\nCONTROL OF ENERGY PRODUCTION\\nFeedback and Feedforward\\nWe have seen how an energy currency that runs low is topped up by conversion from anothercurrency. This is not enough to ensure a constant energy supply, however. Therefore the cellhas more mechanisms that ensure that the supply of cellular energy is accelerated or slowedas appropriate. These mechanisms are of two types called feedforward and feedback. Wewill introduce the terms by analogy with real money. Consider a bank teller. During the daypeople deposit checks but draw out cash to spend. As time passes the stock of banknotes andchange in the till gets low. The teller signals to the supervisor, who opens the bank vault,takes out more cash, and re ﬁlls the teller ’s till. This is an example of negative feedback.\\nIn general negative feedback is said to occur when a change in some parameter activates amechanism that reverses the change in that parameter. We have already met an analogousnegative feedback system in the control of tryptophan biosynthesis (page 116). A downwardchange in the concentration of tryptophan in the bacterial cell activates the mechanism thatcauses the cell to make more tryptophan. Positive feedback is less common in both biology', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='880d61cd-fd24-4e97-b350-b773b89d8028', embedding=None, metadata={'page_label': '310', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\n304 METABOLISM\\nfructose-6-phosphate fructose-1,6-bisphosphate\\ninhibited by ATPactivated by AMP\\nATP+\\n−ve+ve\\nFigure 13.14. Phosphofructokinase is regulated by the binding of ATP or AMP at a regulatory site\\nthat is separate from the active site. Binding of ATP inhibits while binding of AMP activates.\\nand banking. It is said to occur when a change in some parameter activates a mechanism\\nthat accelerates the change. In banking this occurs when a rumor starts that a bank is aboutto fail. The lower a bank ’s reserves of money get, the more its depositors rush to take\\ntheir money out before it is too late. Biological examples of positive feedback are unusual.Quorum sensing in luminescent bacteria (page 116) and the action potential (page 325) aretwo examples of positive feedback that we describe in this book.\\nWhat about feedforward? The bank is especially busy at lunchtime, with lots of cash\\nwithdrawn between 12:30 and 2:00. During this time, everyone is rushed off their feet.The supervisors do not wait until tellers signal that they are short of cash but instead openthe vaults at 12 noon and bring out enough cash to see the tellers through the lunchtimerush. They are preparing for a future drain on cash by stocking up the tills before the drainoccurs —this is feedforward. Feedforward happens in biological systems too, as we will see\\nin this section.\\nNegative Feedback Control of Glycolysis\\nPhosphofructokinase catalyzes the ﬁrst irreversible step in glycolysis after the paths from\\nglucose and glycogen converge (Fig. 13.3). The enzyme is allosterically regulated by ATP(Fig. 13.14). When ATP concentrations are high, ATP binds to regulatory sites on phospho-fructokinase and locks it into an inactive (low af ﬁnity for fructose phosphate) conformation.\\nWhen the concentration of ATP is reduced, levels of AMP increase (because ADP is con-verted to ATP and AMP, page 259). AMP will compete with ATP so an increasing numberof phosphofructokinase molecules come to have AMP in the regulatory sites, which causesthe enzyme to switch to the active, high-af ﬁnity conformation. Fructose-1,6-bisphosphate\\nis produced, feeding the glycolytic pathway that in turn feeds the mitochondria with pyru-vate for the production of ATP. This process is negative feedback because changes in theconcentration of ATP act, through its allosteric action on phosphofructokinase, to reversethe change in ATP concentration.\\nFeedforward Control in Muscle Cells\\nWhen a signal goes out from our brains to the muscles in our legs to tell them to startworking, it causes the endoplasmic reticulum to release calcium ions into the cytosol. Cal-cium is acting as an intracellular messenger, a topic we will cover in more detail in Chapter16. The increase of calcium activates several processes. It causes the muscle cell to con-tract, using the energy released by ATP hydrolysis to do mechanical work (page 393). At', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='98d27c41-293e-4328-912d-8f7ad34a4909', embedding=None, metadata={'page_label': '311', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\nCONTROL OF ENERGY PRODUCTION 305\\ncAMP\\ninactive protein \\nkinase Aactive\\nprotein kinase A \\nphosphorylation of glycogen phosphorylase kinaseinactive glycogen \\nphosphorylase\\nkinasecalmodulin\\nCa\\n++\\nactive glycogen \\nphosphorylase \\nkinase\\nphosphorylation of glycogen \\nphosphorylase\\nactive glycogen \\nphosphorylaseinactive glycogen \\nphosphorylase\\nglucose\\nglucose\\nglucose\\nglucose\\nglycogenglucose\\nglucose\\nglucose\\nglucosePO\\nO−O−\\nto glycolysis and therefore ATP production+\\nFigure 13.15. Calcium and cyclic-AMP both activate glycogen breakdown in muscle and liver.\\nthe same time, other calcium ions pass through a channel into the mitochondrial matrix,\\nattracted by the large negative voltage of the mitochondrion interior. Once there, calciumactivates three key enzymes: pyruvate dehydrogenase, oxoglutarate dehydrogenase, andmalate dehydrogenase (pages 286, 284). The cell has not waited until the ATP concen-tration starts to decrease before activating the Krebs cycle in the mitochondria, so this isfeedforward control. Meanwhile, in the cytosol, calcium ions bind to the protein calmodulin(page 201), which in turn binds to and activates an enzyme called glycogen phosphory-\\nlase kinase (Fig. 13.15). Glycogen phosphorylase kinase is only able to phosphorylate one\\ntarget, glycogen phosphorylase. Glycogen phosphorylase is activated by phosphorylationand proceeds to break down glycogen to release glucose-1-phosphate, which is fed intothe glycolytic pathway. The cell has not waited until glucose concentration falls beforeactivating glycogen breakdown, so this is feedforward control.\\nIn fact, muscles can begin to break down glycogen even before the message goes out\\nfrom the brain to tell them to contract. When the brain realizes that we are in a dangeroussituation and might be going to have to run, it causes the release of the hormone adrenalinefrom the adrenal glands above the kidneys. Adrenaline binds to an integral membrane proteinof the skeletal muscle cells called the β-adrenergic receptor. This causes the production of\\nthe intracellular messenger cyclic AMP (cAMP) within the cytosol of the muscle cell. Thistopic is dealt with in more detail in Chapter 16. cAMP then activates cAMP-dependent\\nprotein kinase, which is given the short name of protein kinase A (Fig. 13.15). Glycogen\\nphosphorylase kinase, the enzyme that phosphorylates glycogen phosphorylase, is itselfphosphorylated by protein kinase A, and is then active even when cytosolic calcium is low.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='57fcafdc-eb9d-4491-a466-ebcb6aed3afd', embedding=None, metadata={'page_label': '312', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\n306 METABOLISM\\nThus even before we know for sure that we need to run, the muscles are breaking down\\nglycogen and making the glucose they will use if running becomes necessary.\\nSUMMARY\\n1.Metabolism is the collective term for all of the reactions going on inside a cell. Thesereactions are divided into catabolic —those that break down chemical compounds to\\nprovide energy —and anabolic —those that build up complex molecules from simpler\\nones. Catabolic reactions are oxidative and anabolic reactions are reductive.\\n2.The Krebs cycle is at the center of the cell ’s metabolism. It can act to oxidize two\\ncarbon units derived from carbohydrates, fats, or amino acids and acts as a centralswitching yard for the molecules in metabolism.\\n3.Glycolysis converts glucose to pyruvate. If the pyruvate is reduced to lactate, glycol-ysis can continue in the absence of oxygen as this reduction regenerates the NAD\\n+\\nneeded for glycolysis.\\n4.Glycogen can act as a reserve of glucose for glycolysis.\\n5.Glucose-6-phosphate can be converted to a pentose sugar for nucleotide manufacturewith production of NADPH for biosynthesis.\\n6.Fats (triacylglycerols) are concentrated fuel stores. Their fatty acid components areoxidized to two-carbon units by βoxidation.\\n7.Amino groups must be removed before excess dietary amino acids can be used asfuels. This is done by transferring them to make aspartate or glutamate, and thenceto urea for excretion.\\n8.Gluconeogenesis allows the synthesis of glucose from noncarbohydrate precursors(but mammals cannot make glucose from fatty acids).\\n9.Biosynthetic pathways for molecules follow different routes from the catabolicpathways. Good examples are fatty acid synthesis and breakdown ( βoxidation) and\\nglycogen synthesis and breakdown.\\n10.Plants use light energy to generate ATP and reducing power as NADPH and hencetoﬁx carbon dioxide into sugars.\\n11.Metabolic reactions are controlled by feedforward and feedback mechanisms, whichmake use of allosteric control and covalent modi ﬁcation of key enzymes.\\nFURTHER READING\\nBender, D. A. 2002. An Introduction to Nutrition and Metabolism, 3rd ed. London: Taylor and Francis.\\nBuchanan, B. B., Gruissem, W., and Jones, R. L. 2000. Biochemistry and Physiology of Plants.\\nRockville, Maryland: American Society of Plant Physiologists.\\nDevlin, T. M. 2002. Textbook of Biochemistry with Clinical Correlations, 5th ed. New York: Wiley.\\nV oet, D., and V oet, J. D. 2003. Biochemistry, 3rd ed. New York: Wiley.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='8d8f58fb-837c-4b06-a256-1336f5c19aa5', embedding=None, metadata={'page_label': '313', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\nREVIEW QUESTIONS 307\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.Metabolism is\\nA. the means by which DNA is stored.\\nB. the sum of all the anabolic and catabolic reactions in a cell.C. the reactions that break fuel molecules down to gain energy.D. the reactions that synthesise molecules in the cell.E. the sum of all the reactions taking place in the mitochondrial matrix.\\n2.The Krebs cycle\\nA. breaks down pyruvate.B. oxidizes two-carbon units to produce carbon dioxide and reduced coenzymes.C. oxidizes glycerol.D. drives H\\n+out of the mitochondrial matrix.\\nE. uses GTP rather than ATP to supply phosphate groups.\\n3.Which of the following statements about the glycolysis pathway in the cytosol is incor-\\nrect?\\nA. It makes ATP.\\nB. It makes acetyl-CoA.C. It interacts with the pentose phosphate pathway.D. All but three steps are reversible and therefore also operate in gluconeogenesis.E. It is oxidative under aerobic conditions.\\n4.βoxidation\\nA. breaks the β-glycosidic bonds in cellulose.\\nB. uses acetyl-CoA, NADH, and FADH 2to produce long-chain fatty acids.\\nC. generates pentose sugars.D. oxidizes fatty acids to give acetyl-CoA, NADH, and FADH\\n2.\\nE. Is part of triacylglycerol biosynthesis.\\n5.Gluconeogenesis is not capable of making glucose from\\nA. alanine.B. lactate.C. glycerol.D. palmitate.E. 2-oxoglutarate.\\n6.Which of the following statements is incorrect concerning photosynthesis in green\\nplants?\\nA. It generates ATP, NADPH, and oxygen.\\nB. It adds carbon dioxide to a ﬁve-carbon sugar that is then cleaved to form two molecules of\\nphosphoglycerate.\\nC. Its enzymes are found in the mitochondria.D. It uses light energy to drive the splitting of water.E. Its enzymes are found in the chloroplasts.\\n7.Positive feedback is operating\\nA. when adenosine monophosphate activates phosphofructokinase.B. when cAMP activates transcription of the lacoperon.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='0f171f4f-55c7-4212-bd63-d71aa9476d9a', embedding=None, metadata={'page_label': '314', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-13 WY001-Bolsover-v2.cls September 15, 2003 21:37\\n308 METABOLISM\\nC. when tryptophan inhibits transcription of the trpoperon.\\nD. when N-acyl-HSL promotes transcription of the luxoperon.\\nE. in all the examples above.\\n8.ATP\\nA. is converted to ADP as H+enters the mitochondrial matrix.\\nB. is a coenzyme.\\nC. passes through the outer mitochondrial membrane by simple diffusion.D. cannot be made in the absence of oxygen.E. concentrations fall to a low level in resting cells.\\nANSWERS TO REVIEW QUESTIONS\\n1.B.It is the sum of all of the catabolic and anabolic reactions in a cell.\\n2.B.The Krebs cycle oxidizes acetyl-CoA and two carbon dioxides are lost for each turn of the cycle.\\nThe cycle does not use pyruvate, rather, pyruvate is ﬁrst converted to acetyl-CoA by pyruvate\\ndehydrogenase, and it is acetyl-CoA that is fed into the cycle. Concerning answer D: the electrontransport chain drives H\\n+out of the mitochondrial matrix. ATP synthase can also (by running in\\nreverse) perform this function, but the enzymes of the Krebs cycle certainly do not. Concerninganswer E: the Krebs cycle does not use either GTP or ATP to supply phosphate groups, though itdoes generate one GTP from GDP (step 5 in Fig. 13.2).\\n3.B.Glycolysis generates pyruvate. Pyruvate can then be converted to acetyl-CoA, but this takes\\nplace in mitochondria, not in the cytosol, and is not regarded as being part of the glycolysis pathway.\\n4.D.βoxidation produces acetyl-CoA from fatty acids and also produces NADH and FADH\\n2.\\n5.D.Glucose cannot be produced from any fatty acid, including palmitate.\\n6.C.Photosynthesis takes place in chloroplasts.\\n7.D.One of the genes within the luxoperon is luxI, which codes for the enzyme that makes N-acyl-\\nHSL. This is therefore an example of positive feedback. Concerning the other examples (A and C):these are examples of negative feedback. Activation of phosphofructokinase will cause more fuelto be sent to the mitochondria and will therefore help the mitochondria convert AMP and ADP toATP. Inhibiting transcription of the trpoperon will reduce the rate at which the cell synthesizes\\ntryptophan. (B) This is a sensible control mechanism, causing the cell to use glucose when it isavailable in preference to lactose. However, there is no feedback involved: the products of the lac\\noperon are concerned with lactose metabolism, while cAMP levels in bacteria are affected by theglucose concentration.\\n8.B.A coenzyme is a molecule that acts as a second substrate for many enzymes; ATP is a substrate\\nfor a wide range of enzymes; some we have already met are polynucleotide kinase (page 139), hex-okinase (page 284), phosphofructokinase (page 285), glycogen phosphorylase kinase (page 305),and cAMP-dependent protein kinase (page 305). Answer A is false: As H\\n+enters the mitochondria\\nADP is converted to ATP, not the other way around. Answer C is false: ATP is a highly charged ionand will certainly not pass through membranes by simple diffusion; it crosses the outer mitochon-drial membrane by passing through porin. Answer D is false: in the absence of oxygen ATP can bemade by anaerobic glycolysis. Answer E is false: in a healthy cell none of the energy currenciesare allowed to run down.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1a705e7a-5e4c-402f-9e5e-94654dcf7c1c', embedding=None, metadata={'page_label': '315', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-14 WY001-Bolsover-v2.cls September 15, 2003 20:6\\n14\\nIONS AND VOLTAGES\\nWe described in Chapter 2 how membranes are composed of phospholipids arranged so that\\ntheir hydrophobic tails are directed toward the center of the membrane, while the polar hy-drophilic head groups face out. Membranes are a barrier to the movement of many solutes. Inparticular, small hydrophilic solutes such as ions and sugars cannot pass through membraneseasily because, to do so, they would have to lose the cloud of water molecules that formstheir hydration shell (page 21). Two consequences follow from the fact that membranes arebarriers. First, the composition of the liquid on one side of a membrane can be different fromthe composition of the liquid on the other side. Indeed, by allowing cells to retain proteins,sugars, ATP, and many other solutes, the barrier property of the cell membrane makes lifepossible. Table 14.1 shows how ﬁve important ions have different concentrations in cytosoland extracellular medium. Second, the cell must make proteins called channels and carrierswhose job it is to help hydrophilic solutes across the membrane. This chapter describes theproperties of membranes, with particular emphasis on their role in energy storage.\\nTHE POTASSIUM GRADIENT AND THE RESTING VOLTAGE\\nIons are electrically charged. This fact has two consequences for membranes. First, themovement of ions across a membrane will tend to change the voltage across that membrane.If positive ions leave the cytosol, they will leave the cytosol with a negative voltage, andvice versa. Second, a voltage across a membrane will exert a force on all the ions present.If the cytosol has a negative voltage, then positive ions such as sodium and potassium will\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n309', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='91a26e8b-4ecf-4a01-9934-cfb3a9c5f8de', embedding=None, metadata={'page_label': '316', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-14 WY001-Bolsover-v2.cls September 15, 2003 20:6\\n310 IONS AND VOLTAGES\\nTable 14.1. Typical Concentrations for Five Important Ions in Mammalian Cytosol and\\nExtracellular Mediuma\\nIon Cytosol Extracellular Medium\\nSodium Na+10 mmol liter−1150 mmol liter−1\\nPotassium K+140 mmol liter−15 mmol liter−1\\nCalcium Ca2+100 nmol liter−11 mmol liter−1\\nChloride Cl−5 mmol liter−1100 mmol liter−1\\nHydrogen ion H+(really H 3O+) 63 nmol liter−1or pH 7.2 40 nmol liter−1or pH 7.4\\naNote the units n for nano is one million times smaller than m for milli.\\nbe attracted in from the extracellular medium. In this chapter, we will begin to address the\\nquestion of how ions and voltages interact by considering the effect of potassium movementson the voltage across the plasma membrane.\\nPotassium Channels Make the Plasma Membrane Permeable\\nto Potassium Ions\\nThe potassium channel (Fig. 14.1) is a protein found in the plasma membrane of almost all\\ncells. It is a tube that links the cytosol with the extracellular ﬂuid. Potassium ions, which\\ncannot pass through the lipid bilayer of the plasma membrane, pass through the potassium\\ncytosolextracellular\\nmedium\\nextracellular mediumK+\\ncytosol\\nK+\\nFigure 14.1. The positively charged potassium ion cannot cross the lipid bilayer but passes easily\\nthrough a water-ﬁlled tube in the potassium channel.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='09be013d-94af-48e7-ba1d-3c5cf04532b4', embedding=None, metadata={'page_label': '317', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-14 WY001-Bolsover-v2.cls September 15, 2003 20:6\\nTHE POTASSIUM GRADIENT AND THE RESTING VOLTAGE 311\\nchannel easily. Other ions cannot go through. The precise shape of the tube, and the position\\nof charged amino acid side chains within the tube, blocks their movement. The channel isselective for potassium.\\nWe saw earlier that the Na\\n+/K+ATPase (page 270) uses the energy of ATP hydrolysis\\nto drive sodium ions out of the cell and, at the same time, brings potassium ions into thecell. This ensures that potassium is much more concentrated in the cytosol than outside —\\ntypically 140 mmol liter\\n−1in the cytosol but only 5 mmol liter−1in the extracellular medium.\\nThere is an apparent paradox here. If potassium can pass through the potassium channel,why is this ion much more concentrated inside the cell than outside? Why doesn ’t all the\\npotassium rush out? To explain why, we must think about the effects of ion movement ontransmembrane voltage. First a word on nomenclature. All cells have a voltage across theirmembrane when they are not being stimulated. This resting voltage is about −80 mV in\\na relaxed skeletal muscle cell. As soon as the muscle is stimulated to contract, there is asudden sharp change in the transmembrane voltage called the action potential (a processdescribed in more detail in Chapter 15). Many cells never change their transmembraneresting voltage.\\nConcentration Gradients and Electrical Voltage Can Balance\\nA few potassium ions do escape from the cell through the potassium channel. As they doso, they carry out their positive charge and leave the cytosol with a negative voltage thatattracts positively charged ions like potassium. There is still a tendency for potassium ionsto leave the cell down the concentration gradient, but there is now an electrical force pullingthe positively charged potassium ions back inside. We have met a similar situation before,where we described how a concentration gradient and an electrical force combine to forman electrochemical gradient down which H\\n+ions will rush into bacteria or mitochondria\\n(page 261). However, in the case of potassium ions at the plasma membrane, the concentra-tion gradient and the electrical force act in opposite directions. As potassium ions continueto leave the cell, carrying out their positive charge and leaving the cytosol at a more andmore negative voltage, the electrical force pulling them back in gets increasingly strong.Soon, the opposing electrical and concentration gradients are equal, and the overall elec-trochemical gradient for potassium is zero. Potassium ions then stop leaving the cell, eventhough they are much more concentrated inside than outside.\\nFor every ion present on both sides of a membrane, it is possible to calculate the\\ntransmembrane voltage that will exactly balance the concentration gradient. This voltage iscalled the equilibrium voltage for that ion at that membrane. For potassium at the plasma\\nmembrane of a normal animal cell, the equilibrium voltage is about −90 mV .\\nThe departure of potassium ions through the potassium channels, leaving negative\\ncharge behind, produces the resting voltage across the plasma membrane. Because thepotassium channels are the major pathway by which ions can cross the plasma membraneof an unstimulated cell, the resting voltage has a value close to the potassium equilibriumvoltage. In some cells, such as white blood cells, potassium channels are the only channels inthe plasma membrane, and potassium ions move out until the transmembrane voltage tendingto pull them back in exactly balances their tendency to move out down their concentrationgradient. The resting voltage of white blood cells therefore has a value equal to the potassiumequilibrium voltage, about −90 mV . In other cells the situation is more complicated. In\\nmuscle cells, for instance, the resting voltage is −80 mV; in nerve cells it is −70 mV . Even\\nin these cells, though, the major in ﬂuence on the resting voltage is potassium movement', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='61ea327c-d92d-4a6d-a5cf-31d7e2b77fdc', embedding=None, metadata={'page_label': '318', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-14 WY001-Bolsover-v2.cls September 15, 2003 20:6\\n312 IONS AND VOLTAGES\\nthrough its channels, so the resting voltage does not deviate very far from the potassium\\nequilibrium voltage.\\nThe resting voltage set up by potassium movement turns the action of the Na+/K+\\nATPase into an energetically asymmetrical one. Consider one conversion cycle: onemolecule of ATP is hydrolyzed, three sodium ions are pushed out of the cell, and twopotassium ions move in. Very little of the energy of ATP hydrolysis is used up in movingthe two potassium ions into the cell because the electrochemical gradient for this ion isclose to zero. Although potassium is being moved up a concentration gradient, it is alsobeing pulled in by the negative voltage of the cytosol, and the two forces cancel. In contrast,pushing the three sodium ions out of the cell requires more energy than would be required ifthe cytosol were at the same voltage as the extracellular ﬂuid. The sodium ions are positively\\ncharged, so they are attracted by the negative voltage inside the cell, which combines withthe concentration gradient to form a large inward electrochemical gradient. All the energyreleased by ATP hydrolysis is needed to push the three sodium ions up this large electro-chemical gradient. The presence of the potassium channels, and the resting voltage thatthey set up, means that almost all the energy of ATP hydrolysis by the Na\\n+/K+ATPase is\\nstored in the sodium gradient, while potassium ions are close to equilibrium.f\\nIN DEPTH 14.1 The Nernst Equation\\nAn ion that can pass across a membrane is acted on by two forces. The ﬁrst derives\\nfrom the concentration gradient. The ion tends to diffuse from a region where it\\nis at high concentration to one where it is at low concentration. The second forcederives from the transmembrane voltage. In the case of positively charged ionssuch as Na\\n+and K+, the ions tend to move toward a negative voltage. Negatively\\ncharged ions such as Cl−tend to move toward a positive voltage. For each ion\\nthere is a value of the transmembrane voltage for which these forces balance,and the ion will not move. The ion is said to be at equilibrium, and this value ofthe transmembrane voltage is called the equilibrium voltage for that ion at thatmembrane.\\nIon I at concentration [I]outside\\nIon I at concentration [I]inside\\ncytosol at voltage V\\nWhen the forces balance, then ions that move in will neither gain nor lose\\nenergy. This way of describing equilibrium is useful because it allows us to setequivalent the effects of the two very different gradients, concentration and volt-age. For concentration, the free energy possessed by a mole of ions I by virtue ofits concentration is\\nG=G\\n0+RTloge[I] joules\\nwhere G0is the standard free energy, Ris the gas constant (8.3 J mol−1degree−1),\\nandTis the absolute temperature.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='8d5ecbb1-30ec-4b6b-9e9d-2d94500cf310', embedding=None, metadata={'page_label': '319', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-14 WY001-Bolsover-v2.cls September 15, 2003 20:6\\nTHE NERNST EQUATION 313\\nA mole of I passing in therefore moves from a region where it had a free\\nenergy of\\nGoutside =G0+RTloge[Ioutside ] joules\\nto one where its free energy is\\nGinside=G0+RTloge[Iinside] joules\\nOne mole of ions I moving inward therefore gains by virtue of the concentration\\ngradient free energy equal to\\nRTloge[Iinside]−RTloge[Ioutside ] joules\\nNow consider the electrical force. The de ﬁnition of a volt means that one coulomb\\nof charge moving across a membrane with a transmembrane voltage of Vvolts\\ngains Vjoules of free energy. However, we are working in moles, not coulombs.\\nOne mole of ions has a charge of zFcoulombs, where zis the charge on the ion\\nin elementary units. For Na+and K+zis 1; for Ca2+zis 2; and for Cl−zis−1. The\\nterm Fis a number that relates the coulomb to the mole. It has the value 96,500.\\nOne mole of ions I moving inward gains by virtue of the transmembrane voltagefree energy equal to\\nzF V joules\\nThis does not mean that an ion always gains energy from the transmembranevoltage when it moves inward: the term zF V can just as easily be negative as posi-\\ntive.\\nWhen the effects of concentration and voltage just balance, a mole of ions\\nmoving inward neither gains nor loses free energy. Hence, at equilibrium\\nRTlog\\ne[Iinside]−RTloge[Ioutside ]+zF V eq=0\\nThis can be simpli ﬁed to\\nVeq=RT\\nzFloge([Ioutside ]\\n[Iinside])\\nvolts\\nThis is the Nernst equation. At a typical mammalian body temperature of 37◦C, the\\nNernst equation can be written in the more convenient form:\\nVeq=62\\nzlog10([Ioutside ]\\n[Iinside])\\nmvolts\\nNote the change in the type of logarithm and the units.\\n“In”and “out”can refer to any two solutions separated by a membrane. At\\nthe plasma membrane inis the cytosol and outis the extracellular medium, but\\nwhen considering equilibria across the inner mitochondrial membrane inis the\\nmitochondrial matrix and outis the intermembrane space.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='01db045e-574b-4c8c-883a-9a1f3d2894d7', embedding=None, metadata={'page_label': '320', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-14 WY001-Bolsover-v2.cls September 15, 2003 20:6\\n314 IONS AND VOLTAGES\\nTHE CHLORIDE GRADIENT\\nChloride ions (Fig. 14.2) are at a lower concentration in the cytosol than in the extracellular\\nmedium. Typically, their concentration in the cytosol is 5 mmol liter−1compared with 100\\nmmol liter−1in the extracellular ﬂuid. This is because of the resting voltage set up by the\\npotassium channels. Chloride ions are repelled by the negative voltage of the cytosol. Theyleave until their tendency to enter the cell down their concentration gradient exactly matchestheir tendency to be repelled by the negative voltage of the cytosol.\\n[Cl−] = 5 mmol liter−1\\n−80 mV cytosol[Cl−] = 100 mmol liter−1\\n.....the negatively charged \\ncytosol pushes the negative chloride ions outwardchloride would move down its concentration but..................\\n0 mV\\nFigure 14.2. Chloride is close to equilibrium across most plasma membranes.\\nGENERAL PROPERTIES OF CHANNELS\\nChannels are integral membrane proteins that form water- ﬁlled tubes through the membrane.\\nWe have already introduced three: the gap junction channel (page 55), porin (page 262), and\\nthe potassium channel. Channels that, like the potassium channel, are selective for particularions can set up transmembrane voltages. The gap junction channel (page 55) is much lessselective than the potassium channel. It forms a tube, 1.5 nm in diameter, through which anysolute of M\\nr≤1000 can pass. The gap junction channel is not always open. It opens only\\nwhen it connects with a second gap junction channel on another cell, forming a tube throughwhich solutes can pass from the cytosol of one cell to the cytosol of the other. Channelsthat are sometimes open and sometimes shut are said to be gated . When a gap junction\\nchannel contacts another on another cell, its gate opens and solute can pass through; at othertimes the gate is shut. The usefulness of gating is obvious: if the gap junction channels notcontacting others were open, many solutes, including ATP and sodium, would leak out intothe extracellular ﬂuid exhausting the cell ’s energy currencies.\\nPorin in the outer mitochondrial membrane plays an important role in energy conver-\\nsion. It forms a very large diameter tube that allows all solutes of M\\nr≤10,000 to pass and\\nseems to spend a large fraction of time open under most circumstances. This is why theouter mitochondrial membrane is permeable to most solutes and ions. Appendix 1, at theend of this book, lists all the different types of channels described in this book. It representsonly a small fraction of the total number known.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='aa949358-34a3-4ca2-8c33-ce5c38b0e31d', embedding=None, metadata={'page_label': '321', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-14 WY001-Bolsover-v2.cls September 15, 2003 20:6\\nGENERAL PROPERTIES OF CHANNELS 315\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 14.1 Cytochrome c—Vital But Deadly\\nWe have described how the electron carrier cytochrome cresides in the intermembrane space be-\\ntween the outer and inner mitochondrial membranes and helps the electron transport chain to convert\\nenergy as NADH to energy as the hydrogen ion electrochemical gradient across the mitochondrialinner membrane (page 266). Although cytochrome cis a soluble protein of relative molecular mass\\n12,270, it cannot escape from the intermembrane space into the cytosol because porin, the channelof the outer mitochondrial membrane, only allows solutes of M\\nr≤10,000 to pass. Although\\ncytochrome cis essential for mitochondrial function, it has another, deadly role. If cytochrome c\\ncomes into contact with a class of cytosolic enzymes called caspases, it activates them, turning on theprocess of cell suicide called apoptosis (page 417). Under certain conditions, porin can associate withother proteins to form a channel of larger diameter; when this happens, cytochrome ccan leak out\\nand the cell dies by apoptosis. This process seems to occur in hearts during heart attacks, and in thebrain during a stroke: there is therefore a considerable research effort aimed at preventing this fromoccurring.\\ncytochrome c   Mr = 12,270\\ncytochrome cporin : cutoff    Mr = 10,000\\nouter mitochondrialmembrane\\ninner mitochondrialmembraneNormal\\nAfter stress\\nApoptosis\\ninactive caspase 9 active caspase 9\\nporin plus other proteins', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e6c477f7-163f-4567-89b4-e292699f0e9b', embedding=None, metadata={'page_label': '322', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-14 WY001-Bolsover-v2.cls September 15, 2003 20:6\\n316 IONS AND VOLTAGES\\nGENERAL PROPERTIES OF CARRIERS\\nWe have already met the three carriers that interconvert the four energy currencies of the cell\\n(page 263). Carriers are like channels in that they are integral membrane proteins that allowsolute to cross the membrane and, like channels, they form a tube across the membrane.However, there is a critical difference. In carriers the tube is never open all the way through;it is always closed at one or other end. Solutes can move into the tube through the openend. When the carrier changes shape, so that the end that was closed is open, the solute canmove into the solution on the other side of the membrane.\\nThe Glucose Carrier\\nOne of the simplest carriers is the glucose carrier (Fig. 14.3). It switches freely between aform that is open to the cytosol and a form that is open to the extracellular medium. Inside thetube is a site to which a glucose molecule can bind. On the left, a glucose molecule is entering\\nNa+/Ca2+\\nexchanger1Ca2+glucose\\ncarrierextracellularmedium\\ncytosol\\nglucose\\n3Na+\\n1Ca2+\\n3Na+\\nglucose\\nFigure 14.3. The glucose carrier and the sodium/calcium exchanger are sometimes open to the\\ncytosol and sometimes open to the extracellular medium.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='903e5875-21bd-40bd-9822-4f8b9de7b96a', embedding=None, metadata={'page_label': '323', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-14 WY001-Bolsover-v2.cls September 15, 2003 20:6\\nGENERAL PROPERTIES OF CARRIERS 317\\nthe tube and binding to the site. Sometimes glucose leaves the binding site before the carrier\\nswitches shape. In Figure 14.3 we see the other possibility: the carrier switches shape beforethe glucose has left. The binding site is now open to the cytosol, and the glucose can escapeinto the cytosol. It has been carried across the plasma membrane. Unlike channels, carriersnever form open tubes all the way across the membrane. Instead, they bind one or moremolecules or ions, then change shape to carry the molecules or ions across the membrane.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 14.2 The Glucose Carrier Is Essential\\nThe cells of our bodies are bathed in a glucose-rich solution. However, glucose cannot cross the\\nplasma membrane by simple diffusion because it is strongly hydrophilic: It can only get in via theglucose carrier. Some cells have glucose carriers in their membranes all the time, but others suchas muscle and fat cells translocate the glucose carrier to the plasma membrane only in the presenceof insulin. Insulin-dependent diabetics cannot produce their own insulin, and unless they injectsynthetic insulin, their muscles and fat cells cannot take up glucose and therefore run out of energy,even though the concentration of glucose in the blood becomes very high. This is why muscularweakness is one symptom of diabetes.\\nThe glucose carrier is very simple, whereas other carriers are more complex. We will\\nnext consider two carriers, the sodium/calcium exchanger and the calcium ATPase, which\\ndo much the same job —they push calcium ions up their concentration gradient out of the\\ncell—but which take their energy from different currencies. The sodium/calcium exchanger\\nuses the sodium concentration gradient while the calcium ATPase uses ATP.\\nThe Sodium –Calcium Exchanger\\nFigure 14.3 shows that like the glucose carrier, the sodium/calcium exchanger can exist in\\ntwo shapes, one open to the extracellular medium and one open to the cytosol. Inside thetube are three sites that can bind sodium ions and one site that can bind a calcium ion. Thesodium/calcium exchanger is not free to switch between its two shapes at any time. Instead,it switches only if either all the sodium sites are ﬁlled and the calcium site is empty or if\\nthe calcium site is ﬁlled and all the sodium sites are empty.\\nOn the left, the carrier is open to the extracellular medium. It can switch its shape so that\\nit is open to the cytosol if one of two things happen. It could, as shown, bind three sodiumions (keeping the calcium site empty) or it could bind one calcium and keep the sodium sitesvacant. This second option does not often happen, since sodium is at high concentration in theextracellular ﬂuid, and one or more of the sodium sites is usually occupied. Therefore nearly\\nall the switches from open-to-outside to open-to-inside are of the type shown. Once the tubehas opened to the low sodium environment of the cytosol, the sodium ions tend to leave.\\nOnce the carrier is open to the cytosol, it can switch back to the open-to-outside form\\nby binding either one calcium or three sodium ions. Since sodium is scarce in the cytosol,the latter event is unlikely. More frequently a calcium ion will bind and will be carried out.The carrier is now ready to bind sodium again and perform another cycle.\\nThe overall effect of one cycle is to carry three sodium ions into the cell down their\\nelectrochemical gradient, and one calcium ion out of the cell up its electrochemical gradient.A simple rule about when the carrier can switch shape has produced a machine that uses theenergy currency of the sodium gradient to do work in pushing calcium ions out of the cell.Figure 14.4 shows a simple way of representing what is happening. The circle representsone cycle of operation, from open to extracellular medium back to open to extracellularmedium again.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6087571d-b62a-4e03-82f1-ab8156d9eaaa', embedding=None, metadata={'page_label': '324', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-14 WY001-Bolsover-v2.cls September 15, 2003 20:6\\n318 IONS AND VOLTAGES\\n1Ca++\\ncytosol3Na+\\nextracellular medium\\nFigure 14.4. Action of the sodium/calcium exchanger.\\nCarriers with an Enzymatic Action: The Calcium ATPase\\nThe electron transport chain, ATP synthase, and the Na+/K+ATPase are carriers with an\\nadditional level of complexity in that they carry out an enzymatic action as well as their\\ncarrier function. Figure 14.5 illustrates another carrier with a linked enzymatic function: theCa\\n2+ATPase. Like the sodium/calcium exchanger this is found in the plasma membrane.\\nThe transmembrane part of this carrier forms a tube that can be open to the cytosol or tothe extracellular medium. Two −COO\\n−groups in the tube can form a binding site for a\\ncalcum ion. Other domains of the protein lie in the cytosol and can hydrolyze ATP. Changesin the shape of the cytosolic region are transmitted to the transmembrane region and forceit between the open-to-cytosol and open-to-outside shapes. Figure 14.5 shows our presentunderstanding of how this might happen. (1) is the relaxed shape of the protein. ATP is heldby noncovalent interactions in a crevice in one of the cytosolic domains. One of the COO\\n−\\ngroups in the tube is protonated at this time. (2) A calcium ion moving in from the cytosolpushes the tube open and binds to both binding sites inside the tube. The positively chargedcalcium ion displaces an H\\n+ion and is then able to interact with both COO−groups. The\\ndistortion in protein shape caused by the calcium ion pushing the tube open is transmittedto the cytosolic region and brings the γphosphate of ATP close to an aspartate residue\\non a neighboring domain. (3) An intrinsic kinase activity of the two domains transfers theγ-phosphate group from ATP to the aspartate residue. As the insert shows, this means that a\\nphosphate group, with two negative charges, is sitting very close to ADP, with three negativecharges. (4) The repulsion between the phosphorylated aspartate and the ADP forces thetwo domains apart. The shape change is transmitted to the transmembrane region, which isforced into a wide open-to-outside shape. The calcium ion, which can no longer bind to theCOO\\n−groups on both sides of the tube, is held only weakly and tends to escape into the\\nextracellular medium. An H+ion moves in to protonate one of the COO−groups. (5) A third\\ncytosolic domain, which has phosphatase catalytic ability, swings in and dephosphorylatesthe aspartate residue. Meanwhile, ATP moves in and displaces ADP at the binding crevice.Once the aspartate has been dephosphorylated, the protein is no longer held in the open-to-outside shape by electrical repulsion and relaxes back to the resting shape shown in (1). Itis now ready to accept another calcium ion from the cytosol and repeat the process.\\nFigure 14.6 summarizes what is happening. The circle represents one cycle of operation.\\nOne molecule of ATP is hydrolyzed to ADP and inorganic phosphate, one calcium ion movesout of the cell, and one H\\n+moves in. The energy released by ATP hydrolysis has been used\\nto drive one calcium ion up its electrochemical gradient out of the cell.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='70a23a39-d05b-45bb-94b6-e02a8a2888bf', embedding=None, metadata={'page_label': '325', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-14 WY001-Bolsover-v2.cls September 15, 2003 20:6\\n1 2 3 4 5\\n++H+ADPphosphatase\\ncatalytic\\nsiteaspartate\\nresidue\\nATP\\nbinding\\ncrevice\\n++C C\\nO OO O\\nCC\\nO OHOOaspartate\\nO P\\nOHOO\\nCC\\nO OOO++C C\\nO OO O\\nC\\nOO\\nC\\nOHOO\\nP\\nOO\\nP\\nOO\\nP\\nOOOOO\\nO\\nP\\nOO\\nP\\nOO\\nP\\nOOOOO\\nO\\nP\\nOO\\nP\\nOO\\nP\\nOOOOOO\\nP\\nOO\\nP\\nOO\\nP\\nOOOOOO P\\nOO P\\nOOO OO P\\nOO P\\nOO P\\nOOO O O\\nO\\nP\\nOO\\nO P\\nOO P\\nOOO O\\nO\\nP\\nOOcytosol\\nCa++\\nH++\\nFigure 14.5. Calcium ATPase undergoes a cycle of phosphorylation and dephosphorylation. These drive shape changes that in turn push calcium\\nions out of the cell.\\n319', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='8ea54081-cb93-409c-8467-6406a7a36571', embedding=None, metadata={'page_label': '326', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-14 WY001-Bolsover-v2.cls September 15, 2003 20:6\\n320 IONS AND VOLTAGES\\n✎\\n✍\\x0c\\n✌Medical\\nRelevance\\n14.1Poisoned Hearts Are Stronger\\nDigitalis is used to treat heart failure. Digitalis inhibits the sodium/potassium\\nATPase and is extremely toxic. Nevertheless, a small dose, which inhibits the\\nsodium/potassium pump just a little, causes the heart muscle to beat more strongly. The reasonis that inhibiting the sodium/potassium pump just a little causes a small increase of cytosolicsodium concentration. Because the sodium/calcium exchanger has three binding sites for sodium,its activity is extremely sensitive to sodium concentration, and even a small increase of cytosolicsodium reduces its activity signi ﬁcantly. The calcium concentration in the cytosol therefore rises.\\nThe mechanical motor that drives heart contraction (Chapter 18) is controlled by calcium, so thata small increase of cytosolic calcium makes the heart beat more strongly.\\nK+Na+\\nNa+/K+\\nATPaseNa+/Ca++\\nExchangerNa+Digitalis\\nextracellularmedium\\nForce\\nof\\ncontractionCytosolATP\\nCa\\n++\\nAll the cells of our bodies have one of these two calcium pumps —the sodium/calcium\\nexchanger or the calcium ATPase —and many have both. Because of the action of these\\ncarriers, the calcium concentration in the cytosol is much less than the concentration in\\nthe extracellular medium: usually about 100 nmol liter−1compared with 1 mmol liter−1.\\nBecause the resting voltage is attracting the positively charged calcium ions inward, theoverall result is a large electrochemical gradient favoring calcium entry into cells.f\\nIN DEPTH 14.2 Measuring the Transmembrane Voltage\\nIn 1949 Gilbert Ling and Ralph Gerard discovered that when a ﬁne glass mi-\\ncropipette ﬁlled with an electrically conducting solution impaled a cell, the plasma\\nmembrane sealed to the glass, so that the transmembrane voltage was not dis-\\ncharged. The voltage difference between a wire inserted into the micropipetteand an electrode in the extracellular ﬂuid could then be measured. By passing\\ncurrent through the micropipette, the transmembrane voltage could be altered.\\nTwenty- ﬁve years later Erwin Neher and Bert Sakmann showed that the mi-\\ncropipette did not have to impale the cell. If it just touched the cell, a slightsuction caused the plasma membrane to seal to the glass. The technique, called', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='392a2056-5ecf-4554-9aed-8ec0b13f6cb6', embedding=None, metadata={'page_label': '327', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-14 WY001-Bolsover-v2.cls September 15, 2003 20:6\\nMEASURING THE TRANSMEMBRANE VOLTAGE 321\\ntip \\ndiameter less than 1µmglass \\nmicropipette\\nelectrode in extracellular mediumvoltmeter\\n(a)\\nchannel\\nammeter, to measure the current flowing\\n(b)\\nvoltmeter\\n(c)\\ncell-attached patch clamping, can measure currents through the few channels\\npresent in the tiny patch of membrane within the pipette.\\nStronger suction bursts the membrane within the pipette. The transmembrane\\nvoltage can now be measured.\\nAlternatively, current can be passed through the micropipette to change the\\ntransmembrane voltage —this is the whole cell patch clamp technique. In 1991,\\nNeher and Sakmann received the Nobel prize for medicine.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='c1d3af9e-dcf3-49d1-a97e-1137997642b8', embedding=None, metadata={'page_label': '328', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-14 WY001-Bolsover-v2.cls September 15, 2003 20:6\\n322 IONS AND VOLTAGES\\n1Ca++\\n1H+\\ncytosol extracellular mediumADP +PiATP\\nFigure 14.6. Action of the calcium ATPase.\\nSUMMARY\\n1.Channels are membrane proteins with a central water- ﬁlled hole through which hy-\\ndrophilic solutes, including ions, can pass from one side of the membrane to the\\nother. Changes in protein structure may act to gate the channel but are not requiredfor movement of the solute from one side to the other.\\n2.The presence of potassium channels in the plasma membrane, and the resting voltagethat they set up, means that almost all the energy of ATP hydrolysis by the Na\\n+/K+\\nATPase is stored in the sodium gradient, while potassium ions are close to equilibrium.\\n3.The resting voltage repels chloride ions from the cell interior.\\n4.Carriers, like channels, form a tube across the membrane, but the tube is alwaysclosed at one end. Solutes can move into the tube through the open end. When thecarrier changes shape, so that the end that was closed is open, the solute can leave tothe solution on the other side of the membrane.\\n5.The glucose carrier is present in the plasma membrane of all human cells.\\n6.The sodium/calcium exchanger is a carrier that uses the energy of the sodium gradientto push calcium ions out of the cell.\\n7.The electron transport chain, ATP synthase, the sodium/potassium ATPase, and cal-cium ATPase are both carriers and enzymes, the two actions being tightly linked.\\nFURTHER READING\\nAshcroft, F. M. 2000. Ion Channels and Disease. San Diego: Academic Press.\\nLevitan, I. B., and Kaczmarek, L. K. 2002. The Neuron. New York: Oxford University Press.\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.Which of the following statements about potassium ions is false?\\nA. Potassium ions are more concentrated in the extracellular medium than in cytosol.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2056d554-f56f-4080-955b-87765c28e2f6', embedding=None, metadata={'page_label': '329', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-14 WY001-Bolsover-v2.cls September 15, 2003 20:6\\nREVIEW QUESTIONS 323\\nB. Potassium ions are moved into cells by the Na+/K+ATPase.\\nC. Most plasma membranes contain potassium-selective channels.\\nD. Potassium would ﬂow out of most cells rapidly but for the effect of the negative resting voltage\\nE. Answers A though D are false.\\n2.Which of the following statements is false?\\nA. The potassium channel allows potassium ions to pass from one side of the plasma membrane\\nto the other.\\nB. Porin allows chloride ions to pass from one side of the inner mitochondrial membrane to the other.C. The glucose carrier allows glucose to pass from one side of the plasma membrane to the\\nother.\\nD. The gap junction channel allows potassium ions to pass from the cytosol of one cell to the\\ncytosol of its neighbor.\\nE. Thermogenin allows H\\n+to pass from one side of the inner mitochondrial membrane to the\\nother.\\n3.The resting voltage\\nA. is between −70 and −90 mV in most cells.\\nB. attracts chloride ions in from the extracellular medium.C. is mainly determined by the chloride concentration gradient across the plasma membrane.D. helps retain ATP and glucose within cells.E. helps keep the sodium concentration low in the cytosol.\\n4.Solute movement through which channels is mainly responsible for the resting voltage?\\nA. PorinB. Gap junction channelsC. GlucoseD. PotassiumE. All of the above contribute equally to the resting voltage.\\n5.Which of the following statements is false?\\nA. Channels form water- ﬁlled tubes through the membrane.\\nB. Channels are often gated.C. Channels are only found in the plasma membrane.D. Channels cannot transport solute up an electrochemical gradient.E. Channels are often selective for a particular solute or type of solute.\\n6.Carrier proteins\\nA. never form a tube open all the way through a membrane.B. can transport solute up an electrochemical gradient.C. can often carry out an enzymatic function in addition to their carrier function.D. are found in the plasma membrane and in intracellular membranes.E. all of the above.\\n7.An ion X+is at equilibrium across the plasma membrane of a cell. The cell is voltage\\nclamped, that is, the value of the transmembrane voltage is set by the experimenter, not\\nby the properties of the membrane. Which of the following would cause X+to no longer\\nbe at equilibrium?\\nA. A change of transmembrane voltage\\nB. A change in the concentration of X+in the extracellular medium\\nC. Either answer A or BD. Closure of the channels that allow X\\n+to cross the membrane\\nE. A change in the selectivity of the channel that allows X+to cross the membrane so that Y+can\\ncross as well\\nF. Either answer D or E', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='fb87184f-f4f6-4b12-ac74-c39b0ec3068e', embedding=None, metadata={'page_label': '330', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-14 WY001-Bolsover-v2.cls September 15, 2003 20:6\\n324 IONS AND VOLTAGES\\n8.Which of the following is not a carrier?\\nA. The electron transport chain\\nB. HexokinaseC. ATP synthaseD. The sodium/potassium ATPaseE. Calcium ATPaseF. The sodium/calcium exchangerG. The ADP/ATP exchanger\\nANSWERS TO REVIEW QUESTIONS\\n1.A.Potassium ions are more concentrated in the cytosol due to the action of the Na+/K+ATPase.\\n2.B.Although porin will certainly allow chloride ions to pass, it is present in the outer mitochondrial\\nmembrane, not the inner. All the other statements are true, although, regarding answer D, note thatthe gap junction channel will allow many different solutes to pass; it is not in any way selectivefor potassium.\\n3.A.All the others are false. In particular, (B) chloride ions are repelled from the negative cytosol; (C)\\nthe resting voltage is mainly determined by the potassium concentration gradient across the plasmamembrane; (D) ATP is negatively charged and is therefore not attracted by a negatively chargedcytosol, it is retained by the cytosol because it is not able to cross the plasma membrane. Glucoseis uncharged and is therefore unaffected by the transmembrane voltage; it is retained within cellsbecause it is rapidly phosphorylated to glucose-6-phosphate, which is not able to cross the plasmamembrane. (E) Sodium is attracted intocells by the negatively charged cytosol; the reason it has\\na low concentration in the cytosol is because it is pushed out of cells by the Na\\n+/K+ATPase.\\n4.D.Because most cells express potassium channels in their plasma membranes, potassium is the ion\\nthat can most easily cross the plasma membrane and therefore is mainly responsible for the restingvoltage. Concerning answer C, note that there is no “glucose channel, ”although large channels,\\nincluding porin and the gap junction channel, can carry glucose along with many other solutes.\\n5.C.Channels are found in all intracellular membranes. We have already discussed porin, a channel\\nof the outer mitochondrial membrane. All the other statements are true. In particular, channelscannot transport solute up an electrochemical gradient; they simply make it easier for solute topass down its electrochemical gradient.\\n6.E.Answers A through D are true. In particular, carrier proteins can transport solute up an\\nelectrochemical gradient by linking the movement with another process with negative /Delta1G.In\\nmost cases this is either the hydrolysis of ATP to ADP and inorganic phosphate or the movementof another solute down its electrochemical gradient. The electron transport chain links themovement of H\\n+up its electrochemical gradient with the reaction\\nNADH +H++1/2O2=NAD++H2O/Delta1G=−206 kJ mol−1\\n7.C.If X+is at equilibrium across the plasma membrane, its tendency to move one way down\\nits concentration gradient is matched by its tendency to move the other way down the voltagegradient. Changing the amplitude of either gradient will cause these two effects to no longerbalance. Neither answer D nor E is true. If X\\n+is at equilibrium, then it will remain at equilibrium\\nirrespective of how easily it or other ions can cross the membrane.\\n8.B.Hexokinase is a soluble protein of the cytosol; it catalyzes the vital metabolic step of phospho-\\nrylating glucose to glucose-6-phosphate but has no carrier action. All the others are carriers.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='28b1ddd6-4cf5-4731-8890-2dec9076fbbc', embedding=None, metadata={'page_label': '331', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-15 WY001-Bolsover-v2.cls August 21, 2003 19:36\\n15\\nTHE ACTION POTENTIAL\\nAn action potential is an explosive change in the voltage across the plasma membrane. The\\nmost sophisticated electrically excitable cells—cells that can produce action potentials—are the nerve cells that allow our brains to carry out the complex electrical data processingcalled thought. However, we will begin with a much simpler electrically excitable cell: thesea urchin egg.\\nTHE CALCIUM ACTION POTENTIAL IN SEA URCHIN EGGS\\nIn most animals, eggs that are fertilized by more than one sperm fail to develop. A number ofmechanisms exist to prevent such “polyspermy,” one of which is based on action potentials.\\nEffect of Egg Transmembrane Voltage on Sperm Fusion\\nAn experiment ﬁrst carried out in 1976 by Laurinda Jaffe is illustrated in Figure 15.1 a.\\nShe impaled a sea urchin egg with a micropipette (page 320). Instead of just measuringthe natural resting transmembrane voltage (around −70 mV), she depolarized the plasma\\nmembrane by passing positive current through the pipette. We use the word depolarization\\nto mean any positive shift in the transmembrane voltage, whatever its size or cause. In thiscase an electronic feedback circuit was used to alter the value of the current passed thoughthe electrode so that the transmembrane voltage was depolarized to +10 mV and held at\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n325', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3e010dd2-a53a-42d9-823e-2e431507c092', embedding=None, metadata={'page_label': '332', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-15 WY001-Bolsover-v2.cls August 21, 2003 19:36\\n326 THE ACTION POTENTIAL\\npass whatever\\nelectrical current isnecessary to keep thetransmembranevoltage at +10 mV\\nno sperm fusepass whateverelectrical current isnecessary to keepthe transmembranevoltage at −70 mV\\nmany sperm fuse-celldivision is abnormaland zygote cannotdevelop\\nsubsequentsperm cannotfuse-celldivision isnormalandzygote willdevelop(a) (b)\\nrecordtransmembranevoltage but donot alter it\\nfirst sperm\\nsucessfully\\nfusestransmembrane\\nvoltage (mV)−70010\\n200 ms 100(c)\\nFigure 15.1. In sea urchin eggs transmembrane voltage controls sperm fusion.\\nthat value. This technique, in which the experimenter, not the cell, determines the value of\\nthe transmembrane voltage, is called voltage clamp.\\nJaffe observed that sperm would not fuse with eggs in which the transmembrane voltage\\nwas clamped at +10 mV , and the eggs remained unfertilized (Fig. 15.1 a). However, an egg\\nclamped at −70 mV (Fig. 15.1 b) was fertilized by multiple sperm, leading to a defective\\nembryo. The transmembrane voltage is able to control the fusion of sperm and egg.\\nWhen no current is passed through the micropipette, the transmembrane voltage is\\nset by the channels in the egg plasma membrane (Fig. 15.1 c). The resting voltage of the\\nunfertilized egg is set by potassium movement through potassium channels. The ﬁrst sperm\\nthat arrives can fuse because the transmembrane voltage is −70 mV . But within 100 ms,\\nthe transmembrane voltage depolarizes to +10 mV and further sperm fusion is blocked.\\nThus transmembrane voltage changes to prevent multiple sperm fusion. The sea urchin', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3034ce09-2644-4897-86e9-dedaafeb5821', embedding=None, metadata={'page_label': '333', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-15 WY001-Bolsover-v2.cls August 21, 2003 19:36\\nTHE CALCIUM ACTION POTENTIAL IN SEA URCHIN EGGS 327\\negg, which when unfertilized has a transmembrane voltage of −70 mV and is receptive\\nto fertilization, depolarizes its membrane following successful fertilization to protect itself\\nfrom subsequent fertilization. The result is shown on the right of Figure 15.1 c. Only one\\nsperm has fused, so the fertilized egg has the correct number of chromosomes and can goon to develop into an embryo and then into a mature sea urchin.\\nThe Voltage-Gated Calcium Channel\\nThe rapid depolarization that occurs upon fertilization of the sea urchin egg is an example ofan action potential. Like many cells, urchin eggs can generate action potentials because theirplasma membranes contain, together with potassium channels, an ion-conducting channelcalled the voltage-gated calcium channel.\\nFigure 15.2 shows how the voltage-gated calcium channel works. On the left is the shape\\nof the protein when the transmembrane voltage is −70 mV , as in an unfertilized egg. The\\nprotein forms a tube, but it is not open all the way through the membrane. If the membranedepolarizes so that the cytosol is less negative or even becomes positive, positively chargedamino acid residues on the protein are repelled, popping the tube into an open state. Calciumions can now pass through the channel. If the membrane is then repolarized, the channelquickly recloses because the positive charges are pulled back toward the inside of the cell.\\nA protein domain in the cytosol called the inactivation plug is constantly jiggling about\\nat the end of a ﬂexible link and can bind to the inside of the open channel. The resulting\\nblockage is called inactivation, and it occurs after about 100 ms. As long as the plasmamembrane remains depolarized, the voltage-gated calcium channel will remain inactivated.When the plasma membrane is repolarized, the positive charges are attracted back towardthe inside of the cell, squeezing the plug out. In summary:\\n1. When the transmembrane voltage is −70 mV , the voltage-gated calcium channel is\\ngated shut.\\n2. When the plasma membrane is depolarized, the channel opens rapidly and after\\nabout 100 ms inactivates.\\nrepolarizationextracellular \\nmedium\\nCytosolCa++spontaneous \\nafter 100 msdepolarization\\ninactivation plugrepolarization\\nFigure 15.2. The voltage-gated calcium channel.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b588fdf7-e7c0-48f2-ab66-ed397d38f94d', embedding=None, metadata={'page_label': '334', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-15 WY001-Bolsover-v2.cls August 21, 2003 19:36\\n328 THE ACTION POTENTIAL\\n3. After the channel has gone through this cycle, it must spend at least 100 ms with the\\ntransmembrane voltage at the resting voltage before it can be opened by a second\\ndepolarization.\\nThe voltage-gated calcium channel goes through this process whether or not calcium ionsare present. To understand the action potential, we must now think about the effect thatmovements of calcium ions through this channel have on the transmembrane voltage.\\nThe Calcium Action Potential\\nAt point A in Figure 15.3 the egg is unfertilized. The potassium channels are open andset the transmembrane voltage to −70 mV . This negative voltage closes the voltage-gated\\ncalcium channels.\\nAt point B we show what happens if an experimenter passes enough current down a\\nmicropipette to depolarize the plasma membrane by 10 mV , to −60 mV , and then stops pass-\\ning current and once again allows the ion channels in the plasma membrane to determinethe transmembrane voltage. This small depolarization opens only a very few voltage-gatedcalcium channels, so what happens next depends almost entirely on the movement of ionsthrough the potassium channels. The slight decrease in the negative voltage of the cytosolreduces its attraction to potassium ions, and they move out of the cell down their concen-tration gradient. In so doing, they carry positive charge away from the cytosol, and thetransmembrane voltage returns to its resting value of −70 mV .\\nAt point C, the experiment is repeated, but this time the membrane is ﬂeetingly depo-\\nlarized by 20 to −50 mV . Again, most of the voltage-gated calcium channels stay shut, but\\nsome open, and calcium ions rush into the cell down their electrochemical gradient. Theﬂow of positive charge inward on the calcium ions outweighs the outward ﬂow of positive\\ncharge on potassium ions, so the cytosol is gaining positive charge and its voltage is movingin the positive direction. As a result more voltage-gated calcium channels open, and thisallows more calcium ions to ﬂood in. This is an example of positive feedback (page 303)\\nbecause every calcium ion that moves in makes the cytosol more positive, causing morecalcium channels to open, and therefore making it easier for the next calcium ion to movein. At point D, all the calcium channels are open, the inward calcium current is much greaterthan the outward potassium current, and the plasma membrane is rapidly depolarizing.\\nHowever, after 100 ms, the voltage-gated calcium channels begin to inactivate and the\\ninward ﬂow of calcium ions stops. The plasma membrane stops depolarizing and returns to\\n−70 mV as potassium leaves through its channels.\\nThis experiment shows the critical feature of an action potential: it is all or nothing.\\nThe depolarization at point B was too small to generate an action potential, and the trans-membrane voltage simply fell back to its resting state. At point C the depolarization was bigenough to start the process, and the action potential took off in an explosive, self-amplifyingway until all the voltage-gated calcium channels were open and the plasma membrane hadgreatly depolarized. In all excitable cells there is a threshold for initiating an action poten-\\ntial; in the sea urchin egg it lies between −60 and −50 mV . Depolarizations to below the\\nthreshold elicit nothing; depolarizations to voltages more positive than the threshold elicitthe complete action potential. At the heart of the action potential is a positive feedback loop.Depolarization causes voltage-gated calcium channels to open, and open calcium channelscause depolarization.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='23345b2b-5cb4-4071-9a81-713504ed9d30', embedding=None, metadata={'page_label': '335', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-15 WY001-Bolsover-v2.cls August 21, 2003 19:36\\nTHE CALCIUM ACTION POTENTIAL IN SEA URCHIN EGGS 329\\nB\\nACD\\nE−10\\n−50\\n−70−3010transmembrane voltage\\nD\\nK+K+Ca2+\\nCa2+Ca2+\\nCa2+\\nCa2+B\\nK+K+1 stime\\nE\\nK+K+Ca2+\\nmembrane \\nof spermC\\nK+K+Ca2+ voltage-gated \\ncalcium channels\\nA\\npotassium \\nchannels−70 mV −60 mV −50 mV\\n−30 mV −50 mV\\nFigure 15.3. The calcium action potential in a sea urchin egg.\\nIn nature the initial depolarization that sets off an action potential is produced by the\\nsperm. This is illustrated at point E. The membrane of the sperm contains an unusual type\\nof calcium channel that is open even at the resting voltage. When the membrane of thesperm is added to that of the egg, the inward current through the sperm ’s calcium channels\\ndepolarizes the plasma membrane of the egg by 20 mV to −50 mV , enough to cause the', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='fecadfe1-c0a9-4899-a44c-8aaac9dc11db', embedding=None, metadata={'page_label': '336', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-15 WY001-Bolsover-v2.cls August 21, 2003 19:36\\n330 THE ACTION POTENTIAL\\ncritical number of the egg ’s own voltage-gated calcium channels to open. The mechanism\\nof the action potential takes over and the plasma membrane depolarizes rapidly to +10 mV .\\nThe graph has been cut off at the right-hand edge (where the voltage-gated calcium channels\\ninactivate) where other mechanisms, not described here, take over and maintain a positivetransmembrane voltage. The action potential has served its purpose in preventing secondand subsequent sperm fusions in the 100 ms following the ﬁrst fertilization.\\nTHE VOLTAGE-GATED SODIUM CHANNEL IN NERVE CELLS\\nThe voltage-gated calcium channel appeared early in evolution and is found in all eukaryotesand in a wide variety of cell types. Later on in evolution, when multicellular organisms arose,there was a need for a system that could send electrical signals rapidly over long distanceswithin the body. This system uses sodium rather than calcium ions.\\nThe Voltage-Gated Sodium Channel\\nThe cells in multicellular animals that are specialized for rapid conduction have a second,voltage-gated channel that is selective for sodium instead of calcium ions. It operates likethe voltage-gated calcium channel, but both opening and inactivation are faster (Fig. 15.4).In summary:\\n1. When the transmembrane voltage is −70 mV , the voltage-gated sodium channel is\\ngated shut.\\n2. When the plasma membrane is depolarized, the channel opens rapidly and then,\\nafter about 1 ms, inactivates.\\n3. After the channel has gone through this cycle, it must spend at least 1 ms with the\\ntransmembrane voltage at the resting voltage before it can be opened by a seconddepolarization.\\nrepolarizationextracellular \\nmedium\\nCytosolNa+spontaneous \\nafter 1 msdepolarization\\ninactivation plugrepolarization\\nFigure 15.4. The voltage-gated sodium channel.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='0caf187e-67e0-4266-a81c-b7ebc3655d6e', embedding=None, metadata={'page_label': '337', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-15 WY001-Bolsover-v2.cls August 21, 2003 19:36\\nTHE VOLTAGE-GATED SODIUM CHANNEL IN NERVE CELLS 331\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 15.1 Chewing off the Inactivation Plug\\nProteases are enzymes that hydrolyze the peptide bonds within proteins. In 1976 Emilio Rojas and\\nBernardo Rudy investigated the effect of introducing protease into squid axons. The membrane ofsquid axons contains voltage-gated sodium channels that, like ours, normally inactivate about 1 msafter they are opened by a depolarization. However, after introduction of the protease, depolariza-tion caused the voltage-gated sodium channels to open and remain open inde ﬁnitely, although the\\nchannels would close if the membrane was repolarized. Back in 1976 Rojas and Rudy had no ideawhy this should be so. Now we can understand the result —the protease cuts the linker between the\\nmain part of the voltage-gated sodium channel and the inactivation plug. The plug then ﬂoats off\\nand is not available to block the open channel.\\ndepolarization\\nrepolarization\\nNa+\\nLike calcium, sodium is at a much higher concentration outside the cell than inside. In a\\nmammal, the sodium concentration in the blood is 150 mmol liter−1, whereas in the cytosol\\nit is 10 mmol liter−1. When voltage-gated sodium channels open, sodium ions rush into the\\ncell carrying positive charge and depolarizing the plasma membrane. This then favors the\\nopening of more voltage-gated sodium channels, and so on. This positive feedback producesa depolarization to +30 mV called the sodium action potential. Because the voltage-gated\\nsodium channels inactivate so quickly, the sodium-based action potential lasts only for 1 ms.\\n✄\\n✂\\x00\\n✁Medical\\nRelevance\\n15.1A Sodium “Channelopathy”\\nEpilepsy, in which nerve cells in the brain ﬁre action potentials in an uncontrolled\\nfashion, is not uncommon in young children but often clears up as the child gets older.\\nHowever, one form of childhood-onset epilepsy, termed GEFS+for“generalized epilepsy with\\nfebrile seizures that extend beyond six years of age, ”does not improve with age. Robyn Wallace\\nand co-workers in Australia showed that this condition could be caused by a mutation that slows\\nthe rate at which the inactivation plug moves to block the voltage-gated sodium channel. The slowrate of inactivation makes it easier for stimuli to trigger action potentials in the nerve cells in thebrain.\\nAlthough enough sodium ions move into the cell to dramatically change the trans-\\nmembrane voltage, the concentration of sodium ions inside the cell is increased only very\\nslightly. The amount depends on the electrical capacitance of the cell membrane and the cellvolume, but, for example, it can be calculated that a single sodium-based action potentialincreases the sodium concentration in the nerve cell axon of Figure 15.7 by only 250 nmolliter\\n−1.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='80ffccf0-e162-44f4-9fc0-7f0770670661', embedding=None, metadata={'page_label': '338', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-15 WY001-Bolsover-v2.cls August 21, 2003 19:36\\n332 THE ACTION POTENTIAL\\nElectrical Transmission down a Nerve Cell Axon\\nFigures 15.5 and 15.6 show how one nerve cell, a pain receptor, uses sodium-based action\\npotentials to signal to the central nervous system. The cell body is close to the spinal cordand extends an axon that branches to the spinal cord and out to the body. The particular pain\\nreceptor illustrated in Figure 15.5 sends its axon for almost a meter to the tip of a ﬁnger,\\nan extraordinary distance for a single cell. Potentially damaging events are detected at theﬁnger, and the message is passed on to another nerve cell (a pain relay cell) in the spinal\\ncord. We will now explain how this function is performed.\\nFigure 15.5 shows the transmembrane voltage of the pain receptor at the two axon\\nterminals, one near the tip of the ﬁnger (green line) and one at the opposite end of the\\nhand in\\nwarm air\\nspinal cordpain relay nerve cell\\nproximal axon terminalrepresents\\ntransmembrane voltage\\nmeasured at proximalaxon terminal\\ncell body ofpain receptor\\naxon of pain receptor\\nrepresents\\ntransmembrane voltagemeasured at distal axonterminalwarmair froma handdrier−70−50−30−1010transmembrane voltage03 0 6 0 m s\\nFigure 15.5. Mild heat stimulus is not felt as pain because the pain receptor is not depolarized to\\nthreshold.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='129c0d24-8d95-4345-8ec1-86ca632840a9', embedding=None, metadata={'page_label': '339', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-15 WY001-Bolsover-v2.cls August 21, 2003 19:36\\nTHE VOLTAGE-GATED SODIUM CHANNEL IN NERVE CELLS 333\\n−70−50−30−101030transmembrane voltage03 0 6 0 m shand in\\nhot air\\npain relay nerve cell\\nproximal axon terminalrepresents\\ntransmembrane voltage\\nmeasured at proximal axonterminal\\nspinal cordcell body ofpain receptor\\naxon of painreceptor\\nrepresents\\ntransmembrane voltagemeasured at distal axonterminalhot airfrom ahairdrier\\nFigure 15.6. Stronger heat stimulus evokes a propagating action potential.\\ncell, in the spinal cord (dashed black line). The axon terminal in the skin is the distal,\\nor far away, terminal and that in the spinal cord is the proximal one. When the ﬁnger is\\npassed through the warm air from a hand dryer, the heat causes the plasma membrane of the\\ndistal terminal to depolarize by 10 mV (continuous green line). This depolarization is notenough to open more than a very few voltage-gated sodium channels. Even in the warm air,the transmembrane voltage of the distal axon terminal is below threshold, and as soon as theﬁnger leaves the warm air jet it sags back to −70 mV under the in ﬂuence of the potassium\\nchannels. The axon is so long that the small voltage change at the distal terminal does notaffect the voltage at the proximal terminal at all (dashed black line). No signal passes to thepain relay cell, and the subject feels no sensation of pain.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4bd54224-7dbe-4cdb-98d9-af518c81b458', embedding=None, metadata={'page_label': '340', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-15 WY001-Bolsover-v2.cls August 21, 2003 19:36\\n334 THE ACTION POTENTIAL\\nFigure 15.6 shows what happens if we pass our ﬁnger through hotter air. The depolar-\\nization caused by the heat is greater and this causes more voltage-gated sodium channels\\nto open (continuous green line). Just as in the sea urchin egg at fertilization, the plasmamembrane then depolarizes rapidly (in this case as more and more sodium enters) un-til the transmembrane voltage is +30 mV . After only 1 ms, the voltage-gated sodium\\nchannels inactivate, and the transmembrane voltage returns to −70 mV , due to the action\\nof the potassium channels. At the proximal terminal (dashed black line), a meter away,there is also an action potential after a short delay. The signal then passes on to the brainand the person notices the hot air in time to pull the hand away before too much dam-age is caused. The pain receptor is one example of how we are quickly made aware ofchanges in our environment by sodium action potentials traveling rapidly along nerve cellaxons.\\nMyelination and Rapid Action Potential Transmission\\nThe axon transmits the action potential so rapidly because for most of its length it isinsulated by a fatty sheath called myelin made by glial cells (the glial cells outside the brainand spinal cord are often called Schwann cells ). Figure 15.7 shows part of the axon of the\\npain receptor and its associated glial cells, each of which wraps around the axon to form anelectrically insulating sheath. Only at short gaps called nodes is the membrane of the nerve\\ncell exposed to extracellular ﬂuid. Note that the vertical and horizontal scales of this diagram\\nare completely different. The axon together with its myelin sheath is only 3 µm across, but\\nthe section of axon between nodes is 1 mm long, a large distance by normal cell standards.Between nodes the axon is an insulated electrical cable: just as voltage changes at one endof an insulated metal wire have an almost instantaneous effect on the voltage at the otherend, so a change in transmembrane voltage at one node has an almost instantaneous effectat the next node along. The plasma membrane at the nodes has potassium and voltage-gatedsodium channels and can therefore generate action potentials.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 15.2 Local Anaesthetic, Overall Well-being\\nNerve conduction is exploited when a dentist uses local anaesthetic. A patient feels pain from the\\ndrill because pain receptors in the tooth are depolarized and transmit action potentials toward thebrain. The site at which they are being depolarized is inside the tooth and therefore inaccessible todrugs until the drill has made a hole. However, the axons of the pain receptors run through thegum. Local anaesthetics injected into the gum close to the axon bind to the nerve cell membraneat the node and prevent the opening of voltage-gated sodium channels. Drilling into the tooth stilldepolarizes the pain receptor membrane, and action potentials begin their journey toward the brain,but they cannot pass the injection site because the nodes there cannot generate an action potential.The patient therefore feels no pain.\\nFigure 15.7 represents a snapshot of one brief instant in the operation of the nerve cell.\\nThe node on the left is generating an action potential. V oltage-gated sodium channels are\\nopen, sodium ions are ﬂooding in, and the transmembrane voltage is +30 mV . Some of\\nthe charge ﬂowing in is carried out again by potassium ions ﬂowing through the potassium\\nchannels. However, some positive charge ﬂows axially up and down the cytosol of the nerve', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='8d65d6b7-2c63-4310-b8e2-9ae1d5ee1cdf', embedding=None, metadata={'page_label': '341', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-15 WY001-Bolsover-v2.cls August 21, 2003 19:36\\nK+Na+\\nNa+sodium channel\\n+30 mV\\ncytoplasm of nerve cell\\nfatty sheath of myelin \\ncreated by glial cell\\nnucleus of glial cell1 mm\\n3 µm\\nnode\\n3 µmpotassium channel\\nNa+axial current flow\\nFigure 15.7. The myelin sheath and the node.\\n335', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='63a8f271-c1a8-445b-a11d-c2575e225f62', embedding=None, metadata={'page_label': '342', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-15 WY001-Bolsover-v2.cls August 21, 2003 19:36\\n336 THE ACTION POTENTIAL\\ncell. Because myelin is an electrical insulator, this charge cannot leave the cytosol until the\\nnext node. At the node on the right the incoming charge depolarizes the plasma membraneto threshold, and voltage-gated sodium channels at this node open as well. Sodium rushesin, and this node generates an action potential. The action potential has jumped from nodeto node in the extremely short time of 50 µs. Now sodium ions will rush into the node on\\nthe right, current will pass axially up the interior of the axon to the next node up, and ina further 50 µs the action potential will have jumped to the next node, and so on, all the\\nway up to the proximal terminal at an overall speed of 20 m s\\n−1. This process, in which the\\naction potential jumps from node to node, is named after the Latin for “to jump, ”saltere,\\nso it is saltatory conduction.\\nThe combination of axon plus myelin sheath is called a nerve ﬁber. The nerve ﬁber\\nshown in Figure 15.7 has an overall diameter of 3 µm. The human body contains myelinated\\nnerve ﬁbers with diameters ranging from 1 µm, conducting a t6ms−1,t o1 0 µm, conducting\\nat 60 m s−1. A number of axons are not myelinated and generally conduct more slowly\\nthan myelinated ﬁbers. The axons of scent-sensitive nerve cells (page 350) are an example.\\nGetting scent information from the nose to the brain as fast as possible is apparently not asigni ﬁcant advantage, given that a scent can take many seconds to waft from its source to\\nour nose.f\\nIN DEPTH 15.1 Frequency Coding in the Nervous System\\nAt the distal axon terminal of the pain receptor, the intensity of the stimulus is\\nrepresented by the amount of depolarization produced. The hotter the air blowingon the skin, the more positive the transmembrane voltage of the terminal becomes.Other sensory cells behave in the same way. For instance, in Chapter 16 we will meetthe scent-sensitive nerve cells in the nose that detect smell chemicals. The higherthe concentration of the smell chemical, the more these cells depolarize.\\nHowever, the signal that passes toward the brain is an all-or-none action po-\\ntential. Either the stimulus produces a depolarization that is below threshold, inwhich case no signal passes to the brain, or the membrane depolarizes beyondthreshold, in which case an action potential is generated. How then is the braintold about the intensity of the stimulus?\\nAction potentials continue to be produced and travel up toward the brain as\\nlong as the stimulus is maintained. The diagrams represent what happens when thehand is being held in a constant airstream. If the stimulus is strong, the membraneof the pain receptor depolarizes rapidly, so that after each action potential thethreshold voltage is soon reached once more. Thus time between successive actionpotentials is short. The stronger the stimulus, the higher the frequency of actionpotentials generated.\\nAmplitude modulation (AM) and frequency modulation (FM) are familiar terms\\nfrom radio. Here we are seeing exactly the same two coding strategies in thenervous system. AM is used in the distal terminals of the pain receptor. The strengthof stimulus is coded for by the amplitude of the depolarization. FM is used inalmost all axons, including those of the pain receptor. The strength of stimulusis coded for by the frequency of action potentials, each of which has the sameamplitude.\\nMan-made systems use exactly the same strategy. On the control unit of a\\ngames console, for instance, the force with which the player presses on a buttonis coded in terms of the amplitude of a voltage change. This is then recoded inall-or-none digital pulses for transmission to the main console.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='94acba30-1f8f-4d01-9e0a-55775ef7aa74', embedding=None, metadata={'page_label': '343', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-15 WY001-Bolsover-v2.cls August 21, 2003 19:36\\nSUMMARY 337\\nstimulus just over thresholdTransmembrane voltage (mV)−70mV01 second\\nstrong stimulusTransmembrane voltage (mV)−70mV0\\nTransmembrane voltage (mV)−70mV0Transmembrane voltage (mV)−70mV01 secondTransmembrane voltage (mV)weak stimulus\\n−70mV01 second\\nTransmembrane voltage (mV)−70mV01 second\\n1 second 1 second\\nSUMMARY\\n1.The voltage-gated calcium channel is shut at the resting voltage but opens upon\\ndepolarization. After about 100 ms the channel inactivates.\\n2.While the calcium channel is open, calcium ions pour into the cell down their elec-trochemical gradient. The mutual effect of current through the channel on trans-membrane voltage, and of transmembrane voltage on current through the channel,constitutes a positive feedback system. If the membrane is initially depolarized tothreshold, the positive feedback of the calcium current system ensures that depo-larization continues in an all-or-nothing fashion. Repolarization occurs when the', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1a44f595-1e17-476a-93f5-fbee2e8b61db', embedding=None, metadata={'page_label': '344', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-15 WY001-Bolsover-v2.cls August 21, 2003 19:36\\n338 THE ACTION POTENTIAL\\ncalcium channel inactivates. The entire cycle of depolarization and repolarization is\\ncalled an action potential.\\n3.The same process occurs in cells expressing voltage-gated sodium channels. But,since these channels inactivate within about 1 ms, the action potential lasts only thislong. Long nerve cell processes called axons transmit action potentials at speeds upto 60 m per second. They can transmit the signal at such high rates because myelin,a fatty sheath, insulates the 1 mm distances between nodes.\\n4.A number of axons are not myelinated; these generally conduct action potentialsmore slowly than do myelinated axons and are found where high conducation speedis not required.\\nFURTHER READING\\nAshcroft, F. M. 2000. Ion Channels and Disease . San Diego: Academic Press.\\nLevitan, I. B., and Kaczmarek, L. K. 2002. The Neuron . New York: Oxford University Press.\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.A sea urchin egg is used in a voltage clamp experiment; that is, the value of the trans-\\nmembrane voltage is set by the experimenter, not by the properties of the membrane.The transmembrane voltage is held at +10 mV for one minute. At the end of that time\\nA. the voltage-gated calcium channels are all open.\\nB. the voltage-gated calcium channels are all inactivated.C. the voltage-gated calcium channels have recovered from inactivation.D. the voltage-gated calcium channels are going through cycles of opening and inactivation as the\\ncell generates repetitive action potentials.\\nE. any of answer A though D above could occur, depending on the density of potassium channels.\\n2.In a calcium action potential\\nA. the in ﬂux of calcium ions depolarizes the plasma membrane.\\nB. depolarization of the plasma membrane opens voltage-gated calcium channels.C. open voltage-gated calcium channels carry an inward current of calcium ions.D. an inward current of calcium ions adds positive charge to the cytosol.E. all of the above occur.\\n3.Action potentials in nerve cell axons last a much shorter time than action potentials in\\nsea urchin eggs because\\nA. sodium has only a single positive charge while calcium has two.\\nB. voltage-gated sodium channels are found only at the nodes, while voltage-gated calcium chan-\\nnels are found over the entire surface of the egg cell.\\nC. sodium ions are removed rapidly from the cytosol by the Na+/K+ATPase.\\nD. voltage-gated sodium channels inactivate more rapidly than do voltage-gated calcium channels.E. calcium has a higher atomic weight than does sodium and therefore diffuses more slowly.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='82a386a3-473b-470d-a636-84c8e158aaf0', embedding=None, metadata={'page_label': '345', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-15 WY001-Bolsover-v2.cls August 21, 2003 19:36\\nANSWERS TO REVIEW QUESTIONS 339\\n4.An action potential is being generated at the node of a myelinated axon. Which of the\\nfollowing statements is false?\\nA. Sodium is entering the axon interior through voltage-gated sodium channels.\\nB. Potassium is leaving the axon interior through potassium channels.C. Calcium is entering the axon interior through voltage-gated calcium channels.D. A net inward current is ﬂowing across the plasma membrane of the node.\\nE. Current is leaving the region of the node axially, through the cytosol of the nerve cell.\\n5.The nodes of a myelinated axon are spaced a distance apart of approximately\\nA. 1 nm.B. 1µm.\\nC. 1 mm.D. 1 cm.E .1m .\\n6.A sperm can trigger an action potential in a sea urchin egg because\\nA. its cytosol contains a lot of calcium ions that, upon fertilization, rush into the egg.B. its membrane contains ion channels that, upon fertilization, pass an inward current that depo-\\nlarizes the egg plasma membrane.\\nC. its membrane contains ion channels that, upon fertilization, pass an outward current that repo-\\nlarizes the egg plasma membrane.\\nD. its cytosol contains a chemical that tends to pop the voltage-gated calcium channels into an\\nopen state.\\nE. its cytosol contains a protease that modi ﬁes the behavior of voltage-gated calcium channels in\\nthe plasma membrane of the egg.\\n7.When a nerve cell axon is depolarized to threshold, the ﬂow of potassium through\\npotassium channels in the plasma membrane is\\nA. outward, and of a greater absolute amplitude than at the resting voltage.B. inward, and of a greater absolute amplitude than at the resting voltage.C. outward, but of a smaller absolute amplitude than at the resting voltage.D. inward, but of a smaller absolute amplitude than at the resting voltage.E. zero.\\nANSWERS TO REVIEW QUESTIONS\\n1.B.They are all inactivated. Upon depolarization they opened and inactivated, and since the trans-\\nmembrane voltage was never returned to the resting value, the channels have not recovered frominactivation. Regarding answer D, note that the cell cannot generate repetitive action potentialsbecause it is voltage clamped: The value of the transmembrane voltage is set by the experimenter,not by the properties of the membrane. For this reason the density of potassium channels (answerE) is irrelevant —the density of potassium channels will affect the current through those channels,\\nbut since the cell is voltage clamped, the current through potassium channels will have no effecton transmembrane voltage and therefore no effect on the behavior of the voltage-gated calciumchannels.\\n2.E.Answers A through D are true. The in ﬂux of calcium acts, through the voltage change it causes,\\nto increase the in ﬂux of calcium —this is why the generation of an action potential is an example\\nof positive feedback.\\n3.D.Statements A, B, C, and E are all true in themselves but are not reasons why action potentials in\\nnerve cell axons are shorter than those in sea urchin eggs. Concerning answer C, the sodium ions', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3b18d81d-75e6-4f2c-abff-733a064b54c7', embedding=None, metadata={'page_label': '346', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-15 WY001-Bolsover-v2.cls August 21, 2003 19:36\\n340 THE ACTION POTENTIAL\\ndo not have to be removed from the cytosol in order for the action potential to terminate. The very\\nsmall amounts of sodium that do enter (see page 331) are removed over a timescale of seconds bythe Na\\n+/K+ATPase. If the Na+/K+ATPase is blocked by the toxin digitalis (page 271), a nerve\\ncell can continue to generate action potentials as normal. Only after many thousands of actionpotentials have occurred in a digitalis-poisoned nerve cell will the cytosolic sodium concentrationrise to a concentration suf ﬁcient to signi ﬁcantly affect the threshold for action potential generation.\\n4.C.There are no voltage-gated calcium channels at the node. All the other statements are true:\\nalthough some charge is being carried out of the axon at the node by potassium, the current carriedby the inward movement of sodium is larger, so there is a net inward current. The excess chargeis carried out of the node axially.\\n5.C.\\n6.B.Statements A, D, and E are untrue. Statement C is true in itself in that, like most cells, the sperm\\nplasma membrane will contain some potassium channels that will pass an outward current, but asthe answer states, the action of these will be to repolarize the egg membrane, not depolarize it.The depolarizing effect of the calcium in ﬂux through the sperm calcium channels, which are open\\neven at the resting voltage, is greater.\\n7.A.Potassium ions are usually attracted to the negative interior of the cell, and this force opposes\\nthe tendency of potassium ions to leave down their concentration gradient. Depolarization reducesthe inward electrical force, so potassium ions leave at a greater rate than at rest. At the same time,voltage-gated sodium channels are carrying an inward current, but that movement is not relevantto the question.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='db0df2ff-9c2a-4f88-a1dc-fc96299ca1a7', embedding=None, metadata={'page_label': '347', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\n16\\nINTRACELLULAR SIGNALING\\nThe behavior of cells is not constant. Cells need to be able to alter their behavior in response\\nto internal changes or to external events, and the internal signaling mechanisms that allowthem to do so are varied and complex. We will begin by continuing the story begun inChapter 15 and explain how calcium ions carry a signal from the plasma membrane ofnerve cells to vesicles deep within the cytosol. The rest of the chapter will describe someother very different methods of intracellular signaling.\\nCALCIUM\\nCalcium ions are present at a very low concentration (about 100 nmol liter−1) in the cytosol\\nof a resting cell. An enormous number of processes in many types of cells are activated whenthe concentration of calcium rises. Calcium can move into the cytosol from two sources:the extracellular medium or the endoplasmic reticulum.\\nCalcium Can Enter from the Extracellular Medium\\nIn Chapter 15, we saw how heating the hand causes action potentials to travel from the handto the spinal cord along the axons of pain receptors. When the action potential reaches theproximal axon terminal, the amino acid glutamate is released onto the surface of anothernerve cell, the pain relay cell. Glutamate is a transmitter that stimulates the pain relay nervecell, so that the message that the ﬁnger is being damaged is passed toward the brain.\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n341', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='36fbb361-b0f9-4051-aafa-c2730075c52e', embedding=None, metadata={'page_label': '348', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\n342 INTRACELLULAR SIGNALING\\n100 nm gap\\n100 nm gapspinal cord\\nvesicles containing\\n100 m mol liter−1\\nglutamatevoltage-gated\\ncalcium channelsvoltage-gated\\nsodium channelspotassium\\nchannelsto skincell body of\\npain receptorpain relay nerve cell\\nproximal axon terminal\\n− 70 mV[Ca2+] = 100 n mol\\nliter−1\\n+ 30 mV[Ca2+] = 1 µ mol\\nliter−1[Ca2+] = 2 m mol liter−1\\n[Ca2+] = 2 m mol liter−1exocytosis of\\nglutamatepainrelaynervecell\\npain\\nrelaynervecell\\nFigure 16.1. Calcium ions entering the cytosol from the extracellular ﬂuid activate regulated\\nexocytosis in the proximal axon terminal of the pain receptor nerve cell.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='babf4323-d6a4-48dc-bac2-1d9a6c2b9aed', embedding=None, metadata={'page_label': '349', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\nCALCIUM 343\\nThe proximal axon terminal of the nerve cell is unmyelinated, so that the plasma\\nmembrane is exposed to the extracellular medium (Fig. 16.1). The membrane contains not\\nonly potassium channels and voltage-gated sodium channels, but also voltage-gated calciumchannels. These are closed in a resting cell, but when an action potential travels in fromthe skin and depolarizes the plasma membrane of the proximal axon terminal, the voltage-gated calcium channels open. Calcium ions pour in, increasing their concentration in thecytosol by 10 times, from the normal concentration of 100 nmol liter\\n−1to 1µmol liter−1.\\nAt the proximal axon terminal the cytosol of the nerve cell contains regulated exocytoticvesicles (page 229). In the case of the pain receptor, these vesicles are ﬁlled with sodium\\nglutamate. In response to the increased concentration of cytosolic calcium the regulatedexocytotic vesicles move to the plasma membrane and fuse with it, releasing their contentsinto the extracellular medium. The glutamate then diffuses across the gap to the pain relaycell, stimulating it (we will describe how on page 372). As the action potential in the axonterminal of the pain receptor cell is over in 1 ms, the voltage-gated calcium channels do nothave time to inactivate. They simply return to the ready-to-open state and can be reopenedimmediately by the next action potential that arrives.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 16.1 Visualizing Calcium Signals\\n(a)\\n(b)\\nCalcium signals in cells can be visualized by using dyes that ﬂuoresce brightly when they bind\\ncalcium. The images in the ﬁgure show a cell body of a pain receptor cell that has been ﬁlled with a\\ncalcium indicator dye. In the ﬁrst image, at rest, the cell is dim. The second image was acquired after\\n100 ms of depolarization. In this the edges of the cell are bright because calcium ions have entered\\nthrough voltage-gated calcium channels. The bright object in the lower part of the cell is the nucleus.\\nIn the process that we have described, calcium ions act as a link between the depolar-\\nization of the plasma membrane and the regulated exocytotic vesicles within the cytosol.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a8c6a531-59b3-42b0-8626-e12648225cef', embedding=None, metadata={'page_label': '350', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\n344 INTRACELLULAR SIGNALING\\nCalcium ions are intracellular messengers. Regulated exocytotic vesicles that are triggered\\nby an increase of cytosolic calcium concentration were ﬁrst discovered in nerve cells but\\nare now known to be a feature of almost all cells.\\nBefore we continue with the topic of intracellular messengers, we will review two\\ngeneral points raised by our description of the pain receptor proximal axon terminal. The ﬁrst\\npoint is that different nerve cells release different transmitters. Glutamate is in the exocytotic\\nvesicles of many nerve cells, but other nerve cells release other transmitters. The second pointconcerns nomenclature. Many nerve cells, like the pain receptor, have their axon terminalsclose to a second cell and release their transmitter onto it. In such cases the completeunit of axon terminal, gap, and the part of the second cell that receives the transmitteris called a synapse. The part of the axon terminal that releases transmitter is called a\\npresynaptic terminal, and the cell upon which the transmitter is released is called the\\npostsynaptic cell. Many nerve cells do not come close enough to a speci ﬁc second cell to\\nform a synapse but simply release transmitter from their axon terminals into the extracellularmedium.\\nCalcium Can Be Released from the Endoplasmic Reticulum\\nCells may show an increase of cytosolic calcium not because of an action potential butbecause of the appearance of a transmitter or other chemical in the extracellular medium.The presence of the chemical is detected by integral membrane proteins, each one a receptorthat recognizes a particular chemical with high af ﬁnity. These receptors then participate in\\na more general mechanism, the end result of which is the release of calcium ions from thesmooth endoplasmic reticulum into the cytosol. This mechanism will be illustrated with theparticular example of blood platelets. We will then discuss how much of the mechanism isgeneral to a wider range of cells.\\nPlatelets are common in the blood. They are small fragments of cells and contain no\\nnucleus, but they do have a plasma membrane and some endoplasmic reticulum. Bloodplatelets use the release of calcium from the endoplasmic reticulum as one step in themechanism of blood clotting (Figs. 16.2 and 16.3). Two new mechanisms are involved incalcium release from the endoplasmic reticulum. The inositol trisphosphate-gated calciumchannel (Fig. 16.3), like the voltage-gated calcium channel, allows only calcium ions topass. Most of the time its gate is shut, and no ions ﬂow. It is not opened by a change in the\\nvoltage across the endoplasmic reticulum membrane. Instead, when the intracellular soluteinositol trisphosphate (IP\\n3for short) binds to the cytosolic face of the channel, the channel\\nchanges to an open shape.\\nThe second new mechanism that we must describe is the one that makes IP 3(Fig. 16.2).\\nIf a blood vessel is cut, cytosol from damaged cells at the edge of the cut can leakinto the blood. The appearance of solutes that normally are found only inside cells is asure sign that damage has occurred. Adenosine diphosphate (ADP) is one such solute,and it acts to stimulate platelets, causing them to begin a blood clot to help plug thedamaged vessel. The plasma membrane of the platelet contains a protein receptor thatbinds ADP, that is, ADP is its ligand. When the ADP has bound, the receptor, which isfree to move in the plasma membrane, becomes a guanine nucleotide exchange factor(page 218) for a trimeric G protein called G\\nq. Like the GTPases Ran, ARF, and Rab\\nthat we have met earlier, trimeric G proteins are GTPases that activate target proteinswhen they have GTP bound, but turn themselves off by hydrolyzing the GTP to GDP.Trimeric G proteins have a slight additional complication in that, as the name indicates,', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='bc84747c-8fcb-4ae0-a7df-80e260cadd62', embedding=None, metadata={'page_label': '351', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\nβ\\nγ OPO\\nO\\nOOO\\nOPO\\nO OOP O α\\nβγGTP\\nGDPGTPPIP2\\ntrimeric\\nG protein\\nGqphospholipase\\nCβADP\\nreceptor\\nGDPADP\\nDAGOH\\nα\\nOP O OO O\\nO\\nOPO\\nO OOP O\\ninositoltrisphosphateblood plateletsADP released from damaged \\ncells in vessel wall\\nFigure 16.2. ADP from damaged cells activates G qand hence phospholipase Cβ in platelets.\\n345', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3e0d886f-9478-4fa8-88e4-c277dd5131a5', embedding=None, metadata={'page_label': '352', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\n346 INTRACELLULAR SIGNALING\\nCa++endoplasmic \\nreticulumblood platelet\\nhigh \\nCa++\\ninterior ofendoplasmicreticulum cytosolCa++\\nCa++IP3 bindsIP3 binding siteADP\\nIP3PIP2DAG\\nFigure 16.3. The inositol trisphosphate-gated calcium channel receptor is a calcium-selective chan-\\nnel in the membrane of the endoplasmic reticulum. An increase of cytosolic calcium concentration\\nin platelets makes them sticky, initiating blood clotting.\\nthey are composed of three subunits. The αsubunit is homologous to the GTPases we\\nhave met before, while the βandγsubunits dissociate from the αsubunit when it has\\nGTP bound and only reassociate when the GTP has been hydrolyzed. GTP-loaded G q\\nactivates the βisoform of an enzyme called phosphoinositide phospholipase C, which we\\nwill call phospholipase C or just PLC. (The capital C refers to the bond hydrolyzed by\\nthe enzyme; A, B, and D phospholipases also exist, but we will not meet them in this\\nbook). PLC speci ﬁcally hydrolyzes phosphatidylinositol bisphosphate (PIP 2for short),\\na phospholipid in the plasma membrane that has phosphorylated inositol as its polar headgroup. Hydrolysis releases the IP\\n3to diffuse freely in the cytosol, leaving behind the glycerol\\nbackbone with its attached fatty acids: this residual lipid is called diacylglycerol, or DAG.As it diffuses through the cytosol, IP\\n3reaches the endoplasmic reticulum and binds to the\\ninositol trisphosphate-gated calcium channels. The inositol trisphosphate-gated calcium', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2b229b88-2552-486c-8455-9543b262d718', embedding=None, metadata={'page_label': '353', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\nCa++\\nATP ADP\\nactivate force \\ngeneration by \\ncytoskeleton glycogen glucose\\nactivate production of \\nglucose ready for use by glycolysis and hence mitochondriaATP\\nGTP\\ncAMP\\nactivate enzymes of Krebs cycle in mitochondria\\nmitochondrial \\ncalcium channel porinadenylate cyclase\\nmuscle cellendoplasmic reticulumadrenaline\\nβ-adrenergic receptor\\nstimulus\\nfrom\\nnerve\\nFigure 16.4. Calcium and cyclic-AMP activate distinct but overlapping sets of target processes in skeletal muscle cells.\\n347', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='431c0d74-3dab-4f6b-93f8-5d7aa5b193ae', embedding=None, metadata={'page_label': '354', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\n348 INTRACELLULAR SIGNALING\\nchannels open, and calcium ions pour out of the endoplasmic reticulum into the cytosol.\\nThe increase of calcium concentration causes the platelet to change shape and to becomevery sticky, so platelets begin to clump together in a clot.\\nPhosphatidylinositol bisphosphate, G\\nq, PLCβ, and the inositol trisphosphate-gated cal-\\ncium channel are found in almost all eukaryotic cells, but the distribution of the ADPreceptor is more restricted. Only cells that express the ADP receptor will show an increaseof cytosolic calcium concentration in response to ADP. Other cells respond to other chem-icals. Each produces a receptor speci ﬁc for that chemical, which then activates G\\nq.O v e r\\n100 such receptors are known. We will meet two more in the next chapter.\\nProcesses Activated by Cytosolic Calcium Are Extremely Diverse\\nThe targets activated by an increase of cytosolic calcium concentration differ betweendifferent cells. An increase of calcium is a crude signal that says “do it”but contains no\\ninformation about what the cell should do. This depends on what the cell was designed todo. Cells and cell regions designed for regulated exocytosis (e.g., salivary gland cells andaxon terminals) exocytose when cytosolic calcium increases. Cells designed to contract(e.g., muscle cells) contract when calcium increases, and so on. In each case, the calciumions bind to a calcium-binding protein; the calcium ion –calcium-binding protein complex\\nactivates the target process.\\nIn skeletal muscle cells (page 14) a transmitter released from nerve axon terminals\\nleads to the escape of calcium from the endoplasmic reticulum (Fig. 16.4). This activatesseveral processes. First, calcium ions bind to a protein called troponin that is attached to thecytoskeleton. This causes the cytoskeleton to contract, using the energy released by ATPhydrolysis to do mechanical work (page 393). Second, the calcium binds to the proteincalmodulin, which in turn activates glycogen phosphorylase kinase and hence glycogenbreakdown as part of the feedforward control of energy metabolism (page 305).\\nMitochondria are also affected by calcium, which passes down its electrochemical\\ngradient from the cytosol into the mitochondrial matrix through a calcium channel. Oncethere, calcium stimulates the mitochondria to increase the production of NADH and ATP(page 305).f\\nIN DEPTH 16.1 Ryanodine Receptors\\nMany cells contain a second type of calcium channel in the membrane of the endo-\\nplasmic reticulum. This channel was initially distinguished from the IP 3receptor by\\nthe fact that it bound the plant alkaloid ryanodine, so it was called the ryanodine\\nreceptor. Ryanodine receptors were ﬁrst identi ﬁed in skeletal muscle. In these cells\\nthey are physically linked to voltage-gated calcium channels in the plasma mem-brane (left-hand side of diagram). When the voltage-gated calcium channels switchto their open con ﬁguration, they induce the ryanodine receptors below them to\\nswitch into an open con ﬁguration, allowing calcium to ﬂow out of the smooth\\nendoplasmic reticulum into the cytosol.\\nNo such direct linkage between ryanodine receptors and plasma membrane\\nchannels is found in other cell types. Instead, ryanodine receptors are openedwhen the concentration of calcium ions in the cytosol rises above a critical level.For example, in heart muscle (right-hand side of diagram) depolarization causesvoltage-gated calcium channels in the plasma membrane to open. The calcium ionsthat enter through this channel bind to the cytosolic aspect of the ryanodine recep-tor, causing it to open too, allowing calcium to ﬂow out of the smooth endoplasmic\\nreticulum into the cytosol.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='eb77b1f2-5007-4535-9f1b-384ebfc685e5', embedding=None, metadata={'page_label': '355', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\nCa++\\nCa++depolarization\\nrepolarizationdepolarization\\nrepolarization\\nlumen of\\nendoplasmicreticulumlumen ofendoplasmicreticulumryanodinereceptorcalcium\\nbinding\\nsite ryanodinereceptor\\nmembrane of\\nendoplasmic\\nreticulumplasmalemmaextracellular medium cardiac muscle\\nvoltage gated calcium channel voltage gated calcium channelskeletal muscle cell\\nCa++\\n349', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='98dc7121-35e1-4183-9617-e89e4b4eb009', embedding=None, metadata={'page_label': '356', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\n350 INTRACELLULAR SIGNALING\\nThe same simple intracellular messenger, calcium, has many different actions inside\\nthe skeletal muscle cell. Under its in ﬂuence the cytoskeleton begins to use ATP, glycogen\\nphosphorylase releases more glucose, and the mitochondria produce more ATP. The skeletal\\nmuscle cell is an example of how diverse mechanisms inside a cell can be integrated by theaction of intracellular messengers.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 16.2 Training for Anaerobic Respiration\\nMuscle cells that are contracted for long periods, cutting off the blood supply —such as those in\\nthe arms of rock climbers —have a much enhanced capacity for anaerobic respiration compared to\\nthose that are used brie ﬂy and then relaxed (such as those in the arms of baseball pitchers). This\\noccurs because the calcium signals that activate muscle contraction (page 393) also act to increase\\nthe transcription of the genes coding for glycolysis (via calcineurin and NFAT, page 222).\\nReturn of Calcium to Resting Levels\\nAs soon as a stimulus, be it depolarization or extracellular chemical, disappears, cytosolic\\ncalcium concentration falls again. Calcium ions are pumped up their electrochemicalgradients from the cytosol into the extracellular medium or into the endoplasmic reticulumby two carriers. The Ca\\n2+ATPase (page 318) uses the energy released by ATP hydrolysis\\nto move calcium ions up their electrochemical gradient out of the cytosol, while theNa\\n+/Ca2+exchanger (page 317) uses the energy released by sodium ion movement into\\nthe cytosol to do the same job.\\nCYCLIC ADENOSINE MONOPHOSPHATE\\nWe have already met the nucleotide cyclic adenosine monophosphate, or cAMP (Fig. 16.5),in the context of the regulation of the lacoperon (page 114). (It is worth mentioning that\\ncAMP is always pronounced “cy-clic A-M-P, ”with ﬁve syllables.) In eukaryotes, cAMP\\nis also important and acts as an intracellular messenger in a great many cells, includingthe scent-sensitive nerve cells in our nose (Fig. 16.6). These cells have their cell bodies inthe skin of the air passages in the nose. Each cell sends an axon into the brain, and shorterprocesses, called dendrites, into the mucus lining the air passages. Scent-sensitive nervecells are stimulated by scents in the air. Particular chemicals in the air stimulate these cellsbecause the cells have protein receptors that speci ﬁcally bind the scent (Fig. 16.7). When\\nthe scent binds, the receptor becomes a guanine nucleotide exchange factor for a trimeric Gprotein called G\\ns. GTP-loaded G sactivates an enzyme called adenylate cyclase that converts\\nATP to cAMP. The next stage in the detection of scents is a channel in the plasma membranecalled the cAMP-gated channel. Like the inositol trisphosphate-gated calcium channel, thecAMP-gated channel is opened by a cytosolic solute, in this case cAMP. When the channelis open, it allows sodium and potassium ions to pass through. The electrochemical gradientpushing sodium ions into the cell is much greater than the electrochemical gradient pushingpotassium ions out of the cell. Thus, when the cAMP-gated channel opens, sodium ionspour in, carrying their positive charge and depolarizing the plasma membrane. The plasmamembrane also contains voltage-gated sodium channels. Thus, when enough cAMP-gatedchannels open, the transmembrane voltage reaches threshold for generation of sodium actionpotentials. These then propagate along the axon to the brain, and the person becomes awareof the particular scent for which that scent-sensitive nerve cell was speci ﬁc.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='96ee127b-99a3-4c46-8d1e-139a5e420033', embedding=None, metadata={'page_label': '357', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\nO\\nC\\nC CCHH\\nOHH H\\nO\\nOCH25′\\n3′O\\nP ON\\nCC\\nN C\\nN C\\nHN\\nC\\nNHH\\nH\\nFigure 16.5. Cyclic adenosine monophosphate, also called cyclic-AMP or just cAMP.\\naxon terminal\\naxon of scent-\\nsensitive nervecellbrain\\nFigure 16.6. Scent-sensitive nerve cells send axons to the brain.\\n351', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='24d96a7c-3423-47ed-bc15-65bac0e972e0', embedding=None, metadata={'page_label': '358', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\n352 INTRACELLULAR SIGNALING\\ncAMP\\nATPscent \\nchemical \\nin airdendrite of \\nscent-sensitive \\nnerve cell\\nscent chemicalcell body \\nof scent-sensitive nerve cellCytosolextracellularfluid\\ncytosolNa+\\nNa+cAMP binding site\\ncAMP binds\\nGDP\\nαGTPGTPα\\nATP\\nGDPadenylate\\ncyclasereceptor for scent chemical\\ntrimeric\\nG protein\\nGscAMP\\nβ\\nγ\\nβγNa+\\nNa+\\nFigure 16.7. Scent chemicals activate G sand hence adenylate cyclase in scent-sensitive nerve cells.\\ncAMP then opens a nonselective cation channel in the plasma membrane.\\nGsand adenylate cyclase are found in many cells, but only scent-sensitive nerve cells\\nare sensitive to scents, because only they have speci ﬁc scent receptors in their plasma\\nmembranes. Other cells that use cAMP as an intracellular messenger are sensitive to other\\nspeci ﬁc chemicals because each makes a receptor that binds the chemical, whatever it may\\nbe, and then activates G s. Once the stimulating chemical has gone, the cAMP concentration', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='65217a08-2dfc-4b4d-85d8-8ff48ba47ea1', embedding=None, metadata={'page_label': '359', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\nBIOCHEMICAL SIGNALING 353\\nreturns to resting levels. The enzyme cAMP phosphodiesterase hydrolyzes cAMP to AMP,\\nwhich is inactive at cAMP-gated channels and other cAMP-binding proteins.\\nMost of the symptoms of the deadly disease cholera are caused by a toxin released by\\nthe gut bacterium Vibrio cholera. The toxin is an enzyme that enters the cytosol of the cells\\nlining the gut and attaches an ADP ribosyl group to the catalytic domain of G s, preventing\\nit from hydrolyzing GTP. G sis therefore locked in the active state, and activates adenylate\\ncyclase nonstop. The cAMP concentration in the cytosol then shoots up. Ion channels in\\nthe plasma membrane are opened by the increase of cAMP, allowing ions to leak from thecells into the gut contents. If untreated this loss of ions, and of the water that accompaniesthem, leads to death from dehydration.\\nCYCLIC GUANOSINE MONOPHOSPHATE\\nAnother nucleotide that acts as a intracellular messenger is cGMP. In light-sensitive nervecells called photoreceptors, cGMP plays a role like that of cAMP in scent-sensitive nervecells. In the dark, the enzyme guanylate cyclase makes cGMP from GTP. cGMP binds toand opens a channel that allows sodium and potassium ions to pass, so that in the dark thephotoreceptor is depolarized to about −40 mV because of the constant in ﬂux of sodium\\nions through the cGMP-gated channel. In the light the concentration of cGMP in the cytosolfalls, and the transmembrane voltage changes to the more typical resting voltage of −70 mV .\\nThe changing transmembrane voltage is transmitted to other nerve cells, making us awareof the pattern of light and dark.\\nMULTIPLE MESSENGERS\\nMany cells use more than one intracellular messenger at once. Skeletal muscle cells area good example (Fig. 16.4). In the excitement before a race, the runner ’s adrenal glands\\nrelease adrenaline into the blood. This binds to a receptor on skeletal muscle cells calledtheβ-adrenergic receptor. The complex of adrenaline plus β-adrenergic receptor can now\\nactivate G\\nsand hence adenylate cyclase. This in turn generates cAMP, which activates\\ncAMP-dependent protein kinase. This enzyme, which is given the short name of protein\\nkinase A, phosphorylates glycogen phosphorylase kinase and turns on the latter enzyme\\neven when cytosolic calcium is low (see Fig. 13.15 on page 305). The end result is thateven before the runner begins to run, the muscles break down glycogen and make theglucose-6-phosphate (page 288) they will need once the race begins.\\nBIOCHEMICAL SIGNALING\\nReceptor Tyrosine Kinases and the MAP Kinase Cascade\\nWhen platelets are stimulated, they not only help the blood to clot, but also trigger thedamaged blood vessel to repair itself. They do this by regulated exocytosis of a proteincalled platelet-derived growth factor (PDGF ) (Fig. 16.8). The plasma membranes of the\\ncells of the blood vessel contain receptors for this protein. PDGF receptors (Fig. 9.15 onpage 198) comprise an extracellular domain, which binds PDGF, a single polypeptide chainthat crosses the plasma membrane, and a cytosolic domain with tyrosine kinase activity.The PDGF receptor is therefore able to phosphorylate tyrosine residues on other proteins.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2f15a5e4-dc38-4222-ac29-450e6365ca7b', embedding=None, metadata={'page_label': '360', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\ntyrosine\\nresiduePDGF receptor\\nGrb2SH2 domainmutual\\nphosphorylation\\nGDP\\nMAPKK MAPKK\\nMAPK MAPKMAPK\\nphosphorylates\\ntranscription\\nfactors\\nmRNA for\\nproteins required\\nfor cell divisionRasSOS\\nGTP\\nMAPKKKPDGF\\n====+\\n+\\n+\\n+kinase\\nsite\\nFigure 16.8. The PDGF receptor, like other growth factor receptors, activates the GTPase Ras and therefore the MAP kinase cascade.\\n354', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b4b7bd30-5c8d-4d38-a150-c6963e42b210', embedding=None, metadata={'page_label': '361', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\nBIOCHEMICAL SIGNALING 355\\nIn the absence of PDGF, the PDGF receptor does not come in contact with proteins to\\nphosphorylate. PDGF can bind to two receptor molecules, drawing them close to each otherand allowing them to phosphorylate each other on tyrosine residues.\\nA number of cytoplasmic proteins have a domain called SH2 that is just the right shape\\nto stick to phosphorylated tyrosine. At the bottom of a deep pocket in the protein surfaceis a positively charged arginine. Although proteins can be phosphorylated on amino acidsother than tyrosine, only tyrosine is long enough to insert down the pocket so that thenegative phosphate can stick to the positive arginine. Proteins with SH2 domains thereforestick to dimerized PDGF receptors. In contrast, SH2 domains do not stick to solitary PDGFreceptors whose tyrosines do not carry the negatively charged phosphate group.\\nOne protein that has an SH2 domain is called growth factor receptor binding protein\\nnumber 2 (Grb2 ). Grb2 has no catalytic function but recruits a second protein called SOS,\\nand SOS is a guanine nucleotide exchange factor for a GTPase called Ras, allowing it\\nto discard GDP and bind GTP. Active Ras turns on a chain of protein kinases, each ofwhich phosphorylates and hence activates the next, culminating in a kinase called mitogen-\\nassociated protein kinase (MAP kinase orMAPK ). The kinase that phosphorylates MAP\\nkinase is called MAP kinase kinase (MAPKK ) while the one that phosphorylates MAPKK\\nis called MAP kinase kinase kinase (MAPKKK ). MAPKKK is activated by Ras.\\nThe word mitogen means a chemical that tends to cause mitosis, re ﬂecting the fact that\\nMAP kinase is activated by transmitters such as PDGF that turn on cell division. WhenMAP kinase is phosphorylated, it moves to the nucleus and phosophorylates transcriptionfactors that in turn stimulate the transcription of the genes for cyclin D (page 413) and other\\nproteins that are required for DNA synthesis and cell division.\\nPlatelet-derived growth factor is one of many growth factors. All work in much the same\\nway: Their receptors are tyrosine kinases that are triggered to dimerize and phosphorylatetheir partners upon tyrosine when the growth factor binds. The general term for this type ofreceptor is receptor tyrosine kinase. Phosphorylated tyrosine then recruits proteins with\\nSH2 domains including Grb2, which in turn allows activation of Ras and the MAP kinasepathway, leading to DNA synthesis and cell division.\\n✄\\n✂\\x00\\n✁Medical\\nRelevance\\n16.1Blocking Growth Factor Receptors\\nActive growth factor receptors cause cell division, by activating the MAP kinase\\npathway, and keep cells alive, by activating PKB (page 417). Turning off growth factor\\nreceptors therefore tends both to stop cells dividing and to kill them. The drug trastuzumab iseffective in slowing down the progression of breast cancer because it prevents a growth factor,called epidermal growth factor (EGF), from binding to its receptor tyrosine kinase. This slowscell division and promotes cell death in the cancer cells.\\n✄\\n✂\\x00\\n✁Medical\\nRelevance\\n16.2R115777 Blocks Ras\\nLike other GTPases, Ras turns itself off by hydrolyzing its bound GTP. Mutantforms of Ras without GTPase activity are therefore constitutively active: they\\nare always in the on state and will therefore be activating the pathway that terminates in MAPkinase and cell division at all times, even in the absence of growth factor. Such mutant forms ofRas are found in about 20% of all human cancers. The drug R115777 prevents Ras from activatingMAPKKK and should therefore help stop the uncontrolled cell division that is characteristic ofcancer cells.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='945a8b12-2551-4206-98b3-c98ccdd63289', embedding=None, metadata={'page_label': '362', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\n356 INTRACELLULAR SIGNALING\\nphosphorylation\\nof PLC γActive PLC γ\\nhydrolyses PIP2= ==+\\n++\\n+P=P=P−\\nP=\\nP=P=\\nFigure 16.9. The PDGF receptor, like other growth factor receptors, phosphorylates and hence\\nactivates phospholipase C γ.\\nGrowth Factors Can Trigger a Calcium Signal\\nA second protein that contains an SH2 domain and that is therefore recruited to phospho-\\nrylated receptor tyrosine kinases is another isoform of phosphoinositide phospholipase C,PLCγ.Binding to the phosphorylated tyrosine holds PLC γat the growth factor receptor\\nlong enough for it itself to be phosphorylated (Fig. 16.9), and this activates its enzymaticaction. Active PLC γhydrolyzes PIP\\n2into diacylglycerol and inositol trisphosphate, and\\nin turn inositol trisphosphate triggers the release of calcium from the smooth endoplasmicreticulum.\\nProtein Kinase B and the Glucose Transporter: How Insulin Works\\nFigure 16.10 shows the insulin receptor. Like other receptor tyrosine kinases, the insulin\\nreceptor has an extracellular domain that can bind the transmitter, in this case the pro-tein insulin, a single polypeptide chain that crosses the plasma membrane, and a cytosolicdomain with tyrosine kinase activity. Unlike growth factor receptors, the insulin receptorexists as a dimer even in the absence of its ligand, insulin. When insulin binds, the shapeand orientation of the individual insulin receptors change a little, and this allows each re-ceptor to phosphorylate its partner upon tyrosine. An associated protein called the insulin\\nreceptor substrate number 1 (IRS-1 ) is also phosphorylated on tyrosine. Proteins with\\nSH2 domains are therefore recruited, either to the phosphorylated tyrosines on the insulinreceptors themselves or to the phosphorylated tyrosines on IRS-1. The most importantprotein to be recruited is phosphoinositide 3-kinase (PI 3-kinase ). Binding to the phos-\\nphorylated tyrosine holds PI 3-kinase close to the insulin receptor long enough for it to', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='fef91bcd-0eba-4e5a-aab0-f3118611f61b', embedding=None, metadata={'page_label': '363', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\nBIOCHEMICAL SIGNALING 357\\nphosphorylation of\\nPI-3 kinaseActive PI-3 kinase\\nphosphorylates PIP2= =\\n==PIP3P−\\nPI-3 kinaseIRS-1kinase siteinsulin\\nreceptorinsulin\\nP−\\nP=P=\\nP=P=\\nP=\\nFigure 16.10. The insulin receptor phosphorylates and hence activates PI 3-kinase.\\nbecome phosphorylated, and this activates its enzymatic action. PI 3-kinase operates on\\nthe same substrate, PIP 2,that PLC does, but instead of hydrolyzing it, PI 3-kinase adds\\nanother phosphate group to the inositol head group, generating the intensely charged lipidphosphatidylinositol trisphosphate (PIP\\n3).\\nJust as phosphotyrosine is bound by a wide variety of proteins that contain SH2 domains,\\nso highly phosphorylated inositols, such as those in PIP 3, are bound by a domain called the\\nPH domain found on many proteins. Most important among these is a protein kinase called\\nprotein kinase B (PKB ). PKB is itself activated by phosphorylation, but the kinase that does\\nthis is located at the plasma membrane and therefore only gets a chance to phosphorylatePKB when PKB is held at the plasma membrane through its binding to PIP\\n3(Fig. 16.11).\\nIn many cell types, particularly fat cells and muscle cells, the ﬁnal stage in traf ﬁcking\\n(page 226) of the glucose carrier (page 316) from the Golgi apparatus to the plasma mem-brane requires active PKB. At the same time, glucose carriers are being endocytosed andare only returned to the plasma membrane if PKB is active. When we eat a large meal,insulin concentrations in the blood increase. Activation of the insulin receptor thereforecauses an increase in the activation of PKB and hence a translocation of glucose carriers tothe plasma membrane. This allows muscle and fat cells to take up large amounts of glucosefrom the extracellular medium. Muscle cells convert the glucose to glycogen (page 298);fat cells convert the glucose to fat (page 300). The action of PKB on a protein called BAD\\nthat is shown in Figure 16.11 will be described in Chapter 19.\\nCrosstalk—Signaling Pathways or Signaling Webs?\\nCell biologists often talk of signaling pathways in which a transmitter activates a chain ofevents culminating in an effect. The vertical arrows in Figure 16.12 show four signalingpathways that we have discussed in this chapter. We have already seen one example of', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='62b83351-0043-44ad-99a0-5d0bd8a24d80', embedding=None, metadata={'page_label': '364', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\n358 INTRACELLULAR SIGNALING\\nserine and threonine\\nresidues \\npHdomain\\nProtein kinase B\\nActivePKB\\nMany other\\ncellular eventsserineresidueBAD+\\n+\\n+\\n++===\\n+14-3-3protein\\ncytochrome Cglucosecarrierkinase specificfor PKB\\nP=\\nP=P=\\nFigure 16.11. PIP3recruits protein kinase B (PKB) to the plasma membrane, where it is activated.\\nActive PKB has many effects including exocytosis of vesicles containing glucose carriers and inac-\\ntivation of BAD.\\nhow these pathways can interact: growth factor receptors can trigger a calcium signal by\\nactivating phospholipase C γ. In fact, the pathways can interact in many ways and at many\\nlevels. The green arrows in Figure 16.12 show some of the most important. PI 3-kinase canbind to phosphotyrosine on growth factor receptors via its SH2 domain and be activatedby phosphorylation, thus growth factors, as well as insulin, will activate protein kinase B(arrow X in Fig. 16.12). In Chapter 19 we will see that this process is literally vital for ourcells—if it stops, they die (page 417).\\nMany of the actions of cAMP occur through the phosphorylation of other proteins by\\nprotein kinase A. In particular, cAMP can activate transcription of many eukaryotic genes(arrow Y in Fig. 16.12), but it does so by causing the phosphorylation of transcriptionfactors, and hence activating them. This is in sharp contrast to the cAMP-CAP system inEscherichia coli (page 114).\\nCalcium ions can activate a number of protein kinases, notably protein kinase C\\nandcalcium-calmodulin–activated protein kinase. These kinases phosphorylate many of\\nthe same targets as protein kinase A and hence can activate the same downstream events(arrows Z in Fig. 16.12). Paradoxically, calcium-calmodulin also activates the phosphatasecalcineurin, which dephosphorylates and hence activates the transcription factor NFAT(page 222). The more we know about intracellular signaling systems, the more they cometo resemble a web of interactions rather than a set of discrete pathways.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='89081985-8c8c-4d9c-9528-64da03cacf70', embedding=None, metadata={'page_label': '365', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\nSUMMARY 359\\ninsulin\\ninsulin\\nreceptor\\nPI3 kinase\\nProtein kinase\\nB\\ntargets 1\\n(e.g. glucosecarrier)growth factor\\ngrowth factor\\nreceptor\\nRas\\nMAP\\nkinase\\ntargets 2(e.g. genetranscription)agonist\\n(eg. ADP , adrenaline)\\nreceptor\\nAdenylate\\ncyclase\\ncAMP\\nProtein kinase\\nA\\ntargets 3(e.g. ionchannels)YZX\\nZagonist\\n(eg. noradrenaline)\\nreceptor\\nPhospholipase\\nCβPhospholipase\\nCγ\\nCa++\\nactivatedproteinkinases andphosphatasesCa\\n++\\ntargets 4\\n(e.g. regulatedexocytosis)\\nFigure 16.12. Interactions of signaling pathways.\\nSUMMARY\\n1.An intracellular messenger is an intracellular solute whose concentration changes in\\nresponse to cell stimulation; it activates or modulates a variety of cellular processes.The most important intracellular messengers are calcium ions, cyclic adenosinemonophosphate (cAMP), and cyclic guanosine monophosphate (cGMP).\\n2.The increased cytosolic calcium may be derived from two sources: the extracellularﬂuid and the smooth endoplasmic reticulum.\\n3.A complex cascade of reactions causes release of calcium from the smooth endo-plasmic reticulum in response to stimulation of the cell. Binding of extracellularchemical to a cell surface receptor activates the trimeric G protein G\\nq, which in\\nturn activates the enzyme phospholipase C β, which cleaves the hydrophilic head\\ngroup inositol trisphosphate (IP 3) from the lipid phosphatidylinositol bisphosphate.\\nInositol trisphosphate then binds to and opens a calcium channel in the membraneof the endoplasmic reticulum.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4de98e2f-7c95-4ba9-9581-02eb1f1a27b5', embedding=None, metadata={'page_label': '366', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\n360 INTRACELLULAR SIGNALING\\n4.A different set of receptors activates G sand hence adenylate cyclase, which makes\\ncyclic AMP from ATP. Many of the actions of cAMP are mediated by cAMP-\\ndependent protein kinase (protein kinase A).\\n5.The majority of transmitters are released from cells by exocytosis induced by anincrease of cytosolic calcium concentration.\\n6.Many nerve cells have their axon terminals close to a second cell, and they releasetheir transmitter onto it. The complete unit of axon terminal, gap, and the part ofthe cell that receives the transmitter is called a synapse.\\n7.Receptor tyrosine kinases are caused to dimerize when their ligand binds. Thisallows each partner to phosphorylate the other upon tyrosine; this in turn recruitsproteins containing SH2 domains.\\n8.Grb2 is an SH2-domain-containing protein that serves to bring together SOSand Ras, allowing activation of Ras and therefore activation of the MAP kinasepathway. This culminates in the transcription of genes necessary for DNA synthesisand cell division.\\n9.Phospholipase C γand phosphoinositide 3-kinase, two enzymes that act on the\\nmembrane lipid phosphatidylinositol bisphosphate (PIP\\n2), both have SH2 domains\\nand are both recruited to, and phosphorylated by, receptor tyrosine kinases: thisactivates them.\\n10.Protein kinase B is recruited to the plasma membrane by PIP\\n3; this allows it to be\\nphosphorylated, which activates it.\\n11.In many cells including fat and skeletal muscle, glucose transporters only appearat the plasma membrane when PKB is kept active through the activation of theinsulin receptor.\\nFURTHER READING\\nDownward, J. 1998. Mechanisms and consequences of activation of protein kinase B/Akt. Curr. Opin.\\nCell. Biol. 10: 262 –267.\\nMarshall, C. J. 1996. Ras effectors. Curr. Opin. Cell Biol. 8: 197 –204.\\nToescu E. C., and Verkhratsky, A. 1998. Integrative Aspects of Calcium Signalling. New York: Plenum\\nPress.\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.Which of the following does not exist?\\nA. Protein kinase A\\nB. Protein kinase BC. Phospholipase C α', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='72cb1c38-5945-4efc-90d7-0b995e14f8cf', embedding=None, metadata={'page_label': '367', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\nANSWERS TO REVIEW QUESTIONS 361\\nD. Phospholipase C β\\nE. Phospholipase C γ\\n2.Which of the following is an enzyme, one of whose substrates is a nucleotide triphos-\\nphate?\\nA. The PDGF receptor\\nB. The insulin receptorC. RasD. Protein kinase AE. All of the above\\n3.Arrival of a scent chemical at a scent-sensitive nerve cell causes that cell to\\nA. activate adenylate cyclase.B. open cAMP-gated channels in the plasma membrane.C. open voltage-gated sodium channels in the plasma membrane.D. generate action potentials.E. all of the above.\\n4.Which of the following statements is false?\\nA. An increase of cytosolic cAMP concentration can activate a protein kinase.B. An increase of cytosolic calcium concentration can activate a protein kinase.C. Dimerization of growth factor receptors can activate a protein kinase.D. Appearance of PIP\\n3at the plasma membrane can activate a protein kinase.\\nE. Hydrolysis of GTP by Ras can activate a protein kinase.\\n5.Depolarization of axon terminals causes calcium ions to ﬂow into the cytosol through\\nA. voltage-gated calcium channels.B. inositol trisphosphate-gated calcium channels.C. ryanodine receptors.D. calcium ATPases.E. cAMP-gated calcium channels.\\n6.Inositol trisphosphate-gated calcium channels are located in\\nA. the plasma membrane.B. the smooth endoplasmic reticulum.C. the inner mitochondrial membrane.D. regulated exocytotic vesicles.E. all of the above.\\n7.Phosphatidylinositol bisphosphate is used in two different signaling pathways. These\\nconvert it, respectively, into\\nA. IP 3and PIP 3.\\nB. IP 3and cAMP.\\nC. PIP 2and PIP 3.\\nD. PIP 2and cAMP.\\nE. cAMP and cGMP.\\nANSWERS TO REVIEW QUESTIONS\\n1.C.While protein kinase A, protein kinase B, phospholipase C β, and phospholipase C γare enzymes\\nwith important roles in signaling, there is no enzyme with the name phospholipase C α.\\n2.E.Protein kinases transfer the γ-phosphate group of ATP to a protein. Thus all protein kinases\\n(including the PDGF receptor and the insulin receptor, both of which are protein kinases) are', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6a3a0701-e6fe-4b44-b630-8d655946cc70', embedding=None, metadata={'page_label': '368', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-16 WY001-Bolsover-v2.cls September 15, 2003 19:40\\n362 INTRACELLULAR SIGNALING\\nenzymes that operate on ATP, a nucleotide triphosphate. Ras, in contrast, hydrolyzes the nucleotide\\ntriphosphate GTP.\\n3.E.These statements constitute a summary of signaling in scent-sensitive nerve cells.\\n4.E.Hydrolysis of GTP by Ras turns Ras off, so that it is no longer able to activate MAPKK kinase.\\nThe other statements are true. In particular (A) cAMP activates protein kinase A, (B) calciumactivates a number of protein kinases, (C) dimerization activates growth factor receptors, and (D)PIP\\n3recruits PKB to the plasma membrane, where it is phosphorylated and hence activated.\\n5.A.Statements B, D, and E are completely false, in particular (B) inositol trisphosphate-gated\\ncalcium channels are opened by inositol trisphosphate, which is not generated by depolarization.(D) Calcium ATPases push calcium ions out of the cytosol —they do not allow it in. (E) cAMP-gated\\ncalcium channels are opened by cAMP, which is not generated by depolarization. Some scientiststhink that (C) may operate in a restricted set of axon terminals, that is, depolarization of the axonterminal allows calcium to ﬂow in through voltage-gated calcium channels, and this calcium then\\nopens ryanodine receptors (page 348). However, this is certainly not a general phenomenon.\\n6.B.Answers C through E are completely false. Some rare cells (the best studied is a sensory cell\\nin lobsters) do express inositol trisphosphate-gated calcium channels in their plasma membranes(answer A), but this is certainly not typical.\\n7.A.Phospholipase C hydrolyzes phosphatidylinositol bisphosphate (PIP\\n2), releasing the head group\\nas IP 3, while phosphatidylinositol 3-kinase phosphorylates the inositol head group of PIP 2, gener-\\nating PIP 3. Each of these products has an important role in cell signaling. IP 3acts to cause release\\nof calcium from the endoplasmic reticulum, while PIP 3causes activation of protein kinase B.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='12ed5728-da62-44a7-bc15-c08858f10398', embedding=None, metadata={'page_label': '369', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\n17\\nINTERCELLULAR COMMUNICATION\\nThe millions of cells that make up a multicellular organism can work together only because\\nthey continually exchange the chemical messages called transmitters. Here we describehow systems of cells use transmitters to cooperate for the good of the organism. Most ofthe many transmitters known are found in all animals and probably evolved with ancestralmulticellular organisms more than a billion years ago. We will introduce the general princi-ples of intercellular communication and then illustrate how transmitters operate in a singletissue, the gastrocnemius muscle.\\nCLASSIFYING TRANSMITTERS AND RECEPTORS\\nTransmitter mechanisms can be classiﬁed in two ways. The ﬁrst depends on their lifetimein the extracellular ﬂuid. A transmitter that is rapidly broken down or taken up into cellsacts only near its release site. One that is broken down slowly can diffuse a long wayand may act on cells a long way away. The shortest lived transmitters of all are thosereleased at synapses (page 344), where the distance from release site to receptor is only100 nm.\\nAt the other extreme are transmitters that last many minutes or, sometimes, even longer.\\nHormones are long-lived transmitters that are released into the blood and travel around the\\nbody before being broken down. Most are released by specialized groups of secretory cellsthat form a structure called an endocrine gland. Paracrine transmitters also last many\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n363', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ff828317-ff29-4399-b2df-7106d6c88e18', embedding=None, metadata={'page_label': '370', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\n364 INTERCELLULAR COMMUNICATION\\nNa+\\nacetylcholine\\nbinding opensthe channelextracellular medium cytosol\\nNa+Na+acetylcholinebinding site\\nacetylcholine\\nbinds\\nFigure 17.1. The nicotinic acetylcholine receptor is an ionotropic cell surface receptor that opens\\nwhen acetylcholine in the extracellular medium binds.\\nminutes before being broken down, but they are released into speci ﬁc tissues rather than\\ninto the blood, and only diffuse within the tissue before they are destroyed.\\nThe second way of classifying transmitter mechanisms depends on the location and\\naction of the receptor on the target cell. There are three types of receptors: ionotropic cell\\nsurface receptors, metabotropic cell surface receptors, andintracellular receptors.\\nIonotropic Cell Surface Receptors\\nIonotropic cell surface receptors are channels that open when a speci ﬁc chemical binds to the\\nextracellular face of the channel protein. The nicotinic acetylcholine receptor (Fig. 17.1),\\nso named because the drug nicotine binds to it, is one example. In the absence of a transmitter\\ncalled acetylcholine, the channel is closed. When acetylcholine binds, the channel opens andallows sodium and potassium ions to pass through. The electrochemical gradient pushingsodium ions into the cell is much greater than that pushing potassium ions out of the cell, sowhen the channel opens, sodium ions pour in carrying positive charge and depolarizing the', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a9d20e5b-0ce4-463c-8585-99152cf3dde6', embedding=None, metadata={'page_label': '371', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\nINTERCELLULAR COMMUNICATION IN ACTION: THE GASTROCNEMIUS MUSCLE 365\\nplasma membrane. This mechanism is similar to that of two channels we met in Chapter 16,\\nthe inositol trisphosphate-gated calcium channel and the cAMP-gated channel. There is,however, a major difference: Those two channels were opened by cytosolic solutes, whileionotropic cell surface receptors are opened by extracellular solutes.\\nMetabotropic Cell Surface Receptors\\nMetabotropic cell surface receptors are linked to enzymes. We have already met a numberof metabotropic cell surface receptors in Chapter 16. When the ADP receptor (page 344)binds extracellular ADP, it activates G\\nqand hence phospholipase C β. The receptor for smell\\nchemicals (page 350) and the β-adrenergic receptor (page 353) are linked to G sand hence\\nadenylate cyclase so ligand binding increases cytosolic cAMP. Receptor tyrosine kinasesare themselves protein kinases activated when their ligand binds.\\nTheα-adrenergic receptor (Fig. 17.2) is another receptor that causes cytosolic calcium\\nconcentration to increase. The α- andβ-adrenergic receptors are distinct proteins that bind\\nthe same transmitters, adrenaline and noradrenaline. To simplify the issue somewhat, wecan say that noradrenaline acts mainly on αreceptors and adrenaline acts mainly on β\\nreceptors. Because the αandβreceptors are distinct proteins, it is possible to design drugs\\n(αandβblockers) that interfere with one or the other.\\nIntracellular Receptors\\nIntracellular receptors lie within the cell (in the cytosol or in the nucleus) and bind transmit-ters that diffuse through the plasma membrane. They always exert their effects by activatingenzymes. The receptors for nitric oxide and steroid hormones are two examples.\\nNitric oxide, or NO, is a transmitter in many tissues. It is not stored ready to be released\\nbut is made from arginine at the time it is needed. NO diffuses easily through the plasmamembrane and binds to various cytosolic proteins that are NO receptors. One particularlyimportant NO receptor is the enzyme guanylate cyclase, which in the presence of NO con-\\nverts the nucleotide GTP to the intracellular messenger cyclic guanosine monophosphate,or cGMP.\\nSteroid hormones have intracellular receptors such as the glucocorticoid receptor (page\\n121). In the absence of hormone this receptor remains in the cytosol and is inactive becauseit is bound to an inhibitor protein. However, when the glucocorticoid hormone binds to itsreceptor, the inhibitor protein is displaced. The complex of the glucocorticoid receptor withits attached hormone now moves into the nucleus. Here two molecules of the complex bindto a 15-bp sequence known as the hormone response element (HRE), which lies upstreamof the TATA box (page 122). The HRE is a transcriptional enhancer sequence.The bindingof the glucocorticoid hormone receptor to the HRE stimulates transcription.\\nINTERCELLULAR COMMUNICATION IN ACTION:\\nTHE GASTROCNEMIUS MUSCLE\\nThe gastrocnemius muscle illustrates how transmitters and receptors operate in a single\\ntissue. This is the calf muscle at the back of the lower leg. When it contracts, it pulls on theAchilles tendon so that the toes push down on the ground. Most of the bulk of the muscleis made up of one type of cell, skeletal muscle cells.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e1309388-f5a2-4f09-b052-c52d14813c89', embedding=None, metadata={'page_label': '372', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\nβ\\nγ OPO\\nO\\nOOO\\nOPO\\nO OOP O αGTP\\nGDPGTPPIP2\\ntrimeric\\nG protein\\nGqphospholipase\\nCβα\\nadrenergicreceptor\\nGDPnoradrenaline\\nDAGOH\\nα\\nOP O OO O\\nO\\nOPO\\nO OOP O\\ninositoltrisphosphateβγ\\nFigure 17.2. Noradrenaline activates G qand hence phospholipase Cβ in many cells including smooth muscle.\\n366', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='95a07279-491d-464f-8baa-09a8aec0fea5', embedding=None, metadata={'page_label': '373', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\nINTERCELLULAR COMMUNICATION IN ACTION: THE GASTROCNEMIUS MUSCLE 367\\nsynaptic vesicle \\ncontaining \\nacetylcholine\\nvoltage-gated \\ncalcium channels\\npotassium \\nchannelsvoltage-gated \\nsodium channels\\nacetylcholine \\nreceptors of nicotinic typeaxon presynaptic terminal\\naxon of motoneurone\\nskeletal muscle cellplasmalemma of muscle cell\\n−80transmembrane voltage of \\nskeletal muscle cell (mV)5 ms time\\n−500\\nthresholdaction potential produced by voltage-gated sodium channels\\ndepolarization due to sodium influx through acetylcholine receptor channelgastrocnemius \\nmusclemotoneurone\\naxon of motoneurone\\nextracellular medium cytosolNa+\\nNa+acetylcholine\\nbinds\\nFigure 17.3. Motoneurones release the transmitter acetylcholine that binds to nicotinic receptors\\non the muscle cells. The plasma membrane of the muscle cell is depolarized to threshold and ﬁres\\nan action potential.\\nTelling the Muscle to Contract: The Action of Motoneurones\\nAll skeletal muscle cells are relaxed until they receive a command to contract from nerve\\ncells called motoneurones. Motoneurone cell bodies are in the spinal cord while the myeli-\\nnated axons run to the muscle. The axon terminal of the motoneurone that controls thegastrocnemius muscle contains vesicles of the transmitter acetylcholine (Fig. 17.3). To', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='aaa7aedd-d7e0-4bf3-ac36-de5727af9197', embedding=None, metadata={'page_label': '374', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\n368 INTERCELLULAR COMMUNICATION\\npress the foot down, action potentials travel from the spinal cord down the motoneurone\\naxon to its terminal, opening voltage-gated calcium channels in the plasma membrane.Calcium ions pour in, raising their cytosolic concentration from 100 nmol liter\\n−1to\\n1µmol liter−1. The calcium torrent causes the synaptic vesicles containing acetylcholine\\nto fuse with the plasma membrane, releasing the transmitter into the extracellular ﬂuid.\\nAcetylcholine survives only 0.2 ms in the extracellular ﬂuid since it is quickly hy-\\ndrolyzed into choline and acetate by the enzyme acetylcholinesterase. However, its target —\\nthe skeletal muscle cell —is only 100 nm away. It hits its mark before it is broken down\\nand binds to the nicotinic acetylcholine receptors present in the plasma membrane of theskeletal muscle cell. These open, allowing both potassium and sodium ions to pass. There isno large electrochemical gradient for potassium, but sodium is at a high concentration in theextracellular ﬂuid and rushes in, depolarizing the plasma membrane of the skeletal muscle\\ncell. The plasma membrane of the skeletal muscle cell also contains voltage-gated sodiumchannels. When the ﬂow of sodium ions through the nicotinic acetylcholine receptors depo-\\nlarizes the plasma membrane to threshold, the skeletal muscle cell ﬁres an action potential,\\nwhich in turn causes release of calcium from the endoplasmic reticulum (page 348). Theresulting increase of calcium concentration in the cytosol of the muscle cell causes it tocontract (page 393).\\nControlling the Blood Supply: Paracrine Transmitters\\nFigure 17.4 is a cut-away drawing of a blood vessel in the muscle. Lining the tube is athin layer of endothelial cells. Wrapped around these are muscle cells of a different type,smooth muscle cells. Both endothelial cells and smooth muscle cells are much smallerthan skeletal muscle cells. A small blood vessel may be only as large as a single skeletalmuscle cell. Smooth muscle cells have no input from motoneurones. Instead, like manyother internal organs, they are supplied by a distinct class of nerve cells that are calledautonomic. Autonomic nerve cells usually use one of two transmitters: acetylcholine or\\nnoradrenaline. Two other transmitters, adrenaline and nitric oxide, also help to adjust theblood ﬂow on a timescale of seconds to minutes. The nerve cells that release noradrenaline\\nonto the smooth muscle of blood vessels are called vasoconstrictors because they cause\\nblood vessels to contract. The smooth muscle cells have α-adrenergic receptors in their\\nplasma membranes. Binding of noradrenaline to α-adrenergic receptors activates PLC β,\\nwhich generates IP\\n3, which in turn releases calcium from the endoplasmic reticulum into\\nthe cytosol. The increase of cytosolic calcium concentration causes the smooth musclecells to contract, constricting the blood vessel and reducing the ﬂow. Vasoconstrictor nerves\\nare used to restrict the ﬂow of blood to muscles and other organs that are not in heavy\\nuse.\\nThe hormone adrenaline is chemically related to noradrenaline but is more stable,\\nlasting a minute or so in the extracellular ﬂuid before being broken down. It is released\\nfrom an endocrine gland (the adrenal gland) during times of stress and spreads around thebody in the blood. Adrenaline that diffuses to the skeletal muscle cells stimulates them tobegin breaking down glycogen to make glucose-6-phosphate (page 305). The smooth musclecells of blood vessels within skeletal muscles also have β-adrenergic receptors connected\\nto adenylate cyclase. However, they do not contain glycogen, and cAMP has another effectin these cells. When cAMP rises, it activates cAMP-dependent protein kinase. This inturn phosphorylates proteins that relax the smooth muscle cell. The action of adrenaline is', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='00306062-2724-47cd-aa79-92605a8c2ef9', embedding=None, metadata={'page_label': '375', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\nINTERCELLULAR COMMUNICATION IN ACTION: THE GASTROCNEMIUS MUSCLE 369\\nblood \\nvessel\\nFGF released from active skeletal muscle cells stimulates cell division in endothelial cells\\nbloodendothelial \\ncellendothelial cells\\nsmooth muscle cells\\nsmooth muscle cell\\nacetylcholine released from vasodilator nerve cells causes endothelial cells to make NONO diffuses easily through many plasma membranes and relaxes smooth muscle cellsskeletal muscle \\ncell\\nskeletal muscle \\ncell\\nadrenaline \\ndiffusing from the blood relaxes smooth muscle and triggers glycogen breakdown in the skeletal muscle\\naxon of autonomic nerve cell\\nnoradrenaline released from vasoconstrictor nerve cells causes smooth muscle to contractBLOODaxon of autonomic nerve \\ncell\\nFigure 17.4. Transmitters regulate the blood supply to muscles.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1acd4ece-d08a-463e-865a-a33b3fb114df', embedding=None, metadata={'page_label': '376', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\n370 INTERCELLULAR COMMUNICATION\\ntherefore to increase the blood supply to all the muscles of the body in preparation for ﬂight\\norﬁght. If we are very frightened and too much adrenaline is released, so much blood is\\ndiverted from the brain to the muscles that we faint.\\nJust as autonomic vasoconstrictor nerve cells are used to shut down blood ﬂow to par-\\nticular regions, autonomic vasodilator nerves are used to dilate blood vessels in muscles\\nthat are about to be used. They release acetylcholine, but neither smooth muscle cells norendothelial cells have nicotinic acetylcholine receptors. Endothelial cells have, instead, adifferent receptor called the muscarinic acetylcholine receptor. This is named for a chem-\\nical that binds it: muscarine, from the poisonous ﬂy agaric mushroom Amanita muscaria\\n(page 376). Like the α-adrenergic receptor, the muscarinic acetylcholine receptor is linked\\nto G\\nqand therefore to phospholipase C β, so that binding of acetylcholine causes an in-\\ncrease of cytosolic calcium concentration. Endothelial cells do not contract in response to acalcium increase. However, they have an enzyme that makes nitric oxide, and this enzymeis activated by calcium. When endothelial cells are stimulated with acetylcholine, theymake nitric oxide. This intercellular transmitter easily passes through the plasma mem-branes of both endothelial and smooth muscle cells and reaches its receptor within thesmooth muscle cells. Here it activates guanylate cyclase, causing an increase of cGMPconcentration.\\nJust as cAMP exerts many of its effects through cAMP-dependent protein kinase, called\\nprotein kinase A for short, so cGMP exerts many of its effects through cGMP-dependentprotein kinase, also called protein kinase G. One of the targets of protein kinase G iscalcium ATPase (page 318). When this is phosphorylated by protein kinase G it worksharder, reducing the concentration of calcium ions in the cytosol. This has the effect ofrelaxing the smooth muscle cells. Nitric oxide lasts for only about 4 s before being brokendown. It is therefore a paracrine transmitter, able to diffuse through and relax all the smoothmuscle cells surrounding the endothelial cells, but without lasting long enough to pass intomore remote tissues.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 17.1 Nitroglycerine Relieves Angina\\nThe discovery in 1987 that nitric oxide was a transmitter explained why nitroglycerine (more familiar\\nas an explosive than as a medicine) relieved angina pectoris. Angina is a pain felt in an overworkedheart. Nitroglycerine spreads throughout the body via the bloodstream and slowly breaks down,releasing nitric oxide that then dilates blood vessels. The heart no longer has to work so hard todrive the blood around the body.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 17.2 Viagra\\nJust as cAMP phosphodiesterase hydrolyzes cAMP to AMP and hence terminates its action as anintracellular messenger, so cGMP is inactivated by cGMP phosphodiesterase. There are a numberof isoforms of this enzyme in different human tissues. The drug sildena ﬁl, sold as Viagra, inhibits\\nthe form of the enzyme found in the penis. If cGMP is not being made, this has little effect onblood ﬂow to the region. However, when cGMP is made in response to a local production of NO,\\nits concentration in blood vessel smooth muscle increases much more than would otherwise occurbecause its hydrolysis to GMP is blocked. This in turn causes a greater activation of protein kinaseG, a greater activation of the calcium ATPase, a lower cytosolic calcium concentration, a greaterrelaxation of blood vessel smooth muscle, and therefore greater blood ﬂow.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e57e50bb-e46e-4968-8496-b942233f3c0a', embedding=None, metadata={'page_label': '377', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\nINTERCELLULAR COMMUNICATION IN ACTION: THE GASTROCNEMIUS MUSCLE 371\\nNew Blood Vessels in Growing Muscle\\nAll the phenomena we have discussed so far occur on a timescale of seconds. However,\\nwhen a muscle is repeatedly exercised over many days, it becomes stronger: the individ-ual skeletal muscle cells enlarge. This is because high cytosolic calcium acts via NFAT(page 222) to stimulate the transcription of genes coding for structural proteins. Further-more, new blood vessels sprout and grow into the enlarging muscle. A growth factor calledFGF is released by stimulated muscle. The receptor tyrosine kinase for FGF, like that forPDGF (page 353) is found on endothelial and smooth muscle cells and like the PDGFreceptor signals via Ras and MAP kinase (page 355) to trigger cell division and hence thegrowth of new blood vessels. (FGF stands for ﬁbroblast growth factor, but FGF is effective\\non a vast range of cell types including both endothelial and smooth muscle cells).\\nIn our study of the gastrocnemius muscle and its blood supply, we have seen examples of\\nall types of transmitter mechanism. Acetylcholine acts as a synaptic transmitter at the axonterminal of the motoneurone. Adrenaline is a hormone. The other transmitters are paracrine.The nicotinic acetylcholine receptor is an ionotropic cell surface receptor. The nitric oxidereceptor is intracellular. The other receptors are metabotropic cell surface receptors. Thereis a wide variety of timescales of action. The acetylcholine released from the axon terminalof the motoneurone causes a contraction of the skeletal muscle cell within 5 ms, by whichtime the acetylcholine has already been destroyed. Adrenaline lasts 1 min and dilates bloodvessels for all this time. FGF lasts 10 min, but its effects are much longer lasting. FGFtriggers the synthesis of proteins, which then act to cause cell proliferation that lasts for\\ntimeone action \\npotential no signalpresynaptic cell = pain receptor nerve cell postsynaptic cell = pain relay nerve cell\\nglutamate\\ntime\\ndepolarization due to \\nsodium ions entering pain relay cell through glutamate receptor channels−70−50\\n−70−5000\\nFigure 17.5. At most synapses, one presynaptic action potential is insuf ﬁcient to depolarize the\\npostsynaptic nerve cell to threshold.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='db8cac8d-0355-4888-b7fe-c2701caa5bc9', embedding=None, metadata={'page_label': '378', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\n372 INTERCELLULAR COMMUNICATION\\ndays. A similar pattern of intercellular communication, using some of the same transmitters\\nand receptors but also many others, is found in every tissue of the body.\\nSYNAPSES BETWEEN NEURONS\\nThe synapse between a motoneurone and a skeletal muscle cell is unusual in that a sin-gle action potential in the presynaptic cell releases enough transmitter to depolarize thepostsynaptic cell to threshold and hence produce an action potential in the postsynap-tic cell. The synapse between a pain receptor and a pain relay nerve cell (page 341)is more typical (Fig. 17.5). An action potential in the axon terminal of the pain recep-tor raises cytosolic calcium and releases the transmitter glutamate. This diffuses to thepain relay cell where it binds to glutamate receptors, which are ionotropic and allow bothpotassium and sodium ions to pass. The postsynaptic nerve cell depolarizes as sodiumions move in through the glutamate receptor channels. However, the depolarization is notenough to take the transmembrane voltage to threshold. As soon as glutamate is removedfrom the extracellular medium, the transmembrane voltage returns to the resting level.In the axon of the relay cell, some distance from the synapse, the transmembrane volt-age does not change, and no message passes on to the brain. The subject does not feelpain.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 17.3 A Toxic Glutamate Analogue\\nThe grasspea is a protein-rich crop that has been cultivated since ancient times and is still an important\\nsource of calories and protein in India, Africa, and China. The peas contain the amino acid β-N-\\noxalyl- L-α-β-diaminopropionic acid. If untreated peas are eaten, the toxin binds to and opens the\\nglutamate receptor on nerve cells. The resulting long-lasting depolarization damages and ﬁnally\\nkills the nerve cells. Boiling the peas during cooking destroys the toxin; but in times of famine,when fuel is scarce, many people are poisoned, and the resulting brain damage is irreversible.\\nOOCCH2\\nCH\\nglutamateCH2COO\\nNH3OOCCH2\\nCH\\nβ-N-oxalyl-L -α-β-\\ndiaminopropionic acidNHC\\nNH3OCOO\\nA subject does feel pain when they move their ﬁnger into the hot air from a hair dryer\\n(Fig. 17.6). This is because the hot air heats a large area and causes many pain receptors\\ntoﬁre action potentials. Many of the pain receptors synapse onto one relay cell, which', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='34d08bca-b46d-482d-9c1f-1fb4471d88fa', embedding=None, metadata={'page_label': '379', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\nSYNAPSES BETWEEN NEURONS 373\\naction potentialpostsynaptic cell = \\nsodium glutamate\\ntime\\ndepolarization due to \\nsodium ions enteringpain relay cell through glutamate receptor channels−50\\n−700action potentialaction potential in each of many pain receptor nerve cells\\npain relay \\nnerve celltransmembrane voltage of \\npostsynaptic cell (mV)\\nFigure 17.6. Spatial summation at a synapse.\\ntherefore receives many doses of glutamate. Because of this spatial summation, enough\\nglutamate receptor channels open to depolarize the relay cell to threshold, and an action\\npotential travels along the axon of the relay cell to the brain. The subject feels pain.\\nAn intense stimulus to a small area is also painful. When the subject is jabbed with\\na needle, only one pain receptor is activated, but that receptor is intensely stimulated andﬁres a rapid barrage of action potentials, each of which causes the release of glutamate and\\nan extra depolarization of the pain relay cell (Fig. 17.7). Soon the transmembrane voltageof the relay cell reaches threshold, and an action potential travels along its axon toward thebrain and the subject feels pain. Such temporal summation only occurs if the presynaptic\\naction potentials are frequent enough to ensure that the depolarizations produced in thepostsynaptic cell add.\\nAnother amino acid, γ-amino butyric acid (GABA) (page 37), is also a synaptic\\ntransmitter. Its receptor is an ionotropic cell surface receptor that forms a channel selective', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1cf77329-1971-46ef-b5d6-42f90d6c0b63', embedding=None, metadata={'page_label': '380', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\n374 INTERCELLULAR COMMUNICATION\\naction potentialpostsynaptic cell = \\nsodium glutamate\\nmany action potentialspain relay \\nnerve cellpresynaptic cell = pain receptor \\nnerve celltransmembrane voltage \\nof presynaptic cell time\\ntransmembrane voltage \\nof presynaptic cell (mV)time\\n−50\\n−700\\ndepolarizations in postsynaptic \\ncell add in timeaction \\npotential\\nFigure 17.7. Temporal summation at a synapse.\\nfor chloride ions. Figure 17.8 illustrates a single nerve cell bearing both glutamate and\\nγ-amino butyric acid receptors.\\nAt◦A an action potential in the GABA-secreting axon releases GABA onto the surface\\nof the postsynaptic cell, causing the GABA receptor channels to open. Although chlorideions could now move into or out of the cell, they do not, because their tendency to travelinto the cell down their concentration gradient is balanced by the repulsive effect of thenegative voltage of the cytosol: chloride ions are at equilibrium (page 314). The openingof GABA channels therefore causes no ion movements and therefore does not alter thetransmembrane voltage.\\nAt\\n◦B , action potentials occur simultaneously in six glutamate-secreting axons. In\\nthis example, this activity provides enough glutamate to depolarize the postsynaptic cell tothreshold, and the postsynaptic nerve cell ﬁres an action potential.\\nAt◦C , action potentials occur simultaneously in six glutamate-secreting axons and\\nalso in the GABA-secreting axon. The same number of glutamate receptor channels open asbefore, and the same number of sodium ions ﬂow into the postsynaptic nerve cell, tending\\nto depolarize it. However, as soon as the transmembrane voltage of the postsynaptic celldeviates from the resting value, chloride ions start to enter through the GABA receptorchannels because the cytosol is no longer negative enough to prevent them from enteringthe cell down their concentration gradient. The inward movement of negatively chargedchloride ions neutralizes some of the positive charge carried in by sodium ions moving inthrough the glutamate receptor channels. The postsynaptic nerve cell therefore does not', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='57b57409-84af-4e56-b444-dac6e7c14a5f', embedding=None, metadata={'page_label': '381', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\nSYNAPSES BETWEEN NEURONS 375\\npostsynaptic nerve cell six axons with terminals that \\nrelease glutamate\\none axon with a terminal that releases GABA\\ntime0\\nA B Ctransmembrane voltage of \\npostsynaptic cell (mV)−50\\n−70\\nFigure 17.8. Inhibition by a GABAergic synapse.\\ndepolarize as much and does not reach threshold. No action potential is generated in the\\npostsynaptic nerve cell.\\nAntianxiety drugs such as Valium act on the GABA receptor and increase the chance of\\nits channel opening. Nerve cells exposed to the drug are less likely to depolarize to threshold.Valium therefore reduces action potential activity in the brain, calming the patient.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a3383acd-d00d-4928-985f-12066c6fff8e', embedding=None, metadata={'page_label': '382', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\n376 INTERCELLULAR COMMUNICATION\\n☛\\n✡✟\\n✠Medical\\nRelevance\\n17.1Cigarettes, Mushrooms, and Insecticides\\nNicotine is one of the most addictive substances known. The reason for this is\\nstill unclear. The muscle weakness that new smokers experience is due to nicotine\\nbinding to and blocking receptors on the muscle cells, but we are still not sure how nicotine actsin the brain to cause a pleasurable experience.\\nTheﬂy agaric mushroom Amanita muscaria is seldom taken recreationally. Its popular use\\nis more prosaic, in killing ﬂies, hence its English and Latin names. However, Robert Graves\\nhas argued that it was used by the classical Greeks and by the Vikings to produce a state ofberserk strength with hallucinations. Any such use would be highly dangerous: muscarine acts bystimulating the muscarinic acetylcholine receptor, and the difference between a dose of muscarinelarge enough to cause pharmacological effects and a lethal one is very small.\\nAmanita muscaria has an even more dangerous relative, A. phalloides, the death cap mush-\\nroom. The toxin it produces, phalloidin, does not affect acetylcholine signaling. Instead, it inter-feres with the actin cytoskeleton (page 390).\\nFarmers and others who suffer insecticide toxicity show similar symptoms to those of\\nmuscarine poisoning. This is because these insecticides work by blocking the enzyme acetyl-cholinesterase, which breaks down acetylcholine. With the enzyme blocked, acetylcholine hangsaround in the extracellular medium, stimulating the receptor for longer than it should. Nerve gasesdeveloped to kill people (like Sarin, released in the Tokyo subway by members of the Aum cultin 1995) work in the same way. Paradoxically, sufferers are treated with another toxin, atropine,from the deadly nightshade plant. Atropine turns off the muscarinic acetylcholine receptor.\\nSUMMARY\\n1.Transmitter molecules can be classi ﬁed in two ways. One depends on their lifetime\\nand the extracellular medium they are found in, the other on the location and action\\nof the receptor on the target cell.\\n2.Synaptic transmitters are extremely short lived. Paracrine transmitters and hormoneshave a longer lifetime and are found in speci ﬁc tissues and in the blood, respectively.\\n3.Receptors can be divided into ionotropic cell surface receptors, metabotropic cellsurface receptors, and intracellular receptors. Ionotropic cell surface receptors areion channels that open in response to ligand binding. The effect upon the target cell iselectrical. Metabotropic cell surface receptors are linked to enzymes. The effect uponthe target cell is mediated through a biochemical process. Intracellular receptors liewithin the target cell and bind transmitters that are able to cross the cell membraneby simple diffusion.\\n4.The gastrocnemius muscle provides examples of all types of intercellular signalingoperating in concert to ﬁt the operation of the tissue to the requirements of the\\norganism.\\n5.Synapses between nerve cells do not generally mediate a 1-to-1 transmission of theaction potential. The presynaptic signal must show summation in time or space toelicit a postsynaptic action potential.\\n6.Stimulation of ionotropic cell surface receptors that pass chloride ions makes it moredifﬁcult for a nerve cell to be depolarized to threshold.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b459d1d9-edcb-40d8-bc53-309fbf49930a', embedding=None, metadata={'page_label': '383', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\nREVIEW QUESTIONS 377\\nFURTHER READING\\nSeeley, R. R., Stephens, T. D., and Tate, P. 1995. Anatomy and Physiology. St. Louis: Mosby.\\nSiegel, G. J., Agranoff, B. W., Albers, R. W., Fisher, S. K., and Uhler, M. D. 1999. Basic Neurochem-\\nistry. Philadelphia: Lippincott-Raven.\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.Ionotropic cell surface receptors\\nA. are ion channels permeable to both sodium and potassium.\\nB. allow the transmembrane voltage of the cell possessing them to change in response to\\ntransmitter.\\nC. are coupled to, or are themselves, enzymes.D. are only found on nerve cells.E. open when the concentration of acetylcholine in the extracellular medium increases.\\n2.Metabotropic cell surface receptors\\nA. are ion channels.B. exert their effects by activating enzymes.C. are tyrosine kinases.D. cause the concentration of cAMP in the cytosol to increase.E. make it more dif ﬁcult for a postsynaptic cell to be depolarized to threshold.\\n3.Application of adrenaline to a tissue tends to increase blood ﬂow through it, while\\nnoradrenaline has the opposite effect. This is because\\nA. noradrenaline, but not adrenaline, is able to diffuse through plasma membranes.B. the receptor at which noradrenaline is most effective is a cAMP-gated channel while the receptor\\nat which adrenaline is most effective is a calcium channel.\\nC. adrenaline is a hormone, while noradrenaline is released close to the target tissue.D. the electrochemical gradient for noradrenaline is inward, so it tends to make the cytosol more\\npositive, while the electrochemical gradient for adrenaline is outward.\\nE. the receptor at which noradrenaline is most effective is coupled to phospholipase C while the\\nreceptor at which adrenaline is most effective is coupled to adenylate cyclase.\\n4.Nitric oxide\\nA. is generated in endothelial cells when the cytosolic calcium concentration increases.B. easily diffuses across plasma membranes.C. acts at intracellular receptors.D. causes an increase of cGMP in the target cell.E. All of the above.\\n5.When seven action potentials arrive in close succession at a particular axon terminal in\\nthe brain, the postsynaptic cell is depolarized to just negative of threshold. If, one minutelater, the same train of seven action potentials arrives again, the postsynaptic cell is morelikely to ﬁre an action potential if\\nA. simultaneously, another nerve cell ﬁres an action potential and releases glutamate onto the same\\npostsynaptic cell.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b55f4ba3-9389-4627-b506-9bbfc01c9557', embedding=None, metadata={'page_label': '384', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\n378 INTERCELLULAR COMMUNICATION\\nB. simultaneously, another nerve cell ﬁres an action potential and releases GABA onto the same\\npostsynaptic cell.\\nC. simultaneously, another nerve cell ﬁres an action potential and releases noradrenaline onto the\\nsame postsynaptic cell.\\nD. the same seven action potentials arrive but spread over a longer period of time, that is, their\\nfrequency is less.\\nE. PDGF is present in the extracellular medium.\\n6.Which of the following statements about the gastrocnemius muscle and its blood supply\\nis untrue?\\nA. Acetylcholine released from motoneurons causes skeletal muscle cells to contract.\\nB. Acetylcholine released from sympathetic neurons causes smooth muscle cells to contract.C. NO released from endothelial cells causes smooth muscle cells to relax.D. PDGF released from platelets causes endothelial cells to enter the cell cycle.E. FGF released from skeletal muscle cells causes endothelial cells to enter the cell cycle.\\n7.The cells in the adrenal gland that secrete adrenaline have, in their plasma membranes,\\nnicotinic acetylcholine receptors, muscarinic acetylcholine receptors, and voltage-gatedcalcium channels. Application of acetylcholine will therefore cause\\nA. an increase of cytosolic calcium concentration, the vast majority of which is due to in ﬂux of\\ncalcium from the extracellular medium.\\nB. an increase of cytosolic calcium concentration, the vast majority of which is due to release of\\ncalcium from the smooth endoplasmic reticulum.\\nC. an increase of cytosolic calcium concentration, due to a combination of in ﬂux of calcium from\\nthe extracellular medium and release of calcium from the smooth endoplasmic reticulum.\\nD. a simultaneous increase of cytosolic calcium concentration and a shift in the transmembrane\\nvoltage to a more negative value.\\nE. a reduction in the cytosolic calcium concentration.\\nANSWERS TO REVIEW QUESTIONS\\n1.B.Ionotropic cell surface receptors pass current when they open and therefore change the trans-\\nmembrane voltage. Considering the other answers: (A) Some ionotropic cell surface receptors are\\nion channels permeable to both sodium and potassium, but others (such as the GABA receptor)are not. (C) Metabotropic, not ionotropic, receptors are coupled to, or are themselves, enzymes.(D) Many other cells have ionotropic cell surface receptors. Skeletal muscle cells are one example.(E) One class of ionotropic cell surface receptors (nicotinic acetylcholine receptors) respond toacetylcholine, but others respond to other transmitters.\\n2.B.This is the de ﬁnition of a metabotropic cell surface receptor. Of the other answers: (A)\\nMetabotropic cell surface receptors are not ion channels. (C) Some metabotropic cell surfacereceptors are tyrosine kinases, but many are not, incuding those that signal to G\\nqor G s. (D) The\\nsubset of metabotropic cell surface receptors that signal to G swill increase the concentration of\\ncAMP, but many others will not. (E) Metabotropic receptors affect many aspects of cell function,so some will indeed make it more dif ﬁcult for a postsynaptic cell to be depolarized to threshold,\\nbut this is not a general rule.\\n3.E.Both receptors are on smooth muscle cells. The receptor at which noradrenaline is most effective\\nis coupled to phospholipase C and therefore causes an increase of cytosolic calcium concentration,leading to contraction and hence a closing of the blood vessel. The receptor at which adrenalineis most effective is coupled to adenylate cyclase and therefore causes an increase of cAMP con-centration, leading to phosphorylation and therefore relaxation of the contractile machinery and', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='846e7fc7-eb13-42f3-be91-5580c4185479', embedding=None, metadata={'page_label': '385', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-17 WY001-Bolsover-v2.cls September 1, 2003 10:8\\nANSWERS TO REVIEW QUESTIONS 379\\nhence an opening of the blood vessel. Of the other answers A, B, and D are nonsense. Statement C\\nis true but is not the underlying reason why adrenaline relaxes, and noradrenaline contracts, thesmooth muscle.\\n4.E.These statements constitute a summary of the role of nitric oxide in blood vessels.\\n5.A.If more glutamate is released onto the postsynaptic cell, more glutamate receptors will open,\\nmore sodium will ﬂow in, and so the cell will depolarize further. Of the other answers: (B) Opening\\nof GABA receptors, allowing chloride to enter the postsynaptic cell, will make it harder for the cellto depolarize to threshold. (C) Noradrenaline acts at metabotropic receptors, so if it does have aneffect, it will be slow and therefore will not affect the ability of the seven other action potentials totrigger an action potential. (D) If the action potentials are spaced further apart, the transmembranevoltage has more time to relax back to the resting voltage between each one, so the maximumvoltage during each action potential will be less, that is, further from threshold. (E) Nerve cellsdo not usually have PDGF receptors. If they did, the effect of this metabotropic receptor wouldbe slow and therefore would not affect the ability of the seven other action potentials to trigger anaction potential.\\n6.B.The smooth muscle cells in blood vessels do not have receptors for acetylcholine. Rather,\\nacetylcholine acts indirectly, via endothelial cells and NO, to relax the smooth muscle. All theother statements are true.\\n7.C.Binding of acetylcholine to nicotinic acetylcholine receptors will depolarize the cells, opening\\nvoltage-gated calcium channels and allowing calcium ions to ﬂow in from the extracellular medium.\\nSimultaneously, binding of acetylcholine to muscarinic acetylcholine receptors will activate phos-pholipase C βand therefore cause release of calcium from the smooth endoplasmic reticulum.\\nAnswers A and B are therefore wrong since neither in ﬂux nor release from endoplasmic reticulum\\ncan be said to constitute the vast majority of the calcium entering the cytosol. Answers D and E arecompletely wrong: the transmembrane voltage becomes more positive and the cytosolic calciumconcentration increases.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='88ac8e4e-52e2-40ef-972e-afc9cb713b45', embedding=None, metadata={'page_label': '386', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\n18\\nMECHANICAL MOLECULES\\nTHE CYTOSKELETON IS BOTH STRONG AND MOTILE\\nEukaryotic cells are supported by a network of struts and cables called the cytoskeleton. In\\nanimal cells, which lack a rigid cell wall, it is the cytoskeleton that determines cell shape.In addition the cytoskeleton is responsible for cell locomotion, for moving componentsfrom place to place within the cell, and for cell division. The cytoskeleton is composedof three cytoplasmic ﬁlament networks: microtubules, microﬁlaments, andintermediate\\nﬁlaments (Fig. 18.1). Although all three are physically strong, microtubules and microﬁla-\\nments have the additional role of organizing movement, both of entire cells and of structureswithin them. The term cytoskeleton implies a rigid set of “bones” within the cell. Nothingcould be further from the truth, and all three ﬁlament systems are highly dynamic, alteringtheir organization in response to the needs of the cell. The molecules that make up thethree ﬁlament systems have been highly conserved throughout evolution; the cytoskeletalproteins present in the cells of complex organisms such as humans are much the same asthose in a simple organism such as a yeast. Although the individual molecules making up thecytoskeleton are below the limit of resolution of the light microscope, the cytoskeleton itselfcan be readily observed within the cell by using ﬂuorescence microscopy (e.g., page 6).\\nMICROTUBULES\\nMicrotubules possess a combination of physical properties that allows them to participatein multiple cellular functions. They can form bundles of rigid ﬁbers that make excellent\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n381', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='64208069-17dc-47bd-9978-93a534d14403', embedding=None, metadata={'page_label': '387', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\n382 MECHANICAL MOLECULES\\nmicrotubulesnucleuscentrosome\\nmicrofilaments\\n(stress fibers)intermediate filaments \\nFigure 18.1. Typical spatial organization of microtubules, stress ﬁbers (one form of microﬁla-\\nments), and intermediate ﬁlaments.\\nstructural scaffolds and hence serve an important role in the determination of cell shape.\\nThey have an inherent structural polarity that de ﬁnes a polarity (front vs. back) to the cell.\\nThey provide a system of intracellular highways that supports a two-way traf ﬁc of organelles\\nand small vesicles powered by enzymes that interact with the microtubule surface. Theycan be rapidly formed and broken down, a property that allows the cell to respond to subtleenvironmental changes. Finally, they play a role in one of the most exquisite and preciseof all movements within the cell, the segregation of chromosomes at mitosis and meiosis(Chapter 19).\\nAnimal cells contain a network of several thousand microtubules, each 25 nm in diame-\\nter. All the cell ’s microtubules can be traced back to a single structure called the centrosome,\\nwhich is tightly attached to the surface of the nucleus at the cell centre (Fig. 18.1). Thecentrosome is the microtubule organizing center of the cell and consists of amorphous ma-terial enclosing a pair of centrioles (Fig. 18.2). Centrioles have a characteristic nine-way\\npattern that we will meet again in cilia and ﬂagella (page 386). Microtubules in plant cells\\nhave a quite different organization, lying immediately beneath the cell membrane, orientedat right angles to the direction of cell expansion. The role of microtubules in plants is todirect the deposition of cellulose ﬁbers on the outside of the cell membrane. Cellulose\\nsynthase, a multiprotein enzyme complex that spans the cell membrane, is thought to move\\nalong microtubules forming new parallel tracks of cellulose micro ﬁbrils on the cell surface\\n(Fig. 18.3). As a consequence of the position of the microtubules beneath the membrane,cellulose is laid down in hoops that encase the plant cell in a rigid corset and allow it toexpand only in one direction. Plant cells do not contain centrioles, and it remains unclearwhether their microtubules arise from a de ﬁned organizing center.\\nMicrotubules are composed of a protein called tubulin that consists of two dissimilar\\nsubunits designated αandβ. In the human genome there are about ﬁveα-tubulin genes and\\nroughly the same number for βtubulin. There is a third member of the tubulin superfamily,\\nγtubulin, which does not itself contribute to microtubule structure but which is found at\\nthe centrosome and plays a role in initiating microtubule assembly. α-Tubulin/ β-tubulin\\ndimers assemble into chains called proto ﬁlaments, 13 of which make up the microtubule\\nwall (Fig. 18.4). Within each proto ﬁlament the tubulin dimers are arranged in a “head-to-\\ntail”manner, αβαβ and so on. This gives the microtubule an built-in molecular polarity\\nthat is re ﬂected in the way it grows. Tubulin subunits are added to, and lost from, one end\\nmuch more rapidly than the other. By convention, the fast growing end is referred to as the(+) end and the slow growing end as the ( −) end.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4be8f4d6-4353-4924-b5c9-f90c2b8a6a76', embedding=None, metadata={'page_label': '388', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\npericentriolar\\nmaterial\\ncentrioles\\nmicrotubulescross\\nsection\\nFigure 18.2. The microtubule organizing center or centrosome consists of amorphous material enclosing a pair of centrioles.\\n383', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='bfb73540-8e89-44b6-892b-2135c4710c0f', embedding=None, metadata={'page_label': '389', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\n384 MECHANICAL MOLECULES\\ncellulose microfibril\\nmicrotubule\\ncellulose\\nsynthaseplasma\\nmembrane(a)\\n(b)cellulose\\nmicrofibrilDirection of cellexpansion\\nFigure 18.3. Cellulose synthase runs on microtubule tracks.\\nIn cells, the minus end of each microtubule is embedded in the centrosome so that only\\nthe plus ends are free to grow or shrink. This process is surprisingly complex. Individual\\nmicrotubules undergo periods of slow growth followed by rapid shrinkage, sometimesdisappearing completely. This phenomenon is referred to as dynamic instability. By chance,the growing ends of certain microtubules may be captured by sites at the cell membrane andstabilized, so that they are protected from shrinkage. Their further growth in ﬂuences the\\nshape of the cell (Fig. 18.5). Groups of microtubules having a common orientation makean excellent structural framework. Because microtubules are dynamic, the framework canbe continually remodeled as the needs of the cell change.\\nOne of the most important tools in establishing microtubule function in cells has been\\nthe plant alkaloid colchicine. Extracted from the corms of the autumn crocus, Colchicum', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='421dc9d5-92a7-449a-b70a-5fbce2257121', embedding=None, metadata={'page_label': '390', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\nMICROTUBULES 385\\nmicrotubule\\nprotofilament(+)( −)\\nα-tubulin/ β-tubulin dimer\\nFigure 18.4. Microtubule structure.\\nautumnale, colchicine has been used since Roman times as a treatment for gout. Cells\\nexposed to colchicine lose their shape, and the movement of organelles within the cytoplasm\\nceases. When the drug is washed away, microtubules reassemble from the centrosome andnormal functions are resumed (Fig. 18.6). Another drug, taxol, obtained from the bark ofthe Paci ﬁcy e w , Taxus brevifolia, has the opposite effect, causing large numbers of very\\nstable microtubules to form in the cell, an effect that is dif ﬁcult to reverse.\\nUnlike animal cells, plant cells do not use their cytoskeletons to de ﬁne their cell shape,\\nbut instead use their extracellular cell walls. The direction in which a growing plant cellextends is determined in the ﬁrst instance by the direction in which the cellulose micro-\\nﬁlaments run. However, these are laid down by cellulose synthase following microtubule\\ntracks so in fact the cytoskeleton has de ﬁned the direction of cell growth, but indirectly\\nrather than by itself acting as a load-bearing component.\\ndynamic instability, \\nmicrotubules grow and shrink, \"sensing\" their environmentthe ends of these two \\nmicrotubules have \\nbeen \"captured\" and \\nstabilized. Now, they \\ndo not shrink but only \\ngrow, altering the cell \\nshape\\ncentrosome\\nnucleus\\nFigure 18.5. Microtubules show dynamic instability.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='fbe6c70c-7bcd-4b23-be6a-074d73a68a2e', embedding=None, metadata={'page_label': '391', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\n386 MECHANICAL MOLECULES\\ncolchicinetaxol\\nFigure 18.6. Effects of taxol and colchicine on microtubules.\\nMICROTUBULE-BASED MOTILITY\\nCells move for a variety of reasons. Human spermatozoa in their millions swim frantically\\ntoward an ovum; the soil amoeba, Acanthamoeba (said to be the most abundant organism\\non Earth), crawls over and between soil particles, engul ﬁng bacteria and small organic\\nparticles as it does so. Cells in the early human embryo show similar crawling movementas they reorganize to form tissues and organs; the invasive properties of some cancer cellsis due to their reverting to this highly motile embryonic state. Of course, not all cellsshow these obvious forms of motility, but careful observation of even the most sedentaryeukaryotic cells often reveals a remarkable repertoire of intracellular movements. Bothmicrotubules and actin ﬁlaments play important roles in cell motility. We will describe\\nmicrotubule-based motility ﬁrst; actin-based motility will be covered following the section\\non micro ﬁlaments.\\nCilia and Flagella\\nCilia and ﬂagella appeared very early in the evolution of eukaryotic cells and have remained\\nessentially unchanged to the present day. The terms cilium (meaning an eyelash) and\\nﬂagellum (meaning a whip) are often used arbitrarily. Generally, cilia are shorter than\\nﬂagella (<10 µm compared to >40µm) and are present on the surface of the cell in much\\ngreater numbers (ciliated cells often have hundreds of cilia but ﬂagellated cells usually\\nhave a single ﬂagellum). The real difference, however, lies in the nature of their movement\\n(Fig. 18.7). Cilia row like oars. The movement is biphasic, consisting of an effective stroke\\nin which the cilium is held rigid and bends only at its base and a recovery stroke in which\\nthe bend formed at the base passes out to the tip. Flagella wriggle like eels. They generatewaves that pass along their length, usually from base to tip at constant amplitude. Thusthe movement of water by a ﬂagellum is parallel to its axis while a cilium moves water\\nperpendicular to its axis and, hence, perpendicular to the surface of the cell.\\nCilia are such a conspicuous feature of some protists that they are called ciliates. The\\nswimming of a paramecium (Fig. 18.8), for example, is generated by the coordinated motionof several thousand cilia on the cell surface. Cilia also play a number of important roles inthe human body. The respiratory tract, for example, is lined with about 0.5 m\\n2of ciliated\\nepithelium, bearing something like 1012cilia. The beating of these cilia moves a belt of\\nmucus containing inhaled particles and microorganisms away from the lungs. This activityis paralyzed by cigarette smoke, causing mucus to accumulate in the smoker ’s lung and\\ncausing the typical cough.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='9e00ebdd-834c-4a39-9edc-f08e8037faf8', embedding=None, metadata={'page_label': '392', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\nMICROTUBULE-BASED MOTILITY 387\\n18 765 43\\n2\\nbasal body1\\n23basal bodycilium\\nflagellummedium pushed this way\\ncell pushed this way\\ncell pushed\\nthis way\\nmedium pushed this way\\nFigure 18.7. How cilia and ﬂagella bend.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ef7c1939-5d5e-4421-bbd8-ddcd90e40867', embedding=None, metadata={'page_label': '393', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\n388 MECHANICAL MOLECULES\\nParamecium  (cilia) sea urchin spermatozoon \\n(flagellum)\\ncross section \\nB microtubule\\nA microtubule\\nouter dynein arm\\ninner dynein arm\\ncentral pair projectionsmembrane\\ncentral pair microtubulesradial spokeinterdoublet link\\ntubulin \\nsubunit\\nFigure 18.8. Cilia and ﬂagella have identical structure.\\nDespite their different pattern of beating, cilia and ﬂagella are indistinguishable struc-\\nturally (Fig. 18.8). Both consist of a regular arrangement of microtubules and associated\\nproteins with the nine-way pattern we have also seen in centrioles. Unlike centrioles, ciliaandﬂagella have a central pair of microtubules, so that the overall structure is called the\\n9+2 axoneme. The axoneme is enclosed by an extension of the cell membrane. Attached\\nto the nine outer doublet microtubules are projections, or arms, composed of a motor proteincalled dynein. Dynein is an ATPase that converts the energy released by ATP hydrolysis into\\nthe mechanical work of ciliary and ﬂagellar beating. Using ATP produced by mitochondria\\nnear the base of the cilium or ﬂagellum as fuel, the dynein arms push on the adjacent outer\\ndoublets, forcing a sliding movement to occur between adjacent outer doublets. Becausethe arms are activated in a strict sequence both around and along the axoneme and becausethe amount of sliding is restricted by the radial spokes and interdoublet links, sliding isconverted into bending.\\nBacterial ﬂagella (page 264) use a fundamentally different mechanism. Like the pro-\\npellor of a boat, the motion of the bacterial ﬂagellum is entirely driven by the rotary motor\\nat its base. The bacterial ﬂagellum itself is a specialized piece of extracellular cell wall,', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e55a833e-14f5-4926-aa3c-10847c98e99f', embedding=None, metadata={'page_label': '394', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\nMICROTUBULE-BASED MOTILITY 389\\nmade of one protein ( ﬂagellin) that has no similarity to tubulin or dynein. Cilia and ﬂagella\\nare full of cytosol all the way to their tips, and use the ATP in that cytosol to generate force\\nall the way along their length.f\\nIN DEPTH 18.1 Calcium and Paramecium Swimming\\nParamecium, although it is a single-celled organism, is a fast-swimming predator.\\nIts surface is covered with hundreds of cilia that are arranged in rows that lie\\nobliquely to the cell axis. Cilia beat with their effective stroke oriented toward therear, propelling the cell forward in a spiral motion. The beating of the individualcilia is coordinated to produce “metachronal waves ”that pass from the front of\\nthe animal to the rear, rather like the wind passing across a corn ﬁeld.Paramecium\\nneeds millisecond-to-millisecond control of its cilia as it swims through pondweedin search of prey. One re ﬂex, the “avoiding reaction, ”wasﬁrst described almost a\\ncentury ago. When a Paramecium is swimming forward, the internal calcium con-\\ncentration is kept low, at about 100 nmol liter\\n−1. When the front of the animal\\nbumps into an object, the plasma membrane depolarizes, opening voltage-gatedcalcium channels in the cilia all the way down the animal as a calcium actionpotential is generated. The in ﬂux of calcium ions raises the intracellular calcium\\nconcentration, and this in turn causes the cilia to swing through 180\\n◦so that the\\neffective stroke is now directed toward the anterior. This causes the Paramecium\\nto reverse a few body lengths, taking it out of contact from the object with whichit collided. Calcium is now pumped out of the cell and the internal calcium con-centration falls back to 100 nmol liter\\n−1. The cilia now swing back to their original\\nposition and forward swimming is resumed.\\n✞\\n✝✆\\n✆Medical\\nRelevance\\n18.1A Sinister Complaint\\nA small number of men seeking the help of fertility clinics turn out to have not\\nonly nonswimming sperm but a failure of all the cilia and ﬂagella in their bodies.\\nOn questioning, they are often found to suffer from chronic respiratory disease because the cilialining their airways are also immobile. Most surprising is the fact that half of these individualsalso exhibit Kartegener ’s syndrome, a condition in which the internal organs have the reversed\\norientation from normal, that is, the heart is on the right, the appendix on the left, and so on. Itturns out that cilia set up a ﬂow from right to left across the early embryo. Transmitters carried in\\nthisﬂow tell the left-hand side to develop its characteristic organs, and vice versa. If a mutation\\nrenders the cilia immobile, the left –right gradient of transmitter does not get set up, and the body\\norgans are positioned by chance and not design.\\nIntracellular Transport\\nThe beating of cilia and ﬂagella is an obvious manifestation of movement generated by\\na microtubule-based structure having a de ﬁned and geometric shape. However, motility\\nis a general property of microtubules within cells. It is seen particularly clearly in some\\nspecialized cell types such as the pigment cells called chromatophores in the skin of\\nﬁsh and amphibia. The inward and outward movement of pigment granules along radial\\narrays of microtubules underlies the remarkable color changes such animals are able todisplay (Fig. 18.9). But this is just an exaggerated example of a process that occurs less', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a1b18b48-188a-4c64-ace3-c19407a5d152', embedding=None, metadata={'page_label': '395', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\n390 MECHANICAL MOLECULES\\nFigure 18.9. Pigment migration in ﬁsh chromatophores.\\nspectacularly in all cells. For instance, nerve cell axons extend up to 1 m from the cell\\nbody. Organelles, small vesicles, and even mRNA are transported in both directions in aphenomenon referred to as axonal transport (Fig. 18.10). This is subdivided into outward\\noranterograde transport and inward or retrograde transport. Both are dependent upon\\nmicrotubules that are abundant in nerve cells.\\nThe two forms of axonal transport are dependent upon different molecular motors.\\nDynein, sometimes referred to as cytoplasmic dynein to distinguish it from its relative in\\ncilia and ﬂagella, moves vesicles and organelles in the retrograde direction while another\\nprotein, kinesin, is the motor for movement along microtubules in the anterograde direction.\\nTheir directionality is speci ﬁed by the polarity of the microtubules. Thus, dynein moves\\nalong a microtubule in a plus to minus direction while kinesin works in the opposite direction.Both proteins consist of a tail that binds to the cargo to be transported and two (kinesin) orthree (dynein) globular heads that interact with the surface of the microtubule, generatingmovement. Speci ﬁcity is imparted to this process by having multiple dyneins and kinesins\\nin a single cell, each responsible for transporting a speci ﬁc type of cargo. This multiplicity\\nof dyneins and kinesins is created by alternative splicing of a smaller number of genes(page 118).\\nMICROFILAMENTS\\nMicro ﬁlaments are ﬁneﬁbers, about 7 nm in diameter, that are made up of subunits of\\nthe protein actin. Because it is a globular protein, the actin monomer is designated G-actin while the ﬁlament that forms from it is referred to as F-actin. Each actin ﬁlament is\\ncomposed of two chains of actin monomers twisted around one another like two strands ofbeads (Fig. 18.11). In animal cells, actin is particularly associated with the cell periphery.When they are grown in plastic cell culture dishes, nonmotile cells have two main types ofmicro ﬁlaments: bundles of actin ﬁlaments called stress ﬁbers run across the cell and help to\\nanchor it to the dish (Fig. 18.1) while under the cell membrane can be seen loose meshworkofﬁlaments that give the edges of the cell structural strength. In actively moving cells,', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='9cea4926-7882-4e39-badd-d8f041ae2d5e', embedding=None, metadata={'page_label': '396', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\nMICROFILAMENTS 391\\naxon terminal\\nvesicle \\nplus \\nkinesin \\nmoleculesvesicle \\nplus dynein moleculesanterograde \\ntransport\\nretrograde transportmicrotubuleaxon\\ncell bodynucleus+\\n−\\nFigure 18.10. Axonal transport.\\nthe stress ﬁbers disappear and actin becomes concentrated at the leading edge. Projections\\nfrom the cell surface such as microvilli (page 13) are maintained by rigid bundles of actin\\nﬁlaments.\\nThe equilibrium between G- and F-actin is affected by many factors including actin-\\nbinding proteins (Fig. 18.11). Filament growth is prevented by proﬁlin,which binds\\nG-actin monomers and prevents their polymerization, and by capping proteins that bindto F-actin ﬁlament ends. In contrast, ﬁlament nucleation proteins such as actin-related\\nprotein (Arp) act as a base on which new ﬁlaments can form. Crosslinking proteins stick\\nto two existing ﬁlaments, forming a mechanically strong lattice. Of these villin, found in', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='daa8bcf3-b034-44cb-b0dd-7bb851114a87', embedding=None, metadata={'page_label': '397', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\n392 MECHANICAL MOLECULES\\nG-actinGrowing filaments push membrane forward\\nprofilin\\nprofilin : actin dimerArp\\ncapped filaments\\ncannot elongatecapping proteinelongation\\nactin flamentelongation\\nFigure 18.11. Actin polymerization is regulated by actin-binding proteins.\\nmicrovilli, generates parallel bundles, while related proteins bind criss-crossing ﬁlaments\\ntogether to form a viscous, three-dimensional cytoplasmic gel.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 18.1 Spearing Eggs\\nUpon contacting an egg the sperm of the marine organism Thyone suddenly extends a needlelike\\nprocess called an acrosome from its front end. This 90- µm-long needle helps the sperm to penetrate\\nthe egg. Prior to activation, a vesicle at the sperm front end contains G-actin at concentrations way\\nabove that which would normally result in spontaneous polymerization into F-actin ﬁlaments. The\\nreason it does not do so is that all the actin is bound to pro ﬁlin. Upon contact with the egg, salts\\nfrom seawater ﬂow into the vacuole and cause dissociation of the pro ﬁlin–actin dimers. The actin\\nthen immediately polymerizes to form the long micro ﬁlaments of the acrosome.\\nCells are anchored to the extracellular matrix through transmembrane proteins such\\nasintegrins. These are dimeric proteins that have an extracellular domain that binds to\\ncollagen and other extracellular matrix proteins and an intracellular domain that attaches to\\nactin micro ﬁlaments on the inside (Fig. 18.12).', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b7450b5f-be1d-4836-b471-82151467b43c', embedding=None, metadata={'page_label': '398', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\nMICROFILAMENTS 393\\nextracellular matrix\\nintegrin\\nadaptor protein\\nactin filament\\ncytosol\\nFigure 18.12. Integrins anchor the actin cytoskeleton to the extracellular matrix.\\n✞\\n✝✆\\n✆Medical\\nRelevance\\n18.2Some Bacteria Highjack the Cytoskeleton for Their Own Purposes\\nWe are all familiar with the ways that bacteria and viruses spread from person to\\nperson by various forms of social or antisocial contact, but how does a bacterium\\nspread through the cells of a human body? A number of bacteria including the important pathogensListeria (responsible for sepsis and meningitis in immunocompromised patients and for infections\\nof the fetus during pregnancy) and Shigella (which causes dysentery), avoid contact with the human\\nbody ’s antibodies and white blood cells by remaining hidden within the cytoplasm of our cells as\\nthey spread through our tissues. The bacteria use actin to power their journey from cell to cell.Listeria has a protein called ActA at one end of its rod-shaped body. Like Arp, ActA promotes\\nthe polymerization of actin ﬁlaments. This generates enough force to drive the bacterium through\\nthe cytoplasm in the opposite direction. Movement is random, but occasionally the bacterium willbump into the cell membrane causing the formation of a ﬁngerlike projection from the cell surface.\\nThe membrane of the projection fuses with the membrane of the neighboring cell, transferring thebacterium in a membrane sac from which it quickly escapes to repeat the procedure. The reallyclever thing (from the bacterium ’s point of view) about such a transfer mechanism is that the\\nbacterium never leaves the safety of the cytoplasm.\\nMuscle Contraction\\nStriated muscle, the kind of muscle found attached to bones or in our hearts, gains that\\nappearance because passing along the cell one encounters in turn regions of parallel mi-croﬁlaments (which are relatively transparent) then regions of thick ﬁlaments of a second\\nprotein called myosin (Fig. 18.13). The complete repeating unit is called a sarcomere and isdelineated by the Z disc, which holds the micro ﬁlaments in a regular pattern, so that striated\\nmuscle has an almost crystalline appearance in transverse section. The myosin moleculehas a distinctive structure, consisting of a tail and two globular heads. The thick ﬁlament is\\nformed from a large number of myosin molecules arranged in a tail-to-tail manner. This de-sign means that the thick ﬁlament is a bipolar structure with myosin heads at both ends. In a\\nresting striated muscle, the myosin heads are unable to operate on the actin micro ﬁlaments.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='8ca34be9-7307-4d9e-b653-76011b1f5132', embedding=None, metadata={'page_label': '399', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\n394 MECHANICAL MOLECULES\\nZ discsarcomere ~2.5 µm\\nthin filament\\nthick filamentZ disc\\nthin filament (actin)sliding sliding\\nlongitudinal section\\nthick filament (myosin)\\nmyosin headsactin monomer\\nmyosin tail\\ntransverse sectioncontraction contractienlarged areaenlarged areaenlarged area\\nenlarged areacontraction\\nFigure 18.13. Muscle contraction.\\nWhen the cytosolic calcium concentration increases, calcium binds to a protein called tro-\\nponin that changes shape to allow myosin access to the actin. The myosin heads now crawlalong the actin (thin) ﬁlaments and generate sliding that is powered by the hydrolysis of\\nATP. Because of the geometry of the system, the two Z discs are pulled toward each otherand the cell shortens (Fig. 18.13).\\nSeveral types of myosin are found in nonmuscle cells. One of them, called myosin II,\\nis very similar to muscle myosin but does not assemble into ﬁlaments to the same extent,', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='838511f8-3f72-47ec-93d9-9fbbca36e6f0', embedding=None, metadata={'page_label': '400', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\nMICROFILAMENTS 395\\nSurfacePseudopodium\\nplasma\\nmembraneendoplasm\\n(sol) ectoplasm (gel)\\nTailHigh calcium,\\nactin depolymerization\\n(gel-sol)\\nmyosin contractionLow calcium,\\nactin polymerization\\n(sol-gel)\\nfocal contacts\\nDirection of movement\\nFigure 18.14. Amoeboid movement resembles the progression of a military tank.\\nprobably because the levels of force required within nonmuscle cells is relatively small. The\\nprimary role of myosin II is in cell division (page 404). Myosin V is also two-headed and\\nis responsible for carrying cargo (vesicles or organelles) along actin ﬁlaments. Unlike the\\nmicrotubule-associated motors dynein and kinesin, which can make long journeys, myosinV can only make short excursions along an actin ﬁlament before falling off.\\nCell Locomotion\\nFor a cell such as an amoeba to crawl across a surface it must generate traction. Less than 1%of the ventral surface of a crawling cell is in contact with the surface over which it is moving,so that the cell does not slither along on its belly but rather “tiptoes ”delicately across the\\nsurface. The transient points of attachment with the surface are called focal contacts. At\\nthe leading edge actin polymerization (Fig. 18.11) causes the cell to extend projections orpseudopodia in the direction of progress. The overall increase in actin polymerization and\\ncrosslinking in this region causes the cytoplasm to form a viscous ectoplasm (Fig. 18.14). In\\ncontrast, raised cytosolic calcium concentrations in the tail activate actin-severing proteins\\nsuch as gelsolin. Calcium also activates myosin II, the resulting contraction driving the now\\nﬂuid endoplasm forward and displacing the existing actin cytoskeleton backward. Since\\nactin is connected via the focal contacts to the surface, the backward movement of theectoplasm drives the cell forward, operating rather like the tracks of a military tank.\\nCytoplasmic Streaming\\nEven though the cells themselves do not move, the cytoplasm of many plant cells also showsactive cytoplasmic streaming. In the epidermal cells of the leaves of the water plant, Elodea,\\na belt of moving cytosol carries the chloroplasts and other organelles in a continuous,unidirectional stream around the central vacuole. In the giant cells of the alga Nitella,\\nthe chloroplasts are embedded in a layer of viscous ectoplasm while other organelles arecarried along in the ﬂowing endoplasm. Cytoplasmic streaming is driven by the interaction\\nof myosin molecules attached to the moving organelles with cables of actin ﬁlaments in the\\nectoplasm.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3cc57d1b-fe98-415d-ab77-5b8774a665bb', embedding=None, metadata={'page_label': '401', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\n396 MECHANICAL MOLECULES\\nINTERMEDIATE FILAMENTS\\nIntermediate ﬁlaments were so named originally because their diameter, 10 nm, lay between\\nthat of the thin and thick muscle ﬁlaments. They are the most stable of the cytoplasmic ﬁl-\\nament systems. Although intermediate ﬁlaments from different mammalian tissues look\\nmuch the same in the electron microscope, they are in fact composed of different protein\\nmonomers (and can therefore be distinguished by immuno ﬂuorescence, page 6). Nerve\\ncells contain neuro ﬁlaments, muscle cells contain desmin ﬁlaments, ﬁbroblasts contain\\nvimentin ﬁlaments, and epithelial cells contain ﬁlaments composed of keratin, the protein\\nthat gives our skin its protective coating, forms our hair and ﬁnger nails, and makes the\\nhorns and hooves of our domestic animals and pets. These different proteins can generatea common structure because all share the same basic design. This consists of a central,α-helical rod and a nonhelical head and tail. Most of the variation between intermediate\\nﬁlament proteins is in the head and tail, and these regions probably confer subtly different\\nproperties on different intermediate ﬁlament classes. The basic building block of interme-\\ndiate ﬁlaments is a tetramer of pairs of subunit proteins joined by their central region to\\nform coiled coils.\\nIn the cell, intermediate ﬁlaments tend to form wavy bundles that extend from the\\nnucleus to the cell surface (Fig. 18.1). It appears that the nucleus is suspended by intermediateﬁlaments stretching to the plasma membrane, rather like a sailor in a hammock. Indeed\\nintermediate ﬁlaments share a high degree of amino acid sequence homology with the\\nnuclear lamins that reinforce the nuclear envelope on the inside (page 57).\\nAnchoring Cell Junctions\\nThe cells that form tissues in multicellular organisms are attached together with anchoringjunctions (Fig. 18.15). Integral membrane proteins called cell adhesion molecules, of which\\ncadherin is an example, extend out from each cell and bind tightly together, while their cy-\\ntosolic domains attach to the cytoskeleton. There are two basic types of anchoring junction.Inadherens junctions the cell adhesion molecules are linked to actin micro ﬁlaments by\\nlinking proteins such as catenin. In desmosomes the cell adhesion molecules are linked to\\nintermediate ﬁlaments. Tissues that need to be mechanically strong, such as the epithelial\\ncells of the gut (page 14) and cardiac muscle, have many anchoring junctions linking thecytoskeletons of the individual cells. Anchoring junctions are one of the three types of celljunctions, the others being tight junctions (page 55) and gap junctions (page 55).\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 18.2 Protected by the Dead\\nThe epidermis of the skin is made up of a layer of living cells called keratinocytes covered by\\na protective layer of their dead bodies. Dead keratinocytes form a good protective layer becausewhile alive they generate a dense internal cytoskeleton of the intermediate ﬁlament keratin, with\\nadjacent cells being linked by desmosomes. When the cells die, the keratin ﬁbers remain because\\nintermediate ﬁlaments are stable. Since the intermediate ﬁlaments were joined by desmosomes, the\\nresulting protective ﬁbers do not stop at the edge of the now dead cell, but are strongly connected\\nwith the ﬁbers in the next cell, and the next, and so on, forming an extremely strong network of\\nﬁbers.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='9911991b-7116-4837-adbb-7717f8ec4aa9', embedding=None, metadata={'page_label': '402', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\ndesmosomeadherens\\njunction\\nmicrofilament\\ncytosol extracellular medium\\ncadherin catenincatenin\\ncadherincadherin\\nactin filament\\nintermediate\\nfilament\\nactin\\nmonomer\\nFigure 18.15. Anchoring junctions attach the cytoskeletons of adjacent cells.\\n397', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6e9ce18c-15b0-4a64-b743-5ea76c2e964b', embedding=None, metadata={'page_label': '403', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\n398 MECHANICAL MOLECULES\\nSUMMARY\\n1.The cytoskeleton is made up of microtubules, micro ﬁlaments, and intermediate\\nﬁlaments.\\n2.Microtubules are composed of equal amounts of αandβtubulin. In animal cells\\ntheir minus ( −) end is stabilized at the centrosome while their plus ( +) end shows\\ndynamic instability.\\n3.Cilia and ﬂagella contain a 9+2 axoneme of microtubules plus the motor protein\\ndynein.\\n4.Dynein and kinesin transport cargo along microtubules.\\n5.Micro ﬁlaments are composed of actin.\\n6.Actin-binding proteins control actin polymerization and determine the organization\\nof the actin lattice.\\n7.The motor protein myosin operates in all cell types but is particularly prominent inmuscle cells. It operates on actin micro ﬁlaments when cytosolic calcium concen-\\ntration rises.\\n8.Cell locomotion is driven by spatially distinct zones of actin polymerization anddepolymerization, aided by myosin. A similar cycle underlies cytoplasmic streamingin plants.\\n9.The proteins that form intermediate ﬁlaments differ in different cells. Intermedi-\\nateﬁlaments are more stable than microtubules and micro ﬁlaments and serve a\\nstructural role only.\\n10.Anchoring cell junctions attach the cytoskeletons of adjacent cells together. Theyare divided into adherens junctions, which link to actin, and desmosomes, whichlink to intermediate ﬁlaments.\\nFURTHER READING\\nHirokawa N. 1998. Kinesin and dynein superfamily proteins and the mechanism of organelle transport.\\nScience 279: 519 –526.\\nPreston, T. M., King, C. A., and Hyams, J. S. 1990. The Cytoskeleton and Cell Motility. Glasgow:\\nBlackie.\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.Which of the following are notcomponents of the cytoskeleton?\\nA. Actin ﬁlaments', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='dfb556b8-a356-4393-9752-f7473d1f4dab', embedding=None, metadata={'page_label': '404', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\nREVIEW QUESTIONS 399\\nB. Collagen ﬁbers\\nC. Microtubules\\nD. Micro ﬁlaments\\nE. Intermediate ﬁlaments\\n2.Which of the following statements are incorrect ?\\nA. Dynein mediates retrograde axonal transport.B. Myosin molecules move along actin ﬁlaments.\\nC. Muscle thin ﬁlaments are composed of actin.\\nD. Bacterial ﬂagella have a 9+2 structure.\\nE. Cilia are abundant in the human respiratory tract.\\n3.Arp, pro ﬁlin, and villin are all\\nA. cell adhesion molecules.B. molecular motors.C. actin-binding proteins.D. intermediate ﬁlament proteins.\\nE. nonmuscle myosins.\\n4.Plant cells lack\\nA. microtubules.B. centrioles.C. actin.D. gelsolin.E. cellulose synthase.\\n5.In cross section cilia and ﬂagella have nine equally spaced sets of micro-\\ntubule structures. This organization —nine equally spaced units —is also shown\\nby\\nA. nuclear lamins.B. the nuclear pore.C. bacterial ﬂagellin.\\nD. centrosomes.E. connexins.\\n6.Motor proteins acting on actin micro ﬁlaments include\\nA. dynamin.B. myosin.C. kinesin.D. dynein.E. all of the above.\\n7.Desmosomes differ from adherens junctions in that\\nA. desmosomes are structurally strong, while adherens junctions block the movement of extracel-\\nlular medium.\\nB. desmosomes block the movement of extracellular medium, while adherens junctions are struc-\\nturally strong.\\nC. desmosomes connect to intermediate ﬁlaments while adherens junctions connect to micro-\\ntubules.\\nD. desmosomes connect to intermediate ﬁlaments while adherens junctions connect to micro ﬁla-\\nments.\\nE. desmosomes anchor the nucleus in position within the cell while adherens junctions connect\\ncells together.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='5a033238-e861-4ead-9489-372854af7d68', embedding=None, metadata={'page_label': '405', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-18 WY001-Bolsover-v2.cls September 4, 2003 13:58\\n400 MECHANICAL MOLECULES\\nANSWERS TO REVIEW QUESTIONS\\n1.B.Collagen is a component of the extracellular matrix.\\n2.D.The bacterial ﬂagellum is made of the protein called ﬂagellin; it does not have a 9 +2 structure.\\nTubulin and dynein, the components of the 9 +2 axoneme, are only found in eukaryotes.\\n3.C.\\n4.B.Plant cells do not contain centrioles, and it remains unclear whether their microtubules arise\\nfrom a de ﬁned organizing center.\\n5.D.Centrosomes are made of tubulin and in cross section have an organization similar to that of\\ncilia and ﬂagella. Bacterial ﬂagellin has no obvious subdivision; nuclear lamins, the nuclear pore,\\nand connexins are formed of subunits arranged in patterns of 4, 8, and 6, respectively.\\n6.B.Myosin acts on actin. In contrast kinesin and dynein act on microtubules, and dynamin causes\\nthe pinching off of clathrin-coated vesicles.\\n7.D.Both desmosomes and adherens junctions are types of anchoring cell junction, desmosomes\\nconnecting to intermediate ﬁlaments while adherens junctions connect to micro ﬁlaments.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='40b1af40-ab25-4e20-9d4f-f6ea694246c9', embedding=None, metadata={'page_label': '406', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\n19\\nCELL CYCLE AND CONTROL\\nOF CELL NUMBER\\nAll cells arise by the division of an existing cell. The life of a cell from the time it is generated\\nby the division of a progenitor cell to the time it in turn divides is called the cell division\\ncycle or just the cell cycle for short. The duration of the cell cycle varies from 2 to 3 h in a\\nsingle-celled organism like the budding yeast Saccharomyces cerevisiae to around 24 h in a\\nhuman cell grown in a culture dish. During this period the cell doubles in mass, duplicatesits genome and organelles, and partitions these between two new progeny cells. Theseevents have to be carried out with great precision and in the correct order, and cells haveestablished sophisticated control processes to ensure that the cell cycle proceeds with therequired accuracy. In humans the cells of some tissues, such as the skin, the lining of the gut,and the bone marrow continue to divide throughout life. Others, such as the light-sensitivecells of the eye and skeletal muscle cells, show almost no replacement. The latter, laid downin infancy, must last a lifetime. Not only must cell division be precisely controlled in termsof which cells divide when, the whole process has to know when to stop. A human being isbigger than a mouse and smaller than an elephant. Allowing for some minor differences inbasic design, this is because humans are made up of more cells than a mouse (and, hence,are the result of more cell cycles) and less than an elephant. How are these differencesachieved? Why don’t humans grow to be the size of elephants or whales? What has becomeincreasingly clear over recent years is that cells contain what have become known as “cellcycle control genes.” The proper functioning of such genes not only determines how bigwe are, it also prevents cell division becoming “out of control” and leading to cancer.\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n401', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='96a4dc6d-bcc7-4cc8-a1f9-72deb0bf2b0c', embedding=None, metadata={'page_label': '407', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\n402 CELL CYCLE AND CONTROL OF CELL NUMBER\\nUnder the microscope it is possible to distinguish two elements of the cell cycle.\\nInterphase occupies about 90% of the cell cycle and is a period of synthesis and growth,\\nduring which the cell roughly doubles in mass but without displaying obvious morphological\\nchanges. Once interphase is complete, the cell enters mitosis, which is a brief period of\\nprofound structural changes. The focal point of mitosis is the behavior of the chromosomes,and it is this that we will deal with ﬁrst.\\nSTAGES OF MITOSIS\\nMitosis is designed to produce two progeny cells each containing an identical set of theprogenitor cell ’s chromosomes. To achieve this, the chromosomes execute a precisely chore-\\nographed sequence of movements that were ﬁrst described well over a century ago. Clas-\\nsically, mitosis is divided into ﬁve stages, each of which is characterized by changes in the\\nappearance of the chromosomes and their organization with respect to a cellular structure,called the mitotic spindle, that is responsible for their segregation. The stages of mitosis\\nare shown diagramatically in Figure 19.1.\\nOne copy of our genome is encoded in 23 chromosomes and is made up of 3 ×10\\n9\\nbase pairs. Human cells contain 46 chromosomes, 23 inherited from each parent. This state,in which each cell has two complete sets of chromosomes, is called diploid. For simplicity\\n(a)\\n(g)\\n(f)\\n(e)(d)(c)(b)\\nInterphase\\nCytokinesis\\nTelophaseProphase\\nPrometaphase\\nMetaphase\\nspindlecentrosomechromatids\\nkinetochore\\nnuclear\\nenvelopemicrotubule\\nAnaphase\\nFigure 19.1. Stages of mitosis. One pair of chromosomes is shown, one chromosome originating\\nfrom the father (light green kinetochores), and the matching one from the mother (dark green\\nkinetochores).', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='70a2bede-7f68-43dc-ab58-59f46dace9e7', embedding=None, metadata={'page_label': '408', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\nSTAGES OF MITOSIS 403\\nFigure 19.1 shows just one pair of chromosomes, one that came from the father (shown with\\nlight green kinetochores), and the matching one from the mother (dark green kinetochores).\\nProphase. Theﬁrst evidence of mitosis in most cells is the compaction of the threads\\nof chromatin that existed through interphase into compact chromosomes that are visible inthe light microscope. As the chromosomes compact, each can be seen to be paired structurescomposed of two chromatids. This is the visible effect of the DNA molecules having been\\nreplicated (page 87) in interphase. Chromosome condensation reduces the chance of longDNA molecules becoming tangled and broken. Each chromosome has a constriction calledthekinetochore, a structure that forms around a region rich in satellite DNA (page 100)\\ncalled the centromere. The kinetochore is the point of attachment of the chromosome to\\nthe spindle. At the same time as the chromosomes are condensing within the nucleus, thecentrosomes, which lie on the cytoplasmic side of the nuclear envelope, begin to separateto establish the mitotic spindle.\\nPrometaphase. At the breakdown of the nuclear envelope, the chromosomes be-\\ncome free to interact with the forming spindle. Microtubule assembly from the centrosomesis random and dynamic. The growing ends of individual microtubules make chance contactwith and are captured by the kinetochores. Because of the random nature of these events,the kinetochores of chromatid pairs are initially associated with different numbers of micro-tubules, and the forces acting upon each chromosome are unbalanced. Initially, therefore,the spindle is highly unstable and chromosomes make frequent excursions toward and awayfrom the poles. Gradually, a balance of forces is established and the chromosomes becomealigned at the equator, with the kinetochores of each member of a chromatid pair orientedtoward opposite poles.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 19.1 Counting Chromosomes\\nThe DNA molecules of an interphase cell can only be observed using an electron microscope\\n(page 8). However, at mitosis, when they condense, individual chromosomes can be made out easilyusing a standard light microscope (page 76). Mitotic cells can then be screened for chromosomeabnormalities such as the presence of three copies of chromosome 21, a condition that causes Downsyndrome, and the “Philadelphia chromosome, ”a rearrangement between chromosomes 9 and 21\\n(which results in a form of leukemia).\\nMetaphase. Metaphase is the most stable period of mitosis. The system can be\\nregarded as being at steady state with the chromosomes lined up rather like athletes at the\\nstart of a race (albeit half facing in one direction and half the other). The metaphase spindleconsists of two major groups of microtubules: those connecting the chromatids to the polesand a second group extending from each pole toward the other. The second group overlapat the spindle equator. If a single chromatid becomes detached from the spindle (Fig. 19.2),it sends a signal that tells the other chromosomes not to start anaphase without it. The otherchromosomes dutifully wait until the missing chromatid reattaches, whereupon all of thechromosomes enter anaphase together.\\nAnaphase. The trigger for the separation of the paired chromatids and the start of\\ntheir journey to the spindle poles is the degradation of the protein cohesin, which acts as the\\nglue holding the pairs of chromatids together. In anaphase A the microtubules holding thechromosomes shorten, pulling the chromosomes to the spindle poles. The chromosomesmove as a “V”with the kinetochores, at which the force for chromosome movement is', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='92be0e87-a367-45fb-ae91-de5bedf28215', embedding=None, metadata={'page_label': '409', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\n404 CELL CYCLE AND CONTROL OF CELL NUMBER\\nforces on\\nchromatids unbalanced\\nstay in\\nmetaphaseSTOPGO\\nstart\\nanaphaseforces on\\nchromatids balanced\\nFigure 19.2. Anaphase only begins when all chromatids are correctly attached to spindle micro-\\ntubules.\\napplied, leading the way. In contrast, in anaphase B the microtubules that overlap at the\\nspindle equator lengthen, extending the distance between the poles. Compared to otherforms of cell motility, the movement of chromosomes at anaphase is extremely slow, lessthan 1 µm per minute.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 19.2 Taxol Stops Mitosis\\nDuring cell division, cytoskeletal microtubules break down and the tubulin monomers reform as the\\nmitotic spindle. Taxol (page 385) stabilizes existing microtubules, making it impossible for the cellto form spindles and therefore preventing cell division. For this reason taxol, usually referred to byits generic name palitaxel, is a valuable anti-cancer drug.\\nTelophase. This stage sees the reversal of many of the events of prophase; the\\nchromosomes decondense, the spindle disassembles, the nuclear envelope reforms, Golgi\\napparatus and endoplasmic reticulum reform, and the nucleolus reappears. Each progenynucleus now contains one complete copy of the genome from the father and one copy fromthe mother. Since in Figure 19.1 we show only one pair of chromosomes, our diagramshows each nucleus containing two chromosomes, one that originated from the father plusone that originated from the mother.\\nCytokinesis. During the last stages of telophase, the cell itself divides. In animal\\ncells, a cleavage furrow made of actin and its motor protein myosin II constricts the middle\\nof the cell; in plants, a structure called the phragmoplast forms at the equator of the spindles\\nwhere it directs the formation of a new cell wall.\\nMEIOSIS AND FERTILIZATION\\nIn sexually reproducing organisms, the germ cells that give rise to the eggs and sperm ariseby a different type of cell division from the somatic cells that make up most of the body.\\nIn regular cell division DNA synthesis is followed by mitosis so that the two progeny end', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ae50df1c-f07f-446e-9320-d20c29fd2d51', embedding=None, metadata={'page_label': '410', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\nMEIOSIS AND FERTILIZATION 405\\nup with an identical set of the original cell ’s chromosomes. To produce germ cells, a single\\nround of DNA synthesis is followed by two nuclear divisions. The result is that an individual\\nsperm or egg cell is haploid; that is, it contains only one copy of each chromosome. Fusion\\nof an egg and sperm at fertilization creates a diploid cell.\\nMeiosis\\nEach meiotic division (known as meiosis I and II ), involves the formation of a meiotic spin-\\ndle and the same sequence of prophase, prometaphase, metaphase, anaphase, and telophase.The steps are illustrated in Figure 19.3. Once again one pair of chromosomes is shown withgreen kinetochores, with the chromosome that came from the father shown with a light greenkinetochore and the matching one from the mother with a dark green kinetochore. We alsoshow a second chromosome, light gray and dark gray kinetochores again indicating paternaland maternal origin. Mitosis is usually a rather brief process, but meiosis is often extended\\nMetaphase IMetaphase IIAnaphase IAnaphase IITelophase I/Cytokinesis I\\nTelophase II/Cytokinesis II\\nFigure 19.3. Stages of meiosis. Two pairs of chromosomes are shown; two paternal chromosomes\\n(light green and light gray kinetochores) and two matching maternal ones (dark green and dark\\ngray kinetochores).', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='783cf583-c97d-41e4-9501-546620231e21', embedding=None, metadata={'page_label': '411', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\n406 CELL CYCLE AND CONTROL OF CELL NUMBER\\nand, in different organisms can last for months or even years. Most of this is occupied by\\na lengthy prophase of meiosis I, prophase I. Prophase I differs from mitotic prophase in\\nthat the paternal and maternal chromosomes (which, because DNA synthesis has occurredin the preceding interphase, are each composed of two chromatids) line up side by side. Asthe cell progresses to metaphase I, the maternal/paternal chromosome pairs line up alongthe metaphase plate. At anaphase I, the homologous pairs separate but, unlike the situationin mitosis, the paired chromatids remain attached and journey together to the pole. The twoprogeny nuclei formed during telophase I almost immediately enter meiosis II. Prophase IIis often so brief as to be undetectable. Metaphase II and anaphase II resemble their mitoticcounterparts, the chromatids ﬁnally separating to give haploid gametes (sperm or eggs).\\nFor each part of the genome, each gamete contains only one copy, derived from either thefather or the mother. Since in Figure 19.3 we show only two pairs of chromosomes, onewith green and one with gray centromeres, the ﬁnal panel in the ﬁgure shows each gamete\\ncontaining two chromosomes.\\nAlthough male and female meiosis in animals follow roughly the same lines, there are\\nsome important distinctions. In males meiosis produces four equal sized haploid cells calledspermatids, each subsequently developing into a spermatozoon. In females, both meioticdivisions are asymmetric, resulting in one large cell that survives, the oocyte, and threesmall cells, called polar bodies, that are discarded.\\nFertilization and Inheritance\\nAt fertilization the sperm cell fuses with the much larger egg cell. The nuclei that originatedin sperm and egg, now called pronuclei, move together and eventually fuse to restore thenormal somatic cell chromosome number. This diploid cell then divides many times bymitosis to generate the multicellular organism.\\nMany of the attributes that make each human being distinct are determined by the\\npattern of genes we inherit. The simplest pattern of inheritance is seen when one form ofa gene fails to make a functional protein. Brown eye color is an example. If you inherit afunctional BEY (brown eye) gene, then you can make brown pigment and your eyes will be\\nbrown. The gene is found on chromosome 15. If you inherit a chromosome from one parentwith a defective BEY gene, and a chromosome from the other parent with a functional BEY\\ngene, then you can make enough of the BEY protein to make brown pigment and your eyeswill be brown. Brown eye color is said to be dominant.\\nIf both of your BEY genes are defective, a different gene, on chromosome 19, becomes\\nimportant. The GEY (green eye) gene codes for a protein that makes green pigment. If you\\nhave a functional BEY gene, then the GEY gene is largely irrelevant, because the brown\\npigment drowns out the more subtle green pigment. However, if you have two nonfunctionalBEY genes and at least one functional GEY gene, you have green eyes. If both BEY genes and\\nboth GEY genes are nonfunctional, you don ’t make brown pigment or green pigment and\\nyour eyes are blue. Green eye color is said to be recessive to brown because an individual\\ncan only have green eyes if they lack any brown eye genes. Similarly, blue eye color isrecessive to brown and green because an individual can only have blue eyes if he or shelacks any functional brown and green eye genes.\\nClearly most of us are not unhappy with green or blue eyes. However, when a more\\nvital protein doesn ’t work, the results are much less benign. Phenylketonuria is a recessive\\ngenetic disease that leads to severe mental retardation (page 294). If an individual inheritsa chromosome from one parent with a defective phenylalanine hydroxylase gene, and a', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='5eb21d57-d5ba-45aa-b56b-14c6e7a2b263', embedding=None, metadata={'page_label': '412', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\nMEIOSIS AND FERTILIZATION 407\\nchromosome from the other parent with a functional phenylalanine hydroxylase gene, then\\nthey can make enough phenylalanine hydroxylase to convert phenylalanine to tyrosineand are perfectly normal. However, when a baby inherits a nonfunctional phenylalaninehydroxylase gene from each parent, then they have the potentially disastrous conditioncalled phenylketonuria (page 294). People with one functional gene and one nonfunctionalone are called carriers. If two carriers of the same defective gene have a child, that child\\nhas a one in four chance of getting a defective gene from both parents and exhibiting therecessive condition, be it nonbrown eyes or phenylketonuria.\\n✎\\n✍\\x0c\\n✌Medical\\nRelevance\\n19.1Why Marrying Your Cousin Is Dangerous\\nMarriage between ﬁrst cousins is illegal in 30 states of the United States. The scienti ﬁc\\nrationale behind these laws is that cousin marriage provides an easy mechanism\\nfor a single carrier of a recessive genetic disease to generate greatgrandchildren with the disease.\\nThe accompanying ﬁgure shows how this could happen in an imaginary family tree. Harry and\\nIngrid are ﬁrst cousins: they share a grandmother, Alice, who was a carrier for phenylketonuria.\\nWe indicate this in the letters below Alice ’s name: P indicates the gene coding for a functional\\nphenylalanine hydroxylase while p indicates the gene coding for a defective protein. Both Alice ’s\\nchildren, Constance and David, inherited her defective gene and were therefore carriers. Of theirfour children, Harry, Ingrid, and Jill inherited the defective gene and are carriers. The childrenof Harry and Ingrid therefore each had a one-in-four chance of inheriting one defective genefrom Harry plus one defective gene from Ingrid. One of their children, Maria, does so and hasphenylketonuria.\\nIn the case we have drawn we know that Constance, David, Harry, and Ingrid are carriers.\\nIf we did not know this, what is the probability that a child of ﬁrst-cousin parents will have\\nphenylketonuria, given that one shared grandparent is a carrier? This is easy to calculate: each childof a carrier has a 1-in-2 chance of inheriting the defective gene, while each child of two carriers hasa 1-in-4 chance of having the disease. The overall probability is therefore\\n1/2×1/2×1/2×1/2×1/4or\\nunder 1.6% per child. The extent to which this risk is signi ﬁcant enough to require legislation, given\\nthe low frequency of cousin marriage in those countries and states where it is allowed, is debatable.\\nAlice\\nPp\\nConstance\\nPpEdward\\nPP\\nGeoff\\nPPHarry\\nPpFrances\\nPPDavid\\nPp\\nIngrid\\nPpJill\\nPpBrad\\nPP\\nKyle\\nPPLucia\\nPpMaria\\npp', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='82a084c9-fa02-4d09-b668-89cf4715ab19', embedding=None, metadata={'page_label': '413', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\n408 CELL CYCLE AND CONTROL OF CELL NUMBER\\nDominant Genetic Disease\\nMore unusually, a defective gene may cause a problem even if the individual has one good\\ncopy, that is, the defective gene is dominant over the normal one. One example is familialCreutzfeldt –Jacob disease. As we described earlier (page 207), the brain damage that occurs\\nin Creutzfeldt –Jacob disease results when an alternatively folded form of PrP protein called\\nPrP\\nsccauses all the PrP protein around it to fold up in the alternative way. Sufferers from\\nfamilial Creutzfeldt –Jacob disease have a mutant form of the PrP gene that generates protein\\nthat spontaneously folds into the PrPscform. The PrPscthen triggers the normal protein,\\ngenerated by the second, normal copy of the gene, to fold into the PrPscshape. Thus one\\ncopy of the defective gene is suf ﬁcient to cause the disease.\\nCrossing Over and Linkage\\nProphase I lasts such a long time because the chromosomes deliberately tie themselvesin knots and then untangle (using topoisomerase II, page 74). Figure 19.4 shows whathappens. At the top we see the paternal and maternal chromosomes, each composed oftwo chromatids, lined up side by side. As before we show the chromosome that originatedfrom the father in light green and the maternal one in dark green. During prophase 1 thechromosomes are cut and resealed at points called chiasmata (singular chiasma ) so that\\nlengths of paternal chromosome are transferred to a maternal one and vice versa. The restof meiosis I proceeds, followed by meiosis II, and the end result is that some gametescontain chromosomes that are neither completely paternal nor completely maternal but arearecombination of the two.\\nThe biological advantage of sexual reproduction is that it allows organisms to possess a\\nrandom selection of the genes from their ancestors. Those individuals with a complement ofgenes that makes them better suited to their environment tend to do better, allowing evolutionby natural selection of the individuals posessing the better genes. Without crossing overthis could not happen: Those genes that are located on the same chromosome would remainlinked down the generations, greatly reducing the number of gene permutations possible\\nat each generation. Crossing over allows a child to inherit, for example, his grandmother ’s\\ngreen eyes without also inheriting her defective sodium channel gene (page 331), althoughboth genes are on chromosome 19. Even with crossing over, genes on the same chromosomeare inherited together more than they would be if they were on different chromosoomes.The closer the genes, the less likely it is that a chisma will form between them, and thereforethe greater the probability that they will be inherited together. This phenomenon is used tohelp identify the genes responsible for speci ﬁc diseases such as cystic ﬁbrosis (page 426).\\nCONTROL OF THE CELL DIVISION CYCLE\\nAlthough mitosis is the most dramatic period of the cell cycle, it is the culmination ofa sequence of biochemical and structural events that occur during interphase. The mostimportant of these is the duplication of the genome in S phase (the“S”stands for DNA\\nsynthesis). In this cell cycle shorthand, mitosis is referred to as M phase. S phase and M\\nphase must occur (a) in the correct order and (b) only once per cell cycle. To make surethis is the case, S phase and M phase do not follow immediately after one another but areseparated by “gaps”that allow the cell to check that everything is in order before going', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='0a52255d-aa5e-46f4-9462-d0d031afe255', embedding=None, metadata={'page_label': '414', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\nCONTROL OF THE CELL DIVISION CYCLE 409\\nchiasmaduring prophase I\\ncompletion of meiosis I\\nmeiosis II follows\\nFigure 19.4. Chiasmata allow crossing over of genetic material during prophase I of meiosis.\\non to the next stage. The gap between M phase and S phase is called G1 (gap 1), and the\\ngap between S phase and M phase is called G2 (gap 2). These four phases, G1, S, G2, and\\nM make up the classic cell cycle “clock ”(Fig. 19.5). Nondividing or quiescent cells are\\nsaid to be in the G0 (gap 0) phase of the cell cycle. Cells can only enter G0 from G1. G0\\ncells can remain viable for months or even years, and most of the cells in the human body\\nare in fact in this nondividing state. Cells are said to be terminally differentiated if they areunable to return to the cell division cycle. Nerve cells are one such example. In contrast', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='39c818d6-39bb-479b-87a6-55ca44f9b050', embedding=None, metadata={'page_label': '415', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\n410 CELL CYCLE AND CONTROL OF CELL NUMBER\\nnon dividing cell\\nG0\\nnucleus\\nCell divisioncell mass is\\ndoubledDNA replication\\nS Phase\\nG2 G1\\nMitosisamount of DNA in\\nnucleus is doubled\\ncell\\nFigure 19.5. Cell division cycle.\\nother differentiated cells, such as glial cells (page 13), can return to the cycle if they receive\\nthe correct signals from their neighbors.\\nFor dividing cells, two points in the cell cycle are particularly critical. The ﬁrst is at\\nthe G1/S boundary at which point the cell is committed to DNA replication. The second isat the G2/M transition when it is committed to mitosis. These are the major control pointsof the cell cycle, and before crossing them the cell must be sure that conditions are suchthat S phase and M phase can be executed successfully. At G1/S, the cell must decidewhether it is big enough and whether nutritional conditions are appropriate to begin thecrucial process of replicating its genome. In G2 the primary concern is that its DNA is inperfect condition before entering mitosis. There are sensitive mechanisms for detecting thepresence of unreplicated or damaged DNA, and cells will not commit themselves to mitosisuntil any defects have been attended to.\\nMolecular Regulation of the G2/M (Interphase/Mitosis)\\nCell Cycle Control Point\\nAs cells enter mitosis, they undergo a remarkable sequence of structural changes. Not\\nonly do the chromosomes condense as we have seen above but, in addition, the nuclearenvelope breaks down, the nucleolus disperses, the membranes of the Golgi apparatusand endoplasmic reticulum fragment, and the cytoskeleton undergoes remodeling to forma completely new cellular structure, the mitotic spindle. How are so many and diverse', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='506db132-e933-462e-8e18-40ba1add6364', embedding=None, metadata={'page_label': '416', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\nCONTROL OF THE CELL DIVISION CYCLE 411\\nSG 1\\n(a)\\n(b)\\n(c)\\n(d)+\\nSG 2+\\nG1 G2\\nG2 M+\\n+\\nFigure 19.6. Cell fusion experiments reveal how constituents of the cytoplasm control the behavior\\nof the nucleus. Mitotic cells are identiﬁed by the presence of the mitotic spindle. Nuclei in S, G1,\\nand G2 phases are represented by different color shades.\\nstructural changes coordinated in a narrow window of the cell cycle, often lasting only a\\nfew minutes? During the 1960s some remarkable experiments started the hunt that led tothe identi ﬁcation of a cytoplasmic master switch that activated all these changes. These\\ninvolved fusing together cultured mammalian cells at different stages of the cell cycle(Fig. 19.6). These experiments demonstrated that cell cycle progress was under the controlof cytoplasmic factors and that cells had to complete certain cell cycle steps before enteringsubsequent ones. In particular, row ( d) shows how fusion of an M-phase cell with a G2\\ncell causes the G2 nucleus to enter mitosis. Similar results were obtained when cytoplasmfrom unfertilized Xenopus oocytes (which are naturally arrested in meiotic M phase) was\\ninjected into interphase oocytes (Fig. 19.7). The implication of these experiments was thatthe cytoplasm of dividing cells contained a factor capable of controlling entry into mitosis.This activity was given the name M-phase promoting factor, orMPF. Deﬁning MPF in\\nbiochemical terms took almost 20 years and culminated in the award of the 2001 Nobelprize in physiology or medicine to Paul Nurse, Tim Hunt, and Leland Hartwell for theirpioneering work on the identi ﬁcation of the molecules that drive the cell division cycle.\\nRemarkably, given the variety of the changes it induces, MPF is a complex of only two\\nproteins. These are an enzyme called cyclin-dependent kinase 1 (Cdk1) and its regulatory\\nsubunit called cyclin B. Cdk1 can act as a cell cycle switch because it is a protein kinase, an', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a9dbbb62-462c-4966-a3ee-edd83295dc91', embedding=None, metadata={'page_label': '417', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\n412 CELL CYCLE AND CONTROL OF CELL NUMBER\\ncell in meiosis interphase cell enters meiosistransfer some cytoplasm\\nto a cell in interphase\\nFigure 19.7. Injection of cytosol from a mitotic cell into an interphase one triggers mitosis in the\\ninterphase nucleus.\\nenzyme that can modify the function of other proteins by phosphorylation. The substrates\\nof Cdk1 are proteins associated with the chromosomes, the nuclear envelope, nucleolus,centrosomes, and so on and phosphorylation of such proteins sets in train the structuralchanges that culminate in mitosis. However, this activity must be activated only at theG2/M boundary and then must be quickly inactivated if the cell is not to undergo multiplerounds of mitosis. Figure 19.8 shows how Cdk1 activity is regulated through the cell cycle.Without cyclin B Cdk1 is very slow at phosphorylating its target but, to ensure that it remainscompletely inactive through interphase, it is itself phosphorylated by another protein kinasecalled Wee1 (because mutants, with unphosphorylated Cdk1, divide too early, while theyare still “wee”—Scottish for small!). Wee1 adds two phosphates to Cdk1, one on a threonine\\nresidue at amino acid number 14, the other on a tyrosine residue at amino acid number 15.These two amino acids lie within the ATP-binding site of Cdk1, and the presence of twophosphates prevents ATP binding, the ﬁrst step of the phosphorylation reaction. Immediately\\nfollowing mitosis, the concentration of cyclin B in the cell is vanishingly small. Cyclin B ismade steadily so its concentration gradually increases, and so more and more Cdk1 comesto have cyclin B attached (Fig. 19.8 b). However, this heterodimer is inactive because it is\\nphosphorylated on T14 and Y15. At the G2/M boundary the phosphates are removed by asecond enzyme called Cdc25. MPF is now active and can phosphorylate its target proteins.\\nOnce mitosis is initiated, it is essential that Cdk1 be deactivated for the reasons outlinedabove. This is achieved by the destruction of cyclin B and the rephosphorylation of Cdk1by Wee1.\\nWhat About the G1/S Control Point?\\nThe decision to begin DNA synthesis is arguably an even more critical decision than entryinto M phase because once a cell begins, it is committed to eventually dividing. (Cells thatreplicate their DNA but then fail to undergo mitosis in a reasonable time commit suicide byapoptosis (page 415) rather than hang around in a polyploid state). Cell fusion experiments\\nagain indicated that factors in the cytoplasm controlled advance to the next stage in the cell', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='cbfeb092-c1e6-43b2-849f-f85b819186da', embedding=None, metadata={'page_label': '418', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\nCONTROL OF THE CELL DIVISION CYCLE 413\\nP\\nPP\\nPT14 T14\\nactive\\ncatalytic\\nsiteY15 Y15\\nCdk1\\nCdk1Cdk1\\ncyclin B\\nsynthesis\\nof cyclin Binactive inactive active kinaseActive Active\\ncyclin B\\nCdk1\\nInactive Inactive\\nCell cycle stageProtein concentration\\ninactive\\ncyclin B\\ncyclin B\\ndestroyedT14\\nY15 P\\nPT14\\nY15\\nCdk1(a)\\n(b)G1 S G2 M G1 S G2 M\\nFigure 19.8. Control of Cdk1 by cyclin B and by phosphorylation.\\ncycle. In particular, fusion of an S-phase cell with a G1 cell caused the G1 nucleus to begin\\nDNA synthesis (Fig. 19.6 a). We now know that entry into S phase is regulated not by one\\nbut by three cyclin-dependent kinases called Cdk2, Cdk4, andCdk6, which, like Cdk1,\\nare active at high ef ﬁciency only when coupled to a cyclin partner, in this case cyclin D\\nfor Cdks 4 and 6 and cyclin E for Cdk2. The function of Cdk4 is most clearly understood\\nand is illustrated in Figure 19.9. DNA replication requires a speci ﬁc set of proteins not\\nused in quiescent cells (page 87), and transcription of the genes coding for these proteinsis activated by a transcription factor called E2F-1. In quiescent cells E2F-1 is prevented\\nfrom activating transcription by being bound to RB, the product of a gene that is mutated\\nin the cancer retinoblastoma. The main function of Cdk4 seems to be to phosphorylate\\nRB, preventing its association with E2F-1, which is therefore freed to bind to enhancersequences, activate transcription, and hence trigger DNA synthesis. Growth factors triggercell division in part through activating transcription of the cyclin-D gene and hence allowing', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='feb92a65-a566-4e1d-970b-a8dc2be1f521', embedding=None, metadata={'page_label': '419', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\ncell-cell\\ncontactDNA\\ndamageUV\\nactivation of\\nDNA repair\\ncyclin D synthesis\\nDNA replicationsynthesis of BAX\\nsynthesis of\\nCKIsMAP KinaseRasGrowth\\nfactor\\nreceptors\\nCdk 4 : cyclin Dapoptosis\\np53 concentration risesPKBP P\\nRB\\nE2F\\n−1\\nE2F\\n−1RB\\nP−−\\nFigure 19.9. Retinoblastoma protein RB sequesters E2F-1, the critical transcription factor controlling entry into S phase.\\n414', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='476381df-ff0f-4f3e-9616-5e4cd3ae641a', embedding=None, metadata={'page_label': '420', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\nAPOPTOSIS 415\\nCdk4 to operate. Once S phase has begun, RB is dephosphorylated to prevent further rounds\\nof DNA replication.\\nThe G1 Cdks have an extra level of regulation above that described for Cdk1 –cyclin\\nB because they are inhibited by an additional class of proteins called CKIs forcyclin-\\ndependent kinase inhibitor proteins. There are many CKIs mediating a number of path-\\nways all with the same outcome: stopping cell division. One of these pathways mediatescontact inhibition. Cells in a culture dish, or at the edge of a wound, divide until they\\ntouch each other. When they contact neighboring cells, they stop dividing because contactcauses production of two CKIs called p16\\nINK4aand p27KIP1. These inhibit the G1 Cdks and\\ntherefore prevent DNA synthesis. Loss of contact inhibition is one of the ﬁrst changes seen\\nin the transformation of normal cells into cancer cells.\\nAnother CKI called p21CIP1is produced in response to the transcription factor p53.\\np53 is produced all the time in cells but is broken down rapidly, so its concentration isusually low. However, if the DNA is damaged, for example, by ultraviolet light (page 94),the destruction of p53 is halted. As p53 concentrations increase, a number of processesare activated. The systems that repair DNA are activated (page 94). p21\\nCIP1is produced,\\ninhibiting the G1 CDKs and therefore preventing replication of the defective DNA. Lastly,if the DNA cannot be repaired within a set time (usually a few hours), the cell is triggeredto self-destruct in the process called apoptosis.\\nAPOPTOSIS\\nAn adult human is made up of about 30 trillion (3 ×1013) cells, all of which originate from\\na single fertilized egg. If this ﬁrst cell divides into 2, the 2 progeny cells into 4, and so\\non, it would take only about 45 rounds of division to produce the number of cells requiredto make an adult human. In fact, cell division occurs constantly through our lifetimes,such that we generate a new complete set of 3 ×10\\n13cells every 2 weeks. The reason that\\nmulticellular organisms do not become in ﬁnitely large is because the proliferation of cells\\nis balanced by cell death. Cells die for two quite different reasons. One is accidental, theresult of mechanical trauma or exposure to some kind of toxic agent, and often referred toasnecrosis. This is the only type of death seen in unicellular organisms. The other type\\nof death is deliberate, the result of an built-in suicide mechanism known as apoptosis or\\nprogrammed cell death. The two types of cell death are quite distinct. In cells that are\\ninjured, ATP concentrations fall so low that the Na\\n+/K+ATPase can no longer operate,\\nand therefore ion concentrations are no longer controled. This causes the cells to swelland then burst. The cell contents then leak out, causing the surrounding tissues to becomeinﬂamed. Cells that die by suicide on the other hand shrink, and their cell contents are\\npackaged into small membrane-bound packets called blebs. The nuclear DNA becomeschopped up into small fragments, each of which becomes enclosed in a portion of thenuclear envelope. The dying cell modi ﬁes its plasma membrane, signaling to macrophages\\n(page 13), which respond by engul ﬁng the blebs and the remaining cell fragments and by\\nsecreting cytokines that inhibit in ﬂammation. The changes that occur during apoptosis are\\nthe result of hydrolysis of cellular proteins by a family of proteases called caspases (short forcysteine-containing, cleaving at aspartate). All the cells of our body contain caspases, butthey are normally locked in an inactive form by an integral inhibitory domain of the protein.Proteolysis cleaves the inhibitory domain off, releasing the active caspases. The advantage', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='98ffbe5c-6c75-412e-a992-c06c7900bee2', embedding=None, metadata={'page_label': '421', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\n416 CELL CYCLE AND CONTROL OF CELL NUMBER\\nDNA\\ndestructioncytochrome c\\nApoptosisinactive\\ncaspase 8Receptor\\ntyrosine\\nkinases\\nCell\\ndivision\\n−Ras\\nsynthesis\\nof CKIsp53 concentration\\nrisesPDeath\\ndomain\\nreceptors\\ninactive\\ncaspase 9DNA\\nDAMAGE\\nProtein\\nKinase\\nBinactive\\neffector\\ncaspases\\nmitochondriainactive\\nBAD:\\n14-3-3\\ncomplexBADsynthesis of\\nBAX\\ncytoskeleton\\ndestructionactive\\neffector\\ncaspases\\nactive\\ncaspase 9active\\ncaspase 8\\n\"eat\"\\nme\\nsignalsP\\nP14-3-3\\nprotein\\nFigure 19.10. Pathways controlling apoptosis.\\nto the cell of this strategy is that no protein sythesis is required to activate the apoptotic\\npathway —all the components are already present. Thus, for example, if a virus infects a\\ncell and takes over all protein synthesis, the cell can still commit suicide and hence preventviral replication. Figure 19.10 summarizes the complex control systems that regulate thedecision to die or survive. Cells activate apoptosis in response to three types of event.\\nInstructed Death: Death Domain Receptors\\nCells are instructed to die when a ligand binds to one of a family of death domain receptors.\\nThis occurs, for example, in the case described above: If a cell is infected by a virus, whiteblood cells recognize viral proteins on the cell surface and activate Fas, a death domain\\nreceptor on the surface of the unlucky cell. On binding its ligand, a death domain receptorcauses caspase 8 to activate. In turn, caspase 8 can hydrolyze and hence activate the effector\\ncaspases that begin the processes of cell destruction.\\nDefault Death: Absence of Growth Factors\\nCells that are not required by the organism die. To make sure that this occurs, death is thedefault option for the cells of a multicellular organism: Only if a cell receives growth factorsfrom other cells will it survive. We have already described the ﬁrst part of this pathway', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='f432fa7c-2b2a-4b87-80ea-1878a272a3f3', embedding=None, metadata={'page_label': '422', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\nAPOPTOSIS 417\\n(page 356). Active receptor tyrosine kinases recruit and in turn activate phosphatidyli-\\nnositol 3-kinase, which generates the highly charged membrane lipid phosphoinositidetrisphosphate. This causes protein kinase B to visit the membrane, where it is itself phos-phorylated and hence activated. Protein kinase B phosphorylates one of a family of bcl-2\\nfamily proteins called bcl-2 associated death promoter, or BAD. Phosphorylated BAD\\nis inactive and is sequestered by binding to a protein called 14-3-3. However, if PKB isever allowed to stop working, BAD loses its phosphates and is then active. Active BADmakes holes in the outer mitochondrial membrane (how it does this is unclear) allowing cy-tochrome cto escape into the cytosol. Although cytochrome cplays a vital role in allowing\\nmitochondria to generate the H\\n+gradient energy currency (page 266), once in the cytosol it\\nis deadly. It activates another caspase, number 9, which then hydrolyzes and hence activatesthe effector caspases.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 19.3 Neurotrophin Trafﬁcking\\nDuring fetal development motoneurons die unless they succeed in growing their axons all the way\\nto an appropriate muscle. Those that do ﬁnd the target are bathed in neurotrophin 3, a growth factor\\nreleased by the muscle cells. Neurotrophin 3 binds to its receptor tyrosine kinase, called Trk C, onthe nerve cell surface. However, the activated kinase then has to activate protein kinase B in the cellbody in order to prevent apoptosis, and the cell body may be many millimeters away. To achievethis, the neurotrophin –receptor complex is endocytosed using the clathrin mechanism (page 229).\\nThe endocytotic vesicles are then transported back to the cell body by dynein (page 390). Once inthe cell body, protein kinase B is activated and apoptosis prevented.\\nThe nerve that contains motoneurons also contains pain receptor axons. Instead of Trk C\\nthese express a related receptor, Trk A, which requires a different neurotrophin (number 1): Trk Acannot be activated by neurotrophin 3. However, both motoneurons and pain receptors express asecond, unrelated receptor called the p75 neurotrophin receptor, which binds both neurotrophin 1and neurotrophin 3. p75 is a death domain receptor. The overall result is that when an axon arrives ata target, it will automatically receive a signal to die. However, if it has arrived at the correct location,it will receive a countermanding signal to survive.\\nThe Sick Are Left to Die: Stress-Activated Apoptosis\\nIf a unicellular organism is damaged, it will try to repair itself since the alternative is the death\\nof the organism. However, if a cell of a multicellular organism is stressed or damaged, itmay be more ef ﬁcient to allow the cell to undergo a quick and polite suicide and replace it by\\ncell division of a healthy neighbor. There are therefore a number of mechanisms that triggerapoptosis in response to cell stress. One mechanism operates directly on mitochondria. Whenmitochondria are stressed, they can spontaneously release cytochrome c:This appears to\\nhappen when porin, a channel of the outer mitochondrial membrane, associates with otherproteins to form a channel of large enough diameter to allow cytochrome cto leak out. This\\nis a major medical problem because it can occur in tissues that are unable to rebuild by celldivision: in hearts during heart attacks and in brain nerve cells during a stroke.\\nA second mechanism operates through the transcription factor p53, which we have\\nalready met in the context of cell cycle control (page 415). p53 concentrations increasewhen DNA is damaged. DNA repair mechanisms are activated, but so is transcription ofthe gene for another bcl-2 family protein called BAX. Like BAD, BAX allows cytochrome', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='bc9a0b65-8aa3-421c-8e0b-384ad9a71130', embedding=None, metadata={'page_label': '423', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\n418 CELL CYCLE AND CONTROL OF CELL NUMBER\\ncto escape from mitochondria, so if the DNA is not repaired in time, BAX concentrations\\nincrease, cytochrome cescapes, and apoptosis results. Without this mechanism, cancers,\\nwhich result from changes in the DNA of our body cells, would be far more abundant.\\n✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁✄\\n✂\\x00\\n✁ Example 19.4 Sunburn, Skin Cancer, and Cell Death\\nWithout knowing it, we have all observed apoptosis in action. All of us at some time or other have\\nbeen out in the sun without proper protection. The ultraviolet (UV) light causes damage to the DNAof the skin cells and activates p53. If the DNA damage is minor, p53 simply arrests the cell cycleuntil the DNA repair machinery has had time to repair the lesions. However, if the damage was moresevere, the skin cells activate the apoptoptic pathway —and the sunburned skin dies and sloughs off.\\nA third mechanism (not shown in Fig. 19.10) operates through a protein kinase called\\np38. Although p38 is related to MAP kinase, it is activated not by growth factors but rather\\nby cell stresses such as swelling, shrinkage, or radiation. The effects of p38 and another\\nstress-activated protein kinase, called JNK, are complex, and we will consider only two.\\np38 phosphorylates p53, which protects it from the rapid breakdown that normally occurs.An increase of p53 concentration then triggers the usual consequences, including synthesisof BAX. p38 also has a direct effect on the founder member of the bcl-2 family, bcl-2itself. Unlike BAD and BAX, bcl-2 cannot trigger cytochrome crelease from mitochondria.\\nIndeed, bcl-2 is antiapoptotic because it can bind to BAD and BAX and prevent them fromreleasing cytochrome c. p38 phosphorylates bcl-2, preventing it from associating with BAD\\nand BAX. When BAD and BAX are freed, they can trigger cytochrome crelease.\\nSince caspases can self-activate by mutual proteolysis, there is a steady slow rate of\\nactivation even in a healthy cell. To prevent this triggering apoptosis, cells produce apoptosis\\ninhibitor proteins, which block caspase action. If the extent of caspase activation exceeds\\nthe capacity of the inhibitory proteins, death results.f\\nIN DEPTH 19.1 A Worm ’s Eye View of Cell Death\\nOur understanding of the mechanisms and importance of cell death has been\\ngreatly in ﬂuenced by work on the nematode worm, Caenorhabditis elegans. This\\ntiny creature (about 1 mm long) is a true animal with nervous, digestive, andmuscular systems but is small enough to be grown in Petri dishes (where it is feda diet of Escherichia coli ). Crucially, every C. elegans worm is made up of a precise\\nnumber of cells. In the newly hatched worm this number is 558, rising to 959somatic cells and a variable number of germ cells in the adult. Because the wormis transparent, it has been possible to trace the lineage of every somatic cell backto the original fertilized egg. This pattern is identical in every worm examined.Strikingly, exactly 131 cells die during the developmental process as the result ofprogrammed cell death.\\nWorkers trying to understand this process isolated ced(=cell death defective)\\nmutants in which various organs contained abnormally high numbers or abnor-mally low numbers of cells because the cell death program was abnormal. Whenthese genes were cloned, it became clear that similar genes were present in otheranimals including humans. For example, ced-3 was found to code for a proteolytic\\nenzyme. Screening the human genome for similar enzymes revealed the family ofapoptotic proteases we now know as caspases.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4e59556f-1e29-463d-9a87-e40b4ab48a1b', embedding=None, metadata={'page_label': '424', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\nSUMMARY 419\\nThe contribution of the simple nematode to biology was recognized by the\\naward of the 2002 Nobel prize in physiology or medicine to the pioneers of nema-\\ntode research, Sydney Brenner, Robert Horvitz, and John Sulston.\\nSUMMARY\\n1.Haploid cells contain only one copy of each chromosome. Diploid cells contain two\\ncopies, one from the organism ’s father, one from the mother.\\n2.The cell cycle comprises S, G1, M, and G2 phases. G2, S, and G1 phases togetherconstitute interphase. DNA is replicated in S phase, so that each chromosome be-comes a pair of identical chromatids. The cell physically divides in mitosis, orM phase.\\n3.Mitosis comprises prophase, prometaphase, metaphase, anaphase, telophase, andcytokinesis. The end result of mitosis is two diploid cells whose chromosome com-plement is the same as that of the original cell before it underwent S phase.\\n4.Meiosis generates haploid germ cells: in vertebrates, eggs and sperm. Like mitosis,it follows an S phase but comprises two cycles of cell division, so that the end resultis four cells whose chromosome complement is only half that of the original cellbefore it underwent S phase.\\n5.During meiosis I homologous chromosomes undergo recombination, a physicalresplicing of homologous chromosomes that allows information on chromosomesoriginating from father and mother to be mixed.\\n6.Recessive genes are usually those that fail to make functional protein. Examplesare the gene for blue eyes and the gene for phenylketonuria. The functional geneis called dominant because an individual need inherit only one copy to be able tomake functional protein.\\n7.In a few cases a defective gene, for example, the gene for familial Creutzfeldt –Jacob\\ndisease, is dominant over a normal one.\\n8.Cells enter mitosis when cyclin-dependent kinase 1 is active. This in turn requiresthat cyclin B be present in high enough concentration and that cyclin-dependentkinase 1 be dephosphorylated by Cdc25. Once mitosis is initiated, cyclin-dependentkinase 1 is rapidly turned off through phosphorylation by WEE1 in parallel withthe proteolytic destruction of cyclin B.\\n9.The entry of cells into S phase is a more complex decision involving cyclin-dependent kinases 2, 4, and 6. The main effect of active cyclin-dependent kinase 4 isto phosphorylate RB, causing it to release the transcription factor E2F-1 and henceallowing the synthesis of proteins required for DNA synthesis. Important compo-nents of the decision are a raised concentration of cyclin D as a result of MAP kinaseactivity and a low concentration of CKIs. Cell –cell contact upregulates CKIs so that\\nwhen an organ has ﬁlled the space available it stops growing.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d385857e-c9ba-40b3-9379-96e9a7e93b4d', embedding=None, metadata={'page_label': '425', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\n420 CELL CYCLE AND CONTROL OF CELL NUMBER\\n10.p53 is continually produced in cells but is as quickly destroyed. An increase in the\\nconcentration of p53 follows DNA damage or other cell stress and has three maineffects: (a) activation of DNA repair mechanisms, (b) synthesis of CKIs, preventingcell division, and (c) activation of apoptosis.\\n11.In contrast to necrosis, which causes in ﬂammation, apoptosis is a regulated\\nmechanism of cell suicide that has little effect on the surrounding tissue. The ﬁnal\\neffectors of apoptosis are a family of proteases called caspases.\\n12.Apoptosis can be triggered in three ways: (a) binding of ligand to death domainreceptors, (b) denial of growth factors, and (c) cell stress.\\nFURTHER READING\\nHyams, J. S. 2002. The cell cycle and mitosis. Biol. Sci. Rev. 14: 37 –41.\\nJacobson, M. D., Weil, M., and Raff, M. C. 1997. Programmed cell death in animal development.\\nCell 88: 347 –354.\\nMichael D., Jacobson, M. D., and McCarthy, N. (eds.) 2002. Apoptosis: The Molecular Biology of\\nProgrammed Cell Death. Oxford: Oxford University Press.\\nMitchison T. J., and Salmon E. D. 2001. Mitosis: A history of division. Nature Cell Biol. 3:\\nE17–E21.\\nNicklas R. B. 1997. How cells get the right chromosomes. Science 275: 632 –637.\\nSharp, D. J., Rogers, G. C., and Scholey, J. M. 2000. Microtubule motors in mitosis. Nature 407:\\n41–47.\\nVaux D. L., and Korsmeyer, S. J. 1999. Cell death in development. Cell Jan 96: 245 –254.\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.Mitosis proceeds in the order\\nA. cytokinesis, prophase, prometaphase, telophase, metaphase, and anaphase.\\nB. telophase, anaphase, prophase, prometaphase, metaphase, and cytokinesis.C. prophase, prometaphase, metaphase, anaphase, telophase, and cytokinesis.D. prophase, prometaphase, metaphase, cytokinesis, anaphase, and telophase.E. prophase, prometaphase, cytokinesis, metaphase, anaphase, and telophase.\\n2.Paired chromatids separate and begin to move toward the spindle poles in mitotic\\nA. prophase.B. prometaphase.C. metaphase.D. anaphase.E. telophase.\\n3.Meiosis differs from mitosis in that\\nA. the centrioles do not separate during the ﬁrst meiotic division.\\nB. meiotic divisions are always asymmetric.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1451f544-da84-4967-9aa6-aaadb8cacb04', embedding=None, metadata={'page_label': '426', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\nANSWERS TO REVIEW QUESTIONS 421\\nC. meiosis generates cells with half as much DNA as the progeny of mitotic division have.\\nD. movement of chromosomes toward the spindles is slower.E. All of the above.\\n4.Cdk1 can only be fully active when\\nA. it is phosphorylated on threonine 14.B. it is phosphorylated on tyrosine 15.C. it is bound to cyclin A.D. it is dephosphorylated by Cdc25.E. All of the above.\\n5.The main target of cdk4 is\\nA. RB.B. E2F-1.C. P53.D. nuclear lamins.E. Wee1.\\n6.Caspases are activated by\\nA. phosphorylation.B. dimerization.C. trimerization.D. proteolysis.E. binding to bcl-2 family members.\\n7.Phenylketonuria is said to be a recessive inherited disease because\\nA. the fraction of babies with the disease falls in successive generations.B. the disease is not observed in individuals with one mutant gene and one normal one.C. phenylketonuria accumulates at the back of the foot.D. myelin recedes from axons, causing failure of action potential transmission.E. the condition only appears following meiotic recombination.\\nANSWERS TO REVIEW QUESTIONS\\n1.C.Mitosis proceeds in the order prophase, prometaphase, metaphase, anaphase, telophase, and\\ncytokinesis.\\n2.D.Anaphase begins when the protein cohesin, the glue holding the pairs of chromatids together,\\nis degraded, allowing the chromosomes to begin traveling toward the spindle poles.\\n3.C.Like mitosis, meiosis is preceded by one round of DNA synthesis but generates four progeny\\ncells rather than two. Concerning the other answers: (A) The centrioles always separate in celldivision, becoming the spindle poles on either side of the nucleus. If you answered answer A youmay have been thinking of the fact that in the ﬁrst meiotic division the centromeres of paired\\nchromatids do not separate, rather, homologous pairs of chromosomes separate. (B) Althoughmeiosis in females is asymmetric, producing one large egg cell and three small polar bodies, inmales meiosis is symmetrical, producing four equally sized spermatids. (D) Meiosis often takesmuch longer than mitosis, but this is because of a lengthy prophase I rather than any change in theduration of anaphase.\\n4.D.For Cdk1 to be fully active, it must be dephosphorylated (not phosphorylated) on threonine 14\\nand tyrosine 15 by Cdc25 and must be bound to cyclin B (not cyclin A, which does play a role inthe cell division cycle but which is not described in this book).', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='101865de-2a07-4aa3-8049-7d764820c10d', embedding=None, metadata={'page_label': '427', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-19 WY001-Bolsover-v2.cls September 1, 2003 10:28\\n422 CELL CYCLE AND CONTROL OF CELL NUMBER\\n5.A.Cdk4 phosphorylates RB, which is then unable to sequester E2F-1. None of the others are\\ntargets of Cdk4, although nuclear lamin is certainly a target of Cdk1.\\n6.D.As generated at the ribosome, caspases are locked into an inactive form by an integral inhibitory\\ndomain. Proteolysis removes this inhibitory domain.\\n7.B.Recessive diseases are so called because the defective gene is overruled by the dominant normal\\ngene if the latter is present.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='236134ff-9169-4824-a82c-dce09b38d616', embedding=None, metadata={'page_label': '428', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-20 WY001-Bolsover-v2.cls September 1, 2003 10:31\\n20\\nCASE STUDY: CYSTIC FIBROSIS\\nINTRODUCTION\\nIn the ﬁnal Chapter of the book we describe the biochemical basis of the disease cystic\\nﬁbrosis (CF), the search for the CFgene, the prospects for CF gene therapy, and prenatal\\ndiagnosis. We have selected this disease as a case study to show how the combined efforts ofbiochemistry, genetics, molecular and cell biology, and physiology were needed to ﬁnd thecause of CF and the function of the protein encoded by the CFgene. The basic principles of\\nmany of the techniques used in achieving this knowledge are described in Chapters 1–19.\\nCYSTIC FIBROSIS IS A SEVERE GENETIC DISEASE\\nAmong Caucasians (white, non-Jewish) about 1 child in 2500 is born with CF. Inheritanceis simple: when both parents are carriers, there is a 1 in 4 chance that a pregnancy willresult in a child with the disease. The disease is a distressing one. Most of its symptomsarise from faults in the way the body moves liquids, leading in particular to a build up ofinadequately hydrated, sticky mucus in various parts of the body. In the lungs this leadsto difﬁculty in breathing, a persistent cough, and a greatly increased risk of infection. Abacterium known as Pseudomonas aerigunosa thrives in the lung mucus and is particularly\\nrecalcitrant to antibiotic treatment. The pancreas—which provides a digestive secretionthat ﬂows to the intestine—is also affected and may be badly damaged (which explains\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n423', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b5345cad-4508-4170-9d5f-f222c64c80dd', embedding=None, metadata={'page_label': '429', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-20 WY001-Bolsover-v2.cls September 1, 2003 10:31\\n424 CASE STUDY: CYSTIC FIBROSIS\\nthe disease ’s full name, cystic ﬁbrosis of the pancreas). Often, there are digestive problems\\nbecause the damaged pancreas is not producing enzymes. The reproductive system is also\\nharmed and most males who survive to adolescence are infertile. Even though there areso many varied symptoms, the simple Mendelian pattern of inheritance led researchers tobelieve that the cause was an abnormality or absence of a single protein.\\nUntil recently babies with CF did not survive to their ﬁrst birthday. Today, the life ex-\\npectancy is 22 years. This remarkable improvement is due to intensive therapy designed to re-verse individual symptoms. Digestive enzymes are taken by mouth to replace those proteinsthe pancreas fails to produce. Physiotherapy —helping patients to cough up the mucus in their\\nlungs by slapping their backs —reduces the severity of the lung disease. Yet, however remark-\\nable the increase in survival, each life that extends only to early adulthood is still a tragedy.\\nTHE FUNDAMENTAL LESION IN CYSTIC FIBROSIS LIES\\nIN CHLORIDE TRANSPORT\\nAccording to the Almanac of Children’s Songs and Games from Switzerland, “the child will\\nsoon die whose brow tastes salty when kissed. ”Cystic ﬁbrosis has long been recognized as a\\nfatal disease of children. The link between a salty brow and a disease known for its effects on\\nthe pancreas and lungs was not, however, made until 1951. During a heat wave in New Yorkthat summer, Paul Di Sant ’Agnese and co-workers noticed that babies with cystic ﬁbrosis\\nwere more likely than others to suffer from heat prostration. This prompted them to test theirsweat, and they found that it contained much more salt than the sweat of unaffected babies.\\nSweat glands have two regions that perform different jobs (Fig. 20.1). The secretory\\nregion deep in the skin produces a ﬂuid that has an ionic composition similar to that of\\nextracellular medium; that is, it is rich in sodium and chloride. If the sweat glands weresimply to pour this liquid onto the surface of the skin, they would do a good job of cooling,but the body would lose large amounts of sodium and chloride. A reabsorptive region closerto the surface removes ions from the sweat, leaving the water (plus a small amount ofsodium chloride) to ﬂow out of the end of the gland. CF patients produce normal volumes\\nnormal sweat gland sweat gland of CF patientNa+\\nCl−\\nCl−\\nNa+\\nH2OCl−\\nNa+\\nH2OH2O H2Owatery\\nsweatsaltysweat\\nNa+    Cl−\\nreabsorbtive\\nregion\\nsecretory\\nregion\\nFigure 20.1. Sodium, chloride, and water transport in the sweat gland.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='686bdc70-06a6-4932-bfec-49e2a7a1e6f0', embedding=None, metadata={'page_label': '430', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-20 WY001-Bolsover-v2.cls September 1, 2003 10:31\\nHOMING IN ON THE CFGENE 425\\nvoltmeter\\nInput Common-30 mVvoltmeter\\nInput Common-70 mV\\nnormal sweat gland sweat gland of CF patient\\nFigure 20.2. Measuring the transepithelial voltage of human sweat glands in situ. The voltmeter\\nmeasures the voltage of the input terminal with respect to the common terminal. In this case\\na reading of −30 mV indicates that the droplet of sweat has a voltage 30 mV negative to the\\nextracellular medium.\\nof sweat, but this contains lots of sodium and chloride, implying that the secretory region\\nis working ﬁne but that the reabsorptive region has failed. The pathways by which sodium\\nand chloride ions are removed are distinct. Which one has failed in CF? A simple electricaltest gave the answer. A normal sweat gland has a small voltage difference across the glandepithelium (Fig. 20.2). In CF patients the inside of the gland is much more negative thanusual. This result tells us at once that it is the chloride transport (and not sodium transport)that has failed. In CF sweat glands the reabsorptive region has the transport systems to movesodium, but the chloride ions remain behind in the sweat, giving it a negative voltage. Thesodium transport system cannot continue to move sodium ions out of the sweat in the faceof this larger electrical force pulling them in, so sodium movement stops too, and sodiumchloride is lost in the sweat, which then tastes salty. Every one of the symptoms sufferedby CF patients is caused by a failure of chloride transport.\\nHOMING IN ON THE CFGENE\\nThe ﬁrst family studies were carried out more than 50 years ago, and family pedigrees\\nshowing CF as a classic case of recessive inheritance were published in 1946. However,there was no obvious way to identify the millions of people who carry a single copy of thedefective CFgene. There were many attempts to ﬁnd a test for CF. For a time it seemed that\\na simple stain might do the job; but this was abandoned. Other tests were more eccentric. Forexample, there was a claim that an extract from the blood of CF patients, and perhaps evenfrom that of their parents, who must be carriers, slowed the beating of the cilia of oysters.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='dfa75944-e76f-4d5c-b5d2-1bc72095be02', embedding=None, metadata={'page_label': '431', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-20 WY001-Bolsover-v2.cls September 1, 2003 10:31\\n426 CASE STUDY: CYSTIC FIBROSIS\\nMost other claims that particular gene products were peculiar to CF carriers also\\nfailed to stand up or proved to be symptoms rather than cause. For instance, trypsinogen,\\na precursor of the digestive enzyme trypsin, is found in ﬂuid bathing fetuses homozygous\\nfor CF, and not in others —but this proves to be because the pancreas is already dying at\\nthis time. In the 1980s there was hope that this result could be used to diagnose affectedpregnancies, but the approach was quickly overtaken by developments in studying thegene itself. As for most inborn diseases, the big problem was that nobody had any real ideawhat the faulty gene did; looking for a defect in an unknown protein is a task that requireshard work and good ideas. After much time-consuming work, in 1985 the research groupled by Lap-Chee Tsui published data indicating that the gene for CF lay on chromosome 7.Researchers then used linkage measurements (page 408) to try to pinpoint the region ofchromosome 7 containing the CFgene. An international scienti ﬁc collaboration analyzed\\nDNA from 200 families. The results indicated that the CFgene must lie between two\\nmarker regions on chromosome 7, named metand D7S8. The distance between the markers\\nwas 2 million base pairs. It was now possible to think about isolating the CFgene.\\nCLONING THE GENE FOR CF\\nSheer hard work won the day when it came to cloning the CFgene. The scientists knew\\nthat the gene was on chromosome 7, somewhere between the metand D7S8 markers. Their\\nstrategy was to “walk ”along the chromosome (page 152), away from each marker, and\\nhopefully approach the CFgene from either side. In the 1980s, the only genomic vectors\\navailable were cosmids (insert size 40,000 bp) and those based on the bacteriophage lambda(insert size 20,000 bp) (page 140). So each genomic clone covered only a tiny part ofchromosome 7. Clever tricks had to be devised to speed up the process, and ﬁnally a series\\nof cosmid clones, thought to possibly contain regions of the CFgene, were isolated. But\\nhow could they tell which clones were the right ones?\\nSince CF affects so many organs, the gene is likely to be a critical one conserved\\nthrough mammalian evolution. A zoo blot (page 146) was therefore used in the hope ofidentifying the gene. In turn, each of the human cosmid clones was radiolabeled and used toprobe a zoo blot. Researchers found three clones that hybridized to the DNA of all the testmammals. The next step was to check whether these clones contained sequences that codefor mRNAs expressed in normal tissues that are affected by CF. Each clone was used toprobe a northern blot (page 148) of sweat gland mRNA. One sequence passed the test —with\\nluck, it would contain part of the CFgene. With hindsight we now know that the scientists ’\\nluck had almost run out: The one positive sequence did contain part of the gene, but only atiny part —113 bp, less than 1% of the total gene.\\nThings moved swiftly from then on. The 113-bp section was used to screen cDNA\\nlibraries (page 134) made from sweat gland mRNA. This gave scientists cDNA for the entireCFmRNA, which could then be sequenced. Meanwhile, the cDNA was used to screen a\\ngenomic library to ﬁnd the rest of the CFgene. The gene is 220,000 bp long and has 24 exons.\\nTHE CFTR GENE CODES FOR A CHLORIDE ION CHANNEL\\nOnce CFcDNA had been sequenced, it could be read, revealing that CFcodes for a\\nprotein of 1480 amino acids. Although we knew that CF was caused by a defect in chloride', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='8ed46dd9-af1b-4776-9eba-a1d500975b9f', embedding=None, metadata={'page_label': '432', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-20 WY001-Bolsover-v2.cls September 1, 2003 10:31\\nGENE THERAPY FOR CF 427\\nregulatory\\ndomain\\nPhenylalanine\\nat position 508F\\nFigure 20.3. The CFTR protein forms a chloride ion channel in the plasma membrane of epithelial\\ncells.\\nion transport, it was not immediately clear that the product of the CFgene was itself\\na chloride channel. Although hydropathy plots (page 204) indicated that the CF protein\\nwas a transmembrane protein, the investigators had to consider the possibility that it was,for example, a cell surface receptor whose activation triggered expression of the proteinsmediating chloride transport. The protein was therefore given the catch-all name “cystic\\nﬁbrosis transmembrane regulator ”(CFTR) and its gene was renamed as the CFTR gene.\\nHowever, for once things were simple: CFTR is indeed itself the chloride ion channel inthe plasma membrane (Fig. 20.3). We know this because if puri ﬁed CFTR is introduced\\ninto lipid bilayers, chloride currents can be measured (see In Depth 20.1). In resting cellsthe channel mouth is blocked by a plug called the regulatory domain. When the regulatorydomain is phosphorylated by cAMP-dependent protein kinase (page 353), the plug leavesthe channel, allowing chloride ions to ﬂow through.\\nComparing the sequence of the normal CFTR gene to the one in people with CF\\nrevealed the mutations that caused the disorder. In 70% of CF patients, three nucleotidesare missing, generating a protein that is missing the amino acid phenylalanine at position 508(Fig. 20.4). The protein is a functional chloride channel, but the mutation affects its abilityto fold properly (page 192), and so the mutant protein fails to be inserted into the plasmamembrane. In the remaining 30% of patients more than 300 CFTR gene mutations have\\nbeen documented. These include missense, nonsense, and frameshift mutations (page 80).In some people mutations that cause incorrect exon –intron splicing (page 118) occur. The\\nseverity of CF symptoms is dependent on the particular mutation. Some cause a completefailure to express the protein and produce severe symptoms. Other mutations produce aprotein that is correctly traf ﬁcked to the plasma membrane and that works, but not as well\\nas the normal protein.\\nGENE THERAPY FOR CF\\nIt is an exciting prospect to think that we might one day treat disease by introducing anormal gene into the cells of patients and correct a defect. However, gene therapy is a', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b71c4a35-0f78-4758-88a7-83abe6e6a923', embedding=None, metadata={'page_label': '433', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-20 WY001-Bolsover-v2.cls September 1, 2003 10:31\\n428 CASE STUDY: CYSTIC FIBROSIS\\ndifﬁcult procedure, still in its infancy, and one that is not without risk to the patient. At\\npresent, permission has only been granted to carry out gene therapy on somatic cells. In\\nthis method the defect is corrected in speci ﬁc cells or tissues rather than in the germ line\\ncells that will give rise to eggs and sperm. Before trying out gene therapy on CF patients,an animal study was conducted. A knock-out mouse (page 157), lacking the CFTR gene,\\nwas created (Fig. 20.5). The mouse showed the symptoms of CF. A plasmid containing theCFTR cDNA was used to prepare an aerosol. When the mice inhaled the aerosol droplets,the results were spectacular. The lung symptoms disappeared for many days. The CFTRcDNA had been expressed into protein and the protein targeted to the plasma membrane.Could this approach work in humans?f\\nIN DEPTH 20.1 Lipid Bilayer Voltage Clamp\\nIn 1992 Christine Bear and co-workers published a deﬁnitive demonstration that\\nthe CFTR is a chloride ion channel. They used a technique called the lipid bilayervoltage clamp, which is illustrated in the accompanying ﬁgure. A lipid bilayer isconstructed from phospholipid and used to plug a hole between two chambersﬁlled with salt solution. Since a lipid bilayer is a barrier to ion movement, nocurrent is recorded when a voltage is imposed across the bilayer. However, whenartiﬁcial vesicles containing integral membrane proteins are added to one bath,the vesicles fuse with the artiﬁcial membrane so that the proteins now span thebarrier bilayer. When Bear and her co-workers did this with puriﬁed CFTR, theyrecorded currents with the direction and amplitude expected of a chloride-selectivechannel. In particular, no current ﬂowed when the voltage difference across themembrane was equal to the calculated chloride equilibrium voltage (page 312).No other integral membrane proteins were present in this experiment, so the CFTRmust itself be a chloride channel.\\ncurrenttime0current\\n−1pAtime0−50mV 0mVartificial\\nbilayerammeter\\nartificial\\nvesiclescontainingpurifiedCFTR\\n−50mVCFTRsalt solution containing cAMP dependent\\nprotein kinase', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='effd4e87-fda7-4728-a273-5b4d47ab45f7', embedding=None, metadata={'page_label': '434', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-20 WY001-Bolsover-v2.cls September 1, 2003 10:31\\nGENE THERAPY FOR CF 429\\nATC ATC TTT GGT GTTsequence of\\nbases in thenormal \\nCFTR  gene\\n506ile\\n507ile\\n508phe\\n509gly\\n510valamino acid sequence of CFTR protein3 base deletion causes CF\\nATC ATT GGT GTTsequence of bases in mutated \\nCFTR  \\ngene\\n506ile\\n507ile\\n508gly\\n509valamino acid sequence of mutant CFTR protein\\nFigure 20.4. The mutation seen in 70% of CF patients: a deletion of three nucleotides from the\\nCFTR gene has removed a phenlyalanine at position 508.\\nIn 1992 the National Institutes of Health in the United States granted permission\\nfor a CFTR gene therapy trial. Because viruses are so good at infecting our cells,\\nthey are useful tools for transferring DNA into somatic cells. Viral vectors for gene\\ntherapy have been engineered to remove their harmful genes. The gene to be trans-ferred into the patient is inserted into the modi ﬁed viral genome. A modi ﬁed aden-\\novirus vector containing the CFTR cDNA was introduced into the lung cells of fourpatients. Each of the patients made CFTR protein, but only for a short time. Somepatients showed adverse reactions to the therapy, probably due to the adenovirus it-self. Subsequent trials have delivered an adenovirus, containing the CFTR cDNA, tothe nasal epithelium of CF patients. However, again adverse reactions to the treatmentdeveloped.\\nOn a more positive note, a method called liposomal-mediated gene transfer shows some\\npromise. This technique has the advantage that it avoids the use of viruses. The plasmidcontaining the CFTR cDNA is coated with a synthetic lipid bilayer to form a liposome(page 229). The liposome can fuse with the epithelial cell membrane and deliver its DNAcontents into the cell. In the case of CF this approach seems to work when the liposomes aredelivered to the lungs in the form of an aerosol spray. The CFTR cDNA is expressed and', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='08e4236b-ac33-4754-a88e-7ce0bb8c001d', embedding=None, metadata={'page_label': '435', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-20 WY001-Bolsover-v2.cls September 1, 2003 10:31\\n(a) normal mouseCl−\\nmRNACFTR\\nproteinchromosomechromosomechromosomelung\\n(b) CFTR knockout\\n(c) gene therapyCl−\\nmRNAdroplets\\ncontainingplasmid\\nCFTR\\nproteinplasmid\\nplasmidplasmidCl−enlarged area\\nFigure 20.5. TheCFTR knock-out mouse is a model that can be used to test gene therapy stategies.\\n430', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a1f11838-74fe-4ee6-be28-7cf55e5e7036', embedding=None, metadata={'page_label': '436', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text=\"P1: GIG\\nWY001-20 WY001-Bolsover-v2.cls September 1, 2003 10:31\\nDIAGNOSTIC TESTS FOR CF 431\\nsome chloride ion channels form in the plasma membrane. The defect is not permanently\\ncorrected and continual use of the aerosol will be necessary to alleviate the lung symptomsof CF. Other organs, such as the pancreas, are affected in CF. Gene therapy on pancreaticcells will prove much more dif ﬁcult. These cells are not as easily accessible as the epithelial\\ncells of the lung.\\nDIAGNOSTIC TESTS FOR CF\\nBecause 70% of CF patients carry the same mutation, the deletion of phe at postion 508,a diagnostic test was developed. The test, based on PCR (page 150), requires a few cells\\nfrom the inside of the mouth and two DNA primers that will amplify a stretch of DNA thatwill include the region encoding amino acid 508 (Fig. 20.6). The PCR products are dottedonto two membrane ﬁlters. One membrane is incubated with an oligonucleotide containing\\nthe sequence encoding phe508 and the other is incubated with an oligonucleotide lackingthe codon for phe508. By comparing the patterns of hybridization, carriers of CF can bedetected. Other mutations that cause CF can also be tested for. However, because there areso many CFTR mutations, many families may require a new predictive test to be designed\\nto match their version of the defective gene.\\nPRENATAL IMPLANTATION DIAGNOSIS FOR CF. In 1992 the ﬁrst preimplan-\\ntation diagnosis for CF was carried out. Three sets of parents took part in the trial. Eachparent was a carrier for the phe508 mutation. Oocytes were removed from the women\\n1234 1234\\n1234 12342. Dot DNA onto membrane filters\\nIndividual 1: CF carrier\\nIndividual 2: both copies of gene are normalIndividual 3: both copies of gene are mutant: CF patientIndividual 4: CF carrier1. PCR amplify section of \\nCFTR  gene\\nspanning codon for phe 508\\nadd 'normal'\\noligonucleotideadd 'mutant'oligonucleotide\\nFigure 20.6. A diagnostic test for the most common CFTR gene mutation.\", start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4295921b-da64-4a40-bb75-e6ff0e2340c6', embedding=None, metadata={'page_label': '437', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-20 WY001-Bolsover-v2.cls September 1, 2003 10:31\\n432 CASE STUDY: CYSTIC FIBROSIS\\nand fertilized with sperm from their partners. Success was reported for only one couple.\\nSix of their oocytes developed into fertilized embryos. A single cell was tested from eachembryo, using PCR, for the presence of the defective CFTR gene. Five embryos were\\ncharacterized. Two embryos each had two copies of the normal CFTR gene, one on each\\nof their chromosome 7 ’s. Two embryos each had two copies of the phe508 deletion. One\\nembryo tested positive for both the normal gene and the phe508 deletion and was there-fore a carrier. One normal/normal embryo and the carrier embryo were implanted into themother. A baby girl was born. Her DNA showed that both copies of her CFTR gene were\\nnormal.\\nTHE FUTURE\\nGene therapy strategies for CF have progressed, but much needs to be improved before thistype of treatment can offer real hope for patients. The best gene therapy can offer is an alle-viation of some of the symptoms. A cure for CF does not seem a likely prospect. Detectionof CF carriers and prenatal diagnosis together with genetic counseling informs parents ofthe risks and gives them the choice of terminating an affected pregnancy. Preimplantationdiagnosis raises ethical questions about selecting a speci ﬁc embryo. In the future it may\\nbe possible to carry out germ line therapy in the egg or sperm of carriers. Most countries\\nregard such interference as unethical.\\n☛\\n✡✟\\n✠Medical\\nRelevance\\n20.1Success and a Setback with SCID\\nChildren with severe combined immunode ﬁciency disease (SCID) show an almost\\ncomplete failure to ﬁght infection and die soon after birth if not kept in sterile\\nconditions. SCID is more widely known as “bubble boy disease ”because con ﬁnement to a germ-\\nfree plastic bubble was the way in which patients were kept alive for a number of years. One rare\\nform of SCID, caused by a failure to make an enzyme called adenosine deaminase, can be treatedby regular injections of recombinant protein (page 149). In 1990 the ﬁrst trial of a genetic therapy\\nbegan at the National Institutes of Health in Washington. White blood cells from patients weretransfected with a plasmid encoding adenosine deaminase, which inserted into random positionsin the genome and began to be transcribed. Although the therapy does seem to have helped thepatients, they still need to be periodically injected with recombinant adenosine deaminase.\\nThe commonest form of the disease, X-SCID, affects only boys and occurs in about 1 in every\\n150,000 births. X-SCID is caused by a defective receptor for interleukin 2 (see page 222 for more oninterleukin 2). In March 1999 a team at INSERM in Paris initiated a trial in which white blood cellsfrom X-SCID patients were transfected with a viral vector containing the normal, interleukin-2receptor gene, which inserted randomly into the genome. So far 10 patients have been treated inParis and one at University College London. Of these, 9 are now essentially cured; they are athome living a normal life and are unlikely to need any further treatment. Tragically, however, oneof the children has developed leukemia —one of the transfected cells began dividing out of control.\\nApparently, the plasmid inserted into the gene for a transcription factor called LMO-2 and theviral promoter caused the transfected cells to make lots of LMO-2 as well as lots of the desiredinterleukin-2 receptor. Expression of LMO-2 causes white blood cells to divide uncontrollably.\\nAt the time of writing, it is uncertain whether this method of treatment for X-SCID will\\ncontinue. In the long term, this tragic accident will increase the drive toward treating geneticdisorders by homologous recombination (page 157) so that only the defective gene is changed.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4b842319-266f-404a-ab0b-eb021ee25160', embedding=None, metadata={'page_label': '438', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-20 WY001-Bolsover-v2.cls September 1, 2003 10:31\\nSUMMARY 433\\nSUMMARY\\n1.Cystic ﬁbrosis is the most common serious single-gene inherited disease in the\\nWestern world. Many organs are affected. Sticky mucus builds up in the reproductive\\ntract and in the lungs. The pancreas is always affected and usually fails completely.\\n2.Electrical measurements show that the basic problem is in chloride transport.\\n3.CF is a classic recessive disorder. The gene is on chromosome 7.\\n4.A combination of hard work and novel techniques helped isolate a small part of theCFTR gene. This sequence helped to isolate the normal CFTR cDNA from a sweat\\ngland clone library.\\n5.From the cDNA it was possible to identify the gene and infer its amino acid se-quence in both normal and mutated forms. The gene codes for a chloride chan-nel protein called CFTR. Hundreds of different CFTR mutations have now been\\nfound.\\n6.For the time being gene therapy only holds out hope of alleviating the lung problems,leaving other less accessible organs untreated. Tests have been devised for prenataldiagnosis and to detect carriers of CF.\\nFURTHER READING\\nBragonzi, A., and Conese, M. 2002. Non-viral approach toward gene therapy of cystic ﬁbrosis lung\\ndisease. Curr. Gene Therapy 2: 295 –305.\\nHandyside, A. H., Lesko, J. G., Tarin, J. J., Winston, R. M., and Hughes, M. R. 1992. Birth of a normal\\ngirl after in vitro fertilization and preimplantation diagnostic testing for cystic ﬁbrosis. N. Engl.\\nJ Med. 327: 905 –909.\\nKerem, B., Rommens, J. M., Buchanan, J. A., Markiewicz, D., Cox, T. K., Chakravarti, A., Buchwald,\\nM., and Tsui, L. C. 1989. Identi ﬁcation of the cystic ﬁbrosis gene: genetic analysis. Science 245:\\n1073 –1080.\\nRich, D. P., Anderson, M. P., Gregory, R. J., Cheng, S. H., Paul, S., Jefferson, D. M., McCann,\\nJ. D., Klinger, K. W., Smith, A. E., and Welsh, M. J. 1990. Expression of cystic ﬁbrosis trans-\\nmembrane conductance regulator corrects defective chloride channel regulation in cystic ﬁbrosis\\nairway epithelial cells. Nature 347: 358 –363.\\nRiordan, J. R., Rommens, J. M., Kerem, B., Alon, N., Rozmahel, R., Grzelczak, Z., Zielenski, J.,\\nLok, S., Plavsic, N., Chou, J. L., Drumm, M. L., Iannuzzi, M. C., Collins, F. S., and Tsui, L-C.\\n1989. Identi ﬁcation of the cystic ﬁbrosis gene: Cloning and characterization of complementary\\nDNA. Science 245: 1066 –1073.\\nWelsh, M. J., and Smith, A. E. 1995. Cystic ﬁbrosis. Sci. Am. 273: 52 –59.\\nBear, C. E., Li, C. H., Kartner, N., Bridges, R. J., Jensen, T. J., Ramjeesingh, M. and Riordan, J. R.\\n1992. Puri ﬁcation and functional reconstitution of the cystic ﬁbrosis transmembrane conductance\\nregulator (CFTR). Cell 68: 809 –818.\\nBenjamini, E., Coico, R. and Sunshine, G. 2000. Immunology, a Short Course, 4th ed. New York:\\nWiley-Liss, pp. 348 –353.\\nCheck, E. 2002. A tragic setback. Nature 420: 116 –118.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6c2f1e19-a879-48da-8e92-d84b1ade50e2', embedding=None, metadata={'page_label': '439', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-20 WY001-Bolsover-v2.cls September 1, 2003 10:31\\n434 CASE STUDY: CYSTIC FIBROSIS\\nREVIEW QUESTIONS\\nFor each question, choose the ONE BEST answer or completion.\\n1.A mother and a father, both CF carriers, have two children that do not suffer from CF.\\nThe chance of a third pregnancy producing a child with the disease is\\nA. zero.\\nB .1i n4 .C .1i n3 .D .1i n2 .E. very high —the child will almost certainly have CF.\\n2.Cystic ﬁbrosis results from a defect in a protein that forms\\nA. a plasma membrane sodium ion channel.B. a plasma membrane potassium ion channel.C. a plasma membrane calcium ion channel.D. a plasma membrane chloride ion channel.E. a plasma membrane phenylalanine carrier.\\n3.A zoo blot helps to detect DNA sequences that\\nA. are mutating at a fast rate.B. are conserved between species.C. show no similarity between species.D. are lost due to species extinction.E. are processed pseudogenes.\\n4.Somatic gene therapy attempts to correct a gene defect by introducing the normal gene\\ninto\\nA. the egg.\\nB. the sperm.C. the fertilized egg.D. cultured embryonic stem (ES) cells.E. cells in the patient ’s body other than eggs or sperm.\\n5.The most common mutation that causes CF\\nA. is a nonsense mutation at codon 508 that causes premature chain termination of the CFTR\\nprotein.\\nB. is a frameshift mutation such that all amino acids following phe at 508 are different from the\\nnormal protein.\\nC. is a deletion of three nucleotides that causes phe508 to be absent in the mutant protein.D. is a nonsense mutation causing failure to express a receptor tyrosine kinase whose activation\\ntriggers expression of the proteins mediating chloride transport.\\nE. only causes disease in boys.\\n6.Cystic ﬁbrosis is said to be “linked ”to another gene if\\nA.CFTR is part of an operon that includes the other gene.\\nB.CFTR and the other gene are located near each other on the same chromosome.\\nC. CFTR is translated as a chimera with the other protein.D. both CFTR and the protein product of the other gene are activated when phosphorylated by\\ncAMP-dependent protein kinase.\\nE. the function of the other gene requires chloride transport.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='9260125b-acac-4b67-9f33-ce0d733402ca', embedding=None, metadata={'page_label': '440', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: GIG\\nWY001-20 WY001-Bolsover-v2.cls September 1, 2003 10:31\\nANSWERS TO REVIEW QUESTIONS 435\\n7.CFTR and the voltage-gated calcium channel are similar in that\\nA. both pass calcium ions.\\nB. both are opened by depolarization.C. both are caused to open by an increase in the concentration of an intracellular messenger.D. both are closed when a cytosolic plug enters the channel lumen and blocks it.E. both play a critical role in brain cells.\\nANSWERS TO REVIEW QUESTIONS\\n1.B.Each pregnancy has a 1-in-4 chance of producing a child with CF. The presence or absence of\\nthe disease in earlier children is irrelevant.\\n2.D.CF is characterized by a defect in chloride ion transport.\\n3.B.Zoo blots detect protein-coding sequences that have been conserved through evolution and are\\ntherefore present in many species.\\n4.E.In somatic gene therapy the normal gene is introduced into a somatic cell where expression of\\nthe new gene will alleviate or cure the disease. This is any cell other than the gametes, the egg,and sperm.\\n5.C.Seventy percent of individuals with CF lack three nucleotides in their CFTR gene. This causes\\nthe phe at 508 in the normal protein to be absent from the mutant protein.\\n6.B.Concerning the other answers: (A) This would certainly produce genetic linkage, but operons\\nare not found in eukaryotes. (C) Chimeras, which are characterized by the physical linkage onthe same polypeptide chain of two proteins that are normally independent, can be produced bynaturally ocurring mutations, but this is not what we mean by linkage. Answers D and E are notrelevant to the concept of linkage.\\n7.D.The voltage-gated calcium channel is inactivated when a cytosolic plug enters the channel\\nlumen and blocks it, while the CFTR is opened when phosphorylation of its cytosolic plug causesthe plug to vacate the channel lumen. Of the other answers: (A) CFTR does not pass calcium ions.(B) CFTR is not opened by depolarization. (C) This answer is not completely wrong because whilethe CFTR opens when phosphorylated by cAMP-dependent protein kinase, phosphorylation bycAMP-dependent protein kinase does also make it easier for depolarization to open the voltage-gated calcium channel. However, phosphorylation alone cannot open the voltage-gated calciumchannel, so this answer is not as good as answer D. (E) As far as we know, CFTR plays no role inthe brain. CF patients are cognitively normal.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4d21546f-0640-4d2b-a44c-a19f2f7b6676', embedding=None, metadata={'page_label': '441', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-APX1 WY001-Bolsover-v2.cls October 23, 2003 15:25\\nAPPENDIX:\\nCHANNELS AND CARRIERS\\nAll the channels and carriers described in the book are listed here for reference.\\nPage for\\nName Details Location Selective for Opened by Comments\\nCHANNELS\\nLarge ChannelsGap junction\\nchannel55 Plasma\\nmembrane ofmany cellsAny solute of\\nM\\nr<1000Contact with\\nsecond gapjunction channelAllows solutes to\\npass fromcytosol to cytosol\\nPorin 262 Outer mito-\\nchondrialmembraneAny solute of\\nM\\nr<10,000Probably spends a\\nlarge fraction oftime open undermostcircumstancesStress can cause\\nswitch to a largerform that allowscytochrome cto\\npass\\nPotassium\\nchannels\\nPotassium\\nchannel310 Plasma\\nmembrane ofall cellsPotassium\\nionsSome isoforms are\\nopened bydepolarization;some are openall the timeResponsible for the\\nresting voltage\\n(Continued )\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n437', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='08a76954-3319-415a-b3c3-45c695c7c983', embedding=None, metadata={'page_label': '442', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-APX1 WY001-Bolsover-v2.cls October 23, 2003 15:25\\n438 APPENDIX: CHANNELS AND CARRIERS\\nPage for\\nName Details Location Selective for Opened by Comments\\nCalcium\\nchannels\\nV oltage-gated\\ncalcium\\nchannel327 Plasma\\nmembrane ofsea urchinegg, axonterminal,muscle, andother cellsCalcium ions Depolarization of\\nplasmamembraneThe inward\\nmovement ofcalcium ionschanges thetransmembranevoltage andraises thecalciumconcentration inthe cytosol\\nInositol\\ntrisphosphate-gated calciumchannel344 Membrane of\\nendoplasmicreticulumCalcium ions Binding of\\ncytosolicinositoltrisphosphatePart of the system\\nthat allows anextracellularsolute to raisecytosolic calcium\\nRyanodine\\nreceptor;skeletalmuscleisoform.348 Membrane of\\nendoplasmicreticulumCalcium ions Direct physical\\nlink tovoltage-gatedcalciumchannels in theplasmamembraneAllows rapid\\nincrease ofcytosolic calciumconcentration inresponse todepolarization ofplasmamembrane\\nRyanodine\\nreceptor;standardisoforms348 Membrane of\\nendoplasmicreticulumCalcium ions Binding of\\ncytosoliccalciumAllows a small\\nincrease ofcytosolic calciumconcentration totrigger a largerincrease\\nMitochondrial\\ncalciumchannel305 Inner mito-\\nchondrialmembraneCalcium ions Not easily studied,\\nprobably alwaysopenAllows an increase\\nof cytosoliccalciumconcentration toactivatemitochondria.\\nSodium and\\nunselectivecationchannels\\nV oltage-gated\\nsodiumchannel330 Plasma\\nmembrane ofnerve andmuscle cellsSodium ions Depolarization of\\nplasmamembraneProduces brief\\naction potentials\\n(Continued )', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='761c2a18-c114-44f9-a586-04824ef64f28', embedding=None, metadata={'page_label': '443', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-APX1 WY001-Bolsover-v2.cls October 23, 2003 15:25\\nAPPENDIX: CHANNELS AND CARRIERS 439\\nPage for\\nName Details Location Selective for Opened by Comments\\nNicotinic\\nacetylcholine\\nreceptorchannel364 Plasma\\nmembrane ofskeletalmuscle cellsand somenerve cellsSodium and\\npotassiumExtracellular\\nacetylcholineAn ionotropic cell\\nsurface receptorthat causes theplasmamembrane todepolarize\\nGlutamate\\nreceptorchannel372 Plasma\\nmembrane ofsome nervecellsSodium and\\npotassium,someisoformsalso passcalciumExtracellular\\nglutamateAn ionotropic cell\\nsurface receptorthat causes theplasmamembrane todepolarize\\ncAMP-gated\\nchannel350 Plasma\\nmembranesof scent-sensitivenerve cellsSodium and\\npotassiumcAMP in the\\ncytosolDepolarizes the\\nplasmamembrane sothat the cellproduces actionpotentials\\ncGMP-gated\\nchannel353 Plasma\\nmembranesof photore-ceptorcellsSodium and\\npotassiumcGMP in the\\ncytosolDepolarizes the\\nplasmamembrane\\nChloride\\nchannels\\nGABA receptor 373 Plasma\\nmembrane ofsome nervecellsChloride ions Extracellular\\nGABAWhen this channel\\nis open, it ismore difﬁcult todepolarize thecell to threshold\\nCFTR 426 Plasma\\nmembrane ofsweat glandcells, airwayepithelialcells, and ofmany othercellsChloride ions Phosphorylation\\nby proteinkinase ARequired for\\nseveral transportprocesses inglands', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='5dd765b3-4ad5-4167-8972-332aeab82e59', embedding=None, metadata={'page_label': '444', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-APX1 WY001-Bolsover-v2.cls October 23, 2003 15:25\\n440 APPENDIX: CHANNELS AND CARRIERS\\nPage for\\nName Details Location Mode of Action Comments\\nCARRIERS\\nCarriers with no\\nenzymatic action\\nGlucose carrier 316 Plasma membrane\\nof all cells GlucoseRequired by all human cells\\nADP/ATP exchanger 271 Inner mitochondrial\\nmembraneADP\\nATPGets ATP and ADP across\\ninner mitochondrialmembrane\\nNa\\n+/Ca2+exchanger 317 Plasma membrane\\nof many cells3 Na+\\nCa2+Pushes calcium ions out of\\nthe cytosol\\nβ-Galactoside permease 111 Bacterial plasma\\nmembrane\\nBrings lactose into the cell.\\nOne of the products ofthelacoperon\\nCarriers with an\\nenzymatic action aswell as a carrieraction\\nATP synthase 272 Inner mitochondrial\\nmembrane\\n10H+\\nInter-\\nmembranespace\\nMatrix3ATP3ADP\\n+ 3PiConverts between energy in\\nthe H+gradient and\\nenergy as ATP\\nCa2+ATPase 318 Plasma membrane\\nof many cells\\nPushes calcium ions out of\\nthe cytosol. All cells\\nhave either this or theNa\\n+/Ca2+exchanger;\\nmany have both\\nNa+/K+ATPase 270 Plasma membrane of\\nall cells\\nConverts between energy\\nas ATP and energy in theNa\\n+gradient\\nElectron transport chain 265 Inner mitochondrial\\nmembrane\\nNAD+\\n12H+Inter-\\nmembranespace\\nMatrix+ H\\n2O+ 1/2O2+ H+NADH Converts between energy\\nas NADH and energy in\\nthe H+gradient', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e3732221-602f-416d-b0fb-047e5f472a61', embedding=None, metadata={'page_label': '445', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY\\n14–3–3: protein that binds phosphorylated BAD, hence preventing it from triggering\\napoptosis.\\n7-methyl guanosine cap: modiﬁed guanosine found at the 5′terminus of eukaryotic mRNA.\\nA guanosine is attached to the mRNA by a 5′-5′-phosphodiester link and is subsequently\\nmethylated on atom number 7 of the guanine.9+2 axoneme: structure of cilium or ﬂagellum; describes the arrangement of nine pe-\\nripheral microtubules surrounding two central microtubules.\\nA (adenine): one of the bases present in DNA and RNA; adenine is a purine.\\nα-amino acid: amino acid in which the carboxyl and amino groups are attached to the\\nsame carbon.\\nαadrenergic receptor: a receptor for the related chemicals adrenaline and noradrenaline.\\nTheαadrenergic receptor activates G\\nqand therefore generates a calcium signal. To a ﬁrst\\napproximation, αreceptors respond to noradrenaline, while adrenaline acts mainly on β\\nadrenergic receptors.\\nαhelix: a common secondary structure in proteins, in which the polypeptide chain is coiled,\\neach turn of the helix taking 3.6 amino acid residues. The nitrogen atom in each peptide\\nbond forms a hydrogen bond with the oxygen four residues ahead of it in the polypeptidechain.\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.\\n441', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ba637f4f-1529-40c1-95a4-c6c9536fe898', embedding=None, metadata={'page_label': '446', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n442 GLOSSARY\\nA site: (aminoacyl site) site on a ribosome occupied by an incoming tRNA and its linked\\namino acid.\\nacceptor: in a hydrogen bond, the atom (oxygen, nitrogen, or sulfur) that is not covalently\\nbonded to the hydrogen but that nevertheless accepts a small share of the electrons.\\nacetone: CH 3—CO—CH 3. A chemical with a fruity smell produced from acetoacetate\\nduring the ketosis that can occur in diabetes.\\nacetylcholine:\\nOCH 3CH 3\\nCH 3CH 2 OC CH 3 CH 2N+\\nA transmitter released by various nerve cells including motoneurones and autonomic va-\\nsodilator neurones.\\nacetylcholine receptor: integral membrane protein that binds acetylcholine. There are two\\ntypes: the nicotinic acetylcholine receptor is ionotropic while the muscarinic acetylcholine\\nreceptor is metabotropic, linked via G qto phospholipase C β.\\nacetylcholinesterase: enzyme that hydrolyzes acetylcholine, terminating its action at\\nsynapses.\\nacid: molecule that readily gives H+to water. Most organic acids are compounds containing\\nthe group —COOH, although —SH is also weakly acidic.\\nacrosome: actin-rich structure that helps the sperm to penetrate the egg at fertilization.\\nActA: protein on the surface of the bacterium Listeria from which actin ﬁlaments\\npolymerize.\\nactin-binding proteins: proteins that bind to and modulate the function of G-actin or\\nF-actin.\\nactin-related protein (Arp): an actin nucleation protein. New actin ﬁlaments can grow out\\nfrom an Arp base.actin-severing protein: one of a group of enzymes that cuts actin micro ﬁlaments. Gelsolin\\nis an example.action potential: explosive depolarization of the plasma membrane.\\nactivation energy: height of the energy barrier between the reactants and the products of\\na chemical reaction.\\nacyl group: group having the general formula\\nOCnHmC\\nAcyl groups are formed when fatty acids are attached to other compounds by ester bonds.\\nadenine (A): one of the bases present in DNA and RNA; adenine is a purine.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3dcf30fe-a176-4efb-9ba2-3f993c2f54e4', embedding=None, metadata={'page_label': '447', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 443\\nadenosine: adenine linked to the sugar ribose. Adenosine is a nucleoside.\\nadenosine diphosphate (ADP): adenosine with two phosphates attached to the 5′carbon\\nof ribose.\\nadenosine monophosphate (AMP): adenosine with one phosphate attached to the 5′carbon\\nof ribose.\\nadenosine triphosphate (ATP): adenosine with three phosphates attached to the 5′carbon\\nof ribose. ATP is a coenzyme and one of the cell ’s energy currencies.\\nadenylate cyclase: enzyme that converts ATP to the intracellular messenger cyclic AMP\\n(cAMP). Also called adenyl cyclase.adenyl cyclase: enzyme that converts ATP to the intracellular messenger cyclic AMP\\n(cAMP). Also called adenylate cyclase.adherens junctions: type of anchoring junction in which the cell adhesion molecules are\\nlinked to actin micro ﬁlaments.\\nadipocytes: cells that store fats (triacylglycerols).\\nadipose tissue: type of fatty connective tissue.\\nADP: adenosine diphosphate; adenosine with two phosphates attached to the 5\\n′carbon of\\nribose.ADP/ATP exchanger: carrier in the inner mitochondrial membrane. ADP is moved in one\\ndirection and ATP in the other.adrenaline: hormone released into the blood when an individual is under stress. Adrenaline\\nacts at β-adrenergic receptors to activate G\\nsand hence adenylate cyclase.\\nadrenergic receptor: receptor for the related chemicals adrenaline and noradrenaline. There\\nare two isoforms, αandβ.T oa ﬁrst approximation, noradrenaline acts mainly on αreceptors\\nlinked to G qand therefore generates a calcium signal, while adrenaline acts mainly on β\\nreceptors linked to G sand therefore generates a cAMP signal.\\nαhelix: common secondary structure in proteins in which the polypeptide chain is coiled,\\neach turn of the helix taking 3.6 amino acid residues. The nitrogen atom in each peptide\\nbond forms a hydrogen bond with the oxygen four residues ahead of it in the polypeptidechain.\\nAkt: earlier name for protein kinase B, a protein kinase that is activated when it is itself\\nphosphorylated; this in turn only occurs when Akt is recruited to the plasma membrane\\nby phosphatidylinositol trisphosphate (PIP\\n3). Akt phosphorylates proteins on serine and\\nthreonine residues (e.g., the bcl-2 family protein BAD).\\nalkali: strong base (that will take H+from water) such as sodium hydroxide or potassium\\nhydroxide.alkaline: of a solution: one with a low concentration of H\\n+(really H 3O+) so that the pH is\\ngreater than 7.0.allolactose: disaccharide sugar. Lactose is converted to allolactose by the enzyme β-\\ngalactosidase. Allolactose is an inducer of lacoperon transcription.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='57e75887-522d-4ecf-bb6f-b2ddfb674269', embedding=None, metadata={'page_label': '448', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n444 GLOSSARY\\nall or nothing: phenomenon that, once initiated, proceeds to completion even if the stimulus\\nis removed. A process that involves positive feedback, such as an action potential, will tend\\nto be all or nothing.\\nallosteric; allostery: when the binding of a ligand to one site on a protein affects the\\nbinding at another site. These interactions can be between sites for the same ligand or\\ndifferent ligands. Molecules that are allosteric almost always have a quaternary structure.\\nalternative splicing: phenomenon in which a single eukaryotic primary mRNA transcript\\ncan be processed to yield a number of different processed mRNAs and can therefore generate\\na number of different proteins.\\nα-amino acid: amino acid in which the carboxyl and amino groups are attached to the\\nsame carbon.amino acid: chemical that has both a —COOH group and an —NH\\n2group. In an α-\\namino acid, both the —COOH and —NH 2groups are attached to the same carbon atom.\\nAll proteins are generated using the genetically encoded palette of 19 α-amino acids plus\\nproline.aminoacyl site (A site): site on a ribosome occupied by an incoming tRNA and its linked\\namino acid.aminoacyl tRNA: tRNA attached to an amino acid via an ester bond.\\naminoacyl tRNA synthases: family of enzymes, each of which attaches an amino acid to\\nthe appropriate tRNA.\\nγ-amino butyric acid: aγ-amino acid that acts to open chloride channels in the plasma\\nmembrane of sensitive nerve cells. Usually called γ-amino butyric acid rather than γ-amino\\nbutyrate, even though the latter (\\n−OOC —CH 2—CH 2—CH 2—NH 3+) is the form in which\\nit is found at neutral pH.\\namino terminal: end of a peptide or polypeptide that has a free α-amino group.\\namorphous: without form. Not a speci ﬁcally scienti ﬁc word.\\nAMP: adenosine monophosphate; adenosine with one phosphate attached to the 5′carbon\\nof ribose.amphipathic: “hating both ”—a molecule with a hydrophobic region and a hydrophilic\\nregion is said to be amphipathic.\\nanabolism: those metabolic reactions that build up molecules: biosynthesis.\\nanaerobic: without air. Obligate anaerobes are poisoned by oxygen and therefore can only\\nfunction anaerobically. Other cells, such as yeast and skeletal muscle, can switch to using\\nanaerobic respiration when they are denied oxygen.\\nanaerobic respiration: partial breakdown of sugars and other cellular fuels that can be\\naccomplished in the absence of oxygen.anaphase: period of mitosis or meiosis during which sister chromatids or homologous\\nchromosome pairs separate; consists of anaphase A and anaphase B.anaphase A: part of anaphase in which the chromosomes move to the spindle poles.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='153ace6a-2f3d-4426-a2b4-265d6200a2f7', embedding=None, metadata={'page_label': '449', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 445\\nanaphase B: part of anaphase in which the spindle poles are separated.\\nanaphase I: anaphase of the ﬁrst meiotic division (meiosis I).\\nanaphase II: anaphase of the second meiotic division (meiosis II).\\nanchoring junction: class of cell junction that attaches the cytoskeleton of one cell to\\nthe cytoskeleton of its neighbor, forming a physically strong connection. There are two\\ntypes: desmosomes, which connect to intermediate ﬁlaments, and adherens junctions, which\\nconnect to actin micro ﬁlaments.\\nanion: negatively charged ion, e.g., chloride, Cl−, or phosphate, HPO 42−.\\nanode: positively charged electrode, e.g., in a gel electrophoresis apparatus used for SDS –\\nPAGE.\\nantenna chlorophyll: chlorophyl molecules that capture light and pass the energy to reac-\\ntion center chlorophylls.anterograde: forward movement; when applied to axonal transport it means away from the\\ncell body.\\nanthrax: lethal bacterial disease. When food supplies are exhausted, anthrax bacteria pro-\\nduce spores that can survive for long periods in the soil or in (illegal) stocks of biological\\nweapons.\\nantibiotic: chemical that is produced by one type of organism and kills others —often\\nby inhibiting protein synthesis. The most useful antibiotics to humans are those that are\\nselective for prokaryotes.\\nantibody: protein formed by the immune system that binds to and helps eliminate another\\nchemical. Antibodies can be extremely selective for their ligand and are useful in many\\naspects of cell biology such as immuno ﬂuorescence microscopy and western blotting.\\nanticodon: three bases on a tRNA molecule that hydrogen-bond to the codon on an mRNA\\nmolecule.\\nantiparallel βsheet: βsheet in which alternate parallel polypeptide chains run in opposite\\ndirections.\\nAP endonuclease: DNA repair enzyme that cleaves the phosphodiester links on either side\\nof a depurinated or depyrimidinated sugar residue. AP stands for apurinic/apyrimidinic.\\napoptosis: process in which a cell actively promotes its own destruction, as distinct from\\nnecrosis. Apoptosis is important in vertebrate development, where tissues and organs are\\nshaped by the death of certain cell lineages.\\napoptosis inhibitor protein: proteins that block the action of caspases and hence help\\nprevent apoptosis.aporepressor: protein that binds to an operator region and represses transcription only\\nwhen it is complexed with another molecule.\\naqueous: watery.\\nARF: GTPase that plays a critical role in the formation of coated vesicles.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d3f76bdc-0e9f-493f-ad35-dc4b772b7cf2', embedding=None, metadata={'page_label': '450', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n446 GLOSSARY\\nArp (actin-related protein): actin nucleation protein. New actin ﬁlaments can grow out\\nfrom an Arp base.\\nA site (aminoacyl site): site on a ribosome occupied by an incoming tRNA and its linked\\namino acid.\\nassay: term for a chemical measurement, e.g., one in which the activity of an enzyme\\nreaction is measured.ATP: adenosine triphosphate; adenosine with three phosphates attached to the 5\\n′carbon of\\nribose. ATP is a coenzyme and one of the cell ’s energy currencies.\\nATP/ADP exchanger: carrier in the inner mitochondrial membrane. ADP is moved in one\\ndirection and ATP in the other.ATP synthase: carrier of the inner mitochondrial membrane that is built around a rotary\\nmotor. Ten H\\n+enter the mitochondrial matrix for every three ATP made.\\nATP synthetase: another name for ATP synthase; a carrier of the inner mitochondrial\\nmembrane that is built around a rotary motor. Ten H+enter the mitochondrial matrix for\\nevery three ATP made.\\nautoinduction: process of self-induction that occurs when a product of a reaction stimulates\\nthe production of more of itself. An example is the activation of the luxoperon by the small\\nmolecule N-acyl-HSL; transcription of the luxoperon then causes the production of more\\nN-acyl-HSL.\\nautonomic nerves: nerves that control processes of which we are usually not consciously\\naware, such as blood vessel size.\\nautoradiography: process that detects a radioactive molecule. For example, in a Southern\\nblot experiment, the membrane that has been hybridized to a radioactive gene probe is\\nplaced in direct contact with a sheet of X-ray ﬁlm. Radioactive decay activates the silver\\ngrains on the emulsion of the X-ray ﬁlm. When the ﬁlm is developed, areas that have been\\nin contact with radioactivity will show as black.\\naxon: long process of a nerve cell, specialized for the rapid conduction of action potentials.\\naxonal transport: movement of material along microtubules within a nerve cell process;\\ncan be outward (anterograde) or inward (retrograde).\\nβadrenergic receptor: a receptor for the related chemicals adrenaline and noradrenaline.\\nTheβadrenergic receptor activates G sand therefore generates a cAMP signal. To a ﬁrst\\napproximation, βreceptors respond to adrenaline, while noradrenaline acts mainly on α\\nadrenergic receptors.\\nβoxidation: process by which fatty acids are broken down into individual two-carbon units\\ncoupled to CoA to form acetyl-CoA. The process, which takes place in the mitochondrial\\nmatrix, generates both NADH and FADH 2.\\nβsheet: common secondary structure in proteins in which lengths of fully extended\\npolypeptide run alongside each other, hydrogen bonds forming between the peptide bondsof the adjoining strands.\\nB-DNA: right-handed DNA double helix.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d6dec79c-aa93-4e82-9ec5-76a3ad3b6505', embedding=None, metadata={'page_label': '451', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 447\\nBAC (bacterial arti ﬁcial chromosome): cloning vector used to propagate DNAs of about\\n300,000 bp in bacterial cells.\\nbacteriophage: (sometimes shortened to phage) a virus that infects bacterial cells.\\nBAD: a bcl-2 family protein that causes the release of cytochrome cfrom mitochondria,\\ntriggering apoptosis. BAD is phosphorylated and thereby inactivated by protein kinase B.\\nbasal body: structure from which cilia and ﬂagella arise; has the same structure as the\\ncentriole.\\nbase: there are two meanings in cell biology: (1) A chemical that will accept an H+. Many\\norganic bases contain the group —NH 2, which accepts an H+to become —NH 3+. (2) One\\nof a group of ring-containing nitrogenous compounds that combine with a sugar to create a\\nnucleoside; the members include adenine, guanine, hypoxanthine, cytosine, thymine, uracil,and nicotinamide.\\nbase excision repair: process that repairs DNA double helices that have lost a purine\\n(depurination) or in which a cytosine has been deaminated to uracil (U). The entire damaged\\nmonomer is removed and a correct deoxyribonucleotide inserted in its place.\\nbasement membrane: thin layer of extracellular ﬁbers that supports epithelial cells.\\nbase pair: The Watson –Crick model of DNA showed that guanine in one DNA strand would\\nﬁt nicely with cytosine in another strand while adenine would ﬁt nicely with thymine. The\\ntwo hydrogen-bonded bases are called a base pair. RNA can also participate in base pairing:\\ninstead of thymine it is uracil that now pairs with adenine. The rare base inosine, found insome tRNAs, can base pair with any of uracil, cytosine, or adenine.\\nBAX: bcl-2 family protein that causes the release of cytochrome cfrom mitochondria,\\ntriggering apoptosis.bcl-2: antiapoptotic protein. By binding to BAD and BAX, it prevents them causing release\\nof cytochrome cfrom mitochondria.\\nbcl-2 family: family of related proteins that regulate cytochrome crelease from mitochon-\\ndria.\\nB-DNA: right-handed DNA double helix.\\nβ-galactosidase: enzyme that cleaves the disaccharide lactose to produce glucose and galac-\\ntose and that also catalyzes the interconversion of lactose and allolactose. βGalactosidase\\nis a product of the lacoperon.\\nbinding site: region of a protein that speci ﬁcally binds a ligand. A property of the protein ’s\\ntertiary structure.\\nbioluminesence: production of light by a living organism.\\nbisphosphate: of a compound, bearing two independent phosphate groups (as opposed to\\na diphosphate, which bears a chain of two phosphates in a line). Fructose-1,6-bisphosphate\\nis an example.\\nblastocyst: early embryo.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b6e921c0-1f62-42eb-ac1c-4d9bfc70861a', embedding=None, metadata={'page_label': '452', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n448 GLOSSARY\\nBloom ’s syndrome: disease resulting from a de ﬁciency in helicase. Affected individuals\\ncannot repair their DNA and are susceptible to developing skin cancer and other cancers.\\nblue-green algae: old name for the photosynthetic prokaryotes now known as cyanobac-\\nteria.\\nblunt ends: ends of a DNA molecule produced by an enzyme that cuts the two DNA strands\\nat sites directly opposite one another.βoxidation: process by which fatty acids are broken down into individual two-carbon units\\ncoupled to CoA to form acetyl-CoA. The process, which takes place in the mitochondrial\\nmatrix, generates both NADH and FADH\\n2.\\nbright- ﬁeld microscopy: most basic form of light microscopy. The specimen appears\\nagainst a bright background and appears darker than the background because of the light ithas absorbed or scattered.\\nbrown fat: tissue specialized for generating heat. Thermogenin in the inner mitochondrial\\nmembrane allows H\\n+to enter down its electrochemical gradient so that the rate at which\\nthe electron transport chain works is not limited by the availability of ADP.βsheet: common secondary structure in proteins in which lengths of fully extended\\npolypeptide run alongside each other, hydrogen bonds forming between the peptide bonds\\nof the adjoining strands.\\nbulky lesion: distortion of the DNA helix caused by a thymine dimer.\\nC (cytosine): one of the bases present in DNA and RNA; cytosine is a pyrimidine.\\nCa\\n2+ATPase: carrier that uses the energy released by ATP hydrolysis to move calcium ions\\nup their concentration gradient out of the cytosol. Different isoforms of calcium ATPase\\nare located at the plasma membrane and in the membrane of the endoplasmic reticulum.\\ncadherin: cell adhesion molecule that helps form adherens junctions.\\ncalcineurin: calcium-calmodulin –activated phosphatase, that is, an enzyme that removes\\nphosphate groups from proteins, opposing the effects of kinases. Calcineurin is inhibited\\nby the immunosuppressant drug cyclosporin.\\ncalcium action potential: action potential driven by the opening of voltage-gated calcium\\nchannels and the resulting calcium in ﬂux.\\ncalcium ATPase: carrier that uses the energy released by ATP hydrolysis to move calcium\\nions up their concentration gradient out of the cytosol. Different isoforms of calcium ATPase\\nare located at the plasma membrane and in the membrane of the endoplasmic reticulum.\\ncalcium-binding protein: any protein that binds calcium. Calmodulin, troponin and cal-\\nreticulin are examples found in the cytosol, attached to actin ﬁlaments in striated muscle,\\nand in the endoplasmic reticulum respectively.calcium-calmodulin –activated protein kinase: important regulatory enzyme, activated\\nwhen calcium-loaded calmodulin binds, which phosphorylates target proteins on serine\\nand threonine residues.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='05feabd4-307f-4bc4-8095-8793fb15f619', embedding=None, metadata={'page_label': '453', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 449\\ncalcium-induced calcium release: process in which a rise of calcium concentration in the\\ncytoplasm triggers the release of more calcium from the endoplasmic reticulum. The best\\nunderstood mechanism of calcium-induced calcium release is via ryanodine receptors.\\ncalcium pump: carrier that moves calcium ions up their electrochemical gradient out of\\nthe cytosol into the extracellular medium or into the endoplasmic reticulum. There are two\\nimportant calcium pumps: the sodium/calcium exchanger is found on the plasma membrane,while different isoforms of the calcium ATPase are found on the plasma membrane and onthe membrane of the endoplasmic reticulum.\\ncalmodulin: calcium-binding protein found in many cells. When calmodulin binds cal-\\ncium, it can then activate other proteins such as the enzymes calcineurin and glycogen\\nphosphorylase kinase.\\nCalvin cycle: series of reactions in the chloroplast that act to ﬁx atmospheric carbon dioxide\\nand build it into larger molecules.\\nCaM-kinase: another name for calcium-calmodulin –activated protein kinase; an important\\nregulatory enzyme, activated when calcium-loaded calmodulin binds, that phosphorylates\\ntarget proteins on serine and threonine residues.\\ncAMP (cyclic adenosine monophosphate): nucleotide produced from ATP by the action\\nof the enzyme adenylate cyclase. cAMP is an intracellular messenger in many cells.cAMP-dependent protein kinase (protein kinase A; PKA): protein kinase that is activated\\nby the intracellular messenger cyclic-AMP. PKA phosphorylates proteins (e.g., glycogen\\nphosphorylase kinase) on serine and threonine residues.\\ncAMP-gated channel: channel found in the plasma membrane of scent-sensitive nerve\\ncells. The channel opens when cAMP binds to its cytoplasmic face and allows sodium and\\npotassium ions to pass.\\ncAMP phosphodiesterase: enzyme that hydrolyzes cyclic AMP, producing AMP and hence\\nturning off signaling through the cAMP system.cap: methylated guanine added to the 5\\n′end of a eukaryotic mRNA.\\nCAP (catabolite activator protein): protein found in prokaryotes that binds to cAMP. The\\nCAP –cAMP complex then binds within the promoter region of some bacterial operons and\\nhelps RNA polymerase to bind to the promoter.\\ncarbohydrates: monosaccharides and all compounds made from monosaccharide\\nmonomers.\\ncarbon ﬁxation: process of reducing atmospheric carbon dioxide and building it into larger\\nmolecules.carboxylation: introduction of a carboxyl group ( —COOH).\\ncarboxyl group: —COOH group. Carboxyl groups give up hydrogen ions to form the\\ndeprotonated group —COO\\n−, so molecules that bear carboxyl groups are usually acids.\\ncarboxyl terminal: end of a peptide or polypeptide that has a free α-carboxyl group. This\\nend is made last at the ribosome.cardiac muscle: form of striated muscle that is found in the heart.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e63b4fb7-ba3e-400a-8d0e-fc6869b7da8d', embedding=None, metadata={'page_label': '454', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n450 GLOSSARY\\ncarrier: there are two meanings used in this book: (1) An integral membrane protein that\\nforms a tube through the membrane that is never open all the way through. Solutes can\\nmove into the tube through the open end. When the channel changes shape, so that the endthat was closed is open, the solute can leave on the other side of the membrane. (2) A personwho has one nonfunctional or mutant copy of a gene, but who shows no effects because theother copy produces suf ﬁcient functional protein.\\ncaspase: cysteine-containing protease that cleaves at aspartate residues. Caspases are re-\\nsponsible for the degradative processes that occur during apoptosis.\\ncatabolism: metabolic reactions that break down molecules to derive chemical energy.\\ncatabolite activator protein (CAP): protein that binds to cAMP. The CAP –cAMP complex\\nthen binds within the promoter region of some bacterial operons and helps RNA polymerase\\nto bind to the promoter.\\ncatalyst: chemical or substance that reduces the activation energy of a reaction, allowing it\\nto proceed more quickly. Many biological reactions would proceed at an in ﬁnitesimal rate\\nwithout the aid of enzymes, which are protein catalysts.catalytic rate constant ( k\\ncat):proportionality constant that relates the maximal initial\\nvelocity ( Vm) of an enzyme-catalyzed reaction to the enzyme concentration. kcat=Vm/ [E].\\nThe units are reciprocal time. See also Turnover number andMaximal velocity .\\ncation: positively charged ion, for example, Na+,K+, and Ca2+.\\ncdc: cell division cycle. The acronym is usually used to denote genes, especially of the\\nyeasts Saccharomyces cerevisiae andSchizosaccharomyces pombe , which, when mutated,\\ncause the cell division cycle to be abnormal.Cdc25: protein phosphatase involved in the regulation of CDK1.\\nCdk1 (cyclin-dependent kinase 1): protein kinase involved in the regulation of the G2/M\\ntransition of the cell cycle. Associates with cyclin B to form MPF.Cdk2 (cyclin-dependent kinase 2): protein kinase involved in the regulation of the G1\\nphase of the cell cycle. Associates with cyclin E.\\nCdk4 (cyclin-dependent kinase 2): protein kinase involved in the regulation of the G1\\nphase of the cell cycle. Associates with cyclin D.\\nCdk6 (cyclin-dependent kinase 2): protein kinase involved in the regulation of the G1\\nphase of the cell cycle. Associates with cyclin D.cDNA (or complementary DNA): DNA copy of an mRNA molecule.\\ncDNA library: collection of bacterial cells each of which contains a different foreign cDNA\\nmolecule.\\ncell: fundamental unit of life. A membrane-bound collection of protein, nucleic acid, and\\nother components that is capable of self-replication using simpler building blocks.\\ncell adhesion molecule: integral membrane protein responsible for sticking cells together.\\nThe extracellular domain binds a cell adhesion molecule on another cell while the intracel-\\nlular domain binds to the cytoskeleton, either directly or via a linker protein.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='551ee5d2-5a70-458c-84a9-bbb1bbef1d87', embedding=None, metadata={'page_label': '455', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 451\\ncell center: point immediately adjacent to the nucleus of eukaryotes where the centrosome\\nand Golgi apparatus are located.\\ncell cycle (cell division cycle): ordered sequence of events that must occur for successful\\ncell division; consists of G1, S, G2, and M phases.\\ncell division cycle: another name for the cell cycle: the ordered sequence of events that\\nmust occur for successful cell division; consists of G1, S, G2, and M phases.\\ncell division cycle (CDC) mutants: mutants, especially of the yeasts Saccharomyces cere-\\nvisiae andSchizosaccharomyces pombe, that are temperature-sensitive for cell division.\\ncell junctions: points of cell –cell interaction in tissues; includes tight junctions; anchoring\\njunctions and gap junctions.cell membrane: membrane that surrounds the cell; also known as the plasmalemma or\\nplasma membrane.cell surface membrane: another name for the plasma membrane.\\ncellulose: major structural polysaccharide of the plant cell wall; a β1→4 polymer of\\nglucose.\\ncellulose synthase: multiprotein complex of the plant plasma membrane that makes cellu-\\nlose.\\ncell wall: rigid case that encloses plant and fungal cells and many prokaryotes. The cell wall\\nlies outside the plasma membrane. Plant cell walls are composed of cellulose plus other\\npolysaccharide molecules such as hemicellulose and pectin.\\ncentral dogma (of molecular biology): “DNA makes RNA makes protein ”—the concept\\nthat the sequence of bases on DNA de ﬁnes the sequence of bases on RNA, and the sequence\\nof bases on RNA then de ﬁnes the sequence of amino acids on protein.\\ncentriole: structure found at the centrosome ( =microtubule organizing center) of animal\\ncells; composed of microtubules.centromere: region of the chromosome at which the kinetochore (where the microtubules\\nof the mitotic or meiotic spindle attach) is formed.\\ncentrosome (microtubule organizing center): structure from which cytoplasmic micro-\\ntubules arise.\\nCF (cystic ﬁbrosis): inherited disease characterized by failure of the pancreas and by thick\\nsticky mucus in the lungs leading to fatal lung infection unless treated. Cystic ﬁbrosis is\\ncaused by failure to make, or properly target, plasma membrane chloride channels.cGMP-dependent protein kinase (PKG; protein kinase G): protein kinase that is acti-\\nvated by the intracellular messenger cyclic GMP. PKG phosphorylates proteins (e.g., the\\ncalcium ATPase) on serine and threonine residues.\\nchain release factors: proteins that occupy the A site in the ribosome when a stop codon\\nis encountered and that act to trigger termination of polypeptide synthesis.channel: integral membrane protein that forms a continuous water- ﬁlled hole through the\\nmembrane.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='3d69e93c-3f66-4891-a5a0-a4d76d0987b6', embedding=None, metadata={'page_label': '456', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n452 GLOSSARY\\nchaotropic reagents: reagents such as urea, which cause proteins to lose all their higher\\nlevels of structure and adopt random, changing conformations.\\nchaperone: protein that helps other proteins to remain unfolded for correct protein targeting,\\nor to fold into their correct three-dimensional structure.charge: (1) excess or de ﬁcit of electrons giving a negative or positive charge, respectively;\\n(2) transfer RNA is said to be charged when it has an amino acid attached.charged tRNA: tRNA attached to an amino acid.\\nchiasmata (singular chiasma): structures formed during crossing over between the chro-\\nmatids of homologous chromosomes during meiosis; the physical manifestation of genetic\\nrecombination.\\nchimera: structure formed from two different parts. Chimeric proteins are generated by\\njoining together all or part of the protein coding sections of two distinct genes. Chimeric\\norganisms are formed by mixing two or more distinct clones of cells.\\nchimeric protein: proteins generated by joining together all or part of the protein coding\\nsections of two distinct genes, for example, GFP and a protein of interest.chiral: structure whose mirror image cannot be superimposed on it. Organic molecules\\nwill be chiral if a carbon atom has four different groups attached to it and is therefore\\nasymmetric.\\nchlorophyll: major photosynthetic pigment of plants and algae.\\nchloroplast: photosynthetic organelle of plant cells.\\nchromatid: complete DNA double helix plus accessory proteins subsequent to DNA repli-\\ncation in eukaryotes. At mitosis, the chromosome is seen to be composed of two chromatids;\\nthese then separate to form the chromosomes of the two progeny cells.\\nchromatin: complex of DNA and certain DNA-binding proteins such as histones.\\nchromatophore: pigment cell in the skin of ﬁsh and amphibia.\\nchromosome: single, enormously long molecule of DNA, together with its accessory pro-\\nteins. Chromosomes are the units of organization of the nuclear chromatin and carry many\\ngenes. In eukaryotes chromosomes are linear; in prokaryotes they are circular.\\nchromosome walking: investigating a chromosome bit by overlapping bit, each bit being\\nused to clone the next.cilium ( plural cilia): locomotory appendage of some epithelial cells and protozoa.\\nCIP1: gene encoding a cyclin-dependent kinase inhibitor protein that prevents cells entering\\nthe S phase of the cell cycle. p21\\nCIP1is upregulated by cell –cell contact.\\ncis:side to which material is added. Of the Golgi complex, the surface that receives vesicles\\nfrom the endoplasmic reticulum.cisternae: ﬂattened membrane-bound sacs, e.g., those that make up the Golgi apparatus.\\nCKI (cyclin-dependent kinase inhibitor): type of cell cycle regulatory protein. Binds to\\nand inactivates CDKs.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='9b5719d5-ab96-46b2-a92d-919d83106fde', embedding=None, metadata={'page_label': '457', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 453\\nclathrin: protein that functions to cause vesicle budding in response to binding of speci ﬁc\\nligand.\\nclathrin adaptor protein: protein that binds to speci ﬁc transmembrane receptors and which\\nin turn recruits clathrin to form a coated vesicle. The vesicle therefore contains the molecule\\nfor which the receptor is speci ﬁc.\\ncleavage furrow: in animal cells, the structure that constricts the middle of the cell during\\ncytokinesis.\\nclone: a number of genetically identical individuals.\\nclone library: collection of bacterial clones where each clone contains a different foreign\\nDNA molecule.\\ncloning: strictly, the creation of a number of genetically identical organisms. In molecular\\ngenetics, the term is used to mean the multiplication of particular sequences of DNA by an\\nasexual process such as bacterial cell division.\\ncloning vector: DNA molecule that carries genes of interest, can be inserted into cells, and\\nwhich will then be replicated inside the cells. Cloning vectors range in size from plasmids\\nto entire arti ﬁcial chromosomes.\\nclosed promoter complex: structure formed when RNA polymerase binds to a promoter\\nsequence prior to the start of transcription.\\ncoatamer: protein complex that encapsulates one class of coated vesicle. Formation of the\\ncoatamer coat on a previously ﬂat membrane forces the membrane into a curved shape and\\ntherefore drives vesicle formation.\\ncoated vesicle: cytoplasmic vesicle encapsulated by a protein coat. There are two types of\\ncoated vesicles, coated by coatamer and clathrin, respectively.codon: sequence of three bases in an mRNA molecule that speci ﬁes a particular amino\\nacid.\\ncoenzyme: molecule that acts as a second substrate for a group of enzymes. ATP/ADP,\\nNADH/NAD\\n+, and Acyl-coenzyme A/coenzyme A are all coenzymes.\\ncofactor: nonprotein molecule or an ion necessary for the activity of a protein. Cofactors are\\nassociated tightly with the protein but can be removed. Examples are pyridoxal phosphate\\nin aminotransferases and zinc in zinc ﬁnger proteins.\\ncohesin: protein that holds two chromatids together. Degradation of cohesin allows the two\\nchromatids to separate at the start of anaphase in mitosis and in meiosis II.\\ncolchicine: plant alkaloid from the autumn crocus, Colchicum autumnale; binds to tubulin.\\ncollagen: major structural protein of the extracellular matrix.\\ncolumnar: taller than it is broad. Used as a description of some types of epithelial cells.\\ncompetent: in molecular genetics this refers to a bacterial culture treated with a solution\\nsuch as calcium chloride so that uptake of foreign DNA is enhanced.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1c45ff08-439d-4c45-83d3-05be5b4f3d7f', embedding=None, metadata={'page_label': '458', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n454 GLOSSARY\\ncomplementary: two structures are said to be complementary when they ﬁt into or associate\\nwith each other. The two strands of the DNA double helix are complementary, as are the\\nanticodon and codon of transfer and messenger RNA.\\ncomplementary DNA (cDNA): DNA copy of an mRNA molecule.\\ncomplex (as a noun): association of molecules that is held together by noncovalent inter-\\nactions and often can be readily dissociated.\\ncomplex I, complex II, complex III, complex IV: large multimolecular complexes that\\nmake up the electron transport chain in the inner mitochondrial membrane.\\ncondenser lens: lens of light and electron microscopes that focuses light (or electrons) onto\\nthe specimen.connective tissue: tissue that contains relatively few cells within a large volume of extra-\\ncellular matrix.connexon (gap junction channel): channel in the plasma membrane with a central hole\\nabout 1.5 nm in diameter. Gap junction channels only open when they contact a second\\nchannel on another cell, in this case they open and form a water- ﬁlled tube that runs all the\\nway through the plasma membrane of the ﬁrst cell, across the small gap between the cells,\\nand through the plasma membrane of the second cell, so allowing passage of solute fromthe cytosol of one cell to the cytosol of the other.\\nconstitutive: operating all the time without obvious regulation. Housekeeping genes, ex-\\npressed all the time, are sometimes called constitutive genes. Proteins that are secreted all\\nthe time are said to use the constitutive route.\\nconstitutively active: mutant or modi ﬁed protein that is always in the “on”state. A con-\\nstitutively active enzyme is active in the absence of its normal regulators. A constitutively\\nactive GTPase activates its downstream targets in the absence of GTP exchange factors.Paradoxically, the easiest way to generate a consitutively active GTPase is to eliminate itsenzymatic activity, so that it does not hydrolyze GTP and therefore remains in the activeGTP-bound state.\\nconstitutive secretion: secretion that continues all the time, without the need for a signal\\nsuch as an increase of cytosolic calcium concentration.\\ncontact inhibition: inhibition of cell division by cell –cell contact. Contact inhibition allows\\ncells to proliferate to ﬁll a gap and then stop dividing.\\ncontigs: overlapping lengths of DNA that can together be used to build up a map of the\\ngenome.\\n—COOH: carboxyl group. Carboxyl groups give up hydrogen ions to form the deprotonated\\ngroup —COO\\n−, so molecules that bear carboxyl groups are usually acids.\\ncorepressor: chemical species that binds to a repressor protein, the complex then binding\\nto an operator region on DNA to prevent transcription. Tryptophan is a corepressor at the\\ntrpoperon.\\ncortex: outer part of any organ or structure. For instance, the tissue outside the vascular\\ntissue of plants, and the tissue that forms the outer region of the brain, are called cortex.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='18adcc61-dea6-4dcf-bbf4-5280b2ecfdf0', embedding=None, metadata={'page_label': '459', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 455\\ncovalent bond: strong bond between two atoms in which electrons are shared.\\ncristae: name given to the folds of the inner membrane of mitochondria.\\ncrossing-over: physical exchange of material that takes place between homologous chro-\\nmosomes during recombination and is manifest in the formation of chiasmata.\\ncrosstalk: two messenger systems show crosstalk when one messenger can produce some\\nor all of the effects of the other.\\nC-terminal (carboxyl terminal): end of a peptide or polypeptide that has a free α-carboxyl\\ngroup. This end is made last on the ribosome.cyanobacteria: photosynthetic prokaryotes that were formerly known as blue-green algae.\\ncyclic adenosine monophosphate (cAMP): nucleotide produced from ATP by the action\\nof the enzyme adenylate cyclase. cAMP is an intracellular messenger in many cells and\\nexerts many of its actions by activating protein kinase A.\\ncyclic guanosine monophosphate (cGMP): nucleotide produced from GTP by the action\\nof the enzyme guanylate cyclase. cGMP is an intracellular messenger in many cells and\\nexerts many of its actions by activating protein kinase G.\\ncyclin: one of a family of proteins whose level oscillates (cycles) through the cell divi-\\nsion cycle. Cyclins associate with and activate cyclin-dependent kinases and hence allow\\nprogression through cell cycle control points.\\ncyclin B: one of the two proteins that make up maturation promoting factor (MPF), the\\nother being CDK1; one of a family of proteins whose level oscillates (cycles) through the\\ncell division cycle.\\ncyclin D: protein that binds to CDK4 and 6 to form a complex that is active at the G1 control\\npoint of the cell division cycle; one of a family of proteins whose level oscillates (cycles)\\nthrough the cell division cycle.\\ncyclin-dependent kinase (CDKs): one of the family of protein kinases that regulate the cell\\ncycle. Cyclin-dependent kinases are only active when bound to one of the family of cyclin\\nproteins. For example, CDK1 associates with cyclin B and regulates the G2/M transitionwhile CDK2 associates with cyclin E and regulates the G1/S transition.\\ncyclin E: protein that binds to CDK2 to form a complex that is active at the G1 control\\npoint of the cell division cycle; one of a family of proteins whose level oscillates (cycles)\\nthrough the cell division cycle.\\ncystic ﬁbrosis (CF): inherited disease characterized by failure of the pancreas and by thick\\nsticky mucus in the lungs leading to fatal lung infection unless treated. Cystic ﬁbrosis is\\ncaused by failure to make, or properly target, plasma membrane chloride channels.cystine: double amino acid formed by two cysteine molecules joined by a disul ﬁde bond.\\ncytochemistry: use of chemical compounds to stain speci ﬁc cell structures and organelles.\\ncytochrome: proteins with a heme prosthetic group that are able to transfer electrons.\\nCytochromes form a critical part of the electron transport chain of mitochondria and also\\nform part of the cytochrome P450 detoxi ﬁcation system in the liver.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='f76bfab0-594c-4d64-a4a6-83d0f50e3fc4', embedding=None, metadata={'page_label': '460', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n456 GLOSSARY\\ncytochrome c:soluble protein of the mitochondrial intermembrane space, often found\\nloosely associated with the inner mitochondrial membrane. Cytochrome ctransports elec-\\ntrons between components of the electron transport chain. If it is allowed to escape from\\nmitochondria cytochrome cactivates caspase 9 and hence triggers apoptosis.\\ncytokinesis: process by which a cell divides in two; part of the M phase of the cell division\\ncycle.\\ncytology: study of cell structure by light microscopy.\\ncytoplasm: semiviscous ground substance of the cell. All the volume outside the nucleus\\nand inside the plasma membrane is cytoplasm.\\ncytoplasmic dynein: motor protein that moves organelles along microtubules in a retrograde\\ndirection.\\ncytoplasmic streaming: movement of cytoplasm that is commonly seen in plant cells and\\nin amoebae; generated by actin and myosin.cytosine: one of the bases present in DNA and RNA; cytosine is a pyrimidine.\\ncytoskeleton: cytoplasmic ﬁlament system consisting of microtubules, micro ﬁlaments, and\\nintermediate ﬁlaments.\\ncytosol: viscous, aqueous medium in which the organelles and the cytoskeleton are bathed.\\nDAG (diacylglycerol): two acyl groups (fatty acid chains) joined by ester bonds to a glycerol\\nbackbone. Diacylglycerol is produced by the action of phospholipase C on phospholipid\\nand helps to activate protein kinase C.\\ndark reactions: reactions occurring in plants that use ATP and reducing power provided\\nby NADPH to ﬁx atmospheric carbon dioxide and use it to make sugar.\\ndeamination: removal of an amino group. Deamination of cytosine to form uracil is a form\\nof DNA damage.death domain: domain found on proteins concerned with regulating apoptosis, such as Fas\\nand the p75 neurotrophin receptor. When activated, death domain proteins turn on caspase\\n8 and hence initiate apoptosis.\\ndenature: to cause to lose three-dimensional structure by breaking noncovalent intramolec-\\nular bonds.\\ndendrite: branching cell process. The term is commonly used to name those processes of\\nnerve cells that are too short to be called axons.deoxyribonucleic acid (DNA): polymer of deoxyribonucleotides. DNA speci ﬁes the in-\\nherited instructions of a cell.deoxyribonucleotide: building block of DNA made up of a nitrogenous base and the sugar\\ndeoxyribose to which a phosphate group is attached.\\ndeoxyribose: ribose that instead of —OH has only —H on carbon 2. Deoxyribose is the\\nsugar used in the nucleotides that make up DNA.\\ndepolarization: any positive shift in the transmembrane voltage, whatever its size or cause.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='cf9d6d2c-70c4-4e4a-83d4-6561ce6b1ce4', embedding=None, metadata={'page_label': '461', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 457\\ndepth of focus: distance toward and away from an object over which components of the\\nobject remain in clear focus.\\ndepurination: removal of either of the purine bases, adenine and guanine, from a DNA\\nmolecule. Depurination is a form of DNA damage.\\ndesmin: protein that makes up the intermediate ﬁlaments in muscle cells.\\ndesmosome: type of anchoring junction that joins the intermediate ﬁlaments of neighboring\\ncells. Desmosomes are common in tissues such as skin.\\ndiabetes: The word diabetes simply refers to conditions in which a patient produces lots\\nof urine. The only form of diabetes we discuss in this book is diabetes mellitus in which\\nthe patient produces large volumes of urine containing sugar. Diabetes mellitus is causedby either a failure in the endocrine gland that produces insulin or a failure of the tissues torespond adequately to insulin.\\ndiacylglycerol (DAG): two acyl groups (fatty acid chains) joined by ester bonds to a glycerol\\nbackbone. Diacylglycerol is produced by the action of phospholipase C on phospholipid\\nand helps to activate protein kinase C.\\ndideoxyribonucleotide: man-made molecule similar to a deoxyribonucleotide but lacking\\na3\\n′-hydroxyl group on its sugar residue. Used in DNA sequencing.\\ndifferentiation: process whereby a cell becomes specialized for a particular function.\\ndiffusion: movement of a substance that results from the individual small random thermal\\nmovements of its molecules.dimeric: formed of two parts. Of a molecule, formed of two parts that are not covalently\\nlinked.diphosphate: of a compound, bearing a chain of two phosphates in a line (as opposed to a\\nbisphosphate, which bears two independent phosphate groups). Adenosine diphosphate is\\nan example.\\ndiploid: containing two sets of chromosomes, in humans, this means two sets of 23, one\\nfrom the father, one from the mother. Most of the cells of the body (the somatic cells) are\\ndiploid.\\ndipolar bond: unusual form of covalent bond found in trimethylamine N-oxide.\\ndisaccharide: dimer of two monosaccharides. Examples are lactose (galactose β1→4\\nglucose) and sucrose (glucose α1→β2 fructose).\\ndistal: far from the center.\\ndisul ﬁde bond (bridge): covalent bond between two sulfur atoms. In proteins disul ﬁde\\nbonds form by oxidation of two thiol ( —SH) groups of cysteine residues. Found chie ﬂyi n\\nextracellular proteins.DNA (deoxyribonucleic acid): polymer of deoxyribonucleotides; DNA speci ﬁes the in-\\nherited instructions of a cell.DNAa: DNA binding protein that causes the two strands of the double helix to separate in\\ntheﬁrst stages of DNA replication.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1a2c70a7-5b94-4601-8d8c-7fea22bdd53c', embedding=None, metadata={'page_label': '462', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n458 GLOSSARY\\nDNAb: helicase that moves along a DNA strand, breaking hydrogen bonds, and in the\\nprocess unwinding the helix.\\nDNAc: DNA-binding protein that serves to bring DNAb to the DNA strands.\\nDNA chip: tiny glass wafer to which cloned DNAs are attached. Also known as gene chips\\nor microarrays.DNA excision: process of cutting out damaged DNA prior to repair.\\nDNA ﬁngerprint: individual pattern of DNA fragments determined by the number and\\nposition of speci ﬁc repeated sequences.\\nDNA ligase: enzyme that joins two DNA molecules by catalyzing the formation of a\\nphosphodiester bond.DNA polymerase: enzyme that synthesizes DNA by catalyzing the formation of a phos-\\nphodiester bond. DNA is always synthesized in the 5\\n′to 3′direction.\\nDNA repair enzymes: enzymes that detect and repair altered DNA.\\nDNA replication: process in which the two strands of the double helix unwind and each\\nacts as a template for the synthesis of a new strand of DNA.DNA sequencing: determining the order of bases on the DNA strand.\\ndocking protein (signal recognition particle receptor): receptor on the endoplasmic retic-\\nulum to which the signal recognition particle binds during the process of polypeptide chain\\nsynthesis and import into the endoplasmic reticulum.\\ndogma: belief. The “central dogma ”of molecular biology is that “DNA makes RNA makes\\nprotein ”—the concept that the sequence of bases on DNA de ﬁnes the sequence of bases\\non RNA, and the sequence of bases on RNA then de ﬁnes the sequence of amino acids on\\nprotein.domain: separately folded segment of the polypeptide chain of a protein.\\ndominant: gene that exerts its effect even when only one copy is present. Most dominant\\ngenes are dominant because they produce functional protein while the recessive gene does\\nnot. However, some mutant genes are dominant because even having 50% of one ’s protein\\nas the mutant form is enough to cause an effect; the gene causing familial Creutzfeldt –Jacob\\ndisease is an example.\\ndonor: that which gives. In a hydrogen bond, the donor is the atom (oxygen, nitrogen, or\\nsulfur) to which the hydrogen is covalently bonded and that gives up some of its share of\\nelectrons to a second electron-grabbing atom.\\ndouble helix: structure formed when two ﬁlaments wind about each other, most commonly\\napplied to DNA, but also applicable to, e.g., F-actin.downstream: general term meaning the direction in which things move. When applied\\nto the DNA within and adjacent to a gene, it means lying on the side of the transcription\\nstart site that is transcribed into RNA. When applied to signaling pathways it means in thedirection in which the signal travels, thus MAP kinase is downstream of Ras.\\nduplication: doubling-up of a particular sequence of the genetic material.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='531cfc8e-9fd1-47eb-b042-c7c85f01fae2', embedding=None, metadata={'page_label': '463', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 459\\ndynamic instability: term that describes the behavior of microtubules in which they switch\\nfrom phases of growth to phases of shortening.\\ndynamin: GTPase that plays a critical role in the formation of clathrin-coated vesicles.\\ndynein: motor protein. Cytoplasmic dynein moves vesicles along microtubules while dynein\\narms power ciliary and ﬂagellar beating by generating sliding between adjacent outer doublet\\nmicrotubules.E2F-1: transcription factor required for DNA synthesis. In quiescent cells E2F-1 is pre-\\nvented from activating transcription by being bound to RB, the product of the retinoblastoma\\ngene. E2F-1 is released when RB is phosphorylated by CDK4.\\nectoplasm: viscous, gel-like outer layer of cytoplasm.\\neffective stroke: part of the beat cycle of a cilium that pushes on the extracellular medium.\\neffector caspase: caspase that digests cellular components in the process of apoptosis.\\nEffector caspases are differentiated from caspases 8 and 9, which do not themselves digest\\ncellular components but which activate effector caspases by hydrolyzing particular peptidebonds in them.\\nelectrically excitable: able to produce action potentials.\\nelectrochemical gradient: free energy gradient for an ion in solution. The arithmetical sum\\nof gradients due to concentration and voltage.electron gun: source of electrons in electron microscopes.\\nelectron microscope: microscope in which the image is formed by passage of electrons\\nthrough, or scattering of electrons by, the object.electron transport chain: series of electron acceptor/donator molecules found in the inner\\nmitochondrial membrane which transport electrons from the reduced carriers NADH and\\nFADH\\n2(thus reoxidizing them) to oxygen. The entire complex forms a carrier that uses the\\nenergy of NADH and FADH 2oxidation to transport hydrogen ions up their electrochemical\\ngradient out of the mitochondrion.\\nelectrophoresis: method of separating charged molecules by drawing them through a ﬁl-\\ntering gel material using an electrical ﬁeld.\\nelectrostatic bond: a strong attraction between ions or charged groups of opposite charge.\\nelectrostatic interaction: attraction or repulsion between ions or charged groups.\\nelongation factors: proteins that speed up the process of protein synthesis at the ribosome.\\nElongation factor tu (EF-tu) is a GTPase.embryonic stem cell: cells derived from an early embryo. Embryonic stem cells have the\\ncapability to divide inde ﬁnitely and to become any cell type in the body.\\nendocytosis: inward budding of plasma membrane to form vesicles. Endocytosis is the pro-\\ncess by which cells retrieve plasma membrane and take up material from their surroundings.endonuclease: enzyme that digests nucleic acids by cleaving internal phosphodiester bonds.\\nendoplasm: ﬂuid, inner layer of cytoplasm that streams during cytoplasmic streaming.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='218548f3-3b09-4b75-971f-a1748a367881', embedding=None, metadata={'page_label': '464', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n460 GLOSSARY\\nendoplasmic reticulum (ER): network (reticulum) of membrane-delimited tubes and sacs\\nthat extends from the outer membrane of the nuclear envelope almost to the plasma mem-\\nbrane. There are two types of ER, rough endoplasmic reticulum (RER) with a surfacecoating of ribosomes and smooth endoplasmic reticulum (SER).\\nendosome: organelle to which newly formed endocytotic vesicles are translocated and with\\nwhich they fuse.\\nendosymbiotic theory: proposal that some of the organelles of eukaryotic cells originated\\nas free-living prokaryotes.endothelial cells: cells that line blood vessels and other body cavities that do not open to\\nthe outside.\\nendothelium: layer of cells that lines blood vessels and other body cavities that do not open\\nto the outside.\\nenergy currency: source of energy that the cell can use to drive processes that would\\notherwise not occur because their /Delta1Gis positive. Energy currencies can be coenzymes such\\nas ATP and NADH, which give up energy on conversion to, respectively, ADP and NAD\\n+,\\nor electrochemical ion gradients such as the hydrogen ion gradient across the mitochondrial\\nmembrane and the sodium gradient across the plasma membrane.\\nenhancer sequence: speci ﬁc DNA sequence to which a protein binds to increase the rate\\nof transcription of a gene.\\nenthalpy: thermodynamic description of heat.\\nentropy: degree of disorder in a system.\\nenvelope: closed sheet enclosing a volume. The term is used to describe, among other\\nthings, the double membrane layer enclosing certain organelles, and the outer membrane\\nof certain viruses.\\nenzyme: catalyst made mainly of protein. Like all catalysts, enzymes work by reducing the\\nactivation energy of the reaction.\\nepidermis: protective outer cell layer of an organism.\\nepithelial cells: cells that make up an epithelium.\\nepithelium: sheet of cells covering the surface of the human body and those internal cavities\\nsuch as the lungs and gut that open to the outside. In contrast endothelium lines internal\\nspaces such as blood vessels.\\nequilibrium: total balance of opposing forces. A process or object is in equilibium if the\\ntendency to go in one direction is exactly equal to the tendency to go in the other direction.\\nFor an ion this condition is equivalent to saying that the electrochemical gradient for thation across the membrane is zero. For a chemical reaction, equilibrium occurs when the rateof the forward reaction is equal to the rate of the reverse reaction, e.g., 2H\\n2O→←OH−+\\nH3O+.\\nequilibrium voltage: transmembrane voltage that will exactly balance the concentration\\ngradient of a particular ion.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='212b1c59-26a2-49f9-b868-e35aa9f19648', embedding=None, metadata={'page_label': '465', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 461\\nER (endoplasmic reticulum): network (reticulum) of membrane channels that extends\\nfrom the outer membrane of the nuclear envelope almost to the plasma membrane; there are\\ntwo types of ER, rough endoplasmic reticulum (RER) with a surface coating of ribosomesand smooth endoplasmic reticulum (SER).\\nES (embryonic stem) cell: cell derived from a very early embryo. ES cells have not\\nyet determined their developmental fate and can therefore, depending on the conditions,\\ngenerate the entire range of tissue types.\\nessential amino acid: amino acid that the organism needs but cannot synthesize. Histidine,\\nisoleucine, leucine, lysine, methionine, phenylalanine, threonine, tryptophan, and valine are\\nessential for humans.\\nessential fatty acid: fatty acid that the organism needs but cannot synthesize. Linoleic and\\nlinolenic acids are essential for humans.ester bond: bond formed between the hydrogen of an alcohol group and the hydroxyl of a\\ncarboxyl group by the elimination of water.euchromatin: that portion of the nuclear chromatin that is not tightly packed. Euchromatin\\ncontains genes that code for proteins that are being actively transcribed.eukaryotic: organism whose cells contain distinct nuclei and other organelles; includes all\\nknown organisms except prokaryotes (bacteria and cyanobacteria).exocytosis: fusion of a vesicle with the plasma membrane. Exocytosis causes the soluble\\ncontents of the vesicle to be released to the extracellular medium, while integral membrane\\nproteins of the vesicle become integral membrane proteins of the plama membrane.\\nexon: in a eukaryotic gene, exons are those parts that after RNA processing leave the\\nnucleus. In contrast introns are spliced out before the RNA leaves the nucleus.\\nexonuclease: enzyme that digests nucleic acids by cleaving phosphodiester links succes-\\nsively from one end of the molecule.expression (of a gene): appearance of the protein which the gene encodes.\\nexpression vector: cloning vector containing a promoter sequence recognized by the host\\ncell, thus enabling a foreign DNA insert to be transcribed into mRNA.extracellular matrix: meshwork of ﬁlaments and ﬁbers that surrounds and supports mam-\\nmalian cells. The major protein of the extracellular matrix is collagen.\\nextracellular medium: aqueous medium outside cells. For a unicellular organism the ex-\\ntracellular medium is the outside world. For a multicellular organism such as a human being\\nthe extracellular medium is the ﬂuid between the cells.\\nextragenic DNA: DNA that can neither be identi ﬁed as coding for protein or RNA nor as\\nbeing promoters or enhancers regulating transcription.\\nFas: metabotropic cell surface receptor that activates caspase 8 and hence triggers apoptosis.\\nfat:triacylglycerol (triglyceride) that is solid at room temperature. In contrast, oils are liquid\\nat room temperature.fat cells (adipocytes): cells that store fats (triacylglycerols).', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d5c13c22-8f7d-4b43-937e-efad6fbebd7e', embedding=None, metadata={'page_label': '466', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n462 GLOSSARY\\nfatty acid: carboxyl group attached to a long chain of carbon atoms with attached hydrogens,\\nthat is, a chemical of the general form\\nOCnHmCO H\\nfeedback: process in which the result of a process modi ﬁes the mechanisms carrying out\\nthat process to increase or decrease their rate. In negative feedback a change in some\\nparameter activates a mechanism that reverses the change in that parameter; an example isthe effect of tryptophan on expression of the trpoperon. In positive feedback a change in\\nsome parameter activates a mechanism that accelerates the change; an example is the effectof depolarization on the opening of voltage-gated sodium channels.\\nfeedforward: control process in which a change in a parameter is predicted, and mechanisms\\ninitiated that will act to reduce the change in that parameter.\\nferredoxin: protein component of the electron transport system of chloroplasts.\\nﬁbroblast: cell found in connective tissue. Fibroblasts synthesize collagen and other com-\\nponents of the extracellular matrix.\\nﬁbroblast growth factor: paracrine transmitter that opposes apoptosis and promotes\\ncell division in target cells. Although named for its effect on ﬁbroblasts, FGF triggers\\nproliferation of many tissues and plays critical roles in determining cell fate during\\ndevelopment.\\nﬁssion: breakage into two parts. The word is used for prokaryote replication. In addition\\nit describes the division of a single membrane-bound organelle into two and the process\\nwhereby a vesicle breaks away from a membrane.\\nﬂagellin: protein that is the bacterial ﬂagellum.\\nﬂagellum: swimming appendage. In eukaryotes ﬂagella are extensions of the cell that use a\\ndynein/microtubule motor system. In prokaryotes ﬂagella are extracellular proteins rotated\\nby a motor at their base.ﬂuid mosaic model: generally accepted hypothesis of how cell membranes are formed\\nfrom a lipid bilayer plus protein.fmet (formyl methionine): methionine modi ﬁed by the attachment of a formyl group; fmet\\nis the ﬁrst amino acid in all newly made bacterial polypeptides.\\nfocal contact: points at which a locomoting cell makes contact with its substrate.\\nformyl methionine (fmet): methionine modi ﬁed by the attachment of a formyl group; fmet\\nis the ﬁrst amino acid in all newly made bacterial polypeptides.\\nfourteen-three-three (14-3-3): protein that binds phosphorylated BAD, hence preventing\\nit from triggering apoptosis.\\nframeshift mutation: mutation that changes the mRNA reading frame, caused by the\\ninsertion or deletion of a nucleotide.\\nG (guanosine): guanine linked to the sugar ribose. Guanosine is a nucleoside.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='fc255241-0743-4b62-8f65-8c394cafdbf0', embedding=None, metadata={'page_label': '467', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 463\\nG0:describes the quiescent state of cells that have left the cell division cycle.\\nG1 (gap 1): period of the cell division cycle that separates mitosis from the following S\\nphase.\\nG2 (gap 2): period of the cell division cycle between the completion of S phase and the\\nstart of cell division or M phase.\\nGABA ( γ-amino butyric acid): γ-amino acid that acts to open chloride channels in the\\nplasma membrane of sensitive nerve cells. Usually called γ-amino butyric acid rather than\\nγ-amino butyrate, even though the latter (−OOC —CH 2—CH 2—CH 2—NH 3+) is the form\\nin which it is found at neutral pH.\\nG-actin: globular, subunit form of actin.\\ngamete: sperm or egg. Gametes are haploid; that is, they contain just one set of chromosomes\\n(23 in humans).gamma-amino butyric acid ( γ-amino butyric acid, GABA): γ-amino acid that acts to\\nopen chloride channels in the plasma membrane of sensitive nerve cells. Usually called\\nγ-amino butyric acid rather than γ-amino butyrate, even though the latter (\\n−OOC —CH 2—\\nCH 2—CH 2—NH 3+) is the form in which it is found at neutral pH.\\nGAP (GTPase activating protein): protein that speeds up the rate at which GTPases\\nhydrolyze GTP and therefore switch from the active to the inactive state.\\ngap 0 (G0): describes the quiescent state of cells that have left the cell division cycle.\\ngap 1 (G1): period of the cell division cycle that separates mitosis from the following S\\nphase.gap 2 (G2): period of the cell division cycle between the completion of S phase and the\\nstart of cell division or M phase.\\ngap junction: type of cell junction that allows solute to pass from the cytosol of one cell to\\nthe cytosol of its neighbor without passing through the extracellular medium. Gap junctions\\nconsist of many paired gap junction channels or connexons.\\ngap junction channel (connexon): channel in the plasma membrane with a central hole\\nabout 1.5 nm in diameter. Gap junction channels only open when they contact a second\\nchannel on another cell, in this case they open and form a water- ﬁlled tube that runs all the\\nway through the plasma membrane of the ﬁrst cell, across the small gap between the cells,\\nand through the plasma membrane of the second cell, so allowing passage of solute fromthe cytosol of one cell to the cytosol of the other.\\ngastrocnemius muscle: muscle at the back of the shin. When it contracts the toes move\\ndown.gated, gating: a channel is gated if it can switch to a shape in which the tube through the\\nmembrane is closed.\\ngated transport: transport of fully folded proteins through intracellular pores that open to\\nallow their passage.\\nGDP: guanosine diphosphate; guanosine with two phosphates attached to the 5\\n′carbon of\\nribose.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='da9fb167-040d-4943-8433-9d27992b12df', embedding=None, metadata={'page_label': '468', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n464 GLOSSARY\\nGEF (guanine nucleotide exchange factor): protein that accelerates the rate at which GDP\\nleaves a GTPase to be replaced by GTP, thus switching the GTPase from its inactive state\\nto its active state.\\nGEFS (generalized epilepsy with febrile seizures): relatively common form of childhood\\nepilepsy, usually clearing up spontaneously. GEFS+is the less common condition in which\\nthe seizures still occur past the age of 6 years.gelsolin: type of actin-binding protein that binds to and fragments actin ﬁlaments.\\ngene: fundamental unit of heredity. In many cases a gene contains the information needed\\nto code for a single polypeptide.\\ngene chip: tiny glass wafer to which cloned DNAs are attached. Also known as microarrays\\nor DNA chips.\\ngene family: group of genes that share sequence similarity and usually code for proteins\\nwith a similar function.gene probe: cDNA or genomic DNA fragment used to detect a speci ﬁc DNA sequence to\\nwhich it is complementary in sequence. The probe is tagged in some way to make it easy\\nto detect. The tag could be, for example, a radioactive isotope or a ﬂuorescent dye.\\ngene therapy: correction or alleviation of a genetic disorder by the introduction of a normal\\ngene copy into an affected individual.\\ngenetically modi ﬁed (GM): organism with a genome that has been modi ﬁed by modern\\nmolecular techniques, usually by the addition of novel gene(s) or by swopping in new DNA\\nto replace existing gene(s).\\ngenetic code: relationship between the sequence of the four bases in DNA and the amino\\nacid sequence of proteins.genome: complete set of genes in an organism.\\ngenomic DNA library: collection of bacterial clones each of which contains a different\\nfragment of foreign genomic DNA.germ cells: cells that give rise to the eggs and sperm.\\nGibbs free energy: if the change of Gibbs free energy during a reaction is negative, the\\nreaction is favored and will proceed if the activation energy is suf ﬁciently low.\\nglial cells: electrically inexcitable cells found in the nervous system.\\nglucocorticoid: steroid hormone produced by the adrenal cortex that forms part of the\\nsystem controlling blood sugar levels.glucocorticoid receptor: intracellular receptor to which glucocorticoid hormone binds.\\ngluconeogenesis: synthesis of glucose from noncarbohydrate precursors such as amino\\nacids and lactate.glucose: hexose monosaccharide. Glucose is the commonest sugar in the blood and is the\\ndominant cellular fuel in animals, being used in glycolysis to generate ATP and pyruvate,\\nthe latter fueling the Krebs cycle.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='11d19137-92f3-4e12-b4de-81da3c4f6e1f', embedding=None, metadata={'page_label': '469', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 465\\nglucose carrier: plasma membrane protein that carries glucose into or out of cells. Some\\ncells, such as skeletal muscle cells, will only translocate glucose carriers to their membranes\\nwhen protein kinase B is active.\\nglyceride: compound formed by attaching units to a glycerol backbone. Triacylglycerols\\n(previously called triglycerides) and phospholipids are glycerides.glycerol: CH\\n2OH—CHOH —CH 2OH. The backbone to which acyl groups (fatty acid\\nchains) are attached to make triacylglycerols and phospholipids.\\nglycogen: glucose polymer that can be quickly hydrolyzed to yield glucose; an α1→4\\npolymer of glucose with α1→6 branches.\\nglycogen phosphorylase: enzyme that releases glucose-1-phosphate monomers from\\nglycogen. The glucose-1-phosphate is then converted to glucose-6-phosphate, which can\\nbe used in respiration or dephosphorylated to glucose for release into the blood.\\nglycogen phosphorylase kinase (phosphorylase kinase): kinase that is activated by the\\ncalcium –calmodulin complex and phosphorylates glycogen phosphorylase, activating the\\nlatter enzyme.glycolysis: breakdown of glucose to pyruvate.\\nglycosidic bond: bond linking monosaccharide residues in which the carbon backbones\\nare linked through oxygen and a water molecule is lost.glycosylation: addition of sugar residues to a molecule. Both proteins and lipids can be\\nglycosylated.glyoxisome: type of peroxisome found in plant cells that is concerned with the conversion\\nof fatty acids to carbohydrate.glyoxylate: CHO —COO\\n−; an intermediate in various metabolic pathways including the\\nglyoxylate shunt.\\nglyoxylate shunt: pathway found in plants and bacteria that allows acetyl-CoA to be con-\\nverted into glucose.\\nGM (genetically modi ﬁed): organism with a genome that has been modi ﬁed by modern\\nmolecular techniques, usually by the addition of novel gene(s) or by swopping in new DNA\\nto replace existing gene(s).\\nGolgi apparatus: system of ﬂattened cisternae concerned with glycosylation and other\\nmodi ﬁcations of proteins.\\nG protein (trimeric G protein): protein that links a class of metabotropic cell surface\\nreceptors with downstream targets. Trimeric G proteins comprise an αsubunit that binds\\nand hydrolyzes GTP and a βγsubunit that dissociates from the αsubunit while the latter\\nis in the GTP-bound state. Important G proteins are G q, which activates phospholipase C β,\\nand G s, which activates adenylate cyclase.\\nGq:isoform of trimeric G protein that activates phospholipase C βand therefore generates\\na calcium signal.grana: distinctive structures within the chloroplast formed by the stacking of the thylakoid\\nmembranes.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='993bee87-7ccd-4bc6-8eb7-2e7e076f37bd', embedding=None, metadata={'page_label': '470', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n466 GLOSSARY\\ngratuitous inducer: inducer of transcription that is not itself metabolized by the resulting\\nenzymes.\\nGrb2 (growth factor receptor binding protein number 2): linker protein that has an SH2\\ndomain and is therefore recruited to phosphotyrosine, e.g., on receptor tyrosine kinases.\\nGrb2 in turn recruits SOS, bringing SOS to the plasma membrane where it can act as aguanine nucleotide exchange protein (GEF) for Ras.\\ngreen ﬂuorescent protein: ﬂuorescent protein made by the jelly ﬁshVictoria victoria .\\nUnlike other colored or ﬂuorescent proteins, it contains no prosthetic groups and therefore\\nwillﬂuoresce when expressed by any cell in which the gene is successfully inserted and\\nexpressed.growth factor: paracrine transmitter that modi ﬁes the developmental pathway of the target\\ncell, often by causing cell division.\\ngrowth factor receptor binding protein number 2 (Grb2): linker protein that has an SH2\\ndomain and is therefore recruited to phosphotyrosine, e.g., on receptor tyrosine kinases.\\nGrb2 in turn recruits SOS, bringing SOS to the plasma membrane where it can act as aguanine nucleotide exchange protein (GEF) for Ras.\\nG\\ns:isoform of trimeric G protein that activates adenylate cyclase and therefore causes an\\nincrease of cAMP concentration.GTP: guanosine triphosphate; guanosine with three phosphates attached to the 5\\n′carbon\\nof ribose.\\nGTPase: enzyme that hydrolyzes GTP. The name is usually restricted to that family of\\nproteins that bind GTP and adopt a new shape that can then activate target proteins. Once\\nthey hydrolyze the GTP they have bound, they switch back to the original form. Examplesare EF-tu, Arf, Ran, dynamin, G\\nq, and G s.\\nGTPase activating protein (GAP): protein that speeds up the rate at which GTPases\\nhydrolyze GTP and therefore switch from the active to the inactive state.\\nguanine: one of the bases found in DNA and RNA; guanine is a purine.\\nguanine nucleotide exchange factor (GEF): protein that accelerates the rate at which GDP\\nleaves a GTPase to be replaced by GTP, thus switching the GTPase from its inactive state\\nto its active state.\\nguanosine: guanine linked to the sugar ribose; guanosine is a nucleoside.\\nguanosine diphosphate (GDP): guanosine with two phosphates attached to the 5′carbon\\nof ribose.\\nguanosine triphosphate (GTP): guanosine with three phosphates attached to the 5′carbon\\nof ribose.\\nguanylate cyclase: An enzyme that converts GTP to the intracellular messenger cyclic\\nGMP (cGMP). One isoform of guanylate cyclase is activated by NO.Guthrie test: test for phenylketonuria. Newborn babies ’blood is tested for the presence of\\nphenylalanine at unusually high concentration.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='10aedf14-baf8-484e-aebc-3ba7c9e43cb1', embedding=None, metadata={'page_label': '471', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 467\\nhairpin loop: loop in which a linear object folds back on itself. Used to describe the loop\\nformed in a RNA molecule due to complementary base pairing.\\nhaploid: containing a single copy of each chromosome, in humans, this means 23 chro-\\nmosomes. Sperm and eggs are haploid while somatic cells contain one set of 23 from each\\nparent and are referred to as being diploid.\\nhead group: hydrophilic group found in phospholipids. The head group is attached to the\\nglycerol backbone by a phosphodiester link. Examples are choline and inositol.\\nheel prick test (Guthrie test): test for phenylketonuria. Newborn babies ’blood is tested\\nfor the presence of phenylalanine at unusually high concentration.\\nhelicase: enzyme that helps unwind the DNA double helix during replication.\\nαhelix: a common secondary structure in proteins, in which the polypeptide chain is coiled,\\neach turn of the helix taking 3.6 amino acid residues. The nitrogen atom in each peptide\\nbond forms a hydrogen bond with the oxygen four residues ahead of it in the polypeptidechain.\\nhemicellulose: polysaccharide component of the plant cell wall that links cellulose ﬁbrils\\ntogether.\\nhemoglobin: oxygen-carrying, iron-containing protein of the blood.\\nheterochromatin: that portion of the nuclear chromatin that is tightly packed. Much of the\\nheterochromatin is repetitive DNA with no coding function.\\nheterozygote: individual whose two copies of a gene differ, e.g., one may be mutant. We\\nhave not used the term in this book but have used the simpler word carrier for the more\\nrestricted case where an individual is a heterozygote for a recessive gene.hexokinase: enzyme that phosphorylates glucose on the number 6 carbon.\\nhistone: positively charged protein that binds to negatively charged DNA and helps to fold\\nDNA into chromatin.histone octamer: two molecules each of histones H2A, H2B, H3, and H4, the whole\\nforming a nucleosome.homologous recombination: process in which a length of DNA with ends that are ho-\\nmologous to a section of chromosome swops in, replacing the existing length of DNA in\\nthe chromosome. Homologous recombination occurs naturally at chiasmata during cross-ing over during meiosis. It can also occur in some cells when they transfected with theappropriate exogenous DNA.\\nhomozygote: individual whose two copies of a gene are identical. Most of us are homozy-\\ngous for most of our genes.\\nhormone: long-lived transmitter that is released into the blood and travels around the body\\nbefore being broken down.\\nhormone response element (HRE): speci ﬁc DNA sequence to which a steroid hormone\\nreceptor binds.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d72f312c-4f36-41d7-9b31-4e15ddb3cc0d', embedding=None, metadata={'page_label': '472', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n468 GLOSSARY\\nhousekeeping gene: gene that is transcribed into mRNA in nearly all the cells of a eukaryotic\\norganism; in bacterial cells a housekeeping gene is one that is always being transcribed.\\nHRE (hormone response element): speci ﬁc DNA sequence to which a steroid hormone\\nreceptor binds.hybridization: association of unlike things. In molecular genetics, the association of two\\nnucleic acid strands (either RNA or DNA) by complementary base pairing.\\nhydration shell: cloud of water molecules that surrounds an ion in solution.\\nhydrocarbon tail: long chain of carbon atoms with attached hydrogens found in phospho-\\nlipids and triacylglycerols. The tail represents all of a molecule of fatty acid except the\\ncarboxyl group.\\nhydrogen bond: relatively weak bond formed between a hydrogen atom and two elec-\\ntronegative atoms (such as nitrogen or oxygen) where the hydrogen is shared between the\\nother atoms.\\nhydrogen ion gradient: energy currency. Hydrogen ions are more concentrated outside\\nbacteria and mitochondria than inside, and this chemical gradient is supplemented by a\\nvoltage gradient pulling hydrogen ions in. If hydrogen ions are allowed to rush in downtheir electrochemical gradient, they release 17,000 J/mol.\\nhydrolysis: breakage of a covalent bond by the addition of water. —H is added to one side,\\n—OH to the other.\\nhydropathy plot: running average of side chain hydrophobicity along a polypeptide chain.\\nFrom the hydropathy plot one can predict, e.g., membrane-spanning domains in integral\\nmembrane proteins.\\nhydrophilic: molecule or part of a molecule that can interact with water.\\nhydrophobic: a molecule or part of a molecule that will associate with other hydrophobic\\nmolecules in preference to water.\\nhydrophobic effect: tendency of hydrophobic molecules or parts of molecules to cluster\\ntogether away from water, such as hydrophobic amino acid residues in the center of a\\nprotein, or the fatty acid chains in lipid bilayers.\\nhydroxyl group: —OH group. The term is speci ﬁcally notused for an —OH that forms\\npart of a carboxyl group —COOH.\\nhypoxanthine: purine that is used to make the nucleotide inosine. Inosine can pair with\\nany of uracil, cytosine, or adenine.imino acid: organic molecule containing both carboxyl ( —COOH) and imino ( —NH—)\\ngroups. Proline is an imino acid although it is usually called an amino acid.\\nimmuno ﬂuorescence: use of ﬂuorescently labeled antibodies to reveal the location of\\nspeci ﬁc chemicals, e.g., in ﬂuorescence microscopy or in western blotting.\\ninactivation (of voltage gated channels): long term closure, often because of blockage of\\nthe open channel with a plug that is attached to the cytosolic face of the protein.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='5525c3c8-1175-4fd9-9b9a-a0aef9e3b7b1', embedding=None, metadata={'page_label': '473', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 469\\nindirect immuno ﬂuorescence microscopy: form of light microscopy in which a ﬂuorescent\\nsecondary antibody is used to label a preapplied primary antibody speci ﬁc for a particular\\nprotein or subcellular structure. Also called secondary immuno ﬂuorescence microscopy.\\ninducible operon: operon that is transcribed only when a speci ﬁc substance is present.\\ninitial velocity (of a reaction): rate at which an enzyme converts substrate to product in\\nthe absence of product, that is, at the onset of a reaction.\\ninositol: cyclic polyalcohol, (CHOH) 6, that forms the headgroup of the phospholipid phos-\\nphatidylinositol. Phosphorylation of inositol yields inositol trisphosphate.inositol trisphosphate (IP\\n3, InsP 3):small phosphorylated cyclic polyalcohol that is re-\\nleased into the cytosol by the action of phospholipase C on the membrane lipid phos-\\nphatidylinositol bisphosphate and that acts to cause release of calcium ions from the endo-plasmic reticulum.\\ninositol trisphosphate-gated calcium channel: channel found in the endoplasmic reticu-\\nlum of many cells. The channel opens when inositol trisphosphate binds to its cytoplasmic\\naspect. It allows only calcium ions to pass.\\ninsertional mutagenesis: insertion of additional nucleotides into a stretch of DNA.\\nin situ hybridization: binding of a particular labeled sequence of RNA (or DNA) to its\\nmatching sequence in the genome as a way of searching for the location of that sequence\\nin a tissue sample.\\nInsP\\n3(inositol trisphosphate, IP 3):small phosphorylated cyclic polyalcohol that is re-\\nleased into the cytosol by the action of phospholipase C on the membrane lipid phos-\\nphatidylinositol bisphosphate and that acts to cause release of calcium ions from the endo-plasmic reticulum.\\ninsulin: hormone produced by endocrine cells in the pancreas. It activates its own receptor\\ntyrosine kinase, which in turn acts mainly through activation of PI-3-kinase and hence\\nprotein kinase B.\\ninsulin receptor: receptor tyrosine kinase speci ﬁc for insulin, which acts mainly through\\nactivation of PI-3-kinase and hence protein kinase B.\\ninsulin receptor substrate number 1 (IRS-1): protein phosphorylated on tyrosine by the\\ninsulin receptor. Once phosphorylated, it recruits PI-3-kinase, which can then be phospho-\\nrylated and hence activated.\\nintegral protein (of a membrane): class of protein that is tightly associated with a mem-\\nbrane, usually because its polypeptide chain crosses the membrane at least once. Integral\\nmembrane proteins can only be isolated by destroying the membrane, e.g., with detergent.In contrast, peripheral membrane proteins are more loosely associated.\\nintegrin: dimeric proteins with an extracellular domain that binds to extracellular matrix\\nproteins and an intracellular domain that attaches to actin micro ﬁlaments.\\nintermediate ﬁlament: one of the ﬁlaments that makes up the cytoskeleton; composed of\\nvarious subunit proteins.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='5644f2ba-7c33-4f18-9abc-a54bbd35efcf', embedding=None, metadata={'page_label': '474', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n470 GLOSSARY\\nintermembrane space: in organelles such as mitochondria, chloroplasts, and nuclei that\\nare bound by two membranes, the intermembrane space is the aqueous space between the\\ninner and outer membranes. The intermembrane space of nuclei is contiguous with thelumen of the ER. The intermembrane space of mitochondria has the ionic composition ofcytosol because porin in the outer mitochondrial membrane allows solutes of M\\nr<10,000\\nto pass.\\ninterphase: period of synthesis and growth that separates one cell division from the next;\\nconsists of the G1, S, and G2 phases of the cell division cycle.\\nintracellular messenger: cytosolic solute that changes in concentration in response to\\nexternal stimuli or internal events, and which acts on intracellular targets to change their\\nbehavior. Calcium ions, cyclic AMP and cyclic GMP are the three common intracellularmessengers.\\nintracellular receptors: receptors that are not on the plasma membrane but that lie within\\nthe cell and bind transmitters that diffuse through the plasma membrane.intron: in a eukaryotic gene, introns are those parts that are spliced out before the RNA\\nleaves the nucleus. In contrast exons are those parts that after RNA processing leave the\\nnucleus.\\nion:charged chemical species. A single atom that has more or less electrons than are required\\nto exactly neutralize the charge on the nucleus is an ion (e.g., Na\\n+,C l−). A molecule with\\none or more charged regions is also an ion (e.g., phosphate, HPO 42−; leucine,−OOC —\\nCH(CH 2.CH(CH 3)2)—NH 3+.\\nionotropic cell surface receptors: channels that open when a speci ﬁc chemical binds to\\nthe extracellular face of the channel protein.IP\\n3(inositol trisphosphate, InsP 3):small phosphorylated cyclic polyalcohol that is re-\\nleased into the cytosol by the action of phospholipase C on the membrane lipid phos-\\nphatidylinositol bisphosphate and that acts to cause release of calcium ions from the endo-plasmic reticulum.\\nIP\\n3receptor: calcium channel in the membrane of the endoplasmic reticulum that opens\\nwhen inositol trisphosphate binds to its cytosolic aspect.\\nIRS-1: protein phosphorylated on tyrosine by the insulin receptor. Once phosphorylated, it\\nrecruits PI-3-kinase, which can then be phosphorylated and hence activated.\\nisoelectric point: pH at which a protein or other molecule has no net charge.\\nisoforms: related proteins that are the products of different genes or of differential splicing\\nof mRNAs from one gene.isomers: different compounds with the same molecular formula. For example, glucose and\\nmannose are both C\\n6H12O6and are therefore isomers.\\nJNK: stress-activated protein kinase that stimulates cell repair but can also trigger apoptosis.\\nK+/Na+ATPase (sodium/potassium ATPase): plasma membrane carrier. For every ATP\\nhydrolyzed, three Na+ions are moved out of the cytosol and two K+ions are moved in.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='feb7071a-bfc6-4feb-85cf-486aabe01eaa', embedding=None, metadata={'page_label': '475', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 471\\nKartegener ’s syndrome: human condition in which the heart is on the wrong side. Male\\nsubjects may also be sterile. Caused by mutations in components of cilia and ﬂagella.\\nkeratin: protein that makes up the intermediate ﬁlaments in epithelial cells.\\nketone: any chemical containing a carbon atom with single bonds to two other carbons and\\na double bond to an oxygen. Acetone (CH 3—CO—CH 3) and acetoacetate (CH 3—CO—\\nCH 2—COO−) are ketones.\\nketone bodies: acetoacetate (CH 3—CO—CH 2—COO−) and 3-hydroxybutyrate (CH 3—\\nCHOH —CH 2—COO−); circulating fuels formed from fats and used to fuel body tissues\\nin times of starvation. Acetoacetate slowly loses carbon dioxide to form acetone.\\nketosis: overproduction of ketone bodies that is seen in extreme starvation and in diabetes.\\nkinase: enzyme that phosphorylates a molecule by transferring a phosphate group from\\nATP to the molecule.\\nkinesin: molecular motor protein responsible for movement along microtubules in the\\nanterograde direction.\\nkinetochore: point of attachment of the chromosome to the spindle. The kinetochore forms\\naround the centromere.\\nKM(Michaelis constant): substrate concentration at which one measures an initial velocity\\nthat is half as fast as the maximal velocity ( Vm) of an enzyme reaction.\\nKrebs cycle (tricarboxylic acid cycle): series of reactions in the mitochondrial matrix\\nin which acetate is completely oxidized to CO 2with the attendant reduction of NAD+to\\nNADH and FAD to FADH 2.\\nlac(lactose) operon: cluster of three bacterial genes that encode enzymes involved in\\nmetabolism of lactose.lactose: disaccharide comprising galactose linked to glucose by a β1→4 glycosidic bond.\\nlactose intolerant: unable to hydrolyze dietary lactose.\\nlactose ( lac) operon: cluster of three bacterial genes that encode enzymes involved in\\nmetabolism of lactose.lagging strand: strand of DNA that grows discontinuously during replication; in contrast\\nto the leading strand, which is synthesized continuously.\\nlamins: proteins that make up the nuclear lamina. Lamins are chemically related to inter-\\nmediate ﬁlaments.\\nleading strand: strand of DNA that grows continuously in the 5\\n′to 3′direction by the\\naddition of deoxyribonucleotides.ligand: when two molecules bind together, one (often the smaller one) is called the ligand,\\nand the other (often the bigger one) is called the receptor.\\nlight reactions: reactions in photosynthesis which use the energy of light to split water\\nto release oxygen, to produce NADPH, and to move hydrogen ions across the thylakoid\\nmembrane setting up a gradient which is then used to drive the formation of ATP from ADP.The NADPH and ATP are used in the dark reactions.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b58408f3-0342-4a70-b1f0-5a7348e88118', embedding=None, metadata={'page_label': '476', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n472 GLOSSARY\\nlinkage, linked (of genes): physical association of genes on the same chromosome. Linked\\ngenes tend to be inherited together.\\nlinker DNA: stretch of DNA that separates two nucleosomes.\\nlipid bilayer: two layers of lipid molecules that form a membrane.\\nlocalization sequence (targeting sequence): stretch of polypeptide that determines the\\ncellular compartment to which a synthesized protein is sent.lumen: inside of a closed structure or tube.\\nlysosome: membrane-bound organelle containing digestive enzymes.\\nM-phase: the period of the cell division cycle during which the cell divides; M phase\\nconsists of mitosis and cytokinesis.M-phase promoting factor (MPF): the complex of CDK1 and cyclin B that regulates the\\nG2/M phase transition of the cell division cycle.\\nmacromolecule: large molecule.\\nmacrophage: phagocytic housekeeping cell that engulfs and digests bacteria and dead cells.\\nmajor groove (of DNA): larger of the two grooves along the surface of the DNA double\\nhelix.mannose: hexose monosaccharide.\\nmannose-6-phosphate: mannose that is phosphorylated on carbon number 6. Mannose-6-\\nphosphate is the sorting signal that identi ﬁes lysosomal proteins.\\nMAPK, MAP kinase (mitogen-associated protein kinase): enzyme that phosphorylates\\nnumerous targets, including transcription factors that trigger transcription of the cyclin-D\\ngene.\\nMAPKK (MAP kinase kinase): enzyme that phosphorylates and activates MAP kinase,\\nit itself being phosphorylated and hence activated by MAPKK kinase.\\nMAPKKK (MAP kinase kinase kinase): enzyme that phosphorylates and activates MAPK\\nkinase. The most important isoform of MAPKKK is also called Raf. Raf is activated by the\\nG protein Ras.\\nmatrix: very vague term meaning a more or less closed location, often but not exclusively\\none that is a solid basis on which things can grow or attach. The term is used of the\\nextracellular matrix in animals, formed of collagen and other ﬁbers, and of the aqueous\\nvolume inside various organelles.\\nmaximal velocity (of an enzyme-catalyzed reaction): limiting value of the initial velocity\\nof a reaction as the substrate concentration is increased at constant enzyme concentration.\\nOccurs when the enzyme is saturated with substrate.\\nmeiosis: form of cell division that produces gametes, each with half the genetic material of\\nthe cells that produce them.\\nmeiosis I and II: ﬁrst and second meiotic divisions.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b7c67554-336f-4ea8-a873-002217ef835a', embedding=None, metadata={'page_label': '477', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 473\\nmeiotic spindle: bipolar, microtubule-based structure on which chromosome segregation\\noccurs during meiosis I and II.\\nmembrane: bilayer formed from lipid plus protein.\\nmessenger RNA (mRNA): RNA molecule that carries the genetic code. The order of bases\\non mRNA speci ﬁes the amino acid sequence of a polypeptide chain. In eukaryotes, the\\nmRNA leaves the nucleus and is translated into protein in the cytoplasm.metabolism: all of the reactions going on inside a cell.\\nmetabotropic cell surface receptors: receptors that are linked to, and activate, enzymes.\\nmetaphase: period of mitosis or meiosis at which the chromosomes align prior to separation\\nat anaphase.metaphase plate: equator of the mitotic or meiotic spindle; the point at which the chromo-\\nsomes congregate at metaphase of mitosis or meiosis.\\n7-methyl guanosine cap: modi ﬁed guanosine found at the 5\\n′terminus of eukaryotic mRNA.\\nA guanosine is attached to the mRNA by a 5′-5′-phosphodiester link and is subsequently\\nmethylated on atom number 7 of the guanine.\\nMichaelis constant ( KM):substrate concentration at which one measures an initial velocity\\nthat is half as fast as the maximal velocity ( Vm) of an enzyme reaction.\\nMichaelis –Menten equation: equation that de ﬁnes the effect of substrate concentration on\\nthe initial velocity of an enzyme-catalyzed reaction:\\nv0=Vm[S]\\nKM+[S]\\nwhere Vmis the limiting initial velocity obtained as the substrate concentration approaches\\ninﬁnity, [S] is the substrate concentration, and KMis a constant called the Michaelis constant,\\nequal to the substrate concentration at which one measures an initial velocity that is half as\\nfast as the maximal velocity.\\nmicroarray: tiny glass wafer to which cloned DNAs are attached. Also known as gene\\nchips or DNA chips.micro ﬁlament: one of the major ﬁlaments of the cytoskeleton; also known as actin ﬁlament\\nor F-actin; synonymous with the thin ﬁlament of striated muscle.\\nmicrosatellite DNA: repetitious DNA of unknown function comprising many repeats of a\\nunit of four or fewer base pairs.\\nmicrotubular molecular motor: protein that moves vesicles along microtubules; examples\\nare cytoplasmic dynein and kinesin.\\nmicrotubule: tubular cytoplasmic ﬁlament composed of tubulin. Tubulin is the major com-\\nponent of 9 +2 cilia and ﬂagella and of the mitotic and meiotic spindle.\\nmicrotubule organizing center (MTOC): structure from which cytoplasmic microtubules\\narise; synonymous with the centrosome.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='12f85e18-7e28-4e2e-9df0-be7900479b60', embedding=None, metadata={'page_label': '478', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n474 GLOSSARY\\nmicrovilli ( sing. microvillus): projections from the surface of epithelial cells that increase\\nthe absorptive surface; contain actin ﬁlaments.\\nminisatellite DNA: repetitious DNA of unknown function comprising up to 20,000 repeats\\nof a unit of about 25 base pairs.\\nminor groove (of DNA) : smaller of the two grooves along the surface of the DNA double\\nhelix.missense mutation: base change in a DNA molecule that changes a codon so that it now\\nspeci ﬁes a different amino acid.\\nmitochondrial inner membrane: inner membrane of mitochondria that is elaborated into\\ncristae. The electron transport chain, and ATP synthase, are integral membrane proteins of\\nthe mitochondrial inner membrane.\\nmitochondrial matrix: aqueous space inside the mitochondrial inner membrane, where\\nthe enzymes of the Krebs cycle are located.mitochondrial outer membrane: outer membrane of mitochondria, permeable to solutes\\nofM\\nr<10,000 because of the presence of the channel porin.\\nmitochondrion: cell organelle concerned with aerobic respiration.\\nmitogen: anything that promotes cell division. FGF and PDGF ( ﬁbroblast growth factor\\nand platelet-derived growth factor) are potent mitogens for endothelial and smooth muscle\\ncells.\\nmitogen-associated protein kinase (MAP kinase, MAPK): enzyme that phosphorylates\\nnumerous targets, including transcription factors that trigger transcription of the cyclin-D\\ngene.\\nmitosis: type of cell division found in somatic cells, in which each daughter cell receives\\nthe full complement of genetic material present in the original cell.\\nmitotic spindle: microtubule-based structure upon which chromosomes are arranged and\\ntranslocated during mitosis.mole: amount of substance comprising 6 .023×10\\n23(Avogadro ’s number) molecules. One\\nmole has a mass equal to the value of the relative molecular mass expressed in grams.\\nmonomer: single unit, usually used to refer to a single building block of a larger molecule.\\nThus DNA is formed of nucleotide monomers. By analogy, the word is sometimes used to\\ndescribe proteins that act as a single unit to distinguish them from related proteins that act aslarger units, so myoglobin is said to be monomeric by comparison with hemoglobin, whichhas four subunits. GTPases such as Ran, Arf, and Ras are sometimes called “monomeric G\\nproteins ”to distinguish them from trimeric G proteins such as G\\nqand G s.\\nmonosaccharide: sweet-tasting chemical with many hydroxyl groups that can adopt a form\\nin which an oxygen atom completes a ring of carbons. All the monosaccharides in this bookhave the general formula C\\nn(H2O)nwhere n=5 (pentoses) or 6 (hexoses).\\nmotif: recognizable conserved sequence of bases (in DNA) or amino acids (in a polypep-\\ntide). A motif in DNA may bind one transcription factor, e.g., 5′AGAACA3′binds the', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='031a7d19-43d2-4bae-b27a-043ddc40bbc5', embedding=None, metadata={'page_label': '479', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 475\\nglucocorticoid receptor. A motif in a polypeptide may, e.g., fold in a particular way (e.g., a\\nhelix-turn-helix motif), or bind to a target (e.g., KDEL).\\nmotoneuron (motoneurone): nerve cell that carries action potentials from the spine to the\\nmuscles. It releases the transmitter acetylcholine onto the muscle cells, causing them to\\ndepolarize and hence contract.\\nM phase: period of the cell division cycle during which the cell divides; consists of mitosis\\nand cytokinesis.\\nMPF (M-phase promoting factor): complex of CDK1 and cyclin B that regulates the\\nG2/M phase transition of the cell division cycle.\\nM-phase promoting factor (MPF): complex of CDK1 and cyclin B that regulates the\\nG2/M phase transition of the cell division cycle.M\\nr:Relative Molecular Mass (sometimes called “molecular weight ”and abbreviated M r\\nor RMM) is the mass of one mole of a substance compared to the mass of one mole of\\nhydrogen. It is dimensionless but is sometimes expressed in Daltons.\\nmRNA (messenger RNA): RNA molecule that carries the genetic code. The order of bases\\non mRNA speci ﬁes the amino acid sequence of a polypeptide chain.\\nMTOC (microtubule organizing center): structure from which cytoplasmic microtubules\\narise; synonymous with the centrosome.muscarinic acetylcholine receptor: integral membrane protein that binds acetylcholine\\nand then becomes a GEF for the trimeric G protein G\\nq, hence leading to the activation\\nof phospholipase C βwhich in turn leads to release of calcium ions from the endoplasmic\\nreticulum.\\nmuscle: tissue specialized for generating contractile force.\\nmutation: inherited change in the structure of a gene or chromosome.\\nmyelin: fatty substance that is wrapped around nerve cell axons by glial cells.\\nmyosin: type of motor protein that moves along, or pulls on, actin ﬁlaments. The thick\\nﬁlaments in skeletal muscle are formed of myosin II molecules.\\nmyosin II: class of myosin that forms the large ﬁlaments seen in striated muscle but that is\\nalso found in other cell types.myosin V: class of myosin that carries cargo along actin ﬁlaments.\\nN-terminal (amino terminal): the end of a peptide or polypeptide that has a free α-amino\\ngroup. This end is made ﬁrst on the ribosome.\\nNADH (nicotinamide adenine dinucleotide, reduced form): combination of two nu-\\ncleotides that is a strong reducing agent and one of the cell ’s energy currencies.\\nNADPH (nicotinamide adenine dinucleotide phosphate, reduced form): phosphorylated\\nNADH; like NADH it is a strong reducing agent, but NADPH is not an energy currency,\\nrather it is used in synthetic reactions in the cytoplasm.\\nNa\\n+/K+ATPase: plasma membrane carrier. For every ATP hydrolyzed, three Na+ions are\\nmoved out of the cytosol and two K+ions are moved in.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='989ae670-c73f-4461-973b-d5df7ff74caf', embedding=None, metadata={'page_label': '480', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n476 GLOSSARY\\nnecrosis: cell death that is due to damage so severe that the cell cannot maintain the level\\nof its energy currencies and therefore falls apart —distinct from apoptosis.\\nnegative feedback: control system in which a change in some parameter activates a mech-\\nanism that reverses the change in that parameter; an example is the effect of tryptophan on\\nexpression of the trpoperon.\\nnegative regulation (of transcription): inhibition of transcription due to the presence of\\na particular substance, which is often the end product of a metabolic pathway.\\nNernst equation: equation that allows the equilibrium voltage of an ion across a membrane\\nto be calculated. The general form is\\nVeq=RT\\nzFloge([Ioutside ]\\n[Iinside])\\nvolts\\nwhere Ris the gas constant (8.3 joules per degree per mole), Tis the absolute temperature,\\nzis the charge on the ion in elementary units, and Fis 96,500 coulombs in a mole of\\nmonovalent ions. At 37◦C the equation approximates to\\nVeq=62\\nzlog10([Ioutside ]\\n[Iinside])\\nmillivolts\\nnerve cell: electrically excitable cell with a long axon specialized for transmission of\\n(usually sodium) action potentials.nerve growth factor (NGF): older name for neurotrophin 1.\\nnervous tissue: tissue formed of nerve and glial cells that carries out electrical data pro-\\ncessing.neuro ﬁlament: type of intermediate ﬁlament found in nerve cells.\\nneuron: another name for a nerve cell.\\nneurone: yet another name for a nerve cell.\\nneurotrophin: one of a family of paracrine transmitters that act upon nerve cells to prevent\\napoptosis and trigger differentiation. Neurotrophin 1 used to be called nerve growth factor\\nor NGF.\\nNFAT: transcription factor that only translocates to the nucleus when dephosphorylated by\\nthe calcium –calmodulin activated phosphatase calcineurin.\\nNGF (nerve growth factor): older name for neurotrophin 1.\\nnicotinamide: base used in the double nucleotides NADH and NADPH.\\nnicotinamide adenine dinucleotide (reduced form) (NADH): combination of two nu-\\ncleotides that is a strong reducing agent and one of the cell ’s energy currencies.\\nnicotinamide adenine dinucleotide phosphate (reduced form) (NADPH): phosphory-\\nlated NADH; like NADH it is a strong reducing agent, but NADPH is not an energy currency,\\nrather it is used in synthetic reactions in the cytoplasm.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a2049344-fa7c-4ee0-8dc3-425246d151dd', embedding=None, metadata={'page_label': '481', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 477\\nnicotinic acetylcholine receptor: integral membrane protein of the plasma membrane that\\nforms an ion channel that allows both sodium and potassium ions to pass. When it binds\\nacetylcholine it opens, which in turn leads to electrical depolarization.\\nnine plus two (9 +2) axoneme: structure of cilium or ﬂagellum; describes the arrangement\\nof nine peripheral microtubules surrounding two central microtubules.nitric oxide (NO): paracrine transmitter that acts on intracellular receptors, the most im-\\nportant of which is guanylate cyclase.node (of a myelinated axon): gap between adjacent lengths of myelin sheath, where the\\nnerve cell plasma membrane directly contacts the extracellular medium. Also called node\\nof Ranvier, after its discoverer, Louis Ranvier.\\nnonpolar: covalent bonds in nonpolar molecules have electrons shared equally so that the\\nconstituent atoms do not carry a charge.nonsense mutation: base change that causes a codon that coded for an amino acid to\\nbecome a stop codon.northern blotting: blotting technique in which RNAs, separated by size, are probed using\\na single-stranded cDNA probe or an antisense RNA probe.N-terminal (amino terminal): end of a peptide or polypeptide that has a free α-amino\\ngroup. This end is made ﬁrst on the ribosome.\\nNTR: neurotrophin receptor. Although the Trk family are receptor tyrosine kinases for\\nneurotrophins, the name NTR is usually reserved for the death domain receptor called p75\\nNTR.\\nnuclear envelope: double-membrane system enclosing the nucleus; it contains nuclear\\npores and is continuous with the endoplasmic reticulum.nuclear lamina: meshwork of proteins lining the inner face of the nuclear envelope.\\nnuclear pores: holes running through the nuclear envelope that regulate traf ﬁc of proteins\\nand nucleic acids between the nucleus and the cytoplasm.nuclear pore complex: multiprotein complex that lies within and around the nuclear pore\\nand that regulates import into and export from the nucleus in a process called gated transport.nuclease: enzyme that degrades nucleic acids.\\nnucleic acid: polymer of nucleotides joined together by phosphodiester bonds; DNA and\\nRNA are nucleic acids.nucleoid: region of a bacterial cell that contains the chromosome.\\nnucleolar organizer regions: region of one or more chromosomes at which the nucleolus\\nis formed.\\nnucleolus: region(s) of the nucleus concerned with the production of ribosomes.\\nnucleoside: purine, pyrimidine, or nicotinamide attached to either ribose or deoxyribose.\\nnucleosome: beadlike structure formed by a stretch of DNA wrapped around a histone\\noctamer.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4fa95ba6-7300-47b9-83fb-b00940427ae6', embedding=None, metadata={'page_label': '482', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n478 GLOSSARY\\nnucleotide: building block of nucleic acids; a nucleoside that is phosphorylated on its 5′\\ncarbon atom.\\nnucleotide excision repair: process that removes a thymine dimer, together with some\\n30 surrounding nucleotides, from DNA. The gap is then repaired by the actions of DNA\\npolymerase I and DNA ligase.\\nnucleus: cell organelle housing the chromosomes; enclosed within a nuclear envelope.\\nobjective lens: lens of a light or electron microscope that forms a magni ﬁed image of a\\nspecimen.obligate anaerobe: organism that is poisoned by oxygen so that it can only function anaer-\\nobically.oil:triacylglycerol (triglyceride) that is liquid at room temperature. In contrast, fats are\\nsolid at room temperature.Okasaki fragment: series of short fragments that are joined together to form the lagging\\nstrand during DNA replication.oleic acid: C\\n17H33—COOH; a fatty acid with a single double bond. The dominant acyl\\ngroup in olive oil is derived from oleic acid.olfactory neuron: nerve cell found in the nose, whose dendrites are sensitive to smell\\nchemicals.oligonucleotide: short fragment of DNA or RNA.\\noligosaccharide: chain of up to 100 or so monosaccharides linked by glycosidic bonds.\\noligosaccharide transferase: enzyme that adds an oligosaccharide group onto a protein.\\noocyte: cell that undergoes meiosis to give rise to an egg.\\nopen complex: complex between DNA and protein that causes the two strands of DNA to\\nseparate, e.g., during DNA replication and transcription.open promoter complex: structure formed when the two strands of the double helix separate\\nso that transcription can commence.operator: DNA sequence to which a repressor protein binds to prevent transcription from\\nan adjacent promoter.\\noperon: cluster of genes that encode proteins involved in the same metabolic pathway and\\nthat are transcribed as one length of mRNA under the control of one promoter. As far as we\\nknow, operons are only found in prokaryotes.\\noptical isomers: two molecules that differ only in that one is the mirror image of the other.\\norganelle: membrane-bound, intracellular structure such as a mitochondrion, chloroplast,\\nlysosome, etc.organic: in chemistry, an organic compound is one that contains carbon atoms. When\\napplied to farming and food, organic means the avoidance of high-intensity agricultural\\ntechniques including genetic engineering.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a186ec43-8f53-4c6b-9a23-dd419bcc1f76', embedding=None, metadata={'page_label': '483', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 479\\norganism: cell or clone of cells that functions as a discrete and integrated whole to maintain\\nand replicate itself.\\norigin of replication: site on a chromosome at which DNA replication can commence.\\nosmolarity: overall strength of a solution; the more solute that is dissolved in a solution,\\nthe higher the osmolarity.\\nosmosis: movement of water down its concentration gradient.\\nouter doublet microtubules: paired microtubules that make the 9 of the 9 +2 axoneme\\nin cilia and ﬂagella.\\noxidation: removal of electrons from a molecule, e.g., by adding oxygen atoms, which tend\\nto take more than their fair share of electrons in any bonds they make. Removal of hydrogen\\natoms from a molecule oxidizes it. (Don ’t confuse with deprotonation, which is the loss\\nof H+).\\nβoxidation: process by which fatty acids are broken down into individual two-carbon units\\ncoupled to CoA to form acetyl-CoA. The process, which takes place in the mitochondrialmatrix, generates both NADH and FADH\\n2.\\noxidizing agent: agent that will act to lessen the share of electrons that the atoms of a\\nmolecule have. Oxidizing agents often work by adding oxygen atoms or by removinghydrogen atoms (Don ’t confuse with a base, which will accept H\\n+, not H). Examples are\\noxygen itself, NAD+, and FAD.\\nP site (peptidyl site): the site on a ribosome that is occupied by the growing polypeptide\\nchain.\\np21CIP1:inhibitor of cyclin-dependent kinases. p21CIP1, which is upregulated by cell –cell\\ncontact, prevents cells entering the S phase of the cell cycle.p38: a stress-activated protein kinase related to mitogen associated protein kinase (MAP\\nkinase) but with a very different role: p38 is activated by cell stress and stimulates both cell\\nrepair and apoptosis.\\np53: transcription factor that stimulates cell repair but also apoptosis. Cancer cells are\\nfrequently found to have mutated, nonfunctional p53 genes.\\np75 neurotrophin receptor: death domain receptor for neurotrophins. Upon binding neu-\\nrotrophins p75 activates caspase 8 and hence initiates apoptosis unless a countermanding\\nsignal to survive is present.\\nPAC (P1 arti ﬁcial chromosome): cloning vector, derived from the bacteriophage P1, that\\nis used to propagate, in E. coli, DNAs of about 150,000 bp.\\npain receptor: nerve cell whose distal axon terminal is depolarized by potentially damaging\\nevents such as heat or stretching.\\npain relay cell: this nerve cell, upon which a pain receptor synapses, carries the message\\non toward the brain.\\nparacrine transmitters: agonists released by cells into the extracellular ﬂuid that can last\\nmany minutes and can therefore diffuse widely within the tissue before they are destroyed.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='935ba785-9139-476a-8482-5fe2a5ea9696', embedding=None, metadata={'page_label': '484', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n480 GLOSSARY\\nparallel βsheet: βsheet in which all the parallel polypeptide chains run in the same\\ndirection.\\nparenchyma cells: unspecialized plant cells whose cell walls are usually thin and bendable;\\nthe major site of metabolic activity and photosynthesis in leaves and green shoots.\\npatch clamp: technique in which a glass micropipette is sealed to the surface of a cell to\\nallow electrical recording of cell properties.\\nPCR (polymerase chain reaction): method for making many copies of a DNA sequence\\nwhere the sequence, at least at each end, is known.\\nPDGF (platelet-derived growth factor): paracrine transmitter that opposes apoptosis and\\npromotes cell division in target cells such as endothelial cells and smooth muscle.pectin: polysaccharide component of the plant cell wall; the amount of pectin in the cell\\nwall determines its thickness.\\npentose: monosaccharide with ﬁve carbon atoms.\\npentose phosphate pathway: pathway that makes pentoses and generates reducing power\\nin the form of NADPH by oxidizing glucose-6-phosphate.peptide: short linear polymer of amino acids.\\npeptide bond: bond between amino acids. The bond is formed between the carboxyl group\\nof one amino acid and the amino group of the next.peptidyl site (P site): site on a ribosome that is occupied by the growing polypeptide chain.\\npeptidyl transferase: enzyme that catalyzes the formation of a peptide bond between two\\namino acids.peripheral membrane protein: class of protein that is easily detached from a cell mem-\\nbrane, unlike integral membrane proteins, which can only be isolated by destroying the\\nmembrane, e.g., with detergent.\\nperoxisome: class of cell organelles of diverse function. Peroxisomes frequently contain\\nthe enzyme catalase, which breaks down hydrogen peroxide into oxygen and water.pH: measure of the acidity of a solution, equal to minus the logarithm to base 10 of the\\nhydrogen ion concentration in moles liter\\n−1. The smaller the pH value, the more acid the\\nsolution. A neutral solution has a pH of 7, that is, the H+concentration is 10−7mol liter−1\\nor 100 nmol liter−1.\\nPH domain: protein domain that binds phosphorylated inositols. The PH group on protein\\nkinase B is recruited to the membrane by the intensely charged lipid phosphatidylinositol\\ntrisphosphate.\\nphage: short for bacteriophage, i.e., a virus that infects bacterial cells.\\nphase-contrast microscopy: type of light microscopy in which differences in the refractive\\nindex of a specimen are converted into differences in contrast.PH domain: protein domain that binds phosphorylated inositols. The PH group on protein\\nkinase B is recruited to the membrane by the intensely charged lipid phosphatidylinositol\\ntrisphosphate.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2f7d17ba-598a-4454-8384-a063dea91ada', embedding=None, metadata={'page_label': '485', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 481\\nphenylketonuria: inherited disease in which the enzyme phenylalanine hydroxylase is\\nmissing or defective. Unless intake of phenylalanine is drastically curtailed, phenylala-\\nnine and its transamination product phenylpyruvate build up in the body and cause braindamage. The heel prick or Guthrie test, performed on all newborn babies, tests for thiscondition.\\nphloem: part of the plant vascular system; conveys the products of photosynthesis to their\\nsites of use or storage.\\nphosphatase: enzyme that removes phosphate groups from substrates. Glucose-6-\\nphosphate phosphatase and calcineurin are phosphatases operating on a sugar and on pro-\\nteins, respectively.\\nphosphate: properly, a name for the ions H\\n2PO4−, HPO 42−, and PO 43−. The word is also\\nvery commonly used to mean the group\\nO–O–\\nOPO\\nand we have used this convention in this book. If one wants to speci ﬁcically describe the\\ngroup, notthe ions, one can refer to a phosphoryl group (a term that does not include the\\nleftmost oxygen in the diagram above).phosphatidylinositol bisphosphate (PIP\\n2):membrane lipid. PIP 2releases inositol trispho-\\nsphate into the cytosol upon hydrolysis by phospholipase C. Alternatively, PIP 2can be\\nfurther phosphorylated to yield phosphatidylinositol trisphosphate.phosphatidylinositol trisphosphate (PIP\\n3):a membrane lipid. Its intensely charged head-\\ngroup can recruit proteins that contain PH domains to the plasma membrane.\\nphosphodiester link: link between two parts of a molecule in which a phosphate is attached\\nthrough an oxygen atom to each of the two parts. In phospholipids, the headgroup is attached\\nto glycerol by a phosphodiester link. In DNA and RNA, successive nucleotides are joinedtogether by phosphodiester links.\\nphosphofructokinase: enzyme that phosphorylates fructose-6-phosphate to generate\\nfructose-1,6-bisphosphate.phosphoinositide 3-kinase (PI-3-kinase): enzyme that phosphorylates phosphatidylinos-\\nitol bisphosphate on the 3 position of the inositol ring to generate the intensely charged\\nlipid phosphatidylinositol trisphosphate (PIP\\n3). PIP 3can then recruit proteins containing\\nPH domains to the plasma membrane.\\nphospholipase C: enzyme that hydrolyzes the membrane lipid phosphatidylinositol bis-\\nphosphate (PIP 2) to release inositol trisphosphate into the cytosol. The βisoform is\\nactivated by the trimeric G protein G q, while the γisoform is activated by tyrosine\\nphosphorylation.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6429f0f1-0a4d-4ec3-b700-eee56ddf3b57', embedding=None, metadata={'page_label': '486', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n482 GLOSSARY\\nphosphoryl group:\\nO–O–\\nPO\\nphosphorylase: enzyme that cleaves a glycosidic bond by adding phosphate.\\nphosphorylase kinase (glycogen phosphorylase kinase): kinase that is activated by the\\ncalcium –calmodulin complex and which phosphorylates glycogen phosphorylase, activat-\\ning the latter enzyme.\\nphosphorylated: having had a phosphate group added. The phosphate groups are usually\\nsubstituted into a hydroxyl group to form a phosphoester bond but are sometimes substi-\\ntuted into an acid to form a phosphoanhydride or attached to a nitrogen atom to form aphosphoimide.\\nphosphorylation: addition of a phosphate group to a molecule. Usually substituted into a\\nhydroxyl group to form a phosphoester bond. Sometimes substituted into an acid to form a\\nphosphoanhydride, or attached to a nitrogen atom to form a phosphoimide.\\nphotosynthesis: synthesis of complex organic molecules, and oxidation of water to release\\noxygen, that is driven by the energy of light.phragmoplast: structure associated with the formation of a new cell wall during cytokinesis\\nin plant cells.PI 3-kinase (phosphoinositide 3-kinase): enzyme that phosphorylates phosphatidylinos-\\nitol bisphosphate on the 3 position of the inositol ring to generate the intensely charged\\nlipid phosphatidylinositol trisphosphate (PIP\\n3). PIP 3can then recruit proteins containing\\nPH domains to the plasma membrane.\\nPIP 2(phosphatidylinositol bisphosphate): membrane lipid that releases inositol trispho-\\nsphate into the cytosol upon hydrolysis by phospholipase C.pK\\na:parameter equal to −log10Kaand representing the pH at which the concentration of\\nthe dissociated acid is equal to the concentration of the undissociated acid.\\nPKA (protein kinase A; cAMP-dependent protein kinase): protein kinase that is activated\\nby the intracellular messenger cyclic AMP. PKA phosphorylates proteins (e.g., glycogen\\nphosphorylase kinase) on serine and threonine residues.\\nPKB (protein kinase B): protein kinase that is activated when it is itself phospho-\\nrylated; this in turn only occurs when PKB is recruited to the plasma membrane\\nby phosphatidylinositol trisphosphate (PIP 3). PKB phosphorylates proteins on serine\\nand threonine residues (e.g., the bcl-2 family protein BAD). An older name for PKBis Akt.\\nPKC (protein kinase C): protein kinase that is activated by a rise of cytosolic calcium\\nconcentration; it is also activated by diacylglycerol. PKC phosphorylates proteins on serine\\nand threonine residues.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1529de30-45c9-4f74-bbb0-a10b42ab74b0', embedding=None, metadata={'page_label': '487', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 483\\nPKG (protein kinase G; cGMP-dependent protein kinase): protein kinase that is ac-\\ntivated by the intracellular messenger cyclic GMP. PKG phosphorylates proteins (e.g.,\\ncalcium ATPase) on serine and threonine residues.\\nplaque: area of dead bacteria in a lawn of live bacteria that is caused by infection of bacterial\\ncells by bacteriophages.plasmalemma: membrane that surrounds the cell .Also called the plasma membrane or the\\ncell membrane.plasma membrane: membrane that surrounds the cell .Also called the plasmalemma or\\nthe cell membrane.\\nplasmid: circular DNA molecule that is replicated independently of the host chromosome\\nin bacterial cells.\\nplasmodesmata ( singular, plasmodesma): type of cell junction unique to plant cells, which\\nprovides a much bigger hole for passage of substances between the cytoplasm of the two\\ncells than gap junctions do.\\nplastoquinone: quinone electron carrier found in chloroplasts.\\nplatelet: small fragment of cell that contains no nucleus but that has a plasma membrane\\nand some endoplasmic reticulum. Platelets are critical in the process of blood clotting, and\\nthey also release platelet-derived growth factor.\\nplatelet-derived growth factor (PDGF): paracrine transmitter that opposes apoptosis and\\npromotes cell division in target cells such as endothelial cells and smooth muscle.\\nPLC (phospholipase C): enzyme that hydrolyzes the membrane lipid phosphatidylinos-\\nitol bisphosphate (PIP\\n2) to release inositol trisphosphate into the cytosol. The βisoform\\nis activated by the trimeric G protein G q, while the γisoform is activated by tyrosine\\nphosphorylation.polar: having covalent bonds in which the electrons are unequally shared, so that atoms\\nhave partial charges. Polar molecules can interact with water by electrostatic interactions\\nand by hydrogen bonding.\\npoly adenosine tail: string of adenine residues added to the 3\\n′end of a eukaryotic mRNA.\\npolyadenylation: process whereby a poly-A tail is added to the 3′end of a eukaryotic\\nmRNA.\\npoly-A tail: string of adenine residues added to the 3′end of a eukaryotic mRNA.\\npolycistronic mRNA: mRNA that, when translated, yields more than one polypeptide.\\npolymer: chemical composed of a long chain of identical or similar subunits.\\npolymerase: enzyme that makes polymers, i.e., long chains of identical or very similar\\nsubunits. DNA and RNA polymerase, respectively, are involved in making DNA and RNA.polymerase chain reaction (PCR): method for making many copies of a DNA sequence\\nwhen the sequence, at least at each end, is known.\\npolynucleotide kinase: enzyme that adds a phosphate group to the 5\\n′end of DNA or RNA.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='70549eaf-2181-404a-929b-dc047624e439', embedding=None, metadata={'page_label': '488', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n484 GLOSSARY\\npolypeptide: polymer of more than 50 amino acids joined by peptide bonds.\\npolyploid: having three or more sets of chromosomes.\\npolyribosome ( orpolysome): chain of ribosomes attached to an mRNA molecule.\\npolysaccharide: chain of more than 100 or so monosaccharides linked by glycosidic bonds.\\npolysome ( orpolyribosome): chain of ribosomes attached to an mRNA molecule.\\npolyunsaturated: having several double bonds. Usually applied to fatty acids; triacylglyc-\\nerols with polyunsaturated acyl (fatty acid) groups are liquid even at low temperatures.\\nporin: channel found in the mitochondrial outer membrane. It spends most of its time open\\nand allows all solutes of Mr<10,000 to pass.\\npositive feedback: process in which the consequences of a change act to increase the\\nmagnitude of that change, so that a small initial change tends to get bigger and bigger.\\npositive regulation (of transcription): process whereby transcription is activated in the\\npresence of a particular substance.\\npostsynaptic cell: cell upon which a nerve cell releases its transmitter at a synapse.\\npotassium channel: channel in the plasma membrane of many cells that allows potassium\\nions to pass.potassium/sodium ATPase (Na\\n+/K+ATPase): plasma membrane carrier. For every ATP\\nhydrolyzed, three Na+ions are moved out of the cytosol and two K+ions are moved in.\\npreinitiation complex: complex formed between transcription factors and RNA polymerase\\nat the promoter of a eukaryotic gene.presynaptic terminal: region of an axon terminal that is specialized for exocytosis of\\ntransmitter.\\nprimary antibody: ﬁrst antibody applied to the preparation. In many cases the primary\\nantibody is not itself labeled but must be revealed by applying a labeled secondary antibody.\\nprimary cell wall: ﬁrst layer of the plant cell wall.\\nprimary endosome: acidic cell compartment with which coated vesicles fuse.\\nprimary immuno ﬂuorescence: technique in which the location or presence of a chemical\\nis revealed by treating the sample with a dye-labeled antibody.primary structure (of a protein): sequence of amino acids held together by peptide bonds\\nmaking up a polypeptide.\\nprimase: enzyme that synthesizes the RNA primers needed for the initiation of synthesis\\nof the leading and lagging DNA strands.\\nprimer: short sequence of nucleic acid (RNA or DNA) that acts as the start point at which\\na polymerase can initiate synthesis of a longer nucleic acid chain.\\nprimosome: protein complex, including the enzyme primase, that is involved in the syn-\\nthesis of RNA primers for DNA replication.\\nprion: protein that is itself an infective agent for a disease.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ebf2cc5b-1f59-4032-b00c-91a91e4778ad', embedding=None, metadata={'page_label': '489', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 485\\nproﬁlin:type of actin-binding protein that regulates the assembly of actin ﬁlaments.\\nprogenitor: ancestor; an individual from which others are descended. In this book we use\\nthe term progenitor cell to mean the cell that divides to give rise to two other cells (in\\nmitosis) or four others (in meiosis).\\nprogeny: children, offspring. In this book we use the term progeny cells to mean the products\\nof cell division.\\nprogrammed cell death (apoptosis): process in which a cell actively promotes its own\\ndestruction, as distinct from necrosis. Apoptosis is important in vertebrate development,\\nwhere tissues and organs are shaped by the death of certain cell lineages.\\nprojector lens: lens of light or electron microscope that carries the image to the eye; more\\ncommonly known as the “eyepiece. ”\\nprokaryotic: type of cellular organization found in bacteria in which the cells lack a distinct\\nnucleus and other organelles.\\nprometaphase: period of mitosis or meiosis that sees the breakdown of the nuclear envelope\\nand the attachment of the chromosomes to the mitotic spindle.\\npromoter: region of DNA to which RNA polymerase binds to initiate transcription.\\npronuclei: nuclei of the egg and sperm prior to fusion.\\nprophase: period of mitosis or meiosis in which the chromosomes condense.\\nprophase I: ﬁrst prophase of meiosis.\\nprophase II: second prophase of meiosis.\\nprosthetic group: nonprotein molecule necessary for the activity of a protein. The concept\\noverlaps with the concept of cofactor. The difference is just how tightly they are bound:\\na prosthetic group is very tightly bound and cannot be removed without at least partialunfolding of the protein. Examples are the heme groups of myoglobin, hemoglobin, andthe cytochromes.\\nprotein: polypeptide (a polymer of α-amino acids) that has a preferred way of folding.\\nprotein engineering: designing a novel protein and then causing it to be built by altering\\nthe sequence of DNA.\\nprotein kinase: enzyme that phosphorylates a protein by transferring a phosphate group\\nfrom ATP to the molecule.\\nprotein kinase A: protein kinase that is activated by the intracellular messenger cAMP.\\nProtein kinase A phosphorylates proteins (e.g., glycogen phosphorylase kinase) on serine\\nand threonine residues.\\nprotein kinase B (PKB): protein kinase that is activated when it is itself phosphorylated; this\\nin turn only occurs when PKB is recruited to the plasma membrane by phosphatidylinositol\\ntrisphosphate (PIP\\n3). PKB phosphorylates proteins on serine and threonine residues (e.g.,\\nthe bcl-2 family protein BAD). An older name for PKB is Akt.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='46fb8ba0-cebc-429f-8075-3672a5dc14f1', embedding=None, metadata={'page_label': '490', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n486 GLOSSARY\\nprotein kinase C (PKC): protein kinase that is activated by a rise of cytosolic calcium\\nconcentration; it is also activated by diacylglycerol. PKC phosphorylates proteins on serine\\nand threonine residues.\\nprotein kinase G (PKG; cGMP-dependent protein kinase): protein kinase that is ac-\\ntivated by the intracellular messenger cyclic GMP. PKG phosphorylates proteins (e.g.,\\ncalcium ATPase) on serine and threonine residues.\\nprotein phosphatase: enzyme that removes phosphate groups from proteins.\\nprotein phosphorylation: addition of a phosphate group to a protein. The addition of\\nthe charge on the phosphoryl group can markedly alter the tertiary structure and therefore\\nfunction of a protein.\\nprotein targeting: delivery of proteins to their correct cellular location.\\nprotein translocator: channel-like protein of the endoplasmic reticulum that allows\\npolypeptide chains to cross the membrane as they are synthesized.\\nproteoglycans: heavily glycosylated proteins that contribute to the extracellular matrix.\\nproteome: complete protein content of the cell.\\nproteomics: study of the proteome. For example, one might compare the protein pro ﬁles\\nof cells in two tissues.\\nproto ﬁlaments: chains of subunits that make up the wall of a microtubule.\\nprotonated: having accepted an H+. For example, the lactate ion, CH 3CH(OH)COO−, will\\nbecome protonated in acid solutions to become lactic acid, CH 3CH(OH)COOH. The word\\nproton is being used as shorthand for hydrogen atom nucleus since, for the commonest\\nisotope of hydrogen, the nucleus is a single proton.proximal: close to the center.\\npseudogene: gene that has mutated such that it no longer codes for a protein.\\npseudopodium: projection extended by an amoeba or other crawling cell in the direction\\nof movement.P site (peptidyl site): site on the ribosome occupied by the growing polypeptide chain.\\npurine: nitrogenous base found in nucleotides and nucleosides; adenine, guanine and hy-\\npoxanthine are purines.pyrimidine: nitrogenous base found in nucleotides and nucleosides; cytosine, thymine, and\\nuracil are pyrimidines.\\npyrophosphate:\\nOHO\\n–OPO PO–\\nO–O', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='221d0dc9-8d0c-4d47-94d8-2e683896c687', embedding=None, metadata={'page_label': '491', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 487\\nthe pyrophosphate ion is stable if kept away from biological tissue but is rapidly hydrolyzed\\nto two phosphate ions by intracellular and extracellular enzymes.\\nquaternary structure: structure in which subunits of a protein, each of which has a tertiary\\nstructure, associate to form a more complex molecule. The subunits associate tightly, but\\nnot covalently and may be the same or different.\\nquiescent: at rest. Nondividing cells are often said to be quiescent.\\nRab family: family of GTPases that mediate fusion of vesicles with other membranes.\\nRaf: isoform of MAPKKK, the enzyme at the top of the mitogen-associated protein kinase\\ncascade.Ras: GTPase that activates the MAP kinase pathway and hence promotes cell division. Ras\\nis activated by SOS, its GTP exchange factor (GEF), which is in turn recruited to the plasma\\nmembrane via receptor tyrosine kinases and the adaptor protein Grb2.\\nRB: protein that binds to the transcription factor E2F-1 and therefore prevents transcription\\nof proteins required for DNA synthesis. Phosphorylation of RB by CDK4 causes it to release\\nE2F-1, allowing entry to S phase. Mutations in RB lead to the formation of the eye cancercalled retinoblastoma.\\nreading frame: reading of the genetic code in blocks of three bases —there are three possible\\nreading frames for each mRNA only one of which will produce the correct protein.\\nreceptor: protein that speci ﬁcally binds a particular solute. Receptors can be transmem-\\nbrane, cytosolic, or nuclear proteins. Particular receptor proteins perform additional func-\\ntions (ion channel, enzyme, activator of endocytosis, transcription factor . . . ) that are acti-vated by the binding of the solute.\\nreceptor-mediated endocytosis: process in which ligands bind to speci ﬁc receptors in the\\nplasma membrane and trigger clathrin-mediated vesicle budding.\\nreceptor tyrosine kinase: integral membrane protein with a binding site for transmitter on\\nthe extracellular aspect and tyrosine kinase catalytic ability on the cytosolic aspect. Binding\\nof transmitter causes self-phosphorylation on tyrosine and hence recruitment of proteinswith SH2 domains such as Grb2, phospholipase C γ, and phosphoinositide 3-kinase, which\\nmay then themselves be phosphorylated. The PDGF receptor, the FGF receptor, the Trkfamily of receptors, and the insulin receptor are all receptor tyrosine kinases.\\nrecessive: gene is recessive if its effects are hidden when the organism posesses a second,\\ndominant, version. Recessive genes usually code for nonfunctional proteins, so that if the\\nindividual can make the protein using the other, working gene, no effects are seen.\\nrecombinant DNA: artiﬁcial DNA molecule formed of DNA from two or more different\\nsources.recombinant plasmid: plasmid into which a foreign DNA sequence has been inserted.\\nrecombinant protein: protein expressed from the foreign DNA inserted into a recombinant\\nplasmid or other cloning vector. Recombinant proteins are often expressed in bacteria, yeast,\\ninsect, or mammalian cells.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4f0bd29a-a951-4be2-bdfa-b5013b9fc057', embedding=None, metadata={'page_label': '492', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n488 GLOSSARY\\nrecombination: “cut-and-paste ”of DNA. Recombination occurs at chiasmata during meio-\\nsis and also in embryonic stem cells, allowing insertional mutagenesis.\\nrecovery stroke: part of the beat cycle of a cilium in which the cilium is moved back into a\\nposition where it can push again; used in contrast to the effective stroke that generates the\\nforce.\\nreduction: addition of electrons to a compound, e.g., by the addition of hydrogen atoms or\\nthe removal of oxygen atoms.refractive index: measure of the capacity of any material to slow down the passage of light.\\nregulated secretion: secretion that only occurs in response to a signal, such as a rise in the\\ncytosolic concentration of calcium ions.\\nRelative Molecular Mass: mass of one mole of a substance compared to the mass of\\none mole of hydrogen. It is abbreviated M\\nror RMM and is sometimes called “molecular\\nweight. ”It is dimensionless but is sometimes expressed in Daltons.\\nrepetitious DNA: DNA sequence that is repeated many times within the genome.\\nreplication (of DNA): The process whereby two DNA molecules are made from one.\\nreplication fork: Y-shaped structure formed when the two strands of the double helix\\nseparate during replication.\\nrepressible operon: operon whose transcription is repressed in the presence of a particular\\nsubstance; often the ﬁnal product of the metabolic pathway.\\nRER (rough endoplasmic reticulum): portion of the endoplasmic reticulum associated\\nwith ribosomes and concerned with the synthesis of secreted proteins. Proteins destined to\\nremain within the majority of single-membrane-bound organelles (Golgi, lysosomes, . . . )are also made on the rough ER, as are integral proteins of these organelles and of the plasmamembrane.\\nresidue: that which is left over. In chemistry, when a molecule is built into a larger molecule\\nwith the loss of some part, the part that remains and forms part of the larger molecule is\\ncalled a residue. For example, an amino acid that has been built into a polypeptide, with theloss of the elements of water, is said to be an amino acid residue.\\nresolving power: measure that de ﬁnes the smallest object that can be distinguished using\\na microscope.\\nresting voltage: voltage across the plasma membrane of an unstimulated cell, typically\\n−70 to−90 mV .\\nrestriction enzyme endonuclease: enzyme that cleaves phosphodiester bonds within a\\nspeci ﬁc sequence in a DNA molecule.\\nretinoblastoma: cancer of the eye, usually caused by a mutation in the RBgene.\\nretrograde: backward movement; when applied to axonal transport it means toward the\\ncell body.retrovirus: virus whose genetic information is stored in RNA.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='c8b7f1ad-6af4-46bd-a99d-b5c74b95f824', embedding=None, metadata={'page_label': '493', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 489\\nreverse genetics: research that begins with a gene of known sequence but unknown effect\\nand works toward deducing its effects and therefore function.\\nreverse transcriptase: enzyme of some viruses that copies RNA into DNA.\\nreverse transcription: process whereby RNA is copied into DNA.\\nribonuclease: enzyme that cleaves phosphodiester links in RNA.\\nribonuclease H: enzyme that cleaves phosphodiester links in an RNA molecule that is\\njoined to a DNA molecule by complementary base pairing.ribonucleic acid (RNA): polymer of ribonucleoside monophosphates; seemRNA ,tRNA ,\\nandrRNA .\\nribonucleic acid polymerase: enzyme that synthesizes RNA.\\nribonucleic acid primer: short length of RNA, complementary in sequence to a DNA\\nstrand, that allows DNA polymerase III to attach and begin DNA synthesis.ribonucleoside monophosphate: nitrogenous base attached to the sugar ribose that has\\none phosphate group on its 5\\n′carbon atom; also known as a ribonucleotide.\\nribose: pentose sugar used to make the nucleotides that form RNA.\\nribosomal RNA (rRNA): RNA component of ribosomes. rRNA forms a major part of the\\nribosome and participates fully in the translation process.ribozyme: RNA molecule with enzyme-like catalytic activity.\\nRNA (ribonucleic acid): polymer of ribonucleoside monophosphates. Cells contain three\\ntypes of RNA: messenger RNA, ribosomal RNA, and transfer RNA.RNA polymerase: enzyme that synthesizes RNA.\\nRNA primer: short length of RNA, complementary in sequence to a DNA strand that allows\\nDNA polymerase III to attach and begin DNA synthesis.RNA splicing: removal of introns from an RNA molecule and the joining together of exons\\nto form the ﬁnal RNA product.\\nrough endoplasmic reticulum (RER): portion of the endoplasmic reticulum associated\\nwith ribosomes and concerned with the synthesis of secreted proteins. Proteins destined to\\nremain within the majority of single-membrane-bound organelles (Golgi, lysosomes, . . . )are also made on the rough ER, as are integral proteins of these organelles and of the plasmamembrane.\\nrRNA (ribosomal RNA): RNA component of ribosomes. rRNA forms a major part of the\\nribosome and participates fully in the translation process.\\nryanodine: plant toxin that binds to the ryanodine receptor, with complex effects on gating\\nof the channel.ryanodine receptor: calcium channel found in the membrane of the endoplasmic reticulum.\\nIn most cells it opens in response to a rise of calcium concentration in the cytosol. In skeletal\\nmuscle ryanodine receptors are directly linked to voltage-gated calcium channels in theplasma membrane and open when the latter open.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='99a8e23e-a1e0-4afd-85bc-b0c3f451357d', embedding=None, metadata={'page_label': '494', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n490 GLOSSARY\\nσfactor (sigma factor): subunit of bacterial RNA polymerase that recognises the promoter\\nsequence.\\nS phase: period of the cell division cycle during which DNA replication occurs.\\nS value (sedimentation coef ﬁcient): a value that describes how fast macromolecules and\\norganelles sediment in a centrifuge.\\nsaltatory conduction: jumping of an action potential from node to node down a myelinated\\naxon.\\nsalt bridge (in protein structure): interaction between a positively charged amino acid\\nresidue (such as arginine) and a negatively charged residue (such as aspartate).sarcomere: contractile unit of striated muscle.\\nsarcoplasmic reticulum: type of smooth endoplasmic reticulum found in striated muscle;\\nconcerned with the regulation of the concentration of calcium ions.\\nsatellite DNA: DNA sequence that is tandemly repeated many times.\\nsaturated (of fatty acids): containing no carbon –carbon double bonds.\\nsaturation kinetics: are said to occur when the rate of a reaction approaches a maximum,\\nlimiting value as the concentration of reactant increases. The rate becomes limited by the\\navailabilty of binding sites on the catalyst for the reactant.\\nscanning electron microscope (SEM): type of electron microscope in which the image is\\nformed from electrons that are re ﬂected back from the surface of a specimen as the electron\\nbeam scans rapidly back and forth over it. The scanning electron microscope is particularly\\nuseful for providing topographical information about the surfaces of cells or tissues.\\nSchwann cell: glial cell of the peripheral (outside the brain and spinal cord) nervous system.\\nSDS–PAGE (sodium dodecyl sulfate –polyacrylamide gel electrophoresis): technique\\nfor separating proteins by their relative molecular mass.\\nsecondary antibody: in immuno ﬂuorescence, a labeled antibody that is used to reveal\\nthe location of an unlabeled primary antibody. Secondary antibodies are produced by one\\nspecies of animal in response to the injection of another animal ’s antibodies, thus a “goat\\nantirabbit ”secondary will bind to any primary antibody that has been generated by a rabbit.\\nLabeled secondary antibodies can be used to study many different questions because thespeci ﬁcity in the ﬁnal image or western blot is provided by the primary antibody, not by the\\nsecondary.\\nsecondary cell wall: layers of a plant cell wall formed external to the primary cell wall.\\nsecondary immuno ﬂuorescence microscopy: form of light microscopy in which a ﬂu-\\norescent secondary antibody is used to label a preapplied primary antibody speci ﬁc for\\na particular protein or subcellular structure. Also called indirect immuno ﬂuorescence mi-\\ncroscopy.secondary structure: regular, repeated folding of the backbone of a polypeptide. The side\\nchains of the amino acids have an in ﬂuence but are not directly involved. There are two\\ncommon types of secondary structue: the αhelix and the βsheet.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='e1251451-9cd9-4cf5-baa7-5ddf4d185be2', embedding=None, metadata={'page_label': '495', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 491\\nsecond messenger: another name for intracellular messenger (the “ﬁrst messenger ”being\\nthe extracellular transmitter chemical). A cytosolic solute that changes in concentration\\nin response to external stimuli or internal events and that acts on intracellular targets tochange their behavior. Calcium ions, cyclic AMP, and cyclic GMP are the three commonintracellular messengers.\\nsecretion: synthesis and release from the cell of a chemical.\\nsecretory vesicles: vesicles derived from the Golgi apparatus that transport secreted proteins\\nto the cell membrane with which they fuse.\\nsedimentation coef ﬁcient (S value): value that describes how fast macromolecules and\\norganelles sediment in a centrifuge.\\nsemiconservative replication: mode of replication used for DNA; both strands of the\\ndouble helix serve as templates for the synthesis of new daughter strands.\\nserine –threonine kinase: enzyme that phosphorylates proteins on serine or threonine\\nresidues, by transferring the terminal phosphate of ATP to the amino acid side chain. With a\\nfew exceptions, protein kinases are either serine –threonine kinases (which can phosphory-\\nlate on serine or on threonine) or tyrosine kinases (which only phosphorylate on tyrosine).\\nseven-methyl guanosine cap: modi ﬁed guanosine found at the 5\\n′terminus of eukary-\\notic mRNA. A guanosine is attached to the mRNA by a 5′-5′-phosphodiester link and is\\nsubsequently methylated on atom number 7 of the guanine.\\nSH2 domain: domain found in a number of proteins that binds to phosphorylated tyrosine.\\nMany proteins with SH2 domains are recruited to receptor tyrosine kinases when the lat-\\nter phosphorylate themselves in response to ligand binding. Important proteins with SH2domains are Grb2, PI-3-kinase, and phospholipase C γ.\\nβsheet: common secondary structure in proteins, in which lengths of fully extended\\npolypeptide run alongside each other, hydrogen bonds forming between the peptide bondsof the adjoining strands.\\nShine –Dalgarno sequence: sequence on a bacterial mRNA molecule to which the ribosome\\nbinds.\\nside chain (of amino acids): group attached to the αcarbon of an amino acid.\\nsigma factor ( σfactor): subunit of bacterial RNA polymerase that recognizes the promoter\\nsequence.\\nsignal peptidase: enzyme that cleaves the signal sequence from a polypeptide as it enters\\nthe lumen of the endoplasmic reticulum.\\nsignal recognition particle: ribonucleoprotein particle that recognizes and binds to the\\nsignal sequence at the N terminus of a polypeptide.\\nsignal recognition particle receptor: receptor on the endoplasmic reticulum to which the\\nsignal recognition particle binds during the process of polypeptide chain synthesis and\\nimport into the endoplasmic reticulum. Also called the “docking protein. ”\\nsignal sequence: short stretch of amino acids found at the N terminus of polypeptides that\\ntargets them to the endoplasmic reticulum.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='133acb07-65b2-46a6-958f-93adecdad50c', embedding=None, metadata={'page_label': '496', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n492 GLOSSARY\\nsimple diffusion: not a speci ﬁc term: we often say “simple diffusion ”to emphasize that a\\nsolute movement is passive, down a concentration gradient, and does not require a carrier\\nor channel.\\nsingle-strand binding protein: protein that binds to the separated DNA strands to keep them\\nin an extended form during replication, thus preventing the double helix from reforming.\\nsite-directed mutagenesis: technique used to change the base sequence of a DNA molecule\\nat a speci ﬁc site.\\nskeletal muscle cells: large multinucleate muscle cells that are attached to bone. Most cuts\\nof meat are mainly skeletal muscle.\\nsmall nuclear RNAs (snRNAs): small RNA molecules found in the nucleus that play a\\nrole in RNA splicing.smooth endoplasmic reticulum (SER): portion of the endoplasmic reticulum without\\nattached ribosomes, among its functions are the synthesis of lipids and the storage and\\nstimulated release of calcium ions.\\nsmooth muscle: nonstriated muscle, found in many places in the body including blood\\nvessels and the intestine.smooth muscle cells: small muscle cells that lack the characteristic striations seen in skeletal\\nand cardiac muscle.\\nSNARES: proteins that mediate fusion of vesicles with other membranes.\\nsnRNAs (small nuclear RNAs): small RNA molecules found in the nucleus that play a\\nrole in RNA splicing.\\nsodium action potential: action potential driven by the opening of sodium channels and\\nthe resulting sodium in ﬂux.\\nsodium/calcium exchanger: carrier in the plasma membrane. Three sodium ions move\\ninto the cell down their electrochemical gradient and one calcium is moved out up its\\nconcentration gradient.\\nsodium gradient: energy currency. Sodium ions are more concentrated outside the cell than\\ninside, and this chemical gradient is usually supplemented by a voltage gradient pulling\\nsodium ions in. If sodium ions are allowed to enter down their electrochemical gradient,they release 15,000 J/mol.\\nsodium/potassium ATPase: plasma membrane carrier. For every ATP hydrolyzed, three\\nNa\\n+ions are moved out of the cytosol and two K+ions are moved in.\\nsodium pump (sodium/potassium ATPase): plasma membrane carrier. For every ATP\\nhydrolyzed, three Na+ions are moved out of the cytosol and two K+ions are moved in.\\nsolute: substance that is dissolved in a liquid.\\nsomatic cells: cells that make up all the normal tissues of the human body; distinct from\\nthe germ cells that form the gametes (eggs and sperm).\\nsorting signal: section of a protein that causes the cell to direct the protein to a speci ﬁc cell\\ncompartment such as the nucleus or mitochondrion. Sorting signals can be lengths of peptide', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='f082636c-5470-4366-91ea-a7f9e5520cd2', embedding=None, metadata={'page_label': '497', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 493\\n(targeting sequences), such as the signal sequence that targets a protein to the endoplasmic\\nreticulum, or can be the result of posttranslational modi ﬁcation, e.g., mannose-6-phosphate,\\nwhich targets a protein to the lysosome.\\nsorting vesicle: vesicle that carries proteins from one membrane compartment to another.\\nSOS: guanine nucleotide exchange factor for a GTPase called Ras. SOS is recruited to the\\nplasma membrane by binding to growth factor receptor binding protein number 2, which\\nin turn binds activated receptor tyrosine kinases.\\nSouthern blotting: blotting technique in which DNAs, separated by size, are probed using\\na single-stranded cDNA probe.\\nspatial summation: the process in which simultaneous release of synaptic transmitter by\\na number of presynaptic nerve cells depolarizes the postsynaptic neuron to threshold.speci ﬁc activity: ratio between activity and amount of material. For an enzyme it is the\\nratio between measured activity and mass of protein. It can be used as an indication of the\\npurity of an enzyme preparation.\\nspeci ﬁcity constant: ratio between the catalytic rate constant ( k\\ncat) and the Michaelis con-\\nstant ( KM) for an enzyme. It is a rate constant with dimensions of liters mol−1s−1and is\\nused to compare different substrates for the same enzyme or to compare the effectiveness\\nof one enzyme with another.\\nspermatid: cell formed by meiosis that differentiates to form a spermatozoon.\\nspermatozoon: motile male gamete.\\nS phase: period of the cell division cycle during which DNA replication occurs.\\nsplicesome: complex of proteins and small RNA molecules involved in RNA splicing.\\nsquamous: ﬂat. A term used of epithelial cells.\\nstandard state: deﬁned set of conditions under which to state the thermodynamics of a\\nreaction. For example, /Delta1G◦′is determined for aqueous solutions at pH 7.0\\nstart signal: start signal for protein synthesis is the codon AUG, specifying the incorporation\\nof methionine.stearic acid: C\\n17H35—COOH; a fatty acid with no double bonds; i.e., it is fully saturated.\\nThe acyl group derived from stearic acid is common in animal fats and phospholipids.\\nstereo isomers: isomers (i.e., different compounds with the same molecular formula) in\\nwhich the atoms have the same connectivity but that differ in the way the atoms are arranged\\nin space.\\nsteroid hormones: these transmitters act on intracellular receptors to activate transcription\\nof particular genes. Glucocorticoids are one type of steroid hormone; the sex hormones\\n(oestradiol, testosterone, progesterone) are another.\\nsteroid hormone receptor: transcription factors that, upon binding their appropriate steroid\\nhormone, activate transcription of particular genes.\\nsticky ends (of DNA): short single-stranded ends produced by cleavage of the two strands\\nof a DNA molecule at sites that are not opposite to one another.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6b49fc16-ee37-4326-8f15-0d12a3c8f664', embedding=None, metadata={'page_label': '498', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n494 GLOSSARY\\nstop codon: codons UAA, UAG, and UGA are codons that signal protein synthesis to stop.\\nAlso known as termination codons.\\nstop signal: signal, to stop protein synthesis, given to the ribosome by a stop codon on\\nmRNA.strati ﬁed epithelium: type of epithelium consisting of several layers, such as the skin.\\nstress-activated protein kinase: kinase that stimulates cell repair but can also trigger\\napoptosis. p38 and JNK are stress-activated protein kinases.stress ﬁber: bundle of actin ﬁlaments commonly seen in cultured, nonmotile animal cells.\\nstriated muscle: striped muscle; includes skeletal and cardiac muscle.\\nstroma: volume within the chloroplast inner membrane but outside the thylakoids.\\nsubstrate: in normal English, a substrate is a solid base. In biology, the term is used to\\nmean (1) a reactant in a reaction catalyzed by an enzyme, e.g., lactose is a substrate for\\nβ-galactosidase; (2) a base that cells grow and move on, e.g., collagen is a good substrate\\nfor cell attachment.\\nsucrose: disaccharide comprising glucose linked to fructose by an α1↔β2 glycosidic\\nbond.summation (at synapses): additive effects of more than one presynaptic action potential\\nupon the postsynaptic voltage.supercoiling: organization of a linear structure into coils at more than one spatial scale.\\nS value (sedimentation coef ﬁcient): value that describes how fast macromolecules and\\norganelles sediment in a centrifuge.Svedberg unit (S value): value that describes how fast macromolecules and organelles\\nsediment in a centrifuge.synapse: structure formed from the axon terminal of a nerve cell and the adjacent region of\\nthe postsynaptic cell. Transmitter released by the axon terminal diffuses across the synapse\\ngap and acts upon the postsynaptic cell.\\nT (thymine): one of the four bases found in DNA; thymine is a pyrimidine.\\ntandem repeats: many copies of the same DNA sequence that lie adjacent to each other\\non the chromosome.targeting sequence: stretch of polypeptide that determines the cellular compartment to\\nwhich a synthesized protein is sent.\\nTATA box: sequence found about 20 bases upstream of the beginning of many eukary-\\notic genes that forms part of the promoter sequence and is involved in positioning RNA\\npolymerase for correct initiation of transcription.\\ntaxol: compound obtained from the bark of the Paci ﬁcy e w , Taxus brevifolia; binds to\\ntubulin. Taxol is a powerful anticancer drug.telomeres: specialized regions at the ends of eukaryotic chromosomes. Telomeres are rich\\nin minisatellite DNA.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='308f800e-a0fd-41a0-aade-30973050d71f', embedding=None, metadata={'page_label': '499', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 495\\ntelophase: ﬁnal period of mitosis or meiosis in which the chromosomes decondense and\\nthe nuclear envelope reforms.\\ntelophase I: telophase of the ﬁrst meiotic division (meiosis I).\\ntelophase II: telophase of the second meiotic division (meiosis II).\\ntemporal summation: the process in which action potentials in a presynaptic nerve cell\\noccur at a high enough frequency that the depolarizations that they evoke in the postsynaptic\\ncell do not have time to decay back to the resting voltage but rather ride on each other anddepolarize the postsynaptic cell to threshold.\\nterminally differentiated: term that describes a cell that cannot return to the cell division\\ncycle. Nerve cells are terminally differentiated; glial cells are not.termination codon: codons UAA, UAG, and UGA are codons that signal protein synthesis\\nto stop. Also known as stop codons.\\nterminator (of transcription): DNA sequence that, when transcribed into mRNA, causes\\ntranscription to terminate.\\ntertiary structure: three-dimensional folding of a polypeptide chain into a biologically\\nactive protein molecule. It usually includes regions of secondary structure. Interactions of\\nthe amino acid side chains are central in its formation.\\nthermodynamics: study of how energy affects matter and chemical reactions.\\nthermogenin: H\\n+channel found in the inner mitochondrial membrane of brown fat cells.\\nIts presence uncouples NADH oxidation from ATP synthesis, that is, the electron transport\\nchain can operate, pumping H+out of the mitochondrial matrix, even when ATP synthase\\nis not allowing H+back in.\\nthick ﬁlament: one of the two ﬁlaments that form the cytoskeleton of striated muscle;\\ncomposed of the motor protein myosin II.\\nthin ﬁlament: one of the two ﬁlaments that form the cytoskeleton of striated muscle;\\ncomposed of actin.\\nthiol group: —SH group.\\nthreshold (voltage): plasma membrane transmembrane voltage at which enough calcium\\nor sodium channels open to initiate an action potential.\\nthylakoid: folded inner membrane of the chloroplast; site of the light reaction of photosyn-\\nthesis. Thylakoids are arranged in stacks called grana.\\nthymine: one of the four bases found in DNA; thymine is a pyrimidine.\\ntight junctions: type of cell junction in which a tight seal is formed between adjacent cells\\noccluding the extracellular space.tissue: group of cells having a common function.\\ntopoisomerases: enzymes that cut and rejoin DNA strands. Topoisomerase I relieves simple\\ntorsional stress by cutting one strand, allowing rotation about the phosphodiester link of the\\nother strand, then rejoining the cut strand. Topoisomerase II cuts both strands of a doublehelix and passes another complete double helix through the gap, keeping hold of the ends', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='09878dcd-309d-4810-9725-bcce78bb93ed', embedding=None, metadata={'page_label': '500', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n496 GLOSSARY\\nand rejoining them when the other strand has passed through. Topoisomerases are essential\\nduring DNA replication.\\ntoxin: poison.\\ntranscription: synthesis of an RNA molecule from a DNA template.\\ntranscription bubble: structure formed when two strands of DNA separate and one acts\\nas the template for synthesis of an RNA molecule.transcription complex: A complex of RNA polymerase and various transcription factors.\\ntranscription factor: protein (other than RNA polymerase) that is required for gene tran-\\nscription.trans: side from which material is removed. Of the Golgi apparatus, the surface from which\\nvesicles bud to pass to the plasma membrane and to lysosomes.transfected, transfection: cell, prokaryotic or eukaryotic, that has been infected by a\\nforeign DNA molecule(s) is said to be transfected. The process of DNA infection is called\\ntransfection.\\ntransfer RNA (tRNA): RNA molecule that carries an amino acid to an mRNA template.\\ntransform, transformation: in addition to its common English meaning, this is used in\\nmolecular genetics to mean introduction of foreign DNA into a cell.transgenic animal: animal carrying a gene from another organism; the foreign gene is\\nusually injected into the nucleus of a fertilized egg.trans-Golgi network: complex network of tubes and sheets that comprise the trans face\\nof the Golgi apparatus. It is in the trans-Golgi network that proteins made on the rough\\nendoplasmic reticulum are sorted as to their ﬁnal destination.\\ntranslation: synthesis of a protein molecule from an mRNA template.\\ntranslocation: movement. When used of the ribosome, it means the movement, three nu-\\ncleotides at a time, of the ribosome on the mRNA molecule.\\ntransmembrane proteins: class of proteins that span the plasma membrane.\\ntransmembrane translocation: form of protein transport in which unfolded polypeptide\\nchains are threaded across one or more membranes as a simple polypeptide chain and then\\n(re)folded at their ﬁnal destination.\\ntransmembrane voltage: voltage difference between one side of a membrane and the other.\\nIt is usually stated as the voltage inside with respect to outside.\\ntransmission electron microscope: type of microscope in which the image is formed by\\nelectrons that are transmitted through the specimen.transmitter: chemical that is released by one cell and that changes the behavior of another\\ncell.transpiration: loss of water from plant leaves.\\ntransport vesicle: membrane vesicle that transports proteins from one membrane compart-\\nment to another.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='cf5cf147-641b-4b01-b6fa-378f39f5bce5', embedding=None, metadata={'page_label': '501', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 497\\ntriacylglycerol (triglyceride): three acyl (i.e., fatty acid) groups attached to a glycerol\\nbackbone by ester bonds. If the compound is liquid at room temperature, it is called an oil;\\nif it is solid, it is called a fat.\\ntricarboxylic acid cycle (Krebs cycle): series of reactions in the mitochondrial matrix\\nin which acetate is completely oxidized to CO 2with the attendant reduction of NAD+to\\nNADH and FAD to FADH 2.\\ntriglyceride (triacylglycerol): three acyl (i.e., fatty acid) groups attached to a glycerol\\nbackbone by ester bonds. If the compound is liquid at room temperature, it is called an oil;\\nif it is solid, it is called a fat.\\ntrimeric: formed of three subunits.\\ntrimeric G protein: protein with three subunits, α,βandγ, where the αsubunit is a GTPase\\nthat dissociates from the βγunits when it is in its GTP bound state. Both the αsubunit,\\nin its GTP bound state, and the now independent βγsubunit, can activate target proteins.\\nExamples are G s, which activates adenylate cyclase, and G q, which activates phospholipase\\nCβ.\\ntrimethylamine: substance that gives rotting ﬁsh its characteristic smell. A base; when\\ndissolved in water, trimethylamine will accept an H+. Produced by the action of intesti-\\nnal bacteria on trimethylamine N-oxide (from dietary ﬁsh), choline (from dietary phos-\\npholipid), or other chemicals containing a trimethylamine group. Trimethylamine is ox-\\nidized in mammalian liver by the action of ﬂavin-containing monooxygenase. A muta-\\ntion in the FMO3 gene causes the disorder trimethylaminuria. Affected individuals ex-\\ncrete large amounts of trimethylamine in their urine, sweat, and breath and consequentlyhave a ﬁshy body odor. The structures of trimethylamine and protonated trimethylamine\\nare\\nCH 3CH 3\\nNC H 3 CH 3CH 3\\nN+CH 3\\nH\\ntrimethylamine N-oxide: The compound\\nCH 3CH 3\\nNC H 3\\nO\\nThe arrow indicates a so-called dipolar bond between nitrogen and oxygen. This bond canalso be represented as N\\n+—O−. Trimethylamine N-oxide is generated from trimethy-\\nlamine by the action of ﬂavin-containing monooxygenase. A mutation in the FMO3\\ngene causes the disorder trimethylaminuria. Affected individuals excrete large amountsof trimethylamine in their urine, sweat, and breath and consequently have a ﬁshy body\\nodor.\\ntrisphosphate: having three phosphate groups attached at three different points on the\\nmolecule. Inositol trisphosphate and phosphatidylinositol trisphosphate are examples.Trk: a family of receptor tyrosine kinases that bind growth factors of the neurotrophin class.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='87bc3a59-4435-4079-b576-8def26fc4d97', embedding=None, metadata={'page_label': '502', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n498 GLOSSARY\\ntRNA (transfer RNA): RNA molecule that carries an amino acid to an mRNA template.\\ntroponin: calcium-binding protein found in muscle cells.\\ntryptophan ( trp) operon: cluster of ﬁve bacterial genes involved in the synthesis of the\\namino acid tryptophan.\\ntubulin: protein that forms microtubules; exists as α,β, andγsubunits.\\ntumour: proliferative cell mass associated with many cancers.\\nturnover number: number of moles of substrate converted to product per mole of enzyme\\nper unit time. Another term for catalytic rate constant, kcat.\\ntyrosine kinase: enzyme that phosphorylates on tyrosine residues in proteins by transferring\\na phosphate group from ATP. In this book all the tyrosine kinases described are receptor\\ntyrosine kinases.\\nU (uracil): one of the four bases found in RNA; uracil is a pyrimidine.\\nultramicrotome: machine for cutting thin sections ( <100 nm) for electron microscopy.\\nultrastructure: ﬁne structure of the cell and its organelles revealed by electron microscopy.\\nuncouples, uncoupler, uncoupled (of mitochondria): mitochondria are uncoupled when\\nthe tight link between the respiratory chain and ATP synthesis is broken. Since the link is\\nthe gradient of H+across the inner mitochondrial membrane, any chemical that allows H+\\nions to cross the inner mitochondrial membrane is an uncoupler. If the chemical is presentin more than a small fraction of the cells of the body, it will kill, but expression of the proteinuncoupler thermogenin in brown fat allows that tissue to generate heat.\\nunsaturated (of fatty acids): containing carbon –carbon double bonds.\\nuntranslated sequence: sequence of bases in an mRNA molecule that does not code for\\nprotein. Untranslated regions are found at the 5\\n′and 3′ends of an mRNA.\\nupstream: general term meaning the direction from which things have come. When applied\\nto the DNA within and adjacent to a gene, it means lying on the side of the transcription\\nstart site that is not transcribed into RNA. When applied to signaling pathways, it meansopposite to the direction in which the signal travels; thus the insulin receptor is upstreamof protein kinase B.\\nuracil: one of the four bases found in RNA; uracil is a pyrimidine.\\nuracil –DNA glycosidase: DNA repair enzyme that recognizes and removes uracil from\\nDNA molecules.urea: H\\n2N—(CO) —NH 2, compound made in the liver that contains two nitrogen atoms but\\nis less toxic than ammonia. Urea is chaotropic: at high concentration it reversibly denatures\\nproteins.\\nurea cycle: metabolic cycle in the liver that makes urea that is then excreted in the urine.\\nvacuole: large membrane-bound compartment. Plant cells often contain a large vacuole\\nﬁlled with sugars and pigments.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='277dfc85-3b48-45fb-ab1c-c0b7664c874a', embedding=None, metadata={'page_label': '503', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\nGLOSSARY 499\\nValium: antianxiety drug that increases the chance that the GABA receptor channel will\\nopen, allowing chloride ions to pass.\\nvan der Waals force: weak close-range attraction between atoms.\\nvascular tissue: blood vessels. The term is also used to describe the water transporting and\\nsupport tissue of plants that is composed of xylem and phloem.vasoconstrictor: substance that constricts blood vessels.\\nvasodilator: substance that dilates (widens) blood vessels.\\nvector: Something that carries something else. The term is often used to describe a plas-\\nmid or bacteriophage that carries a foreign DNA molecule and is capable of independent\\nreplication within a bacterial cell.\\nvesicle: small closed bags made of membrane.\\nvesicular traf ﬁcking: precisely controlled movement of vesicles between different or-\\nganelles and/or the plasma membrane.villin: type of actin-binding protein that crosslinks actin ﬁlaments.\\nvillus ( plural villi): ﬁngerlike extension of an epithelial surface that increases the surface\\narea.vimentin: protein that makes up the intermediate ﬁlaments in cells of mesenchymal origin\\nsuch as ﬁbroblasts.\\nvirus: packaged fragment of DNA or RNA that uses the synthetic machinery of a host cell\\nto replicate its component parts.\\nV\\nm:maximal velocity of an enzyme-catalyzed reaction: the limiting initial velocity ob-\\ntained as the substrate concentration approaches in ﬁnity. Vmis also often used to mean\\ntransmembrane voltage.VNTRs (variable number tandem repeats): DNA sequences that occur many times within\\nthe human genome. Each person carries a different number of these repeats.voltage clamp: technique in which the experimenter passes current to one side of a mem-\\nbrane to arti ﬁcially set the value of the transmembrane voltage to a desired level.\\nvoltage-gated calcium channel: channel that is selective for calcium ions and that opens\\nupon depolarization. Found in the plasma membrane of many cells.voltage-gated sodium channel: channel that is selective for sodium ions and that opens\\nupon depolarization. Found in the plasma membrane of nerve and muscle cells.Wee1: protein kinase that phosphorylates and hence inactivates CDK1.\\nwobble (in tRNA binding): ﬂexibility in the base pairing between the 5\\n′position of the\\nanticodon and the 3′position of the codon.\\nxeroderma pigmentosum: inherited human disease caused by defective DNA repair en-\\nzymes; affected individuals are sensitive to ultraviolet light and contract skin cancer when\\nexposed to sunlight.\\nxylem: part of the plant vascular system; transports water from the roots to the leaves.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b2c1ce6c-7209-4896-b964-5cc717389c59', embedding=None, metadata={'page_label': '504', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: FMK\\nWY001-GSY WY001-Bolsover-v2.cls September 15, 2003 21:41\\n500 GLOSSARY\\nYAC (yeast arti ﬁcial chromosome): cloning vector used to propagate DNAs, of about\\n500,000 bp, in yeast cells.\\nZ disc: disc that is set within and at right angles to the actin micro ﬁlaments in striated\\nmuscle, holding them in a regularly spaced array.Z-DNA: left-handed helical form of DNA.\\nZellweger ’s syndrome: inherited human disease resulting from aberrant targeting of pro-\\nteins to the peroxisome.zinc ﬁngers: structural motif in some families of DNA binding proteins in which a zinc\\nion coordinated by cysteines and/or histidines stabilizes protruding regions that touch the\\nedges of the base pairs exposed in the major groove of DNA.', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6c2a2aa2-567a-404b-9573-c245433742c3', embedding=None, metadata={'page_label': '505', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\nINDEX\\nUnderlined page numbers indicate a glossary entry, often a good place to begin reading about a topic.\\n-10 box, 108\\n-35 box, 1081,3-bisphosphoglycerate, 2862-oxoglutarate\\nin Krebs cycle, 284in nitrogen excretion, 292\\n2-phosphoglycerate, 2863-dimensional structure of proteins, 192–208\\nspeciﬁcation by primary structure, 206\\n3-hydroxybutyrate, 2903-phosphoglycerate, 286, 3027-methyl guanosine cap, 118, 176, 441\\n9+2 axoneme, 388, 441\\n14-3-3 protein, 358, 417, 441\\n30S subunit, of prokaryote ribosome, 168–17540S subunit, of eukaryote ribosome, 168, 17650S subunit, of prokaryote ribosome, 168–17560S subunit, of eukaryote ribosome, 168, 17670S initiation complex, 171, 17680S initiation complex, 176α-adrenergic receptor, 365, 368, 441\\nα-amino acid, 37–39, 184, 441\\nα-blocker, 365\\nαform, of monosaccharide, 27–28\\nα-globin gene family cluster, 100\\nαhelix, 194–196, 441\\nA site, 168, 441\\nAcanthamoeba , 386\\nAcceptor atom in hydrogen bond, 26, 442\\nAcetate, 39\\nbreakdown product of acetylcholine, 368\\nAcetic acid, 39Acetoacetate, 290Acetoacetyl-ACP, 300Acetone, on breath, 299, 442\\nAcetylcholine, 364, 442\\nat motoneurone:muscle synapse, 367–368in control of blood supply, 370receptor, muscarinic, 370, 376, 442\\nreceptor, nicotinic, 364, 439, 442\\nCell Biology: A Short Course, Second Edition, by Stephen R. Bolsover, Jeremy S. Hyams,\\nElizabeth A. Shephard, Hugh A. White, Claudia G. Wiedemann\\nISBN 0-471-26393-1 Copyright C⃝2004 by John Wiley & Sons, Inc.Acetylcholinesterase, 442\\nat synapse, 368\\ninhibition, 376speciﬁcity constant, 247\\nAcetyl-CoA, 290, 291, 295, 300\\nglyoxylate shunt and, 297–298\\nN-Acetyl glucosamine, 32–33\\nAchilles tendon, 365Acid, 21–25, 442\\nas amino acid side group, 185organic, identifying carbon atoms, 37\\nACP (acyl carrier protein), 300Acquired immunodeﬁciency syndrome (AIDS),\\n11–12\\nAcrosome, 392, 442\\nAcrylamide, 166ActA, 393, 442\\nActin, 390–396\\nat cell junctions, 396-binding proteins, 391, 442\\ncell locomotion, 395cytoplasmic streaming, 395F-actin, 390–391G-actin, 390–391muscle, 393–394phalloidin, 376proﬁlin, 391-related protein (Arp), 391, 442\\nsevering proteins, 395, 442\\nvillin, 391–392\\nAction potential, 325–337, 373–375, 442\\nActivation energy, 249, 442\\nAcyl\\ncarrier protein (ACP), 300group, 39, 442\\nN-Acyl-HSL, 116\\nsynthase, 116\\nAdenine (A), 34, 35, 46, 69, 441\\nbonding with thymine, 26, 69\\nAdenosine, 35, 442\\n501', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='6999e4bc-0a4e-496e-ae9a-378ecbde1e70', embedding=None, metadata={'page_label': '506', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\n502 INDEX\\nAdenosine diphosphate (ADP), 258 –261, 263,\\n269, 272 –273, 443\\nADP/ATP exchanger, 271\\nADP-ribosylation, 173receptor, 344sign of cell damage, 344\\nAdenosine monophosphate (AMP), 259,\\n277, 443\\ncAMP, 56, 114, 200, 305, 350 –353, 358\\nregulation of phosphofructokinase, 251, 304\\nAdenosine triphosphate (ATP), 37, 259, 277,\\n443\\nanaerobic generation by glycolysis, 286 –288\\nATP/ADP exchanger, 271, 440ATPase, calcium, 318, 350, 370, 440ATPase, sodium/potassium, 270, 271, 311,\\n312, 318, 320, 440\\ncoenzyme/energy source, 90, 221, 270, 282,\\n290, 293, 295, 298, 302, 318, 388, 394\\nDNA helix unwinding, 90feedforward control of ATP production,\\n304–306\\nnecrosis, 415phosphate source, for kinase, 190, 251, 258,\\n282, 284, 285, 293, 318, 412\\nradioactive, 139regulation of phosphofructokinase, 251, 304RNA synthesis, 105substrate for adenylate cyclase, 350synthase, 269, 272 –273, 274, 275\\nAdenylate cyclase, 350, 352, 353, 443\\nAdherens junction, 396, 443\\nAdipose tissue and cells, 13, 300, 443\\nADP, 258 –261, 263, 269, 272 –273, 443\\nADP/ATP exchanger, 271, 443\\nADP-ribosylation, 173receptor, 344sign of cell damage, 344\\nAdrenal gland, 60, 121, 305, 368Adrenaline, 353, 368 –371, 443\\naction on blood vessels, 368acts mainly at βreceptor, 365\\npromotion of glycogen breakdown, 305\\nAdrenergic receptor, 365, 443\\nαadrenergic receptor, 365, 368\\nβadrenergic receptor, 305, 368\\nAdrenoleukodystrophy, 60Adverse drug reactions, 158Aerobic, 277Aﬂatoxin, 251\\nAgarose gel, 146, 148AGT (alanine glyoxylate aminotransferase),\\n223, 250, 292\\nAkt (protein kinase B), 356 –358, 417, 443\\nAlanine, 185\\naminotransferase (AGT), 223, 250, 292conversion to pyruvate, 250genetic code, 78glyoxylate aminotransferase (AGT), 223,\\n250, 292\\nsynthesis, 302\\nAlexander the Great, 74Algae, 11, 395\\nblue-green algae, 12\\nAlkaline, 25, 443\\nalkaline phosphatase, in DNA cloning,\\n139\\nAllolactose, 113, 443\\nAll or nothing feature, of action potential, 328,\\n444\\nAllosteric, 238 –240, 444\\nenzymes, 251\\nAlpha-adrenergic receptor, 365, 368Alpha-amino acid, 37 –39, 184\\nAlpha-blocker, 365Alpha form, of monosaccharide, 27 –28\\nAlpha-globin gene family cluster, 100Alpha helix, 194 –196\\nAlternative splicing, 68, 118, 444\\nAlu repeat, 101Amanita muscaria , 370, 376\\nAmanita phalloides , 376\\nAmbiguity, in codes, 79Amino acids, 37 –39, 46, 75 –76, 444\\n1-letter abbreviation, 78, 793-letter abbreviation, 78, 79α-amino acids, 37 –38, 444\\nγ-amino acids, 37 –38\\nactivation, 165attachment to tRNA, 163 –165\\nchirality of, 196\\nDAmino acids, 196\\nessential, 302genetic code, 77 –79\\nLAmino acids, 196\\nnonprotein, 192optical isomers, 196polymers of, 184residue, 188side chains, 184 –188\\nsource of metabolic energy, 292 –294\\nsynthesis, 300 –302\\nunique properties of, 188 –191\\nAminoacyl site (A site), 168, 444\\nAminoacyl tRNA, 165, 171, 173, 178, 444\\nsynthase, 165, 444\\nAmino terminal, 184, 444\\nAminotransferases, 249\\nin amino acid synthesis, 302in amino acid breakdown, 292in primary hyperoxaluria type 1, 223mechanism, 249\\nAmoeba, 1, 386, 395Amorphous ground substance, 13', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='35d4f8ce-86e2-4995-b1e7-f6a4ed3e87d3', embedding=None, metadata={'page_label': '507', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\nINDEX 503\\nAMP, 259, 277, 444\\nand regulation of phosphofructokinase, 251,\\n304\\ncAMP, 56, 114, 200, 305, 350 –353, 358\\nAmphipathic, 39, 444\\nAmplitude modulation (AM), 336\\nAnabolism, 281, 444\\nAnaerobic, 275, 350, 444\\ndirection of energy ﬂow, 275\\nglycolysis, 286 –288\\nmode of life (anaerobes), 288\\nAnaesthetic, local, 334Anaphase, 403 –404, 444\\nAnchoring cell junction, 54, 56, 396, 445\\nAnemia, sickle cell, 81, 82Angina, 370Animal cell, summary, 10Anion, 21, 445\\nAnode, 166, 445\\nAntenna chlorophylls, 274, 445\\nAnterograde transport, 390Anthocyanin, 11Anthrax, 89, 445\\nAntianxiety drugs, 375Antibiotics, 445\\nblocking ATP synthase, 269blocking protein synthesis, 176blocking topoisomerase I, 89Cipro, 89contain D amino acids, 196in cystic ﬁbrosis, 423\\nin DNA cloning, 130, 134oligomycin, 269puromycin, 176resistance, 74, 130, 131, 157\\nAntibodies, 7, 139, 445\\nmanufacture in bacteria, 149\\nAnticodon, 164 –165, 178, 445\\nAntiparallel\\nβsheet, 195, 445\\nDNA structure, 69\\nAntisense RNA, 147 –148\\nprobe, 149\\nAnxiety, 375AP1 transcription factor, 124, 200AP endonuclease, 95, 101, 445\\nApoptosis, 315, 412, 415 –418, 420, 445\\ninhibitor proteins, 418, 445\\np53 and DNA damage, 417pathways controlling, 416stress-activated, 417 –418\\nAporepressor, 117, 445\\nAqueous liquid, 21ARF GTPase, 228, 445\\nArginine, 185\\nessential in infants, 302genetic code, 78in histones, 73NO synthesis, 365SH2 domain, 355synthesis, 293, 302\\nArp (actin-related protein), 391, 446\\nArthritis, 124Arum lily, 270A site, 168, 446\\nAsparagine, 185\\ngenetic code, 78glycosylation, 190, 225synthesis, 302\\nAspartate, 185\\nATP synthase, role in, 273calcium ATPase, role in, 241, 318caspase, 415genetic code and wobble, 78, 164nitrogen excretion, role in, 292 –293\\nphosphorylation, 190, 241, 318synthesis, 302urea cycle, 292 –293\\nAssay, 132, 183, 242, 446\\nATP, 37, 259, 277, 446\\nallosteric regulation of phosphofructokinase,\\n251, 304\\nanaerobic generation by glycolysis, 286 –288\\nas coenzyme/energy source, 90, 221, 270,\\n282, 290, 293, 295, 298, 302, 318, 388, 394\\nas phosphate source, for kinase, 190, 251,\\n258, 282, 284, 285, 293, 318, 412\\nATP/ADP exchanger, 271, 440, 446\\nATPase, calcium, 318, 350, 370, 440ATPase, sodium/potassium, 270, 271, 311,\\n312, 318, 320, 440\\nDNA helix unwinding, 90feedforward control of ATP production,\\n304–306\\nnecrosis, 415radioactive, 139RNA synthesis, 105substrate for adenylate cyclase, 350synthase, 269, 272 –273, 274, 275, 446\\nAtropine, 376AUG codon, 79, 80, 150, 168, 171, 176Autoinduction, 116, 446\\nAutonomic nerve cells, 368, 446\\nAutophagic vacuole, 231Autoradiography, 139, 446\\nAutumn crocus, 384Avery, Oswald, 66Axon, 13, 446\\naxonal transport, 390, 446\\nelectrical transmission down, 332 –336\\nAxoneme, 388\\nβadrenergic receptor, 305, 368, 446\\nβbarrel structure, of proteins, 200', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='5291bd2c-2a4d-456c-a37f-1258ac48404a', embedding=None, metadata={'page_label': '508', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\n504 INDEX\\nβblocker, 365\\nβform, of monosaccharide, 27 –28\\nβ-galactosides and galactosidase, 109 –111\\nin pBluescript vector, 132\\nβ-galactoside permease, 111, 440\\nβoxidation of fatty acid, 290, 446\\nβpleated sheet, 195\\nβsheet, 195, 446\\nB-form, of DNA, 71, 446\\nBacillus subtilis ,9 7\\nBack reaction, 243\\nBacteria, bacterial, 9\\nartiﬁcial chromosome (BAC), 140\\ncytoskeleton hijacking by, 393ﬂagellum, 388 –389\\nintroducing foreign DNA molecules into,\\n130–133\\nintroducing recombinant plasmids into,\\n134\\nmotility of, 264producing mammalian proteins in, 149protein synthesis in, 168 –175, 178\\nRNA synthesis, 106 –118\\nBacteriophage, 12, 74, 97, 133, 140, 447\\nBAD protein, 358, 417, 447\\nBasal body, 387, 447\\nBAM H1, 133\\nBase, 25, 35, 447\\namino acid side chain, 185as opposed to acid, 25excision repair, 94 –95, 447\\nin nucleic acids, 35, 94mutation, 94pair, 27, 67, 69, 447\\nBasement membrane, 13, 447\\nBAX protein, 417 –418, 447\\nBcl-2, 418, 447\\nassociated death promoter (BAD), 417family proteins, 417 –418, 447\\nB-DNA, 71, 447\\nBear, Christine, 428Berg, Howard, 272Berserk, 376Beta-adrenergic receptor, 305, 368Beta barrel structure, of proteins, 200Beta blocker, 365Beta form, of monosaccharide, 27 –28\\nBeta-galactosides and galactosidase, 109 –111,\\n447\\nin pBluescript vector, 132\\nBeta-galactoside permease, 111, 440Beta oxidation of fatty acid, 290Beta pleated sheet, 195Beta sheet, 195Bicycle tire, 53Bilayer, 42, 52\\nbilayer voltage clamp, 428Binding, binding site, 447\\nin carriers, 316 –320\\nof ions on proteins, 190of molecules to proteins, 183, 199, 202, 237,\\n412\\nof mRNA on ribosome, 168 –169\\nsingle strand binding proteins, 90\\nBiochemical signaling, 353 –359\\nBiochemistry, 19 –46, 237 –306\\nBioluminescence, 116, 447\\nBisphosphoglycerate, 286Blobel, G ¨unter, 60\\nBlood, 53\\nammonium, 293cells, 5, 81, 171, 287, 311, 416, 432clotting, 149, 344 –348\\nenzyme concentrations in, 248ﬂow regulation, 368 –370\\nfuels for cells, 288, 290, 299, 317hemoglobin, 204 –205, 238 –240, 252\\nhormones, 363leukemia, 154, 432parasites, 288platelets, 149, 344 –348\\nscurvy, 190sickle cell anaemia, 81test for cystic ﬁbrosis, 425\\ntest for phenylketonuria, 295vessel structure and growth, 190, 368 –372\\nBloom ’s syndrome, 98, 448\\nBlotting techniques, 149Blue eyes, 406Blue-green algae, 12, 448\\nBlunt ends, 134, 448\\nBond\\ncovalent, 19disul ﬁde, 191, 192, 199\\ndouble bonds in acyl groups, 42, 300electrostatic, 190, 199ester, 39glycosidic, 28 –31\\nhydrogen, 25 –27, 69, 88, 90, 108, 109, 163,\\n164, 195\\nhydrolysed by phospholipases, 346peptide, 38, 168, 173, 184triple, in nitrogen, 300\\nBorage, 42BoTox, 232Botulism, botulinum toxin, 232, 288-10 Box, 108-35 Box, 108βoxidation, 290 –291, 448\\nBoyer, Paul, 273Breast cancer, 154, 355Breathing, of proteins, 238Brenner, Sydney, 76Bright- ﬁeld microscopy, 3 –5, 448', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='7283a641-3a06-4983-a68b-6bc34db4c3cf', embedding=None, metadata={'page_label': '509', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\nINDEX 505\\nBrown\\neyes, 406\\nfat, 270, 448\\nβsheet, 195, 448\\nBubble\\nboy disease, 432transcription, 108\\nBud formation, 229Budding yeast, 68, 401Bulky lesion, 94, 448\\nC terminal, 173, 185Cadherin, 396, 448\\nCaenorhabditis elegans , 68, 99, 418 –419\\nCalcineurin, 222 –223, 350, 358, 448\\nCalcium\\naction potential, 325 –330, 448\\nactivation of NO synthesis, 370ATPase, 318, 370, 448\\nbinding protein, 200, 216, 448\\nbinding sites, 190, 318calcineurin, 222 –223, 358\\ncell locomotion and mobility, 389, 394, 395channel, IP\\n3gated, 344, 438\\nchannel, mitochondrial, 305, 348, 438channel, ryanodine receptor, 348, 438channel, voltage gated, 327, 389, 438chloride, in DNA cloning, 134concentration, 310control of glycogen breakdown, 305control of mitochondria, 305, 348endoplasmic reticulum, 60, 216, 344,\\n348–349\\ngelsolin, 395indicator dye, 343inﬂux from extracellular medium, 341 –343\\nintracellular messenger, 341 –350, 356,\\n368–370\\nkinases, calcium activated, 305, 358muscle cells, 304 –306, 348 –350, 394\\nnerve cells, 341 –344\\nParamecium , swimming control, 389\\nphosphatase, calcium-activated, 222 –223, 358\\nrelease from endoplasmic reticulum, 344 –348\\nremoval from cytosol, 350sea urchin eggs, 327 –330\\nstimulation of gene transcription, 222, 350,\\n358\\ntrigger for exocytosis, 230, 343, 348\\nCalf muscle, 365Calming, by Valium, 375Calmodulin, 200, 449\\ncalcineurin activation, 222 –223, 358\\nregulator of glycogen phosphorylase kinase,\\n305\\nstructure, 200study by stopped ﬂow, 253Calreticulin, 216Calvin cycle, 302 –303, 449\\ncAMP, 350 –353, 449\\ncontrol of glycogen breakdown, 305dependent protein kinase, 200, 305, 353, 358,\\n427, 449\\ngated channel, 200, 350, 449\\ngene expression control, 114, 200, 358intracellular messenger, 350 –353\\noocyte, 56phosphodiesterase, 353, 449\\nmuscle cells, 305\\nCancer, 401\\nactivation of mutagens, by cytochrome P450,\\n250\\napoptosis, protective, 418classi ﬁcation, 154\\nDNA repair enzyme de ﬁciency, 98\\ndrugs to ﬁght cancer, 355, 404\\nmotility in cancer cells, 386retinoblastoma, 413 –415\\nCAP (catabolite activator protein), 114, 200,\\n204, 449\\nCap, 7-methyl guanosine, on RNA, 118, 176,\\n449\\nCapsaicin, 39Capsid, 12Carbohydrate, 27 –34, 449\\nmetabolism, 284 –289\\nCarbon dioxide\\ndiffusion across membrane, 53generation during anaerobic glycolysis, 288generation in Krebs cycle, 283 –284\\ngeneration in pentose phosphate pathway, 289photosynthesis, 302 –303\\nCarbon ﬁxation, 302 –303, 449\\nCarboxyl group, 22, 27, 449\\nCarboxyl terminal, 173, 185, 449\\nCarcinogens, 250 –251\\nCardiac muscle, 14 –15, 56, 396, 449\\nCarriers, 264, 275, 322, 407, 450\\nglucose, 238, 316 –317, 357\\nlist of, 437 –440\\nproperties, 316 –320\\nwith enzymatic action, 318 –320\\nCaspases, 415, 418, 450\\nCatabolism, 281, 450\\nCatabolite activator protein (CAP), 114, 200,\\n204, 450\\nCatalase, 60\\nfunction, 60k\\ncat, 242\\nlocation, 60speci ﬁcity constant, 247\\nCatalysis, 247 –249, 450\\nCatalytic rate constant ( kcat), 242, 244, 245, 247,\\n450', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='63c74a7a-d726-465a-8b06-8b7630835605', embedding=None, metadata={'page_label': '510', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\n506 INDEX\\nCatenin, 396\\nCation, 21, 450\\nCdc25 enzyme, 412, 419, 450\\nCdk1 enzyme, 251 –252, 411 –412, 450\\nCdk2 enzyme, 413, 450\\nCdk4 enzyme, 413, 450\\nCdk6 enzyme, 413, 450\\ncDNA, 130, 450\\ncloning, 130 –139\\nlibrary and screening, 134 –139, 426,\\n450\\nmicroarray, 154probe, 143, 148synthesis from mRNA, 130\\nCell, 1, 450\\nadhesion molecules, 396, 450\\n-attached patch clamping, 320 –321\\ncenter, 61, 451\\ncycle, 401 –406, 408 –415, 451\\ndeath, 415 –418\\ndivision cycle, 401 –406, 408 –415, 451\\njunction, 54 –56, 396, 451\\nmembranes, 51 –56, 451\\ntheory, 1types, 9 –11\\nwall, 53, 54, 451\\nCell surface membrane (plasma membrane),\\n51\\naction potential, 325 –337\\napoptosis, modi ﬁcation in, 415\\ncrossing by polypeptide, 200, 204diffusion across, 53, 121, 365endocytosis, 229, 230patch clamping, 320 –321\\npermeability to potassium ions, 310 –311\\nresting voltage, 311 –312\\nsodium gradient, as energy currency, 262source of its protein and lipid, 60, 230\\nCella, 1Cellulose, 30 –31, 53, 382, 451\\nsynthase, 382, 384, 451\\nCentral dogma, 39, 451\\nCentral transporter, 216Centriole, 382, 451\\nCentromere, 100, 403, 451\\nCentrosome, 11, 382, 383, 451\\nabsence in plants, 11\\nCF (cystic ﬁbrosis), 423 –432, 451\\nCFgene, CFTR , 425 –427\\ncGMP (cyclic guanosine monophosphate), 353,\\n451\\ndependent protein kinase, 370gated channel, 353phosphodiesterase, 370photoreceptors, role in, 353smooth muscle cells, role in, 370synthesis in response to NO, 365, 370Chain initiation in protein synthesis\\nin eukaryotes, 175 –176\\nin prokaryotes, 169\\nChain release factors, 174, 451\\nChannel, 314, 437 –439, 451\\ncalcium, IP 3gated, 344, 438\\ncalcium, mitochondrial, 305, 348, 438calcium, ryanodine receptor, 348, 438calcium, voltage gated, 327, 389, 438cAMP gated, 200, 350, 439CFTR (cystic ﬁbrosis ...) ,4 2 6 –428, 439\\ncGMP gated, 353, 439chloride, 373 –375, 426 –428, 439\\ncurrents through, measuring, 320gap junction, 55 –56, 314, 437\\nglutamate receptors, 372 –374, 439\\nhydrogen ion channels, in ATP synthase, 272mitochondrial calcium, 305, 348, 438mitochondrial protein import, 221nicotinic acetylcholine receptor, 364, 439porin, 262, 266, 314, 315, 417 –418, 437\\npotassium, 310 –312, 437\\nprotein translocator, 223ryanodine receptor, 348, 438sodium, 200, 330 –336, 408, 438\\nsolute-gated, 344, 350, 364, 438 –439\\nthermogenin, 270\\nChaotropic reagents, 206, 452\\nChaperones, 207, 221, 452\\nChaperonin, 221Chargaff, Erwin, 71Charge, 21, 452\\non amino acid side chain, 190\\nCharged tRNA, 165, 452\\nChemical bond, 19 –21\\nChemistry, 19 –46, 237 –306\\nChiasma, chiasmata, 408, 409, 452\\nChilies, 39Chimeric proteins, 150, 452\\nChiral structure, of amino acids, 196, 452\\nChitin, 33Chloramphenicol, 176Chloride, 21\\ncalcium, in DNA cloning, 134channel, 373 –375, 426 –428\\nconcentration, 310gradient, 314transport, in cystic ﬁbrosis, 424 –425\\nChlorine, 20Chlorophyll, 274, 452\\nChloroplast, 58, 452\\nappearance, 58carbon ﬁxation, 302 –303\\ncytoplasmic streaming, 395DNA, 12energy ﬂux, 271 –275\\norigin as cyanobacteria, 12', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='08e70763-738e-45f2-a447-ab8375199e00', embedding=None, metadata={'page_label': '511', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\nINDEX 507\\nCholera, 353\\nCholine, 45\\nbreakdown product of acetylcholine, 368in phospholipid structure, 40, 82ﬁsh odor syndrome, 82 –83\\ntrimethylaminuria, 82 –83\\nChromatid, 73, 403, 406, 408, 452\\nChromatin, 57, 73, 403, 452\\neuchromatin, 57, 73heterochromatin, 57, 73\\nChromatophore, 389, 452\\nChromosome, 58, 71, 452\\nchanges at mitosis, 403counting, 403eukaryotic, 71 –73\\nhuman, 76, 402packaging of DNA molecules into, 71 –75\\nprokaryotic, 73walking, 152, 452\\nChronic fatigue syndrome, 259Cigarettes, 13, 94, 250, 376, 386Ciliates (cilated protists), 2, 386Cilium, cilia, 13, 386 –389, 398, 452\\nCIP1, 415Cipro, 89ciscisternae, 226\\ncisface of Golgi, 226\\nCisternae, 60, 452\\nCitrate, 284, 297Citric acid cycle, 283Citrulline, 191Citrus fruit, 190CKIs (cyclin dependent kinase inhibitors), 415,\\n452\\nClassi ﬁcation of transmitters, 363\\nClathrin, 228 –229, 453\\nClaude, Albert, 60Cleavage furrow, 404, 453\\nCleavage sites, of restriction endonucleases, 133Clone, 129, 453\\nlibrary, 134, 453\\nCloning, DNA, 129 –130, 453\\nCloning vectors, 130 –132, 453\\njoining foreign DNAs to, 133 –134\\nClosed promoter complex, 108, 453\\nClostridium botulinum , 232\\nClotting of blood, 149, 344 –348\\nCoatamer, 228, 453\\nCoated vesicle, 228 –229, 453\\nCode, genetic, 75 –80\\nCoding, by action potential frequency, 336 –337\\nCodons, 75 –76, 453\\nstart and stop, 79 –80\\nCoenzyme, 259 –261, 282, 453\\ncoenzyme A, 283 –284, 286, 290 –291, 297\\ncoenzyme Q, 265 –268\\nCofactors, 249, 254, 453Cohesin, 403, 453\\nCO 2\\ndiffusion across membrane, 53generation during anaerobic glycolysis, 288generation in Krebs cycle, 283 –284\\ngeneration in pentose phosphate pathway, 289photosynthesis, 302 –303\\nColchicine, 384 –385, 453\\nCollagen, 13, 453\\ncollagenase and arthritis, 124posttranslational modi ﬁcation in, 190\\nColumnar cells, 12 –13, 453\\nCommunication, intercellular, 363 –379\\nvia gap junctions, 15, 55 –56\\nCompetent cells, 134, 453\\nComplementarity of DNA chains, 69 –71\\nComplementary DNA (cDNA), 130, 454\\ncloning, 130 –139\\nlibrary and screening, 134 –139, 426\\nmicroarray, 154probe, 143, 148synthesis from mRNA, 130\\nComplex I, 267, 454\\nComplex II, 267, 454\\nComplex III, 267 –268, 454\\nConcentration gradients, 311 –312\\nCondenser lens, 3, 454\\nConnective tissue, 13, 454\\nConnexon, 56, 454\\nConstitutive secretion, 229 –230, 454\\nContact inhibition, 415, 454\\nContraception, St. John ’s wort and, 124\\nContraction of muscle, 393 –394\\nactivation by calcium, 394\\nCoomassie brilliant blue, 166Cooperative oxygen binding, 240Copper, 205, 265, 268, 275Copying DNA by PCR, 150Core recognition motif, 123Corepressor, 117, 454\\nCortex, plant, 16, 454\\nCorynebacterium diptheriae , 173\\nCousins, marriage between, 407Covalent bonds, 46, 455\\nCreutzfeldt –Jacob disease, 207, 408\\nCrick, Francis, 68, 76Cristae, 58, 455\\nCrossing over, of genetic material, 408, 409,\\n455\\nCrosstalk, 357 –359, 455\\nCTP (cytidine triphosphate) as energy currency,\\n259\\ndCTP, building block of DNA, 65 –67\\nC terminal/terminus, 173, 185, 455\\nCurrency, energy, 257 –264\\nCuticle, 16Cyanide, 271', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1ce42b8e-62f5-48f3-9ad6-41db9ad849e8', embedding=None, metadata={'page_label': '512', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\n508 INDEX\\nCyanobacteria, 12, 455\\nCyclic adenosine monophosphate (cAMP),\\n350–353, 455\\ncontrol of glycogen breakdown, 305\\ndependent protein kinase, 200, 305, 353, 358,\\n427\\ngated channel, 200, 350gene expression control, 114, 200, 358intracellular messenger, 350 –353\\nmuscle cells, 305oocyte, 56phosphodiesterase, 353\\nCyclic guanosine monophosphate (cGMP), 353,\\n455\\ndependent protein kinase, 370gated channel, 353phosphodiesterase, 370photoreceptors, role in, 353smooth muscle cells, role in, 370synthesis in response to NO, 365, 370\\nCyclic photophosphorylation, 275Cyclin B, 251 –252, 411 –412, 419, 455\\nCyclin D, 355, 413, 455\\nCyclin-dependent kinase 1 (Cdk1), 411 –412,\\n419, 455\\nCyclin-dependent kinase inhibitor proteins\\n(CKIs), 415, 419\\nCyclin E, 413, 455\\nCyclosporin A, 222 –223\\nCYP (cytochrome P450), 124, 149, 158,\\n166–167, 249 –251\\nCysteine, 185\\ncaspase, 415disul ﬁde bond, 191, 199\\ngenetic code, 78iron binding, 265synthesis, 302zinc binding, 200\\nCystic ﬁbrosis, 423 –432, 455\\nCystine, 191, 455\\nCytidine triphosphate (CTP) as energy currency,\\n259\\ndCTP, building block of DNA, 65 –67\\nCytochemistry, 5, 455\\nCytochrome, 249, 265, 455\\nc, 266, 268, 315, 417 –418, 456\\ncoxidase, 268\\nelectron transport chain, 265 –268\\nP450 (CYP), 124, 149, 158, 166 –167,\\n249–251\\nCytokinesis, 404, 456\\nCytology, 2, 456\\nCytoplasm, 19, 456\\ncytoplasmic dynein, 390, 456\\ncytoplasmic streaming, 395, 456\\nCytosine (C), 34, 35, 46, 69, 456\\nbase pair with guanine, 69Cytoskeleton, 381 –397, 456\\nchanges at mitosis, 403\\nCytosol, 19, 261, 317, 456\\nCytosolic calcium, 359, 371, 372\\nprocesses activated by, 348 –350, 395\\nDAmino acids, 196\\nDAG (diacylglycerol), 346, 456\\nDark reactions, 271, 302 –303, 456\\ndATP, building block of DNA, 65 –67\\ndCTP, building block of DNA, 65 –67\\nDeadly nightshade, 376Deamination, 94, 456\\nDeath cap mushroom, 376Death domain receptors, 416, 456\\nde Duve, Christian, 60Degeneracy, of genetic code, 79Dehydrated tissue, 8Deletion, of bases in DNA, 82, 431Dalgarno, 169Denatured proteins, 206 –207, 221\\nDendrite, 350, 456\\nDeoxyadenosine-5′-triphosphate (dATP), 65 –67\\nDeoxycytidine-5′-triphosphate (dCTP), 65 –67\\nDeoxyguanosine-5′-triphosphate (dGTP), 65 –67\\nDeoxyribonucleic acid (DNA), 65, 456\\namount, in cells, 67, 68, 71 –73\\nbases found in, 65 –67\\nblotting, 146classes, 99 –101\\ncloning, 129 –143\\ndamage, 94, 415, 417 –418\\ndata storage medium, 87 –103\\ndouble helix, 68 –69\\nexcision enzymes, 94ﬁngerprint, 146\\nforms of, 71helicase, 90, 98introduction into bacteria, 130 –133\\nligase, 90, 92, 97, 130, 134minisatellite, 100notation, 106packaging, 71 –73\\npolymerase I, 92, 95, 97polymerase III, 90 –92\\npolymerase, thermostable, 152proof that it ’s the genetic material, 66\\npuriﬁcation, 129\\npurine and pyrimidine content of, 71recombinant, 129 –158\\nrepair, 94 –98\\nrepetitious, 100replication, 87 –93\\nsatellite, 100, 403sequencing, 143 –146\\nstaining for microscopy, 6 –7\\nstructure, 65 –71', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='eb4afc04-8bd4-411c-a3ea-abb017297b35', embedding=None, metadata={'page_label': '513', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\nINDEX 509\\nsugar-phosphate backbone, 69\\nsynthesis, 87 –93, 130\\ntesting, 157 –158\\nDeoxyribonucleoside triphosphates, 90Deoxyribonucleotides, 92, 456\\nDeoxyribose, 31 –33, 65, 456\\nDeoxythymidine-5′-triphosphate (dTTP), 65 –67\\nDephosphorylation, 190 –191\\nof proteins, 190 –191, 252, 318, 412\\nof sugars, 295\\nDepolarization, 325 –327, 328 –330, 456\\nDeprotonation, 22 –25\\nDepurination, 94, 457\\nDesigner babies, 158Desmin, 396, 457\\nDesmosomes, 396, 457\\nDetoxication, by cytochromes P450, 249dGTP (deoxyguanosine-5\\n′-triphosphate), 65 –67\\nDiabetes, 299, 317, 457\\nDiacylglycerol (DAG), 346, 457\\nDideoxyadenosine triphosphate (ddATP),\\n143–145\\nDideoxy chain termination method, 143 –146\\nDideoxycytidine triphosphate (ddCTP),\\n143–145\\nDideoxyguanosine triphosphate (ddGTP),\\n143–145\\nDideoxynucleotides, 143 –145\\nDideoxythymidine triphosphate (ddTTP),\\n143–145\\nDiet\\nessential amino acids, 302essential fatty acids, 42ﬁsh odor syndrome, 82 –83\\nglyoxylate and kidney stones, 223metabolism of food chemicals, 44 –46, 109,\\n283–295\\nphenylalanine and phenylketonuria, 294 –295\\nniacin and pellagra, 259starvation and ketone bodies, 290, 299trimethylaminuria, 82 –83\\nvitamin C and scurvy, 190\\nDifferentiation, 119, 457\\nDiffusion, 457\\nacross membranes, 53limit to enzyme speci ﬁcity constant, 247\\nthrough gap junctions, 55 –56\\nthrough connective tissue, 13\\nDigitalis, 271, 320Dihydroxyacetone phosphate, 285 –286, 295,\\n300\\nDiphtheria bacterium, protein synthesis and, 173Diplococcus pneumoniae , transformation, 66\\nDiploid, 402, 405, 457\\nDipole, 199Disaccharide, 28, 457\\nDi Sant ’Agnese, Paul, 424Dissolve, 21, 42Distal axon terminal, 333, 336Disul ﬁde bond (bridge), 185, 191, 199, 457\\nSDS-PAGE and, 166\\nDNA (deoxyribonucleic acid), 65, 457\\namount, in cells, 67, 68, 71 –73\\nbases found in, 65 –67\\nblotting, 146classes, 99 –101\\ncloning, 129 –143\\ndamage, 94, 415, 417 –418\\ndata storage medium, 87 –103\\ndouble helix, 68 –69\\nexcision enzymes, 94ﬁngerprint, 146, 458\\nforms of, 71helicase, 90, 98introduction into bacteria, 130 –133\\nligase, 90, 92, 97, 130, 134, 458\\nmicrosatellite, 100, 146minisatellite, 100notation, 106packaging, 71 –73\\npolymerase I, 92, 95, 97polymerase III, 90 –92\\npolymerase, thermostable, 152proof that it ’s the genetic material, 66\\npuriﬁcation, 129\\npurine and pyrimidine content of, 71recombinant, 129 –158\\nrepair, 94 –98\\nrepetitious, 100replication, 87 –93, 458\\nsatellite, 100, 403sequencing, 143 –146, 458\\nstaining for microscopy, 6 –7\\nstructure, 65 –71\\nsugar-phosphate backbone, 69synthesis, 87 –93, 130\\ntesting, 157 –158\\nDnaA protein, 88 –89, 457\\nDnaB protein, 90, 458\\nDnaC protein, 90, 458\\nDocking protein, 223, 458\\nDomain, 99, 200, 458\\nDominant, 458\\ngenes, 406genetic diseases, 408\\nDonor atom, 26, 458\\nDouble bond in acyl group (fatty acids), 42 –43,\\n300\\nDouble helix structure, 68 –69, 458\\nopening during replication, 88 –90\\nDouble-membrane envelopes, 56 –58, 61\\nDown syndrome, 403Downstream sequences, 106, 458\\nDrinking water, ﬁltering, 10', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='75f37484-a9eb-47bc-b4c2-7f413e337fae', embedding=None, metadata={'page_label': '514', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\n510 INDEX\\nDriving energetically unfavourable reactions,\\n258\\nDrosophila ,6 8\\ndTTP (deoxythymidine-5′-triphosphate), 65 –67\\nDuplication, of genes, 100, 458\\nDynamic instability, 384, 459\\nDynamic protein structures, 238\\nDynamin, 229, 459\\nDynein, 118, 388, 398, 390, 459\\nE2F-1 transcription factor, 413, 414, 459\\nEcoRI, 133\\nEctoplasm, 395, 459\\nEndoplasm, 395, 459\\nEffective stroke, 386, 459\\nEffector caspases, 416, 459\\nEgg\\ndimensions, 2oocytes and gap junctions, 56transmembrane voltage and sperm fusion,\\n325–327\\nwhite lysozyme, 192\\nElectrical\\nelectrically excitable cells, 325 –338, 459\\nforce, on ions at a membrane, 311 –314\\nsignalling, via gap junctions, 15, 55 –56\\nsummation, 372 –375\\ntransmission, down a nerve cell axon,\\n332–334\\nvoltage, 311 –312\\nElectrochemical gradient, 261, 459\\nElectron gun, 8, 459\\nElectron microscope, 2 –3, 4, 8 –9, 16, 459\\nElectrons, sharing, 19 –21\\nElectron transport chain, 263, 265 –268, 278,\\n459\\nphotosynthetic, 274\\nElectrophoresis, 166, 177, 459\\nElectrostatic interactions, 199, 459\\nElodea , 395\\nElongation\\nfactors, 173, 459\\nof protein chain at ribosome, 171 –174\\nEmbedding for microscopy, 8Embryonic stem (ES) cells, 157, 459\\nEndocrine gland, 363Endocytosis, 229, 230, 459\\nendocytotic pathway, 226endocytotic vesicle, 229, 417\\nEndonuclease, 459\\nAP endonuclease, 95restriction endonuclease, 133\\nEndoplasmic reticulum, 57, 60, 61, 460\\ncalcium storage and release, 216, 344 –348\\nchanges at mitosis, 410fat synthesis, 300glycosylation and, 225 –226protein synthesis on, 223 –225\\nrough, 60, 223 –225\\nsmooth, 60\\nEndosome, 230, 460\\nEndosymbiotic theory, 12, 16, 261, 460\\nEndothelium, endothelial cells, 368 –371, 460\\nEnergy currencies, 257, 258 –262, 278, 460\\ninterconvertibility of, 263 –278\\nEnergy production, control of, 303 –306\\nEnergy trading, within cells, 257 –280\\nEngineering\\ngenetic, 129 –158\\nprotein, 149 –150\\nEnhancer sequences, 68, 120, 123, 460\\nEnthalpy, 276, 460\\nEntropy, 276, 460\\nEnvelope, double membrane, 56 –58\\nEnzyme, 241, 460\\nassay, 242 –243\\nbiological catalyst, 254carrier action, 265 –274, 318, 440\\ncatalysis, 241 –249\\ncleavage, 133 –134\\ncomplex with substrate, 249concentration, effect of, 245cooperative behavior of, 240DNA excision and repair, 94reaction, initial velocity of, 242 –244\\nregulation of, 251 –252\\nstudy of, 244timer function, 247\\nEpidermal growth factor (EGF), 355Epidermis, 15, 396, 460\\nEpilepsy, 331Epithelium, 12 –13, 55, 386, 396, 425, 429, 460\\nEquilibrium, 257, 276 –277, 460\\nchemical, 242constant, 277voltage, 311 –314, 460\\nER (endoplasmic reticulum), 57, 60, 61, 461\\ncalcium storage and release, 216, 344 –348\\nchanges at mitosis, 410fat synthesis, 300glycosylation and, 225 –226\\nprotein synthesis on, 223 –225\\nrough, 60, 223 –225\\nsmooth, 60\\nES cell, 157, 461\\nES complex, 242, 245Escherichia coli\\nchromosome size, 67Meselson –Stahl experiment and, 92 –93\\nreplication in, 88 –89\\nRNA synthesis in, 106 –109\\ntranscription termination in, 110\\nEssential amino acids, 259, 302, 461\\nEssential fatty acids, 42, 461', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ffb07d7d-4eef-40f2-9549-483ba02ffc20', embedding=None, metadata={'page_label': '515', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\nINDEX 511\\nEster bond, 39, 461\\nEsterase, speci ﬁc for acetylcholine, 368\\nEthics\\nof DNA testing for inherited disease, 157 –158\\nof gene therapy, 432\\nEuchromatin, 57, 461\\nEukaryotes, eukaryotic, 9, 10, 16, 461\\nchromosomes, 71 –73\\ndifferences from prokaryotic cells, 11\\nDNA, major classes of, 99 –101\\ngene, complexity in, 98 –99\\ngene expression, control of, 119 –124\\ngene structure and organization in, 98 –101\\nmRNAs, 125, 176organization of, 213origin of, 12protein-coding genes, 118protein synthesis, 175 –178\\nribosomes, S value of, 168RNA synthesis, 118 –119\\nEuprymna scolopes , 116\\nEvening primrose oil, 42Evolution\\ncrossing over (chiasmata), 408of cAMP gated channel, 200of eukaryotic cell, 12\\nExchange mechanisms for energy currencies,\\n263–264\\nExcision repair, 94 –98\\nExocytosis, 229, 230, 343, 461\\nExocytotic pathway, 226Exon, 98, 101, 118, 461\\nExonuclease, 92, 461\\nExplosive, nitroglycerine, 370Export, from nucleus, 219Expression, gene, 105, 461\\nExpression vector, 139, 149, 461\\nExtracellular matrix, 13, 53 –54, 61, 461\\nExtracellular medium, 2, 19, 461\\ncalcium entry from, 341 –344\\nExtragenic DNA, 100, 461\\nEye color, 406Eyepiece, 3\\nF-actin, 390 –391\\nσfactor, 106, 108\\nFactor VIII, 149, 157\\nFAD ( ﬂavin adenine dinucleotide), 267, 271,\\n284, 290\\nFainting, 370Fallopian tube, 56Fas receptor, 416, 461\\nFat, 39 –43, 461\\nadipose tissue, 13brown fat, 270cell, 39, 461\\nenergy store, 294hydrolysis, 45 –46\\nsynthesis, 300\\nFatty acid, 39 –43, 462\\nβoxidation, 290\\nsynthesis, 300\\nFeedback, 116, 118, 251, 303 –304, 328, 331,\\n462\\nFeedforward, 304 –306, 462\\nFerredoxin, 275, 462\\nFertilization, 406 –407\\nFetal DNA, analysis of, 158FGF ( ﬁbroblast growth factor), 371\\nFibroblast, 13, 396, 462\\ngrowth factor (FGF), 371, 462\\nFibroin, 195Filter\\noptical, 6 –7\\nwater sterilization, 10\\nFiltration, sterilization by, 10Fish odor syndrome, 82Fission, 462\\ncell division, 11vesicular, 226 –229\\nFixation\\ncarbon, 302 –303\\nnitrogen, 300 –302\\ntissue for microscopy, 8\\nFlagellin, 264, 389, 462\\nFlagellum, ﬂagella, 462\\neukaryotic, 386 –389\\nprokaryotic, 264, 388\\nFlavin adenine dinucleotide (FAD), 267, 271,\\n284, 290\\nFlavin-containing monooxygenase, 82, 101Fluidity of membranes, 42 –43\\nFluorescence microscopy, 6 –7\\nFluorescent protein, 150, 200Fly agaric mushroom, 376fmet (formyl methionine), 5, 169 –171, 176,\\n462\\nFMO3 gene, 76, 82, 101\\nFocal contacts, 395, 462\\nFollicle, 56Foreign DNAs, joining to a cloning vector,\\n133–134\\nForms, αandβ, of monosaccharide, 27 –30\\nFormyl group, 169, 171Formyl methionine (fmet), 5, 169 –171, 176,\\n462\\nFourteen-three-three protein, 358, 417, 462\\nFoxglove, 271Frameshift mutation, 80, 82, 462\\nFranklin, Rosalind, 68Free energy, 258, 276 –277\\nof solute at concentration [I], 312\\nFrequency coding, 336Frequency modulation (FM), 336', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1e8b02c7-343c-48f2-9982-9c2b88c85a26', embedding=None, metadata={'page_label': '516', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\n512 INDEX\\nFructose, 27, 29, 243, 303\\n6-phosphate and 1,6-bisphosphate, 284 –285,\\n295, 303\\nin gluconeogenesis, 295\\nin glycolytic pathway, 284 –285\\nFruit ﬂy, 68\\nFumarate, 267, 283 –284, 292 –293\\nFusion, membrane, 231\\nγ-amino butyrate (GABA), 37 –38, 373 –375\\nγ-linolenic acid, 42 –43\\nG (Gibbs free energy), 258, 276 –277\\nof solute at concentration [I], 312\\nG0 (gap 0) phase, 409, 463\\nG1 (gap 1) phase, 409, 463\\nG1/S cell cycle control point, 412 –415\\nG2 (gap 2) phase, 409, 463\\nG2/M cell cycle control point, molecular\\nregulation of, 410 –412\\nGq, 344 –346\\nGs, 350, 353\\nGABA ( γ-amino butyrate), 37 –38, 373 –375,\\n463\\nGalactose, 27, 29\\nlacoperon, 109 –115\\nG-actin, 390 –391, 463\\nβ-Galactosidase, 109 –113, 132\\nin pBluescript vector, 132\\nβ-Galactoside, 111 –112, 115\\nGamete, 406, 408, 463\\nGAP (GTPase activating protein), 218, 463\\nGap junction, 55 –56, 463\\nchannel, 56, 204, 314, 463\\nGap0 (G0) phase, 409, 463\\nGap1 (G1) phase, 409, 463\\nGap2 (G2) phase, 409, 463\\nGastrocnemius muscle, 363, 365 –372, 376, 463\\nGated, gating, of channels, 314, 463\\nGated transport, 213, 214, 463\\nthrough the nuclear pore, 216 –218\\nGC box, 120\\nGDP (guanosine diphosphate), 173, 259, 463\\nGEF (guanine nucleotide exchange factor),\\n218–219, 228, 344, 350, 355, 464\\nGEFS (generalized epilepsy with febrile\\nseizures), 331, 464\\nGelsolin, 395, 464\\nGene, 464\\nchip, 154 , 464\\ndetection, 146 –147\\nexpression, 105, 109 –124\\nfamily, 100, 464\\nknockout, 157nomenclature, 101notation, 106number, 67 –68\\nprobe, 143, 464structure/organization, in eukaryotes, 98 –101\\ntherapy, 158, 428 –431, 432, 464\\ntranscription, 105 –124\\nGeneralized epilepsy with febrile seizures, 331Genetic\\ncode, 75 –80, 464\\ncounseling, 432disease, 82 –83, 294 –295, 406 –408, 423 –424,\\n432\\nengineering, 129 –158\\nﬁngerprinting, 146\\nmaterial, proof that it is DNA, 66modi ﬁcation of plants (GM), 158, 464\\nGenome, 56, 67, 464\\nproject, 68, 99, 144, 154\\nGenomic DNA clones, 139 –143, 159\\ngeneration and selection of, 142\\nGenomic DNA library, 140, 464\\nGerard, Ralph, 320Germ cells, 404 –405, 464\\nGibbs, J. Willard, 276\\nfree energy, 258, 276 –277, 312, 464\\nGland\\nadrenal, 305, 353, 368endocrine, 363salivary, 348sweat, 424 –425\\nGlia, glial, 7, 13, 334, 410, 464\\nGlucocorticoid, 121 –124, 464\\nGluconeogenesis, 295 –297, 306, 464\\nGlucosamine, N-acetyl, 32 –33\\nGlucose, 27 –31, 243, 464\\n6-phosphate, 258, 277breakdown, 284 –288\\ncarrier, 238, 316 –317, 357, 465\\nconcentration in cells, 277diabetes, 317energetics of phosphorylation, 258, 277fats as source, 295 –297\\ngluconeogenesis, 295glycogen as source, 288 –289\\nglyoxylate shunt, 297oils as source, 295 –297\\noxidation in pentose phosphate pathway, 289phosphorylation, 258, 277regulation by feedback and feedforward,\\n304–306\\nrepression of lacoperon, 114\\nsleeping sickness, 288storage as glycogen, 298synthesis, 295\\nGlutamate, 185\\ncharge on side chain, 185, 190conversion to 2-oxoglutarate, 292genetic code, 78phosphorylation of side chain, 190 –191\\nmutation, in sickle cell anaemia, 81', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b757d990-bd9e-4180-b7bd-a2b48aaa59ac', embedding=None, metadata={'page_label': '517', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\nINDEX 513\\nnitrogen metabolism and urea cycle, 292 –293\\nreceptor, 140 –141, 372, 439\\nsynthesis, 302\\ntoxic analogue, 372transmitter action, 341 –343, 372\\nGlutamine, 185\\ngenetic code and wobble, 78, 80, 164nitrogen ﬁxation, 302\\nsynthesis, 302\\nGlutaraldehyde, 8Glyceraldehyde 3-phosphate, 285 –286\\nGlycerides, 39, 465\\nsynthesis, 300\\nGlycerol, 39, 40, 300, 465\\nGlycine, 185, 186, 188Glycogen, 30, 243, 465\\nbreakdown, 288 –289, 305\\nglucose source, 288 –289\\nphosphorylase, 288, 305, 465\\nphosphorylase kinase, 242, 305, 465\\nsynthesis, 298 –299\\nGlycolysis, 251, 284 –290, 306, 465\\nglycogen and, 288negative feedback control of, 304without oxygen, 286 –288\\nGlycosidic bond, 28, 465\\nGlycosylation, 53, 190, 225 –226, 465\\nGlyoxylate, 465\\nshunt, 297 –298, 465\\nGM (genetically modi ﬁed), 158, 465\\nGoldberger, Joseph, 259Golgi apparatus, 60 –61, 215, 225 –231, 357, 465\\ncell center, 61changes at mitosis, 410glycosylation, 226\\nGolgi, Camillo, 60Gordian knot, 74Gout, 385G protein (trimeric), 247, 344 –346, 350, 359,\\n465\\nGq, 344 –346, 465\\nGramicidin A, 196Granum, grana, 58, 465\\nGratuitous inducer, 114, 466\\nGraves, Robert, 376Grb2, 355, 466\\nGreen eyes, 406Green ﬂuorescent protein, 150, 466\\nβbarrel structure, 200\\nchimeric proteins, 150protein traf ﬁcking, 226\\nGrifﬁth, Fred, 66\\nGround substance, 13Growth factor(s), 466\\napoptosis inhibition, 416 –417\\ncell division promotion, 371, 413mechanism and pathways, 353 –356, 358 –359receptor binding protein number 2 (Grb2),\\n355, 466\\nreceptor structure and synthesis, 198, 204,\\n223\\nGrowth hormone, 157G\\ns, 350, 353, 466\\nGTP, 466\\nas energy currency, 259building block of RNA,dGTP, building block of DNA, 65 –67, 90\\ngeneration in Krebs cycle, 284guanylate cyclase, 353, 365in gluconeogenesis, 295in protein synthesis, 173\\nGTPase, 218, 247, 344, 466\\nactivating protein (GAP), 218, 466\\nARF and coatamer coat formation, 228, 229dynamin and vesicle budding, 229G\\nq, 344 –346\\nGs, 350, 353\\nRab and vesicle fusion, 231Ran and nuclear transport, 218 –220\\nRas and MAP kinase, 355timer function, 247\\nGuanine (G), 34, 35, 46, 69, 466\\nGuanine nucleotide exchange factor (GEF),\\n218–219, 228, 344, 350, 355, 466\\nGuanosine diphosphate (GDP), 173, 259, 466\\nGuanosine triphosphate (GTP), 466\\nas energy currency, 259building block of RNA,dGTP, building block of DNA, 65 –67, 90\\ngeneration in Krebs cycle, 284guanylate cyclase, 353, 365in gluconeogenesis, 295in protein synthesis, 173\\nGuanylate cyclase, 353, 365, 466\\nGuthrie test, 295, 466\\nGyrase, 89\\nH+,2 1–25\\nconcentrations, 261, 310\\ngradient, driving bacterial ﬂagellum, 264\\ngradient, energy currency, 261 –262\\nprotonation, 25, 185transport by ATP synthase, 269, 272 –273\\ntransport by electron transport chain, 265 –268\\ntransport by thermogenin, 270\\nH\\n2O2,5 9–60\\nH3O+,2 2\\nHair, 183, 195\\ndryer and pain receptors, 372keratin, 184, 195, 396root hair, 16shampoo, 167\\nHairpin loop, in transcription termination, 109,\\n467', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='b9d65601-5589-49b7-aa5a-024cc111d810', embedding=None, metadata={'page_label': '518', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\n514 INDEX\\nHaploid cells, 405, 419, 467\\nHartridge, Hamilton, 252\\nHartwell, Leland, 411Head group, 42, 467\\nHeart, 14 –15, 56, 290, 315, 320, 348, 370, 393,\\n396, 417\\nHeat\\nas denaturing agent, 206in thermodynamics, 276generation in brown fat, 270-killed bacteria, 66prostration, in CF, 424-shock proteins, 221\\nHeavy DNA, Meselson-Stahl experiment,\\n92–93\\nHeel prick test, 295, 467\\nHelicase, 90, 467\\nHelix\\nactin, 390α-helix, 195, 467\\namphipathic helix, 215chirality, 196destabilizing proteins, 90DNA helix, 68 –71\\nglycogen, 30helix-turn-helix motif, 200recognition helix, 200transmembrane, 200\\nHeme, 205, 249, 265, 268, 274Hemicellulose, 53, 467\\nHemoglobin, 119, 204 –205, 467\\nallosteric effects and, 238 –240\\nglobin gene family, 100oxygen-binding curve for, 239, 240sickle cell anaemia, 81, 82\\nHemophilia, 149Hepatitis C virus, 176Herbicide resistance (GM), 158Heterochromatin, 57, 467\\nHeterozygote, 467\\nHexokinase, 277, 284, 467\\nHexose, 27Higher-level protein structures,\\n206–207\\nHistidine, 185\\nessential amino acid, 302genetic code, 78pH sensitivity, 185, 240 –241\\nphosphorylation, 190, 241protonation, 185, 240 –241\\nHistones, 57, 73, 100, 202, 467\\nH2O2,5 9–60\\nH3O+,2 2\\nHoechst dye, 6 –7\\nHomologous recombination, 157, 467\\nHooke, Robert, 1Hooves, 396Hormone, 363, 467\\nresponse element (HRE), 122 –123, 365, 467\\nsteroid, 53, 60, 121 –124, 249, 365\\nHorns, 396Horseradish peroxidase, 166Housekeeping gene, 120, 468\\nHRE (hormone response element), 122 –123,\\n365, 468\\nHsp90, 122Human Genome Project, 68\\ndisease genes and, 154DNA sequencing and, 144 –146\\nHuman immunode ﬁciency virus (HIV), 12\\nHuman proteome, 178Hunt, Tim, 411Huntington ’s disease, 157\\nHybridization, 139, 468\\nHydrated gel, 190Hydration shell, 21, 468\\nHydrocarbon tail, 39, 468\\nHydrogen bond, 25 –26, 195, 468\\nbetween bases in DNA, 26, 67 –71\\ninαhelix, 195\\ninβsheet, 195\\nin protein structure, 195in water, 27\\nHydrogen ion, 21 –25\\nconcentrations, 261, 310gradient, driving bacterial ﬂagellum, 264\\ngradient, energy currency, 261 –262, 468\\nprotonation, 25, 185transport by ATP synthase, 269, 272 –273\\ntransport by electron transport chain, 265 –268\\ntransport by thermogenin, 270See also pH\\nHydrogen peroxide, 59 –60\\nHydrolysis, 44 –46, 47, 468\\nHydropathy plot, 204, 468\\nHydrophilic, 42, 468\\namino acid side chains, 80 –81, 185, 186, 192,\\n204\\naction of cytochromes P450, 249 –251\\nhead group, of phospholipids, 42solute, transport across membranes,\\n309–320\\nHydrophobic, 39, 468\\nacyl groups, 39amino acid side chains, 80 –81, 185 –186, 192,\\n199, 204\\neffect, 199interaction, 199lipid bilayer interior, 53, 309signal sequence, 215, 216, 223 –225\\nsolvent, 39tail, of ARF, 228tail, of chlorophyll, 274tail, of ubiquinone/coenzyme Q, 265', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='449cb60b-d58b-45d6-95aa-2ce6b75c3fe1', embedding=None, metadata={'page_label': '519', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\nINDEX 515\\ntail, of fatty acid, 39\\ntransmembrane helix, 200, 204, 223 –225\\nHydrostatic pressure, 11, 543-hydroxybutyrate, 290Hydroxyl groups, 27, 28, 468\\nin side chains, 188\\nHydroxylysine, 190Hydroxyproline, 190Hyperforin, 124Hypoxanthine, 34, 35, 46, 164, 468\\nImino acids, 185 –186, 468\\nImmune system, 7, 222, 432Immuno ﬂuorescence, 7, 468\\nImmunoglobulin gene family, 100Immunosuppressants, 222 –223\\nImport\\nER, 223 –225\\nnuclear, 218 –219\\nmitochondrial, 215\\nInactivation, 327, 330, 331, 468\\nInduced ﬁt, 249\\nInducers, gratuitous, 114Inducible operon, 111 –115, 469\\nInheritance, 406 –407\\nInhibition\\nacetylcholine receptor by atropine, 376acetylcholinesterase by Sarin etc., 376contact, 415GABA at synapse, 374phosphofructokinase by ATP, 251, 304\\nInitial rate, 243Initial velocity ( v\\no), 242 –244, 254, 469\\neffect of substrate concentration on, 244 –245\\nmeasurements of, 252\\nInitiation (of)\\ncoatamer assembly, by ARF, 228complex, on DNA, transcriptional, 120, 122complex, 70S, protein synthesis, 171, 176complex, 80S, protein synthesis, 176factors, 171protein synthesis, 168 –171, 175 –176\\ntranscription, 106 –109, 120\\nINK4a , 184, 415\\nInosine, 164Inositol, 469\\nInositol trisphosphate (IP 3), 37, 344 –348, 359,\\n469\\ngated calcium channel, 344, 346 –348, 469\\nInput-output relations at a synapse, 372 –375\\nInsecticide toxicity, 376Insertional mutagenesis, 157, 469\\nIn situ hybridization, 147 –148, 469\\nInsulation, of axons, 334Insulin, 469\\ndiabetes, 299, 317mechanism and pathways, 317, 356 –357receptor, 356 –357, 469\\nreceptor substrate number 1 (IRS-1), 356,\\n469\\nrecombinant, 149\\nInsurance, DNA testing and, 157 –158\\nIntegral membrane protein, 52 –53, 200, 469\\nsynthesis, 223 –225\\ntrafﬁcking, 226, 230\\nIntegration\\nof cell processes by intracellular messengers,\\n350\\nof mRNA back into genome, 100summation at synapse, 372 –375\\nIntegrin, 392 –393, 469\\nIntercellular communication, 55 –56, 363 –375\\nInterleukin 2, 222, 432Intermediate ﬁlament, 396, 469\\nIntermembrane space, 470\\nmitochondrial, 58, 261 –262, 266\\nnuclear, 56\\nInternal ribosome entry site (IRES), 176Interphase, 57, 401, 408, 470\\nInterphase/mitosis cell cycle control, 410 –412\\nIntestine, 12 –13, 14, 44 –46, 55, 82, 109, 423\\nIntracellular\\ncompartments, vesicular traf ﬁcking between,\\n226–232\\nmessengers, 341 –353, 368 –370, 470\\nprotein transport, modes of, 213 –215\\nreceptors, 365, 470\\nsignaling, 341 –362\\ntransport, 389 –390\\nIntron, 68, 98, 101, 118, 470\\nIon, 21, 470\\nanion, 21binding site on protein, 190cation, 21concentrations, 310equilibrium voltage, 311, 312 –313\\nforce due to voltage, 311 –314\\nhydration in solution, 21movement across membranes, 53, 55,\\n309–320\\nprosthetic group, 205\\nIonic compounds, 21, 23Ionotropic cell surface receptor, 364 –365, 470\\nIP3(Inositol trisphosphate), 37, 344 –348, 359,\\n470\\ngated calcium channel, 344, 346 –348\\nIPTG (isopropylthio- β-D-galactoside), 114, 139\\nIRES (internal ribosome entry site), 176Iron, 158, 205, 265 –268, 275, 302\\nIRS-1 (insulin receptor substrate number 1),\\n356, 470\\nIsocitrate, 284, 297Isoelectric point, 25, 178, 470\\nIsoforms, 100, 470', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4852c045-8beb-4d7a-85b9-53204337934b', embedding=None, metadata={'page_label': '520', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\n516 INDEX\\nIsoleucine, 185\\nessential amino acid, 302\\ngenetic code, 78inosine in tRNA, 165\\nIsomers, 27, 196, 470\\nIsopropylthio- β-D-galactoside (IPTG), 114, 139\\nJaffe, Laurinda, 325 –327\\nJelly ﬁsh, 150\\nJNK protein kinase, 418, 470\\nJunctions, 54 –56, 396\\nadherens, 396anchoring, 396desmosome, 396gap, 15, 55 –56, 314, 437\\ntight, 55\\nK\\na,2 4–25\\nkcat, 242, 244, 245, 247\\nKeq, 277\\nKartegener ’s syndrome, 389, 471\\nKDEL, 215, 216Keratin, 184, 195, 396, 471\\nKeratinocytes, 396Ketone bodies, 290, 299, 471\\nKhorana, Gobind, 77Kidney stones, 223Kinase, 190, 471\\nA, 200, 305, 353, 368, 427B, 357, 358, 417C, 358G, 370calcium-calmodulin-activated, 358cAMP-dependent, 200, 305, 353, 368,\\n427\\ncGMP-dependent, 370cyclin dependent, 411 –415\\nglycerol, 300glycogen phosphorylase kinase, 242, 305,\\n348, 353\\nhexokinase, 277, 284intrinsic to Ca\\n++ATPase, 318\\nMAP, 353 –355, 371\\nphosphofructokinase, 251, 285, 304phosphoinositide 3-kinase, 356, 358, 417polynucleotide, 139protein, 190, 241protein kinase B, 357, 358, 417protein kinase C, 358receptor tyrosine, 353 –357, 365, 371, 417\\nstress activated, 418\\nKinesin, 390, 398, 471\\nKinetics, of chemical reaction, 242 –248\\nsaturation kinetics, 245\\nKinetochore, 403, 471\\nKinks, in fatty acid chains, 42 –43\\nKIP1, 415KM, 244 –247, 471\\nof recombinant enzyme, 150\\nKnock-out transgenic mice, 156, 157\\nCFTR , 428, 430\\nKoshland, Daniel, 249Kozak sequence, 176Krebs cycle, 283 –284, 306, 471\\nKrebs, Hans, 283\\nlacoperon and repressor, 109 –114, 238, 471\\nin DNA cloning, 132, 139\\nLactic acid, lactate, 22, 24, 287\\nLactose, 30, 471\\nintolerance, 44, 471\\nlacoperon, 111 –115, 471\\nlacZ gene, 132\\nLagging strand, 90, 471\\nLambda (bacteriophage), 140, 426\\nLAmino acids, 196\\nLamins, 396, 471\\nLead citrate, 8Leader sequence, 169, 176Leading strand, 90, 471\\nLens, 3, 8, 183Leucine, 185\\nessential amino acid, 302genetic code, 77, 78\\nLeukemia, 154Library\\ncDNA, 134genomic DNA, 140in cloning CFgene, 426\\nscreening, 136 –139\\nLigand, 183, 471\\nallosteric effects, 238, 251binding site on protein, 237clathrin-mediated vesicle formation, 229rapid reaction techniques, 252 –253\\nregulation of protein activity, 238, 251\\nLight\\nabsorption, in assays, 243, 252 –253\\nDNA, Meselson-Stahl experiment, 92 –93\\nemission, squid, 116ﬂuorescence, 6 –7\\npolarized, 196, 243microscope, 3 –7\\nreactions of photosynthesis, 274 –275, 471\\nsensitive-nerve cells, 353, 401ultraviolet, DNA damage, 94, 98, 415, 418\\nLignin, 54Limit\\nof resolution in microscopy, 2 –3\\nto enzyme-catalyzed reactions, 245, 247\\nLind, James, 190LINEs (long interspersed nuclear elements), 100Lineweaver –Burk plot, 246\\nLing, Gilbert, 320', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='9b69c898-f6ea-4993-b14b-8c6548473475', embedding=None, metadata={'page_label': '521', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\nINDEX 517\\nLink, linker\\nDNA, 73\\nphosphodiester, 42, 67, 89, 90, 92, 95, 97,\\n105, 109, 118\\npolylinker, 150inactivation plug, 327, 330, 331\\nLinkage, 408, 472\\nLinoleic acid, 42 –43\\nLinolenic acid, 42, 43Lipid, 39 –43\\nβoxidation, 290\\nbilayer, 42, 52 –53, 309, 472\\nbilayer voltage clamp, 428diacylglycerol (DAG), 346ﬁsh odor syndrome, 82 –83\\nhydrolysis, 45liposomes, 229liposomal-mediated gene transfer, 429peroxisomes, 60phosphatidylinositides (PIP\\n2, PIP 3etc), 231,\\n346, 357, 417\\nphospholipid, 40 –45\\nsynthesis, 300translocation to plasma membrane, 230trimethylaminuria, 82 –83\\nLipolysis, 40Listeria , 393\\nLiver (cell)\\nalanine glyoxylate aminotransferase and\\nkidney stones, 223\\ndetoxication (cytochromes P450), 60, 124,\\n149, 166 –167\\nfailure, 293ﬁsh odor syndrome, 82 –83\\nglucose export, 289glycogen, 30, 288 –289\\nhepatitis C, 176identity and gene expression, 119ketone bodies, 290lactate oxidation, 287lipid synthesis, 300specialized biochemical center, 281trimethylaminuria, 82 –83\\nurea cycle, 292 –293\\nLocal anaesthetic, 334Localization\\nnuclear localization signal, 216, 218 –219\\nsequence, 215, 472\\nLong interspersed nuclear elements (LINEs),\\n100\\nLuciferase, 116Lung\\ndiffusion of gases, 53disease in cystic ﬁbrosis, 423, 428 –431\\nepithelium, 12 –13\\nhemoglobin oxygen loading, 238 –240\\nmucus clearance and cilia, 13, 386Luxoperon, 116\\nLuxR, 116Lymphocyte and immunosuppressants, 222Lysine, 185\\ncharge on side chain, 190essential amino acid, 302genetic code, 77, 78hydroxylation, in collagen, 190in histones, 73, 202Lysosomes, 61, 472\\nfunction of, 231targeting proteins to, 230 –231\\nLysozyme, 192 –195\\nspeci ﬁcity constant, 247\\nstructure and shape, 192 –195, 199\\nM phase, 408 –410, 472\\nM phase promoting factor (MPF), 411, 472\\nM13 bacteriophage, 74MacLeod, Colin, 66Macromolecule, 27 –39, 472\\nMacrophage, 13, 61, 415, 472\\nMad cow disease, 207, 208, 408Magnesium, 205, 274Major groove, 69, 200, 202, 472\\nMalate, 284, 293, 297\\ndehydrogenase, regulation by calcium, 305\\nmalonyl-CoA, 300Mammalian proteins, produced in bacteria, 149Manganese, 205, 274Mannose, 27, 29, 472\\n6-phosphate, lysosomal targeting, 32, 33, 231glycosylation in endoplasmic reticulum, 225\\nMAP kinase, 355, 472\\ncascade, 353 –355\\nMAP kinase kinase (MAPKK), 355, 472\\nMAP kinase kinase kinase (MAPKKK), 355,\\n472\\nMarfan ’s syndrome, 13\\nMatrix, 472\\nchloroplast, 58extracellular, 13, 53 –54, 124, 190, 392\\nmitochondrial, 58, 261 –262, 283, 290, 305\\nMaximal velocity ( Vm), 245, 246, 472\\nMcCarty, Maclyn, 66MCS (multiple cloning site), 132, 150Mechanical molecules, 381 –400\\nMedieval saints, 299Meiosis, 404 –406, 419, 472\\nMeiosis I, 405, 472\\nMeiosis II, 405, 472\\nMelting point, of fats, 39, 42 –43\\nMembrane, 42, 51 –53, 473\\nbarrier function, 51, 309, 315, 317basement, 13-bound organelles, 56 –61\\ncell surface, 51', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='5a4b0c1d-0246-4c3f-be49-698bebbce815', embedding=None, metadata={'page_label': '522', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\n518 INDEX\\nMembrane ( Continued )\\ncracking, 42\\ndiffusion across, 53, 121, 365double-membrane envelopes, 56 –58\\nﬁsh odor syndrome, 82 –83\\nﬁssion and fusion, 226 –231\\nﬂuidity, 42\\nintegral membrane proteins, 52 –53, 202,\\n223–225, 226, 230\\nintermembrane space, 56 –58, 261 –262, 266\\nnylon, in genetic engineering, 136, 139, 143,\\n146, 148 –149\\nnylon, in western blotting, 166 –167\\nplasma, 1, 51proteins, tertiary structures in, 200single-membrane envelopes, 58 –61\\nsolidifying, 42structure, 39 –43, 52\\ntransmembrane translocation, 213, 221 –225\\ntrimethylaminuria, 82 –83\\nvoltage, 261, 311 –314, 320 –321, 325 –336,\\n372–375\\nMenten, Maud, 245Meselson, Matthew, 92\\nMeselson –Stahl experiment, 92 –93\\nMessenger RNA, 75 –76, 105 –120, 473\\ncDNA library, 130central dogma, 39northern blotting, 148 –149\\npolycistronic, 109processing after transcription, 68, 98 –99,\\n118–119\\nsplicing, 68, 98 –99, 118 –119\\ntranslation, 163 –176\\nMessenger, intracellular, 341 –353, 368 –370\\nMetabolism, 281 –308, 473\\nMetabotropic cell surface receptors, 365, 473\\nMetalothionein gene, 155Metaphase, 403, 473\\nMethionine, 185\\nessential amino acid, 302formyl, 5, 169, 171genetic code, 78, 79start signal, 79, 168, 175 –176\\nsubtilisin, engineering methionine out, 150\\n7-Methyl guanosine cap, 118, 176, 473\\nMichaelis, Leonor, 245\\nconstant (K M), 244 –247, 473\\nMichaelis –Menten equation, 245, 246, 473\\nMicroarray analysis, 154, 473\\nMicro ﬁlament, 390 –392, 398, 473\\nMicropipette, 320 –321, 325 –326\\nMicrosatellite DNA, 100, 146, 473\\nMicroscope, microscopy, 2 –9\\nbright- ﬁeld, 3 –5\\nelectron, 2 –3, 4, 8 –9\\nﬂuorescence, 6 –7light, 3 –7\\nphase-contrast, 5principles of, 2 –9\\nscanning electron, 9\\nMicrospora, 12Microtubule, 381 –390, 473\\ndynamic instability, 384mitotic spindle, 403, 404molecular motors, 388 –390, 473\\norganizing center, 382, 473\\nMicrovillus, microvilli, 13, 474\\nMinisatellite DNA, 100, 474\\nMinor groove, 69, 474\\nMirror\\ndichroic, 6image structures, 196\\nMissense mutation, 80 –81, 82, 474\\nMitochondria, mitochondrial, 58, 474\\napoptotic role, 315, 417ATPase, 269βoxidation, 290\\ncalcium channel, 305, 348, 438chromosomes, 11electron transport chain, 265 –268\\nheat generation, 270hydrogen ion gradient, 261 –262\\ninner membrane, 261 –269, 272 –273, 474\\nKrebs cycle, 283 –284\\nmatrix, 58, 261 –262, 283, 290, 305, 474\\n[NAD+]/[NADH] ratio, 277\\norigin as bacteria, 12outer membrane, 221, 261 –262, 315, 417, 474\\nprotein targeting and import, 215, 221, 223ribosomes, 11, 168transmembrane voltage, 261 –262\\nunusual genetic code, 80\\nMitogen-associated protein kinase (MAP\\nkinase), 355, 474\\nMitosis, mitotic, 402 –404, 474\\ncontrol by cdk1, 410 –412\\nspindle, 402, 474\\nMolecular biology, central dogma of, 39Molecular motor, 264, 388, 390, 393 –395\\nMolybdenum, 302Monomer, 27, 474\\nMonosaccharide, 27 –29, 474\\nmodi ﬁed, 31\\nMotifs, in tertiary structures, 200, 474\\nMotility, 386 –390, 393 –395\\nMotoneurone, 367 –368, 475\\nMotor proteins, 264, 388, 390, 393 –395\\nMouse genome, 99M phase, 408 –410, 475\\npromoting factor (MPF), 411, 475\\nmRNA, 75 –76, 105 –120, 475\\ncDNA library, 130central dogma, 39', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='135b3d3d-a9ef-4d57-85f4-93e0896a45ce', embedding=None, metadata={'page_label': '523', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\nINDEX 519\\nnorthern blotting, 148 –149\\npolycistronic, 109\\nprocessing after transcription, 68, 98 –99,\\n118–119\\nsplicing, 68, 98 –99, 118 –119\\ntranslation, 163 –176\\nMTOC (microtubule organizing center), 382,\\n475\\nMultiple cloning site (MCS), 132, 150Muscarine, 376Muscarinic acetylcholine receptor, 370, 475\\ninhibition by atropine, 376\\nMuscle, 14, 475\\nactivation, 348 –349, 367 –368\\nanaerobic, 275, 286 –287, 350\\nanchoring junctions, 396blood supply, 368 –372\\ncalcium effects, 348 –350, 394\\ncalf, 365 –372\\ncAMP effects, 353cGMP effects, 370cardiac, 14, 55 –56\\ncramp, 21, 286 –287\\ndesmin, 396digitalis, 320feedforward control, 304 –306\\nﬁber, 15\\ngap junctions, 56gastrocnemius, 365 –372\\nglucose carrier, translocation to plasma\\nmembrane, 317, 357\\nglycogen storage and breakdown, 30, 288,\\n304–306\\nintermediate ﬁlament, 396\\nketone bodies as fuel, 290mechanism of contraction, 393 –394\\nmitochondria, 58neurotrophin synthesis, 417paralysis (BoTox), 232resting voltage, 311ryanodine receptors, 348 –349\\nskeletal, 14, 365smooth, 14, 368striated, 14Viagra, 370\\nMushroom toxicity, 376Mutagenesis\\ninsertional, 157site-directed, 149\\nMutation, 87, 475\\ndetection by Southern blotting, 146FMO3 gene, 82\\nframeshift, 80, 82gene families, generation of, 100missense, 80 –81, 82\\nnonsense, 80, 82pseudogenes, generation of, 100Myelin, myelin sheath, 334 –336, 475\\nMyoglobin, 238 –240\\nMyosin, 393 –395, 398, 475\\nMyosin II, 394, 475\\nMyosin V , 395, 475\\nN-Acetyl glucosamine, 32 –33, 225\\nN-Acyl-HSL, 116\\nN-Acyl-HSL synthase, 116\\nN terminal, 184, 475\\nNa+,2 1\\naction potential, 332 –337\\nbinding site, 190/calcium exchanger, 317, 320channel, voltage gated, 200, 330 –336, 350,\\n368, 438\\nchloride, 21, 424concentration, 310crossing lipid bilayer, 53dodecyl sulfate (SDS), 166, 167glutamate, 343gradient, energy currency, 262hydroxide, 136lauryl sulfate, 167/potassium ATPase, 270, 271, 312\\nNAD\\n+, NADH, 37, 259, 263 –277, 281 –295,\\n348, 475\\nADP ribosylation, 173dehydrogenase, 267in the electron transport chain, 265 –267\\noxidation, 259\\nNADH-Q oxidoreductase, 267NADP\\n+, NADPH, 271 –275, 289, 300, 302, 475\\nNa+/K+ATPase, 270, 271, 475\\nNames, of genes, 101Necrosis, 415, 476\\nNegative\\nfeedback, 118, 251, 303 –304, 476\\nregulation, 114, 476\\nNeher, Erwin, 320Nematode ( c. elegans ), 68, 418\\nNernst equation, 312 –313, 476\\nNerve, nerve cell, 13, 476\\naction potential, 325, 330 –337\\nautonomic, 368 –370\\naxon, 13axonal transmission, 332 –336\\naxonal transport, 390botulinum toxin, 232ELA V , 7electrical signalling, 330 –337\\nﬁber, 336\\nintermediate ﬁlaments, 396\\nlight sensitive, 353motoneurone, 367nerve gas, 376neurotrophin, 417, 476', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='2e8b788b-f9e2-4f1a-8d5d-8db324065304', embedding=None, metadata={'page_label': '524', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\n520 INDEX\\nNerve, nerve cell ( Continued )\\nresting voltage, 311\\nsaltatory conduction, 336scent-sensitive, 350 –353\\nstroke, 417synapse, 341 –344, 367 –368, 372 –375\\nterminal, 332 –334, 341 –344, 367, 371,\\n372–375\\nterminally differentiated, 409urea toxicity, 293voltage-gated sodium channel, 330 –336, 350,\\n368, 438\\nNervous system, frequency coding in, 336Nervous tissue, 13Neuro ﬁlament, 396, 476\\nNeuron (nerve cell), 13, 476\\naction potential, 325, 330 –337\\nautonomic, 368 –370\\naxon, 13axonal transmission, 332 –336\\naxonal transport, 390botulinum toxin, 232ELA V , 7electrical signalling, 330 –337\\nﬁber, 336\\nintermediate ﬁlaments, 396\\nlight sensitive, 353motoneurone, 367neurotrophin, 417resting voltage, 311saltatory conduction, 336scent-sensitive, 350 –353\\nstroke, 417synapse, 341 –344, 367 –368, 372 –375\\nterminal, 332 –334, 341 –344, 367, 371,\\n372–375\\nterminally differentiated, 409urea toxicity, 293voltage-gated sodium channel, 330 –336, 350,\\n368, 438\\nNeurotrophin, 417p75 neurotrophin receptor, 417Neutral pH, 24NFAT, 190 –191, 222 –223, 350, 358, 371, 476\\nN-glycosylation, 225Niacin, 259Nicotinamide, 34, 35, 46, 476\\nNicotinamide adenine dinucleotide (NAD+,\\nNADH), 37, 259, 263 –277, 281 –295, 348,\\n476\\nADP ribosylation, 173dehydrogenase, 267in the electron transport chain, 265 –267\\noxidation, 259\\nNicotinamide adenine dinucleotide phosphate\\n(NADP\\n+, NADPH), 271 –275, 289, 300,\\n302, 476Nicotine, 376Nicotinic acetylcholine receptor, 364, 368, 376,\\n439, 477\\nNightshade, 376Nirenberg, Marshall, 77Nitella , 395\\nNitric oxide, 53, 365, 370, 477\\nreceptor, 365\\nNitrogen, 25 –26, 27, 35, 82, 195, 237, 292 –293\\nexcretion, 292 –293\\nﬁxation, 300 –302\\nNitrogenase, 302Nitroglycerine, 370NO (nitric oxide), 53, 365, 370\\nreceptor, 365\\nNode, 334, 335, 477\\nNomenclature, gene, 101Nondividing cells, 409Nonmuscle myosin, 394 –395\\nNonpolar, 20, 21, 477\\nNonsense mutation, 80, 82, 477\\nNonvirulent bacteria, 66Noradrenaline, 365, 368Northern blot, 148 –149, 477\\nNose and scent receptors, 350 –353\\nN terminal/terminus, 173, 184, 477\\nNucleoid, 73, 477\\nNucleolus, 57, 477\\nchanges at mitosis, 58, 410, 412nucleolar organizer region, 57, 477\\nNucleoporins, 216Nucleoside, 35 –37, 477\\ntriphosphates, 259 –261\\nNucleosome, 73, 477\\nNucleotide, 37, 478\\nbases found in, 34as energy currencies, 258, 259 –261\\nexcision –repair, 97, 478\\nin DNA, 65 –67\\nin RNA, 105 –106\\nNucleus, nuclear, 56 –58, 478\\nchanges at mitosis, 57 –58, 402 –404, 410, 412\\nenvelope, 56, 477\\nimport/export, 216 –220\\nlamina, lamins, 57, 396, 477\\npore, 216 –220, 477\\nNumbering\\namino acids in polypeptides, 192bases in DNA, 106carbons in sugars, 35\\nNurse, Paul, 411\\nObjective lens, 3, 478\\nObligate anaerobes, 288, 478\\nOctane, 21\\nOdor of sanctity, 299-OH (hydroxyl group), 27 –28', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='42ca2721-1156-47a7-bc05-085f200a02ca', embedding=None, metadata={'page_label': '525', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\nINDEX 521\\nOil, 39, 478\\nOkasaki, Reiji, 90\\nfragments, 90, 478\\nOleate, oleic acid, 39, 42 –43, 478\\nOlfactory neurone (scent-sensitive nerve cell),\\n350, 478\\nOligomycin, 269\\nOligonucleotide, 134, 478\\nprimer, for PCR, 152, 431probe for library screening, 136\\nOligosaccharide, 30, 478\\nglycosylation in ER and Golgi, 225 –226, 230\\ntransferase, 225, 478\\nOlive oil, 39Oocyte, 406, 478\\ncAMP, gap junctions, 56cell cycle control, 411gene cloning, 140 –141\\ngeneration by meiosis, 406MPF, 411pre-implantation diagnosis for CF, 431\\nOpen complex, 89, 108, 478\\nDNA replication, 89promoter, 108, 478\\nOperator, in transcriptional control, 111 –117,\\n478\\nOperon, 109, 111, 125, 478\\ninducible, 111 –115\\nrepressible, 116 –118\\nOptical isomers, 196, 478\\nOrganelle, 9 –10, 51, 56 –61, 478\\ncytoplasmic streaming, 395deﬁnition, 51\\ndouble-membrane bounded, 56 –58\\nendosymbiotic theory, 12single-membrane bounded, 58 –61\\ntrafﬁcking, to speci ﬁc organelles, 213 –225,\\n230–231\\ntransport on actin micro ﬁlaments, 395\\ntransport on microtubules, 389 –390\\nOrganic molecules, 36, 478\\nOrganisms, gene number and complexity in, 68Orientation\\nantiparallel in DNA, 69cellulose, 54, 382chromatids, at mitosis, 403ciliary beat, Paramecium , 389\\nmicroscope, 5organs, Kartegener ’s syndrome, 389\\nplant microtubules, 382ribosome on mRNA, 171water in hydration shell, 21\\nOri, origin of replication, 88, 479\\nin cloning vectors, 130, 132, 140\\nOrnithine, 191, 293Osmium tetroxide, 8Osmolarity, 208, 479Osmosis, 53, 208, 479\\nOster, George, 272Outer doublet microtubules, 388, 479\\nOvulation, 56Oxaloacetate\\nde novo generation from pyruvate, 286\\nin gluconeogenesis, 295in glyoxylate shunt, 297in Krebs cycle, 284in nitrogen excretion/urea cycle, 292 –293\\nOxidation, 259, 479\\nβoxidation, 290 –291, 306, 479\\n2-oxoglutarate\\nin Krebs cycle, 284in nitrogen excretion, 292\\nOxoglutarate dehydrogenase, 284\\nactivation by calcium, 305\\nOxygen\\naddition, by cytochromes P450, 249anaerobic, 275, 286 –288\\ncooperative binding, 240diffusion across membranes, 53electron-grabbing, 19generation by photosynthesis, 274heme group, 206hydrogen bond, 25 –27, 195\\nloading, myo- and hemoglobin, 238 –240,\\n252–253\\nmonosaccharide ring, 27removal, by leghemoglobin, 302oxidation, 259why we breathe, 265\\nP site, 168, 479\\nP1 arti ﬁcial chromosomes (PACs), 139, 479\\np16INK4a, 184, 415\\np21CIP1, 415, 479\\np27KIP1, 415\\np38 protein kinase, 418, 479\\np53 transcription factor, 415, 417, 418, 420, 479\\np75 neurotrophin receptor, 417, 479\\nPAC, 140Pain receptors, 334, 336, 344, 372 –375, 479\\nPain relay cell, 332, 341, 372 –375, 479\\nPalade, George, 60Palmitic acid, 300Paracrine transmitters, 363 –364, 368 –370, 479\\nParallel βsheets, 195, 197, 480\\nParamecium , 386, 389\\nParenchyma cells, 16, 480\\nPatch clamp, 320 –321, 480\\nglutamate receptor cloning, 140 –141\\npBluescript cloning vector, 132PBS2 bacteriophage, 97PCR (polymerase chain reaction), 150 –152,\\n158, 431, 432, 480\\nPDGF (platelet derived growth factor), 353, 480', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='1b1c354e-103c-4baa-8a9e-da0a7218d55b', embedding=None, metadata={'page_label': '526', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\n522 INDEX\\nPDGF receptor, 200, 204, 223 –225, 353 –355\\nPectin, 53, 480\\nPellagra, 259\\nPentose, 27, 480\\nphosphate pathway, 289, 480\\nPeptide(s), 38, 184, 480\\nbond, 38, 173, 331, 480\\ngeneration on ribosome, 163 –176\\nhydrolysis, 44, 331mass spectroscopy, 178polypeptide, 38, 163, 184, 213\\nPeptidyl site (P site), 168, 480\\nPeptidyl transferase, 173, 184, 480\\nPericentriolar material, 383Peripheral (membrane) protein, 52 –53, 480\\nPeroxisome, 59 –60, 480\\nalanine glyoxylate aminotransferase (AGT)\\nand kidney stones, 223, 250, 292\\ntransport to, 215, 221, 223\\npH, 24, 480\\ncytoplasmic, 310denaturing effect, 206extracellular, 310/Delta1G\\n◦′standard state, 277\\ngradient, as energy currency, 261 –262\\nisoelectric point, 25, 178neutral, 24protein shape changes, 240receptor/ligand dissociation, 231See also Hydrogen ion\\nPhage (bacteriophage), 12, 74, 97, 133, 140, 480\\nPhalloidin, 376Phase-contrast microscopy, 5, 480\\nPH domain, 357, 480\\nPhenobarbital, 149, 167Phenylalanine, 185\\ndeletion in CF, 427essential amino acid, 302genetic code, 77, 78hydroxylase, 294toxicity in phenylketonuria, 294 –295,\\n406–407\\nPhenylketonuria, 294 –295, 406 –407, 481\\nPhenylpyruvate, 294Philadelphia chromosome, 403Phloem, 16, 481\\nPhosphatase, 190, 223, 481\\nPhosphate, 33, 36, 481\\nα, β andγphosphates, 37\\ngroup, 36radioactive, 139sugar-phosphate backbone, 68See also Phosphorylation\\nPhosphatidylcholine, 40, 41Phosphatidylinositides (PIP\\n2, PIP 3etc), 231,\\n346, 356 –357\\nPhosphoanhydrides, 191Phosphoaspartate, 191Phosphodiester link, 42, 67, 105, 481\\n5′-5′-phosphodiester link, 118\\nDNA, 67, 89, 90, 92, 95, 97phospholipid, 42RNA, 105, 109, 118\\nPhosphoenolpyruvate, 286, 295Phosphoesters, 191Phosphofructokinase, 285, 481\\nallosteric regulation by ATP, AMP, 251, 304\\n2-Phosphoglycerate, 2863-Phosphoglycerate, 286, 302\\n1,3-bisphosphoglycerate, 286\\nPhosphoglutamate, 191Phosphohistidine, 191Phosphoinositide 3-kinase (PI 3-kinase), 356,\\n481\\nPhosphoinositide phospholipase C (PLC), 346\\nβisoform, 346\\nγisoform, 356\\nPhospholipase C (PLC), 346, 481\\nβisoform, 346\\nγisoform, 356\\nPhospholipid, 40 –43, 52\\nartiﬁcial bilayer, 428\\nﬁsh odor syndrome, 82 –83\\nhydrolysis, 45 –46\\nsynthesis, 300trimethylaminuria, 82 –83\\nPhosphoryl group, 482Phosphorylase kinase (glycogen phosphorylase\\nkinase), 242, 305, 348, 353, 482\\nPhosphorylation, 190 –191, 482\\namino acid side group, 190 –191\\nCFTR, 427control of gene expression, 358MPF and cell cycle, 411 –412\\neffect on protein shape, 241enzyme regulation, 251 –252, 305, 356, 357\\nenergy cost, 258generator of motile force, 241mannose and protein targeting, 230 –231\\nsubstrate level, 286\\nPhosphoserine, 191Phosphothreonine, 191Phosphotyrosine, 191Photoreceptor, 353Photosynthesis, 11, 58, 271 –275, 302 –303, 482\\nPhragmoplast, 404, 482\\nPi (inorganic phosphate), 36Pigment cell (chromatophore), 389PIP\\n2(phosphatidylinositol bisphosphate), 346,\\n357, 482\\nPIP 3(phosphatidylinositol trisphosphate), 357\\nPipette, for electrical measurements, 320 –321\\npKa, 25, 482\\nof histidine and cysteine, 185', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='adffda64-2270-4591-8213-50e3c960cf40', embedding=None, metadata={'page_label': '527', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\nINDEX 523\\nPKA (protein kinase A), 305, 353, 358, 427, 482\\nPKB (protein kinase B), 356 –357, 417, 482\\nPKC (protein kinase C), 358, 482\\nPKG (protein kinase G), 370, 483\\nPlant(s), 11\\ncarbon ﬁxation in, 302 –303\\ncells, 11\\ngenetically modi ﬁed, 15\\nglyoxylate shunt in, 297 –298\\ntissues, 15 –16\\nPlaque, 143, 483\\nPlasmalemma (plasma membrane), 51, 483\\naction potential, 325 –337\\napoptosis, modi ﬁcation in, 415\\ncrossing by polypeptide, 200, 204diffusion across, 53, 121, 365endocytosis, 229, 230patch clamping, 320 –321\\npermeability to potassium ions, 310 –311\\nresting voltage, 311 –312\\nsodium gradient, as energy currency, 262source of its protein and lipid, 60, 230\\nPlasma membrane, 1, 51, 483\\naction potential, 325 –337\\napoptosis, modi ﬁcation in, 415\\ncrossing by polypeptide, 200, 204diffusion across, 53, 121, 365endocytosis, 229, 230patch clamping, 320 –321\\npermeability to potassium ions, 310 –311\\nresting voltage, 311 –312\\nsodium gradient, as energy currency, 262source of its protein and lipid, 60, 230\\nPlasmid, 74, 483\\ncloning vectors, 130, 132, 150recombinant, 134in gene therapy, 429, 432\\nPlasmodesma, plasmodesmata, 55, 483\\nPlastoquinone, 274 –275, 483\\nPlatelet, 344, 483\\ncalcium signalling, 344 –348\\nderived growth factor (PDGF), 353derived growth factor receptor, 200, 204,\\n223–225, 353 –355, 483\\nPLC (phospholipase C), 346, 483\\nβisoform, 346\\nγisoform, 356\\nPlug\\nnuclear pore, 216inactivation, 327, 330, 331CFTR regulatory domain, 427\\nPneumococcus transformation experiment, 66Pneumonia, 46Polar, 483\\nbody, 406chemical groups, 199head group, 42, 346molecules, 20solvents, 21\\nPolarity\\nDNA chain, 68Golgi apparatus, 225microtubules, 382, 390\\nPoly-A tail, 100, 118, 483\\nin cDNA cloning, 130\\nPolyadenylation, 118, 483\\nPolycistronic mRNA, 109, 483\\nPolylinker, 150Polymerase, 483\\nchain reaction (PCR), 150 –152, 158, 431,\\n432, 483\\nDNA polymerase I, 92, 95, 97DNA polymerase III, 90, 92DNA polymerase, thermostable, 152poly-A polymerase, 118RNA polymerase, 106 –117, 118\\nRNA polymerase I, 118RNA polymerase II, 118, 120, 122, 252RNA polymerase III, 118\\nPolymer, 27, 483\\nof amino acids, 184 –191\\nPolynucleotide kinase (PNK), 139, 483\\nPolypeptide, 38, 163, 184, 213, 484\\nsynthesis, 163 –176\\nPolyploid, 412, 484\\nPolyribosome, 173 –174, 484\\nPolysaccharide, 30, 484\\nPolysome, 173 –174, 484\\nPolyunsaturated fatty acids, 42, 484\\nPore\\nnuclear, 216 –220\\nstoma, 16\\nPorin, 314, 315, 484\\nPositive feedback, 116, 303 –304, 328, 331, 484\\nPostsynaptic cell, 344, 372 –375, 484\\nPotassium\\nchannel, 310, 437, 484\\nconcentration, 310gradient, 309 –312\\nresting voltage, 311 –312, 326, 328\\nNa\\n+/K+ATPase, 270, 271\\nPreferred state, of proteins, 238Pregnancy test kit, 183Preinitiation complex, 120, 122, 484\\nPrenatal implantation CF diagnosis, 431 –432\\nPresynaptic terminal, 341 –344, 372 –375, 484\\nPribnow box, 108Primary\\nantibody, 7, 166, 484\\ncell wall, 54, 484\\nhyperoxaluria, 223immuno ﬂuorescence, 7, 484\\nlysosome, 231structure, 192, 204, 206, 484', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='d5d539c1-e16f-4938-90f7-9c66b553b024', embedding=None, metadata={'page_label': '528', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\n524 INDEX\\nPrimase, 90, 484\\nPrimer, 152, 484\\nPrion, 207, 484\\nProcessed pseudogene, 100\\nProcessing, of mRNA, 118Product, of reaction, 242Proﬁlin, 391, 485\\nProgenitor, 73, 129, 401, 485\\nProgrammed cell death, 415 –418, 485\\nProjector lens, 3, 485\\nProkaryote, prokaryotic, 9, 10, 16, 485\\ndifferences from eukaryotes, 11chromosomes, 73ribosomes, 165, 168\\nProline, 185\\ngenetic code, 77, 78hydroxylation in collagen, 190steric (shape) effects, 185, 195\\nPrometaphase, 403, 485\\nPromoter, 106, 108, 111, 117, 120, 485\\nbcl-2 associated death promoter, 417\\nPronucleus, 406, 485\\nProof-reading, 91, 92Prophase, 403, 485\\nI, 406, 408, 485\\nII, 406, 485\\nProsthetic group, 205 –206, 249 –251, 265, 485\\nProteases, 331Protein, 183, 485\\nbackbone, 195binding ligand, 183, 190, 199, 200, 202, 237catalysts, 241 –249\\nconformational changes, 240 –241\\ndenaturation, 206, 208, 221domains, 99, 200 –202\\nenergy source, 292engineering, 149 –150, 485\\nfolding, 206 –208, 221\\nfunction, 183infective agent, 207integral membrane, 149 –150\\nkinase, 190, 241, 485\\nkinase A, 305, 353, 358, 427, 485\\nkinase B, 355, 356 –357, 417, 485\\nkinase C, 358, 486\\nkinase G, 370, 486\\nnaming, 184periperal membrane, 52 –53\\nphosphatase, 190, 222 –223, 318, 358, 486\\nphosphorylation, 190 –191, 241, 251 –252,\\n305, 356, 357, 411 –412, 427, 486\\nprimary structure, 192prion, 207quaternary structure, 204 –205\\nrecognition by binding, 202secondary structure, 195 –199\\nsecretion, 229 –230, 231separation, 166 –167, 178\\nsize and molecular weight, 166, 184sorting, 229structure, 183 –207, 238\\nsynthesis, 58, 163 –176, 223 –225\\ntargeting, 213 –232, 486\\ntertiary structure, 200 –202\\nthree-dimensional structures of, 192 –199\\ntranslocation, 213 –231\\ntrafﬁcking, 213 –231\\ntranslocator, 223, 486\\nwestern blot, 166 –167\\nProteoglycans, 53, 486\\nProteome, 177, 486\\nProteomics, 177 –178, 486\\nProtist, 9, 10, 12, 58, 80, 386Proto ﬁlament, 382, 486\\nProton, 22, 25Protonation, 25, 185, 486\\nProximal axon terminal, 333, 343PrP, 207, 208, 408Pseudogene, 100, 101, 486\\nPseudopodium, pseudopodia, 395, 486\\nP site, 168, 486\\nPstI, 133\\nPuffer ﬁsh genome, 99\\nPurines, 35, 65, 71, 486\\nbonding with pyrimidine, 69depurination, 94\\nPuromycin, 176Pyridoxamine, pyridoxal phosphate, 249Pyrimidines, 35, 65, 71, 486\\nbonding with purine, 69\\nPyrophosphate, 44 –46, 486\\nPyruvate, 285\\nconversion to acetaldehyde and ethanol, 288conversion to alanine, 286conversion to lactate, 286conversion to oxaloacetate, 286dehydrogenase, 286, 305generation from glucose, 284in gluconeogenesis, 295uses in cell, 286, 287\\nQuaternary structure, 204 –205, 487\\nQuiescent cells, 409, 487\\nQuorum sensing, 116\\nR bacteria, 66\\nR115777, 355Rab family GTPases, 231, 233, 487\\nRadioactive, radiation\\nDNA damage, 94probes in genetic engineering, 139 –149\\nRaf, 487\\nRan GTPase, 216 –219, 220\\nRandom priming method, 143', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='a39cac92-b2e1-4ca6-abf5-40ecfc4f4524', embedding=None, metadata={'page_label': '529', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\nINDEX 525\\nRan-GAP, 218, 219, 220\\nRapid reaction techniques, 252 –253\\nRas GTPase, 355, 371, 487\\nRB (retinoblastoma), 413, 414, 487\\nReading frame, 79 –80, 169, 487\\nReceptor, 487\\nα-adrenergic, 365, 368\\nADP, 344, 348β-adrenergic, 305, 353, 365, 368\\nclassi ﬁcation, 363 –365\\ncloning, 140 –141\\ndeath domain, 416, 417endocytosis, 229, 417, 487\\nﬁbroblast growth factor, 371\\nﬂuorescent chimera, 150\\nglucocorticoid, 121 –124, 150, 200, 202, 205,\\n365\\nglutamate, 372 –375\\ngrowth factor, 353 –356, 358\\ninsulin, 356 –357\\ninterleukin-2, 432intracellular, 365ionotropic cell surface, 364 –365,\\nmannose-6-phosphate, 231metabotropic cell surface, 365muscarinic acetylcholine, 370, 376neurotrophin, 417nicotinic acetylcholine, 364, 368, 376, 439nitric oxide, 365p75, 417pain, 332, 334, 336, 341 –344, 372 –375\\nphotoreceptor, 353platelet derived growth factor, 200, 204,\\n223\\npregnane X, 124ryanodine, 348 –349\\nscent, 350, 352signal recognition particle, 223sorting signal, 215, 219, 221steroid, 121 –124, 365\\nTrk, 417tyrosine kinase, 353 –358, 417, 487\\nvesicle budding, 229\\nRecessive gene, 406 –407, 425, 487\\nRecognition\\nhelix, 200of AUG codon, 176of ligand, 183, 202of ribosome-binding site, 171motif, 123signal recognition particle, 223site, of restriction enzyme, 133, 146, 150\\nRecombinant, recombination, 488\\nbacteriphage, 140DNA, 129 –158, 487\\ngenetic, 408homologous, 157plasmid, 134, 487\\nprotein, 149, 487\\nRecovery stroke, 386, 488\\nRed blood cell, 53, 81, 119, 204, 287Reduced nicotinamide adenine dinucleotide\\n(NADH), 259, 263 –277, 281 –295, 348\\ndehydrogenase, 267in the electron transport chain, 265 –267\\noxidation, 259\\nReduced nicotinamide adenine dinucleotide\\nphosphate (NADPH), 271 –275, 289, 300,\\n302\\nReduction, reducing agent, 259, 488\\nRefractive index, 5, 488\\nRegulated secretion, 229 –230\\nRegulation\\nof blood ﬂow, 368 –370\\nof enzyme activity, 251 –252\\nof gene expression, 109 –124\\nRelease factors, 174Repair of DNA, 94 –98, 415, 417\\nRepeats, tandem, 100, 146Repetitious DNA, 100, 488\\nReplication of DNA, 87 –93, 488\\nfork, 88, 488\\nG1/S control point, 412origin of replication, 88\\nRepressible operon, 116 –118, 488\\nRepressor (protein), 111 –118, 238\\nRER (rough endoplasmic reticulum), 57, 60,\\n223–225, 488\\nchanges at mitosis, 410glycosylation and, 225 –226\\nprotein synthesis on, 223 –225\\nResidual bodies, 231Residue, 30, 188, 488\\ncharged, 190\\nResistance to antibiotics, 74, 130, 132, 134,\\n157\\nResolving power, 2, 488\\nResting voltage, 311 –312, 314, 326, 488\\nRestriction endonucleases, 133, 134, 158, 488\\nRetention signal, ER, 215Retinoblastoma, 413, 414, 488\\nRetrograde transport, 390, 488\\nRetrovirus, 71, 488\\nReverse\\ngenetics, 152, 489\\ntranscriptase, 100, 130, 489\\ntranscription, 71, 100, 130, 489\\nRheumatoid arthritis, 124Rho, 109\\n-dependent termination, 109-independent termination, 109\\nRibonuclease, 489\\nH, 92, 130, 489\\nintestinal, 199, 247', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='4598ecfd-2ff4-4960-b4d1-3cb472fa1664', embedding=None, metadata={'page_label': '530', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\n526 INDEX\\nRibonucleic acid (RNA), 105, 118, 489\\nantisense, 132, 147\\ncatalytic molecule, 242instability, 118messenger (mRNA), 105 –124\\nnorthern blot, 148polymerase, 106 –109, 118, 120, 489\\nprimer, in DNA synthesis, 90, 92, 489\\nribosomal (rRNA), 57, 99, 105, 163, 165ribozymes, 242splicing, 68, 98, 118synthesis, 73, 106 –123\\ntransfer (tRNA), 99, 163 –165, 169 –176\\ntypes, 105, 163viral, 71, 74\\nRibose, 27, 29, 31, 65, 105, 489\\nRibosome, ribosomal, 57, 60, 165 –176,\\n223–225\\nbinding site, on mRNA, 168, 176recycling, 175\\nRibozyme, 242, 489\\nRibulose, 27, 29\\nbisphosphate, 302\\nRNA (ribonucleic acid), 105, 118, 489\\nantisense, 132, 147catalytic molecule, 242instability, 118messenger (mRNA), 105 –124\\nnorthern blot, 148polymerase, 106 –109, 118, 120, 489\\nprimer, in DNA synthesis, 90, 92ribosomal (rRNA), 57, 99, 105, 163, 165ribozymes, 242splicing, 68, 98, 118, 489\\nsynthesis, 73, 106 –123\\ntransfer (tRNA), 99, 163 –165, 169 –176\\ntypes, 105, 163viral, 71, 74\\nRock climbers, 350Root, 15 –16\\nRotary motion, 264, 272 –273, 388\\nRotenone, 271Rough endoplasmic reticulum (RER), 57, 60,\\n223–225, 489\\nchanges at mitosis, 410glycosylation and, 225 –226\\nprotein synthesis on, 223 –225\\nRoughton, Francis, 252rRNA, 57, 99, 105, 163, 165, 489\\nRuska, Ernst, 9Ryanodine receptor, 348, 489\\nσfactor, 106, 108, 490\\nS bacteria, 66S phase, 408 –410, 412 –415, 490\\nS value, 165 –168, 490\\nSaccharomyces , 68, 401Saints, odor of sanctity, 299Saint John ’s wort, 124\\nSakmann, Bert, 320Salad dressing, 39Salmonella , 264\\nSaltatory conduction, 336, 490\\nSalt, 21, 39\\nbridge, 190, 199, 490\\nsweat, in CF, 424 –425\\nSanctity, odor of, 299Sanger, Frederick, 143\\nDNA sequencing, 143 –146\\nSarcomere, 393, 490\\nSatellite DNA, 100, 403, 490\\nSaturation, saturated\\nenzyme, 245fatty acids, 42, 490\\nkinetics, 245, 490\\nScanning electron microscope, 9, 490\\nScent-sensitive nerve cells, 336, 350, 351, 352Schleiden, Matthias, 1Schwann, Theodor, 1\\nSchwann cells, 334, 490\\nSCID (severe combined immunode ﬁciency\\ndisease), 432\\nScrapie, 207, 208Screening a library, 136Scurvy, 190SDS–PAGE, 166 –167, 490\\nSea urchin egg, 325 –330\\nSecondary\\nantibody, 7, 490\\ncell wall, 54, 490\\nlysosome, 231immuno ﬂuorescence, 7, 490\\nstructure, 195, 490\\nSecretory vesicle, 228, 230, 491\\nSedimentation coef ﬁcient (S value), 165 –168,\\n491\\nSelection, DNA clones, 134 –139\\nSelf-correcting enzyme, 92Semi-conservative replication, 92 –93, 491\\nSequencing of DNA, 143 –146\\nSerine, 185\\ngenetic code, 77, 78phosphorylation of side chain, 120, 190, 241\\n70S initiation complex, formation of, 170, 171Severe combined immunode ﬁciency disease\\n(SCID), 432\\nSH2 domain, 355, 358, 491\\nβsheet, 195, 491\\nShigella , 393\\nShine –Dalgarno sequence, 169, 491\\nShoot, of plant, 15 –16\\nShort interspersed nuclear elements (SINEs),\\n100–101\\nSickle cell anemia, 81, 82', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='27d18e41-e239-4438-9ae7-d52744fa4a49', embedding=None, metadata={'page_label': '531', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\nINDEX 527\\nSide chains, 38, 78, 491\\nhydrophilic and hydrophobic, 185, 186, 187,\\n189\\nhydroxyl groups in, 188\\nposttranslational modi ﬁcations in, 190\\nSigma ( σ) factor, 106, 108, 491\\nSignal\\ndeath, 417ER retention, 215, 216mitochondrial import, 215nuclear localization, 215 –219, 222\\npeptidase, 223, 491\\nrecognition particle, 223, 491\\nsequence, 204, 215, 216, 223 –225, 491\\nsorting, 215, 230 –231\\nstart, 79stop, 78 –80\\nSignalling\\nbiochemical, 353 –358\\nelectrical, 325 –337\\nintracellular, 341 –359\\npathways, 357 –359\\nwebs, 357 –359\\nSildena ﬁl, 370\\nSilk, 183, 195Silver stain, 166SINEs (short interspersed nuclear elements),\\n100–101\\nSingle\\ncelled organisms, 1copy genes, 99-membrane envelopes, 58 –61\\n-strand binding proteins, 90, 492\\nSite-directed mutagenesis, 149, 492\\nSkeletal muscle, 14, 492\\nactivation, 348, 367 –368\\ncalcium signals, 348control of energy production, 305, 348, 350,\\n353\\nenlargement, 371mechanism of contraction, 393 –394\\nresting voltage, 311ryanodine receptor, 348 –349\\nSkin cancer, 418Skou, Jens, 273Sleeping sickness, 288Sma H1, 133\\nSmall nuclear RNA (snRNA), 118, 492\\nSmell detection, 350 –353\\nSmoking, 13, 94, 376, 386Smooth\\nendoplasmic reticulum, 60, 216, 300, 492\\nmuscle, 14, 368 –371, 492\\nSNARE proteins, 231, 232, 492\\nsnRNA (small nuclear RNA), 118, 492\\nSodium, 21\\naction potential, 332 –337, 492binding site, 190/calcium exchanger, 317, 320, 492\\nchannel, voltage gated, 200, 330 –331\\nchloride, 21, 424concentration, 310crossing lipid bilayer, 53dodecyl sulfate (SDS), 166, 167glutamate, 343gradient, energy currency, 262, 492\\nhydroxide, 136lauryl sulfate, 167/potassium ATPase, 270, 271, 312, 492\\nSolubility\\npeptides, 185 –186\\npolar and non-polar solute, 21, 39\\nSolute-gated channel, 344, 350, 365Solvent, 21, 39Somatic cells, 404, 406, 428, 429, 492\\nSorting signal, 215, 230 –231, 492\\nSOS, 355, 493\\nSouthern, Ed, 146\\nSouthern blotting, 146 –147, 493\\nSpatial summation, 373, 493\\nSpeci ﬁc activity, 493\\nSpeci ﬁcity constant, 247, 493\\nSperm, spermatozoon, 493\\nactin in acrosome, 392calcium channel, 329formation, 404 –406\\nfusion, effect of membrane voltage, 325 –327\\nKartegener ’s syndrome, 389\\nmotility, 386, 388\\nSpermatid, 406, 493\\nS phase, 408 –410, 412 –415, 493\\nSp1, 120Spindle, 402, 405, 410Spliceosome, splicing, 68, 118, 390, 493\\nSpongiform encephalopathies, 207Squamous cells, 13, 493\\nSquid\\naxon, 331light emission, 116\\nSSB (single-strand binding protein), 90Stahl, Franklin, 92Stain, for microscopy, 8Standard states, 276 –277, 493\\nStart codon, 79Start signal, 79, 493\\nStarvation metabolism, 299Steady state, 257, 276Stearic acid, 42 –43, 493\\nStereo isomers, 27, 493\\nSterilization, by ﬁltration, 10\\nSteroid hormone, 493\\nreceptor, 121 –124, 365, 493\\ndiffusion across membrane, 53, 365synthesis, 60, 249', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='014c8aeb-75a4-46de-a174-be313d16fb8c', embedding=None, metadata={'page_label': '532', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\n528 INDEX\\nSticky ends, 133, 493\\nSt. John ’s wort, 124\\nStoma, stomata, 16\\nStomach, cell, 61Stop codons, 79 –80, 174, 494\\nStop signals, 79 –80, 109, 173 –174, 494\\nStopped ﬂow, 252 –253\\nStrati ﬁed epithelium, 13, 494\\nStreptomycin, 176Stress\\n-activated apoptosis, 417 –418\\n-activated protein kinase, 418, 494\\nadrenaline in, 368ﬁber, 390 –391, 494\\nheat shock proteins, 221rotational, in DNA, 89\\nStriated muscle, 14, 393, 494\\nactivation, 348 –349, 367 –368\\nanaerobic, 275, 286 –287, 350\\nblood supply, 368 –372\\ncalcium effects, 348 –350, 394\\ncalf, 365 –372\\ncAMP effects, 353desmin, 396digitalis, 320feedforward control, 304 –306\\nﬁber, 15\\ngastrocnemius, 365 –372\\nglucose carrier translocation, 317, 357glycogen storage and breakdown, 30, 288,\\n304–306\\nintermediate ﬁlament, 396\\nketone bodies as fuel, 290mechanism of contraction, 393 –394\\nmitochondria, 58neurotrophin synthesis, 417paralysis (BoTox), 232resting voltage, 311ryanodine receptors, 348 –349\\nStroma, 274, 494\\nSubstrate, 241 –242, 494\\nSubtilisin, 149 –150\\nSuccinate, 267, 284, 297\\nsuccinate-Q reductase, 267succinate dehydrogenase, 267, 284\\nSucrose, 243, 494\\nSugar, 27 –33\\n-phosphate backbone, 68\\nSuicide, cell, 417Sulfur, 25 –26, 185, 191, 195, 265 –256, 275\\nSummation at synapse, 372 –375, 494\\nSunburn, 418Supercoiling, 73, 494\\nS value, 165 –168, 494\\nSvedberg unit (S value), 165 –168, 494\\nSweat gland, 424 –425\\nSynapse, 344, 372 –375, 376, 494Synaptic transmitter, 344, 371Synthesis, 295 –303\\namino acids, 300 –302\\nDNA, 87 –92\\nfatty acids, 300glucose, 295 –297\\nglycogen, 298 –299\\nlipid, 300protein,RNA, 105 –109, 118\\nT lymphocyte, 222T3 promoter sequence, 132T7 promoter sequence, 132Tail\\nhydrocarbon, 39, 42, 228poly A, 118, 130\\nTandemly repeated genes, 100, 494\\nTangling, of chromosomes, 74Targeting\\nsequence, 60, 215, 221, 494\\nto lysosome, 230 –231\\nto mitochondria, 221to nucleus, 215 –220, 222\\nto peroxisome, 221, 223\\nTATA box, 120, 494\\nTATA-binding protein, 120, 125Taxol, 385, 404, 494\\nTBP (TATA-binding protein), 120, 125Telomere, 100, 494\\nTelophase, 404, 495\\nTemporal summation, 373, 374, 495\\nTendon, Achilles, 365Terminally differentiated cells, 409 –410, 495\\nTerminal, axon, 332 –334, 341 –344, 348,\\n367–368, 372 –375\\nTermination\\ndideoxy chain, 143 –146\\nof protein synthesis, 174of RNA synthesis, 109\\nTerminator, 109, 495\\nTertiary structure, 200 –202, 495\\nTetracycline, 176TFII accessory proteins, 120, 121TGN ( trans Golgi network), 226, 228,\\n229–231\\nThermodynamics, 257, 276 –277, 312 –313, 495\\nThermogenin, 270, 495\\nThermostable DNA polymerase, 152Thick ﬁlament, 393, 495\\nThin ﬁlament, 394, 495\\nThree dimensional structure of proteins,\\n192–208\\nspeci ﬁcation by primary structure, 206\\nThreonine, 185\\nessential amino acid, 302genetic code, 78', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='ebcac854-cb50-4e78-ac77-ebb0c894a668', embedding=None, metadata={'page_label': '533', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\nINDEX 529\\nglycosylation, 190\\nphosphorylation of side group, 190\\nThreshold, of action potential, 328, 372 –375,\\n495\\nThylakoid, 58, 274, 495\\nThymidine triphosphate (TTP)\\ndTTP, building block of DNA, 65 –67\\nenergy currency, 259\\nThymine (T), 34, 35, 495\\nbonding with adenine, 70dimer, 94, 97\\nThyone , 392\\nTight junctions, 54, 55, 61, 495\\nTire, 11, 53Tissue, 12, 495\\nepithelium, 12connective, 13muscle, 14 –15\\nnervous, 13plant, 15 –16\\nT lymphocyte, 222Tobacco, 94, 250, 376Topoisomerase I, 89, 495\\nTopoisomerase II, 74 –75, 89, 495\\nTrafﬁcking, 213 –232, 357\\ntrans cisternae, 226\\ntrans Golgi network, 226, 228, 229 –231, 496\\nTransamination, 292, 294Transcription, 105 –124, 496\\nbubble, 108 –109, 496\\ncontrol, 109 –118, 119 –124\\nfactor, 120 –124, 190, 216, 222 –223, 355,\\n358, 413, 415, 417, 432, 496\\npreinitiation complex, 120tissue-speci ﬁc, 122\\nTransepithelial voltage, 425Transfection, 150, 496\\nTransfer\\nRNA (tRNA), 99, 163 –165, 169 –176, 496\\nvesicle, 226\\nTransformation, 66, 134, 415, 496\\nTransgenic\\nanimal, 157, 496\\nplant, 158\\nTransition state, 249Translation, 76, 163 –176, 496\\nTranslocation, 173, 496\\nprotein, 213 –232, 317, 357\\nribosome on mRNA, 173\\nTransmembrane\\nhelix, 200, 204translocation, 213, 221, 223 –225, 496\\nvoltage, 261 –262, 311 –314, 320 –321,\\n325–337, 372 –375, 496\\nTransmission electron microscope, 8, 496\\nTransmitter, 123, 341, 344, 357, 363 –375, 496\\nTranspiration, 16, 496Transport\\nacross membranes, 221 –225, 264, 310 –320\\nintracellular, 389 –390\\nto and from the nucleus, 215 –220\\nto mitochondria, 221to peroxisomes, 221vesicular, 214, 215, 226 –232\\nTrastuzumab, 355Triacylglycerol, 39, 270, 290, 300, 497\\nTricarboxylic acid (TCA) cycle, 283, 497\\nTriglyceride (triacylglycerol), 39, 270, 290, 300,\\n497\\nTrimeric G proteins, 247, 344 –346, 350, 359,\\n497\\nTrimethylamine, 25, 82 –83, 208, 497\\nN-oxide, 82 –83, 208, 497\\nTrimethylaminuria, 82 –83, 101\\nTrioleoylglycerol, 41Trk, 417, 497\\ntRNA, 99, 163 –165, 169 –176, 498\\ntroponin, 394, 498\\ntrpoperon, 116 –118, 498\\nTrypanosomes, 288Trypsin, trypsinogen, 248Tryptophan, 185\\nessential amino acid, 259, 302genetic code, 78, 79, 80operon, 116 –118, 498\\nsynthesis, 302\\ntSNARE proteins, 231, 232Tsui, Lap-Chee, 426TTP (thymidine triphosphate)\\ndTTP, building block of DNA, 65 –67\\nenergy currency, 259\\nTubulin, 382, 498\\nTurgid, turgor, 11, 53Turnover number ( k\\ncat), 242, 244, 245, 247, 498\\nTwo-dimensional electrophoresis, 177Tyrosine, 185\\nﬂuorescence, 253\\ngenetic code, 78kinase, 353 –359, 365, 371, 417, 498\\nphenylketonuria, 294, 406 –407\\nphosphorylation of side chain, 190, 241, 412synthesis, 294\\nUbiquinone, 265UDP-glucose, 298Ultramicrotome, 8, 498\\nUltrastructure, 2, 9, 498\\nUltraviolet (UV) light, 6\\nDNA damage, 94, 98, 415, 418ﬂuorescence, 6\\nthymine dimers, 94\\nUnambiguity of genetic code, 79Uncoupler, 269, 270, 498\\nUnsaturated fatty acids, 42, 498', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='f7b9bf4c-a3cd-4ab3-8ae0-4dec9cfd4852', embedding=None, metadata={'page_label': '534', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\n530 INDEX\\nUntangling chromosomes, 74\\nUntranslated sequence, 169, 498\\nUnwinding of DNA, 87 –90, 108\\nUpstream, 106, 498\\nUracil (U), 34, 35, 46, 94, 498\\nDNA glycosidase, 95, 498\\nUranyl acetate, 8Urea, 498\\ncycle, 292 –293, 498\\ndenaturing agent, 206, 208\\nUridine\\ndiphosphate-glucose, 298dUTP, in DNA, 97triphosphate, 105, 259\\nUV (ultraviolet) light, 6\\nDNA damage, 94, 98, 415, 418ﬂuorescence, 6\\nthymine dimers, 94\\nVm(maximal velocity), 245, 246\\nv\\no(initial velocity), 243 –246\\nVaccines, manufacture in bacteria, 149Vacuole, 11, 498\\nacrosomal, 392autophagic, 231\\nValine, 185\\nessential amino acid, 302genetic code, 77, 78synthesis, 302\\nValinomycin, 196Valium, 375, 499\\nvan der Waals forces, 199, 237, 499\\nvan Leeuwenhoek, Anton, 1Variable number tandem repeats (VNTRs),\\n146\\nVascular tissue, plant, 15 –16\\nVasoconstrictor, 368, 499\\nVasodilator, 370, 499\\nVector, 499\\ncloning, 130 –132\\nexpression, 139, 149genomic DNA, 139 –140\\nVermeer, Johannes, 1Vesicle, 215, 233, 499\\nclathrin-coated, 229coatomer-coated, 228formation of, 228Golgi apparatus and, 226transfer, 226\\nVesicular traf ﬁcking, 214, 215, 499\\nbetween intracellular compartments,\\n226–232\\nViagra, 370Vibrio\\ncholera , 353\\nﬁscheri , 116\\nVikings, 376Villin, 391 –392, 499\\nVillus, villi, 13, 499\\nVimentin, 396, 499\\nVinegar, 39Viral vectors, for gene therapy, 429Virulent bacteria, 66Virus, 11 –12, 71, 74, 97, 176, 416, 429, 499\\nVitamin\\nA, 158B6, 249C, 190niacin (B3), 259\\nVm(maximal velocity), 245, 246, 499\\nVNTRs (variable number tandem repeats), 146,\\n499\\nvo(initial velocity), 243 –246\\nV oltage\\nclamp, 326, 428, 499\\nequilibrium, 311 –313\\n-gated calcium channel, 327 –328, 343,\\n348–349, 389, 499\\n-gated sodium channel, 200, 330 –331, 499\\nmeasurement, 320 –321\\nmitochondrial, 261resting, 311 –312\\nsperm fusion, 325 –327\\nsummation, 372 –375\\nsweat gland, 425thylakoid, 274transmembrane, 261 –262, 311 –314,\\n320–321, 325 –337, 372 –375\\ntransmission down axons, 332 –336\\ntransmission through gap junctions,\\n55–56\\nvSNARES, 231, 232\\nWalker, John, 273\\nWashing powder, 150Water, 19\\nabundance, 19, 183diffusion across membranes, 53, 208ﬁltration, 10\\nH\\n+donor and acceptor, 22 –25\\nhydrogen bonding, 25 –27\\nhydrolysis, 44 –46\\noxidation in chloroplast, 274solvent, 21, 39viscosity, 264\\nWatson, James, 68Watson –Crick model, 69\\nWee1, 412, 499\\nWestern blotting, 166 –167\\nWhite blood cell\\ncalcineurin and immunosuppressants,\\n222\\nFas activation, 416fmet-leu-phe activation, 5, 171', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n'), Document(id_='7ca24976-7091-4527-b123-9abcd2df3235', embedding=None, metadata={'page_label': '535', 'file_name': 'cellbiology.pdf', 'file_path': '/content/data/cellbiology.pdf', 'file_type': 'application/pdf', 'file_size': 11438144, 'creation_date': '2024-05-27', 'last_modified_date': '2024-05-27'}, excluded_embed_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], excluded_llm_metadata_keys=['file_name', 'file_type', 'file_size', 'creation_date', 'last_modified_date', 'last_accessed_date'], relationships={}, text='P1: IOI\\nWY001-IND WY001-Bolsover-v3.cls October 22, 2003 14:56\\nINDEX 531\\nresting voltage, 311\\nsevere combined immunode ﬁciency disease\\n(SCID), 432\\nWilting, 53Wobble hypothesis, 164 –165, 499\\nXenopus oocyte, 140 –141, 411\\nXeroderma pigmentosum, 98, 499\\nX-linked adrenoleukodystrophy, 60X-ray, 68\\nﬁlm, 139, 144, 149\\nX-SCID, 432Xylem, 16, 499Yeast, 1, 68, 275, 288, 401\\nartiﬁcial chromosome (YAC) vectors,\\n139–140\\nYew, 385\\nZ disc, 393 –394, 500\\nZebra ﬁsh genome, 99\\nZellweger ’s syndrome, 500\\nZernike, Frits, 9\\nZ-DNA, 71, 500\\nZinc, 157, 200, 202, 205\\nﬁnger, 200, 500\\nZoo blot, 146, 426', start_char_idx=None, end_char_idx=None, text_template='{metadata_str}\\n\\n{content}', metadata_template='{key}: {value}', metadata_seperator='\\n')]\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Index Creation and Query Engine Creation"
      ],
      "metadata": {
        "id": "_Epknrn_U0PN"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "index1 = VectorStoreIndex.from_documents(documents1)\n",
        "query_engine1 = index1.as_query_engine()"
      ],
      "metadata": {
        "id": "DVX9op2yUk_7"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "index2 = VectorStoreIndex.from_documents(documents2)\n",
        "query_engine2 = index2.as_query_engine()"
      ],
      "metadata": {
        "id": "hTg-L2OuZeZ-"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Running the Query"
      ],
      "metadata": {
        "id": "71s1qCUUU3Mi"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "query1 = \"What is the equation for the overall reaction catalysed by the electron transport chain?\"\n",
        "response = query_engine1.query(query1)\n",
        "display(Markdown(f\"<b>{response}</b>\"))"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 70
        },
        "id": "m4xQ4gnwUnQM",
        "outputId": "851e3fc5-81f3-4c8a-cba6-4708a7de9b0d"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Markdown object>"
            ],
            "text/markdown": "<b>The overall reaction catalyzed by the electron transport chain can be summarized as:\n\nNADH + Q + 5H+ → NAD+ + QH2 + 4H+ matrix intermem</b>"
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "query2 = \"What is the equation for the overall reaction catalysed by the electron transport chain?\"\n",
        "response = query_engine2.query(query2)\n",
        "display(Markdown(f\"<b>{response}</b>\"))"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 46
        },
        "id": "ZuENLZaOYZeB",
        "outputId": "40a42dc7-3631-4500-fe6c-b8505d0ecd58"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Markdown object>"
            ],
            "text/markdown": "<b>NADH + H+ + 1/2O2 → NAD+ + H2O</b>"
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [],
      "metadata": {
        "id": "RI7JDGcRcPr9"
      },
      "execution_count": null,
      "outputs": []
    }
  ]
}